{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "dc67c302-06f6-4b4d-aff1-74d2510862c7",
   "metadata": {},
   "source": [
    "# Utilizing EquiBind for In-Silico Selection of Anti-Cancer Drug Candidates \n",
    "### Andrew Smith\n",
    "--------------"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "2956eb2a-aa09-4f92-b077-13866742997d",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "# Required Imports\n",
    "import os\n",
    "import re\n",
    "import random\n",
    "import subprocess\n",
    "import threading\n",
    "from collections import defaultdict\n",
    "\n",
    "import concurrent.futures\n",
    "from concurrent.futures import ThreadPoolExecutor\n",
    "\n",
    "from rdkit import Chem\n",
    "from rdkit.Chem import AllChem, PandasTools, SDWriter, SDMolSupplier\n",
    "\n",
    "import matplotlib.pyplot as plt\n",
    "import pandas as pd\n",
    "import py3Dmol\n",
    "\n",
    "%config InlineBackend.figure_format='retina'\n",
    "\n",
    "# Define Config Vars\n",
    "NUM_THREADS = os.cpu_count()\n",
    "BATCH_SIZE = 128"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3ae63f5a-952c-44b0-aec5-a233ca03f478",
   "metadata": {},
   "source": [
    "## The Equibind Model\n",
    "EquiBind is a groundbreaking SE(3)-equivariant geometric deep learning model that predicts protein-ligand binding structures rapidly and accurately. One of the core components of EquiBind is the Independent E(3)-Equivariant Graph Matching Network (IEGMN), which plays a crucial role in learning complex molecular interactions while preserving equivariance under rigid transformations.\n",
    "\n",
    "In the context of drug discovery, protein-ligand binding refers to the interaction between a protein (receptor) and a small molecule (ligand). Predicting the binding structure is vital for understanding the potential effects of drugs on their target proteins.\n",
    "\n",
    "IEGMN is a graph neural network that jointly transforms both features (molecular properties) and 3D coordinates (spatial information) to perform intra (within a molecule) and inter (between molecules) neural graph message passing. Its architecture guarantees that any independent rotation and translation of the input structures are precisely reflected in the outputs, making it an effective tool for analyzing molecular structures. In practice, the outputs are obtained by stacking several IEGMN layers.\n",
    "\n",
    "To identify the rigid SE(3) transformation (a combination of rotation and translation) required to dock the ligand in the correct position and orientation, EquiBind employs an SE(3)-equivariant multi-head attention mechanism. This mechanism computes ligand and receptor keypoints, which are critical points on the molecular surface that help guide the alignment of the molecules.\n",
    "\n",
    "These keypoints are trained to match the ground truth binding pocket points (the actual positions of the molecules in their bound state) using an optimal transport loss that recovers their alignment. When the ligand and receptor are separated, the model identifies the corresponding binding sites and their exact matching using the two predicted keypoint sets. If predicted perfectly, the SE(3) transformation to superimpose these sets would precisely correspond to the binding SE(3) transformation required for ligand docking."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c104de3e-d2eb-45cd-855f-8e48e25573a2",
   "metadata": {
    "tags": []
   },
   "source": [
    "----------\n",
    "## The RAS Protein\n",
    "The Ras protein plays a critical role in regulating cell growth, and mutations in the Ras gene have been linked to cancer. Therefore, discovering molecules that can dock and form complexes with Ras is essential because it could lead to the development of new treatments for cancer. By targeting Ras and disrupting its activity, it may be possible to limit the uncontrolled growth of cancer cells. Additionally, understanding the structure and function of Ras and its interactions with other proteins in the signaling network could lead to a better understanding of the mechanisms that regulate cell growth and division, which could have implications for developing treatments for other diseases.\n",
    "\n",
    "The PDB file required for this notebook is available [here](https://www.rcsb.org/structure/5p21). Make sure to download it and add it to `data/ras/ras_protein.pdb`."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "41c5a495-f2f3-4f8c-9bab-1722b9e04d46",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16835744454165597\"  style=\"position: relative; width: 700px; height: 700px\">\n        <p id=\"3dmolwarning_16835744454165597\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n}\n\nvar viewer_16835744454165597 = null;\nvar warn = document.getElementById(\"3dmolwarning_16835744454165597\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16835744454165597 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16835744454165597\"),{backgroundColor:\"white\"});\nviewer_16835744454165597.zoomTo();\n\tviewer_16835744454165597.addModel(\"HEADER    ONCOGENE PROTEIN                        30-APR-90   5P21              \\nTITLE     REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION OF H-RAS   \\nTITLE    2 P21 AT 1.35 ANGSTROMS RESOLUTION: IMPLICATIONS FOR THE MECHANISM OF  \\nTITLE    3 GTP HYDROLYSIS                                                       \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: C-H-RAS P21 PROTEIN;                                       \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    ONCOGENE PROTEIN                                                      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    E.F.PAI,A.WITTINGHOFER,W.KABSCH                                       \\nREVDAT   6   29-NOV-17 5P21    1       HELIX                                    \\nREVDAT   5   24-FEB-09 5P21    1       VERSN                                    \\nREVDAT   4   01-APR-03 5P21    1       JRNL                                     \\nREVDAT   3   20-JUL-95 5P21    1       HET                                      \\nREVDAT   2   30-APR-94 5P21    1       FORMUL                                   \\nREVDAT   1   15-JAN-92 5P21    0                                                \\nJRNL        AUTH   E.F.PAI,U.KRENGEL,G.A.PETSKO,R.S.GOODY,W.KABSCH,             \\nJRNL        AUTH 2 A.WITTINGHOFER                                               \\nJRNL        TITL   REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION   \\nJRNL        TITL 2 OF H-RAS P21 AT 1.35 A RESOLUTION: IMPLICATIONS FOR THE      \\nJRNL        TITL 3 MECHANISM OF GTP HYDROLYSIS.                                 \\nJRNL        REF    EMBO J.                       V.   9  2351 1990              \\nJRNL        REFN                   ISSN 0261-4189                               \\nJRNL        PMID   2196171                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,S.C.ALMO,G.RAPP,K.WILSON,K.PETRATOS,A.LENTFER, \\nREMARK   1  AUTH 2 A.WITTINGHOFER,W.KABSCH,E.F.PAI,G.A.PETSKO,R.S.GOODY         \\nREMARK   1  TITL   TIME-RESOLVED X-RAY CRYSTALLOGRAPHIC STUDY OF THE            \\nREMARK   1  TITL 2 CONFORMATIONAL CHANGE IN HA-RAS P21 PROTEIN ON GTP           \\nREMARK   1  TITL 3 HYDROLYSIS                                                   \\nREMARK   1  REF    NATURE                        V. 345   309 1990              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   A.SCHERER,J.JOHN,R.LINKE,R.S.GOODY,A.WITTINGHOFER,E.F.PAI,   \\nREMARK   1  AUTH 2 K.C.HOLMES                                                   \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY ANALYSIS OF THE HUMAN  \\nREMARK   1  TITL 2 C-H-RAS-ONCOGENE PRODUCT P21 COMPLEXED WITH GTP ANALOGUES    \\nREMARK   1  REF    J.MOL.BIOL.                   V. 206   257 1989              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   E.F.PAI,W.KABSCH,U.KRENGEL,K.C.HOLMES,J.JOHN,A.WITTINGHOFER  \\nREMARK   1  TITL   STRUCTURE OF THE GUANINE-NUCLEOTIDE-BINDING DOMAIN OF THE    \\nREMARK   1  TITL 2 HA-RAS ONCOGENE PRODUCT P21 IN THE TRIPHOSPHATE CONFORMATION \\nREMARK   1  REF    NATURE                        V. 341   209 1989              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,G.RAPP,J.JOHN,A.WITTINGHOFER,E.F.PAI,R.S.GOODY \\nREMARK   1  TITL   BIOCHEMICAL AND CRYSTALLOGRAPHIC CHARACTERIZATION OF A       \\nREMARK   1  TITL 2 COMPLEX OF C-HA-RAS P21 AND CAGED GTP WITH FLASH PHOTOLYSIS  \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  86  7687 1989              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 6.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 26806                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.196                           \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1323                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 33                                      \\nREMARK   3   SOLVENT ATOMS            : 211                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.014                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 2.700                           \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  RESIDUES 61 - 65 (GLN - GLU - GLU - TYR - SER) ADOPT                \\nREMARK   3  SEVERAL CONFORMATIONS IN THE CRYSTAL.  THE COORDINATES              \\nREMARK   3  GIVEN APPROXIMATE ONE OF THESE.                                     \\nREMARK   4                                                                      \\nREMARK   4 5P21 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179747.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 38.93                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.01                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 32 2 1                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+2/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+1/3                                           \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z+1/3                                           \\nREMARK 290       6555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000      108.13333            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       54.06667            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       54.06667            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000      108.13333            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025  0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    ARG A  68   NE  -  CZ  -  NH1 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A  97   NE  -  CZ  -  NH1 ANGL. DEV. =  -3.4 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH1 ANGL. DEV. =   4.1 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500    ARG A 128   NE  -  CZ  -  NH2 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A 135   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500    TYR A 157   CB  -  CG  -  CD2 ANGL. DEV. =  -4.8 DEGREES          \\nREMARK 500    ARG A 164   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ASP A  33       75.86   -157.25                                   \\nREMARK 500    ILE A  36      -64.23   -100.39                                   \\nREMARK 500    GLU A  37      123.75   -171.30                                   \\nREMARK 500    SER A  65     -133.70     53.07                                   \\nREMARK 500    LYS A 117       33.80     71.60                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              MG A 168  MG                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 GNP A 167   O2G                                                    \\nREMARK 620 2 GNP A 167   O2B  95.0                                              \\nREMARK 620 3 HOH A 173   O    87.6  91.7                                        \\nREMARK 620 4 SER A  17   OG  172.9  91.0  88.6                                  \\nREMARK 620 5 THR A  35   OG1  91.3 173.4  90.5  82.8                            \\nREMARK 620 6 HOH A 172   O    97.3  88.0 175.0  86.5  89.3                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE MG A 168                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE GNP A 167                 \\nDBREF  5P21 A    1   166  UNP    P01112   RASH_HUMAN       1    166             \\nSEQRES   1 A  166  MET THR GLU TYR LYS LEU VAL VAL VAL GLY ALA GLY GLY          \\nSEQRES   2 A  166  VAL GLY LYS SER ALA LEU THR ILE GLN LEU ILE GLN ASN          \\nSEQRES   3 A  166  HIS PHE VAL ASP GLU TYR ASP PRO THR ILE GLU ASP SER          \\nSEQRES   4 A  166  TYR ARG LYS GLN VAL VAL ILE ASP GLY GLU THR CYS LEU          \\nSEQRES   5 A  166  LEU ASP ILE LEU ASP THR ALA GLY GLN GLU GLU TYR SER          \\nSEQRES   6 A  166  ALA MET ARG ASP GLN TYR MET ARG THR GLY GLU GLY PHE          \\nSEQRES   7 A  166  LEU CYS VAL PHE ALA ILE ASN ASN THR LYS SER PHE GLU          \\nSEQRES   8 A  166  ASP ILE HIS GLN TYR ARG GLU GLN ILE LYS ARG VAL LYS          \\nSEQRES   9 A  166  ASP SER ASP ASP VAL PRO MET VAL LEU VAL GLY ASN LYS          \\nSEQRES  10 A  166  CYS ASP LEU ALA ALA ARG THR VAL GLU SER ARG GLN ALA          \\nSEQRES  11 A  166  GLN ASP LEU ALA ARG SER TYR GLY ILE PRO TYR ILE GLU          \\nSEQRES  12 A  166  THR SER ALA LYS THR ARG GLN GLY VAL GLU ASP ALA PHE          \\nSEQRES  13 A  166  TYR THR LEU VAL ARG GLU ILE ARG GLN HIS                      \\nHET     MG  A 168       1                                                       \\nHET    GNP  A 167      32                                                       \\nHETNAM      MG MAGNESIUM ION                                                    \\nHETNAM     GNP PHOSPHOAMINOPHOSPHONIC ACID-GUANYLATE ESTER                      \\nFORMUL   2   MG    MG 2+                                                        \\nFORMUL   3  GNP    C10 H17 N6 O13 P3                                            \\nFORMUL   4  HOH   *211(H2 O)                                                    \\nHELIX    1  HA LYS A   16  GLN A   25  1                                  10    \\nHELIX    2  HB SER A   65  THR A   74  1RES 65-67 IN 3/10 CONF            10    \\nHELIX    3  HC THR A   87  VAL A  103  1BREAK AT ASP 92                   17    \\nHELIX    4  HD SER A  127  TYR A  137  1                                  11    \\nHELIX    5  HE VAL A  152  ARG A  164  1                                  13    \\nSHEET    1  S1 6 GLU A  37  ILE A  46  0                                        \\nSHEET    2  S1 6 GLU A  49  THR A  58 -1  O  LEU A  53   N  LYS A  42           \\nSHEET    3  S1 6 THR A   2  VAL A   9  1  N  LEU A   6   O  ASP A  54           \\nSHEET    4  S1 6 GLY A  77  ALA A  83  1  N  LEU A  79   O  VAL A   7           \\nSHEET    5  S1 6 MET A 111  ASN A 116  1  N  VAL A 114   O  CYS A  80           \\nSHEET    6  S1 6 TYR A 141  GLU A 143  1  N  ILE A 142   O  LEU A 113           \\nLINK        MG    MG A 168                 O2G GNP A 167     1555   1555  2.20  \\nLINK        MG    MG A 168                 O2B GNP A 167     1555   1555  2.22  \\nLINK        MG    MG A 168                 O   HOH A 173     1555   1555  2.23  \\nLINK        MG    MG A 168                 OG  SER A  17     1555   1555  2.26  \\nLINK        MG    MG A 168                 OG1 THR A  35     1555   1555  2.26  \\nLINK        MG    MG A 168                 O   HOH A 172     1555   1555  2.30  \\nSITE     1 AC1  5 SER A  17  THR A  35  GNP A 167  HOH A 172                    \\nSITE     2 AC1  5 HOH A 173                                                     \\nSITE     1 AC2 28 GLY A  12  GLY A  13  VAL A  14  GLY A  15                    \\nSITE     2 AC2 28 LYS A  16  SER A  17  ALA A  18  PHE A  28                    \\nSITE     3 AC2 28 VAL A  29  ASP A  30  TYR A  32  PRO A  34                    \\nSITE     4 AC2 28 THR A  35  GLY A  60  ASN A 116  LYS A 117                    \\nSITE     5 AC2 28 ASP A 119  SER A 145  ALA A 146  LYS A 147                    \\nSITE     6 AC2 28  MG A 168  HOH A 170  HOH A 172  HOH A 173                    \\nSITE     7 AC2 28 HOH A 175  HOH A 187  HOH A 281  HOH A 366                    \\nCRYST1   40.300   40.300  162.200  90.00  90.00 120.00 P 32 2 1      6          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.024814  0.014326  0.000000        0.00000                         \\nSCALE2      0.000000  0.028653  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.006165        0.00000                         \\nATOM      1  N   MET A   1      -7.186  32.862  -6.632  1.00 13.92           N  \\nATOM      2  CA  MET A   1      -5.921  32.127  -6.684  1.00 17.10           C  \\nATOM      3  C   MET A   1      -5.845  31.045  -5.621  1.00 16.74           C  \\nATOM      4  O   MET A   1      -6.600  31.083  -4.636  1.00 16.60           O  \\nATOM      5  CB  MET A   1      -4.673  33.058  -6.502  1.00 17.00           C  \\nATOM      6  CG  MET A   1      -4.633  33.804  -5.188  1.00 18.08           C  \\nATOM      7  SD  MET A   1      -3.115  34.766  -4.908  1.00 19.73           S  \\nATOM      8  CE  MET A   1      -3.019  35.851  -6.310  1.00 20.41           C  \\nATOM      9  N   THR A   2      -4.910  30.095  -5.820  1.00 14.70           N  \\nATOM     10  CA  THR A   2      -4.661  29.030  -4.859  1.00 16.92           C  \\nATOM     11  C   THR A   2      -4.230  29.655  -3.547  1.00 14.78           C  \\nATOM     12  O   THR A   2      -3.338  30.501  -3.537  1.00 13.93           O  \\nATOM     13  CB  THR A   2      -3.557  28.107  -5.401  1.00 18.03           C  \\nATOM     14  OG1 THR A   2      -4.059  27.565  -6.629  1.00 20.42           O  \\nATOM     15  CG2 THR A   2      -3.193  26.983  -4.443  1.00 18.51           C  \\nATOM     16  N   GLU A   3      -4.911  29.290  -2.472  1.00 14.80           N  \\nATOM     17  CA  GLU A   3      -4.504  29.730  -1.135  1.00 17.71           C  \\nATOM     18  C   GLU A   3      -4.012  28.553  -0.278  1.00 17.33           C  \\nATOM     19  O   GLU A   3      -4.520  27.427  -0.395  1.00 17.54           O  \\nATOM     20  CB  GLU A   3      -5.655  30.402  -0.432  1.00 20.67           C  \\nATOM     21  CG  GLU A   3      -6.213  31.631  -1.186  1.00 25.56           C  \\nATOM     22  CD  GLU A   3      -7.128  32.563  -0.369  1.00 28.46           C  \\nATOM     23  OE1 GLU A   3      -7.578  32.216   0.728  1.00 27.72           O  \\nATOM     24  OE2 GLU A   3      -7.375  33.680  -0.827  1.00 32.49           O  \\nATOM     25  N   TYR A   4      -2.962  28.744   0.542  1.00 14.50           N  \\nATOM     26  CA  TYR A   4      -2.425  27.654   1.358  1.00 12.29           C  \\nATOM     27  C   TYR A   4      -2.534  28.103   2.807  1.00 12.52           C  \\nATOM     28  O   TYR A   4      -2.016  29.143   3.200  1.00 12.11           O  \\nATOM     29  CB  TYR A   4      -0.975  27.385   1.026  1.00 11.46           C  \\nATOM     30  CG  TYR A   4      -0.726  26.846  -0.387  1.00 12.38           C  \\nATOM     31  CD1 TYR A   4      -0.297  27.698  -1.395  1.00 13.00           C  \\nATOM     32  CD2 TYR A   4      -0.933  25.501  -0.649  1.00 14.09           C  \\nATOM     33  CE1 TYR A   4      -0.070  27.208  -2.674  1.00 13.89           C  \\nATOM     34  CE2 TYR A   4      -0.711  24.993  -1.917  1.00 14.65           C  \\nATOM     35  CZ  TYR A   4      -0.279  25.859  -2.917  1.00 14.84           C  \\nATOM     36  OH  TYR A   4      -0.084  25.357  -4.191  1.00 15.07           O  \\nATOM     37  N   LYS A   5      -3.274  27.305   3.568  1.00 13.07           N  \\nATOM     38  CA  LYS A   5      -3.554  27.569   4.970  1.00 12.96           C  \\nATOM     39  C   LYS A   5      -2.556  26.818   5.852  1.00 12.74           C  \\nATOM     40  O   LYS A   5      -2.645  25.611   6.092  1.00 11.58           O  \\nATOM     41  CB  LYS A   5      -4.972  27.131   5.281  1.00 17.65           C  \\nATOM     42  CG  LYS A   5      -6.053  27.833   4.446  1.00 23.49           C  \\nATOM     43  CD  LYS A   5      -7.186  28.310   5.335  1.00 29.10           C  \\nATOM     44  CE  LYS A   5      -6.655  29.309   6.403  1.00 33.68           C  \\nATOM     45  NZ  LYS A   5      -6.597  28.739   7.755  1.00 36.52           N  \\nATOM     46  N   LEU A   6      -1.543  27.574   6.297  1.00 13.08           N  \\nATOM     47  CA  LEU A   6      -0.418  27.011   7.064  1.00 10.34           C  \\nATOM     48  C   LEU A   6      -0.578  27.382   8.535  1.00  9.68           C  \\nATOM     49  O   LEU A   6      -1.034  28.483   8.863  1.00 11.31           O  \\nATOM     50  CB  LEU A   6       0.910  27.571   6.536  1.00 11.34           C  \\nATOM     51  CG  LEU A   6       1.137  27.565   5.017  1.00 11.62           C  \\nATOM     52  CD1 LEU A   6       2.515  28.119   4.720  1.00 13.60           C  \\nATOM     53  CD2 LEU A   6       0.973  26.171   4.475  1.00 12.19           C  \\nATOM     54  N   VAL A   7      -0.196  26.486   9.432  1.00  9.93           N  \\nATOM     55  CA  VAL A   7      -0.333  26.787  10.852  1.00 10.06           C  \\nATOM     56  C   VAL A   7       1.033  26.513  11.495  1.00 10.37           C  \\nATOM     57  O   VAL A   7       1.597  25.441  11.252  1.00  9.51           O  \\nATOM     58  CB  VAL A   7      -1.474  25.884  11.446  1.00 10.97           C  \\nATOM     59  CG1 VAL A   7      -1.610  26.163  12.943  1.00 11.39           C  \\nATOM     60  CG2 VAL A   7      -2.809  26.161  10.777  1.00 10.14           C  \\nATOM     61  N   VAL A   8       1.576  27.455  12.295  1.00  8.35           N  \\nATOM     62  CA  VAL A   8       2.878  27.262  12.944  1.00  6.92           C  \\nATOM     63  C   VAL A   8       2.609  26.896  14.405  1.00  8.84           C  \\nATOM     64  O   VAL A   8       1.930  27.663  15.092  1.00  9.21           O  \\nATOM     65  CB  VAL A   8       3.684  28.565  12.799  1.00  8.08           C  \\nATOM     66  CG1 VAL A   8       5.111  28.324  13.261  1.00  9.52           C  \\nATOM     67  CG2 VAL A   8       3.729  29.011  11.358  1.00  8.43           C  \\nATOM     68  N   VAL A   9       3.024  25.701  14.881  1.00  8.73           N  \\nATOM     69  CA  VAL A   9       2.779  25.234  16.245  1.00  8.41           C  \\nATOM     70  C   VAL A   9       4.075  24.806  16.919  1.00  8.21           C  \\nATOM     71  O   VAL A   9       5.093  24.551  16.287  1.00 10.97           O  \\nATOM     72  CB  VAL A   9       1.824  24.028  16.286  1.00  9.74           C  \\nATOM     73  CG1 VAL A   9       0.545  24.491  15.680  1.00 11.76           C  \\nATOM     74  CG2 VAL A   9       2.349  22.810  15.535  1.00 10.31           C  \\nATOM     75  N   GLY A  10       4.048  24.707  18.238  1.00  7.34           N  \\nATOM     76  CA  GLY A  10       5.231  24.336  19.009  1.00  7.98           C  \\nATOM     77  C   GLY A  10       5.192  25.053  20.355  1.00  8.31           C  \\nATOM     78  O   GLY A  10       4.417  25.994  20.595  1.00  9.89           O  \\nATOM     79  N   ALA A  11       6.099  24.638  21.226  1.00  9.31           N  \\nATOM     80  CA  ALA A  11       6.121  25.135  22.597  1.00  9.50           C  \\nATOM     81  C   ALA A  11       6.440  26.625  22.711  1.00  9.66           C  \\nATOM     82  O   ALA A  11       6.998  27.250  21.806  1.00  9.46           O  \\nATOM     83  CB  ALA A  11       7.145  24.306  23.378  1.00  9.85           C  \\nATOM     84  N   GLY A  12       6.111  27.227  23.847  1.00 10.36           N  \\nATOM     85  CA  GLY A  12       6.406  28.632  24.054  1.00  9.30           C  \\nATOM     86  C   GLY A  12       7.892  28.950  23.964  1.00 10.23           C  \\nATOM     87  O   GLY A  12       8.785  28.249  24.466  1.00 10.57           O  \\nATOM     88  N   GLY A  13       8.159  30.064  23.266  1.00 11.49           N  \\nATOM     89  CA  GLY A  13       9.511  30.602  23.139  1.00 11.29           C  \\nATOM     90  C   GLY A  13      10.436  29.950  22.108  1.00 12.22           C  \\nATOM     91  O   GLY A  13      11.624  30.321  22.060  1.00 13.62           O  \\nATOM     92  N   VAL A  14       9.961  28.991  21.286  1.00  9.23           N  \\nATOM     93  CA  VAL A  14      10.848  28.327  20.317  1.00 10.38           C  \\nATOM     94  C   VAL A  14      11.169  29.168  19.076  1.00 10.47           C  \\nATOM     95  O   VAL A  14      12.108  28.841  18.353  1.00 12.07           O  \\nATOM     96  CB  VAL A  14      10.246  26.970  19.856  1.00  9.98           C  \\nATOM     97  CG1 VAL A  14      10.187  26.074  21.120  1.00  9.93           C  \\nATOM     98  CG2 VAL A  14       8.866  27.127  19.200  1.00  9.52           C  \\nATOM     99  N   GLY A  15      10.423  30.264  18.834  1.00  9.61           N  \\nATOM    100  CA  GLY A  15      10.666  31.160  17.706  1.00  9.70           C  \\nATOM    101  C   GLY A  15       9.615  31.061  16.626  1.00  8.82           C  \\nATOM    102  O   GLY A  15       9.915  31.403  15.491  1.00  8.99           O  \\nATOM    103  N   LYS A  16       8.395  30.591  16.916  1.00  8.18           N  \\nATOM    104  CA  LYS A  16       7.331  30.521  15.913  1.00  8.44           C  \\nATOM    105  C   LYS A  16       7.060  31.899  15.309  1.00  9.01           C  \\nATOM    106  O   LYS A  16       7.020  32.044  14.086  1.00  8.25           O  \\nATOM    107  CB  LYS A  16       6.014  30.000  16.529  1.00  9.27           C  \\nATOM    108  CG  LYS A  16       6.132  28.585  17.086  1.00  9.11           C  \\nATOM    109  CD  LYS A  16       4.820  28.077  17.726  1.00  9.31           C  \\nATOM    110  CE  LYS A  16       4.351  28.934  18.888  1.00 10.05           C  \\nATOM    111  NZ  LYS A  16       5.221  28.829  20.051  1.00  9.64           N  \\nATOM    112  N   SER A  17       6.913  32.942  16.135  1.00  7.74           N  \\nATOM    113  CA  SER A  17       6.661  34.295  15.649  1.00  8.78           C  \\nATOM    114  C   SER A  17       7.854  34.859  14.919  1.00  9.73           C  \\nATOM    115  O   SER A  17       7.653  35.446  13.868  1.00  9.70           O  \\nATOM    116  CB  SER A  17       6.360  35.264  16.769  1.00  8.22           C  \\nATOM    117  OG  SER A  17       5.083  34.853  17.204  1.00  8.79           O  \\nATOM    118  N   ALA A  18       9.072  34.687  15.444  1.00  8.95           N  \\nATOM    119  CA  ALA A  18      10.253  35.199  14.769  1.00  8.95           C  \\nATOM    120  C   ALA A  18      10.438  34.576  13.374  1.00  8.54           C  \\nATOM    121  O   ALA A  18      10.832  35.254  12.430  1.00  8.75           O  \\nATOM    122  CB  ALA A  18      11.521  34.931  15.640  1.00 10.11           C  \\nATOM    123  N   LEU A  19      10.136  33.284  13.176  1.00  8.31           N  \\nATOM    124  CA  LEU A  19      10.225  32.667  11.837  1.00  9.30           C  \\nATOM    125  C   LEU A  19       9.204  33.281  10.884  1.00 10.98           C  \\nATOM    126  O   LEU A  19       9.534  33.652   9.751  1.00 10.47           O  \\nATOM    127  CB  LEU A  19       9.938  31.165  11.885  1.00  9.41           C  \\nATOM    128  CG  LEU A  19      11.035  30.275  12.487  1.00 11.80           C  \\nATOM    129  CD1 LEU A  19      10.491  28.866  12.723  1.00 10.43           C  \\nATOM    130  CD2 LEU A  19      12.256  30.288  11.554  1.00 13.42           C  \\nATOM    131  N   THR A  20       7.950  33.419  11.335  1.00 10.72           N  \\nATOM    132  CA  THR A  20       6.888  34.015  10.529  1.00 11.87           C  \\nATOM    133  C   THR A  20       7.173  35.492  10.161  1.00 12.65           C  \\nATOM    134  O   THR A  20       7.012  35.917   9.009  1.00 12.60           O  \\nATOM    135  CB  THR A  20       5.547  33.888  11.312  1.00 13.43           C  \\nATOM    136  OG1 THR A  20       5.299  32.509  11.544  1.00 12.67           O  \\nATOM    137  CG2 THR A  20       4.346  34.410  10.518  1.00 14.64           C  \\nATOM    138  N   ILE A  21       7.615  36.326  11.098  1.00 10.48           N  \\nATOM    139  CA  ILE A  21       7.945  37.704  10.795  1.00 12.96           C  \\nATOM    140  C   ILE A  21       9.197  37.831   9.927  1.00 14.26           C  \\nATOM    141  O   ILE A  21       9.237  38.691   9.042  1.00 15.83           O  \\nATOM    142  CB  ILE A  21       8.090  38.451  12.121  1.00 12.57           C  \\nATOM    143  CG1 ILE A  21       6.691  38.551  12.737  1.00 13.24           C  \\nATOM    144  CG2 ILE A  21       8.631  39.852  11.910  1.00 13.40           C  \\nATOM    145  CD1 ILE A  21       6.636  39.185  14.129  1.00 16.46           C  \\nATOM    146  N   GLN A  22      10.206  36.979  10.092  1.00 13.58           N  \\nATOM    147  CA  GLN A  22      11.330  36.983   9.179  1.00 14.91           C  \\nATOM    148  C   GLN A  22      10.817  36.729   7.736  1.00 17.29           C  \\nATOM    149  O   GLN A  22      11.120  37.485   6.792  1.00 15.91           O  \\nATOM    150  CB  GLN A  22      12.270  35.930   9.736  1.00 15.90           C  \\nATOM    151  CG  GLN A  22      13.686  35.878   9.230  1.00 19.36           C  \\nATOM    152  CD  GLN A  22      14.372  37.217   9.091  1.00 18.08           C  \\nATOM    153  OE1 GLN A  22      14.160  37.984   8.143  1.00 19.23           O  \\nATOM    154  NE2 GLN A  22      15.224  37.492  10.050  1.00 17.40           N  \\nATOM    155  N   LEU A  23       9.944  35.718   7.523  1.00 15.95           N  \\nATOM    156  CA  LEU A  23       9.381  35.475   6.194  1.00 18.85           C  \\nATOM    157  C   LEU A  23       8.539  36.622   5.636  1.00 18.21           C  \\nATOM    158  O   LEU A  23       8.695  36.958   4.469  1.00 19.03           O  \\nATOM    159  CB  LEU A  23       8.498  34.235   6.196  1.00 18.10           C  \\nATOM    160  CG  LEU A  23       7.916  33.797   4.863  1.00 18.64           C  \\nATOM    161  CD1 LEU A  23       8.921  32.994   4.030  1.00 18.03           C  \\nATOM    162  CD2 LEU A  23       6.680  32.991   5.183  1.00 21.34           C  \\nATOM    163  N   ILE A  24       7.634  37.197   6.428  1.00 19.25           N  \\nATOM    164  CA  ILE A  24       6.682  38.198   5.953  1.00 20.23           C  \\nATOM    165  C   ILE A  24       7.316  39.590   5.922  1.00 21.89           C  \\nATOM    166  O   ILE A  24       7.125  40.318   4.943  1.00 22.93           O  \\nATOM    167  CB  ILE A  24       5.399  38.222   6.881  1.00 21.20           C  \\nATOM    168  CG1 ILE A  24       4.664  36.907   6.821  1.00 21.27           C  \\nATOM    169  CG2 ILE A  24       4.423  39.307   6.446  1.00 20.00           C  \\nATOM    170  CD1 ILE A  24       4.128  36.442   5.475  1.00 23.98           C  \\nATOM    171  N   GLN A  25       8.074  39.998   6.953  1.00 21.07           N  \\nATOM    172  CA  GLN A  25       8.520  41.380   7.066  1.00 22.09           C  \\nATOM    173  C   GLN A  25      10.003  41.559   6.962  1.00 20.75           C  \\nATOM    174  O   GLN A  25      10.495  42.677   7.054  1.00 20.26           O  \\nATOM    175  CB  GLN A  25       8.039  41.968   8.398  1.00 23.96           C  \\nATOM    176  CG  GLN A  25       6.548  42.276   8.381  1.00 28.69           C  \\nATOM    177  CD  GLN A  25       5.931  42.574   9.744  1.00 30.92           C  \\nATOM    178  OE1 GLN A  25       6.537  43.143  10.657  1.00 31.89           O  \\nATOM    179  NE2 GLN A  25       4.664  42.169   9.867  1.00 32.35           N  \\nATOM    180  N   ASN A  26      10.764  40.482   6.795  1.00 22.13           N  \\nATOM    181  CA  ASN A  26      12.198  40.642   6.628  1.00 24.67           C  \\nATOM    182  C   ASN A  26      12.936  41.362   7.734  1.00 23.62           C  \\nATOM    183  O   ASN A  26      13.603  42.372   7.521  1.00 25.10           O  \\nATOM    184  CB  ASN A  26      12.522  41.369   5.315  1.00 27.68           C  \\nATOM    185  CG  ASN A  26      12.942  40.351   4.288  1.00 32.80           C  \\nATOM    186  OD1 ASN A  26      12.135  39.699   3.602  1.00 33.77           O  \\nATOM    187  ND2 ASN A  26      14.268  40.203   4.237  1.00 34.26           N  \\nATOM    188  N   HIS A  27      12.764  40.842   8.945  1.00 19.95           N  \\nATOM    189  CA  HIS A  27      13.593  41.266  10.057  1.00 17.99           C  \\nATOM    190  C   HIS A  27      13.441  40.299  11.215  1.00 16.84           C  \\nATOM    191  O   HIS A  27      12.411  39.623  11.344  1.00 14.72           O  \\nATOM    192  CB  HIS A  27      13.241  42.680  10.551  1.00 19.80           C  \\nATOM    193  CG  HIS A  27      11.926  42.908  11.258  1.00 20.77           C  \\nATOM    194  ND1 HIS A  27      11.936  43.314  12.510  1.00 22.17           N  \\nATOM    195  CD2 HIS A  27      10.674  42.768  10.729  1.00 22.32           C  \\nATOM    196  CE1 HIS A  27      10.657  43.430  12.785  1.00 23.86           C  \\nATOM    197  NE2 HIS A  27       9.905  43.110  11.738  1.00 23.97           N  \\nATOM    198  N   PHE A  28      14.516  40.222  12.012  1.00 15.44           N  \\nATOM    199  CA  PHE A  28      14.528  39.413  13.223  1.00 13.89           C  \\nATOM    200  C   PHE A  28      14.026  40.196  14.440  1.00 15.33           C  \\nATOM    201  O   PHE A  28      14.528  41.255  14.786  1.00 16.80           O  \\nATOM    202  CB  PHE A  28      15.938  38.906  13.470  1.00 12.73           C  \\nATOM    203  CG  PHE A  28      16.083  38.150  14.786  1.00 13.35           C  \\nATOM    204  CD1 PHE A  28      15.354  36.990  15.012  1.00 13.80           C  \\nATOM    205  CD2 PHE A  28      16.974  38.615  15.740  1.00 14.54           C  \\nATOM    206  CE1 PHE A  28      15.534  36.310  16.194  1.00 14.45           C  \\nATOM    207  CE2 PHE A  28      17.147  37.925  16.920  1.00 14.61           C  \\nATOM    208  CZ  PHE A  28      16.420  36.768  17.137  1.00 14.65           C  \\nATOM    209  N   VAL A  29      12.970  39.697  15.089  1.00 17.22           N  \\nATOM    210  CA  VAL A  29      12.467  40.313  16.294  1.00 19.11           C  \\nATOM    211  C   VAL A  29      13.127  39.606  17.476  1.00 19.36           C  \\nATOM    212  O   VAL A  29      13.245  38.376  17.605  1.00 18.14           O  \\nATOM    213  CB  VAL A  29      10.894  40.205  16.412  1.00 22.03           C  \\nATOM    214  CG1 VAL A  29      10.334  40.869  15.143  1.00 19.66           C  \\nATOM    215  CG2 VAL A  29      10.377  38.776  16.631  1.00 23.15           C  \\nATOM    216  N   ASP A  30      13.615  40.484  18.324  1.00 20.14           N  \\nATOM    217  CA  ASP A  30      14.229  40.019  19.518  1.00 24.04           C  \\nATOM    218  C   ASP A  30      13.676  40.746  20.721  1.00 25.93           C  \\nATOM    219  O   ASP A  30      14.244  41.709  21.238  1.00 29.26           O  \\nATOM    220  CB  ASP A  30      15.719  40.203  19.443  1.00 27.10           C  \\nATOM    221  CG  ASP A  30      16.389  39.731  20.732  1.00 32.46           C  \\nATOM    222  OD1 ASP A  30      15.878  38.859  21.464  1.00 34.28           O  \\nATOM    223  OD2 ASP A  30      17.458  40.264  21.002  1.00 36.46           O  \\nATOM    224  N   GLU A  31      12.472  40.347  21.076  1.00 26.14           N  \\nATOM    225  CA  GLU A  31      11.922  40.804  22.318  1.00 28.72           C  \\nATOM    226  C   GLU A  31      10.870  39.836  22.749  1.00 26.20           C  \\nATOM    227  O   GLU A  31      10.466  38.931  22.014  1.00 23.14           O  \\nATOM    228  CB  GLU A  31      11.337  42.210  22.204  1.00 33.25           C  \\nATOM    229  CG  GLU A  31      10.350  42.618  21.152  1.00 40.08           C  \\nATOM    230  CD  GLU A  31      10.572  44.107  20.887  1.00 44.71           C  \\nATOM    231  OE1 GLU A  31      10.930  44.444  19.755  1.00 47.50           O  \\nATOM    232  OE2 GLU A  31      10.420  44.928  21.806  1.00 47.89           O  \\nATOM    233  N   TYR A  32      10.550  40.044  24.028  1.00 22.26           N  \\nATOM    234  CA  TYR A  32       9.597  39.196  24.685  1.00 18.83           C  \\nATOM    235  C   TYR A  32       8.204  39.688  24.317  1.00 17.53           C  \\nATOM    236  O   TYR A  32       7.794  40.792  24.676  1.00 18.45           O  \\nATOM    237  CB  TYR A  32       9.909  39.259  26.199  1.00 16.84           C  \\nATOM    238  CG  TYR A  32       9.076  38.309  27.052  1.00 15.64           C  \\nATOM    239  CD1 TYR A  32       9.598  37.096  27.433  1.00 16.17           C  \\nATOM    240  CD2 TYR A  32       7.814  38.686  27.459  1.00 15.54           C  \\nATOM    241  CE1 TYR A  32       8.845  36.251  28.239  1.00 17.86           C  \\nATOM    242  CE2 TYR A  32       7.051  37.861  28.257  1.00 14.40           C  \\nATOM    243  CZ  TYR A  32       7.572  36.635  28.652  1.00 16.47           C  \\nATOM    244  OH  TYR A  32       6.821  35.789  29.485  1.00 13.59           O  \\nATOM    245  N   ASP A  33       7.461  38.841  23.604  1.00 13.43           N  \\nATOM    246  CA  ASP A  33       6.128  39.200  23.141  1.00 14.00           C  \\nATOM    247  C   ASP A  33       5.262  37.955  22.854  1.00 13.56           C  \\nATOM    248  O   ASP A  33       5.024  37.586  21.681  1.00 12.47           O  \\nATOM    249  CB  ASP A  33       6.271  40.072  21.864  1.00 15.92           C  \\nATOM    250  CG  ASP A  33       4.993  40.654  21.269  1.00 18.19           C  \\nATOM    251  OD1 ASP A  33       3.923  40.631  21.879  1.00 21.46           O  \\nATOM    252  OD2 ASP A  33       5.074  41.164  20.167  1.00 20.34           O  \\nATOM    253  N   PRO A  34       4.752  37.283  23.903  1.00 12.04           N  \\nATOM    254  CA  PRO A  34       3.949  36.070  23.788  1.00 11.09           C  \\nATOM    255  C   PRO A  34       2.720  36.248  22.895  1.00 11.76           C  \\nATOM    256  O   PRO A  34       1.953  37.220  23.006  1.00 12.63           O  \\nATOM    257  CB  PRO A  34       3.590  35.711  25.219  1.00 12.50           C  \\nATOM    258  CG  PRO A  34       4.643  36.418  26.033  1.00 12.14           C  \\nATOM    259  CD  PRO A  34       4.817  37.717  25.311  1.00 12.82           C  \\nATOM    260  N   THR A  35       2.554  35.282  21.972  1.00 11.00           N  \\nATOM    261  CA  THR A  35       1.421  35.320  21.033  1.00 11.54           C  \\nATOM    262  C   THR A  35       0.140  34.746  21.639  1.00 13.51           C  \\nATOM    263  O   THR A  35       0.174  33.820  22.450  1.00 12.45           O  \\nATOM    264  CB  THR A  35       1.730  34.518  19.751  1.00 11.34           C  \\nATOM    265  OG1 THR A  35       2.900  35.134  19.233  1.00 11.20           O  \\nATOM    266  CG2 THR A  35       0.706  34.611  18.637  1.00 13.10           C  \\nATOM    267  N   ILE A  36      -0.996  35.332  21.233  1.00 15.48           N  \\nATOM    268  CA  ILE A  36      -2.308  34.792  21.567  1.00 19.42           C  \\nATOM    269  C   ILE A  36      -2.663  34.079  20.274  1.00 17.36           C  \\nATOM    270  O   ILE A  36      -2.749  32.863  20.256  1.00 18.29           O  \\nATOM    271  CB  ILE A  36      -3.353  35.914  21.879  1.00 21.30           C  \\nATOM    272  CG1 ILE A  36      -2.930  36.823  23.072  1.00 24.32           C  \\nATOM    273  CG2 ILE A  36      -4.670  35.207  22.136  1.00 21.54           C  \\nATOM    274  CD1 ILE A  36      -2.494  36.167  24.413  1.00 24.70           C  \\nATOM    275  N   GLU A  37      -2.818  34.822  19.181  1.00 19.35           N  \\nATOM    276  CA  GLU A  37      -3.059  34.221  17.889  1.00 21.47           C  \\nATOM    277  C   GLU A  37      -2.957  35.362  16.897  1.00 23.77           C  \\nATOM    278  O   GLU A  37      -3.636  36.393  17.031  1.00 25.49           O  \\nATOM    279  CB  GLU A  37      -4.440  33.611  17.880  1.00 25.97           C  \\nATOM    280  CG  GLU A  37      -4.766  32.776  16.667  1.00 33.81           C  \\nATOM    281  CD  GLU A  37      -6.160  32.154  16.777  1.00 38.37           C  \\nATOM    282  OE1 GLU A  37      -7.095  32.664  16.149  1.00 37.86           O  \\nATOM    283  OE2 GLU A  37      -6.298  31.164  17.515  1.00 42.24           O  \\nATOM    284  N   ASP A  38      -2.070  35.212  15.911  1.00 19.96           N  \\nATOM    285  CA  ASP A  38      -1.899  36.178  14.839  1.00 17.94           C  \\nATOM    286  C   ASP A  38      -2.024  35.526  13.473  1.00 18.21           C  \\nATOM    287  O   ASP A  38      -1.582  34.393  13.294  1.00 16.96           O  \\nATOM    288  CB  ASP A  38      -0.540  36.833  14.965  1.00 18.83           C  \\nATOM    289  CG  ASP A  38      -0.507  37.817  16.133  1.00 20.08           C  \\nATOM    290  OD1 ASP A  38      -1.390  38.660  16.208  1.00 25.47           O  \\nATOM    291  OD2 ASP A  38       0.375  37.781  16.967  1.00 20.08           O  \\nATOM    292  N   SER A  39      -2.698  36.170  12.507  1.00 17.82           N  \\nATOM    293  CA  SER A  39      -2.691  35.658  11.133  1.00 18.16           C  \\nATOM    294  C   SER A  39      -2.012  36.627  10.171  1.00 16.95           C  \\nATOM    295  O   SER A  39      -2.053  37.853  10.333  1.00 15.80           O  \\nATOM    296  CB  SER A  39      -4.094  35.402  10.637  1.00 23.01           C  \\nATOM    297  OG  SER A  39      -4.873  36.571  10.754  1.00 27.13           O  \\nATOM    298  N   TYR A  40      -1.336  36.080   9.168  1.00 15.44           N  \\nATOM    299  CA  TYR A  40      -0.607  36.889   8.196  1.00 16.25           C  \\nATOM    300  C   TYR A  40      -1.018  36.383   6.830  1.00 17.06           C  \\nATOM    301  O   TYR A  40      -1.160  35.172   6.638  1.00 18.29           O  \\nATOM    302  CB  TYR A  40       0.909  36.724   8.370  1.00 16.59           C  \\nATOM    303  CG  TYR A  40       1.439  37.242   9.707  1.00 16.10           C  \\nATOM    304  CD1 TYR A  40       1.473  36.412  10.815  1.00 16.77           C  \\nATOM    305  CD2 TYR A  40       1.883  38.549   9.799  1.00 18.02           C  \\nATOM    306  CE1 TYR A  40       1.950  36.893  12.021  1.00 16.86           C  \\nATOM    307  CE2 TYR A  40       2.368  39.032  11.001  1.00 18.70           C  \\nATOM    308  CZ  TYR A  40       2.396  38.199  12.104  1.00 17.62           C  \\nATOM    309  OH  TYR A  40       2.853  38.695  13.304  1.00 18.37           O  \\nATOM    310  N   ARG A  41      -1.270  37.303   5.900  1.00 15.48           N  \\nATOM    311  CA  ARG A  41      -1.660  36.918   4.558  1.00 15.42           C  \\nATOM    312  C   ARG A  41      -0.730  37.606   3.586  1.00 16.76           C  \\nATOM    313  O   ARG A  41      -0.536  38.829   3.633  1.00 17.01           O  \\nATOM    314  CB  ARG A  41      -3.087  37.342   4.277  1.00 17.64           C  \\nATOM    315  CG  ARG A  41      -4.150  36.507   4.947  1.00 18.09           C  \\nATOM    316  CD  ARG A  41      -5.528  37.181   4.849  1.00 19.64           C  \\nATOM    317  NE  ARG A  41      -6.066  37.318   3.489  1.00 20.11           N  \\nATOM    318  CZ  ARG A  41      -6.660  36.329   2.803  1.00 22.17           C  \\nATOM    319  NH1 ARG A  41      -7.219  36.571   1.617  1.00 22.48           N  \\nATOM    320  NH2 ARG A  41      -6.711  35.090   3.274  1.00 23.55           N  \\nATOM    321  N   LYS A  42      -0.135  36.827   2.679  1.00 16.52           N  \\nATOM    322  CA  LYS A  42       0.835  37.405   1.758  1.00 16.59           C  \\nATOM    323  C   LYS A  42       0.740  36.746   0.387  1.00 16.04           C  \\nATOM    324  O   LYS A  42       0.794  35.528   0.307  1.00 14.74           O  \\nATOM    325  CB  LYS A  42       2.208  37.210   2.401  1.00 18.22           C  \\nATOM    326  CG  LYS A  42       3.388  37.784   1.669  1.00 22.68           C  \\nATOM    327  CD  LYS A  42       3.369  39.274   1.792  1.00 28.09           C  \\nATOM    328  CE  LYS A  42       4.695  39.876   1.317  1.00 31.70           C  \\nATOM    329  NZ  LYS A  42       4.828  39.827  -0.128  1.00 35.70           N  \\nATOM    330  N   GLN A  43       0.561  37.510  -0.697  1.00 16.89           N  \\nATOM    331  CA  GLN A  43       0.579  36.950  -2.047  1.00 19.10           C  \\nATOM    332  C   GLN A  43       2.026  36.729  -2.448  1.00 15.08           C  \\nATOM    333  O   GLN A  43       2.828  37.654  -2.362  1.00 15.72           O  \\nATOM    334  CB  GLN A  43      -0.116  37.908  -3.043  1.00 23.32           C  \\nATOM    335  CG  GLN A  43       0.543  38.332  -4.415  1.00 32.83           C  \\nATOM    336  CD  GLN A  43       1.081  37.332  -5.482  1.00 36.11           C  \\nATOM    337  OE1 GLN A  43       0.457  36.335  -5.865  1.00 37.78           O  \\nATOM    338  NE2 GLN A  43       2.271  37.606  -6.053  1.00 35.45           N  \\nATOM    339  N   VAL A  44       2.394  35.532  -2.922  1.00 12.79           N  \\nATOM    340  CA  VAL A  44       3.771  35.242  -3.311  1.00 11.03           C  \\nATOM    341  C   VAL A  44       3.724  34.425  -4.605  1.00 14.28           C  \\nATOM    342  O   VAL A  44       2.633  34.109  -5.090  1.00 14.90           O  \\nATOM    343  CB  VAL A  44       4.508  34.411  -2.223  1.00 13.21           C  \\nATOM    344  CG1 VAL A  44       4.669  35.282  -0.977  1.00 15.04           C  \\nATOM    345  CG2 VAL A  44       3.761  33.116  -1.889  1.00 12.89           C  \\nATOM    346  N   VAL A  45       4.879  34.127  -5.235  1.00 13.20           N  \\nATOM    347  CA  VAL A  45       4.938  33.277  -6.419  1.00 12.28           C  \\nATOM    348  C   VAL A  45       5.836  32.161  -5.970  1.00 11.08           C  \\nATOM    349  O   VAL A  45       6.988  32.447  -5.636  1.00 13.13           O  \\nATOM    350  CB  VAL A  45       5.587  33.979  -7.641  1.00 13.01           C  \\nATOM    351  CG1 VAL A  45       5.549  33.031  -8.810  1.00 14.51           C  \\nATOM    352  CG2 VAL A  45       4.787  35.173  -8.108  1.00 13.54           C  \\nATOM    353  N   ILE A  46       5.374  30.907  -5.915  1.00  9.89           N  \\nATOM    354  CA  ILE A  46       6.274  29.795  -5.491  1.00 11.03           C  \\nATOM    355  C   ILE A  46       6.369  28.800  -6.662  1.00  8.85           C  \\nATOM    356  O   ILE A  46       5.319  28.296  -7.059  1.00 10.25           O  \\nATOM    357  CB  ILE A  46       5.692  29.115  -4.203  1.00 10.64           C  \\nATOM    358  CG1 ILE A  46       5.652  30.096  -3.042  1.00 11.20           C  \\nATOM    359  CG2 ILE A  46       6.599  27.953  -3.757  1.00 12.93           C  \\nATOM    360  CD1 ILE A  46       4.937  29.536  -1.808  1.00  8.90           C  \\nATOM    361  N   ASP A  47       7.565  28.495  -7.224  1.00  9.41           N  \\nATOM    362  CA  ASP A  47       7.767  27.650  -8.426  1.00  9.70           C  \\nATOM    363  C   ASP A  47       6.838  28.031  -9.572  1.00 11.14           C  \\nATOM    364  O   ASP A  47       6.225  27.211 -10.248  1.00 14.09           O  \\nATOM    365  CB  ASP A  47       7.566  26.165  -8.105  1.00 10.73           C  \\nATOM    366  CG  ASP A  47       8.388  25.783  -6.874  1.00 13.52           C  \\nATOM    367  OD1 ASP A  47       7.805  25.291  -5.898  1.00 12.04           O  \\nATOM    368  OD2 ASP A  47       9.598  26.024  -6.872  1.00 11.31           O  \\nATOM    369  N   GLY A  48       6.724  29.353  -9.735  1.00 10.78           N  \\nATOM    370  CA  GLY A  48       5.930  29.961 -10.785  1.00 13.29           C  \\nATOM    371  C   GLY A  48       4.426  29.989 -10.548  1.00 14.28           C  \\nATOM    372  O   GLY A  48       3.677  30.512 -11.386  1.00 15.19           O  \\nATOM    373  N   GLU A  49       3.944  29.422  -9.439  1.00 13.41           N  \\nATOM    374  CA  GLU A  49       2.520  29.457  -9.143  1.00 13.90           C  \\nATOM    375  C   GLU A  49       2.223  30.700  -8.314  1.00 15.34           C  \\nATOM    376  O   GLU A  49       2.860  30.906  -7.275  1.00 14.56           O  \\nATOM    377  CB  GLU A  49       2.124  28.238  -8.340  1.00 15.50           C  \\nATOM    378  CG  GLU A  49       0.630  28.250  -7.990  1.00 18.10           C  \\nATOM    379  CD  GLU A  49       0.204  27.110  -7.100  1.00 19.13           C  \\nATOM    380  OE1 GLU A  49       0.717  27.004  -5.996  1.00 19.53           O  \\nATOM    381  OE2 GLU A  49      -0.630  26.321  -7.529  1.00 22.12           O  \\nATOM    382  N   THR A  50       1.267  31.536  -8.734  1.00 14.06           N  \\nATOM    383  CA  THR A  50       0.901  32.697  -7.927  1.00 16.15           C  \\nATOM    384  C   THR A  50      -0.001  32.169  -6.819  1.00 13.35           C  \\nATOM    385  O   THR A  50      -0.938  31.409  -7.046  1.00 15.13           O  \\nATOM    386  CB  THR A  50       0.203  33.755  -8.846  1.00 17.88           C  \\nATOM    387  OG1 THR A  50      -0.245  34.813  -8.012  1.00 24.33           O  \\nATOM    388  CG2 THR A  50      -1.028  33.236  -9.518  1.00 19.78           C  \\nATOM    389  N   CYS A  51       0.306  32.442  -5.563  1.00 13.28           N  \\nATOM    390  CA  CYS A  51      -0.566  31.959  -4.508  1.00 15.16           C  \\nATOM    391  C   CYS A  51      -0.579  32.892  -3.303  1.00 14.00           C  \\nATOM    392  O   CYS A  51       0.161  33.874  -3.231  1.00 11.81           O  \\nATOM    393  CB  CYS A  51      -0.132  30.537  -4.079  1.00 14.08           C  \\nATOM    394  SG  CYS A  51       1.597  30.409  -3.575  1.00 16.09           S  \\nATOM    395  N   LEU A  52      -1.521  32.609  -2.405  1.00 13.77           N  \\nATOM    396  CA  LEU A  52      -1.573  33.397  -1.199  1.00 16.01           C  \\nATOM    397  C   LEU A  52      -1.260  32.497  -0.025  1.00 14.53           C  \\nATOM    398  O   LEU A  52      -1.868  31.443   0.113  1.00 16.05           O  \\nATOM    399  CB  LEU A  52      -2.957  33.986  -1.014  1.00 19.71           C  \\nATOM    400  CG  LEU A  52      -3.122  34.885   0.214  1.00 20.84           C  \\nATOM    401  CD1 LEU A  52      -3.070  36.329  -0.220  1.00 21.38           C  \\nATOM    402  CD2 LEU A  52      -4.420  34.526   0.923  1.00 20.90           C  \\nATOM    403  N   LEU A  53      -0.319  32.904   0.800  1.00 12.41           N  \\nATOM    404  CA  LEU A  53       0.019  32.232   2.048  1.00 13.70           C  \\nATOM    405  C   LEU A  53      -0.841  32.852   3.148  1.00 13.85           C  \\nATOM    406  O   LEU A  53      -0.733  34.056   3.413  1.00 12.82           O  \\nATOM    407  CB  LEU A  53       1.485  32.464   2.400  1.00 14.51           C  \\nATOM    408  CG  LEU A  53       2.707  31.589   2.031  1.00 19.29           C  \\nATOM    409  CD1 LEU A  53       2.394  30.553   0.962  1.00 18.56           C  \\nATOM    410  CD2 LEU A  53       3.843  32.557   1.741  1.00 20.00           C  \\nATOM    411  N   ASP A  54      -1.699  32.023   3.751  1.00 11.96           N  \\nATOM    412  CA  ASP A  54      -2.476  32.445   4.884  1.00 15.07           C  \\nATOM    413  C   ASP A  54      -1.882  31.675   6.055  1.00 12.67           C  \\nATOM    414  O   ASP A  54      -2.099  30.477   6.234  1.00 13.99           O  \\nATOM    415  CB  ASP A  54      -3.930  32.088   4.644  1.00 19.33           C  \\nATOM    416  CG  ASP A  54      -4.826  32.609   5.746  1.00 24.24           C  \\nATOM    417  OD1 ASP A  54      -4.412  32.652   6.897  1.00 28.17           O  \\nATOM    418  OD2 ASP A  54      -5.958  32.979   5.450  1.00 27.59           O  \\nATOM    419  N   ILE A  55      -1.127  32.389   6.880  1.00 11.78           N  \\nATOM    420  CA  ILE A  55      -0.346  31.769   7.935  1.00 12.15           C  \\nATOM    421  C   ILE A  55      -0.953  32.099   9.296  1.00 13.10           C  \\nATOM    422  O   ILE A  55      -1.195  33.271   9.587  1.00 13.84           O  \\nATOM    423  CB  ILE A  55       1.109  32.282   7.931  1.00 12.42           C  \\nATOM    424  CG1 ILE A  55       1.859  31.979   6.653  1.00 13.99           C  \\nATOM    425  CG2 ILE A  55       1.859  31.505   9.018  1.00 13.24           C  \\nATOM    426  CD1 ILE A  55       3.176  32.768   6.497  1.00 15.68           C  \\nATOM    427  N   LEU A  56      -1.176  31.083  10.134  1.00 11.64           N  \\nATOM    428  CA  LEU A  56      -1.642  31.341  11.490  1.00 11.98           C  \\nATOM    429  C   LEU A  56      -0.524  31.027  12.431  1.00  9.99           C  \\nATOM    430  O   LEU A  56      -0.017  29.895  12.447  1.00  9.72           O  \\nATOM    431  CB  LEU A  56      -2.836  30.463  11.896  1.00 13.80           C  \\nATOM    432  CG  LEU A  56      -3.470  30.741  13.297  1.00 16.13           C  \\nATOM    433  CD1 LEU A  56      -4.347  31.987  13.260  1.00 15.28           C  \\nATOM    434  CD2 LEU A  56      -4.321  29.580  13.692  1.00 16.84           C  \\nATOM    435  N   ASP A  57      -0.188  32.084  13.182  1.00 10.40           N  \\nATOM    436  CA  ASP A  57       0.815  32.014  14.219  1.00 10.30           C  \\nATOM    437  C   ASP A  57       0.127  31.812  15.571  1.00 11.75           C  \\nATOM    438  O   ASP A  57      -0.652  32.631  16.053  1.00 12.91           O  \\nATOM    439  CB  ASP A  57       1.602  33.296  14.173  1.00 10.66           C  \\nATOM    440  CG  ASP A  57       2.778  33.336  15.132  1.00 11.39           C  \\nATOM    441  OD1 ASP A  57       3.306  32.298  15.535  1.00 11.07           O  \\nATOM    442  OD2 ASP A  57       3.168  34.448  15.455  1.00 13.15           O  \\nATOM    443  N   THR A  58       0.447  30.723  16.244  1.00 12.55           N  \\nATOM    444  CA  THR A  58      -0.324  30.325  17.416  1.00 12.98           C  \\nATOM    445  C   THR A  58       0.368  30.541  18.758  1.00 13.48           C  \\nATOM    446  O   THR A  58       1.568  30.812  18.815  1.00 14.00           O  \\nATOM    447  CB  THR A  58      -0.698  28.851  17.219  1.00  9.54           C  \\nATOM    448  OG1 THR A  58       0.491  28.084  17.369  1.00 11.40           O  \\nATOM    449  CG2 THR A  58      -1.313  28.597  15.848  1.00 12.27           C  \\nATOM    450  N   ALA A  59      -0.371  30.382  19.868  1.00 14.38           N  \\nATOM    451  CA  ALA A  59       0.230  30.510  21.194  1.00 14.79           C  \\nATOM    452  C   ALA A  59       0.928  29.210  21.608  1.00 15.41           C  \\nATOM    453  O   ALA A  59       0.448  28.106  21.324  1.00 13.07           O  \\nATOM    454  CB  ALA A  59      -0.845  30.818  22.238  1.00 15.47           C  \\nATOM    455  N   GLY A  60       2.084  29.341  22.271  1.00 17.71           N  \\nATOM    456  CA  GLY A  60       2.807  28.208  22.814  1.00 24.18           C  \\nATOM    457  C   GLY A  60       2.190  27.751  24.130  1.00 31.21           C  \\nATOM    458  O   GLY A  60       2.538  26.688  24.627  1.00 30.44           O  \\nATOM    459  N   GLN A  61       1.260  28.503  24.726  1.00 39.50           N  \\nATOM    460  CA  GLN A  61       0.671  28.099  25.994  1.00 48.84           C  \\nATOM    461  C   GLN A  61      -0.683  27.395  25.985  1.00 51.52           C  \\nATOM    462  O   GLN A  61      -1.606  27.780  25.276  1.00 52.17           O  \\nATOM    463  CB  GLN A  61       0.605  29.327  26.874  1.00 51.56           C  \\nATOM    464  CG  GLN A  61       0.871  28.861  28.312  1.00 56.74           C  \\nATOM    465  CD  GLN A  61       1.506  29.937  29.177  1.00 58.74           C  \\nATOM    466  OE1 GLN A  61       2.647  29.795  29.616  1.00 60.12           O  \\nATOM    467  NE2 GLN A  61       0.783  31.031  29.429  1.00 60.18           N  \\nATOM    468  N   GLU A  62      -0.804  26.349  26.818  1.00 57.40           N  \\nATOM    469  CA  GLU A  62      -2.008  25.527  26.927  1.00 61.96           C  \\nATOM    470  C   GLU A  62      -3.185  26.196  27.631  1.00 62.20           C  \\nATOM    471  O   GLU A  62      -4.138  25.496  27.975  1.00 62.85           O  \\nATOM    472  CB  GLU A  62      -1.629  24.183  27.632  1.00 65.18           C  \\nATOM    473  CG  GLU A  62      -2.622  22.966  27.761  1.00 68.22           C  \\nATOM    474  CD  GLU A  62      -3.032  22.132  26.528  1.00 70.25           C  \\nATOM    475  OE1 GLU A  62      -2.710  22.494  25.395  1.00 71.45           O  \\nATOM    476  OE2 GLU A  62      -3.691  21.101  26.689  1.00 70.00           O  \\nATOM    477  N   GLU A  63      -3.206  27.520  27.904  1.00 62.78           N  \\nATOM    478  CA  GLU A  63      -4.496  28.139  28.254  1.00 63.62           C  \\nATOM    479  C   GLU A  63      -5.457  27.854  27.085  1.00 63.71           C  \\nATOM    480  O   GLU A  63      -6.688  27.814  27.211  1.00 62.97           O  \\nATOM    481  CB  GLU A  63      -4.372  29.666  28.475  1.00 65.13           C  \\nATOM    482  CG  GLU A  63      -3.456  30.553  27.607  1.00 66.91           C  \\nATOM    483  CD  GLU A  63      -4.012  31.235  26.345  1.00 68.05           C  \\nATOM    484  OE1 GLU A  63      -5.220  31.462  26.220  1.00 68.64           O  \\nATOM    485  OE2 GLU A  63      -3.198  31.568  25.481  1.00 67.62           O  \\nATOM    486  N   TYR A  64      -4.801  27.606  25.934  1.00 63.36           N  \\nATOM    487  CA  TYR A  64      -5.420  27.092  24.743  1.00 63.20           C  \\nATOM    488  C   TYR A  64      -5.905  25.653  24.875  1.00 61.76           C  \\nATOM    489  O   TYR A  64      -7.120  25.486  24.858  1.00 63.04           O  \\nATOM    490  CB  TYR A  64      -4.437  27.208  23.613  1.00 64.83           C  \\nATOM    491  CG  TYR A  64      -4.736  28.459  22.818  1.00 66.98           C  \\nATOM    492  CD1 TYR A  64      -4.300  29.686  23.267  1.00 67.55           C  \\nATOM    493  CD2 TYR A  64      -5.459  28.355  21.646  1.00 67.97           C  \\nATOM    494  CE1 TYR A  64      -4.587  30.825  22.536  1.00 69.49           C  \\nATOM    495  CE2 TYR A  64      -5.752  29.488  20.911  1.00 69.29           C  \\nATOM    496  CZ  TYR A  64      -5.313  30.718  21.359  1.00 70.05           C  \\nATOM    497  OH  TYR A  64      -5.589  31.852  20.614  1.00 70.67           O  \\nATOM    498  N   SER A  65      -5.095  24.590  24.997  1.00 58.67           N  \\nATOM    499  CA  SER A  65      -5.617  23.230  25.127  1.00 56.93           C  \\nATOM    500  C   SER A  65      -6.614  22.757  24.063  1.00 55.33           C  \\nATOM    501  O   SER A  65      -6.321  22.936  22.876  1.00 54.74           O  \\nATOM    502  CB  SER A  65      -6.190  23.122  26.552  1.00 57.99           C  \\nATOM    503  OG  SER A  65      -7.251  24.011  26.900  1.00 59.80           O  \\nATOM    504  N   ALA A  66      -7.762  22.119  24.359  1.00 52.33           N  \\nATOM    505  CA  ALA A  66      -8.766  21.824  23.335  1.00 48.80           C  \\nATOM    506  C   ALA A  66      -9.138  23.050  22.492  1.00 45.71           C  \\nATOM    507  O   ALA A  66      -9.751  22.862  21.449  1.00 44.33           O  \\nATOM    508  CB  ALA A  66     -10.043  21.300  23.971  1.00 49.14           C  \\nATOM    509  N   MET A  67      -8.829  24.310  22.875  1.00 42.85           N  \\nATOM    510  CA  MET A  67      -8.984  25.465  21.967  1.00 40.01           C  \\nATOM    511  C   MET A  67      -8.093  25.362  20.719  1.00 36.79           C  \\nATOM    512  O   MET A  67      -8.221  26.184  19.805  1.00 36.08           O  \\nATOM    513  CB  MET A  67      -8.594  26.804  22.581  1.00 42.50           C  \\nATOM    514  CG  MET A  67      -9.359  27.330  23.772  1.00 45.93           C  \\nATOM    515  SD  MET A  67     -11.140  27.396  23.483  1.00 48.74           S  \\nATOM    516  CE  MET A  67     -11.526  25.880  24.321  1.00 47.64           C  \\nATOM    517  N   ARG A  68      -7.128  24.418  20.665  1.00 31.59           N  \\nATOM    518  CA  ARG A  68      -6.322  24.217  19.467  1.00 26.34           C  \\nATOM    519  C   ARG A  68      -7.140  23.473  18.424  1.00 23.68           C  \\nATOM    520  O   ARG A  68      -6.846  23.617  17.231  1.00 21.93           O  \\nATOM    521  CB  ARG A  68      -5.057  23.394  19.745  1.00 25.86           C  \\nATOM    522  CG  ARG A  68      -4.161  23.898  20.869  1.00 29.36           C  \\nATOM    523  CD  ARG A  68      -2.881  23.061  20.998  1.00 32.88           C  \\nATOM    524  NE  ARG A  68      -2.221  22.990  22.315  1.00 36.64           N  \\nATOM    525  CZ  ARG A  68      -1.700  24.031  23.001  1.00 39.66           C  \\nATOM    526  NH1 ARG A  68      -1.903  25.315  22.664  1.00 42.06           N  \\nATOM    527  NH2 ARG A  68      -0.933  23.798  24.071  1.00 39.61           N  \\nATOM    528  N   ASP A  69      -8.188  22.721  18.793  1.00 19.14           N  \\nATOM    529  CA  ASP A  69      -8.880  21.905  17.822  1.00 20.43           C  \\nATOM    530  C   ASP A  69      -9.437  22.623  16.617  1.00 22.16           C  \\nATOM    531  O   ASP A  69      -9.453  22.109  15.488  1.00 19.58           O  \\nATOM    532  CB  ASP A  69     -10.025  21.148  18.478  1.00 20.58           C  \\nATOM    533  CG  ASP A  69      -9.598  19.935  19.303  1.00 22.81           C  \\nATOM    534  OD1 ASP A  69      -8.409  19.708  19.538  1.00 24.59           O  \\nATOM    535  OD2 ASP A  69     -10.475  19.176  19.715  1.00 25.28           O  \\nATOM    536  N   GLN A  70      -9.834  23.874  16.840  1.00 23.26           N  \\nATOM    537  CA  GLN A  70     -10.451  24.609  15.759  1.00 23.53           C  \\nATOM    538  C   GLN A  70      -9.437  24.925  14.667  1.00 21.20           C  \\nATOM    539  O   GLN A  70      -9.653  24.591  13.493  1.00 20.90           O  \\nATOM    540  CB  GLN A  70     -11.071  25.881  16.341  1.00 27.73           C  \\nATOM    541  CG  GLN A  70     -12.498  26.019  15.832  1.00 34.11           C  \\nATOM    542  CD  GLN A  70     -13.103  27.407  16.008  1.00 37.87           C  \\nATOM    543  OE1 GLN A  70     -12.532  28.284  16.672  1.00 39.77           O  \\nATOM    544  NE2 GLN A  70     -14.283  27.620  15.406  1.00 37.89           N  \\nATOM    545  N   TYR A  71      -8.292  25.515  15.050  1.00 19.09           N  \\nATOM    546  CA  TYR A  71      -7.329  25.873  14.030  1.00 17.69           C  \\nATOM    547  C   TYR A  71      -6.648  24.658  13.416  1.00 17.24           C  \\nATOM    548  O   TYR A  71      -6.247  24.675  12.245  1.00 17.25           O  \\nATOM    549  CB  TYR A  71      -6.302  26.868  14.625  1.00 17.87           C  \\nATOM    550  CG  TYR A  71      -5.311  26.480  15.725  1.00 18.90           C  \\nATOM    551  CD1 TYR A  71      -4.355  25.510  15.492  1.00 19.74           C  \\nATOM    552  CD2 TYR A  71      -5.328  27.162  16.940  1.00 19.21           C  \\nATOM    553  CE1 TYR A  71      -3.402  25.212  16.450  1.00 20.38           C  \\nATOM    554  CE2 TYR A  71      -4.375  26.868  17.901  1.00 21.93           C  \\nATOM    555  CZ  TYR A  71      -3.422  25.897  17.645  1.00 21.91           C  \\nATOM    556  OH  TYR A  71      -2.463  25.590  18.588  1.00 26.44           O  \\nATOM    557  N   MET A  72      -6.557  23.559  14.166  1.00 14.81           N  \\nATOM    558  CA  MET A  72      -5.958  22.393  13.563  1.00 18.59           C  \\nATOM    559  C   MET A  72      -6.849  21.778  12.472  1.00 19.38           C  \\nATOM    560  O   MET A  72      -6.385  21.198  11.492  1.00 17.67           O  \\nATOM    561  CB  MET A  72      -5.658  21.386  14.669  1.00 17.39           C  \\nATOM    562  CG  MET A  72      -4.479  21.796  15.514  1.00 16.35           C  \\nATOM    563  SD  MET A  72      -4.146  20.600  16.820  1.00 17.04           S  \\nATOM    564  CE  MET A  72      -2.635  21.259  17.460  1.00 17.72           C  \\nATOM    565  N   ARG A  73      -8.174  21.931  12.626  1.00 20.31           N  \\nATOM    566  CA  ARG A  73      -9.120  21.350  11.701  1.00 22.24           C  \\nATOM    567  C   ARG A  73      -9.087  22.084  10.370  1.00 20.94           C  \\nATOM    568  O   ARG A  73      -9.183  21.450   9.326  1.00 20.81           O  \\nATOM    569  CB  ARG A  73     -10.499  21.414  12.369  1.00 25.29           C  \\nATOM    570  CG  ARG A  73     -11.553  20.499  11.767  1.00 30.04           C  \\nATOM    571  CD  ARG A  73     -12.775  20.429  12.687  1.00 31.86           C  \\nATOM    572  NE  ARG A  73     -12.483  19.702  13.917  1.00 34.70           N  \\nATOM    573  CZ  ARG A  73     -12.894  20.122  15.130  1.00 36.49           C  \\nATOM    574  NH1 ARG A  73     -12.579  19.402  16.196  1.00 36.93           N  \\nATOM    575  NH2 ARG A  73     -13.609  21.239  15.311  1.00 38.17           N  \\nATOM    576  N   THR A  74      -8.868  23.401  10.413  1.00 21.33           N  \\nATOM    577  CA  THR A  74      -8.813  24.240   9.219  1.00 24.99           C  \\nATOM    578  C   THR A  74      -7.436  24.351   8.544  1.00 23.10           C  \\nATOM    579  O   THR A  74      -7.367  24.738   7.371  1.00 22.59           O  \\nATOM    580  CB  THR A  74      -9.289  25.676   9.538  1.00 25.73           C  \\nATOM    581  OG1 THR A  74      -8.351  26.302  10.409  1.00 31.08           O  \\nATOM    582  CG2 THR A  74     -10.657  25.644  10.201  1.00 27.58           C  \\nATOM    583  N   GLY A  75      -6.345  24.084   9.289  1.00 19.58           N  \\nATOM    584  CA  GLY A  75      -5.013  24.114   8.730  1.00 15.79           C  \\nATOM    585  C   GLY A  75      -4.823  23.023   7.713  1.00 13.66           C  \\nATOM    586  O   GLY A  75      -5.245  21.880   7.884  1.00 14.69           O  \\nATOM    587  N   GLU A  76      -4.228  23.360   6.573  1.00 13.14           N  \\nATOM    588  CA  GLU A  76      -3.914  22.363   5.546  1.00 12.99           C  \\nATOM    589  C   GLU A  76      -2.514  21.740   5.753  1.00 12.22           C  \\nATOM    590  O   GLU A  76      -2.293  20.576   5.426  1.00 13.59           O  \\nATOM    591  CB  GLU A  76      -3.999  23.014   4.145  1.00 13.07           C  \\nATOM    592  CG  GLU A  76      -5.387  23.377   3.676  1.00 14.35           C  \\nATOM    593  CD  GLU A  76      -5.261  24.069   2.323  1.00 20.40           C  \\nATOM    594  OE1 GLU A  76      -5.461  23.403   1.288  1.00 24.32           O  \\nATOM    595  OE2 GLU A  76      -4.913  25.250   2.284  1.00 18.82           O  \\nATOM    596  N   GLY A  77      -1.553  22.503   6.301  1.00 10.73           N  \\nATOM    597  CA  GLY A  77      -0.216  22.018   6.570  1.00  8.07           C  \\nATOM    598  C   GLY A  77       0.302  22.689   7.819  1.00  8.63           C  \\nATOM    599  O   GLY A  77      -0.066  23.823   8.127  1.00  8.55           O  \\nATOM    600  N   PHE A  78       1.248  22.013   8.479  1.00  9.15           N  \\nATOM    601  CA  PHE A  78       1.754  22.468   9.773  1.00  8.87           C  \\nATOM    602  C   PHE A  78       3.268  22.520   9.854  1.00  9.51           C  \\nATOM    603  O   PHE A  78       3.930  21.542   9.486  1.00  9.15           O  \\nATOM    604  CB  PHE A  78       1.247  21.532  10.904  1.00  9.44           C  \\nATOM    605  CG  PHE A  78      -0.267  21.472  11.000  1.00  9.66           C  \\nATOM    606  CD1 PHE A  78      -0.906  22.310  11.901  1.00 10.71           C  \\nATOM    607  CD2 PHE A  78      -0.994  20.645  10.152  1.00 10.20           C  \\nATOM    608  CE1 PHE A  78      -2.295  22.338  11.940  1.00 10.61           C  \\nATOM    609  CE2 PHE A  78      -2.378  20.678  10.192  1.00 12.55           C  \\nATOM    610  CZ  PHE A  78      -3.022  21.529  11.081  1.00 12.80           C  \\nATOM    611  N   LEU A  79       3.813  23.636  10.354  1.00  8.72           N  \\nATOM    612  CA  LEU A  79       5.230  23.626  10.699  1.00  9.93           C  \\nATOM    613  C   LEU A  79       5.279  23.281  12.181  1.00  9.09           C  \\nATOM    614  O   LEU A  79       4.701  24.008  12.988  1.00 11.58           O  \\nATOM    615  CB  LEU A  79       5.921  24.970  10.567  1.00 12.80           C  \\nATOM    616  CG  LEU A  79       6.155  25.659   9.249  1.00 16.39           C  \\nATOM    617  CD1 LEU A  79       7.007  26.911   9.544  1.00 17.02           C  \\nATOM    618  CD2 LEU A  79       6.904  24.756   8.269  1.00 16.98           C  \\nATOM    619  N   CYS A  80       5.950  22.219  12.579  1.00  9.14           N  \\nATOM    620  CA  CYS A  80       6.074  21.821  13.981  1.00 11.81           C  \\nATOM    621  C   CYS A  80       7.448  22.237  14.434  1.00 11.53           C  \\nATOM    622  O   CYS A  80       8.435  21.640  14.000  1.00 12.21           O  \\nATOM    623  CB  CYS A  80       5.903  20.318  14.106  1.00 13.73           C  \\nATOM    624  SG  CYS A  80       4.158  19.900  13.837  1.00 18.26           S  \\nATOM    625  N   VAL A  81       7.486  23.282  15.269  1.00  7.57           N  \\nATOM    626  CA  VAL A  81       8.730  23.956  15.604  1.00  8.45           C  \\nATOM    627  C   VAL A  81       9.248  23.620  16.996  1.00  8.90           C  \\nATOM    628  O   VAL A  81       8.451  23.680  17.939  1.00  8.49           O  \\nATOM    629  CB  VAL A  81       8.544  25.516  15.507  1.00  7.40           C  \\nATOM    630  CG1 VAL A  81       9.917  26.196  15.588  1.00  9.42           C  \\nATOM    631  CG2 VAL A  81       8.004  25.954  14.167  1.00  8.59           C  \\nATOM    632  N   PHE A  82      10.546  23.237  17.119  1.00  9.49           N  \\nATOM    633  CA  PHE A  82      11.195  23.202  18.442  1.00  9.09           C  \\nATOM    634  C   PHE A  82      12.511  23.994  18.350  1.00  9.83           C  \\nATOM    635  O   PHE A  82      12.914  24.440  17.266  1.00  9.02           O  \\nATOM    636  CB  PHE A  82      11.482  21.755  18.902  1.00  9.76           C  \\nATOM    637  CG  PHE A  82      12.536  21.062  18.071  1.00 10.09           C  \\nATOM    638  CD1 PHE A  82      13.867  21.099  18.486  1.00 10.70           C  \\nATOM    639  CD2 PHE A  82      12.192  20.412  16.889  1.00 10.93           C  \\nATOM    640  CE1 PHE A  82      14.838  20.483  17.699  1.00 11.12           C  \\nATOM    641  CE2 PHE A  82      13.181  19.807  16.125  1.00 10.73           C  \\nATOM    642  CZ  PHE A  82      14.505  19.839  16.523  1.00 10.35           C  \\nATOM    643  N   ALA A  83      13.128  24.269  19.516  1.00  9.75           N  \\nATOM    644  CA  ALA A  83      14.407  24.963  19.569  1.00 10.32           C  \\nATOM    645  C   ALA A  83      15.509  23.960  19.916  1.00  9.48           C  \\nATOM    646  O   ALA A  83      15.362  23.142  20.839  1.00  9.61           O  \\nATOM    647  CB  ALA A  83      14.354  26.083  20.620  1.00  9.75           C  \\nATOM    648  N   ILE A  84      16.624  24.042  19.160  1.00 11.41           N  \\nATOM    649  CA  ILE A  84      17.714  23.070  19.299  1.00 13.15           C  \\nATOM    650  C   ILE A  84      18.446  23.223  20.623  1.00 14.57           C  \\nATOM    651  O   ILE A  84      19.234  22.347  20.971  1.00 13.94           O  \\nATOM    652  CB  ILE A  84      18.775  23.151  18.160  1.00 13.29           C  \\nATOM    653  CG1 ILE A  84      19.403  24.509  17.878  1.00 14.90           C  \\nATOM    654  CG2 ILE A  84      18.018  22.701  16.933  1.00 13.09           C  \\nATOM    655  CD1 ILE A  84      20.578  24.957  18.763  1.00 17.59           C  \\nATOM    656  N   ASN A  85      18.200  24.290  21.390  1.00 13.61           N  \\nATOM    657  CA  ASN A  85      18.794  24.394  22.720  1.00 15.52           C  \\nATOM    658  C   ASN A  85      17.758  24.156  23.828  1.00 15.49           C  \\nATOM    659  O   ASN A  85      17.988  24.519  24.999  1.00 16.13           O  \\nATOM    660  CB  ASN A  85      19.446  25.788  22.885  1.00 15.16           C  \\nATOM    661  CG  ASN A  85      18.464  26.946  22.904  1.00 16.42           C  \\nATOM    662  OD1 ASN A  85      17.355  26.878  22.362  1.00 16.01           O  \\nATOM    663  ND2 ASN A  85      18.808  28.030  23.587  1.00 16.83           N  \\nATOM    664  N   ASN A  86      16.616  23.515  23.509  1.00 14.72           N  \\nATOM    665  CA  ASN A  86      15.553  23.303  24.483  1.00 13.01           C  \\nATOM    666  C   ASN A  86      14.987  21.901  24.309  1.00 13.17           C  \\nATOM    667  O   ASN A  86      14.080  21.668  23.520  1.00 13.04           O  \\nATOM    668  CB  ASN A  86      14.478  24.367  24.256  1.00 16.51           C  \\nATOM    669  CG  ASN A  86      13.267  24.415  25.187  1.00 20.74           C  \\nATOM    670  OD1 ASN A  86      12.370  25.241  25.018  1.00 25.41           O  \\nATOM    671  ND2 ASN A  86      13.123  23.596  26.206  1.00 19.96           N  \\nATOM    672  N   THR A  87      15.488  20.950  25.103  1.00 13.80           N  \\nATOM    673  CA  THR A  87      15.095  19.553  25.026  1.00 15.40           C  \\nATOM    674  C   THR A  87      13.589  19.316  25.266  1.00 14.40           C  \\nATOM    675  O   THR A  87      12.953  18.535  24.546  1.00 14.11           O  \\nATOM    676  CB  THR A  87      15.983  18.791  26.046  1.00 16.19           C  \\nATOM    677  OG1 THR A  87      17.325  18.908  25.571  1.00 18.22           O  \\nATOM    678  CG2 THR A  87      15.626  17.327  26.180  1.00 17.82           C  \\nATOM    679  N   LYS A  88      12.983  19.958  26.272  1.00 14.42           N  \\nATOM    680  CA  LYS A  88      11.552  19.842  26.535  1.00 15.84           C  \\nATOM    681  C   LYS A  88      10.726  20.232  25.278  1.00 15.06           C  \\nATOM    682  O   LYS A  88       9.722  19.582  25.003  1.00 14.61           O  \\nATOM    683  CB  LYS A  88      11.207  20.766  27.717  1.00 19.53           C  \\nATOM    684  CG  LYS A  88       9.937  20.401  28.457  1.00 27.71           C  \\nATOM    685  CD  LYS A  88       9.585  21.452  29.529  1.00 33.26           C  \\nATOM    686  CE  LYS A  88       9.061  22.762  28.877  1.00 36.79           C  \\nATOM    687  NZ  LYS A  88       8.642  23.763  29.852  1.00 37.29           N  \\nATOM    688  N   SER A  89      11.119  21.259  24.475  1.00 12.36           N  \\nATOM    689  CA  SER A  89      10.325  21.666  23.309  1.00 11.31           C  \\nATOM    690  C   SER A  89      10.371  20.572  22.268  1.00 12.51           C  \\nATOM    691  O   SER A  89       9.397  20.388  21.547  1.00 12.88           O  \\nATOM    692  CB  SER A  89      10.844  22.951  22.674  1.00 11.64           C  \\nATOM    693  OG  SER A  89      12.160  22.900  22.124  1.00 11.11           O  \\nATOM    694  N   PHE A  90      11.504  19.842  22.204  1.00 11.27           N  \\nATOM    695  CA  PHE A  90      11.680  18.751  21.262  1.00 12.91           C  \\nATOM    696  C   PHE A  90      10.760  17.601  21.667  1.00 14.37           C  \\nATOM    697  O   PHE A  90      10.034  17.057  20.839  1.00 13.24           O  \\nATOM    698  CB  PHE A  90      13.161  18.312  21.260  1.00 12.34           C  \\nATOM    699  CG  PHE A  90      13.411  17.120  20.351  1.00 14.41           C  \\nATOM    700  CD1 PHE A  90      13.890  15.948  20.903  1.00 15.10           C  \\nATOM    701  CD2 PHE A  90      13.136  17.206  18.990  1.00 14.81           C  \\nATOM    702  CE1 PHE A  90      14.093  14.857  20.088  1.00 16.20           C  \\nATOM    703  CE2 PHE A  90      13.340  16.110  18.179  1.00 14.81           C  \\nATOM    704  CZ  PHE A  90      13.818  14.935  18.731  1.00 15.39           C  \\nATOM    705  N   GLU A  91      10.725  17.231  22.949  1.00 14.40           N  \\nATOM    706  CA  GLU A  91       9.863  16.146  23.407  1.00 15.78           C  \\nATOM    707  C   GLU A  91       8.374  16.466  23.301  1.00 16.18           C  \\nATOM    708  O   GLU A  91       7.533  15.582  23.114  1.00 17.50           O  \\nATOM    709  CB  GLU A  91      10.226  15.846  24.835  1.00 19.84           C  \\nATOM    710  CG  GLU A  91      11.654  15.386  25.074  1.00 24.49           C  \\nATOM    711  CD  GLU A  91      12.090  15.310  26.537  1.00 27.97           C  \\nATOM    712  OE1 GLU A  91      13.199  14.842  26.766  1.00 30.72           O  \\nATOM    713  OE2 GLU A  91      11.371  15.730  27.449  1.00 30.38           O  \\nATOM    714  N   ASP A  92       8.019  17.747  23.433  1.00 16.19           N  \\nATOM    715  CA  ASP A  92       6.650  18.216  23.265  1.00 15.23           C  \\nATOM    716  C   ASP A  92       6.141  18.053  21.839  1.00 14.29           C  \\nATOM    717  O   ASP A  92       4.927  18.074  21.633  1.00 14.78           O  \\nATOM    718  CB  ASP A  92       6.536  19.708  23.646  1.00 17.85           C  \\nATOM    719  CG  ASP A  92       6.706  20.080  25.127  1.00 21.39           C  \\nATOM    720  OD1 ASP A  92       6.648  19.209  25.997  1.00 20.06           O  \\nATOM    721  OD2 ASP A  92       6.887  21.272  25.401  1.00 24.39           O  \\nATOM    722  N   ILE A  93       7.013  17.910  20.826  1.00 11.76           N  \\nATOM    723  CA  ILE A  93       6.559  17.749  19.452  1.00 14.77           C  \\nATOM    724  C   ILE A  93       5.574  16.583  19.354  1.00 14.03           C  \\nATOM    725  O   ILE A  93       4.576  16.695  18.644  1.00 14.93           O  \\nATOM    726  CB  ILE A  93       7.790  17.499  18.485  1.00 14.96           C  \\nATOM    727  CG1 ILE A  93       8.506  18.830  18.293  1.00 15.41           C  \\nATOM    728  CG2 ILE A  93       7.369  16.936  17.091  1.00 14.61           C  \\nATOM    729  CD1 ILE A  93       7.654  19.940  17.637  1.00 20.05           C  \\nATOM    730  N   HIS A  94       5.806  15.471  20.049  1.00 15.23           N  \\nATOM    731  CA  HIS A  94       4.940  14.302  19.956  1.00 18.20           C  \\nATOM    732  C   HIS A  94       3.494  14.635  20.287  1.00 16.36           C  \\nATOM    733  O   HIS A  94       2.576  14.183  19.598  1.00 17.20           O  \\nATOM    734  CB  HIS A  94       5.436  13.194  20.900  1.00 23.57           C  \\nATOM    735  CG  HIS A  94       4.685  11.874  20.678  1.00 29.09           C  \\nATOM    736  ND1 HIS A  94       4.923  11.118  19.602  1.00 31.26           N  \\nATOM    737  CD2 HIS A  94       3.705  11.361  21.498  1.00 30.93           C  \\nATOM    738  CE1 HIS A  94       4.078  10.110  19.746  1.00 31.74           C  \\nATOM    739  NE2 HIS A  94       3.366  10.260  20.861  1.00 32.20           N  \\nATOM    740  N   GLN A  95       3.271  15.499  21.281  1.00 16.91           N  \\nATOM    741  CA  GLN A  95       1.916  15.878  21.673  1.00 18.66           C  \\nATOM    742  C   GLN A  95       1.226  16.736  20.625  1.00 16.97           C  \\nATOM    743  O   GLN A  95       0.012  16.562  20.443  1.00 15.36           O  \\nATOM    744  CB  GLN A  95       1.928  16.626  23.018  1.00 21.56           C  \\nATOM    745  CG  GLN A  95       2.457  15.766  24.180  1.00 29.81           C  \\nATOM    746  CD  GLN A  95       3.370  16.540  25.150  1.00 35.29           C  \\nATOM    747  OE1 GLN A  95       3.037  17.647  25.593  1.00 37.94           O  \\nATOM    748  NE2 GLN A  95       4.546  16.006  25.521  1.00 37.04           N  \\nATOM    749  N   TYR A  96       1.930  17.653  19.921  1.00 16.06           N  \\nATOM    750  CA  TYR A  96       1.276  18.417  18.826  1.00 16.07           C  \\nATOM    751  C   TYR A  96       0.963  17.504  17.658  1.00 14.44           C  \\nATOM    752  O   TYR A  96      -0.138  17.565  17.118  1.00 15.79           O  \\nATOM    753  CB  TYR A  96       2.139  19.554  18.264  1.00 17.94           C  \\nATOM    754  CG  TYR A  96       2.287  20.677  19.277  1.00 19.19           C  \\nATOM    755  CD1 TYR A  96       1.368  21.699  19.338  1.00 19.33           C  \\nATOM    756  CD2 TYR A  96       3.348  20.631  20.161  1.00 19.61           C  \\nATOM    757  CE1 TYR A  96       1.526  22.678  20.308  1.00 21.72           C  \\nATOM    758  CE2 TYR A  96       3.512  21.595  21.129  1.00 21.72           C  \\nATOM    759  CZ  TYR A  96       2.594  22.613  21.190  1.00 21.26           C  \\nATOM    760  OH  TYR A  96       2.776  23.582  22.153  1.00 24.09           O  \\nATOM    761  N   ARG A  97       1.904  16.635  17.295  1.00 13.30           N  \\nATOM    762  CA  ARG A  97       1.699  15.650  16.247  1.00 15.46           C  \\nATOM    763  C   ARG A  97       0.457  14.794  16.492  1.00 14.26           C  \\nATOM    764  O   ARG A  97      -0.420  14.676  15.641  1.00 14.10           O  \\nATOM    765  CB  ARG A  97       2.939  14.796  16.196  1.00 17.97           C  \\nATOM    766  CG  ARG A  97       2.886  13.541  15.357  1.00 23.35           C  \\nATOM    767  CD  ARG A  97       3.333  13.765  13.943  1.00 23.59           C  \\nATOM    768  NE  ARG A  97       3.314  12.471  13.314  1.00 24.08           N  \\nATOM    769  CZ  ARG A  97       3.268  12.320  12.007  1.00 25.74           C  \\nATOM    770  NH1 ARG A  97       3.135  11.073  11.548  1.00 26.76           N  \\nATOM    771  NH2 ARG A  97       3.353  13.380  11.181  1.00 26.95           N  \\nATOM    772  N   GLU A  98       0.332  14.218  17.678  1.00 13.69           N  \\nATOM    773  CA  GLU A  98      -0.807  13.361  17.941  1.00 16.64           C  \\nATOM    774  C   GLU A  98      -2.118  14.108  17.983  1.00 15.57           C  \\nATOM    775  O   GLU A  98      -3.157  13.573  17.607  1.00 15.85           O  \\nATOM    776  CB  GLU A  98      -0.593  12.623  19.260  1.00 21.96           C  \\nATOM    777  CG  GLU A  98       0.417  11.455  19.177  1.00 28.02           C  \\nATOM    778  CD  GLU A  98       0.147  10.460  18.038  1.00 31.04           C  \\nATOM    779  OE1 GLU A  98       1.118  10.123  17.373  1.00 33.58           O  \\nATOM    780  OE2 GLU A  98      -0.992  10.025  17.790  1.00 33.61           O  \\nATOM    781  N   GLN A  99      -2.105  15.362  18.446  1.00 14.66           N  \\nATOM    782  CA  GLN A  99      -3.343  16.104  18.468  1.00 13.22           C  \\nATOM    783  C   GLN A  99      -3.779  16.398  17.049  1.00 12.15           C  \\nATOM    784  O   GLN A  99      -4.974  16.274  16.791  1.00 14.09           O  \\nATOM    785  CB  GLN A  99      -3.158  17.403  19.250  1.00 16.29           C  \\nATOM    786  CG  GLN A  99      -4.513  18.118  19.407  1.00 19.67           C  \\nATOM    787  CD  GLN A  99      -4.517  19.240  20.438  1.00 22.19           C  \\nATOM    788  OE1 GLN A  99      -3.537  19.485  21.138  1.00 23.70           O  \\nATOM    789  NE2 GLN A  99      -5.628  19.961  20.537  1.00 22.72           N  \\nATOM    790  N   ILE A 100      -2.861  16.795  16.142  1.00 11.42           N  \\nATOM    791  CA  ILE A 100      -3.197  17.121  14.743  1.00 10.81           C  \\nATOM    792  C   ILE A 100      -3.761  15.893  14.008  1.00 12.39           C  \\nATOM    793  O   ILE A 100      -4.733  15.967  13.243  1.00 12.70           O  \\nATOM    794  CB  ILE A 100      -1.924  17.659  14.038  1.00 10.32           C  \\nATOM    795  CG1 ILE A 100      -1.642  19.075  14.518  1.00 10.37           C  \\nATOM    796  CG2 ILE A 100      -2.131  17.685  12.512  1.00 11.36           C  \\nATOM    797  CD1 ILE A 100      -0.184  19.481  14.392  1.00 10.69           C  \\nATOM    798  N   LYS A 101      -3.143  14.732  14.255  1.00 14.08           N  \\nATOM    799  CA  LYS A 101      -3.613  13.487  13.637  1.00 15.95           C  \\nATOM    800  C   LYS A 101      -5.021  13.150  14.079  1.00 15.72           C  \\nATOM    801  O   LYS A 101      -5.848  12.770  13.262  1.00 14.25           O  \\nATOM    802  CB  LYS A 101      -2.681  12.317  13.996  1.00 17.44           C  \\nATOM    803  CG  LYS A 101      -1.414  12.284  13.115  1.00 19.24           C  \\nATOM    804  CD  LYS A 101      -0.332  11.418  13.764  1.00 23.28           C  \\nATOM    805  CE  LYS A 101      -0.752  10.000  14.188  1.00 23.82           C  \\nATOM    806  NZ  LYS A 101       0.400   9.230  14.662  1.00 24.00           N  \\nATOM    807  N   ARG A 102      -5.278  13.341  15.377  1.00 16.72           N  \\nATOM    808  CA  ARG A 102      -6.586  13.098  15.962  1.00 18.74           C  \\nATOM    809  C   ARG A 102      -7.643  14.087  15.440  1.00 19.07           C  \\nATOM    810  O   ARG A 102      -8.712  13.628  15.002  1.00 20.67           O  \\nATOM    811  CB  ARG A 102      -6.457  13.197  17.462  1.00 20.62           C  \\nATOM    812  CG  ARG A 102      -7.539  12.409  18.132  1.00 26.89           C  \\nATOM    813  CD  ARG A 102      -7.610  12.887  19.585  1.00 30.60           C  \\nATOM    814  NE  ARG A 102      -8.197  14.224  19.645  1.00 35.83           N  \\nATOM    815  CZ  ARG A 102      -7.633  15.255  20.301  1.00 37.61           C  \\nATOM    816  NH1 ARG A 102      -8.243  16.455  20.267  1.00 38.37           N  \\nATOM    817  NH2 ARG A 102      -6.481  15.110  20.983  1.00 38.39           N  \\nATOM    818  N   VAL A 103      -7.418  15.407  15.395  1.00 15.59           N  \\nATOM    819  CA  VAL A 103      -8.452  16.326  14.881  1.00 18.48           C  \\nATOM    820  C   VAL A 103      -8.665  16.200  13.349  1.00 16.72           C  \\nATOM    821  O   VAL A 103      -9.776  16.383  12.848  1.00 15.90           O  \\nATOM    822  CB  VAL A 103      -8.131  17.840  15.215  1.00 20.10           C  \\nATOM    823  CG1 VAL A 103      -7.896  18.005  16.704  1.00 21.34           C  \\nATOM    824  CG2 VAL A 103      -6.899  18.302  14.512  1.00 23.93           C  \\nATOM    825  N   LYS A 104      -7.622  15.902  12.574  1.00 13.84           N  \\nATOM    826  CA  LYS A 104      -7.804  15.755  11.140  1.00 16.11           C  \\nATOM    827  C   LYS A 104      -8.158  14.329  10.716  1.00 16.70           C  \\nATOM    828  O   LYS A 104      -8.260  14.079   9.508  1.00 17.27           O  \\nATOM    829  CB  LYS A 104      -6.525  16.215  10.404  1.00 18.12           C  \\nATOM    830  CG  LYS A 104      -6.197  17.678  10.744  1.00 19.53           C  \\nATOM    831  CD  LYS A 104      -5.250  18.297   9.736  1.00 21.28           C  \\nATOM    832  CE  LYS A 104      -5.921  18.590   8.390  1.00 20.75           C  \\nATOM    833  NZ  LYS A 104      -6.875  19.673   8.462  1.00 19.79           N  \\nATOM    834  N   ASP A 105      -8.372  13.390  11.669  1.00 15.91           N  \\nATOM    835  CA  ASP A 105      -8.666  11.988  11.359  1.00 15.43           C  \\nATOM    836  C   ASP A 105      -7.705  11.372  10.333  1.00 16.04           C  \\nATOM    837  O   ASP A 105      -8.106  10.654   9.415  1.00 15.38           O  \\nATOM    838  CB  ASP A 105     -10.077  11.847  10.810  1.00 18.74           C  \\nATOM    839  CG  ASP A 105     -11.261  12.121  11.696  1.00 21.15           C  \\nATOM    840  OD1 ASP A 105     -11.183  11.972  12.918  1.00 21.36           O  \\nATOM    841  OD2 ASP A 105     -12.281  12.480  11.113  1.00 22.63           O  \\nATOM    842  N   SER A 106      -6.397  11.591  10.469  1.00 15.54           N  \\nATOM    843  CA  SER A 106      -5.470  11.174   9.442  1.00 17.25           C  \\nATOM    844  C   SER A 106      -4.112  10.882  10.011  1.00 17.31           C  \\nATOM    845  O   SER A 106      -3.650  11.556  10.916  1.00 19.11           O  \\nATOM    846  CB  SER A 106      -5.463  12.314   8.398  1.00 19.27           C  \\nATOM    847  OG  SER A 106      -4.326  12.490   7.573  1.00 24.20           O  \\nATOM    848  N   ASP A 107      -3.454   9.891   9.453  1.00 17.29           N  \\nATOM    849  CA  ASP A 107      -2.109   9.557   9.868  1.00 19.80           C  \\nATOM    850  C   ASP A 107      -1.000  10.220   9.063  1.00 20.36           C  \\nATOM    851  O   ASP A 107       0.197  10.118   9.345  1.00 21.26           O  \\nATOM    852  CB  ASP A 107      -1.970   8.045   9.810  1.00 23.15           C  \\nATOM    853  CG  ASP A 107      -2.863   7.310  10.807  1.00 25.12           C  \\nATOM    854  OD1 ASP A 107      -3.262   6.206  10.462  1.00 29.21           O  \\nATOM    855  OD2 ASP A 107      -3.127   7.812  11.903  1.00 26.59           O  \\nATOM    856  N   ASP A 108      -1.383  10.909   8.005  1.00 20.57           N  \\nATOM    857  CA  ASP A 108      -0.430  11.664   7.216  1.00 22.38           C  \\nATOM    858  C   ASP A 108      -1.061  12.982   6.842  1.00 21.37           C  \\nATOM    859  O   ASP A 108      -1.973  13.096   6.030  1.00 24.36           O  \\nATOM    860  CB  ASP A 108      -0.056  10.900   5.967  1.00 27.28           C  \\nATOM    861  CG  ASP A 108      -1.263  10.278   5.320  1.00 32.24           C  \\nATOM    862  OD1 ASP A 108      -1.714   9.272   5.877  1.00 37.07           O  \\nATOM    863  OD2 ASP A 108      -1.747  10.790   4.301  1.00 36.33           O  \\nATOM    864  N   VAL A 109      -0.598  13.970   7.572  1.00 19.33           N  \\nATOM    865  CA  VAL A 109      -1.009  15.354   7.400  1.00 17.49           C  \\nATOM    866  C   VAL A 109       0.215  16.086   6.819  1.00 14.55           C  \\nATOM    867  O   VAL A 109       1.335  15.772   7.245  1.00 13.64           O  \\nATOM    868  CB  VAL A 109      -1.410  15.870   8.784  1.00 18.40           C  \\nATOM    869  CG1 VAL A 109      -1.645  17.366   8.765  1.00 20.21           C  \\nATOM    870  CG2 VAL A 109      -2.676  15.101   9.214  1.00 20.27           C  \\nATOM    871  N   PRO A 110       0.141  17.001   5.825  1.00 12.41           N  \\nATOM    872  CA  PRO A 110       1.297  17.783   5.384  1.00 11.52           C  \\nATOM    873  C   PRO A 110       1.920  18.480   6.605  1.00 11.79           C  \\nATOM    874  O   PRO A 110       1.289  19.245   7.324  1.00  9.55           O  \\nATOM    875  CB  PRO A 110       0.728  18.721   4.370  1.00 11.31           C  \\nATOM    876  CG  PRO A 110      -0.410  17.926   3.779  1.00 11.38           C  \\nATOM    877  CD  PRO A 110      -1.044  17.309   5.020  1.00 12.05           C  \\nATOM    878  N   MET A 111       3.192  18.201   6.874  1.00 11.92           N  \\nATOM    879  CA  MET A 111       3.878  18.724   8.050  1.00 11.49           C  \\nATOM    880  C   MET A 111       5.379  18.749   7.807  1.00 11.68           C  \\nATOM    881  O   MET A 111       5.903  17.900   7.098  1.00 11.11           O  \\nATOM    882  CB  MET A 111       3.494  17.806   9.211  1.00 14.36           C  \\nATOM    883  CG  MET A 111       3.864  18.110  10.627  1.00 17.41           C  \\nATOM    884  SD  MET A 111       3.231  16.825  11.758  1.00 18.36           S  \\nATOM    885  CE  MET A 111       1.528  17.128  11.498  1.00 17.25           C  \\nATOM    886  N   VAL A 112       6.091  19.706   8.404  1.00 10.73           N  \\nATOM    887  CA  VAL A 112       7.545  19.763   8.348  1.00 10.18           C  \\nATOM    888  C   VAL A 112       7.992  19.875   9.794  1.00  9.11           C  \\nATOM    889  O   VAL A 112       7.404  20.633  10.556  1.00 10.77           O  \\nATOM    890  CB  VAL A 112       8.010  21.024   7.542  1.00 11.80           C  \\nATOM    891  CG1 VAL A 112       9.528  21.213   7.558  1.00 13.24           C  \\nATOM    892  CG2 VAL A 112       7.542  20.843   6.104  1.00 14.56           C  \\nATOM    893  N   LEU A 113       9.032  19.167  10.207  1.00  8.47           N  \\nATOM    894  CA  LEU A 113       9.632  19.332  11.535  1.00  9.53           C  \\nATOM    895  C   LEU A 113      10.738  20.384  11.412  1.00  9.33           C  \\nATOM    896  O   LEU A 113      11.621  20.270  10.555  1.00  9.95           O  \\nATOM    897  CB  LEU A 113      10.214  17.983  12.017  1.00  9.56           C  \\nATOM    898  CG  LEU A 113      10.863  17.976  13.381  1.00 11.58           C  \\nATOM    899  CD1 LEU A 113       9.908  18.379  14.503  1.00 13.17           C  \\nATOM    900  CD2 LEU A 113      11.361  16.564  13.597  1.00 12.84           C  \\nATOM    901  N   VAL A 114      10.705  21.442  12.221  1.00  7.62           N  \\nATOM    902  CA  VAL A 114      11.673  22.541  12.109  1.00  7.64           C  \\nATOM    903  C   VAL A 114      12.436  22.667  13.441  1.00  7.65           C  \\nATOM    904  O   VAL A 114      11.872  22.842  14.541  1.00  9.71           O  \\nATOM    905  CB  VAL A 114      10.896  23.836  11.774  1.00  9.37           C  \\nATOM    906  CG1 VAL A 114      11.814  25.030  11.869  1.00 10.27           C  \\nATOM    907  CG2 VAL A 114      10.254  23.719  10.393  1.00  9.64           C  \\nATOM    908  N   GLY A 115      13.745  22.567  13.331  1.00  6.10           N  \\nATOM    909  CA  GLY A 115      14.603  22.742  14.476  1.00  6.69           C  \\nATOM    910  C   GLY A 115      15.120  24.163  14.407  1.00  8.47           C  \\nATOM    911  O   GLY A 115      16.007  24.439  13.594  1.00  9.16           O  \\nATOM    912  N   ASN A 116      14.593  25.089  15.192  1.00  8.01           N  \\nATOM    913  CA  ASN A 116      15.007  26.475  15.111  1.00  8.24           C  \\nATOM    914  C   ASN A 116      16.112  26.873  16.109  1.00  9.61           C  \\nATOM    915  O   ASN A 116      16.453  26.142  17.051  1.00  9.84           O  \\nATOM    916  CB  ASN A 116      13.721  27.307  15.275  1.00  8.01           C  \\nATOM    917  CG  ASN A 116      13.929  28.788  14.992  1.00  9.48           C  \\nATOM    918  OD1 ASN A 116      14.470  29.205  13.972  1.00  9.94           O  \\nATOM    919  ND2 ASN A 116      13.565  29.644  15.940  1.00 10.75           N  \\nATOM    920  N   LYS A 117      16.660  28.087  15.876  1.00 10.22           N  \\nATOM    921  CA  LYS A 117      17.735  28.715  16.674  1.00 12.36           C  \\nATOM    922  C   LYS A 117      19.094  28.021  16.450  1.00 13.39           C  \\nATOM    923  O   LYS A 117      19.977  27.940  17.324  1.00 11.29           O  \\nATOM    924  CB  LYS A 117      17.391  28.706  18.214  1.00 13.23           C  \\nATOM    925  CG  LYS A 117      16.026  29.272  18.611  1.00 12.90           C  \\nATOM    926  CD  LYS A 117      15.932  29.640  20.097  1.00 12.43           C  \\nATOM    927  CE  LYS A 117      14.551  30.217  20.380  1.00 10.55           C  \\nATOM    928  NZ  LYS A 117      14.491  30.647  21.772  1.00 11.10           N  \\nATOM    929  N   CYS A 118      19.254  27.555  15.188  1.00 13.69           N  \\nATOM    930  CA  CYS A 118      20.449  26.806  14.844  1.00 15.67           C  \\nATOM    931  C   CYS A 118      21.716  27.658  14.796  1.00 15.28           C  \\nATOM    932  O   CYS A 118      22.808  27.113  14.629  1.00 15.30           O  \\nATOM    933  CB  CYS A 118      20.246  26.064  13.492  1.00 17.74           C  \\nATOM    934  SG  CYS A 118      20.353  27.009  11.944  1.00 20.74           S  \\nATOM    935  N   ASP A 119      21.578  28.995  14.931  1.00 15.76           N  \\nATOM    936  CA  ASP A 119      22.722  29.904  15.083  1.00 16.09           C  \\nATOM    937  C   ASP A 119      23.426  29.778  16.445  1.00 17.19           C  \\nATOM    938  O   ASP A 119      24.589  30.178  16.553  1.00 18.76           O  \\nATOM    939  CB  ASP A 119      22.268  31.374  14.888  1.00 15.14           C  \\nATOM    940  CG  ASP A 119      21.153  31.785  15.835  1.00 15.81           C  \\nATOM    941  OD1 ASP A 119      20.030  31.294  15.662  1.00 13.87           O  \\nATOM    942  OD2 ASP A 119      21.407  32.596  16.721  1.00 14.85           O  \\nATOM    943  N   LEU A 120      22.732  29.248  17.480  1.00 16.86           N  \\nATOM    944  CA  LEU A 120      23.230  29.099  18.845  1.00 18.76           C  \\nATOM    945  C   LEU A 120      24.217  27.957  18.979  1.00 19.70           C  \\nATOM    946  O   LEU A 120      23.979  26.885  18.431  1.00 20.53           O  \\nATOM    947  CB  LEU A 120      22.103  28.829  19.831  1.00 17.65           C  \\nATOM    948  CG  LEU A 120      21.065  29.933  20.010  1.00 19.43           C  \\nATOM    949  CD1 LEU A 120      20.044  29.418  21.008  1.00 18.46           C  \\nATOM    950  CD2 LEU A 120      21.688  31.239  20.503  1.00 18.73           C  \\nATOM    951  N   ALA A 121      25.313  28.184  19.705  1.00 19.26           N  \\nATOM    952  CA  ALA A 121      26.303  27.143  19.887  1.00 20.48           C  \\nATOM    953  C   ALA A 121      25.942  26.210  21.036  1.00 21.30           C  \\nATOM    954  O   ALA A 121      26.403  25.069  21.067  1.00 24.04           O  \\nATOM    955  CB  ALA A 121      27.652  27.767  20.164  1.00 22.02           C  \\nATOM    956  N   ALA A 122      25.109  26.635  21.982  1.00 22.09           N  \\nATOM    957  CA  ALA A 122      24.683  25.797  23.094  1.00 23.88           C  \\nATOM    958  C   ALA A 122      23.578  24.811  22.720  1.00 23.77           C  \\nATOM    959  O   ALA A 122      22.492  24.835  23.301  1.00 27.82           O  \\nATOM    960  CB  ALA A 122      24.171  26.684  24.249  1.00 25.67           C  \\nATOM    961  N   ARG A 123      23.843  23.908  21.783  1.00 20.99           N  \\nATOM    962  CA  ARG A 123      22.890  22.905  21.311  1.00 19.81           C  \\nATOM    963  C   ARG A 123      22.619  21.761  22.283  1.00 19.56           C  \\nATOM    964  O   ARG A 123      23.590  21.176  22.774  1.00 20.01           O  \\nATOM    965  CB  ARG A 123      23.421  22.333  19.988  1.00 19.81           C  \\nATOM    966  CG  ARG A 123      22.662  21.166  19.398  1.00 21.43           C  \\nATOM    967  CD  ARG A 123      23.273  20.680  18.073  1.00 22.66           C  \\nATOM    968  NE  ARG A 123      23.050  21.590  16.956  1.00 22.61           N  \\nATOM    969  CZ  ARG A 123      22.019  21.487  16.093  1.00 20.95           C  \\nATOM    970  NH1 ARG A 123      21.045  20.568  16.181  1.00 16.36           N  \\nATOM    971  NH2 ARG A 123      21.989  22.370  15.098  1.00 19.65           N  \\nATOM    972  N   THR A 124      21.364  21.389  22.600  1.00 17.96           N  \\nATOM    973  CA  THR A 124      21.118  20.200  23.415  1.00 17.85           C  \\nATOM    974  C   THR A 124      20.413  19.068  22.656  1.00 17.08           C  \\nATOM    975  O   THR A 124      20.459  17.926  23.091  1.00 19.59           O  \\nATOM    976  CB  THR A 124      20.311  20.578  24.672  1.00 19.42           C  \\nATOM    977  OG1 THR A 124      19.109  21.230  24.277  1.00 20.50           O  \\nATOM    978  CG2 THR A 124      21.155  21.458  25.588  1.00 19.06           C  \\nATOM    979  N   VAL A 125      19.788  19.319  21.495  1.00 16.59           N  \\nATOM    980  CA  VAL A 125      19.133  18.324  20.647  1.00 16.09           C  \\nATOM    981  C   VAL A 125      20.032  18.080  19.413  1.00 16.61           C  \\nATOM    982  O   VAL A 125      20.180  18.980  18.573  1.00 14.07           O  \\nATOM    983  CB  VAL A 125      17.720  18.848  20.179  1.00 15.28           C  \\nATOM    984  CG1 VAL A 125      17.097  17.814  19.265  1.00 14.81           C  \\nATOM    985  CG2 VAL A 125      16.824  19.168  21.366  1.00 15.97           C  \\nATOM    986  N   GLU A 126      20.659  16.889  19.280  1.00 17.05           N  \\nATOM    987  CA  GLU A 126      21.487  16.568  18.124  1.00 19.28           C  \\nATOM    988  C   GLU A 126      20.631  16.496  16.876  1.00 18.51           C  \\nATOM    989  O   GLU A 126      19.511  15.976  16.880  1.00 17.68           O  \\nATOM    990  CB  GLU A 126      22.165  15.215  18.219  1.00 23.29           C  \\nATOM    991  CG  GLU A 126      23.244  15.057  19.265  1.00 30.86           C  \\nATOM    992  CD  GLU A 126      24.402  16.054  19.153  1.00 36.59           C  \\nATOM    993  OE1 GLU A 126      25.139  16.012  18.161  1.00 39.64           O  \\nATOM    994  OE2 GLU A 126      24.581  16.885  20.056  1.00 40.44           O  \\nATOM    995  N   SER A 127      21.167  16.985  15.755  1.00 17.73           N  \\nATOM    996  CA  SER A 127      20.426  16.938  14.493  1.00 16.72           C  \\nATOM    997  C   SER A 127      20.040  15.506  14.158  1.00 16.76           C  \\nATOM    998  O   SER A 127      18.931  15.245  13.690  1.00 17.01           O  \\nATOM    999  CB  SER A 127      21.283  17.518  13.371  1.00 17.67           C  \\nATOM   1000  OG  SER A 127      21.730  18.826  13.768  1.00 19.67           O  \\nATOM   1001  N   ARG A 128      20.916  14.558  14.466  1.00 16.72           N  \\nATOM   1002  CA  ARG A 128      20.667  13.154  14.145  1.00 20.94           C  \\nATOM   1003  C   ARG A 128      19.435  12.617  14.862  1.00 19.62           C  \\nATOM   1004  O   ARG A 128      18.641  11.882  14.278  1.00 18.81           O  \\nATOM   1005  CB  ARG A 128      21.910  12.319  14.510  1.00 24.31           C  \\nATOM   1006  CG  ARG A 128      21.969  11.035  13.692  1.00 30.88           C  \\nATOM   1007  CD  ARG A 128      23.381  10.403  13.536  1.00 35.88           C  \\nATOM   1008  NE  ARG A 128      24.384  10.996  12.621  1.00 39.57           N  \\nATOM   1009  CZ  ARG A 128      24.313  11.018  11.267  1.00 40.85           C  \\nATOM   1010  NH1 ARG A 128      25.399  11.388  10.577  1.00 40.83           N  \\nATOM   1011  NH2 ARG A 128      23.202  10.685  10.576  1.00 41.33           N  \\nATOM   1012  N   GLN A 129      19.209  13.016  16.119  1.00 18.27           N  \\nATOM   1013  CA  GLN A 129      18.032  12.519  16.787  1.00 18.53           C  \\nATOM   1014  C   GLN A 129      16.735  13.151  16.224  1.00 17.11           C  \\nATOM   1015  O   GLN A 129      15.691  12.486  16.098  1.00 15.48           O  \\nATOM   1016  CB  GLN A 129      18.229  12.766  18.306  1.00 22.36           C  \\nATOM   1017  CG  GLN A 129      18.041  14.133  18.911  1.00 24.78           C  \\nATOM   1018  CD  GLN A 129      18.283  14.123  20.411  1.00 27.48           C  \\nATOM   1019  OE1 GLN A 129      19.287  14.643  20.906  1.00 28.14           O  \\nATOM   1020  NE2 GLN A 129      17.367  13.497  21.152  1.00 27.21           N  \\nATOM   1021  N   ALA A 130      16.780  14.446  15.837  1.00 14.73           N  \\nATOM   1022  CA  ALA A 130      15.611  15.044  15.210  1.00 13.87           C  \\nATOM   1023  C   ALA A 130      15.342  14.449  13.826  1.00 13.30           C  \\nATOM   1024  O   ALA A 130      14.199  14.194  13.436  1.00 11.42           O  \\nATOM   1025  CB  ALA A 130      15.822  16.536  15.063  1.00 16.21           C  \\nATOM   1026  N   GLN A 131      16.401  14.189  13.057  1.00 14.61           N  \\nATOM   1027  CA  GLN A 131      16.267  13.529  11.739  1.00 16.01           C  \\nATOM   1028  C   GLN A 131      15.637  12.109  11.821  1.00 16.29           C  \\nATOM   1029  O   GLN A 131      14.777  11.708  11.017  1.00 14.37           O  \\nATOM   1030  CB  GLN A 131      17.673  13.507  11.116  1.00 18.12           C  \\nATOM   1031  CG  GLN A 131      17.702  13.017   9.689  1.00 22.19           C  \\nATOM   1032  CD  GLN A 131      16.889  13.922   8.801  1.00 21.61           C  \\nATOM   1033  OE1 GLN A 131      17.292  15.044   8.529  1.00 24.06           O  \\nATOM   1034  NE2 GLN A 131      15.714  13.485   8.370  1.00 23.37           N  \\nATOM   1035  N   ASP A 132      16.038  11.348  12.853  1.00 16.64           N  \\nATOM   1036  CA  ASP A 132      15.467  10.038  13.173  1.00 18.00           C  \\nATOM   1037  C   ASP A 132      13.984  10.175  13.484  1.00 17.28           C  \\nATOM   1038  O   ASP A 132      13.178   9.395  12.985  1.00 16.45           O  \\nATOM   1039  CB  ASP A 132      16.120   9.415  14.405  1.00 20.10           C  \\nATOM   1040  CG  ASP A 132      17.506   8.816  14.234  1.00 24.07           C  \\nATOM   1041  OD1 ASP A 132      18.120   8.459  15.243  1.00 28.35           O  \\nATOM   1042  OD2 ASP A 132      17.975   8.686  13.106  1.00 25.43           O  \\nATOM   1043  N   LEU A 133      13.614  11.168  14.312  1.00 15.58           N  \\nATOM   1044  CA  LEU A 133      12.213  11.390  14.624  1.00 16.28           C  \\nATOM   1045  C   LEU A 133      11.421  11.729  13.366  1.00 14.96           C  \\nATOM   1046  O   LEU A 133      10.345  11.181  13.128  1.00 15.09           O  \\nATOM   1047  CB  LEU A 133      12.052  12.536  15.647  1.00 17.06           C  \\nATOM   1048  CG  LEU A 133      10.617  12.781  16.136  1.00 18.79           C  \\nATOM   1049  CD1 LEU A 133      10.142  11.554  16.881  1.00 19.96           C  \\nATOM   1050  CD2 LEU A 133      10.544  14.013  17.020  1.00 18.45           C  \\nATOM   1051  N   ALA A 134      11.937  12.642  12.545  1.00 13.16           N  \\nATOM   1052  CA  ALA A 134      11.241  13.016  11.331  1.00 14.89           C  \\nATOM   1053  C   ALA A 134      11.092  11.821  10.414  1.00 16.19           C  \\nATOM   1054  O   ALA A 134      10.015  11.643   9.857  1.00 15.70           O  \\nATOM   1055  CB  ALA A 134      12.021  14.089  10.629  1.00 15.12           C  \\nATOM   1056  N   ARG A 135      12.143  10.995  10.279  1.00 17.00           N  \\nATOM   1057  CA  ARG A 135      12.088   9.828   9.438  1.00 19.17           C  \\nATOM   1058  C   ARG A 135      10.990   8.893   9.916  1.00 18.70           C  \\nATOM   1059  O   ARG A 135      10.269   8.352   9.078  1.00 20.29           O  \\nATOM   1060  CB  ARG A 135      13.425   9.091   9.456  1.00 22.48           C  \\nATOM   1061  CG  ARG A 135      13.499   8.297   8.155  1.00 29.02           C  \\nATOM   1062  CD  ARG A 135      14.619   7.270   7.988  1.00 33.22           C  \\nATOM   1063  NE  ARG A 135      14.117   5.888   8.050  1.00 38.80           N  \\nATOM   1064  CZ  ARG A 135      13.281   5.302   7.148  1.00 39.60           C  \\nATOM   1065  NH1 ARG A 135      12.889   4.044   7.384  1.00 40.80           N  \\nATOM   1066  NH2 ARG A 135      12.827   5.913   6.034  1.00 40.11           N  \\nATOM   1067  N   SER A 136      10.815   8.697  11.242  1.00 19.23           N  \\nATOM   1068  CA  SER A 136       9.757   7.841  11.770  1.00 19.12           C  \\nATOM   1069  C   SER A 136       8.374   8.432  11.528  1.00 19.61           C  \\nATOM   1070  O   SER A 136       7.398   7.690  11.522  1.00 21.01           O  \\nATOM   1071  CB  SER A 136       9.930   7.624  13.280  1.00 19.94           C  \\nATOM   1072  OG  SER A 136       9.459   8.686  14.119  1.00 23.21           O  \\nATOM   1073  N   TYR A 137       8.231   9.760  11.380  1.00 17.33           N  \\nATOM   1074  CA  TYR A 137       6.950  10.375  11.078  1.00 18.12           C  \\nATOM   1075  C   TYR A 137       6.683  10.404   9.570  1.00 18.22           C  \\nATOM   1076  O   TYR A 137       5.561  10.656   9.129  1.00 19.27           O  \\nATOM   1077  CB  TYR A 137       6.907  11.812  11.592  1.00 18.21           C  \\nATOM   1078  CG  TYR A 137       6.883  12.057  13.084  1.00 19.08           C  \\nATOM   1079  CD1 TYR A 137       6.619  11.026  13.955  1.00 20.51           C  \\nATOM   1080  CD2 TYR A 137       7.059  13.355  13.548  1.00 20.43           C  \\nATOM   1081  CE1 TYR A 137       6.510  11.285  15.306  1.00 21.89           C  \\nATOM   1082  CE2 TYR A 137       6.945  13.624  14.899  1.00 21.02           C  \\nATOM   1083  CZ  TYR A 137       6.662  12.574  15.763  1.00 22.13           C  \\nATOM   1084  OH  TYR A 137       6.476  12.782  17.112  1.00 22.16           O  \\nATOM   1085  N   GLY A 138       7.689  10.186   8.737  1.00 16.07           N  \\nATOM   1086  CA  GLY A 138       7.520  10.265   7.294  1.00 17.66           C  \\nATOM   1087  C   GLY A 138       7.476  11.725   6.810  1.00 18.01           C  \\nATOM   1088  O   GLY A 138       6.859  12.010   5.774  1.00 17.85           O  \\nATOM   1089  N   ILE A 139       8.128  12.662   7.540  1.00 14.16           N  \\nATOM   1090  CA  ILE A 139       8.117  14.065   7.145  1.00 13.49           C  \\nATOM   1091  C   ILE A 139       9.526  14.662   7.033  1.00 14.35           C  \\nATOM   1092  O   ILE A 139      10.494  14.152   7.610  1.00 16.19           O  \\nATOM   1093  CB  ILE A 139       7.292  14.896   8.161  1.00 14.40           C  \\nATOM   1094  CG1 ILE A 139       7.925  14.811   9.554  1.00 15.44           C  \\nATOM   1095  CG2 ILE A 139       5.821  14.422   8.104  1.00 14.07           C  \\nATOM   1096  CD1 ILE A 139       7.288  15.765  10.587  1.00 15.35           C  \\nATOM   1097  N   PRO A 140       9.706  15.767   6.308  1.00 13.55           N  \\nATOM   1098  CA  PRO A 140      10.984  16.448   6.205  1.00 13.39           C  \\nATOM   1099  C   PRO A 140      11.424  17.065   7.532  1.00 12.74           C  \\nATOM   1100  O   PRO A 140      10.601  17.530   8.310  1.00 11.42           O  \\nATOM   1101  CB  PRO A 140      10.767  17.498   5.127  1.00 13.47           C  \\nATOM   1102  CG  PRO A 140       9.514  17.084   4.410  1.00 14.09           C  \\nATOM   1103  CD  PRO A 140       8.686  16.467   5.519  1.00 13.14           C  \\nATOM   1104  N   TYR A 141      12.744  17.086   7.756  1.00 11.68           N  \\nATOM   1105  CA  TYR A 141      13.336  17.791   8.890  1.00 10.89           C  \\nATOM   1106  C   TYR A 141      14.175  18.945   8.326  1.00 10.09           C  \\nATOM   1107  O   TYR A 141      15.056  18.720   7.468  1.00 10.27           O  \\nATOM   1108  CB  TYR A 141      14.276  16.897   9.691  1.00 12.54           C  \\nATOM   1109  CG  TYR A 141      15.063  17.676  10.733  1.00 10.92           C  \\nATOM   1110  CD1 TYR A 141      14.350  18.396  11.664  1.00 11.75           C  \\nATOM   1111  CD2 TYR A 141      16.453  17.633  10.784  1.00 12.49           C  \\nATOM   1112  CE1 TYR A 141      14.997  19.074  12.670  1.00 12.85           C  \\nATOM   1113  CE2 TYR A 141      17.112  18.307  11.794  1.00 11.33           C  \\nATOM   1114  CZ  TYR A 141      16.369  19.016  12.728  1.00 12.61           C  \\nATOM   1115  OH  TYR A 141      16.944  19.646  13.813  1.00 13.82           O  \\nATOM   1116  N   ILE A 142      13.892  20.174   8.799  1.00  9.35           N  \\nATOM   1117  CA  ILE A 142      14.636  21.355   8.376  1.00  9.05           C  \\nATOM   1118  C   ILE A 142      15.153  22.163   9.551  1.00  9.25           C  \\nATOM   1119  O   ILE A 142      14.409  22.382  10.499  1.00 10.43           O  \\nATOM   1120  CB  ILE A 142      13.756  22.259   7.472  1.00  9.25           C  \\nATOM   1121  CG1 ILE A 142      13.523  21.470   6.180  1.00 10.51           C  \\nATOM   1122  CG2 ILE A 142      14.450  23.586   7.122  1.00  9.08           C  \\nATOM   1123  CD1 ILE A 142      12.719  22.127   5.028  1.00 13.16           C  \\nATOM   1124  N   GLU A 143      16.457  22.513   9.568  1.00  9.01           N  \\nATOM   1125  CA  GLU A 143      16.979  23.404  10.611  1.00  8.52           C  \\nATOM   1126  C   GLU A 143      16.978  24.864  10.193  1.00 10.31           C  \\nATOM   1127  O   GLU A 143      17.278  25.228   9.040  1.00 10.45           O  \\nATOM   1128  CB  GLU A 143      18.397  23.022  11.013  1.00 10.87           C  \\nATOM   1129  CG  GLU A 143      18.363  21.728  11.801  1.00 10.22           C  \\nATOM   1130  CD  GLU A 143      19.592  21.411  12.633  1.00 13.01           C  \\nATOM   1131  OE1 GLU A 143      20.616  22.082  12.521  1.00 13.65           O  \\nATOM   1132  OE2 GLU A 143      19.511  20.487  13.429  1.00 12.70           O  \\nATOM   1133  N   THR A 144      16.608  25.762  11.117  1.00  9.18           N  \\nATOM   1134  CA  THR A 144      16.422  27.169  10.754  1.00  9.71           C  \\nATOM   1135  C   THR A 144      17.014  28.076  11.789  1.00  8.66           C  \\nATOM   1136  O   THR A 144      17.259  27.698  12.927  1.00  9.13           O  \\nATOM   1137  CB  THR A 144      14.919  27.617  10.635  1.00  9.90           C  \\nATOM   1138  OG1 THR A 144      14.336  27.403  11.931  1.00  9.29           O  \\nATOM   1139  CG2 THR A 144      14.120  26.858   9.557  1.00  9.89           C  \\nATOM   1140  N   SER A 145      17.244  29.288  11.360  1.00 10.91           N  \\nATOM   1141  CA  SER A 145      17.558  30.377  12.290  1.00 12.65           C  \\nATOM   1142  C   SER A 145      16.758  31.602  11.849  1.00 10.37           C  \\nATOM   1143  O   SER A 145      16.930  32.094  10.727  1.00 11.76           O  \\nATOM   1144  CB  SER A 145      19.069  30.768  12.308  1.00 11.79           C  \\nATOM   1145  OG  SER A 145      19.248  31.931  13.125  1.00 12.12           O  \\nATOM   1146  N   ALA A 146      15.838  32.046  12.696  1.00  9.49           N  \\nATOM   1147  CA  ALA A 146      15.131  33.268  12.413  1.00 10.90           C  \\nATOM   1148  C   ALA A 146      16.070  34.455  12.553  1.00 13.53           C  \\nATOM   1149  O   ALA A 146      15.799  35.514  11.995  1.00 16.20           O  \\nATOM   1150  CB  ALA A 146      13.976  33.436  13.367  1.00 12.65           C  \\nATOM   1151  N   LYS A 147      17.183  34.302  13.283  1.00 14.35           N  \\nATOM   1152  CA  LYS A 147      18.150  35.351  13.486  1.00 16.25           C  \\nATOM   1153  C   LYS A 147      18.953  35.568  12.215  1.00 17.58           C  \\nATOM   1154  O   LYS A 147      19.124  36.724  11.825  1.00 20.13           O  \\nATOM   1155  CB  LYS A 147      19.068  34.970  14.617  1.00 17.55           C  \\nATOM   1156  CG  LYS A 147      20.003  36.125  14.958  1.00 22.83           C  \\nATOM   1157  CD  LYS A 147      20.581  35.932  16.341  1.00 26.44           C  \\nATOM   1158  CE  LYS A 147      21.683  36.949  16.507  1.00 30.79           C  \\nATOM   1159  NZ  LYS A 147      22.575  36.534  17.574  1.00 33.81           N  \\nATOM   1160  N   THR A 148      19.479  34.529  11.541  1.00 17.16           N  \\nATOM   1161  CA  THR A 148      20.248  34.735  10.300  1.00 15.93           C  \\nATOM   1162  C   THR A 148      19.422  34.607   9.013  1.00 17.72           C  \\nATOM   1163  O   THR A 148      19.926  34.962   7.945  1.00 18.47           O  \\nATOM   1164  CB  THR A 148      21.444  33.745  10.209  1.00 15.86           C  \\nATOM   1165  OG1 THR A 148      20.886  32.435  10.042  1.00 14.56           O  \\nATOM   1166  CG2 THR A 148      22.377  33.839  11.439  1.00 15.25           C  \\nATOM   1167  N   ARG A 149      18.176  34.108   9.100  1.00 15.18           N  \\nATOM   1168  CA  ARG A 149      17.268  33.815   7.994  1.00 19.20           C  \\nATOM   1169  C   ARG A 149      17.652  32.483   7.318  1.00 17.94           C  \\nATOM   1170  O   ARG A 149      17.031  32.079   6.332  1.00 20.28           O  \\nATOM   1171  CB  ARG A 149      17.251  34.964   6.937  1.00 21.40           C  \\nATOM   1172  CG  ARG A 149      15.929  35.106   6.188  1.00 27.00           C  \\nATOM   1173  CD  ARG A 149      15.946  35.904   4.863  1.00 27.99           C  \\nATOM   1174  NE  ARG A 149      14.707  35.687   4.092  1.00 29.40           N  \\nATOM   1175  CZ  ARG A 149      13.550  36.232   4.492  1.00 29.07           C  \\nATOM   1176  NH1 ARG A 149      12.412  36.015   3.828  1.00 29.56           N  \\nATOM   1177  NH2 ARG A 149      13.527  37.004   5.577  1.00 28.99           N  \\nATOM   1178  N   GLN A 150      18.621  31.701   7.838  1.00 15.80           N  \\nATOM   1179  CA  GLN A 150      18.926  30.402   7.273  1.00 16.36           C  \\nATOM   1180  C   GLN A 150      17.737  29.464   7.455  1.00 15.02           C  \\nATOM   1181  O   GLN A 150      17.119  29.351   8.504  1.00 13.57           O  \\nATOM   1182  CB  GLN A 150      20.152  29.729   7.937  1.00 23.05           C  \\nATOM   1183  CG  GLN A 150      20.578  28.448   7.162  1.00 29.37           C  \\nATOM   1184  CD  GLN A 150      21.604  27.530   7.827  1.00 34.84           C  \\nATOM   1185  OE1 GLN A 150      21.301  26.416   8.280  1.00 38.95           O  \\nATOM   1186  NE2 GLN A 150      22.857  27.967   7.883  1.00 35.22           N  \\nATOM   1187  N   GLY A 151      17.412  28.809   6.348  1.00 13.62           N  \\nATOM   1188  CA  GLY A 151      16.357  27.835   6.296  1.00 13.50           C  \\nATOM   1189  C   GLY A 151      14.943  28.395   6.329  1.00 12.36           C  \\nATOM   1190  O   GLY A 151      14.025  27.591   6.148  1.00 11.07           O  \\nATOM   1191  N   VAL A 152      14.706  29.702   6.510  1.00 12.59           N  \\nATOM   1192  CA  VAL A 152      13.339  30.183   6.715  1.00 14.23           C  \\nATOM   1193  C   VAL A 152      12.435  29.971   5.475  1.00 13.41           C  \\nATOM   1194  O   VAL A 152      11.352  29.362   5.577  1.00 12.94           O  \\nATOM   1195  CB  VAL A 152      13.430  31.691   7.167  1.00 12.57           C  \\nATOM   1196  CG1 VAL A 152      12.059  32.316   7.194  1.00 13.14           C  \\nATOM   1197  CG2 VAL A 152      14.044  31.787   8.543  1.00 12.69           C  \\nATOM   1198  N   GLU A 153      12.831  30.410   4.278  1.00 13.52           N  \\nATOM   1199  CA  GLU A 153      12.001  30.213   3.086  1.00 14.81           C  \\nATOM   1200  C   GLU A 153      11.857  28.756   2.742  1.00 11.88           C  \\nATOM   1201  O   GLU A 153      10.797  28.282   2.344  1.00 13.92           O  \\nATOM   1202  CB  GLU A 153      12.592  30.936   1.881  1.00 17.03           C  \\nATOM   1203  CG  GLU A 153      12.454  32.452   2.015  1.00 21.18           C  \\nATOM   1204  CD  GLU A 153      12.870  33.318   0.817  1.00 25.73           C  \\nATOM   1205  OE1 GLU A 153      13.232  32.781  -0.234  1.00 28.32           O  \\nATOM   1206  OE2 GLU A 153      12.822  34.550   0.936  1.00 26.64           O  \\nATOM   1207  N   ASP A 154      12.959  28.031   2.957  1.00 11.31           N  \\nATOM   1208  CA  ASP A 154      13.023  26.593   2.747  1.00 10.16           C  \\nATOM   1209  C   ASP A 154      11.964  25.828   3.561  1.00 10.90           C  \\nATOM   1210  O   ASP A 154      11.224  24.985   3.035  1.00 11.83           O  \\nATOM   1211  CB  ASP A 154      14.420  26.212   3.117  1.00 15.00           C  \\nATOM   1212  CG  ASP A 154      14.919  24.942   2.524  1.00 19.04           C  \\nATOM   1213  OD1 ASP A 154      14.121  24.195   1.938  1.00 22.96           O  \\nATOM   1214  OD2 ASP A 154      16.122  24.697   2.676  1.00 21.26           O  \\nATOM   1215  N   ALA A 155      11.854  26.107   4.881  1.00  8.83           N  \\nATOM   1216  CA  ALA A 155      10.841  25.450   5.716  1.00 10.00           C  \\nATOM   1217  C   ALA A 155       9.424  25.748   5.203  1.00 10.10           C  \\nATOM   1218  O   ALA A 155       8.653  24.814   4.978  1.00 11.14           O  \\nATOM   1219  CB  ALA A 155      10.923  25.933   7.192  1.00  9.50           C  \\nATOM   1220  N   PHE A 156       9.054  27.011   4.949  1.00 10.59           N  \\nATOM   1221  CA  PHE A 156       7.712  27.345   4.475  1.00 10.51           C  \\nATOM   1222  C   PHE A 156       7.453  26.825   3.069  1.00 11.05           C  \\nATOM   1223  O   PHE A 156       6.368  26.302   2.801  1.00 10.80           O  \\nATOM   1224  CB  PHE A 156       7.492  28.890   4.506  1.00 11.54           C  \\nATOM   1225  CG  PHE A 156       7.266  29.450   5.936  1.00 12.42           C  \\nATOM   1226  CD1 PHE A 156       8.319  29.958   6.707  1.00 12.00           C  \\nATOM   1227  CD2 PHE A 156       5.978  29.434   6.478  1.00 12.57           C  \\nATOM   1228  CE1 PHE A 156       8.065  30.434   8.001  1.00 12.45           C  \\nATOM   1229  CE2 PHE A 156       5.747  29.908   7.757  1.00 12.32           C  \\nATOM   1230  CZ  PHE A 156       6.782  30.406   8.513  1.00 10.18           C  \\nATOM   1231  N   TYR A 157       8.408  26.935   2.151  1.00  9.69           N  \\nATOM   1232  CA  TYR A 157       8.148  26.463   0.798  1.00 12.31           C  \\nATOM   1233  C   TYR A 157       8.091  24.945   0.719  1.00 10.42           C  \\nATOM   1234  O   TYR A 157       7.292  24.403  -0.047  1.00 11.35           O  \\nATOM   1235  CB  TYR A 157       9.233  26.978  -0.156  1.00 15.16           C  \\nATOM   1236  CG  TYR A 157       9.257  28.492  -0.367  1.00 18.73           C  \\nATOM   1237  CD1 TYR A 157       8.380  29.366   0.268  1.00 18.52           C  \\nATOM   1238  CD2 TYR A 157      10.211  28.971  -1.242  1.00 20.44           C  \\nATOM   1239  CE1 TYR A 157       8.466  30.724   0.025  1.00 22.04           C  \\nATOM   1240  CE2 TYR A 157      10.305  30.320  -1.488  1.00 22.70           C  \\nATOM   1241  CZ  TYR A 157       9.436  31.178  -0.852  1.00 23.03           C  \\nATOM   1242  OH  TYR A 157       9.572  32.528  -1.089  1.00 26.15           O  \\nATOM   1243  N   THR A 158       8.940  24.198   1.455  1.00 11.35           N  \\nATOM   1244  CA  THR A 158       8.836  22.727   1.522  1.00 10.11           C  \\nATOM   1245  C   THR A 158       7.440  22.347   2.064  1.00 11.11           C  \\nATOM   1246  O   THR A 158       6.811  21.408   1.558  1.00 11.43           O  \\nATOM   1247  CB  THR A 158       9.935  22.129   2.470  1.00 12.09           C  \\nATOM   1248  OG1 THR A 158      11.216  22.451   1.933  1.00 11.36           O  \\nATOM   1249  CG2 THR A 158       9.791  20.657   2.624  1.00 10.87           C  \\nATOM   1250  N   LEU A 159       6.888  23.056   3.072  1.00  9.25           N  \\nATOM   1251  CA  LEU A 159       5.533  22.754   3.506  1.00  9.93           C  \\nATOM   1252  C   LEU A 159       4.489  22.986   2.388  1.00 10.42           C  \\nATOM   1253  O   LEU A 159       3.560  22.185   2.272  1.00 11.22           O  \\nATOM   1254  CB  LEU A 159       5.171  23.608   4.711  1.00  9.03           C  \\nATOM   1255  CG  LEU A 159       3.789  23.318   5.269  1.00 10.96           C  \\nATOM   1256  CD1 LEU A 159       3.635  21.828   5.651  1.00  9.29           C  \\nATOM   1257  CD2 LEU A 159       3.580  24.223   6.469  1.00 13.71           C  \\nATOM   1258  N   VAL A 160       4.596  24.046   1.548  1.00 10.91           N  \\nATOM   1259  CA  VAL A 160       3.662  24.284   0.439  1.00  9.19           C  \\nATOM   1260  C   VAL A 160       3.803  23.137  -0.556  1.00  9.60           C  \\nATOM   1261  O   VAL A 160       2.781  22.638  -1.031  1.00 10.29           O  \\nATOM   1262  CB  VAL A 160       3.981  25.653  -0.211  1.00  8.50           C  \\nATOM   1263  CG1 VAL A 160       3.402  25.754  -1.622  1.00 10.56           C  \\nATOM   1264  CG2 VAL A 160       3.403  26.741   0.659  1.00  7.98           C  \\nATOM   1265  N   ARG A 161       5.033  22.667  -0.839  1.00  9.10           N  \\nATOM   1266  CA  ARG A 161       5.206  21.554  -1.767  1.00  9.82           C  \\nATOM   1267  C   ARG A 161       4.644  20.240  -1.215  1.00 10.44           C  \\nATOM   1268  O   ARG A 161       4.166  19.395  -1.984  1.00 12.88           O  \\nATOM   1269  CB  ARG A 161       6.679  21.398  -2.115  1.00  9.39           C  \\nATOM   1270  CG  ARG A 161       7.146  22.640  -2.816  1.00  9.70           C  \\nATOM   1271  CD  ARG A 161       8.594  22.398  -3.227  1.00 14.12           C  \\nATOM   1272  NE  ARG A 161       9.122  23.635  -3.780  1.00 14.15           N  \\nATOM   1273  CZ  ARG A 161      10.144  24.313  -3.257  1.00 12.45           C  \\nATOM   1274  NH1 ARG A 161      10.823  23.858  -2.204  1.00 13.24           N  \\nATOM   1275  NH2 ARG A 161      10.485  25.475  -3.819  1.00 13.52           N  \\nATOM   1276  N   GLU A 162       4.663  20.046   0.112  1.00 10.71           N  \\nATOM   1277  CA  GLU A 162       4.054  18.877   0.722  1.00 11.80           C  \\nATOM   1278  C   GLU A 162       2.522  18.887   0.568  1.00 11.20           C  \\nATOM   1279  O   GLU A 162       1.921  17.848   0.274  1.00 12.90           O  \\nATOM   1280  CB  GLU A 162       4.430  18.815   2.216  1.00 13.86           C  \\nATOM   1281  CG  GLU A 162       5.870  18.460   2.580  1.00 14.10           C  \\nATOM   1282  CD  GLU A 162       6.268  17.038   2.224  1.00 17.23           C  \\nATOM   1283  OE1 GLU A 162       5.571  16.116   2.627  1.00 19.29           O  \\nATOM   1284  OE2 GLU A 162       7.278  16.827   1.545  1.00 22.14           O  \\nATOM   1285  N   ILE A 163       1.869  20.045   0.731  1.00 10.13           N  \\nATOM   1286  CA  ILE A 163       0.426  20.175   0.542  1.00 10.58           C  \\nATOM   1287  C   ILE A 163       0.128  19.878  -0.933  1.00 13.43           C  \\nATOM   1288  O   ILE A 163      -0.796  19.118  -1.235  1.00 15.01           O  \\nATOM   1289  CB  ILE A 163      -0.015  21.590   0.891  1.00 11.33           C  \\nATOM   1290  CG1 ILE A 163       0.084  21.777   2.400  1.00 11.27           C  \\nATOM   1291  CG2 ILE A 163      -1.486  21.815   0.437  1.00 12.37           C  \\nATOM   1292  CD1 ILE A 163      -0.006  23.245   2.894  1.00 12.56           C  \\nATOM   1293  N   ARG A 164       0.904  20.456  -1.871  1.00 11.72           N  \\nATOM   1294  CA  ARG A 164       0.770  20.163  -3.283  1.00 13.64           C  \\nATOM   1295  C   ARG A 164       0.813  18.657  -3.626  1.00 17.97           C  \\nATOM   1296  O   ARG A 164       0.150  18.197  -4.559  1.00 18.12           O  \\nATOM   1297  CB  ARG A 164       1.897  20.887  -4.029  1.00 12.51           C  \\nATOM   1298  CG  ARG A 164       1.647  22.371  -4.120  1.00 12.49           C  \\nATOM   1299  CD  ARG A 164       2.801  22.888  -4.950  1.00 14.72           C  \\nATOM   1300  NE  ARG A 164       2.730  24.327  -5.113  1.00 12.82           N  \\nATOM   1301  CZ  ARG A 164       3.800  25.071  -5.405  1.00 13.29           C  \\nATOM   1302  NH1 ARG A 164       5.018  24.539  -5.527  1.00 11.56           N  \\nATOM   1303  NH2 ARG A 164       3.595  26.381  -5.572  1.00 12.55           N  \\nATOM   1304  N   GLN A 165       1.590  17.831  -2.935  1.00 20.23           N  \\nATOM   1305  CA  GLN A 165       1.673  16.435  -3.337  1.00 27.50           C  \\nATOM   1306  C   GLN A 165       0.782  15.499  -2.502  1.00 29.12           C  \\nATOM   1307  O   GLN A 165       0.826  14.274  -2.655  1.00 29.30           O  \\nATOM   1308  CB  GLN A 165       3.168  16.070  -3.286  1.00 30.34           C  \\nATOM   1309  CG  GLN A 165       4.018  16.904  -4.309  1.00 35.17           C  \\nATOM   1310  CD  GLN A 165       4.109  16.470  -5.796  1.00 37.71           C  \\nATOM   1311  OE1 GLN A 165       4.256  17.293  -6.719  1.00 36.70           O  \\nATOM   1312  NE2 GLN A 165       4.079  15.166  -6.094  1.00 38.13           N  \\nATOM   1313  N   HIS A 166      -0.068  16.075  -1.638  1.00 29.28           N  \\nATOM   1314  CA  HIS A 166      -0.982  15.296  -0.820  1.00 31.13           C  \\nATOM   1315  C   HIS A 166      -2.214  14.971  -1.665  1.00 33.30           C  \\nATOM   1316  O   HIS A 166      -2.630  15.786  -2.497  1.00 34.96           O  \\nATOM   1317  CB  HIS A 166      -1.420  16.090   0.428  1.00 30.89           C  \\nATOM   1318  CG  HIS A 166      -2.247  15.319   1.475  1.00 32.17           C  \\nATOM   1319  ND1 HIS A 166      -3.579  15.231   1.603  1.00 33.52           N  \\nATOM   1320  CD2 HIS A 166      -1.712  14.560   2.502  1.00 32.95           C  \\nATOM   1321  CE1 HIS A 166      -3.857  14.464   2.647  1.00 33.07           C  \\nATOM   1322  NE2 HIS A 166      -2.727  14.064   3.184  1.00 33.06           N  \\nATOM   1323  OXT HIS A 166      -2.763  13.888  -1.454  1.00 36.27           O  \\nTER    1324      HIS A 166                                                      \\nHETATM 1325 MG    MG A 168       4.771  33.860  19.213  1.00  9.42          MG  \\nHETATM 1326  PG  GNP A 167       5.113  32.223  21.994  1.00  9.66           P  \\nHETATM 1327  O1G GNP A 167       4.844  32.596  23.390  1.00  9.26           O  \\nHETATM 1328  O2G GNP A 167       4.203  32.910  21.112  1.00 11.20           O  \\nHETATM 1329  O3G GNP A 167       4.640  30.769  21.802  1.00 10.09           O  \\nHETATM 1330  N3B GNP A 167       6.701  32.303  21.621  1.00  8.18           N  \\nHETATM 1331  PB  GNP A 167       7.310  32.034  20.111  1.00  8.77           P  \\nHETATM 1332  O1B GNP A 167       7.468  30.605  19.791  1.00  8.35           O  \\nHETATM 1333  O2B GNP A 167       6.696  32.775  19.021  1.00  9.38           O  \\nHETATM 1334  O3A GNP A 167       8.792  32.643  20.152  1.00 10.15           O  \\nHETATM 1335  PA  GNP A 167       9.280  34.140  19.710  1.00 10.18           P  \\nHETATM 1336  O1A GNP A 167       9.263  34.268  18.220  1.00 10.07           O  \\nHETATM 1337  O2A GNP A 167       8.594  35.173  20.510  1.00  9.33           O  \\nHETATM 1338  O5' GNP A 167      10.826  34.056  20.132  1.00 11.46           O  \\nHETATM 1339  C5' GNP A 167      11.170  33.911  21.535  1.00 10.56           C  \\nHETATM 1340  C4' GNP A 167      12.587  34.445  21.811  1.00 11.96           C  \\nHETATM 1341  O4' GNP A 167      13.560  33.649  21.095  1.00 11.66           O  \\nHETATM 1342  C3' GNP A 167      12.813  35.910  21.359  1.00 12.06           C  \\nHETATM 1343  O3' GNP A 167      13.793  36.538  22.221  1.00 14.52           O  \\nHETATM 1344  C2' GNP A 167      13.325  35.679  19.947  1.00 11.90           C  \\nHETATM 1345  O2' GNP A 167      14.151  36.750  19.499  1.00 14.99           O  \\nHETATM 1346  C1' GNP A 167      14.242  34.478  20.156  1.00 12.90           C  \\nHETATM 1347  N9  GNP A 167      14.454  33.739  18.896  1.00 12.98           N  \\nHETATM 1348  C8  GNP A 167      13.518  33.262  17.993  1.00 11.76           C  \\nHETATM 1349  N7  GNP A 167      14.016  32.636  16.979  1.00 12.34           N  \\nHETATM 1350  C5  GNP A 167      15.413  32.694  17.200  1.00 12.36           C  \\nHETATM 1351  C6  GNP A 167      16.521  32.219  16.402  1.00 12.30           C  \\nHETATM 1352  O6  GNP A 167      16.507  31.591  15.338  1.00 12.16           O  \\nHETATM 1353  N1  GNP A 167      17.735  32.503  16.995  1.00 11.60           N  \\nHETATM 1354  C2  GNP A 167      17.909  33.082  18.236  1.00 13.91           C  \\nHETATM 1355  N2  GNP A 167      19.173  33.280  18.658  1.00 14.60           N  \\nHETATM 1356  N3  GNP A 167      16.884  33.498  18.996  1.00 13.01           N  \\nHETATM 1357  C4  GNP A 167      15.675  33.331  18.398  1.00 14.36           C  \\nHETATM 1358  O   HOH A 169      13.588  43.386  17.779  1.00 31.80           O  \\nHETATM 1359  O   HOH A 170       8.248  36.064  23.209  1.00 13.63           O  \\nHETATM 1360  O   HOH A 171       3.254  37.845  19.454  1.00 18.29           O  \\nHETATM 1361  O   HOH A 172       5.945  35.619  20.105  1.00 11.33           O  \\nHETATM 1362  O   HOH A 173       3.669  32.226  18.172  1.00  8.22           O  \\nHETATM 1363  O   HOH A 174       2.896  37.170  15.500  1.00 21.37           O  \\nHETATM 1364  O   HOH A 175       2.102  32.309  24.122  1.00 27.47           O  \\nHETATM 1365  O   HOH A 176      -1.425  37.650  19.667  1.00 24.98           O  \\nHETATM 1366  O   HOH A 177      -3.225  30.031  -8.217  1.00 23.57           O  \\nHETATM 1367  O   HOH A 178      16.678  26.321  26.916  1.00 36.70           O  \\nHETATM 1368  O   HOH A 179      -2.134  40.975   1.551  1.00 81.53           O  \\nHETATM 1369  O   HOH A 180       0.400  40.675  -0.629  1.00 24.66           O  \\nHETATM 1370  O   HOH A 181      10.944  44.793  16.009  1.00 48.47           O  \\nHETATM 1371  O   HOH A 182      17.207  42.570  16.786  1.00 43.23           O  \\nHETATM 1372  O   HOH A 183      14.999  45.167  19.922  1.00 39.52           O  \\nHETATM 1373  O   HOH A 184      11.598  37.755  13.202  1.00 11.72           O  \\nHETATM 1374  O   HOH A 185       6.958  38.922  18.146  1.00 26.17           O  \\nHETATM 1375  O   HOH A 186      14.078  32.033  27.772  1.00 57.73           O  \\nHETATM 1376  O   HOH A 187       9.126  37.906  19.595  1.00 32.42           O  \\nHETATM 1377  O   HOH A 188      -0.049  36.435  27.259  1.00 50.56           O  \\nHETATM 1378  O   HOH A 189       3.843  30.503  25.485  1.00 46.80           O  \\nHETATM 1379  O   HOH A 190       4.931  28.343  27.587  1.00 29.32           O  \\nHETATM 1380  O   HOH A 191       4.881  25.774  26.247  1.00 28.55           O  \\nHETATM 1381  O   HOH A 193       8.729  26.516  26.645  1.00 37.34           O  \\nHETATM 1382  O   HOH A 194      -5.515  23.731  -4.213  1.00 43.74           O  \\nHETATM 1383  O   HOH A 195      -7.006  26.635   0.729  1.00 25.77           O  \\nHETATM 1384  O   HOH A 196      -2.033  23.281  -4.754  1.00 36.89           O  \\nHETATM 1385  O   HOH A 197      -4.326  24.642  -0.854  1.00 28.59           O  \\nHETATM 1386  O   HOH A 198      -6.858  26.995  -2.808  1.00 28.83           O  \\nHETATM 1387  O   HOH A 200      -3.937  29.225   8.289  1.00 24.08           O  \\nHETATM 1388  O   HOH A 201      -8.934  24.037   2.795  1.00 34.16           O  \\nHETATM 1389  O   HOH A 202       1.444  26.302  19.318  1.00 12.37           O  \\nHETATM 1390  O   HOH A 203       2.031  24.632  27.307  1.00 50.09           O  \\nHETATM 1391  O   HOH A 204      15.125  28.341  23.174  1.00 13.88           O  \\nHETATM 1392  O   HOH A 205      12.808  31.541  25.056  1.00 46.88           O  \\nHETATM 1393  O   HOH A 206       8.329  34.933   0.543  1.00 53.21           O  \\nHETATM 1394  O   HOH A 207       0.760  42.969   2.336  1.00 35.42           O  \\nHETATM 1395  O   HOH A 208       1.270  40.298   7.071  1.00 76.61           O  \\nHETATM 1396  O   HOH A 209      -3.135  39.625   7.649  1.00 35.40           O  \\nHETATM 1397  O   HOH A 210       8.699  39.076   2.399  1.00 33.93           O  \\nHETATM 1398  O   HOH A 212      10.040  41.806   3.447  1.00 37.88           O  \\nHETATM 1399  O   HOH A 213      18.914  29.095   3.704  1.00 25.32           O  \\nHETATM 1400  O   HOH A 214      15.966  29.098   2.957  1.00 18.63           O  \\nHETATM 1401  O   HOH A 215      15.506  31.962   4.015  1.00 18.93           O  \\nHETATM 1402  O   HOH A 216      22.935  30.533  10.845  1.00 39.81           O  \\nHETATM 1403  O   HOH A 217       6.476  43.893  23.325  1.00 49.09           O  \\nHETATM 1404  O   HOH A 218      11.746  41.248  28.612  1.00 36.22           O  \\nHETATM 1405  O   HOH A 219      11.480  42.270  25.715  1.00 29.66           O  \\nHETATM 1406  O   HOH A 220       9.344  34.251  -8.278  1.00 36.61           O  \\nHETATM 1407  O   HOH A 221      -0.111  19.667  -8.066  1.00 44.76           O  \\nHETATM 1408  O   HOH A 222      -0.528  30.460 -10.943  1.00 24.29           O  \\nHETATM 1409  O   HOH A 223     -11.352  24.274  19.491  1.00 31.21           O  \\nHETATM 1410  O   HOH A 224      -4.293  30.642 -16.466  1.00127.55           O  \\nHETATM 1411  O   HOH A 225       3.466  29.729 -16.232  1.00 26.15           O  \\nHETATM 1412  O   HOH A 227      -4.288  18.610   4.198  1.00 34.77           O  \\nHETATM 1413  O   HOH A 228       4.626  16.154   5.151  1.00 14.96           O  \\nHETATM 1414  O   HOH A 229       5.681  11.069   2.580  1.00 59.76           O  \\nHETATM 1415  O   HOH A 230       1.173  14.021   3.406  1.00 32.15           O  \\nHETATM 1416  O   HOH A 231       7.935  22.484  20.504  1.00 10.86           O  \\nHETATM 1417  O   HOH A 232       3.071  10.004  14.546  1.00 23.48           O  \\nHETATM 1418  O   HOH A 233       2.437  13.381   8.401  1.00 17.75           O  \\nHETATM 1419  O   HOH A 234       4.779   7.500   6.154  1.00 71.09           O  \\nHETATM 1420  O   HOH A 235      -4.250   8.748   6.880  1.00 24.76           O  \\nHETATM 1421  O   HOH A 236     -13.381  22.830  17.669  1.00 39.09           O  \\nHETATM 1422  O   HOH A 237      16.969  39.052   8.276  1.00 50.25           O  \\nHETATM 1423  O   HOH A 238      -6.315   9.333  13.897  1.00 36.24           O  \\nHETATM 1424  O   HOH A 240      -0.160   4.395  12.468  1.00 38.79           O  \\nHETATM 1425  O   HOH A 241       1.295   7.931  12.335  1.00 49.03           O  \\nHETATM 1426  O   HOH A 242       4.067  11.551   6.968  1.00 46.97           O  \\nHETATM 1427  O   HOH A 243      -5.060   6.579  -0.216  1.00 50.14           O  \\nHETATM 1428  O   HOH A 244      18.164   7.151   5.573  1.00 71.67           O  \\nHETATM 1429  O   HOH A 245       2.068   8.904   7.636  1.00 45.86           O  \\nHETATM 1430  O   HOH A 246      24.661  23.134  12.757  1.00 34.69           O  \\nHETATM 1431  O   HOH A 247      21.051  23.714   6.239  1.00 24.64           O  \\nHETATM 1432  O   HOH A 248      25.855  31.137  20.712  1.00 34.53           O  \\nHETATM 1433  O   HOH A 249      24.868  30.890  26.223  1.00 48.47           O  \\nHETATM 1434  O   HOH A 250      26.967  20.447  19.448  1.00 47.77           O  \\nHETATM 1435  O   HOH A 252      28.453  14.196  14.602  1.00 39.78           O  \\nHETATM 1436  O   HOH A 253      19.639  16.256   9.760  1.00 37.54           O  \\nHETATM 1437  O   HOH A 254      16.629   7.383  10.526  1.00 60.90           O  \\nHETATM 1438  O   HOH A 255       4.972   7.888  14.276  1.00 54.95           O  \\nHETATM 1439  O   HOH A 256      14.184  14.684   5.643  1.00 31.40           O  \\nHETATM 1440  O   HOH A 257      11.766  17.254   1.278  1.00 41.82           O  \\nHETATM 1441  O   HOH A 258      16.274  16.365   6.342  1.00 17.29           O  \\nHETATM 1442  O   HOH A 259      18.346  24.095   6.620  1.00 17.36           O  \\nHETATM 1443  O   HOH A 260      22.479  35.073  19.694  1.00 43.96           O  \\nHETATM 1444  O   HOH A 261      20.580  31.747   4.906  1.00 59.46           O  \\nHETATM 1445  O   HOH A 262      18.196  25.514   4.261  1.00 28.75           O  \\nHETATM 1446  O   HOH A 263      12.857  20.183   1.149  1.00 21.79           O  \\nHETATM 1447  O   HOH A 264      19.514  15.912  24.943  1.00 55.52           O  \\nHETATM 1448  O   HOH A 265       8.925   7.579   5.621  1.00 77.94           O  \\nHETATM 1449  O   HOH A 266      17.622  21.835  26.964  1.00 27.48           O  \\nHETATM 1450  O   HOH A 267      24.258  19.786  14.155  1.00 20.48           O  \\nHETATM 1451  O   HOH A 268      23.806  15.180  15.136  1.00 19.86           O  \\nHETATM 1452  O   HOH A 269      21.478  14.557  11.134  1.00 29.66           O  \\nHETATM 1453  O   HOH A 270      17.293  45.621  24.753  1.00 39.18           O  \\nHETATM 1454  O   HOH A 271      -3.518  40.206  17.649  1.00 44.75           O  \\nHETATM 1455  O   HOH A 272       2.143  19.852  24.090  1.00 40.10           O  \\nHETATM 1456  O   HOH A 273       7.101  35.918  -4.704  1.00 31.76           O  \\nHETATM 1457  O   HOH A 274       5.462  38.756  -5.521  1.00 27.51           O  \\nHETATM 1458  O   HOH A 275       5.748  20.565  28.638  1.00 58.06           O  \\nHETATM 1459  O   HOH A 276       8.814  31.258  -8.964  1.00 23.79           O  \\nHETATM 1460  O   HOH A 277     -10.427  35.895   4.301  1.00 58.91           O  \\nHETATM 1461  O   HOH A 278      -8.713  28.579   2.333  1.00 69.36           O  \\nHETATM 1462  O   HOH A 279      24.472  24.237  17.316  1.00 32.14           O  \\nHETATM 1463  O   HOH A 281      16.968  32.701  21.815  1.00 23.13           O  \\nHETATM 1464  O   HOH A 282      19.541  34.346  21.581  1.00 53.25           O  \\nHETATM 1465  O   HOH A 283      24.847  23.384  26.621  1.00 61.18           O  \\nHETATM 1466  O   HOH A 284       1.409  11.113   2.873  1.00 44.87           O  \\nHETATM 1467  O   HOH A 285       1.556  11.488  -0.914  1.00 37.86           O  \\nHETATM 1468  O   HOH A 286       9.758  12.945   3.889  1.00 36.47           O  \\nHETATM 1469  O   HOH A 287       3.827  13.375   4.622  1.00 44.90           O  \\nHETATM 1470  O   HOH A 288      14.279   6.175  15.391  1.00 78.25           O  \\nHETATM 1471  O   HOH A 289       7.573  13.968   2.191  1.00 76.47           O  \\nHETATM 1472  O   HOH A 290       2.323  15.482   1.418  1.00 35.15           O  \\nHETATM 1473  O   HOH A 291      11.997  14.573   2.962  1.00 49.86           O  \\nHETATM 1474  O   HOH A 292      23.798  33.550  17.880  1.00 24.64           O  \\nHETATM 1475  O   HOH A 293      14.657  36.046  -0.571  1.00 51.86           O  \\nHETATM 1476  O   HOH A 294      24.456  25.366   4.004  1.00 53.23           O  \\nHETATM 1477  O   HOH A 296      -0.603  12.134  -4.185  1.00 52.38           O  \\nHETATM 1478  O   HOH A 297      -8.372  34.415  -4.471  1.00 74.53           O  \\nHETATM 1479  O   HOH A 300     -14.033  36.501   0.062  1.00 50.21           O  \\nHETATM 1480  O   HOH A 301     -17.102  35.735  -2.806  1.00 42.58           O  \\nHETATM 1481  O   HOH A 303     -17.383  40.542   1.431  1.00 77.16           O  \\nHETATM 1482  O   HOH A 304     -19.024  41.557  -3.411  1.00107.16           O  \\nHETATM 1483  O   HOH A 305      -9.876  37.804   0.839  1.00 90.65           O  \\nHETATM 1484  O   HOH A 306     -14.053  36.466   4.552  1.00 82.05           O  \\nHETATM 1485  O   HOH A 307     -14.183  39.558   0.339  1.00 58.66           O  \\nHETATM 1486  O   HOH A 309     -12.547  28.677   8.301  1.00 74.88           O  \\nHETATM 1487  O   HOH A 311     -14.787  32.746  -2.171  1.00 40.79           O  \\nHETATM 1488  O   HOH A 312      -6.916  30.740   3.095  1.00 36.53           O  \\nHETATM 1489  O   HOH A 315      -8.312  33.027   7.340  1.00 83.28           O  \\nHETATM 1490  O   HOH A 316      -6.280  35.646   7.624  1.00 75.97           O  \\nHETATM 1491  O   HOH A 317      -5.230  31.565   9.609  1.00 55.67           O  \\nHETATM 1492  O   HOH A 318       6.747  37.258   1.208  1.00 57.93           O  \\nHETATM 1493  O   HOH A 319       4.747  42.419  12.683  1.00 57.74           O  \\nHETATM 1494  O   HOH A 320       2.984  41.950  16.191  1.00 41.69           O  \\nHETATM 1495  O   HOH A 321       3.867  49.208  12.056  1.00 24.57           O  \\nHETATM 1496  O   HOH A 322       7.530  42.894  17.776  1.00 79.28           O  \\nHETATM 1497  O   HOH A 324      19.822  39.457  17.826  1.00 66.22           O  \\nHETATM 1498  O   HOH A 325      12.391  45.557  24.973  1.00 48.70           O  \\nHETATM 1499  O   HOH A 326      14.944  43.928  23.794  1.00 62.45           O  \\nHETATM 1500  O   HOH A 327     -12.984  24.008  12.563  1.00 40.31           O  \\nHETATM 1501  O   HOH A 330       0.572  41.133  19.277  1.00 50.77           O  \\nHETATM 1502  O   HOH A 331      -6.196  37.979  23.043  1.00 48.93           O  \\nHETATM 1503  O   HOH A 332      -3.526  30.410  19.065  1.00 21.26           O  \\nHETATM 1504  O   HOH A 333      -7.995  32.093  21.685  1.00 54.11           O  \\nHETATM 1505  O   HOH A 334      -8.483  28.891  20.063  1.00 70.37           O  \\nHETATM 1506  O   HOH A 335     -11.670  31.022  16.981  1.00 76.86           O  \\nHETATM 1507  O   HOH A 336     -11.082  34.450  18.910  1.00 43.07           O  \\nHETATM 1508  O   HOH A 339      -6.521  37.738  15.165  1.00 64.11           O  \\nHETATM 1509  O   HOH A 340     -10.829  35.877  15.629  1.00 58.36           O  \\nHETATM 1510  O   HOH A 341      -6.552  41.166  12.231  1.00 55.67           O  \\nHETATM 1511  O   HOH A 342      -1.242  39.671  12.185  1.00 61.04           O  \\nHETATM 1512  O   HOH A 343      -4.281  38.934  13.464  1.00 32.14           O  \\nHETATM 1513  O   HOH A 344       0.386  41.208  14.039  1.00 40.80           O  \\nHETATM 1514  O   HOH A 345       0.777  39.663  21.603  1.00 67.71           O  \\nHETATM 1515  O   HOH A 346      -7.879  34.848  19.044  1.00 52.60           O  \\nHETATM 1516  O   HOH A 347       1.121  32.594 -11.957  1.00 32.30           O  \\nHETATM 1517  O   HOH A 348       0.924  32.597 -14.716  1.00 48.34           O  \\nHETATM 1518  O   HOH A 349      -2.241  27.348  20.565  1.00 26.54           O  \\nHETATM 1519  O   HOH A 350       3.259  26.214  29.881  1.00 69.22           O  \\nHETATM 1520  O   HOH A 351      -8.318  34.095  28.392  1.00 81.96           O  \\nHETATM 1521  O   HOH A 352      -2.927  29.775  31.412  1.00 80.92           O  \\nHETATM 1522  O   HOH A 353     -10.410  32.034  25.972  1.00 60.17           O  \\nHETATM 1523  O   HOH A 354      -8.122  30.393  25.378  1.00 62.33           O  \\nHETATM 1524  O   HOH A 356      -6.138  18.882  24.644  1.00 49.26           O  \\nHETATM 1525  O   HOH A 358      -1.406  19.033  23.084  1.00 82.05           O  \\nHETATM 1526  O   HOH A 359       4.169   7.339  10.280  1.00 66.03           O  \\nHETATM 1527  O   HOH A 360      -9.761  30.165   7.702  1.00 46.15           O  \\nHETATM 1528  O   HOH A 361     -16.002  27.801  11.700  1.00 41.85           O  \\nHETATM 1529  O   HOH A 362      16.215  30.489  29.264  1.00 67.21           O  \\nHETATM 1530  O   HOH A 364      21.417  27.685  26.140  1.00 49.57           O  \\nHETATM 1531  O   HOH A 366      15.824  34.767  30.662  1.00 25.90           O  \\nHETATM 1532  O   HOH A 367      18.527  35.965  24.111  1.00 68.69           O  \\nHETATM 1533  O   HOH A 369      12.103  35.275  24.945  1.00 51.37           O  \\nHETATM 1534  O   HOH A 370      24.782  11.970  23.641  1.00 55.92           O  \\nHETATM 1535  O   HOH A 371      14.337  21.394  28.627  1.00 30.11           O  \\nHETATM 1536  O   HOH A 372      -1.367  12.600  23.053  1.00 44.43           O  \\nHETATM 1537  O   HOH A 373       7.060   8.567  21.416  1.00 47.99           O  \\nHETATM 1538  O   HOH A 374       7.639   8.040  16.420  1.00 38.76           O  \\nHETATM 1539  O   HOH A 376      -1.282  15.566  25.487  1.00 67.79           O  \\nHETATM 1540  O   HOH A 378      17.945  30.924  23.806  1.00 31.50           O  \\nHETATM 1541  O   HOH A 379      17.360  33.209  33.337  1.00 30.46           O  \\nHETATM 1542  O   HOH A 380       3.971   6.567  22.089  1.00 86.80           O  \\nHETATM 1543  O   HOH A 381      -0.065  10.022  26.928  1.00 61.86           O  \\nHETATM 1544  O   HOH A 382      -4.450  12.966  21.222  1.00 42.82           O  \\nHETATM 1545  O   HOH A 383      -2.584   6.719   5.938  1.00 40.64           O  \\nHETATM 1546  O   HOH A 384      25.567  31.936  12.845  1.00 73.17           O  \\nHETATM 1547  O   HOH A 385      24.446  29.383  22.970  1.00 36.61           O  \\nHETATM 1548  O   HOH A 386      24.865  38.167  18.227  1.00 60.74           O  \\nHETATM 1549  O   HOH A 387      23.135  35.012  14.543  1.00 59.19           O  \\nHETATM 1550  O   HOH A 388      19.325   9.556  18.093  1.00 92.18           O  \\nHETATM 1551  O   HOH A 389      25.326  12.873  16.251  1.00 49.27           O  \\nHETATM 1552  O   HOH A 390      20.219  10.405  10.956  1.00 63.05           O  \\nHETATM 1553  O   HOH A 392       9.452   5.453   8.249  1.00 47.81           O  \\nHETATM 1554  O   HOH A 394      10.139  12.252   0.254  1.00 29.37           O  \\nHETATM 1555  O   HOH A 395      28.004  37.416  13.466  1.00 52.29           O  \\nHETATM 1556  O   HOH A 396      24.283  36.829  12.307  1.00 64.74           O  \\nHETATM 1557  O   HOH A 397      13.748  34.031  -3.019  1.00 47.80           O  \\nHETATM 1558  O   HOH A 398      16.981  34.189   2.006  1.00 80.51           O  \\nHETATM 1559  O   HOH A 399      -4.426  24.223  -7.240  1.00 38.09           O  \\nHETATM 1560  O   HOH A 400      19.676  33.981   3.336  1.00 64.86           O  \\nHETATM 1561  O   HOH A 401      14.633  28.745  -0.317  1.00 66.25           O  \\nHETATM 1562  O   HOH A 402      -2.139  15.576  22.716  1.00 65.01           O  \\nHETATM 1563  O   HOH A 403       9.873  11.090  20.760  1.00 53.88           O  \\nHETATM 1564  O   HOH A 404      18.291  36.379  26.927  1.00 57.23           O  \\nHETATM 1565  O   HOH A 405       0.138   6.145  15.269  1.00 82.80           O  \\nHETATM 1566  O   HOH A 406      -5.515  38.166  18.689  1.00 48.28           O  \\nHETATM 1567  O   HOH A 407      -0.399   4.688   8.809  1.00 58.35           O  \\nHETATM 1568  O   HOH A 408       0.813  25.584  22.307  1.00 40.04           O  \\nCONECT  117 1325                                                                \\nCONECT  265 1325                                                                \\nCONECT 1325  117  265 1328 1333                                                 \\nCONECT 1325 1361 1362                                                           \\nCONECT 1326 1327 1328 1329 1330                                                 \\nCONECT 1327 1326                                                                \\nCONECT 1328 1325 1326                                                           \\nCONECT 1329 1326                                                                \\nCONECT 1330 1326 1331                                                           \\nCONECT 1331 1330 1332 1333 1334                                                 \\nCONECT 1332 1331                                                                \\nCONECT 1333 1325 1331                                                           \\nCONECT 1334 1331 1335                                                           \\nCONECT 1335 1334 1336 1337 1338                                                 \\nCONECT 1336 1335                                                                \\nCONECT 1337 1335                                                                \\nCONECT 1338 1335 1339                                                           \\nCONECT 1339 1338 1340                                                           \\nCONECT 1340 1339 1341 1342                                                      \\nCONECT 1341 1340 1346                                                           \\nCONECT 1342 1340 1343 1344                                                      \\nCONECT 1343 1342                                                                \\nCONECT 1344 1342 1345 1346                                                      \\nCONECT 1345 1344                                                                \\nCONECT 1346 1341 1344 1347                                                      \\nCONECT 1347 1346 1348 1357                                                      \\nCONECT 1348 1347 1349                                                           \\nCONECT 1349 1348 1350                                                           \\nCONECT 1350 1349 1351 1357                                                      \\nCONECT 1351 1350 1352 1353                                                      \\nCONECT 1352 1351                                                                \\nCONECT 1353 1351 1354                                                           \\nCONECT 1354 1353 1355 1356                                                      \\nCONECT 1355 1354                                                                \\nCONECT 1356 1354 1357                                                           \\nCONECT 1357 1347 1350 1356                                                      \\nCONECT 1361 1325                                                                \\nCONECT 1362 1325                                                                \\nMASTER      312    0    2    5    6    0    9    6 1567    1   38   13          \\nEND                                                                             \\n\",\"pdb\");\n\tviewer_16835744454165597.setStyle({\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_16835744454165597.addSurface(1,{\"opacity\": 0.7, \"color\": \"white\"});\n\tviewer_16835744454165597.zoomTo();\nviewer_16835744454165597.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_16835744454165597\"  style=\"position: relative; width: 700px; height: 700px\">\n",
       "        <p id=\"3dmolwarning_16835744454165597\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n",
       "}\n",
       "\n",
       "var viewer_16835744454165597 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_16835744454165597\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_16835744454165597 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_16835744454165597\"),{backgroundColor:\"white\"});\n",
       "viewer_16835744454165597.zoomTo();\n",
       "\tviewer_16835744454165597.addModel(\"HEADER    ONCOGENE PROTEIN                        30-APR-90   5P21              \\nTITLE     REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION OF H-RAS   \\nTITLE    2 P21 AT 1.35 ANGSTROMS RESOLUTION: IMPLICATIONS FOR THE MECHANISM OF  \\nTITLE    3 GTP HYDROLYSIS                                                       \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: C-H-RAS P21 PROTEIN;                                       \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    ONCOGENE PROTEIN                                                      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    E.F.PAI,A.WITTINGHOFER,W.KABSCH                                       \\nREVDAT   6   29-NOV-17 5P21    1       HELIX                                    \\nREVDAT   5   24-FEB-09 5P21    1       VERSN                                    \\nREVDAT   4   01-APR-03 5P21    1       JRNL                                     \\nREVDAT   3   20-JUL-95 5P21    1       HET                                      \\nREVDAT   2   30-APR-94 5P21    1       FORMUL                                   \\nREVDAT   1   15-JAN-92 5P21    0                                                \\nJRNL        AUTH   E.F.PAI,U.KRENGEL,G.A.PETSKO,R.S.GOODY,W.KABSCH,             \\nJRNL        AUTH 2 A.WITTINGHOFER                                               \\nJRNL        TITL   REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION   \\nJRNL        TITL 2 OF H-RAS P21 AT 1.35 A RESOLUTION: IMPLICATIONS FOR THE      \\nJRNL        TITL 3 MECHANISM OF GTP HYDROLYSIS.                                 \\nJRNL        REF    EMBO J.                       V.   9  2351 1990              \\nJRNL        REFN                   ISSN 0261-4189                               \\nJRNL        PMID   2196171                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,S.C.ALMO,G.RAPP,K.WILSON,K.PETRATOS,A.LENTFER, \\nREMARK   1  AUTH 2 A.WITTINGHOFER,W.KABSCH,E.F.PAI,G.A.PETSKO,R.S.GOODY         \\nREMARK   1  TITL   TIME-RESOLVED X-RAY CRYSTALLOGRAPHIC STUDY OF THE            \\nREMARK   1  TITL 2 CONFORMATIONAL CHANGE IN HA-RAS P21 PROTEIN ON GTP           \\nREMARK   1  TITL 3 HYDROLYSIS                                                   \\nREMARK   1  REF    NATURE                        V. 345   309 1990              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   A.SCHERER,J.JOHN,R.LINKE,R.S.GOODY,A.WITTINGHOFER,E.F.PAI,   \\nREMARK   1  AUTH 2 K.C.HOLMES                                                   \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY ANALYSIS OF THE HUMAN  \\nREMARK   1  TITL 2 C-H-RAS-ONCOGENE PRODUCT P21 COMPLEXED WITH GTP ANALOGUES    \\nREMARK   1  REF    J.MOL.BIOL.                   V. 206   257 1989              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   E.F.PAI,W.KABSCH,U.KRENGEL,K.C.HOLMES,J.JOHN,A.WITTINGHOFER  \\nREMARK   1  TITL   STRUCTURE OF THE GUANINE-NUCLEOTIDE-BINDING DOMAIN OF THE    \\nREMARK   1  TITL 2 HA-RAS ONCOGENE PRODUCT P21 IN THE TRIPHOSPHATE CONFORMATION \\nREMARK   1  REF    NATURE                        V. 341   209 1989              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,G.RAPP,J.JOHN,A.WITTINGHOFER,E.F.PAI,R.S.GOODY \\nREMARK   1  TITL   BIOCHEMICAL AND CRYSTALLOGRAPHIC CHARACTERIZATION OF A       \\nREMARK   1  TITL 2 COMPLEX OF C-HA-RAS P21 AND CAGED GTP WITH FLASH PHOTOLYSIS  \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  86  7687 1989              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 6.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 26806                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.196                           \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1323                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 33                                      \\nREMARK   3   SOLVENT ATOMS            : 211                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.014                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 2.700                           \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  RESIDUES 61 - 65 (GLN - GLU - GLU - TYR - SER) ADOPT                \\nREMARK   3  SEVERAL CONFORMATIONS IN THE CRYSTAL.  THE COORDINATES              \\nREMARK   3  GIVEN APPROXIMATE ONE OF THESE.                                     \\nREMARK   4                                                                      \\nREMARK   4 5P21 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179747.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 38.93                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.01                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 32 2 1                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+2/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+1/3                                           \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z+1/3                                           \\nREMARK 290       6555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000      108.13333            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       54.06667            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       54.06667            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000      108.13333            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025  0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    ARG A  68   NE  -  CZ  -  NH1 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A  97   NE  -  CZ  -  NH1 ANGL. DEV. =  -3.4 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH1 ANGL. DEV. =   4.1 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500    ARG A 128   NE  -  CZ  -  NH2 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A 135   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500    TYR A 157   CB  -  CG  -  CD2 ANGL. DEV. =  -4.8 DEGREES          \\nREMARK 500    ARG A 164   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ASP A  33       75.86   -157.25                                   \\nREMARK 500    ILE A  36      -64.23   -100.39                                   \\nREMARK 500    GLU A  37      123.75   -171.30                                   \\nREMARK 500    SER A  65     -133.70     53.07                                   \\nREMARK 500    LYS A 117       33.80     71.60                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              MG A 168  MG                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 GNP A 167   O2G                                                    \\nREMARK 620 2 GNP A 167   O2B  95.0                                              \\nREMARK 620 3 HOH A 173   O    87.6  91.7                                        \\nREMARK 620 4 SER A  17   OG  172.9  91.0  88.6                                  \\nREMARK 620 5 THR A  35   OG1  91.3 173.4  90.5  82.8                            \\nREMARK 620 6 HOH A 172   O    97.3  88.0 175.0  86.5  89.3                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE MG A 168                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE GNP A 167                 \\nDBREF  5P21 A    1   166  UNP    P01112   RASH_HUMAN       1    166             \\nSEQRES   1 A  166  MET THR GLU TYR LYS LEU VAL VAL VAL GLY ALA GLY GLY          \\nSEQRES   2 A  166  VAL GLY LYS SER ALA LEU THR ILE GLN LEU ILE GLN ASN          \\nSEQRES   3 A  166  HIS PHE VAL ASP GLU TYR ASP PRO THR ILE GLU ASP SER          \\nSEQRES   4 A  166  TYR ARG LYS GLN VAL VAL ILE ASP GLY GLU THR CYS LEU          \\nSEQRES   5 A  166  LEU ASP ILE LEU ASP THR ALA GLY GLN GLU GLU TYR SER          \\nSEQRES   6 A  166  ALA MET ARG ASP GLN TYR MET ARG THR GLY GLU GLY PHE          \\nSEQRES   7 A  166  LEU CYS VAL PHE ALA ILE ASN ASN THR LYS SER PHE GLU          \\nSEQRES   8 A  166  ASP ILE HIS GLN TYR ARG GLU GLN ILE LYS ARG VAL LYS          \\nSEQRES   9 A  166  ASP SER ASP ASP VAL PRO MET VAL LEU VAL GLY ASN LYS          \\nSEQRES  10 A  166  CYS ASP LEU ALA ALA ARG THR VAL GLU SER ARG GLN ALA          \\nSEQRES  11 A  166  GLN ASP LEU ALA ARG SER TYR GLY ILE PRO TYR ILE GLU          \\nSEQRES  12 A  166  THR SER ALA LYS THR ARG GLN GLY VAL GLU ASP ALA PHE          \\nSEQRES  13 A  166  TYR THR LEU VAL ARG GLU ILE ARG GLN HIS                      \\nHET     MG  A 168       1                                                       \\nHET    GNP  A 167      32                                                       \\nHETNAM      MG MAGNESIUM ION                                                    \\nHETNAM     GNP PHOSPHOAMINOPHOSPHONIC ACID-GUANYLATE ESTER                      \\nFORMUL   2   MG    MG 2+                                                        \\nFORMUL   3  GNP    C10 H17 N6 O13 P3                                            \\nFORMUL   4  HOH   *211(H2 O)                                                    \\nHELIX    1  HA LYS A   16  GLN A   25  1                                  10    \\nHELIX    2  HB SER A   65  THR A   74  1RES 65-67 IN 3/10 CONF            10    \\nHELIX    3  HC THR A   87  VAL A  103  1BREAK AT ASP 92                   17    \\nHELIX    4  HD SER A  127  TYR A  137  1                                  11    \\nHELIX    5  HE VAL A  152  ARG A  164  1                                  13    \\nSHEET    1  S1 6 GLU A  37  ILE A  46  0                                        \\nSHEET    2  S1 6 GLU A  49  THR A  58 -1  O  LEU A  53   N  LYS A  42           \\nSHEET    3  S1 6 THR A   2  VAL A   9  1  N  LEU A   6   O  ASP A  54           \\nSHEET    4  S1 6 GLY A  77  ALA A  83  1  N  LEU A  79   O  VAL A   7           \\nSHEET    5  S1 6 MET A 111  ASN A 116  1  N  VAL A 114   O  CYS A  80           \\nSHEET    6  S1 6 TYR A 141  GLU A 143  1  N  ILE A 142   O  LEU A 113           \\nLINK        MG    MG A 168                 O2G GNP A 167     1555   1555  2.20  \\nLINK        MG    MG A 168                 O2B GNP A 167     1555   1555  2.22  \\nLINK        MG    MG A 168                 O   HOH A 173     1555   1555  2.23  \\nLINK        MG    MG A 168                 OG  SER A  17     1555   1555  2.26  \\nLINK        MG    MG A 168                 OG1 THR A  35     1555   1555  2.26  \\nLINK        MG    MG A 168                 O   HOH A 172     1555   1555  2.30  \\nSITE     1 AC1  5 SER A  17  THR A  35  GNP A 167  HOH A 172                    \\nSITE     2 AC1  5 HOH A 173                                                     \\nSITE     1 AC2 28 GLY A  12  GLY A  13  VAL A  14  GLY A  15                    \\nSITE     2 AC2 28 LYS A  16  SER A  17  ALA A  18  PHE A  28                    \\nSITE     3 AC2 28 VAL A  29  ASP A  30  TYR A  32  PRO A  34                    \\nSITE     4 AC2 28 THR A  35  GLY A  60  ASN A 116  LYS A 117                    \\nSITE     5 AC2 28 ASP A 119  SER A 145  ALA A 146  LYS A 147                    \\nSITE     6 AC2 28  MG A 168  HOH A 170  HOH A 172  HOH A 173                    \\nSITE     7 AC2 28 HOH A 175  HOH A 187  HOH A 281  HOH A 366                    \\nCRYST1   40.300   40.300  162.200  90.00  90.00 120.00 P 32 2 1      6          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.024814  0.014326  0.000000        0.00000                         \\nSCALE2      0.000000  0.028653  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.006165        0.00000                         \\nATOM      1  N   MET A   1      -7.186  32.862  -6.632  1.00 13.92           N  \\nATOM      2  CA  MET A   1      -5.921  32.127  -6.684  1.00 17.10           C  \\nATOM      3  C   MET A   1      -5.845  31.045  -5.621  1.00 16.74           C  \\nATOM      4  O   MET A   1      -6.600  31.083  -4.636  1.00 16.60           O  \\nATOM      5  CB  MET A   1      -4.673  33.058  -6.502  1.00 17.00           C  \\nATOM      6  CG  MET A   1      -4.633  33.804  -5.188  1.00 18.08           C  \\nATOM      7  SD  MET A   1      -3.115  34.766  -4.908  1.00 19.73           S  \\nATOM      8  CE  MET A   1      -3.019  35.851  -6.310  1.00 20.41           C  \\nATOM      9  N   THR A   2      -4.910  30.095  -5.820  1.00 14.70           N  \\nATOM     10  CA  THR A   2      -4.661  29.030  -4.859  1.00 16.92           C  \\nATOM     11  C   THR A   2      -4.230  29.655  -3.547  1.00 14.78           C  \\nATOM     12  O   THR A   2      -3.338  30.501  -3.537  1.00 13.93           O  \\nATOM     13  CB  THR A   2      -3.557  28.107  -5.401  1.00 18.03           C  \\nATOM     14  OG1 THR A   2      -4.059  27.565  -6.629  1.00 20.42           O  \\nATOM     15  CG2 THR A   2      -3.193  26.983  -4.443  1.00 18.51           C  \\nATOM     16  N   GLU A   3      -4.911  29.290  -2.472  1.00 14.80           N  \\nATOM     17  CA  GLU A   3      -4.504  29.730  -1.135  1.00 17.71           C  \\nATOM     18  C   GLU A   3      -4.012  28.553  -0.278  1.00 17.33           C  \\nATOM     19  O   GLU A   3      -4.520  27.427  -0.395  1.00 17.54           O  \\nATOM     20  CB  GLU A   3      -5.655  30.402  -0.432  1.00 20.67           C  \\nATOM     21  CG  GLU A   3      -6.213  31.631  -1.186  1.00 25.56           C  \\nATOM     22  CD  GLU A   3      -7.128  32.563  -0.369  1.00 28.46           C  \\nATOM     23  OE1 GLU A   3      -7.578  32.216   0.728  1.00 27.72           O  \\nATOM     24  OE2 GLU A   3      -7.375  33.680  -0.827  1.00 32.49           O  \\nATOM     25  N   TYR A   4      -2.962  28.744   0.542  1.00 14.50           N  \\nATOM     26  CA  TYR A   4      -2.425  27.654   1.358  1.00 12.29           C  \\nATOM     27  C   TYR A   4      -2.534  28.103   2.807  1.00 12.52           C  \\nATOM     28  O   TYR A   4      -2.016  29.143   3.200  1.00 12.11           O  \\nATOM     29  CB  TYR A   4      -0.975  27.385   1.026  1.00 11.46           C  \\nATOM     30  CG  TYR A   4      -0.726  26.846  -0.387  1.00 12.38           C  \\nATOM     31  CD1 TYR A   4      -0.297  27.698  -1.395  1.00 13.00           C  \\nATOM     32  CD2 TYR A   4      -0.933  25.501  -0.649  1.00 14.09           C  \\nATOM     33  CE1 TYR A   4      -0.070  27.208  -2.674  1.00 13.89           C  \\nATOM     34  CE2 TYR A   4      -0.711  24.993  -1.917  1.00 14.65           C  \\nATOM     35  CZ  TYR A   4      -0.279  25.859  -2.917  1.00 14.84           C  \\nATOM     36  OH  TYR A   4      -0.084  25.357  -4.191  1.00 15.07           O  \\nATOM     37  N   LYS A   5      -3.274  27.305   3.568  1.00 13.07           N  \\nATOM     38  CA  LYS A   5      -3.554  27.569   4.970  1.00 12.96           C  \\nATOM     39  C   LYS A   5      -2.556  26.818   5.852  1.00 12.74           C  \\nATOM     40  O   LYS A   5      -2.645  25.611   6.092  1.00 11.58           O  \\nATOM     41  CB  LYS A   5      -4.972  27.131   5.281  1.00 17.65           C  \\nATOM     42  CG  LYS A   5      -6.053  27.833   4.446  1.00 23.49           C  \\nATOM     43  CD  LYS A   5      -7.186  28.310   5.335  1.00 29.10           C  \\nATOM     44  CE  LYS A   5      -6.655  29.309   6.403  1.00 33.68           C  \\nATOM     45  NZ  LYS A   5      -6.597  28.739   7.755  1.00 36.52           N  \\nATOM     46  N   LEU A   6      -1.543  27.574   6.297  1.00 13.08           N  \\nATOM     47  CA  LEU A   6      -0.418  27.011   7.064  1.00 10.34           C  \\nATOM     48  C   LEU A   6      -0.578  27.382   8.535  1.00  9.68           C  \\nATOM     49  O   LEU A   6      -1.034  28.483   8.863  1.00 11.31           O  \\nATOM     50  CB  LEU A   6       0.910  27.571   6.536  1.00 11.34           C  \\nATOM     51  CG  LEU A   6       1.137  27.565   5.017  1.00 11.62           C  \\nATOM     52  CD1 LEU A   6       2.515  28.119   4.720  1.00 13.60           C  \\nATOM     53  CD2 LEU A   6       0.973  26.171   4.475  1.00 12.19           C  \\nATOM     54  N   VAL A   7      -0.196  26.486   9.432  1.00  9.93           N  \\nATOM     55  CA  VAL A   7      -0.333  26.787  10.852  1.00 10.06           C  \\nATOM     56  C   VAL A   7       1.033  26.513  11.495  1.00 10.37           C  \\nATOM     57  O   VAL A   7       1.597  25.441  11.252  1.00  9.51           O  \\nATOM     58  CB  VAL A   7      -1.474  25.884  11.446  1.00 10.97           C  \\nATOM     59  CG1 VAL A   7      -1.610  26.163  12.943  1.00 11.39           C  \\nATOM     60  CG2 VAL A   7      -2.809  26.161  10.777  1.00 10.14           C  \\nATOM     61  N   VAL A   8       1.576  27.455  12.295  1.00  8.35           N  \\nATOM     62  CA  VAL A   8       2.878  27.262  12.944  1.00  6.92           C  \\nATOM     63  C   VAL A   8       2.609  26.896  14.405  1.00  8.84           C  \\nATOM     64  O   VAL A   8       1.930  27.663  15.092  1.00  9.21           O  \\nATOM     65  CB  VAL A   8       3.684  28.565  12.799  1.00  8.08           C  \\nATOM     66  CG1 VAL A   8       5.111  28.324  13.261  1.00  9.52           C  \\nATOM     67  CG2 VAL A   8       3.729  29.011  11.358  1.00  8.43           C  \\nATOM     68  N   VAL A   9       3.024  25.701  14.881  1.00  8.73           N  \\nATOM     69  CA  VAL A   9       2.779  25.234  16.245  1.00  8.41           C  \\nATOM     70  C   VAL A   9       4.075  24.806  16.919  1.00  8.21           C  \\nATOM     71  O   VAL A   9       5.093  24.551  16.287  1.00 10.97           O  \\nATOM     72  CB  VAL A   9       1.824  24.028  16.286  1.00  9.74           C  \\nATOM     73  CG1 VAL A   9       0.545  24.491  15.680  1.00 11.76           C  \\nATOM     74  CG2 VAL A   9       2.349  22.810  15.535  1.00 10.31           C  \\nATOM     75  N   GLY A  10       4.048  24.707  18.238  1.00  7.34           N  \\nATOM     76  CA  GLY A  10       5.231  24.336  19.009  1.00  7.98           C  \\nATOM     77  C   GLY A  10       5.192  25.053  20.355  1.00  8.31           C  \\nATOM     78  O   GLY A  10       4.417  25.994  20.595  1.00  9.89           O  \\nATOM     79  N   ALA A  11       6.099  24.638  21.226  1.00  9.31           N  \\nATOM     80  CA  ALA A  11       6.121  25.135  22.597  1.00  9.50           C  \\nATOM     81  C   ALA A  11       6.440  26.625  22.711  1.00  9.66           C  \\nATOM     82  O   ALA A  11       6.998  27.250  21.806  1.00  9.46           O  \\nATOM     83  CB  ALA A  11       7.145  24.306  23.378  1.00  9.85           C  \\nATOM     84  N   GLY A  12       6.111  27.227  23.847  1.00 10.36           N  \\nATOM     85  CA  GLY A  12       6.406  28.632  24.054  1.00  9.30           C  \\nATOM     86  C   GLY A  12       7.892  28.950  23.964  1.00 10.23           C  \\nATOM     87  O   GLY A  12       8.785  28.249  24.466  1.00 10.57           O  \\nATOM     88  N   GLY A  13       8.159  30.064  23.266  1.00 11.49           N  \\nATOM     89  CA  GLY A  13       9.511  30.602  23.139  1.00 11.29           C  \\nATOM     90  C   GLY A  13      10.436  29.950  22.108  1.00 12.22           C  \\nATOM     91  O   GLY A  13      11.624  30.321  22.060  1.00 13.62           O  \\nATOM     92  N   VAL A  14       9.961  28.991  21.286  1.00  9.23           N  \\nATOM     93  CA  VAL A  14      10.848  28.327  20.317  1.00 10.38           C  \\nATOM     94  C   VAL A  14      11.169  29.168  19.076  1.00 10.47           C  \\nATOM     95  O   VAL A  14      12.108  28.841  18.353  1.00 12.07           O  \\nATOM     96  CB  VAL A  14      10.246  26.970  19.856  1.00  9.98           C  \\nATOM     97  CG1 VAL A  14      10.187  26.074  21.120  1.00  9.93           C  \\nATOM     98  CG2 VAL A  14       8.866  27.127  19.200  1.00  9.52           C  \\nATOM     99  N   GLY A  15      10.423  30.264  18.834  1.00  9.61           N  \\nATOM    100  CA  GLY A  15      10.666  31.160  17.706  1.00  9.70           C  \\nATOM    101  C   GLY A  15       9.615  31.061  16.626  1.00  8.82           C  \\nATOM    102  O   GLY A  15       9.915  31.403  15.491  1.00  8.99           O  \\nATOM    103  N   LYS A  16       8.395  30.591  16.916  1.00  8.18           N  \\nATOM    104  CA  LYS A  16       7.331  30.521  15.913  1.00  8.44           C  \\nATOM    105  C   LYS A  16       7.060  31.899  15.309  1.00  9.01           C  \\nATOM    106  O   LYS A  16       7.020  32.044  14.086  1.00  8.25           O  \\nATOM    107  CB  LYS A  16       6.014  30.000  16.529  1.00  9.27           C  \\nATOM    108  CG  LYS A  16       6.132  28.585  17.086  1.00  9.11           C  \\nATOM    109  CD  LYS A  16       4.820  28.077  17.726  1.00  9.31           C  \\nATOM    110  CE  LYS A  16       4.351  28.934  18.888  1.00 10.05           C  \\nATOM    111  NZ  LYS A  16       5.221  28.829  20.051  1.00  9.64           N  \\nATOM    112  N   SER A  17       6.913  32.942  16.135  1.00  7.74           N  \\nATOM    113  CA  SER A  17       6.661  34.295  15.649  1.00  8.78           C  \\nATOM    114  C   SER A  17       7.854  34.859  14.919  1.00  9.73           C  \\nATOM    115  O   SER A  17       7.653  35.446  13.868  1.00  9.70           O  \\nATOM    116  CB  SER A  17       6.360  35.264  16.769  1.00  8.22           C  \\nATOM    117  OG  SER A  17       5.083  34.853  17.204  1.00  8.79           O  \\nATOM    118  N   ALA A  18       9.072  34.687  15.444  1.00  8.95           N  \\nATOM    119  CA  ALA A  18      10.253  35.199  14.769  1.00  8.95           C  \\nATOM    120  C   ALA A  18      10.438  34.576  13.374  1.00  8.54           C  \\nATOM    121  O   ALA A  18      10.832  35.254  12.430  1.00  8.75           O  \\nATOM    122  CB  ALA A  18      11.521  34.931  15.640  1.00 10.11           C  \\nATOM    123  N   LEU A  19      10.136  33.284  13.176  1.00  8.31           N  \\nATOM    124  CA  LEU A  19      10.225  32.667  11.837  1.00  9.30           C  \\nATOM    125  C   LEU A  19       9.204  33.281  10.884  1.00 10.98           C  \\nATOM    126  O   LEU A  19       9.534  33.652   9.751  1.00 10.47           O  \\nATOM    127  CB  LEU A  19       9.938  31.165  11.885  1.00  9.41           C  \\nATOM    128  CG  LEU A  19      11.035  30.275  12.487  1.00 11.80           C  \\nATOM    129  CD1 LEU A  19      10.491  28.866  12.723  1.00 10.43           C  \\nATOM    130  CD2 LEU A  19      12.256  30.288  11.554  1.00 13.42           C  \\nATOM    131  N   THR A  20       7.950  33.419  11.335  1.00 10.72           N  \\nATOM    132  CA  THR A  20       6.888  34.015  10.529  1.00 11.87           C  \\nATOM    133  C   THR A  20       7.173  35.492  10.161  1.00 12.65           C  \\nATOM    134  O   THR A  20       7.012  35.917   9.009  1.00 12.60           O  \\nATOM    135  CB  THR A  20       5.547  33.888  11.312  1.00 13.43           C  \\nATOM    136  OG1 THR A  20       5.299  32.509  11.544  1.00 12.67           O  \\nATOM    137  CG2 THR A  20       4.346  34.410  10.518  1.00 14.64           C  \\nATOM    138  N   ILE A  21       7.615  36.326  11.098  1.00 10.48           N  \\nATOM    139  CA  ILE A  21       7.945  37.704  10.795  1.00 12.96           C  \\nATOM    140  C   ILE A  21       9.197  37.831   9.927  1.00 14.26           C  \\nATOM    141  O   ILE A  21       9.237  38.691   9.042  1.00 15.83           O  \\nATOM    142  CB  ILE A  21       8.090  38.451  12.121  1.00 12.57           C  \\nATOM    143  CG1 ILE A  21       6.691  38.551  12.737  1.00 13.24           C  \\nATOM    144  CG2 ILE A  21       8.631  39.852  11.910  1.00 13.40           C  \\nATOM    145  CD1 ILE A  21       6.636  39.185  14.129  1.00 16.46           C  \\nATOM    146  N   GLN A  22      10.206  36.979  10.092  1.00 13.58           N  \\nATOM    147  CA  GLN A  22      11.330  36.983   9.179  1.00 14.91           C  \\nATOM    148  C   GLN A  22      10.817  36.729   7.736  1.00 17.29           C  \\nATOM    149  O   GLN A  22      11.120  37.485   6.792  1.00 15.91           O  \\nATOM    150  CB  GLN A  22      12.270  35.930   9.736  1.00 15.90           C  \\nATOM    151  CG  GLN A  22      13.686  35.878   9.230  1.00 19.36           C  \\nATOM    152  CD  GLN A  22      14.372  37.217   9.091  1.00 18.08           C  \\nATOM    153  OE1 GLN A  22      14.160  37.984   8.143  1.00 19.23           O  \\nATOM    154  NE2 GLN A  22      15.224  37.492  10.050  1.00 17.40           N  \\nATOM    155  N   LEU A  23       9.944  35.718   7.523  1.00 15.95           N  \\nATOM    156  CA  LEU A  23       9.381  35.475   6.194  1.00 18.85           C  \\nATOM    157  C   LEU A  23       8.539  36.622   5.636  1.00 18.21           C  \\nATOM    158  O   LEU A  23       8.695  36.958   4.469  1.00 19.03           O  \\nATOM    159  CB  LEU A  23       8.498  34.235   6.196  1.00 18.10           C  \\nATOM    160  CG  LEU A  23       7.916  33.797   4.863  1.00 18.64           C  \\nATOM    161  CD1 LEU A  23       8.921  32.994   4.030  1.00 18.03           C  \\nATOM    162  CD2 LEU A  23       6.680  32.991   5.183  1.00 21.34           C  \\nATOM    163  N   ILE A  24       7.634  37.197   6.428  1.00 19.25           N  \\nATOM    164  CA  ILE A  24       6.682  38.198   5.953  1.00 20.23           C  \\nATOM    165  C   ILE A  24       7.316  39.590   5.922  1.00 21.89           C  \\nATOM    166  O   ILE A  24       7.125  40.318   4.943  1.00 22.93           O  \\nATOM    167  CB  ILE A  24       5.399  38.222   6.881  1.00 21.20           C  \\nATOM    168  CG1 ILE A  24       4.664  36.907   6.821  1.00 21.27           C  \\nATOM    169  CG2 ILE A  24       4.423  39.307   6.446  1.00 20.00           C  \\nATOM    170  CD1 ILE A  24       4.128  36.442   5.475  1.00 23.98           C  \\nATOM    171  N   GLN A  25       8.074  39.998   6.953  1.00 21.07           N  \\nATOM    172  CA  GLN A  25       8.520  41.380   7.066  1.00 22.09           C  \\nATOM    173  C   GLN A  25      10.003  41.559   6.962  1.00 20.75           C  \\nATOM    174  O   GLN A  25      10.495  42.677   7.054  1.00 20.26           O  \\nATOM    175  CB  GLN A  25       8.039  41.968   8.398  1.00 23.96           C  \\nATOM    176  CG  GLN A  25       6.548  42.276   8.381  1.00 28.69           C  \\nATOM    177  CD  GLN A  25       5.931  42.574   9.744  1.00 30.92           C  \\nATOM    178  OE1 GLN A  25       6.537  43.143  10.657  1.00 31.89           O  \\nATOM    179  NE2 GLN A  25       4.664  42.169   9.867  1.00 32.35           N  \\nATOM    180  N   ASN A  26      10.764  40.482   6.795  1.00 22.13           N  \\nATOM    181  CA  ASN A  26      12.198  40.642   6.628  1.00 24.67           C  \\nATOM    182  C   ASN A  26      12.936  41.362   7.734  1.00 23.62           C  \\nATOM    183  O   ASN A  26      13.603  42.372   7.521  1.00 25.10           O  \\nATOM    184  CB  ASN A  26      12.522  41.369   5.315  1.00 27.68           C  \\nATOM    185  CG  ASN A  26      12.942  40.351   4.288  1.00 32.80           C  \\nATOM    186  OD1 ASN A  26      12.135  39.699   3.602  1.00 33.77           O  \\nATOM    187  ND2 ASN A  26      14.268  40.203   4.237  1.00 34.26           N  \\nATOM    188  N   HIS A  27      12.764  40.842   8.945  1.00 19.95           N  \\nATOM    189  CA  HIS A  27      13.593  41.266  10.057  1.00 17.99           C  \\nATOM    190  C   HIS A  27      13.441  40.299  11.215  1.00 16.84           C  \\nATOM    191  O   HIS A  27      12.411  39.623  11.344  1.00 14.72           O  \\nATOM    192  CB  HIS A  27      13.241  42.680  10.551  1.00 19.80           C  \\nATOM    193  CG  HIS A  27      11.926  42.908  11.258  1.00 20.77           C  \\nATOM    194  ND1 HIS A  27      11.936  43.314  12.510  1.00 22.17           N  \\nATOM    195  CD2 HIS A  27      10.674  42.768  10.729  1.00 22.32           C  \\nATOM    196  CE1 HIS A  27      10.657  43.430  12.785  1.00 23.86           C  \\nATOM    197  NE2 HIS A  27       9.905  43.110  11.738  1.00 23.97           N  \\nATOM    198  N   PHE A  28      14.516  40.222  12.012  1.00 15.44           N  \\nATOM    199  CA  PHE A  28      14.528  39.413  13.223  1.00 13.89           C  \\nATOM    200  C   PHE A  28      14.026  40.196  14.440  1.00 15.33           C  \\nATOM    201  O   PHE A  28      14.528  41.255  14.786  1.00 16.80           O  \\nATOM    202  CB  PHE A  28      15.938  38.906  13.470  1.00 12.73           C  \\nATOM    203  CG  PHE A  28      16.083  38.150  14.786  1.00 13.35           C  \\nATOM    204  CD1 PHE A  28      15.354  36.990  15.012  1.00 13.80           C  \\nATOM    205  CD2 PHE A  28      16.974  38.615  15.740  1.00 14.54           C  \\nATOM    206  CE1 PHE A  28      15.534  36.310  16.194  1.00 14.45           C  \\nATOM    207  CE2 PHE A  28      17.147  37.925  16.920  1.00 14.61           C  \\nATOM    208  CZ  PHE A  28      16.420  36.768  17.137  1.00 14.65           C  \\nATOM    209  N   VAL A  29      12.970  39.697  15.089  1.00 17.22           N  \\nATOM    210  CA  VAL A  29      12.467  40.313  16.294  1.00 19.11           C  \\nATOM    211  C   VAL A  29      13.127  39.606  17.476  1.00 19.36           C  \\nATOM    212  O   VAL A  29      13.245  38.376  17.605  1.00 18.14           O  \\nATOM    213  CB  VAL A  29      10.894  40.205  16.412  1.00 22.03           C  \\nATOM    214  CG1 VAL A  29      10.334  40.869  15.143  1.00 19.66           C  \\nATOM    215  CG2 VAL A  29      10.377  38.776  16.631  1.00 23.15           C  \\nATOM    216  N   ASP A  30      13.615  40.484  18.324  1.00 20.14           N  \\nATOM    217  CA  ASP A  30      14.229  40.019  19.518  1.00 24.04           C  \\nATOM    218  C   ASP A  30      13.676  40.746  20.721  1.00 25.93           C  \\nATOM    219  O   ASP A  30      14.244  41.709  21.238  1.00 29.26           O  \\nATOM    220  CB  ASP A  30      15.719  40.203  19.443  1.00 27.10           C  \\nATOM    221  CG  ASP A  30      16.389  39.731  20.732  1.00 32.46           C  \\nATOM    222  OD1 ASP A  30      15.878  38.859  21.464  1.00 34.28           O  \\nATOM    223  OD2 ASP A  30      17.458  40.264  21.002  1.00 36.46           O  \\nATOM    224  N   GLU A  31      12.472  40.347  21.076  1.00 26.14           N  \\nATOM    225  CA  GLU A  31      11.922  40.804  22.318  1.00 28.72           C  \\nATOM    226  C   GLU A  31      10.870  39.836  22.749  1.00 26.20           C  \\nATOM    227  O   GLU A  31      10.466  38.931  22.014  1.00 23.14           O  \\nATOM    228  CB  GLU A  31      11.337  42.210  22.204  1.00 33.25           C  \\nATOM    229  CG  GLU A  31      10.350  42.618  21.152  1.00 40.08           C  \\nATOM    230  CD  GLU A  31      10.572  44.107  20.887  1.00 44.71           C  \\nATOM    231  OE1 GLU A  31      10.930  44.444  19.755  1.00 47.50           O  \\nATOM    232  OE2 GLU A  31      10.420  44.928  21.806  1.00 47.89           O  \\nATOM    233  N   TYR A  32      10.550  40.044  24.028  1.00 22.26           N  \\nATOM    234  CA  TYR A  32       9.597  39.196  24.685  1.00 18.83           C  \\nATOM    235  C   TYR A  32       8.204  39.688  24.317  1.00 17.53           C  \\nATOM    236  O   TYR A  32       7.794  40.792  24.676  1.00 18.45           O  \\nATOM    237  CB  TYR A  32       9.909  39.259  26.199  1.00 16.84           C  \\nATOM    238  CG  TYR A  32       9.076  38.309  27.052  1.00 15.64           C  \\nATOM    239  CD1 TYR A  32       9.598  37.096  27.433  1.00 16.17           C  \\nATOM    240  CD2 TYR A  32       7.814  38.686  27.459  1.00 15.54           C  \\nATOM    241  CE1 TYR A  32       8.845  36.251  28.239  1.00 17.86           C  \\nATOM    242  CE2 TYR A  32       7.051  37.861  28.257  1.00 14.40           C  \\nATOM    243  CZ  TYR A  32       7.572  36.635  28.652  1.00 16.47           C  \\nATOM    244  OH  TYR A  32       6.821  35.789  29.485  1.00 13.59           O  \\nATOM    245  N   ASP A  33       7.461  38.841  23.604  1.00 13.43           N  \\nATOM    246  CA  ASP A  33       6.128  39.200  23.141  1.00 14.00           C  \\nATOM    247  C   ASP A  33       5.262  37.955  22.854  1.00 13.56           C  \\nATOM    248  O   ASP A  33       5.024  37.586  21.681  1.00 12.47           O  \\nATOM    249  CB  ASP A  33       6.271  40.072  21.864  1.00 15.92           C  \\nATOM    250  CG  ASP A  33       4.993  40.654  21.269  1.00 18.19           C  \\nATOM    251  OD1 ASP A  33       3.923  40.631  21.879  1.00 21.46           O  \\nATOM    252  OD2 ASP A  33       5.074  41.164  20.167  1.00 20.34           O  \\nATOM    253  N   PRO A  34       4.752  37.283  23.903  1.00 12.04           N  \\nATOM    254  CA  PRO A  34       3.949  36.070  23.788  1.00 11.09           C  \\nATOM    255  C   PRO A  34       2.720  36.248  22.895  1.00 11.76           C  \\nATOM    256  O   PRO A  34       1.953  37.220  23.006  1.00 12.63           O  \\nATOM    257  CB  PRO A  34       3.590  35.711  25.219  1.00 12.50           C  \\nATOM    258  CG  PRO A  34       4.643  36.418  26.033  1.00 12.14           C  \\nATOM    259  CD  PRO A  34       4.817  37.717  25.311  1.00 12.82           C  \\nATOM    260  N   THR A  35       2.554  35.282  21.972  1.00 11.00           N  \\nATOM    261  CA  THR A  35       1.421  35.320  21.033  1.00 11.54           C  \\nATOM    262  C   THR A  35       0.140  34.746  21.639  1.00 13.51           C  \\nATOM    263  O   THR A  35       0.174  33.820  22.450  1.00 12.45           O  \\nATOM    264  CB  THR A  35       1.730  34.518  19.751  1.00 11.34           C  \\nATOM    265  OG1 THR A  35       2.900  35.134  19.233  1.00 11.20           O  \\nATOM    266  CG2 THR A  35       0.706  34.611  18.637  1.00 13.10           C  \\nATOM    267  N   ILE A  36      -0.996  35.332  21.233  1.00 15.48           N  \\nATOM    268  CA  ILE A  36      -2.308  34.792  21.567  1.00 19.42           C  \\nATOM    269  C   ILE A  36      -2.663  34.079  20.274  1.00 17.36           C  \\nATOM    270  O   ILE A  36      -2.749  32.863  20.256  1.00 18.29           O  \\nATOM    271  CB  ILE A  36      -3.353  35.914  21.879  1.00 21.30           C  \\nATOM    272  CG1 ILE A  36      -2.930  36.823  23.072  1.00 24.32           C  \\nATOM    273  CG2 ILE A  36      -4.670  35.207  22.136  1.00 21.54           C  \\nATOM    274  CD1 ILE A  36      -2.494  36.167  24.413  1.00 24.70           C  \\nATOM    275  N   GLU A  37      -2.818  34.822  19.181  1.00 19.35           N  \\nATOM    276  CA  GLU A  37      -3.059  34.221  17.889  1.00 21.47           C  \\nATOM    277  C   GLU A  37      -2.957  35.362  16.897  1.00 23.77           C  \\nATOM    278  O   GLU A  37      -3.636  36.393  17.031  1.00 25.49           O  \\nATOM    279  CB  GLU A  37      -4.440  33.611  17.880  1.00 25.97           C  \\nATOM    280  CG  GLU A  37      -4.766  32.776  16.667  1.00 33.81           C  \\nATOM    281  CD  GLU A  37      -6.160  32.154  16.777  1.00 38.37           C  \\nATOM    282  OE1 GLU A  37      -7.095  32.664  16.149  1.00 37.86           O  \\nATOM    283  OE2 GLU A  37      -6.298  31.164  17.515  1.00 42.24           O  \\nATOM    284  N   ASP A  38      -2.070  35.212  15.911  1.00 19.96           N  \\nATOM    285  CA  ASP A  38      -1.899  36.178  14.839  1.00 17.94           C  \\nATOM    286  C   ASP A  38      -2.024  35.526  13.473  1.00 18.21           C  \\nATOM    287  O   ASP A  38      -1.582  34.393  13.294  1.00 16.96           O  \\nATOM    288  CB  ASP A  38      -0.540  36.833  14.965  1.00 18.83           C  \\nATOM    289  CG  ASP A  38      -0.507  37.817  16.133  1.00 20.08           C  \\nATOM    290  OD1 ASP A  38      -1.390  38.660  16.208  1.00 25.47           O  \\nATOM    291  OD2 ASP A  38       0.375  37.781  16.967  1.00 20.08           O  \\nATOM    292  N   SER A  39      -2.698  36.170  12.507  1.00 17.82           N  \\nATOM    293  CA  SER A  39      -2.691  35.658  11.133  1.00 18.16           C  \\nATOM    294  C   SER A  39      -2.012  36.627  10.171  1.00 16.95           C  \\nATOM    295  O   SER A  39      -2.053  37.853  10.333  1.00 15.80           O  \\nATOM    296  CB  SER A  39      -4.094  35.402  10.637  1.00 23.01           C  \\nATOM    297  OG  SER A  39      -4.873  36.571  10.754  1.00 27.13           O  \\nATOM    298  N   TYR A  40      -1.336  36.080   9.168  1.00 15.44           N  \\nATOM    299  CA  TYR A  40      -0.607  36.889   8.196  1.00 16.25           C  \\nATOM    300  C   TYR A  40      -1.018  36.383   6.830  1.00 17.06           C  \\nATOM    301  O   TYR A  40      -1.160  35.172   6.638  1.00 18.29           O  \\nATOM    302  CB  TYR A  40       0.909  36.724   8.370  1.00 16.59           C  \\nATOM    303  CG  TYR A  40       1.439  37.242   9.707  1.00 16.10           C  \\nATOM    304  CD1 TYR A  40       1.473  36.412  10.815  1.00 16.77           C  \\nATOM    305  CD2 TYR A  40       1.883  38.549   9.799  1.00 18.02           C  \\nATOM    306  CE1 TYR A  40       1.950  36.893  12.021  1.00 16.86           C  \\nATOM    307  CE2 TYR A  40       2.368  39.032  11.001  1.00 18.70           C  \\nATOM    308  CZ  TYR A  40       2.396  38.199  12.104  1.00 17.62           C  \\nATOM    309  OH  TYR A  40       2.853  38.695  13.304  1.00 18.37           O  \\nATOM    310  N   ARG A  41      -1.270  37.303   5.900  1.00 15.48           N  \\nATOM    311  CA  ARG A  41      -1.660  36.918   4.558  1.00 15.42           C  \\nATOM    312  C   ARG A  41      -0.730  37.606   3.586  1.00 16.76           C  \\nATOM    313  O   ARG A  41      -0.536  38.829   3.633  1.00 17.01           O  \\nATOM    314  CB  ARG A  41      -3.087  37.342   4.277  1.00 17.64           C  \\nATOM    315  CG  ARG A  41      -4.150  36.507   4.947  1.00 18.09           C  \\nATOM    316  CD  ARG A  41      -5.528  37.181   4.849  1.00 19.64           C  \\nATOM    317  NE  ARG A  41      -6.066  37.318   3.489  1.00 20.11           N  \\nATOM    318  CZ  ARG A  41      -6.660  36.329   2.803  1.00 22.17           C  \\nATOM    319  NH1 ARG A  41      -7.219  36.571   1.617  1.00 22.48           N  \\nATOM    320  NH2 ARG A  41      -6.711  35.090   3.274  1.00 23.55           N  \\nATOM    321  N   LYS A  42      -0.135  36.827   2.679  1.00 16.52           N  \\nATOM    322  CA  LYS A  42       0.835  37.405   1.758  1.00 16.59           C  \\nATOM    323  C   LYS A  42       0.740  36.746   0.387  1.00 16.04           C  \\nATOM    324  O   LYS A  42       0.794  35.528   0.307  1.00 14.74           O  \\nATOM    325  CB  LYS A  42       2.208  37.210   2.401  1.00 18.22           C  \\nATOM    326  CG  LYS A  42       3.388  37.784   1.669  1.00 22.68           C  \\nATOM    327  CD  LYS A  42       3.369  39.274   1.792  1.00 28.09           C  \\nATOM    328  CE  LYS A  42       4.695  39.876   1.317  1.00 31.70           C  \\nATOM    329  NZ  LYS A  42       4.828  39.827  -0.128  1.00 35.70           N  \\nATOM    330  N   GLN A  43       0.561  37.510  -0.697  1.00 16.89           N  \\nATOM    331  CA  GLN A  43       0.579  36.950  -2.047  1.00 19.10           C  \\nATOM    332  C   GLN A  43       2.026  36.729  -2.448  1.00 15.08           C  \\nATOM    333  O   GLN A  43       2.828  37.654  -2.362  1.00 15.72           O  \\nATOM    334  CB  GLN A  43      -0.116  37.908  -3.043  1.00 23.32           C  \\nATOM    335  CG  GLN A  43       0.543  38.332  -4.415  1.00 32.83           C  \\nATOM    336  CD  GLN A  43       1.081  37.332  -5.482  1.00 36.11           C  \\nATOM    337  OE1 GLN A  43       0.457  36.335  -5.865  1.00 37.78           O  \\nATOM    338  NE2 GLN A  43       2.271  37.606  -6.053  1.00 35.45           N  \\nATOM    339  N   VAL A  44       2.394  35.532  -2.922  1.00 12.79           N  \\nATOM    340  CA  VAL A  44       3.771  35.242  -3.311  1.00 11.03           C  \\nATOM    341  C   VAL A  44       3.724  34.425  -4.605  1.00 14.28           C  \\nATOM    342  O   VAL A  44       2.633  34.109  -5.090  1.00 14.90           O  \\nATOM    343  CB  VAL A  44       4.508  34.411  -2.223  1.00 13.21           C  \\nATOM    344  CG1 VAL A  44       4.669  35.282  -0.977  1.00 15.04           C  \\nATOM    345  CG2 VAL A  44       3.761  33.116  -1.889  1.00 12.89           C  \\nATOM    346  N   VAL A  45       4.879  34.127  -5.235  1.00 13.20           N  \\nATOM    347  CA  VAL A  45       4.938  33.277  -6.419  1.00 12.28           C  \\nATOM    348  C   VAL A  45       5.836  32.161  -5.970  1.00 11.08           C  \\nATOM    349  O   VAL A  45       6.988  32.447  -5.636  1.00 13.13           O  \\nATOM    350  CB  VAL A  45       5.587  33.979  -7.641  1.00 13.01           C  \\nATOM    351  CG1 VAL A  45       5.549  33.031  -8.810  1.00 14.51           C  \\nATOM    352  CG2 VAL A  45       4.787  35.173  -8.108  1.00 13.54           C  \\nATOM    353  N   ILE A  46       5.374  30.907  -5.915  1.00  9.89           N  \\nATOM    354  CA  ILE A  46       6.274  29.795  -5.491  1.00 11.03           C  \\nATOM    355  C   ILE A  46       6.369  28.800  -6.662  1.00  8.85           C  \\nATOM    356  O   ILE A  46       5.319  28.296  -7.059  1.00 10.25           O  \\nATOM    357  CB  ILE A  46       5.692  29.115  -4.203  1.00 10.64           C  \\nATOM    358  CG1 ILE A  46       5.652  30.096  -3.042  1.00 11.20           C  \\nATOM    359  CG2 ILE A  46       6.599  27.953  -3.757  1.00 12.93           C  \\nATOM    360  CD1 ILE A  46       4.937  29.536  -1.808  1.00  8.90           C  \\nATOM    361  N   ASP A  47       7.565  28.495  -7.224  1.00  9.41           N  \\nATOM    362  CA  ASP A  47       7.767  27.650  -8.426  1.00  9.70           C  \\nATOM    363  C   ASP A  47       6.838  28.031  -9.572  1.00 11.14           C  \\nATOM    364  O   ASP A  47       6.225  27.211 -10.248  1.00 14.09           O  \\nATOM    365  CB  ASP A  47       7.566  26.165  -8.105  1.00 10.73           C  \\nATOM    366  CG  ASP A  47       8.388  25.783  -6.874  1.00 13.52           C  \\nATOM    367  OD1 ASP A  47       7.805  25.291  -5.898  1.00 12.04           O  \\nATOM    368  OD2 ASP A  47       9.598  26.024  -6.872  1.00 11.31           O  \\nATOM    369  N   GLY A  48       6.724  29.353  -9.735  1.00 10.78           N  \\nATOM    370  CA  GLY A  48       5.930  29.961 -10.785  1.00 13.29           C  \\nATOM    371  C   GLY A  48       4.426  29.989 -10.548  1.00 14.28           C  \\nATOM    372  O   GLY A  48       3.677  30.512 -11.386  1.00 15.19           O  \\nATOM    373  N   GLU A  49       3.944  29.422  -9.439  1.00 13.41           N  \\nATOM    374  CA  GLU A  49       2.520  29.457  -9.143  1.00 13.90           C  \\nATOM    375  C   GLU A  49       2.223  30.700  -8.314  1.00 15.34           C  \\nATOM    376  O   GLU A  49       2.860  30.906  -7.275  1.00 14.56           O  \\nATOM    377  CB  GLU A  49       2.124  28.238  -8.340  1.00 15.50           C  \\nATOM    378  CG  GLU A  49       0.630  28.250  -7.990  1.00 18.10           C  \\nATOM    379  CD  GLU A  49       0.204  27.110  -7.100  1.00 19.13           C  \\nATOM    380  OE1 GLU A  49       0.717  27.004  -5.996  1.00 19.53           O  \\nATOM    381  OE2 GLU A  49      -0.630  26.321  -7.529  1.00 22.12           O  \\nATOM    382  N   THR A  50       1.267  31.536  -8.734  1.00 14.06           N  \\nATOM    383  CA  THR A  50       0.901  32.697  -7.927  1.00 16.15           C  \\nATOM    384  C   THR A  50      -0.001  32.169  -6.819  1.00 13.35           C  \\nATOM    385  O   THR A  50      -0.938  31.409  -7.046  1.00 15.13           O  \\nATOM    386  CB  THR A  50       0.203  33.755  -8.846  1.00 17.88           C  \\nATOM    387  OG1 THR A  50      -0.245  34.813  -8.012  1.00 24.33           O  \\nATOM    388  CG2 THR A  50      -1.028  33.236  -9.518  1.00 19.78           C  \\nATOM    389  N   CYS A  51       0.306  32.442  -5.563  1.00 13.28           N  \\nATOM    390  CA  CYS A  51      -0.566  31.959  -4.508  1.00 15.16           C  \\nATOM    391  C   CYS A  51      -0.579  32.892  -3.303  1.00 14.00           C  \\nATOM    392  O   CYS A  51       0.161  33.874  -3.231  1.00 11.81           O  \\nATOM    393  CB  CYS A  51      -0.132  30.537  -4.079  1.00 14.08           C  \\nATOM    394  SG  CYS A  51       1.597  30.409  -3.575  1.00 16.09           S  \\nATOM    395  N   LEU A  52      -1.521  32.609  -2.405  1.00 13.77           N  \\nATOM    396  CA  LEU A  52      -1.573  33.397  -1.199  1.00 16.01           C  \\nATOM    397  C   LEU A  52      -1.260  32.497  -0.025  1.00 14.53           C  \\nATOM    398  O   LEU A  52      -1.868  31.443   0.113  1.00 16.05           O  \\nATOM    399  CB  LEU A  52      -2.957  33.986  -1.014  1.00 19.71           C  \\nATOM    400  CG  LEU A  52      -3.122  34.885   0.214  1.00 20.84           C  \\nATOM    401  CD1 LEU A  52      -3.070  36.329  -0.220  1.00 21.38           C  \\nATOM    402  CD2 LEU A  52      -4.420  34.526   0.923  1.00 20.90           C  \\nATOM    403  N   LEU A  53      -0.319  32.904   0.800  1.00 12.41           N  \\nATOM    404  CA  LEU A  53       0.019  32.232   2.048  1.00 13.70           C  \\nATOM    405  C   LEU A  53      -0.841  32.852   3.148  1.00 13.85           C  \\nATOM    406  O   LEU A  53      -0.733  34.056   3.413  1.00 12.82           O  \\nATOM    407  CB  LEU A  53       1.485  32.464   2.400  1.00 14.51           C  \\nATOM    408  CG  LEU A  53       2.707  31.589   2.031  1.00 19.29           C  \\nATOM    409  CD1 LEU A  53       2.394  30.553   0.962  1.00 18.56           C  \\nATOM    410  CD2 LEU A  53       3.843  32.557   1.741  1.00 20.00           C  \\nATOM    411  N   ASP A  54      -1.699  32.023   3.751  1.00 11.96           N  \\nATOM    412  CA  ASP A  54      -2.476  32.445   4.884  1.00 15.07           C  \\nATOM    413  C   ASP A  54      -1.882  31.675   6.055  1.00 12.67           C  \\nATOM    414  O   ASP A  54      -2.099  30.477   6.234  1.00 13.99           O  \\nATOM    415  CB  ASP A  54      -3.930  32.088   4.644  1.00 19.33           C  \\nATOM    416  CG  ASP A  54      -4.826  32.609   5.746  1.00 24.24           C  \\nATOM    417  OD1 ASP A  54      -4.412  32.652   6.897  1.00 28.17           O  \\nATOM    418  OD2 ASP A  54      -5.958  32.979   5.450  1.00 27.59           O  \\nATOM    419  N   ILE A  55      -1.127  32.389   6.880  1.00 11.78           N  \\nATOM    420  CA  ILE A  55      -0.346  31.769   7.935  1.00 12.15           C  \\nATOM    421  C   ILE A  55      -0.953  32.099   9.296  1.00 13.10           C  \\nATOM    422  O   ILE A  55      -1.195  33.271   9.587  1.00 13.84           O  \\nATOM    423  CB  ILE A  55       1.109  32.282   7.931  1.00 12.42           C  \\nATOM    424  CG1 ILE A  55       1.859  31.979   6.653  1.00 13.99           C  \\nATOM    425  CG2 ILE A  55       1.859  31.505   9.018  1.00 13.24           C  \\nATOM    426  CD1 ILE A  55       3.176  32.768   6.497  1.00 15.68           C  \\nATOM    427  N   LEU A  56      -1.176  31.083  10.134  1.00 11.64           N  \\nATOM    428  CA  LEU A  56      -1.642  31.341  11.490  1.00 11.98           C  \\nATOM    429  C   LEU A  56      -0.524  31.027  12.431  1.00  9.99           C  \\nATOM    430  O   LEU A  56      -0.017  29.895  12.447  1.00  9.72           O  \\nATOM    431  CB  LEU A  56      -2.836  30.463  11.896  1.00 13.80           C  \\nATOM    432  CG  LEU A  56      -3.470  30.741  13.297  1.00 16.13           C  \\nATOM    433  CD1 LEU A  56      -4.347  31.987  13.260  1.00 15.28           C  \\nATOM    434  CD2 LEU A  56      -4.321  29.580  13.692  1.00 16.84           C  \\nATOM    435  N   ASP A  57      -0.188  32.084  13.182  1.00 10.40           N  \\nATOM    436  CA  ASP A  57       0.815  32.014  14.219  1.00 10.30           C  \\nATOM    437  C   ASP A  57       0.127  31.812  15.571  1.00 11.75           C  \\nATOM    438  O   ASP A  57      -0.652  32.631  16.053  1.00 12.91           O  \\nATOM    439  CB  ASP A  57       1.602  33.296  14.173  1.00 10.66           C  \\nATOM    440  CG  ASP A  57       2.778  33.336  15.132  1.00 11.39           C  \\nATOM    441  OD1 ASP A  57       3.306  32.298  15.535  1.00 11.07           O  \\nATOM    442  OD2 ASP A  57       3.168  34.448  15.455  1.00 13.15           O  \\nATOM    443  N   THR A  58       0.447  30.723  16.244  1.00 12.55           N  \\nATOM    444  CA  THR A  58      -0.324  30.325  17.416  1.00 12.98           C  \\nATOM    445  C   THR A  58       0.368  30.541  18.758  1.00 13.48           C  \\nATOM    446  O   THR A  58       1.568  30.812  18.815  1.00 14.00           O  \\nATOM    447  CB  THR A  58      -0.698  28.851  17.219  1.00  9.54           C  \\nATOM    448  OG1 THR A  58       0.491  28.084  17.369  1.00 11.40           O  \\nATOM    449  CG2 THR A  58      -1.313  28.597  15.848  1.00 12.27           C  \\nATOM    450  N   ALA A  59      -0.371  30.382  19.868  1.00 14.38           N  \\nATOM    451  CA  ALA A  59       0.230  30.510  21.194  1.00 14.79           C  \\nATOM    452  C   ALA A  59       0.928  29.210  21.608  1.00 15.41           C  \\nATOM    453  O   ALA A  59       0.448  28.106  21.324  1.00 13.07           O  \\nATOM    454  CB  ALA A  59      -0.845  30.818  22.238  1.00 15.47           C  \\nATOM    455  N   GLY A  60       2.084  29.341  22.271  1.00 17.71           N  \\nATOM    456  CA  GLY A  60       2.807  28.208  22.814  1.00 24.18           C  \\nATOM    457  C   GLY A  60       2.190  27.751  24.130  1.00 31.21           C  \\nATOM    458  O   GLY A  60       2.538  26.688  24.627  1.00 30.44           O  \\nATOM    459  N   GLN A  61       1.260  28.503  24.726  1.00 39.50           N  \\nATOM    460  CA  GLN A  61       0.671  28.099  25.994  1.00 48.84           C  \\nATOM    461  C   GLN A  61      -0.683  27.395  25.985  1.00 51.52           C  \\nATOM    462  O   GLN A  61      -1.606  27.780  25.276  1.00 52.17           O  \\nATOM    463  CB  GLN A  61       0.605  29.327  26.874  1.00 51.56           C  \\nATOM    464  CG  GLN A  61       0.871  28.861  28.312  1.00 56.74           C  \\nATOM    465  CD  GLN A  61       1.506  29.937  29.177  1.00 58.74           C  \\nATOM    466  OE1 GLN A  61       2.647  29.795  29.616  1.00 60.12           O  \\nATOM    467  NE2 GLN A  61       0.783  31.031  29.429  1.00 60.18           N  \\nATOM    468  N   GLU A  62      -0.804  26.349  26.818  1.00 57.40           N  \\nATOM    469  CA  GLU A  62      -2.008  25.527  26.927  1.00 61.96           C  \\nATOM    470  C   GLU A  62      -3.185  26.196  27.631  1.00 62.20           C  \\nATOM    471  O   GLU A  62      -4.138  25.496  27.975  1.00 62.85           O  \\nATOM    472  CB  GLU A  62      -1.629  24.183  27.632  1.00 65.18           C  \\nATOM    473  CG  GLU A  62      -2.622  22.966  27.761  1.00 68.22           C  \\nATOM    474  CD  GLU A  62      -3.032  22.132  26.528  1.00 70.25           C  \\nATOM    475  OE1 GLU A  62      -2.710  22.494  25.395  1.00 71.45           O  \\nATOM    476  OE2 GLU A  62      -3.691  21.101  26.689  1.00 70.00           O  \\nATOM    477  N   GLU A  63      -3.206  27.520  27.904  1.00 62.78           N  \\nATOM    478  CA  GLU A  63      -4.496  28.139  28.254  1.00 63.62           C  \\nATOM    479  C   GLU A  63      -5.457  27.854  27.085  1.00 63.71           C  \\nATOM    480  O   GLU A  63      -6.688  27.814  27.211  1.00 62.97           O  \\nATOM    481  CB  GLU A  63      -4.372  29.666  28.475  1.00 65.13           C  \\nATOM    482  CG  GLU A  63      -3.456  30.553  27.607  1.00 66.91           C  \\nATOM    483  CD  GLU A  63      -4.012  31.235  26.345  1.00 68.05           C  \\nATOM    484  OE1 GLU A  63      -5.220  31.462  26.220  1.00 68.64           O  \\nATOM    485  OE2 GLU A  63      -3.198  31.568  25.481  1.00 67.62           O  \\nATOM    486  N   TYR A  64      -4.801  27.606  25.934  1.00 63.36           N  \\nATOM    487  CA  TYR A  64      -5.420  27.092  24.743  1.00 63.20           C  \\nATOM    488  C   TYR A  64      -5.905  25.653  24.875  1.00 61.76           C  \\nATOM    489  O   TYR A  64      -7.120  25.486  24.858  1.00 63.04           O  \\nATOM    490  CB  TYR A  64      -4.437  27.208  23.613  1.00 64.83           C  \\nATOM    491  CG  TYR A  64      -4.736  28.459  22.818  1.00 66.98           C  \\nATOM    492  CD1 TYR A  64      -4.300  29.686  23.267  1.00 67.55           C  \\nATOM    493  CD2 TYR A  64      -5.459  28.355  21.646  1.00 67.97           C  \\nATOM    494  CE1 TYR A  64      -4.587  30.825  22.536  1.00 69.49           C  \\nATOM    495  CE2 TYR A  64      -5.752  29.488  20.911  1.00 69.29           C  \\nATOM    496  CZ  TYR A  64      -5.313  30.718  21.359  1.00 70.05           C  \\nATOM    497  OH  TYR A  64      -5.589  31.852  20.614  1.00 70.67           O  \\nATOM    498  N   SER A  65      -5.095  24.590  24.997  1.00 58.67           N  \\nATOM    499  CA  SER A  65      -5.617  23.230  25.127  1.00 56.93           C  \\nATOM    500  C   SER A  65      -6.614  22.757  24.063  1.00 55.33           C  \\nATOM    501  O   SER A  65      -6.321  22.936  22.876  1.00 54.74           O  \\nATOM    502  CB  SER A  65      -6.190  23.122  26.552  1.00 57.99           C  \\nATOM    503  OG  SER A  65      -7.251  24.011  26.900  1.00 59.80           O  \\nATOM    504  N   ALA A  66      -7.762  22.119  24.359  1.00 52.33           N  \\nATOM    505  CA  ALA A  66      -8.766  21.824  23.335  1.00 48.80           C  \\nATOM    506  C   ALA A  66      -9.138  23.050  22.492  1.00 45.71           C  \\nATOM    507  O   ALA A  66      -9.751  22.862  21.449  1.00 44.33           O  \\nATOM    508  CB  ALA A  66     -10.043  21.300  23.971  1.00 49.14           C  \\nATOM    509  N   MET A  67      -8.829  24.310  22.875  1.00 42.85           N  \\nATOM    510  CA  MET A  67      -8.984  25.465  21.967  1.00 40.01           C  \\nATOM    511  C   MET A  67      -8.093  25.362  20.719  1.00 36.79           C  \\nATOM    512  O   MET A  67      -8.221  26.184  19.805  1.00 36.08           O  \\nATOM    513  CB  MET A  67      -8.594  26.804  22.581  1.00 42.50           C  \\nATOM    514  CG  MET A  67      -9.359  27.330  23.772  1.00 45.93           C  \\nATOM    515  SD  MET A  67     -11.140  27.396  23.483  1.00 48.74           S  \\nATOM    516  CE  MET A  67     -11.526  25.880  24.321  1.00 47.64           C  \\nATOM    517  N   ARG A  68      -7.128  24.418  20.665  1.00 31.59           N  \\nATOM    518  CA  ARG A  68      -6.322  24.217  19.467  1.00 26.34           C  \\nATOM    519  C   ARG A  68      -7.140  23.473  18.424  1.00 23.68           C  \\nATOM    520  O   ARG A  68      -6.846  23.617  17.231  1.00 21.93           O  \\nATOM    521  CB  ARG A  68      -5.057  23.394  19.745  1.00 25.86           C  \\nATOM    522  CG  ARG A  68      -4.161  23.898  20.869  1.00 29.36           C  \\nATOM    523  CD  ARG A  68      -2.881  23.061  20.998  1.00 32.88           C  \\nATOM    524  NE  ARG A  68      -2.221  22.990  22.315  1.00 36.64           N  \\nATOM    525  CZ  ARG A  68      -1.700  24.031  23.001  1.00 39.66           C  \\nATOM    526  NH1 ARG A  68      -1.903  25.315  22.664  1.00 42.06           N  \\nATOM    527  NH2 ARG A  68      -0.933  23.798  24.071  1.00 39.61           N  \\nATOM    528  N   ASP A  69      -8.188  22.721  18.793  1.00 19.14           N  \\nATOM    529  CA  ASP A  69      -8.880  21.905  17.822  1.00 20.43           C  \\nATOM    530  C   ASP A  69      -9.437  22.623  16.617  1.00 22.16           C  \\nATOM    531  O   ASP A  69      -9.453  22.109  15.488  1.00 19.58           O  \\nATOM    532  CB  ASP A  69     -10.025  21.148  18.478  1.00 20.58           C  \\nATOM    533  CG  ASP A  69      -9.598  19.935  19.303  1.00 22.81           C  \\nATOM    534  OD1 ASP A  69      -8.409  19.708  19.538  1.00 24.59           O  \\nATOM    535  OD2 ASP A  69     -10.475  19.176  19.715  1.00 25.28           O  \\nATOM    536  N   GLN A  70      -9.834  23.874  16.840  1.00 23.26           N  \\nATOM    537  CA  GLN A  70     -10.451  24.609  15.759  1.00 23.53           C  \\nATOM    538  C   GLN A  70      -9.437  24.925  14.667  1.00 21.20           C  \\nATOM    539  O   GLN A  70      -9.653  24.591  13.493  1.00 20.90           O  \\nATOM    540  CB  GLN A  70     -11.071  25.881  16.341  1.00 27.73           C  \\nATOM    541  CG  GLN A  70     -12.498  26.019  15.832  1.00 34.11           C  \\nATOM    542  CD  GLN A  70     -13.103  27.407  16.008  1.00 37.87           C  \\nATOM    543  OE1 GLN A  70     -12.532  28.284  16.672  1.00 39.77           O  \\nATOM    544  NE2 GLN A  70     -14.283  27.620  15.406  1.00 37.89           N  \\nATOM    545  N   TYR A  71      -8.292  25.515  15.050  1.00 19.09           N  \\nATOM    546  CA  TYR A  71      -7.329  25.873  14.030  1.00 17.69           C  \\nATOM    547  C   TYR A  71      -6.648  24.658  13.416  1.00 17.24           C  \\nATOM    548  O   TYR A  71      -6.247  24.675  12.245  1.00 17.25           O  \\nATOM    549  CB  TYR A  71      -6.302  26.868  14.625  1.00 17.87           C  \\nATOM    550  CG  TYR A  71      -5.311  26.480  15.725  1.00 18.90           C  \\nATOM    551  CD1 TYR A  71      -4.355  25.510  15.492  1.00 19.74           C  \\nATOM    552  CD2 TYR A  71      -5.328  27.162  16.940  1.00 19.21           C  \\nATOM    553  CE1 TYR A  71      -3.402  25.212  16.450  1.00 20.38           C  \\nATOM    554  CE2 TYR A  71      -4.375  26.868  17.901  1.00 21.93           C  \\nATOM    555  CZ  TYR A  71      -3.422  25.897  17.645  1.00 21.91           C  \\nATOM    556  OH  TYR A  71      -2.463  25.590  18.588  1.00 26.44           O  \\nATOM    557  N   MET A  72      -6.557  23.559  14.166  1.00 14.81           N  \\nATOM    558  CA  MET A  72      -5.958  22.393  13.563  1.00 18.59           C  \\nATOM    559  C   MET A  72      -6.849  21.778  12.472  1.00 19.38           C  \\nATOM    560  O   MET A  72      -6.385  21.198  11.492  1.00 17.67           O  \\nATOM    561  CB  MET A  72      -5.658  21.386  14.669  1.00 17.39           C  \\nATOM    562  CG  MET A  72      -4.479  21.796  15.514  1.00 16.35           C  \\nATOM    563  SD  MET A  72      -4.146  20.600  16.820  1.00 17.04           S  \\nATOM    564  CE  MET A  72      -2.635  21.259  17.460  1.00 17.72           C  \\nATOM    565  N   ARG A  73      -8.174  21.931  12.626  1.00 20.31           N  \\nATOM    566  CA  ARG A  73      -9.120  21.350  11.701  1.00 22.24           C  \\nATOM    567  C   ARG A  73      -9.087  22.084  10.370  1.00 20.94           C  \\nATOM    568  O   ARG A  73      -9.183  21.450   9.326  1.00 20.81           O  \\nATOM    569  CB  ARG A  73     -10.499  21.414  12.369  1.00 25.29           C  \\nATOM    570  CG  ARG A  73     -11.553  20.499  11.767  1.00 30.04           C  \\nATOM    571  CD  ARG A  73     -12.775  20.429  12.687  1.00 31.86           C  \\nATOM    572  NE  ARG A  73     -12.483  19.702  13.917  1.00 34.70           N  \\nATOM    573  CZ  ARG A  73     -12.894  20.122  15.130  1.00 36.49           C  \\nATOM    574  NH1 ARG A  73     -12.579  19.402  16.196  1.00 36.93           N  \\nATOM    575  NH2 ARG A  73     -13.609  21.239  15.311  1.00 38.17           N  \\nATOM    576  N   THR A  74      -8.868  23.401  10.413  1.00 21.33           N  \\nATOM    577  CA  THR A  74      -8.813  24.240   9.219  1.00 24.99           C  \\nATOM    578  C   THR A  74      -7.436  24.351   8.544  1.00 23.10           C  \\nATOM    579  O   THR A  74      -7.367  24.738   7.371  1.00 22.59           O  \\nATOM    580  CB  THR A  74      -9.289  25.676   9.538  1.00 25.73           C  \\nATOM    581  OG1 THR A  74      -8.351  26.302  10.409  1.00 31.08           O  \\nATOM    582  CG2 THR A  74     -10.657  25.644  10.201  1.00 27.58           C  \\nATOM    583  N   GLY A  75      -6.345  24.084   9.289  1.00 19.58           N  \\nATOM    584  CA  GLY A  75      -5.013  24.114   8.730  1.00 15.79           C  \\nATOM    585  C   GLY A  75      -4.823  23.023   7.713  1.00 13.66           C  \\nATOM    586  O   GLY A  75      -5.245  21.880   7.884  1.00 14.69           O  \\nATOM    587  N   GLU A  76      -4.228  23.360   6.573  1.00 13.14           N  \\nATOM    588  CA  GLU A  76      -3.914  22.363   5.546  1.00 12.99           C  \\nATOM    589  C   GLU A  76      -2.514  21.740   5.753  1.00 12.22           C  \\nATOM    590  O   GLU A  76      -2.293  20.576   5.426  1.00 13.59           O  \\nATOM    591  CB  GLU A  76      -3.999  23.014   4.145  1.00 13.07           C  \\nATOM    592  CG  GLU A  76      -5.387  23.377   3.676  1.00 14.35           C  \\nATOM    593  CD  GLU A  76      -5.261  24.069   2.323  1.00 20.40           C  \\nATOM    594  OE1 GLU A  76      -5.461  23.403   1.288  1.00 24.32           O  \\nATOM    595  OE2 GLU A  76      -4.913  25.250   2.284  1.00 18.82           O  \\nATOM    596  N   GLY A  77      -1.553  22.503   6.301  1.00 10.73           N  \\nATOM    597  CA  GLY A  77      -0.216  22.018   6.570  1.00  8.07           C  \\nATOM    598  C   GLY A  77       0.302  22.689   7.819  1.00  8.63           C  \\nATOM    599  O   GLY A  77      -0.066  23.823   8.127  1.00  8.55           O  \\nATOM    600  N   PHE A  78       1.248  22.013   8.479  1.00  9.15           N  \\nATOM    601  CA  PHE A  78       1.754  22.468   9.773  1.00  8.87           C  \\nATOM    602  C   PHE A  78       3.268  22.520   9.854  1.00  9.51           C  \\nATOM    603  O   PHE A  78       3.930  21.542   9.486  1.00  9.15           O  \\nATOM    604  CB  PHE A  78       1.247  21.532  10.904  1.00  9.44           C  \\nATOM    605  CG  PHE A  78      -0.267  21.472  11.000  1.00  9.66           C  \\nATOM    606  CD1 PHE A  78      -0.906  22.310  11.901  1.00 10.71           C  \\nATOM    607  CD2 PHE A  78      -0.994  20.645  10.152  1.00 10.20           C  \\nATOM    608  CE1 PHE A  78      -2.295  22.338  11.940  1.00 10.61           C  \\nATOM    609  CE2 PHE A  78      -2.378  20.678  10.192  1.00 12.55           C  \\nATOM    610  CZ  PHE A  78      -3.022  21.529  11.081  1.00 12.80           C  \\nATOM    611  N   LEU A  79       3.813  23.636  10.354  1.00  8.72           N  \\nATOM    612  CA  LEU A  79       5.230  23.626  10.699  1.00  9.93           C  \\nATOM    613  C   LEU A  79       5.279  23.281  12.181  1.00  9.09           C  \\nATOM    614  O   LEU A  79       4.701  24.008  12.988  1.00 11.58           O  \\nATOM    615  CB  LEU A  79       5.921  24.970  10.567  1.00 12.80           C  \\nATOM    616  CG  LEU A  79       6.155  25.659   9.249  1.00 16.39           C  \\nATOM    617  CD1 LEU A  79       7.007  26.911   9.544  1.00 17.02           C  \\nATOM    618  CD2 LEU A  79       6.904  24.756   8.269  1.00 16.98           C  \\nATOM    619  N   CYS A  80       5.950  22.219  12.579  1.00  9.14           N  \\nATOM    620  CA  CYS A  80       6.074  21.821  13.981  1.00 11.81           C  \\nATOM    621  C   CYS A  80       7.448  22.237  14.434  1.00 11.53           C  \\nATOM    622  O   CYS A  80       8.435  21.640  14.000  1.00 12.21           O  \\nATOM    623  CB  CYS A  80       5.903  20.318  14.106  1.00 13.73           C  \\nATOM    624  SG  CYS A  80       4.158  19.900  13.837  1.00 18.26           S  \\nATOM    625  N   VAL A  81       7.486  23.282  15.269  1.00  7.57           N  \\nATOM    626  CA  VAL A  81       8.730  23.956  15.604  1.00  8.45           C  \\nATOM    627  C   VAL A  81       9.248  23.620  16.996  1.00  8.90           C  \\nATOM    628  O   VAL A  81       8.451  23.680  17.939  1.00  8.49           O  \\nATOM    629  CB  VAL A  81       8.544  25.516  15.507  1.00  7.40           C  \\nATOM    630  CG1 VAL A  81       9.917  26.196  15.588  1.00  9.42           C  \\nATOM    631  CG2 VAL A  81       8.004  25.954  14.167  1.00  8.59           C  \\nATOM    632  N   PHE A  82      10.546  23.237  17.119  1.00  9.49           N  \\nATOM    633  CA  PHE A  82      11.195  23.202  18.442  1.00  9.09           C  \\nATOM    634  C   PHE A  82      12.511  23.994  18.350  1.00  9.83           C  \\nATOM    635  O   PHE A  82      12.914  24.440  17.266  1.00  9.02           O  \\nATOM    636  CB  PHE A  82      11.482  21.755  18.902  1.00  9.76           C  \\nATOM    637  CG  PHE A  82      12.536  21.062  18.071  1.00 10.09           C  \\nATOM    638  CD1 PHE A  82      13.867  21.099  18.486  1.00 10.70           C  \\nATOM    639  CD2 PHE A  82      12.192  20.412  16.889  1.00 10.93           C  \\nATOM    640  CE1 PHE A  82      14.838  20.483  17.699  1.00 11.12           C  \\nATOM    641  CE2 PHE A  82      13.181  19.807  16.125  1.00 10.73           C  \\nATOM    642  CZ  PHE A  82      14.505  19.839  16.523  1.00 10.35           C  \\nATOM    643  N   ALA A  83      13.128  24.269  19.516  1.00  9.75           N  \\nATOM    644  CA  ALA A  83      14.407  24.963  19.569  1.00 10.32           C  \\nATOM    645  C   ALA A  83      15.509  23.960  19.916  1.00  9.48           C  \\nATOM    646  O   ALA A  83      15.362  23.142  20.839  1.00  9.61           O  \\nATOM    647  CB  ALA A  83      14.354  26.083  20.620  1.00  9.75           C  \\nATOM    648  N   ILE A  84      16.624  24.042  19.160  1.00 11.41           N  \\nATOM    649  CA  ILE A  84      17.714  23.070  19.299  1.00 13.15           C  \\nATOM    650  C   ILE A  84      18.446  23.223  20.623  1.00 14.57           C  \\nATOM    651  O   ILE A  84      19.234  22.347  20.971  1.00 13.94           O  \\nATOM    652  CB  ILE A  84      18.775  23.151  18.160  1.00 13.29           C  \\nATOM    653  CG1 ILE A  84      19.403  24.509  17.878  1.00 14.90           C  \\nATOM    654  CG2 ILE A  84      18.018  22.701  16.933  1.00 13.09           C  \\nATOM    655  CD1 ILE A  84      20.578  24.957  18.763  1.00 17.59           C  \\nATOM    656  N   ASN A  85      18.200  24.290  21.390  1.00 13.61           N  \\nATOM    657  CA  ASN A  85      18.794  24.394  22.720  1.00 15.52           C  \\nATOM    658  C   ASN A  85      17.758  24.156  23.828  1.00 15.49           C  \\nATOM    659  O   ASN A  85      17.988  24.519  24.999  1.00 16.13           O  \\nATOM    660  CB  ASN A  85      19.446  25.788  22.885  1.00 15.16           C  \\nATOM    661  CG  ASN A  85      18.464  26.946  22.904  1.00 16.42           C  \\nATOM    662  OD1 ASN A  85      17.355  26.878  22.362  1.00 16.01           O  \\nATOM    663  ND2 ASN A  85      18.808  28.030  23.587  1.00 16.83           N  \\nATOM    664  N   ASN A  86      16.616  23.515  23.509  1.00 14.72           N  \\nATOM    665  CA  ASN A  86      15.553  23.303  24.483  1.00 13.01           C  \\nATOM    666  C   ASN A  86      14.987  21.901  24.309  1.00 13.17           C  \\nATOM    667  O   ASN A  86      14.080  21.668  23.520  1.00 13.04           O  \\nATOM    668  CB  ASN A  86      14.478  24.367  24.256  1.00 16.51           C  \\nATOM    669  CG  ASN A  86      13.267  24.415  25.187  1.00 20.74           C  \\nATOM    670  OD1 ASN A  86      12.370  25.241  25.018  1.00 25.41           O  \\nATOM    671  ND2 ASN A  86      13.123  23.596  26.206  1.00 19.96           N  \\nATOM    672  N   THR A  87      15.488  20.950  25.103  1.00 13.80           N  \\nATOM    673  CA  THR A  87      15.095  19.553  25.026  1.00 15.40           C  \\nATOM    674  C   THR A  87      13.589  19.316  25.266  1.00 14.40           C  \\nATOM    675  O   THR A  87      12.953  18.535  24.546  1.00 14.11           O  \\nATOM    676  CB  THR A  87      15.983  18.791  26.046  1.00 16.19           C  \\nATOM    677  OG1 THR A  87      17.325  18.908  25.571  1.00 18.22           O  \\nATOM    678  CG2 THR A  87      15.626  17.327  26.180  1.00 17.82           C  \\nATOM    679  N   LYS A  88      12.983  19.958  26.272  1.00 14.42           N  \\nATOM    680  CA  LYS A  88      11.552  19.842  26.535  1.00 15.84           C  \\nATOM    681  C   LYS A  88      10.726  20.232  25.278  1.00 15.06           C  \\nATOM    682  O   LYS A  88       9.722  19.582  25.003  1.00 14.61           O  \\nATOM    683  CB  LYS A  88      11.207  20.766  27.717  1.00 19.53           C  \\nATOM    684  CG  LYS A  88       9.937  20.401  28.457  1.00 27.71           C  \\nATOM    685  CD  LYS A  88       9.585  21.452  29.529  1.00 33.26           C  \\nATOM    686  CE  LYS A  88       9.061  22.762  28.877  1.00 36.79           C  \\nATOM    687  NZ  LYS A  88       8.642  23.763  29.852  1.00 37.29           N  \\nATOM    688  N   SER A  89      11.119  21.259  24.475  1.00 12.36           N  \\nATOM    689  CA  SER A  89      10.325  21.666  23.309  1.00 11.31           C  \\nATOM    690  C   SER A  89      10.371  20.572  22.268  1.00 12.51           C  \\nATOM    691  O   SER A  89       9.397  20.388  21.547  1.00 12.88           O  \\nATOM    692  CB  SER A  89      10.844  22.951  22.674  1.00 11.64           C  \\nATOM    693  OG  SER A  89      12.160  22.900  22.124  1.00 11.11           O  \\nATOM    694  N   PHE A  90      11.504  19.842  22.204  1.00 11.27           N  \\nATOM    695  CA  PHE A  90      11.680  18.751  21.262  1.00 12.91           C  \\nATOM    696  C   PHE A  90      10.760  17.601  21.667  1.00 14.37           C  \\nATOM    697  O   PHE A  90      10.034  17.057  20.839  1.00 13.24           O  \\nATOM    698  CB  PHE A  90      13.161  18.312  21.260  1.00 12.34           C  \\nATOM    699  CG  PHE A  90      13.411  17.120  20.351  1.00 14.41           C  \\nATOM    700  CD1 PHE A  90      13.890  15.948  20.903  1.00 15.10           C  \\nATOM    701  CD2 PHE A  90      13.136  17.206  18.990  1.00 14.81           C  \\nATOM    702  CE1 PHE A  90      14.093  14.857  20.088  1.00 16.20           C  \\nATOM    703  CE2 PHE A  90      13.340  16.110  18.179  1.00 14.81           C  \\nATOM    704  CZ  PHE A  90      13.818  14.935  18.731  1.00 15.39           C  \\nATOM    705  N   GLU A  91      10.725  17.231  22.949  1.00 14.40           N  \\nATOM    706  CA  GLU A  91       9.863  16.146  23.407  1.00 15.78           C  \\nATOM    707  C   GLU A  91       8.374  16.466  23.301  1.00 16.18           C  \\nATOM    708  O   GLU A  91       7.533  15.582  23.114  1.00 17.50           O  \\nATOM    709  CB  GLU A  91      10.226  15.846  24.835  1.00 19.84           C  \\nATOM    710  CG  GLU A  91      11.654  15.386  25.074  1.00 24.49           C  \\nATOM    711  CD  GLU A  91      12.090  15.310  26.537  1.00 27.97           C  \\nATOM    712  OE1 GLU A  91      13.199  14.842  26.766  1.00 30.72           O  \\nATOM    713  OE2 GLU A  91      11.371  15.730  27.449  1.00 30.38           O  \\nATOM    714  N   ASP A  92       8.019  17.747  23.433  1.00 16.19           N  \\nATOM    715  CA  ASP A  92       6.650  18.216  23.265  1.00 15.23           C  \\nATOM    716  C   ASP A  92       6.141  18.053  21.839  1.00 14.29           C  \\nATOM    717  O   ASP A  92       4.927  18.074  21.633  1.00 14.78           O  \\nATOM    718  CB  ASP A  92       6.536  19.708  23.646  1.00 17.85           C  \\nATOM    719  CG  ASP A  92       6.706  20.080  25.127  1.00 21.39           C  \\nATOM    720  OD1 ASP A  92       6.648  19.209  25.997  1.00 20.06           O  \\nATOM    721  OD2 ASP A  92       6.887  21.272  25.401  1.00 24.39           O  \\nATOM    722  N   ILE A  93       7.013  17.910  20.826  1.00 11.76           N  \\nATOM    723  CA  ILE A  93       6.559  17.749  19.452  1.00 14.77           C  \\nATOM    724  C   ILE A  93       5.574  16.583  19.354  1.00 14.03           C  \\nATOM    725  O   ILE A  93       4.576  16.695  18.644  1.00 14.93           O  \\nATOM    726  CB  ILE A  93       7.790  17.499  18.485  1.00 14.96           C  \\nATOM    727  CG1 ILE A  93       8.506  18.830  18.293  1.00 15.41           C  \\nATOM    728  CG2 ILE A  93       7.369  16.936  17.091  1.00 14.61           C  \\nATOM    729  CD1 ILE A  93       7.654  19.940  17.637  1.00 20.05           C  \\nATOM    730  N   HIS A  94       5.806  15.471  20.049  1.00 15.23           N  \\nATOM    731  CA  HIS A  94       4.940  14.302  19.956  1.00 18.20           C  \\nATOM    732  C   HIS A  94       3.494  14.635  20.287  1.00 16.36           C  \\nATOM    733  O   HIS A  94       2.576  14.183  19.598  1.00 17.20           O  \\nATOM    734  CB  HIS A  94       5.436  13.194  20.900  1.00 23.57           C  \\nATOM    735  CG  HIS A  94       4.685  11.874  20.678  1.00 29.09           C  \\nATOM    736  ND1 HIS A  94       4.923  11.118  19.602  1.00 31.26           N  \\nATOM    737  CD2 HIS A  94       3.705  11.361  21.498  1.00 30.93           C  \\nATOM    738  CE1 HIS A  94       4.078  10.110  19.746  1.00 31.74           C  \\nATOM    739  NE2 HIS A  94       3.366  10.260  20.861  1.00 32.20           N  \\nATOM    740  N   GLN A  95       3.271  15.499  21.281  1.00 16.91           N  \\nATOM    741  CA  GLN A  95       1.916  15.878  21.673  1.00 18.66           C  \\nATOM    742  C   GLN A  95       1.226  16.736  20.625  1.00 16.97           C  \\nATOM    743  O   GLN A  95       0.012  16.562  20.443  1.00 15.36           O  \\nATOM    744  CB  GLN A  95       1.928  16.626  23.018  1.00 21.56           C  \\nATOM    745  CG  GLN A  95       2.457  15.766  24.180  1.00 29.81           C  \\nATOM    746  CD  GLN A  95       3.370  16.540  25.150  1.00 35.29           C  \\nATOM    747  OE1 GLN A  95       3.037  17.647  25.593  1.00 37.94           O  \\nATOM    748  NE2 GLN A  95       4.546  16.006  25.521  1.00 37.04           N  \\nATOM    749  N   TYR A  96       1.930  17.653  19.921  1.00 16.06           N  \\nATOM    750  CA  TYR A  96       1.276  18.417  18.826  1.00 16.07           C  \\nATOM    751  C   TYR A  96       0.963  17.504  17.658  1.00 14.44           C  \\nATOM    752  O   TYR A  96      -0.138  17.565  17.118  1.00 15.79           O  \\nATOM    753  CB  TYR A  96       2.139  19.554  18.264  1.00 17.94           C  \\nATOM    754  CG  TYR A  96       2.287  20.677  19.277  1.00 19.19           C  \\nATOM    755  CD1 TYR A  96       1.368  21.699  19.338  1.00 19.33           C  \\nATOM    756  CD2 TYR A  96       3.348  20.631  20.161  1.00 19.61           C  \\nATOM    757  CE1 TYR A  96       1.526  22.678  20.308  1.00 21.72           C  \\nATOM    758  CE2 TYR A  96       3.512  21.595  21.129  1.00 21.72           C  \\nATOM    759  CZ  TYR A  96       2.594  22.613  21.190  1.00 21.26           C  \\nATOM    760  OH  TYR A  96       2.776  23.582  22.153  1.00 24.09           O  \\nATOM    761  N   ARG A  97       1.904  16.635  17.295  1.00 13.30           N  \\nATOM    762  CA  ARG A  97       1.699  15.650  16.247  1.00 15.46           C  \\nATOM    763  C   ARG A  97       0.457  14.794  16.492  1.00 14.26           C  \\nATOM    764  O   ARG A  97      -0.420  14.676  15.641  1.00 14.10           O  \\nATOM    765  CB  ARG A  97       2.939  14.796  16.196  1.00 17.97           C  \\nATOM    766  CG  ARG A  97       2.886  13.541  15.357  1.00 23.35           C  \\nATOM    767  CD  ARG A  97       3.333  13.765  13.943  1.00 23.59           C  \\nATOM    768  NE  ARG A  97       3.314  12.471  13.314  1.00 24.08           N  \\nATOM    769  CZ  ARG A  97       3.268  12.320  12.007  1.00 25.74           C  \\nATOM    770  NH1 ARG A  97       3.135  11.073  11.548  1.00 26.76           N  \\nATOM    771  NH2 ARG A  97       3.353  13.380  11.181  1.00 26.95           N  \\nATOM    772  N   GLU A  98       0.332  14.218  17.678  1.00 13.69           N  \\nATOM    773  CA  GLU A  98      -0.807  13.361  17.941  1.00 16.64           C  \\nATOM    774  C   GLU A  98      -2.118  14.108  17.983  1.00 15.57           C  \\nATOM    775  O   GLU A  98      -3.157  13.573  17.607  1.00 15.85           O  \\nATOM    776  CB  GLU A  98      -0.593  12.623  19.260  1.00 21.96           C  \\nATOM    777  CG  GLU A  98       0.417  11.455  19.177  1.00 28.02           C  \\nATOM    778  CD  GLU A  98       0.147  10.460  18.038  1.00 31.04           C  \\nATOM    779  OE1 GLU A  98       1.118  10.123  17.373  1.00 33.58           O  \\nATOM    780  OE2 GLU A  98      -0.992  10.025  17.790  1.00 33.61           O  \\nATOM    781  N   GLN A  99      -2.105  15.362  18.446  1.00 14.66           N  \\nATOM    782  CA  GLN A  99      -3.343  16.104  18.468  1.00 13.22           C  \\nATOM    783  C   GLN A  99      -3.779  16.398  17.049  1.00 12.15           C  \\nATOM    784  O   GLN A  99      -4.974  16.274  16.791  1.00 14.09           O  \\nATOM    785  CB  GLN A  99      -3.158  17.403  19.250  1.00 16.29           C  \\nATOM    786  CG  GLN A  99      -4.513  18.118  19.407  1.00 19.67           C  \\nATOM    787  CD  GLN A  99      -4.517  19.240  20.438  1.00 22.19           C  \\nATOM    788  OE1 GLN A  99      -3.537  19.485  21.138  1.00 23.70           O  \\nATOM    789  NE2 GLN A  99      -5.628  19.961  20.537  1.00 22.72           N  \\nATOM    790  N   ILE A 100      -2.861  16.795  16.142  1.00 11.42           N  \\nATOM    791  CA  ILE A 100      -3.197  17.121  14.743  1.00 10.81           C  \\nATOM    792  C   ILE A 100      -3.761  15.893  14.008  1.00 12.39           C  \\nATOM    793  O   ILE A 100      -4.733  15.967  13.243  1.00 12.70           O  \\nATOM    794  CB  ILE A 100      -1.924  17.659  14.038  1.00 10.32           C  \\nATOM    795  CG1 ILE A 100      -1.642  19.075  14.518  1.00 10.37           C  \\nATOM    796  CG2 ILE A 100      -2.131  17.685  12.512  1.00 11.36           C  \\nATOM    797  CD1 ILE A 100      -0.184  19.481  14.392  1.00 10.69           C  \\nATOM    798  N   LYS A 101      -3.143  14.732  14.255  1.00 14.08           N  \\nATOM    799  CA  LYS A 101      -3.613  13.487  13.637  1.00 15.95           C  \\nATOM    800  C   LYS A 101      -5.021  13.150  14.079  1.00 15.72           C  \\nATOM    801  O   LYS A 101      -5.848  12.770  13.262  1.00 14.25           O  \\nATOM    802  CB  LYS A 101      -2.681  12.317  13.996  1.00 17.44           C  \\nATOM    803  CG  LYS A 101      -1.414  12.284  13.115  1.00 19.24           C  \\nATOM    804  CD  LYS A 101      -0.332  11.418  13.764  1.00 23.28           C  \\nATOM    805  CE  LYS A 101      -0.752  10.000  14.188  1.00 23.82           C  \\nATOM    806  NZ  LYS A 101       0.400   9.230  14.662  1.00 24.00           N  \\nATOM    807  N   ARG A 102      -5.278  13.341  15.377  1.00 16.72           N  \\nATOM    808  CA  ARG A 102      -6.586  13.098  15.962  1.00 18.74           C  \\nATOM    809  C   ARG A 102      -7.643  14.087  15.440  1.00 19.07           C  \\nATOM    810  O   ARG A 102      -8.712  13.628  15.002  1.00 20.67           O  \\nATOM    811  CB  ARG A 102      -6.457  13.197  17.462  1.00 20.62           C  \\nATOM    812  CG  ARG A 102      -7.539  12.409  18.132  1.00 26.89           C  \\nATOM    813  CD  ARG A 102      -7.610  12.887  19.585  1.00 30.60           C  \\nATOM    814  NE  ARG A 102      -8.197  14.224  19.645  1.00 35.83           N  \\nATOM    815  CZ  ARG A 102      -7.633  15.255  20.301  1.00 37.61           C  \\nATOM    816  NH1 ARG A 102      -8.243  16.455  20.267  1.00 38.37           N  \\nATOM    817  NH2 ARG A 102      -6.481  15.110  20.983  1.00 38.39           N  \\nATOM    818  N   VAL A 103      -7.418  15.407  15.395  1.00 15.59           N  \\nATOM    819  CA  VAL A 103      -8.452  16.326  14.881  1.00 18.48           C  \\nATOM    820  C   VAL A 103      -8.665  16.200  13.349  1.00 16.72           C  \\nATOM    821  O   VAL A 103      -9.776  16.383  12.848  1.00 15.90           O  \\nATOM    822  CB  VAL A 103      -8.131  17.840  15.215  1.00 20.10           C  \\nATOM    823  CG1 VAL A 103      -7.896  18.005  16.704  1.00 21.34           C  \\nATOM    824  CG2 VAL A 103      -6.899  18.302  14.512  1.00 23.93           C  \\nATOM    825  N   LYS A 104      -7.622  15.902  12.574  1.00 13.84           N  \\nATOM    826  CA  LYS A 104      -7.804  15.755  11.140  1.00 16.11           C  \\nATOM    827  C   LYS A 104      -8.158  14.329  10.716  1.00 16.70           C  \\nATOM    828  O   LYS A 104      -8.260  14.079   9.508  1.00 17.27           O  \\nATOM    829  CB  LYS A 104      -6.525  16.215  10.404  1.00 18.12           C  \\nATOM    830  CG  LYS A 104      -6.197  17.678  10.744  1.00 19.53           C  \\nATOM    831  CD  LYS A 104      -5.250  18.297   9.736  1.00 21.28           C  \\nATOM    832  CE  LYS A 104      -5.921  18.590   8.390  1.00 20.75           C  \\nATOM    833  NZ  LYS A 104      -6.875  19.673   8.462  1.00 19.79           N  \\nATOM    834  N   ASP A 105      -8.372  13.390  11.669  1.00 15.91           N  \\nATOM    835  CA  ASP A 105      -8.666  11.988  11.359  1.00 15.43           C  \\nATOM    836  C   ASP A 105      -7.705  11.372  10.333  1.00 16.04           C  \\nATOM    837  O   ASP A 105      -8.106  10.654   9.415  1.00 15.38           O  \\nATOM    838  CB  ASP A 105     -10.077  11.847  10.810  1.00 18.74           C  \\nATOM    839  CG  ASP A 105     -11.261  12.121  11.696  1.00 21.15           C  \\nATOM    840  OD1 ASP A 105     -11.183  11.972  12.918  1.00 21.36           O  \\nATOM    841  OD2 ASP A 105     -12.281  12.480  11.113  1.00 22.63           O  \\nATOM    842  N   SER A 106      -6.397  11.591  10.469  1.00 15.54           N  \\nATOM    843  CA  SER A 106      -5.470  11.174   9.442  1.00 17.25           C  \\nATOM    844  C   SER A 106      -4.112  10.882  10.011  1.00 17.31           C  \\nATOM    845  O   SER A 106      -3.650  11.556  10.916  1.00 19.11           O  \\nATOM    846  CB  SER A 106      -5.463  12.314   8.398  1.00 19.27           C  \\nATOM    847  OG  SER A 106      -4.326  12.490   7.573  1.00 24.20           O  \\nATOM    848  N   ASP A 107      -3.454   9.891   9.453  1.00 17.29           N  \\nATOM    849  CA  ASP A 107      -2.109   9.557   9.868  1.00 19.80           C  \\nATOM    850  C   ASP A 107      -1.000  10.220   9.063  1.00 20.36           C  \\nATOM    851  O   ASP A 107       0.197  10.118   9.345  1.00 21.26           O  \\nATOM    852  CB  ASP A 107      -1.970   8.045   9.810  1.00 23.15           C  \\nATOM    853  CG  ASP A 107      -2.863   7.310  10.807  1.00 25.12           C  \\nATOM    854  OD1 ASP A 107      -3.262   6.206  10.462  1.00 29.21           O  \\nATOM    855  OD2 ASP A 107      -3.127   7.812  11.903  1.00 26.59           O  \\nATOM    856  N   ASP A 108      -1.383  10.909   8.005  1.00 20.57           N  \\nATOM    857  CA  ASP A 108      -0.430  11.664   7.216  1.00 22.38           C  \\nATOM    858  C   ASP A 108      -1.061  12.982   6.842  1.00 21.37           C  \\nATOM    859  O   ASP A 108      -1.973  13.096   6.030  1.00 24.36           O  \\nATOM    860  CB  ASP A 108      -0.056  10.900   5.967  1.00 27.28           C  \\nATOM    861  CG  ASP A 108      -1.263  10.278   5.320  1.00 32.24           C  \\nATOM    862  OD1 ASP A 108      -1.714   9.272   5.877  1.00 37.07           O  \\nATOM    863  OD2 ASP A 108      -1.747  10.790   4.301  1.00 36.33           O  \\nATOM    864  N   VAL A 109      -0.598  13.970   7.572  1.00 19.33           N  \\nATOM    865  CA  VAL A 109      -1.009  15.354   7.400  1.00 17.49           C  \\nATOM    866  C   VAL A 109       0.215  16.086   6.819  1.00 14.55           C  \\nATOM    867  O   VAL A 109       1.335  15.772   7.245  1.00 13.64           O  \\nATOM    868  CB  VAL A 109      -1.410  15.870   8.784  1.00 18.40           C  \\nATOM    869  CG1 VAL A 109      -1.645  17.366   8.765  1.00 20.21           C  \\nATOM    870  CG2 VAL A 109      -2.676  15.101   9.214  1.00 20.27           C  \\nATOM    871  N   PRO A 110       0.141  17.001   5.825  1.00 12.41           N  \\nATOM    872  CA  PRO A 110       1.297  17.783   5.384  1.00 11.52           C  \\nATOM    873  C   PRO A 110       1.920  18.480   6.605  1.00 11.79           C  \\nATOM    874  O   PRO A 110       1.289  19.245   7.324  1.00  9.55           O  \\nATOM    875  CB  PRO A 110       0.728  18.721   4.370  1.00 11.31           C  \\nATOM    876  CG  PRO A 110      -0.410  17.926   3.779  1.00 11.38           C  \\nATOM    877  CD  PRO A 110      -1.044  17.309   5.020  1.00 12.05           C  \\nATOM    878  N   MET A 111       3.192  18.201   6.874  1.00 11.92           N  \\nATOM    879  CA  MET A 111       3.878  18.724   8.050  1.00 11.49           C  \\nATOM    880  C   MET A 111       5.379  18.749   7.807  1.00 11.68           C  \\nATOM    881  O   MET A 111       5.903  17.900   7.098  1.00 11.11           O  \\nATOM    882  CB  MET A 111       3.494  17.806   9.211  1.00 14.36           C  \\nATOM    883  CG  MET A 111       3.864  18.110  10.627  1.00 17.41           C  \\nATOM    884  SD  MET A 111       3.231  16.825  11.758  1.00 18.36           S  \\nATOM    885  CE  MET A 111       1.528  17.128  11.498  1.00 17.25           C  \\nATOM    886  N   VAL A 112       6.091  19.706   8.404  1.00 10.73           N  \\nATOM    887  CA  VAL A 112       7.545  19.763   8.348  1.00 10.18           C  \\nATOM    888  C   VAL A 112       7.992  19.875   9.794  1.00  9.11           C  \\nATOM    889  O   VAL A 112       7.404  20.633  10.556  1.00 10.77           O  \\nATOM    890  CB  VAL A 112       8.010  21.024   7.542  1.00 11.80           C  \\nATOM    891  CG1 VAL A 112       9.528  21.213   7.558  1.00 13.24           C  \\nATOM    892  CG2 VAL A 112       7.542  20.843   6.104  1.00 14.56           C  \\nATOM    893  N   LEU A 113       9.032  19.167  10.207  1.00  8.47           N  \\nATOM    894  CA  LEU A 113       9.632  19.332  11.535  1.00  9.53           C  \\nATOM    895  C   LEU A 113      10.738  20.384  11.412  1.00  9.33           C  \\nATOM    896  O   LEU A 113      11.621  20.270  10.555  1.00  9.95           O  \\nATOM    897  CB  LEU A 113      10.214  17.983  12.017  1.00  9.56           C  \\nATOM    898  CG  LEU A 113      10.863  17.976  13.381  1.00 11.58           C  \\nATOM    899  CD1 LEU A 113       9.908  18.379  14.503  1.00 13.17           C  \\nATOM    900  CD2 LEU A 113      11.361  16.564  13.597  1.00 12.84           C  \\nATOM    901  N   VAL A 114      10.705  21.442  12.221  1.00  7.62           N  \\nATOM    902  CA  VAL A 114      11.673  22.541  12.109  1.00  7.64           C  \\nATOM    903  C   VAL A 114      12.436  22.667  13.441  1.00  7.65           C  \\nATOM    904  O   VAL A 114      11.872  22.842  14.541  1.00  9.71           O  \\nATOM    905  CB  VAL A 114      10.896  23.836  11.774  1.00  9.37           C  \\nATOM    906  CG1 VAL A 114      11.814  25.030  11.869  1.00 10.27           C  \\nATOM    907  CG2 VAL A 114      10.254  23.719  10.393  1.00  9.64           C  \\nATOM    908  N   GLY A 115      13.745  22.567  13.331  1.00  6.10           N  \\nATOM    909  CA  GLY A 115      14.603  22.742  14.476  1.00  6.69           C  \\nATOM    910  C   GLY A 115      15.120  24.163  14.407  1.00  8.47           C  \\nATOM    911  O   GLY A 115      16.007  24.439  13.594  1.00  9.16           O  \\nATOM    912  N   ASN A 116      14.593  25.089  15.192  1.00  8.01           N  \\nATOM    913  CA  ASN A 116      15.007  26.475  15.111  1.00  8.24           C  \\nATOM    914  C   ASN A 116      16.112  26.873  16.109  1.00  9.61           C  \\nATOM    915  O   ASN A 116      16.453  26.142  17.051  1.00  9.84           O  \\nATOM    916  CB  ASN A 116      13.721  27.307  15.275  1.00  8.01           C  \\nATOM    917  CG  ASN A 116      13.929  28.788  14.992  1.00  9.48           C  \\nATOM    918  OD1 ASN A 116      14.470  29.205  13.972  1.00  9.94           O  \\nATOM    919  ND2 ASN A 116      13.565  29.644  15.940  1.00 10.75           N  \\nATOM    920  N   LYS A 117      16.660  28.087  15.876  1.00 10.22           N  \\nATOM    921  CA  LYS A 117      17.735  28.715  16.674  1.00 12.36           C  \\nATOM    922  C   LYS A 117      19.094  28.021  16.450  1.00 13.39           C  \\nATOM    923  O   LYS A 117      19.977  27.940  17.324  1.00 11.29           O  \\nATOM    924  CB  LYS A 117      17.391  28.706  18.214  1.00 13.23           C  \\nATOM    925  CG  LYS A 117      16.026  29.272  18.611  1.00 12.90           C  \\nATOM    926  CD  LYS A 117      15.932  29.640  20.097  1.00 12.43           C  \\nATOM    927  CE  LYS A 117      14.551  30.217  20.380  1.00 10.55           C  \\nATOM    928  NZ  LYS A 117      14.491  30.647  21.772  1.00 11.10           N  \\nATOM    929  N   CYS A 118      19.254  27.555  15.188  1.00 13.69           N  \\nATOM    930  CA  CYS A 118      20.449  26.806  14.844  1.00 15.67           C  \\nATOM    931  C   CYS A 118      21.716  27.658  14.796  1.00 15.28           C  \\nATOM    932  O   CYS A 118      22.808  27.113  14.629  1.00 15.30           O  \\nATOM    933  CB  CYS A 118      20.246  26.064  13.492  1.00 17.74           C  \\nATOM    934  SG  CYS A 118      20.353  27.009  11.944  1.00 20.74           S  \\nATOM    935  N   ASP A 119      21.578  28.995  14.931  1.00 15.76           N  \\nATOM    936  CA  ASP A 119      22.722  29.904  15.083  1.00 16.09           C  \\nATOM    937  C   ASP A 119      23.426  29.778  16.445  1.00 17.19           C  \\nATOM    938  O   ASP A 119      24.589  30.178  16.553  1.00 18.76           O  \\nATOM    939  CB  ASP A 119      22.268  31.374  14.888  1.00 15.14           C  \\nATOM    940  CG  ASP A 119      21.153  31.785  15.835  1.00 15.81           C  \\nATOM    941  OD1 ASP A 119      20.030  31.294  15.662  1.00 13.87           O  \\nATOM    942  OD2 ASP A 119      21.407  32.596  16.721  1.00 14.85           O  \\nATOM    943  N   LEU A 120      22.732  29.248  17.480  1.00 16.86           N  \\nATOM    944  CA  LEU A 120      23.230  29.099  18.845  1.00 18.76           C  \\nATOM    945  C   LEU A 120      24.217  27.957  18.979  1.00 19.70           C  \\nATOM    946  O   LEU A 120      23.979  26.885  18.431  1.00 20.53           O  \\nATOM    947  CB  LEU A 120      22.103  28.829  19.831  1.00 17.65           C  \\nATOM    948  CG  LEU A 120      21.065  29.933  20.010  1.00 19.43           C  \\nATOM    949  CD1 LEU A 120      20.044  29.418  21.008  1.00 18.46           C  \\nATOM    950  CD2 LEU A 120      21.688  31.239  20.503  1.00 18.73           C  \\nATOM    951  N   ALA A 121      25.313  28.184  19.705  1.00 19.26           N  \\nATOM    952  CA  ALA A 121      26.303  27.143  19.887  1.00 20.48           C  \\nATOM    953  C   ALA A 121      25.942  26.210  21.036  1.00 21.30           C  \\nATOM    954  O   ALA A 121      26.403  25.069  21.067  1.00 24.04           O  \\nATOM    955  CB  ALA A 121      27.652  27.767  20.164  1.00 22.02           C  \\nATOM    956  N   ALA A 122      25.109  26.635  21.982  1.00 22.09           N  \\nATOM    957  CA  ALA A 122      24.683  25.797  23.094  1.00 23.88           C  \\nATOM    958  C   ALA A 122      23.578  24.811  22.720  1.00 23.77           C  \\nATOM    959  O   ALA A 122      22.492  24.835  23.301  1.00 27.82           O  \\nATOM    960  CB  ALA A 122      24.171  26.684  24.249  1.00 25.67           C  \\nATOM    961  N   ARG A 123      23.843  23.908  21.783  1.00 20.99           N  \\nATOM    962  CA  ARG A 123      22.890  22.905  21.311  1.00 19.81           C  \\nATOM    963  C   ARG A 123      22.619  21.761  22.283  1.00 19.56           C  \\nATOM    964  O   ARG A 123      23.590  21.176  22.774  1.00 20.01           O  \\nATOM    965  CB  ARG A 123      23.421  22.333  19.988  1.00 19.81           C  \\nATOM    966  CG  ARG A 123      22.662  21.166  19.398  1.00 21.43           C  \\nATOM    967  CD  ARG A 123      23.273  20.680  18.073  1.00 22.66           C  \\nATOM    968  NE  ARG A 123      23.050  21.590  16.956  1.00 22.61           N  \\nATOM    969  CZ  ARG A 123      22.019  21.487  16.093  1.00 20.95           C  \\nATOM    970  NH1 ARG A 123      21.045  20.568  16.181  1.00 16.36           N  \\nATOM    971  NH2 ARG A 123      21.989  22.370  15.098  1.00 19.65           N  \\nATOM    972  N   THR A 124      21.364  21.389  22.600  1.00 17.96           N  \\nATOM    973  CA  THR A 124      21.118  20.200  23.415  1.00 17.85           C  \\nATOM    974  C   THR A 124      20.413  19.068  22.656  1.00 17.08           C  \\nATOM    975  O   THR A 124      20.459  17.926  23.091  1.00 19.59           O  \\nATOM    976  CB  THR A 124      20.311  20.578  24.672  1.00 19.42           C  \\nATOM    977  OG1 THR A 124      19.109  21.230  24.277  1.00 20.50           O  \\nATOM    978  CG2 THR A 124      21.155  21.458  25.588  1.00 19.06           C  \\nATOM    979  N   VAL A 125      19.788  19.319  21.495  1.00 16.59           N  \\nATOM    980  CA  VAL A 125      19.133  18.324  20.647  1.00 16.09           C  \\nATOM    981  C   VAL A 125      20.032  18.080  19.413  1.00 16.61           C  \\nATOM    982  O   VAL A 125      20.180  18.980  18.573  1.00 14.07           O  \\nATOM    983  CB  VAL A 125      17.720  18.848  20.179  1.00 15.28           C  \\nATOM    984  CG1 VAL A 125      17.097  17.814  19.265  1.00 14.81           C  \\nATOM    985  CG2 VAL A 125      16.824  19.168  21.366  1.00 15.97           C  \\nATOM    986  N   GLU A 126      20.659  16.889  19.280  1.00 17.05           N  \\nATOM    987  CA  GLU A 126      21.487  16.568  18.124  1.00 19.28           C  \\nATOM    988  C   GLU A 126      20.631  16.496  16.876  1.00 18.51           C  \\nATOM    989  O   GLU A 126      19.511  15.976  16.880  1.00 17.68           O  \\nATOM    990  CB  GLU A 126      22.165  15.215  18.219  1.00 23.29           C  \\nATOM    991  CG  GLU A 126      23.244  15.057  19.265  1.00 30.86           C  \\nATOM    992  CD  GLU A 126      24.402  16.054  19.153  1.00 36.59           C  \\nATOM    993  OE1 GLU A 126      25.139  16.012  18.161  1.00 39.64           O  \\nATOM    994  OE2 GLU A 126      24.581  16.885  20.056  1.00 40.44           O  \\nATOM    995  N   SER A 127      21.167  16.985  15.755  1.00 17.73           N  \\nATOM    996  CA  SER A 127      20.426  16.938  14.493  1.00 16.72           C  \\nATOM    997  C   SER A 127      20.040  15.506  14.158  1.00 16.76           C  \\nATOM    998  O   SER A 127      18.931  15.245  13.690  1.00 17.01           O  \\nATOM    999  CB  SER A 127      21.283  17.518  13.371  1.00 17.67           C  \\nATOM   1000  OG  SER A 127      21.730  18.826  13.768  1.00 19.67           O  \\nATOM   1001  N   ARG A 128      20.916  14.558  14.466  1.00 16.72           N  \\nATOM   1002  CA  ARG A 128      20.667  13.154  14.145  1.00 20.94           C  \\nATOM   1003  C   ARG A 128      19.435  12.617  14.862  1.00 19.62           C  \\nATOM   1004  O   ARG A 128      18.641  11.882  14.278  1.00 18.81           O  \\nATOM   1005  CB  ARG A 128      21.910  12.319  14.510  1.00 24.31           C  \\nATOM   1006  CG  ARG A 128      21.969  11.035  13.692  1.00 30.88           C  \\nATOM   1007  CD  ARG A 128      23.381  10.403  13.536  1.00 35.88           C  \\nATOM   1008  NE  ARG A 128      24.384  10.996  12.621  1.00 39.57           N  \\nATOM   1009  CZ  ARG A 128      24.313  11.018  11.267  1.00 40.85           C  \\nATOM   1010  NH1 ARG A 128      25.399  11.388  10.577  1.00 40.83           N  \\nATOM   1011  NH2 ARG A 128      23.202  10.685  10.576  1.00 41.33           N  \\nATOM   1012  N   GLN A 129      19.209  13.016  16.119  1.00 18.27           N  \\nATOM   1013  CA  GLN A 129      18.032  12.519  16.787  1.00 18.53           C  \\nATOM   1014  C   GLN A 129      16.735  13.151  16.224  1.00 17.11           C  \\nATOM   1015  O   GLN A 129      15.691  12.486  16.098  1.00 15.48           O  \\nATOM   1016  CB  GLN A 129      18.229  12.766  18.306  1.00 22.36           C  \\nATOM   1017  CG  GLN A 129      18.041  14.133  18.911  1.00 24.78           C  \\nATOM   1018  CD  GLN A 129      18.283  14.123  20.411  1.00 27.48           C  \\nATOM   1019  OE1 GLN A 129      19.287  14.643  20.906  1.00 28.14           O  \\nATOM   1020  NE2 GLN A 129      17.367  13.497  21.152  1.00 27.21           N  \\nATOM   1021  N   ALA A 130      16.780  14.446  15.837  1.00 14.73           N  \\nATOM   1022  CA  ALA A 130      15.611  15.044  15.210  1.00 13.87           C  \\nATOM   1023  C   ALA A 130      15.342  14.449  13.826  1.00 13.30           C  \\nATOM   1024  O   ALA A 130      14.199  14.194  13.436  1.00 11.42           O  \\nATOM   1025  CB  ALA A 130      15.822  16.536  15.063  1.00 16.21           C  \\nATOM   1026  N   GLN A 131      16.401  14.189  13.057  1.00 14.61           N  \\nATOM   1027  CA  GLN A 131      16.267  13.529  11.739  1.00 16.01           C  \\nATOM   1028  C   GLN A 131      15.637  12.109  11.821  1.00 16.29           C  \\nATOM   1029  O   GLN A 131      14.777  11.708  11.017  1.00 14.37           O  \\nATOM   1030  CB  GLN A 131      17.673  13.507  11.116  1.00 18.12           C  \\nATOM   1031  CG  GLN A 131      17.702  13.017   9.689  1.00 22.19           C  \\nATOM   1032  CD  GLN A 131      16.889  13.922   8.801  1.00 21.61           C  \\nATOM   1033  OE1 GLN A 131      17.292  15.044   8.529  1.00 24.06           O  \\nATOM   1034  NE2 GLN A 131      15.714  13.485   8.370  1.00 23.37           N  \\nATOM   1035  N   ASP A 132      16.038  11.348  12.853  1.00 16.64           N  \\nATOM   1036  CA  ASP A 132      15.467  10.038  13.173  1.00 18.00           C  \\nATOM   1037  C   ASP A 132      13.984  10.175  13.484  1.00 17.28           C  \\nATOM   1038  O   ASP A 132      13.178   9.395  12.985  1.00 16.45           O  \\nATOM   1039  CB  ASP A 132      16.120   9.415  14.405  1.00 20.10           C  \\nATOM   1040  CG  ASP A 132      17.506   8.816  14.234  1.00 24.07           C  \\nATOM   1041  OD1 ASP A 132      18.120   8.459  15.243  1.00 28.35           O  \\nATOM   1042  OD2 ASP A 132      17.975   8.686  13.106  1.00 25.43           O  \\nATOM   1043  N   LEU A 133      13.614  11.168  14.312  1.00 15.58           N  \\nATOM   1044  CA  LEU A 133      12.213  11.390  14.624  1.00 16.28           C  \\nATOM   1045  C   LEU A 133      11.421  11.729  13.366  1.00 14.96           C  \\nATOM   1046  O   LEU A 133      10.345  11.181  13.128  1.00 15.09           O  \\nATOM   1047  CB  LEU A 133      12.052  12.536  15.647  1.00 17.06           C  \\nATOM   1048  CG  LEU A 133      10.617  12.781  16.136  1.00 18.79           C  \\nATOM   1049  CD1 LEU A 133      10.142  11.554  16.881  1.00 19.96           C  \\nATOM   1050  CD2 LEU A 133      10.544  14.013  17.020  1.00 18.45           C  \\nATOM   1051  N   ALA A 134      11.937  12.642  12.545  1.00 13.16           N  \\nATOM   1052  CA  ALA A 134      11.241  13.016  11.331  1.00 14.89           C  \\nATOM   1053  C   ALA A 134      11.092  11.821  10.414  1.00 16.19           C  \\nATOM   1054  O   ALA A 134      10.015  11.643   9.857  1.00 15.70           O  \\nATOM   1055  CB  ALA A 134      12.021  14.089  10.629  1.00 15.12           C  \\nATOM   1056  N   ARG A 135      12.143  10.995  10.279  1.00 17.00           N  \\nATOM   1057  CA  ARG A 135      12.088   9.828   9.438  1.00 19.17           C  \\nATOM   1058  C   ARG A 135      10.990   8.893   9.916  1.00 18.70           C  \\nATOM   1059  O   ARG A 135      10.269   8.352   9.078  1.00 20.29           O  \\nATOM   1060  CB  ARG A 135      13.425   9.091   9.456  1.00 22.48           C  \\nATOM   1061  CG  ARG A 135      13.499   8.297   8.155  1.00 29.02           C  \\nATOM   1062  CD  ARG A 135      14.619   7.270   7.988  1.00 33.22           C  \\nATOM   1063  NE  ARG A 135      14.117   5.888   8.050  1.00 38.80           N  \\nATOM   1064  CZ  ARG A 135      13.281   5.302   7.148  1.00 39.60           C  \\nATOM   1065  NH1 ARG A 135      12.889   4.044   7.384  1.00 40.80           N  \\nATOM   1066  NH2 ARG A 135      12.827   5.913   6.034  1.00 40.11           N  \\nATOM   1067  N   SER A 136      10.815   8.697  11.242  1.00 19.23           N  \\nATOM   1068  CA  SER A 136       9.757   7.841  11.770  1.00 19.12           C  \\nATOM   1069  C   SER A 136       8.374   8.432  11.528  1.00 19.61           C  \\nATOM   1070  O   SER A 136       7.398   7.690  11.522  1.00 21.01           O  \\nATOM   1071  CB  SER A 136       9.930   7.624  13.280  1.00 19.94           C  \\nATOM   1072  OG  SER A 136       9.459   8.686  14.119  1.00 23.21           O  \\nATOM   1073  N   TYR A 137       8.231   9.760  11.380  1.00 17.33           N  \\nATOM   1074  CA  TYR A 137       6.950  10.375  11.078  1.00 18.12           C  \\nATOM   1075  C   TYR A 137       6.683  10.404   9.570  1.00 18.22           C  \\nATOM   1076  O   TYR A 137       5.561  10.656   9.129  1.00 19.27           O  \\nATOM   1077  CB  TYR A 137       6.907  11.812  11.592  1.00 18.21           C  \\nATOM   1078  CG  TYR A 137       6.883  12.057  13.084  1.00 19.08           C  \\nATOM   1079  CD1 TYR A 137       6.619  11.026  13.955  1.00 20.51           C  \\nATOM   1080  CD2 TYR A 137       7.059  13.355  13.548  1.00 20.43           C  \\nATOM   1081  CE1 TYR A 137       6.510  11.285  15.306  1.00 21.89           C  \\nATOM   1082  CE2 TYR A 137       6.945  13.624  14.899  1.00 21.02           C  \\nATOM   1083  CZ  TYR A 137       6.662  12.574  15.763  1.00 22.13           C  \\nATOM   1084  OH  TYR A 137       6.476  12.782  17.112  1.00 22.16           O  \\nATOM   1085  N   GLY A 138       7.689  10.186   8.737  1.00 16.07           N  \\nATOM   1086  CA  GLY A 138       7.520  10.265   7.294  1.00 17.66           C  \\nATOM   1087  C   GLY A 138       7.476  11.725   6.810  1.00 18.01           C  \\nATOM   1088  O   GLY A 138       6.859  12.010   5.774  1.00 17.85           O  \\nATOM   1089  N   ILE A 139       8.128  12.662   7.540  1.00 14.16           N  \\nATOM   1090  CA  ILE A 139       8.117  14.065   7.145  1.00 13.49           C  \\nATOM   1091  C   ILE A 139       9.526  14.662   7.033  1.00 14.35           C  \\nATOM   1092  O   ILE A 139      10.494  14.152   7.610  1.00 16.19           O  \\nATOM   1093  CB  ILE A 139       7.292  14.896   8.161  1.00 14.40           C  \\nATOM   1094  CG1 ILE A 139       7.925  14.811   9.554  1.00 15.44           C  \\nATOM   1095  CG2 ILE A 139       5.821  14.422   8.104  1.00 14.07           C  \\nATOM   1096  CD1 ILE A 139       7.288  15.765  10.587  1.00 15.35           C  \\nATOM   1097  N   PRO A 140       9.706  15.767   6.308  1.00 13.55           N  \\nATOM   1098  CA  PRO A 140      10.984  16.448   6.205  1.00 13.39           C  \\nATOM   1099  C   PRO A 140      11.424  17.065   7.532  1.00 12.74           C  \\nATOM   1100  O   PRO A 140      10.601  17.530   8.310  1.00 11.42           O  \\nATOM   1101  CB  PRO A 140      10.767  17.498   5.127  1.00 13.47           C  \\nATOM   1102  CG  PRO A 140       9.514  17.084   4.410  1.00 14.09           C  \\nATOM   1103  CD  PRO A 140       8.686  16.467   5.519  1.00 13.14           C  \\nATOM   1104  N   TYR A 141      12.744  17.086   7.756  1.00 11.68           N  \\nATOM   1105  CA  TYR A 141      13.336  17.791   8.890  1.00 10.89           C  \\nATOM   1106  C   TYR A 141      14.175  18.945   8.326  1.00 10.09           C  \\nATOM   1107  O   TYR A 141      15.056  18.720   7.468  1.00 10.27           O  \\nATOM   1108  CB  TYR A 141      14.276  16.897   9.691  1.00 12.54           C  \\nATOM   1109  CG  TYR A 141      15.063  17.676  10.733  1.00 10.92           C  \\nATOM   1110  CD1 TYR A 141      14.350  18.396  11.664  1.00 11.75           C  \\nATOM   1111  CD2 TYR A 141      16.453  17.633  10.784  1.00 12.49           C  \\nATOM   1112  CE1 TYR A 141      14.997  19.074  12.670  1.00 12.85           C  \\nATOM   1113  CE2 TYR A 141      17.112  18.307  11.794  1.00 11.33           C  \\nATOM   1114  CZ  TYR A 141      16.369  19.016  12.728  1.00 12.61           C  \\nATOM   1115  OH  TYR A 141      16.944  19.646  13.813  1.00 13.82           O  \\nATOM   1116  N   ILE A 142      13.892  20.174   8.799  1.00  9.35           N  \\nATOM   1117  CA  ILE A 142      14.636  21.355   8.376  1.00  9.05           C  \\nATOM   1118  C   ILE A 142      15.153  22.163   9.551  1.00  9.25           C  \\nATOM   1119  O   ILE A 142      14.409  22.382  10.499  1.00 10.43           O  \\nATOM   1120  CB  ILE A 142      13.756  22.259   7.472  1.00  9.25           C  \\nATOM   1121  CG1 ILE A 142      13.523  21.470   6.180  1.00 10.51           C  \\nATOM   1122  CG2 ILE A 142      14.450  23.586   7.122  1.00  9.08           C  \\nATOM   1123  CD1 ILE A 142      12.719  22.127   5.028  1.00 13.16           C  \\nATOM   1124  N   GLU A 143      16.457  22.513   9.568  1.00  9.01           N  \\nATOM   1125  CA  GLU A 143      16.979  23.404  10.611  1.00  8.52           C  \\nATOM   1126  C   GLU A 143      16.978  24.864  10.193  1.00 10.31           C  \\nATOM   1127  O   GLU A 143      17.278  25.228   9.040  1.00 10.45           O  \\nATOM   1128  CB  GLU A 143      18.397  23.022  11.013  1.00 10.87           C  \\nATOM   1129  CG  GLU A 143      18.363  21.728  11.801  1.00 10.22           C  \\nATOM   1130  CD  GLU A 143      19.592  21.411  12.633  1.00 13.01           C  \\nATOM   1131  OE1 GLU A 143      20.616  22.082  12.521  1.00 13.65           O  \\nATOM   1132  OE2 GLU A 143      19.511  20.487  13.429  1.00 12.70           O  \\nATOM   1133  N   THR A 144      16.608  25.762  11.117  1.00  9.18           N  \\nATOM   1134  CA  THR A 144      16.422  27.169  10.754  1.00  9.71           C  \\nATOM   1135  C   THR A 144      17.014  28.076  11.789  1.00  8.66           C  \\nATOM   1136  O   THR A 144      17.259  27.698  12.927  1.00  9.13           O  \\nATOM   1137  CB  THR A 144      14.919  27.617  10.635  1.00  9.90           C  \\nATOM   1138  OG1 THR A 144      14.336  27.403  11.931  1.00  9.29           O  \\nATOM   1139  CG2 THR A 144      14.120  26.858   9.557  1.00  9.89           C  \\nATOM   1140  N   SER A 145      17.244  29.288  11.360  1.00 10.91           N  \\nATOM   1141  CA  SER A 145      17.558  30.377  12.290  1.00 12.65           C  \\nATOM   1142  C   SER A 145      16.758  31.602  11.849  1.00 10.37           C  \\nATOM   1143  O   SER A 145      16.930  32.094  10.727  1.00 11.76           O  \\nATOM   1144  CB  SER A 145      19.069  30.768  12.308  1.00 11.79           C  \\nATOM   1145  OG  SER A 145      19.248  31.931  13.125  1.00 12.12           O  \\nATOM   1146  N   ALA A 146      15.838  32.046  12.696  1.00  9.49           N  \\nATOM   1147  CA  ALA A 146      15.131  33.268  12.413  1.00 10.90           C  \\nATOM   1148  C   ALA A 146      16.070  34.455  12.553  1.00 13.53           C  \\nATOM   1149  O   ALA A 146      15.799  35.514  11.995  1.00 16.20           O  \\nATOM   1150  CB  ALA A 146      13.976  33.436  13.367  1.00 12.65           C  \\nATOM   1151  N   LYS A 147      17.183  34.302  13.283  1.00 14.35           N  \\nATOM   1152  CA  LYS A 147      18.150  35.351  13.486  1.00 16.25           C  \\nATOM   1153  C   LYS A 147      18.953  35.568  12.215  1.00 17.58           C  \\nATOM   1154  O   LYS A 147      19.124  36.724  11.825  1.00 20.13           O  \\nATOM   1155  CB  LYS A 147      19.068  34.970  14.617  1.00 17.55           C  \\nATOM   1156  CG  LYS A 147      20.003  36.125  14.958  1.00 22.83           C  \\nATOM   1157  CD  LYS A 147      20.581  35.932  16.341  1.00 26.44           C  \\nATOM   1158  CE  LYS A 147      21.683  36.949  16.507  1.00 30.79           C  \\nATOM   1159  NZ  LYS A 147      22.575  36.534  17.574  1.00 33.81           N  \\nATOM   1160  N   THR A 148      19.479  34.529  11.541  1.00 17.16           N  \\nATOM   1161  CA  THR A 148      20.248  34.735  10.300  1.00 15.93           C  \\nATOM   1162  C   THR A 148      19.422  34.607   9.013  1.00 17.72           C  \\nATOM   1163  O   THR A 148      19.926  34.962   7.945  1.00 18.47           O  \\nATOM   1164  CB  THR A 148      21.444  33.745  10.209  1.00 15.86           C  \\nATOM   1165  OG1 THR A 148      20.886  32.435  10.042  1.00 14.56           O  \\nATOM   1166  CG2 THR A 148      22.377  33.839  11.439  1.00 15.25           C  \\nATOM   1167  N   ARG A 149      18.176  34.108   9.100  1.00 15.18           N  \\nATOM   1168  CA  ARG A 149      17.268  33.815   7.994  1.00 19.20           C  \\nATOM   1169  C   ARG A 149      17.652  32.483   7.318  1.00 17.94           C  \\nATOM   1170  O   ARG A 149      17.031  32.079   6.332  1.00 20.28           O  \\nATOM   1171  CB  ARG A 149      17.251  34.964   6.937  1.00 21.40           C  \\nATOM   1172  CG  ARG A 149      15.929  35.106   6.188  1.00 27.00           C  \\nATOM   1173  CD  ARG A 149      15.946  35.904   4.863  1.00 27.99           C  \\nATOM   1174  NE  ARG A 149      14.707  35.687   4.092  1.00 29.40           N  \\nATOM   1175  CZ  ARG A 149      13.550  36.232   4.492  1.00 29.07           C  \\nATOM   1176  NH1 ARG A 149      12.412  36.015   3.828  1.00 29.56           N  \\nATOM   1177  NH2 ARG A 149      13.527  37.004   5.577  1.00 28.99           N  \\nATOM   1178  N   GLN A 150      18.621  31.701   7.838  1.00 15.80           N  \\nATOM   1179  CA  GLN A 150      18.926  30.402   7.273  1.00 16.36           C  \\nATOM   1180  C   GLN A 150      17.737  29.464   7.455  1.00 15.02           C  \\nATOM   1181  O   GLN A 150      17.119  29.351   8.504  1.00 13.57           O  \\nATOM   1182  CB  GLN A 150      20.152  29.729   7.937  1.00 23.05           C  \\nATOM   1183  CG  GLN A 150      20.578  28.448   7.162  1.00 29.37           C  \\nATOM   1184  CD  GLN A 150      21.604  27.530   7.827  1.00 34.84           C  \\nATOM   1185  OE1 GLN A 150      21.301  26.416   8.280  1.00 38.95           O  \\nATOM   1186  NE2 GLN A 150      22.857  27.967   7.883  1.00 35.22           N  \\nATOM   1187  N   GLY A 151      17.412  28.809   6.348  1.00 13.62           N  \\nATOM   1188  CA  GLY A 151      16.357  27.835   6.296  1.00 13.50           C  \\nATOM   1189  C   GLY A 151      14.943  28.395   6.329  1.00 12.36           C  \\nATOM   1190  O   GLY A 151      14.025  27.591   6.148  1.00 11.07           O  \\nATOM   1191  N   VAL A 152      14.706  29.702   6.510  1.00 12.59           N  \\nATOM   1192  CA  VAL A 152      13.339  30.183   6.715  1.00 14.23           C  \\nATOM   1193  C   VAL A 152      12.435  29.971   5.475  1.00 13.41           C  \\nATOM   1194  O   VAL A 152      11.352  29.362   5.577  1.00 12.94           O  \\nATOM   1195  CB  VAL A 152      13.430  31.691   7.167  1.00 12.57           C  \\nATOM   1196  CG1 VAL A 152      12.059  32.316   7.194  1.00 13.14           C  \\nATOM   1197  CG2 VAL A 152      14.044  31.787   8.543  1.00 12.69           C  \\nATOM   1198  N   GLU A 153      12.831  30.410   4.278  1.00 13.52           N  \\nATOM   1199  CA  GLU A 153      12.001  30.213   3.086  1.00 14.81           C  \\nATOM   1200  C   GLU A 153      11.857  28.756   2.742  1.00 11.88           C  \\nATOM   1201  O   GLU A 153      10.797  28.282   2.344  1.00 13.92           O  \\nATOM   1202  CB  GLU A 153      12.592  30.936   1.881  1.00 17.03           C  \\nATOM   1203  CG  GLU A 153      12.454  32.452   2.015  1.00 21.18           C  \\nATOM   1204  CD  GLU A 153      12.870  33.318   0.817  1.00 25.73           C  \\nATOM   1205  OE1 GLU A 153      13.232  32.781  -0.234  1.00 28.32           O  \\nATOM   1206  OE2 GLU A 153      12.822  34.550   0.936  1.00 26.64           O  \\nATOM   1207  N   ASP A 154      12.959  28.031   2.957  1.00 11.31           N  \\nATOM   1208  CA  ASP A 154      13.023  26.593   2.747  1.00 10.16           C  \\nATOM   1209  C   ASP A 154      11.964  25.828   3.561  1.00 10.90           C  \\nATOM   1210  O   ASP A 154      11.224  24.985   3.035  1.00 11.83           O  \\nATOM   1211  CB  ASP A 154      14.420  26.212   3.117  1.00 15.00           C  \\nATOM   1212  CG  ASP A 154      14.919  24.942   2.524  1.00 19.04           C  \\nATOM   1213  OD1 ASP A 154      14.121  24.195   1.938  1.00 22.96           O  \\nATOM   1214  OD2 ASP A 154      16.122  24.697   2.676  1.00 21.26           O  \\nATOM   1215  N   ALA A 155      11.854  26.107   4.881  1.00  8.83           N  \\nATOM   1216  CA  ALA A 155      10.841  25.450   5.716  1.00 10.00           C  \\nATOM   1217  C   ALA A 155       9.424  25.748   5.203  1.00 10.10           C  \\nATOM   1218  O   ALA A 155       8.653  24.814   4.978  1.00 11.14           O  \\nATOM   1219  CB  ALA A 155      10.923  25.933   7.192  1.00  9.50           C  \\nATOM   1220  N   PHE A 156       9.054  27.011   4.949  1.00 10.59           N  \\nATOM   1221  CA  PHE A 156       7.712  27.345   4.475  1.00 10.51           C  \\nATOM   1222  C   PHE A 156       7.453  26.825   3.069  1.00 11.05           C  \\nATOM   1223  O   PHE A 156       6.368  26.302   2.801  1.00 10.80           O  \\nATOM   1224  CB  PHE A 156       7.492  28.890   4.506  1.00 11.54           C  \\nATOM   1225  CG  PHE A 156       7.266  29.450   5.936  1.00 12.42           C  \\nATOM   1226  CD1 PHE A 156       8.319  29.958   6.707  1.00 12.00           C  \\nATOM   1227  CD2 PHE A 156       5.978  29.434   6.478  1.00 12.57           C  \\nATOM   1228  CE1 PHE A 156       8.065  30.434   8.001  1.00 12.45           C  \\nATOM   1229  CE2 PHE A 156       5.747  29.908   7.757  1.00 12.32           C  \\nATOM   1230  CZ  PHE A 156       6.782  30.406   8.513  1.00 10.18           C  \\nATOM   1231  N   TYR A 157       8.408  26.935   2.151  1.00  9.69           N  \\nATOM   1232  CA  TYR A 157       8.148  26.463   0.798  1.00 12.31           C  \\nATOM   1233  C   TYR A 157       8.091  24.945   0.719  1.00 10.42           C  \\nATOM   1234  O   TYR A 157       7.292  24.403  -0.047  1.00 11.35           O  \\nATOM   1235  CB  TYR A 157       9.233  26.978  -0.156  1.00 15.16           C  \\nATOM   1236  CG  TYR A 157       9.257  28.492  -0.367  1.00 18.73           C  \\nATOM   1237  CD1 TYR A 157       8.380  29.366   0.268  1.00 18.52           C  \\nATOM   1238  CD2 TYR A 157      10.211  28.971  -1.242  1.00 20.44           C  \\nATOM   1239  CE1 TYR A 157       8.466  30.724   0.025  1.00 22.04           C  \\nATOM   1240  CE2 TYR A 157      10.305  30.320  -1.488  1.00 22.70           C  \\nATOM   1241  CZ  TYR A 157       9.436  31.178  -0.852  1.00 23.03           C  \\nATOM   1242  OH  TYR A 157       9.572  32.528  -1.089  1.00 26.15           O  \\nATOM   1243  N   THR A 158       8.940  24.198   1.455  1.00 11.35           N  \\nATOM   1244  CA  THR A 158       8.836  22.727   1.522  1.00 10.11           C  \\nATOM   1245  C   THR A 158       7.440  22.347   2.064  1.00 11.11           C  \\nATOM   1246  O   THR A 158       6.811  21.408   1.558  1.00 11.43           O  \\nATOM   1247  CB  THR A 158       9.935  22.129   2.470  1.00 12.09           C  \\nATOM   1248  OG1 THR A 158      11.216  22.451   1.933  1.00 11.36           O  \\nATOM   1249  CG2 THR A 158       9.791  20.657   2.624  1.00 10.87           C  \\nATOM   1250  N   LEU A 159       6.888  23.056   3.072  1.00  9.25           N  \\nATOM   1251  CA  LEU A 159       5.533  22.754   3.506  1.00  9.93           C  \\nATOM   1252  C   LEU A 159       4.489  22.986   2.388  1.00 10.42           C  \\nATOM   1253  O   LEU A 159       3.560  22.185   2.272  1.00 11.22           O  \\nATOM   1254  CB  LEU A 159       5.171  23.608   4.711  1.00  9.03           C  \\nATOM   1255  CG  LEU A 159       3.789  23.318   5.269  1.00 10.96           C  \\nATOM   1256  CD1 LEU A 159       3.635  21.828   5.651  1.00  9.29           C  \\nATOM   1257  CD2 LEU A 159       3.580  24.223   6.469  1.00 13.71           C  \\nATOM   1258  N   VAL A 160       4.596  24.046   1.548  1.00 10.91           N  \\nATOM   1259  CA  VAL A 160       3.662  24.284   0.439  1.00  9.19           C  \\nATOM   1260  C   VAL A 160       3.803  23.137  -0.556  1.00  9.60           C  \\nATOM   1261  O   VAL A 160       2.781  22.638  -1.031  1.00 10.29           O  \\nATOM   1262  CB  VAL A 160       3.981  25.653  -0.211  1.00  8.50           C  \\nATOM   1263  CG1 VAL A 160       3.402  25.754  -1.622  1.00 10.56           C  \\nATOM   1264  CG2 VAL A 160       3.403  26.741   0.659  1.00  7.98           C  \\nATOM   1265  N   ARG A 161       5.033  22.667  -0.839  1.00  9.10           N  \\nATOM   1266  CA  ARG A 161       5.206  21.554  -1.767  1.00  9.82           C  \\nATOM   1267  C   ARG A 161       4.644  20.240  -1.215  1.00 10.44           C  \\nATOM   1268  O   ARG A 161       4.166  19.395  -1.984  1.00 12.88           O  \\nATOM   1269  CB  ARG A 161       6.679  21.398  -2.115  1.00  9.39           C  \\nATOM   1270  CG  ARG A 161       7.146  22.640  -2.816  1.00  9.70           C  \\nATOM   1271  CD  ARG A 161       8.594  22.398  -3.227  1.00 14.12           C  \\nATOM   1272  NE  ARG A 161       9.122  23.635  -3.780  1.00 14.15           N  \\nATOM   1273  CZ  ARG A 161      10.144  24.313  -3.257  1.00 12.45           C  \\nATOM   1274  NH1 ARG A 161      10.823  23.858  -2.204  1.00 13.24           N  \\nATOM   1275  NH2 ARG A 161      10.485  25.475  -3.819  1.00 13.52           N  \\nATOM   1276  N   GLU A 162       4.663  20.046   0.112  1.00 10.71           N  \\nATOM   1277  CA  GLU A 162       4.054  18.877   0.722  1.00 11.80           C  \\nATOM   1278  C   GLU A 162       2.522  18.887   0.568  1.00 11.20           C  \\nATOM   1279  O   GLU A 162       1.921  17.848   0.274  1.00 12.90           O  \\nATOM   1280  CB  GLU A 162       4.430  18.815   2.216  1.00 13.86           C  \\nATOM   1281  CG  GLU A 162       5.870  18.460   2.580  1.00 14.10           C  \\nATOM   1282  CD  GLU A 162       6.268  17.038   2.224  1.00 17.23           C  \\nATOM   1283  OE1 GLU A 162       5.571  16.116   2.627  1.00 19.29           O  \\nATOM   1284  OE2 GLU A 162       7.278  16.827   1.545  1.00 22.14           O  \\nATOM   1285  N   ILE A 163       1.869  20.045   0.731  1.00 10.13           N  \\nATOM   1286  CA  ILE A 163       0.426  20.175   0.542  1.00 10.58           C  \\nATOM   1287  C   ILE A 163       0.128  19.878  -0.933  1.00 13.43           C  \\nATOM   1288  O   ILE A 163      -0.796  19.118  -1.235  1.00 15.01           O  \\nATOM   1289  CB  ILE A 163      -0.015  21.590   0.891  1.00 11.33           C  \\nATOM   1290  CG1 ILE A 163       0.084  21.777   2.400  1.00 11.27           C  \\nATOM   1291  CG2 ILE A 163      -1.486  21.815   0.437  1.00 12.37           C  \\nATOM   1292  CD1 ILE A 163      -0.006  23.245   2.894  1.00 12.56           C  \\nATOM   1293  N   ARG A 164       0.904  20.456  -1.871  1.00 11.72           N  \\nATOM   1294  CA  ARG A 164       0.770  20.163  -3.283  1.00 13.64           C  \\nATOM   1295  C   ARG A 164       0.813  18.657  -3.626  1.00 17.97           C  \\nATOM   1296  O   ARG A 164       0.150  18.197  -4.559  1.00 18.12           O  \\nATOM   1297  CB  ARG A 164       1.897  20.887  -4.029  1.00 12.51           C  \\nATOM   1298  CG  ARG A 164       1.647  22.371  -4.120  1.00 12.49           C  \\nATOM   1299  CD  ARG A 164       2.801  22.888  -4.950  1.00 14.72           C  \\nATOM   1300  NE  ARG A 164       2.730  24.327  -5.113  1.00 12.82           N  \\nATOM   1301  CZ  ARG A 164       3.800  25.071  -5.405  1.00 13.29           C  \\nATOM   1302  NH1 ARG A 164       5.018  24.539  -5.527  1.00 11.56           N  \\nATOM   1303  NH2 ARG A 164       3.595  26.381  -5.572  1.00 12.55           N  \\nATOM   1304  N   GLN A 165       1.590  17.831  -2.935  1.00 20.23           N  \\nATOM   1305  CA  GLN A 165       1.673  16.435  -3.337  1.00 27.50           C  \\nATOM   1306  C   GLN A 165       0.782  15.499  -2.502  1.00 29.12           C  \\nATOM   1307  O   GLN A 165       0.826  14.274  -2.655  1.00 29.30           O  \\nATOM   1308  CB  GLN A 165       3.168  16.070  -3.286  1.00 30.34           C  \\nATOM   1309  CG  GLN A 165       4.018  16.904  -4.309  1.00 35.17           C  \\nATOM   1310  CD  GLN A 165       4.109  16.470  -5.796  1.00 37.71           C  \\nATOM   1311  OE1 GLN A 165       4.256  17.293  -6.719  1.00 36.70           O  \\nATOM   1312  NE2 GLN A 165       4.079  15.166  -6.094  1.00 38.13           N  \\nATOM   1313  N   HIS A 166      -0.068  16.075  -1.638  1.00 29.28           N  \\nATOM   1314  CA  HIS A 166      -0.982  15.296  -0.820  1.00 31.13           C  \\nATOM   1315  C   HIS A 166      -2.214  14.971  -1.665  1.00 33.30           C  \\nATOM   1316  O   HIS A 166      -2.630  15.786  -2.497  1.00 34.96           O  \\nATOM   1317  CB  HIS A 166      -1.420  16.090   0.428  1.00 30.89           C  \\nATOM   1318  CG  HIS A 166      -2.247  15.319   1.475  1.00 32.17           C  \\nATOM   1319  ND1 HIS A 166      -3.579  15.231   1.603  1.00 33.52           N  \\nATOM   1320  CD2 HIS A 166      -1.712  14.560   2.502  1.00 32.95           C  \\nATOM   1321  CE1 HIS A 166      -3.857  14.464   2.647  1.00 33.07           C  \\nATOM   1322  NE2 HIS A 166      -2.727  14.064   3.184  1.00 33.06           N  \\nATOM   1323  OXT HIS A 166      -2.763  13.888  -1.454  1.00 36.27           O  \\nTER    1324      HIS A 166                                                      \\nHETATM 1325 MG    MG A 168       4.771  33.860  19.213  1.00  9.42          MG  \\nHETATM 1326  PG  GNP A 167       5.113  32.223  21.994  1.00  9.66           P  \\nHETATM 1327  O1G GNP A 167       4.844  32.596  23.390  1.00  9.26           O  \\nHETATM 1328  O2G GNP A 167       4.203  32.910  21.112  1.00 11.20           O  \\nHETATM 1329  O3G GNP A 167       4.640  30.769  21.802  1.00 10.09           O  \\nHETATM 1330  N3B GNP A 167       6.701  32.303  21.621  1.00  8.18           N  \\nHETATM 1331  PB  GNP A 167       7.310  32.034  20.111  1.00  8.77           P  \\nHETATM 1332  O1B GNP A 167       7.468  30.605  19.791  1.00  8.35           O  \\nHETATM 1333  O2B GNP A 167       6.696  32.775  19.021  1.00  9.38           O  \\nHETATM 1334  O3A GNP A 167       8.792  32.643  20.152  1.00 10.15           O  \\nHETATM 1335  PA  GNP A 167       9.280  34.140  19.710  1.00 10.18           P  \\nHETATM 1336  O1A GNP A 167       9.263  34.268  18.220  1.00 10.07           O  \\nHETATM 1337  O2A GNP A 167       8.594  35.173  20.510  1.00  9.33           O  \\nHETATM 1338  O5' GNP A 167      10.826  34.056  20.132  1.00 11.46           O  \\nHETATM 1339  C5' GNP A 167      11.170  33.911  21.535  1.00 10.56           C  \\nHETATM 1340  C4' GNP A 167      12.587  34.445  21.811  1.00 11.96           C  \\nHETATM 1341  O4' GNP A 167      13.560  33.649  21.095  1.00 11.66           O  \\nHETATM 1342  C3' GNP A 167      12.813  35.910  21.359  1.00 12.06           C  \\nHETATM 1343  O3' GNP A 167      13.793  36.538  22.221  1.00 14.52           O  \\nHETATM 1344  C2' GNP A 167      13.325  35.679  19.947  1.00 11.90           C  \\nHETATM 1345  O2' GNP A 167      14.151  36.750  19.499  1.00 14.99           O  \\nHETATM 1346  C1' GNP A 167      14.242  34.478  20.156  1.00 12.90           C  \\nHETATM 1347  N9  GNP A 167      14.454  33.739  18.896  1.00 12.98           N  \\nHETATM 1348  C8  GNP A 167      13.518  33.262  17.993  1.00 11.76           C  \\nHETATM 1349  N7  GNP A 167      14.016  32.636  16.979  1.00 12.34           N  \\nHETATM 1350  C5  GNP A 167      15.413  32.694  17.200  1.00 12.36           C  \\nHETATM 1351  C6  GNP A 167      16.521  32.219  16.402  1.00 12.30           C  \\nHETATM 1352  O6  GNP A 167      16.507  31.591  15.338  1.00 12.16           O  \\nHETATM 1353  N1  GNP A 167      17.735  32.503  16.995  1.00 11.60           N  \\nHETATM 1354  C2  GNP A 167      17.909  33.082  18.236  1.00 13.91           C  \\nHETATM 1355  N2  GNP A 167      19.173  33.280  18.658  1.00 14.60           N  \\nHETATM 1356  N3  GNP A 167      16.884  33.498  18.996  1.00 13.01           N  \\nHETATM 1357  C4  GNP A 167      15.675  33.331  18.398  1.00 14.36           C  \\nHETATM 1358  O   HOH A 169      13.588  43.386  17.779  1.00 31.80           O  \\nHETATM 1359  O   HOH A 170       8.248  36.064  23.209  1.00 13.63           O  \\nHETATM 1360  O   HOH A 171       3.254  37.845  19.454  1.00 18.29           O  \\nHETATM 1361  O   HOH A 172       5.945  35.619  20.105  1.00 11.33           O  \\nHETATM 1362  O   HOH A 173       3.669  32.226  18.172  1.00  8.22           O  \\nHETATM 1363  O   HOH A 174       2.896  37.170  15.500  1.00 21.37           O  \\nHETATM 1364  O   HOH A 175       2.102  32.309  24.122  1.00 27.47           O  \\nHETATM 1365  O   HOH A 176      -1.425  37.650  19.667  1.00 24.98           O  \\nHETATM 1366  O   HOH A 177      -3.225  30.031  -8.217  1.00 23.57           O  \\nHETATM 1367  O   HOH A 178      16.678  26.321  26.916  1.00 36.70           O  \\nHETATM 1368  O   HOH A 179      -2.134  40.975   1.551  1.00 81.53           O  \\nHETATM 1369  O   HOH A 180       0.400  40.675  -0.629  1.00 24.66           O  \\nHETATM 1370  O   HOH A 181      10.944  44.793  16.009  1.00 48.47           O  \\nHETATM 1371  O   HOH A 182      17.207  42.570  16.786  1.00 43.23           O  \\nHETATM 1372  O   HOH A 183      14.999  45.167  19.922  1.00 39.52           O  \\nHETATM 1373  O   HOH A 184      11.598  37.755  13.202  1.00 11.72           O  \\nHETATM 1374  O   HOH A 185       6.958  38.922  18.146  1.00 26.17           O  \\nHETATM 1375  O   HOH A 186      14.078  32.033  27.772  1.00 57.73           O  \\nHETATM 1376  O   HOH A 187       9.126  37.906  19.595  1.00 32.42           O  \\nHETATM 1377  O   HOH A 188      -0.049  36.435  27.259  1.00 50.56           O  \\nHETATM 1378  O   HOH A 189       3.843  30.503  25.485  1.00 46.80           O  \\nHETATM 1379  O   HOH A 190       4.931  28.343  27.587  1.00 29.32           O  \\nHETATM 1380  O   HOH A 191       4.881  25.774  26.247  1.00 28.55           O  \\nHETATM 1381  O   HOH A 193       8.729  26.516  26.645  1.00 37.34           O  \\nHETATM 1382  O   HOH A 194      -5.515  23.731  -4.213  1.00 43.74           O  \\nHETATM 1383  O   HOH A 195      -7.006  26.635   0.729  1.00 25.77           O  \\nHETATM 1384  O   HOH A 196      -2.033  23.281  -4.754  1.00 36.89           O  \\nHETATM 1385  O   HOH A 197      -4.326  24.642  -0.854  1.00 28.59           O  \\nHETATM 1386  O   HOH A 198      -6.858  26.995  -2.808  1.00 28.83           O  \\nHETATM 1387  O   HOH A 200      -3.937  29.225   8.289  1.00 24.08           O  \\nHETATM 1388  O   HOH A 201      -8.934  24.037   2.795  1.00 34.16           O  \\nHETATM 1389  O   HOH A 202       1.444  26.302  19.318  1.00 12.37           O  \\nHETATM 1390  O   HOH A 203       2.031  24.632  27.307  1.00 50.09           O  \\nHETATM 1391  O   HOH A 204      15.125  28.341  23.174  1.00 13.88           O  \\nHETATM 1392  O   HOH A 205      12.808  31.541  25.056  1.00 46.88           O  \\nHETATM 1393  O   HOH A 206       8.329  34.933   0.543  1.00 53.21           O  \\nHETATM 1394  O   HOH A 207       0.760  42.969   2.336  1.00 35.42           O  \\nHETATM 1395  O   HOH A 208       1.270  40.298   7.071  1.00 76.61           O  \\nHETATM 1396  O   HOH A 209      -3.135  39.625   7.649  1.00 35.40           O  \\nHETATM 1397  O   HOH A 210       8.699  39.076   2.399  1.00 33.93           O  \\nHETATM 1398  O   HOH A 212      10.040  41.806   3.447  1.00 37.88           O  \\nHETATM 1399  O   HOH A 213      18.914  29.095   3.704  1.00 25.32           O  \\nHETATM 1400  O   HOH A 214      15.966  29.098   2.957  1.00 18.63           O  \\nHETATM 1401  O   HOH A 215      15.506  31.962   4.015  1.00 18.93           O  \\nHETATM 1402  O   HOH A 216      22.935  30.533  10.845  1.00 39.81           O  \\nHETATM 1403  O   HOH A 217       6.476  43.893  23.325  1.00 49.09           O  \\nHETATM 1404  O   HOH A 218      11.746  41.248  28.612  1.00 36.22           O  \\nHETATM 1405  O   HOH A 219      11.480  42.270  25.715  1.00 29.66           O  \\nHETATM 1406  O   HOH A 220       9.344  34.251  -8.278  1.00 36.61           O  \\nHETATM 1407  O   HOH A 221      -0.111  19.667  -8.066  1.00 44.76           O  \\nHETATM 1408  O   HOH A 222      -0.528  30.460 -10.943  1.00 24.29           O  \\nHETATM 1409  O   HOH A 223     -11.352  24.274  19.491  1.00 31.21           O  \\nHETATM 1410  O   HOH A 224      -4.293  30.642 -16.466  1.00127.55           O  \\nHETATM 1411  O   HOH A 225       3.466  29.729 -16.232  1.00 26.15           O  \\nHETATM 1412  O   HOH A 227      -4.288  18.610   4.198  1.00 34.77           O  \\nHETATM 1413  O   HOH A 228       4.626  16.154   5.151  1.00 14.96           O  \\nHETATM 1414  O   HOH A 229       5.681  11.069   2.580  1.00 59.76           O  \\nHETATM 1415  O   HOH A 230       1.173  14.021   3.406  1.00 32.15           O  \\nHETATM 1416  O   HOH A 231       7.935  22.484  20.504  1.00 10.86           O  \\nHETATM 1417  O   HOH A 232       3.071  10.004  14.546  1.00 23.48           O  \\nHETATM 1418  O   HOH A 233       2.437  13.381   8.401  1.00 17.75           O  \\nHETATM 1419  O   HOH A 234       4.779   7.500   6.154  1.00 71.09           O  \\nHETATM 1420  O   HOH A 235      -4.250   8.748   6.880  1.00 24.76           O  \\nHETATM 1421  O   HOH A 236     -13.381  22.830  17.669  1.00 39.09           O  \\nHETATM 1422  O   HOH A 237      16.969  39.052   8.276  1.00 50.25           O  \\nHETATM 1423  O   HOH A 238      -6.315   9.333  13.897  1.00 36.24           O  \\nHETATM 1424  O   HOH A 240      -0.160   4.395  12.468  1.00 38.79           O  \\nHETATM 1425  O   HOH A 241       1.295   7.931  12.335  1.00 49.03           O  \\nHETATM 1426  O   HOH A 242       4.067  11.551   6.968  1.00 46.97           O  \\nHETATM 1427  O   HOH A 243      -5.060   6.579  -0.216  1.00 50.14           O  \\nHETATM 1428  O   HOH A 244      18.164   7.151   5.573  1.00 71.67           O  \\nHETATM 1429  O   HOH A 245       2.068   8.904   7.636  1.00 45.86           O  \\nHETATM 1430  O   HOH A 246      24.661  23.134  12.757  1.00 34.69           O  \\nHETATM 1431  O   HOH A 247      21.051  23.714   6.239  1.00 24.64           O  \\nHETATM 1432  O   HOH A 248      25.855  31.137  20.712  1.00 34.53           O  \\nHETATM 1433  O   HOH A 249      24.868  30.890  26.223  1.00 48.47           O  \\nHETATM 1434  O   HOH A 250      26.967  20.447  19.448  1.00 47.77           O  \\nHETATM 1435  O   HOH A 252      28.453  14.196  14.602  1.00 39.78           O  \\nHETATM 1436  O   HOH A 253      19.639  16.256   9.760  1.00 37.54           O  \\nHETATM 1437  O   HOH A 254      16.629   7.383  10.526  1.00 60.90           O  \\nHETATM 1438  O   HOH A 255       4.972   7.888  14.276  1.00 54.95           O  \\nHETATM 1439  O   HOH A 256      14.184  14.684   5.643  1.00 31.40           O  \\nHETATM 1440  O   HOH A 257      11.766  17.254   1.278  1.00 41.82           O  \\nHETATM 1441  O   HOH A 258      16.274  16.365   6.342  1.00 17.29           O  \\nHETATM 1442  O   HOH A 259      18.346  24.095   6.620  1.00 17.36           O  \\nHETATM 1443  O   HOH A 260      22.479  35.073  19.694  1.00 43.96           O  \\nHETATM 1444  O   HOH A 261      20.580  31.747   4.906  1.00 59.46           O  \\nHETATM 1445  O   HOH A 262      18.196  25.514   4.261  1.00 28.75           O  \\nHETATM 1446  O   HOH A 263      12.857  20.183   1.149  1.00 21.79           O  \\nHETATM 1447  O   HOH A 264      19.514  15.912  24.943  1.00 55.52           O  \\nHETATM 1448  O   HOH A 265       8.925   7.579   5.621  1.00 77.94           O  \\nHETATM 1449  O   HOH A 266      17.622  21.835  26.964  1.00 27.48           O  \\nHETATM 1450  O   HOH A 267      24.258  19.786  14.155  1.00 20.48           O  \\nHETATM 1451  O   HOH A 268      23.806  15.180  15.136  1.00 19.86           O  \\nHETATM 1452  O   HOH A 269      21.478  14.557  11.134  1.00 29.66           O  \\nHETATM 1453  O   HOH A 270      17.293  45.621  24.753  1.00 39.18           O  \\nHETATM 1454  O   HOH A 271      -3.518  40.206  17.649  1.00 44.75           O  \\nHETATM 1455  O   HOH A 272       2.143  19.852  24.090  1.00 40.10           O  \\nHETATM 1456  O   HOH A 273       7.101  35.918  -4.704  1.00 31.76           O  \\nHETATM 1457  O   HOH A 274       5.462  38.756  -5.521  1.00 27.51           O  \\nHETATM 1458  O   HOH A 275       5.748  20.565  28.638  1.00 58.06           O  \\nHETATM 1459  O   HOH A 276       8.814  31.258  -8.964  1.00 23.79           O  \\nHETATM 1460  O   HOH A 277     -10.427  35.895   4.301  1.00 58.91           O  \\nHETATM 1461  O   HOH A 278      -8.713  28.579   2.333  1.00 69.36           O  \\nHETATM 1462  O   HOH A 279      24.472  24.237  17.316  1.00 32.14           O  \\nHETATM 1463  O   HOH A 281      16.968  32.701  21.815  1.00 23.13           O  \\nHETATM 1464  O   HOH A 282      19.541  34.346  21.581  1.00 53.25           O  \\nHETATM 1465  O   HOH A 283      24.847  23.384  26.621  1.00 61.18           O  \\nHETATM 1466  O   HOH A 284       1.409  11.113   2.873  1.00 44.87           O  \\nHETATM 1467  O   HOH A 285       1.556  11.488  -0.914  1.00 37.86           O  \\nHETATM 1468  O   HOH A 286       9.758  12.945   3.889  1.00 36.47           O  \\nHETATM 1469  O   HOH A 287       3.827  13.375   4.622  1.00 44.90           O  \\nHETATM 1470  O   HOH A 288      14.279   6.175  15.391  1.00 78.25           O  \\nHETATM 1471  O   HOH A 289       7.573  13.968   2.191  1.00 76.47           O  \\nHETATM 1472  O   HOH A 290       2.323  15.482   1.418  1.00 35.15           O  \\nHETATM 1473  O   HOH A 291      11.997  14.573   2.962  1.00 49.86           O  \\nHETATM 1474  O   HOH A 292      23.798  33.550  17.880  1.00 24.64           O  \\nHETATM 1475  O   HOH A 293      14.657  36.046  -0.571  1.00 51.86           O  \\nHETATM 1476  O   HOH A 294      24.456  25.366   4.004  1.00 53.23           O  \\nHETATM 1477  O   HOH A 296      -0.603  12.134  -4.185  1.00 52.38           O  \\nHETATM 1478  O   HOH A 297      -8.372  34.415  -4.471  1.00 74.53           O  \\nHETATM 1479  O   HOH A 300     -14.033  36.501   0.062  1.00 50.21           O  \\nHETATM 1480  O   HOH A 301     -17.102  35.735  -2.806  1.00 42.58           O  \\nHETATM 1481  O   HOH A 303     -17.383  40.542   1.431  1.00 77.16           O  \\nHETATM 1482  O   HOH A 304     -19.024  41.557  -3.411  1.00107.16           O  \\nHETATM 1483  O   HOH A 305      -9.876  37.804   0.839  1.00 90.65           O  \\nHETATM 1484  O   HOH A 306     -14.053  36.466   4.552  1.00 82.05           O  \\nHETATM 1485  O   HOH A 307     -14.183  39.558   0.339  1.00 58.66           O  \\nHETATM 1486  O   HOH A 309     -12.547  28.677   8.301  1.00 74.88           O  \\nHETATM 1487  O   HOH A 311     -14.787  32.746  -2.171  1.00 40.79           O  \\nHETATM 1488  O   HOH A 312      -6.916  30.740   3.095  1.00 36.53           O  \\nHETATM 1489  O   HOH A 315      -8.312  33.027   7.340  1.00 83.28           O  \\nHETATM 1490  O   HOH A 316      -6.280  35.646   7.624  1.00 75.97           O  \\nHETATM 1491  O   HOH A 317      -5.230  31.565   9.609  1.00 55.67           O  \\nHETATM 1492  O   HOH A 318       6.747  37.258   1.208  1.00 57.93           O  \\nHETATM 1493  O   HOH A 319       4.747  42.419  12.683  1.00 57.74           O  \\nHETATM 1494  O   HOH A 320       2.984  41.950  16.191  1.00 41.69           O  \\nHETATM 1495  O   HOH A 321       3.867  49.208  12.056  1.00 24.57           O  \\nHETATM 1496  O   HOH A 322       7.530  42.894  17.776  1.00 79.28           O  \\nHETATM 1497  O   HOH A 324      19.822  39.457  17.826  1.00 66.22           O  \\nHETATM 1498  O   HOH A 325      12.391  45.557  24.973  1.00 48.70           O  \\nHETATM 1499  O   HOH A 326      14.944  43.928  23.794  1.00 62.45           O  \\nHETATM 1500  O   HOH A 327     -12.984  24.008  12.563  1.00 40.31           O  \\nHETATM 1501  O   HOH A 330       0.572  41.133  19.277  1.00 50.77           O  \\nHETATM 1502  O   HOH A 331      -6.196  37.979  23.043  1.00 48.93           O  \\nHETATM 1503  O   HOH A 332      -3.526  30.410  19.065  1.00 21.26           O  \\nHETATM 1504  O   HOH A 333      -7.995  32.093  21.685  1.00 54.11           O  \\nHETATM 1505  O   HOH A 334      -8.483  28.891  20.063  1.00 70.37           O  \\nHETATM 1506  O   HOH A 335     -11.670  31.022  16.981  1.00 76.86           O  \\nHETATM 1507  O   HOH A 336     -11.082  34.450  18.910  1.00 43.07           O  \\nHETATM 1508  O   HOH A 339      -6.521  37.738  15.165  1.00 64.11           O  \\nHETATM 1509  O   HOH A 340     -10.829  35.877  15.629  1.00 58.36           O  \\nHETATM 1510  O   HOH A 341      -6.552  41.166  12.231  1.00 55.67           O  \\nHETATM 1511  O   HOH A 342      -1.242  39.671  12.185  1.00 61.04           O  \\nHETATM 1512  O   HOH A 343      -4.281  38.934  13.464  1.00 32.14           O  \\nHETATM 1513  O   HOH A 344       0.386  41.208  14.039  1.00 40.80           O  \\nHETATM 1514  O   HOH A 345       0.777  39.663  21.603  1.00 67.71           O  \\nHETATM 1515  O   HOH A 346      -7.879  34.848  19.044  1.00 52.60           O  \\nHETATM 1516  O   HOH A 347       1.121  32.594 -11.957  1.00 32.30           O  \\nHETATM 1517  O   HOH A 348       0.924  32.597 -14.716  1.00 48.34           O  \\nHETATM 1518  O   HOH A 349      -2.241  27.348  20.565  1.00 26.54           O  \\nHETATM 1519  O   HOH A 350       3.259  26.214  29.881  1.00 69.22           O  \\nHETATM 1520  O   HOH A 351      -8.318  34.095  28.392  1.00 81.96           O  \\nHETATM 1521  O   HOH A 352      -2.927  29.775  31.412  1.00 80.92           O  \\nHETATM 1522  O   HOH A 353     -10.410  32.034  25.972  1.00 60.17           O  \\nHETATM 1523  O   HOH A 354      -8.122  30.393  25.378  1.00 62.33           O  \\nHETATM 1524  O   HOH A 356      -6.138  18.882  24.644  1.00 49.26           O  \\nHETATM 1525  O   HOH A 358      -1.406  19.033  23.084  1.00 82.05           O  \\nHETATM 1526  O   HOH A 359       4.169   7.339  10.280  1.00 66.03           O  \\nHETATM 1527  O   HOH A 360      -9.761  30.165   7.702  1.00 46.15           O  \\nHETATM 1528  O   HOH A 361     -16.002  27.801  11.700  1.00 41.85           O  \\nHETATM 1529  O   HOH A 362      16.215  30.489  29.264  1.00 67.21           O  \\nHETATM 1530  O   HOH A 364      21.417  27.685  26.140  1.00 49.57           O  \\nHETATM 1531  O   HOH A 366      15.824  34.767  30.662  1.00 25.90           O  \\nHETATM 1532  O   HOH A 367      18.527  35.965  24.111  1.00 68.69           O  \\nHETATM 1533  O   HOH A 369      12.103  35.275  24.945  1.00 51.37           O  \\nHETATM 1534  O   HOH A 370      24.782  11.970  23.641  1.00 55.92           O  \\nHETATM 1535  O   HOH A 371      14.337  21.394  28.627  1.00 30.11           O  \\nHETATM 1536  O   HOH A 372      -1.367  12.600  23.053  1.00 44.43           O  \\nHETATM 1537  O   HOH A 373       7.060   8.567  21.416  1.00 47.99           O  \\nHETATM 1538  O   HOH A 374       7.639   8.040  16.420  1.00 38.76           O  \\nHETATM 1539  O   HOH A 376      -1.282  15.566  25.487  1.00 67.79           O  \\nHETATM 1540  O   HOH A 378      17.945  30.924  23.806  1.00 31.50           O  \\nHETATM 1541  O   HOH A 379      17.360  33.209  33.337  1.00 30.46           O  \\nHETATM 1542  O   HOH A 380       3.971   6.567  22.089  1.00 86.80           O  \\nHETATM 1543  O   HOH A 381      -0.065  10.022  26.928  1.00 61.86           O  \\nHETATM 1544  O   HOH A 382      -4.450  12.966  21.222  1.00 42.82           O  \\nHETATM 1545  O   HOH A 383      -2.584   6.719   5.938  1.00 40.64           O  \\nHETATM 1546  O   HOH A 384      25.567  31.936  12.845  1.00 73.17           O  \\nHETATM 1547  O   HOH A 385      24.446  29.383  22.970  1.00 36.61           O  \\nHETATM 1548  O   HOH A 386      24.865  38.167  18.227  1.00 60.74           O  \\nHETATM 1549  O   HOH A 387      23.135  35.012  14.543  1.00 59.19           O  \\nHETATM 1550  O   HOH A 388      19.325   9.556  18.093  1.00 92.18           O  \\nHETATM 1551  O   HOH A 389      25.326  12.873  16.251  1.00 49.27           O  \\nHETATM 1552  O   HOH A 390      20.219  10.405  10.956  1.00 63.05           O  \\nHETATM 1553  O   HOH A 392       9.452   5.453   8.249  1.00 47.81           O  \\nHETATM 1554  O   HOH A 394      10.139  12.252   0.254  1.00 29.37           O  \\nHETATM 1555  O   HOH A 395      28.004  37.416  13.466  1.00 52.29           O  \\nHETATM 1556  O   HOH A 396      24.283  36.829  12.307  1.00 64.74           O  \\nHETATM 1557  O   HOH A 397      13.748  34.031  -3.019  1.00 47.80           O  \\nHETATM 1558  O   HOH A 398      16.981  34.189   2.006  1.00 80.51           O  \\nHETATM 1559  O   HOH A 399      -4.426  24.223  -7.240  1.00 38.09           O  \\nHETATM 1560  O   HOH A 400      19.676  33.981   3.336  1.00 64.86           O  \\nHETATM 1561  O   HOH A 401      14.633  28.745  -0.317  1.00 66.25           O  \\nHETATM 1562  O   HOH A 402      -2.139  15.576  22.716  1.00 65.01           O  \\nHETATM 1563  O   HOH A 403       9.873  11.090  20.760  1.00 53.88           O  \\nHETATM 1564  O   HOH A 404      18.291  36.379  26.927  1.00 57.23           O  \\nHETATM 1565  O   HOH A 405       0.138   6.145  15.269  1.00 82.80           O  \\nHETATM 1566  O   HOH A 406      -5.515  38.166  18.689  1.00 48.28           O  \\nHETATM 1567  O   HOH A 407      -0.399   4.688   8.809  1.00 58.35           O  \\nHETATM 1568  O   HOH A 408       0.813  25.584  22.307  1.00 40.04           O  \\nCONECT  117 1325                                                                \\nCONECT  265 1325                                                                \\nCONECT 1325  117  265 1328 1333                                                 \\nCONECT 1325 1361 1362                                                           \\nCONECT 1326 1327 1328 1329 1330                                                 \\nCONECT 1327 1326                                                                \\nCONECT 1328 1325 1326                                                           \\nCONECT 1329 1326                                                                \\nCONECT 1330 1326 1331                                                           \\nCONECT 1331 1330 1332 1333 1334                                                 \\nCONECT 1332 1331                                                                \\nCONECT 1333 1325 1331                                                           \\nCONECT 1334 1331 1335                                                           \\nCONECT 1335 1334 1336 1337 1338                                                 \\nCONECT 1336 1335                                                                \\nCONECT 1337 1335                                                                \\nCONECT 1338 1335 1339                                                           \\nCONECT 1339 1338 1340                                                           \\nCONECT 1340 1339 1341 1342                                                      \\nCONECT 1341 1340 1346                                                           \\nCONECT 1342 1340 1343 1344                                                      \\nCONECT 1343 1342                                                                \\nCONECT 1344 1342 1345 1346                                                      \\nCONECT 1345 1344                                                                \\nCONECT 1346 1341 1344 1347                                                      \\nCONECT 1347 1346 1348 1357                                                      \\nCONECT 1348 1347 1349                                                           \\nCONECT 1349 1348 1350                                                           \\nCONECT 1350 1349 1351 1357                                                      \\nCONECT 1351 1350 1352 1353                                                      \\nCONECT 1352 1351                                                                \\nCONECT 1353 1351 1354                                                           \\nCONECT 1354 1353 1355 1356                                                      \\nCONECT 1355 1354                                                                \\nCONECT 1356 1354 1357                                                           \\nCONECT 1357 1347 1350 1356                                                      \\nCONECT 1361 1325                                                                \\nCONECT 1362 1325                                                                \\nMASTER      312    0    2    5    6    0    9    6 1567    1   38   13          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewer_16835744454165597.setStyle({\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewer_16835744454165597.addSurface(1,{\"opacity\": 0.7, \"color\": \"white\"});\n",
       "\tviewer_16835744454165597.zoomTo();\n",
       "viewer_16835744454165597.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pdb_file = 'data/ras/ras_protein.pdb'\n",
    "\n",
    "with open(pdb_file, 'r') as f:\n",
    "    pdb_data = f.read()\n",
    "\n",
    "view = py3Dmol.view(width=700, height=700)\n",
    "view.addModel(pdb_data, 'pdb')\n",
    "view.setStyle({'cartoon': {'color': 'spectrum'}})\n",
    "view.addSurface(py3Dmol.VDW,{'opacity':0.7,'color':'white'})\n",
    "\n",
    "view.zoomTo()\n",
    "view.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2b7ffbf9-716f-40ae-b97d-9d0d92689299",
   "metadata": {},
   "source": [
    "--------\n",
    "## Inference: Multiple Ligands w/ Single Protein"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "99ae25ea",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Download copy of zinc250k from github\n",
    "url = 'https://raw.githubusercontent.com/divelab/DIG_storage/main/ggraph/zinc250k_property.csv'\n",
    "df = pd.read_csv(url)\n",
    "smiles = df['smile']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "6d60801e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>qed</th>\n",
       "      <th>penalized_logp</th>\n",
       "      <th>smile</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0.731901</td>\n",
       "      <td>3.139906</td>\n",
       "      <td>CC(C)(C)c1ccc2occ(CC(=O)Nc3ccccc3F)c2c1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0.941112</td>\n",
       "      <td>0.172386</td>\n",
       "      <td>C[C@@H]1CC(Nc2cncc(-c3nncn3C)c2)C[C@@H](C)C1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0.626105</td>\n",
       "      <td>2.618423</td>\n",
       "      <td>N#Cc1ccc(-c2ccc(O[C@@H](C(=O)N3CCCC3)c3ccccc3)...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0.716225</td>\n",
       "      <td>-1.974847</td>\n",
       "      <td>CCOC(=O)[C@@H]1CCCN(C(=O)c2nc(-c3ccc(C)cc3)n3c...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0.809572</td>\n",
       "      <td>-0.205555</td>\n",
       "      <td>N#CC1=C(SCC(=O)Nc2cccc(Cl)c2)N=C([O-])[C@H](C#...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "        qed  penalized_logp                                              smile\n",
       "0  0.731901        3.139906            CC(C)(C)c1ccc2occ(CC(=O)Nc3ccccc3F)c2c1\n",
       "1  0.941112        0.172386       C[C@@H]1CC(Nc2cncc(-c3nncn3C)c2)C[C@@H](C)C1\n",
       "2  0.626105        2.618423  N#Cc1ccc(-c2ccc(O[C@@H](C(=O)N3CCCC3)c3ccccc3)...\n",
       "3  0.716225       -1.974847  CCOC(=O)[C@@H]1CCCN(C(=O)c2nc(-c3ccc(C)cc3)n3c...\n",
       "4  0.809572       -0.205555  N#CC1=C(SCC(=O)Nc2cccc(Cl)c2)N=C([O-])[C@H](C#..."
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1a02144a-b88a-4c45-a66b-28cec60d655f",
   "metadata": {},
   "source": [
    "### Process the Data\n",
    "While EquiBind comes with a preprocessed dataset that consists of tens of thousands of ligand and protein combinations, the Zinc250k dataset only consisted of SMILES strings. In the code cell directly below I was able to convert the strings to RDKit Molecule objects adding explicit hydrogens per the requirements of the model. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "62bc2c35",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[13:35:18] UFFTYPER: Unrecognized charge state for atom: 5\n",
      "[13:35:22] UFFTYPER: Unrecognized charge state for atom: 6\n",
      "[13:38:14] UFFTYPER: Unrecognized charge state for atom: 2\n",
      "[13:39:02] UFFTYPER: Unrecognized charge state for atom: 18\n",
      "[13:39:21] UFFTYPER: Warning: hybridization set to SP3 for atom 7\n",
      "[13:39:47] UFFTYPER: Warning: hybridization set to SP3 for atom 3\n",
      "[13:40:47] UFFTYPER: Unrecognized charge state for atom: 1\n",
      "[13:42:38] UFFTYPER: Unrecognized charge state for atom: 17\n",
      "[13:42:43] UFFTYPER: Unrecognized charge state for atom: 18\n"
     ]
    }
   ],
   "source": [
    "def process_smiles(smi):\n",
    "    \"\"\"\n",
    "    Converts a SMILES string to a 3D molecule object.\n",
    "\n",
    "    Args:\n",
    "    - smi (str): The SMILES string to convert.\n",
    "\n",
    "    Returns:\n",
    "    - An RDKit 3D molecule object, or None if the SMILES string could not be converted.\n",
    "    \"\"\"\n",
    "    try:\n",
    "        mol = Chem.MolFromSmiles(smi)\n",
    "        mol = Chem.AddHs(mol)  # Add explicit Hs\n",
    "        AllChem.EmbedMolecule(mol)\n",
    "        return mol\n",
    "    except:\n",
    "        # Skip if unrecognized charge state or similar\n",
    "        return None\n",
    "\n",
    "# Create a list of threads to process SMILES strings\n",
    "threads = []\n",
    "molecules = [None] * len(smiles)\n",
    "for i, smi in enumerate(smiles):\n",
    "    t = threading.Thread(\n",
    "        target=lambda index, smi: molecules.__setitem__(index, process_smiles(smi)), args=(i, smi)\n",
    "    )\n",
    "    t.start()\n",
    "    threads.append(t)\n",
    "\n",
    "# Wait for threads to finish\n",
    "for t in threads:\n",
    "    t.join()\n",
    "\n",
    "# Write molecules to SDF file\n",
    "writer = Chem.SDWriter('molecules.sdf')\n",
    "for mol in molecules:\n",
    "    writer.write(mol)\n",
    "writer.close()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3ac3fdee-9178-4a91-8265-5a0b7278bd1b",
   "metadata": {},
   "source": [
    "--------------\n",
    "### Changes to EquiBind\n",
    "There were several issues that came up while trying to use the existing model to perform inference. In order to get around this, several changes to the code base had to be done. Here are the specific changes and the files that were modified.\n",
    "\n",
    "\n",
    ">`commons/process_mols.py`\n",
    ">\n",
    ">Update the lig_atom_featurizer so that it will skip atoms with unknown chiral tags by setting their feature vector to all zeros.\n",
    ">\n",
    ">```python\n",
    ">def lig_atom_featurizer(mol):\n",
    ">    ComputeGasteigerCharges(mol)\n",
    ">    ringinfo = mol.GetRingInfo()\n",
    ">    atom_features_list = []\n",
    ">    for idx, atom in enumerate(mol.GetAtoms()):\n",
    ">        g_charge = atom.GetDoubleProp('_GasteigerCharge')\n",
    ">        chiral_tag_index = safe_index(allowable_features['possible_chirality_list'], str(atom.GetChiralTag()))\n",
    ">\n",
    ">        if chiral_tag_index != -1:\n",
    ">            atom_features_list.append([\n",
    ">                safe_index(allowable_features['possible_atomic_num_list'], atom.GetAtomicNum()),\n",
    ">                chiral_tag_index,\n",
    ">                safe_index(allowable_features['possible_degree_list'], atom.GetTotalDegree()),\n",
    ">                safe_index(allowable_features['possible_formal_charge_list'], atom.GetFormalCharge()),\n",
    ">                safe_index(allowable_features['possible_implicit_valence_list'], atom.GetImplicitValence()),\n",
    ">                safe_index(allowable_features['possible_numH_list'], atom.GetTotalNumHs()),\n",
    ">                safe_index(allowable_features['possible_number_radical_e_list'], atom.GetNumRadicalElectrons()),\n",
    ">                safe_index(allowable_features['possible_hybridization_list'], str(atom.GetHybridization())),\n",
    ">                allowable_features['possible_is_aromatic_list'].index(atom.GetIsAromatic()),\n",
    ">                safe_index(allowable_features['possible_numring_list'], ringinfo.NumAtomRings(idx)),\n",
    ">                allowable_features['possible_is_in_ring3_list'].index(ringinfo.IsAtomInRingOfSize(idx, 3)),\n",
    ">                allowable_features['possible_is_in_ring4_list'].index(ringinfo.IsAtomInRingOfSize(idx, 4)),\n",
    ">                allowable_features['possible_is_in_ring5_list'].index(ringinfo.IsAtomInRingOfSize(idx, 5)),\n",
    ">                allowable_features['possible_is_in_ring6_list'].index(ringinfo.IsAtomInRingOfSize(idx, 6)),\n",
    ">                allowable_features['possible_is_in_ring7_list'].index(ringinfo.IsAtomInRingOfSize(idx, 7)),\n",
    ">                allowable_features['possible_is_in_ring8_list'].index(ringinfo.IsAtomInRingOfSize(idx, 8)),\n",
    ">                g_charge if not np.isnan(g_charge) and not np.isinf(g_charge) else 0.\n",
    ">            ])\n",
    ">        else:\n",
    ">            atom_features_list.append([0] * 17)\n",
    ">\n",
    ">    return torch.tensor(atom_features_list)\n",
    ">```\n",
    "\n",
    "<br>\n",
    "\n",
    ">`commons/geometry_utils.py`\n",
    ">\n",
    "> When performing SVD in the rigid_transform_Kabsch_3D calculation, it was common for certain molecules to fail to converge. Thus, I had to add error handling when finding the rotation. If Numpy's SVD function failed to converge, I would skip the molecule.\n",
    ">```python\n",
    ">def rigid_transform_Kabsch_3D(A, B):\n",
    ">    assert A.shape[1] == B.shape[1]\n",
    ">    num_rows, num_cols = A.shape\n",
    ">    if num_rows != 3:\n",
    ">        raise Exception(f\"matrix A is not 3xN, it is {num_rows}x{num_cols}\")\n",
    ">    num_rows, num_cols = B.shape\n",
    ">    if num_rows != 3:\n",
    ">        raise Exception(f\"matrix B is not 3xN, it is {num_rows}x{num_cols}\")\n",
    ">\n",
    ">    # find mean column wise: 3 x 1\n",
    ">    centroid_A = np.mean(A, axis=1, keepdims=True)\n",
    ">    centroid_B = np.mean(B, axis=1, keepdims=True)\n",
    ">\n",
    ">    # subtract mean\n",
    ">    Am = A - centroid_A\n",
    ">    Bm = B - centroid_B\n",
    ">\n",
    ">    H = Am @ Bm.T\n",
    ">\n",
    ">    # find rotation\n",
    ">    try:\n",
    ">        U, S, Vt = np.linalg.svd(H)\n",
    ">    except np.linalg.LinAlgError:\n",
    ">        # Handle the error here, e.g., return None or a special value\n",
    ">        return None, None\n",
    ">\n",
    ">    R = Vt.T @ U.T\n",
    ">\n",
    ">    # special reflection case\n",
    ">    if np.linalg.det(R) < 0:\n",
    ">        # print(\"det(R) < R, reflection detected!, correcting for it ...\")\n",
    ">        SS = np.diag([1.,1.,-1.])\n",
    ">        R = (Vt.T @ SS) @ U.T\n",
    ">    assert math.fabs(np.linalg.det(R) - 1) < 1e-5\n",
    ">\n",
    ">    t = -R @ centroid_A + centroid_B\n",
    ">    return R, t\n",
    ">```\n",
    "\n",
    "<br>\n",
    "\n",
    ">`multiligand_inference.py`\n",
    ">The modifications above result in None types being returned when a convergence or molecule issue arises. Thus, two additional functions in the main script had to be updated to process None types.\n",
    ">```python\n",
    "> def run_corrections(lig, lig_coord, ligs_coords_pred_untuned):\n",
    ">     input_coords = lig_coord.detach().cpu()\n",
    ">     prediction = ligs_coords_pred_untuned.detach().cpu()\n",
    ">     lig_input = deepcopy(lig)\n",
    ">     conf = lig_input.GetConformer()\n",
    ">     for i in range(lig_input.GetNumAtoms()):\n",
    ">         x, y, z = input_coords.numpy()[i]\n",
    ">         conf.SetAtomPosition(i, Point3D(float(x), float(y), float(z)))\n",
    ">\n",
    ">     lig_equibind = deepcopy(lig)\n",
    ">     conf = lig_equibind.GetConformer()\n",
    ">     for i in range(lig_equibind.GetNumAtoms()):\n",
    ">         x, y, z = prediction.numpy()[i]\n",
    ">         conf.SetAtomPosition(i, Point3D(float(x), float(y), float(z)))\n",
    ">\n",
    ">     coords_pred = lig_equibind.GetConformer().GetPositions()\n",
    ">\n",
    ">     Z_pt_cloud = coords_pred\n",
    ">     rotable_bonds = get_torsions([lig_input])\n",
    ">     new_dihedrals = np.zeros(len(rotable_bonds))\n",
    ">     for idx, r in enumerate(rotable_bonds):\n",
    ">         new_dihedrals[idx] = get_dihedral_vonMises(lig_input, lig_input.GetConformer(), r, Z_pt_cloud)\n",
    ">     optimized_mol = apply_changes(lig_input, new_dihedrals, rotable_bonds)\n",
    ">     optimized_conf = optimized_mol.GetConformer()\n",
    ">     coords_pred_optimized = optimized_conf.GetPositions()\n",
    ">     \n",
    ">     ####################### ADD THIS ##########################\n",
    ">     R, t = rigid_transform_Kabsch_3D(coords_pred_optimized.T, coords_pred.T)\n",
    ">     if R is None or t is None:\n",
    ">         # Handle the situation when SVD doesn't converge, e.g., skip this case\n",
    ">         return None\n",
    ">     ###########################################################\n",
    ">     \n",
    ">     coords_pred_optimized = (R @ (coords_pred_optimized).T).T + t.squeeze()\n",
    ">     for i in range(optimized_mol.GetNumAtoms()):\n",
    ">         x, y, z = coords_pred_optimized[i]\n",
    ">         optimized_conf.SetAtomPosition(i, Point3D(float(x), float(y), float(z)))\n",
    ">     return optimized_mol\n",
    ">\n",
    "> def write_while_inferring(dataloader, model, args):\n",
    ">     \n",
    ">     full_output_path = os.path.join(args.output_directory, \"output.sdf\")\n",
    ">     full_failed_path = os.path.join(args.output_directory, \"failed.txt\")\n",
    ">     full_success_path = os.path.join(args.output_directory, \"success.txt\")\n",
    ">\n",
    ">     w_or_a = \"a\" if args.skip_in_output else \"w\"\n",
    ">     with torch.no_grad(), open(full_output_path, w_or_a) as file, open(\n",
    ">         full_failed_path, \"a\") as failed_file, open(full_success_path, w_or_a) as success_file:\n",
    ">         with Chem.SDWriter(file) as writer:\n",
    ">             i = 0\n",
    ">             total_ligs = len(dataloader.dataset)\n",
    ">             for batch in dataloader:\n",
    ">                 i += args.batch_size\n",
    ">                 if i % 1000 == 0:\n",
    ">                     print(f\"Entering batch ending in index {min(i, total_ligs)}/{len(dataloader.dataset)}\")\n",
    ">                 ligs, lig_coords, lig_graphs, rec_graphs, geometry_graphs, true_indices, failed_in_batch = batch\n",
    ">                 for failure in failed_in_batch:\n",
    ">                     if failure[1] == \"Skipped\":\n",
    ">                        continue\n",
    ">                    failed_file.write(f\"{failure[0]} {failure[1]}\")\n",
    ">                    failed_file.write(\"\\n\")\n",
    ">                if ligs is None:\n",
    ">                    continue\n",
    ">                lig_graphs = lig_graphs.to(args.device)\n",
    ">                rec_graphs = rec_graphs.to(args.device)\n",
    ">                geometry_graphs = geometry_graphs.to(args.device)\n",
    ">\n",
    ">                out_ligs, out_lig_coords, predictions, successes, failures = run_batch(\n",
    ">                    model, ligs, lig_coords, lig_graphs, \n",
    ">                    rec_graphs, geometry_graphs, true_indices,\n",
    ">                )\n",
    ">\n",
    ">                ####################### MODIFY THIS ##########################\n",
    ">                opt_mols = []\n",
    ">                for lig, lig_coord, prediction in zip(\n",
    ">                    out_ligs, out_lig_coords, predictions\n",
    ">                ):\n",
    ">                    opt_mol = run_corrections(lig, lig_coord, prediction)\n",
    ">                    if opt_mol is not None:\n",
    ">                       opt_mols.append(opt_mol)\n",
    ">              ##############################################################\n",
    ">\n",
    ">                for mol, success in zip(opt_mols, successes):\n",
    ">                    writer.write(mol)\n",
    ">                    success_file.write(f\"{success[0]} {success[1]}\")\n",
    ">                    success_file.write(\"\\n\")\n",
    ">                for failure in failures:\n",
    ">                    failed_file.write(f\"{failure[0]} {failure[1]}\")\n",
    ">                    failed_file.write(\"\\n\")\n",
    ">```                "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e87a9f30-142f-4e1f-9e23-1eebd2e9ba83",
   "metadata": {},
   "source": [
    "-----------\n",
    "### Execute Equibind"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "a8f4b135-91c2-4cd2-8e82-5ced6b554005",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[2023-05-08 13:43:42.302639] [ Using Seed :  1  ]\n",
      "Found 249455 previously calculated ligands\n",
      "device = cuda:0\n"
     ]
    }
   ],
   "source": [
    "!python multiligand_inference.py -o data/output/ -r data/ras/ras_protein.pdb -l molecules.sdf --batch_size $BATCH_SIZE"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "91ab18bc-9e5b-4720-83ec-98c524b557a3",
   "metadata": {},
   "source": [
    "------------\n",
    "\n",
    "### Inspection Functions\n",
    "After using the multiligand_inference.py script that comes with the equibind model to infer potential ligands for the RAS protein from the Zinc250k dataset, I was provided with an output.sdf file that appeared to consist of the same number of molecules that I input. This observation was perplexing, and to better understand what was happening, I had to develop various functions to examine the contents of the output.sdf file. These functions were necessary to inspect the molecular positions of each atom in the ligand. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "762fd26a",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "def count_molecules_in_chunk(sdf_file, start, end):\n",
    "    \"\"\"\n",
    "    Counts the number of molecules in a chunk of an SDF file.\n",
    "\n",
    "    Args:\n",
    "    - sdf_file (str): The path to the SDF file.\n",
    "    - start (int): The index of the first molecule in the chunk.\n",
    "    - end (int): The index of the last molecule in the chunk (exclusive).\n",
    "\n",
    "    Returns:\n",
    "    - The number of molecules in the chunk.\n",
    "    \"\"\"\n",
    "    sdf_supplier = Chem.SDMolSupplier(sdf_file)\n",
    "    count = 0\n",
    "    for i in range(start, end):\n",
    "        mol = sdf_supplier[i]\n",
    "        if mol is not None:\n",
    "            count += 1\n",
    "    return count\n",
    "\n",
    "def count_molecules(sdf_file, num_processes):\n",
    "    \"\"\"\n",
    "    Counts the total number of molecules in an SDF file, using multiple processes.\n",
    "\n",
    "    Args:\n",
    "    - sdf_file (str): The path to the SDF file.\n",
    "    - num_processes (int): The number of processes to use.\n",
    "\n",
    "    Returns:\n",
    "    - The total number of molecules in the SDF file.\n",
    "    \"\"\"\n",
    "    sdf_supplier = Chem.SDMolSupplier(sdf_file)\n",
    "    total_molecules = len(sdf_supplier)\n",
    "\n",
    "    chunk_size = total_molecules // num_processes\n",
    "    chunks = [(i * chunk_size, (i + 1) * chunk_size) for i in range(num_processes - 1)]\n",
    "    chunks.append(((num_processes - 1) * chunk_size, total_molecules))\n",
    "\n",
    "    molecule_count = 0\n",
    "\n",
    "    with concurrent.futures.ProcessPoolExecutor(max_workers=num_processes) as executor:\n",
    "        futures = [executor.submit(count_molecules_in_chunk, sdf_file, start, end) for start, end in chunks]\n",
    "        for future in concurrent.futures.as_completed(futures):\n",
    "            molecule_count += future.result()\n",
    "\n",
    "    return molecule_count\n",
    "\n",
    "def mol_to_smiles(mol):\n",
    "    \"\"\"\n",
    "    Converts a molecule object to a SMILES string.\n",
    "\n",
    "    Args:\n",
    "    - mol (rdkit.Chem.rdchem.Mol): The molecule object to convert.\n",
    "\n",
    "    Returns:\n",
    "    - A SMILES string representing the molecule.\n",
    "    \"\"\"\n",
    "    return Chem.MolToSmiles(mol) if mol is not None else None\n",
    "\n",
    "def sdf_to_smiles_set(sdf_file, num_threads=os.cpu_count()):\n",
    "    \"\"\"\n",
    "    Converts an SDF file to a set of unique SMILES strings, using multiple threads.\n",
    "\n",
    "    Args:\n",
    "    - sdf_file (str): The path to the SDF file.\n",
    "    - num_threads (int): The number of threads to use (default is the number of CPU cores).\n",
    "\n",
    "    Returns:\n",
    "    - A set containing the unique SMILES strings in the SDF file.\n",
    "    \"\"\"\n",
    "    sdf_supplier = Chem.SDMolSupplier(sdf_file)\n",
    "    mols = [mol for mol in sdf_supplier]\n",
    "\n",
    "    with concurrent.futures.ThreadPoolExecutor(max_workers=num_threads) as executor:\n",
    "        smiles_list = list(executor.map(mol_to_smiles, mols))\n",
    "\n",
    "    return set(smiles_list)\n",
    "\n",
    "def compare_sdf_files(sdf_file1, sdf_file2):\n",
    "    \"\"\"\n",
    "    Compares the molecules in two SDF files and reports the differences.\n",
    "\n",
    "    Args:\n",
    "    - sdf_file1 (str): The path to the first SDF file.\n",
    "    - sdf_file2 (str): The path to the second SDF file.\n",
    "\n",
    "    Returns:\n",
    "    - None. Prints a message indicating the differences between the files.\n",
    "    \"\"\"\n",
    "    with concurrent.futures.ThreadPoolExecutor(max_workers=2) as executor:\n",
    "        smiles_set1_future = executor.submit(sdf_to_smiles_set, sdf_file1)\n",
    "        smiles_set2_future = executor.submit(sdf_to_smiles_set, sdf_file2)\n",
    "\n",
    "        smiles_set1 = smiles_set1_future.result()\n",
    "        smiles_set2 = smiles_set2_future.result()\n",
    "\n",
    "    difference1 = smiles_set1.difference(smiles_set2)\n",
    "    difference2 = smiles_set2.difference(smiles_set1)\n",
    "\n",
    "    if not difference1 and not difference2:\n",
    "        print(\"The two SDF files are identical.\")\n",
    "    else:\n",
    "        print(f\"Number of Molecules in {sdf_file1} not found in {sdf_file2}: {len(difference1)}\")\n",
    "        print(f\"Number of Molecules in {sdf_file2} not found in {sdf_file1}: {len(difference2)}\")\n",
    "        \n",
    "def process_molecule(mol, x_freq, y_freq, z_freq):\n",
    "    \"\"\"\n",
    "    Processes the input molecule and extracts the x, y, and z coordinates of each atom. \n",
    "    Updates the frequency dictionaries for each coordinate with the counts of atoms at that coordinate.\n",
    "\n",
    "    Args:\n",
    "    - mol (rdkit.Chem.rdchem.Mol): the input molecule to be processed\n",
    "    - x_freq (collections.defaultdict): a defaultdict representing the frequency of atoms at each x coordinate\n",
    "    - y_freq (collections.defaultdict): a defaultdict representing the frequency of atoms at each y coordinate\n",
    "    - z_freq (collections.defaultdict): a defaultdict representing the frequency of atoms at each z coordinate\n",
    "\n",
    "    Returns:\n",
    "    - None\n",
    "    \"\"\"\n",
    "    if mol is None:\n",
    "        return\n",
    "\n",
    "    conf = mol.GetConformer()\n",
    "    for atom in mol.GetAtoms():\n",
    "        pos = conf.GetAtomPosition(atom.GetIdx())\n",
    "        x, y, z = pos.x, pos.y, pos.z\n",
    "\n",
    "        x_freq[round(x)] += 1\n",
    "        y_freq[round(y)] += 1\n",
    "        z_freq[round(z)] += 1\n",
    "\n",
    "def process_sdf(sdf_file):\n",
    "    \"\"\"\n",
    "    Processes an SDF file and extracts the frequency of atoms at each x, y, and z coordinate.\n",
    "    Uses a ThreadPoolExecutor to parallelize the processing of each molecule in the SDF file.\n",
    "\n",
    "    Args:\n",
    "    - sdf_file (str): the path to the SDF file to be processed\n",
    "\n",
    "    Returns:\n",
    "    - x_freq (collections.defaultdict): a defaultdict representing the frequency of atoms at each x coordinate\n",
    "    - y_freq (collections.defaultdict): a defaultdict representing the frequency of atoms at each y coordinate\n",
    "    - z_freq (collections.defaultdict): a defaultdict representing the frequency of atoms at each z coordinate\n",
    "    \"\"\"\n",
    "    suppl = Chem.SDMolSupplier(sdf_file)\n",
    "    molecules = [mol for mol in suppl if mol is not None]\n",
    "\n",
    "    x_freq = defaultdict(int)\n",
    "    y_freq = defaultdict(int)\n",
    "    z_freq = defaultdict(int)\n",
    "\n",
    "    with concurrent.futures.ThreadPoolExecutor() as executor:\n",
    "        futures = [executor.submit(process_molecule, mol, x_freq, y_freq, z_freq) for mol in molecules]\n",
    "        concurrent.futures.wait(futures)\n",
    "    \n",
    "    return x_freq, y_freq, z_freq\n",
    "\n",
    "def plot_frequency(freq_dicts, title, ax):\n",
    "    \"\"\"\n",
    "    Plots the frequency of atoms at each x, y, and z coordinate for a list of frequency dictionaries.\n",
    "\n",
    "    Args:\n",
    "    - freq_dicts (list): a list of defaultdicts representing the frequency of atoms at each coordinate.\n",
    "    - title (str): the title of the plot.\n",
    "    - ax (matplotlib.axes._subplots.AxesSubplot): the axes to plot on.\n",
    "\n",
    "    Returns:\n",
    "    - None\n",
    "    \"\"\"\n",
    "    colors = ['blue', 'green', 'red']\n",
    "    labels = ['X', 'Y', 'Z']\n",
    "    \n",
    "    for freq_dict, color, label in zip(freq_dicts, colors, labels):\n",
    "        keys = sorted(freq_dict.keys())\n",
    "        values = [freq_dict[key] for key in keys]\n",
    "        ax.bar(keys, values, color=color, alpha=0.5, label=label)\n",
    "\n",
    "    ax.set_xlabel('Coordinate')\n",
    "    ax.set_ylabel('Frequency')\n",
    "    ax.set_title(title)\n",
    "    ax.legend()\n",
    "\n",
    "def plot_sdf_files(sdf_file1, sdf_file2):\n",
    "    \"\"\"\n",
    "    Compares the frequency of atoms at each x, y, and z coordinate for two SDF files.\n",
    "    Plots the frequency distributions for each coordinate side by side.\n",
    "\n",
    "    Args:\n",
    "    - sdf_file1 (str): the path to the first SDF file.\n",
    "    - sdf_file2 (str): the path to the second SDF file.\n",
    "\n",
    "    Returns:\n",
    "    - None\n",
    "    \"\"\"\n",
    "    x_freq1, y_freq1, z_freq1 = process_sdf(sdf_file1)\n",
    "    x_freq2, y_freq2, z_freq2 = process_sdf(sdf_file2)\n",
    "\n",
    "    fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(15, 5), sharey=True)\n",
    "    \n",
    "    plot_frequency([x_freq1, y_freq1, z_freq1], 'File 1: Frequency of X, Y, Z Coordinates', ax1)\n",
    "    plot_frequency([x_freq2, y_freq2, z_freq2], 'File 2: Frequency of X, Y, Z Coordinates', ax2)\n",
    "\n",
    "    plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "af510fef",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[13:43:52] Conflicting single bond directions around double bond at index 1.\n",
      "[13:43:52]   BondStereo set to STEREONONE and single bond directions set to NONE.\n",
      "[13:43:55] WARNING: not removing hydrogen atom with neighbor that has non-tetrahedral stereochemistry\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The number of molecules in molecules.sdf is 249455\n"
     ]
    }
   ],
   "source": [
    "sdf_file1 = 'molecules.sdf'\n",
    "print(f\"The number of molecules in {sdf_file1} is {count_molecules(sdf_file1, NUM_THREADS)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "c24a213c-73a6-424a-b942-05217cb58aee",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[13:44:10] WARNING: not removing hydrogen atom with neighbor that has non-tetrahedral stereochemistry\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The number of molecules in data/output/output.sdf is 249455\n"
     ]
    }
   ],
   "source": [
    "sdf_file2 = 'data/output/output.sdf'\n",
    "print(f\"The number of molecules in {sdf_file2} is {count_molecules(sdf_file2, NUM_THREADS)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "eb6c7524",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[13:44:38] Conflicting single bond directions around double bond at index 1.\n",
      "[13:44:38]   BondStereo set to STEREONONE and single bond directions set to NONE.\n",
      "[13:45:15] WARNING: not removing hydrogen atom with neighbor that has non-tetrahedral stereochemistry\n",
      "[13:45:16] WARNING: not removing hydrogen atom with neighbor that has non-tetrahedral stereochemistry\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of Molecules in molecules.sdf not found in data/output/output.sdf: 9453\n",
      "Number of Molecules in data/output/output.sdf not found in molecules.sdf: 9446\n"
     ]
    }
   ],
   "source": [
    "compare_sdf_files(sdf_file1, sdf_file2)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ce40c874-ae93-40aa-a5ef-4919291c81f3",
   "metadata": {},
   "source": [
    "------------\n",
    "### Atom Position Distribution\n",
    "Upon examining the output generated by the Equibind model, it became apparent that the model was capable of generating predicted positions for any molecule it was presented with. The implementation details surrounding this feature were not explicated in the original publication nor on the Github repository. The multiligand_inference.py script was a post-publication addition by the authors. Upon visual inspection of the output, it was observed that the positions of individual atoms in the molecule were substantially shifted. The observed distributions in the Cartesian coordinates showed a leftward shift in X, an upward shift in Y, and a forward shift in Z. It is likely that these shifts in the predicted atomic positions were influenced by the protein's spatial orientation along the three dimensions."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "0744b2dc-e59d-440e-9bb2-bf5d9f8f86cd",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[13:47:12] Conflicting single bond directions around double bond at index 1.\n",
      "[13:47:12]   BondStereo set to STEREONONE and single bond directions set to NONE.\n",
      "[13:47:31] WARNING: not removing hydrogen atom with neighbor that has non-tetrahedral stereochemistry\n",
      "[13:49:10] WARNING: not removing hydrogen atom with neighbor that has non-tetrahedral stereochemistry\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAACbUAAAOrCAYAAABqKvlaAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjUuMywgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/NK7nSAAAACXBIWXMAAB7CAAAewgFu0HU+AADjNElEQVR4nOzdd5hU5fk//vcC0gREpVlQlGjsihE0ooLGXmI3YgN7jJpoYuwFYzSamOgnJragYu/GWBIlKqIixhJrbNFIREFEIiIoIMv8/uDHfHeFbbDsjOzrdV1zXefMec7z3FP2zMy993NORaFQKAQAAAAAAAAAAADKQItSBwAAAAAAAAAAAADzKGoDAAAAAAAAAACgbChqAwAAAAAAAAAAoGwoagMAAAAAAAAAAKBsKGoDAAAAAAAAAACgbChqAwAAAAAAAAAAoGwoagMAAAAAAAAAAKBsKGoDAAAAAAAAAACgbChqAwAAAAAAAAAAoGwoagMAAAAAAAAAAKBsKGoDAAAAAAAAAACgbChqAwAAAAAAAAAAoGwoagMAAAAAAAAAAKBsKGoDAAAAAAAAAACgbChqAwAAAAAAAAAAoGwoagMAAAAAAAAAAKBsKGoDAAAAAAAAAACgbChqAwAAAAAAAAAAoGwoagMAAAAAAAAAAKBsKGoDKGNjx45NRUVFKioq0qtXrxrb9erVq9hu7NixTRYfQJK89dZbOfbYY7PuuuumY8eOxeORYxI0vaFDhxb//oYOHbrANsOHDy+2GTJkSJPGBwAASzr5POCbQD4Pyod8HkDNFLUBLEYDBw6s9mOwvreXXnqp1KEvFuPHj89f/vKXnHnmmdlxxx2z/PLLl92P5ao/DBpyO+GEE0odOpTEAw88kI022iiXX355Xn/99UybNm2R+vvNb35T/Lvq2LFj/vvf/9Z734svvri4b4cOHRbrMeWggw4qjrXmmmvmyy+/bND+11xzTXH/Nm3a5PXXX19MkSZDhgxZqOPagm41JVUWxWeffZbbbrsthx9+eDbaaKOstNJKadOmTTp27JhVVlkl2223XU4//fSMGTOm0ccGAAD4Ovm8/2f69Ol54IEHcuKJJ2bAgAFZYYUV0qZNmyy99NJZddVVs8cee+Sqq67K9OnTSxqnfB40jHyefN6iks8DoKm0KnUAADQPK664YiZMmFDqMIBGNH369AwePDgzZsxIkqywwgrZYost0rVr11RUVCRJOnXq1KA+f/rTn+aOO+7I888/n2nTpuWoo47Kww8/XOd+77zzTs4+++zi+q9+9ataZ8Qvqv/7v//LI488kokTJ+bf//53zjnnnPz617+u174TJkzISSedVFw/++yzs8466yyuUMvWF198kUsvvTQXX3xxPv300/m2z5o1K9OmTcu4cePyyCOP5Fe/+lXWXHPNDB06NPvvv3/xPUbzNmTIkFx//fVJkuuuu85MVQAAaCQHHXRQ/vznP+eLL76Yb9usWbPy/vvv5/333y9OYL3qqquy1157lSBSoCHk8+TzFoV8Ho1BPg9oCEVtAE2kb9++6devX73adu3adTFH0/S+iQVtHTt2zCGHHFKvtltuueVijgbKz/3335///e9/SZJ11103zz33XNq1a7dIfbZs2TLXXHNNNtlkk3z11VcZMWJEhg8fXusP20KhkCOOOKI4u7J///459thjFymOuiy//PL5wx/+kH333TdJ8rvf/S777bdfNtlkkzr3PfbYYzNlypQkSZ8+fXLKKacszlCz7bbbpkOHDgu17913352PPvqouL7eeus1Skzvv/9+dtttt7zyyivV7l9llVWywQYbpGvXrqmsrMxHH32Ul19+ORMnTkySvP322znggAMybty4nHzyyY0SCwAAQE2acz7vrrvuysyZM4vryy67bPr165cVV1wxhUIhb7zxRp599tkUCoV88skn2XvvvXPFFVfkhz/8YQmjls+DusjnyectLPk8AEpBURtAE9l5550bfJrnXr16pVAoLJ6ASqBt27bZaKON0rdv3/Tt2zfdu3fPDjvsUOqwarTccsvlD3/4Q6nDgLL1z3/+s7g8aNCgRU6AzbPBBhvk1FNPzXnnnZdk7mzPHXfcMT169Fhg+yuuuCKjRo1KkrRp0ybDhg1LixYtGiWW2uyzzz7Ze++9c/fdd6eysjKHHnpoXnjhhbRu3brGfe666678+c9/TpK0atUq1157bVq1WrxfyQ866KAcdNBBDd7vz3/+c/74xz8W14888sjss88+ixzP2LFj893vfreYXKuoqMigQYNy+umnZ911152vfaFQyPPPP5/LLrssN998c+bMmbPAMwVQf0OGDDEDEgAA6qG55/Pat2+fQYMG5bDDDstmm20232/t1157LQcffHDx0qvHHXdc+vfvn/XXX78E0c4lnwe1k8+Tz1sY8nmlJ58HNFeL/9sBAGTuj+XPP/88Y8aMye9///scfPDBWXPNNUsdFrAIqp5ifoUVVmjUvs8888xiQuTTTz+tcabm+++/n1NPPbW4fs4552SttdZq1Fhq88c//jHLL798krnJ/AsuuKDGtp9++mmOO+644vqpp56ajTbaaHGHuFDeeeedHHroocX1Pn365Pe///0i9ztr1qzsu+++xQRY27Ztc8899+Tmm29eYAIsmZsk69u3b2644Ya8/PLLjTa7FAAAgJodf/zx+c9//pNhw4Zl8803X2CxyXrrrZfHHnssq666apKksrKy1t/FQOnJ58nnNZR8HgClpKgNgCbRp0+fxT57CWhaX331VXG5sWdStm7dOtdcc02x33vuuSd33333fO2OPvrofP7550nmHmd+/vOfN2ocdenevXsuvfTS4voFF1yQV199dYFtTzzxxOJp99ddd92cddZZTRFig82YMSP77rtvPvvssyRJ586dc9ddd6Vt27aL3Pevf/3rPP/888X166+/PnvssUe9919vvfXyzDPPZLvttlvkWAAAAKjZb37zm3Tv3r3Odssuu2y1y/D99a9/XZxhAYtIPk8+r6Hk8wAoJUVtAGVs7NixqaioSEVFRXr16tVo/X711Ve58cYbs99++2X11VdPx44ds/TSS2e11VbLoEGD8uc//3mJuUxCKfTq1av4uo0dOzZJ8u677+aMM85Inz590rVr17Ro0aLGGV3Tp0/PFVdckd122y2rrrpq2rdvn44dO2aNNdbIYYcdlscee6xB8XzwwQc55ZRTsu6666ZDhw5Zdtlls8EGG+SMM87I+++/nyR5/PHHizEPHDhwgf0MHz682KY+p7lemPfvuHHjct5552XLLbfMiiuumDZt2mS55ZZLnz59ctJJJ+Xtt9+us48hQ4YUxx0+fHiSuc/p5Zdfni222CLdu3dPmzZt0rNnzwwaNCijR4+uV2xVPfXUU/nJT36SPn36pFu3bllqqaXSqVOnrL/++hk8eHBuvfXWfPnll8X2s2bNSteuXYtxPfPMM/Uea/PNNy/u96c//anBsS7I+++/n7PPPjubbbZZunfvntatW6d79+7ZbLPNcs4552TcuHE17jt06NBiPNdff33x/kMPPbR4/7zb448/vkhxbrrppvnJT35SXD/22GOrzSYdPnx4HnrooSRNd+r/BTnooIOyyy67JJl7fD3ssMNSWVlZrc2IESOKz1fLli1z7bXX1npZg1I69thji5eOmfd3tPrqqy9yv19++WW12aF77bVX9ttvvwb3s/TSS6d///61tlmU9/iCFAqF3HnnnRk0aFB69+6dDh06pEOHDundu3cOOOCA3HXXXfX63Bw4cOB8fx8TJkzIBRdckH79+qVHjx5p2bJlOnfuvMD9n3322Rx66KFZbbXV0q5du/To0SP9+/fPZZddlmnTptX78dTneF7T58Jjjz2W/fffP6uvvnratm2b5ZdfPltttVX+8Ic/VEuM1+aNN97IJZdckr322ivf/va307Fjxyy11FLp2rVrNtlkk5x44ol5/fXXa+1j3mdtXcehioqKWi8X1ZjfiZ577rkcd9xx2XjjjbPssstmqaWWynLLLZc111wzW2yxRX7605/m4Ycfrvb5AADAkqc55vOq/kabOnVq/ve//5UkjkUhnzeXfJ58nnze/yOfJ5+XyOfNI58nnwclUwBgsRkwYEAhSSFJ4Zxzzmnw/u+9915x/1VXXbXGdquuumqx3XvvvVdrnyNHjiz07t272L6m22abbVb44IMPGhxzQ1R9fPWJvSlcd9119XrOa/P11+Oqq64qtG3bdr7neMMNN5xv3zvuuKPQo0ePOl+fXXfdtTBlypQ6Y7n99tsLnTp1qrGfTp06Ff785z8XRo4cWbxvwIABdT43gwcPrnPs+r5/C4VCobKysnDWWWct8HmqemvVqlXh9NNPL8yZM6fGvgYPHlxsf9111xVef/31wtprr11rv2effXadj6dQKBTGjRtX2G677ep8fZIUNt1002r7/uxnPytuO+KII+o13htvvFHcZ+mlly5MnTq1XvvV5vzzz6/zeW7btm3hV7/61QL3P+ecc+r1+JMURo4cucjxTp8+vbD66qvP996bMGFCYdllly3ef/rppy/yWIvigw8+KCyzzDLFeC666KLitmnTphV69epV3HbSSSeVMNLaXXvttdVew5NPPrnR+r7hhhuq9f3UU081Wt9VLep7/OvefvvtQp8+fep8v3/nO98pvPPOO7X2VfV7wciRIwv33ntvtffxvNsyyywz374nn3xyoUWLFjWO/+1vf7vw+uuvV/sbrem7R32O51//XJg5c2bhqKOOqvU52HjjjQuTJk2q9TnYd99963X8qKioKJxwwgmF2bNnL7Cfqp+1dd1qeh4a6zvRV199VedzU/V2xhln1PocAQBQWvJ5DffKK69Ui+Pjjz9u0vHl8+TzaiOfJ59XE/m8usnnyecVCvJ5iXwelJLrwAE0I3feeWcOPPDA4syLtm3bZrPNNkuvXr3SsmXLvP322xkzZkxmz56dZ555Jt/97nfz3HPP1etSA6UwdOjQnHvuuUmSVVddtTiLspzceeedOfnkk5MkK620UjbffPMss8wyGT9+/HyzVi+55JL87Gc/K84g6dixY7773e+mZ8+eqayszOuvv57nnnsuhUIhDzzwQAYMGJCnn3467du3X+DY9913Xw444IDiDLOWLVtmyy23TO/evfPZZ5/l8ccfzyeffJL99tsvF1xwwWJ8FmpXWVmZH/zgB9VORb/CCitk0003Tbdu3TJt2rT84x//yLvvvpvZs2fnggsuyKRJk3L11VfX2ff48eOz7bbbZvz48encuXO23HLL9OjRI5988kkee+yx4unYf/GLX2SdddbJD37wgxr7+te//pXtttsuEyZMKN7XrVu3bL755unatWtmzJiRd999Ny+++GK+/PLLzJgxo9r+Rx11VH77298mSW6//fZceumlWXrppWuN/5prriku77vvvunYsWOdj7k2xx13XP74xz8W15deeulss8026dGjRz766KOMHDky06ZNy4wZM3Laaadl4sSJueSSS6r10a9fvxx77LFJkkcffTRvvvlmkuR73/te1lprrWptV1pppUWKN0nat2+fP/3pT/ne976XZO7p7QcNGpSrrrqqOMtzrbXWytlnn73IYy2KlVZaKRdffHGOPPLIJMk555yTPfbYI2uuuWZOP/304vFpjTXWyC9+8YsSRlqzV155pfjaJslWW22V888/v9H6rzorfZVVVqlzdubCaIz3eFVvvPFGBgwYkEmTJhXvW3/99bPRRhuloqIiL774YvHyFC+88EL69++fJ554ImuuuWadsT799NMZOnRovvrqq+LsyC5duuTjjz/Oiy++WK3tz3/+81x88cXF9Y4dO2brrbdOt27d8uGHH2bkyJF56623svPOOzfo8g8NcfTRR2f48OFp0aJFNt1006y11lqZM2dOnnnmmbz11ltJkn/+85855JBDar3c0LwzCrRq1SrrrLNO1lhjjXTu3DktW7bMxx9/nOeeey4ffvhhCoVCLr300sycOTOXX375fP0MHjw4kydPrvM4lMw9bn1dY34nOumkk6p9Jq200krp169funTpkkKhkMmTJ+f1118vPk8AANAQ34R8XtXL9rVr1y5dunSpsa18nnxeXeTzqpPPW3zk8+omn1edfJ58HlACpaunA1jyldPMztdee63Qvn37YrsTTjihMHny5Pnavfvuu4Utttii2G6nnXZqcNz1tahnaqs6e2VhZ2F+XWPP7GzVqlWhdevWhauvvnq+2YgzZswoLj/yyCPFGTtLLbVU4Ze//GVh2rRp8/X94osvFtZZZ51i/8ccc8wCY5g0aVKhS5cuxXZ9+vQpvP3229XazJw5s3DyyScXkhRat25dbQbPgiyumZ1nnXVWsV23bt0Kt99+e6GysnK+dnfeeWe1mXO33377AvurOrOzTZs2xdlp06dPr9Zu8uTJhW222abYdvXVV69xxuhnn31WWGONNYptu3TpUrj11lsX2H7atGmFm2++uXDooYfOt63qMeHaa6+t8TkpFObOFOrevXux/aLOgrv99tur/b0dcsghhc8++2y+x3nQQQdVa3fXXXfV2OfXZ9EuTkcccURxrKqzlVu0aFEYPXr0Yh27IbbddttibFtssUVh9OjRxb/tioqKwhNPPFHqEBfo6+/x7t27F8aPH9+oY1SdQbfvvvs2at+FQuO/x2fOnFnYcMMNi+26du1aePjhh+dr9/DDD1c73m688caFWbNmLbDPqseAVq1aFSoqKgrnnXfefO2rfj6MHDmyUFFRUdzvgAMOmO9xTZw4sbD99tvPdzxvrJmd846lffv2LbzxxhvV2s2ZM6dw6aWXVntOR40atcA+C4VC4dRTTy3ccccd8z2Gqv3dd999ha5duxb7e/LJJ2vsb2GOQ435nWjSpEmFVq1aFZIUWrZsWRg+fHiNnyXjx48v/P73vy8MGzasXnECAFAa8nkNV/UsWLvsskutbeXz5PMWRD5vweTzmoZ8Xs3k8+TzCgX5PPk8KC1FbQCLUdUvu3379i0ce+yxdd6qng63MZNgVX/s//KXv6w17mnTplU7vfszzzyzMA+/TuVe1NaxY8d6vWZf/wH19VMo33TTTbWOWVlZWe3HZ13tJ0yYUOjWrVsxYTZu3Lj52px22mnF/nr06FHr6aN/+MMfVou3KZNg7733XqFly5bF5/vrP66+7tFHHy32ufbaay/wh0bVH0RJCqeddlqN/X300UeFpZdeus73+hlnnFFss8wyyxTefPPNWuOsyU033VTsp3///rW2/fOf/1xsu9Zaay3UePNUVlYWVltttWJ/e++9d40/0ubMmVPYfffdi2179+69wKRkodC0SbApU6YUVlxxxWqvbZLCj3/848U6bkO99957hQ4dOlQ7jsxbPu6440odXo322muvYpwtW7ZslEtNfN28REGSwtChQxu178XxHq966Yalllqq8Pzzz9c4/rPPPlvt8V1//fULbFf1e0F9Po8LhULhu9/9brH99ttvX+Pf45dfflnYaKONqvXfWEmwJIU11lij8Pnnn9cY5z777FNs+8Mf/rDOx1WXZ555ptjffvvtV2O7hTkONeZ3ovvvv7+47cADD6zX+AAAlDf5vIZ54IEHqv12+Otf/1pre/k8+Tz5vPqRz2s68nk1k8+Tz2sI+TxgcVDUBrAYff3Lbn1uL774YnH/xkqCvfTSS8Xta665Zo3Xs6/q1ltvLe5z/PHHL8Sjr1u5F7XV9/aTn/ykWh9VX49+/frVOea9995bbP+9732vXnH+6le/Ku7z29/+ttq2ysrKQo8ePYrbr7zyylr7mjJlSrUf7U2ZBDvhhBOKbc4777w6+ywUCoUddtihuM8LL7ww3/aqP4i6du1a+PLLL2vtb7/99iu2v+yyy+bbPmPGjELnzp2Lba644op6xbkgM2bMKCy33HLFvmpL+u26667Fdr/5zW8WesxCoVD429/+VuyrdevWdc7Y++CDDwpLLbVUcZ+HHnpoge2aMglWKBQK11xzTbW/vV69ei1wBnSpXXbZZfMdJ3r16lVrAqGUfvvb31aL9YILLmj0MT777LNqY1x66aWN2v/ieI9vuummDfocPOaYY4rtN9tsswW2qfq9YKWVVip89dVXtfb5r3/9q9rz1pB/FDR2Euzuu++udey//vWvxbbf+c53am1bX/OST8svv3yNbRp6HGrs70Q333xzcdsJJ5xQZ18AAJQ/+bz6Gz9+fGGFFVYojrnddtvVuY98nnyefF79yOc1Lfm8+cnnzSWf1zDyeUBjaxFYSB9//HEeeOCBnH322dlpp53SpUuXVFRUpKKiIkOGDClZXG+88UZ+/vOfZ4MNNsiyyy6bdu3apVevXtl2221zwQUXZOzYsSWLDUrlr3/9a3F53333TcuWLevcZ5tttikuP/XUU4slrkU1dOjQFOYWaJft3/b+++9fZ5uqr0992ie1vz5vvPFGPvrooyRJq1at6uxzmWWWye67716vcRtbYz/2r9ttt93Stm3bWtv06dOnuLyg99EzzzyTKVOmJEk6duyYwYMH1yvOBWnTpk0OOeSQ4vq11167wHYTJkzIQw89lCRZaqmlqu2zMB577LHi8k477ZQVVlih1vYrrbRSdtxxx+L6yJEjF2n8xlAoFHLjjTdWu2/e945yc+yxx2bLLbesdt+f/vSndOjQoUQR1ezpp5/OqaeeWlzfddddq603ls8//7zaemM/F439Hp82bVqef/754vphhx1WZwxHHHFEcfm5557L9OnTa22/9957p1WrVrW2qRpX3759s9Zaa9Xafuutt87KK69cZ6wN1bZt2+y66661tqnrWLogb7/9dm655Zacd955Oemkk3L88cfnuOOOK94+++yzJMnkyZMzbty4hY6/qsb+TrTKKqsUl+++++7i5y8AACyqcs/nzZo1K/vss08mTJiQJFluueVy3XXX1bmffN5c8nk1k8+bSz6vacnnzU8+b37yefJ5QNOr/agLtejevXupQ6imUCjkrLPOykUXXZTZs2dX2/bf//43//3vf/Poo4+mffv2OeGEE0oTJM3aOeeck6FDh5Zk7DFjxlRbPu644+rcp1AoFJcb64vnN82qq666yMm173znO3W2qfr6PPjgg3nppZfq3GfeD4Nk/ten6v5rrbVWlllmmTr722yzzXLzzTfX2a4xTZ48OW+//XZx/ZJLLqlXQuP1118vLtf13lx//fXr7G/55ZcvLld9Xud55plnisubbbZZ2rVrV2eftTnyyCNz6aWXJkluuOGGXHDBBfP9EL7++uuLn2W77bZbunXrtkhjvvjii8Xl/v3712uf/v375/7770+S/POf/1yk8RvD1Vdfnccff7zafe+9917OOuus/Pa3vy1NUDWoqKjIYYcdlieffDLJ3GPJtttuW+Ko5jdp0qTst99++eqrr5IkvXr1yg033LBYEosdO3astj5t2rRG7b+x3+Mvv/xyKisrk8xN2G2wwQZ19rfRRhtl6aWXzvTp01NZWZmXX345m2++eY3t6/P5UPV4vummm9bZvqKiIptuumk++OCDOts2xLe//e20bt261jZ1HUurevDBB3PWWWdVe93q8sknn6Rnz571bl+Txv5OtOmmm2bVVVfNf//734wbNy7rrrtuBg8enN12261RPjMAACgt+byaxxk8eHCefvrpJHMLeG655ZastNJKi23M+pLPW7zk8+TzGkI+r/HJ58nn1Zd8nnweLCkUtdEoevbsmbXXXjsjRowoWQxHHXVUhg0bliTZcMMNc/jhh2fDDTdMu3btMm7cuLz++uu56667ynIGCCxu48ePLy4/9thj1WbA1Menn37a2CE1G127dq2zTdXX5957723wGF9/fSZNmlRcrjrjpDaN8cOioebN5J3n8ssvb3Afdb0365MAXGqppYrL85IBVU2cOLG4vPrqqzcgugVbZ5110r9//4wePToTJ07MAw88kD322KNam6ozmw8//PBFHrPqe2LVVVet1z69evUqLn/yySeLHMOi+OCDD3LyyScX17/3ve/l0UcfTZL83//9XwYNGpRNNtmkVOF9I82ZMycHHHBAPvzwwyRzZx3feeedWXbZZRfLeJ06dUqrVq2Kyd15s6UbS2O/x6v217Nnz7RoUfcJrlu0aJGePXvmzTffXGCfX1efz4dyOZ439Fj69QkmVQ0dOjTnnntug2P4+uzghdXY34mWWmqp3HTTTdlll10yderU/O9//8sll1ySSy65JK1bt87GG2+crbbaKrvssku23HJLv0UAAKi3cs7nHXfccbntttuSzP0tdP3112eHHXZYbOM1Nfm8msnnyefVl3xe45PPk89rCPm8/0c+D77ZXH6UhXb22Wfn/vvvz0cffZT3338/V111Vcliufbaa4sFbSeccEL++c9/5vjjj89WW22Vvn37Zq+99sqZZ56Zl156Kcccc0zJ4oRSqWuGRV3mzW6h4eozo2NRX5+v/9ioOmOqffv29epj6aWXXqQYFsaiPu6k9h9aSRrlx0bVH16NdYr1o446qrj89UsWPPnkk8UZryuttFKjJIWrvifq+1pXbddYPz4X1jHHHJOpU6cmSTbZZJM8/PDDxeelsrIyRxxxRJ3vBaobOnRoHnnkkeL6JZdcstgTiVWTU1VnaDeGxn6PL0x/dfX5dfX5fCiX43ljJW7+/ve/V0uAbbHFFvnTn/6UF198MZ988klmzJhRvAxRoVDIgAEDim3nzJnTKDEsju9EW2yxRV5++eUcdthh1Z7/WbNm5Zlnnsmvf/3rDBgwIGuttVb+/Oc/L9L4AAA0H+Wazzv99NOrFTL98Y9/zKBBgxbLWKUin1cz+by55PPqJp/X+OTz5PMaQj7v/5HPg282RW0stHPPPTe77rpryS9D+vnnn+ekk05KkgwcODCXXHJJrdX3dZ1qFZZEVb+Q3XvvvdW+YNb3xuJT9fV56aWXGvzafP2SClUTNV988UW9Ypg+fXqjPJaq6vrBUvVxd+7ceaHel8OHD2/0uL+u6mnWG+sU6/vuu286d+6cJPnrX/9abZbrNddcU1w+9NBD07Jly0Uer+p7or6vddV2Xz/VfFO65ZZb8sADDySZO4PqmmuuScuWLXPVVVcV30Mvv/xyfvOb35Qsxm+ahx9+OOeff35x/YADDmiSov8tttiiuPyPf/yjUftu7Pf4wvRXV58Lo1yO542l6t/pUUcdlSeffDJHHHFENtpooyy//PJp06ZNtfaLIwG/uL4T9erVK9dcc00mTZqURx55JGeffXa23nrrasnOt99+O3vttVd+97vfNfrjAgBgyVOO+bzzzz8/v/rVr4rrF154YX74wx82+jjfBPJ58nnyeTWTz2t88nnzt5PPaxryefJ5UGqK2igLs2fPzjXXXJOdd945K664Ytq0aZMuXbpkq622yqWXXpoZM2bUuO/NN99cPG3o0KFDmyhi+GapWnz673//u4SRsCCN/fpUPQX2+++/X699xo0bV2eb+p6Kep66Zs9UfdxTpkypdlruclI1zvfee69R+mzXrl0OOuigJHNnCV1//fVJ5v7gu+uuu5LMnUl12GGHNcp4C/Oe+O9//1tc7tKlS6PE0VCTJk3KT37yk+L6ySefnA022CDJ3FmCVRM5v/jFLxzf6mHcuHE58MADi0nqddZZJ1dffXWTjL3NNtsUl//73//m6aefbrS+G/s9XrW/cePG1eufQXPmzKl2LG2Mv5vFdTwvhcrKyowaNSrJ3Es7/PKXv6xzn/o+5oZY3N+J2rVrl+9973s599xz89hjj2Xy5Mm5++67s+GGGxbbnHbaacVLhQAAQE3KLZ936aWX5swzzyyun3nmmTnllFNKGFFpyefJ58nnLZh8XuOTz/t/5POalnyefB6UA0VtlNy7776bDTfcMEcccUT+9re/ZcKECZk1a1YmT56cJ598MieeeGI22GCDGj+k7rzzziRJt27diqc0LRQK+eijj/Luu+822iwc+CbbdNNNi8sPP/xwCSNhQRr79dloo42Ky2+++Wa9Ts38zDPP1Nmm6iylyZMn19n+1VdfrXX7CiuskFVWWaW4PmLEiDr7LIXNNtusuDxmzJh8+eWXjdLvgi5ZcNtttxVnZW2zzTZZbbXVGmWsPn36FJfrm3gYPXp0cXnjjTdulDga6sc//nE++eSTJMlaa62Vs846q9r2448/vvj6zJgxI0ceeaQzS9biq6++yn777Vf8++3QoUPuuuuuJrtcyb777lstMdSYs9sa+z2+4YYbFmdVT5s2rc7jWTJ3hvG8v9+WLVtWS3osrKrH8/ocpwuFQqPPmm0sn3zySWbNmpVk7vf2qgm+BXn99deLf/+1aeilFJr6O1G7du2y1157ZeTIkenRo0eSuZcx8H0MAIC6lFM+7+qrr86JJ55YXP/JT36S8847r4QRlZ583lzyefJ5Xyef17jk86qTz2ta8nnyeVAOFLVRUhMmTEj//v3z+uuvp2PHjvnZz36Wv/3tb/nnP/+ZkSNH5rTTTkv79u3z73//OzvuuON8P+TmzJmT5557LsncHymzZs3Kueeem5VWWikrrLBCvvWtb6VTp07ZZJNNcuONN/piTLO16667FpcfffTRen2Zp+lUfX1uueWWfPzxx4vU39prr12cuTJ79uzcdttttbb/7LPP8pe//KXOfqsmZF5++eU6j6l33HFHnX3usssuxeVLL720LI/Tm222WZZddtkkc2de3nDDDY3S7/rrr19M4Pz73//OE088UUyGJcnhhx/eKOMk1WfU/fWvf63zPfbRRx/loYceWuD+TeX+++8vvncrKioybNiw+U5l3qJFiwwbNqw463jUqFEZNmxYk8f6TfHTn/60WiLl6quvztprr91k47dr1y4//vGPi+t333137r777gb3M3369PkSXY39Hu/QoUM22WST4np9Lo1y3XXXFZf79evXKMnFrbfeurj8/PPP580336y1/WOPPZYPPvhgkcddHFq0+H8/Pb/88ss6j/dXXHFFvfpt27Ztcfmrr76qs32pvhMtu+yy2XzzzYvrEydObJJxAQD45iqXfN5NN91U7RJ3hx9+eC655JKSxFJO5PPmks+Tz6tKPq/xyef9P/J5TU8+Tz4PyoGiNkrqqKOOysSJE9OzZ8+89NJLufjii7PjjjumT58+GThwYC644II89dRTWXrppfOf//wnF198cbX9x40bV7w2d7du3bLVVltl6NChmTBhQrFNoVDICy+8kEMOOST7779/vU6xDUuafv36ZeDAgUnm/k0cdNBBmTp1ar32nTVrVvESvywee++9d771rW8lSb744oscdNBB9foin8ydcTRvJtE8LVq0yKGHHlpcP/fcc2udHXPqqafW66yWa6+9dnF254QJE2qdifnggw/mwQcfrLPPn/3sZ8XZU88//3zOPffcOveZ56OPPqp320XRpk2b/OhHPyqun3LKKXnrrbcape8jjzyyuHzqqacWExTLLrts9tprr0YZI0m23377YhJz5syZOeGEE2psWygU8uMf/7g4A6t3797ZdtttGy2W+pg6dWq1hPmPfvSj9O/ff4Ft11133Zx22mnF9ZNPPrna9wDmuuOOO/KHP/yhuH7sscdm0KBBTR7HySefXG0W5cEHH5z777+/3vu/+uqr2XTTTec7/iyO9/jRRx9dXP7jH/+YV155pcY+X3zxxVx55ZXF9R/+8If1ejx1WWeddarNLj/hhBOKl5r4uhkzZuSkk05qlHEXh+WXXz6dOnVKMvefL/MuXbAgo0ePrncSbPnlly8u1+cSAI39nag+ZzqYp+rlF+qa2QoAAOWQz7vnnnsyZMiQ4u+QQYMG5eqrr27wGVaWRPJ58nmJfF5V8nmNTz7v/5HPKw35PPk8KAeK2iiZ1157LQ888ECS5A9/+ENWX331Bbbr06dPjj322CSpNuMlSf73v/8Vl2+88cb84x//yNprr50HHnggn3/+eT7//PPcf//9+fa3v51k7hfAM888c3E8HCh7l112WTp06JAkeeWVV9KvX7888sgjNbZ/5513cv7552e11VardkrncjJ06NBUVFSkoqIivXr1KnU4C61ly5a54oorismgv//979lqq62KZ6JckFdeeSWnnXZaVllllbz33nvzbf/pT39a/GEwYcKEbL/99nnnnXeqtZk1a1ZOPfXUXHnllWndunWdcbZq1Sr77rtvcf3II4/M66+/Xq1NoVDIjTfemP3222++WXgL0rt372rH5XPPPTdDhgypcWZSZWVlHnnkkRxyyCFNegr9k08+Ob17904y98fbFltskdtuu22BM5O++OKL3HrrrTnssMPq7Hf//fcv/igcM2ZM8f6DDjqoXs9ffbVo0SIXXnhhcf3WW2/NkUceOV/y8/PPP8/hhx9evLR3klx00UXVZmQ1hZNOOqn4Y3aVVVapFvuCnHHGGVlnnXWSJFOmTMlxxx1XY9uxY8cWjxsVFRV5/PHHGy3uxjRw4MBijEOGDFmkvt56660cccQRxfW+ffs26qUCGqJNmza58847061btyRzZ/jtscceOeSQQ/LGG28scJ9CoZDnnnsugwcPzkYbbZR//etf87VZHO/xAw88sHjJgVmzZmWHHXbIyJEj52v36KOPZscddyz+82LjjTdu1ATj+eefX1x++OGHc8ghh8yXtPn444+zxx575KWXXqrX8bwUWrRokZ133rm4fuihh+bZZ5+dr90dd9yRnXfeOZWVlfWaHbv++usXl++9995icrM2jfmd6LLLLstGG22Uyy+/vMYE/Oeff55TTjklzz//fJK5n/s77LBDnXECAEAp83kPPfRQBg0alMrKyiTJ7rvvnhtuuGGRcwTyefJ5TUU+Tz6vqcnn/T/yefJ5tZHPAxqiVakDoPmad2rs9u3bVztd9YJstdVW+fWvf53x48dn3Lhx6dmzZ5JUm800c+bM9OzZM0899VSWW2654v277rprNttss2y44YYZP358Lrnkkhx//PFZaaWVFsOjgvK13nrr5dZbb80PfvCDfPHFF3nrrbey3XbbpWfPnunbt2+6dOmSWbNmZdKkSXn55Zcb/XTHV155ZbVZL0nm+6K68847z/fl/Yc//GGjzZApZ9tuu22uuOKKHHPMMamsrMwzzzyTfv36ZY011kifPn2y7LLL5ssvv8xHH32Ul156qc5TcXft2jV/+tOfsu+++6aysjIvvvhi1lprrQwYMCCrr756pk6dmpEjR2bSpElZaqml8stf/jInn3xynXGeddZZuf322zN9+vSMGzcuG220UbU+n3766bz//vtp2bJlrrrqqmo/vGtyzjnnZOzYsbn++uuTJNdff31uuumm9OnTJ2uttVY6dOiQqVOn5r///W9efvnl4o/aqrN5FrdOnTrlnnvuyXbbbZePP/44n3zySQYNGpQTTjghm2++ebp27ZoZM2bk3XffzT//+c98+eWXxR/PtWnfvn0OPPDA+WYw1ed5a6j99tsvTzzxRP74xz8mSYYNG5bbb789W2+9dbp3756PP/44jz32WPEMqMncWWR77713o8dSm5EjR1a75MCVV15Z/LFak9atW2fYsGHZYostMmfOnNxzzz3585//nD333HNxh/uNsM8++1R7XVdYYYX89Kc/Xai+ll9++QbNwF6Q1VdfPf/4xz+y22675bXXXsucOXNy44035sYbb0yvXr2ywQYbpEuXLqmsrCwe875+avd5s8yrauz3eOvWrXPrrbdmwIABmTRpUj766KNss8022XDDDbPRRhslSV566aW8/PLLxX26deuWW2+9tXgJjcawzTbb5MQTTyxe3ufmm2/Offfdl2222SbdunXLhx9+mMceeywzZsxIr169svvuu+f//u//Gm38xnTWWWflL3/5S7788suMHTs2m222Wb773e9mzTXXzKxZszJmzJjiP3aOPPLIvP3227XOAE2SnXbaKe3bt88XX3yRl19+OWuvvXYGDhyYzp07F88esf3222f77bcv7tPY34lefvnlHHvssTnuuOPSu3fvrLfeeunSpUu++uqrjB8/Pk8//XS13y2nnnpq8fcMAADUplT5vE8++SR77713MXfXsmXLdO3atdaz6FT1k5/8JGussUajxFLO5PPk86qSz5PPa0zyefJ55UI+by75PCihAjSS9957r5CkkKQwePDgOtvvsccexfYNuT377LPFPp577rlq26688soax/vDH/5QbPf73/++MR4y1GnAgAHF990555zT4P2r/l2tuuqqNbZbddVVi+3ee++9Wvt86aWXCt/5znfq/TfXq1evwosvvtjg2L/unHPOWai/+dqet6p91vb8NMR11123yH025PX4uscee6ywxhpr1Pv5WXfddQsffvhhjf3dcssthQ4dOtS4f8eOHQt33313YeTIkcX7BgwYUGuMf/vb3wrt27evsc9OnToV7r777nq/f+e57LLLCssuu2y9HndFRUXh+9///gL7GTx4cLHdddddV+e4VV/zuj6/xo4dW9hqq63qFWP//v3rHLtQKBRefPHFavttsskm9dpvYZ133nmFNm3a1Bp727ZtCxdccEGdfTX0ua7LF198Uejdu3exz4MOOqhB+x933HHFfVdcccXClClT5mvzn//8p9pjfeKJJxY57gVZ1GNJ1ffZYYcdtkixLMyxt6ZbYx1rC4VC4fPPPy/84he/KHTu3Lne42+44YaFe+65p9Z+G/M9XigUCm+99VahT58+dca28cYbF955551a+6r6vWDkyJH1faoKc+bMKfzsZz8rVFRU1Dj+GmusUfjXv/5V7bOxps/Q+hz3GvK5ME/VeGpy77331voZkqRw1FFHFWbMmFHv5+vqq68utGjRosb+anoeGuM70cUXX1zv/Vu3bl0499xz6/VcAgBQOvJ51R/Dwtxq+/4unyeftyDyebWTz5PPW5SbfF7NN/k8+by6bvJ5UHrO1EbJ1DUrqSZffPFFcfnrVf077bRTjftVPSVobacAhyXdhhtumOeffz4jRozIvffem9GjR2f8+PGZMmVK2rRpk65du2bNNdfMZpttlh122CHf/e53izMjWPy23nrrvPnmm/nzn/+cBx98MM8880w++uijTJ06Ne3bt0/37t2z1lprZfPNN89OO+1UnF1Uk0GDBmWLLbbI73//+zz44IN5//3306pVq/Ts2TM777xzfvjDH2a11VZr0Cnbd9xxx7z55pu5+OKL8/DDD2fcuHFp2bJlVlllley222455phjssoqq2Ts2LENeuzHHXdcBg8enBtvvDF///vf8/LLL2fSpEmZMWNGOnbsmJVXXjnrrrtuBg4cmJ133rkks2JWXXXVjBo1Ko8++mjuvPPOPPnkk5kwYUKmTp2apZdeOquuumq+853vZJdddsn3v//9evW50UYbZfXVV89//vOfJItnVmdVZ555Zg4++OAMGzYsDz/8cN57771MmTIlnTt3zuqrr54ddtghRxxxRFZZZZXFGkdNsb377rtJ5s5OvvTSSxu0/69+9avcd999ef/99zN+/PicfPLJueqqq6q1efXVV4vLvXr1ymabbbbIcTe2QqFQ7ZT8jXnq+3LSoUOHnHXWWfnxj3+cBx98MH//+9/zwgsvZNKkSfnf//6X1q1bZ7nllstaa62VTTfdNHvssUe9LlPS2O/xNddcM88//3zuuuuu3H333Xn22WeL32O7deuWTTfdNPvss0/23nvvxfZ5WVFRkYsvvjj77rtvLr/88jz++OOZOHFiOnXqlN69e2e//fbL4YcfXrz8STnbfffd89prr+V3v/tdRowYUfxcWnHFFdO/f/8MGTIkW221VYP6PPLII7PeeuvlyiuvzDPPPJMPP/wwX3zxxQIvKVNVY3wn+tnPfpa99947f//73/P000/n1VdfzdixYzN16tS0aNEinTt3ztprr51tttkmhxxySFZdddUGP2cAACCfV97k8+Tz5PPk8+Tz5PO+Tj6vdvJ5QH1VFOo6MkA9jR07NquttlqSZPDgwRk+fHit7TfbbLP84x//yGqrrZb77ruv3uOsttpqxetxT58+PR07dix+wM2ePTstW7Zc4H4zZ85M27Ztk8wtcHvooYfqPSbAku7xxx/P1ltvnSQZMGBAg5JiLJr33nsvvXv3TqFQSPv27TNhwoRvxA/Zb6oTTzyxmFwbNmxYDj/88NIGtAAvv/xyMcG9xRZb5MknnyxtQAAAAEDZkc8rHfm8piWfBwDNlzO1UTLLL798kmTixIlZa6210qpVw9+O82bRzJs9NGfOnBqL2iorK4vLCzMWACwO11xzTbE4e99995UAW8wee+yxJHOL5AcPHlziaBZsXoxJcu6555YwEgAAAAC+Tj6vacnnAUDz1aLUAdB89enTJ8ncy4mOHj16ofupejrTeac4XpCq21ZaaaWFHg8AGsuXX36ZP/3pT8X1H/7whyWMZsk3efLk4uUKzjrrrLItch85cmSSud9xttlmmxJHAwAAAMA88nlNSz4PAJo3RW2UzO67715c/vWvf73Q/ey9997F5XvuuafGdlW3bbnllgs9HgA0ljPPPDMff/xxkuS73/1uNttssxJHtGR7/PHHUygU0rt37xx88MGlDmeBKisr88QTTyQxqxMAAACg3MjnNS35PABo3sqznJ1moW/fvtl+++0zYsSI/PWvf80555xT65e9sWPHZsyYMRk0aFC1+3fdddest956ee2113LRRRdl7733zre//e1qbd54441cfPHFSZKuXbtmr732avwHBAB1GDFiREaMGJEvv/wy//jHP/LCCy8Ut/3qV78qYWTNw9577128NES5atmyZaZMmVLqMAAAAACIfF6pyecBQPOmqI2F9tRTT+Wdd94prn/yySfF5XfeeSfDhw+v1n7IkCHz9XHddddlk002yYQJE/KLX/wiDz/8cA477LCsv/76adu2bSZPnpxXXnklDz30UB577LHsscce8xW1tWjRIldccUW+973vZerUqdl8881zyimnZMCAASkUChk1alQuuuiiTJs2LUly2WWXpX379o33RABAPT399NP57W9/O9/9P//5zzNgwIASRAQAAAAA1EQ+DwCgdBS1sdCGDRuW66+/foHbRo8endGjR1e7b0FFbSuuuGLGjBmTfffdN88991z+8Y9/5B//+EeNY3bq1GmB92+xxRa55ZZbcuihh+Z///tfTjnllPnaLLXUUvn973+fH/zgB7U8KgBoGu3bt8/666+fY445JoMHDy51OAAAAABALeTzAACaVkWh3M/ZStkaMmRIjUVtC1LbW61QKOQvf/lLbr/99vzjH//IxIkT89VXX6Vz585ZY4018t3vfjff//73s+WWW6aioqLGft5///1cdtllefDBBzNu3LjMmTMnPXv2zLbbbpsTTjgh3/rWtxr0GAEAAAAAAAAAgKalqA0AAAAAAAAAAICy0aLUAQAAAAAAAAAAAMA8itoAAAAAAAAAAAAoG4raAAAAAAAAAAAAKBuK2gAAAAAAAAAAACgbitoAAAAAAAAAAAAoG61KHQDfDDNmzMirr76aJOnatWtatfLWAQAAABrf7NmzM2nSpCTJ+uuvn7Zt25Y4Ivhmks8DAAAAmsLiyufJZFAvr776avr161fqMAAAAIBm5Nlnn03fvn1LHQZ8I8nnAQAAAE2tMfN5Lj8KAAAAAAAAAABA2XCmNuqla9euxeVnn302K6ywQgmjAQAAAJZUEyZMKJ5dqmo+AmgY+TwAAACgKSyufJ6iNuqlVav/91ZZYYUVsvLKK5cwGgAAAKA5qJqPABpGPg8AAABoao2Zz3P5UQAAAAAAAAAAAMqGojYAAAAAAAAAAADKhqI2AAAAAAAAAAAAyoaiNgAAAAAAAAAAAMqGojYAAAAAAAAAAADKhqI2AAAAAAAAAAAAyoaiNgAAAAAAAAAAAMqGojYAAAAAAAAAAADKhqI2AAAAAAAAAAAAyoaiNgAAAAAAAAAAAMpGq1IHAAAAADDPnDlzMm3atEydOjWzZs1KZWVlqUOigVq2bJn27dunc+fOadu2banDAQAAAGAxks/75ivXfJ6iNgAAAKAsfP755/nwww9TKBRKHQqLYPbs2Zk5c2Y+/fTTLLPMMllhhRVSUVFR6rAAAAAAaGTyeUuGcs3nKWoDAAAASm5BCbCKioq0bNmyhFGxMGbPnl1c/uyzz9K6det06dKlhBEBAAAA0Njk85Yc5ZrPU9QGAAAAlNScOXOqJcA6dOiQ5ZZbLu3bty+LGYE0TGVlZaZMmZKPP/44STJp0qR06tQprVu3LnFkAAAAADQG+bwlS7nm81qUdHQAAACg2Zs2bVq1BNjKK6+cpZdeWgLsG6ply5ZZfvnls/zyyxfvmzZtWgkjAgAAAKAxyectWco1n6eoDQAAACipqVOnFpeXW245ya8lRKdOnYrL06dPL2EkAAAAADQm+bwlU7nl8xS1AQAAACU1a9asJElFRUXat29f4mhoLG3atCkmNOe9xgAAAAB888nnLZnKLZ+nqA0AAAAoqcrKyiRzT3NvVueSo6KiIi1btkySzJkzp8TRAAAAANBY5POWTOWWz1PUBgAAAAAAAAAAQNlQ1AYAAAAAAAAAAEDZUNQGAAAAAAAAAABA2VDUBgAAAAAAAAAAQNlQ1AYAAAAAAAAAAEDZUNQGAAAAAAAAAABA2VDUBgAAANCMTJs2LauuumoqKirSpUuXTJo0qc59fv7zn6eioiIVFRW58sormyBKAAAAACBpvvm8VqUOAAAAAKAhhg4tdQSL1+J+fB06dMiVV16ZnXfeOZMnT85PfvKT3HLLLTW2f/7553PJJZckSbbaaqscffTRizdAAAAAAJY4S3JOTz5v8XCmNgAAAIBmZqeddsqBBx6YJLn11lvz4IMPLrDdV199lcMPPzyVlZVp27Zt/vSnP6WioqIpQwUAAACAZq855vMUtQEAAAA0Q5deemm6du2aJDnmmGPy+eefz9fmwgsvzCuvvJIkOeecc7Lmmms2aYwAAAAAwFzNLZ/XbIva/vnPf+aCCy7ITjvtlJ49e6ZNmzbp0KFD1lxzzQwZMiRPPvlko4952223ZYcddsgKK6yQtm3bplevXjn44IPzzDPP1LuPyZMn55xzzsmGG26YZZZZJp06dcqGG26Yc845J5MnT270mAEAAIAlU5cuXXLppZcmScaNG5dTTz212vY33ngj559/fpKkT58+Oemkk5o6RAAAAADg/9fc8nnNsqhtwIAB+c53vpMzzjgjDz30UD744IPMmjUr06dPz7///e9cf/312WqrrXLIIYdk1qxZizzejBkzsttuu2XQoEEZMWJEPvroo8ycOTP//e9/c9NNN6V///4577zz6uznueeey/rrr59f/OIXeeWVVzJ16tR8/vnneeWVV/KLX/wiG2ywQZ5//vlFjhcAAABoHg444IDsvPPOSZIrrrgiTz31VJJkzpw5OfzwwzNz5sy0atUq11xzTVq1alXKUAEAAACg2WtO+bxmWdT24YcfJklWXHHF/OQnP8ldd92VZ599NmPGjMnvfve7rLTSSkmSG2+8MUOGDFnk8Q4//PA88MADSZKtt9469957b5599tlcc8016d27d+bMmZOzzz47w4YNqzXm3XbbLRMmTEirVq1y8skn54knnsgTTzyRk08+Oa1atcr48eOz6667Fh8fAAAAQF2uvPLKdOzYMYVCIUceeWRmzpyZyy67LGPGjEmSnHTSSenTp0+JowQAAAAAkuaTz2uWRW1rrbVWbr/99rz//vu59NJLs/fee6dv377ZbLPNcuKJJ+all14qXlP21ltvXaRLkY4aNSq33HJLkmS33XbL3//+9+y+++7p27dvDjvssDzzzDNZZZVVkiQnn3xypkyZssB+zjjjjEycODFJcsstt+Siiy7KlltumS233DIXXXRRcYyJEyfmrLPOWuh4AQAAgOalZ8+e+dWvfpUkefPNN3PMMcfkjDPOSJKsueaaOeecc0oZHgAAAABQRXPJ5zXLorYHHngg++23X1q2bLnA7V26dMlvf/vb4vpdd9210GP9+te/TpK0bNkyl19++XxjdunSJRdddFGS5NNPP80111wzXx8TJ07MTTfdlCTZYYcdsu+++87XZt99980OO+yQJLnhhhuKBXAAAAAAdfnRj36U/v37J0muu+66TJ8+PRUVFfnTn/6Utm3bljg6AAAAAKCq5pDPa5ZFbfUxcODA4vK77767UH1MmzYtjz76aJJku+22y8orr7zAdnvttVc6deqUJLnnnnvm237fffelsrIySXLooYfWON68S6VWVlbmvvvuW6iYAQAAgOZnXsKroqKieN/RRx+drbbaqoRRAQAAAAAL0hzyea1KHUC5mjVrVnG5RYuFq/179tlnM3PmzCTJgAEDamzXunXrbLbZZhkxYkSeffbZfPXVV1lqqaWK26te/rS2fqpue+qpp3LkkUcuVNwAAAs0dGjzGBMAmqnRo0enUCgU11dcccUSRgMAAAAA1GZJz+c5U1sNRo0aVVxea621FqqPN954o959zNs+e/bs/Pvf/15gP8sss0x69OhRYx8rrLBC8YxvVccGAAAAqM2ECRPy85//vNp9559/ft5+++0SRQQAAAAA1KQ55PMUtS3AnDlzcuGFFxbX99tvv4XqZ9y4ccXlmi49Ok/Pnj0XuF/V9br6qNrP1/uoywcffFDrbcKECQ3qDwAAAPjm+NGPfpQpU6akoqIiv/nNb9KiRYvMnDkzRx11VLXZnkD5kM8DAACA5qs55PNcfnQBLrnkkjz77LNJkj333DObbLLJQvXz+eefF5c7dOhQa9ull166uDxt2rQF9lNXH1X7+XofdalaVAcAAAA0H3fddVfuvffeJMnRRx+dk046KePGjcvvf//7jBo1Ktdcc02OOOKI0gYJzEc+DwAAAJqn5pLPc6a2rxk1alROPfXUJEm3bt1yxRVXLHRfM2bMKC63bt261rZt2rQpLn/55ZcL7KeuPqr28/U+AAAAAL7u008/zfHHH58kWWmllXLRRRclmXupglVWWSVJcvLJJ2fixIklixEAAAAAmKs55fOcqa2Kf/3rX9lzzz0ze/bstGnTJnfccUe6d+++0P21bdu2uDxr1qxa286cObO43K5du/n6+eKLL+rso2o/X++jLnVdrnTChAnp169fg/oEAAAAyttPf/rTfPTRR0mSP/7xj+nUqVOSuWeLv/zyy7Prrrvm008/zY9//OPcfvvtpQwV+Br5PAAAAGh+mlM+T1Hb/++9997L9ttvn08//TQtW7bMrbfemgEDBixSnx07diwu13U50OnTpxeXv36Z0Y4dO+aLL76o1yVF5/VTn0uVVrXyyis3qD0AAADwzfbII49k+PDhSZJ99tknu+++e7Xtu+yyS/bff//cdtttueOOO3LwwQdn1113LUGkwILI5wEAAEDz0tzyeS4/mmT8+PHZdtttM378+FRUVOTaa6/Nnnvuucj9Vk0sffDBB7W2rTqzsmfPngvsp64+qvbz9T4AAAAA5pk+fXqOOuqoJMmyyy6byy67bIHt/u///i/LLbdckuRHP/pRvSbcAQAAAACNqznm85p9Udsnn3yS7bbbLv/5z3+SJJdddlkOOeSQRul7nXXWKS6/+eabtbadt71Vq1b51re+tcB+Pvvss+IpBBdkwoQJmTp1apJk7bXXXqiYAQAAgCXfmWeemffeey9JcvHFF6dHjx4LbNetW7dcfPHFSeZOpDvjjDOaLEYAAAAAYK7mmM9r1kVtn332WXbYYYe8/vrrSZILL7wwxx57bKP137dv37Ru3TpJMmrUqBrbzZo1K88888x8+8yzxRZbFJdr66fqtv79+y9UzAAAAMCS7dlnn83vf//7JMk222yTww47rNb2hx56aL73ve8lSf7whz/k2WefXewxAgAAAABzNdd8XqtSB1AqX3zxRXbZZZf885//TJKcccYZOeWUUxp1jI4dO+Z73/te/va3v+WRRx7JBx98UO2SpPPcc889xTOsLeiyp9///vdzzDHHZM6cObnuuuvygx/8YIHjzbtubosWLfL973+/8R4IAAAAlJGhQ0sdwTfXV199lcMPPzxz5sxJu3btcvXVV9drv6uuuirrr79+vvzyyxx55JF54YUX0qpVs00rAQAAANBAcnoLpznn85rlmdpmzZqVPffcM6NHj06S/OQnP8kvf/nLBvczfPjwVFRUpKKiIkNr+Os76aSTkiSzZ8/Osccem8rKymrbP/nkk2IxXefOnXPEEUfM10ePHj1y4IEHJkkefvjh3HXXXfO1ufPOO/Pwww8nSQ4++OAaTzMIAAAANF9LLbVUXn311RQKhXzxxRfp3bt3vfbr3bt3vvjiixQKhbz88svfuAQYAAAAAHwTNed83jcv4kYwaNCgjBgxIsnc0/Idfvjhee2112ps37p166y55poLNdY222yT/fffP7fddlvuu+++bLfddjnhhBOy4oor5tVXX83555+f999/P8ncy58uu+yyC+zn/PPPz0MPPZRJkyZl0KBBef7557PrrrsmSR544IH89re/TZJ07dp1oQr0AAAAAAAAAAAAykGzLGq75557isuPPfZYNthgg1rbr7rqqhk7duxCj3fttddm6tSp+etf/5qRI0dm5MiR1ba3aNEiZ511Vo4++uga++jZs2fuv//+7LHHHvnoo49y0UUX5aKLLqrWpkePHrn33nsXeIlTAAAAAAAAAACAb4JmefnRptauXbs8+OCDufnmm7PddtulW7duad26dXr27JkDDjggTz31VI2XL61q0003zauvvpozzzwz6623Xjp06JAOHTpk/fXXz5lnnpnXXnstm2666eJ/QAAAAAAAAAAAAItJszxTW6FQaJR+hgwZkiFDhtS7/QEHHJADDjhgkcbs0qVLzjvvvJx33nmL1A8AAAAAAAAAAEA5cqY2AAAAAAAAAAAAyoaiNgAAAAAAAAAAAMqGojYAAAAAAAAAAADKhqI2AAAAAAAAAAAAyoaiNgAAAAAAAAAAAMqGojYAAAAAAAAAAADKhqI2AAAAAAAAAAAAyoaiNgAAAAAAAAAAAMqGojYAAAAAAAAAAADKhqI2AAAAAAAAAAAAyoaiNgAAAAAAAAAAAMqGojYAAAAAAAAAAADKhqI2AAAAAAAAAAAAyoaiNgAAAAAAAAAAAMqGojYAAAAAAAAAAADKhqI2AAAAAAAAAAAAyoaiNgAAAIBmYpdddklFRUVatGiRp556ql77PPXUU2nRokUqKiqy6667LuYIAQAAAIB5mnM+r1WpAwAAAABoiKGPDy11CIvV0IFDF1vfV111VdZdd91MnTo1RxxxRF566aW0bdu2xvYzZ87MEUcckUKhkE6dOuXKK69cbLEBAAAAsORaknN68nmLhzO1AQAAADQTK6+8ci666KIkyVtvvZVzzz231va/+MUv8tZbbyVJfv3rX2fllVde7DECAAAAAHM153yeojYAAACAZuToo4/OgAEDkiQXX3xxXnzxxQW2e/nll/PrX/86STJw4MAcddRRTRYjAAAAADBXc83nKWoDAAAAaEYqKioybNiwtGvXLrNnz85hhx2W2bNnV2tTWVmZww8/PLNnz067du0ybNiwVFRUlChiAAAAAGi+mms+T1EbAAAAQDPzrW99K7/4xS+SJC+99FJ+85vfVNv+u9/9Li+88EKS5Lzzzkvv3r2bPEYAAAAAYK7mmM9T1AYAAADQDJ144onp27dvkuTcc8/NW2+9lSR59913c8455yRJ+vbtmxNOOKFUIQIAAAAA/7/mls9T1AYAAADQDLVs2TLXXHNNllpqqcycOTOHH3545syZkyOPPDJffvllllpqqVx77bVp2bJlqUMFAAAAgGavueXzFLUBAAAANFPrr79+TjvttCTJ6NGjs/3222fkyJFJktNPPz3rrbdeKcMDAAAAAKpoTvk8RW0AAAAAzdgZZ5yRddddN0ny6KOPJknWW2+9nH766aUMCwAAAABYgOaSz1PUBgAAANCMtW7dutplCeZdxqB169YljgwAAAAA+Lrmks9T1AYAAADQzPXr1y8rr7xykmTllVdOv379ShwRAAAAAFCT5pDPU9QGAAAAAAAAAABA2VDUBgAAAAAAAAAAQNlQ1AYAAAAAAAAAAEDZUNQGAAAAAAAAAABA2VDUBgAAAAAAAAAAQNlQ1AYAAAAAAAAAAEDZUNQGAAAAAAAAAABA2VDUBgAAAAAAAAAAQNloVeoAAAAAABpi6MChpQ4BAAAAAGgAOT0aSlEbAAAAABk7dmypQwAAAAAA6mlJz+e5/CgAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAQDMxfPjwVFRULPRt6NChpX4IAAAAANCsNNecnqI2AAAAAOqloqKi1CEAAAAAAA3wTc3ptSp1AAAAAAAN8g2dWVhvi/Hx7bHHHtlkk03q3f6ll17KIYcckkKhkI4dO2bQoEGLLTYAAAAAlmBLck5vMT+25prTU9QGAAAA0Ex07tw5nTt3rlfbzz77LHvssUcKhUKSZNiwYVlzzTUXY3QAAAAAwNc115yey48CAAAAMJ/DDjss7777bpLk2GOPzX777VfiiAAAAACA2ixJOT1FbQAAAABUc8kll+See+5JkmyyySb53e9+V+KIAAAAAIDaLGk5PUVtAAAAABSNGTMmp5xySpK5lza444470rp16xJHBQAAAADUZEnM6SlqAwAAACBJ8sknn2S//fbLV199lSQZPnx4VltttRJHBQAAAADUZEnN6SlqAwAAACCFQiEHHXRQPvjggyTJz372s+y+++4ljgoAAAAAqMmSnNNT1AYAAABAfvnLX+bhhx9Okmy++ea58MILSxwRAAAAAFCbJTmnp6gNAAAAoJl79NFHM3To0CRJly5dcvvtt6dVq1alDQoAAAAAqNGSntNT1AYAAADQjI0fPz4HHHBA5syZk4qKitx4441ZeeWVSx0WAAAAAFCD5pDTa7ZFbR9//HEeeOCBnH322dlpp53SpUuXVFRUpKKiIkOGDGmUMR5//PFin/W9DRw4cIF99erVq1779+rVq1FiBwAAAJZ8s2fPzv7775+PP/44SXL66adnxx13LHFUAAAAAEBNmktOb8k551wDde/evdQhLNC3v/3tUocAAAAANBNnnHFGnnzyySTJwIEDc+6555Y4IgAAAACgNs0lp9dsi9qq6tmzZ9Zee+2MGDGiUfvt27dvXn311TrbHXfccRk1alSSZPDgwbW23X333fPLX/6yxu2tW7duWJAAAABAs3T//ffnN7/5TZKkR48eufXWW9OyZcsSRwUAAAAA1KQ55fSabVHb2Wefnb59+6Zv377p3r17xo4dm9VWW61Rx1h66aWz3nrr1dpmypQpeeaZZ5Ik3/rWt7L55pvX2r5z58519gkAAABQm7Fjx2bw4MEpFApp2bJlbr311vTo0aPUYQEAAAAANWhuOb1mW9RWLqfeu/322zNz5swkycEHH1ziaAAAAIAl3axZs7Lvvvvm008/TTI3RzJw4MDSBgUAAAAA1Kg55vRalDqA5u6GG25IklRUVChqAwAAABa7E088Mc8//3ySZMcdd8zpp59e4ogAAAAAgNo0x5xesz1TWzl499138/TTTydJttxyy0a//CkAAABAVbfddlsuv/zyJEnHjh1zyimn5F//+le991966aXlLwAAAACgCTXXnJ6ithKad5a2JBk8eHC99nniiSeywQYb5N13302hUEj37t3Tr1+/DBo0KLvvvnsqKioWV7gAAADAN9xVV11VXP7888+z9dZbN2j/AQMG5PHHH2/kqAAAAACAmjTXnJ6ithK66aabkiTt2rXLPvvsU6993nvvvWrrY8eOzdixY3PHHXekf//+uf3227PSSis1OJYPPvig1u0TJkxocJ8AAABAeSkUCqUOAWgk8nkAAADQPDTXnJ6ithJ58skn85///CdJsueee6ZTp061tm/dunW+//3vZ/vtt896662XZZZZJlOmTMmYMWNyxRVXZNy4cRk9enS22267jBkzJssss0yD4unZs+dCPxYAAABoUkOHljqCb6xv4oxMYMHk8wAAAPhGkdNbaM01p6eorURuvPHG4vIhhxxSZ/tnn302nTt3nu/+gQMH5rjjjss+++yTESNG5I033si5556b3/3ud40ZLgAAAAAAAAAAQJNQ1FYCM2fOzJ133pkkWXHFFbPtttvWuc+CCtrm6dixY+6444707t07kydPztVXX50LL7wwrVu3rndM48aNq3X7hAkT0q9fv3r3BwAAAAAsPvJ5AAAAwJJMUVsJ/OUvf8mUKVOSJAceeGBatmy5yH0us8wy2X///fPHP/4x06dPz/PPP5/NN9+83vuvvPLKixwDAAAAANA05PMAAACAJVmLUgfQHN1www3F5fpcerS+1llnneLyhx9+2Gj9AgAAAAAAAAAANBVFbU3s448/zsMPP5wk2XjjjbPeeus1Wt+FQqHR+gIAAAAAAAAAACgFRW1N7JZbbsns2bOTNO5Z2pLk9ddfLy6vuOKKjdo3AAAAAAAAAABAU1DU1sTmXXq0VatWOeCAAxqt388++yy33357kqR9+/bZZJNNGq1vAAAAAAAAAACApqKobREMHz48FRUVqaioyNChQ+ts/69//SsvvvhikmSnnXZK165d6zXOQw89lC+//LLG7Z9//nn222+/TJ48OUly+OGHp02bNvXqGwAAAAAAAAAAoJy0KnUApfLUU0/lnXfeKa5/8sknxeV33nknw4cPr9Z+yJAhizzm9ddfX1wePHhwvfe78MILc+CBB2avvfbKFltskd69e6dDhw6ZMmVKxowZkyuuuCLjxo1Lknz729+uV4EdAAAAAAAAAABAOWq2RW3Dhg2rVmRW1ejRozN69Ohq9y1qUducOXNyyy23JEmWXXbZ7Lrrrg3a/3//+1+GDRuWYcOG1dhmq622yi233JLllltukWIFAAAAAAAAAAAolWZb1NbUHn300Xz44YdJkh/84AcNujzoxRdfnEcffTRjxozJW2+9lU8++SRTpkxJ+/bts+KKK2bTTTfNoEGDsv3226eiomJxPQQAAAAAAAAAAIDFrtkWtQ0fPny+S4w21JAhQ+p9BrftttsuhUJhocbZZJNNsskmmyzUvgAAAAAAAAAAAN8kLUodAAAAANC8tWzZMkkye/bsVFZWljgaGsucOXOKr+e81xgAAACAbz75vCVTueXzFLUBAAAAJdW+ffvi8pQpU0oXCI1q2rRpxbPWt2vXrsTRAAAAANBY5POWTOWWz1PUBgAAAJRU586di8sff/xxPv7448yYMaOYQOGbZc6cOZk6dWo++uij4n0dO3YsYUQAAAAANCb5vCVLuebzWpU6AAAAAKB5a9u2bZZZZpl89tlnSZLJkydn8uTJqaioKIvT3NMwlZWV1RKY7dq1y9JLL13CiAAAAABoTPJ5S5ZyzecpagMAAABKboUVVkjr1q0zadKk4n2FQiGzZ88uYVQsqnbt2mWVVVZJRUVFqUMBAAAAoBHJ5y2Zyimfp6gNAAAAKLmKiop06dIlnTp1yrRp0zJ9+vTMmjUrc+bMKXVoNFDLli3Trl27dOzYMUsvvXRZJMAAAAAAaFzyeUuOcs3nKWoDAAAAykbr1q2z3HLLZbnllit1KAAAAABAHeTzWFxalDoAAAAAAAAAAAAAmEdRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlQ1EbAAAAAAAAAAAAZUNRGwAAAAAAAAAAAGVDURsAAAAAAAAAAABlo1WpAwAA4Jvh8bGPN/mYA5t8RAAAAAAAAKDUnKkNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAstFsi9o+/vjjPPDAAzn77LOz0047pUuXLqmoqEhFRUWGDBnSaOMMHTq02G9dt8cff7zO/iZPnpxzzjknG264YZZZZpl06tQpG264Yc4555xMnjy50eIGAAAAAAAAAAAohValDqBUunfvXuoQGuy5557L7rvvngkTJlS7/5VXXskrr7ySYcOG5S9/+Us22WSTEkUIAAAAAAAAAACwaJptUVtVPXv2zNprr50RI0Ys1nFeffXVWrevttpqNW778MMPs9tuu2XixIlp1apVfvrTn2bXXXdNkjzwwAP53e9+l/Hjx2fXXXfNCy+8kJVWWqlRYwcAAAAAAAAAAGgKzbao7eyzz07fvn3Tt2/fdO/ePWPHjq21qKwxrLfeegu97xlnnJGJEycmSW655Zbsu+++xW1bbrllNtlkk+y3336ZOHFizjrrrFx77bWLHC8AAAAAAAAAAEBTa1HqAErl3HPPza677vqNuAzpxIkTc9NNNyVJdthhh2oFbfPsu+++2WGHHZIkN9xwQ7EADgAAAAAAAAAA4Juk2Ra1fZPcd999qaysTJIceuihNbYbMmRIkqSysjL33XdfU4QGAAAAAAAAAADQqBS1fQM8+eSTxeUBAwbU2K7qtqeeemqxxgQAAAAAAAAAALA4KGprQtttt12WX375tG7dOt26dcvAgQNz4YUX5tNPP611vzfeeCNJsswyy6RHjx41tlthhRXSqVOnavsAAAAAAAAAAAB8k7QqdQDNySOPPFJcnjRpUkaNGpVRo0bloosuyvDhw7P77rsvcL9x48YlSVZeeeU6x+jZs2f+9a9/Ffeprw8++KDW7RMmTGhQfwAAAADA4iOfBwAAACzJFLU1gfXXXz977LFH+vXrlxVXXDFfffVV3nrrrdx8880ZMWJEpkyZkr333jv3339/dtppp/n2//zzz5MkHTp0qHOspZdeOkkybdq0BsXYs2fPBrUHAAAAAEpHPg8AAABYkilqW8xOOOGEDB06dL77N9100xxyyCG56qqr8sMf/jCVlZU54ogj8s4776Rdu3bV2s6YMSNJ0rp16zrHa9OmTZLkyy+/XPTgAQAAAAAAAAAAmpiitsWsc+fOtW4/+uij8/zzz2fYsGEZP3587rnnnhx44IHV2rRt2zZffPFFZs2aVed4M2fOTJL5CuPqUtflSidMmJB+/fo1qE8AAAAAYPGQzwMAAACWZIraysDRRx+dYcOGJUlGjRo1X1Fbx44d88UXX9TrkqLTp09PUr9LlVa18sorN6g9AAAAAFA68nkAAADAkqxFqQMgWWeddYrLH3744Xzb5yWoPvjggzr7mjdDs2fPno0UHQAAAAAAAAAAQNNR1FYGCoVCrdvnFb199tln+eijj2psN2HChEydOjVJsvbaazdegAAAAAAAAAAAAE1EUVsZeP3114vLK6644nzbt9hii+LyqFGjauyn6rb+/fs3UnQAAAAAAAAAAABNR1FbGbjqqquKywMGDJhv+/e///20aDH3pbruuutq7Gf48OFJkhYtWuT73/9+4wYJAAAAAAAAAADQBBS1LYLhw4enoqIiFRUVGTp06HzbX3311bzzzju19nHVVVflmmuuSZL06NEje+6553xtevTokQMPPDBJ8vDDD+euu+6ar82dd96Zhx9+OEly8MEHp0ePHg19OAAAAAAAAAAAACXXqtQBlMpTTz1VreDsk08+KS6/8847xbOezTNkyJAGj/HCCy/kiCOOyNZbb52ddtop66+/fpZffvnMnj07b775Zm666ab8/e9/T5K0bNkyV111VZZeeukF9nX++efnoYceyqRJkzJo0KA8//zz2XXXXZMkDzzwQH77298mSbp27Zpf/vKXDY4VAAAAAAAAAACgHDTborZhw4bl+uuvX+C20aNHZ/To0dXuW5iitiSprKzMI488kkceeaTGNssvv3yuueaaWi8Z2rNnz9x///3ZY4898tFHH+Wiiy7KRRddVK1Njx49cu+992bllVdeqFgBAAAAAAAAAABKrdkWtTWFnXfeOddcc03GjBmTF198MRMnTszkyZNTKBSy3HLLZcMNN8yOO+6YIUOGpFOnTnX2t+mmm+bVV1/N//3f/+Xee+/N2LFjkySrrbZadt9995xwwglZfvnlF/OjAgAAAAAAAAAAWHwqCoVCodRBUP4++OCD9OzZM0kybtw4Z4MDgGbo8SEDm3zMgcMfb/IxAYDSkoOAxuFvCQAAAGgKiysH0aJRegEAAAAAAAAAAIBGoKgNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICyoagNAAAAAAAAAACAsqGoDQAAAAAAAAAAgLKhqA0AAAAAAAAAAICy0WyL2j7++OM88MADOfvss7PTTjulS5cuqaioSEVFRYYMGdJo40ydOjW33XZbjjzyyGy88cbp3LlzWrduna5du2bgwIG5+OKLM2XKlDr76dWrVzG+2m69evVqtNgBAAAAAAAAAACaWqtSB1Aq3bt3X+xj/O1vf8uee+6ZmTNnzrftk08+yahRozJq1KhcfPHFufXWW7P11lsv9pgAAAAAAAAAAADKWbMtaquqZ8+eWXvttTNixIhG7Xfy5MmZOXNmWrRoke222y477rhjNtxww3Tu3DkffPBBbr755tx+++2ZOHFidt1114wePTobbbRRrX3uvvvu+eUvf1nj9tatWzfqYwAAAAAAAAAAAGhKzbao7eyzz07fvn3Tt2/fdO/ePWPHjs1qq63WqGMstdRSOfroo3P66adnlVVWqbatT58+2W233dK/f//8+Mc/zhdffJGf/exnefTRR2vts3PnzllvvfUaNU4AAAAAAAAAAIBy0WyL2s4999zFPsYPfvCD/OAHP6i1zfHHH58bbrghzz//fB5//PFMnjw5yy+//GKPDQAAAAAAAAAAoBy1KHUAJAMHDkySzJkzJ++9915pgwEAAAAAAAAAACghRW1lYObMmcXlFi28JAAAAAAAAAAAQPPVbC8/Wk5GjRqVJGnVqlW+9a1v1dr2iSeeyAYbbJB33303hUIh3bt3T79+/TJo0KDsvvvuqaioWKgYPvjgg1q3T5gwYaH6BQAAAAAan3weAAAAsCRT1FZiDz74YF555ZUkyQ477JBOnTrV2v7rlycdO3Zsxo4dmzvuuCP9+/fP7bffnpVWWqnBcfTs2bPB+wAAAAAApSGfBwAAACzJFLWV0P/+978ce+yxSZKWLVvmvPPOq7Ft69at8/3vfz/bb7991ltvvSyzzDKZMmVKxowZkyuuuCLjxo3L6NGjs91222XMmDFZZpllmuphAAAAAAAAAAAANBpFbSVSWVmZAw88MP/973+TJGeeeWb69OlTY/tnn302nTt3nu/+gQMH5rjjjss+++yTESNG5I033si5556b3/3udw2KZ9y4cbVunzBhQvr169egPgEAAACAxUM+DwAAAFiSKWorkR/96Ed56KGHkiS77LJLzjrrrFrbL6igbZ6OHTvmjjvuSO/evTN58uRcffXVufDCC9O6det6x7PyyivXuy0AAAAAUFryeQAAAMCSrEWpA2iOTjvttFx99dVJki222CJ33nlnWrZsuUh9LrPMMtl///2TJNOnT8/zzz+/yHECAAAAAAAAAAA0NUVtTeyiiy7KhRdemCTZeOON88ADD6Rdu3aN0vc666xTXP7www8bpU8AAAAAAAAAAICm1ORFbTvuuGPuvPPOfPXVV009dMldfvnlOfXUU5Mka6+9dh5++OEss8wyjdZ/oVBotL4AAAAAgNo151wnAAAAwOLU5EVtI0aMyP77758VVlghJ5xwQl5++eWmDqEkbrzxxhx33HFJktVXXz2PPPJIunTp0qhjvP7668XlFVdcsVH7BgAAAACqa665TgAAAIDFrcmL2rp165ZCoZD//e9/ueyyy7LxxhvnO9/5Ti6//PJMmTKlqcNpEvfcc08OPfTQFAqFrLzyynn00Ucbvejss88+y+23354kad++fTbZZJNG7R8AAAAAqK455joBAAAAmkKTF7V9+OGH+ctf/pI99tgjrVq1SqFQyIsvvpjjjz8+K664Yg488MA88sgjTR3WQhk+fHgqKipSUVGRoUOHLrDNiBEjMmjQoFRWVqZbt2555JFH0qtXrwaN89BDD+XLL7+scfvnn3+e/fbbL5MnT06SHH744WnTpk2DxgAAAAAAGmZJynUCAAAAlJNWTT1gy5Yts9tuu2W33XbLpEmTcuONN2b48OF57bXXMmPGjNx222257bbbssoqq+TQQw/NkCFDssoqqzR6HE899VTeeeed4vonn3xSXH7nnXcyfPjwau2HDBnS4DGeeeaZ7Lnnnpk1a1aWWmqpXHLJJfnqq6/y2muv1bjPyiuvnM6dO1e778ILL8yBBx6YvfbaK1tssUV69+6dDh06ZMqUKRkzZkyuuOKKjBs3Lkny7W9/u8YCOwAAAACg8ZRLrhMAAABgSVNRKBQKpQ4iSV544YVcc801uf322/Ppp58mSfEsaFtvvXWOOOKI7LnnnmndunWjjDdkyJBcf/319W6/oKdp+PDhOfTQQ5Mk55xzznzFZEOHDs25557boLiuu+66+QroBg4cmFGjRtW571ZbbZVbbrklK620UoPGrI8PPvggPXv2TJKMGzcuK6+8cqOPAQCUt8eHDGzyMQcOf7zJxwQASksOgiVBU+c6F8TfEgAAANAUFlcOoskvP1qT73znO7n88sszfvz43HLLLdl2221TUVGROXPm5LHHHssBBxyQFVZYIccff3xefPHFUofbpC6++OJceOGF2X333bPWWmulS5cuadWqVTp16pS11lorgwcPzkMPPZTHH398sRS0AQAAAAD1J9cJAAAAsGjK5kxtC/L444/ngAMOyMSJE5PMPVtaRUVFkmSTTTbJaaedlj322KOEETYfZnYCAM7UBgA0BTkIllRNnev0twQAAAA0hSX+TG3zfPHFF7nhhhuy9dZb53vf+14mTpyYQqGQQqGQddZZJ+3atUuhUMhzzz2XvffeO7vvvntmzJhR6rAB/j/27j7O6rLOH//rwIjcg4oKCnkTqfCzdU3EewELzRVFXcWMUsyb7tw0cS3d1LEstVXUeqx3oRCVd7lGCiWurpC0mOi2ogauloQgohgqMogC5/eH35kFmYEZmDnnwDyfj8c8Hp+Zz/tzXe+xizMz797nugAAAADWotYJAAAAsHEqpqnt97//fc4666z06tUrZ5xxRqZNm5ZisZiuXbvmK1/5Sp566qk899xzee2113LrrbemX79+KRaLmTRpUq6++upypw8AAAAAkEStEwAAAGBTlbWpbcGCBbnqqquy55575vDDD8+4ceOydOnSFIvFHHLIIRk/fnxeffXV3HTTTfnUpz6VJOncuXPOPvvsPPvssznllFNSLBZz5513lvPbAAAAAABaObVOAAAAgOZTVeoJ33///UycODHjxo3LI488ktWrV6dYLCZJtt9++5x22mk566yzsueee653nDZt2uSb3/xm7rnnnvz1r38tReoAAAAAAHXUOgEAAABaRsmb2nr16pW33norSVIsFtOmTZsMHTo0Z511VoYPH56tttqq0WNtt912SZKVK1e2RKoAAAAAAA1S6wQAAABoGSVvaluyZEmSpE+fPjnjjDPypS99KR/72Mc2aqxtt902l19+eXOmBwAAAADQKGqdAAAAAC2j5E1tJ5xwQs4666x89rOfTaFQ2KSxttlmG4UeAAAAAKAs1DoBAAAAWkbJm9r+/d//vdRTAgAAAAA0O7VOAAAAgJbRptwJAAAAAAAAAAAAQK2S79S2dOnSXH/99UmSc845Jz179lxv/MKFC/OTn/wkSfLP//zP6dChQ4vnCAAAAACwIWqdAAAAAC2j5Du1TZw4MdXV1fnFL36xwSJPkvTs2TO/+MUvcsUVV+TBBx8sQYYAAAAAABum1gkAAADQMkre1Hb//fenUChkxIgRjYovFAr53Oc+l2KxmF/+8pctnB0AAAAAQOOodQIAAAC0jJIfPzpnzpwkycEHH9zoZw466KAkyZ/+9KcWyQkAgM1QdXXrmBMAgIql1gkAAADQMkq+U9v8+fOTJL169Wr0M7Vb9y9YsKBFcgIAAAAAaCq1TgAAAICWUfKmtjZtPpyypqam0c/Uxq5cubJFcgIAAAAAaCq1TgAAAICWUfLjR3v16pUXX3wxTz31VKO35X/qqaeS/N+7GAEAWo1SH3fpeE0AAGg0tU4AAACAllHyndoOO+ywFIvF3HTTTfnggw82GP/BBx/kpptuSqFQyKGHHlqCDAEAAAAANkytEwAAAKBllLyp7YwzzkiSvPjii/n85z+/3q35a2pqcuqpp+Z///d/13oWAAAAAKDc1DoBAAAAWkbJjx89+OCD87nPfS5333137r///vzhD3/I2WefncMPPzy9evVKoVDIq6++mt/97ncZO3Zs5s+fn0KhkJNOOimDBg0qdboAAAAAAPVS6wQAAABoGSVvakuSO+64I4sXL84jjzySBQsWpLq6ut64YrGYJBk6dGh++tOfljBDAAAAAIANU+sEAAAAaH4lP340Sdq3b58pU6bk+uuvz0477ZRisVjvR58+ffKjH/0oDz30UNq3b1+OVAEAAAAAGqTWCQAAAND8yrJTW5IUCoWcd955+cY3vpH/+Z//yR//+McsXrw4SdKjR4986lOfyj777JNCoVCuFAEAAAAANkitEwAAAKB5la2prVahUMi+++6bfffdt9ypAAAAAABsNLVOAAAAgOZR9qY2AAAaNnXu1JLON7ikswEAAAAAAACsq025EwAAAAAAAAAAAIBaZd2p7Zlnnsnjjz+ev/zlL1m6dGlWrVq13vhCoZDbb7+9RNkBAAAAADSOWicAAABA8ylLU9sLL7yQL33pS3niiSca/UyxWFToAQAAAAAqilonAAAAQPMreVPbggULcvjhh2fx4sUpFotJks6dO2ebbbZJmzZOQwUAAAAANg9qnQAAAAAto+RNbd///vfzxhtvpFAo5KyzzsqFF16YPfbYo9RpAAAAAABsErVOAAAAgJZR8qa2hx56KIVCIaeddlpuu+22Uk8PAAAAANAs1DoBAAAAWkbJ98B/9dVXkySnnXZaqacGAAAAAGg2ap0AAAAALaPkTW3bbLNNkqR79+6lnhoAAAAAoNmodQIAAAC0jJI3tQ0YMCBJ8r//+7+lnhoAAAAAoNmodQIAAAC0jJI3tX3jG99IsVjMbbfdVuqpAQAAAACajVonAAAAQMsoeVPb0KFDc9FFF+Wxxx7LV7/61XzwwQelTgEAAAAAYJOpdQIAAAC0jKpSTzhhwoT0798/Bx98cG677bY8+OCDOemkk7LXXnulY8eOG3z+tNNOK0GWAAAAAADrp9YJAAAA0DJK3tQ2atSoFAqFus8XLlyYH//4x416tlAoKPQAAAAAABVBrRMAAACgZZS8qS1JisViOaYFAGALMnXu1JLPObjkMwIAUOnUOgEAAACaX8mb2l5++eVSTwkAAAAA0OzUOgEAAABaRsmb2nbZZZdSTwkAAAAA0OzUOgEAAABaRptyJwAAAAAAAAAAAAC1NLUBAAAAAAAAAABQMUp+/OiaXnrppUyYMCEzZszIa6+9luXLl+ehhx5K375962Kee+65zJs3L506dcqgQYPKmC0AAAAAQP3UOgEAAACaT1ma2lavXp1vfetbueGGG7J69eoUi8UkSaFQyPvvv79W7CuvvJJhw4alqqoqL7/8cnbeeedypAwAAAAAsA61TgAAAIDmV5bjR7/85S9nzJgxWbVqVXbaaaecdNJJDcYeffTR2X333bNq1arcd999JcwSAAAAAGD91DoBAAAAml/Jm9qmTp2a22+/PUlyySWXZO7cubn33nvX+8zJJ5+cYrGYxx57rBQpAgAAAABskFonAAAAQMso+fGjt9xyS5LkH/7hH3LllVc26pmBAwcmSZ5//vkWywsAAAAAoCnUOgEAAABaRsl3apsxY0YKhULOPPPMRj/Tu3fvJMlrr73WUmkBAAAAADSJWicAAABAyyh5U9vrr7+eJNltt90a/UxV1Ycbyn3wwQctkhMAAAAAQFOpdQIAAAC0jJI3tXXo0CFJUlNT0+hn5s2blyTZZpttWiQnAAAAAICmUusEAAAAaBklb2qrfdfiH//4x0Y/M2nSpCRJ//79WyQnAAAAAICmUusEAAAAaBklb2o78sgjUywWc9ttt2X16tUbjH/66afzs5/9LIVCIZ/97GdLkCEAAAAAwIapdQIAAAC0jJI3tZ177rnp0KFDnn322Zx99tn54IMPGoz993//93z2s5/N+++/n65du+acc84pYaYAAAAAAA1T6wQAAABoGVWlnnDnnXfOj370o5x99tkZP358Hn744Rx77LF192+//fbU1NTkkUceyV/+8pcUi8UUCoXcdttt6datW6nTBQAAAACol1onAAAAQMsoeVNbkpx55pkpFAr5xje+kQULFuTWW29NoVBIktxwww1JkmKxmCTZeuutc8stt+Tkk08uR6oAAAAAAA1S6wQAAABofiU/frTWl770pcyZMycXXHBBPv7xj6dYLK71sfPOO+erX/1qZs+endNPP71caQIAAAAArJdaJwAAAEDzKstObbV69+6da6+9Ntdee23eeeedvP7661m1alW222679OjRo5ypAQAAAAA0mlonAAAAQPMpa1Pbmrp27ZquXbuWOw0AAAAAgE2i1gkAAACwacp2/CgAAAAAAAAAAAB8lKY2AAAAAAAAAAAAKkbJjx894ogjNvrZQqGQRx99tBmzAQAAAADYOGqdAAAbobq6ZWIBgC1KyZvapk6dmkKhkGKx2GBMoVBY6/Pa2I9+HQAAAACgXNQ6AQAAAFpGyZvaDj/88A0WbJYtW5YXX3wxb7/9dgqFQvbYY4/06tWrRBkCAAAAAGyYWicAAABAyyjLTm2NUSwWM3ny5Jx33nn529/+lrFjx+bQQw9t2eQAAAAAABpJrRMAAACgZbQpdwINKRQKGTZsWKZPn562bdvmhBNOyIIFC8qdFgAAAABAk6h1AgAAADRNyXdqa6pevXrlggsuyLe+9a388Ic/zI033ljulAAAAAAAmkytEwDYolRXt0wsAEAqeKe2NdVuxT958uQyZwIAAAAAsPHUOgEAAAA2bLNoamvXrl2S5NVXXy1zJgAAAAAAG0+tEwAAAGDDNoumtunTpydJOnbsWOZMAAAAAAA2nlonAAAAwIZVfFPbjBkz8t3vfjeFQiEDBw5stnFff/31TJo0KZdddlmOPvro9OjRI4VCIYVCIaNGjWq2edZ0991356ijjkqvXr3Svn377LrrrvniF7+YJ554otFjvPnmm7n88suzzz77pFu3bunatWv22WefXH755XnzzTdbJG8AAAAAYNO1VK0TAAAAYEtTVeoJv/vd724wZvXq1VmyZEmeeuqp/OEPf8jq1atTKBTyzW9+s9ny2HHHHZttrA157733cvLJJ2fSpElrff2vf/1r/vrXv+bOO+9MdXV1Lr300vWOM3PmzAwfPjwLFy5c6+uzZs3KrFmzMnbs2Pz617/OgAEDmv17AAAAAADWVim1TgAAAIAtTcmb2qqrq1MoFBodXywWU1VVlR/+8IcZOnRoi+TUp0+f9OvXLw8//HCLjH/mmWfWNbQNGTIk5513Xnbaaac8++yz+cEPfpA///nPueyyy9KrV6+cddZZ9Y6xYMGCHHvssVm0aFGqqqpywQUXZNiwYUmSSZMmZcyYMXn11VczbNiwPP3009l5551b5HsBAAAAAD5UibVOAAAAgC1ByZvakg+LN+tTKBTSpUuX7Lbbbhk0aFDOOeec9O/fv1lzuOyyy7L//vtn//33z4477pi5c+dmt912a9Y5kmTatGm58847kyTHHntsfvWrX6Vt27ZJkv333z/HHXdc9ttvv8ybNy8XXXRRTjrppHTv3n2dcf7lX/4lixYtSpLceeedOfnkk+vuHXbYYRkwYEBGjBiRRYsW5dJLL80dd9zR7N8LAAAAALC2Sqh1AgAAAGxpSt7Utnr16lJPWa8rrriiJPP88Ic/TJK0bds2N910U11DW60ePXrkmmuuyamnnpolS5bk9ttvz+jRo9eKWbRoUX7+858nSY466qi1GtpqnXzyyTnqqKMyZcqUTJgwIVdddVVJj1gFAAAAgNamUmqdAAAAAFuaNuVOYEv27rvv5tFHH02SDB06NL1796437sQTT0zXrl2TJPfff/869x944IGsWrUqSXLGGWc0ON+oUaOSJKtWrcoDDzywKakDAAAAAAAAAACUhaa2FvTkk09mxYoVSZJBgwY1GNeuXbsceOCBdc988MEHa91//PHH667XN86a96ZPn75ROQMAAAAAAAAAAJSTprYWNHv27Lrrvfbaa72xtfdXrlyZF198sd5xunXrlp49ezY4Rq9evep2fFtzbgAAAAAAAAAAgM1FVaknnDdvXouM+7GPfaxFxt0Ur7zySt11Q0eP1urTp89az/Xv33+dcTY0Ru04zz///FpzN8b8+fPXe3/hwoVNGg8AAAAAtnTlrHWq5wEAAABbspI3te22227NPmahUMjKlSubfdxNtXTp0rrrzp07rze2U6dOddfvvvtuveNsaIw1x/noGBuyZlMdAAAAALBh5ax1qucBAAAAW7KSN7UVi8VST1k27733Xt11u3bt1hu79dZb110vX7683nE2NMaa43x0DAAAAACgebWmWicAAABAKZW8qW3cuHFJkptuuikzZ87MVlttlSOPPDIDBw7MjjvumGKxmNdffz0zZ87Mww8/nA8++CD7779/vvrVr5Y61U3Wvn37uuv3339/vbErVqyou+7QocM649TU1GxwjDXH+egYG7Kh40oXLlyYgQMHNmlMAAAAANiSlbPWqZ4HAAAAbMlK3tR2+umn56yzzspTTz2VI488Mrfffnt23nnnemMXLFiQs88+O1OmTMknP/nJ/OQnPylxtpumS5cuddcbOg502bJlddcfPWa0S5cuqampadSRorXjNOao0jX17t27SfEAAAAA0NqVs9apngcAAABsydqUesL77rsvd9xxRwYMGJDJkyc3WORJkp133jkPPvhg9ttvv9xxxx259957S5jppluzsDR//vz1xq75zso+ffrUO86GxlhznI+OAQAAAAA0r9ZU6wQAAAAopZI3td16660pFAq54IIL0rZt2w3Gt23bNqNHj06xWMxtt91WggybT//+/euu58yZs97Y2vtVVVXp27dvveO8/fbbee211xocY+HChXnnnXeSJP369duonAEAAACAxmlNtU4AAACAUip5U9usWbOSJHvssUejn6mNffbZZ1skp5ay//77p127dkmSadOmNRj3/vvv54knnljnmVqHHnpo3fX6xlnz3iGHHLJROQMAAAAAjdOaap0AAAAApVTypralS5cmSV5//fVGP1MbW/vs5qJLly759Kc/nSR55JFHGjw+9P7776/bYe2EE05Y5/5xxx2XNm0+/J9q3LhxDc43fvz4JEmbNm1y3HHHbUrqAAAAAMAGtKZaJwAAAEAplbypbZdddkmSTJgwodHP1MZ+7GMfa5GcNtb48eNTKBRSKBRSXV1db8yFF16YJFm5cmW+/vWvZ9WqVWvdX7x4cb71rW8lSbp3756zzjprnTF69uyZkSNHJkmmTJmS++67b52YX/7yl5kyZUqS5Itf/GJ69uy50d8XAAAAALBhW1KtEwAAAKCSVJV6wuHDh+eHP/xh7r777uyzzz656KKL1ht/7bXX5q677kqhUKh3F7ONNX369Lz00kt1ny9evLju+qWXXqrb9azWqFGjNmqeI444Ip/73Ody991354EHHsjQoUNz/vnnZ6eddsqzzz6b73//+5k3b16S5Oqrr84222xT7zjf//7389BDD+WNN97IqaeemqeeeirDhg1LkkyaNCnXXXddkmT77bfPlVdeuVG5AgAAAACNVym1TgAAAIAtTcmb2r797W9nwoQJWbRoUS6++OLcddddOf3007P//vtnhx12SKFQyKJFizJz5sz87Gc/y//8z/8k+XC3stodzZrD2LFj89Of/rTee7///e/z+9//fq2vbWxTW5Lccccdeeedd/Kb3/wmjz32WB577LG17rdp0yaXXnppvvzlLzc4Rp8+ffLggw/m+OOPz2uvvZZrrrkm11xzzVoxPXv2zMSJE9O7d++NzhUAAAAAaJxKqXUCAAAAbGlK3tTWvXv3PPLIIznqqKOyYMGCzJo1K6NHj24wvlgspnfv3nnooYfSvXv30iXajDp06JDJkyfnzjvvzPjx4/PMM8/krbfeyo477pjDDjss5557bg466KANjnPAAQfk2WefzY033piJEydm7ty5SZLddtstw4cPz/nnn5/tttuuhb8bAAAAACBpnbVOAAAAgFJoU45J+/fvn+effz7f/OY307179xSLxXo/unfvngsuuCDPPfdc+vfv36w5jB8/vsF56/uoz6hRo+ruV1dXb3DOz3/+83n44YezaNGirFixIvPmzcsvfvGLRjW01erRo0e+973v5dlnn83SpUuzdOnSzJo1K9/73vc0tAEAAABAiVVCrRMAAABgS1Pyndpqde3aNdddd12uuuqqPP3003n22WezZMmSFIvFbLvttvnkJz+Z/fbbL+3atStXigAAAAAAG6TWCQAAANC8ytbUVqtdu3Y56KCDmrRbGQAAAABApVHrBAAAAGgeZTl+FAAAAAAAAAAAAOpT9p3a/vKXv2TGjBl57bXXUlNTk69+9avp0aNHudMCAAAAAGgStU4AAACA5lG2prY//vGPOf/88zN9+vS1vv6P//iPaxV6/u3f/i1XXHFFunXrlj/96U/ZaqutSp0qAAAAAECD1DoBAAAAmldZjh+dPHlyDj744EyfPj3FYrHuoz6nn356li9fnr/85S+ZNGlSiTMFAAAAAGiYWicAAABA8yt5U9trr72WU089NStWrEj//v3z29/+NkuXLm0wvnPnzjn++OOTJL/97W9LlCUAAAAAwPqpdQIAAAC0jJI3tV1//fV59913s8suu+Txxx/PUUcdlU6dOq33mcGDB6dYLObpp58uUZYAAAAAAOun1gkAAADQMkre1DZlypQUCoWMHj063bt3b9Qze+65Z5Jk7ty5LZcYAAAAAEATqHUCAAAAtIySN7W9/PLLSZKBAwc2+pkuXbokSd59990WyQkAAAAAoKnUOgEAAABaRsmb2j744IMkyVZbbdXoZ956660k2eDW/QAAAAAApaLWCQAAANAySt7U1rNnzyT/9y7GxpgxY0aSpHfv3i2SEwAAAABAU6l1AgAAALSMkje1HXLIIUmSX/3qV42Kr6mpyS233JJCoZDDDz+8JVMDAAAAAGg0tU4AAACAllHyprbTTz89xWIxd911Vx5++OH1xr777rsZMWJE5s2blyQ588wzS5EiAAAAAMAGqXUCAAAAtIySN7V95jOfyfHHH5/Vq1fnuOOOyz//8z/nySefrLv/t7/9LX/4wx/yve99L3vuuWd++9vfplAo5LTTTsu+++5b6nQBAAAAAOql1gkAAADQMqrKMenPf/7zDBs2LFOnTs2YMWMyZsyYFAqFJMmgQYPq4orFYpLk05/+dG655ZZypAoAAAAA0CC1TgAAAIDmV/Kd2pKkY8eOeeSRR/Kv//qv6dmzZ4rFYr0f2267bX7wgx9kypQp2XrrrcuRKgAAAABAg9Q6AQAAAJpfWXZqS5I2bdpk9OjROe+88/Lkk0/mqaeeyuuvv55Vq1Zlu+22y7777ptDDz1UgQcAAAAAqGhqnQAAAADNq+RNbRMmTEiS7LnnnjnggANSVVWVgw8+OAcffHCpUwEAAAAA2GhqnQAAAAAto+THj44aNSpnnHFG/vrXv5Z6agAAAACAZqPWCQAAANAySt7U1q1btyTJJz7xiVJPDQAAAADQbNQ6AQAAAFpGyZvadttttyTJkiVLSj01AAAAAECzUesEAAAAaBklb2o74YQTUiwW8+CDD5Z6agAAAACAZqPWCQAAANAySt7Udt5552WXXXbJzTffnP/8z/8s9fQAAAAAAM1CrRMAAACgZZS8qa1r1675j//4j+y111456qijcs4552Tq1Kn529/+lmKxWOp0AAAAAAA2ilonAAAAQMuoKvWEbdu2rbsuFou5/fbbc/vttzfq2UKhkJUrV7ZUagAAAAAAjabWCQAAANAySt7U9tF3KHrHIgAAAACwOVLrBAAAAGgZJW9qu/zyy0s9JQAAAABAs1PrBAAAAGgZLdrUNmHChCTJ8ccfn65duyZR6AEAAAAANj9qnQAAAACl06JNbaNGjUqhUMiAAQPSv3//de6/8cYbufnmm1MoFHLppZe2ZCoAAAAAABtNrRMAAACgdEp+/OiaXn/99VRXVyv0AAAAAACbNbVOAAAAgObTptwJAAAAAAAAAAAAQC1NbQAAAAAAAAAAAFQMTW0AAAAAAAAAAABUDE1tAAAAAAAAAAAAVAxNbQAAAAAAAAAAAFQMTW0AAAAAAAAAAABUjKpSTHLTTTdlhx12WOfrr7/+et31d7/73UaNddlllzVbXgAAAAAATaHWCQAAANDyStLUdvPNNzd4r1AoJEmuuOKKRo2l0AMAAAAAlItaJwAAAEDLa/GmtmKx2Gxj1RaFAAAAAABKTa0TAAAAoDRatKntsccea8nhAQAAAABKQq0TAAAAoHRatKlt0KBBLTk8AAAAAEBJqHUCAAAAlE6bcicAAAAAAAAAAAAAtTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxqsqdAAAAAAAAAACsV3V1y8YDABXFTm0AAAAAAAAAAABUDE1tAAAAAAAAAAAAVAxNbQAAAAAAAAAAAFQMTW0AAAAAAAAAAABUDE1tAAAAAAAAAAAAVAxNbQAAAAAAAAAAAFQMTW0AAAAAAAAAAABUDE1tAAAAAAAAAAAAVAxNbQAAAAAAAAAAAFQMTW0AAAAAAAAAAABUDE1tAAAAAAAAAAAAVAxNbQAAAAAAAAAAAFQMTW0AAAAAAAAAAABUDE1tAAAAAAAAAAAAVAxNbQAAAAAAAAAAAFQMTW0AAAAAAAAAAABUDE1tAAAAAAAAAAAAVAxNbQAAAAAAAAAAAFQMTW0AAAAAAAAAAABUjKpyJwAAAAAAAABAGVRXt2w8AMBGslNbknnz5uXCCy9Mv3790qlTp2y77bYZOHBgrr322tTU1Gz0uFOnTk2hUGjSx+DBg+sda9ddd23U87vuuutG5wsAAAAAAAAAAFBurX6ntsmTJ2fkyJF5++23675WU1OTmTNnZubMmRk7dmx+85vfZPfddy9JPnvuuWdJ5gEAAAAAAAAAAKhErbqp7ZlnnsmIESNSU1OTzp075+KLL86QIUOyfPny3H333fnJT36SF154Icccc0xmzpyZzp07N2n8/fffP88+++wG484999xMmzYtSXL66aevN3b48OG58sorG7zfrl27JuUIAAAAAAAAAABQSVp1U9v555+fmpqaVFVV5eGHH85BBx1Ud++II47IJz7xiVx00UWZM2dOxowZk8suu6xJ43fq1Cl77733emPeeuutPPHEE0mSvn375uCDD15vfPfu3Tc4JgAAAAAAAAAAwOaqTbkTKJeZM2dm6tSpSZIzzzxzrYa2WqNHj06/fv2SJDfccEM++OCDZs/jnnvuyYoVK5IkX/ziF5t9fAAAAAAAAAAAgM1Jq21qmzhxYt31GWecUW9MmzZtctpppyVJlixZUtcE15wmTJiQJCkUCpraAAAAAAAAAACAVq/VNrU9/vjjST48InS//fZrMG7QoEF119OnT2/WHP785z/nv/7rv5Ikhx12WHbbbbdmHR8AAAAAAAAAAGBz02qb2mbPnp0k6du3b6qqqhqM22uvvdZ5prnU7tKWJKeffnqjnvnd736Xv/u7v0unTp3SsWPH7LbbbjnllFMyceLEFIvFZs0PAAAAAAAAAACg1Bru5tqCvffee1m8eHGSpHfv3uuN3WabbdKpU6csW7Ysr7zySrPm8fOf/zxJ0qFDh5x00kmNeubll19e6/O5c+dm7ty5uffee3PIIYfknnvuyc4779zkXObPn7/e+wsXLmzymAAAAABAy1DPAwAAALZkrbKpbenSpXXXnTt33mB8bVPbu+++22w5PP744/nLX/6SJDnhhBPStWvX9ca3a9cuxx13XI488sjsvffe6datW956663MmDEjN998c1555ZX8/ve/z9ChQzNjxox069atSfn06dNno78XAAAAAKC01PMAAACALVmrbGp777336q7btWu3wfitt946SbJ8+fJmy+FnP/tZ3fVpp522wfgnn3wy3bt3X+frgwcPzrnnnpuTTjopDz/8cGbPnp0rrrgiY8aMabZcAQAAAAAAAAAASqVVNrW1b9++7vr999/fYPyKFSuSfHhMaHNYsWJFfvnLXyZJdtppp3zmM5/Z4DP1NbTV6tKlS+699958/OMfz5tvvpnbbrstV199daMa9mpt6GjVhQsXZuDAgY0eDwAAAABoOep5AAAAwJasVTa1denSpe66MUeKLlu2LEnjjiptjF//+td56623kiQjR45M27ZtN3nMbt265XOf+1z+7d/+LcuWLctTTz2Vgw8+uNHP9+7de5NzAAAAAABKQz0PAAAA2JK1KXcC5dC+ffv06NEjSTJ//vz1xi5ZsqSuqa1Pnz7NMv+ECRPqrhtz9Ghj9e/fv+56wYIFzTYuAAAAAAAAAABAqbTKprYk6devX5LkpZdeysqVKxuMmzNnzjrPbIrXX389U6ZMSZJ86lOfyt57773JY9YqFovNNhYAAAAAAAAAAEA5tNqmtkMPPTTJh0eLPv300w3GTZs2re76kEMO2eR577zzzromuubcpS1J/vSnP9Vd77TTTs06NgAAAAAAAAAAQCm02qa2448/vu563Lhx9casXr267qjQ7t27Z8iQIZs8b+14VVVV+fznP7/J49V6++23c8899yRJOnbsmAEDBjTb2AAAAAAAAAAAAKXSapvaBg4cmMMOOyxJcvvtt2fGjBnrxFx33XWZPXt2kuS8887LVltttdb98ePHp1AopFAopLq6eoNzPv/88/njH/+YJDn66KOz/fbbNyrXhx56KMuXL2/w/tKlSzNixIi8+eabSZIzzzwzW2+9daPGBgAAAAAAAAAAqCRV5U6gnG688cYccsghWb58eY488shccsklGTJkSJYvX5677747t912W5Jkjz32yOjRozd5vp/+9Kd116effnqjn7v66qszcuTInHjiiTn00EPz8Y9/PJ07d85bb72VGTNm5Oabb84rr7ySJNlzzz0b1WAHAAAAAAAAAABQiVp1U9u+++6be+65J1/4whfyzjvv5JJLLlknZo899sjkyZPTpUuXTZpr9erVufPOO5Mk22yzTYYNG9ak5//2t79l7NixGTt2bIMxhx9+eO68885su+22m5QrAAAAAAAAAABAubTqprYkOfbYYzNr1qzceOONmTx5cubPn5927dqlb9++Ofnkk3PuueemY8eOmzzPo48+mgULFiRJTjnllCYdD3rttdfm0UcfzYwZM/LCCy9k8eLFeeutt9KxY8fstNNOOeCAA3LqqafmyCOPTKFQ2ORcAQAAAAAAAAAAyqXVN7UlyS677JIxY8ZkzJgxTXpu1KhRGTVqVKNihw4dmmKxuBHZJQMGDMiAAQM26lkAAAAAAAAAAIDNSZtyJwAAAAAAAAAAAAC1NLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDGqyp0AAAAAAAAAAABJ9dTqpsUP/r/4pjy75nMAlchObQAAAAAAAAAAAFQMTW0AAAAAAAAAAABUDE1tAAAAAAAAAAAAVAxNbQAAAAAAAAAAAFQMTW0AAAAAAAAAAABUDE1tAAAAAAAAAAAAVAxNbQAAAAAAAAAAAFQMTW0AAAAAAAAAAABUDE1tAAAAAAAAAAAAVAxNbQAAAAAAAAAAAFSMqnInAAAAAAAAAABAeVRPrW5a/OCmxQNsDDu1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxqsqdAABAJameWl3a+QaXdj4AAAAAAACASmentiTz5s3LhRdemH79+qVTp07ZdtttM3DgwFx77bWpqanZpLGrq6tTKBQa9TF16tQNjvfmm2/m8ssvzz777JNu3bqla9eu2WeffXL55ZfnzTff3KRcAQAAAAAAAAAAyq3V79Q2efLkjBw5Mm+//Xbd12pqajJz5szMnDkzY8eOzW9+85vsvvvuZczyQzNnzszw4cOzcOHCtb4+a9aszJo1K2PHjs2vf/3rDBgwoEwZAgC0XqXe5S+x0x8AAAAAAABbplbd1PbMM89kxIgRqampSefOnXPxxRdnyJAhWb58ee6+++785Cc/yQsvvJBjjjkmM2fOTOfOnTdpvmeffXa993fbbbcG7y1YsCDHHntsFi1alKqqqlxwwQUZNmxYkmTSpEkZM2ZMXn311QwbNixPP/10dt55503KFQAAAAAAAAAAoBxadVPb+eefn5qamlRVVeXhhx/OQQcdVHfviCOOyCc+8YlcdNFFmTNnTsaMGZPLLrtsk+bbe++9N/rZf/mXf8miRYuSJHfeeWdOPvnkunuHHXZYBgwYkBEjRmTRokW59NJLc8cdd2xSrgAAAAAAAAAAAOXQapvaZs6cmalTpyZJzjzzzLUa2mqNHj0648aNy+zZs3PDDTfk4osvzlZbbVXiTJNFixbl5z//eZLkqKOOWquhrdbJJ5+co446KlOmTMmECRNy1VVXZccddyx1qgAAAAAAAADQ6lVPrW587ODGxwK0Fm3KnUC5TJw4se76jDPOqDemTZs2Oe2005IkS5YsqWuCK7UHHnggq1atStJwrkkyatSoJMmqVavywAMPlCI1AAAAAAAAAACAZtVqm9oef/zxJEmnTp2y3377NRg3aNCguuvp06e3eF71qc01WTufj6qEXAEAAAAAAAAAADZFq21qmz17dpKkb9++qapq+BTWvfbaa51nNtbQoUOz3XbbpV27dtlhhx0yePDgXH311VmyZEmjcu3WrVt69uzZYFyvXr3StWvXZskVAAAAAAAAAACgHBru5tqCvffee1m8eHGSpHfv3uuN3WabbdKpU6csW7Ysr7zyyibN+8gjj9Rdv/HGG5k2bVqmTZuWa665JuPHj8/w4cPrfa523g3lmiR9+vTJ888/3+Rc58+fv977CxcubNJ4ANBY1dWtY04AAIDmpJ4HANAETSkKKyADQEVolU1tS5curbvu3LnzBuNrm9refffdjZrvk5/8ZI4//vgMHDgwO+20Uz744IO88MIL+cUvfpGHH344b731Vv7xH/8xDz74YI4++ugG821srkmanGufPn2aFA8AAAAAlI96HgAAALAla5VNbe+9917ddbt27TYYv/XWWydJli9f3uS5zj///FTX081/wAEH5LTTTsutt96ar3zlK1m1alXOOuusvPTSS+nQoUO9+bZ0rgAAAAAAAAAAAOXWKpva2rdvX3f9/vvvbzB+xYoVSbJOs1ljdO/efb33v/zlL+epp57K2LFj8+qrr+b+++/PyJEj18m3pqamRXPd0HGlCxcuzMCBA5s0JgAAAADQMtTzAAAAgC1Zq2xq69KlS911Y47pXLZsWZLGHf+5Mb785S9n7NixSZJp06at09TWpUuX1NTUtGiuvXv3blI8AAAAAFA+6nkAAADAlqxVNrW1b98+PXr0yOLFizN//vz1xi5ZsqSuUaxPnz4tkk///v3rrhcsWLDO/d69e2fRokUbzDX5v3dotlSuALClmzq1xBMOLvF8AAAAAAAAABWuTbkTKJd+/folSV566aWsXLmywbg5c+as80xzKxaL671f2/T29ttv57XXXmswbuHChXnnnXeStFyuAAAAAAAAAAAALanVNrUdeuihST48rvPpp59uMG7atGl114ccckiL5PKnP/2p7nqnnXZa535trh/N56NKkSsAAAAAAAAAAEBLapXHjybJ8ccfn6uuuipJMm7cuBxwwAHrxKxevToTJkxIknTv3j1DhgxpkVxuvfXWuutBgwatc/+4447LV7/61axevTrjxo3LKaecUu8448ePT5K0adMmxx13XIvkCgAAAAAANK/q6paNBwAA2Ny02p3aBg4cmMMOOyxJcvvtt2fGjBnrxFx33XWZPXt2kuS8887LVltttdb98ePHp1AopFAopLqevyCfffbZvPTSS+vN49Zbb83tt9+eJOnZs2dOOOGEdWJ69uyZkSNHJkmmTJmS++67b52YX/7yl5kyZUqS5Itf/GJ69uy53nkBAAAAAAAAAAAqUavdqS1JbrzxxhxyyCFZvnx5jjzyyFxyySUZMmRIli9fnrvvvju33XZbkmSPPfbI6NGjmzz+008/nbPOOitDhgzJ0UcfnU9+8pPZbrvtsnLlysyZMyc///nP8x//8R9JkrZt2+bWW29Np06d6h3r+9//fh566KG88cYbOfXUU/PUU09l2LBhSZJJkybluuuuS5Jsv/32ufLKKzfmPwcAAAAAAAAAAEDZteqmtn333Tf33HNPvvCFL+Sdd97JJZdcsk7MHnvskcmTJ6dLly4bNceqVavyyCOP5JFHHmkwZrvttsvtt9++3iND+/TpkwcffDDHH398XnvttVxzzTW55ppr1orp2bNnJk6cmN69e29UrgAAAAAAAAAAAOXWqpvakuTYY4/NrFmzcuONN2by5MmZP39+2rVrl759++bkk0/Oueeem44dO27U2P/wD/9Qd7TpH//4xyxatChvvvlmisVitt122+yzzz757Gc/m1GjRqVr164bHO+AAw7Is88+mxtvvDETJ07M3LlzkyS77bZbhg8fnvPPPz/bbbfdRuUKAAAAAABsfqqrWyYWAACgnFp9U1uS7LLLLhkzZkzGjBnTpOdGjRqVUaNGNXh/hx12yJe+9KV86Utf2sQM/0+PHj3yve99L9/73veabUwAAAAAAAAAAIBK0abcCQAAAAAAAAAAAEAtTW0AAAAAAAAAAABUDMePAgAAAAAAAGzOqqtbJhYAoEzs1AYAAAAAAAAAAEDFsFMbALQy5XgTnjf+AQAAAAAAANBYmtoAAAAAAACaQVPf2OeNgAAAAPXT1AYAAAAAAAAAQJNUT61uWvzgpsUDrVubcicAAAAAAAAAAAAAtTS1AQAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxNLUBAAAAAAAAAABQMarKnQAAAA2bO7fcGQAAAAAAAACUlqY2AAAAAACAVqa6umXjAQAANoWmNgAAAAAAAACAj6ieWt20+MFNiwegYW3KnQAAAAAAAAAAAADU0tQGAAAAAAAAAABAxXD8KAAAbIGqq1vHnAAAAAAAAGx5NLUBAEAzmDq1DJMOLsOcG6nUDW8a7AAAAAAAADZfmtoAAAAAAABotKa8kcibjgAAgI2hqQ0AYA2jyrLdFgAAAAAAAAC12pQ7AQAAAAAAAAAAAKhlpzYAAGgGdvkDAAAAAACA5qGpDQAAAAAAYA3V1S0TCwAAQOM4fhQAAAAAAAAAAICKoakNAAAAAAAAAACAiuH4UQAANktz55Y7AwAAAAAAAKAl2KkNAAAAAAAAAACAiqGpDQAAAAAAAAAAgIrh+FEAABrFcZ8AAAAAAABAKdipDQAAAAAAAAAAgIqhqQ0AAAAAAAAAAICKoakNAAAAAAAAAACAiqGpDQAAAAAAAAAAgIqhqQ0AAAAAAAAAAICKUVXuBAAAAAAAANjyVVe3bDwAALDlsFMbAAAAAAAAAAAAFUNTGwAAAAAAAAAAABXD8aMAAECrU+ojbByZAwAAAAAA0Hh2agMAAAAAAAAAAKBiaGoDAAAAAAAAAACgYmhqAwAAAAAAAAAAoGJUlTsBAAAAAAAAAICWUj21umnxg5sWD0Dzs1MbAAAAAAAAAAAAFUNTGwAAAAAAAAAAABXD8aMAAAAAAMAWp7q6ZeMBAABoOXZqAwAAAAAAAAAAoGLYqQ0AWlg53uXrncUAAAAAAAAAbK7s1AYAAAAAAAAAAEDFsFMbAAAAAAAAFauppxI4xQAAKl/11OrGxw5ufCyw5bBTGwAAAAAAAAAAABVDUxsAAAAAAAAAAAAVQ1MbAAAAAAAAAAAAFUNTGwAAAAAAAAAAABVDUxsAAAAAAAAAAAAVQ1MbAAAAAAAAAAAAFUNTGwAAAAAAAAAAABVDUxsAAAAAAAAAAAAVQ1MbAAAAAAAAAAAAFUNTGwAAAAAAAAAAABVDUxsAAAAAAAAAAAAVo6rcCQAAAAAAAEBLqK5umVgAAKBl2akNAAAAAAAAAACAiqGpDQAAAAAAAAAAgIqhqQ0AAAAAAAAAAICKoakNAAAAAAAAAACAiqGpDQAAAAAAAAAAgIpRVe4EAAAAWrPq6tYxJwAAAAAAQGPZqQ0AAAAAAAAAAICKYac2AAAAAAAAAKDiVU+tbnzs4MbHAlB57NSWZN68ebnwwgvTr1+/dOrUKdtuu20GDhyYa6+9NjU1NZs09jvvvJO77747Z599dj71qU+le/fuadeuXbbffvsMHjw41157bd56660NjrPrrrumUChs8GPXXXfdpHwBAAAAAAAAAADKqdXv1DZ58uSMHDkyb7/9dt3XampqMnPmzMycOTNjx47Nb37zm+y+++5NHvu3v/1tTjjhhKxYsWKde4sXL860adMybdq0XHvttbnrrrsyZMiQTfpeAAAAAAAAAAAANnetuqntmWeeyYgRI1JTU5POnTvn4osvzpAhQ7J8+fLcfffd+clPfpIXXnghxxxzTGbOnJnOnTs3afw333wzK1asSJs2bTJ06NB89rOfzT777JPu3btn/vz5+cUvfpF77rknixYtyrBhw/L73/8+f//3f7/eMYcPH54rr7yywfvt2rVrUo4AAAAAAAAAAACVpFU3tZ1//vmpqalJVVVVHn744Rx00EF194444oh84hOfyEUXXZQ5c+ZkzJgxueyyy5o0/lZbbZUvf/nLueSSS/Kxj31srXv77rtvjj322BxyyCH5xje+kZqamowePTqPPvroesfs3r179t577yblAQAAAAAAAAAAsLlotU1tM2fOzNSpU5MkZ5555loNbbVGjx6dcePGZfbs2bnhhhty8cUXZ6uttmr0HKecckpOOeWU9cb80z/9UyZMmJCnnnoqU6dOzZtvvpntttuuSd8LAAAAAAAAzae6umXjAQCA9WtT7gTKZeLEiXXXZ5xxRr0xbdq0yWmnnZYkWbJkSV0TXHMbPHhwkmT16tV5+eWXW2QOAAAAAAAAAACAzUGr3ant8ccfT5J06tQp++23X4NxgwYNqruePn16hg4d2uy5rFixou66TZtW22cIAAAAAADraMoOWHbLAgAA2DK02g6q2bNnJ0n69u2bqqqGe/v22muvdZ5pbtOmTUuSVFVVpW/fvuuN/d3vfpe/+7u/S6dOndKxY8fstttuOeWUUzJx4sQUi8UWyQ8AAAAAAAAAAKBUWuVObe+9914WL16cJOndu/d6Y7fZZpt06tQpy5YtyyuvvNLsuUyePDmzZs1Kkhx11FHp2rXreuM/ejzp3LlzM3fu3Nx777055JBDcs8992TnnXduch7z589f7/2FCxc2eUwAAAAAoGWo5wEAAABbslbZ1LZ06dK6686dO28wvrap7d13323WPP72t7/l61//epKkbdu2+d73vtdgbLt27XLcccflyCOPzN57751u3brlrbfeyowZM3LzzTfnlVdeye9///sMHTo0M2bMSLdu3ZqUS58+fTbpewEAAAAASkc9DwCgBJp6rrVzsAGg2bTKprb33nuv7rpdu3YbjN96662TJMuXL2+2HFatWpWRI0fmr3/9a5LkO9/5Tvbdd98G45988sl07959na8PHjw45557bk466aQ8/PDDmT17dq644oqMGTOm2XIFAAAAAAAAAAAolVbZ1Na+ffu66/fff3+D8StWrEiSdOjQodly+NrXvpaHHnooSXLMMcfk0ksvXW98fQ1ttbp06ZJ77703H//4x/Pmm2/mtttuy9VXX92ohr1aGzpadeHChRk4cGCjxwMAAAAAWo56HgAAALAla5VNbV26dKm7bsyRosuWLUvSuKNKG+Piiy/ObbfdliQ59NBD88tf/jJt27bdpDG7deuWz33uc/m3f/u3LFu2LE899VQOPvjgRj/fu3fvTZofAAAAACgd9TwA2AI56hIAoE6bcidQDu3bt0+PHj2SJPPnz19v7JIlS+qa2vr06bPJc19zzTW5+uqrkySf+tSnMmnSpGbbAa5///511wsWLGiWMQEAAAAAAAAAAEqpVTa1JUm/fv2SJC+99FJWrlzZYNycOXPWeWZj3XTTTfn2t79dN9aUKVPSrVu3TRpzTcVisdnGAgAAAAAAAAAAKIdW29R26KGHJvnwaNGnn366wbhp06bVXR9yyCEbPd/PfvaznHvuuUmS3XffPY888kjdbnHN5U9/+lPd9U477dSsYwMAAAAAAAAAAJRCq21qO/744+uux40bV2/M6tWrM2HChCRJ9+7dM2TIkI2a6/77788ZZ5yRYrGY3r1759FHH232prO3334799xzT5KkY8eOGTBgQLOODwAAAAAAAAAAUApV5U6gXAYOHJjDDjssjz/+eG6//facfvrpOeigg9aKue666zJ79uwkyXnnnZetttpqrfvjx4/PGWeckSS5/PLLU11dvc48Dz/8cE499dSsWrUqO+ywQx555JHsuuuuTcr1oYceyqBBg9KhQ4d67y9dujQjRozIm2++mSQ588wzs/XWWzdpDgAAAAAAADZdPf93UbPEAgBAa9Jqm9qS5MYbb8whhxyS5cuX58gjj8wll1ySIUOGZPny5bn77rtz2223JUn22GOPjB49usnjP/HEEznhhBPy/vvvZ6uttsr111+fDz74IM8991yDz/Tu3Tvdu3df62tXX311Ro4cmRNPPDGHHnpoPv7xj6dz58556623MmPGjNx888155ZVXkiR77rlnvc11AAAAAAAAAAAAm4NW3dS277775p577skXvvCFvPPOO7nkkkvWidljjz0yefLkdOnSpcnjP/TQQ6mpqUmSfPDBBxk5cuQGnxk3blxGjRq1ztf/9re/ZezYsRk7dmyDzx5++OG58847s+222zY5V4AtSTl6e/UTAwAAAAAAAEDzaNVNbUly7LHHZtasWbnxxhszefLkzJ8/P+3atUvfvn1z8skn59xzz03Hjh3LmuO1116bRx99NDNmzMgLL7yQxYsX56233krHjh2z00475YADDsipp56aI488MoVCoay5AgAAAAAAAAAAbIpW39SWJLvsskvGjBmTMWPGNOm5UaNG1burWq3q6upmOQp0wIABGTBgwCaPAwAAAAAAAADlVD21umnxg5sWD8CWQVMbAAAAAAAAAAAVT1MktB5typ0AAAAAAAAAAAAA1NLUBgAAAAAAAAAAQMXQ1AYAAAAAAAAAAEDF0NQGAAAAAAAAAABAxdDUBgAAAAAAAAAAQMXQ1AYAAAAAAAAAAEDF0NQGAAAAAAAAAABAxdDUBgAAAAAAAAAAQMXQ1AYAAAAAAAAAAEDF0NQGAAAAAAAAAABAxdDUBgAAAAAAAAAAQMXQ1AYAAAAAAAAAAEDFqCp3AgAAAAAAANDaVVe3TCwAAGyO7NQGAAAAAAAAAABAxbBTGwBQVlNTXYZZyzEnAAAAtG52oQIAAKCxNLUBAAAAAAAAAE1SPbW68bGDGx8LAImmNgAA2CLZBREAAAAAAIDNlaY2AAAA6pTjmCdHSwEAAAAAAGtqU+4EAAAAAAAAAAAAoJamNgAAAAAAAAAAACqGpjYAAAAAAAAAAAAqhqY2AAAAAAAAAAAAKoamNgAAAAAAAAAAACqGpjYAAAAAAAAAAAAqhqY2AAAAAAAAAAAAKkZVuRMAAAAAAAAAAMqjemp142MHNz4WADaFndoAAAAAAAAAAACoGJraAAAAAAAAAAAAqBia2gAAAAAAAAAAAKgYmtoAAAAAAAAAAACoGFXlTgAAAAAAAADYONXVLRsPAADlYKc2AAAAAAAAAAAAKoamNgAAAAAAAAAAACqGpjYAAAAAAAAAAAAqRlW5EwAAAAAAAAAANl711OqmxQ9uWjxsCZry78S/ESg/TW0AAAAAAAAAzaW6umXjAQBaAcePAgAAAAAAAAAAUDE0tQEAAAAAAAAAAFAxHD8KwCYrx87odmMHAAAAAAAAgC2TpjYAAAAAAAAAaC5NfWe+d/IDwDocPwoAAAAAAAAAAEDFsFMbAAAAAAAAAFSA6qnVjY8d3PhYANjc2KkNAAAAAAAAAACAimGnNgAAAAAAoFGqq1s2HgAAABI7tQEAAAAAAAAAAFBB7NQGAJTVqKlTy50CbJH82wIAAAAAAGBzpakNAAAAAAAAWqGmHBHsOGFovKlzpzYpfnCLZAEAmzfHjwIAAAAAAAAAAFAxNLUBAAAAAAAAAABQMTS1AQAAAAAAAAAAUDGqyp0AAAAAAAAAAFSaqXOnNil+cItkAQCtk6Y2AAAAKlZ1deuYEwAAAAAA+D+OHwUAAAAAAAAAAKBiaGoDAAAAAAAAAACgYjh+FAAAAAAAAIAt1tS5UxsdO7jFsgAAmsJObQAAAAAAAAAAAFQMTW0AAAAAAAAAAABUDMePAgAAAAAAAABAPaqnVjc+dnDjY4H109QGAAAAAAAAQEWbOndqk+IHt0gWAECpaGoDAAAAAAAAGq26umXjK0ZTEt9sv0kAgMrUptwJAAAAAAAAAAAAQC07tQEAAC1uaqpLPGOp5wMAAAAAAKC5aGoDgFam9I0lieYSAAAAqCxO1AOgXKbOndro2MEtlkXlGjx+ahOCWyoLACg/x48CAAAAAAAAAABQMezUBrCZKvU7ZL0jFwAAAAAAAAAoBTu1AQAAAAAAAAAAUDHs1AYAAAAAAABAk0ydO7XRsYNbLAsAYEulqQ0AAAAAAABocdXVLRsPAMCWw/GjAAAAAAAAAAAAVAw7tSWZN29efvSjH2Xy5MmZN29ett566/Tt2zcjRozI1772tXTs2LFZ5rn77rszbty4zJo1K0uWLEnPnj1z2GGH5etf/3oOPPDARo3x5ptv5kc/+lEmTpyYuXPnplgsZrfddsvxxx+fb3zjG9luu+2aJVcAAAAAAABgy9aUI0QTx4gCNEX11OqmxQ9uWjxs6Vp9U9vkyZMzcuTIvP3223Vfq6mpycyZMzNz5syMHTs2v/nNb7L77rtv9BzvvfdeTj755EyaNGmtr//1r3/NX//619x5552prq7OpZdeut5xZs6cmeHDh2fhwoVrfX3WrFmZNWtWxo4dm1//+tcZMGDARucKAABAw8px9I3jdgAAADaBM08BADZLrbqp7ZlnnsmIESNSU1OTzp075+KLL86QIUOyfPny3H333fnJT36SF154Icccc0xmzpyZzp07b9Q8Z555Zl1D25AhQ3Leeedlp512yrPPPpsf/OAH+fOf/5zLLrssvXr1yllnnVXvGAsWLMixxx6bRYsWpaqqKhdccEGGDRuWJJk0aVLGjBmTV199NcOGDcvTTz+dnXfeeeP+owAAAAAAAECFaUqvmb40AIDNX6tuajv//PNTU1OTqqqqPPzwwznooIPq7h1xxBH5xCc+kYsuuihz5szJmDFjctlllzV5jmnTpuXOO+9Mkhx77LH51a9+lbZt2yZJ9t9//xx33HHZb7/9Mm/evFx00UU56aST0r1793XG+Zd/+ZcsWrQoSXLnnXfm5JNPrrt32GGHZcCAARkxYkQWLVqUSy+9NHfccUeTcwUAAAAAYPNh8yEAajXlGNHBLZYF5TR4/NQmPtASWQBA82m1TW0zZ87M1KlTk3y4k9qaDW21Ro8enXHjxmX27Nm54YYbcvHFF2errbZq0jw//OEPkyRt27bNTTfdVNfQVqtHjx655pprcuqpp2bJkiW5/fbbM3r06LViFi1alJ///OdJkqOOOmqthrZaJ598co466qhMmTIlEyZMyFVXXZUdd9yxSbkC0DqM+n8//wBKqdJee6amusQzlno+AAAAAACAzVebcidQLhMnTqy7PuOMM+qNadOmTU477bQkyZIlS+qa4Brr3XffzaOPPpokGTp0aHr37l1v3IknnpiuXbsmSe6///517j/wwANZtWrVenNNklGjRiVJVq1alQceeKBJuQIAAAAAAMCWZurUpn0AAFAZWu1ObY8//niSpFOnTtlvv/0ajBs0aFDd9fTp0zN06NBGz/Hkk09mxYoV64zzUe3atcuBBx6Yhx9+OE8++WQ++OCDtXaEq811Q+N8NNezzz670bkCG1bq4xkcB7HlKP1uQIkdgQAAAACALcngqdVNiG5KbMOa0uQ2uDnma8IRos01JwCVqboJP/eqBzc+FjYnrbapbfbs2UmSvn37pqqq4f8Me+211zrPNHWOj47T0DwPP/xwVq5cmRdffDH9+/dfZ5xu3bqlZ8+eDY7Rq1evdO3aNe+8806Tc50/f/5677/yyit11wsXLmzS2LCleOed0s63gX+WFZVPqXNJNq98VpQhofW9rv/t/zVcl5J8Grahn8GVlE9r/98qkc+GVFI+G/q3VerX5vXlU2k/t+SzeeUzZkzp8qh1wQWln5PWZc26w8qVK8uYCVQ29bwtS1N/ptf+PG7q7yZrLpumPLuxz5Vjzs0p1zWf3ZxyLcecm1OupZpzc8p1zWc3Jdc3VjT+4TV/TjbluU15ds3nZkxp2pwHnfXhs28sa1r9ZK1cm/Dsxj5Xjjk3p1zXfHZzyjVJ/r9/m9L45/qeVXf9zhsb92+kKc9tyrPNMefmlGs55tycci3VnOXOFcqhpep5hWKxWGy20TYT7733Xjp06JAkOeaYYzJp0qT1xnfu3DnLli3LgQcemBkzZjR6nm9/+9u55pprkiQzZ87MgAEDGoy99tpr88///M9JkoceeihHHXVU3b2ePXtm0aJF+f/+v/8vzz333Hrn3HvvvfP888+nZ8+eTSpWFQqFRscCAAAAlMKTTz6Z/fffv9xpQEVSzwMAAAAqTXPW89o0yyibmaVLl9Zdd+7ceYPxnTp1SpK8++67LTZP7Rz1zVM7TkvmCgAAAAAAAAAAUAla5fGj7733Xt11u3btNhi/9dZbJ0mWL1/eYvPUzlHfPLXjtGSuax5HUJ/33nsvc+bMyY477pjtt99+vUe2QrksXLgwAwcOTPJh92+vXr3KnBFsHGuZLYn1zJbCWmZLYS2zOVi5cmXeeOONJMknP/nJMmcDlUs9r+n8HGRjWTtsCuuHTWH9sCmsHzaF9cOmsH5an5aq57XKSkb79u3rrt9///0Nxq9Y8eH547VHlrbEPLVz1DdP+/btU1NT06K59u7de4Mxffv2bdKYUE69evVq1LqGSmctsyWxntlSWMtsKaxlKtmuu+5a7hSg4qnnbRo/B9lY1g6bwvphU1g/bArrh01h/bAprJ/WoyXqea3y+NEuXbrUXTfmmM5ly5Yladzxnxs7T+0c9c1TO05L5goAAAAAAAAAAFAJWmVTW/v27dOjR48kyfz589cbu2TJkrpGsT59+jRpnjW7TTc0z5rHBXx0ntpxNjTGmuM0NVcAAAAAAAAAAIBK0Cqb2pKkX79+SZKXXnopK1eubDBuzpw56zzTWP379693nPXNU1VVtc6xALXjvP3223nttdcaHGPhwoV55513NipXAAAAAAAAAACAStBqm9oOPfTQJB8e1/n00083GDdt2rS660MOOaRJc+y///5p167dOuN81Pvvv58nnnhinWc+muuGxtmUXAEAAAAAAAAAACpBq21qO/744+uux40bV2/M6tWrM2HChCRJ9+7dM2TIkCbN0aVLl3z6059OkjzyyCMNHh96//331+2wdsIJJ6xz/7jjjkubNm3Wm2uSjB8/PknSpk2bHHfccU3KFQAAAAAAAAAAoBK02qa2gQMH5rDDDkuS3H777ZkxY8Y6Mdddd11mz56dJDnvvPOy1VZbrXV//PjxKRQKKRQKqa6urneeCy+8MEmycuXKfP3rX8+qVavWur948eJ861vfSvJh49xZZ521zhg9e/bMyJEjkyRTpkzJfffdt07ML3/5y0yZMiVJ8sUvfjE9e/Zs8HsHAAAAAAAAAACoVK22qS1JbrzxxnTo0CErV67MkUcemauuuipPPPFEHnvssXz5y1/ORRddlCTZY489Mnr06I2a44gjjsjnPve5JMkDDzyQoUOH5oEHHshTTz2VcePG5cADD8y8efOSJFdffXW22Wabesf5/ve/n+233z5Jcuqpp+bb3/52pk+fnunTp+fb3/52Pv/5zydJtt9++1x55ZUblSsAAAAAAAAAAEC5FYrFYrHcSZTTgw8+mC984Qt1x39+1B577JHJkyenb9++69wbP358zjjjjCTJ5Zdf3uBubcuXL89JJ52U3/zmN/Xeb9OmTS699NIGn6/1hz/8Iccff3xee+21eu/37NkzEydOzAEHHLDecQAAAAAAAAAAACpVq96pLUmOPfbYzJo1K9/85jezxx57pGPHjunevXsGDBiQa665Jn/84x/rbWhrig4dOmTy5Mn5xS9+kaFDh2aHHXZIu3bt0qdPn3z+85/P9OnTN9jQliQHHHBAnn322XznO9/J3nvvnc6dO6dz58755Cc/me985zt57rnnNLQBAAAAAAAAAACbtVa/UxsAAAAAAAAAAACVo9Xv1AYAAAAAAAAAAEDl0NQGAAAAAAAAAABAxdDUBgAAAAAAAAAAQMXQ1AYAAAAAAAAAAEDF0NQGAAAAAAAAAABAxdDUBgAAAAAAAAAAQMXQ1AYAAAAAAAAAAEDF0NQGAAAAAAAAAABAxdDUBmwR3n333fzud7/LtddemxEjRmS33XZLoVBIoVDIrrvu2uTxnn/++XzlK19J375906FDh2y//fY5/PDDc+utt2blypXN/w1AEwwePLhufW/oA8pp3rx5ufDCC9OvX7906tQp2267bQYOHJhrr702NTU15U4P1quxr7ODBw8ud6q0cq+//nomTZqUyy67LEcffXR69OhRtz5HjRrV5PEeeuihnHjiiendu3e23nrr9O7dOyeeeGIeeuih5k8eAEpk3rx5ufnmm3PKKadkzz33TKdOndK+ffv07t07w4cPz1133dWkeo+6Ueui7simUBvho/wNx8b67//+7/zgBz/I0UcfnT59+mTrrbdO586ds8cee2TUqFF5/PHHmzSetdN6vPPOO7n77rszevToDBo0KH379k23bt3Srl277LDDDhk8eHB++MMf5s0332zUeNYOtS666KK1auVTp07d4DPWD01VKBaLxXInAbCphgwZ0uAPyl122SVz585t9Fi33357vv71r2fFihX13j/wwAMzadKkbLfddhuRKWy6wYMHZ9q0aY2K9WOecpk8eXJGjhyZt99+u977e+65Z37zm99k9913L3Fm0DiNbQweNGhQo/5Yh5ayvrV6+umnZ/z48Y0ap1gs5itf+Upuu+22BmPOOeec3HLLLRrnAdisXHbZZbnyyis3+PfxgAED8u///u/52Mc+tt44daPWR92RjaU2Qn38DcfGGDRoUH73u99tMO6LX/xixo4dm3bt2jUYY+20Po888kiGDh26wbgePXrk5z//eY466qh671s7rOmZZ57JgAED1npTxmOPPdbgm8CtHzaWndqALcKahcltttkmQ4cOTefOnZs8zpQpU3LOOedkxYoV2XHHHfOjH/0of/jDH/Lb3/42J554YpLkiSeeyIknnpjVq1c3W/6wMQYMGJBnn312vR9QDs8880xGjBiRt99+O507d873v//9/Nd//VceffTRnH322UmSF154Icccc0zefffdMmcL6/fVr351va+z48aNK3eKUKdPnz458sgjN+rZ73znO3VFpX333Td33XVXnnzyydx1113Zd999kyS33XZbLr300mbLFwBK4dVXX02xWEynTp3yhS98IePGjcv06dPz1FNP5Wc/+1n233//JMlTTz2Vz3zmM+v9G0XdqHVSd2RjqI3QGP6Go7EWLFiQJNlpp51y3nnn5b777suTTz6ZGTNmZMyYMdl5552TJD/72c82uOOftdM69enTJ6eddlpuvPHG3H///ZkxY0Z+//vf55577snJJ5+ctm3bZvHixTnuuOMya9asesewdqi1evXqnH322Vm5cmV22GGHRj1j/bDRigBbgFtvvbX4i1/8ovjiiy/WfW2XXXYpJinusssujRrjgw8+KPbt27eYpNi1a9fiSy+9tE7M1772tWKSYpLiT3/60+ZKH5pk0KBBxSTFQYMGlTsVqNfgwYOLSYpVVVXF//qv/1rn/g9/+MO619IrrriiDBnChtWu0csvv7zcqcB6XXbZZcUHH3yw+NprrxWLxWLx5Zdfrlu/p59+eqPGePHFF4tVVVXFJMUBAwYUa2pq1rq/bNmy4oABA+pe2+v7PRkAKtVFF11UvOaaa4rvvPNOvfdXrlxZHDFiRN3Pz+9+97v1xqkbtV7qjmwMtREa4m84NsYxxxxTvOeee4orV66s9/4bb7xR3GOPPerW0u9+97t646yd1qmhdbOmX/3qV3Xr58QTT1znvrXDmq6//vpikuJee+1VvPjii+vWzmOPPVZvvPXDprBTG7BFOOecc/L5z38+ffv23egxfvWrX+Wll15Kklx88cX5+Mc/vk7Mv/7rv2abbbapuwZgbTNnzqw7luXMM8/MQQcdtE7M6NGj069fvyTJDTfckA8++KCUKQJsUa644ooMGzYsO+6440aPcf3119cdFfDjH/84HTp0WOt+x44d8+Mf/zhJsnLlytxwww0bPRcAlNo111yTiy66KF26dKn3ftu2bXPTTTfVHdN133331RunbtR6qTvSVGojrI+/4dgYkyZNyogRI9K2bdt67/fo0SPXXXdd3ecN/T5j7bRODa2bNR1//PHZa6+9kqTeo26tHWq98sordbup3Xzzzes97riW9cOm0NQG8P9MnDix7rqh7Zk7duyYESNGJEmee+65vPjiiyXIDGDzseZr6RlnnFFvTJs2bXLaaaclSZYsWVJX6AWg9IrFYn79618nSfbaa68ceOCB9cYdeOCB2XPPPZN8+FpfXOMYLgDY3G233Xb5u7/7uyTJn//853pj1I3YFNZP66I2QkvyNxwNGTx4cN11fb/PWDtsSKdOnZIk77333lpft3ZY09e+9rW8++67Of3009d63WmI9cOm0tQG8P88/vjjSZI999wzPXv2bDBu0KBBddfTp09v8bwANie1r6WdOnXKfvvt12Cc11KAyvDyyy9nwYIFSdZ+ba5P7f358+dn7ty5LZ0aAJTUihUrknzYaFIfdSM2hfXTuqiN0JL8DUdD3n///brr+n6fsXZYn9mzZ+d//ud/kqRux7Za1g617r333kyaNCnbbrtto3cWtn7YVJraAJK8++67mT9/fpJ1f1n7qDXvz549u0XzgvWZM2dO9t9//3Tp0iXt27dP7969M3z48EyYMMGRBZRN7eti3759U1VV1WCc11I2F7/85S+z5557pkOHDunSpUs+8YlP5PTTT89jjz1W7tSgWaz5Guz3YABaq9dff73uZ1t9Pw/VjdgU1k/rozZCS/I3HA2ZNm1a3XV9a8Pa4aNqamry4osvZsyYMRkyZEhWrVqVJDnvvPPWirN2SJK33nqrbm1cc8012X777Rv1nPXDpmr4t2mAVmT+/Pl125j27t17vbF9+vSpu37llVdaNC9Yn0WLFmXRokV1ny9YsCALFizIAw88kGuuuSb33Xdf+vXrV8YMaW3ee++9LF68OMmGX0u32WabdOrUKcuWLfNaSkX705/+tNbnL730Ul566aVMmDAhxx9/fMaPH59u3bqVKTvYdGu+Bvs9GIDW6l//9V+zcuXKJKk7/nFN6kZsCuundVEboaX5G476rF69OldffXXd5/X9PmPtkCTjx49v8GjsJLnwwgszcuTItb5m7ZAkF110UV577bUcfPDBOfPMMxv9nPXDptLUBpBk6dKlddedO3deb2ztmfLJh++0hFJr06ZNPv3pT+cf/uEfss8++2S77bbL0qVL89///d+59dZbM3v27PzpT3/KkCFD8uSTT+ZjH/tYuVOmlWjKa2mSusKt11IqUceOHXPcccfl05/+dPbaa6907tw5b7zxRqZNm5Zbbrklb775ZiZOnJjhw4fnP/7jP7LVVluVO2XYKH4PBqC1+8Mf/pAbbrghyYf/J8vXvva1dWL8vGRTWD+ti9oILc1rCvW5/vrr8+STTyZJTjjhhAwYMGCdGGuH9fn7v//73HLLLTnggAPWuWftMH369IwdOzZVVVW55ZZbUigUGv2s9cOm0tQGkA/fQVerXbt2643deuut666XL1/eYjlBQ+6///507959na8fdthh+drXvpazzz47P/3pT7No0aKcf/75uf/++0ufJK1SU15Lk/97PfVaSiVasGBBva+1Q4cOzT/90z/l6KOPzh//+MdMmzYtN998c77xjW+UPkloBn4PBqA1W7RoUU466aSsXLkyhUIhP/3pT9OxY8d14vy8ZFNYP62L2ggtzWsKHzVt2rR8+9vfTpLssMMOufnmm+uNs3ZIkuOPP76u6XH58uX585//nHvvvTe/+tWvMnLkyNxwww0ZNmzYWs9YO63b+++/n3POOSfFYjHf/OY388lPfrJJz1s/bKo25U4AaD1qC4Sb+jF+/Phmz619+/Z11++///56Y1esWFF33aFDh2bPhS1DS673+posam211VYZO3Zs3bnzv/rVr7JgwYIW+i5hbU15LU3+7/XUaymVaH2vtTvuuGPuu+++uj/Cf/zjH5coK2h+fg8GoFKUum60dOnSHHPMMZk/f36S5Ac/+EGOOOKIemP9vKx86o5UCrURWprXFNb0/PPP54QTTsjKlSuz9dZb5957782OO+5Yb6y1Q/JhzXPvvffO3nvvnf333z+f+9zncv/992fChAn5y1/+kuHDh6/z+5C107r94Ac/yOzZs/Oxj30sl19+eZOft37YVJraAJJ06dKl7npD25kuW7as7roxW8hDqVVVVa11nv20adPKmA2tSVNeS5P/ez31WsrmaPfdd8/QoUOTJC+99FJeffXVMmcEG8fvwQC0Ru+9916GDx+ep59+OklywQUX1O1wUh8/L9kU1k/rojZCS/OaQq2XX345Rx55ZJYsWZK2bdvmrrvuyqBBgxqMt3ZYny9+8Ys5+eSTs3r16px77rlZsmRJ3T1rp/WaM2dOrrrqqiQfvrF7zeNBG8v6YVM5fhQomaqqqsyePXuTx+nVq1czZLO23r17113XvkO3Ia+88krddZ8+fZo9F7YM5V7v/fv3r7u2Uxul0r59+/To0SOLFy/e4GvpkiVL6v5A8VrK5qp///6ZPHlykg9fa3faaacyZwRN5/dgACpFqf6OXrlyZUaMGJHHHnssSXLWWWfluuuuW+8zfl5WvnLXYdbH+mld1EZoaV5TSJJXX301n/nMZ/Lqq6+mUCjkjjvuyAknnLDeZ6wdNmT48OG59957s2zZsvz2t7/N5z//+STWTmt2/fXX5/3338/uu++empqa3H333evEPPfcc3XX//mf/5nXXnstSXLsscemU6dO1g+bTFMbUFK1RyJWms6dO6dPnz555ZVXMmfOnPXGrnm/X79+LZ0am7FyrvdisVi2uWnd+vXrl8cffzwvvfRSVq5cmaqq+n/d9Fr6/7d350FVnfcfxz8XZFHccYuI4BIUIiK2osbkhyuaRGsQo7UuUONSYhNtbWprnKhTW81U625coqCJHY3RTDQa1wGjHY1ESUVxi2hdSFQWN0DDcn5/EM5AuBcQUK7h/Zq5M+fe8zzP+Z7rM9d7v3yfc/BzwGctfg4KF8LzPRgAUNUe9+/ovLw8jR49Wjt27JAkDR8+XKtWrSq1H3mjpwN5R9gLciN4nPgNh5SUFPXr109JSUmS8q+eNGbMmFL7MXdQmsaNG5vb//vf/8xt5k71VXA70KSkJI0YMaLU9n/729/M7UuXLsnNzY35gwrj9qMA8KMXXnhBknTu3Dmzityawrdy7NGjx2OPCyiPxMREc5srB+FJKvgszcjIMG/lYw2fpfg54LMWPwetWrUy529ptyz/8ssvJUkeHh7y9vZ+3KEBAFDpJk6caF5dYODAgfrwww/l4FC2FDl5I1QE86d6ITeCx4nfcNXbnTt31L9/fzMnNW/ePE2aNKlMfZk7KE3hu/4UvvUjcwcVwfxBRVHUBgA/evXVV83t6Ohoq20yMzP18ccfS8pfmeDj4/MEIgMeTU5OjtatW2c+/7//+78qjAbVTeHP0qioKKtt8vLytGHDBklS/fr11atXrycRGlCpkpKStG/fPklS69at5eHhUcURAeVjsVg0ePBgSfmrIY8ePWq13dGjR83VkoMHD5bFYnliMQIAUBn++Mc/6oMPPpAk9enTR5988omcnJzK3J+8ESqC+VO9kBvB48RvuOorMzNTr7zyik6cOCFJeueddzRt2rQy92fuoDRbtmwxt/39/c1t5k71FR0dLcMwSnzMnDnTbB8TE2O+XlCUxvxBRVHUBgA/Cg0NVZs2bSRJc+fO1cWLF4u1efvtt5Wenm5uA09aTEyMbt++bXN/dna2xo0bZ37xGzRoEPedxxMVFBSkF198UZK0du1aHTlypFibBQsW6MyZM5KkyZMnP9IfkoAnYceOHcrJybG5/8aNGxo6dKiys7MlqcwrYgF7NWXKFPOWSG+++aaysrKK7M/KytKbb74pSapRo4amTJnypEMEAKBCZs2apYULF0qSnn/+eX322WdycXF5pDHIG6EimD/VC7kRPG78hqt+fvjhB4WGhuo///mPpPzPjTlz5jzyOMyd6ik6OloPHjwosc3ChQu1a9cuSZK3t7d51dECzB1UBPMHFWExDMOo6iAAoKK+/fZbHT58uMhrf/rTn5Samip3d3fNnz+/yL4BAwaoWbNmxcbZtWuXBg0apLy8PDVt2lQzZsxQUFCQ0tPTtWbNGm3dulVS/iXkY2Nj5ejo+PhOCrAiIiJCW7du1a9+9Sv17NlT7dq1U926dXX//n0dP35cq1atMhNiTZo00dGjR9WqVasqjhrVTXx8vHr06KGsrCzVrl1b06dPV69evZSVlaVNmzZp9erVkiQfHx99/fXXqlOnThVHDBTl7e2t7OxshYWFqXv37vL29lbNmjWVkpKi2NhYrVy5UqmpqZLyvxPs37//kf8oClSWw4cP69tvvzWfp6SkmH8E7dGjh8aNG1ekfUREhNVx/vrXv2revHmSpMDAQE2bNk1t2rTRxYsX9d577yk+Pt5s949//OMxnAkAAI/H0qVL9dZbb0nKv43N5s2bVa9evRL7tGvXzmqBCXmj6om8I8qD3Ahs4TccyiMsLEzbtm2TJPXu3VuLFi0q8SpGzs7ONq/4ydypfry9vXXv3j2FhYXphRdeUJs2bVS7dm3du3dPCQkJ2rhxo1kw6ezsrJ07d6pv377FxmHuwJpZs2Zp9uzZkvIvzNGzZ0+r7Zg/KDcDAH4GoqKiDEllfsTExNgca/Xq1Yazs7PNvkFBQcatW7ee3MkBhYSHh5dpjvv7+xunT5+u6nBRjW3fvt2oW7euzTnq4+NjXLhwoarDBKzy8vIq02dtWFiYkZ6eXtXhopor63eDgoctubm5xtixY0vs+/rrrxu5ublP8OwAAKi44ODgR/q/UpJx6dIlm+ORN6p+yDuivMiNwBp+w6E8HvW7jJeXl82xmDvVT1lznS1atDD27t1rcxzmDqyZOXNmmb4HM39QXvnX+AMAmMaPH6/u3btryZIlOnDggJKTk+Xm5iZfX1+NHDlS48aNMy+RCjxp06ZNU6dOnXTkyBElJibq1q1bSktLk4uLi5o2bapf/vKXGjp0qEJDQ1nRiyo1aNAgnTx5UosXL9bOnTt17do1OTs7q23btnrttdf0+9//XrVq1arqMAGr1q9fr4MHD+rIkSNKSkpSSkqK7t69q9q1a8vT01PPP/+8wsPD1b1796oOFag0Dg4OWrt2rcLCwrR69WrFxcUpJSVFjRo1UpcuXTRx4kS99NJLVR0mAABVjrwRKoL5U72QG8HjxG84lBdzp/o5cOCA9u/fr5iYGJ05c0Y3btxQamqqXF1d1bRpU3Xq1EkDBw7UsGHDSvx/ibmDimD+oLy4/SgAAAAAAAAAAAAAAAAAwG44VHUAAAAAAAAAAAAAAAAAAAAUoKgNAAAAAAAAAAAAAAAAAGA3KGoDAAAAAAAAAAAAAAAAANgNitoAAAAAAAAAAAAAAAAAAHaDojYAAAAAAAAAAAAAAAAAgN2gqA0AAAAAAAAAAAAAAAAAYDcoagMAAAAAAAAAAAAAAAAA2A2K2gAAAAAAAAAAAAAAAAAAdoOiNgAAAAAAAAAAAAAAAACA3aCoDQAAAAAAAAAAAAAAAABgNyhqAwAAAAAAAAAAAAAAAADYDYraAAAAAAAAAAAAAAAAAAB2g6I2AAAAAAAAAAAAAAAAAIDdoKgNAAAAAAAAAAAAAAAAAGA3KGoDAAAAAAAAAAAAAAAAANgNitoAAAAAAAAAAAAAAAAAAHaDojYAAAA8VrGxsbJYLLJYLIqNjS22PyIiQhaLRd7e3k88NgAAAAAAAABFkc8DAAD2gKI2AACASpSdna1NmzYpPDxcvr6+cnd3l5OTkxo1aqRf/OIXioyM1P79+5WXl1fVoQIAAAAAAADVHvk8AAAA+0RRGwAAQCX57LPP1L59e40YMUIbNmzQ2bNnlZaWppycHKWmpurEiRNauXKl+vXrJ19fX+3cubOqQ8YjKG2FKgAAAAAAAJ4u5PN+3sjnAQDwdKOoDQAAoBLMnTtXoaGhSkpKkiT17dtXS5cu1YEDB3T8+HHt27dPy5YtU//+/eXg4KDz58/rnXfeqeKo7UN0dLQMw9Dly5erOhQAAAAAAABUE+Tzyo98HgAAeBJqVHUAAAAAT7sPP/xQ06dPlyQ1btxYmzdvVq9evYq169u3ryZNmqSEhARNmTJFqampTzpUAAAAAAAAoNojnwcAAGD/KGoDAACogOTkZEVGRkqSatWqpdjYWPn5+ZXYx9/fX/v27dO///3vJxEiAAAAAAAAgB+RzwMAAHg6cPtRAACACli4cKEyMjIkSbNnzy41AVbAwcFBo0aNsrrv8OHDGj16tLy9veXq6qr69esrMDBQM2bM0K1bt8o0/o4dOzR06FC1aNFCLi4ucnd3V/fu3TVv3jzdv3/fZr/o6GhZLBZZLBZdvnxZDx8+1KJFi9StWzc1atRIFotFs2bNKtInKytLf//73xUQECA3Nze5u7urR48eWrNmjfLy8kqNNSIiQhaLRd7e3lb3F8RTcNy4uDiNGDHCPDcPDw+NHj1aZ86cKfE4SUlJWrBggQYNGiRvb2/VrFlTNWvWlJeXl4YPH67du3db7Xf58mVZLJYiq3V79eplxlXwiI6Ottr/2LFjGj9+vHx8fFS7dm25ubmpffv2mjRpki5cuFDq+wMAAAAAAIDKQz6PfB75PAAAnhIGAAAAyiUvL89o3LixIclwc3Mz7ty5U6HxcnNzjUmTJhmSbD7q1atn7N271+YYWVlZRmhoaIljNG/e3IiPj7faPyoqymwXFxdndOrUqVj/mTNnmu2vX79utG/f3uaxBgwYYOzZs8d8HhMTU+yY4eHhhiTDy8vLakyFj7t06VKjRo0aVo9Vq1Yt4+DBg1bHSEpKKvE9KXiMGjXKyM7OLtL30qVLZeobFRVVpF92drYRGRlZYh8nJydj9erVtv45AQAAAAAAUInI55HPI58HAMDTg9uPAgAAlFNiYqK50vLFF19U3bp1KzTeX/7yFy1fvlyS1KpVK02bNk2dO3dWRkaGtm/frmXLlunOnTsaOHCgjh07poCAgGJjhIeH69NPP5UkBQQEaOrUqfL19VVaWpo2bdqk6OhoJScnq0+fPjp58qQ8PDxsxvP6668rISFBY8aM0fDhw9WsWTNduXJFLi4ukqScnBwNHDhQZ8+elSSFhIQoMjJSnp6eunLlilasWKHdu3crNTW1Qu9LgT179uirr75Sx44dNXnyZPn7+ysrK0uffvqpFi9erMzMTI0ePVoXLlyQs7Nzkb65ublydnZW//791a9fP/n5+alhw4ZKS0vT+fPntXz5cp0+fVofffSRWrdurdmzZ5t9PTw8lJCQoLi4OI0dO1aStG7dOnXp0qXIMVq0aFHs/duwYYMk6aWXXtLIkSPl4+Mji8Wib775RosWLdLp06c1YcIENWvWTIMGDaqU9wkAAAAAAADWkc8jn1cY+TwAAOxcVVfVAQAAPK02btxortCbPn16hcY6efKk4eDgYEgyOnToYKSnpxdr88UXX5htgoKCiu3//PPPzXj69OljPHz4sFib1atXm22GDRtWbH/hlZ2SjLVr19qMecmSJWa7CRMmWG0zduzYIuNVZGWnJOPll1+2el5z5swx22zbtq3Y/vv37xvJyck2zyUvL8+IiIgwV+nevn27WJuYmJgSz6OwTz75xGy7Zs0aq22ysrKM3r17G5IMb2/vYitKAQAAAAAAULnI55HPs4V8HgAA9seh8srjAAAAqpeUlBRzu2nTphUa6/3331deXp4kac2aNapfv36xNgMGDDBXFh47dkxxcXFF9hesCnVyclJUVFSx1Y2SNH78ePXt21eStG3bNn333Xc2Y+rdu7d5PFsxS/nnvnDhQqttFi9erMaNG9sc41G4urraPK+33nrLfP3QoUPF9ru5uemZZ56xObbFYtGCBQvk6OiojIwM7d+/v0Kxzp07V5IUGhqqcePGWW3j6uqqZcuWSZIuX76s2NjYCh0TAAAAAAAAJSOfRz7PFvJ5AADYH4raAAAAyunevXvmtpubW4XGKki6+Pn5qVu3bjbbjR8/vlgfKf/WAQcPHpQk9evXT56enqWOkZOTU2LiZeTIkTb3JScn68yZM5KkYcOGqVatWlbb1a5dW8OGDbM5zqPo16+fmjRpYnVfnTp19Oyzz0qSkpKSSh0rOztb165d05kzZ3Tq1CmdOnVKycnJcnd3lyT997//LXec169f1/HjxyWp1HP39fVVo0aNJElHjhwp9zEBAAAAAABQOvJ55POsIZ8HAIB9oqgNAACgnOrUqWNuZ2RklHuchw8f6sKFC5Kkrl27ltg2MDBQTk5OkqRTp06ZryclJSkzM7NMYxTeX3iMn+rYsaPNfQkJCeZ2ly5dSjxeUFBQifvLqn379iXub9iwoaSiycnCsrOztXz5cnXr1k21a9eWp6en/Pz85O/vbz5u3rwpqeiq3Uf19ddfm9sjRoyQxWIp8VFwrO+//77cxwQAAAAAAEDpyOflI59XFPk8AADsU42qDgAAAOBpVbAiT5Ju3LhR7nHS09PN7dJue+Dk5CR3d3d9//33SktLM18vvF3aGM2aNbPa76caNGhQpphtrbYsazxlZWv1aAEHh/z1Grm5ucX2paWlKSQkxFxxWZqsrKxHD/BHBYm0R1WQxAQAAAAAAMDjQT4vH/m8osjnAQBgnyhqAwAAKKeAgABz+8SJE5UypsViKbWNYRgVHqMsHB0dyxRDaccrLd4nYfLkyWYC7NVXX9XYsWPVsWNHNWnSRK6uruY5tGzZUlevXq1QzIWTcBs3bixxhWxhJSUdAQAAAAAAUHHk88p2PPJ55PMAALAHFLUBAACUk5+fnxo1aqSUlBQdOnRId+/eVd26dR95nMLJj9IuWZ+Tk2Ouxiy4PP9Pt0sbo/D+wv0eReF+pa1qLe9Kx8py9+5dbd68WZL0m9/8Rhs3brTZtvCK1fJyd3c3ty0Wizp06FDhMQEAAAAAAFBx5PPykc8rinweAAD2yaGqAwAAAHhaWSwWRURESJIyMjL0wQcflGscFxcXPfvss5Kkr776qsS28fHxys7OlqQiyZXWrVubl/MvbYxjx46Z2+VN0Pj7+5vbcXFxJbYtbf/jduHCBfM9+/Wvf22z3blz53T//n2b+8u6YjYwMNDc3rt3bxmjBAAAAAAAwONGPi8f+byiyOcBAGCfKGoDAACogClTppjJp3fffVdnz54tU7+8vDx99NFH5vO+fftKkhITE3X06FGb/Qon2gr6SFKNGjUUHBwsSdq3b5+uXr1a6hiOjo7q2bNnmeL9qebNm8vX11eStGXLFmVlZVltl5GRoY8//rhcx6gsOTk55nZmZqbNditXrixxHFdXV3P74cOHNtu1bdtWfn5+kqRNmzbpypUrZQ0VAAAAAAAAjxn5PPJ5P0U+DwAA+0RRGwAAQAV4eHho2bJlkvITPsHBwTp48GCJfRITE9W/f3/Nnz/ffC0yMlIODvlfzSZMmKA7d+4U67d3716tXbtWkhQUFKQuXboU2T9p0iRJUnZ2tsaOHasffvih2Bjr1q0zVxuGhYXpmWeeKeupFhMZGSkp//YHU6dOtdrmD3/4Q5XfrqBt27bmqswNGzZYbfP5559r6dKlJY5T+L26ePFiiW1nzJghSXrw4IGGDBmiW7du2Wz78OFDrVixQg8ePChxTAAAAAAAAFQc+TzyedaQzwMAwP7UqOoAAAAAnna//e1vde3aNb377ru6efOmevbsqZCQEA0ePFi+vr6qX7++0tLSdP78ee3cuVO7d+9Wbm6uAgICzDH8/f01depU/fOf/1RCQoI6d+6sadOmKTAwUJmZmdqxY4eWLFmi3NxcOTs7a9WqVcXieOWVV/Taa69py5Yt2r9/v7p27aqpU6fK19dX6enp2rRpk9atWydJatiwof71r39V6LwjIyMVFRWl+Ph4vf/++7p06ZJ+97vfydPTU1evXtWKFSu0d+9edenSpUpvWeDu7q6XX35ZO3fu1K5duzRgwABNnDhRLVu21M2bN7V161ZFR0erdevWun37ts2EVcuWLdWiRQtdu3ZN8+fPl4eHh9q1a6caNfK/Ujdt2lR16tSRJI0YMUJ79uzR+vXrdfz4cfn5+WnixIkKDg5W48aNlZGRoYsXL+rQoUPatm2b0tLSNGbMmCf2ngAAAAAAAFRn5PPI50nk8wAAsHsGAAAAKsXWrVsNb29vQ1Kpj+eee87Ys2dPkf65ubnGG2+8UWK/evXqFetXWFZWlhEaGlriGM2bNzfi4+Ot9o+KijLbXbp0qdRzvn79utGuXTubxwoJCTH27NljPo+JiSk2Rnh4uCHJ8PLysnqMgr4zZ84sMZbg4GBDkhEcHFxs35UrV4yWLVvajLNly5bG6dOnDS8vL0OSER4ebvUYK1assDlGVFRUkbY5OTnGn//8Z8PR0bHU+eDm5mZkZmaWeH4AAAAAAACoXOTzyOcVRj4PAAD7wu1HAQAAKsmQIUN07tw5bdy4UaNGjVK7du3UoEED1ahRQw0bNlTnzp31xhtv6MCBA0pISFBISEiR/g4ODlq+fLm+/PJLjRw5Ui1btpSLi4vq1q2rTp06afr06bpw4UKxfoW5urpq27Zt2r59u4YMGaLmzZvL2dlZDRo0UNeuXTV37lydO3dOnTp1qpRzbt68ueLj4zVnzhx16NBBNWvWVP369dWtWzetWLFCX3zxhZydnSvlWBXh6empEydO6O2335aPj49cXFxUr149BQQEaObMmfrmm2/k5+dX6jiRkZHaunWrQkJC1KRJE3NVpzWOjo567733lJiYqKlTpyowMFANGjSQo6Oj6tSpo+eee04jR47U+vXr9d1336lmzZqVecoAAAAAAAAoBfk88nmFkc8DAMC+WAzDMKo6CAAAAAAAAAAAAAAAAAAAJIkrtQEAAAAAAAAAAAAAAAAA7AZFbQAAAAAAAAAAAAAAAAAAu0FRGwAAAAAAAAAAAAAAAADAblDUBgAAAAAAAAAAAAAAAACwGxS1AQAAAAAAAAAAAAAAAADsBkVtAAAAAAAAAAAAAAAAAAC7QVEbAAAAAAAAAAAAAAAAAMBuUNQGAAAAAAAAAAAAAAAAALAbFLUBAAAAAAAAAAAAAAAAAOwGRW0AAAAAAAAAAAAAAAAAALtBURsAAAAAAAAAAAAAAAAAwG5Q1AYAAAAAAAAAAAAAAAAAsBsUtQEAAAAAAAAAAAAAAAAA7AZFbQAAAAAAAAAAAAAAAAAAu0FRGwAAAAAAAAAAAAAAAADAblDUBgAAAAAAAAAAAAAAAACwGxS1AQAAAAAAAAAAAAAAAADsBkVtAAAAAAAAAAAAAAAAAAC7QVEbAAAAAAAAAAAAAAAAAMBuUNQGAAAAAAAAAAAAAAAAALAbFLUBAAAAAAAAAAAAAAAAAOwGRW0AAAAAAAAAAAAAAAAAALtBURsAAAAAAAAAAAAAAAAAwG5Q1AYAAAAAAAAAAAAAAAAAsBsUtQEAAAAAAAAAAAAAAAAA7Mb/A1a9rRb4zMxBAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 1500x500 with 2 Axes>"
      ]
     },
     "metadata": {
      "image/png": {
       "height": 469,
       "width": 1242
      }
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "plot_sdf_files(sdf_file1, sdf_file2)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "774d9985-28c4-4e31-8d91-5c6fd01ab346",
   "metadata": {},
   "source": [
    "-------------\n",
    "### Scoring and Ranking of Ligand Positions Using SMINA\n",
    "While the Equibind model is able to generate predicted positions for any molecule it is presented with, it lacks the ability to rank or score these positions in terms of their binding affinity to the target protein. To overcome this limitation, I turned to the discussions by the author on the Github repository's issue section, where the use of SMINA was suggested to score a series of molecules.\n",
    "\n",
    "SMINA is a freely available software for molecular docking and scoring, designed for high-performance energy minimization and scoring of protein-ligand complexes. SMINA has been widely used in the field of drug discovery and virtual screening, offering a customizable scoring function and an efficient search algorithm.\n",
    "\n",
    "Given the time-consuming nature of the scoring process, I opted to use a random sample of 1000 molecules from the model output file. This allowed me to obtain a representative sampling of the data while keeping computational time manageable. To streamline the scoring process, I created custom Python functions that interact with the SMINA binary file installed via mamba. These functions handle the necessary file input/output operations and perform the actual docking and scoring tasks using the SMINA executable."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "2e8f9594-a9a0-4ce8-aaa4-370288dfea4f",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "==============================\n",
      "*** Open Babel Warning  in Init\n",
      "  Unable to open data file 'space-groups.txt'\n",
      "==============================\n",
      "*** Open Babel Warning  in Init\n",
      "  Cannot initialize database 'space-groups.txt' which may cause further errors.\n",
      "1 molecule converted\n"
     ]
    }
   ],
   "source": [
    "# Convert the RAS protein to the required pdbqt file\n",
    "!obabel data/ras/ras_protein.pdb -O data/ras/ras_protein.pdbqt -xr\n",
    "\n",
    "# Create directories used in my functions\n",
    "!mkdir -p data/temp data/ligs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "bc999364-6a44-4957-8ad9-33f28acba8c4",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[13:50:55] WARNING: not removing hydrogen atom with neighbor that has non-tetrahedral stereochemistry\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Molecule index</th>\n",
       "      <th>Score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>282</td>\n",
       "      <td>-4.7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>844</td>\n",
       "      <td>-4.6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>671</td>\n",
       "      <td>-4.5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>27</td>\n",
       "      <td>-4.4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>253</td>\n",
       "      <td>-4.4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>425</td>\n",
       "      <td>-4.4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>446</td>\n",
       "      <td>-4.4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>505</td>\n",
       "      <td>-4.4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>519</td>\n",
       "      <td>-4.4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>745</td>\n",
       "      <td>-4.4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>214</td>\n",
       "      <td>-4.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>309</td>\n",
       "      <td>-4.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>786</td>\n",
       "      <td>-4.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>841</td>\n",
       "      <td>-4.3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>259</td>\n",
       "      <td>-4.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>311</td>\n",
       "      <td>-4.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>993</td>\n",
       "      <td>-4.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>996</td>\n",
       "      <td>-4.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>1</td>\n",
       "      <td>-4.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>67</td>\n",
       "      <td>-4.1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    Molecule index  Score\n",
       "0              282   -4.7\n",
       "1              844   -4.6\n",
       "2              671   -4.5\n",
       "3               27   -4.4\n",
       "4              253   -4.4\n",
       "5              425   -4.4\n",
       "6              446   -4.4\n",
       "7              505   -4.4\n",
       "8              519   -4.4\n",
       "9              745   -4.4\n",
       "10             214   -4.3\n",
       "11             309   -4.3\n",
       "12             786   -4.3\n",
       "13             841   -4.3\n",
       "14             259   -4.2\n",
       "15             311   -4.2\n",
       "16             993   -4.2\n",
       "17             996   -4.2\n",
       "18               1   -4.1\n",
       "19              67   -4.1"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def extract_best_score(output):\n",
    "    \"\"\"\n",
    "    Extracts the best docking score from the output of the smina program.\n",
    "\n",
    "    Args:\n",
    "        output (str): The output of the smina program as a string.\n",
    "\n",
    "    Returns:\n",
    "        float or None: The best docking score as a float if found in the output, or None if not found.\n",
    "    \"\"\"\n",
    "    pattern = re.compile(r\"^\\s*1\\s+(-?\\d+\\.\\d+)\", re.MULTILINE)\n",
    "    match = pattern.search(output)\n",
    "    if match:\n",
    "        return float(match.group(1))\n",
    "    return None\n",
    "\n",
    "def perform_docking(i, mol):\n",
    "    \"\"\"\n",
    "    Performs docking of a ligand molecule to a protein receptor using the smina program.\n",
    "\n",
    "    Args:\n",
    "        i (int): The index of the ligand molecule.\n",
    "        mol (rdkit.Chem.rdchem.Mol): The ligand molecule as an RDKit Mol object.\n",
    "\n",
    "    Returns:\n",
    "        tuple or None: A tuple containing the index of the ligand molecule and its best docking score,\n",
    "        or None if the best docking score was not found in the output.\n",
    "    \"\"\"\n",
    "    mol_file = f\"data/ligs/ligand_{i}.sdf\"\n",
    "    writer = SDWriter(mol_file)\n",
    "    writer.write(mol)\n",
    "    writer.close()\n",
    "\n",
    "    output_file = f\"data/temp/docking_results_{i}.sdf\"\n",
    "\n",
    "    cmd = f\"smina -r {receptor_file} -l {mol_file} --center_x {center_x} --center_y {center_y} --center_z {center_z} --size_x {size_x} --size_y {size_y} --size_z {size_z} -o {output_file}\"\n",
    "    result = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True, text=True)\n",
    "\n",
    "    best_score = extract_best_score(result.stdout)\n",
    "    if best_score is not None:\n",
    "        return (i, best_score)\n",
    "    return None\n",
    "\n",
    "# Create an SDMolSupplier object to read molecules from an SDF file\n",
    "sdf_supplier = SDMolSupplier(sdf_file2)\n",
    "\n",
    "# Define the receptor file and docking box parameters\n",
    "receptor_file = \"data/ras/ras_protein.pdbqt\"\n",
    "center_x, center_y, center_z = 0, 0, 0\n",
    "size_x, size_y, size_z = 30, 30, 30\n",
    "\n",
    "# Read all molecules from the SDF file and sample 1000 random molecules\n",
    "molecules = list(sdf_supplier)\n",
    "random.seed(123)\n",
    "sampled_molecules = random.sample(molecules, 1000)\n",
    "\n",
    "# Perform docking calculations for the sampled molecules in parallel using a thread pool\n",
    "scores = []\n",
    "with ThreadPoolExecutor(max_workers=NUM_THREADS) as executor:\n",
    "    futures = [executor.submit(perform_docking, i, mol) for i, mol in enumerate(sampled_molecules)]\n",
    "    for future in futures:\n",
    "        score = future.result()\n",
    "        if score is not None:\n",
    "            scores.append(score)\n",
    "\n",
    "# Sort the docking scores and select the top 20\n",
    "sorted_scores = sorted(scores, key=lambda x: x[1])[:20]\n",
    "\n",
    "# Convert the sorted_scores list to a pandas DataFrame\n",
    "score_df = pd.DataFrame(sorted_scores, columns=['Molecule index', 'Score'])\n",
    "docking_scores = [score for _, score in scores]\n",
    "\n",
    "# Display the DataFrame\n",
    "score_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "fde95d7b-5930-488c-b6f8-d2bd3f0db4e9",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABHcAAAOMCAYAAADHXV9PAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjUuMywgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/NK7nSAAAACXBIWXMAAB7CAAAewgFu0HU+AACeX0lEQVR4nOzdeVhV5f7+8XsrgijihDPklAN2smOCc6KW5oCZNhxtckrLTiaW2nDKodLUTEm/lYcc65TZ4HDSNHOeU9Q0C1RQExDDCVMREFm/P/yxDlumzbRh4ft1XVzXYq9nPeuz9wZr3zyDzTAMQwAAAAAAALCkUkVdAAAAAAAAAPKOcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcA4FT16tWTzWaTzWbTyZMni7ocIN+io6M1btw4tWjRQhUrVlSpUqXMn/HNmzcXdXmWc/LkSfP1q1evXr7769SpE+8HAKDEI9wBAJjSfwiaOHFirq6dOHGieW2nTp0KpT6guAkNDVXz5s31/vvv65dfftFff/0lwzDy1Wf636Vbv1xcXFS1alU1aNBALVu21KBBgzRnzhzt37+/gJ4RSqKjR4/qzTffVJcuXVS7dm25u7vL1dVVlStX1l133aXevXtrwoQJWrt2ra5du1bU5QIA8sClqAsAAKAwbN68WZ07d5YkBQQE8Bd7FDjDMPT000/r4sWLkqRKlSqpS5cuqlGjhkqVuvn3szp16hToPW/cuKELFy7owoULOnHihPbv36/FixdLkpo3b67nn39ew4YNk4sL/4sH6dKlSxo9erQWLlyY6fn4+HjFx8fr999/16pVqyRJZcuW1YEDB9S0aVNnlgoAyCf+yw8AAJAHP//8s8LDwyVJ1apV0++//y4vL68CvUft2rXVt29fu8cuX76s+Ph4nTx5UocPH1Zqaqok6dChQ3rhhRe0cOFCff7552rSpEmB1gJruXTpku6//37t27fPfMzd3V3+/v6qX7++ypYtq0uXLikiIkK//vqrkpKSJEmJiYlKTEwsqrIBAHlEuAMAcCrW2UFJkX4qVJ8+fQo82JGkRo0a6f/+7/+yPH/58mWtWrVKM2fOVGhoqCRp7969atOmjXbv3k3AI922o/ZeeeUVM9hxcXHR22+/rZdeeknly5fP0DYpKUk//fSTli5dqm+++cbZpQIACgBr7gAAAORB2nQsSapVq1aR1FChQgUNGDBAe/bs0ZQpU1S6dGlJN6fbBAYGKj4+vkjqQtGKi4uzm4oVEhKi119/PdNgR5Lc3NwUGBiozz//XFFRUWrQoIGzSgUAFBDCHQAAgDy4fv26eZy2xk5Rsdlsev311/Xee++Zj0VERCg4OLjoikKRWb9+vTldr2bNmho0aJDD11arVk2enp6FVBkAoLAQ7gAAnMrRrdCvXLmiuXPnqlevXrrjjjtUrlw5lS1bVt7e3vr73/+uvn376uOPP1ZERITddWk7DaUtpixJW7ZsyXTnoey2Wb5+/boWLlyohx9+WHXr1pW7u7s8PT3VpEkTDR06VD/99FOunrdhGFqyZIl69OihWrVqqWzZsqpbt64CAwP17bffmjssObJtc2ZtYmNjNWXKFLVq1Uo1a9ZU6dKlValSpQzX/vHHH/rkk080YMAA/e1vf1PFihVVpkwZVa1aVc2bN9eIESO0e/duh57ToEGDzDoWLVokSbpw4YKmT58uf39/eXl5yc3NTY0bN9Yrr7yic+fOZejj1KlTevXVV3XPPfeoYsWK8vT0lL+/v4KDg+3Ck4J07tw5TZ06VQEBAapVq5bc3Nzk5eWlFi1aaOzYsfr999+zvHbRokXmc540aZL5+KRJkzL8fKW9Js40ZswYtW/f3vx+9uzZunz5co7X7d69Wy+++KLuuusuVa5c2fxd6969u/7v//5PV69ezXUt27dv16hRo9SiRQtVr15dZcqUkaenp+6++24NHDhQS5YsyffOTNeuXdNDDz1kvuY1a9bUL7/8YtfGkd+pzH6Wr169qo8//lgdOnRQjRo15ObmJh8fHw0YMEA7duzIVZ2HDx/WiBEj1KhRI5UrV05eXl7y8/PTe++9Z/5epP/Zyk0Yk5no6GjzuG7durLZbPnqLysF9R7/+OOPGjJkiBo3bixPT0+5u7urbt266tu3rxYuXOjQvwWZvYfx8fH68MMP1bFjR9WpU0cuLi6y2WyZjmgzDEPLly/XwIED1bhxY1WsWFFly5aVj4+PHn74YS1evFgpKSkOvS7h4eEaN26c2rRpIy8vL7m6uqpixYq688471aZNG73wwgtasWKF/vrrL4f6AwCHGAAA/H8BAQGGJEOSMWHChFxdO2HCBPPagICALNvVrVvXbHfixIlM2+zcudOoU6eO2S6nr+vXr2daR05fdevWzfT+u3fvNho2bJjj9V27djXi4uJyfG0uXLhgdOrUKdu+HnroIePSpUt278GmTZsy7e/WNitWrDAqV66coc+KFSvaXTdmzBjDZrM59NoMGDDAuHr1arbPa+DAgWb7hQsXGnv37jV8fHyy7NPHx8fuPQ8JCTHc3NyybN++fXvjypUrOb6+uTF//nyjYsWK2T730qVLG0FBQUZKSkqG6xcuXOjwz9fChQvzVKOjv0tZ+e677+zqWLZsWZZtr1y5YvzjH//I8bnUqlXL+OGHHxy6f1RUlNG1a1eHXqPWrVtnuP7EiRM5/o4ahmHEx8cb9913n9m2fv36xrFjxzK0c+R36taf5d9//93w9fXNtvbx48c79HpMmzbNKFOmTLav7Y4dO+x+tgYOHOhQ31mZPn262Vf16tXz1Vdm8vsep/nzzz+N+++/P8c+GjVqZOzduzfbmm59D7dv357lv0cXL160u/bgwYPG3//+9xzraNKkifHbb79lW8eECRMMFxcXh16bJ5980uHXHABywoLKAIBiJSoqSg8++KA52qBMmTLy9/fXnXfeqXLlyunq1as6efKkDh48mOlfPVu1aqV//vOfiomJ0YoVKyRlvuOQJFWtWjXDY1u3blWPHj2UkJBg1+ddd92l5ORk7d69W5GRkZKkn376SR06dND27dtVrVq1TJ9PYmKiHnzwQe3du9d8zMfHRx06dFC5cuV05MgR7dy5U//97381ePBgx1+o/2/nzp2aOHGirl+/rqpVq6pjx47y8vJSXFycDhw4YNc2KipKhmHIZrOpSZMmatKkiapWraoyZcro/PnzOnDggPnclixZokuXLmnVqlUO/dU/KipKY8eO1blz51SjRg117NhRlSpV0rFjx7R161alpqYqKipKPXr00OHDh7VkyRINHz5cktS0aVP5+fnJ1dVVoaGhOnTokCRpx44dCgoK0qeffprr1yUzM2bM0NixY83v3dzcFBAQoDvuuEMXL17Upk2bdOHCBd24cUPBwcH6448/9N1339k9f19fX/3zn/+UJO3Zs8d8X/39/dWqVSu7+/n6+hZI3bnVu3dvlS1b1tzxaOvWrZn+/CckJKhLly7as2eP+Vjt2rV13333ycPDQxEREdq+fbtu3Lih2NhYPfTQQ1qyZIkeffTRLO/922+/qWvXroqNjTUfq169utq1a6dq1aopMTFRkZGROnDggK5du5bnXZliY2PVvXt382fl7rvv1o8//lggax+dPn1aDzzwgE6fPq1KlSrpvvvuU82aNXXu3Dlt3LhRly5dkiS9/fbbatasmf7xj39k2dfMmTP16quvmt+XLVtWnTt3lre3t86dO6dNmzYpNjZWgYGBCgoKynftae68807zOC4uTp999pmeeeaZAum7oN7jP//8U+3btzf/zZGkhg0bqnXr1nJzc9Pvv/+un3/+WZJ07Ngxde7cWWvXrrUbmZaViIgIBQUF6dKlS6pQoYI6duyo2rVr6+LFi9q6datd261bt6p3797mf09cXFzk5+enJk2aqEyZMjp58qS2b9+uxMREHTlyRO3atdOuXbsy/f0ODg62G9Hn5eWlNm3aqFatWrLZbLpw4YLCw8MVFhamGzdu5Pg8ACBXijpdAgAUH8Vh5M6oUaPM8/fdd58RExOTaT/Xr183Nm/ebDz55JOZjrDYtGlTrkc/XLhwwW7EUMOGDTP9a/F//vMfw93d3WzXu3fvLPt8/fXX7UaEzJkzx0hNTbVrc/jwYXOUQPqRLI6M3HFxcTFsNpvxzjvvGMnJyXbtEhMT7b6fPn26sXDhQuPs2bNZ1rt161bjzjvvNPv//PPPs2yb/i/lrq6uhs1mM9599127kVSGYRjbt283KlSoYLadPHmy4eHhYVSsWNH473//m6Hf2bNnm21tNptx/PjxLGtw1M6dO43SpUub/Xbv3t2IjY21a5OYmGiMHTvW7i/rH3zwQZZ9pv+Zz+3vS3byO3LHMAyjbdu2Zh9t27bNtM2IESPsfjZnzpxp3Lhxw67N0aNHjZYtW5rtPD09s3w/Ll26ZDRq1Mhs6+XlZSxZsiTDz7th3Bwx9MUXXxiDBw/OcC6nkTsRERFG/fr1zTbt27fPMBIjvdyO3En7HRw3blyG0Wvnz583unTpYrZt0KBBps/PMG7+Xru6uppte/XqlWGk39WrV43nn38+w+9+fkfuXLp0yfD09DT7K1OmjBEUFGQcPnw43/0WxHtsGIbRo0cPs59y5coZX3zxRYY2e/fuNRo0aGC28/HxyfK9Tv8epo2c+ec//2lcvnzZrl1ycrL5cx4bG2vUqFHDvG7AgAFGdHR0hr7PnDlj9O3b12x39913Z/jvzvXr142qVauabd57770M/yanOX/+vLFgwQJj2rRpmZ4HgLwg3AEAmNJ/CPL39zf++c9/Ovzl7+9fIOFO+g+SmU2xcFRewp3x48eb11SqVMn4448/smy7bNkyuxBgy5YtGdqcP3/e7gNbdkFBdHS0UalSJbs+HQl3JBnvvvuuQ8/PUSdOnDDKli1rSDJatWqVZbv0H6YkGRMnTsyy7XvvvWfXtlSpUsbmzZuzbN+tWzez7dSpU/P1fAzDMDp27Gj216ZNGyMpKSnLti+99JJdmPHXX39l2q44hzuDBg2yCyBuFRERYZQqVcpsM3v27Cz7unDhglGvXj2zbVYf1v/1r3+ZbSpWrGiEh4fnqfbswp0DBw7YfRjv2bOnkZCQkG1/uQ13JBmvv/56lv2dOXPGKF++vNl29+7dmbZ79NFHzTZ+fn7Z/sw99thjdvfPb7hjGIYxa9Ysuz7TvmrVqmU8/PDDxrvvvmusW7cuQ/iRnYJ6jzdu3GhX08qVK7Nse+LECbuplJMmTcq03a3v4bPPPptjHUOGDHG4fUpKitG5c2ez/VdffWV3/tdff7ULHAHA2Qh3AACmW0ODvH7lJ9xJ/1fh+Pj4PD+X3IY7qampRs2aNR0KYtKk/8tz//79M5yfM2eO3QfszEYYpTd58uRchzt16tTJMFKmIKQ9N5vNZly6dCnTNuk/TNWoUSPbD6/Hjx+3e279+vXL9v4LFiww2z7yyCP5ei6///673b337duXbfsrV64YXl5eZvu5c+dm2q44hzvpR8BVrlw5w/lXX33VPN+8efMsR5+kWbp0qdnezc0tw+9mYmKiXTj5ySef5Kluw8g63Nm8ebPdaJSnnnrKoZ/93IY71apVM65du5Ztn48//rjZfs6cORnOnz9/3m6dnY0bN2bb36lTp+zCtoIIdwzDMKZMmWI3Yi2zLxcXF6NLly7G559/nu2/UQX5Hqdf5ym7kY9ppk2bZhdOZfbzmv49LFu2rHHhwoVs+4yLizPD94oVKzoUcu3atSvLunfs2GGee/jhh3PsCwAKGrtlAQCKlTvuuMM8/uijj5x237CwMJ05c0bSzW2tBw4cmOM1zz77rHmc2S486R97/PHHVbp06Wz7e/LJJx0rNp1HHnlELi65X0Lv1KlT+vbbbzVlyhSNGzdOI0eO1Isvvmh+nThxQpJkGIYOHjyYY3+9e/eWq6trlufr16+v8uXL29Wdnb/97W/mcVotebVp0ybz+J577tG9996bbfvy5ctrwIABmV5vFR4eHuZxZrtlbdy40TwePHhwjusq9e3bV1WqVJEkJSUladeuXXbnd+/ebe5AVKFCBYd+f3JjxYoV6t69u7kuyqhRo/TZZ5/l6Wc/J2lrFmWnRYsW5nFmu/7t3LnT3OGpVq1a6tSpU7b9+fj4qGPHjrmuNSevv/66fvvtNw0ePFgVKlTItE1KSoo2btyop59+Ws2bN9evv/6aabuCfI/T/04NGTIkx/aDBw9WqVI3P7bExsbqyJEj2bbv1q2bKleunG2b9evXKykpSZIUGBho9zuTldatW6tcuXKSbu4Ull76/3Zt3LhRYWFhOfYHAAWJBZUBAJmaMGGCJk6c6HD7iRMn2i0kmVf/+Mc/tGHDBknSv/71L61bt05PPvmkunbtmu3W5fmVfvHhpk2bZrrY8q3SL+x55swZnT59WrVr1zYfS78l862L7Wambt26qlatms6ePetg1VLLli0dbitJu3bt0muvvaZt27aZ26/nJLMtzG9111135dimUqVK5pbazZo1y7Zt+g9m+d0uOP1768hirGnt5syZI0nav39/vu5fFNIHOp6ennbnDMOw+9l05DUpU6aMWrVqpbVr10q6+Zp0797dPL97927zuE2bNnJ3d89r6RnMnz9fzz33nLkA7TvvvKM333yzwPq/1d13351jm/T/PqQtsJxe+tfX39/foUXJ/f39s9yqPT+aNGmiBQsW6OOPP9bOnTu1fft2hYaGav/+/YqJibFr+/vvv6tdu3basWOHmjdvbneuoN7jmJgYxcXFmd+3a9cux2uqVaumxo0bKzw8XNLNn7+mTZtm2d6RfxfTB5RHjx7Viy++mOM1ksz38uLFi7p69aoZWnt7e6tdu3bauXOn/vrrL/n5+enJJ59U37591aFDhyzDNQAoKIQ7AIBiZejQoVq3bp2+/fZbSdKWLVu0ZcsWSf/byadLly7q27dvljtU5UX6QKVu3boOXVOjRg27XYnOnTtnF+6kD0W8vb0d6rNOnTq5Cndy8xosWLBAzz77rMOhTprMRn7cqmLFijm2ST/KIqf26dumjYDIq7y8t+mDREfCreImfeCQNuIm/bn0r2lBvCZ//vmnedygQYPclJqtmJgYuxFyn3zyiZ5//vkC6z8zjvwslylTxjzO7Oczr7/7hals2bLq0qWLunTpYj52/PhxffvttwoODjZ3v7py5Yqefvpp/fLLL3ahVEG9x+l/H93d3VW9enWHrqtXr54Z7uT0O+nIv4unT582j/fu3Wu3o6GjLl68aDciccGCBercubNiY2OVkJCgTz/9VJ9++qlKly6tu+++Wx07dlSPHj30wAMPFMqoMwC3N6ZlAQCKlVKlSunrr7/WokWLMvzl+PTp01q6dKmee+451a5dW88++6wuXLhQIPe9cuWKeZz+f9Zzkr7trSFI+j7ThvLnpj9HOPrX87CwMD333HNmsNO8eXPNnj1be/fuVVxcnK5duybj5lp8MgzDbspFampqjv07MjIhP+3zIy/vbXbvqxWkfQiWpJo1a9qdS/96SAXzmqT/3pHpLY4qVaqUOR1HujmypLAVxM+mM373C0KDBg00btw4hYWF2U0LO3ToUIZRRAX1HhfGv7W3cuTfxcxGXOVWSkqK3fdNmjTRwYMHNXr0aLtQ9caNG/rll180e/Zs9ejRQ3Xr1tWnn36a7/sDQHqEOwCAYsdms2ngwIE6ePCgIiIiNH/+fA0cONDur8UpKSmaP3++WrVqlauRLllJ/2ElbeqQI9K3vXXYffoPIwkJCbnuryDNmjXL/CDSs2dPhYaGauTIkfLz81O1atUyrDFixUAjK3l5b7N7X4u75ORku2lBbdq0sTt/6wfzgnhN0n9/a3iUH7Vq1dJnn31mBjxz5szRqFGjCqz/wlKcfvcdUbFiRf3nP/+xWxds27Ztdm0K6j0ujH9r8yL9exQcHGwXbjv6ldlU4WrVqmnmzJk6c+aMtm3bpsmTJ6tHjx520yNPnz6t4cOH66WXXsr38wCANIQ7AIBirWHDhhoyZIgWLVqkyMhIHT16VGPGjDGHtEdGRhbIWj/ph/GfOnXKoWvi4uLMKVmS5OXlZXc+/ffR0dEO9XnrGhgFJW0dI+nmmiXpp5Vk5o8//iiUOopCXt7b9M//1ve1uFu5cqW5UKwk3XfffXbnK1asaPf+F8RrUqNGDfM4vwtg3+rJJ5+0C3hmz56toKCgAr1HQStOv/uO8vHxsVs7K22aVpqCeo/T/z5eu3bN4WmPBf07mf75HDt2LN/93apMmTLq0KGD3njjDf3www86d+6c1q5dq4CAALPNnDlz8jQdDAAyQ7gDALCURo0a6f3339fbb79tPvbf//43Q7vcTq1Iv/tNeHi4Q9O9duzYYR7XrFnTbr0dSfr73/9uHu/ZsyfH/v74448CGYWUmfTrS+S0+PGlS5d06NChQqmjKKR/b3fu3OnQNenf25x21ypODMPQrFmzzO+9vLx0//3327Wx2Wx2P5uOvCYpKSl2P8O3vibpRwft2rVL165dy23p2XryySe1ePFiM+D58MMPNXr06AK9R0FK//qGhoY6tM5VcfiQn34En5ubm925gnqP69SpY7fOjiM/f+fPn7fbIasgfidbt25tHv/444/57i8nZcqU0YMPPqh169bZLdr9/fffF/q9AdweCHcAAJYUGBhoHqdf6DNN+g8pjizI6+vra65NcuPGDf3nP//J8ZqFCxeax507d85wPv32x19//bW5209WvvjiixzvmVfp1y3JaZrIvHnz8r2IcXGSfgHZAwcO5Li1+7Vr1/TVV19len1xN336dLtdgIKCgjJd1yT9c1q8eHGO4cN///tfnT9/XtLN3622bdvanW/Tpo25w9nly5f12Wef5fk5ZOWpp56yC3iCg4OLbcDTrl07c3TU6dOnc9wFKyoqKsM0KGdLSkqyW6sp/dbeUsG+x+n/vVy0aFGO7RcvXmyu/VW7dm01adIkz/dO8+CDD5ojQCMiIrRq1ap89+kIV1dXde3a1fw+s/9+AUBeEO4AAIoVR4fop59KktnOKOm3KnZkuoPNZtPw4cPN799+++1sr1u9erXdX1wz28HniSeeMP/6ffz4cc2ePTvL/mJiYvT+++/nWGdepV+vaOXKlVm2O3bsWIFMcytOmjZtardY7MiRI7MNr9566y1zq2ZPT0898cQThV5jfhmGoWnTpulf//qX+VjTpk2zXNNj2LBhZkiyf/9+hYSEZNn3pUuXNG7cOPP7AQMGZNhRys3NTS+88IL5/auvvmo30qKgPPXUU1q0aJFdwPPyyy8X+H3yq2rVqnrooYfM78eNG6fk5OQs248ZMybH8Dc3vv/++1yHtFOnTtVff/1lfp9+q3upYN/j5557zjxevny51q5dm2XbqKgovfvuu3bXFsSi13Xq1NFTTz1lfv/88887PDUuNTU1wyjLixcvOrT4vJTzf78AIC8IdwAAxcodd9yh4cOHa/PmzVl+2Nm9e7defPFF8/uePXtmaNOgQQNzxMIff/zh0LSooKAgczvi8+fP6/7777dbmDbN0qVL9Y9//MP8vnfv3nbhQZqqVavaLf46duxYffTRRxlGSfz222/q2rWr4uPjM0yFKCjpRzq9/PLLmU5D2LBhgzp16qTLly8Xyc49hWnq1KnmYrHbtm3TI488YgY4aZKTk/Wvf/1LH3zwgfnYhAkTCnT3p4J25coVffXVV2rdurVee+0183ematWqWrVqVZYLzzZs2NDuA/aLL76ojz76KMOH08jISHXr1k2RkZGSboZdb731VqZ9jhs3Tg0bNpR0MxDq0KGDvvrqq0xHBSUkJGjJkiUaMmRIrp/z008/bRfwzJo1q1gGPBMmTDBH74SGhqpfv34ZAoGEhASNGDFCX3/9dYH+7sfExGjYsGFq0KCB3nzzzWx3GYuLi9OoUaM0ceJE87GHHnpIzZo1y9C2oN7jzp07q0ePHub3jz32mL755psM7Q4cOKD7779fFy9elHRzXaCCXIR4ypQpqlWrlqSbr5m/v7++/fbbLEOamJgYffjhh2ratKmWLl1qd27lypXmtOGs1iRKTExUcHCwvvvuO/OxzP77BQB54VLUBQAAkN61a9f06aef6tNPP1WFChX097//XXfccYfKly+vc+fOKTw83O6DSrVq1ew+lKQpVaqUHn74YXOqU+fOndW9e3fdcccd5of8KlWq6I033jCvqVy5sr788kv16NFDCQkJOnLkiO699161bt1azZo1U3Jysn7++We7xTcbNWqk+fPnZ/l8Jk6cqA0bNmjfvn26ceOGXnzxRU2fPl3t27dXuXLldPToUe3YsUOpqanq16+fzp8/ry1btpjPoaCMHj1a8+fP19mzZ3Xx4kV1795d9957r5o1ayabzab9+/frt99+k3RzukL16tX1+eefF9j9i1rbtm01depUjR07VtLNkQ133HGHOnfuLB8fH128eFGbN2+2GznWt2/fIp/2c+zYMbsgU7oZ6MTHx+vkyZM6fPhwhhC0bdu2+vzzz80P4VmZMWOGQkNDtXfvXqWkpOjFF1/U1KlT1aFDB3l4eCgyMlJbt241+3dxcdH8+fNVv379TPvz9PTUsmXL1LVrV8XFxencuXMaMGCAgoKC1K5dO1WrVk2JiYmKjIzU/v37de3aNd1zzz15el2efvppGYahwYMHKzU1VbNmzZLNZrML5ora3XffrcmTJ5ujnlavXq26deuqc+fOqlOnjs6fP6+NGzcqPj5elSpV0ujRozVhwgRJBfe7Hx0drcmTJ2vy5MmqWrWq7r33XtWoUUMeHh66fPmyjh49qv3799v9DDVu3Fhz587NtL+CfI8XLlyo9u3bKzIyUleuXNHjjz+uRo0aqXXr1nJ1dVVYWJh2795tBkfly5fXkiVLVKlSpQJ5baSbu7GtXLlSPXv21Llz5xQbG6vHHntM1atXV+vWrVWjRg2lpqbq/PnzOnz4sI4fP57tFMbjx49r3LhxGjdunO644w41b95c1atXl2EYOnPmjHbv3m0GVdLNtaTatWtXYM8HwG3OAADg/wsICDAkGZKMCRMm5OraCRMmmNcGBARk2a5u3bpmuxMnTmQ47+HhYZ7P6euee+4xwsLCsrzXqVOnjNq1a2d5fd26dTO9bteuXUaDBg1yvP8DDzxgxMXF5fjanD9/3ujYsWO2ffXu3du4dOmS0a5dO/Ox/fv3Z9pf+vdp06ZNOd4/zc6dOw0vL69s63j44YeN+Ph4Y+DAgeZjCxcuzLQ/R9qkl9N7n96JEydyfJ/yYt68eYanp2e2r0Hp0qWNUaNGGSkpKdn2lf5nPre/L472m5uvFi1aGP/+979zrDu9y5cvG48//niOfdeqVcv44YcfHOrz5MmTOf68p321b98+w/W5ee8XL15slCpVymz/yiuvZGjjyO9Lbn+WFy5caLYfOHBgtm2nTJliuLi4ZPva7tixwwgJCTEfe+mll3KsITs7duww/P39DZvNlqufoaeeesqhf9Py+x6nOXPmjNGlS5cc+7jzzjuNPXv2ZFtTbt/DW5/P/fff7/DrVKNGDWPt2rV2fXzzzTcOv96lSpUyXnjhBSM5OTlXdQJAdhi5AwAoVs6fP6+tW7dqy5Yt2rt3r44dO6Y///xTiYmJKleunLy9vdWyZUs98sgjeuihh7L9C7ePj48OHjyoOXPmaN26dTpy5IguX76slJSUbGto06aNwsLC9J///EcrVqzQL7/8ori4OJUpU0Y1a9ZUhw4dNGDAAHXr1s2h51SlShVt3rxZX375pT7//HMdOHBA8fHxqlGjhpo3b65BgwbpkUcekc1ms9ulqyD/Qi3dHNHx22+/KTg4WN9//72OHz8u6eZfr1u2bKmnnnpKvXv3LtB7FjdDhw5Vnz599Omnn2rNmjU6evSoLly4oAoVKsjHx0cPPPCAhgwZkumUlOKgdOnS8vT0lKenp6pUqaK7775bfn5+uu++++x2aHKUh4eHli5dqqCgIH3++efavHmzTp8+rWvXrsnLy0t/+9vfFBgYqCFDhjg8Va9u3brasmWLNmzYoG+++Ubbtm1TbGys/vrrL5UvX15169ZVy5Yt1atXL7t1afLimWeekSRzBM8HH3wgm81WqOtX5dbrr7+uwMBA/d///Z/Wr1+v06dPq1y5cqpXr54eeeQRDR8+XF5eXnYLKuf3d79du3bas2ePYmJitGnTJu3YsUO//fabjh8/rosXLyopKUkeHh6qWrWqmjVrprZt26p///52a3Nlp6De4xo1amjDhg368ccf9dVXX2n79u06c+aMrl+/rurVq6tFixZ6+OGH9dRTT5lT3ApD3bp1tX79eu3atUvffPONtm7dqqioKF28eFEuLi6qWrWqGjVqJD8/P3Xr1k2dOnUyF2NO8+ijjyo2Nlbr1q3Tjh07dPDgQR0/flzx8fGSpIoVK6px48bq0KGDnnnmmWL7bwwA67IZhgN7MwIAgEJ37do1eXp6KiUlReXKldNff/1lTiEDULI9+eST+vLLLyVJS5YsUf/+/Yu4IgCAlbCgMgAAxcSyZcvMUUX33nsvwQ5wm7h69ap++OEH83t/f/8irAYAYEWEOwAAFAPx8fF2uxANGDCgCKsB4ExvvvmmOX3H398/x8WwAQC4FeEOAACF7IUXXtCiRYt05cqVTM///PPPuu+++8ztc2vVqqWnnnrKmSUCKATffvutxo4dq4iIiEzPnzt3Ti+88IKCg4PNx9J2dAMAIDdYcwcAgELWqVMnbdmyRWXLllWLFi105513qly5crp06ZIOHDigI0eOmG1dXFz0/fffq3v37kVYMYCCsGjRIg0ePFjSzS3G7777blWtWlVJSUk6ceKEfv75ZyUlJZntn3zySf3nP/8pqnIBABbGblkAADhJYmKidu3apV27dmV6vnr16lq8eDHBDlACHT16VEePHs30XOnSpfXPf/5TM2fOdHJVAICSgpE7AAAUsjNnzmj58uXavHmzjh49qrNnz+r8+fMqVaqUqlatqnvuuUfdu3fX4MGDVa5cuaIuF0ABSU5O1k8//aS1a9dq//79+vPPP3Xu3DklJCSocuXKqlevnjp16qQhQ4aoSZMmRV0uAMDCCHcAAAAAAAAsjAWVAQAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwlyKugAUvcTERP3666+SpGrVqsnFhR8LAAAAAAAKWkpKis6ePStJuvvuu1W2bNkC6ZdP8dCvv/6qVq1aFXUZAAAAAADcNvbs2SN/f/8C6YtpWQAAAAAAABbGyB2oWrVq5vGePXtUq1atIqwGAAAAAICSKTY21pw5k/6zeH4R7sBujZ1atWrJ29u7CKsBAAAAAKDkK8j1bpmWBQAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABbmUtQFAAAAAMi9eq+tLuoS7Jyc2quoSwCA2xYjdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACzstg134uLitGrVKo0fP149evSQl5eXbDabbDabBg0alKc+9+zZoxdeeEG+vr7y9PSUh4eHGjZsqF69emnmzJk6e/ZsttcnJCTo/fffV6tWrVSlShV5eHjI19dXY8aM0alTp/JUEwAAAAAAKNlcirqAolKjRo0C6yspKUkvvvii5s+fL8Mw7M4dP35cx48f1w8//KAGDRro4YcfzrSPyMhI9erVS0eOHLF7PDw8XOHh4Zo3b56+/PJL9ezZs8DqBgAAAAAA1nfbhjvp+fj4yNfXV+vWrcv1tcnJyerbt6/WrFkjSbrvvvv0zDPPyNfXVy4uLvrjjz908OBBffPNN1n2ceXKFQUGBprBzrBhw9S/f3+5u7tr06ZNeu+993Tp0iU99thj2rVrl5o3b563JwoAAAAAAEqc2zbcGT9+vPz9/eXv768aNWro5MmTql+/fq77effdd81gZ8aMGXrllVfszrdu3VqPP/64Jk+erOvXr2fax4wZMxQeHi5Jmj59usaOHWuea9u2rTp37qyOHTsqISFBQUFB2rhxY67rBAAAAAAAJdNtu+bOpEmTFBgYmK/pWcePH9fUqVMlSYMGDcoQ7NyqTJkyGR67fv26PvzwQ0mSr69vpn20bdtWQ4cOlSRt2rRJ+/bty3PNAAAAAACgZLltw52CEBISouvXr8tms2n8+PF56mPz5s2Kj4+XJA0cOFClSmX+lqRf5HnZsmV5uhcAAAAAACh5CHfyIW0dHT8/P3NKV2pqqqKjo3XixAldu3Ytxz62bdtmHgcEBGTZzs/PT+XLl5ckbd++PT9lAwAAAACAEoRwJ4/Onj2r48ePS7o5beqvv/5SUFCQvLy85OPjowYNGsjT01MBAQFavXp1lv2EhYWZx02bNs2ynYuLixo2bJjhGgAAAAAAcHu7bRdUzq/ff//dPHZ3d9e9996ryMhIuzYpKSnaunWrtm7dqtGjR2vmzJkZ+omKipIklS9fXpUqVcr2nj4+Pjp06JDOnj2rpKQkubm5OVRrdHR0tudjY2Md6gcAAAAAABQ/hDt5dOHCBfM4ODhYSUlJateund577z35+/srMTFRa9as0ZgxYxQbG6tZs2apcePGev755+36uXz5siTJw8Mjx3umTcuSbm6f7mi44+Pj41A7AAAAAABgPUzLyqOrV6+ax0lJSWrZsqU2bNigjh07yt3dXZUrV9YTTzyhLVu2mKHM+PHjM6zDk5iYKElydXXN8Z7pwxxH1vMBAAAAAAAlH+FOHpUtW9bu+8mTJ2d4TJIaNWqkESNGSLq5Ts/69esz7Sc5OTnHeyYlJZnH7u7uDtcaFRWV7deePXsc7gsAAAAAABQvTMvKowoVKpjHrq6u6ty5c5ZtH3zwQc2YMUOStHfvXvXu3TtDP1euXMnxnulHCzkyjSuNt7e3w20BAAAAAIC1MHInj9KvY1OjRo1sp1WlbxsXF2d3Li14uXr1quLj47O9Z9riy9WqVXN4vR0AAAAAAFCyEe7kUaNGjVSmTBlJ0o0bN7Jtm/68i4v9YKlmzZqZx+Hh4Vn2kZKSYu7G5evrm+t6AQAAAABAyUS4k0dlypRR27ZtJUl//vmn3ZSpW6XfIr1OnTp25zp06GAeb9myJcs+QkNDzXu0b98+TzUDAAAAAICSh3AnHx555BFJN0fmrFy5Mst2y5YtM4/vu+8+u3OdOnVSxYoVJUmLFy+WYRiZ9rFo0SLzuG/fvnktGQAAAAAAlDCEO/kwZMgQVa9eXZL0xhtv6M8//8zQZvPmzfr8888lSX/7298yjLpxdXXVSy+9JEkKCwszF15Ob9euXZo/f74kKSAgQP7+/gX6PAAAAAAAgHXdtrtlbd++XREREeb3586dM48jIiLsRspI0qBBgzL04eHhodmzZ2vAgAH6448/5O/vr9dee02tWrVSYmKi1qxZo1mzZunGjRtycXHR3LlzZbPZMvQzduxYLV26VEePHtW4ceMUERGh/v37y93dXZs2bdKUKVOUkpIid3d3BQcHF9RLAAAAAAAASgCbkdU8oBJu0KBBWrx4scPts3uZPvroI7388stKTk7O9LyHh4f+85//qE+fPln2ERERoZ49e+rYsWOZnvf09NQXX3yhwMBAh2t2VHR0tLmjV1RUFFunAwAAWEC911YXdQl2Tk7tVdQlAECxV1ifv5mWVQD++c9/av/+/RoxYoTuvPNOubu7y8PDQ82bN9e4ceN09OjRbIMdSbrzzjt14MABTZs2TX5+fqpUqZLKlSunJk2aaPTo0Tp06FChBDsAAAAAAMDabtuRO/gfRu4AAABYDyN3AMB6GLkDAAAAAACADAh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwm7bcCcuLk6rVq3S+PHj1aNHD3l5eclms8lms2nQoEH56jshIUENGjQw+6tXr57D173//vtq1aqVqlSpIg8PD/n6+mrMmDE6depUvmoCAAAAAAAlk0tRF1BUatSoUWh9jx8/XidOnMjVNZGRkerVq5eOHDli93h4eLjCw8M1b948ffnll+rZs2dBlgoAAAAAACzuth25k56Pj4+6detWIH0dOHBAwcHBKlu2rCpUqODQNVeuXFFgYKAZ7AwbNkwbNmzQzp07NXnyZHl4eOjSpUt67LHHdOjQoQKpEwAAAAAAlAy3bbgzfvx4ff/99zpz5oxOnTqlf//73/nu88aNGxo2bJhu3LihN954Q1WqVHHouhkzZig8PFySNH36dIWEhKhLly5q27at3njjDa1bt04uLi5KSEhQUFBQvusEAAAAAAAlx20b7kyaNEmBgYEFOj3rww8/1L59+9SkSRO9+uqrDl1z/fp1ffjhh5IkX19fvfLKKxnatG3bVkOHDpUkbdq0Sfv27SuwmgEAAAAAgLXdtuFOQfvjjz80fvx4SdInn3wiV1dXh67bvHmz4uPjJUkDBw5UqVKZvyXpF3letmxZvmoFAAAAAAAlB+FOAXnhhRd09epVPf300+rcubPD123bts08DggIyLKdn5+fypcvL0navn173gsFAAAAAAAlCuFOAfjqq6/0ww8/qHLlypoxY0aurg0LCzOPmzZtmmU7FxcXNWzYMMM1AAAAAADg9nbbboVeUC5evGgucjx16lRVr149V9dHRUVJksqXL69KlSpl29bHx0eHDh3S2bNnlZSUJDc3N4fuER0dne352NhYh/oBAAAAAADFD+FOPo0dO1Z//vmn2rZtq2HDhuX6+suXL0uSPDw8cmybNi1Lurl9uqPhjo+PT67rAgAAAAAA1sC0rHzYunWrFixYIBcXF82dO1c2my3XfSQmJkqSQwswpw9zrl27lut7AQAAAACAkoeRO3mUlJSk4cOHyzAMjRo1Ss2bN89TP2XLlpUkJScnO3TPNO7u7g7fI23qV1ZiY2PVqlUrh/sDAAAAAADFB+FOHk2ePFlHjhyRj4+PJk6cmOd+KlSoIOnmNKucXL161Tx2ZBpXGm9v79wXBgAAAAAALIFwJ4+mTZsmSXrggQe0atWqTNukhTFXr17VV199JUmqXr26unTpYrbx9vbWzz//rKtXryo+Pj7bRZXTRuBUq1bN4fV2AAAAAABAyUa4k0dp06gWLlyohQsXZtv23LlzGjBggCQpICDALtxp1qyZvvvuO0lSeHi42rRpk2kfKSkpioyMlCT5+vrmu34AAAAAAFAysKByEevQoYN5vGXLlizbhYaGmiOB2rdvX+h1AQAAAAAAayDcySPDMHL8qlu3riSpbt265mObN2+266dTp06qWLGiJGnx4sUyDCPT+y1atMg87tu3b6E8JwAAAAAAYD2EO0XM1dVVL730kiQpLCxMM2bMyNBm165dmj9/vqSb07r8/f2dWiMAAAAAACi+bts1d7Zv366IiAjz+3PnzpnHERERdiNlJGnQoEGFVsvYsWO1dOlSHT16VOPGjVNERIT69+8vd3d3bdq0SVOmTFFKSorc3d0VHBxcaHUAAAAAAADruW3DnXnz5mnx4sWZntuxY4d27Nhh91hhhjsVKlTQ6tWr1bNnTx07dkwhISEKCQmxa+Pp6akvvvhCf//73wutDgAAAAAAYD1Myyom7rzzTh04cEDTpk2Tn5+fKlWqpHLlyqlJkyYaPXq0Dh06pMDAwKIuEwAAAAAAFDM2I6sVfHHbiI6Olo+PjyQpKipK3t7eRVwRAAAAclLvtdVFXYKdk1N7FXUJAFDsFdbnb0buAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWNhtG+7ExcVp1apVGj9+vHr06CEvLy/ZbDbZbDYNGjTIoT4SExO1cuVKjRw5Uq1bt1aVKlVUpkwZValSRW3bttXEiRMVGxvrcE0JCQl6//331apVK1WpUkUeHh7y9fXVmDFjdOrUqTw+UwAAAAAAUJK5FHUBRaVGjRr5uv7QoUPq0KGDLl++nOHcxYsXtXv3bu3evVszZ87UvHnz9Pjjj2fbX2RkpHr16qUjR47YPR4eHq7w8HDNmzdPX375pXr27JmvugEAAAAAQMly247cSc/Hx0fdunXL1TV//fWXGey0b99e7733nn766Sft379fP/74o5577jmVLl1aly9f1hNPPKE1a9Zk2deVK1cUGBhoBjvDhg3Thg0btHPnTk2ePFkeHh66dOmSHnvsMR06dCjvTxQAAAAAAJQ4t+3InfHjx8vf31/+/v6qUaOGTp48qfr16zt8falSpfT4449rwoQJatasWYbz3bp1U48ePdS3b1/duHFDI0eO1LFjx2Sz2TK0nTFjhsLDwyVJ06dP19ixY81zbdu2VefOndWxY0clJCQoKChIGzduzMMzBgAAAAAAJdFtO3Jn0qRJCgwMzPP0rHbt2mnp0qWZBjtp+vTpo379+km6Oe3ql19+ydDm+vXr+vDDDyVJvr6+euWVVzK0adu2rYYOHSpJ2rRpk/bt25enmgEAAAAAQMlz24Y7ztK5c2fzODIyMsP5zZs3Kz4+XpI0cOBAlSqV+VuSfpHnZcuWFWiNAAAAAADAugh3CllSUpJ5nFlws23bNvM4ICAgy378/PxUvnx5SdL27dsLsEIAAAAAAGBlhDuFbMuWLeZx06ZNM5wPCwvL9nwaFxcXNWzYMMM1AAAAAADg9nbbLqjsDAcPHtTq1aslSXfddVem6/NERUVJksqXL69KlSpl25+Pj48OHTqks2fPKikpSW5ubg7VER0dne352NhYh/oBAAAAAADFD+FOIUlKStKzzz6rGzduSJKmTJmSabu07dQ9PDxy7DNtWpZ0c/t0R8MdHx8fh9oBAAAAAADrYVpWIXnxxRcVGhoq6eZCyQ899FCm7RITEyVJrq6uOfaZPsy5du1aAVQJAAAAAACsjpE7heC9997TvHnzJEktW7bURx99lGXbsmXLSpKSk5Nz7Df94szu7u4O15M29SsrsbGxatWqlcP9AQAAAACA4oNwp4D9+9//1htvvCFJatKkidasWWM3nepWFSpUkHRzmlVOrl69ah47Mo0rjbe3t8NtAQAAAACAtTAtqwAtWbJEL7zwgiSpbt26Wr9+vapVq5btNWnBy9WrVxUfH59t27QRONWqVXN4vR0AAAAAAFCyEe4UkP/+97965plnlJqaqlq1amnDhg0OjZhJv4NWeHh4lu1SUlIUGRkpSfL19c1/wQAAAAAAoEQg3CkAGzZs0OOPP66UlBRVrVpVP/30kxo2bOjQtR06dDCPt2zZkmW70NBQc1pW+/bt81cwAAAAAAAoMQh38mnnzp3q06ePkpKS5OnpqR9//FF33XWXw9d36tRJFStWlCQtXrxYhmFk2m7RokXmcd++ffNVMwAAAAAAKDkId/Lhl19+Ua9evXT16lWVL19eP/zwg1q2bJmrPlxdXfXSSy9JksLCwjRjxowMbXbt2qX58+dLkgICAuTv75//4gEAAAAAQIlw2+6WtX37dkVERJjfnzt3zjyOiIiwGykjSYMGDbL7PjIyUg8++KC5CPK7776rihUr6vDhw1nes3r16qpevXqGx8eOHaulS5fq6NGjGjdunCIiItS/f3+5u7tr06ZNmjJlilJSUuTu7q7g4OBcP1cAAAAAAFBy2Yys5gGVcIMGDdLixYsdbn/ry7Ro0SINHjw4V/ecMGGCJk6cmOm5iIgI9ezZU8eOHcv0vKenp7744gsFBgbm6p6OiI6Olo+Pj6SbO3KxdToAAEDxV++11UVdgp2TU3sVdQkAUOwV1udvpmUVE3feeacOHDigadOmyc/PT5UqVVK5cuXUpEkTjR49WocOHSqUYAcAAAAAAFjbbTtyB//DyB0AAADrYeQOAFgPI3cAAAAAAACQAeEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFOT3c6d69u7755htdv37d2bcGAAAAAAAocZwe7qxbt079+/dXrVq1FBQUpIMHDzq7BAAAAAAAgBLD6eFO9erVZRiGLly4oDlz5ujee+9Vy5Yt9fHHHys+Pt7Z5QAAAAAAAFia08OdmJgYrVy5Ug8//LBcXFxkGIYOHDigkSNHqnbt2nryySe1fv16Z5cFAAAAAABgSU4Pd0qXLq3evXtr2bJlio6O1owZM/S3v/1NhmEoMTFRX331lR588EHVr19fb7/9tk6dOuXsEgEAAAAAACyjSHfLqlatml5++WUdOnRIe/fu1fPPP69KlSrJMAz98ccfmjRpkho0aKCuXbtq6dKlSk5OLspyAQAAAAAAip1isxV62ro7p0+f1pdffqkHHnhANptNqamp2rhxo5544gnVqlVLI0eO1IEDB4q6XAAAAAAAgGKh2IQ7adzc3NS/f3+tW7dO69evV82aNc1zFy9e1Mcffyw/Pz+1bt1aK1asKLpCAQAAAAAAioFiF+4kJCTos88+U+fOnXX//ffrzz//lGEYMgxDzZo1k7u7uwzD0N69e/XII4+oT58+SkxMLOqyAQAAAAAAikSxCXd27NihZ599VrVq1dLgwYO1ZcsWGYYhT09PPf/88woNDdXhw4d15swZ/fvf/5avr68Mw9CqVas0derUoi4fAAAAAACgSBRpuBMTE6P33ntPTZo0UceOHbVw4UJdvnxZhmGoffv2WrRokU6fPq2PP/5Y9957ryTJw8NDw4YN06+//qp//OMfMgxDX375ZVE+DQAAAAAAgCLj4uwbJicna8WKFVq4cKHWr1+v1NRUGYYh6ebuWc8884yeffZZNWnSJNt+SpUqpdGjR2vp0qX6448/nFE6AAAAAABAseP0cKdWrVqKj4+XJBmGoVKlSqlr16569tln1adPH5UpU8bhvqpWrSpJSklJKYxSAQAAAAAAij2nT8u6ePGiDMOQt7e3xo8fr+PHj2vt2rV69NFHcxXsSFKVKlU0YcIEjR8/Ptd1xMXFadWqVRo/frx69OghLy8v2Ww22Ww2DRo0KNf9rV27Vv369ZO3t7fc3Nzk7e2tfv36ae3atQ73kZCQoPfff1+tWrVSlSpV5OHhIV9fX40ZM0anTp3KdU0AAAAAAKDkc/rInb59++rZZ59V9+7dZbPZ8tVX5cqVNWHChDxdW6NGjXzdO41hGHr++ecVEhJi93hMTIyWL1+u5cuXa/jw4Zo7d262zzcyMlK9evXSkSNH7B4PDw9XeHi45s2bpy+//FI9e/YskLoBAAAAAEDJ4PSRO99995169OiR72CnIPn4+Khbt255uvbNN980g50WLVpoyZIl2rNnj5YsWaIWLVpIkkJCQvTWW29l2ceVK1cUGBhoBjvDhg3Thg0btHPnTk2ePFkeHh66dOmSHnvsMR06dChPdQIAAAAAgJLJ6SN3iovx48fL399f/v7+qlGjhk6ePKn69evnqo+IiAhNnz5dkuTn56etW7fK3d1dkuTv76+HHnpIAQEBCg0N1bRp0zR48GA1bNgwQz8zZsxQeHi4JGn69OkaO3asea5t27bq3LmzOnbsqISEBAUFBWnjxo15fdoAAAAAAKCEcfrIncuXL+vtt9/W22+/rTNnzuTYPjY21mx/7dq1Aqtj0qRJCgwMzNf0rFmzZpmLOc+ZM8cMdtKUK1dOc+bMkXRz0efg4OAMfVy/fl0ffvihJMnX11evvPJKhjZt27bV0KFDJUmbNm3Svn378lwzAAAAAAAoWZwe7qxYsUITJ07UF198oZo1a+bYvmbNmvriiy80adIkff/9906o0DGGYWjlypWSpKZNm6pNmzaZtmvTpo25rfuKFSvMbd/TbN682dw9bODAgSpVKvO3JP0iz8uWLctn9QAAAAAAoKRwerizbNky2Ww2Pf744w61t9ls6t+/vwzD0DfffFPI1TnuxIkTiomJkSQFBARk2zbtfHR0tE6ePGl3btu2bRnaZcbPz0/ly5eXJG3fvj0vJQMAAAAAgBLI6WvupK0t065dO4evadu2rSTp999/L5Sa8iIsLMw8btq0abZt058PCwuzW9vH0X5cXFzUsGFDHTp0yO4aR0RHR2d7PjY2Nlf9AQAAAACA4sPp4U5a0FCrVi2Hr0mbvpU2UqY4iIqKMo+9vb2zbevj45Ppdem/L1++vCpVqpRjP4cOHdLZs2eVlJQkNzc3h2pNf38AAAAAAFCyOH1aVtqaMgkJCQ5fk9Y2bfHi4uDy5cvmsYeHR7Zt06ZTSTe3Pc+sn5z6yKkfAAAAAABwe3L6yJ1atWrp2LFjCg0NdXhqVmhoqCQ5tACzsyQmJprHrq6u2bZNP8Lm1h2/0vrJqY+c+snOraOFbhUbG6tWrVo53B8AAAAAACg+nB7u3HfffTp69Kg+/vhjjRgxQmXKlMm2/fXr1/Xxxx/LZrOpQ4cOTqoyZ2XLljWPk5OTs22blJRkHt+6XXpaPzn1kVM/2clp2hgAAAAAALAup0/LGjx4sCTp2LFjeuKJJ7KdnpWQkKABAwbo6NGjdtcWBxUqVDCPc5oidfXqVfP41ulXaf04Ms0qu34AAAAAAMDtyenhTrt27cytzZctW6amTZvqnXfe0ZYtW3T06FEdO3ZMW7Zs0TvvvCNfX18tX75cNptNjz76aI5bjjtT+tEwOe1GlX5a1K2LG6f1c/XqVcXHxzvUT7Vq1RxeTBkAAAAAAJRsTp+WJUkLFizQuXPntH79esXExGjixImZtjMMQ5LUtWtXLV682IkV5qxZs2bmcdr27llJf97X1zdDP999953Zrk2bNpn2kZKSosjIyEz7AAAAAAAAty+nj9yRbq4z8+OPP2rWrFmqXbu2DMPI9MvHx0ezZ8/W2rVr7da4KQ7q16+v2rVrS5K2bNmSbdutW7dKkurUqaN69erZnUu/jlB2/YSGhprTstq3b5+XkgEAAAAAQAlUJCN3JMlms2nUqFF66aWX9Msvv+jAgQM6d+6cJMnLy0v33nuv7rnnHtlstqIqMVs2m019+vTRJ598ovDwcO3evTvTUTe7d+82R+706dMnw/Pp1KmTKlasqEuXLmnx4sUaN25cps950aJF5nHfvn0L9skAAAAAAADLKrJwJ43NZlOLFi3UokWLoi4l14KCgvTpp58qJSVFI0eO1NatW+12sbp27ZpGjhwpSXJxcVFQUFCGPlxdXfXSSy/pnXfeUVhYmGbMmKGxY8fatdm1a5fmz58vSQoICJC/v3/hPSkAAAAAAGApRR7uFJXt27crIiLC/D5t1JAkRURE2I2UkaRBgwZl6KNx48YaM2aMpk6dqtDQULVv316vvvqqGjZsqMjISE2bNk0HDhyQJI0dO1aNGjXKtJaxY8dq6dKlOnr0qMaNG6eIiAj1799f7u7u2rRpk6ZMmaKUlBS5u7srODg4388dAAAAAACUHDYjbdXi28ygQYNytUhzVi9Tamqqhg0bpgULFmR57dChQxUSEqJSpbJe4igiIkI9e/bUsWPHMj3v6empL774QoGBgQ7X7Kjo6GhzF6+oqCi7ncAAAABQPNV7bXVRl2Dn5NReRV0CABR7hfX5u0hH7hw8eFDbtm3T8ePHdfnyZd24cSPb9jabzZyeVFyUKlVK8+fP1yOPPKKQkBDt3btX586dk5eXl/z9/fXcc8+pR48eOfZz55136sCBA/roo4/0zTffKCIiQsnJyfLx8VHPnj01atQo1a1b1wnPCAAAAAAAWEmRjNw5cuSIhgwZot27dzt8jWEYstlsOQZAyD1G7gAAAFgPI3cAwHpKzMidmJgYdezYUefOnTOnOnl4eKhy5crZTlsCAAAAAABARk4PdyZPnqyzZ8/KZrPp2Wef1ZgxY9S4cWNnlwEAAAAAAFAiOD3cWbt2rWw2m5555hmFhIQ4+/YAAAAAAAAlitPnQZ0+fVqS9Mwzzzj71gAAAAAAACWO08OdypUrS5IqVark7FsDAAAAAACUOE4Pd/z8/CRJR48edfatAQAAAAAAShynhzsvvfSSDMNgvR0AAAAAAIAC4PRwp2vXrho3bpw2bdqkESNG6Pr1684uAQAAAAAAoMRw+m5Zn332mZo1a6Z27dopJCRE33//vR599FE1bdpU5cqVy/F6FmIGAAAAAAD4H6eHO4MGDZLNZjO/j42N1Zw5cxy6Nm0LdQAAAAAAANzk9HBHkgzDKIrbAgAAAAAAlDhOD3dOnDjh7FsCAAAAAACUWE4Pd+rWrevsWwIAAAAAAJRYTt8tCwAAAAAAAAWHcAcAAAAAAMDCimRB5TQRERH67LPPtGvXLp05c0bXrl3T2rVrdeedd5ptDh8+rFOnTql8+fIKCAgowmoBAAAAAACKnyIJd1JTU/Xqq68qODhYqamp5u5ZNptNycnJdm2joqIUGBgoFxcXnThxQnXq1CmKkgEAAAAAAIqlIpmW9dxzz2nmzJm6ceOGateurUcffTTLtj169FCDBg1048YNffvtt06sEgAAAAAAoPhzerizefNmzZ8/X5L0xhtv6OTJk/r666+zveaxxx6TYRjatGmTM0oEAAAAAACwDKdPy5o7d64kqWfPnnr33XcduqZVq1aSpN9++63Q6gIAAAAAALAip4/c2bVrl2w2m4YOHerwNd7e3pKkM2fOFFZZAAAAAAAAluT0cCcuLk6SVL9+fYevcXG5OcDo+vXrhVITAAAAAACAVTk93HF3d5ckJSQkOHzNqVOnJEmVK1culJoAAAAAAACsyunhTtqInQMHDjh8zapVqyRJzZo1K5SaAAAAAAAArMrp4U63bt1kGIZCQkKUmpqaY/t9+/bp888/l81mU/fu3Z1QIQAAAAAAgHU4Pdx58cUX5e7url9//VXDhg3Ldh2d7777Tt27d1dycrI8PT01fPhwJ1YKAAAAAABQ/Dl9K/Q6depo9uzZGjZsmBYtWqR169apd+/e5vn58+crISFB69ev1/Hjx2UYhmw2m0JCQlSxYkVnlwsAAAAAAFCsOT3ckaShQ4fKZrPppZdeUkxMjP7973/LZrNJkoKDgyVJhmFIktzc3DR37lw99thjRVEqAAAAAABAseb0aVlphgwZovDwcL388stq2LChDMOw+6pTp45GjBihsLAwDRw4sKjKBAAAAAAAKNaKZOROGm9vb82YMUMzZszQX3/9pbi4ON24cUNVq1aVl5dXUZYGAAAAAABgCUUa7qTn6ekpT0/Poi4DAAAAAADAUopsWhYAAAAAAADyj3AHAAAAAADAwpw+LatLly55vtZms2nDhg0FWA0AAAAAAIC1OT3c2bx5s2w2m7nVeWbStkVPk9b21scBAAAAAABud04Pdzp27JhjSHP16lUdO3ZMly5dks1mU+PGjVWrVi0nVQgAAAAAAGAdRTJyxxGGYWj16tUaNWqULly4oHnz5qlDhw6FWxwAAAAAAIDFFNsFlW02mwIDA7V9+3aVLl1affv2VUxMTFGXBQAAAAAAUKwU23AnTa1atfTyyy/r/Pnzmj59elGXAwAAAAAAUKwU+3BHkjkda/Xq1UVcCQAAAAAAQPFiiXDH1dVVknT69OkirgQAAAAAAKB4sUS4s337dklSuXLlirgSAAAAAACA4qXYhzu7du3S22+/LZvNplatWhV1OQAAAAAAAMWK07dCf/vtt3Nsk5qaqosXLyo0NFQ///yzUlNTZbPZNHr0aCdUCAAAAAAAYB1OD3cmTpwom83mcHvDMOTi4qLp06era9euhVgZAAAAAACA9Tg93JFuBjbZsdlsqlChgurXr6+AgAANHz5czZo1c1J1AAAAt596rxW/XUlPTu1V1CUAAGAJTg93UlNTnX1LAAAAAACAEqvYL6gMAAAAAACArBHuAAAAAAAAWBjhDgAAAAAAgIU5fc2dU6dOFUq/d9xxR6H0CwAAAAAAUJw5PdypX79+gfdps9mUkpJS4P0CAAAAAAAUd04Pd3LaBh0AAAAAAACOc3q4s3DhQknSxx9/rL1796pMmTLq1q2bWrVqpRo1asgwDMXFxWnv3r1at26drl+/Ln9/f40YMcLZpQIAAAAAABR7Tg93Bg4cqGeffVahoaHq1q2b5s+frzp16mTaNiYmRsOGDdOPP/6ou+++W59++qmTqwUAAEBRqffa6qIuwc7Jqb2KugQAADLl9N2yvv32Wy1YsEB+fn5avXp1lsGOJNWpU0fff/+9WrZsqQULFujrr792YqUAAAAAAADFn9PDnX//+9+y2Wx6+eWXVbp06Rzbly5dWq+88ooMw1BISIgTKgQAAAAAALAOp4c7hw4dkiQ1btzY4WvS2v7666+FUhMAAAAAAIBVOT3cuXz5siQpLi7O4WvS2qZdCwAAAAAAgJucHu7UrVtXkvTZZ585fE1a2zvuuKNQagIAAAAAALAqp4c7ffr0kWEY+uqrrzR9+vQc28+YMUNLliyRzWZT3759nVAhAAAAAACAdTg93HnttddUs2ZNSdLrr7+uFi1aKDg4WDt27NCxY8cUERGhHTt2KDg4WC1bttSrr74qSapZs6Z5XBwlJydr/vz56t69u2rVqiU3Nzd5eHioSZMmGjJkiHbv3u1QP2vXrlW/fv3k7e0tNzc3eXt7q1+/flq7dm0hPwMAAAAAAGBFNsMwDGff9Pfff9eDDz6omJgY2Wy2bNsahiFvb2+tXbtWzZo1c1KFuRMVFaVevXrluODz6NGj9cEHH2T6nA3D0PPPP5/tjmDDhw/X3Llzc3zNcis6Olo+Pj6Sbj4Xb2/vAu0fAAAUf/VeW13UJRR7J6f2KuoS7BS396y4vT4AUBwV1udvp4/ckaRmzZrpt99+0+jRo1WpUiUZhpHpV6VKlfTyyy/r8OHDxTbYSUlJsQt2mjdvrkWLFmnXrl1at26dxo8fr/Lly0uSZs2apRkzZmTaz5tvvmkGOy1atNCSJUu0Z88eLVmyRC1atJAkhYSE6K233nLCswIAAAAAAFZRJCN30ktOTta+ffv066+/6uLFizIMQ1WqVNHdd9+tli1bytXVtSjLy9F3332nRx99VJLUtm1bbdu2TaVLl7Zrs2/fPrVt21bXr19X5cqVFRcXJxcXF/N8RESEfH19lZKSIj8/P23dulXu7u7m+YSEBAUEBCg0NFQuLi4KDw9Xw4YNC+w5MHIHAAAUt1EgxVFxG5lS3N6z4vb6AEBxVFifv11yblK4XF1d1bZtW7Vt27aoS8mTHTt2mMevv/56hmBHklq2bKnAwEAtX75cFy9eVHh4uP72t7+Z52fNmqWUlBRJ0pw5c+yCHUkqV66c5syZo7Zt2yolJUXBwcGaM2dOIT0jAAAAAABgJUUe7lhdcnKyedygQYMs26UfaZOUlGQeG4ahlStXSpKaNm2qNm3aZHp9mzZt1KRJEx05ckQrVqzQ7NmzC3ztHQAAAGStuI2UAQAgTZGHO8ePH9euXbt05swZJSQkaMSIEfLy8irqshzWuHFj8/j48eO66667Mm0XGRkpSbLZbGrUqJH5+IkTJxQTEyNJCggIyPZeAQEBOnLkiKKjo3Xy5EnVr18/v+UDAAAAAACLK7Jw58CBAwoKCtL27dvtHn/kkUfswp2PPvpIkyZNUsWKFfX777+rTJkyzi41WwMGDNBbb72lv/76S9OmTVPPnj0zTM06cOCAVq+++Zee/v37y9PT0zwXFhZmHjdt2jTbe6U/HxYW5nC4Ex0dne352NhYh/oBAAAAAADFT5GEO6tXr9ajjz6q5ORkpV/PObNpRgMHDtRrr72m8+fPa9WqVerbt68zS81RtWrVtGjRIj355JPasWOH/P39FRQUpMaNG+vKlSvasWOHPvjgAyUnJ+vvf/+7Zs6caXd9VFSUeZzTQkppiy7del1O0l8HAAAAAABKFqdvhX7mzBkNGDBASUlJatasmdasWaPLly9n2d7Dw0MPP/ywJGnNmjVOqjJ3+vbtq9DQUA0dOlS//PKLBg4cqLZt26pr166aOHGiypUrp5kzZ2r79u2qWbOm3bXpn7uHh0e290nbUl2Srly5UrBPAgAAAAAAWJLTR+7MmjVLV65cUd26dbVt2zZVqlQpx2s6deqkL774Qvv27Sv8AvPg+vXr+vLLL/X9998rs53l//zzTy1ZskSNGzdWr172W0QmJiaaxzlt++7m5mYeX7t2zeH6chrlExsbq1atWjncHwAAAAAAKD6cHu78+OOPstlseuWVVxwKdiSpSZMmkqSTJ08WXmF5dPXqVfXs2VNbt25V6dKlNW7cOA0ePFgNGjRQYmKifv75Z7399tvavn27evfurVmzZmnUqFHm9WXLljWP0++8lZn0u2zdul16dnKa7gUAAAAAAKzL6dOyTpw4IUm5GilSoUIFScVzKtKECRO0detWSdL8+fM1bdo0NW3aVK6urvL09FTXrl21adMmde7cWYZh6OWXX9ahQ4fM69Oem5Tz87t69ap5nNMULgAAAAAAcHtwerhz/fp1ScrVrlfx8fGS7NecKQ4Mw9DChQsl3dwSfeDAgZm2c3Fx0TvvvCNJSk1NNa+R7EfV5LSrVfrpVSySDAAAAAAApCIId9IWFE4bweOIXbt2SSp+04v+/PNPXbhwQZLUokWLbNu2bNnSPA4PDzePmzVrlunjmUl/3tfXN1e1AgAAAACAksnp4U779u0lScuXL3eofUJCgubOnSubzaaOHTsWZmm55uLyvyWLUlJSsm2bNmLp1uvq16+v2rVrS5K2bNmSbR9p07/q1KmjevXq5bZcAAAAAABQAjk93Bk4cKAMw9CSJUu0bt26bNteuXJFjz/+uE6dOiVJGjp0qDNKdFiVKlXk6ekp6eboouwCnvTBTf369c1jm82mPn36SLo5Mmf37t2ZXr97925z5E6fPn1ks9nyXT8AAAAAALA+p4c7DzzwgB5++GGlpqbqoYce0tixY7Vnzx7z/IULF/Tzzz/rnXfeUZMmTbRmzRrZbDY988wzOU59crZSpUqZW5ufPn1akydPzrTdxYsX9eqrr5rfBwYG2p0PCgoyR/OMHDkywzbn165d08iRIyXdHPUTFBRUUE8BAAAAAABYnM0wDMPZN01ISFBgYKA2b96c7QiUtNLuv/9+rVq1Sm5ubs4q0WHh4eFq2bKlEhISJEm9e/fWwIEDza3Qd+/ereDgYHP00f3336/169dn6Of111/X1KlTJd1cv+fVV19Vw4YNFRkZqWnTpunAgQNmuylTphToc4iOjjYXaI6Kiip2axsBAIDCV++11UVdAizu5NReRV0CABR7hfX5u0jCHenmrlGzZs3SzJkzFRsbm2mbKlWqaMyYMRo3bpxKlXL6ICOHrV+/XgMGDNC5c+eybdelSxd9++23qly5coZzqampGjZsmBYsWJDl9UOHDlVISEiBvxaEOwAAgHAH+UW4AwA5K3HhTpqUlBTt2bNHoaGhiouL040bN1S1alW1aNFCHTp0KJajdTJz/vx5zZ8/X2vWrNFvv/2m+Ph4ubi4qGbNmvL399cTTzyhhx56KMe1cn744QeFhIRo7969OnfunLy8vOTv76/nnntOPXr0KJTaCXcAAADhDvKLcAcAclZiwp3PPvtMktSkSRO1bt3ambdGFgh3AAAA4Q7yi3AHAHJWWJ+/nT7XadCgQRo8eLD++OMPZ98aAAAAAACgxHF6uFOxYkVJUqNGjZx9awAAAAAAgBLH6eFO/fr1Jd3cHhwAAAAAAAD54/Rwp2/fvjIMQ99//72zbw0AAAAAAFDiuDj7hqNGjdKCBQv0ySefqHfv3urSpYuzSwAAAChyLGAMAAAKitNH7nh6euqnn35S06ZN9eCDD2r48OHavHmzLly4oCLelR0AAAAAAMBynD5yp3Tp0uaxYRiaP3++5s+f79C1NptNKSkphVUaAAAAAACA5Tg93Ll1dA6jdQAAAAAAAPLO6eHOhAkTnH1LAAAAAACAEqtQw53PPvtMkvTwww/L09NTEuEOAAAAAABAQSrUcGfQoEGy2Wzy8/NTs2bNMpw/e/asPvnkE9lsNr311luFWQoAAAAAAECJ5PRpWenFxcVp4sSJhDsAAAAAAAB55PSt0AEAAAAAAFBwCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACzMKVuhf/zxx6pevXqGx+Pi4szjt99+26G+xo8fX2B1AQAAAAAAWJ1Twp1PPvkky3M2m02SNGnSJIf6ItwBAAAAAAD4n0IPdwzDKLC+0oIgAAAAAAAA3FSo4c6mTZsKs3sAAAAAAIDbXqGGOwEBAYXZPQAAAAAAwG2P3bIAAAAAAAAszCkLKgMAAAAo2eq9trqoS7Bzcmqvoi4BAJyGkTsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuFOAzp07p+nTp6t9+/aqWbOm3NzcVLt2bbVu3Vpjx47Vrl27cuxj7dq16tevn7y9veXm5iZvb2/169dPa9eudcIzAAAAAAAAVuNS1AWUFN98841GjBih8+fP2z0eGxur2NhY7dmzR8eOHdOKFSsyvd4wDD3//PMKCQmxezwmJkbLly/X8uXLNXz4cM2dO1c2m62wngYAAAAAALAYwp0C8Nlnn2nw4MFKTU1V9erVNWLECHXo0EFVqlTRmTNnFBkZqe+//15lypTJso8333zTDHZatGihcePGqWHDhoqMjNT06dN14MABhYSEqFq1anr33Xed9dQAAAAAAEAxZzMMwyjqIqwsLCxMLVq0UFJSku677z59//33qlixYqZtk5OT5erqmuHxiIgI+fr6KiUlRX5+ftq6davc3d3N8wkJCQoICFBoaKhcXFwUHh6uhg0bFthziI6Olo+PjyQpKipK3t7eBdY3AADIXL3XVhd1CUCJdnJqr6IuAQAyKKzP36y5k08jR45UUlKSvLy8tGzZsiyDHUmZBjuSNGvWLKWkpEiS5syZYxfsSFK5cuU0Z84cSVJKSoqCg4MLpngAAAAAAGB5hDv5EB4erg0bNkiSXnzxRXl5eeW6D8MwtHLlSklS06ZN1aZNm0zbtWnTRk2aNJEkrVixQgy4AgAAAAAAEuFOvnzzzTfm8WOPPWYeX7x4UceOHcuwuHJmTpw4oZiYGElSQEBAtm3TzkdHR+vkyZN5qBgAAAAAAJQ0hDv5sHv3bklSxYoV5evrqy+++EL33HOPqlSposaNG8vLy0sNGjTQpEmTdOXKlUz7CAsLM4+bNm2a7f3Sn09/HQAAAAAAuH2xW1Y+/P7775KkevXqaeTIkfroo48ytDlx4oQmTpyob7/9Vj/++KNq165tdz4qKso8zmkhpbRFl269LifR0dHZno+NjXW4LwAAAAAAULwQ7uTDhQsXJN1ce+fgwYOqVKmSpk6dqn79+snT01O//vqrxo8frzVr1ujw4cN67LHHtG3bNpUq9b8BU5cvXzaPPTw8sr1f+fLlzeOsRgJlJn0oBAAAAAAAShbCnXy4evWqJCkpKUmlS5fWmjVr7BZE9vPz06pVqxQYGKg1a9Zo586dWrZsmR599FGzTWJionmc1W5aadzc3Mzja9euFdTTAACgUBS3rb7ZFhm4vfBvEIDbCeFOPpQtW9YMeB577LFMd7oqVaqU3n//fa1Zs0aStGTJErtwp2zZsuZxcnJytvdLSkoyj2/dLj07OU3hio2NVatWrRzuDwAAAAAAFB+EO/lQoUIFM9zp0aNHlu3uuusu1alTRzExMdq7d2+GPtLkNNUq7V5SzlO40stpLR8AAAAAAGBd7JaVD+nXsnF0MeS4uDi7x9Nfl9PCx+lH4LCODgAAAAAAkBi5ky933XWXORLnxo0b2bZNO+/iYv+SN2vWzDwODw/Pto/05319fXNVKwAAt7vitv4GAABAQWHkTj507NjRPI6MjMy27fHjxyVJderUsXu8fv365vboW7ZsybaPrVu3mn3Uq1cvt+UCAAAAAIASiHAnHx566CGVKVNGkrRs2bIs223ZskXnz5+XJN13331252w2m/r06SPp5sic3bt3Z9rH7t27zZE7ffr0kc1my3f9AAAAAADA+gh38qFq1ap69tlnJUk//fSTvvrqqwxtLl++rKCgIPP75557LkOboKAgc7rWyJEjM2xzfu3aNY0cOVLSzWld6fsDAAAAAAC3N8KdfJo0aZLuuOMOSdLTTz+tkSNHatOmTdq3b58WLVqkVq1a6ZdffpEkjRgxQv7+/hn6aNy4scaMGSNJCg0NVfv27bV06VKFhoZq6dKlat++vUJDQyVJY8eOVaNGjZzz5AAAAAAAQLFnMwzDKOoirC4sLEwPPfSQIiIismwzZMgQzZ0715zGdavU1FQNGzZMCxYsyLKPoUOHKiQkRKVKFWwmFx0dbe6+FRUVxdbpAIACwQLGAPA/J6f2KuoSABQDhfX5m5E7BcDX11e//PKL3n//fbVu3VpVqlSRq6urvL299Y9//EMbN27U/Pnzswx2JKlUqVKaP3++Vq9erT59+qh27dpydXVV7dq11adPH/3www+aN29egQc7AAAAAADA2hi5A0buAAAKBSN3AOB/GLkDQCq8z98uBdILAAC3oeIWXvDBAQAA4PbEHB8AAAAAAAALI9wBAAAAAACwMMIdAAAAAAAACyPcAQAAAAAAsDDCHQAAAAAAAAsj3AEAAAAAALAwwh0AAAAAAAALI9wBAAAAAACwMMIdAAAAAAAACyPcAQAAAAAAsDDCHQAAAAAAAAsj3AEAAAAAALAwwh0AAAAAAAALI9wBAAAAAACwMMIdAAAAAAAACyPcAQAAAAAAsDDCHQAAAAAAAAsj3AEAAAAAALAwwh0AAAAAAAALI9wBAAAAAACwMMIdAAAAAAAACyPcAQAAAAAAsDDCHQAAAAAAAAsj3AEAAAAAALAwwh0AAAAAAAALI9wBAAAAAACwMMIdAAAAAAAACyPcAQAAAAAAsDDCHQAAAAAAAAsj3AEAAAAAALAwwh0AAAAAAAALI9wBAAAAAACwMMIdAAAAAAAACyPcAQAAAAAAsDDCHQAAAAAAAAsj3AEAAAAAALAwwh0AAAAAAAALI9wBAAAAAACwMMIdAAAAAAAACyPcAQAAAAAAsDDCHQAAAAAAAAsj3AEAAAAAALAwwh0AAAAAAAALI9wBAAAAAACwMMIdAAAAAAAACyPcAQAAAAAAsDDCHQAAAAAAAAsj3AEAAAAAALAwwh0AAAAAAAALI9wBAAAAAACwMMIdAAAAAAAACyPcAQAAAAAAsDDCHQAAAAAAAAsj3AEAAAAAALAwwh0AAAAAAAALI9wBAAAAAACwMMIdAAAAAAAACyPcAQAAAAAAsDDCHQAAAAAAAAsj3AEAAAAAALAwwp1CMm7cONlsNvNr8+bNOV6zdu1a9evXT97e3nJzc5O3t7f69euntWvXFn7BAAAAAADAkgh3CsHBgwc1a9Ysh9sbhqHnnntOPXr00PLlyxUTE6Pk5GTFxMRo+fLl6tGjh5577jkZhlGIVQMAAAAAACsi3ClgqampGjZsmFJSUlS9enWHrnnzzTcVEhIiSWrRooWWLFmiPXv2aMmSJWrRooUkKSQkRG+99Vah1Q0AAAAAAKyJcKeAzZ49W3v37lXTpk01dOjQHNtHRERo+vTpkiQ/Pz/t2LFD/fv3l7+/v/r376/t27fLz89PkjRt2jRFRkYWav0AAAAAAMBaCHcKUFRUlDm65pNPPpGrq2uO18yaNUspKSmSpDlz5sjd3d3ufLly5TRnzhxJUkpKioKDgwu2aAAAAAAAYGmEOwXohRde0JUrVzRw4EB16tQpx/aGYWjlypWSpKZNm6pNmzaZtmvTpo2aNGkiSVqxYgVr7wAAAAAAABPhTgH5+uuvtWrVKlWpUkXvv/++Q9ecOHFCMTExkqSAgIBs26adj46O1smTJ/NVKwAAAAAAKDkIdwpAfHy8Ro0aJenmujjVqlVz6LqwsDDzuGnTptm2TX8+/XUAAAAAAOD25lLUBZQE48aN05kzZ9SuXTuHFlFOExUVZR57e3tn29bHxyfT6xwRHR2d7fnY2Nhc9QcAAAAAAIoPwp182r59u+bNmycXFxfNnTtXNpvN4WsvX75sHnt4eGTbtnz58ubxlStXclVj+mAIAAAAAACULEzLyofk5GQNHz5chmFo9OjRuvvuu3N1fWJionmc085abm5u5vG1a9dyVygAAAAAACixGLmTD1OmTFFYWJjuuOMOTZgwIdfXly1b1jxOTk7Otm1SUpJ5fOt26TnJaRpXbGysWrVqlas+AQAAAABA8UC4k0fh4eF67733JElz5syxmzblqAoVKpjHOU21unr1qnmc0xSuW+W0ng8AAAAAALAuwp08mjVrlpKTk9WgQQMlJCToq6++ytDm8OHD5vHGjRt15swZSVLv3r1Vvnx5u9Alp0WP04++YQ0dAAAAAACQhnAnj9KmSR0/flwDBgzIsf0777xjHp84cULly5dXs2bNzMfCw8OzvT79eV9f39yWCwAAAAAASigWVC5C9evXV+3atSVJW7Zsybbt1q1bJUl16tRRvXr1Crs0AAAAAABgEYQ7ebRo0SIZhpHtV/pFljdt2mQ+nhbO2Gw29enTR9LNkTm7d+/O9F67d+82R+706dMnV9utAwAAAACAko1wp4gFBQXJxeXm7LiRI0dm2Ob82rVrGjlypCTJxcVFQUFBzi4RAAAAAAAUY4Q7Raxx48YaM2aMJCk0NFTt27fX0qVLFRoaqqVLl6p9+/YKDQ2VJI0dO1aNGjUqynIBAAAAAEAxw4LKxcDkyZMVFxenBQsW6MCBA+rfv3+GNkOHDtW7775bBNUBAAAAAIDijJE7xUCpUqU0f/58rV69Wn369FHt2rXl6uqq2rVrq0+fPvrhhx80b948lSrF2wUAAAAAAOwxcqcQTZw4URMnTnS4fc+ePdWzZ8/CKwgAAAAAAJQ4hDsAAJQQ9V5bXdQlAAAAoAgwzwcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALIxwBwAAAAAAwMIIdwAAAAAAACyMcAcAAAAAAMDCCHcAAAAAAAAsjHAHAAAAAADAwgh3AAAAAAAALMylqAsAAAAAgJKu3muri7oEOyen9irqEgAUIEbuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYGOEOAAAAAACAhRHuAAAAAAAAWBjhDgAAAAAAgIUR7gAAAAAAAFgY4Q4AAAAAAICFEe4AAAAAAABYmEtRFwAAAAAAcK56r60u6hLsnJzaq6hLACyNkTsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRrgDAAAAAABgYYQ7AAAAAAAAFka4AwAAAAAAYGGEOwAAAAAAABZGuAMAAAAAAGBhhDsAAAAAAAAWRriTT/v379eUKVPUo0cP+fj4yM3NTR4eHmrcuLEGDRqkbdu25aq/tWvXql+/fvL29pabm5u8vb3Vr18/rV27tpCeAQAAAAAAsDKXoi7AygICArR169YMjycnJ+vYsWM6duyYFi9erKefflrz5s2Tq6trln0ZhqHnn39eISEhdo/HxMRo+fLlWr58uYYPH665c+fKZrMV+HMBAAAAAADWxMidfIiJiZEk1a5dW6NGjdK3336rPXv2aNeuXZo5c6bq1KkjSfr88881aNCgbPt68803zWCnRYsWWrJkifbs2aMlS5aoRYsWkqSQkBC99dZbhfeEAAAAAACA5dgMwzCKugirCgwM1DPPPKNHHnlEpUuXznD+3Llzat++vY4ePSpJ2rp1q+67774M7SIiIuTr66uUlBT5+flp69atcnd3N88nJCQoICBAoaGhcnFxUXh4uBo2bFhgzyM6Olo+Pj6SpKioKHl7exdY3wBQktV7bXVRlwAAQIlwcmqvoi4BcIrC+vzNyJ18WLVqlR5//PFMgx1J8vLy0gcffGB+/+2332babtasWUpJSZEkzZkzxy7YkaRy5cppzpw5kqSUlBQFBwcXQPUAAAAAAKAkYM2dQtapUyfzODIyMsN5wzC0cuVKSVLTpk3Vpk2bTPtp06aNmjRpoiNHjmjFihWaPXs2a+8AAAAAKBGK22hYRhLBahi5U8iSk5PN41KlMr7cJ06cMNfuCQgIyLavtPPR0dE6efJkwRUJAAAAAAAsi3CnkG3ZssU8btq0aYbzYWFh2Z5PL/359NcBAAAAAIDbF9OyClFqaqqmTp1qfv/4449naBMVFWUe57SQUtqiS7del5Po6Ohsz8fGxjrcFwAAAAAAKF4IdwrRrFmztGfPHklS37595efnl6HN5cuXzWMPD49s+ytfvrx5fOXKFYfrSB8KAQAAAACAkoVpWYVky5Yteu211yRJ1atX1yeffJJpu8TERPPY1dU12z7d3NzM42vXrhVAlQAAAAAAwOoYuVMIfvvtN/Xt21cpKSlyc3PT119/rRo1amTatmzZsuZx+sWXM5OUlGQe37pdenZymsIVGxurVq1aOdwfAAAAAAAoPgh3CtiJEyfUrVs3Xbx4UaVLl9aSJUuy3QWrQoUK5nFOU62uXr1qHuc0hSu9nNbyAQAAAP5fe3ceHlWV53/8U9lICJKwOgSQRQiERUAkYJM2hgA6RIalFRtpNgERUBsF2WwERkVoXHBDRZYGe5QRBDREkC1BYEDCIjQDiCwJS9DAGLZsJOT+/siT+lVIKltVqurC+/U8eZ7bdc4951tVx9Pkm3POBQCYF9uynCglJUXdu3dXSkqKLBaLlixZon79+pV4j23ipbSDj21X4HCODgAAAAAAkEjuOM2lS5fUo0cPnTp1SpL0wQcfaMiQIaXe16pVK+v1sWPHSqxrWx4WFlbBSAEAAAAAwO2E5I4TXLlyRY888oiOHDkiSZozZ47GjRtXpnubNGmikJAQSfmHMJfkhx9+kCTVr19fjRs3rnjAAAAAAADgtkFyx0EZGRmKiYnR/v37JUmvvPKKJk+eXOb7LRaL+vTpIyl/Zc7u3buLrbd7927ryp0+ffrIYrE4GDkAAAAAALgdkNxxwI0bN9SvXz/t3LlTkvTXv/5Vr7/+ernbGT9+vHx88s+2fv7554s85jwzM1PPP/+8JMnHx0fjx493LHAAAAAAAHDb4GlZDhg4cKA2btwoSerWrZtGjBihw4cP263v5+en0NDQIq+HhoZq4sSJmjNnjvbu3auuXbtq8uTJuvfee3Xy5EnNnTtXBw4ckCS9/PLLat68eeW8IQAAAAAAYDoWwzAMdwdhVuXdGtWoUSMlJSUVW5aXl6dRo0ZpyZIldu8fMWKEFi5cKC8v5y64OnfunPXpW2fPnuXR6QBQRo2nxLk7BAAAUAmS5sS4OwTcpirr92+2ZXkILy8vLV68WHFxcerTp49CQkLk5+enkJAQ9enTR999950WLVrk9MQOAAAAAAAwN7ZlOaAyFj316tVLvXr1cnq7AAAAAADg9sQyEAAAAAAAABMjuQMAAAAAAGBiJHcAAAAAAABMjOQOAAAAAACAiZHcAQAAAAAAMDGSOwAAAAAAACZGcgcAAAAAAMDESO4AAAAAAACYmI+7AwAAAAAAwJM0nhLn7hAKSZoT4+4Q4OFYuQMAAAAAAGBiJHcAAAAAAABMjOQOAAAAAACAiZHcAQAAAAAAMDGSOwAAAAAAACZGcgcAAAAAAMDESO4AAAAAAACYGMkdAAAAAAAAEyO5AwAAAAAAYGIkdwAAAAAAAEyM5A4AAAAAAICJkdwBAAAAAAAwMR93BwAAQFk1nhLn7hAAAAAAj8PKHQAAAAAAABMjuQMAAAAAAGBiJHcAAAAAAABMjOQOAAAAAACAiZHcAQAAAAAAMDGSOwAAAAAAACZGcgcAAAAAAMDEfNwdAADAczWeEufuEAAAAACUgpU7AAAAAAAAJkZyBwAAAAAAwMRI7gAAAAAAAJgYZ+4AAAAAAIAy87RzGZPmxLg7BLdj5Q4AAAAAAICJkdwBAAAAAAAwMZI7AAAAAAAAJsaZOwAAAAAAeDBPO+MGnoeVOwAAAAAAACZGcgcAAAAAAMDESO4AAAAAAACYGGfuALc5T9ufmzQnxt0hAAAAAMBthZU7AAAAAAAAJkZyBwAAAAAAwMRI7gAAAAAAAJgYyR0AAAAAAAATI7kDAAAAAABgYiR3AAAAAAAATIzkDgAAAAAAgImR3AEAAAAAADAxkjsAAAAAAAAmRnIHAAAAAADAxEjuAAAAAAAAmBjJHQAAAAAAABPzcXcAAO4sjafEuTsEj5Y0J8bdIQAAAAAwGVbuAAAAAAAAmBjJHQAAAAAAABMjuQMAAAAAAGBinLkDAB6EM4kAAAAAlBcrdwAAAAAAAEyM5A4AAAAAAICJkdzxMGfOnNHEiRMVFhamwMBA1axZU+Hh4XrrrbeUkZHh7vAAAAAAAICH4cwdDxIXF6dBgwbpypUr1tcyMjKUmJioxMRELVq0SN99952aNm3qxig9j6edUZI0J8bdIQAAAAAA7iCs3PEQBw8e1IABA3TlyhVVq1ZNb7zxhv7nf/5HW7Zs0ahRoyRJP//8s2JiYnT9+nU3RwsAAAAAADwFK3c8xPjx45WRkSEfHx9t3LhRDz74oLWsW7duat68uSZNmqRjx47pnXfe0auvvurGaAEAAAAAgKdg5Y4HSExMVEJCgiRpxIgRhRI7BSZMmKCwsDBJ0vz585WTk+PKEAEAAAAAgIdi5Y4HWLt2rfV6+PDhxdbx8vLSkCFDNHXqVKWlpSkhIUE9evRwUYQoD087AwgAAAAAcHtj5Y4H2L59uyQpMDBQHTt2tFsvMjLSer1jx45KjwsAAAAAAHg+Vu54gKNHj0qSmjVrJh8f+19Jy5Yti9xTFufOnSux/OzZs9brCxculLldT5F79ZK7QwAAAAAAuElpv/N6EtvfuXNzc53WLskdN8vKytKlS/nJiQYNGpRYt0aNGgoMDFR6enqhhExpGjZsWOa64eHhZa4LAAAAAIC7NfzY3RFUzMWLF9W4cWOntMW2LDe7du2a9bpatWql1g8MDJQkHocOAAAAAAAksXLH7bKysqzXfn5+pdavUqWKJCkzM7PMfZS2yicrK0vHjh3T3XffrTp16pS4NQye48KFC9aVVnv27FG9evXcHBHuNIxBuBtjEO7GGIS7MQbhbozB8svNzdXFixclSW3btnVau/wW72b+/v7W6xs3bpRaPzs7W5IUEBBQ5j5K2+4l5Z/3A/OqV69emb5noLIwBuFujEG4G2MQ7sYYhLsxBsvOWVuxbLEty83uuusu63VZtlqlp6dLKtsWLgAAAAAAcPsjueNm/v7+ql27tqTST/hOS0uzJnfKc0gyAAAAAAC4fZHc8QBhYWGSpBMnTpT4KLRjx44VuQcAAAAAANzZSO54gIiICEn5W6727dtnt962bdus1127dq30uAAAAAAAgOcjueMB+vbta71eunRpsXXy8vK0fPlySVJwcLCioqJcERoAAAAAAPBwJHc8QHh4uP74xz9KkhYvXqxdu3YVqfP222/r6NGjkqS//vWv8vX1dWmMAAAAAADAM/EodA/x3nvvqWvXrsrMzFTPnj01bdo0RUVFKTMzUytWrNDChQslSaGhoZowYYKbowUAAAAAAJ7CYhiG4e4gkC82NlZ/+ctfdPXq1WLLQ0NDFRcXp2bNmrk4MgAAAAAA4KlI7niY5ORkvffee4qLi9O5c+fk5+enZs2a6YknntBzzz2nqlWrujtEAAAAAADgQUjuAAAAAAAAmBgHKgMAAAAAAJgYyR0AAAAAAAATI7kDAAAAAABgYiR3AAAAAAAATIzkDgAAAAAAgImR3AEAAAAAADAxkjsAAAAAAAAmRnIHAAAAAADAxEjuACa1fv16WSwW68/MmTMdai8hIaFQeyX9ONoXbg/OHoO2du3apcGDB6tx48by9/dXvXr19Oijj2rFihVO6wPmERcXp5kzZyomJkZhYWGqXbu2fH19VaNGDXXs2FETJkzQzz//7HA/zIOwx1Vj0BbzIGydOXNGH3/8sZ588km1aNFCgYGB8vf3V4MGDdSnTx99+eWXys3Ndbgf5kHY46oxaIt5sJwMAKZz/fp1o1GjRoYk68+MGTMcajM+Pr5QeyX9ONoXzK8yxmCBWbNmGV5eXnbHX+/evY3MzEyn9AXPl5OTU6Z5ydfX13jzzTcd6ot5EMVx5RgswDwIW9OnTzcsFkupY/CBBx4wkpOTHeqLeRDFceUYLMA8WH4+ZU8DAfAU06dPV3JysurWravU1FSnt79kyRJ16tTJbnndunWd3ifMpbLG4KJFizRjxgxJ0r333qtp06apbdu2SklJ0Xvvvaf4+HjFxsZq5MiR+uc//+m0fuHZgoKC9PDDD6tz585q2rSp6tWrp6pVqyolJUUJCQlasmSJrly5oqlTpyo4OFjPPvusw30yD8KWK8cg8yBulZKSIsMwFBgYqH79+ik6OlrNmzeXv7+/jh49qvfff1+JiYnau3evunfvrv3796tatWoO98s8iAKuHoPMgxXk7uwSgPLZt2+f4e3tbVSpUsVYuHBhpazciY+Pd0qsuD1V1hhMS0szgoODDUnGPffcY1y8eLFQeW5urtG7d29rf9u2bXOoP5hHbm5uieWnTp0yatSoYUgy6tSpU2p9e5gHYY+rxiDzIIozadIkY+7cucbVq1eLLc/NzTUGDBhgHRf/+Z//WeG+mAdRHFeOQebBiuPMHcBEbt68qVGjRunmzZuaNm2amjdv7u6QcIepzDH42Wef6fLly5KkuXPnqnbt2oXKvb29tWDBAnl7e0uS5s2b57S+4dkKvnN7mjRpoieffFKSdPHiRR07dswVYeEO4qoxyDyI4sydO1eTJk3SXXfdVWx5wbjw8/OTJK1atcqV4eEO4MoxyDxYcSR3ABN59913tX//foWGhmry5MnuDgd3oMocg2vXrpUkVa9eXf379y+2ToMGDdS9e3dJ0qZNm3T9+nWnxgDzCgwMtF5nZWW5MRLcqZwxBpkHUVG1atXSfffdJ0k6efKkm6PBnchZY5B5sOJI7gAmkZSUZN17umDBAlWpUsXNEeFOU5lj8MaNG9qzZ48k6cEHH7T+5ac4kZGRkqTs7GwlJiY6LQaYV2Zmpr755htJkpeXl0JDQ90cEe40zhiDzINwVHZ2tqT8MQi4g6NjkHnQMfyXD5jEmDFjlJGRoUGDBik6OrpS+5o2bZoaNGggPz8/1ahRQx06dNCLL76o48ePV2q/8GyVOQZ/+eUX6+MzW7ZsWWJd2/KjR486NQ6YR05Ojs6cOaMVK1boD3/4g06cOCFJGj58uN1l4+XBPIjSOHsMMg/CEampqdaxUNr4KSvmQZSHM8Yg86BjSO4AJvDFF19ow4YNCg4O1ttvv13p/e3atUvnz59XTk6OLl++rJ9++knz589XWFiYZs6cKcMwKj0GeJbKHoNnz561Xjdo0KDEug0bNiz2Ptz+kpKSZLFYZLFY5Ofnp0aNGmngwIH66aefJEndu3d32vhkHkRxKnMMMg/CEfPmzbP+UjxgwACntMk8iPJwxhhkHnQMj0IHPNzvv/+uF198UZL05ptv6u677660vurVq6f+/fsrIiJCTZs2lY+Pj86cOaPY2Fh9/vnnysnJ0axZs3Tjxg3Nnj270uKAZ3HFGLx27Zr1urRHZ9qea8Eea0j5+/w//PBDPfHEE6UefFsa5kFUhDPGIPMgKurHH3/U/PnzJeX/Qjx27FiH2mMeRHk5awwyDzqG5A7g4SZOnKjU1FR17txZzzzzTKX106lTJyUnJ8vX17fQ6/fff7/69u2r0aNHq2fPnrpy5YrmzJmjAQMGqH379pUWDzyHK8ag7eGjJe2vllTorJ/MzMxKiQeeqX79+vrXv/4lScrNzdX58+e1YcMGLV68WGPHjtWpU6c0depUWSyWCrXPPIjSVOYYZB5ERfz22296/PHHlZubK4vFomXLlqlq1aoVbo95EOXlzDHIPOgYtmUBTlAwmTn6849//KNQuwkJCVq6dKm8vb31ySefVOoBeYGBgUX+j9xWeHi4PvroI0mSYRjWa3gGs49Bf39/6/WNGzdKrFtwWJ8kBQQEVEo8KL/KGoO2fH191aZNG7Vp00bt27dXTEyMPvjgA+3evVsWi0WvvPKKRowYUeH3wDxobmYfg8yD5ueKMWjr2rVriomJ0blz5yRJs2fPVrdu3Rx6D8yD5mb2Mcg86BiSO4CHys7O1ujRoyVJL7zwgkf8VeTJJ59UUFCQJGnbtm1ujgaVzZVj0Pbw0dKW1qanp1uvS1uyizvDfffdp9dff12StHTpUm3cuLHS+mIeRHGcMQaZB1EeWVlZ6tOnj/bt2ydJeumllzRlyhSX9M08CKlyxiDzoGPYlgU4gY+Pj1NOaa9Xr571evXq1Tp+/Lh8fHzUqlUrrVixokj9I0eOWK8PHz5srdO5c2c1adLE4Xhu5ePjo9DQUCUmJur8+fNObx8VZ/YxaHtoXsFff+yxPTTP9jA9uFdljMHy6NOnj3WP/6pVq9SzZ0+HYykO86DnMvsYZB40P1eNwdzcXA0YMEDx8fGSpJEjR7rkgRsFmAc9l9nHIPOgY0juAE7irMdOFihYapibm6tRo0aVWv/rr7/W119/LSn/r4aVkdyRxJMRPJiZx2BoaKi8vb118+ZNHTt2rMS6tuVhYWFl7gOVz9ljsDzq1KljvU5OTq7UvpgHPZeZxyDz4O2hssdgXl6eBg8erNjYWEn5q2g+/fTTSu2zOMyDnsvMY5B50DFsywJQZrm5uTp+/LgkKSQkxM3R4Hbi5+en8PBwSfmPXi1pn3XBEvAqVarogQcecEl88Hy2fz2uzOXZzIOwx9ExyDyIshg9erR1lexjjz2mzz//vFLPZCwO8+CdrTLHIPOgY0juAB5q2LBhMgyjxJ+CpZCSNGPGDOvrw4YNq5SYVqxYoatXr0qSIiMjK6UPeA5Xj8G+fftKkq5evarVq1cXW+fcuXPavHmzJCk6OrrQ3mzc2VauXGm9btu2baX1wzwIe5wxBpkHUZKXXnpJixYtkpT/3a9atarEw48rC/PgncsVY5B50AEGANOKj483JBmSjBkzZtitd/r0aWu9yMjIIuW///67ER8fX2JfP/74oxEcHGxIMiwWi5GYmOhY8LgtOGsMGoZh/N///Z8RFBRkSDIaNWpkXLp0qVB5bm6u0bt3b2s7W7dudeI7gadas2aNkZKSUmKdbdu2GdWqVTMkGT4+PsaxY8eK1GEeREW5agwaBvMg7JsxY4b1e//DH/5gXL9+vdxtMA/CEa4Yg4bBPOgIztwBoCtXrigqKkr33Xef+vbtq44dO6pevXry9vbWmTNnFBsbq88//1w5OTmSpIkTJ7L8EU5Xs2ZNzZ07V88++6ySk5PVuXNnvfLKK2rbtq1SUlI0f/5860qhgQMHKioqys0RwxXWrl2rJ598UjExMYqOjlbr1q0VHBys7OxsnTx5UrGxsfrqq6+Ul5cnSZo+fbpatGhR7n6YB2GPq8agxDyI4n3wwQeaNWuWJKl+/fr6+9//rtOnT5d4T4sWLcq9ooJ5EPa4agxKzIOOILkDwOrQoUM6dOiQ3XJvb29Nnz5dr776qgujwp1k9OjRSklJ0WuvvaaTJ0/q6aefLlKnV69eWrJkiRuig7vcuHFDa9as0Zo1a+zWCQgI0GuvvaYJEyY41BfzIIrjyjHIPIhbFTysQMo/2ykiIqLUe06fPq3GjRtXqD/mQdzK1WOQebBiSO4AUEhIiFauXKldu3Zpz549On/+vC5duqSsrCwFBQWpRYsWevjhhzVy5MgKT9JAWc2aNUuPPPKIPvroI23fvl2//fabgoOD1a5dOw0fPlwDBw50d4hwobfeeku9evXS1q1btX//fv36669KTU2Vl5eXatasqdatW6tbt24aMmRIhR9hLTEPwj5XjUFbzINwB+ZBeBLmwfKzGAbPsQMAAAAAADArnpYFAAAAAABgYiR3AAAAAAAATIzkDgAAAAAAgImR3AEAAAAAADAxkjsAAAAAAAAmRnIHAAAAAADAxEjuAAAAAAAAmBjJHQAAAAAAABMjuQMAAAAAAGBiJHcAAAAAAABMjOQOAAAAAACAiZHcAQAAAAAAMDGSOwAAAAAAACZGcgcAAAAAAMDESO4AAAAAAACYGMkdAAAAAAAAEyO5AwAAcAuLxSKLxaKZM2dW6P7GjRvLYrFo2LBhTo0LAACgOD7uDgAAAJhXQkKCoqKiirzu7e2t6tWrKygoSA0bNlTHjh0VERGh3r17y8/Pzw2Rwhl++OEHLVu2TLt27dK5c+eUlZWl2rVrq27dumratKkeeughRUZGql27dvLy4m+IAAC4CskdAADgdDdv3lRaWprS0tKUlJSk7du3a/78+apTp45eeOEFTZkyRT4+/DPELDIyMjR06FCtWrWqSNmFCxd04cIFHTx4UGvWrJEkrV+/Xo8++qirwwQA4I7Fv6oAAIBTjBkzRmPHjrX+7+vXrystLU2HDh3Sli1btHnzZl28eFHTp09XbGys1q1bpzp16rgx4sqTlJTk7hCc6oknntB3330nSWrWrJlGjRqlTp06qUaNGkpPT9cvv/yinTt36ttvv1VqaqqbowUA4M5DcgcAADhF3bp11aZNmyKv//u//7smT56s//3f/9XgwYN14MAB7dmzR/3799eWLVvYpuXh1q9fb03sPPLII/rmm29UpUqVQnW6du2qYcOG6ZNPPtHatWvVsGFDd4QKAMAdi83QAADAJVq3bq2dO3eqQ4cOkqQdO3ZowYIFbo4KpVm7dq31+u233y6S2LHl7e2tP/3pT2rdurULIgMAAAVI7gAAAJcJCAjQ559/LovFIkl66623lJOTY7f+jh07NHjwYDVu3Fj+/v4KDg5Whw4d9Le//U0XL14sU59JSUmaPHmyOnbsqFq1asnf319NmjRRVFSU3n77bZ05c6bc78MwDD333HPWp2qNGTNGeXl51vKSnpaVkJBgvS8hIUGS9NVXXyk6Olp16tRRQECAWrRooUmTJun3338vNZbk5GQ9++yz1s8oJCREffv2VXx8vCRp5syZ1v4qIjk52XrdrFmzCrVxq507d2rkyJFq0aKFqlevrmrVqqlly5bq27evli9frqtXr9q9NzY2Vo8//rgaNGigKlWqqFatWnrwwQc1Z84cXb9+3e59//jHP6yfQ1JSkrKzszV//nx16dJFtWvXtvt0tD179mjUqFEKDQ1VtWrVFBgYqJYtW2rcuHH65ZdfnPFxAADgOAMAAKCC4uPjDUmGJGPGjBllvq9nz57W+3bu3Fmk/ObNm8a4ceOsdYr7CQoKMjZu3FhiP/PmzTN8fX1LbCcyMrLIfSW9p5ycHOOpp56y1pk6dWqROo0aNTIkGUOHDi1SZvuZbd68uVBbt/40a9bMuHDhgt33t3HjRiMwMLDYey0Wi/HGG28YM2bMsL5WEb1797bef+DAgQq1USAjI8MYOHBgid+Hvc89MzPT6NevX4n3hYSE2I1x6dKl1nqJiYlG+/btS+w3JyfHGDNmTIn9+fr6GgsXLnToMwEAwBlYuQMAAFyue/fu1uvt27cXKZ8yZYo++ugjSVKTJk30ySefaM+ePYqPj9eLL74oX19fXblyRY899pgOHjxYbB+vvfaaXn75ZeXk5Cg4OFjTpk3Tpk2btH//fm3dulVvvfWWunbtWq4VLZmZmerTp4+++OILSfkrj2bPnl2et17Iq6++qi+++EJ9+/bV6tWrtW/fPn333XeKiYmRJJ04cUIvvvhisfeeOHFCffv2VXp6ury9vTVu3Dht2bJFiYmJWrp0qcLCwvTKK69o/fr1FY5PknUbnSQ999xzZV4xdau8vDz16dNHX375pSSpefPmevfdd7V9+3bt27dP69at07Rp0+yuDho6dKj1aVzt2rXT8uXLlZiYqO+//17Dhw+XxWJRSkqKoqOjdf78+RJjGTFihA4ePKghQ4YoLi5O+/bt05o1a9S5c+dCdT7++GNJ+edG/fOf/9SePXuUmJiozz77TK1bt1ZOTo6eeeYZxcbGVugzAQDAadydXQIAAOZV0ZU7mzdvtt739NNPFyo7dOiQ4eXlZUgy2rRpY6SlpRW5f/369dY64eHhRcr37dtnLQ8NDTXOnj1rN5biyop7T5cvXzYiIiIMSYa3t7exZMkSu22WdeWOJOP1118vUicvL8+6usnHx8dITU0tUuc//uM/rG2sXLmySHl6eroRHh5eqK+KSE5ONqpWrWptIyAgwOjfv78xf/58Y9euXUZWVlaZ2pk/f761jX79+tm97+bNm8b58+cLvbZu3TrrvdHR0UZ2dnaR+xYuXGitM2DAgCLltit3JBmLFy+2G+uqVaus9T777LNi62RmZhrdunUzJBmNGzc2cnJySnr7AABUKlbuAAAAl6tVq5b1Oi0trVDZxx9/bD2/5rPPPlNwcHCR+x999FE9/fTTkmRdTWFr3rx5ysvLk8Vi0YoVK9SgQQO7sZRUVuC3335TZGSkduzYoSpVqmjlypUaPnx4qfeVpmPHjpo2bVqR1y0Wi1566SVJUm5urnbt2lWo/Pz581q3bp0kqV+/fnr88ceLtFG1alUtXLjQ4Rjvuece/fd//7eqVasmKX/10urVqzV+/Hg9+OCDCgoK0kMPPaR3333X7hlBeXl5mjdvniSpfv36Wr58ud2Dmb28vBQSElLotYJVXL6+vlq6dGmxT1gbNWqUdUXY6tWrdeHCBbvvqVu3btbxU5w333xTUv5nO3LkyGLr+Pv768MPP5SUf65TwflJAAC4A8kdAADgcgWJAkm6du1aobLNmzdLklq1aqUuXbrYbWPUqFFF7pHyEwkbNmyQJEVGRhbaVlQRSUlJioiI0MGDB1WtWjXFxcWpX79+DrVZ4KmnnrK7Laxjx47W61OnThUqS0hIsCbAhg4darf9du3aqV27dg7H+dhjj+nIkSMaN26catasWagsOztb27dv10svvaR7771Xy5cvL3L/Tz/9ZN0qNWrUqELff2lyc3O1bds2SVKPHj1KfMx6wZjIzc0tMdkyaNAgu2Xnz5/Xvn37JEkDBgwoMbawsDDVrl1bkook4AAAcCWSOwAAwOVsEzrVq1e3XmdnZ1ufQGR7/klxOnToIF9fX0nS4cOHra+fPn1aly9fliQ99NBDDsV59OhRRURE6MSJE6pVq5a2bNmi6Ohoh9q01bJlS7tltkmUWxNgtu/XNglUnAceeKCC0RXWsGFDffjhh/rtt9/0448/6v3339fw4cPVvHlza53Lly9r6NChWrp0aaF7Dxw4YL0u73dy6tQpZWRkSCp9TNiW235Gt7rvvvvslu3du9d6PXDgQOsTtuz9XLp0SZL066+/lun9AABQGUjuAAAAlyv4hVgqnMSw3aJ19913l9iGr6+vdXuX7XYg27br1avnUJxfffWVdcXJxx9/rPDwcIfau1XVqlXtlnl5/f9/pt28ebNQme3nVLdu3RL7qFOnTgWjK56Pj4/Cw8P1/PPPa8mSJTp+/Lj27t2riIgIa50JEyYUSkg58p3YfreljYl/+7d/K/a+W9WoUcNuWWpqajmi+/8KElAAALiDj7sDAAAAdx7blRwtWrQotk5ZnmJlGEaJ5eV5ElZxHnnkEe3YsUPp6el67rnn1Lp1a7Vq1cqhNm9HHTt21IYNG9S+fXudOHFCaWlp2rx5c7Hb1xz5Thz9Pgt4e3vbLbNNpP3Xf/1Xiat8bJWUMAIAoLKR3AEAAC63adMm67Xtig/bX5BL2+aSm5trXZ1hu/qn4AwUSUpJSXEozi5dumjq1Knq1auXUlNTFR0drYSEBLsJKVex/ZxSU1NLPBS6oo8uL6/AwEANHDhQr732mqT8R7UXuPU7Kc/nZ/vdljYmbMtvPRuorGwP+7ZYLGrTpk2F2gEAwJXYlgUAAFzq8OHD2rJli6T8c1xsz4SpUqWK9QyXH3/8scR2Dhw4oJycHEkq9At4kyZNrMmPH374weF4IyMjtW7dOgUEBOjXX39VVFSU9Vwgd2ndurX12vaMmOKUVu5Mtk+5st1Wdv/991uvy/udNG3a1Lp9rbQxsWfPHut1RZMytgdwb9y4sUJtAADgaiR3AACAy2RmZmrIkCHW7VQTJ06Uj0/hhcQFj7M+cuSIdu/ebbetRYsWFblHyk8q9OrVS5K0bdu2QlvAKioqKkqxsbEKCAjQhQsXFBUVVWhliqtFRUVZkyfFPZ2qwMGDB3Xw4EGH+ipt65st20RSkyZNrNft2rWzPuVq0aJFun79epnb9PHxUWRkpKT8FV9nz561W7dgTHh7e+vhhx8ucx+2mjVrZt16t2LFCp05c6ZC7QAA4EokdwAAgEscOXJEERER1mRLZGSkxowZU6TemDFjrImLZ555RleuXClSZ+PGjVq8eLEkKTw8XJ06dSpUPnHiRHl5eckwDP35z3/WuXPn7MZVUpmt6OhoffPNN/L399f58+fVrVu3Io8od5X69esrJiZGkrRmzRqtWrWqSJ3MzEw988wzDvc1ZswYzZ49u8QDiqX8xMuyZcsk5W/RujXh9vLLL0vK/7yHDBmiGzduFNtOXl5eke1048aNkyTl5OTo6aefLvbeJUuWWFfa/OlPf3LoMO2//e1vkqSsrCz179+/xK1t2dnZWrBggbKysircHwAAjuLMHQAA4BSpqamFHj+dnp6utLQ0HTp0SFu2bNGmTZusq0C6dOmiVatWWR9lbqtt27aaMGGC5s2bp3/961+6//77NXnyZHXo0EEZGRmKjY3V+++/r5s3b8rPz0+ffvppkTbat2+vWbNmafr06Tp+/Ljatm2rcePGKSoqSrVq1dLly5f1008/afXq1fL29lZ8fHyZ3mOPHj20Zs0a9e3bV2fPnlW3bt20bds2NWrUqIKfWsW988472rJlizIyMvTnP/9ZY8aMUf/+/VW9enUdPnxYf//733XkyBF16tRJiYmJFe7n0qVL+vTTTzVr1iz16tVLDz/8sFq3bq1atWopNzdXJ06c0LfffquvvvpKeXl5kqTXX3+90CPupfwETWxsrDZt2qQ1a9aobdu2Gjt2rB544AFVrVpVv/76q3bv3q0vv/xSTz31lGbOnGm9NyYmRk888YRWrlypzZs3q3PnzpowYYLCwsKUlpamFStWaMmSJZLyz9p55513Kvx+pfxHoH///fdatmyZ9u3bp1atWmn06NGKjIxUnTp1lJ6erpMnT2r79u1avXq1fv/9dw0ZMsShPgEAcIgBAABQQfHx8YakMv/UqVPHeOONN4ycnJwS271586YxduzYEtsKCgoyvv/++xLbmT17tuHj41NiO5GRkUXuKyibMWNGse3GxcUZfn5+hiSjSZMmRnJycqHyRo0aGZKMoUOHlviZxcfHlxh/aXFs3LjRCAwMtPveZsyYYUyfPt2QZPj7+5fYlz0vvPBCmb9ff39/Y+7cuXbbSk9PNx5//PFS2ynu/WZmZhr9+vUr8b6QkBDjwIEDxfa9dOlSa73Tp0+X+r5zc3ONSZMmGd7e3qXGGxgYaGRkZJTxEwUAwPnYlgUAAJzOy8tLQUFBuueee/THP/5R48eP19dff61z585p2rRpRc7ZKe7+jz76SD/88IMGDRqke+65R1WqVFH16tXVvn17TZs2Tb/88ot69uxZYjtTp07VkSNHNH78eLVp00bVq1eXv7+/mjZtqujoaM2fP18rVqwo9/vr1auXvv76a/n5+en06dPq1q1bmbd3OVOPHj10+PBhjR49Wo0aNZKfn5/uvvtuxcTEaMOGDZo5c6auXr0qSQoKCqpQH++9956Sk5P16aef6i9/+Yvat2+vGjVqyMfHRwEBAQoJCVH37t31xhtv6Pjx45o0aZLdtqpWraqVK1dq69atGjx4sJo0aaKAgADdddddatmypfr3768vvvjCuoXLlr+/v1avXq1vv/1W/fv3V0hIiPz8/FSjRg117txZb775pn7++We1b9++Qu/zVt7e3po7d66OHDmiCRMmqEOHDqpRo4a8vb111113qXXr1ho0aJCWLVumCxcuKCAgwCn9AgBQERbDKMcpeQAAADCV7t27a8uWLYqIiND27dvdHQ4AAKgErNwBAAC4TaWkpFgfPd6lSxc3RwMAACoLyR0AAACTKulx7JmZmRo2bJhycnIkiQN/AQC4jfG0LAAAAJMaOXKk0tPTNWDAAHXs2FE1a9bUtWvXtHfvXi1YsMCa/BkxYoTatm3r5mgBAEBlIbkDAABgYnv37tXevXvtlvfr108ffPCBCyMCAACuxoHKAAAAJrV//36tWbNGW7du1blz53Tx4kUZhqG6deuqS5cuGjJkiGJiYtwdJgAAqGQkdwAAAAAAAEyMA5UBAAAAAABMjOQOAAAAAACAiZHcAQAAAAAAMDGSOwAAAAAAACZGcgcAAAAAAMDESO4AAAAAAACYGMkdAAAAAAAAEyO5AwAAAAAAYGIkdwAAAAAAAEyM5A4AAAAAAICJkdwBAAAAAAAwMZI7AAAAAAAAJkZyBwAAAAAAwMRI7gAAAAAAAJgYyR0AAAAAAAATI7kDAAAAAABgYiR3AAAAAAAATIzkDgAAAAAAgImR3AEAAAAAADCx/weTjaZnvH/tiQAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<Figure size 640x480 with 1 Axes>"
      ]
     },
     "metadata": {
      "image/png": {
       "height": 454,
       "width": 571
      }
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Generate plot of all scores\n",
    "plt.hist(docking_scores, bins=25)\n",
    "plt.xlabel('Docking Score')\n",
    "plt.ylabel('Frequency')\n",
    "plt.title('Histogram of Docking Scores')\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "300838b5-5c6c-43f6-9dcd-20977982bdcc",
   "metadata": {},
   "source": [
    "------------\n",
    "#### Visual Inspection of Top-Ranking Molecules\n",
    "\n",
    "After utilizing SMINA to score the binding affinities of the sampled molecules, I proceeded to analyze the top-ranking molecules to better understand their structural characteristics and potential interactions with the target protein. Visual inspection of these high-scoring ligands can provide valuable insights into the binding modes and molecular features that contribute to their predicted binding affinities."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "1eb99ed4-a2d7-4bf8-90db-60c307fa38a4",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16835771918631198\"  style=\"position: relative; width: 1000px; height: 1000px\">\n        <p id=\"3dmolwarning_16835771918631198\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n}\n\nvar viewer_16835771918631198 = null;\nvar warn = document.getElementById(\"3dmolwarning_16835771918631198\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nvar viewergrid_16835771918631198 = null;\nviewergrid_16835771918631198 = $3Dmol.createViewerGrid(document.getElementById(\"3dmolviewer_16835771918631198\"),{rows: 2, cols: 2, control_all: false},{backgroundColor:\"white\"});\nviewer_16835771918631198 = viewergrid_16835771918631198[0][0];\nviewergrid_16835771918631198[0][0].zoomTo();viewergrid_16835771918631198[0][1].zoomTo();viewergrid_16835771918631198[1][0].zoomTo();viewergrid_16835771918631198[1][1].zoomTo();\tviewergrid_16835771918631198[0][0].setViewStyle({\"style\": \"outline\", \"color\": \"black\", \"width\": 0.1});\n\tviewergrid_16835771918631198[0][1].setViewStyle({\"style\": \"outline\", \"color\": \"black\", \"width\": 0.1});\n\tviewergrid_16835771918631198[1][0].setViewStyle({\"style\": \"outline\", \"color\": \"black\", \"width\": 0.1});\n\tviewergrid_16835771918631198[1][1].setViewStyle({\"style\": \"outline\", \"color\": \"black\", \"width\": 0.1});\n\tviewergrid_16835771918631198[0][0].addModel(\"HEADER    ONCOGENE PROTEIN                        30-APR-90   5P21              \\nTITLE     REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION OF H-RAS   \\nTITLE    2 P21 AT 1.35 ANGSTROMS RESOLUTION: IMPLICATIONS FOR THE MECHANISM OF  \\nTITLE    3 GTP HYDROLYSIS                                                       \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: C-H-RAS P21 PROTEIN;                                       \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    ONCOGENE PROTEIN                                                      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    E.F.PAI,A.WITTINGHOFER,W.KABSCH                                       \\nREVDAT   6   29-NOV-17 5P21    1       HELIX                                    \\nREVDAT   5   24-FEB-09 5P21    1       VERSN                                    \\nREVDAT   4   01-APR-03 5P21    1       JRNL                                     \\nREVDAT   3   20-JUL-95 5P21    1       HET                                      \\nREVDAT   2   30-APR-94 5P21    1       FORMUL                                   \\nREVDAT   1   15-JAN-92 5P21    0                                                \\nJRNL        AUTH   E.F.PAI,U.KRENGEL,G.A.PETSKO,R.S.GOODY,W.KABSCH,             \\nJRNL        AUTH 2 A.WITTINGHOFER                                               \\nJRNL        TITL   REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION   \\nJRNL        TITL 2 OF H-RAS P21 AT 1.35 A RESOLUTION: IMPLICATIONS FOR THE      \\nJRNL        TITL 3 MECHANISM OF GTP HYDROLYSIS.                                 \\nJRNL        REF    EMBO J.                       V.   9  2351 1990              \\nJRNL        REFN                   ISSN 0261-4189                               \\nJRNL        PMID   2196171                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,S.C.ALMO,G.RAPP,K.WILSON,K.PETRATOS,A.LENTFER, \\nREMARK   1  AUTH 2 A.WITTINGHOFER,W.KABSCH,E.F.PAI,G.A.PETSKO,R.S.GOODY         \\nREMARK   1  TITL   TIME-RESOLVED X-RAY CRYSTALLOGRAPHIC STUDY OF THE            \\nREMARK   1  TITL 2 CONFORMATIONAL CHANGE IN HA-RAS P21 PROTEIN ON GTP           \\nREMARK   1  TITL 3 HYDROLYSIS                                                   \\nREMARK   1  REF    NATURE                        V. 345   309 1990              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   A.SCHERER,J.JOHN,R.LINKE,R.S.GOODY,A.WITTINGHOFER,E.F.PAI,   \\nREMARK   1  AUTH 2 K.C.HOLMES                                                   \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY ANALYSIS OF THE HUMAN  \\nREMARK   1  TITL 2 C-H-RAS-ONCOGENE PRODUCT P21 COMPLEXED WITH GTP ANALOGUES    \\nREMARK   1  REF    J.MOL.BIOL.                   V. 206   257 1989              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   E.F.PAI,W.KABSCH,U.KRENGEL,K.C.HOLMES,J.JOHN,A.WITTINGHOFER  \\nREMARK   1  TITL   STRUCTURE OF THE GUANINE-NUCLEOTIDE-BINDING DOMAIN OF THE    \\nREMARK   1  TITL 2 HA-RAS ONCOGENE PRODUCT P21 IN THE TRIPHOSPHATE CONFORMATION \\nREMARK   1  REF    NATURE                        V. 341   209 1989              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,G.RAPP,J.JOHN,A.WITTINGHOFER,E.F.PAI,R.S.GOODY \\nREMARK   1  TITL   BIOCHEMICAL AND CRYSTALLOGRAPHIC CHARACTERIZATION OF A       \\nREMARK   1  TITL 2 COMPLEX OF C-HA-RAS P21 AND CAGED GTP WITH FLASH PHOTOLYSIS  \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  86  7687 1989              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 6.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 26806                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.196                           \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1323                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 33                                      \\nREMARK   3   SOLVENT ATOMS            : 211                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.014                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 2.700                           \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  RESIDUES 61 - 65 (GLN - GLU - GLU - TYR - SER) ADOPT                \\nREMARK   3  SEVERAL CONFORMATIONS IN THE CRYSTAL.  THE COORDINATES              \\nREMARK   3  GIVEN APPROXIMATE ONE OF THESE.                                     \\nREMARK   4                                                                      \\nREMARK   4 5P21 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179747.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 38.93                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.01                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 32 2 1                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+2/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+1/3                                           \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z+1/3                                           \\nREMARK 290       6555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000      108.13333            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       54.06667            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       54.06667            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000      108.13333            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025  0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    ARG A  68   NE  -  CZ  -  NH1 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A  97   NE  -  CZ  -  NH1 ANGL. DEV. =  -3.4 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH1 ANGL. DEV. =   4.1 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500    ARG A 128   NE  -  CZ  -  NH2 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A 135   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500    TYR A 157   CB  -  CG  -  CD2 ANGL. DEV. =  -4.8 DEGREES          \\nREMARK 500    ARG A 164   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ASP A  33       75.86   -157.25                                   \\nREMARK 500    ILE A  36      -64.23   -100.39                                   \\nREMARK 500    GLU A  37      123.75   -171.30                                   \\nREMARK 500    SER A  65     -133.70     53.07                                   \\nREMARK 500    LYS A 117       33.80     71.60                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              MG A 168  MG                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 GNP A 167   O2G                                                    \\nREMARK 620 2 GNP A 167   O2B  95.0                                              \\nREMARK 620 3 HOH A 173   O    87.6  91.7                                        \\nREMARK 620 4 SER A  17   OG  172.9  91.0  88.6                                  \\nREMARK 620 5 THR A  35   OG1  91.3 173.4  90.5  82.8                            \\nREMARK 620 6 HOH A 172   O    97.3  88.0 175.0  86.5  89.3                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE MG A 168                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE GNP A 167                 \\nDBREF  5P21 A    1   166  UNP    P01112   RASH_HUMAN       1    166             \\nSEQRES   1 A  166  MET THR GLU TYR LYS LEU VAL VAL VAL GLY ALA GLY GLY          \\nSEQRES   2 A  166  VAL GLY LYS SER ALA LEU THR ILE GLN LEU ILE GLN ASN          \\nSEQRES   3 A  166  HIS PHE VAL ASP GLU TYR ASP PRO THR ILE GLU ASP SER          \\nSEQRES   4 A  166  TYR ARG LYS GLN VAL VAL ILE ASP GLY GLU THR CYS LEU          \\nSEQRES   5 A  166  LEU ASP ILE LEU ASP THR ALA GLY GLN GLU GLU TYR SER          \\nSEQRES   6 A  166  ALA MET ARG ASP GLN TYR MET ARG THR GLY GLU GLY PHE          \\nSEQRES   7 A  166  LEU CYS VAL PHE ALA ILE ASN ASN THR LYS SER PHE GLU          \\nSEQRES   8 A  166  ASP ILE HIS GLN TYR ARG GLU GLN ILE LYS ARG VAL LYS          \\nSEQRES   9 A  166  ASP SER ASP ASP VAL PRO MET VAL LEU VAL GLY ASN LYS          \\nSEQRES  10 A  166  CYS ASP LEU ALA ALA ARG THR VAL GLU SER ARG GLN ALA          \\nSEQRES  11 A  166  GLN ASP LEU ALA ARG SER TYR GLY ILE PRO TYR ILE GLU          \\nSEQRES  12 A  166  THR SER ALA LYS THR ARG GLN GLY VAL GLU ASP ALA PHE          \\nSEQRES  13 A  166  TYR THR LEU VAL ARG GLU ILE ARG GLN HIS                      \\nHET     MG  A 168       1                                                       \\nHET    GNP  A 167      32                                                       \\nHETNAM      MG MAGNESIUM ION                                                    \\nHETNAM     GNP PHOSPHOAMINOPHOSPHONIC ACID-GUANYLATE ESTER                      \\nFORMUL   2   MG    MG 2+                                                        \\nFORMUL   3  GNP    C10 H17 N6 O13 P3                                            \\nFORMUL   4  HOH   *211(H2 O)                                                    \\nHELIX    1  HA LYS A   16  GLN A   25  1                                  10    \\nHELIX    2  HB SER A   65  THR A   74  1RES 65-67 IN 3/10 CONF            10    \\nHELIX    3  HC THR A   87  VAL A  103  1BREAK AT ASP 92                   17    \\nHELIX    4  HD SER A  127  TYR A  137  1                                  11    \\nHELIX    5  HE VAL A  152  ARG A  164  1                                  13    \\nSHEET    1  S1 6 GLU A  37  ILE A  46  0                                        \\nSHEET    2  S1 6 GLU A  49  THR A  58 -1  O  LEU A  53   N  LYS A  42           \\nSHEET    3  S1 6 THR A   2  VAL A   9  1  N  LEU A   6   O  ASP A  54           \\nSHEET    4  S1 6 GLY A  77  ALA A  83  1  N  LEU A  79   O  VAL A   7           \\nSHEET    5  S1 6 MET A 111  ASN A 116  1  N  VAL A 114   O  CYS A  80           \\nSHEET    6  S1 6 TYR A 141  GLU A 143  1  N  ILE A 142   O  LEU A 113           \\nLINK        MG    MG A 168                 O2G GNP A 167     1555   1555  2.20  \\nLINK        MG    MG A 168                 O2B GNP A 167     1555   1555  2.22  \\nLINK        MG    MG A 168                 O   HOH A 173     1555   1555  2.23  \\nLINK        MG    MG A 168                 OG  SER A  17     1555   1555  2.26  \\nLINK        MG    MG A 168                 OG1 THR A  35     1555   1555  2.26  \\nLINK        MG    MG A 168                 O   HOH A 172     1555   1555  2.30  \\nSITE     1 AC1  5 SER A  17  THR A  35  GNP A 167  HOH A 172                    \\nSITE     2 AC1  5 HOH A 173                                                     \\nSITE     1 AC2 28 GLY A  12  GLY A  13  VAL A  14  GLY A  15                    \\nSITE     2 AC2 28 LYS A  16  SER A  17  ALA A  18  PHE A  28                    \\nSITE     3 AC2 28 VAL A  29  ASP A  30  TYR A  32  PRO A  34                    \\nSITE     4 AC2 28 THR A  35  GLY A  60  ASN A 116  LYS A 117                    \\nSITE     5 AC2 28 ASP A 119  SER A 145  ALA A 146  LYS A 147                    \\nSITE     6 AC2 28  MG A 168  HOH A 170  HOH A 172  HOH A 173                    \\nSITE     7 AC2 28 HOH A 175  HOH A 187  HOH A 281  HOH A 366                    \\nCRYST1   40.300   40.300  162.200  90.00  90.00 120.00 P 32 2 1      6          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.024814  0.014326  0.000000        0.00000                         \\nSCALE2      0.000000  0.028653  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.006165        0.00000                         \\nATOM      1  N   MET A   1      -7.186  32.862  -6.632  1.00 13.92           N  \\nATOM      2  CA  MET A   1      -5.921  32.127  -6.684  1.00 17.10           C  \\nATOM      3  C   MET A   1      -5.845  31.045  -5.621  1.00 16.74           C  \\nATOM      4  O   MET A   1      -6.600  31.083  -4.636  1.00 16.60           O  \\nATOM      5  CB  MET A   1      -4.673  33.058  -6.502  1.00 17.00           C  \\nATOM      6  CG  MET A   1      -4.633  33.804  -5.188  1.00 18.08           C  \\nATOM      7  SD  MET A   1      -3.115  34.766  -4.908  1.00 19.73           S  \\nATOM      8  CE  MET A   1      -3.019  35.851  -6.310  1.00 20.41           C  \\nATOM      9  N   THR A   2      -4.910  30.095  -5.820  1.00 14.70           N  \\nATOM     10  CA  THR A   2      -4.661  29.030  -4.859  1.00 16.92           C  \\nATOM     11  C   THR A   2      -4.230  29.655  -3.547  1.00 14.78           C  \\nATOM     12  O   THR A   2      -3.338  30.501  -3.537  1.00 13.93           O  \\nATOM     13  CB  THR A   2      -3.557  28.107  -5.401  1.00 18.03           C  \\nATOM     14  OG1 THR A   2      -4.059  27.565  -6.629  1.00 20.42           O  \\nATOM     15  CG2 THR A   2      -3.193  26.983  -4.443  1.00 18.51           C  \\nATOM     16  N   GLU A   3      -4.911  29.290  -2.472  1.00 14.80           N  \\nATOM     17  CA  GLU A   3      -4.504  29.730  -1.135  1.00 17.71           C  \\nATOM     18  C   GLU A   3      -4.012  28.553  -0.278  1.00 17.33           C  \\nATOM     19  O   GLU A   3      -4.520  27.427  -0.395  1.00 17.54           O  \\nATOM     20  CB  GLU A   3      -5.655  30.402  -0.432  1.00 20.67           C  \\nATOM     21  CG  GLU A   3      -6.213  31.631  -1.186  1.00 25.56           C  \\nATOM     22  CD  GLU A   3      -7.128  32.563  -0.369  1.00 28.46           C  \\nATOM     23  OE1 GLU A   3      -7.578  32.216   0.728  1.00 27.72           O  \\nATOM     24  OE2 GLU A   3      -7.375  33.680  -0.827  1.00 32.49           O  \\nATOM     25  N   TYR A   4      -2.962  28.744   0.542  1.00 14.50           N  \\nATOM     26  CA  TYR A   4      -2.425  27.654   1.358  1.00 12.29           C  \\nATOM     27  C   TYR A   4      -2.534  28.103   2.807  1.00 12.52           C  \\nATOM     28  O   TYR A   4      -2.016  29.143   3.200  1.00 12.11           O  \\nATOM     29  CB  TYR A   4      -0.975  27.385   1.026  1.00 11.46           C  \\nATOM     30  CG  TYR A   4      -0.726  26.846  -0.387  1.00 12.38           C  \\nATOM     31  CD1 TYR A   4      -0.297  27.698  -1.395  1.00 13.00           C  \\nATOM     32  CD2 TYR A   4      -0.933  25.501  -0.649  1.00 14.09           C  \\nATOM     33  CE1 TYR A   4      -0.070  27.208  -2.674  1.00 13.89           C  \\nATOM     34  CE2 TYR A   4      -0.711  24.993  -1.917  1.00 14.65           C  \\nATOM     35  CZ  TYR A   4      -0.279  25.859  -2.917  1.00 14.84           C  \\nATOM     36  OH  TYR A   4      -0.084  25.357  -4.191  1.00 15.07           O  \\nATOM     37  N   LYS A   5      -3.274  27.305   3.568  1.00 13.07           N  \\nATOM     38  CA  LYS A   5      -3.554  27.569   4.970  1.00 12.96           C  \\nATOM     39  C   LYS A   5      -2.556  26.818   5.852  1.00 12.74           C  \\nATOM     40  O   LYS A   5      -2.645  25.611   6.092  1.00 11.58           O  \\nATOM     41  CB  LYS A   5      -4.972  27.131   5.281  1.00 17.65           C  \\nATOM     42  CG  LYS A   5      -6.053  27.833   4.446  1.00 23.49           C  \\nATOM     43  CD  LYS A   5      -7.186  28.310   5.335  1.00 29.10           C  \\nATOM     44  CE  LYS A   5      -6.655  29.309   6.403  1.00 33.68           C  \\nATOM     45  NZ  LYS A   5      -6.597  28.739   7.755  1.00 36.52           N  \\nATOM     46  N   LEU A   6      -1.543  27.574   6.297  1.00 13.08           N  \\nATOM     47  CA  LEU A   6      -0.418  27.011   7.064  1.00 10.34           C  \\nATOM     48  C   LEU A   6      -0.578  27.382   8.535  1.00  9.68           C  \\nATOM     49  O   LEU A   6      -1.034  28.483   8.863  1.00 11.31           O  \\nATOM     50  CB  LEU A   6       0.910  27.571   6.536  1.00 11.34           C  \\nATOM     51  CG  LEU A   6       1.137  27.565   5.017  1.00 11.62           C  \\nATOM     52  CD1 LEU A   6       2.515  28.119   4.720  1.00 13.60           C  \\nATOM     53  CD2 LEU A   6       0.973  26.171   4.475  1.00 12.19           C  \\nATOM     54  N   VAL A   7      -0.196  26.486   9.432  1.00  9.93           N  \\nATOM     55  CA  VAL A   7      -0.333  26.787  10.852  1.00 10.06           C  \\nATOM     56  C   VAL A   7       1.033  26.513  11.495  1.00 10.37           C  \\nATOM     57  O   VAL A   7       1.597  25.441  11.252  1.00  9.51           O  \\nATOM     58  CB  VAL A   7      -1.474  25.884  11.446  1.00 10.97           C  \\nATOM     59  CG1 VAL A   7      -1.610  26.163  12.943  1.00 11.39           C  \\nATOM     60  CG2 VAL A   7      -2.809  26.161  10.777  1.00 10.14           C  \\nATOM     61  N   VAL A   8       1.576  27.455  12.295  1.00  8.35           N  \\nATOM     62  CA  VAL A   8       2.878  27.262  12.944  1.00  6.92           C  \\nATOM     63  C   VAL A   8       2.609  26.896  14.405  1.00  8.84           C  \\nATOM     64  O   VAL A   8       1.930  27.663  15.092  1.00  9.21           O  \\nATOM     65  CB  VAL A   8       3.684  28.565  12.799  1.00  8.08           C  \\nATOM     66  CG1 VAL A   8       5.111  28.324  13.261  1.00  9.52           C  \\nATOM     67  CG2 VAL A   8       3.729  29.011  11.358  1.00  8.43           C  \\nATOM     68  N   VAL A   9       3.024  25.701  14.881  1.00  8.73           N  \\nATOM     69  CA  VAL A   9       2.779  25.234  16.245  1.00  8.41           C  \\nATOM     70  C   VAL A   9       4.075  24.806  16.919  1.00  8.21           C  \\nATOM     71  O   VAL A   9       5.093  24.551  16.287  1.00 10.97           O  \\nATOM     72  CB  VAL A   9       1.824  24.028  16.286  1.00  9.74           C  \\nATOM     73  CG1 VAL A   9       0.545  24.491  15.680  1.00 11.76           C  \\nATOM     74  CG2 VAL A   9       2.349  22.810  15.535  1.00 10.31           C  \\nATOM     75  N   GLY A  10       4.048  24.707  18.238  1.00  7.34           N  \\nATOM     76  CA  GLY A  10       5.231  24.336  19.009  1.00  7.98           C  \\nATOM     77  C   GLY A  10       5.192  25.053  20.355  1.00  8.31           C  \\nATOM     78  O   GLY A  10       4.417  25.994  20.595  1.00  9.89           O  \\nATOM     79  N   ALA A  11       6.099  24.638  21.226  1.00  9.31           N  \\nATOM     80  CA  ALA A  11       6.121  25.135  22.597  1.00  9.50           C  \\nATOM     81  C   ALA A  11       6.440  26.625  22.711  1.00  9.66           C  \\nATOM     82  O   ALA A  11       6.998  27.250  21.806  1.00  9.46           O  \\nATOM     83  CB  ALA A  11       7.145  24.306  23.378  1.00  9.85           C  \\nATOM     84  N   GLY A  12       6.111  27.227  23.847  1.00 10.36           N  \\nATOM     85  CA  GLY A  12       6.406  28.632  24.054  1.00  9.30           C  \\nATOM     86  C   GLY A  12       7.892  28.950  23.964  1.00 10.23           C  \\nATOM     87  O   GLY A  12       8.785  28.249  24.466  1.00 10.57           O  \\nATOM     88  N   GLY A  13       8.159  30.064  23.266  1.00 11.49           N  \\nATOM     89  CA  GLY A  13       9.511  30.602  23.139  1.00 11.29           C  \\nATOM     90  C   GLY A  13      10.436  29.950  22.108  1.00 12.22           C  \\nATOM     91  O   GLY A  13      11.624  30.321  22.060  1.00 13.62           O  \\nATOM     92  N   VAL A  14       9.961  28.991  21.286  1.00  9.23           N  \\nATOM     93  CA  VAL A  14      10.848  28.327  20.317  1.00 10.38           C  \\nATOM     94  C   VAL A  14      11.169  29.168  19.076  1.00 10.47           C  \\nATOM     95  O   VAL A  14      12.108  28.841  18.353  1.00 12.07           O  \\nATOM     96  CB  VAL A  14      10.246  26.970  19.856  1.00  9.98           C  \\nATOM     97  CG1 VAL A  14      10.187  26.074  21.120  1.00  9.93           C  \\nATOM     98  CG2 VAL A  14       8.866  27.127  19.200  1.00  9.52           C  \\nATOM     99  N   GLY A  15      10.423  30.264  18.834  1.00  9.61           N  \\nATOM    100  CA  GLY A  15      10.666  31.160  17.706  1.00  9.70           C  \\nATOM    101  C   GLY A  15       9.615  31.061  16.626  1.00  8.82           C  \\nATOM    102  O   GLY A  15       9.915  31.403  15.491  1.00  8.99           O  \\nATOM    103  N   LYS A  16       8.395  30.591  16.916  1.00  8.18           N  \\nATOM    104  CA  LYS A  16       7.331  30.521  15.913  1.00  8.44           C  \\nATOM    105  C   LYS A  16       7.060  31.899  15.309  1.00  9.01           C  \\nATOM    106  O   LYS A  16       7.020  32.044  14.086  1.00  8.25           O  \\nATOM    107  CB  LYS A  16       6.014  30.000  16.529  1.00  9.27           C  \\nATOM    108  CG  LYS A  16       6.132  28.585  17.086  1.00  9.11           C  \\nATOM    109  CD  LYS A  16       4.820  28.077  17.726  1.00  9.31           C  \\nATOM    110  CE  LYS A  16       4.351  28.934  18.888  1.00 10.05           C  \\nATOM    111  NZ  LYS A  16       5.221  28.829  20.051  1.00  9.64           N  \\nATOM    112  N   SER A  17       6.913  32.942  16.135  1.00  7.74           N  \\nATOM    113  CA  SER A  17       6.661  34.295  15.649  1.00  8.78           C  \\nATOM    114  C   SER A  17       7.854  34.859  14.919  1.00  9.73           C  \\nATOM    115  O   SER A  17       7.653  35.446  13.868  1.00  9.70           O  \\nATOM    116  CB  SER A  17       6.360  35.264  16.769  1.00  8.22           C  \\nATOM    117  OG  SER A  17       5.083  34.853  17.204  1.00  8.79           O  \\nATOM    118  N   ALA A  18       9.072  34.687  15.444  1.00  8.95           N  \\nATOM    119  CA  ALA A  18      10.253  35.199  14.769  1.00  8.95           C  \\nATOM    120  C   ALA A  18      10.438  34.576  13.374  1.00  8.54           C  \\nATOM    121  O   ALA A  18      10.832  35.254  12.430  1.00  8.75           O  \\nATOM    122  CB  ALA A  18      11.521  34.931  15.640  1.00 10.11           C  \\nATOM    123  N   LEU A  19      10.136  33.284  13.176  1.00  8.31           N  \\nATOM    124  CA  LEU A  19      10.225  32.667  11.837  1.00  9.30           C  \\nATOM    125  C   LEU A  19       9.204  33.281  10.884  1.00 10.98           C  \\nATOM    126  O   LEU A  19       9.534  33.652   9.751  1.00 10.47           O  \\nATOM    127  CB  LEU A  19       9.938  31.165  11.885  1.00  9.41           C  \\nATOM    128  CG  LEU A  19      11.035  30.275  12.487  1.00 11.80           C  \\nATOM    129  CD1 LEU A  19      10.491  28.866  12.723  1.00 10.43           C  \\nATOM    130  CD2 LEU A  19      12.256  30.288  11.554  1.00 13.42           C  \\nATOM    131  N   THR A  20       7.950  33.419  11.335  1.00 10.72           N  \\nATOM    132  CA  THR A  20       6.888  34.015  10.529  1.00 11.87           C  \\nATOM    133  C   THR A  20       7.173  35.492  10.161  1.00 12.65           C  \\nATOM    134  O   THR A  20       7.012  35.917   9.009  1.00 12.60           O  \\nATOM    135  CB  THR A  20       5.547  33.888  11.312  1.00 13.43           C  \\nATOM    136  OG1 THR A  20       5.299  32.509  11.544  1.00 12.67           O  \\nATOM    137  CG2 THR A  20       4.346  34.410  10.518  1.00 14.64           C  \\nATOM    138  N   ILE A  21       7.615  36.326  11.098  1.00 10.48           N  \\nATOM    139  CA  ILE A  21       7.945  37.704  10.795  1.00 12.96           C  \\nATOM    140  C   ILE A  21       9.197  37.831   9.927  1.00 14.26           C  \\nATOM    141  O   ILE A  21       9.237  38.691   9.042  1.00 15.83           O  \\nATOM    142  CB  ILE A  21       8.090  38.451  12.121  1.00 12.57           C  \\nATOM    143  CG1 ILE A  21       6.691  38.551  12.737  1.00 13.24           C  \\nATOM    144  CG2 ILE A  21       8.631  39.852  11.910  1.00 13.40           C  \\nATOM    145  CD1 ILE A  21       6.636  39.185  14.129  1.00 16.46           C  \\nATOM    146  N   GLN A  22      10.206  36.979  10.092  1.00 13.58           N  \\nATOM    147  CA  GLN A  22      11.330  36.983   9.179  1.00 14.91           C  \\nATOM    148  C   GLN A  22      10.817  36.729   7.736  1.00 17.29           C  \\nATOM    149  O   GLN A  22      11.120  37.485   6.792  1.00 15.91           O  \\nATOM    150  CB  GLN A  22      12.270  35.930   9.736  1.00 15.90           C  \\nATOM    151  CG  GLN A  22      13.686  35.878   9.230  1.00 19.36           C  \\nATOM    152  CD  GLN A  22      14.372  37.217   9.091  1.00 18.08           C  \\nATOM    153  OE1 GLN A  22      14.160  37.984   8.143  1.00 19.23           O  \\nATOM    154  NE2 GLN A  22      15.224  37.492  10.050  1.00 17.40           N  \\nATOM    155  N   LEU A  23       9.944  35.718   7.523  1.00 15.95           N  \\nATOM    156  CA  LEU A  23       9.381  35.475   6.194  1.00 18.85           C  \\nATOM    157  C   LEU A  23       8.539  36.622   5.636  1.00 18.21           C  \\nATOM    158  O   LEU A  23       8.695  36.958   4.469  1.00 19.03           O  \\nATOM    159  CB  LEU A  23       8.498  34.235   6.196  1.00 18.10           C  \\nATOM    160  CG  LEU A  23       7.916  33.797   4.863  1.00 18.64           C  \\nATOM    161  CD1 LEU A  23       8.921  32.994   4.030  1.00 18.03           C  \\nATOM    162  CD2 LEU A  23       6.680  32.991   5.183  1.00 21.34           C  \\nATOM    163  N   ILE A  24       7.634  37.197   6.428  1.00 19.25           N  \\nATOM    164  CA  ILE A  24       6.682  38.198   5.953  1.00 20.23           C  \\nATOM    165  C   ILE A  24       7.316  39.590   5.922  1.00 21.89           C  \\nATOM    166  O   ILE A  24       7.125  40.318   4.943  1.00 22.93           O  \\nATOM    167  CB  ILE A  24       5.399  38.222   6.881  1.00 21.20           C  \\nATOM    168  CG1 ILE A  24       4.664  36.907   6.821  1.00 21.27           C  \\nATOM    169  CG2 ILE A  24       4.423  39.307   6.446  1.00 20.00           C  \\nATOM    170  CD1 ILE A  24       4.128  36.442   5.475  1.00 23.98           C  \\nATOM    171  N   GLN A  25       8.074  39.998   6.953  1.00 21.07           N  \\nATOM    172  CA  GLN A  25       8.520  41.380   7.066  1.00 22.09           C  \\nATOM    173  C   GLN A  25      10.003  41.559   6.962  1.00 20.75           C  \\nATOM    174  O   GLN A  25      10.495  42.677   7.054  1.00 20.26           O  \\nATOM    175  CB  GLN A  25       8.039  41.968   8.398  1.00 23.96           C  \\nATOM    176  CG  GLN A  25       6.548  42.276   8.381  1.00 28.69           C  \\nATOM    177  CD  GLN A  25       5.931  42.574   9.744  1.00 30.92           C  \\nATOM    178  OE1 GLN A  25       6.537  43.143  10.657  1.00 31.89           O  \\nATOM    179  NE2 GLN A  25       4.664  42.169   9.867  1.00 32.35           N  \\nATOM    180  N   ASN A  26      10.764  40.482   6.795  1.00 22.13           N  \\nATOM    181  CA  ASN A  26      12.198  40.642   6.628  1.00 24.67           C  \\nATOM    182  C   ASN A  26      12.936  41.362   7.734  1.00 23.62           C  \\nATOM    183  O   ASN A  26      13.603  42.372   7.521  1.00 25.10           O  \\nATOM    184  CB  ASN A  26      12.522  41.369   5.315  1.00 27.68           C  \\nATOM    185  CG  ASN A  26      12.942  40.351   4.288  1.00 32.80           C  \\nATOM    186  OD1 ASN A  26      12.135  39.699   3.602  1.00 33.77           O  \\nATOM    187  ND2 ASN A  26      14.268  40.203   4.237  1.00 34.26           N  \\nATOM    188  N   HIS A  27      12.764  40.842   8.945  1.00 19.95           N  \\nATOM    189  CA  HIS A  27      13.593  41.266  10.057  1.00 17.99           C  \\nATOM    190  C   HIS A  27      13.441  40.299  11.215  1.00 16.84           C  \\nATOM    191  O   HIS A  27      12.411  39.623  11.344  1.00 14.72           O  \\nATOM    192  CB  HIS A  27      13.241  42.680  10.551  1.00 19.80           C  \\nATOM    193  CG  HIS A  27      11.926  42.908  11.258  1.00 20.77           C  \\nATOM    194  ND1 HIS A  27      11.936  43.314  12.510  1.00 22.17           N  \\nATOM    195  CD2 HIS A  27      10.674  42.768  10.729  1.00 22.32           C  \\nATOM    196  CE1 HIS A  27      10.657  43.430  12.785  1.00 23.86           C  \\nATOM    197  NE2 HIS A  27       9.905  43.110  11.738  1.00 23.97           N  \\nATOM    198  N   PHE A  28      14.516  40.222  12.012  1.00 15.44           N  \\nATOM    199  CA  PHE A  28      14.528  39.413  13.223  1.00 13.89           C  \\nATOM    200  C   PHE A  28      14.026  40.196  14.440  1.00 15.33           C  \\nATOM    201  O   PHE A  28      14.528  41.255  14.786  1.00 16.80           O  \\nATOM    202  CB  PHE A  28      15.938  38.906  13.470  1.00 12.73           C  \\nATOM    203  CG  PHE A  28      16.083  38.150  14.786  1.00 13.35           C  \\nATOM    204  CD1 PHE A  28      15.354  36.990  15.012  1.00 13.80           C  \\nATOM    205  CD2 PHE A  28      16.974  38.615  15.740  1.00 14.54           C  \\nATOM    206  CE1 PHE A  28      15.534  36.310  16.194  1.00 14.45           C  \\nATOM    207  CE2 PHE A  28      17.147  37.925  16.920  1.00 14.61           C  \\nATOM    208  CZ  PHE A  28      16.420  36.768  17.137  1.00 14.65           C  \\nATOM    209  N   VAL A  29      12.970  39.697  15.089  1.00 17.22           N  \\nATOM    210  CA  VAL A  29      12.467  40.313  16.294  1.00 19.11           C  \\nATOM    211  C   VAL A  29      13.127  39.606  17.476  1.00 19.36           C  \\nATOM    212  O   VAL A  29      13.245  38.376  17.605  1.00 18.14           O  \\nATOM    213  CB  VAL A  29      10.894  40.205  16.412  1.00 22.03           C  \\nATOM    214  CG1 VAL A  29      10.334  40.869  15.143  1.00 19.66           C  \\nATOM    215  CG2 VAL A  29      10.377  38.776  16.631  1.00 23.15           C  \\nATOM    216  N   ASP A  30      13.615  40.484  18.324  1.00 20.14           N  \\nATOM    217  CA  ASP A  30      14.229  40.019  19.518  1.00 24.04           C  \\nATOM    218  C   ASP A  30      13.676  40.746  20.721  1.00 25.93           C  \\nATOM    219  O   ASP A  30      14.244  41.709  21.238  1.00 29.26           O  \\nATOM    220  CB  ASP A  30      15.719  40.203  19.443  1.00 27.10           C  \\nATOM    221  CG  ASP A  30      16.389  39.731  20.732  1.00 32.46           C  \\nATOM    222  OD1 ASP A  30      15.878  38.859  21.464  1.00 34.28           O  \\nATOM    223  OD2 ASP A  30      17.458  40.264  21.002  1.00 36.46           O  \\nATOM    224  N   GLU A  31      12.472  40.347  21.076  1.00 26.14           N  \\nATOM    225  CA  GLU A  31      11.922  40.804  22.318  1.00 28.72           C  \\nATOM    226  C   GLU A  31      10.870  39.836  22.749  1.00 26.20           C  \\nATOM    227  O   GLU A  31      10.466  38.931  22.014  1.00 23.14           O  \\nATOM    228  CB  GLU A  31      11.337  42.210  22.204  1.00 33.25           C  \\nATOM    229  CG  GLU A  31      10.350  42.618  21.152  1.00 40.08           C  \\nATOM    230  CD  GLU A  31      10.572  44.107  20.887  1.00 44.71           C  \\nATOM    231  OE1 GLU A  31      10.930  44.444  19.755  1.00 47.50           O  \\nATOM    232  OE2 GLU A  31      10.420  44.928  21.806  1.00 47.89           O  \\nATOM    233  N   TYR A  32      10.550  40.044  24.028  1.00 22.26           N  \\nATOM    234  CA  TYR A  32       9.597  39.196  24.685  1.00 18.83           C  \\nATOM    235  C   TYR A  32       8.204  39.688  24.317  1.00 17.53           C  \\nATOM    236  O   TYR A  32       7.794  40.792  24.676  1.00 18.45           O  \\nATOM    237  CB  TYR A  32       9.909  39.259  26.199  1.00 16.84           C  \\nATOM    238  CG  TYR A  32       9.076  38.309  27.052  1.00 15.64           C  \\nATOM    239  CD1 TYR A  32       9.598  37.096  27.433  1.00 16.17           C  \\nATOM    240  CD2 TYR A  32       7.814  38.686  27.459  1.00 15.54           C  \\nATOM    241  CE1 TYR A  32       8.845  36.251  28.239  1.00 17.86           C  \\nATOM    242  CE2 TYR A  32       7.051  37.861  28.257  1.00 14.40           C  \\nATOM    243  CZ  TYR A  32       7.572  36.635  28.652  1.00 16.47           C  \\nATOM    244  OH  TYR A  32       6.821  35.789  29.485  1.00 13.59           O  \\nATOM    245  N   ASP A  33       7.461  38.841  23.604  1.00 13.43           N  \\nATOM    246  CA  ASP A  33       6.128  39.200  23.141  1.00 14.00           C  \\nATOM    247  C   ASP A  33       5.262  37.955  22.854  1.00 13.56           C  \\nATOM    248  O   ASP A  33       5.024  37.586  21.681  1.00 12.47           O  \\nATOM    249  CB  ASP A  33       6.271  40.072  21.864  1.00 15.92           C  \\nATOM    250  CG  ASP A  33       4.993  40.654  21.269  1.00 18.19           C  \\nATOM    251  OD1 ASP A  33       3.923  40.631  21.879  1.00 21.46           O  \\nATOM    252  OD2 ASP A  33       5.074  41.164  20.167  1.00 20.34           O  \\nATOM    253  N   PRO A  34       4.752  37.283  23.903  1.00 12.04           N  \\nATOM    254  CA  PRO A  34       3.949  36.070  23.788  1.00 11.09           C  \\nATOM    255  C   PRO A  34       2.720  36.248  22.895  1.00 11.76           C  \\nATOM    256  O   PRO A  34       1.953  37.220  23.006  1.00 12.63           O  \\nATOM    257  CB  PRO A  34       3.590  35.711  25.219  1.00 12.50           C  \\nATOM    258  CG  PRO A  34       4.643  36.418  26.033  1.00 12.14           C  \\nATOM    259  CD  PRO A  34       4.817  37.717  25.311  1.00 12.82           C  \\nATOM    260  N   THR A  35       2.554  35.282  21.972  1.00 11.00           N  \\nATOM    261  CA  THR A  35       1.421  35.320  21.033  1.00 11.54           C  \\nATOM    262  C   THR A  35       0.140  34.746  21.639  1.00 13.51           C  \\nATOM    263  O   THR A  35       0.174  33.820  22.450  1.00 12.45           O  \\nATOM    264  CB  THR A  35       1.730  34.518  19.751  1.00 11.34           C  \\nATOM    265  OG1 THR A  35       2.900  35.134  19.233  1.00 11.20           O  \\nATOM    266  CG2 THR A  35       0.706  34.611  18.637  1.00 13.10           C  \\nATOM    267  N   ILE A  36      -0.996  35.332  21.233  1.00 15.48           N  \\nATOM    268  CA  ILE A  36      -2.308  34.792  21.567  1.00 19.42           C  \\nATOM    269  C   ILE A  36      -2.663  34.079  20.274  1.00 17.36           C  \\nATOM    270  O   ILE A  36      -2.749  32.863  20.256  1.00 18.29           O  \\nATOM    271  CB  ILE A  36      -3.353  35.914  21.879  1.00 21.30           C  \\nATOM    272  CG1 ILE A  36      -2.930  36.823  23.072  1.00 24.32           C  \\nATOM    273  CG2 ILE A  36      -4.670  35.207  22.136  1.00 21.54           C  \\nATOM    274  CD1 ILE A  36      -2.494  36.167  24.413  1.00 24.70           C  \\nATOM    275  N   GLU A  37      -2.818  34.822  19.181  1.00 19.35           N  \\nATOM    276  CA  GLU A  37      -3.059  34.221  17.889  1.00 21.47           C  \\nATOM    277  C   GLU A  37      -2.957  35.362  16.897  1.00 23.77           C  \\nATOM    278  O   GLU A  37      -3.636  36.393  17.031  1.00 25.49           O  \\nATOM    279  CB  GLU A  37      -4.440  33.611  17.880  1.00 25.97           C  \\nATOM    280  CG  GLU A  37      -4.766  32.776  16.667  1.00 33.81           C  \\nATOM    281  CD  GLU A  37      -6.160  32.154  16.777  1.00 38.37           C  \\nATOM    282  OE1 GLU A  37      -7.095  32.664  16.149  1.00 37.86           O  \\nATOM    283  OE2 GLU A  37      -6.298  31.164  17.515  1.00 42.24           O  \\nATOM    284  N   ASP A  38      -2.070  35.212  15.911  1.00 19.96           N  \\nATOM    285  CA  ASP A  38      -1.899  36.178  14.839  1.00 17.94           C  \\nATOM    286  C   ASP A  38      -2.024  35.526  13.473  1.00 18.21           C  \\nATOM    287  O   ASP A  38      -1.582  34.393  13.294  1.00 16.96           O  \\nATOM    288  CB  ASP A  38      -0.540  36.833  14.965  1.00 18.83           C  \\nATOM    289  CG  ASP A  38      -0.507  37.817  16.133  1.00 20.08           C  \\nATOM    290  OD1 ASP A  38      -1.390  38.660  16.208  1.00 25.47           O  \\nATOM    291  OD2 ASP A  38       0.375  37.781  16.967  1.00 20.08           O  \\nATOM    292  N   SER A  39      -2.698  36.170  12.507  1.00 17.82           N  \\nATOM    293  CA  SER A  39      -2.691  35.658  11.133  1.00 18.16           C  \\nATOM    294  C   SER A  39      -2.012  36.627  10.171  1.00 16.95           C  \\nATOM    295  O   SER A  39      -2.053  37.853  10.333  1.00 15.80           O  \\nATOM    296  CB  SER A  39      -4.094  35.402  10.637  1.00 23.01           C  \\nATOM    297  OG  SER A  39      -4.873  36.571  10.754  1.00 27.13           O  \\nATOM    298  N   TYR A  40      -1.336  36.080   9.168  1.00 15.44           N  \\nATOM    299  CA  TYR A  40      -0.607  36.889   8.196  1.00 16.25           C  \\nATOM    300  C   TYR A  40      -1.018  36.383   6.830  1.00 17.06           C  \\nATOM    301  O   TYR A  40      -1.160  35.172   6.638  1.00 18.29           O  \\nATOM    302  CB  TYR A  40       0.909  36.724   8.370  1.00 16.59           C  \\nATOM    303  CG  TYR A  40       1.439  37.242   9.707  1.00 16.10           C  \\nATOM    304  CD1 TYR A  40       1.473  36.412  10.815  1.00 16.77           C  \\nATOM    305  CD2 TYR A  40       1.883  38.549   9.799  1.00 18.02           C  \\nATOM    306  CE1 TYR A  40       1.950  36.893  12.021  1.00 16.86           C  \\nATOM    307  CE2 TYR A  40       2.368  39.032  11.001  1.00 18.70           C  \\nATOM    308  CZ  TYR A  40       2.396  38.199  12.104  1.00 17.62           C  \\nATOM    309  OH  TYR A  40       2.853  38.695  13.304  1.00 18.37           O  \\nATOM    310  N   ARG A  41      -1.270  37.303   5.900  1.00 15.48           N  \\nATOM    311  CA  ARG A  41      -1.660  36.918   4.558  1.00 15.42           C  \\nATOM    312  C   ARG A  41      -0.730  37.606   3.586  1.00 16.76           C  \\nATOM    313  O   ARG A  41      -0.536  38.829   3.633  1.00 17.01           O  \\nATOM    314  CB  ARG A  41      -3.087  37.342   4.277  1.00 17.64           C  \\nATOM    315  CG  ARG A  41      -4.150  36.507   4.947  1.00 18.09           C  \\nATOM    316  CD  ARG A  41      -5.528  37.181   4.849  1.00 19.64           C  \\nATOM    317  NE  ARG A  41      -6.066  37.318   3.489  1.00 20.11           N  \\nATOM    318  CZ  ARG A  41      -6.660  36.329   2.803  1.00 22.17           C  \\nATOM    319  NH1 ARG A  41      -7.219  36.571   1.617  1.00 22.48           N  \\nATOM    320  NH2 ARG A  41      -6.711  35.090   3.274  1.00 23.55           N  \\nATOM    321  N   LYS A  42      -0.135  36.827   2.679  1.00 16.52           N  \\nATOM    322  CA  LYS A  42       0.835  37.405   1.758  1.00 16.59           C  \\nATOM    323  C   LYS A  42       0.740  36.746   0.387  1.00 16.04           C  \\nATOM    324  O   LYS A  42       0.794  35.528   0.307  1.00 14.74           O  \\nATOM    325  CB  LYS A  42       2.208  37.210   2.401  1.00 18.22           C  \\nATOM    326  CG  LYS A  42       3.388  37.784   1.669  1.00 22.68           C  \\nATOM    327  CD  LYS A  42       3.369  39.274   1.792  1.00 28.09           C  \\nATOM    328  CE  LYS A  42       4.695  39.876   1.317  1.00 31.70           C  \\nATOM    329  NZ  LYS A  42       4.828  39.827  -0.128  1.00 35.70           N  \\nATOM    330  N   GLN A  43       0.561  37.510  -0.697  1.00 16.89           N  \\nATOM    331  CA  GLN A  43       0.579  36.950  -2.047  1.00 19.10           C  \\nATOM    332  C   GLN A  43       2.026  36.729  -2.448  1.00 15.08           C  \\nATOM    333  O   GLN A  43       2.828  37.654  -2.362  1.00 15.72           O  \\nATOM    334  CB  GLN A  43      -0.116  37.908  -3.043  1.00 23.32           C  \\nATOM    335  CG  GLN A  43       0.543  38.332  -4.415  1.00 32.83           C  \\nATOM    336  CD  GLN A  43       1.081  37.332  -5.482  1.00 36.11           C  \\nATOM    337  OE1 GLN A  43       0.457  36.335  -5.865  1.00 37.78           O  \\nATOM    338  NE2 GLN A  43       2.271  37.606  -6.053  1.00 35.45           N  \\nATOM    339  N   VAL A  44       2.394  35.532  -2.922  1.00 12.79           N  \\nATOM    340  CA  VAL A  44       3.771  35.242  -3.311  1.00 11.03           C  \\nATOM    341  C   VAL A  44       3.724  34.425  -4.605  1.00 14.28           C  \\nATOM    342  O   VAL A  44       2.633  34.109  -5.090  1.00 14.90           O  \\nATOM    343  CB  VAL A  44       4.508  34.411  -2.223  1.00 13.21           C  \\nATOM    344  CG1 VAL A  44       4.669  35.282  -0.977  1.00 15.04           C  \\nATOM    345  CG2 VAL A  44       3.761  33.116  -1.889  1.00 12.89           C  \\nATOM    346  N   VAL A  45       4.879  34.127  -5.235  1.00 13.20           N  \\nATOM    347  CA  VAL A  45       4.938  33.277  -6.419  1.00 12.28           C  \\nATOM    348  C   VAL A  45       5.836  32.161  -5.970  1.00 11.08           C  \\nATOM    349  O   VAL A  45       6.988  32.447  -5.636  1.00 13.13           O  \\nATOM    350  CB  VAL A  45       5.587  33.979  -7.641  1.00 13.01           C  \\nATOM    351  CG1 VAL A  45       5.549  33.031  -8.810  1.00 14.51           C  \\nATOM    352  CG2 VAL A  45       4.787  35.173  -8.108  1.00 13.54           C  \\nATOM    353  N   ILE A  46       5.374  30.907  -5.915  1.00  9.89           N  \\nATOM    354  CA  ILE A  46       6.274  29.795  -5.491  1.00 11.03           C  \\nATOM    355  C   ILE A  46       6.369  28.800  -6.662  1.00  8.85           C  \\nATOM    356  O   ILE A  46       5.319  28.296  -7.059  1.00 10.25           O  \\nATOM    357  CB  ILE A  46       5.692  29.115  -4.203  1.00 10.64           C  \\nATOM    358  CG1 ILE A  46       5.652  30.096  -3.042  1.00 11.20           C  \\nATOM    359  CG2 ILE A  46       6.599  27.953  -3.757  1.00 12.93           C  \\nATOM    360  CD1 ILE A  46       4.937  29.536  -1.808  1.00  8.90           C  \\nATOM    361  N   ASP A  47       7.565  28.495  -7.224  1.00  9.41           N  \\nATOM    362  CA  ASP A  47       7.767  27.650  -8.426  1.00  9.70           C  \\nATOM    363  C   ASP A  47       6.838  28.031  -9.572  1.00 11.14           C  \\nATOM    364  O   ASP A  47       6.225  27.211 -10.248  1.00 14.09           O  \\nATOM    365  CB  ASP A  47       7.566  26.165  -8.105  1.00 10.73           C  \\nATOM    366  CG  ASP A  47       8.388  25.783  -6.874  1.00 13.52           C  \\nATOM    367  OD1 ASP A  47       7.805  25.291  -5.898  1.00 12.04           O  \\nATOM    368  OD2 ASP A  47       9.598  26.024  -6.872  1.00 11.31           O  \\nATOM    369  N   GLY A  48       6.724  29.353  -9.735  1.00 10.78           N  \\nATOM    370  CA  GLY A  48       5.930  29.961 -10.785  1.00 13.29           C  \\nATOM    371  C   GLY A  48       4.426  29.989 -10.548  1.00 14.28           C  \\nATOM    372  O   GLY A  48       3.677  30.512 -11.386  1.00 15.19           O  \\nATOM    373  N   GLU A  49       3.944  29.422  -9.439  1.00 13.41           N  \\nATOM    374  CA  GLU A  49       2.520  29.457  -9.143  1.00 13.90           C  \\nATOM    375  C   GLU A  49       2.223  30.700  -8.314  1.00 15.34           C  \\nATOM    376  O   GLU A  49       2.860  30.906  -7.275  1.00 14.56           O  \\nATOM    377  CB  GLU A  49       2.124  28.238  -8.340  1.00 15.50           C  \\nATOM    378  CG  GLU A  49       0.630  28.250  -7.990  1.00 18.10           C  \\nATOM    379  CD  GLU A  49       0.204  27.110  -7.100  1.00 19.13           C  \\nATOM    380  OE1 GLU A  49       0.717  27.004  -5.996  1.00 19.53           O  \\nATOM    381  OE2 GLU A  49      -0.630  26.321  -7.529  1.00 22.12           O  \\nATOM    382  N   THR A  50       1.267  31.536  -8.734  1.00 14.06           N  \\nATOM    383  CA  THR A  50       0.901  32.697  -7.927  1.00 16.15           C  \\nATOM    384  C   THR A  50      -0.001  32.169  -6.819  1.00 13.35           C  \\nATOM    385  O   THR A  50      -0.938  31.409  -7.046  1.00 15.13           O  \\nATOM    386  CB  THR A  50       0.203  33.755  -8.846  1.00 17.88           C  \\nATOM    387  OG1 THR A  50      -0.245  34.813  -8.012  1.00 24.33           O  \\nATOM    388  CG2 THR A  50      -1.028  33.236  -9.518  1.00 19.78           C  \\nATOM    389  N   CYS A  51       0.306  32.442  -5.563  1.00 13.28           N  \\nATOM    390  CA  CYS A  51      -0.566  31.959  -4.508  1.00 15.16           C  \\nATOM    391  C   CYS A  51      -0.579  32.892  -3.303  1.00 14.00           C  \\nATOM    392  O   CYS A  51       0.161  33.874  -3.231  1.00 11.81           O  \\nATOM    393  CB  CYS A  51      -0.132  30.537  -4.079  1.00 14.08           C  \\nATOM    394  SG  CYS A  51       1.597  30.409  -3.575  1.00 16.09           S  \\nATOM    395  N   LEU A  52      -1.521  32.609  -2.405  1.00 13.77           N  \\nATOM    396  CA  LEU A  52      -1.573  33.397  -1.199  1.00 16.01           C  \\nATOM    397  C   LEU A  52      -1.260  32.497  -0.025  1.00 14.53           C  \\nATOM    398  O   LEU A  52      -1.868  31.443   0.113  1.00 16.05           O  \\nATOM    399  CB  LEU A  52      -2.957  33.986  -1.014  1.00 19.71           C  \\nATOM    400  CG  LEU A  52      -3.122  34.885   0.214  1.00 20.84           C  \\nATOM    401  CD1 LEU A  52      -3.070  36.329  -0.220  1.00 21.38           C  \\nATOM    402  CD2 LEU A  52      -4.420  34.526   0.923  1.00 20.90           C  \\nATOM    403  N   LEU A  53      -0.319  32.904   0.800  1.00 12.41           N  \\nATOM    404  CA  LEU A  53       0.019  32.232   2.048  1.00 13.70           C  \\nATOM    405  C   LEU A  53      -0.841  32.852   3.148  1.00 13.85           C  \\nATOM    406  O   LEU A  53      -0.733  34.056   3.413  1.00 12.82           O  \\nATOM    407  CB  LEU A  53       1.485  32.464   2.400  1.00 14.51           C  \\nATOM    408  CG  LEU A  53       2.707  31.589   2.031  1.00 19.29           C  \\nATOM    409  CD1 LEU A  53       2.394  30.553   0.962  1.00 18.56           C  \\nATOM    410  CD2 LEU A  53       3.843  32.557   1.741  1.00 20.00           C  \\nATOM    411  N   ASP A  54      -1.699  32.023   3.751  1.00 11.96           N  \\nATOM    412  CA  ASP A  54      -2.476  32.445   4.884  1.00 15.07           C  \\nATOM    413  C   ASP A  54      -1.882  31.675   6.055  1.00 12.67           C  \\nATOM    414  O   ASP A  54      -2.099  30.477   6.234  1.00 13.99           O  \\nATOM    415  CB  ASP A  54      -3.930  32.088   4.644  1.00 19.33           C  \\nATOM    416  CG  ASP A  54      -4.826  32.609   5.746  1.00 24.24           C  \\nATOM    417  OD1 ASP A  54      -4.412  32.652   6.897  1.00 28.17           O  \\nATOM    418  OD2 ASP A  54      -5.958  32.979   5.450  1.00 27.59           O  \\nATOM    419  N   ILE A  55      -1.127  32.389   6.880  1.00 11.78           N  \\nATOM    420  CA  ILE A  55      -0.346  31.769   7.935  1.00 12.15           C  \\nATOM    421  C   ILE A  55      -0.953  32.099   9.296  1.00 13.10           C  \\nATOM    422  O   ILE A  55      -1.195  33.271   9.587  1.00 13.84           O  \\nATOM    423  CB  ILE A  55       1.109  32.282   7.931  1.00 12.42           C  \\nATOM    424  CG1 ILE A  55       1.859  31.979   6.653  1.00 13.99           C  \\nATOM    425  CG2 ILE A  55       1.859  31.505   9.018  1.00 13.24           C  \\nATOM    426  CD1 ILE A  55       3.176  32.768   6.497  1.00 15.68           C  \\nATOM    427  N   LEU A  56      -1.176  31.083  10.134  1.00 11.64           N  \\nATOM    428  CA  LEU A  56      -1.642  31.341  11.490  1.00 11.98           C  \\nATOM    429  C   LEU A  56      -0.524  31.027  12.431  1.00  9.99           C  \\nATOM    430  O   LEU A  56      -0.017  29.895  12.447  1.00  9.72           O  \\nATOM    431  CB  LEU A  56      -2.836  30.463  11.896  1.00 13.80           C  \\nATOM    432  CG  LEU A  56      -3.470  30.741  13.297  1.00 16.13           C  \\nATOM    433  CD1 LEU A  56      -4.347  31.987  13.260  1.00 15.28           C  \\nATOM    434  CD2 LEU A  56      -4.321  29.580  13.692  1.00 16.84           C  \\nATOM    435  N   ASP A  57      -0.188  32.084  13.182  1.00 10.40           N  \\nATOM    436  CA  ASP A  57       0.815  32.014  14.219  1.00 10.30           C  \\nATOM    437  C   ASP A  57       0.127  31.812  15.571  1.00 11.75           C  \\nATOM    438  O   ASP A  57      -0.652  32.631  16.053  1.00 12.91           O  \\nATOM    439  CB  ASP A  57       1.602  33.296  14.173  1.00 10.66           C  \\nATOM    440  CG  ASP A  57       2.778  33.336  15.132  1.00 11.39           C  \\nATOM    441  OD1 ASP A  57       3.306  32.298  15.535  1.00 11.07           O  \\nATOM    442  OD2 ASP A  57       3.168  34.448  15.455  1.00 13.15           O  \\nATOM    443  N   THR A  58       0.447  30.723  16.244  1.00 12.55           N  \\nATOM    444  CA  THR A  58      -0.324  30.325  17.416  1.00 12.98           C  \\nATOM    445  C   THR A  58       0.368  30.541  18.758  1.00 13.48           C  \\nATOM    446  O   THR A  58       1.568  30.812  18.815  1.00 14.00           O  \\nATOM    447  CB  THR A  58      -0.698  28.851  17.219  1.00  9.54           C  \\nATOM    448  OG1 THR A  58       0.491  28.084  17.369  1.00 11.40           O  \\nATOM    449  CG2 THR A  58      -1.313  28.597  15.848  1.00 12.27           C  \\nATOM    450  N   ALA A  59      -0.371  30.382  19.868  1.00 14.38           N  \\nATOM    451  CA  ALA A  59       0.230  30.510  21.194  1.00 14.79           C  \\nATOM    452  C   ALA A  59       0.928  29.210  21.608  1.00 15.41           C  \\nATOM    453  O   ALA A  59       0.448  28.106  21.324  1.00 13.07           O  \\nATOM    454  CB  ALA A  59      -0.845  30.818  22.238  1.00 15.47           C  \\nATOM    455  N   GLY A  60       2.084  29.341  22.271  1.00 17.71           N  \\nATOM    456  CA  GLY A  60       2.807  28.208  22.814  1.00 24.18           C  \\nATOM    457  C   GLY A  60       2.190  27.751  24.130  1.00 31.21           C  \\nATOM    458  O   GLY A  60       2.538  26.688  24.627  1.00 30.44           O  \\nATOM    459  N   GLN A  61       1.260  28.503  24.726  1.00 39.50           N  \\nATOM    460  CA  GLN A  61       0.671  28.099  25.994  1.00 48.84           C  \\nATOM    461  C   GLN A  61      -0.683  27.395  25.985  1.00 51.52           C  \\nATOM    462  O   GLN A  61      -1.606  27.780  25.276  1.00 52.17           O  \\nATOM    463  CB  GLN A  61       0.605  29.327  26.874  1.00 51.56           C  \\nATOM    464  CG  GLN A  61       0.871  28.861  28.312  1.00 56.74           C  \\nATOM    465  CD  GLN A  61       1.506  29.937  29.177  1.00 58.74           C  \\nATOM    466  OE1 GLN A  61       2.647  29.795  29.616  1.00 60.12           O  \\nATOM    467  NE2 GLN A  61       0.783  31.031  29.429  1.00 60.18           N  \\nATOM    468  N   GLU A  62      -0.804  26.349  26.818  1.00 57.40           N  \\nATOM    469  CA  GLU A  62      -2.008  25.527  26.927  1.00 61.96           C  \\nATOM    470  C   GLU A  62      -3.185  26.196  27.631  1.00 62.20           C  \\nATOM    471  O   GLU A  62      -4.138  25.496  27.975  1.00 62.85           O  \\nATOM    472  CB  GLU A  62      -1.629  24.183  27.632  1.00 65.18           C  \\nATOM    473  CG  GLU A  62      -2.622  22.966  27.761  1.00 68.22           C  \\nATOM    474  CD  GLU A  62      -3.032  22.132  26.528  1.00 70.25           C  \\nATOM    475  OE1 GLU A  62      -2.710  22.494  25.395  1.00 71.45           O  \\nATOM    476  OE2 GLU A  62      -3.691  21.101  26.689  1.00 70.00           O  \\nATOM    477  N   GLU A  63      -3.206  27.520  27.904  1.00 62.78           N  \\nATOM    478  CA  GLU A  63      -4.496  28.139  28.254  1.00 63.62           C  \\nATOM    479  C   GLU A  63      -5.457  27.854  27.085  1.00 63.71           C  \\nATOM    480  O   GLU A  63      -6.688  27.814  27.211  1.00 62.97           O  \\nATOM    481  CB  GLU A  63      -4.372  29.666  28.475  1.00 65.13           C  \\nATOM    482  CG  GLU A  63      -3.456  30.553  27.607  1.00 66.91           C  \\nATOM    483  CD  GLU A  63      -4.012  31.235  26.345  1.00 68.05           C  \\nATOM    484  OE1 GLU A  63      -5.220  31.462  26.220  1.00 68.64           O  \\nATOM    485  OE2 GLU A  63      -3.198  31.568  25.481  1.00 67.62           O  \\nATOM    486  N   TYR A  64      -4.801  27.606  25.934  1.00 63.36           N  \\nATOM    487  CA  TYR A  64      -5.420  27.092  24.743  1.00 63.20           C  \\nATOM    488  C   TYR A  64      -5.905  25.653  24.875  1.00 61.76           C  \\nATOM    489  O   TYR A  64      -7.120  25.486  24.858  1.00 63.04           O  \\nATOM    490  CB  TYR A  64      -4.437  27.208  23.613  1.00 64.83           C  \\nATOM    491  CG  TYR A  64      -4.736  28.459  22.818  1.00 66.98           C  \\nATOM    492  CD1 TYR A  64      -4.300  29.686  23.267  1.00 67.55           C  \\nATOM    493  CD2 TYR A  64      -5.459  28.355  21.646  1.00 67.97           C  \\nATOM    494  CE1 TYR A  64      -4.587  30.825  22.536  1.00 69.49           C  \\nATOM    495  CE2 TYR A  64      -5.752  29.488  20.911  1.00 69.29           C  \\nATOM    496  CZ  TYR A  64      -5.313  30.718  21.359  1.00 70.05           C  \\nATOM    497  OH  TYR A  64      -5.589  31.852  20.614  1.00 70.67           O  \\nATOM    498  N   SER A  65      -5.095  24.590  24.997  1.00 58.67           N  \\nATOM    499  CA  SER A  65      -5.617  23.230  25.127  1.00 56.93           C  \\nATOM    500  C   SER A  65      -6.614  22.757  24.063  1.00 55.33           C  \\nATOM    501  O   SER A  65      -6.321  22.936  22.876  1.00 54.74           O  \\nATOM    502  CB  SER A  65      -6.190  23.122  26.552  1.00 57.99           C  \\nATOM    503  OG  SER A  65      -7.251  24.011  26.900  1.00 59.80           O  \\nATOM    504  N   ALA A  66      -7.762  22.119  24.359  1.00 52.33           N  \\nATOM    505  CA  ALA A  66      -8.766  21.824  23.335  1.00 48.80           C  \\nATOM    506  C   ALA A  66      -9.138  23.050  22.492  1.00 45.71           C  \\nATOM    507  O   ALA A  66      -9.751  22.862  21.449  1.00 44.33           O  \\nATOM    508  CB  ALA A  66     -10.043  21.300  23.971  1.00 49.14           C  \\nATOM    509  N   MET A  67      -8.829  24.310  22.875  1.00 42.85           N  \\nATOM    510  CA  MET A  67      -8.984  25.465  21.967  1.00 40.01           C  \\nATOM    511  C   MET A  67      -8.093  25.362  20.719  1.00 36.79           C  \\nATOM    512  O   MET A  67      -8.221  26.184  19.805  1.00 36.08           O  \\nATOM    513  CB  MET A  67      -8.594  26.804  22.581  1.00 42.50           C  \\nATOM    514  CG  MET A  67      -9.359  27.330  23.772  1.00 45.93           C  \\nATOM    515  SD  MET A  67     -11.140  27.396  23.483  1.00 48.74           S  \\nATOM    516  CE  MET A  67     -11.526  25.880  24.321  1.00 47.64           C  \\nATOM    517  N   ARG A  68      -7.128  24.418  20.665  1.00 31.59           N  \\nATOM    518  CA  ARG A  68      -6.322  24.217  19.467  1.00 26.34           C  \\nATOM    519  C   ARG A  68      -7.140  23.473  18.424  1.00 23.68           C  \\nATOM    520  O   ARG A  68      -6.846  23.617  17.231  1.00 21.93           O  \\nATOM    521  CB  ARG A  68      -5.057  23.394  19.745  1.00 25.86           C  \\nATOM    522  CG  ARG A  68      -4.161  23.898  20.869  1.00 29.36           C  \\nATOM    523  CD  ARG A  68      -2.881  23.061  20.998  1.00 32.88           C  \\nATOM    524  NE  ARG A  68      -2.221  22.990  22.315  1.00 36.64           N  \\nATOM    525  CZ  ARG A  68      -1.700  24.031  23.001  1.00 39.66           C  \\nATOM    526  NH1 ARG A  68      -1.903  25.315  22.664  1.00 42.06           N  \\nATOM    527  NH2 ARG A  68      -0.933  23.798  24.071  1.00 39.61           N  \\nATOM    528  N   ASP A  69      -8.188  22.721  18.793  1.00 19.14           N  \\nATOM    529  CA  ASP A  69      -8.880  21.905  17.822  1.00 20.43           C  \\nATOM    530  C   ASP A  69      -9.437  22.623  16.617  1.00 22.16           C  \\nATOM    531  O   ASP A  69      -9.453  22.109  15.488  1.00 19.58           O  \\nATOM    532  CB  ASP A  69     -10.025  21.148  18.478  1.00 20.58           C  \\nATOM    533  CG  ASP A  69      -9.598  19.935  19.303  1.00 22.81           C  \\nATOM    534  OD1 ASP A  69      -8.409  19.708  19.538  1.00 24.59           O  \\nATOM    535  OD2 ASP A  69     -10.475  19.176  19.715  1.00 25.28           O  \\nATOM    536  N   GLN A  70      -9.834  23.874  16.840  1.00 23.26           N  \\nATOM    537  CA  GLN A  70     -10.451  24.609  15.759  1.00 23.53           C  \\nATOM    538  C   GLN A  70      -9.437  24.925  14.667  1.00 21.20           C  \\nATOM    539  O   GLN A  70      -9.653  24.591  13.493  1.00 20.90           O  \\nATOM    540  CB  GLN A  70     -11.071  25.881  16.341  1.00 27.73           C  \\nATOM    541  CG  GLN A  70     -12.498  26.019  15.832  1.00 34.11           C  \\nATOM    542  CD  GLN A  70     -13.103  27.407  16.008  1.00 37.87           C  \\nATOM    543  OE1 GLN A  70     -12.532  28.284  16.672  1.00 39.77           O  \\nATOM    544  NE2 GLN A  70     -14.283  27.620  15.406  1.00 37.89           N  \\nATOM    545  N   TYR A  71      -8.292  25.515  15.050  1.00 19.09           N  \\nATOM    546  CA  TYR A  71      -7.329  25.873  14.030  1.00 17.69           C  \\nATOM    547  C   TYR A  71      -6.648  24.658  13.416  1.00 17.24           C  \\nATOM    548  O   TYR A  71      -6.247  24.675  12.245  1.00 17.25           O  \\nATOM    549  CB  TYR A  71      -6.302  26.868  14.625  1.00 17.87           C  \\nATOM    550  CG  TYR A  71      -5.311  26.480  15.725  1.00 18.90           C  \\nATOM    551  CD1 TYR A  71      -4.355  25.510  15.492  1.00 19.74           C  \\nATOM    552  CD2 TYR A  71      -5.328  27.162  16.940  1.00 19.21           C  \\nATOM    553  CE1 TYR A  71      -3.402  25.212  16.450  1.00 20.38           C  \\nATOM    554  CE2 TYR A  71      -4.375  26.868  17.901  1.00 21.93           C  \\nATOM    555  CZ  TYR A  71      -3.422  25.897  17.645  1.00 21.91           C  \\nATOM    556  OH  TYR A  71      -2.463  25.590  18.588  1.00 26.44           O  \\nATOM    557  N   MET A  72      -6.557  23.559  14.166  1.00 14.81           N  \\nATOM    558  CA  MET A  72      -5.958  22.393  13.563  1.00 18.59           C  \\nATOM    559  C   MET A  72      -6.849  21.778  12.472  1.00 19.38           C  \\nATOM    560  O   MET A  72      -6.385  21.198  11.492  1.00 17.67           O  \\nATOM    561  CB  MET A  72      -5.658  21.386  14.669  1.00 17.39           C  \\nATOM    562  CG  MET A  72      -4.479  21.796  15.514  1.00 16.35           C  \\nATOM    563  SD  MET A  72      -4.146  20.600  16.820  1.00 17.04           S  \\nATOM    564  CE  MET A  72      -2.635  21.259  17.460  1.00 17.72           C  \\nATOM    565  N   ARG A  73      -8.174  21.931  12.626  1.00 20.31           N  \\nATOM    566  CA  ARG A  73      -9.120  21.350  11.701  1.00 22.24           C  \\nATOM    567  C   ARG A  73      -9.087  22.084  10.370  1.00 20.94           C  \\nATOM    568  O   ARG A  73      -9.183  21.450   9.326  1.00 20.81           O  \\nATOM    569  CB  ARG A  73     -10.499  21.414  12.369  1.00 25.29           C  \\nATOM    570  CG  ARG A  73     -11.553  20.499  11.767  1.00 30.04           C  \\nATOM    571  CD  ARG A  73     -12.775  20.429  12.687  1.00 31.86           C  \\nATOM    572  NE  ARG A  73     -12.483  19.702  13.917  1.00 34.70           N  \\nATOM    573  CZ  ARG A  73     -12.894  20.122  15.130  1.00 36.49           C  \\nATOM    574  NH1 ARG A  73     -12.579  19.402  16.196  1.00 36.93           N  \\nATOM    575  NH2 ARG A  73     -13.609  21.239  15.311  1.00 38.17           N  \\nATOM    576  N   THR A  74      -8.868  23.401  10.413  1.00 21.33           N  \\nATOM    577  CA  THR A  74      -8.813  24.240   9.219  1.00 24.99           C  \\nATOM    578  C   THR A  74      -7.436  24.351   8.544  1.00 23.10           C  \\nATOM    579  O   THR A  74      -7.367  24.738   7.371  1.00 22.59           O  \\nATOM    580  CB  THR A  74      -9.289  25.676   9.538  1.00 25.73           C  \\nATOM    581  OG1 THR A  74      -8.351  26.302  10.409  1.00 31.08           O  \\nATOM    582  CG2 THR A  74     -10.657  25.644  10.201  1.00 27.58           C  \\nATOM    583  N   GLY A  75      -6.345  24.084   9.289  1.00 19.58           N  \\nATOM    584  CA  GLY A  75      -5.013  24.114   8.730  1.00 15.79           C  \\nATOM    585  C   GLY A  75      -4.823  23.023   7.713  1.00 13.66           C  \\nATOM    586  O   GLY A  75      -5.245  21.880   7.884  1.00 14.69           O  \\nATOM    587  N   GLU A  76      -4.228  23.360   6.573  1.00 13.14           N  \\nATOM    588  CA  GLU A  76      -3.914  22.363   5.546  1.00 12.99           C  \\nATOM    589  C   GLU A  76      -2.514  21.740   5.753  1.00 12.22           C  \\nATOM    590  O   GLU A  76      -2.293  20.576   5.426  1.00 13.59           O  \\nATOM    591  CB  GLU A  76      -3.999  23.014   4.145  1.00 13.07           C  \\nATOM    592  CG  GLU A  76      -5.387  23.377   3.676  1.00 14.35           C  \\nATOM    593  CD  GLU A  76      -5.261  24.069   2.323  1.00 20.40           C  \\nATOM    594  OE1 GLU A  76      -5.461  23.403   1.288  1.00 24.32           O  \\nATOM    595  OE2 GLU A  76      -4.913  25.250   2.284  1.00 18.82           O  \\nATOM    596  N   GLY A  77      -1.553  22.503   6.301  1.00 10.73           N  \\nATOM    597  CA  GLY A  77      -0.216  22.018   6.570  1.00  8.07           C  \\nATOM    598  C   GLY A  77       0.302  22.689   7.819  1.00  8.63           C  \\nATOM    599  O   GLY A  77      -0.066  23.823   8.127  1.00  8.55           O  \\nATOM    600  N   PHE A  78       1.248  22.013   8.479  1.00  9.15           N  \\nATOM    601  CA  PHE A  78       1.754  22.468   9.773  1.00  8.87           C  \\nATOM    602  C   PHE A  78       3.268  22.520   9.854  1.00  9.51           C  \\nATOM    603  O   PHE A  78       3.930  21.542   9.486  1.00  9.15           O  \\nATOM    604  CB  PHE A  78       1.247  21.532  10.904  1.00  9.44           C  \\nATOM    605  CG  PHE A  78      -0.267  21.472  11.000  1.00  9.66           C  \\nATOM    606  CD1 PHE A  78      -0.906  22.310  11.901  1.00 10.71           C  \\nATOM    607  CD2 PHE A  78      -0.994  20.645  10.152  1.00 10.20           C  \\nATOM    608  CE1 PHE A  78      -2.295  22.338  11.940  1.00 10.61           C  \\nATOM    609  CE2 PHE A  78      -2.378  20.678  10.192  1.00 12.55           C  \\nATOM    610  CZ  PHE A  78      -3.022  21.529  11.081  1.00 12.80           C  \\nATOM    611  N   LEU A  79       3.813  23.636  10.354  1.00  8.72           N  \\nATOM    612  CA  LEU A  79       5.230  23.626  10.699  1.00  9.93           C  \\nATOM    613  C   LEU A  79       5.279  23.281  12.181  1.00  9.09           C  \\nATOM    614  O   LEU A  79       4.701  24.008  12.988  1.00 11.58           O  \\nATOM    615  CB  LEU A  79       5.921  24.970  10.567  1.00 12.80           C  \\nATOM    616  CG  LEU A  79       6.155  25.659   9.249  1.00 16.39           C  \\nATOM    617  CD1 LEU A  79       7.007  26.911   9.544  1.00 17.02           C  \\nATOM    618  CD2 LEU A  79       6.904  24.756   8.269  1.00 16.98           C  \\nATOM    619  N   CYS A  80       5.950  22.219  12.579  1.00  9.14           N  \\nATOM    620  CA  CYS A  80       6.074  21.821  13.981  1.00 11.81           C  \\nATOM    621  C   CYS A  80       7.448  22.237  14.434  1.00 11.53           C  \\nATOM    622  O   CYS A  80       8.435  21.640  14.000  1.00 12.21           O  \\nATOM    623  CB  CYS A  80       5.903  20.318  14.106  1.00 13.73           C  \\nATOM    624  SG  CYS A  80       4.158  19.900  13.837  1.00 18.26           S  \\nATOM    625  N   VAL A  81       7.486  23.282  15.269  1.00  7.57           N  \\nATOM    626  CA  VAL A  81       8.730  23.956  15.604  1.00  8.45           C  \\nATOM    627  C   VAL A  81       9.248  23.620  16.996  1.00  8.90           C  \\nATOM    628  O   VAL A  81       8.451  23.680  17.939  1.00  8.49           O  \\nATOM    629  CB  VAL A  81       8.544  25.516  15.507  1.00  7.40           C  \\nATOM    630  CG1 VAL A  81       9.917  26.196  15.588  1.00  9.42           C  \\nATOM    631  CG2 VAL A  81       8.004  25.954  14.167  1.00  8.59           C  \\nATOM    632  N   PHE A  82      10.546  23.237  17.119  1.00  9.49           N  \\nATOM    633  CA  PHE A  82      11.195  23.202  18.442  1.00  9.09           C  \\nATOM    634  C   PHE A  82      12.511  23.994  18.350  1.00  9.83           C  \\nATOM    635  O   PHE A  82      12.914  24.440  17.266  1.00  9.02           O  \\nATOM    636  CB  PHE A  82      11.482  21.755  18.902  1.00  9.76           C  \\nATOM    637  CG  PHE A  82      12.536  21.062  18.071  1.00 10.09           C  \\nATOM    638  CD1 PHE A  82      13.867  21.099  18.486  1.00 10.70           C  \\nATOM    639  CD2 PHE A  82      12.192  20.412  16.889  1.00 10.93           C  \\nATOM    640  CE1 PHE A  82      14.838  20.483  17.699  1.00 11.12           C  \\nATOM    641  CE2 PHE A  82      13.181  19.807  16.125  1.00 10.73           C  \\nATOM    642  CZ  PHE A  82      14.505  19.839  16.523  1.00 10.35           C  \\nATOM    643  N   ALA A  83      13.128  24.269  19.516  1.00  9.75           N  \\nATOM    644  CA  ALA A  83      14.407  24.963  19.569  1.00 10.32           C  \\nATOM    645  C   ALA A  83      15.509  23.960  19.916  1.00  9.48           C  \\nATOM    646  O   ALA A  83      15.362  23.142  20.839  1.00  9.61           O  \\nATOM    647  CB  ALA A  83      14.354  26.083  20.620  1.00  9.75           C  \\nATOM    648  N   ILE A  84      16.624  24.042  19.160  1.00 11.41           N  \\nATOM    649  CA  ILE A  84      17.714  23.070  19.299  1.00 13.15           C  \\nATOM    650  C   ILE A  84      18.446  23.223  20.623  1.00 14.57           C  \\nATOM    651  O   ILE A  84      19.234  22.347  20.971  1.00 13.94           O  \\nATOM    652  CB  ILE A  84      18.775  23.151  18.160  1.00 13.29           C  \\nATOM    653  CG1 ILE A  84      19.403  24.509  17.878  1.00 14.90           C  \\nATOM    654  CG2 ILE A  84      18.018  22.701  16.933  1.00 13.09           C  \\nATOM    655  CD1 ILE A  84      20.578  24.957  18.763  1.00 17.59           C  \\nATOM    656  N   ASN A  85      18.200  24.290  21.390  1.00 13.61           N  \\nATOM    657  CA  ASN A  85      18.794  24.394  22.720  1.00 15.52           C  \\nATOM    658  C   ASN A  85      17.758  24.156  23.828  1.00 15.49           C  \\nATOM    659  O   ASN A  85      17.988  24.519  24.999  1.00 16.13           O  \\nATOM    660  CB  ASN A  85      19.446  25.788  22.885  1.00 15.16           C  \\nATOM    661  CG  ASN A  85      18.464  26.946  22.904  1.00 16.42           C  \\nATOM    662  OD1 ASN A  85      17.355  26.878  22.362  1.00 16.01           O  \\nATOM    663  ND2 ASN A  85      18.808  28.030  23.587  1.00 16.83           N  \\nATOM    664  N   ASN A  86      16.616  23.515  23.509  1.00 14.72           N  \\nATOM    665  CA  ASN A  86      15.553  23.303  24.483  1.00 13.01           C  \\nATOM    666  C   ASN A  86      14.987  21.901  24.309  1.00 13.17           C  \\nATOM    667  O   ASN A  86      14.080  21.668  23.520  1.00 13.04           O  \\nATOM    668  CB  ASN A  86      14.478  24.367  24.256  1.00 16.51           C  \\nATOM    669  CG  ASN A  86      13.267  24.415  25.187  1.00 20.74           C  \\nATOM    670  OD1 ASN A  86      12.370  25.241  25.018  1.00 25.41           O  \\nATOM    671  ND2 ASN A  86      13.123  23.596  26.206  1.00 19.96           N  \\nATOM    672  N   THR A  87      15.488  20.950  25.103  1.00 13.80           N  \\nATOM    673  CA  THR A  87      15.095  19.553  25.026  1.00 15.40           C  \\nATOM    674  C   THR A  87      13.589  19.316  25.266  1.00 14.40           C  \\nATOM    675  O   THR A  87      12.953  18.535  24.546  1.00 14.11           O  \\nATOM    676  CB  THR A  87      15.983  18.791  26.046  1.00 16.19           C  \\nATOM    677  OG1 THR A  87      17.325  18.908  25.571  1.00 18.22           O  \\nATOM    678  CG2 THR A  87      15.626  17.327  26.180  1.00 17.82           C  \\nATOM    679  N   LYS A  88      12.983  19.958  26.272  1.00 14.42           N  \\nATOM    680  CA  LYS A  88      11.552  19.842  26.535  1.00 15.84           C  \\nATOM    681  C   LYS A  88      10.726  20.232  25.278  1.00 15.06           C  \\nATOM    682  O   LYS A  88       9.722  19.582  25.003  1.00 14.61           O  \\nATOM    683  CB  LYS A  88      11.207  20.766  27.717  1.00 19.53           C  \\nATOM    684  CG  LYS A  88       9.937  20.401  28.457  1.00 27.71           C  \\nATOM    685  CD  LYS A  88       9.585  21.452  29.529  1.00 33.26           C  \\nATOM    686  CE  LYS A  88       9.061  22.762  28.877  1.00 36.79           C  \\nATOM    687  NZ  LYS A  88       8.642  23.763  29.852  1.00 37.29           N  \\nATOM    688  N   SER A  89      11.119  21.259  24.475  1.00 12.36           N  \\nATOM    689  CA  SER A  89      10.325  21.666  23.309  1.00 11.31           C  \\nATOM    690  C   SER A  89      10.371  20.572  22.268  1.00 12.51           C  \\nATOM    691  O   SER A  89       9.397  20.388  21.547  1.00 12.88           O  \\nATOM    692  CB  SER A  89      10.844  22.951  22.674  1.00 11.64           C  \\nATOM    693  OG  SER A  89      12.160  22.900  22.124  1.00 11.11           O  \\nATOM    694  N   PHE A  90      11.504  19.842  22.204  1.00 11.27           N  \\nATOM    695  CA  PHE A  90      11.680  18.751  21.262  1.00 12.91           C  \\nATOM    696  C   PHE A  90      10.760  17.601  21.667  1.00 14.37           C  \\nATOM    697  O   PHE A  90      10.034  17.057  20.839  1.00 13.24           O  \\nATOM    698  CB  PHE A  90      13.161  18.312  21.260  1.00 12.34           C  \\nATOM    699  CG  PHE A  90      13.411  17.120  20.351  1.00 14.41           C  \\nATOM    700  CD1 PHE A  90      13.890  15.948  20.903  1.00 15.10           C  \\nATOM    701  CD2 PHE A  90      13.136  17.206  18.990  1.00 14.81           C  \\nATOM    702  CE1 PHE A  90      14.093  14.857  20.088  1.00 16.20           C  \\nATOM    703  CE2 PHE A  90      13.340  16.110  18.179  1.00 14.81           C  \\nATOM    704  CZ  PHE A  90      13.818  14.935  18.731  1.00 15.39           C  \\nATOM    705  N   GLU A  91      10.725  17.231  22.949  1.00 14.40           N  \\nATOM    706  CA  GLU A  91       9.863  16.146  23.407  1.00 15.78           C  \\nATOM    707  C   GLU A  91       8.374  16.466  23.301  1.00 16.18           C  \\nATOM    708  O   GLU A  91       7.533  15.582  23.114  1.00 17.50           O  \\nATOM    709  CB  GLU A  91      10.226  15.846  24.835  1.00 19.84           C  \\nATOM    710  CG  GLU A  91      11.654  15.386  25.074  1.00 24.49           C  \\nATOM    711  CD  GLU A  91      12.090  15.310  26.537  1.00 27.97           C  \\nATOM    712  OE1 GLU A  91      13.199  14.842  26.766  1.00 30.72           O  \\nATOM    713  OE2 GLU A  91      11.371  15.730  27.449  1.00 30.38           O  \\nATOM    714  N   ASP A  92       8.019  17.747  23.433  1.00 16.19           N  \\nATOM    715  CA  ASP A  92       6.650  18.216  23.265  1.00 15.23           C  \\nATOM    716  C   ASP A  92       6.141  18.053  21.839  1.00 14.29           C  \\nATOM    717  O   ASP A  92       4.927  18.074  21.633  1.00 14.78           O  \\nATOM    718  CB  ASP A  92       6.536  19.708  23.646  1.00 17.85           C  \\nATOM    719  CG  ASP A  92       6.706  20.080  25.127  1.00 21.39           C  \\nATOM    720  OD1 ASP A  92       6.648  19.209  25.997  1.00 20.06           O  \\nATOM    721  OD2 ASP A  92       6.887  21.272  25.401  1.00 24.39           O  \\nATOM    722  N   ILE A  93       7.013  17.910  20.826  1.00 11.76           N  \\nATOM    723  CA  ILE A  93       6.559  17.749  19.452  1.00 14.77           C  \\nATOM    724  C   ILE A  93       5.574  16.583  19.354  1.00 14.03           C  \\nATOM    725  O   ILE A  93       4.576  16.695  18.644  1.00 14.93           O  \\nATOM    726  CB  ILE A  93       7.790  17.499  18.485  1.00 14.96           C  \\nATOM    727  CG1 ILE A  93       8.506  18.830  18.293  1.00 15.41           C  \\nATOM    728  CG2 ILE A  93       7.369  16.936  17.091  1.00 14.61           C  \\nATOM    729  CD1 ILE A  93       7.654  19.940  17.637  1.00 20.05           C  \\nATOM    730  N   HIS A  94       5.806  15.471  20.049  1.00 15.23           N  \\nATOM    731  CA  HIS A  94       4.940  14.302  19.956  1.00 18.20           C  \\nATOM    732  C   HIS A  94       3.494  14.635  20.287  1.00 16.36           C  \\nATOM    733  O   HIS A  94       2.576  14.183  19.598  1.00 17.20           O  \\nATOM    734  CB  HIS A  94       5.436  13.194  20.900  1.00 23.57           C  \\nATOM    735  CG  HIS A  94       4.685  11.874  20.678  1.00 29.09           C  \\nATOM    736  ND1 HIS A  94       4.923  11.118  19.602  1.00 31.26           N  \\nATOM    737  CD2 HIS A  94       3.705  11.361  21.498  1.00 30.93           C  \\nATOM    738  CE1 HIS A  94       4.078  10.110  19.746  1.00 31.74           C  \\nATOM    739  NE2 HIS A  94       3.366  10.260  20.861  1.00 32.20           N  \\nATOM    740  N   GLN A  95       3.271  15.499  21.281  1.00 16.91           N  \\nATOM    741  CA  GLN A  95       1.916  15.878  21.673  1.00 18.66           C  \\nATOM    742  C   GLN A  95       1.226  16.736  20.625  1.00 16.97           C  \\nATOM    743  O   GLN A  95       0.012  16.562  20.443  1.00 15.36           O  \\nATOM    744  CB  GLN A  95       1.928  16.626  23.018  1.00 21.56           C  \\nATOM    745  CG  GLN A  95       2.457  15.766  24.180  1.00 29.81           C  \\nATOM    746  CD  GLN A  95       3.370  16.540  25.150  1.00 35.29           C  \\nATOM    747  OE1 GLN A  95       3.037  17.647  25.593  1.00 37.94           O  \\nATOM    748  NE2 GLN A  95       4.546  16.006  25.521  1.00 37.04           N  \\nATOM    749  N   TYR A  96       1.930  17.653  19.921  1.00 16.06           N  \\nATOM    750  CA  TYR A  96       1.276  18.417  18.826  1.00 16.07           C  \\nATOM    751  C   TYR A  96       0.963  17.504  17.658  1.00 14.44           C  \\nATOM    752  O   TYR A  96      -0.138  17.565  17.118  1.00 15.79           O  \\nATOM    753  CB  TYR A  96       2.139  19.554  18.264  1.00 17.94           C  \\nATOM    754  CG  TYR A  96       2.287  20.677  19.277  1.00 19.19           C  \\nATOM    755  CD1 TYR A  96       1.368  21.699  19.338  1.00 19.33           C  \\nATOM    756  CD2 TYR A  96       3.348  20.631  20.161  1.00 19.61           C  \\nATOM    757  CE1 TYR A  96       1.526  22.678  20.308  1.00 21.72           C  \\nATOM    758  CE2 TYR A  96       3.512  21.595  21.129  1.00 21.72           C  \\nATOM    759  CZ  TYR A  96       2.594  22.613  21.190  1.00 21.26           C  \\nATOM    760  OH  TYR A  96       2.776  23.582  22.153  1.00 24.09           O  \\nATOM    761  N   ARG A  97       1.904  16.635  17.295  1.00 13.30           N  \\nATOM    762  CA  ARG A  97       1.699  15.650  16.247  1.00 15.46           C  \\nATOM    763  C   ARG A  97       0.457  14.794  16.492  1.00 14.26           C  \\nATOM    764  O   ARG A  97      -0.420  14.676  15.641  1.00 14.10           O  \\nATOM    765  CB  ARG A  97       2.939  14.796  16.196  1.00 17.97           C  \\nATOM    766  CG  ARG A  97       2.886  13.541  15.357  1.00 23.35           C  \\nATOM    767  CD  ARG A  97       3.333  13.765  13.943  1.00 23.59           C  \\nATOM    768  NE  ARG A  97       3.314  12.471  13.314  1.00 24.08           N  \\nATOM    769  CZ  ARG A  97       3.268  12.320  12.007  1.00 25.74           C  \\nATOM    770  NH1 ARG A  97       3.135  11.073  11.548  1.00 26.76           N  \\nATOM    771  NH2 ARG A  97       3.353  13.380  11.181  1.00 26.95           N  \\nATOM    772  N   GLU A  98       0.332  14.218  17.678  1.00 13.69           N  \\nATOM    773  CA  GLU A  98      -0.807  13.361  17.941  1.00 16.64           C  \\nATOM    774  C   GLU A  98      -2.118  14.108  17.983  1.00 15.57           C  \\nATOM    775  O   GLU A  98      -3.157  13.573  17.607  1.00 15.85           O  \\nATOM    776  CB  GLU A  98      -0.593  12.623  19.260  1.00 21.96           C  \\nATOM    777  CG  GLU A  98       0.417  11.455  19.177  1.00 28.02           C  \\nATOM    778  CD  GLU A  98       0.147  10.460  18.038  1.00 31.04           C  \\nATOM    779  OE1 GLU A  98       1.118  10.123  17.373  1.00 33.58           O  \\nATOM    780  OE2 GLU A  98      -0.992  10.025  17.790  1.00 33.61           O  \\nATOM    781  N   GLN A  99      -2.105  15.362  18.446  1.00 14.66           N  \\nATOM    782  CA  GLN A  99      -3.343  16.104  18.468  1.00 13.22           C  \\nATOM    783  C   GLN A  99      -3.779  16.398  17.049  1.00 12.15           C  \\nATOM    784  O   GLN A  99      -4.974  16.274  16.791  1.00 14.09           O  \\nATOM    785  CB  GLN A  99      -3.158  17.403  19.250  1.00 16.29           C  \\nATOM    786  CG  GLN A  99      -4.513  18.118  19.407  1.00 19.67           C  \\nATOM    787  CD  GLN A  99      -4.517  19.240  20.438  1.00 22.19           C  \\nATOM    788  OE1 GLN A  99      -3.537  19.485  21.138  1.00 23.70           O  \\nATOM    789  NE2 GLN A  99      -5.628  19.961  20.537  1.00 22.72           N  \\nATOM    790  N   ILE A 100      -2.861  16.795  16.142  1.00 11.42           N  \\nATOM    791  CA  ILE A 100      -3.197  17.121  14.743  1.00 10.81           C  \\nATOM    792  C   ILE A 100      -3.761  15.893  14.008  1.00 12.39           C  \\nATOM    793  O   ILE A 100      -4.733  15.967  13.243  1.00 12.70           O  \\nATOM    794  CB  ILE A 100      -1.924  17.659  14.038  1.00 10.32           C  \\nATOM    795  CG1 ILE A 100      -1.642  19.075  14.518  1.00 10.37           C  \\nATOM    796  CG2 ILE A 100      -2.131  17.685  12.512  1.00 11.36           C  \\nATOM    797  CD1 ILE A 100      -0.184  19.481  14.392  1.00 10.69           C  \\nATOM    798  N   LYS A 101      -3.143  14.732  14.255  1.00 14.08           N  \\nATOM    799  CA  LYS A 101      -3.613  13.487  13.637  1.00 15.95           C  \\nATOM    800  C   LYS A 101      -5.021  13.150  14.079  1.00 15.72           C  \\nATOM    801  O   LYS A 101      -5.848  12.770  13.262  1.00 14.25           O  \\nATOM    802  CB  LYS A 101      -2.681  12.317  13.996  1.00 17.44           C  \\nATOM    803  CG  LYS A 101      -1.414  12.284  13.115  1.00 19.24           C  \\nATOM    804  CD  LYS A 101      -0.332  11.418  13.764  1.00 23.28           C  \\nATOM    805  CE  LYS A 101      -0.752  10.000  14.188  1.00 23.82           C  \\nATOM    806  NZ  LYS A 101       0.400   9.230  14.662  1.00 24.00           N  \\nATOM    807  N   ARG A 102      -5.278  13.341  15.377  1.00 16.72           N  \\nATOM    808  CA  ARG A 102      -6.586  13.098  15.962  1.00 18.74           C  \\nATOM    809  C   ARG A 102      -7.643  14.087  15.440  1.00 19.07           C  \\nATOM    810  O   ARG A 102      -8.712  13.628  15.002  1.00 20.67           O  \\nATOM    811  CB  ARG A 102      -6.457  13.197  17.462  1.00 20.62           C  \\nATOM    812  CG  ARG A 102      -7.539  12.409  18.132  1.00 26.89           C  \\nATOM    813  CD  ARG A 102      -7.610  12.887  19.585  1.00 30.60           C  \\nATOM    814  NE  ARG A 102      -8.197  14.224  19.645  1.00 35.83           N  \\nATOM    815  CZ  ARG A 102      -7.633  15.255  20.301  1.00 37.61           C  \\nATOM    816  NH1 ARG A 102      -8.243  16.455  20.267  1.00 38.37           N  \\nATOM    817  NH2 ARG A 102      -6.481  15.110  20.983  1.00 38.39           N  \\nATOM    818  N   VAL A 103      -7.418  15.407  15.395  1.00 15.59           N  \\nATOM    819  CA  VAL A 103      -8.452  16.326  14.881  1.00 18.48           C  \\nATOM    820  C   VAL A 103      -8.665  16.200  13.349  1.00 16.72           C  \\nATOM    821  O   VAL A 103      -9.776  16.383  12.848  1.00 15.90           O  \\nATOM    822  CB  VAL A 103      -8.131  17.840  15.215  1.00 20.10           C  \\nATOM    823  CG1 VAL A 103      -7.896  18.005  16.704  1.00 21.34           C  \\nATOM    824  CG2 VAL A 103      -6.899  18.302  14.512  1.00 23.93           C  \\nATOM    825  N   LYS A 104      -7.622  15.902  12.574  1.00 13.84           N  \\nATOM    826  CA  LYS A 104      -7.804  15.755  11.140  1.00 16.11           C  \\nATOM    827  C   LYS A 104      -8.158  14.329  10.716  1.00 16.70           C  \\nATOM    828  O   LYS A 104      -8.260  14.079   9.508  1.00 17.27           O  \\nATOM    829  CB  LYS A 104      -6.525  16.215  10.404  1.00 18.12           C  \\nATOM    830  CG  LYS A 104      -6.197  17.678  10.744  1.00 19.53           C  \\nATOM    831  CD  LYS A 104      -5.250  18.297   9.736  1.00 21.28           C  \\nATOM    832  CE  LYS A 104      -5.921  18.590   8.390  1.00 20.75           C  \\nATOM    833  NZ  LYS A 104      -6.875  19.673   8.462  1.00 19.79           N  \\nATOM    834  N   ASP A 105      -8.372  13.390  11.669  1.00 15.91           N  \\nATOM    835  CA  ASP A 105      -8.666  11.988  11.359  1.00 15.43           C  \\nATOM    836  C   ASP A 105      -7.705  11.372  10.333  1.00 16.04           C  \\nATOM    837  O   ASP A 105      -8.106  10.654   9.415  1.00 15.38           O  \\nATOM    838  CB  ASP A 105     -10.077  11.847  10.810  1.00 18.74           C  \\nATOM    839  CG  ASP A 105     -11.261  12.121  11.696  1.00 21.15           C  \\nATOM    840  OD1 ASP A 105     -11.183  11.972  12.918  1.00 21.36           O  \\nATOM    841  OD2 ASP A 105     -12.281  12.480  11.113  1.00 22.63           O  \\nATOM    842  N   SER A 106      -6.397  11.591  10.469  1.00 15.54           N  \\nATOM    843  CA  SER A 106      -5.470  11.174   9.442  1.00 17.25           C  \\nATOM    844  C   SER A 106      -4.112  10.882  10.011  1.00 17.31           C  \\nATOM    845  O   SER A 106      -3.650  11.556  10.916  1.00 19.11           O  \\nATOM    846  CB  SER A 106      -5.463  12.314   8.398  1.00 19.27           C  \\nATOM    847  OG  SER A 106      -4.326  12.490   7.573  1.00 24.20           O  \\nATOM    848  N   ASP A 107      -3.454   9.891   9.453  1.00 17.29           N  \\nATOM    849  CA  ASP A 107      -2.109   9.557   9.868  1.00 19.80           C  \\nATOM    850  C   ASP A 107      -1.000  10.220   9.063  1.00 20.36           C  \\nATOM    851  O   ASP A 107       0.197  10.118   9.345  1.00 21.26           O  \\nATOM    852  CB  ASP A 107      -1.970   8.045   9.810  1.00 23.15           C  \\nATOM    853  CG  ASP A 107      -2.863   7.310  10.807  1.00 25.12           C  \\nATOM    854  OD1 ASP A 107      -3.262   6.206  10.462  1.00 29.21           O  \\nATOM    855  OD2 ASP A 107      -3.127   7.812  11.903  1.00 26.59           O  \\nATOM    856  N   ASP A 108      -1.383  10.909   8.005  1.00 20.57           N  \\nATOM    857  CA  ASP A 108      -0.430  11.664   7.216  1.00 22.38           C  \\nATOM    858  C   ASP A 108      -1.061  12.982   6.842  1.00 21.37           C  \\nATOM    859  O   ASP A 108      -1.973  13.096   6.030  1.00 24.36           O  \\nATOM    860  CB  ASP A 108      -0.056  10.900   5.967  1.00 27.28           C  \\nATOM    861  CG  ASP A 108      -1.263  10.278   5.320  1.00 32.24           C  \\nATOM    862  OD1 ASP A 108      -1.714   9.272   5.877  1.00 37.07           O  \\nATOM    863  OD2 ASP A 108      -1.747  10.790   4.301  1.00 36.33           O  \\nATOM    864  N   VAL A 109      -0.598  13.970   7.572  1.00 19.33           N  \\nATOM    865  CA  VAL A 109      -1.009  15.354   7.400  1.00 17.49           C  \\nATOM    866  C   VAL A 109       0.215  16.086   6.819  1.00 14.55           C  \\nATOM    867  O   VAL A 109       1.335  15.772   7.245  1.00 13.64           O  \\nATOM    868  CB  VAL A 109      -1.410  15.870   8.784  1.00 18.40           C  \\nATOM    869  CG1 VAL A 109      -1.645  17.366   8.765  1.00 20.21           C  \\nATOM    870  CG2 VAL A 109      -2.676  15.101   9.214  1.00 20.27           C  \\nATOM    871  N   PRO A 110       0.141  17.001   5.825  1.00 12.41           N  \\nATOM    872  CA  PRO A 110       1.297  17.783   5.384  1.00 11.52           C  \\nATOM    873  C   PRO A 110       1.920  18.480   6.605  1.00 11.79           C  \\nATOM    874  O   PRO A 110       1.289  19.245   7.324  1.00  9.55           O  \\nATOM    875  CB  PRO A 110       0.728  18.721   4.370  1.00 11.31           C  \\nATOM    876  CG  PRO A 110      -0.410  17.926   3.779  1.00 11.38           C  \\nATOM    877  CD  PRO A 110      -1.044  17.309   5.020  1.00 12.05           C  \\nATOM    878  N   MET A 111       3.192  18.201   6.874  1.00 11.92           N  \\nATOM    879  CA  MET A 111       3.878  18.724   8.050  1.00 11.49           C  \\nATOM    880  C   MET A 111       5.379  18.749   7.807  1.00 11.68           C  \\nATOM    881  O   MET A 111       5.903  17.900   7.098  1.00 11.11           O  \\nATOM    882  CB  MET A 111       3.494  17.806   9.211  1.00 14.36           C  \\nATOM    883  CG  MET A 111       3.864  18.110  10.627  1.00 17.41           C  \\nATOM    884  SD  MET A 111       3.231  16.825  11.758  1.00 18.36           S  \\nATOM    885  CE  MET A 111       1.528  17.128  11.498  1.00 17.25           C  \\nATOM    886  N   VAL A 112       6.091  19.706   8.404  1.00 10.73           N  \\nATOM    887  CA  VAL A 112       7.545  19.763   8.348  1.00 10.18           C  \\nATOM    888  C   VAL A 112       7.992  19.875   9.794  1.00  9.11           C  \\nATOM    889  O   VAL A 112       7.404  20.633  10.556  1.00 10.77           O  \\nATOM    890  CB  VAL A 112       8.010  21.024   7.542  1.00 11.80           C  \\nATOM    891  CG1 VAL A 112       9.528  21.213   7.558  1.00 13.24           C  \\nATOM    892  CG2 VAL A 112       7.542  20.843   6.104  1.00 14.56           C  \\nATOM    893  N   LEU A 113       9.032  19.167  10.207  1.00  8.47           N  \\nATOM    894  CA  LEU A 113       9.632  19.332  11.535  1.00  9.53           C  \\nATOM    895  C   LEU A 113      10.738  20.384  11.412  1.00  9.33           C  \\nATOM    896  O   LEU A 113      11.621  20.270  10.555  1.00  9.95           O  \\nATOM    897  CB  LEU A 113      10.214  17.983  12.017  1.00  9.56           C  \\nATOM    898  CG  LEU A 113      10.863  17.976  13.381  1.00 11.58           C  \\nATOM    899  CD1 LEU A 113       9.908  18.379  14.503  1.00 13.17           C  \\nATOM    900  CD2 LEU A 113      11.361  16.564  13.597  1.00 12.84           C  \\nATOM    901  N   VAL A 114      10.705  21.442  12.221  1.00  7.62           N  \\nATOM    902  CA  VAL A 114      11.673  22.541  12.109  1.00  7.64           C  \\nATOM    903  C   VAL A 114      12.436  22.667  13.441  1.00  7.65           C  \\nATOM    904  O   VAL A 114      11.872  22.842  14.541  1.00  9.71           O  \\nATOM    905  CB  VAL A 114      10.896  23.836  11.774  1.00  9.37           C  \\nATOM    906  CG1 VAL A 114      11.814  25.030  11.869  1.00 10.27           C  \\nATOM    907  CG2 VAL A 114      10.254  23.719  10.393  1.00  9.64           C  \\nATOM    908  N   GLY A 115      13.745  22.567  13.331  1.00  6.10           N  \\nATOM    909  CA  GLY A 115      14.603  22.742  14.476  1.00  6.69           C  \\nATOM    910  C   GLY A 115      15.120  24.163  14.407  1.00  8.47           C  \\nATOM    911  O   GLY A 115      16.007  24.439  13.594  1.00  9.16           O  \\nATOM    912  N   ASN A 116      14.593  25.089  15.192  1.00  8.01           N  \\nATOM    913  CA  ASN A 116      15.007  26.475  15.111  1.00  8.24           C  \\nATOM    914  C   ASN A 116      16.112  26.873  16.109  1.00  9.61           C  \\nATOM    915  O   ASN A 116      16.453  26.142  17.051  1.00  9.84           O  \\nATOM    916  CB  ASN A 116      13.721  27.307  15.275  1.00  8.01           C  \\nATOM    917  CG  ASN A 116      13.929  28.788  14.992  1.00  9.48           C  \\nATOM    918  OD1 ASN A 116      14.470  29.205  13.972  1.00  9.94           O  \\nATOM    919  ND2 ASN A 116      13.565  29.644  15.940  1.00 10.75           N  \\nATOM    920  N   LYS A 117      16.660  28.087  15.876  1.00 10.22           N  \\nATOM    921  CA  LYS A 117      17.735  28.715  16.674  1.00 12.36           C  \\nATOM    922  C   LYS A 117      19.094  28.021  16.450  1.00 13.39           C  \\nATOM    923  O   LYS A 117      19.977  27.940  17.324  1.00 11.29           O  \\nATOM    924  CB  LYS A 117      17.391  28.706  18.214  1.00 13.23           C  \\nATOM    925  CG  LYS A 117      16.026  29.272  18.611  1.00 12.90           C  \\nATOM    926  CD  LYS A 117      15.932  29.640  20.097  1.00 12.43           C  \\nATOM    927  CE  LYS A 117      14.551  30.217  20.380  1.00 10.55           C  \\nATOM    928  NZ  LYS A 117      14.491  30.647  21.772  1.00 11.10           N  \\nATOM    929  N   CYS A 118      19.254  27.555  15.188  1.00 13.69           N  \\nATOM    930  CA  CYS A 118      20.449  26.806  14.844  1.00 15.67           C  \\nATOM    931  C   CYS A 118      21.716  27.658  14.796  1.00 15.28           C  \\nATOM    932  O   CYS A 118      22.808  27.113  14.629  1.00 15.30           O  \\nATOM    933  CB  CYS A 118      20.246  26.064  13.492  1.00 17.74           C  \\nATOM    934  SG  CYS A 118      20.353  27.009  11.944  1.00 20.74           S  \\nATOM    935  N   ASP A 119      21.578  28.995  14.931  1.00 15.76           N  \\nATOM    936  CA  ASP A 119      22.722  29.904  15.083  1.00 16.09           C  \\nATOM    937  C   ASP A 119      23.426  29.778  16.445  1.00 17.19           C  \\nATOM    938  O   ASP A 119      24.589  30.178  16.553  1.00 18.76           O  \\nATOM    939  CB  ASP A 119      22.268  31.374  14.888  1.00 15.14           C  \\nATOM    940  CG  ASP A 119      21.153  31.785  15.835  1.00 15.81           C  \\nATOM    941  OD1 ASP A 119      20.030  31.294  15.662  1.00 13.87           O  \\nATOM    942  OD2 ASP A 119      21.407  32.596  16.721  1.00 14.85           O  \\nATOM    943  N   LEU A 120      22.732  29.248  17.480  1.00 16.86           N  \\nATOM    944  CA  LEU A 120      23.230  29.099  18.845  1.00 18.76           C  \\nATOM    945  C   LEU A 120      24.217  27.957  18.979  1.00 19.70           C  \\nATOM    946  O   LEU A 120      23.979  26.885  18.431  1.00 20.53           O  \\nATOM    947  CB  LEU A 120      22.103  28.829  19.831  1.00 17.65           C  \\nATOM    948  CG  LEU A 120      21.065  29.933  20.010  1.00 19.43           C  \\nATOM    949  CD1 LEU A 120      20.044  29.418  21.008  1.00 18.46           C  \\nATOM    950  CD2 LEU A 120      21.688  31.239  20.503  1.00 18.73           C  \\nATOM    951  N   ALA A 121      25.313  28.184  19.705  1.00 19.26           N  \\nATOM    952  CA  ALA A 121      26.303  27.143  19.887  1.00 20.48           C  \\nATOM    953  C   ALA A 121      25.942  26.210  21.036  1.00 21.30           C  \\nATOM    954  O   ALA A 121      26.403  25.069  21.067  1.00 24.04           O  \\nATOM    955  CB  ALA A 121      27.652  27.767  20.164  1.00 22.02           C  \\nATOM    956  N   ALA A 122      25.109  26.635  21.982  1.00 22.09           N  \\nATOM    957  CA  ALA A 122      24.683  25.797  23.094  1.00 23.88           C  \\nATOM    958  C   ALA A 122      23.578  24.811  22.720  1.00 23.77           C  \\nATOM    959  O   ALA A 122      22.492  24.835  23.301  1.00 27.82           O  \\nATOM    960  CB  ALA A 122      24.171  26.684  24.249  1.00 25.67           C  \\nATOM    961  N   ARG A 123      23.843  23.908  21.783  1.00 20.99           N  \\nATOM    962  CA  ARG A 123      22.890  22.905  21.311  1.00 19.81           C  \\nATOM    963  C   ARG A 123      22.619  21.761  22.283  1.00 19.56           C  \\nATOM    964  O   ARG A 123      23.590  21.176  22.774  1.00 20.01           O  \\nATOM    965  CB  ARG A 123      23.421  22.333  19.988  1.00 19.81           C  \\nATOM    966  CG  ARG A 123      22.662  21.166  19.398  1.00 21.43           C  \\nATOM    967  CD  ARG A 123      23.273  20.680  18.073  1.00 22.66           C  \\nATOM    968  NE  ARG A 123      23.050  21.590  16.956  1.00 22.61           N  \\nATOM    969  CZ  ARG A 123      22.019  21.487  16.093  1.00 20.95           C  \\nATOM    970  NH1 ARG A 123      21.045  20.568  16.181  1.00 16.36           N  \\nATOM    971  NH2 ARG A 123      21.989  22.370  15.098  1.00 19.65           N  \\nATOM    972  N   THR A 124      21.364  21.389  22.600  1.00 17.96           N  \\nATOM    973  CA  THR A 124      21.118  20.200  23.415  1.00 17.85           C  \\nATOM    974  C   THR A 124      20.413  19.068  22.656  1.00 17.08           C  \\nATOM    975  O   THR A 124      20.459  17.926  23.091  1.00 19.59           O  \\nATOM    976  CB  THR A 124      20.311  20.578  24.672  1.00 19.42           C  \\nATOM    977  OG1 THR A 124      19.109  21.230  24.277  1.00 20.50           O  \\nATOM    978  CG2 THR A 124      21.155  21.458  25.588  1.00 19.06           C  \\nATOM    979  N   VAL A 125      19.788  19.319  21.495  1.00 16.59           N  \\nATOM    980  CA  VAL A 125      19.133  18.324  20.647  1.00 16.09           C  \\nATOM    981  C   VAL A 125      20.032  18.080  19.413  1.00 16.61           C  \\nATOM    982  O   VAL A 125      20.180  18.980  18.573  1.00 14.07           O  \\nATOM    983  CB  VAL A 125      17.720  18.848  20.179  1.00 15.28           C  \\nATOM    984  CG1 VAL A 125      17.097  17.814  19.265  1.00 14.81           C  \\nATOM    985  CG2 VAL A 125      16.824  19.168  21.366  1.00 15.97           C  \\nATOM    986  N   GLU A 126      20.659  16.889  19.280  1.00 17.05           N  \\nATOM    987  CA  GLU A 126      21.487  16.568  18.124  1.00 19.28           C  \\nATOM    988  C   GLU A 126      20.631  16.496  16.876  1.00 18.51           C  \\nATOM    989  O   GLU A 126      19.511  15.976  16.880  1.00 17.68           O  \\nATOM    990  CB  GLU A 126      22.165  15.215  18.219  1.00 23.29           C  \\nATOM    991  CG  GLU A 126      23.244  15.057  19.265  1.00 30.86           C  \\nATOM    992  CD  GLU A 126      24.402  16.054  19.153  1.00 36.59           C  \\nATOM    993  OE1 GLU A 126      25.139  16.012  18.161  1.00 39.64           O  \\nATOM    994  OE2 GLU A 126      24.581  16.885  20.056  1.00 40.44           O  \\nATOM    995  N   SER A 127      21.167  16.985  15.755  1.00 17.73           N  \\nATOM    996  CA  SER A 127      20.426  16.938  14.493  1.00 16.72           C  \\nATOM    997  C   SER A 127      20.040  15.506  14.158  1.00 16.76           C  \\nATOM    998  O   SER A 127      18.931  15.245  13.690  1.00 17.01           O  \\nATOM    999  CB  SER A 127      21.283  17.518  13.371  1.00 17.67           C  \\nATOM   1000  OG  SER A 127      21.730  18.826  13.768  1.00 19.67           O  \\nATOM   1001  N   ARG A 128      20.916  14.558  14.466  1.00 16.72           N  \\nATOM   1002  CA  ARG A 128      20.667  13.154  14.145  1.00 20.94           C  \\nATOM   1003  C   ARG A 128      19.435  12.617  14.862  1.00 19.62           C  \\nATOM   1004  O   ARG A 128      18.641  11.882  14.278  1.00 18.81           O  \\nATOM   1005  CB  ARG A 128      21.910  12.319  14.510  1.00 24.31           C  \\nATOM   1006  CG  ARG A 128      21.969  11.035  13.692  1.00 30.88           C  \\nATOM   1007  CD  ARG A 128      23.381  10.403  13.536  1.00 35.88           C  \\nATOM   1008  NE  ARG A 128      24.384  10.996  12.621  1.00 39.57           N  \\nATOM   1009  CZ  ARG A 128      24.313  11.018  11.267  1.00 40.85           C  \\nATOM   1010  NH1 ARG A 128      25.399  11.388  10.577  1.00 40.83           N  \\nATOM   1011  NH2 ARG A 128      23.202  10.685  10.576  1.00 41.33           N  \\nATOM   1012  N   GLN A 129      19.209  13.016  16.119  1.00 18.27           N  \\nATOM   1013  CA  GLN A 129      18.032  12.519  16.787  1.00 18.53           C  \\nATOM   1014  C   GLN A 129      16.735  13.151  16.224  1.00 17.11           C  \\nATOM   1015  O   GLN A 129      15.691  12.486  16.098  1.00 15.48           O  \\nATOM   1016  CB  GLN A 129      18.229  12.766  18.306  1.00 22.36           C  \\nATOM   1017  CG  GLN A 129      18.041  14.133  18.911  1.00 24.78           C  \\nATOM   1018  CD  GLN A 129      18.283  14.123  20.411  1.00 27.48           C  \\nATOM   1019  OE1 GLN A 129      19.287  14.643  20.906  1.00 28.14           O  \\nATOM   1020  NE2 GLN A 129      17.367  13.497  21.152  1.00 27.21           N  \\nATOM   1021  N   ALA A 130      16.780  14.446  15.837  1.00 14.73           N  \\nATOM   1022  CA  ALA A 130      15.611  15.044  15.210  1.00 13.87           C  \\nATOM   1023  C   ALA A 130      15.342  14.449  13.826  1.00 13.30           C  \\nATOM   1024  O   ALA A 130      14.199  14.194  13.436  1.00 11.42           O  \\nATOM   1025  CB  ALA A 130      15.822  16.536  15.063  1.00 16.21           C  \\nATOM   1026  N   GLN A 131      16.401  14.189  13.057  1.00 14.61           N  \\nATOM   1027  CA  GLN A 131      16.267  13.529  11.739  1.00 16.01           C  \\nATOM   1028  C   GLN A 131      15.637  12.109  11.821  1.00 16.29           C  \\nATOM   1029  O   GLN A 131      14.777  11.708  11.017  1.00 14.37           O  \\nATOM   1030  CB  GLN A 131      17.673  13.507  11.116  1.00 18.12           C  \\nATOM   1031  CG  GLN A 131      17.702  13.017   9.689  1.00 22.19           C  \\nATOM   1032  CD  GLN A 131      16.889  13.922   8.801  1.00 21.61           C  \\nATOM   1033  OE1 GLN A 131      17.292  15.044   8.529  1.00 24.06           O  \\nATOM   1034  NE2 GLN A 131      15.714  13.485   8.370  1.00 23.37           N  \\nATOM   1035  N   ASP A 132      16.038  11.348  12.853  1.00 16.64           N  \\nATOM   1036  CA  ASP A 132      15.467  10.038  13.173  1.00 18.00           C  \\nATOM   1037  C   ASP A 132      13.984  10.175  13.484  1.00 17.28           C  \\nATOM   1038  O   ASP A 132      13.178   9.395  12.985  1.00 16.45           O  \\nATOM   1039  CB  ASP A 132      16.120   9.415  14.405  1.00 20.10           C  \\nATOM   1040  CG  ASP A 132      17.506   8.816  14.234  1.00 24.07           C  \\nATOM   1041  OD1 ASP A 132      18.120   8.459  15.243  1.00 28.35           O  \\nATOM   1042  OD2 ASP A 132      17.975   8.686  13.106  1.00 25.43           O  \\nATOM   1043  N   LEU A 133      13.614  11.168  14.312  1.00 15.58           N  \\nATOM   1044  CA  LEU A 133      12.213  11.390  14.624  1.00 16.28           C  \\nATOM   1045  C   LEU A 133      11.421  11.729  13.366  1.00 14.96           C  \\nATOM   1046  O   LEU A 133      10.345  11.181  13.128  1.00 15.09           O  \\nATOM   1047  CB  LEU A 133      12.052  12.536  15.647  1.00 17.06           C  \\nATOM   1048  CG  LEU A 133      10.617  12.781  16.136  1.00 18.79           C  \\nATOM   1049  CD1 LEU A 133      10.142  11.554  16.881  1.00 19.96           C  \\nATOM   1050  CD2 LEU A 133      10.544  14.013  17.020  1.00 18.45           C  \\nATOM   1051  N   ALA A 134      11.937  12.642  12.545  1.00 13.16           N  \\nATOM   1052  CA  ALA A 134      11.241  13.016  11.331  1.00 14.89           C  \\nATOM   1053  C   ALA A 134      11.092  11.821  10.414  1.00 16.19           C  \\nATOM   1054  O   ALA A 134      10.015  11.643   9.857  1.00 15.70           O  \\nATOM   1055  CB  ALA A 134      12.021  14.089  10.629  1.00 15.12           C  \\nATOM   1056  N   ARG A 135      12.143  10.995  10.279  1.00 17.00           N  \\nATOM   1057  CA  ARG A 135      12.088   9.828   9.438  1.00 19.17           C  \\nATOM   1058  C   ARG A 135      10.990   8.893   9.916  1.00 18.70           C  \\nATOM   1059  O   ARG A 135      10.269   8.352   9.078  1.00 20.29           O  \\nATOM   1060  CB  ARG A 135      13.425   9.091   9.456  1.00 22.48           C  \\nATOM   1061  CG  ARG A 135      13.499   8.297   8.155  1.00 29.02           C  \\nATOM   1062  CD  ARG A 135      14.619   7.270   7.988  1.00 33.22           C  \\nATOM   1063  NE  ARG A 135      14.117   5.888   8.050  1.00 38.80           N  \\nATOM   1064  CZ  ARG A 135      13.281   5.302   7.148  1.00 39.60           C  \\nATOM   1065  NH1 ARG A 135      12.889   4.044   7.384  1.00 40.80           N  \\nATOM   1066  NH2 ARG A 135      12.827   5.913   6.034  1.00 40.11           N  \\nATOM   1067  N   SER A 136      10.815   8.697  11.242  1.00 19.23           N  \\nATOM   1068  CA  SER A 136       9.757   7.841  11.770  1.00 19.12           C  \\nATOM   1069  C   SER A 136       8.374   8.432  11.528  1.00 19.61           C  \\nATOM   1070  O   SER A 136       7.398   7.690  11.522  1.00 21.01           O  \\nATOM   1071  CB  SER A 136       9.930   7.624  13.280  1.00 19.94           C  \\nATOM   1072  OG  SER A 136       9.459   8.686  14.119  1.00 23.21           O  \\nATOM   1073  N   TYR A 137       8.231   9.760  11.380  1.00 17.33           N  \\nATOM   1074  CA  TYR A 137       6.950  10.375  11.078  1.00 18.12           C  \\nATOM   1075  C   TYR A 137       6.683  10.404   9.570  1.00 18.22           C  \\nATOM   1076  O   TYR A 137       5.561  10.656   9.129  1.00 19.27           O  \\nATOM   1077  CB  TYR A 137       6.907  11.812  11.592  1.00 18.21           C  \\nATOM   1078  CG  TYR A 137       6.883  12.057  13.084  1.00 19.08           C  \\nATOM   1079  CD1 TYR A 137       6.619  11.026  13.955  1.00 20.51           C  \\nATOM   1080  CD2 TYR A 137       7.059  13.355  13.548  1.00 20.43           C  \\nATOM   1081  CE1 TYR A 137       6.510  11.285  15.306  1.00 21.89           C  \\nATOM   1082  CE2 TYR A 137       6.945  13.624  14.899  1.00 21.02           C  \\nATOM   1083  CZ  TYR A 137       6.662  12.574  15.763  1.00 22.13           C  \\nATOM   1084  OH  TYR A 137       6.476  12.782  17.112  1.00 22.16           O  \\nATOM   1085  N   GLY A 138       7.689  10.186   8.737  1.00 16.07           N  \\nATOM   1086  CA  GLY A 138       7.520  10.265   7.294  1.00 17.66           C  \\nATOM   1087  C   GLY A 138       7.476  11.725   6.810  1.00 18.01           C  \\nATOM   1088  O   GLY A 138       6.859  12.010   5.774  1.00 17.85           O  \\nATOM   1089  N   ILE A 139       8.128  12.662   7.540  1.00 14.16           N  \\nATOM   1090  CA  ILE A 139       8.117  14.065   7.145  1.00 13.49           C  \\nATOM   1091  C   ILE A 139       9.526  14.662   7.033  1.00 14.35           C  \\nATOM   1092  O   ILE A 139      10.494  14.152   7.610  1.00 16.19           O  \\nATOM   1093  CB  ILE A 139       7.292  14.896   8.161  1.00 14.40           C  \\nATOM   1094  CG1 ILE A 139       7.925  14.811   9.554  1.00 15.44           C  \\nATOM   1095  CG2 ILE A 139       5.821  14.422   8.104  1.00 14.07           C  \\nATOM   1096  CD1 ILE A 139       7.288  15.765  10.587  1.00 15.35           C  \\nATOM   1097  N   PRO A 140       9.706  15.767   6.308  1.00 13.55           N  \\nATOM   1098  CA  PRO A 140      10.984  16.448   6.205  1.00 13.39           C  \\nATOM   1099  C   PRO A 140      11.424  17.065   7.532  1.00 12.74           C  \\nATOM   1100  O   PRO A 140      10.601  17.530   8.310  1.00 11.42           O  \\nATOM   1101  CB  PRO A 140      10.767  17.498   5.127  1.00 13.47           C  \\nATOM   1102  CG  PRO A 140       9.514  17.084   4.410  1.00 14.09           C  \\nATOM   1103  CD  PRO A 140       8.686  16.467   5.519  1.00 13.14           C  \\nATOM   1104  N   TYR A 141      12.744  17.086   7.756  1.00 11.68           N  \\nATOM   1105  CA  TYR A 141      13.336  17.791   8.890  1.00 10.89           C  \\nATOM   1106  C   TYR A 141      14.175  18.945   8.326  1.00 10.09           C  \\nATOM   1107  O   TYR A 141      15.056  18.720   7.468  1.00 10.27           O  \\nATOM   1108  CB  TYR A 141      14.276  16.897   9.691  1.00 12.54           C  \\nATOM   1109  CG  TYR A 141      15.063  17.676  10.733  1.00 10.92           C  \\nATOM   1110  CD1 TYR A 141      14.350  18.396  11.664  1.00 11.75           C  \\nATOM   1111  CD2 TYR A 141      16.453  17.633  10.784  1.00 12.49           C  \\nATOM   1112  CE1 TYR A 141      14.997  19.074  12.670  1.00 12.85           C  \\nATOM   1113  CE2 TYR A 141      17.112  18.307  11.794  1.00 11.33           C  \\nATOM   1114  CZ  TYR A 141      16.369  19.016  12.728  1.00 12.61           C  \\nATOM   1115  OH  TYR A 141      16.944  19.646  13.813  1.00 13.82           O  \\nATOM   1116  N   ILE A 142      13.892  20.174   8.799  1.00  9.35           N  \\nATOM   1117  CA  ILE A 142      14.636  21.355   8.376  1.00  9.05           C  \\nATOM   1118  C   ILE A 142      15.153  22.163   9.551  1.00  9.25           C  \\nATOM   1119  O   ILE A 142      14.409  22.382  10.499  1.00 10.43           O  \\nATOM   1120  CB  ILE A 142      13.756  22.259   7.472  1.00  9.25           C  \\nATOM   1121  CG1 ILE A 142      13.523  21.470   6.180  1.00 10.51           C  \\nATOM   1122  CG2 ILE A 142      14.450  23.586   7.122  1.00  9.08           C  \\nATOM   1123  CD1 ILE A 142      12.719  22.127   5.028  1.00 13.16           C  \\nATOM   1124  N   GLU A 143      16.457  22.513   9.568  1.00  9.01           N  \\nATOM   1125  CA  GLU A 143      16.979  23.404  10.611  1.00  8.52           C  \\nATOM   1126  C   GLU A 143      16.978  24.864  10.193  1.00 10.31           C  \\nATOM   1127  O   GLU A 143      17.278  25.228   9.040  1.00 10.45           O  \\nATOM   1128  CB  GLU A 143      18.397  23.022  11.013  1.00 10.87           C  \\nATOM   1129  CG  GLU A 143      18.363  21.728  11.801  1.00 10.22           C  \\nATOM   1130  CD  GLU A 143      19.592  21.411  12.633  1.00 13.01           C  \\nATOM   1131  OE1 GLU A 143      20.616  22.082  12.521  1.00 13.65           O  \\nATOM   1132  OE2 GLU A 143      19.511  20.487  13.429  1.00 12.70           O  \\nATOM   1133  N   THR A 144      16.608  25.762  11.117  1.00  9.18           N  \\nATOM   1134  CA  THR A 144      16.422  27.169  10.754  1.00  9.71           C  \\nATOM   1135  C   THR A 144      17.014  28.076  11.789  1.00  8.66           C  \\nATOM   1136  O   THR A 144      17.259  27.698  12.927  1.00  9.13           O  \\nATOM   1137  CB  THR A 144      14.919  27.617  10.635  1.00  9.90           C  \\nATOM   1138  OG1 THR A 144      14.336  27.403  11.931  1.00  9.29           O  \\nATOM   1139  CG2 THR A 144      14.120  26.858   9.557  1.00  9.89           C  \\nATOM   1140  N   SER A 145      17.244  29.288  11.360  1.00 10.91           N  \\nATOM   1141  CA  SER A 145      17.558  30.377  12.290  1.00 12.65           C  \\nATOM   1142  C   SER A 145      16.758  31.602  11.849  1.00 10.37           C  \\nATOM   1143  O   SER A 145      16.930  32.094  10.727  1.00 11.76           O  \\nATOM   1144  CB  SER A 145      19.069  30.768  12.308  1.00 11.79           C  \\nATOM   1145  OG  SER A 145      19.248  31.931  13.125  1.00 12.12           O  \\nATOM   1146  N   ALA A 146      15.838  32.046  12.696  1.00  9.49           N  \\nATOM   1147  CA  ALA A 146      15.131  33.268  12.413  1.00 10.90           C  \\nATOM   1148  C   ALA A 146      16.070  34.455  12.553  1.00 13.53           C  \\nATOM   1149  O   ALA A 146      15.799  35.514  11.995  1.00 16.20           O  \\nATOM   1150  CB  ALA A 146      13.976  33.436  13.367  1.00 12.65           C  \\nATOM   1151  N   LYS A 147      17.183  34.302  13.283  1.00 14.35           N  \\nATOM   1152  CA  LYS A 147      18.150  35.351  13.486  1.00 16.25           C  \\nATOM   1153  C   LYS A 147      18.953  35.568  12.215  1.00 17.58           C  \\nATOM   1154  O   LYS A 147      19.124  36.724  11.825  1.00 20.13           O  \\nATOM   1155  CB  LYS A 147      19.068  34.970  14.617  1.00 17.55           C  \\nATOM   1156  CG  LYS A 147      20.003  36.125  14.958  1.00 22.83           C  \\nATOM   1157  CD  LYS A 147      20.581  35.932  16.341  1.00 26.44           C  \\nATOM   1158  CE  LYS A 147      21.683  36.949  16.507  1.00 30.79           C  \\nATOM   1159  NZ  LYS A 147      22.575  36.534  17.574  1.00 33.81           N  \\nATOM   1160  N   THR A 148      19.479  34.529  11.541  1.00 17.16           N  \\nATOM   1161  CA  THR A 148      20.248  34.735  10.300  1.00 15.93           C  \\nATOM   1162  C   THR A 148      19.422  34.607   9.013  1.00 17.72           C  \\nATOM   1163  O   THR A 148      19.926  34.962   7.945  1.00 18.47           O  \\nATOM   1164  CB  THR A 148      21.444  33.745  10.209  1.00 15.86           C  \\nATOM   1165  OG1 THR A 148      20.886  32.435  10.042  1.00 14.56           O  \\nATOM   1166  CG2 THR A 148      22.377  33.839  11.439  1.00 15.25           C  \\nATOM   1167  N   ARG A 149      18.176  34.108   9.100  1.00 15.18           N  \\nATOM   1168  CA  ARG A 149      17.268  33.815   7.994  1.00 19.20           C  \\nATOM   1169  C   ARG A 149      17.652  32.483   7.318  1.00 17.94           C  \\nATOM   1170  O   ARG A 149      17.031  32.079   6.332  1.00 20.28           O  \\nATOM   1171  CB  ARG A 149      17.251  34.964   6.937  1.00 21.40           C  \\nATOM   1172  CG  ARG A 149      15.929  35.106   6.188  1.00 27.00           C  \\nATOM   1173  CD  ARG A 149      15.946  35.904   4.863  1.00 27.99           C  \\nATOM   1174  NE  ARG A 149      14.707  35.687   4.092  1.00 29.40           N  \\nATOM   1175  CZ  ARG A 149      13.550  36.232   4.492  1.00 29.07           C  \\nATOM   1176  NH1 ARG A 149      12.412  36.015   3.828  1.00 29.56           N  \\nATOM   1177  NH2 ARG A 149      13.527  37.004   5.577  1.00 28.99           N  \\nATOM   1178  N   GLN A 150      18.621  31.701   7.838  1.00 15.80           N  \\nATOM   1179  CA  GLN A 150      18.926  30.402   7.273  1.00 16.36           C  \\nATOM   1180  C   GLN A 150      17.737  29.464   7.455  1.00 15.02           C  \\nATOM   1181  O   GLN A 150      17.119  29.351   8.504  1.00 13.57           O  \\nATOM   1182  CB  GLN A 150      20.152  29.729   7.937  1.00 23.05           C  \\nATOM   1183  CG  GLN A 150      20.578  28.448   7.162  1.00 29.37           C  \\nATOM   1184  CD  GLN A 150      21.604  27.530   7.827  1.00 34.84           C  \\nATOM   1185  OE1 GLN A 150      21.301  26.416   8.280  1.00 38.95           O  \\nATOM   1186  NE2 GLN A 150      22.857  27.967   7.883  1.00 35.22           N  \\nATOM   1187  N   GLY A 151      17.412  28.809   6.348  1.00 13.62           N  \\nATOM   1188  CA  GLY A 151      16.357  27.835   6.296  1.00 13.50           C  \\nATOM   1189  C   GLY A 151      14.943  28.395   6.329  1.00 12.36           C  \\nATOM   1190  O   GLY A 151      14.025  27.591   6.148  1.00 11.07           O  \\nATOM   1191  N   VAL A 152      14.706  29.702   6.510  1.00 12.59           N  \\nATOM   1192  CA  VAL A 152      13.339  30.183   6.715  1.00 14.23           C  \\nATOM   1193  C   VAL A 152      12.435  29.971   5.475  1.00 13.41           C  \\nATOM   1194  O   VAL A 152      11.352  29.362   5.577  1.00 12.94           O  \\nATOM   1195  CB  VAL A 152      13.430  31.691   7.167  1.00 12.57           C  \\nATOM   1196  CG1 VAL A 152      12.059  32.316   7.194  1.00 13.14           C  \\nATOM   1197  CG2 VAL A 152      14.044  31.787   8.543  1.00 12.69           C  \\nATOM   1198  N   GLU A 153      12.831  30.410   4.278  1.00 13.52           N  \\nATOM   1199  CA  GLU A 153      12.001  30.213   3.086  1.00 14.81           C  \\nATOM   1200  C   GLU A 153      11.857  28.756   2.742  1.00 11.88           C  \\nATOM   1201  O   GLU A 153      10.797  28.282   2.344  1.00 13.92           O  \\nATOM   1202  CB  GLU A 153      12.592  30.936   1.881  1.00 17.03           C  \\nATOM   1203  CG  GLU A 153      12.454  32.452   2.015  1.00 21.18           C  \\nATOM   1204  CD  GLU A 153      12.870  33.318   0.817  1.00 25.73           C  \\nATOM   1205  OE1 GLU A 153      13.232  32.781  -0.234  1.00 28.32           O  \\nATOM   1206  OE2 GLU A 153      12.822  34.550   0.936  1.00 26.64           O  \\nATOM   1207  N   ASP A 154      12.959  28.031   2.957  1.00 11.31           N  \\nATOM   1208  CA  ASP A 154      13.023  26.593   2.747  1.00 10.16           C  \\nATOM   1209  C   ASP A 154      11.964  25.828   3.561  1.00 10.90           C  \\nATOM   1210  O   ASP A 154      11.224  24.985   3.035  1.00 11.83           O  \\nATOM   1211  CB  ASP A 154      14.420  26.212   3.117  1.00 15.00           C  \\nATOM   1212  CG  ASP A 154      14.919  24.942   2.524  1.00 19.04           C  \\nATOM   1213  OD1 ASP A 154      14.121  24.195   1.938  1.00 22.96           O  \\nATOM   1214  OD2 ASP A 154      16.122  24.697   2.676  1.00 21.26           O  \\nATOM   1215  N   ALA A 155      11.854  26.107   4.881  1.00  8.83           N  \\nATOM   1216  CA  ALA A 155      10.841  25.450   5.716  1.00 10.00           C  \\nATOM   1217  C   ALA A 155       9.424  25.748   5.203  1.00 10.10           C  \\nATOM   1218  O   ALA A 155       8.653  24.814   4.978  1.00 11.14           O  \\nATOM   1219  CB  ALA A 155      10.923  25.933   7.192  1.00  9.50           C  \\nATOM   1220  N   PHE A 156       9.054  27.011   4.949  1.00 10.59           N  \\nATOM   1221  CA  PHE A 156       7.712  27.345   4.475  1.00 10.51           C  \\nATOM   1222  C   PHE A 156       7.453  26.825   3.069  1.00 11.05           C  \\nATOM   1223  O   PHE A 156       6.368  26.302   2.801  1.00 10.80           O  \\nATOM   1224  CB  PHE A 156       7.492  28.890   4.506  1.00 11.54           C  \\nATOM   1225  CG  PHE A 156       7.266  29.450   5.936  1.00 12.42           C  \\nATOM   1226  CD1 PHE A 156       8.319  29.958   6.707  1.00 12.00           C  \\nATOM   1227  CD2 PHE A 156       5.978  29.434   6.478  1.00 12.57           C  \\nATOM   1228  CE1 PHE A 156       8.065  30.434   8.001  1.00 12.45           C  \\nATOM   1229  CE2 PHE A 156       5.747  29.908   7.757  1.00 12.32           C  \\nATOM   1230  CZ  PHE A 156       6.782  30.406   8.513  1.00 10.18           C  \\nATOM   1231  N   TYR A 157       8.408  26.935   2.151  1.00  9.69           N  \\nATOM   1232  CA  TYR A 157       8.148  26.463   0.798  1.00 12.31           C  \\nATOM   1233  C   TYR A 157       8.091  24.945   0.719  1.00 10.42           C  \\nATOM   1234  O   TYR A 157       7.292  24.403  -0.047  1.00 11.35           O  \\nATOM   1235  CB  TYR A 157       9.233  26.978  -0.156  1.00 15.16           C  \\nATOM   1236  CG  TYR A 157       9.257  28.492  -0.367  1.00 18.73           C  \\nATOM   1237  CD1 TYR A 157       8.380  29.366   0.268  1.00 18.52           C  \\nATOM   1238  CD2 TYR A 157      10.211  28.971  -1.242  1.00 20.44           C  \\nATOM   1239  CE1 TYR A 157       8.466  30.724   0.025  1.00 22.04           C  \\nATOM   1240  CE2 TYR A 157      10.305  30.320  -1.488  1.00 22.70           C  \\nATOM   1241  CZ  TYR A 157       9.436  31.178  -0.852  1.00 23.03           C  \\nATOM   1242  OH  TYR A 157       9.572  32.528  -1.089  1.00 26.15           O  \\nATOM   1243  N   THR A 158       8.940  24.198   1.455  1.00 11.35           N  \\nATOM   1244  CA  THR A 158       8.836  22.727   1.522  1.00 10.11           C  \\nATOM   1245  C   THR A 158       7.440  22.347   2.064  1.00 11.11           C  \\nATOM   1246  O   THR A 158       6.811  21.408   1.558  1.00 11.43           O  \\nATOM   1247  CB  THR A 158       9.935  22.129   2.470  1.00 12.09           C  \\nATOM   1248  OG1 THR A 158      11.216  22.451   1.933  1.00 11.36           O  \\nATOM   1249  CG2 THR A 158       9.791  20.657   2.624  1.00 10.87           C  \\nATOM   1250  N   LEU A 159       6.888  23.056   3.072  1.00  9.25           N  \\nATOM   1251  CA  LEU A 159       5.533  22.754   3.506  1.00  9.93           C  \\nATOM   1252  C   LEU A 159       4.489  22.986   2.388  1.00 10.42           C  \\nATOM   1253  O   LEU A 159       3.560  22.185   2.272  1.00 11.22           O  \\nATOM   1254  CB  LEU A 159       5.171  23.608   4.711  1.00  9.03           C  \\nATOM   1255  CG  LEU A 159       3.789  23.318   5.269  1.00 10.96           C  \\nATOM   1256  CD1 LEU A 159       3.635  21.828   5.651  1.00  9.29           C  \\nATOM   1257  CD2 LEU A 159       3.580  24.223   6.469  1.00 13.71           C  \\nATOM   1258  N   VAL A 160       4.596  24.046   1.548  1.00 10.91           N  \\nATOM   1259  CA  VAL A 160       3.662  24.284   0.439  1.00  9.19           C  \\nATOM   1260  C   VAL A 160       3.803  23.137  -0.556  1.00  9.60           C  \\nATOM   1261  O   VAL A 160       2.781  22.638  -1.031  1.00 10.29           O  \\nATOM   1262  CB  VAL A 160       3.981  25.653  -0.211  1.00  8.50           C  \\nATOM   1263  CG1 VAL A 160       3.402  25.754  -1.622  1.00 10.56           C  \\nATOM   1264  CG2 VAL A 160       3.403  26.741   0.659  1.00  7.98           C  \\nATOM   1265  N   ARG A 161       5.033  22.667  -0.839  1.00  9.10           N  \\nATOM   1266  CA  ARG A 161       5.206  21.554  -1.767  1.00  9.82           C  \\nATOM   1267  C   ARG A 161       4.644  20.240  -1.215  1.00 10.44           C  \\nATOM   1268  O   ARG A 161       4.166  19.395  -1.984  1.00 12.88           O  \\nATOM   1269  CB  ARG A 161       6.679  21.398  -2.115  1.00  9.39           C  \\nATOM   1270  CG  ARG A 161       7.146  22.640  -2.816  1.00  9.70           C  \\nATOM   1271  CD  ARG A 161       8.594  22.398  -3.227  1.00 14.12           C  \\nATOM   1272  NE  ARG A 161       9.122  23.635  -3.780  1.00 14.15           N  \\nATOM   1273  CZ  ARG A 161      10.144  24.313  -3.257  1.00 12.45           C  \\nATOM   1274  NH1 ARG A 161      10.823  23.858  -2.204  1.00 13.24           N  \\nATOM   1275  NH2 ARG A 161      10.485  25.475  -3.819  1.00 13.52           N  \\nATOM   1276  N   GLU A 162       4.663  20.046   0.112  1.00 10.71           N  \\nATOM   1277  CA  GLU A 162       4.054  18.877   0.722  1.00 11.80           C  \\nATOM   1278  C   GLU A 162       2.522  18.887   0.568  1.00 11.20           C  \\nATOM   1279  O   GLU A 162       1.921  17.848   0.274  1.00 12.90           O  \\nATOM   1280  CB  GLU A 162       4.430  18.815   2.216  1.00 13.86           C  \\nATOM   1281  CG  GLU A 162       5.870  18.460   2.580  1.00 14.10           C  \\nATOM   1282  CD  GLU A 162       6.268  17.038   2.224  1.00 17.23           C  \\nATOM   1283  OE1 GLU A 162       5.571  16.116   2.627  1.00 19.29           O  \\nATOM   1284  OE2 GLU A 162       7.278  16.827   1.545  1.00 22.14           O  \\nATOM   1285  N   ILE A 163       1.869  20.045   0.731  1.00 10.13           N  \\nATOM   1286  CA  ILE A 163       0.426  20.175   0.542  1.00 10.58           C  \\nATOM   1287  C   ILE A 163       0.128  19.878  -0.933  1.00 13.43           C  \\nATOM   1288  O   ILE A 163      -0.796  19.118  -1.235  1.00 15.01           O  \\nATOM   1289  CB  ILE A 163      -0.015  21.590   0.891  1.00 11.33           C  \\nATOM   1290  CG1 ILE A 163       0.084  21.777   2.400  1.00 11.27           C  \\nATOM   1291  CG2 ILE A 163      -1.486  21.815   0.437  1.00 12.37           C  \\nATOM   1292  CD1 ILE A 163      -0.006  23.245   2.894  1.00 12.56           C  \\nATOM   1293  N   ARG A 164       0.904  20.456  -1.871  1.00 11.72           N  \\nATOM   1294  CA  ARG A 164       0.770  20.163  -3.283  1.00 13.64           C  \\nATOM   1295  C   ARG A 164       0.813  18.657  -3.626  1.00 17.97           C  \\nATOM   1296  O   ARG A 164       0.150  18.197  -4.559  1.00 18.12           O  \\nATOM   1297  CB  ARG A 164       1.897  20.887  -4.029  1.00 12.51           C  \\nATOM   1298  CG  ARG A 164       1.647  22.371  -4.120  1.00 12.49           C  \\nATOM   1299  CD  ARG A 164       2.801  22.888  -4.950  1.00 14.72           C  \\nATOM   1300  NE  ARG A 164       2.730  24.327  -5.113  1.00 12.82           N  \\nATOM   1301  CZ  ARG A 164       3.800  25.071  -5.405  1.00 13.29           C  \\nATOM   1302  NH1 ARG A 164       5.018  24.539  -5.527  1.00 11.56           N  \\nATOM   1303  NH2 ARG A 164       3.595  26.381  -5.572  1.00 12.55           N  \\nATOM   1304  N   GLN A 165       1.590  17.831  -2.935  1.00 20.23           N  \\nATOM   1305  CA  GLN A 165       1.673  16.435  -3.337  1.00 27.50           C  \\nATOM   1306  C   GLN A 165       0.782  15.499  -2.502  1.00 29.12           C  \\nATOM   1307  O   GLN A 165       0.826  14.274  -2.655  1.00 29.30           O  \\nATOM   1308  CB  GLN A 165       3.168  16.070  -3.286  1.00 30.34           C  \\nATOM   1309  CG  GLN A 165       4.018  16.904  -4.309  1.00 35.17           C  \\nATOM   1310  CD  GLN A 165       4.109  16.470  -5.796  1.00 37.71           C  \\nATOM   1311  OE1 GLN A 165       4.256  17.293  -6.719  1.00 36.70           O  \\nATOM   1312  NE2 GLN A 165       4.079  15.166  -6.094  1.00 38.13           N  \\nATOM   1313  N   HIS A 166      -0.068  16.075  -1.638  1.00 29.28           N  \\nATOM   1314  CA  HIS A 166      -0.982  15.296  -0.820  1.00 31.13           C  \\nATOM   1315  C   HIS A 166      -2.214  14.971  -1.665  1.00 33.30           C  \\nATOM   1316  O   HIS A 166      -2.630  15.786  -2.497  1.00 34.96           O  \\nATOM   1317  CB  HIS A 166      -1.420  16.090   0.428  1.00 30.89           C  \\nATOM   1318  CG  HIS A 166      -2.247  15.319   1.475  1.00 32.17           C  \\nATOM   1319  ND1 HIS A 166      -3.579  15.231   1.603  1.00 33.52           N  \\nATOM   1320  CD2 HIS A 166      -1.712  14.560   2.502  1.00 32.95           C  \\nATOM   1321  CE1 HIS A 166      -3.857  14.464   2.647  1.00 33.07           C  \\nATOM   1322  NE2 HIS A 166      -2.727  14.064   3.184  1.00 33.06           N  \\nATOM   1323  OXT HIS A 166      -2.763  13.888  -1.454  1.00 36.27           O  \\nTER    1324      HIS A 166                                                      \\nHETATM 1325 MG    MG A 168       4.771  33.860  19.213  1.00  9.42          MG  \\nHETATM 1326  PG  GNP A 167       5.113  32.223  21.994  1.00  9.66           P  \\nHETATM 1327  O1G GNP A 167       4.844  32.596  23.390  1.00  9.26           O  \\nHETATM 1328  O2G GNP A 167       4.203  32.910  21.112  1.00 11.20           O  \\nHETATM 1329  O3G GNP A 167       4.640  30.769  21.802  1.00 10.09           O  \\nHETATM 1330  N3B GNP A 167       6.701  32.303  21.621  1.00  8.18           N  \\nHETATM 1331  PB  GNP A 167       7.310  32.034  20.111  1.00  8.77           P  \\nHETATM 1332  O1B GNP A 167       7.468  30.605  19.791  1.00  8.35           O  \\nHETATM 1333  O2B GNP A 167       6.696  32.775  19.021  1.00  9.38           O  \\nHETATM 1334  O3A GNP A 167       8.792  32.643  20.152  1.00 10.15           O  \\nHETATM 1335  PA  GNP A 167       9.280  34.140  19.710  1.00 10.18           P  \\nHETATM 1336  O1A GNP A 167       9.263  34.268  18.220  1.00 10.07           O  \\nHETATM 1337  O2A GNP A 167       8.594  35.173  20.510  1.00  9.33           O  \\nHETATM 1338  O5' GNP A 167      10.826  34.056  20.132  1.00 11.46           O  \\nHETATM 1339  C5' GNP A 167      11.170  33.911  21.535  1.00 10.56           C  \\nHETATM 1340  C4' GNP A 167      12.587  34.445  21.811  1.00 11.96           C  \\nHETATM 1341  O4' GNP A 167      13.560  33.649  21.095  1.00 11.66           O  \\nHETATM 1342  C3' GNP A 167      12.813  35.910  21.359  1.00 12.06           C  \\nHETATM 1343  O3' GNP A 167      13.793  36.538  22.221  1.00 14.52           O  \\nHETATM 1344  C2' GNP A 167      13.325  35.679  19.947  1.00 11.90           C  \\nHETATM 1345  O2' GNP A 167      14.151  36.750  19.499  1.00 14.99           O  \\nHETATM 1346  C1' GNP A 167      14.242  34.478  20.156  1.00 12.90           C  \\nHETATM 1347  N9  GNP A 167      14.454  33.739  18.896  1.00 12.98           N  \\nHETATM 1348  C8  GNP A 167      13.518  33.262  17.993  1.00 11.76           C  \\nHETATM 1349  N7  GNP A 167      14.016  32.636  16.979  1.00 12.34           N  \\nHETATM 1350  C5  GNP A 167      15.413  32.694  17.200  1.00 12.36           C  \\nHETATM 1351  C6  GNP A 167      16.521  32.219  16.402  1.00 12.30           C  \\nHETATM 1352  O6  GNP A 167      16.507  31.591  15.338  1.00 12.16           O  \\nHETATM 1353  N1  GNP A 167      17.735  32.503  16.995  1.00 11.60           N  \\nHETATM 1354  C2  GNP A 167      17.909  33.082  18.236  1.00 13.91           C  \\nHETATM 1355  N2  GNP A 167      19.173  33.280  18.658  1.00 14.60           N  \\nHETATM 1356  N3  GNP A 167      16.884  33.498  18.996  1.00 13.01           N  \\nHETATM 1357  C4  GNP A 167      15.675  33.331  18.398  1.00 14.36           C  \\nHETATM 1358  O   HOH A 169      13.588  43.386  17.779  1.00 31.80           O  \\nHETATM 1359  O   HOH A 170       8.248  36.064  23.209  1.00 13.63           O  \\nHETATM 1360  O   HOH A 171       3.254  37.845  19.454  1.00 18.29           O  \\nHETATM 1361  O   HOH A 172       5.945  35.619  20.105  1.00 11.33           O  \\nHETATM 1362  O   HOH A 173       3.669  32.226  18.172  1.00  8.22           O  \\nHETATM 1363  O   HOH A 174       2.896  37.170  15.500  1.00 21.37           O  \\nHETATM 1364  O   HOH A 175       2.102  32.309  24.122  1.00 27.47           O  \\nHETATM 1365  O   HOH A 176      -1.425  37.650  19.667  1.00 24.98           O  \\nHETATM 1366  O   HOH A 177      -3.225  30.031  -8.217  1.00 23.57           O  \\nHETATM 1367  O   HOH A 178      16.678  26.321  26.916  1.00 36.70           O  \\nHETATM 1368  O   HOH A 179      -2.134  40.975   1.551  1.00 81.53           O  \\nHETATM 1369  O   HOH A 180       0.400  40.675  -0.629  1.00 24.66           O  \\nHETATM 1370  O   HOH A 181      10.944  44.793  16.009  1.00 48.47           O  \\nHETATM 1371  O   HOH A 182      17.207  42.570  16.786  1.00 43.23           O  \\nHETATM 1372  O   HOH A 183      14.999  45.167  19.922  1.00 39.52           O  \\nHETATM 1373  O   HOH A 184      11.598  37.755  13.202  1.00 11.72           O  \\nHETATM 1374  O   HOH A 185       6.958  38.922  18.146  1.00 26.17           O  \\nHETATM 1375  O   HOH A 186      14.078  32.033  27.772  1.00 57.73           O  \\nHETATM 1376  O   HOH A 187       9.126  37.906  19.595  1.00 32.42           O  \\nHETATM 1377  O   HOH A 188      -0.049  36.435  27.259  1.00 50.56           O  \\nHETATM 1378  O   HOH A 189       3.843  30.503  25.485  1.00 46.80           O  \\nHETATM 1379  O   HOH A 190       4.931  28.343  27.587  1.00 29.32           O  \\nHETATM 1380  O   HOH A 191       4.881  25.774  26.247  1.00 28.55           O  \\nHETATM 1381  O   HOH A 193       8.729  26.516  26.645  1.00 37.34           O  \\nHETATM 1382  O   HOH A 194      -5.515  23.731  -4.213  1.00 43.74           O  \\nHETATM 1383  O   HOH A 195      -7.006  26.635   0.729  1.00 25.77           O  \\nHETATM 1384  O   HOH A 196      -2.033  23.281  -4.754  1.00 36.89           O  \\nHETATM 1385  O   HOH A 197      -4.326  24.642  -0.854  1.00 28.59           O  \\nHETATM 1386  O   HOH A 198      -6.858  26.995  -2.808  1.00 28.83           O  \\nHETATM 1387  O   HOH A 200      -3.937  29.225   8.289  1.00 24.08           O  \\nHETATM 1388  O   HOH A 201      -8.934  24.037   2.795  1.00 34.16           O  \\nHETATM 1389  O   HOH A 202       1.444  26.302  19.318  1.00 12.37           O  \\nHETATM 1390  O   HOH A 203       2.031  24.632  27.307  1.00 50.09           O  \\nHETATM 1391  O   HOH A 204      15.125  28.341  23.174  1.00 13.88           O  \\nHETATM 1392  O   HOH A 205      12.808  31.541  25.056  1.00 46.88           O  \\nHETATM 1393  O   HOH A 206       8.329  34.933   0.543  1.00 53.21           O  \\nHETATM 1394  O   HOH A 207       0.760  42.969   2.336  1.00 35.42           O  \\nHETATM 1395  O   HOH A 208       1.270  40.298   7.071  1.00 76.61           O  \\nHETATM 1396  O   HOH A 209      -3.135  39.625   7.649  1.00 35.40           O  \\nHETATM 1397  O   HOH A 210       8.699  39.076   2.399  1.00 33.93           O  \\nHETATM 1398  O   HOH A 212      10.040  41.806   3.447  1.00 37.88           O  \\nHETATM 1399  O   HOH A 213      18.914  29.095   3.704  1.00 25.32           O  \\nHETATM 1400  O   HOH A 214      15.966  29.098   2.957  1.00 18.63           O  \\nHETATM 1401  O   HOH A 215      15.506  31.962   4.015  1.00 18.93           O  \\nHETATM 1402  O   HOH A 216      22.935  30.533  10.845  1.00 39.81           O  \\nHETATM 1403  O   HOH A 217       6.476  43.893  23.325  1.00 49.09           O  \\nHETATM 1404  O   HOH A 218      11.746  41.248  28.612  1.00 36.22           O  \\nHETATM 1405  O   HOH A 219      11.480  42.270  25.715  1.00 29.66           O  \\nHETATM 1406  O   HOH A 220       9.344  34.251  -8.278  1.00 36.61           O  \\nHETATM 1407  O   HOH A 221      -0.111  19.667  -8.066  1.00 44.76           O  \\nHETATM 1408  O   HOH A 222      -0.528  30.460 -10.943  1.00 24.29           O  \\nHETATM 1409  O   HOH A 223     -11.352  24.274  19.491  1.00 31.21           O  \\nHETATM 1410  O   HOH A 224      -4.293  30.642 -16.466  1.00127.55           O  \\nHETATM 1411  O   HOH A 225       3.466  29.729 -16.232  1.00 26.15           O  \\nHETATM 1412  O   HOH A 227      -4.288  18.610   4.198  1.00 34.77           O  \\nHETATM 1413  O   HOH A 228       4.626  16.154   5.151  1.00 14.96           O  \\nHETATM 1414  O   HOH A 229       5.681  11.069   2.580  1.00 59.76           O  \\nHETATM 1415  O   HOH A 230       1.173  14.021   3.406  1.00 32.15           O  \\nHETATM 1416  O   HOH A 231       7.935  22.484  20.504  1.00 10.86           O  \\nHETATM 1417  O   HOH A 232       3.071  10.004  14.546  1.00 23.48           O  \\nHETATM 1418  O   HOH A 233       2.437  13.381   8.401  1.00 17.75           O  \\nHETATM 1419  O   HOH A 234       4.779   7.500   6.154  1.00 71.09           O  \\nHETATM 1420  O   HOH A 235      -4.250   8.748   6.880  1.00 24.76           O  \\nHETATM 1421  O   HOH A 236     -13.381  22.830  17.669  1.00 39.09           O  \\nHETATM 1422  O   HOH A 237      16.969  39.052   8.276  1.00 50.25           O  \\nHETATM 1423  O   HOH A 238      -6.315   9.333  13.897  1.00 36.24           O  \\nHETATM 1424  O   HOH A 240      -0.160   4.395  12.468  1.00 38.79           O  \\nHETATM 1425  O   HOH A 241       1.295   7.931  12.335  1.00 49.03           O  \\nHETATM 1426  O   HOH A 242       4.067  11.551   6.968  1.00 46.97           O  \\nHETATM 1427  O   HOH A 243      -5.060   6.579  -0.216  1.00 50.14           O  \\nHETATM 1428  O   HOH A 244      18.164   7.151   5.573  1.00 71.67           O  \\nHETATM 1429  O   HOH A 245       2.068   8.904   7.636  1.00 45.86           O  \\nHETATM 1430  O   HOH A 246      24.661  23.134  12.757  1.00 34.69           O  \\nHETATM 1431  O   HOH A 247      21.051  23.714   6.239  1.00 24.64           O  \\nHETATM 1432  O   HOH A 248      25.855  31.137  20.712  1.00 34.53           O  \\nHETATM 1433  O   HOH A 249      24.868  30.890  26.223  1.00 48.47           O  \\nHETATM 1434  O   HOH A 250      26.967  20.447  19.448  1.00 47.77           O  \\nHETATM 1435  O   HOH A 252      28.453  14.196  14.602  1.00 39.78           O  \\nHETATM 1436  O   HOH A 253      19.639  16.256   9.760  1.00 37.54           O  \\nHETATM 1437  O   HOH A 254      16.629   7.383  10.526  1.00 60.90           O  \\nHETATM 1438  O   HOH A 255       4.972   7.888  14.276  1.00 54.95           O  \\nHETATM 1439  O   HOH A 256      14.184  14.684   5.643  1.00 31.40           O  \\nHETATM 1440  O   HOH A 257      11.766  17.254   1.278  1.00 41.82           O  \\nHETATM 1441  O   HOH A 258      16.274  16.365   6.342  1.00 17.29           O  \\nHETATM 1442  O   HOH A 259      18.346  24.095   6.620  1.00 17.36           O  \\nHETATM 1443  O   HOH A 260      22.479  35.073  19.694  1.00 43.96           O  \\nHETATM 1444  O   HOH A 261      20.580  31.747   4.906  1.00 59.46           O  \\nHETATM 1445  O   HOH A 262      18.196  25.514   4.261  1.00 28.75           O  \\nHETATM 1446  O   HOH A 263      12.857  20.183   1.149  1.00 21.79           O  \\nHETATM 1447  O   HOH A 264      19.514  15.912  24.943  1.00 55.52           O  \\nHETATM 1448  O   HOH A 265       8.925   7.579   5.621  1.00 77.94           O  \\nHETATM 1449  O   HOH A 266      17.622  21.835  26.964  1.00 27.48           O  \\nHETATM 1450  O   HOH A 267      24.258  19.786  14.155  1.00 20.48           O  \\nHETATM 1451  O   HOH A 268      23.806  15.180  15.136  1.00 19.86           O  \\nHETATM 1452  O   HOH A 269      21.478  14.557  11.134  1.00 29.66           O  \\nHETATM 1453  O   HOH A 270      17.293  45.621  24.753  1.00 39.18           O  \\nHETATM 1454  O   HOH A 271      -3.518  40.206  17.649  1.00 44.75           O  \\nHETATM 1455  O   HOH A 272       2.143  19.852  24.090  1.00 40.10           O  \\nHETATM 1456  O   HOH A 273       7.101  35.918  -4.704  1.00 31.76           O  \\nHETATM 1457  O   HOH A 274       5.462  38.756  -5.521  1.00 27.51           O  \\nHETATM 1458  O   HOH A 275       5.748  20.565  28.638  1.00 58.06           O  \\nHETATM 1459  O   HOH A 276       8.814  31.258  -8.964  1.00 23.79           O  \\nHETATM 1460  O   HOH A 277     -10.427  35.895   4.301  1.00 58.91           O  \\nHETATM 1461  O   HOH A 278      -8.713  28.579   2.333  1.00 69.36           O  \\nHETATM 1462  O   HOH A 279      24.472  24.237  17.316  1.00 32.14           O  \\nHETATM 1463  O   HOH A 281      16.968  32.701  21.815  1.00 23.13           O  \\nHETATM 1464  O   HOH A 282      19.541  34.346  21.581  1.00 53.25           O  \\nHETATM 1465  O   HOH A 283      24.847  23.384  26.621  1.00 61.18           O  \\nHETATM 1466  O   HOH A 284       1.409  11.113   2.873  1.00 44.87           O  \\nHETATM 1467  O   HOH A 285       1.556  11.488  -0.914  1.00 37.86           O  \\nHETATM 1468  O   HOH A 286       9.758  12.945   3.889  1.00 36.47           O  \\nHETATM 1469  O   HOH A 287       3.827  13.375   4.622  1.00 44.90           O  \\nHETATM 1470  O   HOH A 288      14.279   6.175  15.391  1.00 78.25           O  \\nHETATM 1471  O   HOH A 289       7.573  13.968   2.191  1.00 76.47           O  \\nHETATM 1472  O   HOH A 290       2.323  15.482   1.418  1.00 35.15           O  \\nHETATM 1473  O   HOH A 291      11.997  14.573   2.962  1.00 49.86           O  \\nHETATM 1474  O   HOH A 292      23.798  33.550  17.880  1.00 24.64           O  \\nHETATM 1475  O   HOH A 293      14.657  36.046  -0.571  1.00 51.86           O  \\nHETATM 1476  O   HOH A 294      24.456  25.366   4.004  1.00 53.23           O  \\nHETATM 1477  O   HOH A 296      -0.603  12.134  -4.185  1.00 52.38           O  \\nHETATM 1478  O   HOH A 297      -8.372  34.415  -4.471  1.00 74.53           O  \\nHETATM 1479  O   HOH A 300     -14.033  36.501   0.062  1.00 50.21           O  \\nHETATM 1480  O   HOH A 301     -17.102  35.735  -2.806  1.00 42.58           O  \\nHETATM 1481  O   HOH A 303     -17.383  40.542   1.431  1.00 77.16           O  \\nHETATM 1482  O   HOH A 304     -19.024  41.557  -3.411  1.00107.16           O  \\nHETATM 1483  O   HOH A 305      -9.876  37.804   0.839  1.00 90.65           O  \\nHETATM 1484  O   HOH A 306     -14.053  36.466   4.552  1.00 82.05           O  \\nHETATM 1485  O   HOH A 307     -14.183  39.558   0.339  1.00 58.66           O  \\nHETATM 1486  O   HOH A 309     -12.547  28.677   8.301  1.00 74.88           O  \\nHETATM 1487  O   HOH A 311     -14.787  32.746  -2.171  1.00 40.79           O  \\nHETATM 1488  O   HOH A 312      -6.916  30.740   3.095  1.00 36.53           O  \\nHETATM 1489  O   HOH A 315      -8.312  33.027   7.340  1.00 83.28           O  \\nHETATM 1490  O   HOH A 316      -6.280  35.646   7.624  1.00 75.97           O  \\nHETATM 1491  O   HOH A 317      -5.230  31.565   9.609  1.00 55.67           O  \\nHETATM 1492  O   HOH A 318       6.747  37.258   1.208  1.00 57.93           O  \\nHETATM 1493  O   HOH A 319       4.747  42.419  12.683  1.00 57.74           O  \\nHETATM 1494  O   HOH A 320       2.984  41.950  16.191  1.00 41.69           O  \\nHETATM 1495  O   HOH A 321       3.867  49.208  12.056  1.00 24.57           O  \\nHETATM 1496  O   HOH A 322       7.530  42.894  17.776  1.00 79.28           O  \\nHETATM 1497  O   HOH A 324      19.822  39.457  17.826  1.00 66.22           O  \\nHETATM 1498  O   HOH A 325      12.391  45.557  24.973  1.00 48.70           O  \\nHETATM 1499  O   HOH A 326      14.944  43.928  23.794  1.00 62.45           O  \\nHETATM 1500  O   HOH A 327     -12.984  24.008  12.563  1.00 40.31           O  \\nHETATM 1501  O   HOH A 330       0.572  41.133  19.277  1.00 50.77           O  \\nHETATM 1502  O   HOH A 331      -6.196  37.979  23.043  1.00 48.93           O  \\nHETATM 1503  O   HOH A 332      -3.526  30.410  19.065  1.00 21.26           O  \\nHETATM 1504  O   HOH A 333      -7.995  32.093  21.685  1.00 54.11           O  \\nHETATM 1505  O   HOH A 334      -8.483  28.891  20.063  1.00 70.37           O  \\nHETATM 1506  O   HOH A 335     -11.670  31.022  16.981  1.00 76.86           O  \\nHETATM 1507  O   HOH A 336     -11.082  34.450  18.910  1.00 43.07           O  \\nHETATM 1508  O   HOH A 339      -6.521  37.738  15.165  1.00 64.11           O  \\nHETATM 1509  O   HOH A 340     -10.829  35.877  15.629  1.00 58.36           O  \\nHETATM 1510  O   HOH A 341      -6.552  41.166  12.231  1.00 55.67           O  \\nHETATM 1511  O   HOH A 342      -1.242  39.671  12.185  1.00 61.04           O  \\nHETATM 1512  O   HOH A 343      -4.281  38.934  13.464  1.00 32.14           O  \\nHETATM 1513  O   HOH A 344       0.386  41.208  14.039  1.00 40.80           O  \\nHETATM 1514  O   HOH A 345       0.777  39.663  21.603  1.00 67.71           O  \\nHETATM 1515  O   HOH A 346      -7.879  34.848  19.044  1.00 52.60           O  \\nHETATM 1516  O   HOH A 347       1.121  32.594 -11.957  1.00 32.30           O  \\nHETATM 1517  O   HOH A 348       0.924  32.597 -14.716  1.00 48.34           O  \\nHETATM 1518  O   HOH A 349      -2.241  27.348  20.565  1.00 26.54           O  \\nHETATM 1519  O   HOH A 350       3.259  26.214  29.881  1.00 69.22           O  \\nHETATM 1520  O   HOH A 351      -8.318  34.095  28.392  1.00 81.96           O  \\nHETATM 1521  O   HOH A 352      -2.927  29.775  31.412  1.00 80.92           O  \\nHETATM 1522  O   HOH A 353     -10.410  32.034  25.972  1.00 60.17           O  \\nHETATM 1523  O   HOH A 354      -8.122  30.393  25.378  1.00 62.33           O  \\nHETATM 1524  O   HOH A 356      -6.138  18.882  24.644  1.00 49.26           O  \\nHETATM 1525  O   HOH A 358      -1.406  19.033  23.084  1.00 82.05           O  \\nHETATM 1526  O   HOH A 359       4.169   7.339  10.280  1.00 66.03           O  \\nHETATM 1527  O   HOH A 360      -9.761  30.165   7.702  1.00 46.15           O  \\nHETATM 1528  O   HOH A 361     -16.002  27.801  11.700  1.00 41.85           O  \\nHETATM 1529  O   HOH A 362      16.215  30.489  29.264  1.00 67.21           O  \\nHETATM 1530  O   HOH A 364      21.417  27.685  26.140  1.00 49.57           O  \\nHETATM 1531  O   HOH A 366      15.824  34.767  30.662  1.00 25.90           O  \\nHETATM 1532  O   HOH A 367      18.527  35.965  24.111  1.00 68.69           O  \\nHETATM 1533  O   HOH A 369      12.103  35.275  24.945  1.00 51.37           O  \\nHETATM 1534  O   HOH A 370      24.782  11.970  23.641  1.00 55.92           O  \\nHETATM 1535  O   HOH A 371      14.337  21.394  28.627  1.00 30.11           O  \\nHETATM 1536  O   HOH A 372      -1.367  12.600  23.053  1.00 44.43           O  \\nHETATM 1537  O   HOH A 373       7.060   8.567  21.416  1.00 47.99           O  \\nHETATM 1538  O   HOH A 374       7.639   8.040  16.420  1.00 38.76           O  \\nHETATM 1539  O   HOH A 376      -1.282  15.566  25.487  1.00 67.79           O  \\nHETATM 1540  O   HOH A 378      17.945  30.924  23.806  1.00 31.50           O  \\nHETATM 1541  O   HOH A 379      17.360  33.209  33.337  1.00 30.46           O  \\nHETATM 1542  O   HOH A 380       3.971   6.567  22.089  1.00 86.80           O  \\nHETATM 1543  O   HOH A 381      -0.065  10.022  26.928  1.00 61.86           O  \\nHETATM 1544  O   HOH A 382      -4.450  12.966  21.222  1.00 42.82           O  \\nHETATM 1545  O   HOH A 383      -2.584   6.719   5.938  1.00 40.64           O  \\nHETATM 1546  O   HOH A 384      25.567  31.936  12.845  1.00 73.17           O  \\nHETATM 1547  O   HOH A 385      24.446  29.383  22.970  1.00 36.61           O  \\nHETATM 1548  O   HOH A 386      24.865  38.167  18.227  1.00 60.74           O  \\nHETATM 1549  O   HOH A 387      23.135  35.012  14.543  1.00 59.19           O  \\nHETATM 1550  O   HOH A 388      19.325   9.556  18.093  1.00 92.18           O  \\nHETATM 1551  O   HOH A 389      25.326  12.873  16.251  1.00 49.27           O  \\nHETATM 1552  O   HOH A 390      20.219  10.405  10.956  1.00 63.05           O  \\nHETATM 1553  O   HOH A 392       9.452   5.453   8.249  1.00 47.81           O  \\nHETATM 1554  O   HOH A 394      10.139  12.252   0.254  1.00 29.37           O  \\nHETATM 1555  O   HOH A 395      28.004  37.416  13.466  1.00 52.29           O  \\nHETATM 1556  O   HOH A 396      24.283  36.829  12.307  1.00 64.74           O  \\nHETATM 1557  O   HOH A 397      13.748  34.031  -3.019  1.00 47.80           O  \\nHETATM 1558  O   HOH A 398      16.981  34.189   2.006  1.00 80.51           O  \\nHETATM 1559  O   HOH A 399      -4.426  24.223  -7.240  1.00 38.09           O  \\nHETATM 1560  O   HOH A 400      19.676  33.981   3.336  1.00 64.86           O  \\nHETATM 1561  O   HOH A 401      14.633  28.745  -0.317  1.00 66.25           O  \\nHETATM 1562  O   HOH A 402      -2.139  15.576  22.716  1.00 65.01           O  \\nHETATM 1563  O   HOH A 403       9.873  11.090  20.760  1.00 53.88           O  \\nHETATM 1564  O   HOH A 404      18.291  36.379  26.927  1.00 57.23           O  \\nHETATM 1565  O   HOH A 405       0.138   6.145  15.269  1.00 82.80           O  \\nHETATM 1566  O   HOH A 406      -5.515  38.166  18.689  1.00 48.28           O  \\nHETATM 1567  O   HOH A 407      -0.399   4.688   8.809  1.00 58.35           O  \\nHETATM 1568  O   HOH A 408       0.813  25.584  22.307  1.00 40.04           O  \\nCONECT  117 1325                                                                \\nCONECT  265 1325                                                                \\nCONECT 1325  117  265 1328 1333                                                 \\nCONECT 1325 1361 1362                                                           \\nCONECT 1326 1327 1328 1329 1330                                                 \\nCONECT 1327 1326                                                                \\nCONECT 1328 1325 1326                                                           \\nCONECT 1329 1326                                                                \\nCONECT 1330 1326 1331                                                           \\nCONECT 1331 1330 1332 1333 1334                                                 \\nCONECT 1332 1331                                                                \\nCONECT 1333 1325 1331                                                           \\nCONECT 1334 1331 1335                                                           \\nCONECT 1335 1334 1336 1337 1338                                                 \\nCONECT 1336 1335                                                                \\nCONECT 1337 1335                                                                \\nCONECT 1338 1335 1339                                                           \\nCONECT 1339 1338 1340                                                           \\nCONECT 1340 1339 1341 1342                                                      \\nCONECT 1341 1340 1346                                                           \\nCONECT 1342 1340 1343 1344                                                      \\nCONECT 1343 1342                                                                \\nCONECT 1344 1342 1345 1346                                                      \\nCONECT 1345 1344                                                                \\nCONECT 1346 1341 1344 1347                                                      \\nCONECT 1347 1346 1348 1357                                                      \\nCONECT 1348 1347 1349                                                           \\nCONECT 1349 1348 1350                                                           \\nCONECT 1350 1349 1351 1357                                                      \\nCONECT 1351 1350 1352 1353                                                      \\nCONECT 1352 1351                                                                \\nCONECT 1353 1351 1354                                                           \\nCONECT 1354 1353 1355 1356                                                      \\nCONECT 1355 1354                                                                \\nCONECT 1356 1354 1357                                                           \\nCONECT 1357 1347 1350 1356                                                      \\nCONECT 1361 1325                                                                \\nCONECT 1362 1325                                                                \\nMASTER      312    0    2    5    6    0    9    6 1567    1   38   13          \\nEND                                                                             \\n\",\"pdb\");\n\tviewergrid_16835771918631198[0][0].addModel(\"\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    5.3278   11.9619   -1.5862 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4918   12.0625   -0.4513 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5527   13.0816    0.5011 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7707   13.0853    1.5453 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4957   14.1194    0.2981 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3069   14.0310   -0.8665 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2721   14.9285   -1.2734 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.7924   14.4907   -2.4207 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.7723   14.9976   -3.2383 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.1222   14.3073   -4.4052 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.4811   13.1363   -4.7128 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4813   12.6157   -3.8820 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1577   13.3143   -2.7429 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2349   13.0415   -1.7643 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8145   11.3598   -0.3222 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1195   12.3585    1.6752 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8318   13.8149    2.2035 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5851   14.8663    0.9332 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7640    9.3703   -2.6645 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8250   10.2866   -3.3296 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1718    9.8621   -4.4958 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2879   10.6862   -5.1442 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.0572   11.9404   -4.6231 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7079   12.3479   -3.4716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5995   11.5321   -2.8072 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-4.68549\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -7.2548    9.0375    4.5817 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0448    8.4019    4.2228 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.8292    9.0547    4.0095 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.7612    8.4021    3.6418 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.7899   10.4573    4.2059 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0025   11.0841    4.6043 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1942   12.4245    4.8676 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4672   12.6023    5.2235 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1730   13.7240    5.5829 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.5292   13.6094    5.9119 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.1308   12.3793    5.8698 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.4149   11.2328    5.5039 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0847   11.3744    5.1862 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.1585   10.4388    4.7983 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0601    7.4234    4.1148 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8013    7.4275    3.5086 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9150    8.8841    3.4976 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9622   10.9735    4.0730 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.7008    7.7755    3.8786 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.2098    7.6907    5.2625 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.9715    6.9952    6.2128 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.5471    6.8921    7.5129 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3553    7.4879    7.8630 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6125    8.1716    6.9162 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0245    8.2848    5.6050 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-4.21517\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -5.2059    9.8913    1.4701 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.4393    8.7205    1.6663 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9500    8.2799    2.8975 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.2150    7.2064    2.9934 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.2783    9.0388    4.0481 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0823   10.1944    3.8472 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.5352   11.0665    4.8152 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2676   12.0083    4.2190 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9393   13.0997    4.7121 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6386   13.9345    3.8321 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6414   13.6492    2.4922 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9578   12.5374    1.9849 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2799   11.7347    2.8716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.5365   10.6013    2.6562 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.2343    8.1735    0.8739 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9938    6.6876    2.1865 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.8815    6.9151    3.8727 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9646    8.7783    4.9442 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0780   11.0707   -1.1110 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2599   10.2829   -0.7287 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3094   10.1417   -1.6483 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4274    9.4138   -1.3304 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4951    8.8250   -0.0867 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4530    8.9717    0.8122 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3219    9.7000    0.5088 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-4.01159\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -6.9498   10.0239    3.9638 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6927    9.4677    4.2914 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.8273    8.8533    3.3843 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6974    8.3277    3.7703 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.2156    8.8174    2.0222 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.4645    9.4132    1.6947 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.0326    9.5098    0.4413 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1973   10.1489    0.5589 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.1492   10.5022   -0.3655 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.2995   11.1814    0.0542 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.4572   11.4825    1.3813 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.4891   11.1240    2.3275 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3695   10.4583    1.8875 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2812    9.9929    2.5820 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4050    9.5174    5.2316 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.4388    8.3591    4.7197 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.1033    7.8998    3.1121 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.6476    8.3998    1.3350 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.9625   10.1830    5.9622 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1934   11.3853    5.6068 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4816   12.5938    6.2575 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7858   13.7364    5.9555 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7975   13.6689    4.9981 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5224   12.4705    4.3628 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2122   11.3124    4.6543 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.94575\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    6.7047   12.9704   -2.2899 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4325   13.2928   -0.9412 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1589   13.3474   -0.3716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9846   13.6911    0.8748 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0535   13.0157   -1.1934 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3391   12.6553   -2.5390 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4301   12.2813   -3.5068 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1009   12.0104   -4.6273 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6860   11.6021   -5.8709 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6341   11.3923   -6.8797 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9614   11.5982   -6.6099 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3889   12.0147   -5.3431 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4366   12.2126   -4.3715 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5684   12.6144   -3.0658 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1999   13.4943   -0.3584 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7609   13.9193    1.4356 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0768   13.7223    1.2545 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1313   13.0340   -0.8491 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.0760   13.9697   -3.4934 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.0229   12.5708   -3.0417 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.1470   11.7530   -3.2282 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.1377   10.4440   -2.8192 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.9984    9.9516   -2.2213 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8955   10.7685   -2.0429 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8879   12.0867   -2.4480 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.69027\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -0.0133    7.1993    3.0780 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9633    8.1816    3.3577 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.9099    8.1021    4.3810 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7566    9.0696    4.6019 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.9237    6.9307    5.1778 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9627    5.9302    4.8653 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.8033    4.7110    5.4906 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.1959    4.0619    4.8915 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.7575    2.8294    5.1174 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8208    2.3984    4.3147 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.2843    3.2138    3.3162 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.7147    4.4715    3.0823 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.6719    4.8690    3.8852 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.0605    6.0297    3.8822 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9772    8.9839    2.7872 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7370    9.8786    4.0412 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4153    8.9908    5.3294 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.5681    6.8148    5.9130 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.3767    8.0951    1.7813 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.3467    7.3852    1.0076 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.5618    7.1538   -0.3589 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.6250    6.4970   -1.1150 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.5318    6.0700   -0.5007 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.7326    6.3043    0.8485 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.1989    6.9630    1.6234 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.62684\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    6.0766   11.2367   -1.2749 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8567   12.0362   -0.1307 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9469   13.0926   -0.0540 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8276   13.8112    1.0283 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1513   13.3627   -1.1948 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3516   12.5291   -2.3293 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6835   12.5959   -3.5342 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1547   11.6324   -4.3269 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8413   11.2642   -5.6121 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5154   10.1899   -6.2056 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4777    9.5230   -5.4943 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8006    9.8951   -4.1835 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1228   10.9531   -3.6255 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2336   11.5243   -2.3825 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3856   11.8328    0.6743 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3862   13.6135    1.8145 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1789   14.5511    1.0589 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4910   14.0929   -1.2017 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.5729    9.9538   -1.7136 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4410    9.1803   -1.1807 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.6171    7.8152   -0.9115 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5868    7.0600   -0.4124 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3752    7.6740   -0.1820 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2137    9.0219   -0.4517 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2390    9.7957   -0.9537 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.57606\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    5.0653   12.1568   -3.9872 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8604   12.4012   -4.6839 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9703   11.4160   -5.1156 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.8943   11.7181   -5.7887 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2724   10.0686   -4.7985 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4672    9.8343   -4.0639 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9490    8.6156   -3.6332 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.0860    8.8271   -2.9690 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9587    7.9704   -2.3445 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0986    8.4851   -1.7151 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.3255    9.8361   -1.7324 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4383   10.7134   -2.3681 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3270   10.1808   -2.9778 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3092   10.7944   -3.6642 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6309   13.3383   -4.8799 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.6954   12.6582   -6.0029 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2813   11.0060   -6.0825 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.6830    9.3295   -5.0736 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.0458   14.1722   -4.2793 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2733   14.0867   -3.0305 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5425   14.9999   -2.0006 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8430   14.9541   -0.8219 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8701   13.9900   -0.6735 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6136   13.0944   -1.6971 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3075   13.1265   -2.8885 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.48728\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -4.9024   11.4840    0.7331 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6126   11.9069    0.3400 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.4179   11.2889    0.7142 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.2607   11.7621    0.3415 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.5003   10.1240    1.5163 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8035    9.6828    1.8758 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.1233    8.5730    2.6300 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4517    8.5025    2.7266 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2812    7.6082    3.3572 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6681    7.7885    3.2902 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1758    8.8551    2.5964 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3337    9.7696    1.9519 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.9762    9.5681    2.0338 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.9361   10.2941    1.5097 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.5518   12.6983   -0.2425 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.2166   12.5724   -0.2159 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.4315   11.3089    0.6177 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.6936    9.6427    1.8109 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9700   13.4305    0.8105 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8997   12.4880   -0.3166 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8688   12.5637   -1.3277 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8367   11.7042   -2.3958 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8305   10.7648   -2.4519 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.8800   10.6999   -1.4480 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.8974   11.5564   -0.3673 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.46691\\n\\n$$$$\\n\",\"sdf\");\n\tviewergrid_16835771918631198[0][0].setStyle({\"model\": 0},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewergrid_16835771918631198[0][0].addSurface(1,{\"opacity\": 0.5, \"color\": \"white\"},{\"model\": 0});\n\tviewergrid_16835771918631198[0][0].setStyle({\"model\": 1},{\"stick\": {}});\n\tviewergrid_16835771918631198[0][0].zoomTo();\n\tviewergrid_16835771918631198[0][1].addModel(\"HEADER    ONCOGENE PROTEIN                        30-APR-90   5P21              \\nTITLE     REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION OF H-RAS   \\nTITLE    2 P21 AT 1.35 ANGSTROMS RESOLUTION: IMPLICATIONS FOR THE MECHANISM OF  \\nTITLE    3 GTP HYDROLYSIS                                                       \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: C-H-RAS P21 PROTEIN;                                       \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    ONCOGENE PROTEIN                                                      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    E.F.PAI,A.WITTINGHOFER,W.KABSCH                                       \\nREVDAT   6   29-NOV-17 5P21    1       HELIX                                    \\nREVDAT   5   24-FEB-09 5P21    1       VERSN                                    \\nREVDAT   4   01-APR-03 5P21    1       JRNL                                     \\nREVDAT   3   20-JUL-95 5P21    1       HET                                      \\nREVDAT   2   30-APR-94 5P21    1       FORMUL                                   \\nREVDAT   1   15-JAN-92 5P21    0                                                \\nJRNL        AUTH   E.F.PAI,U.KRENGEL,G.A.PETSKO,R.S.GOODY,W.KABSCH,             \\nJRNL        AUTH 2 A.WITTINGHOFER                                               \\nJRNL        TITL   REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION   \\nJRNL        TITL 2 OF H-RAS P21 AT 1.35 A RESOLUTION: IMPLICATIONS FOR THE      \\nJRNL        TITL 3 MECHANISM OF GTP HYDROLYSIS.                                 \\nJRNL        REF    EMBO J.                       V.   9  2351 1990              \\nJRNL        REFN                   ISSN 0261-4189                               \\nJRNL        PMID   2196171                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,S.C.ALMO,G.RAPP,K.WILSON,K.PETRATOS,A.LENTFER, \\nREMARK   1  AUTH 2 A.WITTINGHOFER,W.KABSCH,E.F.PAI,G.A.PETSKO,R.S.GOODY         \\nREMARK   1  TITL   TIME-RESOLVED X-RAY CRYSTALLOGRAPHIC STUDY OF THE            \\nREMARK   1  TITL 2 CONFORMATIONAL CHANGE IN HA-RAS P21 PROTEIN ON GTP           \\nREMARK   1  TITL 3 HYDROLYSIS                                                   \\nREMARK   1  REF    NATURE                        V. 345   309 1990              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   A.SCHERER,J.JOHN,R.LINKE,R.S.GOODY,A.WITTINGHOFER,E.F.PAI,   \\nREMARK   1  AUTH 2 K.C.HOLMES                                                   \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY ANALYSIS OF THE HUMAN  \\nREMARK   1  TITL 2 C-H-RAS-ONCOGENE PRODUCT P21 COMPLEXED WITH GTP ANALOGUES    \\nREMARK   1  REF    J.MOL.BIOL.                   V. 206   257 1989              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   E.F.PAI,W.KABSCH,U.KRENGEL,K.C.HOLMES,J.JOHN,A.WITTINGHOFER  \\nREMARK   1  TITL   STRUCTURE OF THE GUANINE-NUCLEOTIDE-BINDING DOMAIN OF THE    \\nREMARK   1  TITL 2 HA-RAS ONCOGENE PRODUCT P21 IN THE TRIPHOSPHATE CONFORMATION \\nREMARK   1  REF    NATURE                        V. 341   209 1989              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,G.RAPP,J.JOHN,A.WITTINGHOFER,E.F.PAI,R.S.GOODY \\nREMARK   1  TITL   BIOCHEMICAL AND CRYSTALLOGRAPHIC CHARACTERIZATION OF A       \\nREMARK   1  TITL 2 COMPLEX OF C-HA-RAS P21 AND CAGED GTP WITH FLASH PHOTOLYSIS  \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  86  7687 1989              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 6.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 26806                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.196                           \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1323                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 33                                      \\nREMARK   3   SOLVENT ATOMS            : 211                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.014                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 2.700                           \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  RESIDUES 61 - 65 (GLN - GLU - GLU - TYR - SER) ADOPT                \\nREMARK   3  SEVERAL CONFORMATIONS IN THE CRYSTAL.  THE COORDINATES              \\nREMARK   3  GIVEN APPROXIMATE ONE OF THESE.                                     \\nREMARK   4                                                                      \\nREMARK   4 5P21 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179747.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 38.93                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.01                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 32 2 1                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+2/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+1/3                                           \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z+1/3                                           \\nREMARK 290       6555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000      108.13333            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       54.06667            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       54.06667            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000      108.13333            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025  0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    ARG A  68   NE  -  CZ  -  NH1 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A  97   NE  -  CZ  -  NH1 ANGL. DEV. =  -3.4 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH1 ANGL. DEV. =   4.1 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500    ARG A 128   NE  -  CZ  -  NH2 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A 135   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500    TYR A 157   CB  -  CG  -  CD2 ANGL. DEV. =  -4.8 DEGREES          \\nREMARK 500    ARG A 164   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ASP A  33       75.86   -157.25                                   \\nREMARK 500    ILE A  36      -64.23   -100.39                                   \\nREMARK 500    GLU A  37      123.75   -171.30                                   \\nREMARK 500    SER A  65     -133.70     53.07                                   \\nREMARK 500    LYS A 117       33.80     71.60                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              MG A 168  MG                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 GNP A 167   O2G                                                    \\nREMARK 620 2 GNP A 167   O2B  95.0                                              \\nREMARK 620 3 HOH A 173   O    87.6  91.7                                        \\nREMARK 620 4 SER A  17   OG  172.9  91.0  88.6                                  \\nREMARK 620 5 THR A  35   OG1  91.3 173.4  90.5  82.8                            \\nREMARK 620 6 HOH A 172   O    97.3  88.0 175.0  86.5  89.3                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE MG A 168                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE GNP A 167                 \\nDBREF  5P21 A    1   166  UNP    P01112   RASH_HUMAN       1    166             \\nSEQRES   1 A  166  MET THR GLU TYR LYS LEU VAL VAL VAL GLY ALA GLY GLY          \\nSEQRES   2 A  166  VAL GLY LYS SER ALA LEU THR ILE GLN LEU ILE GLN ASN          \\nSEQRES   3 A  166  HIS PHE VAL ASP GLU TYR ASP PRO THR ILE GLU ASP SER          \\nSEQRES   4 A  166  TYR ARG LYS GLN VAL VAL ILE ASP GLY GLU THR CYS LEU          \\nSEQRES   5 A  166  LEU ASP ILE LEU ASP THR ALA GLY GLN GLU GLU TYR SER          \\nSEQRES   6 A  166  ALA MET ARG ASP GLN TYR MET ARG THR GLY GLU GLY PHE          \\nSEQRES   7 A  166  LEU CYS VAL PHE ALA ILE ASN ASN THR LYS SER PHE GLU          \\nSEQRES   8 A  166  ASP ILE HIS GLN TYR ARG GLU GLN ILE LYS ARG VAL LYS          \\nSEQRES   9 A  166  ASP SER ASP ASP VAL PRO MET VAL LEU VAL GLY ASN LYS          \\nSEQRES  10 A  166  CYS ASP LEU ALA ALA ARG THR VAL GLU SER ARG GLN ALA          \\nSEQRES  11 A  166  GLN ASP LEU ALA ARG SER TYR GLY ILE PRO TYR ILE GLU          \\nSEQRES  12 A  166  THR SER ALA LYS THR ARG GLN GLY VAL GLU ASP ALA PHE          \\nSEQRES  13 A  166  TYR THR LEU VAL ARG GLU ILE ARG GLN HIS                      \\nHET     MG  A 168       1                                                       \\nHET    GNP  A 167      32                                                       \\nHETNAM      MG MAGNESIUM ION                                                    \\nHETNAM     GNP PHOSPHOAMINOPHOSPHONIC ACID-GUANYLATE ESTER                      \\nFORMUL   2   MG    MG 2+                                                        \\nFORMUL   3  GNP    C10 H17 N6 O13 P3                                            \\nFORMUL   4  HOH   *211(H2 O)                                                    \\nHELIX    1  HA LYS A   16  GLN A   25  1                                  10    \\nHELIX    2  HB SER A   65  THR A   74  1RES 65-67 IN 3/10 CONF            10    \\nHELIX    3  HC THR A   87  VAL A  103  1BREAK AT ASP 92                   17    \\nHELIX    4  HD SER A  127  TYR A  137  1                                  11    \\nHELIX    5  HE VAL A  152  ARG A  164  1                                  13    \\nSHEET    1  S1 6 GLU A  37  ILE A  46  0                                        \\nSHEET    2  S1 6 GLU A  49  THR A  58 -1  O  LEU A  53   N  LYS A  42           \\nSHEET    3  S1 6 THR A   2  VAL A   9  1  N  LEU A   6   O  ASP A  54           \\nSHEET    4  S1 6 GLY A  77  ALA A  83  1  N  LEU A  79   O  VAL A   7           \\nSHEET    5  S1 6 MET A 111  ASN A 116  1  N  VAL A 114   O  CYS A  80           \\nSHEET    6  S1 6 TYR A 141  GLU A 143  1  N  ILE A 142   O  LEU A 113           \\nLINK        MG    MG A 168                 O2G GNP A 167     1555   1555  2.20  \\nLINK        MG    MG A 168                 O2B GNP A 167     1555   1555  2.22  \\nLINK        MG    MG A 168                 O   HOH A 173     1555   1555  2.23  \\nLINK        MG    MG A 168                 OG  SER A  17     1555   1555  2.26  \\nLINK        MG    MG A 168                 OG1 THR A  35     1555   1555  2.26  \\nLINK        MG    MG A 168                 O   HOH A 172     1555   1555  2.30  \\nSITE     1 AC1  5 SER A  17  THR A  35  GNP A 167  HOH A 172                    \\nSITE     2 AC1  5 HOH A 173                                                     \\nSITE     1 AC2 28 GLY A  12  GLY A  13  VAL A  14  GLY A  15                    \\nSITE     2 AC2 28 LYS A  16  SER A  17  ALA A  18  PHE A  28                    \\nSITE     3 AC2 28 VAL A  29  ASP A  30  TYR A  32  PRO A  34                    \\nSITE     4 AC2 28 THR A  35  GLY A  60  ASN A 116  LYS A 117                    \\nSITE     5 AC2 28 ASP A 119  SER A 145  ALA A 146  LYS A 147                    \\nSITE     6 AC2 28  MG A 168  HOH A 170  HOH A 172  HOH A 173                    \\nSITE     7 AC2 28 HOH A 175  HOH A 187  HOH A 281  HOH A 366                    \\nCRYST1   40.300   40.300  162.200  90.00  90.00 120.00 P 32 2 1      6          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.024814  0.014326  0.000000        0.00000                         \\nSCALE2      0.000000  0.028653  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.006165        0.00000                         \\nATOM      1  N   MET A   1      -7.186  32.862  -6.632  1.00 13.92           N  \\nATOM      2  CA  MET A   1      -5.921  32.127  -6.684  1.00 17.10           C  \\nATOM      3  C   MET A   1      -5.845  31.045  -5.621  1.00 16.74           C  \\nATOM      4  O   MET A   1      -6.600  31.083  -4.636  1.00 16.60           O  \\nATOM      5  CB  MET A   1      -4.673  33.058  -6.502  1.00 17.00           C  \\nATOM      6  CG  MET A   1      -4.633  33.804  -5.188  1.00 18.08           C  \\nATOM      7  SD  MET A   1      -3.115  34.766  -4.908  1.00 19.73           S  \\nATOM      8  CE  MET A   1      -3.019  35.851  -6.310  1.00 20.41           C  \\nATOM      9  N   THR A   2      -4.910  30.095  -5.820  1.00 14.70           N  \\nATOM     10  CA  THR A   2      -4.661  29.030  -4.859  1.00 16.92           C  \\nATOM     11  C   THR A   2      -4.230  29.655  -3.547  1.00 14.78           C  \\nATOM     12  O   THR A   2      -3.338  30.501  -3.537  1.00 13.93           O  \\nATOM     13  CB  THR A   2      -3.557  28.107  -5.401  1.00 18.03           C  \\nATOM     14  OG1 THR A   2      -4.059  27.565  -6.629  1.00 20.42           O  \\nATOM     15  CG2 THR A   2      -3.193  26.983  -4.443  1.00 18.51           C  \\nATOM     16  N   GLU A   3      -4.911  29.290  -2.472  1.00 14.80           N  \\nATOM     17  CA  GLU A   3      -4.504  29.730  -1.135  1.00 17.71           C  \\nATOM     18  C   GLU A   3      -4.012  28.553  -0.278  1.00 17.33           C  \\nATOM     19  O   GLU A   3      -4.520  27.427  -0.395  1.00 17.54           O  \\nATOM     20  CB  GLU A   3      -5.655  30.402  -0.432  1.00 20.67           C  \\nATOM     21  CG  GLU A   3      -6.213  31.631  -1.186  1.00 25.56           C  \\nATOM     22  CD  GLU A   3      -7.128  32.563  -0.369  1.00 28.46           C  \\nATOM     23  OE1 GLU A   3      -7.578  32.216   0.728  1.00 27.72           O  \\nATOM     24  OE2 GLU A   3      -7.375  33.680  -0.827  1.00 32.49           O  \\nATOM     25  N   TYR A   4      -2.962  28.744   0.542  1.00 14.50           N  \\nATOM     26  CA  TYR A   4      -2.425  27.654   1.358  1.00 12.29           C  \\nATOM     27  C   TYR A   4      -2.534  28.103   2.807  1.00 12.52           C  \\nATOM     28  O   TYR A   4      -2.016  29.143   3.200  1.00 12.11           O  \\nATOM     29  CB  TYR A   4      -0.975  27.385   1.026  1.00 11.46           C  \\nATOM     30  CG  TYR A   4      -0.726  26.846  -0.387  1.00 12.38           C  \\nATOM     31  CD1 TYR A   4      -0.297  27.698  -1.395  1.00 13.00           C  \\nATOM     32  CD2 TYR A   4      -0.933  25.501  -0.649  1.00 14.09           C  \\nATOM     33  CE1 TYR A   4      -0.070  27.208  -2.674  1.00 13.89           C  \\nATOM     34  CE2 TYR A   4      -0.711  24.993  -1.917  1.00 14.65           C  \\nATOM     35  CZ  TYR A   4      -0.279  25.859  -2.917  1.00 14.84           C  \\nATOM     36  OH  TYR A   4      -0.084  25.357  -4.191  1.00 15.07           O  \\nATOM     37  N   LYS A   5      -3.274  27.305   3.568  1.00 13.07           N  \\nATOM     38  CA  LYS A   5      -3.554  27.569   4.970  1.00 12.96           C  \\nATOM     39  C   LYS A   5      -2.556  26.818   5.852  1.00 12.74           C  \\nATOM     40  O   LYS A   5      -2.645  25.611   6.092  1.00 11.58           O  \\nATOM     41  CB  LYS A   5      -4.972  27.131   5.281  1.00 17.65           C  \\nATOM     42  CG  LYS A   5      -6.053  27.833   4.446  1.00 23.49           C  \\nATOM     43  CD  LYS A   5      -7.186  28.310   5.335  1.00 29.10           C  \\nATOM     44  CE  LYS A   5      -6.655  29.309   6.403  1.00 33.68           C  \\nATOM     45  NZ  LYS A   5      -6.597  28.739   7.755  1.00 36.52           N  \\nATOM     46  N   LEU A   6      -1.543  27.574   6.297  1.00 13.08           N  \\nATOM     47  CA  LEU A   6      -0.418  27.011   7.064  1.00 10.34           C  \\nATOM     48  C   LEU A   6      -0.578  27.382   8.535  1.00  9.68           C  \\nATOM     49  O   LEU A   6      -1.034  28.483   8.863  1.00 11.31           O  \\nATOM     50  CB  LEU A   6       0.910  27.571   6.536  1.00 11.34           C  \\nATOM     51  CG  LEU A   6       1.137  27.565   5.017  1.00 11.62           C  \\nATOM     52  CD1 LEU A   6       2.515  28.119   4.720  1.00 13.60           C  \\nATOM     53  CD2 LEU A   6       0.973  26.171   4.475  1.00 12.19           C  \\nATOM     54  N   VAL A   7      -0.196  26.486   9.432  1.00  9.93           N  \\nATOM     55  CA  VAL A   7      -0.333  26.787  10.852  1.00 10.06           C  \\nATOM     56  C   VAL A   7       1.033  26.513  11.495  1.00 10.37           C  \\nATOM     57  O   VAL A   7       1.597  25.441  11.252  1.00  9.51           O  \\nATOM     58  CB  VAL A   7      -1.474  25.884  11.446  1.00 10.97           C  \\nATOM     59  CG1 VAL A   7      -1.610  26.163  12.943  1.00 11.39           C  \\nATOM     60  CG2 VAL A   7      -2.809  26.161  10.777  1.00 10.14           C  \\nATOM     61  N   VAL A   8       1.576  27.455  12.295  1.00  8.35           N  \\nATOM     62  CA  VAL A   8       2.878  27.262  12.944  1.00  6.92           C  \\nATOM     63  C   VAL A   8       2.609  26.896  14.405  1.00  8.84           C  \\nATOM     64  O   VAL A   8       1.930  27.663  15.092  1.00  9.21           O  \\nATOM     65  CB  VAL A   8       3.684  28.565  12.799  1.00  8.08           C  \\nATOM     66  CG1 VAL A   8       5.111  28.324  13.261  1.00  9.52           C  \\nATOM     67  CG2 VAL A   8       3.729  29.011  11.358  1.00  8.43           C  \\nATOM     68  N   VAL A   9       3.024  25.701  14.881  1.00  8.73           N  \\nATOM     69  CA  VAL A   9       2.779  25.234  16.245  1.00  8.41           C  \\nATOM     70  C   VAL A   9       4.075  24.806  16.919  1.00  8.21           C  \\nATOM     71  O   VAL A   9       5.093  24.551  16.287  1.00 10.97           O  \\nATOM     72  CB  VAL A   9       1.824  24.028  16.286  1.00  9.74           C  \\nATOM     73  CG1 VAL A   9       0.545  24.491  15.680  1.00 11.76           C  \\nATOM     74  CG2 VAL A   9       2.349  22.810  15.535  1.00 10.31           C  \\nATOM     75  N   GLY A  10       4.048  24.707  18.238  1.00  7.34           N  \\nATOM     76  CA  GLY A  10       5.231  24.336  19.009  1.00  7.98           C  \\nATOM     77  C   GLY A  10       5.192  25.053  20.355  1.00  8.31           C  \\nATOM     78  O   GLY A  10       4.417  25.994  20.595  1.00  9.89           O  \\nATOM     79  N   ALA A  11       6.099  24.638  21.226  1.00  9.31           N  \\nATOM     80  CA  ALA A  11       6.121  25.135  22.597  1.00  9.50           C  \\nATOM     81  C   ALA A  11       6.440  26.625  22.711  1.00  9.66           C  \\nATOM     82  O   ALA A  11       6.998  27.250  21.806  1.00  9.46           O  \\nATOM     83  CB  ALA A  11       7.145  24.306  23.378  1.00  9.85           C  \\nATOM     84  N   GLY A  12       6.111  27.227  23.847  1.00 10.36           N  \\nATOM     85  CA  GLY A  12       6.406  28.632  24.054  1.00  9.30           C  \\nATOM     86  C   GLY A  12       7.892  28.950  23.964  1.00 10.23           C  \\nATOM     87  O   GLY A  12       8.785  28.249  24.466  1.00 10.57           O  \\nATOM     88  N   GLY A  13       8.159  30.064  23.266  1.00 11.49           N  \\nATOM     89  CA  GLY A  13       9.511  30.602  23.139  1.00 11.29           C  \\nATOM     90  C   GLY A  13      10.436  29.950  22.108  1.00 12.22           C  \\nATOM     91  O   GLY A  13      11.624  30.321  22.060  1.00 13.62           O  \\nATOM     92  N   VAL A  14       9.961  28.991  21.286  1.00  9.23           N  \\nATOM     93  CA  VAL A  14      10.848  28.327  20.317  1.00 10.38           C  \\nATOM     94  C   VAL A  14      11.169  29.168  19.076  1.00 10.47           C  \\nATOM     95  O   VAL A  14      12.108  28.841  18.353  1.00 12.07           O  \\nATOM     96  CB  VAL A  14      10.246  26.970  19.856  1.00  9.98           C  \\nATOM     97  CG1 VAL A  14      10.187  26.074  21.120  1.00  9.93           C  \\nATOM     98  CG2 VAL A  14       8.866  27.127  19.200  1.00  9.52           C  \\nATOM     99  N   GLY A  15      10.423  30.264  18.834  1.00  9.61           N  \\nATOM    100  CA  GLY A  15      10.666  31.160  17.706  1.00  9.70           C  \\nATOM    101  C   GLY A  15       9.615  31.061  16.626  1.00  8.82           C  \\nATOM    102  O   GLY A  15       9.915  31.403  15.491  1.00  8.99           O  \\nATOM    103  N   LYS A  16       8.395  30.591  16.916  1.00  8.18           N  \\nATOM    104  CA  LYS A  16       7.331  30.521  15.913  1.00  8.44           C  \\nATOM    105  C   LYS A  16       7.060  31.899  15.309  1.00  9.01           C  \\nATOM    106  O   LYS A  16       7.020  32.044  14.086  1.00  8.25           O  \\nATOM    107  CB  LYS A  16       6.014  30.000  16.529  1.00  9.27           C  \\nATOM    108  CG  LYS A  16       6.132  28.585  17.086  1.00  9.11           C  \\nATOM    109  CD  LYS A  16       4.820  28.077  17.726  1.00  9.31           C  \\nATOM    110  CE  LYS A  16       4.351  28.934  18.888  1.00 10.05           C  \\nATOM    111  NZ  LYS A  16       5.221  28.829  20.051  1.00  9.64           N  \\nATOM    112  N   SER A  17       6.913  32.942  16.135  1.00  7.74           N  \\nATOM    113  CA  SER A  17       6.661  34.295  15.649  1.00  8.78           C  \\nATOM    114  C   SER A  17       7.854  34.859  14.919  1.00  9.73           C  \\nATOM    115  O   SER A  17       7.653  35.446  13.868  1.00  9.70           O  \\nATOM    116  CB  SER A  17       6.360  35.264  16.769  1.00  8.22           C  \\nATOM    117  OG  SER A  17       5.083  34.853  17.204  1.00  8.79           O  \\nATOM    118  N   ALA A  18       9.072  34.687  15.444  1.00  8.95           N  \\nATOM    119  CA  ALA A  18      10.253  35.199  14.769  1.00  8.95           C  \\nATOM    120  C   ALA A  18      10.438  34.576  13.374  1.00  8.54           C  \\nATOM    121  O   ALA A  18      10.832  35.254  12.430  1.00  8.75           O  \\nATOM    122  CB  ALA A  18      11.521  34.931  15.640  1.00 10.11           C  \\nATOM    123  N   LEU A  19      10.136  33.284  13.176  1.00  8.31           N  \\nATOM    124  CA  LEU A  19      10.225  32.667  11.837  1.00  9.30           C  \\nATOM    125  C   LEU A  19       9.204  33.281  10.884  1.00 10.98           C  \\nATOM    126  O   LEU A  19       9.534  33.652   9.751  1.00 10.47           O  \\nATOM    127  CB  LEU A  19       9.938  31.165  11.885  1.00  9.41           C  \\nATOM    128  CG  LEU A  19      11.035  30.275  12.487  1.00 11.80           C  \\nATOM    129  CD1 LEU A  19      10.491  28.866  12.723  1.00 10.43           C  \\nATOM    130  CD2 LEU A  19      12.256  30.288  11.554  1.00 13.42           C  \\nATOM    131  N   THR A  20       7.950  33.419  11.335  1.00 10.72           N  \\nATOM    132  CA  THR A  20       6.888  34.015  10.529  1.00 11.87           C  \\nATOM    133  C   THR A  20       7.173  35.492  10.161  1.00 12.65           C  \\nATOM    134  O   THR A  20       7.012  35.917   9.009  1.00 12.60           O  \\nATOM    135  CB  THR A  20       5.547  33.888  11.312  1.00 13.43           C  \\nATOM    136  OG1 THR A  20       5.299  32.509  11.544  1.00 12.67           O  \\nATOM    137  CG2 THR A  20       4.346  34.410  10.518  1.00 14.64           C  \\nATOM    138  N   ILE A  21       7.615  36.326  11.098  1.00 10.48           N  \\nATOM    139  CA  ILE A  21       7.945  37.704  10.795  1.00 12.96           C  \\nATOM    140  C   ILE A  21       9.197  37.831   9.927  1.00 14.26           C  \\nATOM    141  O   ILE A  21       9.237  38.691   9.042  1.00 15.83           O  \\nATOM    142  CB  ILE A  21       8.090  38.451  12.121  1.00 12.57           C  \\nATOM    143  CG1 ILE A  21       6.691  38.551  12.737  1.00 13.24           C  \\nATOM    144  CG2 ILE A  21       8.631  39.852  11.910  1.00 13.40           C  \\nATOM    145  CD1 ILE A  21       6.636  39.185  14.129  1.00 16.46           C  \\nATOM    146  N   GLN A  22      10.206  36.979  10.092  1.00 13.58           N  \\nATOM    147  CA  GLN A  22      11.330  36.983   9.179  1.00 14.91           C  \\nATOM    148  C   GLN A  22      10.817  36.729   7.736  1.00 17.29           C  \\nATOM    149  O   GLN A  22      11.120  37.485   6.792  1.00 15.91           O  \\nATOM    150  CB  GLN A  22      12.270  35.930   9.736  1.00 15.90           C  \\nATOM    151  CG  GLN A  22      13.686  35.878   9.230  1.00 19.36           C  \\nATOM    152  CD  GLN A  22      14.372  37.217   9.091  1.00 18.08           C  \\nATOM    153  OE1 GLN A  22      14.160  37.984   8.143  1.00 19.23           O  \\nATOM    154  NE2 GLN A  22      15.224  37.492  10.050  1.00 17.40           N  \\nATOM    155  N   LEU A  23       9.944  35.718   7.523  1.00 15.95           N  \\nATOM    156  CA  LEU A  23       9.381  35.475   6.194  1.00 18.85           C  \\nATOM    157  C   LEU A  23       8.539  36.622   5.636  1.00 18.21           C  \\nATOM    158  O   LEU A  23       8.695  36.958   4.469  1.00 19.03           O  \\nATOM    159  CB  LEU A  23       8.498  34.235   6.196  1.00 18.10           C  \\nATOM    160  CG  LEU A  23       7.916  33.797   4.863  1.00 18.64           C  \\nATOM    161  CD1 LEU A  23       8.921  32.994   4.030  1.00 18.03           C  \\nATOM    162  CD2 LEU A  23       6.680  32.991   5.183  1.00 21.34           C  \\nATOM    163  N   ILE A  24       7.634  37.197   6.428  1.00 19.25           N  \\nATOM    164  CA  ILE A  24       6.682  38.198   5.953  1.00 20.23           C  \\nATOM    165  C   ILE A  24       7.316  39.590   5.922  1.00 21.89           C  \\nATOM    166  O   ILE A  24       7.125  40.318   4.943  1.00 22.93           O  \\nATOM    167  CB  ILE A  24       5.399  38.222   6.881  1.00 21.20           C  \\nATOM    168  CG1 ILE A  24       4.664  36.907   6.821  1.00 21.27           C  \\nATOM    169  CG2 ILE A  24       4.423  39.307   6.446  1.00 20.00           C  \\nATOM    170  CD1 ILE A  24       4.128  36.442   5.475  1.00 23.98           C  \\nATOM    171  N   GLN A  25       8.074  39.998   6.953  1.00 21.07           N  \\nATOM    172  CA  GLN A  25       8.520  41.380   7.066  1.00 22.09           C  \\nATOM    173  C   GLN A  25      10.003  41.559   6.962  1.00 20.75           C  \\nATOM    174  O   GLN A  25      10.495  42.677   7.054  1.00 20.26           O  \\nATOM    175  CB  GLN A  25       8.039  41.968   8.398  1.00 23.96           C  \\nATOM    176  CG  GLN A  25       6.548  42.276   8.381  1.00 28.69           C  \\nATOM    177  CD  GLN A  25       5.931  42.574   9.744  1.00 30.92           C  \\nATOM    178  OE1 GLN A  25       6.537  43.143  10.657  1.00 31.89           O  \\nATOM    179  NE2 GLN A  25       4.664  42.169   9.867  1.00 32.35           N  \\nATOM    180  N   ASN A  26      10.764  40.482   6.795  1.00 22.13           N  \\nATOM    181  CA  ASN A  26      12.198  40.642   6.628  1.00 24.67           C  \\nATOM    182  C   ASN A  26      12.936  41.362   7.734  1.00 23.62           C  \\nATOM    183  O   ASN A  26      13.603  42.372   7.521  1.00 25.10           O  \\nATOM    184  CB  ASN A  26      12.522  41.369   5.315  1.00 27.68           C  \\nATOM    185  CG  ASN A  26      12.942  40.351   4.288  1.00 32.80           C  \\nATOM    186  OD1 ASN A  26      12.135  39.699   3.602  1.00 33.77           O  \\nATOM    187  ND2 ASN A  26      14.268  40.203   4.237  1.00 34.26           N  \\nATOM    188  N   HIS A  27      12.764  40.842   8.945  1.00 19.95           N  \\nATOM    189  CA  HIS A  27      13.593  41.266  10.057  1.00 17.99           C  \\nATOM    190  C   HIS A  27      13.441  40.299  11.215  1.00 16.84           C  \\nATOM    191  O   HIS A  27      12.411  39.623  11.344  1.00 14.72           O  \\nATOM    192  CB  HIS A  27      13.241  42.680  10.551  1.00 19.80           C  \\nATOM    193  CG  HIS A  27      11.926  42.908  11.258  1.00 20.77           C  \\nATOM    194  ND1 HIS A  27      11.936  43.314  12.510  1.00 22.17           N  \\nATOM    195  CD2 HIS A  27      10.674  42.768  10.729  1.00 22.32           C  \\nATOM    196  CE1 HIS A  27      10.657  43.430  12.785  1.00 23.86           C  \\nATOM    197  NE2 HIS A  27       9.905  43.110  11.738  1.00 23.97           N  \\nATOM    198  N   PHE A  28      14.516  40.222  12.012  1.00 15.44           N  \\nATOM    199  CA  PHE A  28      14.528  39.413  13.223  1.00 13.89           C  \\nATOM    200  C   PHE A  28      14.026  40.196  14.440  1.00 15.33           C  \\nATOM    201  O   PHE A  28      14.528  41.255  14.786  1.00 16.80           O  \\nATOM    202  CB  PHE A  28      15.938  38.906  13.470  1.00 12.73           C  \\nATOM    203  CG  PHE A  28      16.083  38.150  14.786  1.00 13.35           C  \\nATOM    204  CD1 PHE A  28      15.354  36.990  15.012  1.00 13.80           C  \\nATOM    205  CD2 PHE A  28      16.974  38.615  15.740  1.00 14.54           C  \\nATOM    206  CE1 PHE A  28      15.534  36.310  16.194  1.00 14.45           C  \\nATOM    207  CE2 PHE A  28      17.147  37.925  16.920  1.00 14.61           C  \\nATOM    208  CZ  PHE A  28      16.420  36.768  17.137  1.00 14.65           C  \\nATOM    209  N   VAL A  29      12.970  39.697  15.089  1.00 17.22           N  \\nATOM    210  CA  VAL A  29      12.467  40.313  16.294  1.00 19.11           C  \\nATOM    211  C   VAL A  29      13.127  39.606  17.476  1.00 19.36           C  \\nATOM    212  O   VAL A  29      13.245  38.376  17.605  1.00 18.14           O  \\nATOM    213  CB  VAL A  29      10.894  40.205  16.412  1.00 22.03           C  \\nATOM    214  CG1 VAL A  29      10.334  40.869  15.143  1.00 19.66           C  \\nATOM    215  CG2 VAL A  29      10.377  38.776  16.631  1.00 23.15           C  \\nATOM    216  N   ASP A  30      13.615  40.484  18.324  1.00 20.14           N  \\nATOM    217  CA  ASP A  30      14.229  40.019  19.518  1.00 24.04           C  \\nATOM    218  C   ASP A  30      13.676  40.746  20.721  1.00 25.93           C  \\nATOM    219  O   ASP A  30      14.244  41.709  21.238  1.00 29.26           O  \\nATOM    220  CB  ASP A  30      15.719  40.203  19.443  1.00 27.10           C  \\nATOM    221  CG  ASP A  30      16.389  39.731  20.732  1.00 32.46           C  \\nATOM    222  OD1 ASP A  30      15.878  38.859  21.464  1.00 34.28           O  \\nATOM    223  OD2 ASP A  30      17.458  40.264  21.002  1.00 36.46           O  \\nATOM    224  N   GLU A  31      12.472  40.347  21.076  1.00 26.14           N  \\nATOM    225  CA  GLU A  31      11.922  40.804  22.318  1.00 28.72           C  \\nATOM    226  C   GLU A  31      10.870  39.836  22.749  1.00 26.20           C  \\nATOM    227  O   GLU A  31      10.466  38.931  22.014  1.00 23.14           O  \\nATOM    228  CB  GLU A  31      11.337  42.210  22.204  1.00 33.25           C  \\nATOM    229  CG  GLU A  31      10.350  42.618  21.152  1.00 40.08           C  \\nATOM    230  CD  GLU A  31      10.572  44.107  20.887  1.00 44.71           C  \\nATOM    231  OE1 GLU A  31      10.930  44.444  19.755  1.00 47.50           O  \\nATOM    232  OE2 GLU A  31      10.420  44.928  21.806  1.00 47.89           O  \\nATOM    233  N   TYR A  32      10.550  40.044  24.028  1.00 22.26           N  \\nATOM    234  CA  TYR A  32       9.597  39.196  24.685  1.00 18.83           C  \\nATOM    235  C   TYR A  32       8.204  39.688  24.317  1.00 17.53           C  \\nATOM    236  O   TYR A  32       7.794  40.792  24.676  1.00 18.45           O  \\nATOM    237  CB  TYR A  32       9.909  39.259  26.199  1.00 16.84           C  \\nATOM    238  CG  TYR A  32       9.076  38.309  27.052  1.00 15.64           C  \\nATOM    239  CD1 TYR A  32       9.598  37.096  27.433  1.00 16.17           C  \\nATOM    240  CD2 TYR A  32       7.814  38.686  27.459  1.00 15.54           C  \\nATOM    241  CE1 TYR A  32       8.845  36.251  28.239  1.00 17.86           C  \\nATOM    242  CE2 TYR A  32       7.051  37.861  28.257  1.00 14.40           C  \\nATOM    243  CZ  TYR A  32       7.572  36.635  28.652  1.00 16.47           C  \\nATOM    244  OH  TYR A  32       6.821  35.789  29.485  1.00 13.59           O  \\nATOM    245  N   ASP A  33       7.461  38.841  23.604  1.00 13.43           N  \\nATOM    246  CA  ASP A  33       6.128  39.200  23.141  1.00 14.00           C  \\nATOM    247  C   ASP A  33       5.262  37.955  22.854  1.00 13.56           C  \\nATOM    248  O   ASP A  33       5.024  37.586  21.681  1.00 12.47           O  \\nATOM    249  CB  ASP A  33       6.271  40.072  21.864  1.00 15.92           C  \\nATOM    250  CG  ASP A  33       4.993  40.654  21.269  1.00 18.19           C  \\nATOM    251  OD1 ASP A  33       3.923  40.631  21.879  1.00 21.46           O  \\nATOM    252  OD2 ASP A  33       5.074  41.164  20.167  1.00 20.34           O  \\nATOM    253  N   PRO A  34       4.752  37.283  23.903  1.00 12.04           N  \\nATOM    254  CA  PRO A  34       3.949  36.070  23.788  1.00 11.09           C  \\nATOM    255  C   PRO A  34       2.720  36.248  22.895  1.00 11.76           C  \\nATOM    256  O   PRO A  34       1.953  37.220  23.006  1.00 12.63           O  \\nATOM    257  CB  PRO A  34       3.590  35.711  25.219  1.00 12.50           C  \\nATOM    258  CG  PRO A  34       4.643  36.418  26.033  1.00 12.14           C  \\nATOM    259  CD  PRO A  34       4.817  37.717  25.311  1.00 12.82           C  \\nATOM    260  N   THR A  35       2.554  35.282  21.972  1.00 11.00           N  \\nATOM    261  CA  THR A  35       1.421  35.320  21.033  1.00 11.54           C  \\nATOM    262  C   THR A  35       0.140  34.746  21.639  1.00 13.51           C  \\nATOM    263  O   THR A  35       0.174  33.820  22.450  1.00 12.45           O  \\nATOM    264  CB  THR A  35       1.730  34.518  19.751  1.00 11.34           C  \\nATOM    265  OG1 THR A  35       2.900  35.134  19.233  1.00 11.20           O  \\nATOM    266  CG2 THR A  35       0.706  34.611  18.637  1.00 13.10           C  \\nATOM    267  N   ILE A  36      -0.996  35.332  21.233  1.00 15.48           N  \\nATOM    268  CA  ILE A  36      -2.308  34.792  21.567  1.00 19.42           C  \\nATOM    269  C   ILE A  36      -2.663  34.079  20.274  1.00 17.36           C  \\nATOM    270  O   ILE A  36      -2.749  32.863  20.256  1.00 18.29           O  \\nATOM    271  CB  ILE A  36      -3.353  35.914  21.879  1.00 21.30           C  \\nATOM    272  CG1 ILE A  36      -2.930  36.823  23.072  1.00 24.32           C  \\nATOM    273  CG2 ILE A  36      -4.670  35.207  22.136  1.00 21.54           C  \\nATOM    274  CD1 ILE A  36      -2.494  36.167  24.413  1.00 24.70           C  \\nATOM    275  N   GLU A  37      -2.818  34.822  19.181  1.00 19.35           N  \\nATOM    276  CA  GLU A  37      -3.059  34.221  17.889  1.00 21.47           C  \\nATOM    277  C   GLU A  37      -2.957  35.362  16.897  1.00 23.77           C  \\nATOM    278  O   GLU A  37      -3.636  36.393  17.031  1.00 25.49           O  \\nATOM    279  CB  GLU A  37      -4.440  33.611  17.880  1.00 25.97           C  \\nATOM    280  CG  GLU A  37      -4.766  32.776  16.667  1.00 33.81           C  \\nATOM    281  CD  GLU A  37      -6.160  32.154  16.777  1.00 38.37           C  \\nATOM    282  OE1 GLU A  37      -7.095  32.664  16.149  1.00 37.86           O  \\nATOM    283  OE2 GLU A  37      -6.298  31.164  17.515  1.00 42.24           O  \\nATOM    284  N   ASP A  38      -2.070  35.212  15.911  1.00 19.96           N  \\nATOM    285  CA  ASP A  38      -1.899  36.178  14.839  1.00 17.94           C  \\nATOM    286  C   ASP A  38      -2.024  35.526  13.473  1.00 18.21           C  \\nATOM    287  O   ASP A  38      -1.582  34.393  13.294  1.00 16.96           O  \\nATOM    288  CB  ASP A  38      -0.540  36.833  14.965  1.00 18.83           C  \\nATOM    289  CG  ASP A  38      -0.507  37.817  16.133  1.00 20.08           C  \\nATOM    290  OD1 ASP A  38      -1.390  38.660  16.208  1.00 25.47           O  \\nATOM    291  OD2 ASP A  38       0.375  37.781  16.967  1.00 20.08           O  \\nATOM    292  N   SER A  39      -2.698  36.170  12.507  1.00 17.82           N  \\nATOM    293  CA  SER A  39      -2.691  35.658  11.133  1.00 18.16           C  \\nATOM    294  C   SER A  39      -2.012  36.627  10.171  1.00 16.95           C  \\nATOM    295  O   SER A  39      -2.053  37.853  10.333  1.00 15.80           O  \\nATOM    296  CB  SER A  39      -4.094  35.402  10.637  1.00 23.01           C  \\nATOM    297  OG  SER A  39      -4.873  36.571  10.754  1.00 27.13           O  \\nATOM    298  N   TYR A  40      -1.336  36.080   9.168  1.00 15.44           N  \\nATOM    299  CA  TYR A  40      -0.607  36.889   8.196  1.00 16.25           C  \\nATOM    300  C   TYR A  40      -1.018  36.383   6.830  1.00 17.06           C  \\nATOM    301  O   TYR A  40      -1.160  35.172   6.638  1.00 18.29           O  \\nATOM    302  CB  TYR A  40       0.909  36.724   8.370  1.00 16.59           C  \\nATOM    303  CG  TYR A  40       1.439  37.242   9.707  1.00 16.10           C  \\nATOM    304  CD1 TYR A  40       1.473  36.412  10.815  1.00 16.77           C  \\nATOM    305  CD2 TYR A  40       1.883  38.549   9.799  1.00 18.02           C  \\nATOM    306  CE1 TYR A  40       1.950  36.893  12.021  1.00 16.86           C  \\nATOM    307  CE2 TYR A  40       2.368  39.032  11.001  1.00 18.70           C  \\nATOM    308  CZ  TYR A  40       2.396  38.199  12.104  1.00 17.62           C  \\nATOM    309  OH  TYR A  40       2.853  38.695  13.304  1.00 18.37           O  \\nATOM    310  N   ARG A  41      -1.270  37.303   5.900  1.00 15.48           N  \\nATOM    311  CA  ARG A  41      -1.660  36.918   4.558  1.00 15.42           C  \\nATOM    312  C   ARG A  41      -0.730  37.606   3.586  1.00 16.76           C  \\nATOM    313  O   ARG A  41      -0.536  38.829   3.633  1.00 17.01           O  \\nATOM    314  CB  ARG A  41      -3.087  37.342   4.277  1.00 17.64           C  \\nATOM    315  CG  ARG A  41      -4.150  36.507   4.947  1.00 18.09           C  \\nATOM    316  CD  ARG A  41      -5.528  37.181   4.849  1.00 19.64           C  \\nATOM    317  NE  ARG A  41      -6.066  37.318   3.489  1.00 20.11           N  \\nATOM    318  CZ  ARG A  41      -6.660  36.329   2.803  1.00 22.17           C  \\nATOM    319  NH1 ARG A  41      -7.219  36.571   1.617  1.00 22.48           N  \\nATOM    320  NH2 ARG A  41      -6.711  35.090   3.274  1.00 23.55           N  \\nATOM    321  N   LYS A  42      -0.135  36.827   2.679  1.00 16.52           N  \\nATOM    322  CA  LYS A  42       0.835  37.405   1.758  1.00 16.59           C  \\nATOM    323  C   LYS A  42       0.740  36.746   0.387  1.00 16.04           C  \\nATOM    324  O   LYS A  42       0.794  35.528   0.307  1.00 14.74           O  \\nATOM    325  CB  LYS A  42       2.208  37.210   2.401  1.00 18.22           C  \\nATOM    326  CG  LYS A  42       3.388  37.784   1.669  1.00 22.68           C  \\nATOM    327  CD  LYS A  42       3.369  39.274   1.792  1.00 28.09           C  \\nATOM    328  CE  LYS A  42       4.695  39.876   1.317  1.00 31.70           C  \\nATOM    329  NZ  LYS A  42       4.828  39.827  -0.128  1.00 35.70           N  \\nATOM    330  N   GLN A  43       0.561  37.510  -0.697  1.00 16.89           N  \\nATOM    331  CA  GLN A  43       0.579  36.950  -2.047  1.00 19.10           C  \\nATOM    332  C   GLN A  43       2.026  36.729  -2.448  1.00 15.08           C  \\nATOM    333  O   GLN A  43       2.828  37.654  -2.362  1.00 15.72           O  \\nATOM    334  CB  GLN A  43      -0.116  37.908  -3.043  1.00 23.32           C  \\nATOM    335  CG  GLN A  43       0.543  38.332  -4.415  1.00 32.83           C  \\nATOM    336  CD  GLN A  43       1.081  37.332  -5.482  1.00 36.11           C  \\nATOM    337  OE1 GLN A  43       0.457  36.335  -5.865  1.00 37.78           O  \\nATOM    338  NE2 GLN A  43       2.271  37.606  -6.053  1.00 35.45           N  \\nATOM    339  N   VAL A  44       2.394  35.532  -2.922  1.00 12.79           N  \\nATOM    340  CA  VAL A  44       3.771  35.242  -3.311  1.00 11.03           C  \\nATOM    341  C   VAL A  44       3.724  34.425  -4.605  1.00 14.28           C  \\nATOM    342  O   VAL A  44       2.633  34.109  -5.090  1.00 14.90           O  \\nATOM    343  CB  VAL A  44       4.508  34.411  -2.223  1.00 13.21           C  \\nATOM    344  CG1 VAL A  44       4.669  35.282  -0.977  1.00 15.04           C  \\nATOM    345  CG2 VAL A  44       3.761  33.116  -1.889  1.00 12.89           C  \\nATOM    346  N   VAL A  45       4.879  34.127  -5.235  1.00 13.20           N  \\nATOM    347  CA  VAL A  45       4.938  33.277  -6.419  1.00 12.28           C  \\nATOM    348  C   VAL A  45       5.836  32.161  -5.970  1.00 11.08           C  \\nATOM    349  O   VAL A  45       6.988  32.447  -5.636  1.00 13.13           O  \\nATOM    350  CB  VAL A  45       5.587  33.979  -7.641  1.00 13.01           C  \\nATOM    351  CG1 VAL A  45       5.549  33.031  -8.810  1.00 14.51           C  \\nATOM    352  CG2 VAL A  45       4.787  35.173  -8.108  1.00 13.54           C  \\nATOM    353  N   ILE A  46       5.374  30.907  -5.915  1.00  9.89           N  \\nATOM    354  CA  ILE A  46       6.274  29.795  -5.491  1.00 11.03           C  \\nATOM    355  C   ILE A  46       6.369  28.800  -6.662  1.00  8.85           C  \\nATOM    356  O   ILE A  46       5.319  28.296  -7.059  1.00 10.25           O  \\nATOM    357  CB  ILE A  46       5.692  29.115  -4.203  1.00 10.64           C  \\nATOM    358  CG1 ILE A  46       5.652  30.096  -3.042  1.00 11.20           C  \\nATOM    359  CG2 ILE A  46       6.599  27.953  -3.757  1.00 12.93           C  \\nATOM    360  CD1 ILE A  46       4.937  29.536  -1.808  1.00  8.90           C  \\nATOM    361  N   ASP A  47       7.565  28.495  -7.224  1.00  9.41           N  \\nATOM    362  CA  ASP A  47       7.767  27.650  -8.426  1.00  9.70           C  \\nATOM    363  C   ASP A  47       6.838  28.031  -9.572  1.00 11.14           C  \\nATOM    364  O   ASP A  47       6.225  27.211 -10.248  1.00 14.09           O  \\nATOM    365  CB  ASP A  47       7.566  26.165  -8.105  1.00 10.73           C  \\nATOM    366  CG  ASP A  47       8.388  25.783  -6.874  1.00 13.52           C  \\nATOM    367  OD1 ASP A  47       7.805  25.291  -5.898  1.00 12.04           O  \\nATOM    368  OD2 ASP A  47       9.598  26.024  -6.872  1.00 11.31           O  \\nATOM    369  N   GLY A  48       6.724  29.353  -9.735  1.00 10.78           N  \\nATOM    370  CA  GLY A  48       5.930  29.961 -10.785  1.00 13.29           C  \\nATOM    371  C   GLY A  48       4.426  29.989 -10.548  1.00 14.28           C  \\nATOM    372  O   GLY A  48       3.677  30.512 -11.386  1.00 15.19           O  \\nATOM    373  N   GLU A  49       3.944  29.422  -9.439  1.00 13.41           N  \\nATOM    374  CA  GLU A  49       2.520  29.457  -9.143  1.00 13.90           C  \\nATOM    375  C   GLU A  49       2.223  30.700  -8.314  1.00 15.34           C  \\nATOM    376  O   GLU A  49       2.860  30.906  -7.275  1.00 14.56           O  \\nATOM    377  CB  GLU A  49       2.124  28.238  -8.340  1.00 15.50           C  \\nATOM    378  CG  GLU A  49       0.630  28.250  -7.990  1.00 18.10           C  \\nATOM    379  CD  GLU A  49       0.204  27.110  -7.100  1.00 19.13           C  \\nATOM    380  OE1 GLU A  49       0.717  27.004  -5.996  1.00 19.53           O  \\nATOM    381  OE2 GLU A  49      -0.630  26.321  -7.529  1.00 22.12           O  \\nATOM    382  N   THR A  50       1.267  31.536  -8.734  1.00 14.06           N  \\nATOM    383  CA  THR A  50       0.901  32.697  -7.927  1.00 16.15           C  \\nATOM    384  C   THR A  50      -0.001  32.169  -6.819  1.00 13.35           C  \\nATOM    385  O   THR A  50      -0.938  31.409  -7.046  1.00 15.13           O  \\nATOM    386  CB  THR A  50       0.203  33.755  -8.846  1.00 17.88           C  \\nATOM    387  OG1 THR A  50      -0.245  34.813  -8.012  1.00 24.33           O  \\nATOM    388  CG2 THR A  50      -1.028  33.236  -9.518  1.00 19.78           C  \\nATOM    389  N   CYS A  51       0.306  32.442  -5.563  1.00 13.28           N  \\nATOM    390  CA  CYS A  51      -0.566  31.959  -4.508  1.00 15.16           C  \\nATOM    391  C   CYS A  51      -0.579  32.892  -3.303  1.00 14.00           C  \\nATOM    392  O   CYS A  51       0.161  33.874  -3.231  1.00 11.81           O  \\nATOM    393  CB  CYS A  51      -0.132  30.537  -4.079  1.00 14.08           C  \\nATOM    394  SG  CYS A  51       1.597  30.409  -3.575  1.00 16.09           S  \\nATOM    395  N   LEU A  52      -1.521  32.609  -2.405  1.00 13.77           N  \\nATOM    396  CA  LEU A  52      -1.573  33.397  -1.199  1.00 16.01           C  \\nATOM    397  C   LEU A  52      -1.260  32.497  -0.025  1.00 14.53           C  \\nATOM    398  O   LEU A  52      -1.868  31.443   0.113  1.00 16.05           O  \\nATOM    399  CB  LEU A  52      -2.957  33.986  -1.014  1.00 19.71           C  \\nATOM    400  CG  LEU A  52      -3.122  34.885   0.214  1.00 20.84           C  \\nATOM    401  CD1 LEU A  52      -3.070  36.329  -0.220  1.00 21.38           C  \\nATOM    402  CD2 LEU A  52      -4.420  34.526   0.923  1.00 20.90           C  \\nATOM    403  N   LEU A  53      -0.319  32.904   0.800  1.00 12.41           N  \\nATOM    404  CA  LEU A  53       0.019  32.232   2.048  1.00 13.70           C  \\nATOM    405  C   LEU A  53      -0.841  32.852   3.148  1.00 13.85           C  \\nATOM    406  O   LEU A  53      -0.733  34.056   3.413  1.00 12.82           O  \\nATOM    407  CB  LEU A  53       1.485  32.464   2.400  1.00 14.51           C  \\nATOM    408  CG  LEU A  53       2.707  31.589   2.031  1.00 19.29           C  \\nATOM    409  CD1 LEU A  53       2.394  30.553   0.962  1.00 18.56           C  \\nATOM    410  CD2 LEU A  53       3.843  32.557   1.741  1.00 20.00           C  \\nATOM    411  N   ASP A  54      -1.699  32.023   3.751  1.00 11.96           N  \\nATOM    412  CA  ASP A  54      -2.476  32.445   4.884  1.00 15.07           C  \\nATOM    413  C   ASP A  54      -1.882  31.675   6.055  1.00 12.67           C  \\nATOM    414  O   ASP A  54      -2.099  30.477   6.234  1.00 13.99           O  \\nATOM    415  CB  ASP A  54      -3.930  32.088   4.644  1.00 19.33           C  \\nATOM    416  CG  ASP A  54      -4.826  32.609   5.746  1.00 24.24           C  \\nATOM    417  OD1 ASP A  54      -4.412  32.652   6.897  1.00 28.17           O  \\nATOM    418  OD2 ASP A  54      -5.958  32.979   5.450  1.00 27.59           O  \\nATOM    419  N   ILE A  55      -1.127  32.389   6.880  1.00 11.78           N  \\nATOM    420  CA  ILE A  55      -0.346  31.769   7.935  1.00 12.15           C  \\nATOM    421  C   ILE A  55      -0.953  32.099   9.296  1.00 13.10           C  \\nATOM    422  O   ILE A  55      -1.195  33.271   9.587  1.00 13.84           O  \\nATOM    423  CB  ILE A  55       1.109  32.282   7.931  1.00 12.42           C  \\nATOM    424  CG1 ILE A  55       1.859  31.979   6.653  1.00 13.99           C  \\nATOM    425  CG2 ILE A  55       1.859  31.505   9.018  1.00 13.24           C  \\nATOM    426  CD1 ILE A  55       3.176  32.768   6.497  1.00 15.68           C  \\nATOM    427  N   LEU A  56      -1.176  31.083  10.134  1.00 11.64           N  \\nATOM    428  CA  LEU A  56      -1.642  31.341  11.490  1.00 11.98           C  \\nATOM    429  C   LEU A  56      -0.524  31.027  12.431  1.00  9.99           C  \\nATOM    430  O   LEU A  56      -0.017  29.895  12.447  1.00  9.72           O  \\nATOM    431  CB  LEU A  56      -2.836  30.463  11.896  1.00 13.80           C  \\nATOM    432  CG  LEU A  56      -3.470  30.741  13.297  1.00 16.13           C  \\nATOM    433  CD1 LEU A  56      -4.347  31.987  13.260  1.00 15.28           C  \\nATOM    434  CD2 LEU A  56      -4.321  29.580  13.692  1.00 16.84           C  \\nATOM    435  N   ASP A  57      -0.188  32.084  13.182  1.00 10.40           N  \\nATOM    436  CA  ASP A  57       0.815  32.014  14.219  1.00 10.30           C  \\nATOM    437  C   ASP A  57       0.127  31.812  15.571  1.00 11.75           C  \\nATOM    438  O   ASP A  57      -0.652  32.631  16.053  1.00 12.91           O  \\nATOM    439  CB  ASP A  57       1.602  33.296  14.173  1.00 10.66           C  \\nATOM    440  CG  ASP A  57       2.778  33.336  15.132  1.00 11.39           C  \\nATOM    441  OD1 ASP A  57       3.306  32.298  15.535  1.00 11.07           O  \\nATOM    442  OD2 ASP A  57       3.168  34.448  15.455  1.00 13.15           O  \\nATOM    443  N   THR A  58       0.447  30.723  16.244  1.00 12.55           N  \\nATOM    444  CA  THR A  58      -0.324  30.325  17.416  1.00 12.98           C  \\nATOM    445  C   THR A  58       0.368  30.541  18.758  1.00 13.48           C  \\nATOM    446  O   THR A  58       1.568  30.812  18.815  1.00 14.00           O  \\nATOM    447  CB  THR A  58      -0.698  28.851  17.219  1.00  9.54           C  \\nATOM    448  OG1 THR A  58       0.491  28.084  17.369  1.00 11.40           O  \\nATOM    449  CG2 THR A  58      -1.313  28.597  15.848  1.00 12.27           C  \\nATOM    450  N   ALA A  59      -0.371  30.382  19.868  1.00 14.38           N  \\nATOM    451  CA  ALA A  59       0.230  30.510  21.194  1.00 14.79           C  \\nATOM    452  C   ALA A  59       0.928  29.210  21.608  1.00 15.41           C  \\nATOM    453  O   ALA A  59       0.448  28.106  21.324  1.00 13.07           O  \\nATOM    454  CB  ALA A  59      -0.845  30.818  22.238  1.00 15.47           C  \\nATOM    455  N   GLY A  60       2.084  29.341  22.271  1.00 17.71           N  \\nATOM    456  CA  GLY A  60       2.807  28.208  22.814  1.00 24.18           C  \\nATOM    457  C   GLY A  60       2.190  27.751  24.130  1.00 31.21           C  \\nATOM    458  O   GLY A  60       2.538  26.688  24.627  1.00 30.44           O  \\nATOM    459  N   GLN A  61       1.260  28.503  24.726  1.00 39.50           N  \\nATOM    460  CA  GLN A  61       0.671  28.099  25.994  1.00 48.84           C  \\nATOM    461  C   GLN A  61      -0.683  27.395  25.985  1.00 51.52           C  \\nATOM    462  O   GLN A  61      -1.606  27.780  25.276  1.00 52.17           O  \\nATOM    463  CB  GLN A  61       0.605  29.327  26.874  1.00 51.56           C  \\nATOM    464  CG  GLN A  61       0.871  28.861  28.312  1.00 56.74           C  \\nATOM    465  CD  GLN A  61       1.506  29.937  29.177  1.00 58.74           C  \\nATOM    466  OE1 GLN A  61       2.647  29.795  29.616  1.00 60.12           O  \\nATOM    467  NE2 GLN A  61       0.783  31.031  29.429  1.00 60.18           N  \\nATOM    468  N   GLU A  62      -0.804  26.349  26.818  1.00 57.40           N  \\nATOM    469  CA  GLU A  62      -2.008  25.527  26.927  1.00 61.96           C  \\nATOM    470  C   GLU A  62      -3.185  26.196  27.631  1.00 62.20           C  \\nATOM    471  O   GLU A  62      -4.138  25.496  27.975  1.00 62.85           O  \\nATOM    472  CB  GLU A  62      -1.629  24.183  27.632  1.00 65.18           C  \\nATOM    473  CG  GLU A  62      -2.622  22.966  27.761  1.00 68.22           C  \\nATOM    474  CD  GLU A  62      -3.032  22.132  26.528  1.00 70.25           C  \\nATOM    475  OE1 GLU A  62      -2.710  22.494  25.395  1.00 71.45           O  \\nATOM    476  OE2 GLU A  62      -3.691  21.101  26.689  1.00 70.00           O  \\nATOM    477  N   GLU A  63      -3.206  27.520  27.904  1.00 62.78           N  \\nATOM    478  CA  GLU A  63      -4.496  28.139  28.254  1.00 63.62           C  \\nATOM    479  C   GLU A  63      -5.457  27.854  27.085  1.00 63.71           C  \\nATOM    480  O   GLU A  63      -6.688  27.814  27.211  1.00 62.97           O  \\nATOM    481  CB  GLU A  63      -4.372  29.666  28.475  1.00 65.13           C  \\nATOM    482  CG  GLU A  63      -3.456  30.553  27.607  1.00 66.91           C  \\nATOM    483  CD  GLU A  63      -4.012  31.235  26.345  1.00 68.05           C  \\nATOM    484  OE1 GLU A  63      -5.220  31.462  26.220  1.00 68.64           O  \\nATOM    485  OE2 GLU A  63      -3.198  31.568  25.481  1.00 67.62           O  \\nATOM    486  N   TYR A  64      -4.801  27.606  25.934  1.00 63.36           N  \\nATOM    487  CA  TYR A  64      -5.420  27.092  24.743  1.00 63.20           C  \\nATOM    488  C   TYR A  64      -5.905  25.653  24.875  1.00 61.76           C  \\nATOM    489  O   TYR A  64      -7.120  25.486  24.858  1.00 63.04           O  \\nATOM    490  CB  TYR A  64      -4.437  27.208  23.613  1.00 64.83           C  \\nATOM    491  CG  TYR A  64      -4.736  28.459  22.818  1.00 66.98           C  \\nATOM    492  CD1 TYR A  64      -4.300  29.686  23.267  1.00 67.55           C  \\nATOM    493  CD2 TYR A  64      -5.459  28.355  21.646  1.00 67.97           C  \\nATOM    494  CE1 TYR A  64      -4.587  30.825  22.536  1.00 69.49           C  \\nATOM    495  CE2 TYR A  64      -5.752  29.488  20.911  1.00 69.29           C  \\nATOM    496  CZ  TYR A  64      -5.313  30.718  21.359  1.00 70.05           C  \\nATOM    497  OH  TYR A  64      -5.589  31.852  20.614  1.00 70.67           O  \\nATOM    498  N   SER A  65      -5.095  24.590  24.997  1.00 58.67           N  \\nATOM    499  CA  SER A  65      -5.617  23.230  25.127  1.00 56.93           C  \\nATOM    500  C   SER A  65      -6.614  22.757  24.063  1.00 55.33           C  \\nATOM    501  O   SER A  65      -6.321  22.936  22.876  1.00 54.74           O  \\nATOM    502  CB  SER A  65      -6.190  23.122  26.552  1.00 57.99           C  \\nATOM    503  OG  SER A  65      -7.251  24.011  26.900  1.00 59.80           O  \\nATOM    504  N   ALA A  66      -7.762  22.119  24.359  1.00 52.33           N  \\nATOM    505  CA  ALA A  66      -8.766  21.824  23.335  1.00 48.80           C  \\nATOM    506  C   ALA A  66      -9.138  23.050  22.492  1.00 45.71           C  \\nATOM    507  O   ALA A  66      -9.751  22.862  21.449  1.00 44.33           O  \\nATOM    508  CB  ALA A  66     -10.043  21.300  23.971  1.00 49.14           C  \\nATOM    509  N   MET A  67      -8.829  24.310  22.875  1.00 42.85           N  \\nATOM    510  CA  MET A  67      -8.984  25.465  21.967  1.00 40.01           C  \\nATOM    511  C   MET A  67      -8.093  25.362  20.719  1.00 36.79           C  \\nATOM    512  O   MET A  67      -8.221  26.184  19.805  1.00 36.08           O  \\nATOM    513  CB  MET A  67      -8.594  26.804  22.581  1.00 42.50           C  \\nATOM    514  CG  MET A  67      -9.359  27.330  23.772  1.00 45.93           C  \\nATOM    515  SD  MET A  67     -11.140  27.396  23.483  1.00 48.74           S  \\nATOM    516  CE  MET A  67     -11.526  25.880  24.321  1.00 47.64           C  \\nATOM    517  N   ARG A  68      -7.128  24.418  20.665  1.00 31.59           N  \\nATOM    518  CA  ARG A  68      -6.322  24.217  19.467  1.00 26.34           C  \\nATOM    519  C   ARG A  68      -7.140  23.473  18.424  1.00 23.68           C  \\nATOM    520  O   ARG A  68      -6.846  23.617  17.231  1.00 21.93           O  \\nATOM    521  CB  ARG A  68      -5.057  23.394  19.745  1.00 25.86           C  \\nATOM    522  CG  ARG A  68      -4.161  23.898  20.869  1.00 29.36           C  \\nATOM    523  CD  ARG A  68      -2.881  23.061  20.998  1.00 32.88           C  \\nATOM    524  NE  ARG A  68      -2.221  22.990  22.315  1.00 36.64           N  \\nATOM    525  CZ  ARG A  68      -1.700  24.031  23.001  1.00 39.66           C  \\nATOM    526  NH1 ARG A  68      -1.903  25.315  22.664  1.00 42.06           N  \\nATOM    527  NH2 ARG A  68      -0.933  23.798  24.071  1.00 39.61           N  \\nATOM    528  N   ASP A  69      -8.188  22.721  18.793  1.00 19.14           N  \\nATOM    529  CA  ASP A  69      -8.880  21.905  17.822  1.00 20.43           C  \\nATOM    530  C   ASP A  69      -9.437  22.623  16.617  1.00 22.16           C  \\nATOM    531  O   ASP A  69      -9.453  22.109  15.488  1.00 19.58           O  \\nATOM    532  CB  ASP A  69     -10.025  21.148  18.478  1.00 20.58           C  \\nATOM    533  CG  ASP A  69      -9.598  19.935  19.303  1.00 22.81           C  \\nATOM    534  OD1 ASP A  69      -8.409  19.708  19.538  1.00 24.59           O  \\nATOM    535  OD2 ASP A  69     -10.475  19.176  19.715  1.00 25.28           O  \\nATOM    536  N   GLN A  70      -9.834  23.874  16.840  1.00 23.26           N  \\nATOM    537  CA  GLN A  70     -10.451  24.609  15.759  1.00 23.53           C  \\nATOM    538  C   GLN A  70      -9.437  24.925  14.667  1.00 21.20           C  \\nATOM    539  O   GLN A  70      -9.653  24.591  13.493  1.00 20.90           O  \\nATOM    540  CB  GLN A  70     -11.071  25.881  16.341  1.00 27.73           C  \\nATOM    541  CG  GLN A  70     -12.498  26.019  15.832  1.00 34.11           C  \\nATOM    542  CD  GLN A  70     -13.103  27.407  16.008  1.00 37.87           C  \\nATOM    543  OE1 GLN A  70     -12.532  28.284  16.672  1.00 39.77           O  \\nATOM    544  NE2 GLN A  70     -14.283  27.620  15.406  1.00 37.89           N  \\nATOM    545  N   TYR A  71      -8.292  25.515  15.050  1.00 19.09           N  \\nATOM    546  CA  TYR A  71      -7.329  25.873  14.030  1.00 17.69           C  \\nATOM    547  C   TYR A  71      -6.648  24.658  13.416  1.00 17.24           C  \\nATOM    548  O   TYR A  71      -6.247  24.675  12.245  1.00 17.25           O  \\nATOM    549  CB  TYR A  71      -6.302  26.868  14.625  1.00 17.87           C  \\nATOM    550  CG  TYR A  71      -5.311  26.480  15.725  1.00 18.90           C  \\nATOM    551  CD1 TYR A  71      -4.355  25.510  15.492  1.00 19.74           C  \\nATOM    552  CD2 TYR A  71      -5.328  27.162  16.940  1.00 19.21           C  \\nATOM    553  CE1 TYR A  71      -3.402  25.212  16.450  1.00 20.38           C  \\nATOM    554  CE2 TYR A  71      -4.375  26.868  17.901  1.00 21.93           C  \\nATOM    555  CZ  TYR A  71      -3.422  25.897  17.645  1.00 21.91           C  \\nATOM    556  OH  TYR A  71      -2.463  25.590  18.588  1.00 26.44           O  \\nATOM    557  N   MET A  72      -6.557  23.559  14.166  1.00 14.81           N  \\nATOM    558  CA  MET A  72      -5.958  22.393  13.563  1.00 18.59           C  \\nATOM    559  C   MET A  72      -6.849  21.778  12.472  1.00 19.38           C  \\nATOM    560  O   MET A  72      -6.385  21.198  11.492  1.00 17.67           O  \\nATOM    561  CB  MET A  72      -5.658  21.386  14.669  1.00 17.39           C  \\nATOM    562  CG  MET A  72      -4.479  21.796  15.514  1.00 16.35           C  \\nATOM    563  SD  MET A  72      -4.146  20.600  16.820  1.00 17.04           S  \\nATOM    564  CE  MET A  72      -2.635  21.259  17.460  1.00 17.72           C  \\nATOM    565  N   ARG A  73      -8.174  21.931  12.626  1.00 20.31           N  \\nATOM    566  CA  ARG A  73      -9.120  21.350  11.701  1.00 22.24           C  \\nATOM    567  C   ARG A  73      -9.087  22.084  10.370  1.00 20.94           C  \\nATOM    568  O   ARG A  73      -9.183  21.450   9.326  1.00 20.81           O  \\nATOM    569  CB  ARG A  73     -10.499  21.414  12.369  1.00 25.29           C  \\nATOM    570  CG  ARG A  73     -11.553  20.499  11.767  1.00 30.04           C  \\nATOM    571  CD  ARG A  73     -12.775  20.429  12.687  1.00 31.86           C  \\nATOM    572  NE  ARG A  73     -12.483  19.702  13.917  1.00 34.70           N  \\nATOM    573  CZ  ARG A  73     -12.894  20.122  15.130  1.00 36.49           C  \\nATOM    574  NH1 ARG A  73     -12.579  19.402  16.196  1.00 36.93           N  \\nATOM    575  NH2 ARG A  73     -13.609  21.239  15.311  1.00 38.17           N  \\nATOM    576  N   THR A  74      -8.868  23.401  10.413  1.00 21.33           N  \\nATOM    577  CA  THR A  74      -8.813  24.240   9.219  1.00 24.99           C  \\nATOM    578  C   THR A  74      -7.436  24.351   8.544  1.00 23.10           C  \\nATOM    579  O   THR A  74      -7.367  24.738   7.371  1.00 22.59           O  \\nATOM    580  CB  THR A  74      -9.289  25.676   9.538  1.00 25.73           C  \\nATOM    581  OG1 THR A  74      -8.351  26.302  10.409  1.00 31.08           O  \\nATOM    582  CG2 THR A  74     -10.657  25.644  10.201  1.00 27.58           C  \\nATOM    583  N   GLY A  75      -6.345  24.084   9.289  1.00 19.58           N  \\nATOM    584  CA  GLY A  75      -5.013  24.114   8.730  1.00 15.79           C  \\nATOM    585  C   GLY A  75      -4.823  23.023   7.713  1.00 13.66           C  \\nATOM    586  O   GLY A  75      -5.245  21.880   7.884  1.00 14.69           O  \\nATOM    587  N   GLU A  76      -4.228  23.360   6.573  1.00 13.14           N  \\nATOM    588  CA  GLU A  76      -3.914  22.363   5.546  1.00 12.99           C  \\nATOM    589  C   GLU A  76      -2.514  21.740   5.753  1.00 12.22           C  \\nATOM    590  O   GLU A  76      -2.293  20.576   5.426  1.00 13.59           O  \\nATOM    591  CB  GLU A  76      -3.999  23.014   4.145  1.00 13.07           C  \\nATOM    592  CG  GLU A  76      -5.387  23.377   3.676  1.00 14.35           C  \\nATOM    593  CD  GLU A  76      -5.261  24.069   2.323  1.00 20.40           C  \\nATOM    594  OE1 GLU A  76      -5.461  23.403   1.288  1.00 24.32           O  \\nATOM    595  OE2 GLU A  76      -4.913  25.250   2.284  1.00 18.82           O  \\nATOM    596  N   GLY A  77      -1.553  22.503   6.301  1.00 10.73           N  \\nATOM    597  CA  GLY A  77      -0.216  22.018   6.570  1.00  8.07           C  \\nATOM    598  C   GLY A  77       0.302  22.689   7.819  1.00  8.63           C  \\nATOM    599  O   GLY A  77      -0.066  23.823   8.127  1.00  8.55           O  \\nATOM    600  N   PHE A  78       1.248  22.013   8.479  1.00  9.15           N  \\nATOM    601  CA  PHE A  78       1.754  22.468   9.773  1.00  8.87           C  \\nATOM    602  C   PHE A  78       3.268  22.520   9.854  1.00  9.51           C  \\nATOM    603  O   PHE A  78       3.930  21.542   9.486  1.00  9.15           O  \\nATOM    604  CB  PHE A  78       1.247  21.532  10.904  1.00  9.44           C  \\nATOM    605  CG  PHE A  78      -0.267  21.472  11.000  1.00  9.66           C  \\nATOM    606  CD1 PHE A  78      -0.906  22.310  11.901  1.00 10.71           C  \\nATOM    607  CD2 PHE A  78      -0.994  20.645  10.152  1.00 10.20           C  \\nATOM    608  CE1 PHE A  78      -2.295  22.338  11.940  1.00 10.61           C  \\nATOM    609  CE2 PHE A  78      -2.378  20.678  10.192  1.00 12.55           C  \\nATOM    610  CZ  PHE A  78      -3.022  21.529  11.081  1.00 12.80           C  \\nATOM    611  N   LEU A  79       3.813  23.636  10.354  1.00  8.72           N  \\nATOM    612  CA  LEU A  79       5.230  23.626  10.699  1.00  9.93           C  \\nATOM    613  C   LEU A  79       5.279  23.281  12.181  1.00  9.09           C  \\nATOM    614  O   LEU A  79       4.701  24.008  12.988  1.00 11.58           O  \\nATOM    615  CB  LEU A  79       5.921  24.970  10.567  1.00 12.80           C  \\nATOM    616  CG  LEU A  79       6.155  25.659   9.249  1.00 16.39           C  \\nATOM    617  CD1 LEU A  79       7.007  26.911   9.544  1.00 17.02           C  \\nATOM    618  CD2 LEU A  79       6.904  24.756   8.269  1.00 16.98           C  \\nATOM    619  N   CYS A  80       5.950  22.219  12.579  1.00  9.14           N  \\nATOM    620  CA  CYS A  80       6.074  21.821  13.981  1.00 11.81           C  \\nATOM    621  C   CYS A  80       7.448  22.237  14.434  1.00 11.53           C  \\nATOM    622  O   CYS A  80       8.435  21.640  14.000  1.00 12.21           O  \\nATOM    623  CB  CYS A  80       5.903  20.318  14.106  1.00 13.73           C  \\nATOM    624  SG  CYS A  80       4.158  19.900  13.837  1.00 18.26           S  \\nATOM    625  N   VAL A  81       7.486  23.282  15.269  1.00  7.57           N  \\nATOM    626  CA  VAL A  81       8.730  23.956  15.604  1.00  8.45           C  \\nATOM    627  C   VAL A  81       9.248  23.620  16.996  1.00  8.90           C  \\nATOM    628  O   VAL A  81       8.451  23.680  17.939  1.00  8.49           O  \\nATOM    629  CB  VAL A  81       8.544  25.516  15.507  1.00  7.40           C  \\nATOM    630  CG1 VAL A  81       9.917  26.196  15.588  1.00  9.42           C  \\nATOM    631  CG2 VAL A  81       8.004  25.954  14.167  1.00  8.59           C  \\nATOM    632  N   PHE A  82      10.546  23.237  17.119  1.00  9.49           N  \\nATOM    633  CA  PHE A  82      11.195  23.202  18.442  1.00  9.09           C  \\nATOM    634  C   PHE A  82      12.511  23.994  18.350  1.00  9.83           C  \\nATOM    635  O   PHE A  82      12.914  24.440  17.266  1.00  9.02           O  \\nATOM    636  CB  PHE A  82      11.482  21.755  18.902  1.00  9.76           C  \\nATOM    637  CG  PHE A  82      12.536  21.062  18.071  1.00 10.09           C  \\nATOM    638  CD1 PHE A  82      13.867  21.099  18.486  1.00 10.70           C  \\nATOM    639  CD2 PHE A  82      12.192  20.412  16.889  1.00 10.93           C  \\nATOM    640  CE1 PHE A  82      14.838  20.483  17.699  1.00 11.12           C  \\nATOM    641  CE2 PHE A  82      13.181  19.807  16.125  1.00 10.73           C  \\nATOM    642  CZ  PHE A  82      14.505  19.839  16.523  1.00 10.35           C  \\nATOM    643  N   ALA A  83      13.128  24.269  19.516  1.00  9.75           N  \\nATOM    644  CA  ALA A  83      14.407  24.963  19.569  1.00 10.32           C  \\nATOM    645  C   ALA A  83      15.509  23.960  19.916  1.00  9.48           C  \\nATOM    646  O   ALA A  83      15.362  23.142  20.839  1.00  9.61           O  \\nATOM    647  CB  ALA A  83      14.354  26.083  20.620  1.00  9.75           C  \\nATOM    648  N   ILE A  84      16.624  24.042  19.160  1.00 11.41           N  \\nATOM    649  CA  ILE A  84      17.714  23.070  19.299  1.00 13.15           C  \\nATOM    650  C   ILE A  84      18.446  23.223  20.623  1.00 14.57           C  \\nATOM    651  O   ILE A  84      19.234  22.347  20.971  1.00 13.94           O  \\nATOM    652  CB  ILE A  84      18.775  23.151  18.160  1.00 13.29           C  \\nATOM    653  CG1 ILE A  84      19.403  24.509  17.878  1.00 14.90           C  \\nATOM    654  CG2 ILE A  84      18.018  22.701  16.933  1.00 13.09           C  \\nATOM    655  CD1 ILE A  84      20.578  24.957  18.763  1.00 17.59           C  \\nATOM    656  N   ASN A  85      18.200  24.290  21.390  1.00 13.61           N  \\nATOM    657  CA  ASN A  85      18.794  24.394  22.720  1.00 15.52           C  \\nATOM    658  C   ASN A  85      17.758  24.156  23.828  1.00 15.49           C  \\nATOM    659  O   ASN A  85      17.988  24.519  24.999  1.00 16.13           O  \\nATOM    660  CB  ASN A  85      19.446  25.788  22.885  1.00 15.16           C  \\nATOM    661  CG  ASN A  85      18.464  26.946  22.904  1.00 16.42           C  \\nATOM    662  OD1 ASN A  85      17.355  26.878  22.362  1.00 16.01           O  \\nATOM    663  ND2 ASN A  85      18.808  28.030  23.587  1.00 16.83           N  \\nATOM    664  N   ASN A  86      16.616  23.515  23.509  1.00 14.72           N  \\nATOM    665  CA  ASN A  86      15.553  23.303  24.483  1.00 13.01           C  \\nATOM    666  C   ASN A  86      14.987  21.901  24.309  1.00 13.17           C  \\nATOM    667  O   ASN A  86      14.080  21.668  23.520  1.00 13.04           O  \\nATOM    668  CB  ASN A  86      14.478  24.367  24.256  1.00 16.51           C  \\nATOM    669  CG  ASN A  86      13.267  24.415  25.187  1.00 20.74           C  \\nATOM    670  OD1 ASN A  86      12.370  25.241  25.018  1.00 25.41           O  \\nATOM    671  ND2 ASN A  86      13.123  23.596  26.206  1.00 19.96           N  \\nATOM    672  N   THR A  87      15.488  20.950  25.103  1.00 13.80           N  \\nATOM    673  CA  THR A  87      15.095  19.553  25.026  1.00 15.40           C  \\nATOM    674  C   THR A  87      13.589  19.316  25.266  1.00 14.40           C  \\nATOM    675  O   THR A  87      12.953  18.535  24.546  1.00 14.11           O  \\nATOM    676  CB  THR A  87      15.983  18.791  26.046  1.00 16.19           C  \\nATOM    677  OG1 THR A  87      17.325  18.908  25.571  1.00 18.22           O  \\nATOM    678  CG2 THR A  87      15.626  17.327  26.180  1.00 17.82           C  \\nATOM    679  N   LYS A  88      12.983  19.958  26.272  1.00 14.42           N  \\nATOM    680  CA  LYS A  88      11.552  19.842  26.535  1.00 15.84           C  \\nATOM    681  C   LYS A  88      10.726  20.232  25.278  1.00 15.06           C  \\nATOM    682  O   LYS A  88       9.722  19.582  25.003  1.00 14.61           O  \\nATOM    683  CB  LYS A  88      11.207  20.766  27.717  1.00 19.53           C  \\nATOM    684  CG  LYS A  88       9.937  20.401  28.457  1.00 27.71           C  \\nATOM    685  CD  LYS A  88       9.585  21.452  29.529  1.00 33.26           C  \\nATOM    686  CE  LYS A  88       9.061  22.762  28.877  1.00 36.79           C  \\nATOM    687  NZ  LYS A  88       8.642  23.763  29.852  1.00 37.29           N  \\nATOM    688  N   SER A  89      11.119  21.259  24.475  1.00 12.36           N  \\nATOM    689  CA  SER A  89      10.325  21.666  23.309  1.00 11.31           C  \\nATOM    690  C   SER A  89      10.371  20.572  22.268  1.00 12.51           C  \\nATOM    691  O   SER A  89       9.397  20.388  21.547  1.00 12.88           O  \\nATOM    692  CB  SER A  89      10.844  22.951  22.674  1.00 11.64           C  \\nATOM    693  OG  SER A  89      12.160  22.900  22.124  1.00 11.11           O  \\nATOM    694  N   PHE A  90      11.504  19.842  22.204  1.00 11.27           N  \\nATOM    695  CA  PHE A  90      11.680  18.751  21.262  1.00 12.91           C  \\nATOM    696  C   PHE A  90      10.760  17.601  21.667  1.00 14.37           C  \\nATOM    697  O   PHE A  90      10.034  17.057  20.839  1.00 13.24           O  \\nATOM    698  CB  PHE A  90      13.161  18.312  21.260  1.00 12.34           C  \\nATOM    699  CG  PHE A  90      13.411  17.120  20.351  1.00 14.41           C  \\nATOM    700  CD1 PHE A  90      13.890  15.948  20.903  1.00 15.10           C  \\nATOM    701  CD2 PHE A  90      13.136  17.206  18.990  1.00 14.81           C  \\nATOM    702  CE1 PHE A  90      14.093  14.857  20.088  1.00 16.20           C  \\nATOM    703  CE2 PHE A  90      13.340  16.110  18.179  1.00 14.81           C  \\nATOM    704  CZ  PHE A  90      13.818  14.935  18.731  1.00 15.39           C  \\nATOM    705  N   GLU A  91      10.725  17.231  22.949  1.00 14.40           N  \\nATOM    706  CA  GLU A  91       9.863  16.146  23.407  1.00 15.78           C  \\nATOM    707  C   GLU A  91       8.374  16.466  23.301  1.00 16.18           C  \\nATOM    708  O   GLU A  91       7.533  15.582  23.114  1.00 17.50           O  \\nATOM    709  CB  GLU A  91      10.226  15.846  24.835  1.00 19.84           C  \\nATOM    710  CG  GLU A  91      11.654  15.386  25.074  1.00 24.49           C  \\nATOM    711  CD  GLU A  91      12.090  15.310  26.537  1.00 27.97           C  \\nATOM    712  OE1 GLU A  91      13.199  14.842  26.766  1.00 30.72           O  \\nATOM    713  OE2 GLU A  91      11.371  15.730  27.449  1.00 30.38           O  \\nATOM    714  N   ASP A  92       8.019  17.747  23.433  1.00 16.19           N  \\nATOM    715  CA  ASP A  92       6.650  18.216  23.265  1.00 15.23           C  \\nATOM    716  C   ASP A  92       6.141  18.053  21.839  1.00 14.29           C  \\nATOM    717  O   ASP A  92       4.927  18.074  21.633  1.00 14.78           O  \\nATOM    718  CB  ASP A  92       6.536  19.708  23.646  1.00 17.85           C  \\nATOM    719  CG  ASP A  92       6.706  20.080  25.127  1.00 21.39           C  \\nATOM    720  OD1 ASP A  92       6.648  19.209  25.997  1.00 20.06           O  \\nATOM    721  OD2 ASP A  92       6.887  21.272  25.401  1.00 24.39           O  \\nATOM    722  N   ILE A  93       7.013  17.910  20.826  1.00 11.76           N  \\nATOM    723  CA  ILE A  93       6.559  17.749  19.452  1.00 14.77           C  \\nATOM    724  C   ILE A  93       5.574  16.583  19.354  1.00 14.03           C  \\nATOM    725  O   ILE A  93       4.576  16.695  18.644  1.00 14.93           O  \\nATOM    726  CB  ILE A  93       7.790  17.499  18.485  1.00 14.96           C  \\nATOM    727  CG1 ILE A  93       8.506  18.830  18.293  1.00 15.41           C  \\nATOM    728  CG2 ILE A  93       7.369  16.936  17.091  1.00 14.61           C  \\nATOM    729  CD1 ILE A  93       7.654  19.940  17.637  1.00 20.05           C  \\nATOM    730  N   HIS A  94       5.806  15.471  20.049  1.00 15.23           N  \\nATOM    731  CA  HIS A  94       4.940  14.302  19.956  1.00 18.20           C  \\nATOM    732  C   HIS A  94       3.494  14.635  20.287  1.00 16.36           C  \\nATOM    733  O   HIS A  94       2.576  14.183  19.598  1.00 17.20           O  \\nATOM    734  CB  HIS A  94       5.436  13.194  20.900  1.00 23.57           C  \\nATOM    735  CG  HIS A  94       4.685  11.874  20.678  1.00 29.09           C  \\nATOM    736  ND1 HIS A  94       4.923  11.118  19.602  1.00 31.26           N  \\nATOM    737  CD2 HIS A  94       3.705  11.361  21.498  1.00 30.93           C  \\nATOM    738  CE1 HIS A  94       4.078  10.110  19.746  1.00 31.74           C  \\nATOM    739  NE2 HIS A  94       3.366  10.260  20.861  1.00 32.20           N  \\nATOM    740  N   GLN A  95       3.271  15.499  21.281  1.00 16.91           N  \\nATOM    741  CA  GLN A  95       1.916  15.878  21.673  1.00 18.66           C  \\nATOM    742  C   GLN A  95       1.226  16.736  20.625  1.00 16.97           C  \\nATOM    743  O   GLN A  95       0.012  16.562  20.443  1.00 15.36           O  \\nATOM    744  CB  GLN A  95       1.928  16.626  23.018  1.00 21.56           C  \\nATOM    745  CG  GLN A  95       2.457  15.766  24.180  1.00 29.81           C  \\nATOM    746  CD  GLN A  95       3.370  16.540  25.150  1.00 35.29           C  \\nATOM    747  OE1 GLN A  95       3.037  17.647  25.593  1.00 37.94           O  \\nATOM    748  NE2 GLN A  95       4.546  16.006  25.521  1.00 37.04           N  \\nATOM    749  N   TYR A  96       1.930  17.653  19.921  1.00 16.06           N  \\nATOM    750  CA  TYR A  96       1.276  18.417  18.826  1.00 16.07           C  \\nATOM    751  C   TYR A  96       0.963  17.504  17.658  1.00 14.44           C  \\nATOM    752  O   TYR A  96      -0.138  17.565  17.118  1.00 15.79           O  \\nATOM    753  CB  TYR A  96       2.139  19.554  18.264  1.00 17.94           C  \\nATOM    754  CG  TYR A  96       2.287  20.677  19.277  1.00 19.19           C  \\nATOM    755  CD1 TYR A  96       1.368  21.699  19.338  1.00 19.33           C  \\nATOM    756  CD2 TYR A  96       3.348  20.631  20.161  1.00 19.61           C  \\nATOM    757  CE1 TYR A  96       1.526  22.678  20.308  1.00 21.72           C  \\nATOM    758  CE2 TYR A  96       3.512  21.595  21.129  1.00 21.72           C  \\nATOM    759  CZ  TYR A  96       2.594  22.613  21.190  1.00 21.26           C  \\nATOM    760  OH  TYR A  96       2.776  23.582  22.153  1.00 24.09           O  \\nATOM    761  N   ARG A  97       1.904  16.635  17.295  1.00 13.30           N  \\nATOM    762  CA  ARG A  97       1.699  15.650  16.247  1.00 15.46           C  \\nATOM    763  C   ARG A  97       0.457  14.794  16.492  1.00 14.26           C  \\nATOM    764  O   ARG A  97      -0.420  14.676  15.641  1.00 14.10           O  \\nATOM    765  CB  ARG A  97       2.939  14.796  16.196  1.00 17.97           C  \\nATOM    766  CG  ARG A  97       2.886  13.541  15.357  1.00 23.35           C  \\nATOM    767  CD  ARG A  97       3.333  13.765  13.943  1.00 23.59           C  \\nATOM    768  NE  ARG A  97       3.314  12.471  13.314  1.00 24.08           N  \\nATOM    769  CZ  ARG A  97       3.268  12.320  12.007  1.00 25.74           C  \\nATOM    770  NH1 ARG A  97       3.135  11.073  11.548  1.00 26.76           N  \\nATOM    771  NH2 ARG A  97       3.353  13.380  11.181  1.00 26.95           N  \\nATOM    772  N   GLU A  98       0.332  14.218  17.678  1.00 13.69           N  \\nATOM    773  CA  GLU A  98      -0.807  13.361  17.941  1.00 16.64           C  \\nATOM    774  C   GLU A  98      -2.118  14.108  17.983  1.00 15.57           C  \\nATOM    775  O   GLU A  98      -3.157  13.573  17.607  1.00 15.85           O  \\nATOM    776  CB  GLU A  98      -0.593  12.623  19.260  1.00 21.96           C  \\nATOM    777  CG  GLU A  98       0.417  11.455  19.177  1.00 28.02           C  \\nATOM    778  CD  GLU A  98       0.147  10.460  18.038  1.00 31.04           C  \\nATOM    779  OE1 GLU A  98       1.118  10.123  17.373  1.00 33.58           O  \\nATOM    780  OE2 GLU A  98      -0.992  10.025  17.790  1.00 33.61           O  \\nATOM    781  N   GLN A  99      -2.105  15.362  18.446  1.00 14.66           N  \\nATOM    782  CA  GLN A  99      -3.343  16.104  18.468  1.00 13.22           C  \\nATOM    783  C   GLN A  99      -3.779  16.398  17.049  1.00 12.15           C  \\nATOM    784  O   GLN A  99      -4.974  16.274  16.791  1.00 14.09           O  \\nATOM    785  CB  GLN A  99      -3.158  17.403  19.250  1.00 16.29           C  \\nATOM    786  CG  GLN A  99      -4.513  18.118  19.407  1.00 19.67           C  \\nATOM    787  CD  GLN A  99      -4.517  19.240  20.438  1.00 22.19           C  \\nATOM    788  OE1 GLN A  99      -3.537  19.485  21.138  1.00 23.70           O  \\nATOM    789  NE2 GLN A  99      -5.628  19.961  20.537  1.00 22.72           N  \\nATOM    790  N   ILE A 100      -2.861  16.795  16.142  1.00 11.42           N  \\nATOM    791  CA  ILE A 100      -3.197  17.121  14.743  1.00 10.81           C  \\nATOM    792  C   ILE A 100      -3.761  15.893  14.008  1.00 12.39           C  \\nATOM    793  O   ILE A 100      -4.733  15.967  13.243  1.00 12.70           O  \\nATOM    794  CB  ILE A 100      -1.924  17.659  14.038  1.00 10.32           C  \\nATOM    795  CG1 ILE A 100      -1.642  19.075  14.518  1.00 10.37           C  \\nATOM    796  CG2 ILE A 100      -2.131  17.685  12.512  1.00 11.36           C  \\nATOM    797  CD1 ILE A 100      -0.184  19.481  14.392  1.00 10.69           C  \\nATOM    798  N   LYS A 101      -3.143  14.732  14.255  1.00 14.08           N  \\nATOM    799  CA  LYS A 101      -3.613  13.487  13.637  1.00 15.95           C  \\nATOM    800  C   LYS A 101      -5.021  13.150  14.079  1.00 15.72           C  \\nATOM    801  O   LYS A 101      -5.848  12.770  13.262  1.00 14.25           O  \\nATOM    802  CB  LYS A 101      -2.681  12.317  13.996  1.00 17.44           C  \\nATOM    803  CG  LYS A 101      -1.414  12.284  13.115  1.00 19.24           C  \\nATOM    804  CD  LYS A 101      -0.332  11.418  13.764  1.00 23.28           C  \\nATOM    805  CE  LYS A 101      -0.752  10.000  14.188  1.00 23.82           C  \\nATOM    806  NZ  LYS A 101       0.400   9.230  14.662  1.00 24.00           N  \\nATOM    807  N   ARG A 102      -5.278  13.341  15.377  1.00 16.72           N  \\nATOM    808  CA  ARG A 102      -6.586  13.098  15.962  1.00 18.74           C  \\nATOM    809  C   ARG A 102      -7.643  14.087  15.440  1.00 19.07           C  \\nATOM    810  O   ARG A 102      -8.712  13.628  15.002  1.00 20.67           O  \\nATOM    811  CB  ARG A 102      -6.457  13.197  17.462  1.00 20.62           C  \\nATOM    812  CG  ARG A 102      -7.539  12.409  18.132  1.00 26.89           C  \\nATOM    813  CD  ARG A 102      -7.610  12.887  19.585  1.00 30.60           C  \\nATOM    814  NE  ARG A 102      -8.197  14.224  19.645  1.00 35.83           N  \\nATOM    815  CZ  ARG A 102      -7.633  15.255  20.301  1.00 37.61           C  \\nATOM    816  NH1 ARG A 102      -8.243  16.455  20.267  1.00 38.37           N  \\nATOM    817  NH2 ARG A 102      -6.481  15.110  20.983  1.00 38.39           N  \\nATOM    818  N   VAL A 103      -7.418  15.407  15.395  1.00 15.59           N  \\nATOM    819  CA  VAL A 103      -8.452  16.326  14.881  1.00 18.48           C  \\nATOM    820  C   VAL A 103      -8.665  16.200  13.349  1.00 16.72           C  \\nATOM    821  O   VAL A 103      -9.776  16.383  12.848  1.00 15.90           O  \\nATOM    822  CB  VAL A 103      -8.131  17.840  15.215  1.00 20.10           C  \\nATOM    823  CG1 VAL A 103      -7.896  18.005  16.704  1.00 21.34           C  \\nATOM    824  CG2 VAL A 103      -6.899  18.302  14.512  1.00 23.93           C  \\nATOM    825  N   LYS A 104      -7.622  15.902  12.574  1.00 13.84           N  \\nATOM    826  CA  LYS A 104      -7.804  15.755  11.140  1.00 16.11           C  \\nATOM    827  C   LYS A 104      -8.158  14.329  10.716  1.00 16.70           C  \\nATOM    828  O   LYS A 104      -8.260  14.079   9.508  1.00 17.27           O  \\nATOM    829  CB  LYS A 104      -6.525  16.215  10.404  1.00 18.12           C  \\nATOM    830  CG  LYS A 104      -6.197  17.678  10.744  1.00 19.53           C  \\nATOM    831  CD  LYS A 104      -5.250  18.297   9.736  1.00 21.28           C  \\nATOM    832  CE  LYS A 104      -5.921  18.590   8.390  1.00 20.75           C  \\nATOM    833  NZ  LYS A 104      -6.875  19.673   8.462  1.00 19.79           N  \\nATOM    834  N   ASP A 105      -8.372  13.390  11.669  1.00 15.91           N  \\nATOM    835  CA  ASP A 105      -8.666  11.988  11.359  1.00 15.43           C  \\nATOM    836  C   ASP A 105      -7.705  11.372  10.333  1.00 16.04           C  \\nATOM    837  O   ASP A 105      -8.106  10.654   9.415  1.00 15.38           O  \\nATOM    838  CB  ASP A 105     -10.077  11.847  10.810  1.00 18.74           C  \\nATOM    839  CG  ASP A 105     -11.261  12.121  11.696  1.00 21.15           C  \\nATOM    840  OD1 ASP A 105     -11.183  11.972  12.918  1.00 21.36           O  \\nATOM    841  OD2 ASP A 105     -12.281  12.480  11.113  1.00 22.63           O  \\nATOM    842  N   SER A 106      -6.397  11.591  10.469  1.00 15.54           N  \\nATOM    843  CA  SER A 106      -5.470  11.174   9.442  1.00 17.25           C  \\nATOM    844  C   SER A 106      -4.112  10.882  10.011  1.00 17.31           C  \\nATOM    845  O   SER A 106      -3.650  11.556  10.916  1.00 19.11           O  \\nATOM    846  CB  SER A 106      -5.463  12.314   8.398  1.00 19.27           C  \\nATOM    847  OG  SER A 106      -4.326  12.490   7.573  1.00 24.20           O  \\nATOM    848  N   ASP A 107      -3.454   9.891   9.453  1.00 17.29           N  \\nATOM    849  CA  ASP A 107      -2.109   9.557   9.868  1.00 19.80           C  \\nATOM    850  C   ASP A 107      -1.000  10.220   9.063  1.00 20.36           C  \\nATOM    851  O   ASP A 107       0.197  10.118   9.345  1.00 21.26           O  \\nATOM    852  CB  ASP A 107      -1.970   8.045   9.810  1.00 23.15           C  \\nATOM    853  CG  ASP A 107      -2.863   7.310  10.807  1.00 25.12           C  \\nATOM    854  OD1 ASP A 107      -3.262   6.206  10.462  1.00 29.21           O  \\nATOM    855  OD2 ASP A 107      -3.127   7.812  11.903  1.00 26.59           O  \\nATOM    856  N   ASP A 108      -1.383  10.909   8.005  1.00 20.57           N  \\nATOM    857  CA  ASP A 108      -0.430  11.664   7.216  1.00 22.38           C  \\nATOM    858  C   ASP A 108      -1.061  12.982   6.842  1.00 21.37           C  \\nATOM    859  O   ASP A 108      -1.973  13.096   6.030  1.00 24.36           O  \\nATOM    860  CB  ASP A 108      -0.056  10.900   5.967  1.00 27.28           C  \\nATOM    861  CG  ASP A 108      -1.263  10.278   5.320  1.00 32.24           C  \\nATOM    862  OD1 ASP A 108      -1.714   9.272   5.877  1.00 37.07           O  \\nATOM    863  OD2 ASP A 108      -1.747  10.790   4.301  1.00 36.33           O  \\nATOM    864  N   VAL A 109      -0.598  13.970   7.572  1.00 19.33           N  \\nATOM    865  CA  VAL A 109      -1.009  15.354   7.400  1.00 17.49           C  \\nATOM    866  C   VAL A 109       0.215  16.086   6.819  1.00 14.55           C  \\nATOM    867  O   VAL A 109       1.335  15.772   7.245  1.00 13.64           O  \\nATOM    868  CB  VAL A 109      -1.410  15.870   8.784  1.00 18.40           C  \\nATOM    869  CG1 VAL A 109      -1.645  17.366   8.765  1.00 20.21           C  \\nATOM    870  CG2 VAL A 109      -2.676  15.101   9.214  1.00 20.27           C  \\nATOM    871  N   PRO A 110       0.141  17.001   5.825  1.00 12.41           N  \\nATOM    872  CA  PRO A 110       1.297  17.783   5.384  1.00 11.52           C  \\nATOM    873  C   PRO A 110       1.920  18.480   6.605  1.00 11.79           C  \\nATOM    874  O   PRO A 110       1.289  19.245   7.324  1.00  9.55           O  \\nATOM    875  CB  PRO A 110       0.728  18.721   4.370  1.00 11.31           C  \\nATOM    876  CG  PRO A 110      -0.410  17.926   3.779  1.00 11.38           C  \\nATOM    877  CD  PRO A 110      -1.044  17.309   5.020  1.00 12.05           C  \\nATOM    878  N   MET A 111       3.192  18.201   6.874  1.00 11.92           N  \\nATOM    879  CA  MET A 111       3.878  18.724   8.050  1.00 11.49           C  \\nATOM    880  C   MET A 111       5.379  18.749   7.807  1.00 11.68           C  \\nATOM    881  O   MET A 111       5.903  17.900   7.098  1.00 11.11           O  \\nATOM    882  CB  MET A 111       3.494  17.806   9.211  1.00 14.36           C  \\nATOM    883  CG  MET A 111       3.864  18.110  10.627  1.00 17.41           C  \\nATOM    884  SD  MET A 111       3.231  16.825  11.758  1.00 18.36           S  \\nATOM    885  CE  MET A 111       1.528  17.128  11.498  1.00 17.25           C  \\nATOM    886  N   VAL A 112       6.091  19.706   8.404  1.00 10.73           N  \\nATOM    887  CA  VAL A 112       7.545  19.763   8.348  1.00 10.18           C  \\nATOM    888  C   VAL A 112       7.992  19.875   9.794  1.00  9.11           C  \\nATOM    889  O   VAL A 112       7.404  20.633  10.556  1.00 10.77           O  \\nATOM    890  CB  VAL A 112       8.010  21.024   7.542  1.00 11.80           C  \\nATOM    891  CG1 VAL A 112       9.528  21.213   7.558  1.00 13.24           C  \\nATOM    892  CG2 VAL A 112       7.542  20.843   6.104  1.00 14.56           C  \\nATOM    893  N   LEU A 113       9.032  19.167  10.207  1.00  8.47           N  \\nATOM    894  CA  LEU A 113       9.632  19.332  11.535  1.00  9.53           C  \\nATOM    895  C   LEU A 113      10.738  20.384  11.412  1.00  9.33           C  \\nATOM    896  O   LEU A 113      11.621  20.270  10.555  1.00  9.95           O  \\nATOM    897  CB  LEU A 113      10.214  17.983  12.017  1.00  9.56           C  \\nATOM    898  CG  LEU A 113      10.863  17.976  13.381  1.00 11.58           C  \\nATOM    899  CD1 LEU A 113       9.908  18.379  14.503  1.00 13.17           C  \\nATOM    900  CD2 LEU A 113      11.361  16.564  13.597  1.00 12.84           C  \\nATOM    901  N   VAL A 114      10.705  21.442  12.221  1.00  7.62           N  \\nATOM    902  CA  VAL A 114      11.673  22.541  12.109  1.00  7.64           C  \\nATOM    903  C   VAL A 114      12.436  22.667  13.441  1.00  7.65           C  \\nATOM    904  O   VAL A 114      11.872  22.842  14.541  1.00  9.71           O  \\nATOM    905  CB  VAL A 114      10.896  23.836  11.774  1.00  9.37           C  \\nATOM    906  CG1 VAL A 114      11.814  25.030  11.869  1.00 10.27           C  \\nATOM    907  CG2 VAL A 114      10.254  23.719  10.393  1.00  9.64           C  \\nATOM    908  N   GLY A 115      13.745  22.567  13.331  1.00  6.10           N  \\nATOM    909  CA  GLY A 115      14.603  22.742  14.476  1.00  6.69           C  \\nATOM    910  C   GLY A 115      15.120  24.163  14.407  1.00  8.47           C  \\nATOM    911  O   GLY A 115      16.007  24.439  13.594  1.00  9.16           O  \\nATOM    912  N   ASN A 116      14.593  25.089  15.192  1.00  8.01           N  \\nATOM    913  CA  ASN A 116      15.007  26.475  15.111  1.00  8.24           C  \\nATOM    914  C   ASN A 116      16.112  26.873  16.109  1.00  9.61           C  \\nATOM    915  O   ASN A 116      16.453  26.142  17.051  1.00  9.84           O  \\nATOM    916  CB  ASN A 116      13.721  27.307  15.275  1.00  8.01           C  \\nATOM    917  CG  ASN A 116      13.929  28.788  14.992  1.00  9.48           C  \\nATOM    918  OD1 ASN A 116      14.470  29.205  13.972  1.00  9.94           O  \\nATOM    919  ND2 ASN A 116      13.565  29.644  15.940  1.00 10.75           N  \\nATOM    920  N   LYS A 117      16.660  28.087  15.876  1.00 10.22           N  \\nATOM    921  CA  LYS A 117      17.735  28.715  16.674  1.00 12.36           C  \\nATOM    922  C   LYS A 117      19.094  28.021  16.450  1.00 13.39           C  \\nATOM    923  O   LYS A 117      19.977  27.940  17.324  1.00 11.29           O  \\nATOM    924  CB  LYS A 117      17.391  28.706  18.214  1.00 13.23           C  \\nATOM    925  CG  LYS A 117      16.026  29.272  18.611  1.00 12.90           C  \\nATOM    926  CD  LYS A 117      15.932  29.640  20.097  1.00 12.43           C  \\nATOM    927  CE  LYS A 117      14.551  30.217  20.380  1.00 10.55           C  \\nATOM    928  NZ  LYS A 117      14.491  30.647  21.772  1.00 11.10           N  \\nATOM    929  N   CYS A 118      19.254  27.555  15.188  1.00 13.69           N  \\nATOM    930  CA  CYS A 118      20.449  26.806  14.844  1.00 15.67           C  \\nATOM    931  C   CYS A 118      21.716  27.658  14.796  1.00 15.28           C  \\nATOM    932  O   CYS A 118      22.808  27.113  14.629  1.00 15.30           O  \\nATOM    933  CB  CYS A 118      20.246  26.064  13.492  1.00 17.74           C  \\nATOM    934  SG  CYS A 118      20.353  27.009  11.944  1.00 20.74           S  \\nATOM    935  N   ASP A 119      21.578  28.995  14.931  1.00 15.76           N  \\nATOM    936  CA  ASP A 119      22.722  29.904  15.083  1.00 16.09           C  \\nATOM    937  C   ASP A 119      23.426  29.778  16.445  1.00 17.19           C  \\nATOM    938  O   ASP A 119      24.589  30.178  16.553  1.00 18.76           O  \\nATOM    939  CB  ASP A 119      22.268  31.374  14.888  1.00 15.14           C  \\nATOM    940  CG  ASP A 119      21.153  31.785  15.835  1.00 15.81           C  \\nATOM    941  OD1 ASP A 119      20.030  31.294  15.662  1.00 13.87           O  \\nATOM    942  OD2 ASP A 119      21.407  32.596  16.721  1.00 14.85           O  \\nATOM    943  N   LEU A 120      22.732  29.248  17.480  1.00 16.86           N  \\nATOM    944  CA  LEU A 120      23.230  29.099  18.845  1.00 18.76           C  \\nATOM    945  C   LEU A 120      24.217  27.957  18.979  1.00 19.70           C  \\nATOM    946  O   LEU A 120      23.979  26.885  18.431  1.00 20.53           O  \\nATOM    947  CB  LEU A 120      22.103  28.829  19.831  1.00 17.65           C  \\nATOM    948  CG  LEU A 120      21.065  29.933  20.010  1.00 19.43           C  \\nATOM    949  CD1 LEU A 120      20.044  29.418  21.008  1.00 18.46           C  \\nATOM    950  CD2 LEU A 120      21.688  31.239  20.503  1.00 18.73           C  \\nATOM    951  N   ALA A 121      25.313  28.184  19.705  1.00 19.26           N  \\nATOM    952  CA  ALA A 121      26.303  27.143  19.887  1.00 20.48           C  \\nATOM    953  C   ALA A 121      25.942  26.210  21.036  1.00 21.30           C  \\nATOM    954  O   ALA A 121      26.403  25.069  21.067  1.00 24.04           O  \\nATOM    955  CB  ALA A 121      27.652  27.767  20.164  1.00 22.02           C  \\nATOM    956  N   ALA A 122      25.109  26.635  21.982  1.00 22.09           N  \\nATOM    957  CA  ALA A 122      24.683  25.797  23.094  1.00 23.88           C  \\nATOM    958  C   ALA A 122      23.578  24.811  22.720  1.00 23.77           C  \\nATOM    959  O   ALA A 122      22.492  24.835  23.301  1.00 27.82           O  \\nATOM    960  CB  ALA A 122      24.171  26.684  24.249  1.00 25.67           C  \\nATOM    961  N   ARG A 123      23.843  23.908  21.783  1.00 20.99           N  \\nATOM    962  CA  ARG A 123      22.890  22.905  21.311  1.00 19.81           C  \\nATOM    963  C   ARG A 123      22.619  21.761  22.283  1.00 19.56           C  \\nATOM    964  O   ARG A 123      23.590  21.176  22.774  1.00 20.01           O  \\nATOM    965  CB  ARG A 123      23.421  22.333  19.988  1.00 19.81           C  \\nATOM    966  CG  ARG A 123      22.662  21.166  19.398  1.00 21.43           C  \\nATOM    967  CD  ARG A 123      23.273  20.680  18.073  1.00 22.66           C  \\nATOM    968  NE  ARG A 123      23.050  21.590  16.956  1.00 22.61           N  \\nATOM    969  CZ  ARG A 123      22.019  21.487  16.093  1.00 20.95           C  \\nATOM    970  NH1 ARG A 123      21.045  20.568  16.181  1.00 16.36           N  \\nATOM    971  NH2 ARG A 123      21.989  22.370  15.098  1.00 19.65           N  \\nATOM    972  N   THR A 124      21.364  21.389  22.600  1.00 17.96           N  \\nATOM    973  CA  THR A 124      21.118  20.200  23.415  1.00 17.85           C  \\nATOM    974  C   THR A 124      20.413  19.068  22.656  1.00 17.08           C  \\nATOM    975  O   THR A 124      20.459  17.926  23.091  1.00 19.59           O  \\nATOM    976  CB  THR A 124      20.311  20.578  24.672  1.00 19.42           C  \\nATOM    977  OG1 THR A 124      19.109  21.230  24.277  1.00 20.50           O  \\nATOM    978  CG2 THR A 124      21.155  21.458  25.588  1.00 19.06           C  \\nATOM    979  N   VAL A 125      19.788  19.319  21.495  1.00 16.59           N  \\nATOM    980  CA  VAL A 125      19.133  18.324  20.647  1.00 16.09           C  \\nATOM    981  C   VAL A 125      20.032  18.080  19.413  1.00 16.61           C  \\nATOM    982  O   VAL A 125      20.180  18.980  18.573  1.00 14.07           O  \\nATOM    983  CB  VAL A 125      17.720  18.848  20.179  1.00 15.28           C  \\nATOM    984  CG1 VAL A 125      17.097  17.814  19.265  1.00 14.81           C  \\nATOM    985  CG2 VAL A 125      16.824  19.168  21.366  1.00 15.97           C  \\nATOM    986  N   GLU A 126      20.659  16.889  19.280  1.00 17.05           N  \\nATOM    987  CA  GLU A 126      21.487  16.568  18.124  1.00 19.28           C  \\nATOM    988  C   GLU A 126      20.631  16.496  16.876  1.00 18.51           C  \\nATOM    989  O   GLU A 126      19.511  15.976  16.880  1.00 17.68           O  \\nATOM    990  CB  GLU A 126      22.165  15.215  18.219  1.00 23.29           C  \\nATOM    991  CG  GLU A 126      23.244  15.057  19.265  1.00 30.86           C  \\nATOM    992  CD  GLU A 126      24.402  16.054  19.153  1.00 36.59           C  \\nATOM    993  OE1 GLU A 126      25.139  16.012  18.161  1.00 39.64           O  \\nATOM    994  OE2 GLU A 126      24.581  16.885  20.056  1.00 40.44           O  \\nATOM    995  N   SER A 127      21.167  16.985  15.755  1.00 17.73           N  \\nATOM    996  CA  SER A 127      20.426  16.938  14.493  1.00 16.72           C  \\nATOM    997  C   SER A 127      20.040  15.506  14.158  1.00 16.76           C  \\nATOM    998  O   SER A 127      18.931  15.245  13.690  1.00 17.01           O  \\nATOM    999  CB  SER A 127      21.283  17.518  13.371  1.00 17.67           C  \\nATOM   1000  OG  SER A 127      21.730  18.826  13.768  1.00 19.67           O  \\nATOM   1001  N   ARG A 128      20.916  14.558  14.466  1.00 16.72           N  \\nATOM   1002  CA  ARG A 128      20.667  13.154  14.145  1.00 20.94           C  \\nATOM   1003  C   ARG A 128      19.435  12.617  14.862  1.00 19.62           C  \\nATOM   1004  O   ARG A 128      18.641  11.882  14.278  1.00 18.81           O  \\nATOM   1005  CB  ARG A 128      21.910  12.319  14.510  1.00 24.31           C  \\nATOM   1006  CG  ARG A 128      21.969  11.035  13.692  1.00 30.88           C  \\nATOM   1007  CD  ARG A 128      23.381  10.403  13.536  1.00 35.88           C  \\nATOM   1008  NE  ARG A 128      24.384  10.996  12.621  1.00 39.57           N  \\nATOM   1009  CZ  ARG A 128      24.313  11.018  11.267  1.00 40.85           C  \\nATOM   1010  NH1 ARG A 128      25.399  11.388  10.577  1.00 40.83           N  \\nATOM   1011  NH2 ARG A 128      23.202  10.685  10.576  1.00 41.33           N  \\nATOM   1012  N   GLN A 129      19.209  13.016  16.119  1.00 18.27           N  \\nATOM   1013  CA  GLN A 129      18.032  12.519  16.787  1.00 18.53           C  \\nATOM   1014  C   GLN A 129      16.735  13.151  16.224  1.00 17.11           C  \\nATOM   1015  O   GLN A 129      15.691  12.486  16.098  1.00 15.48           O  \\nATOM   1016  CB  GLN A 129      18.229  12.766  18.306  1.00 22.36           C  \\nATOM   1017  CG  GLN A 129      18.041  14.133  18.911  1.00 24.78           C  \\nATOM   1018  CD  GLN A 129      18.283  14.123  20.411  1.00 27.48           C  \\nATOM   1019  OE1 GLN A 129      19.287  14.643  20.906  1.00 28.14           O  \\nATOM   1020  NE2 GLN A 129      17.367  13.497  21.152  1.00 27.21           N  \\nATOM   1021  N   ALA A 130      16.780  14.446  15.837  1.00 14.73           N  \\nATOM   1022  CA  ALA A 130      15.611  15.044  15.210  1.00 13.87           C  \\nATOM   1023  C   ALA A 130      15.342  14.449  13.826  1.00 13.30           C  \\nATOM   1024  O   ALA A 130      14.199  14.194  13.436  1.00 11.42           O  \\nATOM   1025  CB  ALA A 130      15.822  16.536  15.063  1.00 16.21           C  \\nATOM   1026  N   GLN A 131      16.401  14.189  13.057  1.00 14.61           N  \\nATOM   1027  CA  GLN A 131      16.267  13.529  11.739  1.00 16.01           C  \\nATOM   1028  C   GLN A 131      15.637  12.109  11.821  1.00 16.29           C  \\nATOM   1029  O   GLN A 131      14.777  11.708  11.017  1.00 14.37           O  \\nATOM   1030  CB  GLN A 131      17.673  13.507  11.116  1.00 18.12           C  \\nATOM   1031  CG  GLN A 131      17.702  13.017   9.689  1.00 22.19           C  \\nATOM   1032  CD  GLN A 131      16.889  13.922   8.801  1.00 21.61           C  \\nATOM   1033  OE1 GLN A 131      17.292  15.044   8.529  1.00 24.06           O  \\nATOM   1034  NE2 GLN A 131      15.714  13.485   8.370  1.00 23.37           N  \\nATOM   1035  N   ASP A 132      16.038  11.348  12.853  1.00 16.64           N  \\nATOM   1036  CA  ASP A 132      15.467  10.038  13.173  1.00 18.00           C  \\nATOM   1037  C   ASP A 132      13.984  10.175  13.484  1.00 17.28           C  \\nATOM   1038  O   ASP A 132      13.178   9.395  12.985  1.00 16.45           O  \\nATOM   1039  CB  ASP A 132      16.120   9.415  14.405  1.00 20.10           C  \\nATOM   1040  CG  ASP A 132      17.506   8.816  14.234  1.00 24.07           C  \\nATOM   1041  OD1 ASP A 132      18.120   8.459  15.243  1.00 28.35           O  \\nATOM   1042  OD2 ASP A 132      17.975   8.686  13.106  1.00 25.43           O  \\nATOM   1043  N   LEU A 133      13.614  11.168  14.312  1.00 15.58           N  \\nATOM   1044  CA  LEU A 133      12.213  11.390  14.624  1.00 16.28           C  \\nATOM   1045  C   LEU A 133      11.421  11.729  13.366  1.00 14.96           C  \\nATOM   1046  O   LEU A 133      10.345  11.181  13.128  1.00 15.09           O  \\nATOM   1047  CB  LEU A 133      12.052  12.536  15.647  1.00 17.06           C  \\nATOM   1048  CG  LEU A 133      10.617  12.781  16.136  1.00 18.79           C  \\nATOM   1049  CD1 LEU A 133      10.142  11.554  16.881  1.00 19.96           C  \\nATOM   1050  CD2 LEU A 133      10.544  14.013  17.020  1.00 18.45           C  \\nATOM   1051  N   ALA A 134      11.937  12.642  12.545  1.00 13.16           N  \\nATOM   1052  CA  ALA A 134      11.241  13.016  11.331  1.00 14.89           C  \\nATOM   1053  C   ALA A 134      11.092  11.821  10.414  1.00 16.19           C  \\nATOM   1054  O   ALA A 134      10.015  11.643   9.857  1.00 15.70           O  \\nATOM   1055  CB  ALA A 134      12.021  14.089  10.629  1.00 15.12           C  \\nATOM   1056  N   ARG A 135      12.143  10.995  10.279  1.00 17.00           N  \\nATOM   1057  CA  ARG A 135      12.088   9.828   9.438  1.00 19.17           C  \\nATOM   1058  C   ARG A 135      10.990   8.893   9.916  1.00 18.70           C  \\nATOM   1059  O   ARG A 135      10.269   8.352   9.078  1.00 20.29           O  \\nATOM   1060  CB  ARG A 135      13.425   9.091   9.456  1.00 22.48           C  \\nATOM   1061  CG  ARG A 135      13.499   8.297   8.155  1.00 29.02           C  \\nATOM   1062  CD  ARG A 135      14.619   7.270   7.988  1.00 33.22           C  \\nATOM   1063  NE  ARG A 135      14.117   5.888   8.050  1.00 38.80           N  \\nATOM   1064  CZ  ARG A 135      13.281   5.302   7.148  1.00 39.60           C  \\nATOM   1065  NH1 ARG A 135      12.889   4.044   7.384  1.00 40.80           N  \\nATOM   1066  NH2 ARG A 135      12.827   5.913   6.034  1.00 40.11           N  \\nATOM   1067  N   SER A 136      10.815   8.697  11.242  1.00 19.23           N  \\nATOM   1068  CA  SER A 136       9.757   7.841  11.770  1.00 19.12           C  \\nATOM   1069  C   SER A 136       8.374   8.432  11.528  1.00 19.61           C  \\nATOM   1070  O   SER A 136       7.398   7.690  11.522  1.00 21.01           O  \\nATOM   1071  CB  SER A 136       9.930   7.624  13.280  1.00 19.94           C  \\nATOM   1072  OG  SER A 136       9.459   8.686  14.119  1.00 23.21           O  \\nATOM   1073  N   TYR A 137       8.231   9.760  11.380  1.00 17.33           N  \\nATOM   1074  CA  TYR A 137       6.950  10.375  11.078  1.00 18.12           C  \\nATOM   1075  C   TYR A 137       6.683  10.404   9.570  1.00 18.22           C  \\nATOM   1076  O   TYR A 137       5.561  10.656   9.129  1.00 19.27           O  \\nATOM   1077  CB  TYR A 137       6.907  11.812  11.592  1.00 18.21           C  \\nATOM   1078  CG  TYR A 137       6.883  12.057  13.084  1.00 19.08           C  \\nATOM   1079  CD1 TYR A 137       6.619  11.026  13.955  1.00 20.51           C  \\nATOM   1080  CD2 TYR A 137       7.059  13.355  13.548  1.00 20.43           C  \\nATOM   1081  CE1 TYR A 137       6.510  11.285  15.306  1.00 21.89           C  \\nATOM   1082  CE2 TYR A 137       6.945  13.624  14.899  1.00 21.02           C  \\nATOM   1083  CZ  TYR A 137       6.662  12.574  15.763  1.00 22.13           C  \\nATOM   1084  OH  TYR A 137       6.476  12.782  17.112  1.00 22.16           O  \\nATOM   1085  N   GLY A 138       7.689  10.186   8.737  1.00 16.07           N  \\nATOM   1086  CA  GLY A 138       7.520  10.265   7.294  1.00 17.66           C  \\nATOM   1087  C   GLY A 138       7.476  11.725   6.810  1.00 18.01           C  \\nATOM   1088  O   GLY A 138       6.859  12.010   5.774  1.00 17.85           O  \\nATOM   1089  N   ILE A 139       8.128  12.662   7.540  1.00 14.16           N  \\nATOM   1090  CA  ILE A 139       8.117  14.065   7.145  1.00 13.49           C  \\nATOM   1091  C   ILE A 139       9.526  14.662   7.033  1.00 14.35           C  \\nATOM   1092  O   ILE A 139      10.494  14.152   7.610  1.00 16.19           O  \\nATOM   1093  CB  ILE A 139       7.292  14.896   8.161  1.00 14.40           C  \\nATOM   1094  CG1 ILE A 139       7.925  14.811   9.554  1.00 15.44           C  \\nATOM   1095  CG2 ILE A 139       5.821  14.422   8.104  1.00 14.07           C  \\nATOM   1096  CD1 ILE A 139       7.288  15.765  10.587  1.00 15.35           C  \\nATOM   1097  N   PRO A 140       9.706  15.767   6.308  1.00 13.55           N  \\nATOM   1098  CA  PRO A 140      10.984  16.448   6.205  1.00 13.39           C  \\nATOM   1099  C   PRO A 140      11.424  17.065   7.532  1.00 12.74           C  \\nATOM   1100  O   PRO A 140      10.601  17.530   8.310  1.00 11.42           O  \\nATOM   1101  CB  PRO A 140      10.767  17.498   5.127  1.00 13.47           C  \\nATOM   1102  CG  PRO A 140       9.514  17.084   4.410  1.00 14.09           C  \\nATOM   1103  CD  PRO A 140       8.686  16.467   5.519  1.00 13.14           C  \\nATOM   1104  N   TYR A 141      12.744  17.086   7.756  1.00 11.68           N  \\nATOM   1105  CA  TYR A 141      13.336  17.791   8.890  1.00 10.89           C  \\nATOM   1106  C   TYR A 141      14.175  18.945   8.326  1.00 10.09           C  \\nATOM   1107  O   TYR A 141      15.056  18.720   7.468  1.00 10.27           O  \\nATOM   1108  CB  TYR A 141      14.276  16.897   9.691  1.00 12.54           C  \\nATOM   1109  CG  TYR A 141      15.063  17.676  10.733  1.00 10.92           C  \\nATOM   1110  CD1 TYR A 141      14.350  18.396  11.664  1.00 11.75           C  \\nATOM   1111  CD2 TYR A 141      16.453  17.633  10.784  1.00 12.49           C  \\nATOM   1112  CE1 TYR A 141      14.997  19.074  12.670  1.00 12.85           C  \\nATOM   1113  CE2 TYR A 141      17.112  18.307  11.794  1.00 11.33           C  \\nATOM   1114  CZ  TYR A 141      16.369  19.016  12.728  1.00 12.61           C  \\nATOM   1115  OH  TYR A 141      16.944  19.646  13.813  1.00 13.82           O  \\nATOM   1116  N   ILE A 142      13.892  20.174   8.799  1.00  9.35           N  \\nATOM   1117  CA  ILE A 142      14.636  21.355   8.376  1.00  9.05           C  \\nATOM   1118  C   ILE A 142      15.153  22.163   9.551  1.00  9.25           C  \\nATOM   1119  O   ILE A 142      14.409  22.382  10.499  1.00 10.43           O  \\nATOM   1120  CB  ILE A 142      13.756  22.259   7.472  1.00  9.25           C  \\nATOM   1121  CG1 ILE A 142      13.523  21.470   6.180  1.00 10.51           C  \\nATOM   1122  CG2 ILE A 142      14.450  23.586   7.122  1.00  9.08           C  \\nATOM   1123  CD1 ILE A 142      12.719  22.127   5.028  1.00 13.16           C  \\nATOM   1124  N   GLU A 143      16.457  22.513   9.568  1.00  9.01           N  \\nATOM   1125  CA  GLU A 143      16.979  23.404  10.611  1.00  8.52           C  \\nATOM   1126  C   GLU A 143      16.978  24.864  10.193  1.00 10.31           C  \\nATOM   1127  O   GLU A 143      17.278  25.228   9.040  1.00 10.45           O  \\nATOM   1128  CB  GLU A 143      18.397  23.022  11.013  1.00 10.87           C  \\nATOM   1129  CG  GLU A 143      18.363  21.728  11.801  1.00 10.22           C  \\nATOM   1130  CD  GLU A 143      19.592  21.411  12.633  1.00 13.01           C  \\nATOM   1131  OE1 GLU A 143      20.616  22.082  12.521  1.00 13.65           O  \\nATOM   1132  OE2 GLU A 143      19.511  20.487  13.429  1.00 12.70           O  \\nATOM   1133  N   THR A 144      16.608  25.762  11.117  1.00  9.18           N  \\nATOM   1134  CA  THR A 144      16.422  27.169  10.754  1.00  9.71           C  \\nATOM   1135  C   THR A 144      17.014  28.076  11.789  1.00  8.66           C  \\nATOM   1136  O   THR A 144      17.259  27.698  12.927  1.00  9.13           O  \\nATOM   1137  CB  THR A 144      14.919  27.617  10.635  1.00  9.90           C  \\nATOM   1138  OG1 THR A 144      14.336  27.403  11.931  1.00  9.29           O  \\nATOM   1139  CG2 THR A 144      14.120  26.858   9.557  1.00  9.89           C  \\nATOM   1140  N   SER A 145      17.244  29.288  11.360  1.00 10.91           N  \\nATOM   1141  CA  SER A 145      17.558  30.377  12.290  1.00 12.65           C  \\nATOM   1142  C   SER A 145      16.758  31.602  11.849  1.00 10.37           C  \\nATOM   1143  O   SER A 145      16.930  32.094  10.727  1.00 11.76           O  \\nATOM   1144  CB  SER A 145      19.069  30.768  12.308  1.00 11.79           C  \\nATOM   1145  OG  SER A 145      19.248  31.931  13.125  1.00 12.12           O  \\nATOM   1146  N   ALA A 146      15.838  32.046  12.696  1.00  9.49           N  \\nATOM   1147  CA  ALA A 146      15.131  33.268  12.413  1.00 10.90           C  \\nATOM   1148  C   ALA A 146      16.070  34.455  12.553  1.00 13.53           C  \\nATOM   1149  O   ALA A 146      15.799  35.514  11.995  1.00 16.20           O  \\nATOM   1150  CB  ALA A 146      13.976  33.436  13.367  1.00 12.65           C  \\nATOM   1151  N   LYS A 147      17.183  34.302  13.283  1.00 14.35           N  \\nATOM   1152  CA  LYS A 147      18.150  35.351  13.486  1.00 16.25           C  \\nATOM   1153  C   LYS A 147      18.953  35.568  12.215  1.00 17.58           C  \\nATOM   1154  O   LYS A 147      19.124  36.724  11.825  1.00 20.13           O  \\nATOM   1155  CB  LYS A 147      19.068  34.970  14.617  1.00 17.55           C  \\nATOM   1156  CG  LYS A 147      20.003  36.125  14.958  1.00 22.83           C  \\nATOM   1157  CD  LYS A 147      20.581  35.932  16.341  1.00 26.44           C  \\nATOM   1158  CE  LYS A 147      21.683  36.949  16.507  1.00 30.79           C  \\nATOM   1159  NZ  LYS A 147      22.575  36.534  17.574  1.00 33.81           N  \\nATOM   1160  N   THR A 148      19.479  34.529  11.541  1.00 17.16           N  \\nATOM   1161  CA  THR A 148      20.248  34.735  10.300  1.00 15.93           C  \\nATOM   1162  C   THR A 148      19.422  34.607   9.013  1.00 17.72           C  \\nATOM   1163  O   THR A 148      19.926  34.962   7.945  1.00 18.47           O  \\nATOM   1164  CB  THR A 148      21.444  33.745  10.209  1.00 15.86           C  \\nATOM   1165  OG1 THR A 148      20.886  32.435  10.042  1.00 14.56           O  \\nATOM   1166  CG2 THR A 148      22.377  33.839  11.439  1.00 15.25           C  \\nATOM   1167  N   ARG A 149      18.176  34.108   9.100  1.00 15.18           N  \\nATOM   1168  CA  ARG A 149      17.268  33.815   7.994  1.00 19.20           C  \\nATOM   1169  C   ARG A 149      17.652  32.483   7.318  1.00 17.94           C  \\nATOM   1170  O   ARG A 149      17.031  32.079   6.332  1.00 20.28           O  \\nATOM   1171  CB  ARG A 149      17.251  34.964   6.937  1.00 21.40           C  \\nATOM   1172  CG  ARG A 149      15.929  35.106   6.188  1.00 27.00           C  \\nATOM   1173  CD  ARG A 149      15.946  35.904   4.863  1.00 27.99           C  \\nATOM   1174  NE  ARG A 149      14.707  35.687   4.092  1.00 29.40           N  \\nATOM   1175  CZ  ARG A 149      13.550  36.232   4.492  1.00 29.07           C  \\nATOM   1176  NH1 ARG A 149      12.412  36.015   3.828  1.00 29.56           N  \\nATOM   1177  NH2 ARG A 149      13.527  37.004   5.577  1.00 28.99           N  \\nATOM   1178  N   GLN A 150      18.621  31.701   7.838  1.00 15.80           N  \\nATOM   1179  CA  GLN A 150      18.926  30.402   7.273  1.00 16.36           C  \\nATOM   1180  C   GLN A 150      17.737  29.464   7.455  1.00 15.02           C  \\nATOM   1181  O   GLN A 150      17.119  29.351   8.504  1.00 13.57           O  \\nATOM   1182  CB  GLN A 150      20.152  29.729   7.937  1.00 23.05           C  \\nATOM   1183  CG  GLN A 150      20.578  28.448   7.162  1.00 29.37           C  \\nATOM   1184  CD  GLN A 150      21.604  27.530   7.827  1.00 34.84           C  \\nATOM   1185  OE1 GLN A 150      21.301  26.416   8.280  1.00 38.95           O  \\nATOM   1186  NE2 GLN A 150      22.857  27.967   7.883  1.00 35.22           N  \\nATOM   1187  N   GLY A 151      17.412  28.809   6.348  1.00 13.62           N  \\nATOM   1188  CA  GLY A 151      16.357  27.835   6.296  1.00 13.50           C  \\nATOM   1189  C   GLY A 151      14.943  28.395   6.329  1.00 12.36           C  \\nATOM   1190  O   GLY A 151      14.025  27.591   6.148  1.00 11.07           O  \\nATOM   1191  N   VAL A 152      14.706  29.702   6.510  1.00 12.59           N  \\nATOM   1192  CA  VAL A 152      13.339  30.183   6.715  1.00 14.23           C  \\nATOM   1193  C   VAL A 152      12.435  29.971   5.475  1.00 13.41           C  \\nATOM   1194  O   VAL A 152      11.352  29.362   5.577  1.00 12.94           O  \\nATOM   1195  CB  VAL A 152      13.430  31.691   7.167  1.00 12.57           C  \\nATOM   1196  CG1 VAL A 152      12.059  32.316   7.194  1.00 13.14           C  \\nATOM   1197  CG2 VAL A 152      14.044  31.787   8.543  1.00 12.69           C  \\nATOM   1198  N   GLU A 153      12.831  30.410   4.278  1.00 13.52           N  \\nATOM   1199  CA  GLU A 153      12.001  30.213   3.086  1.00 14.81           C  \\nATOM   1200  C   GLU A 153      11.857  28.756   2.742  1.00 11.88           C  \\nATOM   1201  O   GLU A 153      10.797  28.282   2.344  1.00 13.92           O  \\nATOM   1202  CB  GLU A 153      12.592  30.936   1.881  1.00 17.03           C  \\nATOM   1203  CG  GLU A 153      12.454  32.452   2.015  1.00 21.18           C  \\nATOM   1204  CD  GLU A 153      12.870  33.318   0.817  1.00 25.73           C  \\nATOM   1205  OE1 GLU A 153      13.232  32.781  -0.234  1.00 28.32           O  \\nATOM   1206  OE2 GLU A 153      12.822  34.550   0.936  1.00 26.64           O  \\nATOM   1207  N   ASP A 154      12.959  28.031   2.957  1.00 11.31           N  \\nATOM   1208  CA  ASP A 154      13.023  26.593   2.747  1.00 10.16           C  \\nATOM   1209  C   ASP A 154      11.964  25.828   3.561  1.00 10.90           C  \\nATOM   1210  O   ASP A 154      11.224  24.985   3.035  1.00 11.83           O  \\nATOM   1211  CB  ASP A 154      14.420  26.212   3.117  1.00 15.00           C  \\nATOM   1212  CG  ASP A 154      14.919  24.942   2.524  1.00 19.04           C  \\nATOM   1213  OD1 ASP A 154      14.121  24.195   1.938  1.00 22.96           O  \\nATOM   1214  OD2 ASP A 154      16.122  24.697   2.676  1.00 21.26           O  \\nATOM   1215  N   ALA A 155      11.854  26.107   4.881  1.00  8.83           N  \\nATOM   1216  CA  ALA A 155      10.841  25.450   5.716  1.00 10.00           C  \\nATOM   1217  C   ALA A 155       9.424  25.748   5.203  1.00 10.10           C  \\nATOM   1218  O   ALA A 155       8.653  24.814   4.978  1.00 11.14           O  \\nATOM   1219  CB  ALA A 155      10.923  25.933   7.192  1.00  9.50           C  \\nATOM   1220  N   PHE A 156       9.054  27.011   4.949  1.00 10.59           N  \\nATOM   1221  CA  PHE A 156       7.712  27.345   4.475  1.00 10.51           C  \\nATOM   1222  C   PHE A 156       7.453  26.825   3.069  1.00 11.05           C  \\nATOM   1223  O   PHE A 156       6.368  26.302   2.801  1.00 10.80           O  \\nATOM   1224  CB  PHE A 156       7.492  28.890   4.506  1.00 11.54           C  \\nATOM   1225  CG  PHE A 156       7.266  29.450   5.936  1.00 12.42           C  \\nATOM   1226  CD1 PHE A 156       8.319  29.958   6.707  1.00 12.00           C  \\nATOM   1227  CD2 PHE A 156       5.978  29.434   6.478  1.00 12.57           C  \\nATOM   1228  CE1 PHE A 156       8.065  30.434   8.001  1.00 12.45           C  \\nATOM   1229  CE2 PHE A 156       5.747  29.908   7.757  1.00 12.32           C  \\nATOM   1230  CZ  PHE A 156       6.782  30.406   8.513  1.00 10.18           C  \\nATOM   1231  N   TYR A 157       8.408  26.935   2.151  1.00  9.69           N  \\nATOM   1232  CA  TYR A 157       8.148  26.463   0.798  1.00 12.31           C  \\nATOM   1233  C   TYR A 157       8.091  24.945   0.719  1.00 10.42           C  \\nATOM   1234  O   TYR A 157       7.292  24.403  -0.047  1.00 11.35           O  \\nATOM   1235  CB  TYR A 157       9.233  26.978  -0.156  1.00 15.16           C  \\nATOM   1236  CG  TYR A 157       9.257  28.492  -0.367  1.00 18.73           C  \\nATOM   1237  CD1 TYR A 157       8.380  29.366   0.268  1.00 18.52           C  \\nATOM   1238  CD2 TYR A 157      10.211  28.971  -1.242  1.00 20.44           C  \\nATOM   1239  CE1 TYR A 157       8.466  30.724   0.025  1.00 22.04           C  \\nATOM   1240  CE2 TYR A 157      10.305  30.320  -1.488  1.00 22.70           C  \\nATOM   1241  CZ  TYR A 157       9.436  31.178  -0.852  1.00 23.03           C  \\nATOM   1242  OH  TYR A 157       9.572  32.528  -1.089  1.00 26.15           O  \\nATOM   1243  N   THR A 158       8.940  24.198   1.455  1.00 11.35           N  \\nATOM   1244  CA  THR A 158       8.836  22.727   1.522  1.00 10.11           C  \\nATOM   1245  C   THR A 158       7.440  22.347   2.064  1.00 11.11           C  \\nATOM   1246  O   THR A 158       6.811  21.408   1.558  1.00 11.43           O  \\nATOM   1247  CB  THR A 158       9.935  22.129   2.470  1.00 12.09           C  \\nATOM   1248  OG1 THR A 158      11.216  22.451   1.933  1.00 11.36           O  \\nATOM   1249  CG2 THR A 158       9.791  20.657   2.624  1.00 10.87           C  \\nATOM   1250  N   LEU A 159       6.888  23.056   3.072  1.00  9.25           N  \\nATOM   1251  CA  LEU A 159       5.533  22.754   3.506  1.00  9.93           C  \\nATOM   1252  C   LEU A 159       4.489  22.986   2.388  1.00 10.42           C  \\nATOM   1253  O   LEU A 159       3.560  22.185   2.272  1.00 11.22           O  \\nATOM   1254  CB  LEU A 159       5.171  23.608   4.711  1.00  9.03           C  \\nATOM   1255  CG  LEU A 159       3.789  23.318   5.269  1.00 10.96           C  \\nATOM   1256  CD1 LEU A 159       3.635  21.828   5.651  1.00  9.29           C  \\nATOM   1257  CD2 LEU A 159       3.580  24.223   6.469  1.00 13.71           C  \\nATOM   1258  N   VAL A 160       4.596  24.046   1.548  1.00 10.91           N  \\nATOM   1259  CA  VAL A 160       3.662  24.284   0.439  1.00  9.19           C  \\nATOM   1260  C   VAL A 160       3.803  23.137  -0.556  1.00  9.60           C  \\nATOM   1261  O   VAL A 160       2.781  22.638  -1.031  1.00 10.29           O  \\nATOM   1262  CB  VAL A 160       3.981  25.653  -0.211  1.00  8.50           C  \\nATOM   1263  CG1 VAL A 160       3.402  25.754  -1.622  1.00 10.56           C  \\nATOM   1264  CG2 VAL A 160       3.403  26.741   0.659  1.00  7.98           C  \\nATOM   1265  N   ARG A 161       5.033  22.667  -0.839  1.00  9.10           N  \\nATOM   1266  CA  ARG A 161       5.206  21.554  -1.767  1.00  9.82           C  \\nATOM   1267  C   ARG A 161       4.644  20.240  -1.215  1.00 10.44           C  \\nATOM   1268  O   ARG A 161       4.166  19.395  -1.984  1.00 12.88           O  \\nATOM   1269  CB  ARG A 161       6.679  21.398  -2.115  1.00  9.39           C  \\nATOM   1270  CG  ARG A 161       7.146  22.640  -2.816  1.00  9.70           C  \\nATOM   1271  CD  ARG A 161       8.594  22.398  -3.227  1.00 14.12           C  \\nATOM   1272  NE  ARG A 161       9.122  23.635  -3.780  1.00 14.15           N  \\nATOM   1273  CZ  ARG A 161      10.144  24.313  -3.257  1.00 12.45           C  \\nATOM   1274  NH1 ARG A 161      10.823  23.858  -2.204  1.00 13.24           N  \\nATOM   1275  NH2 ARG A 161      10.485  25.475  -3.819  1.00 13.52           N  \\nATOM   1276  N   GLU A 162       4.663  20.046   0.112  1.00 10.71           N  \\nATOM   1277  CA  GLU A 162       4.054  18.877   0.722  1.00 11.80           C  \\nATOM   1278  C   GLU A 162       2.522  18.887   0.568  1.00 11.20           C  \\nATOM   1279  O   GLU A 162       1.921  17.848   0.274  1.00 12.90           O  \\nATOM   1280  CB  GLU A 162       4.430  18.815   2.216  1.00 13.86           C  \\nATOM   1281  CG  GLU A 162       5.870  18.460   2.580  1.00 14.10           C  \\nATOM   1282  CD  GLU A 162       6.268  17.038   2.224  1.00 17.23           C  \\nATOM   1283  OE1 GLU A 162       5.571  16.116   2.627  1.00 19.29           O  \\nATOM   1284  OE2 GLU A 162       7.278  16.827   1.545  1.00 22.14           O  \\nATOM   1285  N   ILE A 163       1.869  20.045   0.731  1.00 10.13           N  \\nATOM   1286  CA  ILE A 163       0.426  20.175   0.542  1.00 10.58           C  \\nATOM   1287  C   ILE A 163       0.128  19.878  -0.933  1.00 13.43           C  \\nATOM   1288  O   ILE A 163      -0.796  19.118  -1.235  1.00 15.01           O  \\nATOM   1289  CB  ILE A 163      -0.015  21.590   0.891  1.00 11.33           C  \\nATOM   1290  CG1 ILE A 163       0.084  21.777   2.400  1.00 11.27           C  \\nATOM   1291  CG2 ILE A 163      -1.486  21.815   0.437  1.00 12.37           C  \\nATOM   1292  CD1 ILE A 163      -0.006  23.245   2.894  1.00 12.56           C  \\nATOM   1293  N   ARG A 164       0.904  20.456  -1.871  1.00 11.72           N  \\nATOM   1294  CA  ARG A 164       0.770  20.163  -3.283  1.00 13.64           C  \\nATOM   1295  C   ARG A 164       0.813  18.657  -3.626  1.00 17.97           C  \\nATOM   1296  O   ARG A 164       0.150  18.197  -4.559  1.00 18.12           O  \\nATOM   1297  CB  ARG A 164       1.897  20.887  -4.029  1.00 12.51           C  \\nATOM   1298  CG  ARG A 164       1.647  22.371  -4.120  1.00 12.49           C  \\nATOM   1299  CD  ARG A 164       2.801  22.888  -4.950  1.00 14.72           C  \\nATOM   1300  NE  ARG A 164       2.730  24.327  -5.113  1.00 12.82           N  \\nATOM   1301  CZ  ARG A 164       3.800  25.071  -5.405  1.00 13.29           C  \\nATOM   1302  NH1 ARG A 164       5.018  24.539  -5.527  1.00 11.56           N  \\nATOM   1303  NH2 ARG A 164       3.595  26.381  -5.572  1.00 12.55           N  \\nATOM   1304  N   GLN A 165       1.590  17.831  -2.935  1.00 20.23           N  \\nATOM   1305  CA  GLN A 165       1.673  16.435  -3.337  1.00 27.50           C  \\nATOM   1306  C   GLN A 165       0.782  15.499  -2.502  1.00 29.12           C  \\nATOM   1307  O   GLN A 165       0.826  14.274  -2.655  1.00 29.30           O  \\nATOM   1308  CB  GLN A 165       3.168  16.070  -3.286  1.00 30.34           C  \\nATOM   1309  CG  GLN A 165       4.018  16.904  -4.309  1.00 35.17           C  \\nATOM   1310  CD  GLN A 165       4.109  16.470  -5.796  1.00 37.71           C  \\nATOM   1311  OE1 GLN A 165       4.256  17.293  -6.719  1.00 36.70           O  \\nATOM   1312  NE2 GLN A 165       4.079  15.166  -6.094  1.00 38.13           N  \\nATOM   1313  N   HIS A 166      -0.068  16.075  -1.638  1.00 29.28           N  \\nATOM   1314  CA  HIS A 166      -0.982  15.296  -0.820  1.00 31.13           C  \\nATOM   1315  C   HIS A 166      -2.214  14.971  -1.665  1.00 33.30           C  \\nATOM   1316  O   HIS A 166      -2.630  15.786  -2.497  1.00 34.96           O  \\nATOM   1317  CB  HIS A 166      -1.420  16.090   0.428  1.00 30.89           C  \\nATOM   1318  CG  HIS A 166      -2.247  15.319   1.475  1.00 32.17           C  \\nATOM   1319  ND1 HIS A 166      -3.579  15.231   1.603  1.00 33.52           N  \\nATOM   1320  CD2 HIS A 166      -1.712  14.560   2.502  1.00 32.95           C  \\nATOM   1321  CE1 HIS A 166      -3.857  14.464   2.647  1.00 33.07           C  \\nATOM   1322  NE2 HIS A 166      -2.727  14.064   3.184  1.00 33.06           N  \\nATOM   1323  OXT HIS A 166      -2.763  13.888  -1.454  1.00 36.27           O  \\nTER    1324      HIS A 166                                                      \\nHETATM 1325 MG    MG A 168       4.771  33.860  19.213  1.00  9.42          MG  \\nHETATM 1326  PG  GNP A 167       5.113  32.223  21.994  1.00  9.66           P  \\nHETATM 1327  O1G GNP A 167       4.844  32.596  23.390  1.00  9.26           O  \\nHETATM 1328  O2G GNP A 167       4.203  32.910  21.112  1.00 11.20           O  \\nHETATM 1329  O3G GNP A 167       4.640  30.769  21.802  1.00 10.09           O  \\nHETATM 1330  N3B GNP A 167       6.701  32.303  21.621  1.00  8.18           N  \\nHETATM 1331  PB  GNP A 167       7.310  32.034  20.111  1.00  8.77           P  \\nHETATM 1332  O1B GNP A 167       7.468  30.605  19.791  1.00  8.35           O  \\nHETATM 1333  O2B GNP A 167       6.696  32.775  19.021  1.00  9.38           O  \\nHETATM 1334  O3A GNP A 167       8.792  32.643  20.152  1.00 10.15           O  \\nHETATM 1335  PA  GNP A 167       9.280  34.140  19.710  1.00 10.18           P  \\nHETATM 1336  O1A GNP A 167       9.263  34.268  18.220  1.00 10.07           O  \\nHETATM 1337  O2A GNP A 167       8.594  35.173  20.510  1.00  9.33           O  \\nHETATM 1338  O5' GNP A 167      10.826  34.056  20.132  1.00 11.46           O  \\nHETATM 1339  C5' GNP A 167      11.170  33.911  21.535  1.00 10.56           C  \\nHETATM 1340  C4' GNP A 167      12.587  34.445  21.811  1.00 11.96           C  \\nHETATM 1341  O4' GNP A 167      13.560  33.649  21.095  1.00 11.66           O  \\nHETATM 1342  C3' GNP A 167      12.813  35.910  21.359  1.00 12.06           C  \\nHETATM 1343  O3' GNP A 167      13.793  36.538  22.221  1.00 14.52           O  \\nHETATM 1344  C2' GNP A 167      13.325  35.679  19.947  1.00 11.90           C  \\nHETATM 1345  O2' GNP A 167      14.151  36.750  19.499  1.00 14.99           O  \\nHETATM 1346  C1' GNP A 167      14.242  34.478  20.156  1.00 12.90           C  \\nHETATM 1347  N9  GNP A 167      14.454  33.739  18.896  1.00 12.98           N  \\nHETATM 1348  C8  GNP A 167      13.518  33.262  17.993  1.00 11.76           C  \\nHETATM 1349  N7  GNP A 167      14.016  32.636  16.979  1.00 12.34           N  \\nHETATM 1350  C5  GNP A 167      15.413  32.694  17.200  1.00 12.36           C  \\nHETATM 1351  C6  GNP A 167      16.521  32.219  16.402  1.00 12.30           C  \\nHETATM 1352  O6  GNP A 167      16.507  31.591  15.338  1.00 12.16           O  \\nHETATM 1353  N1  GNP A 167      17.735  32.503  16.995  1.00 11.60           N  \\nHETATM 1354  C2  GNP A 167      17.909  33.082  18.236  1.00 13.91           C  \\nHETATM 1355  N2  GNP A 167      19.173  33.280  18.658  1.00 14.60           N  \\nHETATM 1356  N3  GNP A 167      16.884  33.498  18.996  1.00 13.01           N  \\nHETATM 1357  C4  GNP A 167      15.675  33.331  18.398  1.00 14.36           C  \\nHETATM 1358  O   HOH A 169      13.588  43.386  17.779  1.00 31.80           O  \\nHETATM 1359  O   HOH A 170       8.248  36.064  23.209  1.00 13.63           O  \\nHETATM 1360  O   HOH A 171       3.254  37.845  19.454  1.00 18.29           O  \\nHETATM 1361  O   HOH A 172       5.945  35.619  20.105  1.00 11.33           O  \\nHETATM 1362  O   HOH A 173       3.669  32.226  18.172  1.00  8.22           O  \\nHETATM 1363  O   HOH A 174       2.896  37.170  15.500  1.00 21.37           O  \\nHETATM 1364  O   HOH A 175       2.102  32.309  24.122  1.00 27.47           O  \\nHETATM 1365  O   HOH A 176      -1.425  37.650  19.667  1.00 24.98           O  \\nHETATM 1366  O   HOH A 177      -3.225  30.031  -8.217  1.00 23.57           O  \\nHETATM 1367  O   HOH A 178      16.678  26.321  26.916  1.00 36.70           O  \\nHETATM 1368  O   HOH A 179      -2.134  40.975   1.551  1.00 81.53           O  \\nHETATM 1369  O   HOH A 180       0.400  40.675  -0.629  1.00 24.66           O  \\nHETATM 1370  O   HOH A 181      10.944  44.793  16.009  1.00 48.47           O  \\nHETATM 1371  O   HOH A 182      17.207  42.570  16.786  1.00 43.23           O  \\nHETATM 1372  O   HOH A 183      14.999  45.167  19.922  1.00 39.52           O  \\nHETATM 1373  O   HOH A 184      11.598  37.755  13.202  1.00 11.72           O  \\nHETATM 1374  O   HOH A 185       6.958  38.922  18.146  1.00 26.17           O  \\nHETATM 1375  O   HOH A 186      14.078  32.033  27.772  1.00 57.73           O  \\nHETATM 1376  O   HOH A 187       9.126  37.906  19.595  1.00 32.42           O  \\nHETATM 1377  O   HOH A 188      -0.049  36.435  27.259  1.00 50.56           O  \\nHETATM 1378  O   HOH A 189       3.843  30.503  25.485  1.00 46.80           O  \\nHETATM 1379  O   HOH A 190       4.931  28.343  27.587  1.00 29.32           O  \\nHETATM 1380  O   HOH A 191       4.881  25.774  26.247  1.00 28.55           O  \\nHETATM 1381  O   HOH A 193       8.729  26.516  26.645  1.00 37.34           O  \\nHETATM 1382  O   HOH A 194      -5.515  23.731  -4.213  1.00 43.74           O  \\nHETATM 1383  O   HOH A 195      -7.006  26.635   0.729  1.00 25.77           O  \\nHETATM 1384  O   HOH A 196      -2.033  23.281  -4.754  1.00 36.89           O  \\nHETATM 1385  O   HOH A 197      -4.326  24.642  -0.854  1.00 28.59           O  \\nHETATM 1386  O   HOH A 198      -6.858  26.995  -2.808  1.00 28.83           O  \\nHETATM 1387  O   HOH A 200      -3.937  29.225   8.289  1.00 24.08           O  \\nHETATM 1388  O   HOH A 201      -8.934  24.037   2.795  1.00 34.16           O  \\nHETATM 1389  O   HOH A 202       1.444  26.302  19.318  1.00 12.37           O  \\nHETATM 1390  O   HOH A 203       2.031  24.632  27.307  1.00 50.09           O  \\nHETATM 1391  O   HOH A 204      15.125  28.341  23.174  1.00 13.88           O  \\nHETATM 1392  O   HOH A 205      12.808  31.541  25.056  1.00 46.88           O  \\nHETATM 1393  O   HOH A 206       8.329  34.933   0.543  1.00 53.21           O  \\nHETATM 1394  O   HOH A 207       0.760  42.969   2.336  1.00 35.42           O  \\nHETATM 1395  O   HOH A 208       1.270  40.298   7.071  1.00 76.61           O  \\nHETATM 1396  O   HOH A 209      -3.135  39.625   7.649  1.00 35.40           O  \\nHETATM 1397  O   HOH A 210       8.699  39.076   2.399  1.00 33.93           O  \\nHETATM 1398  O   HOH A 212      10.040  41.806   3.447  1.00 37.88           O  \\nHETATM 1399  O   HOH A 213      18.914  29.095   3.704  1.00 25.32           O  \\nHETATM 1400  O   HOH A 214      15.966  29.098   2.957  1.00 18.63           O  \\nHETATM 1401  O   HOH A 215      15.506  31.962   4.015  1.00 18.93           O  \\nHETATM 1402  O   HOH A 216      22.935  30.533  10.845  1.00 39.81           O  \\nHETATM 1403  O   HOH A 217       6.476  43.893  23.325  1.00 49.09           O  \\nHETATM 1404  O   HOH A 218      11.746  41.248  28.612  1.00 36.22           O  \\nHETATM 1405  O   HOH A 219      11.480  42.270  25.715  1.00 29.66           O  \\nHETATM 1406  O   HOH A 220       9.344  34.251  -8.278  1.00 36.61           O  \\nHETATM 1407  O   HOH A 221      -0.111  19.667  -8.066  1.00 44.76           O  \\nHETATM 1408  O   HOH A 222      -0.528  30.460 -10.943  1.00 24.29           O  \\nHETATM 1409  O   HOH A 223     -11.352  24.274  19.491  1.00 31.21           O  \\nHETATM 1410  O   HOH A 224      -4.293  30.642 -16.466  1.00127.55           O  \\nHETATM 1411  O   HOH A 225       3.466  29.729 -16.232  1.00 26.15           O  \\nHETATM 1412  O   HOH A 227      -4.288  18.610   4.198  1.00 34.77           O  \\nHETATM 1413  O   HOH A 228       4.626  16.154   5.151  1.00 14.96           O  \\nHETATM 1414  O   HOH A 229       5.681  11.069   2.580  1.00 59.76           O  \\nHETATM 1415  O   HOH A 230       1.173  14.021   3.406  1.00 32.15           O  \\nHETATM 1416  O   HOH A 231       7.935  22.484  20.504  1.00 10.86           O  \\nHETATM 1417  O   HOH A 232       3.071  10.004  14.546  1.00 23.48           O  \\nHETATM 1418  O   HOH A 233       2.437  13.381   8.401  1.00 17.75           O  \\nHETATM 1419  O   HOH A 234       4.779   7.500   6.154  1.00 71.09           O  \\nHETATM 1420  O   HOH A 235      -4.250   8.748   6.880  1.00 24.76           O  \\nHETATM 1421  O   HOH A 236     -13.381  22.830  17.669  1.00 39.09           O  \\nHETATM 1422  O   HOH A 237      16.969  39.052   8.276  1.00 50.25           O  \\nHETATM 1423  O   HOH A 238      -6.315   9.333  13.897  1.00 36.24           O  \\nHETATM 1424  O   HOH A 240      -0.160   4.395  12.468  1.00 38.79           O  \\nHETATM 1425  O   HOH A 241       1.295   7.931  12.335  1.00 49.03           O  \\nHETATM 1426  O   HOH A 242       4.067  11.551   6.968  1.00 46.97           O  \\nHETATM 1427  O   HOH A 243      -5.060   6.579  -0.216  1.00 50.14           O  \\nHETATM 1428  O   HOH A 244      18.164   7.151   5.573  1.00 71.67           O  \\nHETATM 1429  O   HOH A 245       2.068   8.904   7.636  1.00 45.86           O  \\nHETATM 1430  O   HOH A 246      24.661  23.134  12.757  1.00 34.69           O  \\nHETATM 1431  O   HOH A 247      21.051  23.714   6.239  1.00 24.64           O  \\nHETATM 1432  O   HOH A 248      25.855  31.137  20.712  1.00 34.53           O  \\nHETATM 1433  O   HOH A 249      24.868  30.890  26.223  1.00 48.47           O  \\nHETATM 1434  O   HOH A 250      26.967  20.447  19.448  1.00 47.77           O  \\nHETATM 1435  O   HOH A 252      28.453  14.196  14.602  1.00 39.78           O  \\nHETATM 1436  O   HOH A 253      19.639  16.256   9.760  1.00 37.54           O  \\nHETATM 1437  O   HOH A 254      16.629   7.383  10.526  1.00 60.90           O  \\nHETATM 1438  O   HOH A 255       4.972   7.888  14.276  1.00 54.95           O  \\nHETATM 1439  O   HOH A 256      14.184  14.684   5.643  1.00 31.40           O  \\nHETATM 1440  O   HOH A 257      11.766  17.254   1.278  1.00 41.82           O  \\nHETATM 1441  O   HOH A 258      16.274  16.365   6.342  1.00 17.29           O  \\nHETATM 1442  O   HOH A 259      18.346  24.095   6.620  1.00 17.36           O  \\nHETATM 1443  O   HOH A 260      22.479  35.073  19.694  1.00 43.96           O  \\nHETATM 1444  O   HOH A 261      20.580  31.747   4.906  1.00 59.46           O  \\nHETATM 1445  O   HOH A 262      18.196  25.514   4.261  1.00 28.75           O  \\nHETATM 1446  O   HOH A 263      12.857  20.183   1.149  1.00 21.79           O  \\nHETATM 1447  O   HOH A 264      19.514  15.912  24.943  1.00 55.52           O  \\nHETATM 1448  O   HOH A 265       8.925   7.579   5.621  1.00 77.94           O  \\nHETATM 1449  O   HOH A 266      17.622  21.835  26.964  1.00 27.48           O  \\nHETATM 1450  O   HOH A 267      24.258  19.786  14.155  1.00 20.48           O  \\nHETATM 1451  O   HOH A 268      23.806  15.180  15.136  1.00 19.86           O  \\nHETATM 1452  O   HOH A 269      21.478  14.557  11.134  1.00 29.66           O  \\nHETATM 1453  O   HOH A 270      17.293  45.621  24.753  1.00 39.18           O  \\nHETATM 1454  O   HOH A 271      -3.518  40.206  17.649  1.00 44.75           O  \\nHETATM 1455  O   HOH A 272       2.143  19.852  24.090  1.00 40.10           O  \\nHETATM 1456  O   HOH A 273       7.101  35.918  -4.704  1.00 31.76           O  \\nHETATM 1457  O   HOH A 274       5.462  38.756  -5.521  1.00 27.51           O  \\nHETATM 1458  O   HOH A 275       5.748  20.565  28.638  1.00 58.06           O  \\nHETATM 1459  O   HOH A 276       8.814  31.258  -8.964  1.00 23.79           O  \\nHETATM 1460  O   HOH A 277     -10.427  35.895   4.301  1.00 58.91           O  \\nHETATM 1461  O   HOH A 278      -8.713  28.579   2.333  1.00 69.36           O  \\nHETATM 1462  O   HOH A 279      24.472  24.237  17.316  1.00 32.14           O  \\nHETATM 1463  O   HOH A 281      16.968  32.701  21.815  1.00 23.13           O  \\nHETATM 1464  O   HOH A 282      19.541  34.346  21.581  1.00 53.25           O  \\nHETATM 1465  O   HOH A 283      24.847  23.384  26.621  1.00 61.18           O  \\nHETATM 1466  O   HOH A 284       1.409  11.113   2.873  1.00 44.87           O  \\nHETATM 1467  O   HOH A 285       1.556  11.488  -0.914  1.00 37.86           O  \\nHETATM 1468  O   HOH A 286       9.758  12.945   3.889  1.00 36.47           O  \\nHETATM 1469  O   HOH A 287       3.827  13.375   4.622  1.00 44.90           O  \\nHETATM 1470  O   HOH A 288      14.279   6.175  15.391  1.00 78.25           O  \\nHETATM 1471  O   HOH A 289       7.573  13.968   2.191  1.00 76.47           O  \\nHETATM 1472  O   HOH A 290       2.323  15.482   1.418  1.00 35.15           O  \\nHETATM 1473  O   HOH A 291      11.997  14.573   2.962  1.00 49.86           O  \\nHETATM 1474  O   HOH A 292      23.798  33.550  17.880  1.00 24.64           O  \\nHETATM 1475  O   HOH A 293      14.657  36.046  -0.571  1.00 51.86           O  \\nHETATM 1476  O   HOH A 294      24.456  25.366   4.004  1.00 53.23           O  \\nHETATM 1477  O   HOH A 296      -0.603  12.134  -4.185  1.00 52.38           O  \\nHETATM 1478  O   HOH A 297      -8.372  34.415  -4.471  1.00 74.53           O  \\nHETATM 1479  O   HOH A 300     -14.033  36.501   0.062  1.00 50.21           O  \\nHETATM 1480  O   HOH A 301     -17.102  35.735  -2.806  1.00 42.58           O  \\nHETATM 1481  O   HOH A 303     -17.383  40.542   1.431  1.00 77.16           O  \\nHETATM 1482  O   HOH A 304     -19.024  41.557  -3.411  1.00107.16           O  \\nHETATM 1483  O   HOH A 305      -9.876  37.804   0.839  1.00 90.65           O  \\nHETATM 1484  O   HOH A 306     -14.053  36.466   4.552  1.00 82.05           O  \\nHETATM 1485  O   HOH A 307     -14.183  39.558   0.339  1.00 58.66           O  \\nHETATM 1486  O   HOH A 309     -12.547  28.677   8.301  1.00 74.88           O  \\nHETATM 1487  O   HOH A 311     -14.787  32.746  -2.171  1.00 40.79           O  \\nHETATM 1488  O   HOH A 312      -6.916  30.740   3.095  1.00 36.53           O  \\nHETATM 1489  O   HOH A 315      -8.312  33.027   7.340  1.00 83.28           O  \\nHETATM 1490  O   HOH A 316      -6.280  35.646   7.624  1.00 75.97           O  \\nHETATM 1491  O   HOH A 317      -5.230  31.565   9.609  1.00 55.67           O  \\nHETATM 1492  O   HOH A 318       6.747  37.258   1.208  1.00 57.93           O  \\nHETATM 1493  O   HOH A 319       4.747  42.419  12.683  1.00 57.74           O  \\nHETATM 1494  O   HOH A 320       2.984  41.950  16.191  1.00 41.69           O  \\nHETATM 1495  O   HOH A 321       3.867  49.208  12.056  1.00 24.57           O  \\nHETATM 1496  O   HOH A 322       7.530  42.894  17.776  1.00 79.28           O  \\nHETATM 1497  O   HOH A 324      19.822  39.457  17.826  1.00 66.22           O  \\nHETATM 1498  O   HOH A 325      12.391  45.557  24.973  1.00 48.70           O  \\nHETATM 1499  O   HOH A 326      14.944  43.928  23.794  1.00 62.45           O  \\nHETATM 1500  O   HOH A 327     -12.984  24.008  12.563  1.00 40.31           O  \\nHETATM 1501  O   HOH A 330       0.572  41.133  19.277  1.00 50.77           O  \\nHETATM 1502  O   HOH A 331      -6.196  37.979  23.043  1.00 48.93           O  \\nHETATM 1503  O   HOH A 332      -3.526  30.410  19.065  1.00 21.26           O  \\nHETATM 1504  O   HOH A 333      -7.995  32.093  21.685  1.00 54.11           O  \\nHETATM 1505  O   HOH A 334      -8.483  28.891  20.063  1.00 70.37           O  \\nHETATM 1506  O   HOH A 335     -11.670  31.022  16.981  1.00 76.86           O  \\nHETATM 1507  O   HOH A 336     -11.082  34.450  18.910  1.00 43.07           O  \\nHETATM 1508  O   HOH A 339      -6.521  37.738  15.165  1.00 64.11           O  \\nHETATM 1509  O   HOH A 340     -10.829  35.877  15.629  1.00 58.36           O  \\nHETATM 1510  O   HOH A 341      -6.552  41.166  12.231  1.00 55.67           O  \\nHETATM 1511  O   HOH A 342      -1.242  39.671  12.185  1.00 61.04           O  \\nHETATM 1512  O   HOH A 343      -4.281  38.934  13.464  1.00 32.14           O  \\nHETATM 1513  O   HOH A 344       0.386  41.208  14.039  1.00 40.80           O  \\nHETATM 1514  O   HOH A 345       0.777  39.663  21.603  1.00 67.71           O  \\nHETATM 1515  O   HOH A 346      -7.879  34.848  19.044  1.00 52.60           O  \\nHETATM 1516  O   HOH A 347       1.121  32.594 -11.957  1.00 32.30           O  \\nHETATM 1517  O   HOH A 348       0.924  32.597 -14.716  1.00 48.34           O  \\nHETATM 1518  O   HOH A 349      -2.241  27.348  20.565  1.00 26.54           O  \\nHETATM 1519  O   HOH A 350       3.259  26.214  29.881  1.00 69.22           O  \\nHETATM 1520  O   HOH A 351      -8.318  34.095  28.392  1.00 81.96           O  \\nHETATM 1521  O   HOH A 352      -2.927  29.775  31.412  1.00 80.92           O  \\nHETATM 1522  O   HOH A 353     -10.410  32.034  25.972  1.00 60.17           O  \\nHETATM 1523  O   HOH A 354      -8.122  30.393  25.378  1.00 62.33           O  \\nHETATM 1524  O   HOH A 356      -6.138  18.882  24.644  1.00 49.26           O  \\nHETATM 1525  O   HOH A 358      -1.406  19.033  23.084  1.00 82.05           O  \\nHETATM 1526  O   HOH A 359       4.169   7.339  10.280  1.00 66.03           O  \\nHETATM 1527  O   HOH A 360      -9.761  30.165   7.702  1.00 46.15           O  \\nHETATM 1528  O   HOH A 361     -16.002  27.801  11.700  1.00 41.85           O  \\nHETATM 1529  O   HOH A 362      16.215  30.489  29.264  1.00 67.21           O  \\nHETATM 1530  O   HOH A 364      21.417  27.685  26.140  1.00 49.57           O  \\nHETATM 1531  O   HOH A 366      15.824  34.767  30.662  1.00 25.90           O  \\nHETATM 1532  O   HOH A 367      18.527  35.965  24.111  1.00 68.69           O  \\nHETATM 1533  O   HOH A 369      12.103  35.275  24.945  1.00 51.37           O  \\nHETATM 1534  O   HOH A 370      24.782  11.970  23.641  1.00 55.92           O  \\nHETATM 1535  O   HOH A 371      14.337  21.394  28.627  1.00 30.11           O  \\nHETATM 1536  O   HOH A 372      -1.367  12.600  23.053  1.00 44.43           O  \\nHETATM 1537  O   HOH A 373       7.060   8.567  21.416  1.00 47.99           O  \\nHETATM 1538  O   HOH A 374       7.639   8.040  16.420  1.00 38.76           O  \\nHETATM 1539  O   HOH A 376      -1.282  15.566  25.487  1.00 67.79           O  \\nHETATM 1540  O   HOH A 378      17.945  30.924  23.806  1.00 31.50           O  \\nHETATM 1541  O   HOH A 379      17.360  33.209  33.337  1.00 30.46           O  \\nHETATM 1542  O   HOH A 380       3.971   6.567  22.089  1.00 86.80           O  \\nHETATM 1543  O   HOH A 381      -0.065  10.022  26.928  1.00 61.86           O  \\nHETATM 1544  O   HOH A 382      -4.450  12.966  21.222  1.00 42.82           O  \\nHETATM 1545  O   HOH A 383      -2.584   6.719   5.938  1.00 40.64           O  \\nHETATM 1546  O   HOH A 384      25.567  31.936  12.845  1.00 73.17           O  \\nHETATM 1547  O   HOH A 385      24.446  29.383  22.970  1.00 36.61           O  \\nHETATM 1548  O   HOH A 386      24.865  38.167  18.227  1.00 60.74           O  \\nHETATM 1549  O   HOH A 387      23.135  35.012  14.543  1.00 59.19           O  \\nHETATM 1550  O   HOH A 388      19.325   9.556  18.093  1.00 92.18           O  \\nHETATM 1551  O   HOH A 389      25.326  12.873  16.251  1.00 49.27           O  \\nHETATM 1552  O   HOH A 390      20.219  10.405  10.956  1.00 63.05           O  \\nHETATM 1553  O   HOH A 392       9.452   5.453   8.249  1.00 47.81           O  \\nHETATM 1554  O   HOH A 394      10.139  12.252   0.254  1.00 29.37           O  \\nHETATM 1555  O   HOH A 395      28.004  37.416  13.466  1.00 52.29           O  \\nHETATM 1556  O   HOH A 396      24.283  36.829  12.307  1.00 64.74           O  \\nHETATM 1557  O   HOH A 397      13.748  34.031  -3.019  1.00 47.80           O  \\nHETATM 1558  O   HOH A 398      16.981  34.189   2.006  1.00 80.51           O  \\nHETATM 1559  O   HOH A 399      -4.426  24.223  -7.240  1.00 38.09           O  \\nHETATM 1560  O   HOH A 400      19.676  33.981   3.336  1.00 64.86           O  \\nHETATM 1561  O   HOH A 401      14.633  28.745  -0.317  1.00 66.25           O  \\nHETATM 1562  O   HOH A 402      -2.139  15.576  22.716  1.00 65.01           O  \\nHETATM 1563  O   HOH A 403       9.873  11.090  20.760  1.00 53.88           O  \\nHETATM 1564  O   HOH A 404      18.291  36.379  26.927  1.00 57.23           O  \\nHETATM 1565  O   HOH A 405       0.138   6.145  15.269  1.00 82.80           O  \\nHETATM 1566  O   HOH A 406      -5.515  38.166  18.689  1.00 48.28           O  \\nHETATM 1567  O   HOH A 407      -0.399   4.688   8.809  1.00 58.35           O  \\nHETATM 1568  O   HOH A 408       0.813  25.584  22.307  1.00 40.04           O  \\nCONECT  117 1325                                                                \\nCONECT  265 1325                                                                \\nCONECT 1325  117  265 1328 1333                                                 \\nCONECT 1325 1361 1362                                                           \\nCONECT 1326 1327 1328 1329 1330                                                 \\nCONECT 1327 1326                                                                \\nCONECT 1328 1325 1326                                                           \\nCONECT 1329 1326                                                                \\nCONECT 1330 1326 1331                                                           \\nCONECT 1331 1330 1332 1333 1334                                                 \\nCONECT 1332 1331                                                                \\nCONECT 1333 1325 1331                                                           \\nCONECT 1334 1331 1335                                                           \\nCONECT 1335 1334 1336 1337 1338                                                 \\nCONECT 1336 1335                                                                \\nCONECT 1337 1335                                                                \\nCONECT 1338 1335 1339                                                           \\nCONECT 1339 1338 1340                                                           \\nCONECT 1340 1339 1341 1342                                                      \\nCONECT 1341 1340 1346                                                           \\nCONECT 1342 1340 1343 1344                                                      \\nCONECT 1343 1342                                                                \\nCONECT 1344 1342 1345 1346                                                      \\nCONECT 1345 1344                                                                \\nCONECT 1346 1341 1344 1347                                                      \\nCONECT 1347 1346 1348 1357                                                      \\nCONECT 1348 1347 1349                                                           \\nCONECT 1349 1348 1350                                                           \\nCONECT 1350 1349 1351 1357                                                      \\nCONECT 1351 1350 1352 1353                                                      \\nCONECT 1352 1351                                                                \\nCONECT 1353 1351 1354                                                           \\nCONECT 1354 1353 1355 1356                                                      \\nCONECT 1355 1354                                                                \\nCONECT 1356 1354 1357                                                           \\nCONECT 1357 1347 1350 1356                                                      \\nCONECT 1361 1325                                                                \\nCONECT 1362 1325                                                                \\nMASTER      312    0    2    5    6    0    9    6 1567    1   38   13          \\nEND                                                                             \\n\",\"pdb\");\n\tviewergrid_16835771918631198[0][1].addModel(\"\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n    8.1498    9.9395    0.8183 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.1695    9.3371    1.5922 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.4103    8.1108    1.4000 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.9316   10.0209    2.5654 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.6967   10.7493    1.1472 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.6864   10.9350    2.8365 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0759    9.0817   -0.2489 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4561    9.6329   -0.2770 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5017    8.5444   -0.1653 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.7693    9.3636   -0.4637 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2842   10.4214   -1.3759 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0242   11.2860   -1.9323 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8490   10.2763   -1.5004 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1270   10.7110   -2.6308 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0018    9.9037   -3.7685 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3069   10.2965   -4.8903 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7101   11.5464   -4.8813 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8178   12.3608   -3.7734 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5166   11.9402   -2.6711 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.0759    9.3652    3.1600 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   12.0671   10.3169    3.7560 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.9955   11.5604    3.1551 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   13.4528    9.7553    3.5843 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   14.4727   10.6536    4.2325 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   14.1515   10.9003    5.6810 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   12.7439   11.3926    5.8652 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.7439   10.4670    5.2278 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   12.2011   11.4757    2.2110 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-4.32704\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n   -5.5591    6.5652    5.3801 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4601    7.2046    4.1219 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3728    6.9851    3.2746 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.4052    8.0696    3.7554 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.7503    6.4196    5.9223 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6134    8.1603    4.3334 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0424    4.3506    9.0315 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1768    5.4043    7.9916 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5582    4.8251    6.6463 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8550    6.1223    5.8745 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3423    7.0404    6.9260 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7729    8.2128    6.7140 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2327    6.3568    8.1977 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0500    6.6586    9.3063 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.3668    6.1884    9.3885 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.1936    6.4652   10.4541 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.6899    7.2434   11.4832 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3963    7.7185   11.4271 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.5976    7.4246   10.3520 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.4994    8.8210    2.5228 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.9032   10.2495    2.7231 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8859   11.1153    2.3654 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.2580   10.4621    4.1700 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.7350   11.8721    4.3970 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8824   12.2209    3.4892 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5596   11.9421    2.0485 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1069   10.5220    1.8454 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.1092   10.9492    2.9221 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-4.22165\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n   -2.0336    7.3002    3.2585 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.0649    8.1827    2.8594 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.1311    8.4991    1.6369 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.0214    8.7315    3.7421 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.7263    7.2945    4.1938 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9237    8.6161    4.7149 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.1505    6.3186    3.2716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.6701    6.1875    3.2975 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.0188    6.9057    2.1353 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.4261    6.4003    2.2883 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.2557    5.0542    2.8758 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.2024    4.2437    3.1029 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.1559    4.8563    3.1297 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.7591    3.5824    3.1699 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.0041    2.9318    4.3858 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.5930    1.6896    4.4628 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.9543    1.0666    3.2798 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.7236    1.6854    2.0689 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.1345    2.9228    2.0258 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.1459    9.4649    3.2031 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.7133   10.4731    4.1546 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.7644   10.8922    5.0692 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8777    9.8598    4.8844 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.5178   10.8671    5.8024 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9231   12.1117    5.0615 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7821   12.6884    4.2721 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1747   11.6688    3.3478 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.4618   10.1332    5.5919 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-4.07662\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n   -7.1997    7.9665    8.2440 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4895    8.3884    6.9251 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0297    7.5540    6.1435 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2111    9.6816    6.4295 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.4990    8.4233    8.7633 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6743   10.3126    6.9614 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3429    6.4030   12.1545 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.0660    6.9950   10.9986 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9185    6.1539    9.7490 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9360    6.8579    8.8347 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.9513    7.3653    9.7822 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.0204    7.9543    9.4433 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4958    7.0654   11.1235 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.3730    6.9003   12.2150 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.9823    5.6669   12.4784 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.8453    5.4702   13.5329 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -11.1149    6.5462   14.3622 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.5276    7.7714   14.1253 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.6717    7.9350   13.0665 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7220   10.0699    5.1330 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9418   11.5450    4.9946 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.7161   11.8372    3.8866 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6084   12.2315    4.8704 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7860   13.7250    4.7975 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.5695   14.2468    5.9705 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.8705   13.5153    6.1433 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.6636   12.0281    6.2352 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.2625   11.5316    3.0855 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-4.06176\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n    7.8347   10.1795    0.6909 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.7621    9.4932    1.5097 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.8104    8.2329    1.4189 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.6337   10.1291    2.4215 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.5151   11.0732    0.9527 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.5346   11.0890    2.6166 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0092   12.0862   -1.9268 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2385   11.4206   -1.4213 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9269   10.1170   -0.7184 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.3521    9.5663   -0.5384 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0788   10.1150   -1.7033 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.2837    9.8487   -1.9897 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1718   10.9844   -2.4230 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2934   11.2507   -3.8023 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7850   10.3656   -4.7615 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.8878   10.5921   -6.1156 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.5236   11.7489   -6.5352 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0325   12.6373   -5.6112 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.9136   12.3832   -4.2690 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.6733    9.3566    3.0658 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.8421   10.1812    3.5104 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.8395   11.4254    2.9066 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   13.1151    9.4574    3.1636 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   14.3126   10.2253    3.6569 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   14.2205   10.5064    5.1316 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   12.9193   11.1641    5.4947 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.7394   10.3666    5.0099 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.9050   11.3181    1.9447 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.99217\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n    2.9603    8.8680    1.1997 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0437    9.3540    0.4305 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0325    8.5864    0.2521 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0879   10.6423   -0.1472 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.3534    9.5001    1.6483 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4022   11.3118    0.0782 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2273    7.0677    4.6963 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.4629    7.4669    3.2840 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.8735    7.1514    2.8354 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7415    7.4343    1.3290 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.3288    7.1046    1.0431 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.8044    7.1268   -0.1098 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.6898    6.7646    2.2971 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.4328    5.9159    2.3833 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.2884    4.5256    2.4720 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.3614    3.6669    2.5551 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.6339    4.2137    2.5498 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.8038    5.5797    2.4639 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.7144    6.4084    2.3829 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1504   10.9687   -1.0731 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1634   12.4090   -1.4843 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0063   12.7502   -2.1605 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3543   12.6568   -2.3702 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4392   14.1116   -2.7493 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4587   14.9994   -1.5354 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3114   14.7065   -0.6105 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2620   13.2519   -0.2300 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.9332   12.2155   -2.9665 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.94954\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n   -9.2444    8.5266    7.4724 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.2625    7.6554    8.0002 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6100    6.9395    7.1874 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9693    7.5361    9.3769 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.9882    8.8310    8.0411 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.5421    7.9746   10.0468 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  -11.4592   11.8798    5.9487 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.3335   10.9796    6.3122 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.5387    9.5764    5.7835 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.1567    8.9738    6.0895 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.2478   10.1352    5.9841 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9871   10.0738    6.0920 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.0568   11.3114    5.7424 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.5709   12.4560    5.0777 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0699   13.5530    5.7900 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.5900   14.6872    5.1746 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6086   14.7335    3.7904 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0962   13.6666    3.0651 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.5668   12.5506    3.7080 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8192    6.7635    9.7930 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8444    6.3878   11.2428 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.9345    7.1315   11.9718 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5195    4.9246   11.3775 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6021    4.4938   12.8179 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9426    4.8200   13.4169 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3052    6.2650   13.2214 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.2360    6.6640   11.7727 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0389    6.9619   11.6402 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.74039\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n    2.6681    5.6237    6.8498 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.3542    4.7346    7.7102 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7488    3.6891    8.0837 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6707    4.9446    8.1774 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9325    6.5712    6.8100 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1233    5.8042    8.0181 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.0009    7.8945    3.6041 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.6382    6.7965    4.3773 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.8179    6.4085    5.5886 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.5716    5.1397    6.0231 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.0916    4.5994    4.7488 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7205    3.5056    4.6348 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.7509    5.5353    3.6979 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.5917    5.1539    2.3499 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.3915    4.5993    1.8877 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.2016    4.2136    0.5798 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2505    4.3865   -0.3081 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.4428    4.9309    0.1212 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.6003    5.3057    1.4307 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3476    3.8845    8.8922 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1056    4.3624   10.0925 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4737    5.6889    9.9605 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.3369    3.5146   10.2662 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0905    3.9221   11.5044 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2130    3.8796   12.7252 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9472    4.6654   12.5311 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2028    4.2245   11.3005 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.0528    5.7885    9.1888 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.71957\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n   -8.1030    7.8964    8.9253 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8174    7.7427    7.5482 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9203    6.5853    7.0494 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4235    8.8010    6.6996 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7214    8.6542    9.4248 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2046    9.6842    7.0755 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.6448    8.5241   12.8875 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.5734    8.2653   11.4255 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.6438    7.1162   11.0999 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.9602    6.9356    9.6052 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.3754    7.3514    9.5039 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -11.0689    7.2901    8.4456 O   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.7932    7.8247   10.8069 N   0  0  0  0  0  0  0  0  0  0  0  0\\n  -12.1339    7.7953   11.2425 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -12.9502    8.9306   11.1612 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -14.2620    8.9388   11.5790 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -14.7860    7.7675   12.1005 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -14.0036    6.6353   12.1911 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -12.7003    6.6593   11.7658 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3448    8.5749    5.2729 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7115    9.7060    4.5225 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.3685    9.4688    4.2929 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8686   10.9734    5.3187 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3030   12.1470    4.5637 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9169   12.2731    3.1965 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8277   10.9879    2.4232 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4186    9.8363    3.1897 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.9056    9.3836    5.1411 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.67500\\n\\n$$$$\\n\",\"sdf\");\n\tviewergrid_16835771918631198[0][1].setStyle({\"model\": 0},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewergrid_16835771918631198[0][1].addSurface(1,{\"opacity\": 0.5, \"color\": \"white\"},{\"model\": 0});\n\tviewergrid_16835771918631198[0][1].setStyle({\"model\": 1},{\"stick\": {}});\n\tviewergrid_16835771918631198[0][1].zoomTo();\n\tviewergrid_16835771918631198[1][0].addModel(\"HEADER    ONCOGENE PROTEIN                        30-APR-90   5P21              \\nTITLE     REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION OF H-RAS   \\nTITLE    2 P21 AT 1.35 ANGSTROMS RESOLUTION: IMPLICATIONS FOR THE MECHANISM OF  \\nTITLE    3 GTP HYDROLYSIS                                                       \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: C-H-RAS P21 PROTEIN;                                       \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    ONCOGENE PROTEIN                                                      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    E.F.PAI,A.WITTINGHOFER,W.KABSCH                                       \\nREVDAT   6   29-NOV-17 5P21    1       HELIX                                    \\nREVDAT   5   24-FEB-09 5P21    1       VERSN                                    \\nREVDAT   4   01-APR-03 5P21    1       JRNL                                     \\nREVDAT   3   20-JUL-95 5P21    1       HET                                      \\nREVDAT   2   30-APR-94 5P21    1       FORMUL                                   \\nREVDAT   1   15-JAN-92 5P21    0                                                \\nJRNL        AUTH   E.F.PAI,U.KRENGEL,G.A.PETSKO,R.S.GOODY,W.KABSCH,             \\nJRNL        AUTH 2 A.WITTINGHOFER                                               \\nJRNL        TITL   REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION   \\nJRNL        TITL 2 OF H-RAS P21 AT 1.35 A RESOLUTION: IMPLICATIONS FOR THE      \\nJRNL        TITL 3 MECHANISM OF GTP HYDROLYSIS.                                 \\nJRNL        REF    EMBO J.                       V.   9  2351 1990              \\nJRNL        REFN                   ISSN 0261-4189                               \\nJRNL        PMID   2196171                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,S.C.ALMO,G.RAPP,K.WILSON,K.PETRATOS,A.LENTFER, \\nREMARK   1  AUTH 2 A.WITTINGHOFER,W.KABSCH,E.F.PAI,G.A.PETSKO,R.S.GOODY         \\nREMARK   1  TITL   TIME-RESOLVED X-RAY CRYSTALLOGRAPHIC STUDY OF THE            \\nREMARK   1  TITL 2 CONFORMATIONAL CHANGE IN HA-RAS P21 PROTEIN ON GTP           \\nREMARK   1  TITL 3 HYDROLYSIS                                                   \\nREMARK   1  REF    NATURE                        V. 345   309 1990              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   A.SCHERER,J.JOHN,R.LINKE,R.S.GOODY,A.WITTINGHOFER,E.F.PAI,   \\nREMARK   1  AUTH 2 K.C.HOLMES                                                   \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY ANALYSIS OF THE HUMAN  \\nREMARK   1  TITL 2 C-H-RAS-ONCOGENE PRODUCT P21 COMPLEXED WITH GTP ANALOGUES    \\nREMARK   1  REF    J.MOL.BIOL.                   V. 206   257 1989              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   E.F.PAI,W.KABSCH,U.KRENGEL,K.C.HOLMES,J.JOHN,A.WITTINGHOFER  \\nREMARK   1  TITL   STRUCTURE OF THE GUANINE-NUCLEOTIDE-BINDING DOMAIN OF THE    \\nREMARK   1  TITL 2 HA-RAS ONCOGENE PRODUCT P21 IN THE TRIPHOSPHATE CONFORMATION \\nREMARK   1  REF    NATURE                        V. 341   209 1989              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,G.RAPP,J.JOHN,A.WITTINGHOFER,E.F.PAI,R.S.GOODY \\nREMARK   1  TITL   BIOCHEMICAL AND CRYSTALLOGRAPHIC CHARACTERIZATION OF A       \\nREMARK   1  TITL 2 COMPLEX OF C-HA-RAS P21 AND CAGED GTP WITH FLASH PHOTOLYSIS  \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  86  7687 1989              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 6.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 26806                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.196                           \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1323                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 33                                      \\nREMARK   3   SOLVENT ATOMS            : 211                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.014                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 2.700                           \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  RESIDUES 61 - 65 (GLN - GLU - GLU - TYR - SER) ADOPT                \\nREMARK   3  SEVERAL CONFORMATIONS IN THE CRYSTAL.  THE COORDINATES              \\nREMARK   3  GIVEN APPROXIMATE ONE OF THESE.                                     \\nREMARK   4                                                                      \\nREMARK   4 5P21 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179747.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 38.93                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.01                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 32 2 1                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+2/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+1/3                                           \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z+1/3                                           \\nREMARK 290       6555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000      108.13333            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       54.06667            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       54.06667            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000      108.13333            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025  0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    ARG A  68   NE  -  CZ  -  NH1 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A  97   NE  -  CZ  -  NH1 ANGL. DEV. =  -3.4 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH1 ANGL. DEV. =   4.1 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500    ARG A 128   NE  -  CZ  -  NH2 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A 135   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500    TYR A 157   CB  -  CG  -  CD2 ANGL. DEV. =  -4.8 DEGREES          \\nREMARK 500    ARG A 164   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ASP A  33       75.86   -157.25                                   \\nREMARK 500    ILE A  36      -64.23   -100.39                                   \\nREMARK 500    GLU A  37      123.75   -171.30                                   \\nREMARK 500    SER A  65     -133.70     53.07                                   \\nREMARK 500    LYS A 117       33.80     71.60                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              MG A 168  MG                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 GNP A 167   O2G                                                    \\nREMARK 620 2 GNP A 167   O2B  95.0                                              \\nREMARK 620 3 HOH A 173   O    87.6  91.7                                        \\nREMARK 620 4 SER A  17   OG  172.9  91.0  88.6                                  \\nREMARK 620 5 THR A  35   OG1  91.3 173.4  90.5  82.8                            \\nREMARK 620 6 HOH A 172   O    97.3  88.0 175.0  86.5  89.3                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE MG A 168                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE GNP A 167                 \\nDBREF  5P21 A    1   166  UNP    P01112   RASH_HUMAN       1    166             \\nSEQRES   1 A  166  MET THR GLU TYR LYS LEU VAL VAL VAL GLY ALA GLY GLY          \\nSEQRES   2 A  166  VAL GLY LYS SER ALA LEU THR ILE GLN LEU ILE GLN ASN          \\nSEQRES   3 A  166  HIS PHE VAL ASP GLU TYR ASP PRO THR ILE GLU ASP SER          \\nSEQRES   4 A  166  TYR ARG LYS GLN VAL VAL ILE ASP GLY GLU THR CYS LEU          \\nSEQRES   5 A  166  LEU ASP ILE LEU ASP THR ALA GLY GLN GLU GLU TYR SER          \\nSEQRES   6 A  166  ALA MET ARG ASP GLN TYR MET ARG THR GLY GLU GLY PHE          \\nSEQRES   7 A  166  LEU CYS VAL PHE ALA ILE ASN ASN THR LYS SER PHE GLU          \\nSEQRES   8 A  166  ASP ILE HIS GLN TYR ARG GLU GLN ILE LYS ARG VAL LYS          \\nSEQRES   9 A  166  ASP SER ASP ASP VAL PRO MET VAL LEU VAL GLY ASN LYS          \\nSEQRES  10 A  166  CYS ASP LEU ALA ALA ARG THR VAL GLU SER ARG GLN ALA          \\nSEQRES  11 A  166  GLN ASP LEU ALA ARG SER TYR GLY ILE PRO TYR ILE GLU          \\nSEQRES  12 A  166  THR SER ALA LYS THR ARG GLN GLY VAL GLU ASP ALA PHE          \\nSEQRES  13 A  166  TYR THR LEU VAL ARG GLU ILE ARG GLN HIS                      \\nHET     MG  A 168       1                                                       \\nHET    GNP  A 167      32                                                       \\nHETNAM      MG MAGNESIUM ION                                                    \\nHETNAM     GNP PHOSPHOAMINOPHOSPHONIC ACID-GUANYLATE ESTER                      \\nFORMUL   2   MG    MG 2+                                                        \\nFORMUL   3  GNP    C10 H17 N6 O13 P3                                            \\nFORMUL   4  HOH   *211(H2 O)                                                    \\nHELIX    1  HA LYS A   16  GLN A   25  1                                  10    \\nHELIX    2  HB SER A   65  THR A   74  1RES 65-67 IN 3/10 CONF            10    \\nHELIX    3  HC THR A   87  VAL A  103  1BREAK AT ASP 92                   17    \\nHELIX    4  HD SER A  127  TYR A  137  1                                  11    \\nHELIX    5  HE VAL A  152  ARG A  164  1                                  13    \\nSHEET    1  S1 6 GLU A  37  ILE A  46  0                                        \\nSHEET    2  S1 6 GLU A  49  THR A  58 -1  O  LEU A  53   N  LYS A  42           \\nSHEET    3  S1 6 THR A   2  VAL A   9  1  N  LEU A   6   O  ASP A  54           \\nSHEET    4  S1 6 GLY A  77  ALA A  83  1  N  LEU A  79   O  VAL A   7           \\nSHEET    5  S1 6 MET A 111  ASN A 116  1  N  VAL A 114   O  CYS A  80           \\nSHEET    6  S1 6 TYR A 141  GLU A 143  1  N  ILE A 142   O  LEU A 113           \\nLINK        MG    MG A 168                 O2G GNP A 167     1555   1555  2.20  \\nLINK        MG    MG A 168                 O2B GNP A 167     1555   1555  2.22  \\nLINK        MG    MG A 168                 O   HOH A 173     1555   1555  2.23  \\nLINK        MG    MG A 168                 OG  SER A  17     1555   1555  2.26  \\nLINK        MG    MG A 168                 OG1 THR A  35     1555   1555  2.26  \\nLINK        MG    MG A 168                 O   HOH A 172     1555   1555  2.30  \\nSITE     1 AC1  5 SER A  17  THR A  35  GNP A 167  HOH A 172                    \\nSITE     2 AC1  5 HOH A 173                                                     \\nSITE     1 AC2 28 GLY A  12  GLY A  13  VAL A  14  GLY A  15                    \\nSITE     2 AC2 28 LYS A  16  SER A  17  ALA A  18  PHE A  28                    \\nSITE     3 AC2 28 VAL A  29  ASP A  30  TYR A  32  PRO A  34                    \\nSITE     4 AC2 28 THR A  35  GLY A  60  ASN A 116  LYS A 117                    \\nSITE     5 AC2 28 ASP A 119  SER A 145  ALA A 146  LYS A 147                    \\nSITE     6 AC2 28  MG A 168  HOH A 170  HOH A 172  HOH A 173                    \\nSITE     7 AC2 28 HOH A 175  HOH A 187  HOH A 281  HOH A 366                    \\nCRYST1   40.300   40.300  162.200  90.00  90.00 120.00 P 32 2 1      6          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.024814  0.014326  0.000000        0.00000                         \\nSCALE2      0.000000  0.028653  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.006165        0.00000                         \\nATOM      1  N   MET A   1      -7.186  32.862  -6.632  1.00 13.92           N  \\nATOM      2  CA  MET A   1      -5.921  32.127  -6.684  1.00 17.10           C  \\nATOM      3  C   MET A   1      -5.845  31.045  -5.621  1.00 16.74           C  \\nATOM      4  O   MET A   1      -6.600  31.083  -4.636  1.00 16.60           O  \\nATOM      5  CB  MET A   1      -4.673  33.058  -6.502  1.00 17.00           C  \\nATOM      6  CG  MET A   1      -4.633  33.804  -5.188  1.00 18.08           C  \\nATOM      7  SD  MET A   1      -3.115  34.766  -4.908  1.00 19.73           S  \\nATOM      8  CE  MET A   1      -3.019  35.851  -6.310  1.00 20.41           C  \\nATOM      9  N   THR A   2      -4.910  30.095  -5.820  1.00 14.70           N  \\nATOM     10  CA  THR A   2      -4.661  29.030  -4.859  1.00 16.92           C  \\nATOM     11  C   THR A   2      -4.230  29.655  -3.547  1.00 14.78           C  \\nATOM     12  O   THR A   2      -3.338  30.501  -3.537  1.00 13.93           O  \\nATOM     13  CB  THR A   2      -3.557  28.107  -5.401  1.00 18.03           C  \\nATOM     14  OG1 THR A   2      -4.059  27.565  -6.629  1.00 20.42           O  \\nATOM     15  CG2 THR A   2      -3.193  26.983  -4.443  1.00 18.51           C  \\nATOM     16  N   GLU A   3      -4.911  29.290  -2.472  1.00 14.80           N  \\nATOM     17  CA  GLU A   3      -4.504  29.730  -1.135  1.00 17.71           C  \\nATOM     18  C   GLU A   3      -4.012  28.553  -0.278  1.00 17.33           C  \\nATOM     19  O   GLU A   3      -4.520  27.427  -0.395  1.00 17.54           O  \\nATOM     20  CB  GLU A   3      -5.655  30.402  -0.432  1.00 20.67           C  \\nATOM     21  CG  GLU A   3      -6.213  31.631  -1.186  1.00 25.56           C  \\nATOM     22  CD  GLU A   3      -7.128  32.563  -0.369  1.00 28.46           C  \\nATOM     23  OE1 GLU A   3      -7.578  32.216   0.728  1.00 27.72           O  \\nATOM     24  OE2 GLU A   3      -7.375  33.680  -0.827  1.00 32.49           O  \\nATOM     25  N   TYR A   4      -2.962  28.744   0.542  1.00 14.50           N  \\nATOM     26  CA  TYR A   4      -2.425  27.654   1.358  1.00 12.29           C  \\nATOM     27  C   TYR A   4      -2.534  28.103   2.807  1.00 12.52           C  \\nATOM     28  O   TYR A   4      -2.016  29.143   3.200  1.00 12.11           O  \\nATOM     29  CB  TYR A   4      -0.975  27.385   1.026  1.00 11.46           C  \\nATOM     30  CG  TYR A   4      -0.726  26.846  -0.387  1.00 12.38           C  \\nATOM     31  CD1 TYR A   4      -0.297  27.698  -1.395  1.00 13.00           C  \\nATOM     32  CD2 TYR A   4      -0.933  25.501  -0.649  1.00 14.09           C  \\nATOM     33  CE1 TYR A   4      -0.070  27.208  -2.674  1.00 13.89           C  \\nATOM     34  CE2 TYR A   4      -0.711  24.993  -1.917  1.00 14.65           C  \\nATOM     35  CZ  TYR A   4      -0.279  25.859  -2.917  1.00 14.84           C  \\nATOM     36  OH  TYR A   4      -0.084  25.357  -4.191  1.00 15.07           O  \\nATOM     37  N   LYS A   5      -3.274  27.305   3.568  1.00 13.07           N  \\nATOM     38  CA  LYS A   5      -3.554  27.569   4.970  1.00 12.96           C  \\nATOM     39  C   LYS A   5      -2.556  26.818   5.852  1.00 12.74           C  \\nATOM     40  O   LYS A   5      -2.645  25.611   6.092  1.00 11.58           O  \\nATOM     41  CB  LYS A   5      -4.972  27.131   5.281  1.00 17.65           C  \\nATOM     42  CG  LYS A   5      -6.053  27.833   4.446  1.00 23.49           C  \\nATOM     43  CD  LYS A   5      -7.186  28.310   5.335  1.00 29.10           C  \\nATOM     44  CE  LYS A   5      -6.655  29.309   6.403  1.00 33.68           C  \\nATOM     45  NZ  LYS A   5      -6.597  28.739   7.755  1.00 36.52           N  \\nATOM     46  N   LEU A   6      -1.543  27.574   6.297  1.00 13.08           N  \\nATOM     47  CA  LEU A   6      -0.418  27.011   7.064  1.00 10.34           C  \\nATOM     48  C   LEU A   6      -0.578  27.382   8.535  1.00  9.68           C  \\nATOM     49  O   LEU A   6      -1.034  28.483   8.863  1.00 11.31           O  \\nATOM     50  CB  LEU A   6       0.910  27.571   6.536  1.00 11.34           C  \\nATOM     51  CG  LEU A   6       1.137  27.565   5.017  1.00 11.62           C  \\nATOM     52  CD1 LEU A   6       2.515  28.119   4.720  1.00 13.60           C  \\nATOM     53  CD2 LEU A   6       0.973  26.171   4.475  1.00 12.19           C  \\nATOM     54  N   VAL A   7      -0.196  26.486   9.432  1.00  9.93           N  \\nATOM     55  CA  VAL A   7      -0.333  26.787  10.852  1.00 10.06           C  \\nATOM     56  C   VAL A   7       1.033  26.513  11.495  1.00 10.37           C  \\nATOM     57  O   VAL A   7       1.597  25.441  11.252  1.00  9.51           O  \\nATOM     58  CB  VAL A   7      -1.474  25.884  11.446  1.00 10.97           C  \\nATOM     59  CG1 VAL A   7      -1.610  26.163  12.943  1.00 11.39           C  \\nATOM     60  CG2 VAL A   7      -2.809  26.161  10.777  1.00 10.14           C  \\nATOM     61  N   VAL A   8       1.576  27.455  12.295  1.00  8.35           N  \\nATOM     62  CA  VAL A   8       2.878  27.262  12.944  1.00  6.92           C  \\nATOM     63  C   VAL A   8       2.609  26.896  14.405  1.00  8.84           C  \\nATOM     64  O   VAL A   8       1.930  27.663  15.092  1.00  9.21           O  \\nATOM     65  CB  VAL A   8       3.684  28.565  12.799  1.00  8.08           C  \\nATOM     66  CG1 VAL A   8       5.111  28.324  13.261  1.00  9.52           C  \\nATOM     67  CG2 VAL A   8       3.729  29.011  11.358  1.00  8.43           C  \\nATOM     68  N   VAL A   9       3.024  25.701  14.881  1.00  8.73           N  \\nATOM     69  CA  VAL A   9       2.779  25.234  16.245  1.00  8.41           C  \\nATOM     70  C   VAL A   9       4.075  24.806  16.919  1.00  8.21           C  \\nATOM     71  O   VAL A   9       5.093  24.551  16.287  1.00 10.97           O  \\nATOM     72  CB  VAL A   9       1.824  24.028  16.286  1.00  9.74           C  \\nATOM     73  CG1 VAL A   9       0.545  24.491  15.680  1.00 11.76           C  \\nATOM     74  CG2 VAL A   9       2.349  22.810  15.535  1.00 10.31           C  \\nATOM     75  N   GLY A  10       4.048  24.707  18.238  1.00  7.34           N  \\nATOM     76  CA  GLY A  10       5.231  24.336  19.009  1.00  7.98           C  \\nATOM     77  C   GLY A  10       5.192  25.053  20.355  1.00  8.31           C  \\nATOM     78  O   GLY A  10       4.417  25.994  20.595  1.00  9.89           O  \\nATOM     79  N   ALA A  11       6.099  24.638  21.226  1.00  9.31           N  \\nATOM     80  CA  ALA A  11       6.121  25.135  22.597  1.00  9.50           C  \\nATOM     81  C   ALA A  11       6.440  26.625  22.711  1.00  9.66           C  \\nATOM     82  O   ALA A  11       6.998  27.250  21.806  1.00  9.46           O  \\nATOM     83  CB  ALA A  11       7.145  24.306  23.378  1.00  9.85           C  \\nATOM     84  N   GLY A  12       6.111  27.227  23.847  1.00 10.36           N  \\nATOM     85  CA  GLY A  12       6.406  28.632  24.054  1.00  9.30           C  \\nATOM     86  C   GLY A  12       7.892  28.950  23.964  1.00 10.23           C  \\nATOM     87  O   GLY A  12       8.785  28.249  24.466  1.00 10.57           O  \\nATOM     88  N   GLY A  13       8.159  30.064  23.266  1.00 11.49           N  \\nATOM     89  CA  GLY A  13       9.511  30.602  23.139  1.00 11.29           C  \\nATOM     90  C   GLY A  13      10.436  29.950  22.108  1.00 12.22           C  \\nATOM     91  O   GLY A  13      11.624  30.321  22.060  1.00 13.62           O  \\nATOM     92  N   VAL A  14       9.961  28.991  21.286  1.00  9.23           N  \\nATOM     93  CA  VAL A  14      10.848  28.327  20.317  1.00 10.38           C  \\nATOM     94  C   VAL A  14      11.169  29.168  19.076  1.00 10.47           C  \\nATOM     95  O   VAL A  14      12.108  28.841  18.353  1.00 12.07           O  \\nATOM     96  CB  VAL A  14      10.246  26.970  19.856  1.00  9.98           C  \\nATOM     97  CG1 VAL A  14      10.187  26.074  21.120  1.00  9.93           C  \\nATOM     98  CG2 VAL A  14       8.866  27.127  19.200  1.00  9.52           C  \\nATOM     99  N   GLY A  15      10.423  30.264  18.834  1.00  9.61           N  \\nATOM    100  CA  GLY A  15      10.666  31.160  17.706  1.00  9.70           C  \\nATOM    101  C   GLY A  15       9.615  31.061  16.626  1.00  8.82           C  \\nATOM    102  O   GLY A  15       9.915  31.403  15.491  1.00  8.99           O  \\nATOM    103  N   LYS A  16       8.395  30.591  16.916  1.00  8.18           N  \\nATOM    104  CA  LYS A  16       7.331  30.521  15.913  1.00  8.44           C  \\nATOM    105  C   LYS A  16       7.060  31.899  15.309  1.00  9.01           C  \\nATOM    106  O   LYS A  16       7.020  32.044  14.086  1.00  8.25           O  \\nATOM    107  CB  LYS A  16       6.014  30.000  16.529  1.00  9.27           C  \\nATOM    108  CG  LYS A  16       6.132  28.585  17.086  1.00  9.11           C  \\nATOM    109  CD  LYS A  16       4.820  28.077  17.726  1.00  9.31           C  \\nATOM    110  CE  LYS A  16       4.351  28.934  18.888  1.00 10.05           C  \\nATOM    111  NZ  LYS A  16       5.221  28.829  20.051  1.00  9.64           N  \\nATOM    112  N   SER A  17       6.913  32.942  16.135  1.00  7.74           N  \\nATOM    113  CA  SER A  17       6.661  34.295  15.649  1.00  8.78           C  \\nATOM    114  C   SER A  17       7.854  34.859  14.919  1.00  9.73           C  \\nATOM    115  O   SER A  17       7.653  35.446  13.868  1.00  9.70           O  \\nATOM    116  CB  SER A  17       6.360  35.264  16.769  1.00  8.22           C  \\nATOM    117  OG  SER A  17       5.083  34.853  17.204  1.00  8.79           O  \\nATOM    118  N   ALA A  18       9.072  34.687  15.444  1.00  8.95           N  \\nATOM    119  CA  ALA A  18      10.253  35.199  14.769  1.00  8.95           C  \\nATOM    120  C   ALA A  18      10.438  34.576  13.374  1.00  8.54           C  \\nATOM    121  O   ALA A  18      10.832  35.254  12.430  1.00  8.75           O  \\nATOM    122  CB  ALA A  18      11.521  34.931  15.640  1.00 10.11           C  \\nATOM    123  N   LEU A  19      10.136  33.284  13.176  1.00  8.31           N  \\nATOM    124  CA  LEU A  19      10.225  32.667  11.837  1.00  9.30           C  \\nATOM    125  C   LEU A  19       9.204  33.281  10.884  1.00 10.98           C  \\nATOM    126  O   LEU A  19       9.534  33.652   9.751  1.00 10.47           O  \\nATOM    127  CB  LEU A  19       9.938  31.165  11.885  1.00  9.41           C  \\nATOM    128  CG  LEU A  19      11.035  30.275  12.487  1.00 11.80           C  \\nATOM    129  CD1 LEU A  19      10.491  28.866  12.723  1.00 10.43           C  \\nATOM    130  CD2 LEU A  19      12.256  30.288  11.554  1.00 13.42           C  \\nATOM    131  N   THR A  20       7.950  33.419  11.335  1.00 10.72           N  \\nATOM    132  CA  THR A  20       6.888  34.015  10.529  1.00 11.87           C  \\nATOM    133  C   THR A  20       7.173  35.492  10.161  1.00 12.65           C  \\nATOM    134  O   THR A  20       7.012  35.917   9.009  1.00 12.60           O  \\nATOM    135  CB  THR A  20       5.547  33.888  11.312  1.00 13.43           C  \\nATOM    136  OG1 THR A  20       5.299  32.509  11.544  1.00 12.67           O  \\nATOM    137  CG2 THR A  20       4.346  34.410  10.518  1.00 14.64           C  \\nATOM    138  N   ILE A  21       7.615  36.326  11.098  1.00 10.48           N  \\nATOM    139  CA  ILE A  21       7.945  37.704  10.795  1.00 12.96           C  \\nATOM    140  C   ILE A  21       9.197  37.831   9.927  1.00 14.26           C  \\nATOM    141  O   ILE A  21       9.237  38.691   9.042  1.00 15.83           O  \\nATOM    142  CB  ILE A  21       8.090  38.451  12.121  1.00 12.57           C  \\nATOM    143  CG1 ILE A  21       6.691  38.551  12.737  1.00 13.24           C  \\nATOM    144  CG2 ILE A  21       8.631  39.852  11.910  1.00 13.40           C  \\nATOM    145  CD1 ILE A  21       6.636  39.185  14.129  1.00 16.46           C  \\nATOM    146  N   GLN A  22      10.206  36.979  10.092  1.00 13.58           N  \\nATOM    147  CA  GLN A  22      11.330  36.983   9.179  1.00 14.91           C  \\nATOM    148  C   GLN A  22      10.817  36.729   7.736  1.00 17.29           C  \\nATOM    149  O   GLN A  22      11.120  37.485   6.792  1.00 15.91           O  \\nATOM    150  CB  GLN A  22      12.270  35.930   9.736  1.00 15.90           C  \\nATOM    151  CG  GLN A  22      13.686  35.878   9.230  1.00 19.36           C  \\nATOM    152  CD  GLN A  22      14.372  37.217   9.091  1.00 18.08           C  \\nATOM    153  OE1 GLN A  22      14.160  37.984   8.143  1.00 19.23           O  \\nATOM    154  NE2 GLN A  22      15.224  37.492  10.050  1.00 17.40           N  \\nATOM    155  N   LEU A  23       9.944  35.718   7.523  1.00 15.95           N  \\nATOM    156  CA  LEU A  23       9.381  35.475   6.194  1.00 18.85           C  \\nATOM    157  C   LEU A  23       8.539  36.622   5.636  1.00 18.21           C  \\nATOM    158  O   LEU A  23       8.695  36.958   4.469  1.00 19.03           O  \\nATOM    159  CB  LEU A  23       8.498  34.235   6.196  1.00 18.10           C  \\nATOM    160  CG  LEU A  23       7.916  33.797   4.863  1.00 18.64           C  \\nATOM    161  CD1 LEU A  23       8.921  32.994   4.030  1.00 18.03           C  \\nATOM    162  CD2 LEU A  23       6.680  32.991   5.183  1.00 21.34           C  \\nATOM    163  N   ILE A  24       7.634  37.197   6.428  1.00 19.25           N  \\nATOM    164  CA  ILE A  24       6.682  38.198   5.953  1.00 20.23           C  \\nATOM    165  C   ILE A  24       7.316  39.590   5.922  1.00 21.89           C  \\nATOM    166  O   ILE A  24       7.125  40.318   4.943  1.00 22.93           O  \\nATOM    167  CB  ILE A  24       5.399  38.222   6.881  1.00 21.20           C  \\nATOM    168  CG1 ILE A  24       4.664  36.907   6.821  1.00 21.27           C  \\nATOM    169  CG2 ILE A  24       4.423  39.307   6.446  1.00 20.00           C  \\nATOM    170  CD1 ILE A  24       4.128  36.442   5.475  1.00 23.98           C  \\nATOM    171  N   GLN A  25       8.074  39.998   6.953  1.00 21.07           N  \\nATOM    172  CA  GLN A  25       8.520  41.380   7.066  1.00 22.09           C  \\nATOM    173  C   GLN A  25      10.003  41.559   6.962  1.00 20.75           C  \\nATOM    174  O   GLN A  25      10.495  42.677   7.054  1.00 20.26           O  \\nATOM    175  CB  GLN A  25       8.039  41.968   8.398  1.00 23.96           C  \\nATOM    176  CG  GLN A  25       6.548  42.276   8.381  1.00 28.69           C  \\nATOM    177  CD  GLN A  25       5.931  42.574   9.744  1.00 30.92           C  \\nATOM    178  OE1 GLN A  25       6.537  43.143  10.657  1.00 31.89           O  \\nATOM    179  NE2 GLN A  25       4.664  42.169   9.867  1.00 32.35           N  \\nATOM    180  N   ASN A  26      10.764  40.482   6.795  1.00 22.13           N  \\nATOM    181  CA  ASN A  26      12.198  40.642   6.628  1.00 24.67           C  \\nATOM    182  C   ASN A  26      12.936  41.362   7.734  1.00 23.62           C  \\nATOM    183  O   ASN A  26      13.603  42.372   7.521  1.00 25.10           O  \\nATOM    184  CB  ASN A  26      12.522  41.369   5.315  1.00 27.68           C  \\nATOM    185  CG  ASN A  26      12.942  40.351   4.288  1.00 32.80           C  \\nATOM    186  OD1 ASN A  26      12.135  39.699   3.602  1.00 33.77           O  \\nATOM    187  ND2 ASN A  26      14.268  40.203   4.237  1.00 34.26           N  \\nATOM    188  N   HIS A  27      12.764  40.842   8.945  1.00 19.95           N  \\nATOM    189  CA  HIS A  27      13.593  41.266  10.057  1.00 17.99           C  \\nATOM    190  C   HIS A  27      13.441  40.299  11.215  1.00 16.84           C  \\nATOM    191  O   HIS A  27      12.411  39.623  11.344  1.00 14.72           O  \\nATOM    192  CB  HIS A  27      13.241  42.680  10.551  1.00 19.80           C  \\nATOM    193  CG  HIS A  27      11.926  42.908  11.258  1.00 20.77           C  \\nATOM    194  ND1 HIS A  27      11.936  43.314  12.510  1.00 22.17           N  \\nATOM    195  CD2 HIS A  27      10.674  42.768  10.729  1.00 22.32           C  \\nATOM    196  CE1 HIS A  27      10.657  43.430  12.785  1.00 23.86           C  \\nATOM    197  NE2 HIS A  27       9.905  43.110  11.738  1.00 23.97           N  \\nATOM    198  N   PHE A  28      14.516  40.222  12.012  1.00 15.44           N  \\nATOM    199  CA  PHE A  28      14.528  39.413  13.223  1.00 13.89           C  \\nATOM    200  C   PHE A  28      14.026  40.196  14.440  1.00 15.33           C  \\nATOM    201  O   PHE A  28      14.528  41.255  14.786  1.00 16.80           O  \\nATOM    202  CB  PHE A  28      15.938  38.906  13.470  1.00 12.73           C  \\nATOM    203  CG  PHE A  28      16.083  38.150  14.786  1.00 13.35           C  \\nATOM    204  CD1 PHE A  28      15.354  36.990  15.012  1.00 13.80           C  \\nATOM    205  CD2 PHE A  28      16.974  38.615  15.740  1.00 14.54           C  \\nATOM    206  CE1 PHE A  28      15.534  36.310  16.194  1.00 14.45           C  \\nATOM    207  CE2 PHE A  28      17.147  37.925  16.920  1.00 14.61           C  \\nATOM    208  CZ  PHE A  28      16.420  36.768  17.137  1.00 14.65           C  \\nATOM    209  N   VAL A  29      12.970  39.697  15.089  1.00 17.22           N  \\nATOM    210  CA  VAL A  29      12.467  40.313  16.294  1.00 19.11           C  \\nATOM    211  C   VAL A  29      13.127  39.606  17.476  1.00 19.36           C  \\nATOM    212  O   VAL A  29      13.245  38.376  17.605  1.00 18.14           O  \\nATOM    213  CB  VAL A  29      10.894  40.205  16.412  1.00 22.03           C  \\nATOM    214  CG1 VAL A  29      10.334  40.869  15.143  1.00 19.66           C  \\nATOM    215  CG2 VAL A  29      10.377  38.776  16.631  1.00 23.15           C  \\nATOM    216  N   ASP A  30      13.615  40.484  18.324  1.00 20.14           N  \\nATOM    217  CA  ASP A  30      14.229  40.019  19.518  1.00 24.04           C  \\nATOM    218  C   ASP A  30      13.676  40.746  20.721  1.00 25.93           C  \\nATOM    219  O   ASP A  30      14.244  41.709  21.238  1.00 29.26           O  \\nATOM    220  CB  ASP A  30      15.719  40.203  19.443  1.00 27.10           C  \\nATOM    221  CG  ASP A  30      16.389  39.731  20.732  1.00 32.46           C  \\nATOM    222  OD1 ASP A  30      15.878  38.859  21.464  1.00 34.28           O  \\nATOM    223  OD2 ASP A  30      17.458  40.264  21.002  1.00 36.46           O  \\nATOM    224  N   GLU A  31      12.472  40.347  21.076  1.00 26.14           N  \\nATOM    225  CA  GLU A  31      11.922  40.804  22.318  1.00 28.72           C  \\nATOM    226  C   GLU A  31      10.870  39.836  22.749  1.00 26.20           C  \\nATOM    227  O   GLU A  31      10.466  38.931  22.014  1.00 23.14           O  \\nATOM    228  CB  GLU A  31      11.337  42.210  22.204  1.00 33.25           C  \\nATOM    229  CG  GLU A  31      10.350  42.618  21.152  1.00 40.08           C  \\nATOM    230  CD  GLU A  31      10.572  44.107  20.887  1.00 44.71           C  \\nATOM    231  OE1 GLU A  31      10.930  44.444  19.755  1.00 47.50           O  \\nATOM    232  OE2 GLU A  31      10.420  44.928  21.806  1.00 47.89           O  \\nATOM    233  N   TYR A  32      10.550  40.044  24.028  1.00 22.26           N  \\nATOM    234  CA  TYR A  32       9.597  39.196  24.685  1.00 18.83           C  \\nATOM    235  C   TYR A  32       8.204  39.688  24.317  1.00 17.53           C  \\nATOM    236  O   TYR A  32       7.794  40.792  24.676  1.00 18.45           O  \\nATOM    237  CB  TYR A  32       9.909  39.259  26.199  1.00 16.84           C  \\nATOM    238  CG  TYR A  32       9.076  38.309  27.052  1.00 15.64           C  \\nATOM    239  CD1 TYR A  32       9.598  37.096  27.433  1.00 16.17           C  \\nATOM    240  CD2 TYR A  32       7.814  38.686  27.459  1.00 15.54           C  \\nATOM    241  CE1 TYR A  32       8.845  36.251  28.239  1.00 17.86           C  \\nATOM    242  CE2 TYR A  32       7.051  37.861  28.257  1.00 14.40           C  \\nATOM    243  CZ  TYR A  32       7.572  36.635  28.652  1.00 16.47           C  \\nATOM    244  OH  TYR A  32       6.821  35.789  29.485  1.00 13.59           O  \\nATOM    245  N   ASP A  33       7.461  38.841  23.604  1.00 13.43           N  \\nATOM    246  CA  ASP A  33       6.128  39.200  23.141  1.00 14.00           C  \\nATOM    247  C   ASP A  33       5.262  37.955  22.854  1.00 13.56           C  \\nATOM    248  O   ASP A  33       5.024  37.586  21.681  1.00 12.47           O  \\nATOM    249  CB  ASP A  33       6.271  40.072  21.864  1.00 15.92           C  \\nATOM    250  CG  ASP A  33       4.993  40.654  21.269  1.00 18.19           C  \\nATOM    251  OD1 ASP A  33       3.923  40.631  21.879  1.00 21.46           O  \\nATOM    252  OD2 ASP A  33       5.074  41.164  20.167  1.00 20.34           O  \\nATOM    253  N   PRO A  34       4.752  37.283  23.903  1.00 12.04           N  \\nATOM    254  CA  PRO A  34       3.949  36.070  23.788  1.00 11.09           C  \\nATOM    255  C   PRO A  34       2.720  36.248  22.895  1.00 11.76           C  \\nATOM    256  O   PRO A  34       1.953  37.220  23.006  1.00 12.63           O  \\nATOM    257  CB  PRO A  34       3.590  35.711  25.219  1.00 12.50           C  \\nATOM    258  CG  PRO A  34       4.643  36.418  26.033  1.00 12.14           C  \\nATOM    259  CD  PRO A  34       4.817  37.717  25.311  1.00 12.82           C  \\nATOM    260  N   THR A  35       2.554  35.282  21.972  1.00 11.00           N  \\nATOM    261  CA  THR A  35       1.421  35.320  21.033  1.00 11.54           C  \\nATOM    262  C   THR A  35       0.140  34.746  21.639  1.00 13.51           C  \\nATOM    263  O   THR A  35       0.174  33.820  22.450  1.00 12.45           O  \\nATOM    264  CB  THR A  35       1.730  34.518  19.751  1.00 11.34           C  \\nATOM    265  OG1 THR A  35       2.900  35.134  19.233  1.00 11.20           O  \\nATOM    266  CG2 THR A  35       0.706  34.611  18.637  1.00 13.10           C  \\nATOM    267  N   ILE A  36      -0.996  35.332  21.233  1.00 15.48           N  \\nATOM    268  CA  ILE A  36      -2.308  34.792  21.567  1.00 19.42           C  \\nATOM    269  C   ILE A  36      -2.663  34.079  20.274  1.00 17.36           C  \\nATOM    270  O   ILE A  36      -2.749  32.863  20.256  1.00 18.29           O  \\nATOM    271  CB  ILE A  36      -3.353  35.914  21.879  1.00 21.30           C  \\nATOM    272  CG1 ILE A  36      -2.930  36.823  23.072  1.00 24.32           C  \\nATOM    273  CG2 ILE A  36      -4.670  35.207  22.136  1.00 21.54           C  \\nATOM    274  CD1 ILE A  36      -2.494  36.167  24.413  1.00 24.70           C  \\nATOM    275  N   GLU A  37      -2.818  34.822  19.181  1.00 19.35           N  \\nATOM    276  CA  GLU A  37      -3.059  34.221  17.889  1.00 21.47           C  \\nATOM    277  C   GLU A  37      -2.957  35.362  16.897  1.00 23.77           C  \\nATOM    278  O   GLU A  37      -3.636  36.393  17.031  1.00 25.49           O  \\nATOM    279  CB  GLU A  37      -4.440  33.611  17.880  1.00 25.97           C  \\nATOM    280  CG  GLU A  37      -4.766  32.776  16.667  1.00 33.81           C  \\nATOM    281  CD  GLU A  37      -6.160  32.154  16.777  1.00 38.37           C  \\nATOM    282  OE1 GLU A  37      -7.095  32.664  16.149  1.00 37.86           O  \\nATOM    283  OE2 GLU A  37      -6.298  31.164  17.515  1.00 42.24           O  \\nATOM    284  N   ASP A  38      -2.070  35.212  15.911  1.00 19.96           N  \\nATOM    285  CA  ASP A  38      -1.899  36.178  14.839  1.00 17.94           C  \\nATOM    286  C   ASP A  38      -2.024  35.526  13.473  1.00 18.21           C  \\nATOM    287  O   ASP A  38      -1.582  34.393  13.294  1.00 16.96           O  \\nATOM    288  CB  ASP A  38      -0.540  36.833  14.965  1.00 18.83           C  \\nATOM    289  CG  ASP A  38      -0.507  37.817  16.133  1.00 20.08           C  \\nATOM    290  OD1 ASP A  38      -1.390  38.660  16.208  1.00 25.47           O  \\nATOM    291  OD2 ASP A  38       0.375  37.781  16.967  1.00 20.08           O  \\nATOM    292  N   SER A  39      -2.698  36.170  12.507  1.00 17.82           N  \\nATOM    293  CA  SER A  39      -2.691  35.658  11.133  1.00 18.16           C  \\nATOM    294  C   SER A  39      -2.012  36.627  10.171  1.00 16.95           C  \\nATOM    295  O   SER A  39      -2.053  37.853  10.333  1.00 15.80           O  \\nATOM    296  CB  SER A  39      -4.094  35.402  10.637  1.00 23.01           C  \\nATOM    297  OG  SER A  39      -4.873  36.571  10.754  1.00 27.13           O  \\nATOM    298  N   TYR A  40      -1.336  36.080   9.168  1.00 15.44           N  \\nATOM    299  CA  TYR A  40      -0.607  36.889   8.196  1.00 16.25           C  \\nATOM    300  C   TYR A  40      -1.018  36.383   6.830  1.00 17.06           C  \\nATOM    301  O   TYR A  40      -1.160  35.172   6.638  1.00 18.29           O  \\nATOM    302  CB  TYR A  40       0.909  36.724   8.370  1.00 16.59           C  \\nATOM    303  CG  TYR A  40       1.439  37.242   9.707  1.00 16.10           C  \\nATOM    304  CD1 TYR A  40       1.473  36.412  10.815  1.00 16.77           C  \\nATOM    305  CD2 TYR A  40       1.883  38.549   9.799  1.00 18.02           C  \\nATOM    306  CE1 TYR A  40       1.950  36.893  12.021  1.00 16.86           C  \\nATOM    307  CE2 TYR A  40       2.368  39.032  11.001  1.00 18.70           C  \\nATOM    308  CZ  TYR A  40       2.396  38.199  12.104  1.00 17.62           C  \\nATOM    309  OH  TYR A  40       2.853  38.695  13.304  1.00 18.37           O  \\nATOM    310  N   ARG A  41      -1.270  37.303   5.900  1.00 15.48           N  \\nATOM    311  CA  ARG A  41      -1.660  36.918   4.558  1.00 15.42           C  \\nATOM    312  C   ARG A  41      -0.730  37.606   3.586  1.00 16.76           C  \\nATOM    313  O   ARG A  41      -0.536  38.829   3.633  1.00 17.01           O  \\nATOM    314  CB  ARG A  41      -3.087  37.342   4.277  1.00 17.64           C  \\nATOM    315  CG  ARG A  41      -4.150  36.507   4.947  1.00 18.09           C  \\nATOM    316  CD  ARG A  41      -5.528  37.181   4.849  1.00 19.64           C  \\nATOM    317  NE  ARG A  41      -6.066  37.318   3.489  1.00 20.11           N  \\nATOM    318  CZ  ARG A  41      -6.660  36.329   2.803  1.00 22.17           C  \\nATOM    319  NH1 ARG A  41      -7.219  36.571   1.617  1.00 22.48           N  \\nATOM    320  NH2 ARG A  41      -6.711  35.090   3.274  1.00 23.55           N  \\nATOM    321  N   LYS A  42      -0.135  36.827   2.679  1.00 16.52           N  \\nATOM    322  CA  LYS A  42       0.835  37.405   1.758  1.00 16.59           C  \\nATOM    323  C   LYS A  42       0.740  36.746   0.387  1.00 16.04           C  \\nATOM    324  O   LYS A  42       0.794  35.528   0.307  1.00 14.74           O  \\nATOM    325  CB  LYS A  42       2.208  37.210   2.401  1.00 18.22           C  \\nATOM    326  CG  LYS A  42       3.388  37.784   1.669  1.00 22.68           C  \\nATOM    327  CD  LYS A  42       3.369  39.274   1.792  1.00 28.09           C  \\nATOM    328  CE  LYS A  42       4.695  39.876   1.317  1.00 31.70           C  \\nATOM    329  NZ  LYS A  42       4.828  39.827  -0.128  1.00 35.70           N  \\nATOM    330  N   GLN A  43       0.561  37.510  -0.697  1.00 16.89           N  \\nATOM    331  CA  GLN A  43       0.579  36.950  -2.047  1.00 19.10           C  \\nATOM    332  C   GLN A  43       2.026  36.729  -2.448  1.00 15.08           C  \\nATOM    333  O   GLN A  43       2.828  37.654  -2.362  1.00 15.72           O  \\nATOM    334  CB  GLN A  43      -0.116  37.908  -3.043  1.00 23.32           C  \\nATOM    335  CG  GLN A  43       0.543  38.332  -4.415  1.00 32.83           C  \\nATOM    336  CD  GLN A  43       1.081  37.332  -5.482  1.00 36.11           C  \\nATOM    337  OE1 GLN A  43       0.457  36.335  -5.865  1.00 37.78           O  \\nATOM    338  NE2 GLN A  43       2.271  37.606  -6.053  1.00 35.45           N  \\nATOM    339  N   VAL A  44       2.394  35.532  -2.922  1.00 12.79           N  \\nATOM    340  CA  VAL A  44       3.771  35.242  -3.311  1.00 11.03           C  \\nATOM    341  C   VAL A  44       3.724  34.425  -4.605  1.00 14.28           C  \\nATOM    342  O   VAL A  44       2.633  34.109  -5.090  1.00 14.90           O  \\nATOM    343  CB  VAL A  44       4.508  34.411  -2.223  1.00 13.21           C  \\nATOM    344  CG1 VAL A  44       4.669  35.282  -0.977  1.00 15.04           C  \\nATOM    345  CG2 VAL A  44       3.761  33.116  -1.889  1.00 12.89           C  \\nATOM    346  N   VAL A  45       4.879  34.127  -5.235  1.00 13.20           N  \\nATOM    347  CA  VAL A  45       4.938  33.277  -6.419  1.00 12.28           C  \\nATOM    348  C   VAL A  45       5.836  32.161  -5.970  1.00 11.08           C  \\nATOM    349  O   VAL A  45       6.988  32.447  -5.636  1.00 13.13           O  \\nATOM    350  CB  VAL A  45       5.587  33.979  -7.641  1.00 13.01           C  \\nATOM    351  CG1 VAL A  45       5.549  33.031  -8.810  1.00 14.51           C  \\nATOM    352  CG2 VAL A  45       4.787  35.173  -8.108  1.00 13.54           C  \\nATOM    353  N   ILE A  46       5.374  30.907  -5.915  1.00  9.89           N  \\nATOM    354  CA  ILE A  46       6.274  29.795  -5.491  1.00 11.03           C  \\nATOM    355  C   ILE A  46       6.369  28.800  -6.662  1.00  8.85           C  \\nATOM    356  O   ILE A  46       5.319  28.296  -7.059  1.00 10.25           O  \\nATOM    357  CB  ILE A  46       5.692  29.115  -4.203  1.00 10.64           C  \\nATOM    358  CG1 ILE A  46       5.652  30.096  -3.042  1.00 11.20           C  \\nATOM    359  CG2 ILE A  46       6.599  27.953  -3.757  1.00 12.93           C  \\nATOM    360  CD1 ILE A  46       4.937  29.536  -1.808  1.00  8.90           C  \\nATOM    361  N   ASP A  47       7.565  28.495  -7.224  1.00  9.41           N  \\nATOM    362  CA  ASP A  47       7.767  27.650  -8.426  1.00  9.70           C  \\nATOM    363  C   ASP A  47       6.838  28.031  -9.572  1.00 11.14           C  \\nATOM    364  O   ASP A  47       6.225  27.211 -10.248  1.00 14.09           O  \\nATOM    365  CB  ASP A  47       7.566  26.165  -8.105  1.00 10.73           C  \\nATOM    366  CG  ASP A  47       8.388  25.783  -6.874  1.00 13.52           C  \\nATOM    367  OD1 ASP A  47       7.805  25.291  -5.898  1.00 12.04           O  \\nATOM    368  OD2 ASP A  47       9.598  26.024  -6.872  1.00 11.31           O  \\nATOM    369  N   GLY A  48       6.724  29.353  -9.735  1.00 10.78           N  \\nATOM    370  CA  GLY A  48       5.930  29.961 -10.785  1.00 13.29           C  \\nATOM    371  C   GLY A  48       4.426  29.989 -10.548  1.00 14.28           C  \\nATOM    372  O   GLY A  48       3.677  30.512 -11.386  1.00 15.19           O  \\nATOM    373  N   GLU A  49       3.944  29.422  -9.439  1.00 13.41           N  \\nATOM    374  CA  GLU A  49       2.520  29.457  -9.143  1.00 13.90           C  \\nATOM    375  C   GLU A  49       2.223  30.700  -8.314  1.00 15.34           C  \\nATOM    376  O   GLU A  49       2.860  30.906  -7.275  1.00 14.56           O  \\nATOM    377  CB  GLU A  49       2.124  28.238  -8.340  1.00 15.50           C  \\nATOM    378  CG  GLU A  49       0.630  28.250  -7.990  1.00 18.10           C  \\nATOM    379  CD  GLU A  49       0.204  27.110  -7.100  1.00 19.13           C  \\nATOM    380  OE1 GLU A  49       0.717  27.004  -5.996  1.00 19.53           O  \\nATOM    381  OE2 GLU A  49      -0.630  26.321  -7.529  1.00 22.12           O  \\nATOM    382  N   THR A  50       1.267  31.536  -8.734  1.00 14.06           N  \\nATOM    383  CA  THR A  50       0.901  32.697  -7.927  1.00 16.15           C  \\nATOM    384  C   THR A  50      -0.001  32.169  -6.819  1.00 13.35           C  \\nATOM    385  O   THR A  50      -0.938  31.409  -7.046  1.00 15.13           O  \\nATOM    386  CB  THR A  50       0.203  33.755  -8.846  1.00 17.88           C  \\nATOM    387  OG1 THR A  50      -0.245  34.813  -8.012  1.00 24.33           O  \\nATOM    388  CG2 THR A  50      -1.028  33.236  -9.518  1.00 19.78           C  \\nATOM    389  N   CYS A  51       0.306  32.442  -5.563  1.00 13.28           N  \\nATOM    390  CA  CYS A  51      -0.566  31.959  -4.508  1.00 15.16           C  \\nATOM    391  C   CYS A  51      -0.579  32.892  -3.303  1.00 14.00           C  \\nATOM    392  O   CYS A  51       0.161  33.874  -3.231  1.00 11.81           O  \\nATOM    393  CB  CYS A  51      -0.132  30.537  -4.079  1.00 14.08           C  \\nATOM    394  SG  CYS A  51       1.597  30.409  -3.575  1.00 16.09           S  \\nATOM    395  N   LEU A  52      -1.521  32.609  -2.405  1.00 13.77           N  \\nATOM    396  CA  LEU A  52      -1.573  33.397  -1.199  1.00 16.01           C  \\nATOM    397  C   LEU A  52      -1.260  32.497  -0.025  1.00 14.53           C  \\nATOM    398  O   LEU A  52      -1.868  31.443   0.113  1.00 16.05           O  \\nATOM    399  CB  LEU A  52      -2.957  33.986  -1.014  1.00 19.71           C  \\nATOM    400  CG  LEU A  52      -3.122  34.885   0.214  1.00 20.84           C  \\nATOM    401  CD1 LEU A  52      -3.070  36.329  -0.220  1.00 21.38           C  \\nATOM    402  CD2 LEU A  52      -4.420  34.526   0.923  1.00 20.90           C  \\nATOM    403  N   LEU A  53      -0.319  32.904   0.800  1.00 12.41           N  \\nATOM    404  CA  LEU A  53       0.019  32.232   2.048  1.00 13.70           C  \\nATOM    405  C   LEU A  53      -0.841  32.852   3.148  1.00 13.85           C  \\nATOM    406  O   LEU A  53      -0.733  34.056   3.413  1.00 12.82           O  \\nATOM    407  CB  LEU A  53       1.485  32.464   2.400  1.00 14.51           C  \\nATOM    408  CG  LEU A  53       2.707  31.589   2.031  1.00 19.29           C  \\nATOM    409  CD1 LEU A  53       2.394  30.553   0.962  1.00 18.56           C  \\nATOM    410  CD2 LEU A  53       3.843  32.557   1.741  1.00 20.00           C  \\nATOM    411  N   ASP A  54      -1.699  32.023   3.751  1.00 11.96           N  \\nATOM    412  CA  ASP A  54      -2.476  32.445   4.884  1.00 15.07           C  \\nATOM    413  C   ASP A  54      -1.882  31.675   6.055  1.00 12.67           C  \\nATOM    414  O   ASP A  54      -2.099  30.477   6.234  1.00 13.99           O  \\nATOM    415  CB  ASP A  54      -3.930  32.088   4.644  1.00 19.33           C  \\nATOM    416  CG  ASP A  54      -4.826  32.609   5.746  1.00 24.24           C  \\nATOM    417  OD1 ASP A  54      -4.412  32.652   6.897  1.00 28.17           O  \\nATOM    418  OD2 ASP A  54      -5.958  32.979   5.450  1.00 27.59           O  \\nATOM    419  N   ILE A  55      -1.127  32.389   6.880  1.00 11.78           N  \\nATOM    420  CA  ILE A  55      -0.346  31.769   7.935  1.00 12.15           C  \\nATOM    421  C   ILE A  55      -0.953  32.099   9.296  1.00 13.10           C  \\nATOM    422  O   ILE A  55      -1.195  33.271   9.587  1.00 13.84           O  \\nATOM    423  CB  ILE A  55       1.109  32.282   7.931  1.00 12.42           C  \\nATOM    424  CG1 ILE A  55       1.859  31.979   6.653  1.00 13.99           C  \\nATOM    425  CG2 ILE A  55       1.859  31.505   9.018  1.00 13.24           C  \\nATOM    426  CD1 ILE A  55       3.176  32.768   6.497  1.00 15.68           C  \\nATOM    427  N   LEU A  56      -1.176  31.083  10.134  1.00 11.64           N  \\nATOM    428  CA  LEU A  56      -1.642  31.341  11.490  1.00 11.98           C  \\nATOM    429  C   LEU A  56      -0.524  31.027  12.431  1.00  9.99           C  \\nATOM    430  O   LEU A  56      -0.017  29.895  12.447  1.00  9.72           O  \\nATOM    431  CB  LEU A  56      -2.836  30.463  11.896  1.00 13.80           C  \\nATOM    432  CG  LEU A  56      -3.470  30.741  13.297  1.00 16.13           C  \\nATOM    433  CD1 LEU A  56      -4.347  31.987  13.260  1.00 15.28           C  \\nATOM    434  CD2 LEU A  56      -4.321  29.580  13.692  1.00 16.84           C  \\nATOM    435  N   ASP A  57      -0.188  32.084  13.182  1.00 10.40           N  \\nATOM    436  CA  ASP A  57       0.815  32.014  14.219  1.00 10.30           C  \\nATOM    437  C   ASP A  57       0.127  31.812  15.571  1.00 11.75           C  \\nATOM    438  O   ASP A  57      -0.652  32.631  16.053  1.00 12.91           O  \\nATOM    439  CB  ASP A  57       1.602  33.296  14.173  1.00 10.66           C  \\nATOM    440  CG  ASP A  57       2.778  33.336  15.132  1.00 11.39           C  \\nATOM    441  OD1 ASP A  57       3.306  32.298  15.535  1.00 11.07           O  \\nATOM    442  OD2 ASP A  57       3.168  34.448  15.455  1.00 13.15           O  \\nATOM    443  N   THR A  58       0.447  30.723  16.244  1.00 12.55           N  \\nATOM    444  CA  THR A  58      -0.324  30.325  17.416  1.00 12.98           C  \\nATOM    445  C   THR A  58       0.368  30.541  18.758  1.00 13.48           C  \\nATOM    446  O   THR A  58       1.568  30.812  18.815  1.00 14.00           O  \\nATOM    447  CB  THR A  58      -0.698  28.851  17.219  1.00  9.54           C  \\nATOM    448  OG1 THR A  58       0.491  28.084  17.369  1.00 11.40           O  \\nATOM    449  CG2 THR A  58      -1.313  28.597  15.848  1.00 12.27           C  \\nATOM    450  N   ALA A  59      -0.371  30.382  19.868  1.00 14.38           N  \\nATOM    451  CA  ALA A  59       0.230  30.510  21.194  1.00 14.79           C  \\nATOM    452  C   ALA A  59       0.928  29.210  21.608  1.00 15.41           C  \\nATOM    453  O   ALA A  59       0.448  28.106  21.324  1.00 13.07           O  \\nATOM    454  CB  ALA A  59      -0.845  30.818  22.238  1.00 15.47           C  \\nATOM    455  N   GLY A  60       2.084  29.341  22.271  1.00 17.71           N  \\nATOM    456  CA  GLY A  60       2.807  28.208  22.814  1.00 24.18           C  \\nATOM    457  C   GLY A  60       2.190  27.751  24.130  1.00 31.21           C  \\nATOM    458  O   GLY A  60       2.538  26.688  24.627  1.00 30.44           O  \\nATOM    459  N   GLN A  61       1.260  28.503  24.726  1.00 39.50           N  \\nATOM    460  CA  GLN A  61       0.671  28.099  25.994  1.00 48.84           C  \\nATOM    461  C   GLN A  61      -0.683  27.395  25.985  1.00 51.52           C  \\nATOM    462  O   GLN A  61      -1.606  27.780  25.276  1.00 52.17           O  \\nATOM    463  CB  GLN A  61       0.605  29.327  26.874  1.00 51.56           C  \\nATOM    464  CG  GLN A  61       0.871  28.861  28.312  1.00 56.74           C  \\nATOM    465  CD  GLN A  61       1.506  29.937  29.177  1.00 58.74           C  \\nATOM    466  OE1 GLN A  61       2.647  29.795  29.616  1.00 60.12           O  \\nATOM    467  NE2 GLN A  61       0.783  31.031  29.429  1.00 60.18           N  \\nATOM    468  N   GLU A  62      -0.804  26.349  26.818  1.00 57.40           N  \\nATOM    469  CA  GLU A  62      -2.008  25.527  26.927  1.00 61.96           C  \\nATOM    470  C   GLU A  62      -3.185  26.196  27.631  1.00 62.20           C  \\nATOM    471  O   GLU A  62      -4.138  25.496  27.975  1.00 62.85           O  \\nATOM    472  CB  GLU A  62      -1.629  24.183  27.632  1.00 65.18           C  \\nATOM    473  CG  GLU A  62      -2.622  22.966  27.761  1.00 68.22           C  \\nATOM    474  CD  GLU A  62      -3.032  22.132  26.528  1.00 70.25           C  \\nATOM    475  OE1 GLU A  62      -2.710  22.494  25.395  1.00 71.45           O  \\nATOM    476  OE2 GLU A  62      -3.691  21.101  26.689  1.00 70.00           O  \\nATOM    477  N   GLU A  63      -3.206  27.520  27.904  1.00 62.78           N  \\nATOM    478  CA  GLU A  63      -4.496  28.139  28.254  1.00 63.62           C  \\nATOM    479  C   GLU A  63      -5.457  27.854  27.085  1.00 63.71           C  \\nATOM    480  O   GLU A  63      -6.688  27.814  27.211  1.00 62.97           O  \\nATOM    481  CB  GLU A  63      -4.372  29.666  28.475  1.00 65.13           C  \\nATOM    482  CG  GLU A  63      -3.456  30.553  27.607  1.00 66.91           C  \\nATOM    483  CD  GLU A  63      -4.012  31.235  26.345  1.00 68.05           C  \\nATOM    484  OE1 GLU A  63      -5.220  31.462  26.220  1.00 68.64           O  \\nATOM    485  OE2 GLU A  63      -3.198  31.568  25.481  1.00 67.62           O  \\nATOM    486  N   TYR A  64      -4.801  27.606  25.934  1.00 63.36           N  \\nATOM    487  CA  TYR A  64      -5.420  27.092  24.743  1.00 63.20           C  \\nATOM    488  C   TYR A  64      -5.905  25.653  24.875  1.00 61.76           C  \\nATOM    489  O   TYR A  64      -7.120  25.486  24.858  1.00 63.04           O  \\nATOM    490  CB  TYR A  64      -4.437  27.208  23.613  1.00 64.83           C  \\nATOM    491  CG  TYR A  64      -4.736  28.459  22.818  1.00 66.98           C  \\nATOM    492  CD1 TYR A  64      -4.300  29.686  23.267  1.00 67.55           C  \\nATOM    493  CD2 TYR A  64      -5.459  28.355  21.646  1.00 67.97           C  \\nATOM    494  CE1 TYR A  64      -4.587  30.825  22.536  1.00 69.49           C  \\nATOM    495  CE2 TYR A  64      -5.752  29.488  20.911  1.00 69.29           C  \\nATOM    496  CZ  TYR A  64      -5.313  30.718  21.359  1.00 70.05           C  \\nATOM    497  OH  TYR A  64      -5.589  31.852  20.614  1.00 70.67           O  \\nATOM    498  N   SER A  65      -5.095  24.590  24.997  1.00 58.67           N  \\nATOM    499  CA  SER A  65      -5.617  23.230  25.127  1.00 56.93           C  \\nATOM    500  C   SER A  65      -6.614  22.757  24.063  1.00 55.33           C  \\nATOM    501  O   SER A  65      -6.321  22.936  22.876  1.00 54.74           O  \\nATOM    502  CB  SER A  65      -6.190  23.122  26.552  1.00 57.99           C  \\nATOM    503  OG  SER A  65      -7.251  24.011  26.900  1.00 59.80           O  \\nATOM    504  N   ALA A  66      -7.762  22.119  24.359  1.00 52.33           N  \\nATOM    505  CA  ALA A  66      -8.766  21.824  23.335  1.00 48.80           C  \\nATOM    506  C   ALA A  66      -9.138  23.050  22.492  1.00 45.71           C  \\nATOM    507  O   ALA A  66      -9.751  22.862  21.449  1.00 44.33           O  \\nATOM    508  CB  ALA A  66     -10.043  21.300  23.971  1.00 49.14           C  \\nATOM    509  N   MET A  67      -8.829  24.310  22.875  1.00 42.85           N  \\nATOM    510  CA  MET A  67      -8.984  25.465  21.967  1.00 40.01           C  \\nATOM    511  C   MET A  67      -8.093  25.362  20.719  1.00 36.79           C  \\nATOM    512  O   MET A  67      -8.221  26.184  19.805  1.00 36.08           O  \\nATOM    513  CB  MET A  67      -8.594  26.804  22.581  1.00 42.50           C  \\nATOM    514  CG  MET A  67      -9.359  27.330  23.772  1.00 45.93           C  \\nATOM    515  SD  MET A  67     -11.140  27.396  23.483  1.00 48.74           S  \\nATOM    516  CE  MET A  67     -11.526  25.880  24.321  1.00 47.64           C  \\nATOM    517  N   ARG A  68      -7.128  24.418  20.665  1.00 31.59           N  \\nATOM    518  CA  ARG A  68      -6.322  24.217  19.467  1.00 26.34           C  \\nATOM    519  C   ARG A  68      -7.140  23.473  18.424  1.00 23.68           C  \\nATOM    520  O   ARG A  68      -6.846  23.617  17.231  1.00 21.93           O  \\nATOM    521  CB  ARG A  68      -5.057  23.394  19.745  1.00 25.86           C  \\nATOM    522  CG  ARG A  68      -4.161  23.898  20.869  1.00 29.36           C  \\nATOM    523  CD  ARG A  68      -2.881  23.061  20.998  1.00 32.88           C  \\nATOM    524  NE  ARG A  68      -2.221  22.990  22.315  1.00 36.64           N  \\nATOM    525  CZ  ARG A  68      -1.700  24.031  23.001  1.00 39.66           C  \\nATOM    526  NH1 ARG A  68      -1.903  25.315  22.664  1.00 42.06           N  \\nATOM    527  NH2 ARG A  68      -0.933  23.798  24.071  1.00 39.61           N  \\nATOM    528  N   ASP A  69      -8.188  22.721  18.793  1.00 19.14           N  \\nATOM    529  CA  ASP A  69      -8.880  21.905  17.822  1.00 20.43           C  \\nATOM    530  C   ASP A  69      -9.437  22.623  16.617  1.00 22.16           C  \\nATOM    531  O   ASP A  69      -9.453  22.109  15.488  1.00 19.58           O  \\nATOM    532  CB  ASP A  69     -10.025  21.148  18.478  1.00 20.58           C  \\nATOM    533  CG  ASP A  69      -9.598  19.935  19.303  1.00 22.81           C  \\nATOM    534  OD1 ASP A  69      -8.409  19.708  19.538  1.00 24.59           O  \\nATOM    535  OD2 ASP A  69     -10.475  19.176  19.715  1.00 25.28           O  \\nATOM    536  N   GLN A  70      -9.834  23.874  16.840  1.00 23.26           N  \\nATOM    537  CA  GLN A  70     -10.451  24.609  15.759  1.00 23.53           C  \\nATOM    538  C   GLN A  70      -9.437  24.925  14.667  1.00 21.20           C  \\nATOM    539  O   GLN A  70      -9.653  24.591  13.493  1.00 20.90           O  \\nATOM    540  CB  GLN A  70     -11.071  25.881  16.341  1.00 27.73           C  \\nATOM    541  CG  GLN A  70     -12.498  26.019  15.832  1.00 34.11           C  \\nATOM    542  CD  GLN A  70     -13.103  27.407  16.008  1.00 37.87           C  \\nATOM    543  OE1 GLN A  70     -12.532  28.284  16.672  1.00 39.77           O  \\nATOM    544  NE2 GLN A  70     -14.283  27.620  15.406  1.00 37.89           N  \\nATOM    545  N   TYR A  71      -8.292  25.515  15.050  1.00 19.09           N  \\nATOM    546  CA  TYR A  71      -7.329  25.873  14.030  1.00 17.69           C  \\nATOM    547  C   TYR A  71      -6.648  24.658  13.416  1.00 17.24           C  \\nATOM    548  O   TYR A  71      -6.247  24.675  12.245  1.00 17.25           O  \\nATOM    549  CB  TYR A  71      -6.302  26.868  14.625  1.00 17.87           C  \\nATOM    550  CG  TYR A  71      -5.311  26.480  15.725  1.00 18.90           C  \\nATOM    551  CD1 TYR A  71      -4.355  25.510  15.492  1.00 19.74           C  \\nATOM    552  CD2 TYR A  71      -5.328  27.162  16.940  1.00 19.21           C  \\nATOM    553  CE1 TYR A  71      -3.402  25.212  16.450  1.00 20.38           C  \\nATOM    554  CE2 TYR A  71      -4.375  26.868  17.901  1.00 21.93           C  \\nATOM    555  CZ  TYR A  71      -3.422  25.897  17.645  1.00 21.91           C  \\nATOM    556  OH  TYR A  71      -2.463  25.590  18.588  1.00 26.44           O  \\nATOM    557  N   MET A  72      -6.557  23.559  14.166  1.00 14.81           N  \\nATOM    558  CA  MET A  72      -5.958  22.393  13.563  1.00 18.59           C  \\nATOM    559  C   MET A  72      -6.849  21.778  12.472  1.00 19.38           C  \\nATOM    560  O   MET A  72      -6.385  21.198  11.492  1.00 17.67           O  \\nATOM    561  CB  MET A  72      -5.658  21.386  14.669  1.00 17.39           C  \\nATOM    562  CG  MET A  72      -4.479  21.796  15.514  1.00 16.35           C  \\nATOM    563  SD  MET A  72      -4.146  20.600  16.820  1.00 17.04           S  \\nATOM    564  CE  MET A  72      -2.635  21.259  17.460  1.00 17.72           C  \\nATOM    565  N   ARG A  73      -8.174  21.931  12.626  1.00 20.31           N  \\nATOM    566  CA  ARG A  73      -9.120  21.350  11.701  1.00 22.24           C  \\nATOM    567  C   ARG A  73      -9.087  22.084  10.370  1.00 20.94           C  \\nATOM    568  O   ARG A  73      -9.183  21.450   9.326  1.00 20.81           O  \\nATOM    569  CB  ARG A  73     -10.499  21.414  12.369  1.00 25.29           C  \\nATOM    570  CG  ARG A  73     -11.553  20.499  11.767  1.00 30.04           C  \\nATOM    571  CD  ARG A  73     -12.775  20.429  12.687  1.00 31.86           C  \\nATOM    572  NE  ARG A  73     -12.483  19.702  13.917  1.00 34.70           N  \\nATOM    573  CZ  ARG A  73     -12.894  20.122  15.130  1.00 36.49           C  \\nATOM    574  NH1 ARG A  73     -12.579  19.402  16.196  1.00 36.93           N  \\nATOM    575  NH2 ARG A  73     -13.609  21.239  15.311  1.00 38.17           N  \\nATOM    576  N   THR A  74      -8.868  23.401  10.413  1.00 21.33           N  \\nATOM    577  CA  THR A  74      -8.813  24.240   9.219  1.00 24.99           C  \\nATOM    578  C   THR A  74      -7.436  24.351   8.544  1.00 23.10           C  \\nATOM    579  O   THR A  74      -7.367  24.738   7.371  1.00 22.59           O  \\nATOM    580  CB  THR A  74      -9.289  25.676   9.538  1.00 25.73           C  \\nATOM    581  OG1 THR A  74      -8.351  26.302  10.409  1.00 31.08           O  \\nATOM    582  CG2 THR A  74     -10.657  25.644  10.201  1.00 27.58           C  \\nATOM    583  N   GLY A  75      -6.345  24.084   9.289  1.00 19.58           N  \\nATOM    584  CA  GLY A  75      -5.013  24.114   8.730  1.00 15.79           C  \\nATOM    585  C   GLY A  75      -4.823  23.023   7.713  1.00 13.66           C  \\nATOM    586  O   GLY A  75      -5.245  21.880   7.884  1.00 14.69           O  \\nATOM    587  N   GLU A  76      -4.228  23.360   6.573  1.00 13.14           N  \\nATOM    588  CA  GLU A  76      -3.914  22.363   5.546  1.00 12.99           C  \\nATOM    589  C   GLU A  76      -2.514  21.740   5.753  1.00 12.22           C  \\nATOM    590  O   GLU A  76      -2.293  20.576   5.426  1.00 13.59           O  \\nATOM    591  CB  GLU A  76      -3.999  23.014   4.145  1.00 13.07           C  \\nATOM    592  CG  GLU A  76      -5.387  23.377   3.676  1.00 14.35           C  \\nATOM    593  CD  GLU A  76      -5.261  24.069   2.323  1.00 20.40           C  \\nATOM    594  OE1 GLU A  76      -5.461  23.403   1.288  1.00 24.32           O  \\nATOM    595  OE2 GLU A  76      -4.913  25.250   2.284  1.00 18.82           O  \\nATOM    596  N   GLY A  77      -1.553  22.503   6.301  1.00 10.73           N  \\nATOM    597  CA  GLY A  77      -0.216  22.018   6.570  1.00  8.07           C  \\nATOM    598  C   GLY A  77       0.302  22.689   7.819  1.00  8.63           C  \\nATOM    599  O   GLY A  77      -0.066  23.823   8.127  1.00  8.55           O  \\nATOM    600  N   PHE A  78       1.248  22.013   8.479  1.00  9.15           N  \\nATOM    601  CA  PHE A  78       1.754  22.468   9.773  1.00  8.87           C  \\nATOM    602  C   PHE A  78       3.268  22.520   9.854  1.00  9.51           C  \\nATOM    603  O   PHE A  78       3.930  21.542   9.486  1.00  9.15           O  \\nATOM    604  CB  PHE A  78       1.247  21.532  10.904  1.00  9.44           C  \\nATOM    605  CG  PHE A  78      -0.267  21.472  11.000  1.00  9.66           C  \\nATOM    606  CD1 PHE A  78      -0.906  22.310  11.901  1.00 10.71           C  \\nATOM    607  CD2 PHE A  78      -0.994  20.645  10.152  1.00 10.20           C  \\nATOM    608  CE1 PHE A  78      -2.295  22.338  11.940  1.00 10.61           C  \\nATOM    609  CE2 PHE A  78      -2.378  20.678  10.192  1.00 12.55           C  \\nATOM    610  CZ  PHE A  78      -3.022  21.529  11.081  1.00 12.80           C  \\nATOM    611  N   LEU A  79       3.813  23.636  10.354  1.00  8.72           N  \\nATOM    612  CA  LEU A  79       5.230  23.626  10.699  1.00  9.93           C  \\nATOM    613  C   LEU A  79       5.279  23.281  12.181  1.00  9.09           C  \\nATOM    614  O   LEU A  79       4.701  24.008  12.988  1.00 11.58           O  \\nATOM    615  CB  LEU A  79       5.921  24.970  10.567  1.00 12.80           C  \\nATOM    616  CG  LEU A  79       6.155  25.659   9.249  1.00 16.39           C  \\nATOM    617  CD1 LEU A  79       7.007  26.911   9.544  1.00 17.02           C  \\nATOM    618  CD2 LEU A  79       6.904  24.756   8.269  1.00 16.98           C  \\nATOM    619  N   CYS A  80       5.950  22.219  12.579  1.00  9.14           N  \\nATOM    620  CA  CYS A  80       6.074  21.821  13.981  1.00 11.81           C  \\nATOM    621  C   CYS A  80       7.448  22.237  14.434  1.00 11.53           C  \\nATOM    622  O   CYS A  80       8.435  21.640  14.000  1.00 12.21           O  \\nATOM    623  CB  CYS A  80       5.903  20.318  14.106  1.00 13.73           C  \\nATOM    624  SG  CYS A  80       4.158  19.900  13.837  1.00 18.26           S  \\nATOM    625  N   VAL A  81       7.486  23.282  15.269  1.00  7.57           N  \\nATOM    626  CA  VAL A  81       8.730  23.956  15.604  1.00  8.45           C  \\nATOM    627  C   VAL A  81       9.248  23.620  16.996  1.00  8.90           C  \\nATOM    628  O   VAL A  81       8.451  23.680  17.939  1.00  8.49           O  \\nATOM    629  CB  VAL A  81       8.544  25.516  15.507  1.00  7.40           C  \\nATOM    630  CG1 VAL A  81       9.917  26.196  15.588  1.00  9.42           C  \\nATOM    631  CG2 VAL A  81       8.004  25.954  14.167  1.00  8.59           C  \\nATOM    632  N   PHE A  82      10.546  23.237  17.119  1.00  9.49           N  \\nATOM    633  CA  PHE A  82      11.195  23.202  18.442  1.00  9.09           C  \\nATOM    634  C   PHE A  82      12.511  23.994  18.350  1.00  9.83           C  \\nATOM    635  O   PHE A  82      12.914  24.440  17.266  1.00  9.02           O  \\nATOM    636  CB  PHE A  82      11.482  21.755  18.902  1.00  9.76           C  \\nATOM    637  CG  PHE A  82      12.536  21.062  18.071  1.00 10.09           C  \\nATOM    638  CD1 PHE A  82      13.867  21.099  18.486  1.00 10.70           C  \\nATOM    639  CD2 PHE A  82      12.192  20.412  16.889  1.00 10.93           C  \\nATOM    640  CE1 PHE A  82      14.838  20.483  17.699  1.00 11.12           C  \\nATOM    641  CE2 PHE A  82      13.181  19.807  16.125  1.00 10.73           C  \\nATOM    642  CZ  PHE A  82      14.505  19.839  16.523  1.00 10.35           C  \\nATOM    643  N   ALA A  83      13.128  24.269  19.516  1.00  9.75           N  \\nATOM    644  CA  ALA A  83      14.407  24.963  19.569  1.00 10.32           C  \\nATOM    645  C   ALA A  83      15.509  23.960  19.916  1.00  9.48           C  \\nATOM    646  O   ALA A  83      15.362  23.142  20.839  1.00  9.61           O  \\nATOM    647  CB  ALA A  83      14.354  26.083  20.620  1.00  9.75           C  \\nATOM    648  N   ILE A  84      16.624  24.042  19.160  1.00 11.41           N  \\nATOM    649  CA  ILE A  84      17.714  23.070  19.299  1.00 13.15           C  \\nATOM    650  C   ILE A  84      18.446  23.223  20.623  1.00 14.57           C  \\nATOM    651  O   ILE A  84      19.234  22.347  20.971  1.00 13.94           O  \\nATOM    652  CB  ILE A  84      18.775  23.151  18.160  1.00 13.29           C  \\nATOM    653  CG1 ILE A  84      19.403  24.509  17.878  1.00 14.90           C  \\nATOM    654  CG2 ILE A  84      18.018  22.701  16.933  1.00 13.09           C  \\nATOM    655  CD1 ILE A  84      20.578  24.957  18.763  1.00 17.59           C  \\nATOM    656  N   ASN A  85      18.200  24.290  21.390  1.00 13.61           N  \\nATOM    657  CA  ASN A  85      18.794  24.394  22.720  1.00 15.52           C  \\nATOM    658  C   ASN A  85      17.758  24.156  23.828  1.00 15.49           C  \\nATOM    659  O   ASN A  85      17.988  24.519  24.999  1.00 16.13           O  \\nATOM    660  CB  ASN A  85      19.446  25.788  22.885  1.00 15.16           C  \\nATOM    661  CG  ASN A  85      18.464  26.946  22.904  1.00 16.42           C  \\nATOM    662  OD1 ASN A  85      17.355  26.878  22.362  1.00 16.01           O  \\nATOM    663  ND2 ASN A  85      18.808  28.030  23.587  1.00 16.83           N  \\nATOM    664  N   ASN A  86      16.616  23.515  23.509  1.00 14.72           N  \\nATOM    665  CA  ASN A  86      15.553  23.303  24.483  1.00 13.01           C  \\nATOM    666  C   ASN A  86      14.987  21.901  24.309  1.00 13.17           C  \\nATOM    667  O   ASN A  86      14.080  21.668  23.520  1.00 13.04           O  \\nATOM    668  CB  ASN A  86      14.478  24.367  24.256  1.00 16.51           C  \\nATOM    669  CG  ASN A  86      13.267  24.415  25.187  1.00 20.74           C  \\nATOM    670  OD1 ASN A  86      12.370  25.241  25.018  1.00 25.41           O  \\nATOM    671  ND2 ASN A  86      13.123  23.596  26.206  1.00 19.96           N  \\nATOM    672  N   THR A  87      15.488  20.950  25.103  1.00 13.80           N  \\nATOM    673  CA  THR A  87      15.095  19.553  25.026  1.00 15.40           C  \\nATOM    674  C   THR A  87      13.589  19.316  25.266  1.00 14.40           C  \\nATOM    675  O   THR A  87      12.953  18.535  24.546  1.00 14.11           O  \\nATOM    676  CB  THR A  87      15.983  18.791  26.046  1.00 16.19           C  \\nATOM    677  OG1 THR A  87      17.325  18.908  25.571  1.00 18.22           O  \\nATOM    678  CG2 THR A  87      15.626  17.327  26.180  1.00 17.82           C  \\nATOM    679  N   LYS A  88      12.983  19.958  26.272  1.00 14.42           N  \\nATOM    680  CA  LYS A  88      11.552  19.842  26.535  1.00 15.84           C  \\nATOM    681  C   LYS A  88      10.726  20.232  25.278  1.00 15.06           C  \\nATOM    682  O   LYS A  88       9.722  19.582  25.003  1.00 14.61           O  \\nATOM    683  CB  LYS A  88      11.207  20.766  27.717  1.00 19.53           C  \\nATOM    684  CG  LYS A  88       9.937  20.401  28.457  1.00 27.71           C  \\nATOM    685  CD  LYS A  88       9.585  21.452  29.529  1.00 33.26           C  \\nATOM    686  CE  LYS A  88       9.061  22.762  28.877  1.00 36.79           C  \\nATOM    687  NZ  LYS A  88       8.642  23.763  29.852  1.00 37.29           N  \\nATOM    688  N   SER A  89      11.119  21.259  24.475  1.00 12.36           N  \\nATOM    689  CA  SER A  89      10.325  21.666  23.309  1.00 11.31           C  \\nATOM    690  C   SER A  89      10.371  20.572  22.268  1.00 12.51           C  \\nATOM    691  O   SER A  89       9.397  20.388  21.547  1.00 12.88           O  \\nATOM    692  CB  SER A  89      10.844  22.951  22.674  1.00 11.64           C  \\nATOM    693  OG  SER A  89      12.160  22.900  22.124  1.00 11.11           O  \\nATOM    694  N   PHE A  90      11.504  19.842  22.204  1.00 11.27           N  \\nATOM    695  CA  PHE A  90      11.680  18.751  21.262  1.00 12.91           C  \\nATOM    696  C   PHE A  90      10.760  17.601  21.667  1.00 14.37           C  \\nATOM    697  O   PHE A  90      10.034  17.057  20.839  1.00 13.24           O  \\nATOM    698  CB  PHE A  90      13.161  18.312  21.260  1.00 12.34           C  \\nATOM    699  CG  PHE A  90      13.411  17.120  20.351  1.00 14.41           C  \\nATOM    700  CD1 PHE A  90      13.890  15.948  20.903  1.00 15.10           C  \\nATOM    701  CD2 PHE A  90      13.136  17.206  18.990  1.00 14.81           C  \\nATOM    702  CE1 PHE A  90      14.093  14.857  20.088  1.00 16.20           C  \\nATOM    703  CE2 PHE A  90      13.340  16.110  18.179  1.00 14.81           C  \\nATOM    704  CZ  PHE A  90      13.818  14.935  18.731  1.00 15.39           C  \\nATOM    705  N   GLU A  91      10.725  17.231  22.949  1.00 14.40           N  \\nATOM    706  CA  GLU A  91       9.863  16.146  23.407  1.00 15.78           C  \\nATOM    707  C   GLU A  91       8.374  16.466  23.301  1.00 16.18           C  \\nATOM    708  O   GLU A  91       7.533  15.582  23.114  1.00 17.50           O  \\nATOM    709  CB  GLU A  91      10.226  15.846  24.835  1.00 19.84           C  \\nATOM    710  CG  GLU A  91      11.654  15.386  25.074  1.00 24.49           C  \\nATOM    711  CD  GLU A  91      12.090  15.310  26.537  1.00 27.97           C  \\nATOM    712  OE1 GLU A  91      13.199  14.842  26.766  1.00 30.72           O  \\nATOM    713  OE2 GLU A  91      11.371  15.730  27.449  1.00 30.38           O  \\nATOM    714  N   ASP A  92       8.019  17.747  23.433  1.00 16.19           N  \\nATOM    715  CA  ASP A  92       6.650  18.216  23.265  1.00 15.23           C  \\nATOM    716  C   ASP A  92       6.141  18.053  21.839  1.00 14.29           C  \\nATOM    717  O   ASP A  92       4.927  18.074  21.633  1.00 14.78           O  \\nATOM    718  CB  ASP A  92       6.536  19.708  23.646  1.00 17.85           C  \\nATOM    719  CG  ASP A  92       6.706  20.080  25.127  1.00 21.39           C  \\nATOM    720  OD1 ASP A  92       6.648  19.209  25.997  1.00 20.06           O  \\nATOM    721  OD2 ASP A  92       6.887  21.272  25.401  1.00 24.39           O  \\nATOM    722  N   ILE A  93       7.013  17.910  20.826  1.00 11.76           N  \\nATOM    723  CA  ILE A  93       6.559  17.749  19.452  1.00 14.77           C  \\nATOM    724  C   ILE A  93       5.574  16.583  19.354  1.00 14.03           C  \\nATOM    725  O   ILE A  93       4.576  16.695  18.644  1.00 14.93           O  \\nATOM    726  CB  ILE A  93       7.790  17.499  18.485  1.00 14.96           C  \\nATOM    727  CG1 ILE A  93       8.506  18.830  18.293  1.00 15.41           C  \\nATOM    728  CG2 ILE A  93       7.369  16.936  17.091  1.00 14.61           C  \\nATOM    729  CD1 ILE A  93       7.654  19.940  17.637  1.00 20.05           C  \\nATOM    730  N   HIS A  94       5.806  15.471  20.049  1.00 15.23           N  \\nATOM    731  CA  HIS A  94       4.940  14.302  19.956  1.00 18.20           C  \\nATOM    732  C   HIS A  94       3.494  14.635  20.287  1.00 16.36           C  \\nATOM    733  O   HIS A  94       2.576  14.183  19.598  1.00 17.20           O  \\nATOM    734  CB  HIS A  94       5.436  13.194  20.900  1.00 23.57           C  \\nATOM    735  CG  HIS A  94       4.685  11.874  20.678  1.00 29.09           C  \\nATOM    736  ND1 HIS A  94       4.923  11.118  19.602  1.00 31.26           N  \\nATOM    737  CD2 HIS A  94       3.705  11.361  21.498  1.00 30.93           C  \\nATOM    738  CE1 HIS A  94       4.078  10.110  19.746  1.00 31.74           C  \\nATOM    739  NE2 HIS A  94       3.366  10.260  20.861  1.00 32.20           N  \\nATOM    740  N   GLN A  95       3.271  15.499  21.281  1.00 16.91           N  \\nATOM    741  CA  GLN A  95       1.916  15.878  21.673  1.00 18.66           C  \\nATOM    742  C   GLN A  95       1.226  16.736  20.625  1.00 16.97           C  \\nATOM    743  O   GLN A  95       0.012  16.562  20.443  1.00 15.36           O  \\nATOM    744  CB  GLN A  95       1.928  16.626  23.018  1.00 21.56           C  \\nATOM    745  CG  GLN A  95       2.457  15.766  24.180  1.00 29.81           C  \\nATOM    746  CD  GLN A  95       3.370  16.540  25.150  1.00 35.29           C  \\nATOM    747  OE1 GLN A  95       3.037  17.647  25.593  1.00 37.94           O  \\nATOM    748  NE2 GLN A  95       4.546  16.006  25.521  1.00 37.04           N  \\nATOM    749  N   TYR A  96       1.930  17.653  19.921  1.00 16.06           N  \\nATOM    750  CA  TYR A  96       1.276  18.417  18.826  1.00 16.07           C  \\nATOM    751  C   TYR A  96       0.963  17.504  17.658  1.00 14.44           C  \\nATOM    752  O   TYR A  96      -0.138  17.565  17.118  1.00 15.79           O  \\nATOM    753  CB  TYR A  96       2.139  19.554  18.264  1.00 17.94           C  \\nATOM    754  CG  TYR A  96       2.287  20.677  19.277  1.00 19.19           C  \\nATOM    755  CD1 TYR A  96       1.368  21.699  19.338  1.00 19.33           C  \\nATOM    756  CD2 TYR A  96       3.348  20.631  20.161  1.00 19.61           C  \\nATOM    757  CE1 TYR A  96       1.526  22.678  20.308  1.00 21.72           C  \\nATOM    758  CE2 TYR A  96       3.512  21.595  21.129  1.00 21.72           C  \\nATOM    759  CZ  TYR A  96       2.594  22.613  21.190  1.00 21.26           C  \\nATOM    760  OH  TYR A  96       2.776  23.582  22.153  1.00 24.09           O  \\nATOM    761  N   ARG A  97       1.904  16.635  17.295  1.00 13.30           N  \\nATOM    762  CA  ARG A  97       1.699  15.650  16.247  1.00 15.46           C  \\nATOM    763  C   ARG A  97       0.457  14.794  16.492  1.00 14.26           C  \\nATOM    764  O   ARG A  97      -0.420  14.676  15.641  1.00 14.10           O  \\nATOM    765  CB  ARG A  97       2.939  14.796  16.196  1.00 17.97           C  \\nATOM    766  CG  ARG A  97       2.886  13.541  15.357  1.00 23.35           C  \\nATOM    767  CD  ARG A  97       3.333  13.765  13.943  1.00 23.59           C  \\nATOM    768  NE  ARG A  97       3.314  12.471  13.314  1.00 24.08           N  \\nATOM    769  CZ  ARG A  97       3.268  12.320  12.007  1.00 25.74           C  \\nATOM    770  NH1 ARG A  97       3.135  11.073  11.548  1.00 26.76           N  \\nATOM    771  NH2 ARG A  97       3.353  13.380  11.181  1.00 26.95           N  \\nATOM    772  N   GLU A  98       0.332  14.218  17.678  1.00 13.69           N  \\nATOM    773  CA  GLU A  98      -0.807  13.361  17.941  1.00 16.64           C  \\nATOM    774  C   GLU A  98      -2.118  14.108  17.983  1.00 15.57           C  \\nATOM    775  O   GLU A  98      -3.157  13.573  17.607  1.00 15.85           O  \\nATOM    776  CB  GLU A  98      -0.593  12.623  19.260  1.00 21.96           C  \\nATOM    777  CG  GLU A  98       0.417  11.455  19.177  1.00 28.02           C  \\nATOM    778  CD  GLU A  98       0.147  10.460  18.038  1.00 31.04           C  \\nATOM    779  OE1 GLU A  98       1.118  10.123  17.373  1.00 33.58           O  \\nATOM    780  OE2 GLU A  98      -0.992  10.025  17.790  1.00 33.61           O  \\nATOM    781  N   GLN A  99      -2.105  15.362  18.446  1.00 14.66           N  \\nATOM    782  CA  GLN A  99      -3.343  16.104  18.468  1.00 13.22           C  \\nATOM    783  C   GLN A  99      -3.779  16.398  17.049  1.00 12.15           C  \\nATOM    784  O   GLN A  99      -4.974  16.274  16.791  1.00 14.09           O  \\nATOM    785  CB  GLN A  99      -3.158  17.403  19.250  1.00 16.29           C  \\nATOM    786  CG  GLN A  99      -4.513  18.118  19.407  1.00 19.67           C  \\nATOM    787  CD  GLN A  99      -4.517  19.240  20.438  1.00 22.19           C  \\nATOM    788  OE1 GLN A  99      -3.537  19.485  21.138  1.00 23.70           O  \\nATOM    789  NE2 GLN A  99      -5.628  19.961  20.537  1.00 22.72           N  \\nATOM    790  N   ILE A 100      -2.861  16.795  16.142  1.00 11.42           N  \\nATOM    791  CA  ILE A 100      -3.197  17.121  14.743  1.00 10.81           C  \\nATOM    792  C   ILE A 100      -3.761  15.893  14.008  1.00 12.39           C  \\nATOM    793  O   ILE A 100      -4.733  15.967  13.243  1.00 12.70           O  \\nATOM    794  CB  ILE A 100      -1.924  17.659  14.038  1.00 10.32           C  \\nATOM    795  CG1 ILE A 100      -1.642  19.075  14.518  1.00 10.37           C  \\nATOM    796  CG2 ILE A 100      -2.131  17.685  12.512  1.00 11.36           C  \\nATOM    797  CD1 ILE A 100      -0.184  19.481  14.392  1.00 10.69           C  \\nATOM    798  N   LYS A 101      -3.143  14.732  14.255  1.00 14.08           N  \\nATOM    799  CA  LYS A 101      -3.613  13.487  13.637  1.00 15.95           C  \\nATOM    800  C   LYS A 101      -5.021  13.150  14.079  1.00 15.72           C  \\nATOM    801  O   LYS A 101      -5.848  12.770  13.262  1.00 14.25           O  \\nATOM    802  CB  LYS A 101      -2.681  12.317  13.996  1.00 17.44           C  \\nATOM    803  CG  LYS A 101      -1.414  12.284  13.115  1.00 19.24           C  \\nATOM    804  CD  LYS A 101      -0.332  11.418  13.764  1.00 23.28           C  \\nATOM    805  CE  LYS A 101      -0.752  10.000  14.188  1.00 23.82           C  \\nATOM    806  NZ  LYS A 101       0.400   9.230  14.662  1.00 24.00           N  \\nATOM    807  N   ARG A 102      -5.278  13.341  15.377  1.00 16.72           N  \\nATOM    808  CA  ARG A 102      -6.586  13.098  15.962  1.00 18.74           C  \\nATOM    809  C   ARG A 102      -7.643  14.087  15.440  1.00 19.07           C  \\nATOM    810  O   ARG A 102      -8.712  13.628  15.002  1.00 20.67           O  \\nATOM    811  CB  ARG A 102      -6.457  13.197  17.462  1.00 20.62           C  \\nATOM    812  CG  ARG A 102      -7.539  12.409  18.132  1.00 26.89           C  \\nATOM    813  CD  ARG A 102      -7.610  12.887  19.585  1.00 30.60           C  \\nATOM    814  NE  ARG A 102      -8.197  14.224  19.645  1.00 35.83           N  \\nATOM    815  CZ  ARG A 102      -7.633  15.255  20.301  1.00 37.61           C  \\nATOM    816  NH1 ARG A 102      -8.243  16.455  20.267  1.00 38.37           N  \\nATOM    817  NH2 ARG A 102      -6.481  15.110  20.983  1.00 38.39           N  \\nATOM    818  N   VAL A 103      -7.418  15.407  15.395  1.00 15.59           N  \\nATOM    819  CA  VAL A 103      -8.452  16.326  14.881  1.00 18.48           C  \\nATOM    820  C   VAL A 103      -8.665  16.200  13.349  1.00 16.72           C  \\nATOM    821  O   VAL A 103      -9.776  16.383  12.848  1.00 15.90           O  \\nATOM    822  CB  VAL A 103      -8.131  17.840  15.215  1.00 20.10           C  \\nATOM    823  CG1 VAL A 103      -7.896  18.005  16.704  1.00 21.34           C  \\nATOM    824  CG2 VAL A 103      -6.899  18.302  14.512  1.00 23.93           C  \\nATOM    825  N   LYS A 104      -7.622  15.902  12.574  1.00 13.84           N  \\nATOM    826  CA  LYS A 104      -7.804  15.755  11.140  1.00 16.11           C  \\nATOM    827  C   LYS A 104      -8.158  14.329  10.716  1.00 16.70           C  \\nATOM    828  O   LYS A 104      -8.260  14.079   9.508  1.00 17.27           O  \\nATOM    829  CB  LYS A 104      -6.525  16.215  10.404  1.00 18.12           C  \\nATOM    830  CG  LYS A 104      -6.197  17.678  10.744  1.00 19.53           C  \\nATOM    831  CD  LYS A 104      -5.250  18.297   9.736  1.00 21.28           C  \\nATOM    832  CE  LYS A 104      -5.921  18.590   8.390  1.00 20.75           C  \\nATOM    833  NZ  LYS A 104      -6.875  19.673   8.462  1.00 19.79           N  \\nATOM    834  N   ASP A 105      -8.372  13.390  11.669  1.00 15.91           N  \\nATOM    835  CA  ASP A 105      -8.666  11.988  11.359  1.00 15.43           C  \\nATOM    836  C   ASP A 105      -7.705  11.372  10.333  1.00 16.04           C  \\nATOM    837  O   ASP A 105      -8.106  10.654   9.415  1.00 15.38           O  \\nATOM    838  CB  ASP A 105     -10.077  11.847  10.810  1.00 18.74           C  \\nATOM    839  CG  ASP A 105     -11.261  12.121  11.696  1.00 21.15           C  \\nATOM    840  OD1 ASP A 105     -11.183  11.972  12.918  1.00 21.36           O  \\nATOM    841  OD2 ASP A 105     -12.281  12.480  11.113  1.00 22.63           O  \\nATOM    842  N   SER A 106      -6.397  11.591  10.469  1.00 15.54           N  \\nATOM    843  CA  SER A 106      -5.470  11.174   9.442  1.00 17.25           C  \\nATOM    844  C   SER A 106      -4.112  10.882  10.011  1.00 17.31           C  \\nATOM    845  O   SER A 106      -3.650  11.556  10.916  1.00 19.11           O  \\nATOM    846  CB  SER A 106      -5.463  12.314   8.398  1.00 19.27           C  \\nATOM    847  OG  SER A 106      -4.326  12.490   7.573  1.00 24.20           O  \\nATOM    848  N   ASP A 107      -3.454   9.891   9.453  1.00 17.29           N  \\nATOM    849  CA  ASP A 107      -2.109   9.557   9.868  1.00 19.80           C  \\nATOM    850  C   ASP A 107      -1.000  10.220   9.063  1.00 20.36           C  \\nATOM    851  O   ASP A 107       0.197  10.118   9.345  1.00 21.26           O  \\nATOM    852  CB  ASP A 107      -1.970   8.045   9.810  1.00 23.15           C  \\nATOM    853  CG  ASP A 107      -2.863   7.310  10.807  1.00 25.12           C  \\nATOM    854  OD1 ASP A 107      -3.262   6.206  10.462  1.00 29.21           O  \\nATOM    855  OD2 ASP A 107      -3.127   7.812  11.903  1.00 26.59           O  \\nATOM    856  N   ASP A 108      -1.383  10.909   8.005  1.00 20.57           N  \\nATOM    857  CA  ASP A 108      -0.430  11.664   7.216  1.00 22.38           C  \\nATOM    858  C   ASP A 108      -1.061  12.982   6.842  1.00 21.37           C  \\nATOM    859  O   ASP A 108      -1.973  13.096   6.030  1.00 24.36           O  \\nATOM    860  CB  ASP A 108      -0.056  10.900   5.967  1.00 27.28           C  \\nATOM    861  CG  ASP A 108      -1.263  10.278   5.320  1.00 32.24           C  \\nATOM    862  OD1 ASP A 108      -1.714   9.272   5.877  1.00 37.07           O  \\nATOM    863  OD2 ASP A 108      -1.747  10.790   4.301  1.00 36.33           O  \\nATOM    864  N   VAL A 109      -0.598  13.970   7.572  1.00 19.33           N  \\nATOM    865  CA  VAL A 109      -1.009  15.354   7.400  1.00 17.49           C  \\nATOM    866  C   VAL A 109       0.215  16.086   6.819  1.00 14.55           C  \\nATOM    867  O   VAL A 109       1.335  15.772   7.245  1.00 13.64           O  \\nATOM    868  CB  VAL A 109      -1.410  15.870   8.784  1.00 18.40           C  \\nATOM    869  CG1 VAL A 109      -1.645  17.366   8.765  1.00 20.21           C  \\nATOM    870  CG2 VAL A 109      -2.676  15.101   9.214  1.00 20.27           C  \\nATOM    871  N   PRO A 110       0.141  17.001   5.825  1.00 12.41           N  \\nATOM    872  CA  PRO A 110       1.297  17.783   5.384  1.00 11.52           C  \\nATOM    873  C   PRO A 110       1.920  18.480   6.605  1.00 11.79           C  \\nATOM    874  O   PRO A 110       1.289  19.245   7.324  1.00  9.55           O  \\nATOM    875  CB  PRO A 110       0.728  18.721   4.370  1.00 11.31           C  \\nATOM    876  CG  PRO A 110      -0.410  17.926   3.779  1.00 11.38           C  \\nATOM    877  CD  PRO A 110      -1.044  17.309   5.020  1.00 12.05           C  \\nATOM    878  N   MET A 111       3.192  18.201   6.874  1.00 11.92           N  \\nATOM    879  CA  MET A 111       3.878  18.724   8.050  1.00 11.49           C  \\nATOM    880  C   MET A 111       5.379  18.749   7.807  1.00 11.68           C  \\nATOM    881  O   MET A 111       5.903  17.900   7.098  1.00 11.11           O  \\nATOM    882  CB  MET A 111       3.494  17.806   9.211  1.00 14.36           C  \\nATOM    883  CG  MET A 111       3.864  18.110  10.627  1.00 17.41           C  \\nATOM    884  SD  MET A 111       3.231  16.825  11.758  1.00 18.36           S  \\nATOM    885  CE  MET A 111       1.528  17.128  11.498  1.00 17.25           C  \\nATOM    886  N   VAL A 112       6.091  19.706   8.404  1.00 10.73           N  \\nATOM    887  CA  VAL A 112       7.545  19.763   8.348  1.00 10.18           C  \\nATOM    888  C   VAL A 112       7.992  19.875   9.794  1.00  9.11           C  \\nATOM    889  O   VAL A 112       7.404  20.633  10.556  1.00 10.77           O  \\nATOM    890  CB  VAL A 112       8.010  21.024   7.542  1.00 11.80           C  \\nATOM    891  CG1 VAL A 112       9.528  21.213   7.558  1.00 13.24           C  \\nATOM    892  CG2 VAL A 112       7.542  20.843   6.104  1.00 14.56           C  \\nATOM    893  N   LEU A 113       9.032  19.167  10.207  1.00  8.47           N  \\nATOM    894  CA  LEU A 113       9.632  19.332  11.535  1.00  9.53           C  \\nATOM    895  C   LEU A 113      10.738  20.384  11.412  1.00  9.33           C  \\nATOM    896  O   LEU A 113      11.621  20.270  10.555  1.00  9.95           O  \\nATOM    897  CB  LEU A 113      10.214  17.983  12.017  1.00  9.56           C  \\nATOM    898  CG  LEU A 113      10.863  17.976  13.381  1.00 11.58           C  \\nATOM    899  CD1 LEU A 113       9.908  18.379  14.503  1.00 13.17           C  \\nATOM    900  CD2 LEU A 113      11.361  16.564  13.597  1.00 12.84           C  \\nATOM    901  N   VAL A 114      10.705  21.442  12.221  1.00  7.62           N  \\nATOM    902  CA  VAL A 114      11.673  22.541  12.109  1.00  7.64           C  \\nATOM    903  C   VAL A 114      12.436  22.667  13.441  1.00  7.65           C  \\nATOM    904  O   VAL A 114      11.872  22.842  14.541  1.00  9.71           O  \\nATOM    905  CB  VAL A 114      10.896  23.836  11.774  1.00  9.37           C  \\nATOM    906  CG1 VAL A 114      11.814  25.030  11.869  1.00 10.27           C  \\nATOM    907  CG2 VAL A 114      10.254  23.719  10.393  1.00  9.64           C  \\nATOM    908  N   GLY A 115      13.745  22.567  13.331  1.00  6.10           N  \\nATOM    909  CA  GLY A 115      14.603  22.742  14.476  1.00  6.69           C  \\nATOM    910  C   GLY A 115      15.120  24.163  14.407  1.00  8.47           C  \\nATOM    911  O   GLY A 115      16.007  24.439  13.594  1.00  9.16           O  \\nATOM    912  N   ASN A 116      14.593  25.089  15.192  1.00  8.01           N  \\nATOM    913  CA  ASN A 116      15.007  26.475  15.111  1.00  8.24           C  \\nATOM    914  C   ASN A 116      16.112  26.873  16.109  1.00  9.61           C  \\nATOM    915  O   ASN A 116      16.453  26.142  17.051  1.00  9.84           O  \\nATOM    916  CB  ASN A 116      13.721  27.307  15.275  1.00  8.01           C  \\nATOM    917  CG  ASN A 116      13.929  28.788  14.992  1.00  9.48           C  \\nATOM    918  OD1 ASN A 116      14.470  29.205  13.972  1.00  9.94           O  \\nATOM    919  ND2 ASN A 116      13.565  29.644  15.940  1.00 10.75           N  \\nATOM    920  N   LYS A 117      16.660  28.087  15.876  1.00 10.22           N  \\nATOM    921  CA  LYS A 117      17.735  28.715  16.674  1.00 12.36           C  \\nATOM    922  C   LYS A 117      19.094  28.021  16.450  1.00 13.39           C  \\nATOM    923  O   LYS A 117      19.977  27.940  17.324  1.00 11.29           O  \\nATOM    924  CB  LYS A 117      17.391  28.706  18.214  1.00 13.23           C  \\nATOM    925  CG  LYS A 117      16.026  29.272  18.611  1.00 12.90           C  \\nATOM    926  CD  LYS A 117      15.932  29.640  20.097  1.00 12.43           C  \\nATOM    927  CE  LYS A 117      14.551  30.217  20.380  1.00 10.55           C  \\nATOM    928  NZ  LYS A 117      14.491  30.647  21.772  1.00 11.10           N  \\nATOM    929  N   CYS A 118      19.254  27.555  15.188  1.00 13.69           N  \\nATOM    930  CA  CYS A 118      20.449  26.806  14.844  1.00 15.67           C  \\nATOM    931  C   CYS A 118      21.716  27.658  14.796  1.00 15.28           C  \\nATOM    932  O   CYS A 118      22.808  27.113  14.629  1.00 15.30           O  \\nATOM    933  CB  CYS A 118      20.246  26.064  13.492  1.00 17.74           C  \\nATOM    934  SG  CYS A 118      20.353  27.009  11.944  1.00 20.74           S  \\nATOM    935  N   ASP A 119      21.578  28.995  14.931  1.00 15.76           N  \\nATOM    936  CA  ASP A 119      22.722  29.904  15.083  1.00 16.09           C  \\nATOM    937  C   ASP A 119      23.426  29.778  16.445  1.00 17.19           C  \\nATOM    938  O   ASP A 119      24.589  30.178  16.553  1.00 18.76           O  \\nATOM    939  CB  ASP A 119      22.268  31.374  14.888  1.00 15.14           C  \\nATOM    940  CG  ASP A 119      21.153  31.785  15.835  1.00 15.81           C  \\nATOM    941  OD1 ASP A 119      20.030  31.294  15.662  1.00 13.87           O  \\nATOM    942  OD2 ASP A 119      21.407  32.596  16.721  1.00 14.85           O  \\nATOM    943  N   LEU A 120      22.732  29.248  17.480  1.00 16.86           N  \\nATOM    944  CA  LEU A 120      23.230  29.099  18.845  1.00 18.76           C  \\nATOM    945  C   LEU A 120      24.217  27.957  18.979  1.00 19.70           C  \\nATOM    946  O   LEU A 120      23.979  26.885  18.431  1.00 20.53           O  \\nATOM    947  CB  LEU A 120      22.103  28.829  19.831  1.00 17.65           C  \\nATOM    948  CG  LEU A 120      21.065  29.933  20.010  1.00 19.43           C  \\nATOM    949  CD1 LEU A 120      20.044  29.418  21.008  1.00 18.46           C  \\nATOM    950  CD2 LEU A 120      21.688  31.239  20.503  1.00 18.73           C  \\nATOM    951  N   ALA A 121      25.313  28.184  19.705  1.00 19.26           N  \\nATOM    952  CA  ALA A 121      26.303  27.143  19.887  1.00 20.48           C  \\nATOM    953  C   ALA A 121      25.942  26.210  21.036  1.00 21.30           C  \\nATOM    954  O   ALA A 121      26.403  25.069  21.067  1.00 24.04           O  \\nATOM    955  CB  ALA A 121      27.652  27.767  20.164  1.00 22.02           C  \\nATOM    956  N   ALA A 122      25.109  26.635  21.982  1.00 22.09           N  \\nATOM    957  CA  ALA A 122      24.683  25.797  23.094  1.00 23.88           C  \\nATOM    958  C   ALA A 122      23.578  24.811  22.720  1.00 23.77           C  \\nATOM    959  O   ALA A 122      22.492  24.835  23.301  1.00 27.82           O  \\nATOM    960  CB  ALA A 122      24.171  26.684  24.249  1.00 25.67           C  \\nATOM    961  N   ARG A 123      23.843  23.908  21.783  1.00 20.99           N  \\nATOM    962  CA  ARG A 123      22.890  22.905  21.311  1.00 19.81           C  \\nATOM    963  C   ARG A 123      22.619  21.761  22.283  1.00 19.56           C  \\nATOM    964  O   ARG A 123      23.590  21.176  22.774  1.00 20.01           O  \\nATOM    965  CB  ARG A 123      23.421  22.333  19.988  1.00 19.81           C  \\nATOM    966  CG  ARG A 123      22.662  21.166  19.398  1.00 21.43           C  \\nATOM    967  CD  ARG A 123      23.273  20.680  18.073  1.00 22.66           C  \\nATOM    968  NE  ARG A 123      23.050  21.590  16.956  1.00 22.61           N  \\nATOM    969  CZ  ARG A 123      22.019  21.487  16.093  1.00 20.95           C  \\nATOM    970  NH1 ARG A 123      21.045  20.568  16.181  1.00 16.36           N  \\nATOM    971  NH2 ARG A 123      21.989  22.370  15.098  1.00 19.65           N  \\nATOM    972  N   THR A 124      21.364  21.389  22.600  1.00 17.96           N  \\nATOM    973  CA  THR A 124      21.118  20.200  23.415  1.00 17.85           C  \\nATOM    974  C   THR A 124      20.413  19.068  22.656  1.00 17.08           C  \\nATOM    975  O   THR A 124      20.459  17.926  23.091  1.00 19.59           O  \\nATOM    976  CB  THR A 124      20.311  20.578  24.672  1.00 19.42           C  \\nATOM    977  OG1 THR A 124      19.109  21.230  24.277  1.00 20.50           O  \\nATOM    978  CG2 THR A 124      21.155  21.458  25.588  1.00 19.06           C  \\nATOM    979  N   VAL A 125      19.788  19.319  21.495  1.00 16.59           N  \\nATOM    980  CA  VAL A 125      19.133  18.324  20.647  1.00 16.09           C  \\nATOM    981  C   VAL A 125      20.032  18.080  19.413  1.00 16.61           C  \\nATOM    982  O   VAL A 125      20.180  18.980  18.573  1.00 14.07           O  \\nATOM    983  CB  VAL A 125      17.720  18.848  20.179  1.00 15.28           C  \\nATOM    984  CG1 VAL A 125      17.097  17.814  19.265  1.00 14.81           C  \\nATOM    985  CG2 VAL A 125      16.824  19.168  21.366  1.00 15.97           C  \\nATOM    986  N   GLU A 126      20.659  16.889  19.280  1.00 17.05           N  \\nATOM    987  CA  GLU A 126      21.487  16.568  18.124  1.00 19.28           C  \\nATOM    988  C   GLU A 126      20.631  16.496  16.876  1.00 18.51           C  \\nATOM    989  O   GLU A 126      19.511  15.976  16.880  1.00 17.68           O  \\nATOM    990  CB  GLU A 126      22.165  15.215  18.219  1.00 23.29           C  \\nATOM    991  CG  GLU A 126      23.244  15.057  19.265  1.00 30.86           C  \\nATOM    992  CD  GLU A 126      24.402  16.054  19.153  1.00 36.59           C  \\nATOM    993  OE1 GLU A 126      25.139  16.012  18.161  1.00 39.64           O  \\nATOM    994  OE2 GLU A 126      24.581  16.885  20.056  1.00 40.44           O  \\nATOM    995  N   SER A 127      21.167  16.985  15.755  1.00 17.73           N  \\nATOM    996  CA  SER A 127      20.426  16.938  14.493  1.00 16.72           C  \\nATOM    997  C   SER A 127      20.040  15.506  14.158  1.00 16.76           C  \\nATOM    998  O   SER A 127      18.931  15.245  13.690  1.00 17.01           O  \\nATOM    999  CB  SER A 127      21.283  17.518  13.371  1.00 17.67           C  \\nATOM   1000  OG  SER A 127      21.730  18.826  13.768  1.00 19.67           O  \\nATOM   1001  N   ARG A 128      20.916  14.558  14.466  1.00 16.72           N  \\nATOM   1002  CA  ARG A 128      20.667  13.154  14.145  1.00 20.94           C  \\nATOM   1003  C   ARG A 128      19.435  12.617  14.862  1.00 19.62           C  \\nATOM   1004  O   ARG A 128      18.641  11.882  14.278  1.00 18.81           O  \\nATOM   1005  CB  ARG A 128      21.910  12.319  14.510  1.00 24.31           C  \\nATOM   1006  CG  ARG A 128      21.969  11.035  13.692  1.00 30.88           C  \\nATOM   1007  CD  ARG A 128      23.381  10.403  13.536  1.00 35.88           C  \\nATOM   1008  NE  ARG A 128      24.384  10.996  12.621  1.00 39.57           N  \\nATOM   1009  CZ  ARG A 128      24.313  11.018  11.267  1.00 40.85           C  \\nATOM   1010  NH1 ARG A 128      25.399  11.388  10.577  1.00 40.83           N  \\nATOM   1011  NH2 ARG A 128      23.202  10.685  10.576  1.00 41.33           N  \\nATOM   1012  N   GLN A 129      19.209  13.016  16.119  1.00 18.27           N  \\nATOM   1013  CA  GLN A 129      18.032  12.519  16.787  1.00 18.53           C  \\nATOM   1014  C   GLN A 129      16.735  13.151  16.224  1.00 17.11           C  \\nATOM   1015  O   GLN A 129      15.691  12.486  16.098  1.00 15.48           O  \\nATOM   1016  CB  GLN A 129      18.229  12.766  18.306  1.00 22.36           C  \\nATOM   1017  CG  GLN A 129      18.041  14.133  18.911  1.00 24.78           C  \\nATOM   1018  CD  GLN A 129      18.283  14.123  20.411  1.00 27.48           C  \\nATOM   1019  OE1 GLN A 129      19.287  14.643  20.906  1.00 28.14           O  \\nATOM   1020  NE2 GLN A 129      17.367  13.497  21.152  1.00 27.21           N  \\nATOM   1021  N   ALA A 130      16.780  14.446  15.837  1.00 14.73           N  \\nATOM   1022  CA  ALA A 130      15.611  15.044  15.210  1.00 13.87           C  \\nATOM   1023  C   ALA A 130      15.342  14.449  13.826  1.00 13.30           C  \\nATOM   1024  O   ALA A 130      14.199  14.194  13.436  1.00 11.42           O  \\nATOM   1025  CB  ALA A 130      15.822  16.536  15.063  1.00 16.21           C  \\nATOM   1026  N   GLN A 131      16.401  14.189  13.057  1.00 14.61           N  \\nATOM   1027  CA  GLN A 131      16.267  13.529  11.739  1.00 16.01           C  \\nATOM   1028  C   GLN A 131      15.637  12.109  11.821  1.00 16.29           C  \\nATOM   1029  O   GLN A 131      14.777  11.708  11.017  1.00 14.37           O  \\nATOM   1030  CB  GLN A 131      17.673  13.507  11.116  1.00 18.12           C  \\nATOM   1031  CG  GLN A 131      17.702  13.017   9.689  1.00 22.19           C  \\nATOM   1032  CD  GLN A 131      16.889  13.922   8.801  1.00 21.61           C  \\nATOM   1033  OE1 GLN A 131      17.292  15.044   8.529  1.00 24.06           O  \\nATOM   1034  NE2 GLN A 131      15.714  13.485   8.370  1.00 23.37           N  \\nATOM   1035  N   ASP A 132      16.038  11.348  12.853  1.00 16.64           N  \\nATOM   1036  CA  ASP A 132      15.467  10.038  13.173  1.00 18.00           C  \\nATOM   1037  C   ASP A 132      13.984  10.175  13.484  1.00 17.28           C  \\nATOM   1038  O   ASP A 132      13.178   9.395  12.985  1.00 16.45           O  \\nATOM   1039  CB  ASP A 132      16.120   9.415  14.405  1.00 20.10           C  \\nATOM   1040  CG  ASP A 132      17.506   8.816  14.234  1.00 24.07           C  \\nATOM   1041  OD1 ASP A 132      18.120   8.459  15.243  1.00 28.35           O  \\nATOM   1042  OD2 ASP A 132      17.975   8.686  13.106  1.00 25.43           O  \\nATOM   1043  N   LEU A 133      13.614  11.168  14.312  1.00 15.58           N  \\nATOM   1044  CA  LEU A 133      12.213  11.390  14.624  1.00 16.28           C  \\nATOM   1045  C   LEU A 133      11.421  11.729  13.366  1.00 14.96           C  \\nATOM   1046  O   LEU A 133      10.345  11.181  13.128  1.00 15.09           O  \\nATOM   1047  CB  LEU A 133      12.052  12.536  15.647  1.00 17.06           C  \\nATOM   1048  CG  LEU A 133      10.617  12.781  16.136  1.00 18.79           C  \\nATOM   1049  CD1 LEU A 133      10.142  11.554  16.881  1.00 19.96           C  \\nATOM   1050  CD2 LEU A 133      10.544  14.013  17.020  1.00 18.45           C  \\nATOM   1051  N   ALA A 134      11.937  12.642  12.545  1.00 13.16           N  \\nATOM   1052  CA  ALA A 134      11.241  13.016  11.331  1.00 14.89           C  \\nATOM   1053  C   ALA A 134      11.092  11.821  10.414  1.00 16.19           C  \\nATOM   1054  O   ALA A 134      10.015  11.643   9.857  1.00 15.70           O  \\nATOM   1055  CB  ALA A 134      12.021  14.089  10.629  1.00 15.12           C  \\nATOM   1056  N   ARG A 135      12.143  10.995  10.279  1.00 17.00           N  \\nATOM   1057  CA  ARG A 135      12.088   9.828   9.438  1.00 19.17           C  \\nATOM   1058  C   ARG A 135      10.990   8.893   9.916  1.00 18.70           C  \\nATOM   1059  O   ARG A 135      10.269   8.352   9.078  1.00 20.29           O  \\nATOM   1060  CB  ARG A 135      13.425   9.091   9.456  1.00 22.48           C  \\nATOM   1061  CG  ARG A 135      13.499   8.297   8.155  1.00 29.02           C  \\nATOM   1062  CD  ARG A 135      14.619   7.270   7.988  1.00 33.22           C  \\nATOM   1063  NE  ARG A 135      14.117   5.888   8.050  1.00 38.80           N  \\nATOM   1064  CZ  ARG A 135      13.281   5.302   7.148  1.00 39.60           C  \\nATOM   1065  NH1 ARG A 135      12.889   4.044   7.384  1.00 40.80           N  \\nATOM   1066  NH2 ARG A 135      12.827   5.913   6.034  1.00 40.11           N  \\nATOM   1067  N   SER A 136      10.815   8.697  11.242  1.00 19.23           N  \\nATOM   1068  CA  SER A 136       9.757   7.841  11.770  1.00 19.12           C  \\nATOM   1069  C   SER A 136       8.374   8.432  11.528  1.00 19.61           C  \\nATOM   1070  O   SER A 136       7.398   7.690  11.522  1.00 21.01           O  \\nATOM   1071  CB  SER A 136       9.930   7.624  13.280  1.00 19.94           C  \\nATOM   1072  OG  SER A 136       9.459   8.686  14.119  1.00 23.21           O  \\nATOM   1073  N   TYR A 137       8.231   9.760  11.380  1.00 17.33           N  \\nATOM   1074  CA  TYR A 137       6.950  10.375  11.078  1.00 18.12           C  \\nATOM   1075  C   TYR A 137       6.683  10.404   9.570  1.00 18.22           C  \\nATOM   1076  O   TYR A 137       5.561  10.656   9.129  1.00 19.27           O  \\nATOM   1077  CB  TYR A 137       6.907  11.812  11.592  1.00 18.21           C  \\nATOM   1078  CG  TYR A 137       6.883  12.057  13.084  1.00 19.08           C  \\nATOM   1079  CD1 TYR A 137       6.619  11.026  13.955  1.00 20.51           C  \\nATOM   1080  CD2 TYR A 137       7.059  13.355  13.548  1.00 20.43           C  \\nATOM   1081  CE1 TYR A 137       6.510  11.285  15.306  1.00 21.89           C  \\nATOM   1082  CE2 TYR A 137       6.945  13.624  14.899  1.00 21.02           C  \\nATOM   1083  CZ  TYR A 137       6.662  12.574  15.763  1.00 22.13           C  \\nATOM   1084  OH  TYR A 137       6.476  12.782  17.112  1.00 22.16           O  \\nATOM   1085  N   GLY A 138       7.689  10.186   8.737  1.00 16.07           N  \\nATOM   1086  CA  GLY A 138       7.520  10.265   7.294  1.00 17.66           C  \\nATOM   1087  C   GLY A 138       7.476  11.725   6.810  1.00 18.01           C  \\nATOM   1088  O   GLY A 138       6.859  12.010   5.774  1.00 17.85           O  \\nATOM   1089  N   ILE A 139       8.128  12.662   7.540  1.00 14.16           N  \\nATOM   1090  CA  ILE A 139       8.117  14.065   7.145  1.00 13.49           C  \\nATOM   1091  C   ILE A 139       9.526  14.662   7.033  1.00 14.35           C  \\nATOM   1092  O   ILE A 139      10.494  14.152   7.610  1.00 16.19           O  \\nATOM   1093  CB  ILE A 139       7.292  14.896   8.161  1.00 14.40           C  \\nATOM   1094  CG1 ILE A 139       7.925  14.811   9.554  1.00 15.44           C  \\nATOM   1095  CG2 ILE A 139       5.821  14.422   8.104  1.00 14.07           C  \\nATOM   1096  CD1 ILE A 139       7.288  15.765  10.587  1.00 15.35           C  \\nATOM   1097  N   PRO A 140       9.706  15.767   6.308  1.00 13.55           N  \\nATOM   1098  CA  PRO A 140      10.984  16.448   6.205  1.00 13.39           C  \\nATOM   1099  C   PRO A 140      11.424  17.065   7.532  1.00 12.74           C  \\nATOM   1100  O   PRO A 140      10.601  17.530   8.310  1.00 11.42           O  \\nATOM   1101  CB  PRO A 140      10.767  17.498   5.127  1.00 13.47           C  \\nATOM   1102  CG  PRO A 140       9.514  17.084   4.410  1.00 14.09           C  \\nATOM   1103  CD  PRO A 140       8.686  16.467   5.519  1.00 13.14           C  \\nATOM   1104  N   TYR A 141      12.744  17.086   7.756  1.00 11.68           N  \\nATOM   1105  CA  TYR A 141      13.336  17.791   8.890  1.00 10.89           C  \\nATOM   1106  C   TYR A 141      14.175  18.945   8.326  1.00 10.09           C  \\nATOM   1107  O   TYR A 141      15.056  18.720   7.468  1.00 10.27           O  \\nATOM   1108  CB  TYR A 141      14.276  16.897   9.691  1.00 12.54           C  \\nATOM   1109  CG  TYR A 141      15.063  17.676  10.733  1.00 10.92           C  \\nATOM   1110  CD1 TYR A 141      14.350  18.396  11.664  1.00 11.75           C  \\nATOM   1111  CD2 TYR A 141      16.453  17.633  10.784  1.00 12.49           C  \\nATOM   1112  CE1 TYR A 141      14.997  19.074  12.670  1.00 12.85           C  \\nATOM   1113  CE2 TYR A 141      17.112  18.307  11.794  1.00 11.33           C  \\nATOM   1114  CZ  TYR A 141      16.369  19.016  12.728  1.00 12.61           C  \\nATOM   1115  OH  TYR A 141      16.944  19.646  13.813  1.00 13.82           O  \\nATOM   1116  N   ILE A 142      13.892  20.174   8.799  1.00  9.35           N  \\nATOM   1117  CA  ILE A 142      14.636  21.355   8.376  1.00  9.05           C  \\nATOM   1118  C   ILE A 142      15.153  22.163   9.551  1.00  9.25           C  \\nATOM   1119  O   ILE A 142      14.409  22.382  10.499  1.00 10.43           O  \\nATOM   1120  CB  ILE A 142      13.756  22.259   7.472  1.00  9.25           C  \\nATOM   1121  CG1 ILE A 142      13.523  21.470   6.180  1.00 10.51           C  \\nATOM   1122  CG2 ILE A 142      14.450  23.586   7.122  1.00  9.08           C  \\nATOM   1123  CD1 ILE A 142      12.719  22.127   5.028  1.00 13.16           C  \\nATOM   1124  N   GLU A 143      16.457  22.513   9.568  1.00  9.01           N  \\nATOM   1125  CA  GLU A 143      16.979  23.404  10.611  1.00  8.52           C  \\nATOM   1126  C   GLU A 143      16.978  24.864  10.193  1.00 10.31           C  \\nATOM   1127  O   GLU A 143      17.278  25.228   9.040  1.00 10.45           O  \\nATOM   1128  CB  GLU A 143      18.397  23.022  11.013  1.00 10.87           C  \\nATOM   1129  CG  GLU A 143      18.363  21.728  11.801  1.00 10.22           C  \\nATOM   1130  CD  GLU A 143      19.592  21.411  12.633  1.00 13.01           C  \\nATOM   1131  OE1 GLU A 143      20.616  22.082  12.521  1.00 13.65           O  \\nATOM   1132  OE2 GLU A 143      19.511  20.487  13.429  1.00 12.70           O  \\nATOM   1133  N   THR A 144      16.608  25.762  11.117  1.00  9.18           N  \\nATOM   1134  CA  THR A 144      16.422  27.169  10.754  1.00  9.71           C  \\nATOM   1135  C   THR A 144      17.014  28.076  11.789  1.00  8.66           C  \\nATOM   1136  O   THR A 144      17.259  27.698  12.927  1.00  9.13           O  \\nATOM   1137  CB  THR A 144      14.919  27.617  10.635  1.00  9.90           C  \\nATOM   1138  OG1 THR A 144      14.336  27.403  11.931  1.00  9.29           O  \\nATOM   1139  CG2 THR A 144      14.120  26.858   9.557  1.00  9.89           C  \\nATOM   1140  N   SER A 145      17.244  29.288  11.360  1.00 10.91           N  \\nATOM   1141  CA  SER A 145      17.558  30.377  12.290  1.00 12.65           C  \\nATOM   1142  C   SER A 145      16.758  31.602  11.849  1.00 10.37           C  \\nATOM   1143  O   SER A 145      16.930  32.094  10.727  1.00 11.76           O  \\nATOM   1144  CB  SER A 145      19.069  30.768  12.308  1.00 11.79           C  \\nATOM   1145  OG  SER A 145      19.248  31.931  13.125  1.00 12.12           O  \\nATOM   1146  N   ALA A 146      15.838  32.046  12.696  1.00  9.49           N  \\nATOM   1147  CA  ALA A 146      15.131  33.268  12.413  1.00 10.90           C  \\nATOM   1148  C   ALA A 146      16.070  34.455  12.553  1.00 13.53           C  \\nATOM   1149  O   ALA A 146      15.799  35.514  11.995  1.00 16.20           O  \\nATOM   1150  CB  ALA A 146      13.976  33.436  13.367  1.00 12.65           C  \\nATOM   1151  N   LYS A 147      17.183  34.302  13.283  1.00 14.35           N  \\nATOM   1152  CA  LYS A 147      18.150  35.351  13.486  1.00 16.25           C  \\nATOM   1153  C   LYS A 147      18.953  35.568  12.215  1.00 17.58           C  \\nATOM   1154  O   LYS A 147      19.124  36.724  11.825  1.00 20.13           O  \\nATOM   1155  CB  LYS A 147      19.068  34.970  14.617  1.00 17.55           C  \\nATOM   1156  CG  LYS A 147      20.003  36.125  14.958  1.00 22.83           C  \\nATOM   1157  CD  LYS A 147      20.581  35.932  16.341  1.00 26.44           C  \\nATOM   1158  CE  LYS A 147      21.683  36.949  16.507  1.00 30.79           C  \\nATOM   1159  NZ  LYS A 147      22.575  36.534  17.574  1.00 33.81           N  \\nATOM   1160  N   THR A 148      19.479  34.529  11.541  1.00 17.16           N  \\nATOM   1161  CA  THR A 148      20.248  34.735  10.300  1.00 15.93           C  \\nATOM   1162  C   THR A 148      19.422  34.607   9.013  1.00 17.72           C  \\nATOM   1163  O   THR A 148      19.926  34.962   7.945  1.00 18.47           O  \\nATOM   1164  CB  THR A 148      21.444  33.745  10.209  1.00 15.86           C  \\nATOM   1165  OG1 THR A 148      20.886  32.435  10.042  1.00 14.56           O  \\nATOM   1166  CG2 THR A 148      22.377  33.839  11.439  1.00 15.25           C  \\nATOM   1167  N   ARG A 149      18.176  34.108   9.100  1.00 15.18           N  \\nATOM   1168  CA  ARG A 149      17.268  33.815   7.994  1.00 19.20           C  \\nATOM   1169  C   ARG A 149      17.652  32.483   7.318  1.00 17.94           C  \\nATOM   1170  O   ARG A 149      17.031  32.079   6.332  1.00 20.28           O  \\nATOM   1171  CB  ARG A 149      17.251  34.964   6.937  1.00 21.40           C  \\nATOM   1172  CG  ARG A 149      15.929  35.106   6.188  1.00 27.00           C  \\nATOM   1173  CD  ARG A 149      15.946  35.904   4.863  1.00 27.99           C  \\nATOM   1174  NE  ARG A 149      14.707  35.687   4.092  1.00 29.40           N  \\nATOM   1175  CZ  ARG A 149      13.550  36.232   4.492  1.00 29.07           C  \\nATOM   1176  NH1 ARG A 149      12.412  36.015   3.828  1.00 29.56           N  \\nATOM   1177  NH2 ARG A 149      13.527  37.004   5.577  1.00 28.99           N  \\nATOM   1178  N   GLN A 150      18.621  31.701   7.838  1.00 15.80           N  \\nATOM   1179  CA  GLN A 150      18.926  30.402   7.273  1.00 16.36           C  \\nATOM   1180  C   GLN A 150      17.737  29.464   7.455  1.00 15.02           C  \\nATOM   1181  O   GLN A 150      17.119  29.351   8.504  1.00 13.57           O  \\nATOM   1182  CB  GLN A 150      20.152  29.729   7.937  1.00 23.05           C  \\nATOM   1183  CG  GLN A 150      20.578  28.448   7.162  1.00 29.37           C  \\nATOM   1184  CD  GLN A 150      21.604  27.530   7.827  1.00 34.84           C  \\nATOM   1185  OE1 GLN A 150      21.301  26.416   8.280  1.00 38.95           O  \\nATOM   1186  NE2 GLN A 150      22.857  27.967   7.883  1.00 35.22           N  \\nATOM   1187  N   GLY A 151      17.412  28.809   6.348  1.00 13.62           N  \\nATOM   1188  CA  GLY A 151      16.357  27.835   6.296  1.00 13.50           C  \\nATOM   1189  C   GLY A 151      14.943  28.395   6.329  1.00 12.36           C  \\nATOM   1190  O   GLY A 151      14.025  27.591   6.148  1.00 11.07           O  \\nATOM   1191  N   VAL A 152      14.706  29.702   6.510  1.00 12.59           N  \\nATOM   1192  CA  VAL A 152      13.339  30.183   6.715  1.00 14.23           C  \\nATOM   1193  C   VAL A 152      12.435  29.971   5.475  1.00 13.41           C  \\nATOM   1194  O   VAL A 152      11.352  29.362   5.577  1.00 12.94           O  \\nATOM   1195  CB  VAL A 152      13.430  31.691   7.167  1.00 12.57           C  \\nATOM   1196  CG1 VAL A 152      12.059  32.316   7.194  1.00 13.14           C  \\nATOM   1197  CG2 VAL A 152      14.044  31.787   8.543  1.00 12.69           C  \\nATOM   1198  N   GLU A 153      12.831  30.410   4.278  1.00 13.52           N  \\nATOM   1199  CA  GLU A 153      12.001  30.213   3.086  1.00 14.81           C  \\nATOM   1200  C   GLU A 153      11.857  28.756   2.742  1.00 11.88           C  \\nATOM   1201  O   GLU A 153      10.797  28.282   2.344  1.00 13.92           O  \\nATOM   1202  CB  GLU A 153      12.592  30.936   1.881  1.00 17.03           C  \\nATOM   1203  CG  GLU A 153      12.454  32.452   2.015  1.00 21.18           C  \\nATOM   1204  CD  GLU A 153      12.870  33.318   0.817  1.00 25.73           C  \\nATOM   1205  OE1 GLU A 153      13.232  32.781  -0.234  1.00 28.32           O  \\nATOM   1206  OE2 GLU A 153      12.822  34.550   0.936  1.00 26.64           O  \\nATOM   1207  N   ASP A 154      12.959  28.031   2.957  1.00 11.31           N  \\nATOM   1208  CA  ASP A 154      13.023  26.593   2.747  1.00 10.16           C  \\nATOM   1209  C   ASP A 154      11.964  25.828   3.561  1.00 10.90           C  \\nATOM   1210  O   ASP A 154      11.224  24.985   3.035  1.00 11.83           O  \\nATOM   1211  CB  ASP A 154      14.420  26.212   3.117  1.00 15.00           C  \\nATOM   1212  CG  ASP A 154      14.919  24.942   2.524  1.00 19.04           C  \\nATOM   1213  OD1 ASP A 154      14.121  24.195   1.938  1.00 22.96           O  \\nATOM   1214  OD2 ASP A 154      16.122  24.697   2.676  1.00 21.26           O  \\nATOM   1215  N   ALA A 155      11.854  26.107   4.881  1.00  8.83           N  \\nATOM   1216  CA  ALA A 155      10.841  25.450   5.716  1.00 10.00           C  \\nATOM   1217  C   ALA A 155       9.424  25.748   5.203  1.00 10.10           C  \\nATOM   1218  O   ALA A 155       8.653  24.814   4.978  1.00 11.14           O  \\nATOM   1219  CB  ALA A 155      10.923  25.933   7.192  1.00  9.50           C  \\nATOM   1220  N   PHE A 156       9.054  27.011   4.949  1.00 10.59           N  \\nATOM   1221  CA  PHE A 156       7.712  27.345   4.475  1.00 10.51           C  \\nATOM   1222  C   PHE A 156       7.453  26.825   3.069  1.00 11.05           C  \\nATOM   1223  O   PHE A 156       6.368  26.302   2.801  1.00 10.80           O  \\nATOM   1224  CB  PHE A 156       7.492  28.890   4.506  1.00 11.54           C  \\nATOM   1225  CG  PHE A 156       7.266  29.450   5.936  1.00 12.42           C  \\nATOM   1226  CD1 PHE A 156       8.319  29.958   6.707  1.00 12.00           C  \\nATOM   1227  CD2 PHE A 156       5.978  29.434   6.478  1.00 12.57           C  \\nATOM   1228  CE1 PHE A 156       8.065  30.434   8.001  1.00 12.45           C  \\nATOM   1229  CE2 PHE A 156       5.747  29.908   7.757  1.00 12.32           C  \\nATOM   1230  CZ  PHE A 156       6.782  30.406   8.513  1.00 10.18           C  \\nATOM   1231  N   TYR A 157       8.408  26.935   2.151  1.00  9.69           N  \\nATOM   1232  CA  TYR A 157       8.148  26.463   0.798  1.00 12.31           C  \\nATOM   1233  C   TYR A 157       8.091  24.945   0.719  1.00 10.42           C  \\nATOM   1234  O   TYR A 157       7.292  24.403  -0.047  1.00 11.35           O  \\nATOM   1235  CB  TYR A 157       9.233  26.978  -0.156  1.00 15.16           C  \\nATOM   1236  CG  TYR A 157       9.257  28.492  -0.367  1.00 18.73           C  \\nATOM   1237  CD1 TYR A 157       8.380  29.366   0.268  1.00 18.52           C  \\nATOM   1238  CD2 TYR A 157      10.211  28.971  -1.242  1.00 20.44           C  \\nATOM   1239  CE1 TYR A 157       8.466  30.724   0.025  1.00 22.04           C  \\nATOM   1240  CE2 TYR A 157      10.305  30.320  -1.488  1.00 22.70           C  \\nATOM   1241  CZ  TYR A 157       9.436  31.178  -0.852  1.00 23.03           C  \\nATOM   1242  OH  TYR A 157       9.572  32.528  -1.089  1.00 26.15           O  \\nATOM   1243  N   THR A 158       8.940  24.198   1.455  1.00 11.35           N  \\nATOM   1244  CA  THR A 158       8.836  22.727   1.522  1.00 10.11           C  \\nATOM   1245  C   THR A 158       7.440  22.347   2.064  1.00 11.11           C  \\nATOM   1246  O   THR A 158       6.811  21.408   1.558  1.00 11.43           O  \\nATOM   1247  CB  THR A 158       9.935  22.129   2.470  1.00 12.09           C  \\nATOM   1248  OG1 THR A 158      11.216  22.451   1.933  1.00 11.36           O  \\nATOM   1249  CG2 THR A 158       9.791  20.657   2.624  1.00 10.87           C  \\nATOM   1250  N   LEU A 159       6.888  23.056   3.072  1.00  9.25           N  \\nATOM   1251  CA  LEU A 159       5.533  22.754   3.506  1.00  9.93           C  \\nATOM   1252  C   LEU A 159       4.489  22.986   2.388  1.00 10.42           C  \\nATOM   1253  O   LEU A 159       3.560  22.185   2.272  1.00 11.22           O  \\nATOM   1254  CB  LEU A 159       5.171  23.608   4.711  1.00  9.03           C  \\nATOM   1255  CG  LEU A 159       3.789  23.318   5.269  1.00 10.96           C  \\nATOM   1256  CD1 LEU A 159       3.635  21.828   5.651  1.00  9.29           C  \\nATOM   1257  CD2 LEU A 159       3.580  24.223   6.469  1.00 13.71           C  \\nATOM   1258  N   VAL A 160       4.596  24.046   1.548  1.00 10.91           N  \\nATOM   1259  CA  VAL A 160       3.662  24.284   0.439  1.00  9.19           C  \\nATOM   1260  C   VAL A 160       3.803  23.137  -0.556  1.00  9.60           C  \\nATOM   1261  O   VAL A 160       2.781  22.638  -1.031  1.00 10.29           O  \\nATOM   1262  CB  VAL A 160       3.981  25.653  -0.211  1.00  8.50           C  \\nATOM   1263  CG1 VAL A 160       3.402  25.754  -1.622  1.00 10.56           C  \\nATOM   1264  CG2 VAL A 160       3.403  26.741   0.659  1.00  7.98           C  \\nATOM   1265  N   ARG A 161       5.033  22.667  -0.839  1.00  9.10           N  \\nATOM   1266  CA  ARG A 161       5.206  21.554  -1.767  1.00  9.82           C  \\nATOM   1267  C   ARG A 161       4.644  20.240  -1.215  1.00 10.44           C  \\nATOM   1268  O   ARG A 161       4.166  19.395  -1.984  1.00 12.88           O  \\nATOM   1269  CB  ARG A 161       6.679  21.398  -2.115  1.00  9.39           C  \\nATOM   1270  CG  ARG A 161       7.146  22.640  -2.816  1.00  9.70           C  \\nATOM   1271  CD  ARG A 161       8.594  22.398  -3.227  1.00 14.12           C  \\nATOM   1272  NE  ARG A 161       9.122  23.635  -3.780  1.00 14.15           N  \\nATOM   1273  CZ  ARG A 161      10.144  24.313  -3.257  1.00 12.45           C  \\nATOM   1274  NH1 ARG A 161      10.823  23.858  -2.204  1.00 13.24           N  \\nATOM   1275  NH2 ARG A 161      10.485  25.475  -3.819  1.00 13.52           N  \\nATOM   1276  N   GLU A 162       4.663  20.046   0.112  1.00 10.71           N  \\nATOM   1277  CA  GLU A 162       4.054  18.877   0.722  1.00 11.80           C  \\nATOM   1278  C   GLU A 162       2.522  18.887   0.568  1.00 11.20           C  \\nATOM   1279  O   GLU A 162       1.921  17.848   0.274  1.00 12.90           O  \\nATOM   1280  CB  GLU A 162       4.430  18.815   2.216  1.00 13.86           C  \\nATOM   1281  CG  GLU A 162       5.870  18.460   2.580  1.00 14.10           C  \\nATOM   1282  CD  GLU A 162       6.268  17.038   2.224  1.00 17.23           C  \\nATOM   1283  OE1 GLU A 162       5.571  16.116   2.627  1.00 19.29           O  \\nATOM   1284  OE2 GLU A 162       7.278  16.827   1.545  1.00 22.14           O  \\nATOM   1285  N   ILE A 163       1.869  20.045   0.731  1.00 10.13           N  \\nATOM   1286  CA  ILE A 163       0.426  20.175   0.542  1.00 10.58           C  \\nATOM   1287  C   ILE A 163       0.128  19.878  -0.933  1.00 13.43           C  \\nATOM   1288  O   ILE A 163      -0.796  19.118  -1.235  1.00 15.01           O  \\nATOM   1289  CB  ILE A 163      -0.015  21.590   0.891  1.00 11.33           C  \\nATOM   1290  CG1 ILE A 163       0.084  21.777   2.400  1.00 11.27           C  \\nATOM   1291  CG2 ILE A 163      -1.486  21.815   0.437  1.00 12.37           C  \\nATOM   1292  CD1 ILE A 163      -0.006  23.245   2.894  1.00 12.56           C  \\nATOM   1293  N   ARG A 164       0.904  20.456  -1.871  1.00 11.72           N  \\nATOM   1294  CA  ARG A 164       0.770  20.163  -3.283  1.00 13.64           C  \\nATOM   1295  C   ARG A 164       0.813  18.657  -3.626  1.00 17.97           C  \\nATOM   1296  O   ARG A 164       0.150  18.197  -4.559  1.00 18.12           O  \\nATOM   1297  CB  ARG A 164       1.897  20.887  -4.029  1.00 12.51           C  \\nATOM   1298  CG  ARG A 164       1.647  22.371  -4.120  1.00 12.49           C  \\nATOM   1299  CD  ARG A 164       2.801  22.888  -4.950  1.00 14.72           C  \\nATOM   1300  NE  ARG A 164       2.730  24.327  -5.113  1.00 12.82           N  \\nATOM   1301  CZ  ARG A 164       3.800  25.071  -5.405  1.00 13.29           C  \\nATOM   1302  NH1 ARG A 164       5.018  24.539  -5.527  1.00 11.56           N  \\nATOM   1303  NH2 ARG A 164       3.595  26.381  -5.572  1.00 12.55           N  \\nATOM   1304  N   GLN A 165       1.590  17.831  -2.935  1.00 20.23           N  \\nATOM   1305  CA  GLN A 165       1.673  16.435  -3.337  1.00 27.50           C  \\nATOM   1306  C   GLN A 165       0.782  15.499  -2.502  1.00 29.12           C  \\nATOM   1307  O   GLN A 165       0.826  14.274  -2.655  1.00 29.30           O  \\nATOM   1308  CB  GLN A 165       3.168  16.070  -3.286  1.00 30.34           C  \\nATOM   1309  CG  GLN A 165       4.018  16.904  -4.309  1.00 35.17           C  \\nATOM   1310  CD  GLN A 165       4.109  16.470  -5.796  1.00 37.71           C  \\nATOM   1311  OE1 GLN A 165       4.256  17.293  -6.719  1.00 36.70           O  \\nATOM   1312  NE2 GLN A 165       4.079  15.166  -6.094  1.00 38.13           N  \\nATOM   1313  N   HIS A 166      -0.068  16.075  -1.638  1.00 29.28           N  \\nATOM   1314  CA  HIS A 166      -0.982  15.296  -0.820  1.00 31.13           C  \\nATOM   1315  C   HIS A 166      -2.214  14.971  -1.665  1.00 33.30           C  \\nATOM   1316  O   HIS A 166      -2.630  15.786  -2.497  1.00 34.96           O  \\nATOM   1317  CB  HIS A 166      -1.420  16.090   0.428  1.00 30.89           C  \\nATOM   1318  CG  HIS A 166      -2.247  15.319   1.475  1.00 32.17           C  \\nATOM   1319  ND1 HIS A 166      -3.579  15.231   1.603  1.00 33.52           N  \\nATOM   1320  CD2 HIS A 166      -1.712  14.560   2.502  1.00 32.95           C  \\nATOM   1321  CE1 HIS A 166      -3.857  14.464   2.647  1.00 33.07           C  \\nATOM   1322  NE2 HIS A 166      -2.727  14.064   3.184  1.00 33.06           N  \\nATOM   1323  OXT HIS A 166      -2.763  13.888  -1.454  1.00 36.27           O  \\nTER    1324      HIS A 166                                                      \\nHETATM 1325 MG    MG A 168       4.771  33.860  19.213  1.00  9.42          MG  \\nHETATM 1326  PG  GNP A 167       5.113  32.223  21.994  1.00  9.66           P  \\nHETATM 1327  O1G GNP A 167       4.844  32.596  23.390  1.00  9.26           O  \\nHETATM 1328  O2G GNP A 167       4.203  32.910  21.112  1.00 11.20           O  \\nHETATM 1329  O3G GNP A 167       4.640  30.769  21.802  1.00 10.09           O  \\nHETATM 1330  N3B GNP A 167       6.701  32.303  21.621  1.00  8.18           N  \\nHETATM 1331  PB  GNP A 167       7.310  32.034  20.111  1.00  8.77           P  \\nHETATM 1332  O1B GNP A 167       7.468  30.605  19.791  1.00  8.35           O  \\nHETATM 1333  O2B GNP A 167       6.696  32.775  19.021  1.00  9.38           O  \\nHETATM 1334  O3A GNP A 167       8.792  32.643  20.152  1.00 10.15           O  \\nHETATM 1335  PA  GNP A 167       9.280  34.140  19.710  1.00 10.18           P  \\nHETATM 1336  O1A GNP A 167       9.263  34.268  18.220  1.00 10.07           O  \\nHETATM 1337  O2A GNP A 167       8.594  35.173  20.510  1.00  9.33           O  \\nHETATM 1338  O5' GNP A 167      10.826  34.056  20.132  1.00 11.46           O  \\nHETATM 1339  C5' GNP A 167      11.170  33.911  21.535  1.00 10.56           C  \\nHETATM 1340  C4' GNP A 167      12.587  34.445  21.811  1.00 11.96           C  \\nHETATM 1341  O4' GNP A 167      13.560  33.649  21.095  1.00 11.66           O  \\nHETATM 1342  C3' GNP A 167      12.813  35.910  21.359  1.00 12.06           C  \\nHETATM 1343  O3' GNP A 167      13.793  36.538  22.221  1.00 14.52           O  \\nHETATM 1344  C2' GNP A 167      13.325  35.679  19.947  1.00 11.90           C  \\nHETATM 1345  O2' GNP A 167      14.151  36.750  19.499  1.00 14.99           O  \\nHETATM 1346  C1' GNP A 167      14.242  34.478  20.156  1.00 12.90           C  \\nHETATM 1347  N9  GNP A 167      14.454  33.739  18.896  1.00 12.98           N  \\nHETATM 1348  C8  GNP A 167      13.518  33.262  17.993  1.00 11.76           C  \\nHETATM 1349  N7  GNP A 167      14.016  32.636  16.979  1.00 12.34           N  \\nHETATM 1350  C5  GNP A 167      15.413  32.694  17.200  1.00 12.36           C  \\nHETATM 1351  C6  GNP A 167      16.521  32.219  16.402  1.00 12.30           C  \\nHETATM 1352  O6  GNP A 167      16.507  31.591  15.338  1.00 12.16           O  \\nHETATM 1353  N1  GNP A 167      17.735  32.503  16.995  1.00 11.60           N  \\nHETATM 1354  C2  GNP A 167      17.909  33.082  18.236  1.00 13.91           C  \\nHETATM 1355  N2  GNP A 167      19.173  33.280  18.658  1.00 14.60           N  \\nHETATM 1356  N3  GNP A 167      16.884  33.498  18.996  1.00 13.01           N  \\nHETATM 1357  C4  GNP A 167      15.675  33.331  18.398  1.00 14.36           C  \\nHETATM 1358  O   HOH A 169      13.588  43.386  17.779  1.00 31.80           O  \\nHETATM 1359  O   HOH A 170       8.248  36.064  23.209  1.00 13.63           O  \\nHETATM 1360  O   HOH A 171       3.254  37.845  19.454  1.00 18.29           O  \\nHETATM 1361  O   HOH A 172       5.945  35.619  20.105  1.00 11.33           O  \\nHETATM 1362  O   HOH A 173       3.669  32.226  18.172  1.00  8.22           O  \\nHETATM 1363  O   HOH A 174       2.896  37.170  15.500  1.00 21.37           O  \\nHETATM 1364  O   HOH A 175       2.102  32.309  24.122  1.00 27.47           O  \\nHETATM 1365  O   HOH A 176      -1.425  37.650  19.667  1.00 24.98           O  \\nHETATM 1366  O   HOH A 177      -3.225  30.031  -8.217  1.00 23.57           O  \\nHETATM 1367  O   HOH A 178      16.678  26.321  26.916  1.00 36.70           O  \\nHETATM 1368  O   HOH A 179      -2.134  40.975   1.551  1.00 81.53           O  \\nHETATM 1369  O   HOH A 180       0.400  40.675  -0.629  1.00 24.66           O  \\nHETATM 1370  O   HOH A 181      10.944  44.793  16.009  1.00 48.47           O  \\nHETATM 1371  O   HOH A 182      17.207  42.570  16.786  1.00 43.23           O  \\nHETATM 1372  O   HOH A 183      14.999  45.167  19.922  1.00 39.52           O  \\nHETATM 1373  O   HOH A 184      11.598  37.755  13.202  1.00 11.72           O  \\nHETATM 1374  O   HOH A 185       6.958  38.922  18.146  1.00 26.17           O  \\nHETATM 1375  O   HOH A 186      14.078  32.033  27.772  1.00 57.73           O  \\nHETATM 1376  O   HOH A 187       9.126  37.906  19.595  1.00 32.42           O  \\nHETATM 1377  O   HOH A 188      -0.049  36.435  27.259  1.00 50.56           O  \\nHETATM 1378  O   HOH A 189       3.843  30.503  25.485  1.00 46.80           O  \\nHETATM 1379  O   HOH A 190       4.931  28.343  27.587  1.00 29.32           O  \\nHETATM 1380  O   HOH A 191       4.881  25.774  26.247  1.00 28.55           O  \\nHETATM 1381  O   HOH A 193       8.729  26.516  26.645  1.00 37.34           O  \\nHETATM 1382  O   HOH A 194      -5.515  23.731  -4.213  1.00 43.74           O  \\nHETATM 1383  O   HOH A 195      -7.006  26.635   0.729  1.00 25.77           O  \\nHETATM 1384  O   HOH A 196      -2.033  23.281  -4.754  1.00 36.89           O  \\nHETATM 1385  O   HOH A 197      -4.326  24.642  -0.854  1.00 28.59           O  \\nHETATM 1386  O   HOH A 198      -6.858  26.995  -2.808  1.00 28.83           O  \\nHETATM 1387  O   HOH A 200      -3.937  29.225   8.289  1.00 24.08           O  \\nHETATM 1388  O   HOH A 201      -8.934  24.037   2.795  1.00 34.16           O  \\nHETATM 1389  O   HOH A 202       1.444  26.302  19.318  1.00 12.37           O  \\nHETATM 1390  O   HOH A 203       2.031  24.632  27.307  1.00 50.09           O  \\nHETATM 1391  O   HOH A 204      15.125  28.341  23.174  1.00 13.88           O  \\nHETATM 1392  O   HOH A 205      12.808  31.541  25.056  1.00 46.88           O  \\nHETATM 1393  O   HOH A 206       8.329  34.933   0.543  1.00 53.21           O  \\nHETATM 1394  O   HOH A 207       0.760  42.969   2.336  1.00 35.42           O  \\nHETATM 1395  O   HOH A 208       1.270  40.298   7.071  1.00 76.61           O  \\nHETATM 1396  O   HOH A 209      -3.135  39.625   7.649  1.00 35.40           O  \\nHETATM 1397  O   HOH A 210       8.699  39.076   2.399  1.00 33.93           O  \\nHETATM 1398  O   HOH A 212      10.040  41.806   3.447  1.00 37.88           O  \\nHETATM 1399  O   HOH A 213      18.914  29.095   3.704  1.00 25.32           O  \\nHETATM 1400  O   HOH A 214      15.966  29.098   2.957  1.00 18.63           O  \\nHETATM 1401  O   HOH A 215      15.506  31.962   4.015  1.00 18.93           O  \\nHETATM 1402  O   HOH A 216      22.935  30.533  10.845  1.00 39.81           O  \\nHETATM 1403  O   HOH A 217       6.476  43.893  23.325  1.00 49.09           O  \\nHETATM 1404  O   HOH A 218      11.746  41.248  28.612  1.00 36.22           O  \\nHETATM 1405  O   HOH A 219      11.480  42.270  25.715  1.00 29.66           O  \\nHETATM 1406  O   HOH A 220       9.344  34.251  -8.278  1.00 36.61           O  \\nHETATM 1407  O   HOH A 221      -0.111  19.667  -8.066  1.00 44.76           O  \\nHETATM 1408  O   HOH A 222      -0.528  30.460 -10.943  1.00 24.29           O  \\nHETATM 1409  O   HOH A 223     -11.352  24.274  19.491  1.00 31.21           O  \\nHETATM 1410  O   HOH A 224      -4.293  30.642 -16.466  1.00127.55           O  \\nHETATM 1411  O   HOH A 225       3.466  29.729 -16.232  1.00 26.15           O  \\nHETATM 1412  O   HOH A 227      -4.288  18.610   4.198  1.00 34.77           O  \\nHETATM 1413  O   HOH A 228       4.626  16.154   5.151  1.00 14.96           O  \\nHETATM 1414  O   HOH A 229       5.681  11.069   2.580  1.00 59.76           O  \\nHETATM 1415  O   HOH A 230       1.173  14.021   3.406  1.00 32.15           O  \\nHETATM 1416  O   HOH A 231       7.935  22.484  20.504  1.00 10.86           O  \\nHETATM 1417  O   HOH A 232       3.071  10.004  14.546  1.00 23.48           O  \\nHETATM 1418  O   HOH A 233       2.437  13.381   8.401  1.00 17.75           O  \\nHETATM 1419  O   HOH A 234       4.779   7.500   6.154  1.00 71.09           O  \\nHETATM 1420  O   HOH A 235      -4.250   8.748   6.880  1.00 24.76           O  \\nHETATM 1421  O   HOH A 236     -13.381  22.830  17.669  1.00 39.09           O  \\nHETATM 1422  O   HOH A 237      16.969  39.052   8.276  1.00 50.25           O  \\nHETATM 1423  O   HOH A 238      -6.315   9.333  13.897  1.00 36.24           O  \\nHETATM 1424  O   HOH A 240      -0.160   4.395  12.468  1.00 38.79           O  \\nHETATM 1425  O   HOH A 241       1.295   7.931  12.335  1.00 49.03           O  \\nHETATM 1426  O   HOH A 242       4.067  11.551   6.968  1.00 46.97           O  \\nHETATM 1427  O   HOH A 243      -5.060   6.579  -0.216  1.00 50.14           O  \\nHETATM 1428  O   HOH A 244      18.164   7.151   5.573  1.00 71.67           O  \\nHETATM 1429  O   HOH A 245       2.068   8.904   7.636  1.00 45.86           O  \\nHETATM 1430  O   HOH A 246      24.661  23.134  12.757  1.00 34.69           O  \\nHETATM 1431  O   HOH A 247      21.051  23.714   6.239  1.00 24.64           O  \\nHETATM 1432  O   HOH A 248      25.855  31.137  20.712  1.00 34.53           O  \\nHETATM 1433  O   HOH A 249      24.868  30.890  26.223  1.00 48.47           O  \\nHETATM 1434  O   HOH A 250      26.967  20.447  19.448  1.00 47.77           O  \\nHETATM 1435  O   HOH A 252      28.453  14.196  14.602  1.00 39.78           O  \\nHETATM 1436  O   HOH A 253      19.639  16.256   9.760  1.00 37.54           O  \\nHETATM 1437  O   HOH A 254      16.629   7.383  10.526  1.00 60.90           O  \\nHETATM 1438  O   HOH A 255       4.972   7.888  14.276  1.00 54.95           O  \\nHETATM 1439  O   HOH A 256      14.184  14.684   5.643  1.00 31.40           O  \\nHETATM 1440  O   HOH A 257      11.766  17.254   1.278  1.00 41.82           O  \\nHETATM 1441  O   HOH A 258      16.274  16.365   6.342  1.00 17.29           O  \\nHETATM 1442  O   HOH A 259      18.346  24.095   6.620  1.00 17.36           O  \\nHETATM 1443  O   HOH A 260      22.479  35.073  19.694  1.00 43.96           O  \\nHETATM 1444  O   HOH A 261      20.580  31.747   4.906  1.00 59.46           O  \\nHETATM 1445  O   HOH A 262      18.196  25.514   4.261  1.00 28.75           O  \\nHETATM 1446  O   HOH A 263      12.857  20.183   1.149  1.00 21.79           O  \\nHETATM 1447  O   HOH A 264      19.514  15.912  24.943  1.00 55.52           O  \\nHETATM 1448  O   HOH A 265       8.925   7.579   5.621  1.00 77.94           O  \\nHETATM 1449  O   HOH A 266      17.622  21.835  26.964  1.00 27.48           O  \\nHETATM 1450  O   HOH A 267      24.258  19.786  14.155  1.00 20.48           O  \\nHETATM 1451  O   HOH A 268      23.806  15.180  15.136  1.00 19.86           O  \\nHETATM 1452  O   HOH A 269      21.478  14.557  11.134  1.00 29.66           O  \\nHETATM 1453  O   HOH A 270      17.293  45.621  24.753  1.00 39.18           O  \\nHETATM 1454  O   HOH A 271      -3.518  40.206  17.649  1.00 44.75           O  \\nHETATM 1455  O   HOH A 272       2.143  19.852  24.090  1.00 40.10           O  \\nHETATM 1456  O   HOH A 273       7.101  35.918  -4.704  1.00 31.76           O  \\nHETATM 1457  O   HOH A 274       5.462  38.756  -5.521  1.00 27.51           O  \\nHETATM 1458  O   HOH A 275       5.748  20.565  28.638  1.00 58.06           O  \\nHETATM 1459  O   HOH A 276       8.814  31.258  -8.964  1.00 23.79           O  \\nHETATM 1460  O   HOH A 277     -10.427  35.895   4.301  1.00 58.91           O  \\nHETATM 1461  O   HOH A 278      -8.713  28.579   2.333  1.00 69.36           O  \\nHETATM 1462  O   HOH A 279      24.472  24.237  17.316  1.00 32.14           O  \\nHETATM 1463  O   HOH A 281      16.968  32.701  21.815  1.00 23.13           O  \\nHETATM 1464  O   HOH A 282      19.541  34.346  21.581  1.00 53.25           O  \\nHETATM 1465  O   HOH A 283      24.847  23.384  26.621  1.00 61.18           O  \\nHETATM 1466  O   HOH A 284       1.409  11.113   2.873  1.00 44.87           O  \\nHETATM 1467  O   HOH A 285       1.556  11.488  -0.914  1.00 37.86           O  \\nHETATM 1468  O   HOH A 286       9.758  12.945   3.889  1.00 36.47           O  \\nHETATM 1469  O   HOH A 287       3.827  13.375   4.622  1.00 44.90           O  \\nHETATM 1470  O   HOH A 288      14.279   6.175  15.391  1.00 78.25           O  \\nHETATM 1471  O   HOH A 289       7.573  13.968   2.191  1.00 76.47           O  \\nHETATM 1472  O   HOH A 290       2.323  15.482   1.418  1.00 35.15           O  \\nHETATM 1473  O   HOH A 291      11.997  14.573   2.962  1.00 49.86           O  \\nHETATM 1474  O   HOH A 292      23.798  33.550  17.880  1.00 24.64           O  \\nHETATM 1475  O   HOH A 293      14.657  36.046  -0.571  1.00 51.86           O  \\nHETATM 1476  O   HOH A 294      24.456  25.366   4.004  1.00 53.23           O  \\nHETATM 1477  O   HOH A 296      -0.603  12.134  -4.185  1.00 52.38           O  \\nHETATM 1478  O   HOH A 297      -8.372  34.415  -4.471  1.00 74.53           O  \\nHETATM 1479  O   HOH A 300     -14.033  36.501   0.062  1.00 50.21           O  \\nHETATM 1480  O   HOH A 301     -17.102  35.735  -2.806  1.00 42.58           O  \\nHETATM 1481  O   HOH A 303     -17.383  40.542   1.431  1.00 77.16           O  \\nHETATM 1482  O   HOH A 304     -19.024  41.557  -3.411  1.00107.16           O  \\nHETATM 1483  O   HOH A 305      -9.876  37.804   0.839  1.00 90.65           O  \\nHETATM 1484  O   HOH A 306     -14.053  36.466   4.552  1.00 82.05           O  \\nHETATM 1485  O   HOH A 307     -14.183  39.558   0.339  1.00 58.66           O  \\nHETATM 1486  O   HOH A 309     -12.547  28.677   8.301  1.00 74.88           O  \\nHETATM 1487  O   HOH A 311     -14.787  32.746  -2.171  1.00 40.79           O  \\nHETATM 1488  O   HOH A 312      -6.916  30.740   3.095  1.00 36.53           O  \\nHETATM 1489  O   HOH A 315      -8.312  33.027   7.340  1.00 83.28           O  \\nHETATM 1490  O   HOH A 316      -6.280  35.646   7.624  1.00 75.97           O  \\nHETATM 1491  O   HOH A 317      -5.230  31.565   9.609  1.00 55.67           O  \\nHETATM 1492  O   HOH A 318       6.747  37.258   1.208  1.00 57.93           O  \\nHETATM 1493  O   HOH A 319       4.747  42.419  12.683  1.00 57.74           O  \\nHETATM 1494  O   HOH A 320       2.984  41.950  16.191  1.00 41.69           O  \\nHETATM 1495  O   HOH A 321       3.867  49.208  12.056  1.00 24.57           O  \\nHETATM 1496  O   HOH A 322       7.530  42.894  17.776  1.00 79.28           O  \\nHETATM 1497  O   HOH A 324      19.822  39.457  17.826  1.00 66.22           O  \\nHETATM 1498  O   HOH A 325      12.391  45.557  24.973  1.00 48.70           O  \\nHETATM 1499  O   HOH A 326      14.944  43.928  23.794  1.00 62.45           O  \\nHETATM 1500  O   HOH A 327     -12.984  24.008  12.563  1.00 40.31           O  \\nHETATM 1501  O   HOH A 330       0.572  41.133  19.277  1.00 50.77           O  \\nHETATM 1502  O   HOH A 331      -6.196  37.979  23.043  1.00 48.93           O  \\nHETATM 1503  O   HOH A 332      -3.526  30.410  19.065  1.00 21.26           O  \\nHETATM 1504  O   HOH A 333      -7.995  32.093  21.685  1.00 54.11           O  \\nHETATM 1505  O   HOH A 334      -8.483  28.891  20.063  1.00 70.37           O  \\nHETATM 1506  O   HOH A 335     -11.670  31.022  16.981  1.00 76.86           O  \\nHETATM 1507  O   HOH A 336     -11.082  34.450  18.910  1.00 43.07           O  \\nHETATM 1508  O   HOH A 339      -6.521  37.738  15.165  1.00 64.11           O  \\nHETATM 1509  O   HOH A 340     -10.829  35.877  15.629  1.00 58.36           O  \\nHETATM 1510  O   HOH A 341      -6.552  41.166  12.231  1.00 55.67           O  \\nHETATM 1511  O   HOH A 342      -1.242  39.671  12.185  1.00 61.04           O  \\nHETATM 1512  O   HOH A 343      -4.281  38.934  13.464  1.00 32.14           O  \\nHETATM 1513  O   HOH A 344       0.386  41.208  14.039  1.00 40.80           O  \\nHETATM 1514  O   HOH A 345       0.777  39.663  21.603  1.00 67.71           O  \\nHETATM 1515  O   HOH A 346      -7.879  34.848  19.044  1.00 52.60           O  \\nHETATM 1516  O   HOH A 347       1.121  32.594 -11.957  1.00 32.30           O  \\nHETATM 1517  O   HOH A 348       0.924  32.597 -14.716  1.00 48.34           O  \\nHETATM 1518  O   HOH A 349      -2.241  27.348  20.565  1.00 26.54           O  \\nHETATM 1519  O   HOH A 350       3.259  26.214  29.881  1.00 69.22           O  \\nHETATM 1520  O   HOH A 351      -8.318  34.095  28.392  1.00 81.96           O  \\nHETATM 1521  O   HOH A 352      -2.927  29.775  31.412  1.00 80.92           O  \\nHETATM 1522  O   HOH A 353     -10.410  32.034  25.972  1.00 60.17           O  \\nHETATM 1523  O   HOH A 354      -8.122  30.393  25.378  1.00 62.33           O  \\nHETATM 1524  O   HOH A 356      -6.138  18.882  24.644  1.00 49.26           O  \\nHETATM 1525  O   HOH A 358      -1.406  19.033  23.084  1.00 82.05           O  \\nHETATM 1526  O   HOH A 359       4.169   7.339  10.280  1.00 66.03           O  \\nHETATM 1527  O   HOH A 360      -9.761  30.165   7.702  1.00 46.15           O  \\nHETATM 1528  O   HOH A 361     -16.002  27.801  11.700  1.00 41.85           O  \\nHETATM 1529  O   HOH A 362      16.215  30.489  29.264  1.00 67.21           O  \\nHETATM 1530  O   HOH A 364      21.417  27.685  26.140  1.00 49.57           O  \\nHETATM 1531  O   HOH A 366      15.824  34.767  30.662  1.00 25.90           O  \\nHETATM 1532  O   HOH A 367      18.527  35.965  24.111  1.00 68.69           O  \\nHETATM 1533  O   HOH A 369      12.103  35.275  24.945  1.00 51.37           O  \\nHETATM 1534  O   HOH A 370      24.782  11.970  23.641  1.00 55.92           O  \\nHETATM 1535  O   HOH A 371      14.337  21.394  28.627  1.00 30.11           O  \\nHETATM 1536  O   HOH A 372      -1.367  12.600  23.053  1.00 44.43           O  \\nHETATM 1537  O   HOH A 373       7.060   8.567  21.416  1.00 47.99           O  \\nHETATM 1538  O   HOH A 374       7.639   8.040  16.420  1.00 38.76           O  \\nHETATM 1539  O   HOH A 376      -1.282  15.566  25.487  1.00 67.79           O  \\nHETATM 1540  O   HOH A 378      17.945  30.924  23.806  1.00 31.50           O  \\nHETATM 1541  O   HOH A 379      17.360  33.209  33.337  1.00 30.46           O  \\nHETATM 1542  O   HOH A 380       3.971   6.567  22.089  1.00 86.80           O  \\nHETATM 1543  O   HOH A 381      -0.065  10.022  26.928  1.00 61.86           O  \\nHETATM 1544  O   HOH A 382      -4.450  12.966  21.222  1.00 42.82           O  \\nHETATM 1545  O   HOH A 383      -2.584   6.719   5.938  1.00 40.64           O  \\nHETATM 1546  O   HOH A 384      25.567  31.936  12.845  1.00 73.17           O  \\nHETATM 1547  O   HOH A 385      24.446  29.383  22.970  1.00 36.61           O  \\nHETATM 1548  O   HOH A 386      24.865  38.167  18.227  1.00 60.74           O  \\nHETATM 1549  O   HOH A 387      23.135  35.012  14.543  1.00 59.19           O  \\nHETATM 1550  O   HOH A 388      19.325   9.556  18.093  1.00 92.18           O  \\nHETATM 1551  O   HOH A 389      25.326  12.873  16.251  1.00 49.27           O  \\nHETATM 1552  O   HOH A 390      20.219  10.405  10.956  1.00 63.05           O  \\nHETATM 1553  O   HOH A 392       9.452   5.453   8.249  1.00 47.81           O  \\nHETATM 1554  O   HOH A 394      10.139  12.252   0.254  1.00 29.37           O  \\nHETATM 1555  O   HOH A 395      28.004  37.416  13.466  1.00 52.29           O  \\nHETATM 1556  O   HOH A 396      24.283  36.829  12.307  1.00 64.74           O  \\nHETATM 1557  O   HOH A 397      13.748  34.031  -3.019  1.00 47.80           O  \\nHETATM 1558  O   HOH A 398      16.981  34.189   2.006  1.00 80.51           O  \\nHETATM 1559  O   HOH A 399      -4.426  24.223  -7.240  1.00 38.09           O  \\nHETATM 1560  O   HOH A 400      19.676  33.981   3.336  1.00 64.86           O  \\nHETATM 1561  O   HOH A 401      14.633  28.745  -0.317  1.00 66.25           O  \\nHETATM 1562  O   HOH A 402      -2.139  15.576  22.716  1.00 65.01           O  \\nHETATM 1563  O   HOH A 403       9.873  11.090  20.760  1.00 53.88           O  \\nHETATM 1564  O   HOH A 404      18.291  36.379  26.927  1.00 57.23           O  \\nHETATM 1565  O   HOH A 405       0.138   6.145  15.269  1.00 82.80           O  \\nHETATM 1566  O   HOH A 406      -5.515  38.166  18.689  1.00 48.28           O  \\nHETATM 1567  O   HOH A 407      -0.399   4.688   8.809  1.00 58.35           O  \\nHETATM 1568  O   HOH A 408       0.813  25.584  22.307  1.00 40.04           O  \\nCONECT  117 1325                                                                \\nCONECT  265 1325                                                                \\nCONECT 1325  117  265 1328 1333                                                 \\nCONECT 1325 1361 1362                                                           \\nCONECT 1326 1327 1328 1329 1330                                                 \\nCONECT 1327 1326                                                                \\nCONECT 1328 1325 1326                                                           \\nCONECT 1329 1326                                                                \\nCONECT 1330 1326 1331                                                           \\nCONECT 1331 1330 1332 1333 1334                                                 \\nCONECT 1332 1331                                                                \\nCONECT 1333 1325 1331                                                           \\nCONECT 1334 1331 1335                                                           \\nCONECT 1335 1334 1336 1337 1338                                                 \\nCONECT 1336 1335                                                                \\nCONECT 1337 1335                                                                \\nCONECT 1338 1335 1339                                                           \\nCONECT 1339 1338 1340                                                           \\nCONECT 1340 1339 1341 1342                                                      \\nCONECT 1341 1340 1346                                                           \\nCONECT 1342 1340 1343 1344                                                      \\nCONECT 1343 1342                                                                \\nCONECT 1344 1342 1345 1346                                                      \\nCONECT 1345 1344                                                                \\nCONECT 1346 1341 1344 1347                                                      \\nCONECT 1347 1346 1348 1357                                                      \\nCONECT 1348 1347 1349                                                           \\nCONECT 1349 1348 1350                                                           \\nCONECT 1350 1349 1351 1357                                                      \\nCONECT 1351 1350 1352 1353                                                      \\nCONECT 1352 1351                                                                \\nCONECT 1353 1351 1354                                                           \\nCONECT 1354 1353 1355 1356                                                      \\nCONECT 1355 1354                                                                \\nCONECT 1356 1354 1357                                                           \\nCONECT 1357 1347 1350 1356                                                      \\nCONECT 1361 1325                                                                \\nCONECT 1362 1325                                                                \\nMASTER      312    0    2    5    6    0    9    6 1567    1   38   13          \\nEND                                                                             \\n\",\"pdb\");\n\tviewergrid_16835771918631198[1][0].addModel(\"\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n   -9.4114   12.4715    7.2804 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4411   12.3024    6.2602 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7949   13.4149    5.7632 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8582   13.2305    4.7753 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5557   11.9767    4.2780 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.7162   11.5413    3.3522 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.7686   10.2482    3.2133 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6926    9.8146    4.0964 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.1966   10.8621    4.7693 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1359   11.0475    5.7603 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.6142   13.3721    7.6225 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.8823   11.6882    7.6466 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0255    9.4196    2.3278 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.3159    8.4875    2.2011 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8700    9.8820    1.6098 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.8639    8.7148    1.5301 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.6979    7.8986    2.5605 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8107    6.8678    2.4813 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.1439    7.0309    0.8493 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.1103    8.4783    0.3910 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.21977\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n   -5.9911    6.7304   12.3325 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9191    6.9212   11.2774 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.2647    6.7642   11.5359 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.1479    6.9534   10.5007 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.7239    7.2913    9.2292 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.3552    7.5235    8.0893 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.5317    7.8152    7.1245 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2947    7.7714    7.6628 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3827    7.4498    8.9640 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.4972    7.2600   10.0026 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3110    6.4898   13.2320 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0269    6.8384   12.1656 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.8272    8.1235    5.7679 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.8559    7.3880    5.1141 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.0857    9.4602    5.3088 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7450   10.0707    4.8498 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4635   11.3409    5.1007 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2760   11.8718    4.6959 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4727   10.5094    3.9001 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7969    9.3211    4.1710 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.13625\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n   -9.2752   13.0914    6.8146 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4983   12.1348    6.1129 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.8716   10.8079    6.1603 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1063    9.8965    5.4740 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9907   10.2745    4.7505 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1083    9.6096    4.0219 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.2035   10.3945    3.5126 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.5111   11.6404    3.9310 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6124   11.5968    4.6987 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3803   12.5106    5.3872 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.0666   12.8041    7.3249 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.0231   14.0428    6.7916 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.1003   10.0556    2.6814 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.5163    9.3118    2.9553 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.7997   10.7501    1.4601 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9148    9.8273    0.5962 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.1986    9.6277   -0.6825 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.4103    8.8148   -1.4399 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.1690    8.2752   -0.2986 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8029    9.1854    1.1187 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.05286\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n    3.8277   12.8535    0.7057 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8945   13.2451   -0.1425 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0823   14.5865   -0.4024 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1212   14.9457   -1.2265 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9663   14.0083   -1.7903 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0127   14.0920   -2.5962 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.5106   12.9221   -2.8747 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.7484   12.0247   -2.2149 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7848   12.6680   -1.5353 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7427   12.3081   -0.7086 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2355   13.5350    1.0984 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6791   11.9003    0.9018 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.6260   12.6053   -3.6985 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.1601   11.8076   -3.4803 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.0081   13.3972   -4.8348 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.9410   13.1976   -5.9313 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.7807   12.0148   -6.5063 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8337   11.8451   -7.4709 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.0965   13.4495   -7.6023 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.1191   14.2339   -6.3461 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.03419\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n    3.8796   12.8280    0.5341 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6839   13.3276   -0.5216 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3512   14.5215   -0.3442 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1254   14.9888   -1.3785 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2471   14.3003   -2.5709 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9120   14.5294   -3.6921 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7287   13.5869   -4.5709 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8988   12.6931   -3.9929 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5842   13.1083   -2.7548 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8078   12.6399   -1.7171 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8044   13.3319    1.3765 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.3963   11.9773    0.4245 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2696   13.4781   -5.8818 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.9442   14.1354   -6.1681 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.8756   12.4518   -6.8069 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5310   13.1310   -8.1489 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.8433   14.4013   -8.3593 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5395   14.9931   -9.5482 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7557   13.6934  -10.4598 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8944   12.4447   -9.1711 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-2.95533\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n    3.7671   12.7959    0.6769 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8542   13.1811   -0.1483 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0647   14.5222   -0.3915 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1230   14.8752   -1.1933 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9653   13.9318   -1.7512 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0271   14.0091   -2.5375 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.5142   12.8353   -2.8186 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.7282   11.9420   -2.1814 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7614   12.5918   -1.5127 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.6998   12.2382   -0.7083 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1773   13.4816    1.0657 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6023   11.8429    0.8609 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.6398   12.5112   -3.6254 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.7621   12.9910   -4.4764 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.6064   11.5151   -3.2543 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   12.0024   12.1710   -3.2972 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   12.2320   13.3064   -2.6541 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   13.4683   13.8776   -2.6932 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   14.4000   12.7594   -3.7013 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   13.0544   11.6015   -3.9974 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-2.89640\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n    4.0202   13.0896    0.3514 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8424   13.4745   -0.7380 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5672   14.6443   -0.6461 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3577   15.0000   -1.7119 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4401   14.2246   -2.8532 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1096   14.3421   -3.9888 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.8767   13.3503   -4.7990 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.0078   12.5403   -4.1582 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7197   13.0560   -2.9519 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9270   12.6995   -1.8826 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.9736   13.6543    1.1565 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4961   12.2577    0.3027 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4052   13.1239   -6.0999 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.3793   13.1849   -6.2289 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5811   12.8100   -7.2343 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7451   11.3058   -7.5368 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7932   10.6405   -8.1745 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9468    9.3134   -8.4417 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.5376    8.9435   -7.7581 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8812   10.6062   -7.1611 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-2.74515\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n   12.9015    6.6939    2.9198 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.8323    7.6012    2.7084 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.6940    8.6789    3.5579 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.6520    9.5464    3.3367 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.7560    9.3652    2.2998 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.6955   10.0385    1.8832 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.1434    9.4863    0.8419 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.8831    8.3924    0.5625 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.8880    8.2921    1.4481 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.9333    7.4222    1.6704 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   13.5302    6.8387    3.6635 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   13.0151    5.9193    2.3229 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9971    9.9109    0.1148 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1080    9.7643    0.5112 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.0786   10.5444   -1.1722 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4416   11.9446   -1.0518 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1787   12.0786   -0.6741 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6243   13.3184   -0.5662 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9515   14.3975   -1.0230 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1593   13.0982   -1.3267 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-2.73742\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n   -8.9631   12.4957    7.1107 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1675   11.9876    6.0525 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.5706   12.8747    5.1814 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8038   12.3631    4.1627 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6216   11.0031    3.9957 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.9490   10.2743    3.1192 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0633    8.9990    3.3532 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8534    8.8880    4.4418 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2141   10.1120    4.8612 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9830   10.6256    5.8828 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.0820   13.4674    7.2144 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.3965   11.8766    7.7417 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4922    7.9105    2.6376 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6511    7.8503    1.6679 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.6940    6.8948    3.2666 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6150    5.6984    3.5848 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5858    5.3565    2.7504 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3924    4.2965    3.0362 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7654    3.7159    4.5868 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4596    4.9452    4.7380 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-2.73579\\n\\n$$$$\\n\",\"sdf\");\n\tviewergrid_16835771918631198[1][0].setStyle({\"model\": 0},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewergrid_16835771918631198[1][0].addSurface(1,{\"opacity\": 0.5, \"color\": \"white\"},{\"model\": 0});\n\tviewergrid_16835771918631198[1][0].setStyle({\"model\": 1},{\"stick\": {}});\n\tviewergrid_16835771918631198[1][0].zoomTo();\n\tviewergrid_16835771918631198[1][1].addModel(\"HEADER    ONCOGENE PROTEIN                        30-APR-90   5P21              \\nTITLE     REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION OF H-RAS   \\nTITLE    2 P21 AT 1.35 ANGSTROMS RESOLUTION: IMPLICATIONS FOR THE MECHANISM OF  \\nTITLE    3 GTP HYDROLYSIS                                                       \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: C-H-RAS P21 PROTEIN;                                       \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    ONCOGENE PROTEIN                                                      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    E.F.PAI,A.WITTINGHOFER,W.KABSCH                                       \\nREVDAT   6   29-NOV-17 5P21    1       HELIX                                    \\nREVDAT   5   24-FEB-09 5P21    1       VERSN                                    \\nREVDAT   4   01-APR-03 5P21    1       JRNL                                     \\nREVDAT   3   20-JUL-95 5P21    1       HET                                      \\nREVDAT   2   30-APR-94 5P21    1       FORMUL                                   \\nREVDAT   1   15-JAN-92 5P21    0                                                \\nJRNL        AUTH   E.F.PAI,U.KRENGEL,G.A.PETSKO,R.S.GOODY,W.KABSCH,             \\nJRNL        AUTH 2 A.WITTINGHOFER                                               \\nJRNL        TITL   REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION   \\nJRNL        TITL 2 OF H-RAS P21 AT 1.35 A RESOLUTION: IMPLICATIONS FOR THE      \\nJRNL        TITL 3 MECHANISM OF GTP HYDROLYSIS.                                 \\nJRNL        REF    EMBO J.                       V.   9  2351 1990              \\nJRNL        REFN                   ISSN 0261-4189                               \\nJRNL        PMID   2196171                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,S.C.ALMO,G.RAPP,K.WILSON,K.PETRATOS,A.LENTFER, \\nREMARK   1  AUTH 2 A.WITTINGHOFER,W.KABSCH,E.F.PAI,G.A.PETSKO,R.S.GOODY         \\nREMARK   1  TITL   TIME-RESOLVED X-RAY CRYSTALLOGRAPHIC STUDY OF THE            \\nREMARK   1  TITL 2 CONFORMATIONAL CHANGE IN HA-RAS P21 PROTEIN ON GTP           \\nREMARK   1  TITL 3 HYDROLYSIS                                                   \\nREMARK   1  REF    NATURE                        V. 345   309 1990              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   A.SCHERER,J.JOHN,R.LINKE,R.S.GOODY,A.WITTINGHOFER,E.F.PAI,   \\nREMARK   1  AUTH 2 K.C.HOLMES                                                   \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY ANALYSIS OF THE HUMAN  \\nREMARK   1  TITL 2 C-H-RAS-ONCOGENE PRODUCT P21 COMPLEXED WITH GTP ANALOGUES    \\nREMARK   1  REF    J.MOL.BIOL.                   V. 206   257 1989              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   E.F.PAI,W.KABSCH,U.KRENGEL,K.C.HOLMES,J.JOHN,A.WITTINGHOFER  \\nREMARK   1  TITL   STRUCTURE OF THE GUANINE-NUCLEOTIDE-BINDING DOMAIN OF THE    \\nREMARK   1  TITL 2 HA-RAS ONCOGENE PRODUCT P21 IN THE TRIPHOSPHATE CONFORMATION \\nREMARK   1  REF    NATURE                        V. 341   209 1989              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,G.RAPP,J.JOHN,A.WITTINGHOFER,E.F.PAI,R.S.GOODY \\nREMARK   1  TITL   BIOCHEMICAL AND CRYSTALLOGRAPHIC CHARACTERIZATION OF A       \\nREMARK   1  TITL 2 COMPLEX OF C-HA-RAS P21 AND CAGED GTP WITH FLASH PHOTOLYSIS  \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  86  7687 1989              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 6.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 26806                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.196                           \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1323                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 33                                      \\nREMARK   3   SOLVENT ATOMS            : 211                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.014                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 2.700                           \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  RESIDUES 61 - 65 (GLN - GLU - GLU - TYR - SER) ADOPT                \\nREMARK   3  SEVERAL CONFORMATIONS IN THE CRYSTAL.  THE COORDINATES              \\nREMARK   3  GIVEN APPROXIMATE ONE OF THESE.                                     \\nREMARK   4                                                                      \\nREMARK   4 5P21 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179747.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 38.93                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.01                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 32 2 1                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+2/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+1/3                                           \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z+1/3                                           \\nREMARK 290       6555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000      108.13333            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       54.06667            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       54.06667            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000      108.13333            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025  0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    ARG A  68   NE  -  CZ  -  NH1 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A  97   NE  -  CZ  -  NH1 ANGL. DEV. =  -3.4 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH1 ANGL. DEV. =   4.1 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500    ARG A 128   NE  -  CZ  -  NH2 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A 135   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500    TYR A 157   CB  -  CG  -  CD2 ANGL. DEV. =  -4.8 DEGREES          \\nREMARK 500    ARG A 164   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ASP A  33       75.86   -157.25                                   \\nREMARK 500    ILE A  36      -64.23   -100.39                                   \\nREMARK 500    GLU A  37      123.75   -171.30                                   \\nREMARK 500    SER A  65     -133.70     53.07                                   \\nREMARK 500    LYS A 117       33.80     71.60                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              MG A 168  MG                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 GNP A 167   O2G                                                    \\nREMARK 620 2 GNP A 167   O2B  95.0                                              \\nREMARK 620 3 HOH A 173   O    87.6  91.7                                        \\nREMARK 620 4 SER A  17   OG  172.9  91.0  88.6                                  \\nREMARK 620 5 THR A  35   OG1  91.3 173.4  90.5  82.8                            \\nREMARK 620 6 HOH A 172   O    97.3  88.0 175.0  86.5  89.3                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE MG A 168                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE GNP A 167                 \\nDBREF  5P21 A    1   166  UNP    P01112   RASH_HUMAN       1    166             \\nSEQRES   1 A  166  MET THR GLU TYR LYS LEU VAL VAL VAL GLY ALA GLY GLY          \\nSEQRES   2 A  166  VAL GLY LYS SER ALA LEU THR ILE GLN LEU ILE GLN ASN          \\nSEQRES   3 A  166  HIS PHE VAL ASP GLU TYR ASP PRO THR ILE GLU ASP SER          \\nSEQRES   4 A  166  TYR ARG LYS GLN VAL VAL ILE ASP GLY GLU THR CYS LEU          \\nSEQRES   5 A  166  LEU ASP ILE LEU ASP THR ALA GLY GLN GLU GLU TYR SER          \\nSEQRES   6 A  166  ALA MET ARG ASP GLN TYR MET ARG THR GLY GLU GLY PHE          \\nSEQRES   7 A  166  LEU CYS VAL PHE ALA ILE ASN ASN THR LYS SER PHE GLU          \\nSEQRES   8 A  166  ASP ILE HIS GLN TYR ARG GLU GLN ILE LYS ARG VAL LYS          \\nSEQRES   9 A  166  ASP SER ASP ASP VAL PRO MET VAL LEU VAL GLY ASN LYS          \\nSEQRES  10 A  166  CYS ASP LEU ALA ALA ARG THR VAL GLU SER ARG GLN ALA          \\nSEQRES  11 A  166  GLN ASP LEU ALA ARG SER TYR GLY ILE PRO TYR ILE GLU          \\nSEQRES  12 A  166  THR SER ALA LYS THR ARG GLN GLY VAL GLU ASP ALA PHE          \\nSEQRES  13 A  166  TYR THR LEU VAL ARG GLU ILE ARG GLN HIS                      \\nHET     MG  A 168       1                                                       \\nHET    GNP  A 167      32                                                       \\nHETNAM      MG MAGNESIUM ION                                                    \\nHETNAM     GNP PHOSPHOAMINOPHOSPHONIC ACID-GUANYLATE ESTER                      \\nFORMUL   2   MG    MG 2+                                                        \\nFORMUL   3  GNP    C10 H17 N6 O13 P3                                            \\nFORMUL   4  HOH   *211(H2 O)                                                    \\nHELIX    1  HA LYS A   16  GLN A   25  1                                  10    \\nHELIX    2  HB SER A   65  THR A   74  1RES 65-67 IN 3/10 CONF            10    \\nHELIX    3  HC THR A   87  VAL A  103  1BREAK AT ASP 92                   17    \\nHELIX    4  HD SER A  127  TYR A  137  1                                  11    \\nHELIX    5  HE VAL A  152  ARG A  164  1                                  13    \\nSHEET    1  S1 6 GLU A  37  ILE A  46  0                                        \\nSHEET    2  S1 6 GLU A  49  THR A  58 -1  O  LEU A  53   N  LYS A  42           \\nSHEET    3  S1 6 THR A   2  VAL A   9  1  N  LEU A   6   O  ASP A  54           \\nSHEET    4  S1 6 GLY A  77  ALA A  83  1  N  LEU A  79   O  VAL A   7           \\nSHEET    5  S1 6 MET A 111  ASN A 116  1  N  VAL A 114   O  CYS A  80           \\nSHEET    6  S1 6 TYR A 141  GLU A 143  1  N  ILE A 142   O  LEU A 113           \\nLINK        MG    MG A 168                 O2G GNP A 167     1555   1555  2.20  \\nLINK        MG    MG A 168                 O2B GNP A 167     1555   1555  2.22  \\nLINK        MG    MG A 168                 O   HOH A 173     1555   1555  2.23  \\nLINK        MG    MG A 168                 OG  SER A  17     1555   1555  2.26  \\nLINK        MG    MG A 168                 OG1 THR A  35     1555   1555  2.26  \\nLINK        MG    MG A 168                 O   HOH A 172     1555   1555  2.30  \\nSITE     1 AC1  5 SER A  17  THR A  35  GNP A 167  HOH A 172                    \\nSITE     2 AC1  5 HOH A 173                                                     \\nSITE     1 AC2 28 GLY A  12  GLY A  13  VAL A  14  GLY A  15                    \\nSITE     2 AC2 28 LYS A  16  SER A  17  ALA A  18  PHE A  28                    \\nSITE     3 AC2 28 VAL A  29  ASP A  30  TYR A  32  PRO A  34                    \\nSITE     4 AC2 28 THR A  35  GLY A  60  ASN A 116  LYS A 117                    \\nSITE     5 AC2 28 ASP A 119  SER A 145  ALA A 146  LYS A 147                    \\nSITE     6 AC2 28  MG A 168  HOH A 170  HOH A 172  HOH A 173                    \\nSITE     7 AC2 28 HOH A 175  HOH A 187  HOH A 281  HOH A 366                    \\nCRYST1   40.300   40.300  162.200  90.00  90.00 120.00 P 32 2 1      6          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.024814  0.014326  0.000000        0.00000                         \\nSCALE2      0.000000  0.028653  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.006165        0.00000                         \\nATOM      1  N   MET A   1      -7.186  32.862  -6.632  1.00 13.92           N  \\nATOM      2  CA  MET A   1      -5.921  32.127  -6.684  1.00 17.10           C  \\nATOM      3  C   MET A   1      -5.845  31.045  -5.621  1.00 16.74           C  \\nATOM      4  O   MET A   1      -6.600  31.083  -4.636  1.00 16.60           O  \\nATOM      5  CB  MET A   1      -4.673  33.058  -6.502  1.00 17.00           C  \\nATOM      6  CG  MET A   1      -4.633  33.804  -5.188  1.00 18.08           C  \\nATOM      7  SD  MET A   1      -3.115  34.766  -4.908  1.00 19.73           S  \\nATOM      8  CE  MET A   1      -3.019  35.851  -6.310  1.00 20.41           C  \\nATOM      9  N   THR A   2      -4.910  30.095  -5.820  1.00 14.70           N  \\nATOM     10  CA  THR A   2      -4.661  29.030  -4.859  1.00 16.92           C  \\nATOM     11  C   THR A   2      -4.230  29.655  -3.547  1.00 14.78           C  \\nATOM     12  O   THR A   2      -3.338  30.501  -3.537  1.00 13.93           O  \\nATOM     13  CB  THR A   2      -3.557  28.107  -5.401  1.00 18.03           C  \\nATOM     14  OG1 THR A   2      -4.059  27.565  -6.629  1.00 20.42           O  \\nATOM     15  CG2 THR A   2      -3.193  26.983  -4.443  1.00 18.51           C  \\nATOM     16  N   GLU A   3      -4.911  29.290  -2.472  1.00 14.80           N  \\nATOM     17  CA  GLU A   3      -4.504  29.730  -1.135  1.00 17.71           C  \\nATOM     18  C   GLU A   3      -4.012  28.553  -0.278  1.00 17.33           C  \\nATOM     19  O   GLU A   3      -4.520  27.427  -0.395  1.00 17.54           O  \\nATOM     20  CB  GLU A   3      -5.655  30.402  -0.432  1.00 20.67           C  \\nATOM     21  CG  GLU A   3      -6.213  31.631  -1.186  1.00 25.56           C  \\nATOM     22  CD  GLU A   3      -7.128  32.563  -0.369  1.00 28.46           C  \\nATOM     23  OE1 GLU A   3      -7.578  32.216   0.728  1.00 27.72           O  \\nATOM     24  OE2 GLU A   3      -7.375  33.680  -0.827  1.00 32.49           O  \\nATOM     25  N   TYR A   4      -2.962  28.744   0.542  1.00 14.50           N  \\nATOM     26  CA  TYR A   4      -2.425  27.654   1.358  1.00 12.29           C  \\nATOM     27  C   TYR A   4      -2.534  28.103   2.807  1.00 12.52           C  \\nATOM     28  O   TYR A   4      -2.016  29.143   3.200  1.00 12.11           O  \\nATOM     29  CB  TYR A   4      -0.975  27.385   1.026  1.00 11.46           C  \\nATOM     30  CG  TYR A   4      -0.726  26.846  -0.387  1.00 12.38           C  \\nATOM     31  CD1 TYR A   4      -0.297  27.698  -1.395  1.00 13.00           C  \\nATOM     32  CD2 TYR A   4      -0.933  25.501  -0.649  1.00 14.09           C  \\nATOM     33  CE1 TYR A   4      -0.070  27.208  -2.674  1.00 13.89           C  \\nATOM     34  CE2 TYR A   4      -0.711  24.993  -1.917  1.00 14.65           C  \\nATOM     35  CZ  TYR A   4      -0.279  25.859  -2.917  1.00 14.84           C  \\nATOM     36  OH  TYR A   4      -0.084  25.357  -4.191  1.00 15.07           O  \\nATOM     37  N   LYS A   5      -3.274  27.305   3.568  1.00 13.07           N  \\nATOM     38  CA  LYS A   5      -3.554  27.569   4.970  1.00 12.96           C  \\nATOM     39  C   LYS A   5      -2.556  26.818   5.852  1.00 12.74           C  \\nATOM     40  O   LYS A   5      -2.645  25.611   6.092  1.00 11.58           O  \\nATOM     41  CB  LYS A   5      -4.972  27.131   5.281  1.00 17.65           C  \\nATOM     42  CG  LYS A   5      -6.053  27.833   4.446  1.00 23.49           C  \\nATOM     43  CD  LYS A   5      -7.186  28.310   5.335  1.00 29.10           C  \\nATOM     44  CE  LYS A   5      -6.655  29.309   6.403  1.00 33.68           C  \\nATOM     45  NZ  LYS A   5      -6.597  28.739   7.755  1.00 36.52           N  \\nATOM     46  N   LEU A   6      -1.543  27.574   6.297  1.00 13.08           N  \\nATOM     47  CA  LEU A   6      -0.418  27.011   7.064  1.00 10.34           C  \\nATOM     48  C   LEU A   6      -0.578  27.382   8.535  1.00  9.68           C  \\nATOM     49  O   LEU A   6      -1.034  28.483   8.863  1.00 11.31           O  \\nATOM     50  CB  LEU A   6       0.910  27.571   6.536  1.00 11.34           C  \\nATOM     51  CG  LEU A   6       1.137  27.565   5.017  1.00 11.62           C  \\nATOM     52  CD1 LEU A   6       2.515  28.119   4.720  1.00 13.60           C  \\nATOM     53  CD2 LEU A   6       0.973  26.171   4.475  1.00 12.19           C  \\nATOM     54  N   VAL A   7      -0.196  26.486   9.432  1.00  9.93           N  \\nATOM     55  CA  VAL A   7      -0.333  26.787  10.852  1.00 10.06           C  \\nATOM     56  C   VAL A   7       1.033  26.513  11.495  1.00 10.37           C  \\nATOM     57  O   VAL A   7       1.597  25.441  11.252  1.00  9.51           O  \\nATOM     58  CB  VAL A   7      -1.474  25.884  11.446  1.00 10.97           C  \\nATOM     59  CG1 VAL A   7      -1.610  26.163  12.943  1.00 11.39           C  \\nATOM     60  CG2 VAL A   7      -2.809  26.161  10.777  1.00 10.14           C  \\nATOM     61  N   VAL A   8       1.576  27.455  12.295  1.00  8.35           N  \\nATOM     62  CA  VAL A   8       2.878  27.262  12.944  1.00  6.92           C  \\nATOM     63  C   VAL A   8       2.609  26.896  14.405  1.00  8.84           C  \\nATOM     64  O   VAL A   8       1.930  27.663  15.092  1.00  9.21           O  \\nATOM     65  CB  VAL A   8       3.684  28.565  12.799  1.00  8.08           C  \\nATOM     66  CG1 VAL A   8       5.111  28.324  13.261  1.00  9.52           C  \\nATOM     67  CG2 VAL A   8       3.729  29.011  11.358  1.00  8.43           C  \\nATOM     68  N   VAL A   9       3.024  25.701  14.881  1.00  8.73           N  \\nATOM     69  CA  VAL A   9       2.779  25.234  16.245  1.00  8.41           C  \\nATOM     70  C   VAL A   9       4.075  24.806  16.919  1.00  8.21           C  \\nATOM     71  O   VAL A   9       5.093  24.551  16.287  1.00 10.97           O  \\nATOM     72  CB  VAL A   9       1.824  24.028  16.286  1.00  9.74           C  \\nATOM     73  CG1 VAL A   9       0.545  24.491  15.680  1.00 11.76           C  \\nATOM     74  CG2 VAL A   9       2.349  22.810  15.535  1.00 10.31           C  \\nATOM     75  N   GLY A  10       4.048  24.707  18.238  1.00  7.34           N  \\nATOM     76  CA  GLY A  10       5.231  24.336  19.009  1.00  7.98           C  \\nATOM     77  C   GLY A  10       5.192  25.053  20.355  1.00  8.31           C  \\nATOM     78  O   GLY A  10       4.417  25.994  20.595  1.00  9.89           O  \\nATOM     79  N   ALA A  11       6.099  24.638  21.226  1.00  9.31           N  \\nATOM     80  CA  ALA A  11       6.121  25.135  22.597  1.00  9.50           C  \\nATOM     81  C   ALA A  11       6.440  26.625  22.711  1.00  9.66           C  \\nATOM     82  O   ALA A  11       6.998  27.250  21.806  1.00  9.46           O  \\nATOM     83  CB  ALA A  11       7.145  24.306  23.378  1.00  9.85           C  \\nATOM     84  N   GLY A  12       6.111  27.227  23.847  1.00 10.36           N  \\nATOM     85  CA  GLY A  12       6.406  28.632  24.054  1.00  9.30           C  \\nATOM     86  C   GLY A  12       7.892  28.950  23.964  1.00 10.23           C  \\nATOM     87  O   GLY A  12       8.785  28.249  24.466  1.00 10.57           O  \\nATOM     88  N   GLY A  13       8.159  30.064  23.266  1.00 11.49           N  \\nATOM     89  CA  GLY A  13       9.511  30.602  23.139  1.00 11.29           C  \\nATOM     90  C   GLY A  13      10.436  29.950  22.108  1.00 12.22           C  \\nATOM     91  O   GLY A  13      11.624  30.321  22.060  1.00 13.62           O  \\nATOM     92  N   VAL A  14       9.961  28.991  21.286  1.00  9.23           N  \\nATOM     93  CA  VAL A  14      10.848  28.327  20.317  1.00 10.38           C  \\nATOM     94  C   VAL A  14      11.169  29.168  19.076  1.00 10.47           C  \\nATOM     95  O   VAL A  14      12.108  28.841  18.353  1.00 12.07           O  \\nATOM     96  CB  VAL A  14      10.246  26.970  19.856  1.00  9.98           C  \\nATOM     97  CG1 VAL A  14      10.187  26.074  21.120  1.00  9.93           C  \\nATOM     98  CG2 VAL A  14       8.866  27.127  19.200  1.00  9.52           C  \\nATOM     99  N   GLY A  15      10.423  30.264  18.834  1.00  9.61           N  \\nATOM    100  CA  GLY A  15      10.666  31.160  17.706  1.00  9.70           C  \\nATOM    101  C   GLY A  15       9.615  31.061  16.626  1.00  8.82           C  \\nATOM    102  O   GLY A  15       9.915  31.403  15.491  1.00  8.99           O  \\nATOM    103  N   LYS A  16       8.395  30.591  16.916  1.00  8.18           N  \\nATOM    104  CA  LYS A  16       7.331  30.521  15.913  1.00  8.44           C  \\nATOM    105  C   LYS A  16       7.060  31.899  15.309  1.00  9.01           C  \\nATOM    106  O   LYS A  16       7.020  32.044  14.086  1.00  8.25           O  \\nATOM    107  CB  LYS A  16       6.014  30.000  16.529  1.00  9.27           C  \\nATOM    108  CG  LYS A  16       6.132  28.585  17.086  1.00  9.11           C  \\nATOM    109  CD  LYS A  16       4.820  28.077  17.726  1.00  9.31           C  \\nATOM    110  CE  LYS A  16       4.351  28.934  18.888  1.00 10.05           C  \\nATOM    111  NZ  LYS A  16       5.221  28.829  20.051  1.00  9.64           N  \\nATOM    112  N   SER A  17       6.913  32.942  16.135  1.00  7.74           N  \\nATOM    113  CA  SER A  17       6.661  34.295  15.649  1.00  8.78           C  \\nATOM    114  C   SER A  17       7.854  34.859  14.919  1.00  9.73           C  \\nATOM    115  O   SER A  17       7.653  35.446  13.868  1.00  9.70           O  \\nATOM    116  CB  SER A  17       6.360  35.264  16.769  1.00  8.22           C  \\nATOM    117  OG  SER A  17       5.083  34.853  17.204  1.00  8.79           O  \\nATOM    118  N   ALA A  18       9.072  34.687  15.444  1.00  8.95           N  \\nATOM    119  CA  ALA A  18      10.253  35.199  14.769  1.00  8.95           C  \\nATOM    120  C   ALA A  18      10.438  34.576  13.374  1.00  8.54           C  \\nATOM    121  O   ALA A  18      10.832  35.254  12.430  1.00  8.75           O  \\nATOM    122  CB  ALA A  18      11.521  34.931  15.640  1.00 10.11           C  \\nATOM    123  N   LEU A  19      10.136  33.284  13.176  1.00  8.31           N  \\nATOM    124  CA  LEU A  19      10.225  32.667  11.837  1.00  9.30           C  \\nATOM    125  C   LEU A  19       9.204  33.281  10.884  1.00 10.98           C  \\nATOM    126  O   LEU A  19       9.534  33.652   9.751  1.00 10.47           O  \\nATOM    127  CB  LEU A  19       9.938  31.165  11.885  1.00  9.41           C  \\nATOM    128  CG  LEU A  19      11.035  30.275  12.487  1.00 11.80           C  \\nATOM    129  CD1 LEU A  19      10.491  28.866  12.723  1.00 10.43           C  \\nATOM    130  CD2 LEU A  19      12.256  30.288  11.554  1.00 13.42           C  \\nATOM    131  N   THR A  20       7.950  33.419  11.335  1.00 10.72           N  \\nATOM    132  CA  THR A  20       6.888  34.015  10.529  1.00 11.87           C  \\nATOM    133  C   THR A  20       7.173  35.492  10.161  1.00 12.65           C  \\nATOM    134  O   THR A  20       7.012  35.917   9.009  1.00 12.60           O  \\nATOM    135  CB  THR A  20       5.547  33.888  11.312  1.00 13.43           C  \\nATOM    136  OG1 THR A  20       5.299  32.509  11.544  1.00 12.67           O  \\nATOM    137  CG2 THR A  20       4.346  34.410  10.518  1.00 14.64           C  \\nATOM    138  N   ILE A  21       7.615  36.326  11.098  1.00 10.48           N  \\nATOM    139  CA  ILE A  21       7.945  37.704  10.795  1.00 12.96           C  \\nATOM    140  C   ILE A  21       9.197  37.831   9.927  1.00 14.26           C  \\nATOM    141  O   ILE A  21       9.237  38.691   9.042  1.00 15.83           O  \\nATOM    142  CB  ILE A  21       8.090  38.451  12.121  1.00 12.57           C  \\nATOM    143  CG1 ILE A  21       6.691  38.551  12.737  1.00 13.24           C  \\nATOM    144  CG2 ILE A  21       8.631  39.852  11.910  1.00 13.40           C  \\nATOM    145  CD1 ILE A  21       6.636  39.185  14.129  1.00 16.46           C  \\nATOM    146  N   GLN A  22      10.206  36.979  10.092  1.00 13.58           N  \\nATOM    147  CA  GLN A  22      11.330  36.983   9.179  1.00 14.91           C  \\nATOM    148  C   GLN A  22      10.817  36.729   7.736  1.00 17.29           C  \\nATOM    149  O   GLN A  22      11.120  37.485   6.792  1.00 15.91           O  \\nATOM    150  CB  GLN A  22      12.270  35.930   9.736  1.00 15.90           C  \\nATOM    151  CG  GLN A  22      13.686  35.878   9.230  1.00 19.36           C  \\nATOM    152  CD  GLN A  22      14.372  37.217   9.091  1.00 18.08           C  \\nATOM    153  OE1 GLN A  22      14.160  37.984   8.143  1.00 19.23           O  \\nATOM    154  NE2 GLN A  22      15.224  37.492  10.050  1.00 17.40           N  \\nATOM    155  N   LEU A  23       9.944  35.718   7.523  1.00 15.95           N  \\nATOM    156  CA  LEU A  23       9.381  35.475   6.194  1.00 18.85           C  \\nATOM    157  C   LEU A  23       8.539  36.622   5.636  1.00 18.21           C  \\nATOM    158  O   LEU A  23       8.695  36.958   4.469  1.00 19.03           O  \\nATOM    159  CB  LEU A  23       8.498  34.235   6.196  1.00 18.10           C  \\nATOM    160  CG  LEU A  23       7.916  33.797   4.863  1.00 18.64           C  \\nATOM    161  CD1 LEU A  23       8.921  32.994   4.030  1.00 18.03           C  \\nATOM    162  CD2 LEU A  23       6.680  32.991   5.183  1.00 21.34           C  \\nATOM    163  N   ILE A  24       7.634  37.197   6.428  1.00 19.25           N  \\nATOM    164  CA  ILE A  24       6.682  38.198   5.953  1.00 20.23           C  \\nATOM    165  C   ILE A  24       7.316  39.590   5.922  1.00 21.89           C  \\nATOM    166  O   ILE A  24       7.125  40.318   4.943  1.00 22.93           O  \\nATOM    167  CB  ILE A  24       5.399  38.222   6.881  1.00 21.20           C  \\nATOM    168  CG1 ILE A  24       4.664  36.907   6.821  1.00 21.27           C  \\nATOM    169  CG2 ILE A  24       4.423  39.307   6.446  1.00 20.00           C  \\nATOM    170  CD1 ILE A  24       4.128  36.442   5.475  1.00 23.98           C  \\nATOM    171  N   GLN A  25       8.074  39.998   6.953  1.00 21.07           N  \\nATOM    172  CA  GLN A  25       8.520  41.380   7.066  1.00 22.09           C  \\nATOM    173  C   GLN A  25      10.003  41.559   6.962  1.00 20.75           C  \\nATOM    174  O   GLN A  25      10.495  42.677   7.054  1.00 20.26           O  \\nATOM    175  CB  GLN A  25       8.039  41.968   8.398  1.00 23.96           C  \\nATOM    176  CG  GLN A  25       6.548  42.276   8.381  1.00 28.69           C  \\nATOM    177  CD  GLN A  25       5.931  42.574   9.744  1.00 30.92           C  \\nATOM    178  OE1 GLN A  25       6.537  43.143  10.657  1.00 31.89           O  \\nATOM    179  NE2 GLN A  25       4.664  42.169   9.867  1.00 32.35           N  \\nATOM    180  N   ASN A  26      10.764  40.482   6.795  1.00 22.13           N  \\nATOM    181  CA  ASN A  26      12.198  40.642   6.628  1.00 24.67           C  \\nATOM    182  C   ASN A  26      12.936  41.362   7.734  1.00 23.62           C  \\nATOM    183  O   ASN A  26      13.603  42.372   7.521  1.00 25.10           O  \\nATOM    184  CB  ASN A  26      12.522  41.369   5.315  1.00 27.68           C  \\nATOM    185  CG  ASN A  26      12.942  40.351   4.288  1.00 32.80           C  \\nATOM    186  OD1 ASN A  26      12.135  39.699   3.602  1.00 33.77           O  \\nATOM    187  ND2 ASN A  26      14.268  40.203   4.237  1.00 34.26           N  \\nATOM    188  N   HIS A  27      12.764  40.842   8.945  1.00 19.95           N  \\nATOM    189  CA  HIS A  27      13.593  41.266  10.057  1.00 17.99           C  \\nATOM    190  C   HIS A  27      13.441  40.299  11.215  1.00 16.84           C  \\nATOM    191  O   HIS A  27      12.411  39.623  11.344  1.00 14.72           O  \\nATOM    192  CB  HIS A  27      13.241  42.680  10.551  1.00 19.80           C  \\nATOM    193  CG  HIS A  27      11.926  42.908  11.258  1.00 20.77           C  \\nATOM    194  ND1 HIS A  27      11.936  43.314  12.510  1.00 22.17           N  \\nATOM    195  CD2 HIS A  27      10.674  42.768  10.729  1.00 22.32           C  \\nATOM    196  CE1 HIS A  27      10.657  43.430  12.785  1.00 23.86           C  \\nATOM    197  NE2 HIS A  27       9.905  43.110  11.738  1.00 23.97           N  \\nATOM    198  N   PHE A  28      14.516  40.222  12.012  1.00 15.44           N  \\nATOM    199  CA  PHE A  28      14.528  39.413  13.223  1.00 13.89           C  \\nATOM    200  C   PHE A  28      14.026  40.196  14.440  1.00 15.33           C  \\nATOM    201  O   PHE A  28      14.528  41.255  14.786  1.00 16.80           O  \\nATOM    202  CB  PHE A  28      15.938  38.906  13.470  1.00 12.73           C  \\nATOM    203  CG  PHE A  28      16.083  38.150  14.786  1.00 13.35           C  \\nATOM    204  CD1 PHE A  28      15.354  36.990  15.012  1.00 13.80           C  \\nATOM    205  CD2 PHE A  28      16.974  38.615  15.740  1.00 14.54           C  \\nATOM    206  CE1 PHE A  28      15.534  36.310  16.194  1.00 14.45           C  \\nATOM    207  CE2 PHE A  28      17.147  37.925  16.920  1.00 14.61           C  \\nATOM    208  CZ  PHE A  28      16.420  36.768  17.137  1.00 14.65           C  \\nATOM    209  N   VAL A  29      12.970  39.697  15.089  1.00 17.22           N  \\nATOM    210  CA  VAL A  29      12.467  40.313  16.294  1.00 19.11           C  \\nATOM    211  C   VAL A  29      13.127  39.606  17.476  1.00 19.36           C  \\nATOM    212  O   VAL A  29      13.245  38.376  17.605  1.00 18.14           O  \\nATOM    213  CB  VAL A  29      10.894  40.205  16.412  1.00 22.03           C  \\nATOM    214  CG1 VAL A  29      10.334  40.869  15.143  1.00 19.66           C  \\nATOM    215  CG2 VAL A  29      10.377  38.776  16.631  1.00 23.15           C  \\nATOM    216  N   ASP A  30      13.615  40.484  18.324  1.00 20.14           N  \\nATOM    217  CA  ASP A  30      14.229  40.019  19.518  1.00 24.04           C  \\nATOM    218  C   ASP A  30      13.676  40.746  20.721  1.00 25.93           C  \\nATOM    219  O   ASP A  30      14.244  41.709  21.238  1.00 29.26           O  \\nATOM    220  CB  ASP A  30      15.719  40.203  19.443  1.00 27.10           C  \\nATOM    221  CG  ASP A  30      16.389  39.731  20.732  1.00 32.46           C  \\nATOM    222  OD1 ASP A  30      15.878  38.859  21.464  1.00 34.28           O  \\nATOM    223  OD2 ASP A  30      17.458  40.264  21.002  1.00 36.46           O  \\nATOM    224  N   GLU A  31      12.472  40.347  21.076  1.00 26.14           N  \\nATOM    225  CA  GLU A  31      11.922  40.804  22.318  1.00 28.72           C  \\nATOM    226  C   GLU A  31      10.870  39.836  22.749  1.00 26.20           C  \\nATOM    227  O   GLU A  31      10.466  38.931  22.014  1.00 23.14           O  \\nATOM    228  CB  GLU A  31      11.337  42.210  22.204  1.00 33.25           C  \\nATOM    229  CG  GLU A  31      10.350  42.618  21.152  1.00 40.08           C  \\nATOM    230  CD  GLU A  31      10.572  44.107  20.887  1.00 44.71           C  \\nATOM    231  OE1 GLU A  31      10.930  44.444  19.755  1.00 47.50           O  \\nATOM    232  OE2 GLU A  31      10.420  44.928  21.806  1.00 47.89           O  \\nATOM    233  N   TYR A  32      10.550  40.044  24.028  1.00 22.26           N  \\nATOM    234  CA  TYR A  32       9.597  39.196  24.685  1.00 18.83           C  \\nATOM    235  C   TYR A  32       8.204  39.688  24.317  1.00 17.53           C  \\nATOM    236  O   TYR A  32       7.794  40.792  24.676  1.00 18.45           O  \\nATOM    237  CB  TYR A  32       9.909  39.259  26.199  1.00 16.84           C  \\nATOM    238  CG  TYR A  32       9.076  38.309  27.052  1.00 15.64           C  \\nATOM    239  CD1 TYR A  32       9.598  37.096  27.433  1.00 16.17           C  \\nATOM    240  CD2 TYR A  32       7.814  38.686  27.459  1.00 15.54           C  \\nATOM    241  CE1 TYR A  32       8.845  36.251  28.239  1.00 17.86           C  \\nATOM    242  CE2 TYR A  32       7.051  37.861  28.257  1.00 14.40           C  \\nATOM    243  CZ  TYR A  32       7.572  36.635  28.652  1.00 16.47           C  \\nATOM    244  OH  TYR A  32       6.821  35.789  29.485  1.00 13.59           O  \\nATOM    245  N   ASP A  33       7.461  38.841  23.604  1.00 13.43           N  \\nATOM    246  CA  ASP A  33       6.128  39.200  23.141  1.00 14.00           C  \\nATOM    247  C   ASP A  33       5.262  37.955  22.854  1.00 13.56           C  \\nATOM    248  O   ASP A  33       5.024  37.586  21.681  1.00 12.47           O  \\nATOM    249  CB  ASP A  33       6.271  40.072  21.864  1.00 15.92           C  \\nATOM    250  CG  ASP A  33       4.993  40.654  21.269  1.00 18.19           C  \\nATOM    251  OD1 ASP A  33       3.923  40.631  21.879  1.00 21.46           O  \\nATOM    252  OD2 ASP A  33       5.074  41.164  20.167  1.00 20.34           O  \\nATOM    253  N   PRO A  34       4.752  37.283  23.903  1.00 12.04           N  \\nATOM    254  CA  PRO A  34       3.949  36.070  23.788  1.00 11.09           C  \\nATOM    255  C   PRO A  34       2.720  36.248  22.895  1.00 11.76           C  \\nATOM    256  O   PRO A  34       1.953  37.220  23.006  1.00 12.63           O  \\nATOM    257  CB  PRO A  34       3.590  35.711  25.219  1.00 12.50           C  \\nATOM    258  CG  PRO A  34       4.643  36.418  26.033  1.00 12.14           C  \\nATOM    259  CD  PRO A  34       4.817  37.717  25.311  1.00 12.82           C  \\nATOM    260  N   THR A  35       2.554  35.282  21.972  1.00 11.00           N  \\nATOM    261  CA  THR A  35       1.421  35.320  21.033  1.00 11.54           C  \\nATOM    262  C   THR A  35       0.140  34.746  21.639  1.00 13.51           C  \\nATOM    263  O   THR A  35       0.174  33.820  22.450  1.00 12.45           O  \\nATOM    264  CB  THR A  35       1.730  34.518  19.751  1.00 11.34           C  \\nATOM    265  OG1 THR A  35       2.900  35.134  19.233  1.00 11.20           O  \\nATOM    266  CG2 THR A  35       0.706  34.611  18.637  1.00 13.10           C  \\nATOM    267  N   ILE A  36      -0.996  35.332  21.233  1.00 15.48           N  \\nATOM    268  CA  ILE A  36      -2.308  34.792  21.567  1.00 19.42           C  \\nATOM    269  C   ILE A  36      -2.663  34.079  20.274  1.00 17.36           C  \\nATOM    270  O   ILE A  36      -2.749  32.863  20.256  1.00 18.29           O  \\nATOM    271  CB  ILE A  36      -3.353  35.914  21.879  1.00 21.30           C  \\nATOM    272  CG1 ILE A  36      -2.930  36.823  23.072  1.00 24.32           C  \\nATOM    273  CG2 ILE A  36      -4.670  35.207  22.136  1.00 21.54           C  \\nATOM    274  CD1 ILE A  36      -2.494  36.167  24.413  1.00 24.70           C  \\nATOM    275  N   GLU A  37      -2.818  34.822  19.181  1.00 19.35           N  \\nATOM    276  CA  GLU A  37      -3.059  34.221  17.889  1.00 21.47           C  \\nATOM    277  C   GLU A  37      -2.957  35.362  16.897  1.00 23.77           C  \\nATOM    278  O   GLU A  37      -3.636  36.393  17.031  1.00 25.49           O  \\nATOM    279  CB  GLU A  37      -4.440  33.611  17.880  1.00 25.97           C  \\nATOM    280  CG  GLU A  37      -4.766  32.776  16.667  1.00 33.81           C  \\nATOM    281  CD  GLU A  37      -6.160  32.154  16.777  1.00 38.37           C  \\nATOM    282  OE1 GLU A  37      -7.095  32.664  16.149  1.00 37.86           O  \\nATOM    283  OE2 GLU A  37      -6.298  31.164  17.515  1.00 42.24           O  \\nATOM    284  N   ASP A  38      -2.070  35.212  15.911  1.00 19.96           N  \\nATOM    285  CA  ASP A  38      -1.899  36.178  14.839  1.00 17.94           C  \\nATOM    286  C   ASP A  38      -2.024  35.526  13.473  1.00 18.21           C  \\nATOM    287  O   ASP A  38      -1.582  34.393  13.294  1.00 16.96           O  \\nATOM    288  CB  ASP A  38      -0.540  36.833  14.965  1.00 18.83           C  \\nATOM    289  CG  ASP A  38      -0.507  37.817  16.133  1.00 20.08           C  \\nATOM    290  OD1 ASP A  38      -1.390  38.660  16.208  1.00 25.47           O  \\nATOM    291  OD2 ASP A  38       0.375  37.781  16.967  1.00 20.08           O  \\nATOM    292  N   SER A  39      -2.698  36.170  12.507  1.00 17.82           N  \\nATOM    293  CA  SER A  39      -2.691  35.658  11.133  1.00 18.16           C  \\nATOM    294  C   SER A  39      -2.012  36.627  10.171  1.00 16.95           C  \\nATOM    295  O   SER A  39      -2.053  37.853  10.333  1.00 15.80           O  \\nATOM    296  CB  SER A  39      -4.094  35.402  10.637  1.00 23.01           C  \\nATOM    297  OG  SER A  39      -4.873  36.571  10.754  1.00 27.13           O  \\nATOM    298  N   TYR A  40      -1.336  36.080   9.168  1.00 15.44           N  \\nATOM    299  CA  TYR A  40      -0.607  36.889   8.196  1.00 16.25           C  \\nATOM    300  C   TYR A  40      -1.018  36.383   6.830  1.00 17.06           C  \\nATOM    301  O   TYR A  40      -1.160  35.172   6.638  1.00 18.29           O  \\nATOM    302  CB  TYR A  40       0.909  36.724   8.370  1.00 16.59           C  \\nATOM    303  CG  TYR A  40       1.439  37.242   9.707  1.00 16.10           C  \\nATOM    304  CD1 TYR A  40       1.473  36.412  10.815  1.00 16.77           C  \\nATOM    305  CD2 TYR A  40       1.883  38.549   9.799  1.00 18.02           C  \\nATOM    306  CE1 TYR A  40       1.950  36.893  12.021  1.00 16.86           C  \\nATOM    307  CE2 TYR A  40       2.368  39.032  11.001  1.00 18.70           C  \\nATOM    308  CZ  TYR A  40       2.396  38.199  12.104  1.00 17.62           C  \\nATOM    309  OH  TYR A  40       2.853  38.695  13.304  1.00 18.37           O  \\nATOM    310  N   ARG A  41      -1.270  37.303   5.900  1.00 15.48           N  \\nATOM    311  CA  ARG A  41      -1.660  36.918   4.558  1.00 15.42           C  \\nATOM    312  C   ARG A  41      -0.730  37.606   3.586  1.00 16.76           C  \\nATOM    313  O   ARG A  41      -0.536  38.829   3.633  1.00 17.01           O  \\nATOM    314  CB  ARG A  41      -3.087  37.342   4.277  1.00 17.64           C  \\nATOM    315  CG  ARG A  41      -4.150  36.507   4.947  1.00 18.09           C  \\nATOM    316  CD  ARG A  41      -5.528  37.181   4.849  1.00 19.64           C  \\nATOM    317  NE  ARG A  41      -6.066  37.318   3.489  1.00 20.11           N  \\nATOM    318  CZ  ARG A  41      -6.660  36.329   2.803  1.00 22.17           C  \\nATOM    319  NH1 ARG A  41      -7.219  36.571   1.617  1.00 22.48           N  \\nATOM    320  NH2 ARG A  41      -6.711  35.090   3.274  1.00 23.55           N  \\nATOM    321  N   LYS A  42      -0.135  36.827   2.679  1.00 16.52           N  \\nATOM    322  CA  LYS A  42       0.835  37.405   1.758  1.00 16.59           C  \\nATOM    323  C   LYS A  42       0.740  36.746   0.387  1.00 16.04           C  \\nATOM    324  O   LYS A  42       0.794  35.528   0.307  1.00 14.74           O  \\nATOM    325  CB  LYS A  42       2.208  37.210   2.401  1.00 18.22           C  \\nATOM    326  CG  LYS A  42       3.388  37.784   1.669  1.00 22.68           C  \\nATOM    327  CD  LYS A  42       3.369  39.274   1.792  1.00 28.09           C  \\nATOM    328  CE  LYS A  42       4.695  39.876   1.317  1.00 31.70           C  \\nATOM    329  NZ  LYS A  42       4.828  39.827  -0.128  1.00 35.70           N  \\nATOM    330  N   GLN A  43       0.561  37.510  -0.697  1.00 16.89           N  \\nATOM    331  CA  GLN A  43       0.579  36.950  -2.047  1.00 19.10           C  \\nATOM    332  C   GLN A  43       2.026  36.729  -2.448  1.00 15.08           C  \\nATOM    333  O   GLN A  43       2.828  37.654  -2.362  1.00 15.72           O  \\nATOM    334  CB  GLN A  43      -0.116  37.908  -3.043  1.00 23.32           C  \\nATOM    335  CG  GLN A  43       0.543  38.332  -4.415  1.00 32.83           C  \\nATOM    336  CD  GLN A  43       1.081  37.332  -5.482  1.00 36.11           C  \\nATOM    337  OE1 GLN A  43       0.457  36.335  -5.865  1.00 37.78           O  \\nATOM    338  NE2 GLN A  43       2.271  37.606  -6.053  1.00 35.45           N  \\nATOM    339  N   VAL A  44       2.394  35.532  -2.922  1.00 12.79           N  \\nATOM    340  CA  VAL A  44       3.771  35.242  -3.311  1.00 11.03           C  \\nATOM    341  C   VAL A  44       3.724  34.425  -4.605  1.00 14.28           C  \\nATOM    342  O   VAL A  44       2.633  34.109  -5.090  1.00 14.90           O  \\nATOM    343  CB  VAL A  44       4.508  34.411  -2.223  1.00 13.21           C  \\nATOM    344  CG1 VAL A  44       4.669  35.282  -0.977  1.00 15.04           C  \\nATOM    345  CG2 VAL A  44       3.761  33.116  -1.889  1.00 12.89           C  \\nATOM    346  N   VAL A  45       4.879  34.127  -5.235  1.00 13.20           N  \\nATOM    347  CA  VAL A  45       4.938  33.277  -6.419  1.00 12.28           C  \\nATOM    348  C   VAL A  45       5.836  32.161  -5.970  1.00 11.08           C  \\nATOM    349  O   VAL A  45       6.988  32.447  -5.636  1.00 13.13           O  \\nATOM    350  CB  VAL A  45       5.587  33.979  -7.641  1.00 13.01           C  \\nATOM    351  CG1 VAL A  45       5.549  33.031  -8.810  1.00 14.51           C  \\nATOM    352  CG2 VAL A  45       4.787  35.173  -8.108  1.00 13.54           C  \\nATOM    353  N   ILE A  46       5.374  30.907  -5.915  1.00  9.89           N  \\nATOM    354  CA  ILE A  46       6.274  29.795  -5.491  1.00 11.03           C  \\nATOM    355  C   ILE A  46       6.369  28.800  -6.662  1.00  8.85           C  \\nATOM    356  O   ILE A  46       5.319  28.296  -7.059  1.00 10.25           O  \\nATOM    357  CB  ILE A  46       5.692  29.115  -4.203  1.00 10.64           C  \\nATOM    358  CG1 ILE A  46       5.652  30.096  -3.042  1.00 11.20           C  \\nATOM    359  CG2 ILE A  46       6.599  27.953  -3.757  1.00 12.93           C  \\nATOM    360  CD1 ILE A  46       4.937  29.536  -1.808  1.00  8.90           C  \\nATOM    361  N   ASP A  47       7.565  28.495  -7.224  1.00  9.41           N  \\nATOM    362  CA  ASP A  47       7.767  27.650  -8.426  1.00  9.70           C  \\nATOM    363  C   ASP A  47       6.838  28.031  -9.572  1.00 11.14           C  \\nATOM    364  O   ASP A  47       6.225  27.211 -10.248  1.00 14.09           O  \\nATOM    365  CB  ASP A  47       7.566  26.165  -8.105  1.00 10.73           C  \\nATOM    366  CG  ASP A  47       8.388  25.783  -6.874  1.00 13.52           C  \\nATOM    367  OD1 ASP A  47       7.805  25.291  -5.898  1.00 12.04           O  \\nATOM    368  OD2 ASP A  47       9.598  26.024  -6.872  1.00 11.31           O  \\nATOM    369  N   GLY A  48       6.724  29.353  -9.735  1.00 10.78           N  \\nATOM    370  CA  GLY A  48       5.930  29.961 -10.785  1.00 13.29           C  \\nATOM    371  C   GLY A  48       4.426  29.989 -10.548  1.00 14.28           C  \\nATOM    372  O   GLY A  48       3.677  30.512 -11.386  1.00 15.19           O  \\nATOM    373  N   GLU A  49       3.944  29.422  -9.439  1.00 13.41           N  \\nATOM    374  CA  GLU A  49       2.520  29.457  -9.143  1.00 13.90           C  \\nATOM    375  C   GLU A  49       2.223  30.700  -8.314  1.00 15.34           C  \\nATOM    376  O   GLU A  49       2.860  30.906  -7.275  1.00 14.56           O  \\nATOM    377  CB  GLU A  49       2.124  28.238  -8.340  1.00 15.50           C  \\nATOM    378  CG  GLU A  49       0.630  28.250  -7.990  1.00 18.10           C  \\nATOM    379  CD  GLU A  49       0.204  27.110  -7.100  1.00 19.13           C  \\nATOM    380  OE1 GLU A  49       0.717  27.004  -5.996  1.00 19.53           O  \\nATOM    381  OE2 GLU A  49      -0.630  26.321  -7.529  1.00 22.12           O  \\nATOM    382  N   THR A  50       1.267  31.536  -8.734  1.00 14.06           N  \\nATOM    383  CA  THR A  50       0.901  32.697  -7.927  1.00 16.15           C  \\nATOM    384  C   THR A  50      -0.001  32.169  -6.819  1.00 13.35           C  \\nATOM    385  O   THR A  50      -0.938  31.409  -7.046  1.00 15.13           O  \\nATOM    386  CB  THR A  50       0.203  33.755  -8.846  1.00 17.88           C  \\nATOM    387  OG1 THR A  50      -0.245  34.813  -8.012  1.00 24.33           O  \\nATOM    388  CG2 THR A  50      -1.028  33.236  -9.518  1.00 19.78           C  \\nATOM    389  N   CYS A  51       0.306  32.442  -5.563  1.00 13.28           N  \\nATOM    390  CA  CYS A  51      -0.566  31.959  -4.508  1.00 15.16           C  \\nATOM    391  C   CYS A  51      -0.579  32.892  -3.303  1.00 14.00           C  \\nATOM    392  O   CYS A  51       0.161  33.874  -3.231  1.00 11.81           O  \\nATOM    393  CB  CYS A  51      -0.132  30.537  -4.079  1.00 14.08           C  \\nATOM    394  SG  CYS A  51       1.597  30.409  -3.575  1.00 16.09           S  \\nATOM    395  N   LEU A  52      -1.521  32.609  -2.405  1.00 13.77           N  \\nATOM    396  CA  LEU A  52      -1.573  33.397  -1.199  1.00 16.01           C  \\nATOM    397  C   LEU A  52      -1.260  32.497  -0.025  1.00 14.53           C  \\nATOM    398  O   LEU A  52      -1.868  31.443   0.113  1.00 16.05           O  \\nATOM    399  CB  LEU A  52      -2.957  33.986  -1.014  1.00 19.71           C  \\nATOM    400  CG  LEU A  52      -3.122  34.885   0.214  1.00 20.84           C  \\nATOM    401  CD1 LEU A  52      -3.070  36.329  -0.220  1.00 21.38           C  \\nATOM    402  CD2 LEU A  52      -4.420  34.526   0.923  1.00 20.90           C  \\nATOM    403  N   LEU A  53      -0.319  32.904   0.800  1.00 12.41           N  \\nATOM    404  CA  LEU A  53       0.019  32.232   2.048  1.00 13.70           C  \\nATOM    405  C   LEU A  53      -0.841  32.852   3.148  1.00 13.85           C  \\nATOM    406  O   LEU A  53      -0.733  34.056   3.413  1.00 12.82           O  \\nATOM    407  CB  LEU A  53       1.485  32.464   2.400  1.00 14.51           C  \\nATOM    408  CG  LEU A  53       2.707  31.589   2.031  1.00 19.29           C  \\nATOM    409  CD1 LEU A  53       2.394  30.553   0.962  1.00 18.56           C  \\nATOM    410  CD2 LEU A  53       3.843  32.557   1.741  1.00 20.00           C  \\nATOM    411  N   ASP A  54      -1.699  32.023   3.751  1.00 11.96           N  \\nATOM    412  CA  ASP A  54      -2.476  32.445   4.884  1.00 15.07           C  \\nATOM    413  C   ASP A  54      -1.882  31.675   6.055  1.00 12.67           C  \\nATOM    414  O   ASP A  54      -2.099  30.477   6.234  1.00 13.99           O  \\nATOM    415  CB  ASP A  54      -3.930  32.088   4.644  1.00 19.33           C  \\nATOM    416  CG  ASP A  54      -4.826  32.609   5.746  1.00 24.24           C  \\nATOM    417  OD1 ASP A  54      -4.412  32.652   6.897  1.00 28.17           O  \\nATOM    418  OD2 ASP A  54      -5.958  32.979   5.450  1.00 27.59           O  \\nATOM    419  N   ILE A  55      -1.127  32.389   6.880  1.00 11.78           N  \\nATOM    420  CA  ILE A  55      -0.346  31.769   7.935  1.00 12.15           C  \\nATOM    421  C   ILE A  55      -0.953  32.099   9.296  1.00 13.10           C  \\nATOM    422  O   ILE A  55      -1.195  33.271   9.587  1.00 13.84           O  \\nATOM    423  CB  ILE A  55       1.109  32.282   7.931  1.00 12.42           C  \\nATOM    424  CG1 ILE A  55       1.859  31.979   6.653  1.00 13.99           C  \\nATOM    425  CG2 ILE A  55       1.859  31.505   9.018  1.00 13.24           C  \\nATOM    426  CD1 ILE A  55       3.176  32.768   6.497  1.00 15.68           C  \\nATOM    427  N   LEU A  56      -1.176  31.083  10.134  1.00 11.64           N  \\nATOM    428  CA  LEU A  56      -1.642  31.341  11.490  1.00 11.98           C  \\nATOM    429  C   LEU A  56      -0.524  31.027  12.431  1.00  9.99           C  \\nATOM    430  O   LEU A  56      -0.017  29.895  12.447  1.00  9.72           O  \\nATOM    431  CB  LEU A  56      -2.836  30.463  11.896  1.00 13.80           C  \\nATOM    432  CG  LEU A  56      -3.470  30.741  13.297  1.00 16.13           C  \\nATOM    433  CD1 LEU A  56      -4.347  31.987  13.260  1.00 15.28           C  \\nATOM    434  CD2 LEU A  56      -4.321  29.580  13.692  1.00 16.84           C  \\nATOM    435  N   ASP A  57      -0.188  32.084  13.182  1.00 10.40           N  \\nATOM    436  CA  ASP A  57       0.815  32.014  14.219  1.00 10.30           C  \\nATOM    437  C   ASP A  57       0.127  31.812  15.571  1.00 11.75           C  \\nATOM    438  O   ASP A  57      -0.652  32.631  16.053  1.00 12.91           O  \\nATOM    439  CB  ASP A  57       1.602  33.296  14.173  1.00 10.66           C  \\nATOM    440  CG  ASP A  57       2.778  33.336  15.132  1.00 11.39           C  \\nATOM    441  OD1 ASP A  57       3.306  32.298  15.535  1.00 11.07           O  \\nATOM    442  OD2 ASP A  57       3.168  34.448  15.455  1.00 13.15           O  \\nATOM    443  N   THR A  58       0.447  30.723  16.244  1.00 12.55           N  \\nATOM    444  CA  THR A  58      -0.324  30.325  17.416  1.00 12.98           C  \\nATOM    445  C   THR A  58       0.368  30.541  18.758  1.00 13.48           C  \\nATOM    446  O   THR A  58       1.568  30.812  18.815  1.00 14.00           O  \\nATOM    447  CB  THR A  58      -0.698  28.851  17.219  1.00  9.54           C  \\nATOM    448  OG1 THR A  58       0.491  28.084  17.369  1.00 11.40           O  \\nATOM    449  CG2 THR A  58      -1.313  28.597  15.848  1.00 12.27           C  \\nATOM    450  N   ALA A  59      -0.371  30.382  19.868  1.00 14.38           N  \\nATOM    451  CA  ALA A  59       0.230  30.510  21.194  1.00 14.79           C  \\nATOM    452  C   ALA A  59       0.928  29.210  21.608  1.00 15.41           C  \\nATOM    453  O   ALA A  59       0.448  28.106  21.324  1.00 13.07           O  \\nATOM    454  CB  ALA A  59      -0.845  30.818  22.238  1.00 15.47           C  \\nATOM    455  N   GLY A  60       2.084  29.341  22.271  1.00 17.71           N  \\nATOM    456  CA  GLY A  60       2.807  28.208  22.814  1.00 24.18           C  \\nATOM    457  C   GLY A  60       2.190  27.751  24.130  1.00 31.21           C  \\nATOM    458  O   GLY A  60       2.538  26.688  24.627  1.00 30.44           O  \\nATOM    459  N   GLN A  61       1.260  28.503  24.726  1.00 39.50           N  \\nATOM    460  CA  GLN A  61       0.671  28.099  25.994  1.00 48.84           C  \\nATOM    461  C   GLN A  61      -0.683  27.395  25.985  1.00 51.52           C  \\nATOM    462  O   GLN A  61      -1.606  27.780  25.276  1.00 52.17           O  \\nATOM    463  CB  GLN A  61       0.605  29.327  26.874  1.00 51.56           C  \\nATOM    464  CG  GLN A  61       0.871  28.861  28.312  1.00 56.74           C  \\nATOM    465  CD  GLN A  61       1.506  29.937  29.177  1.00 58.74           C  \\nATOM    466  OE1 GLN A  61       2.647  29.795  29.616  1.00 60.12           O  \\nATOM    467  NE2 GLN A  61       0.783  31.031  29.429  1.00 60.18           N  \\nATOM    468  N   GLU A  62      -0.804  26.349  26.818  1.00 57.40           N  \\nATOM    469  CA  GLU A  62      -2.008  25.527  26.927  1.00 61.96           C  \\nATOM    470  C   GLU A  62      -3.185  26.196  27.631  1.00 62.20           C  \\nATOM    471  O   GLU A  62      -4.138  25.496  27.975  1.00 62.85           O  \\nATOM    472  CB  GLU A  62      -1.629  24.183  27.632  1.00 65.18           C  \\nATOM    473  CG  GLU A  62      -2.622  22.966  27.761  1.00 68.22           C  \\nATOM    474  CD  GLU A  62      -3.032  22.132  26.528  1.00 70.25           C  \\nATOM    475  OE1 GLU A  62      -2.710  22.494  25.395  1.00 71.45           O  \\nATOM    476  OE2 GLU A  62      -3.691  21.101  26.689  1.00 70.00           O  \\nATOM    477  N   GLU A  63      -3.206  27.520  27.904  1.00 62.78           N  \\nATOM    478  CA  GLU A  63      -4.496  28.139  28.254  1.00 63.62           C  \\nATOM    479  C   GLU A  63      -5.457  27.854  27.085  1.00 63.71           C  \\nATOM    480  O   GLU A  63      -6.688  27.814  27.211  1.00 62.97           O  \\nATOM    481  CB  GLU A  63      -4.372  29.666  28.475  1.00 65.13           C  \\nATOM    482  CG  GLU A  63      -3.456  30.553  27.607  1.00 66.91           C  \\nATOM    483  CD  GLU A  63      -4.012  31.235  26.345  1.00 68.05           C  \\nATOM    484  OE1 GLU A  63      -5.220  31.462  26.220  1.00 68.64           O  \\nATOM    485  OE2 GLU A  63      -3.198  31.568  25.481  1.00 67.62           O  \\nATOM    486  N   TYR A  64      -4.801  27.606  25.934  1.00 63.36           N  \\nATOM    487  CA  TYR A  64      -5.420  27.092  24.743  1.00 63.20           C  \\nATOM    488  C   TYR A  64      -5.905  25.653  24.875  1.00 61.76           C  \\nATOM    489  O   TYR A  64      -7.120  25.486  24.858  1.00 63.04           O  \\nATOM    490  CB  TYR A  64      -4.437  27.208  23.613  1.00 64.83           C  \\nATOM    491  CG  TYR A  64      -4.736  28.459  22.818  1.00 66.98           C  \\nATOM    492  CD1 TYR A  64      -4.300  29.686  23.267  1.00 67.55           C  \\nATOM    493  CD2 TYR A  64      -5.459  28.355  21.646  1.00 67.97           C  \\nATOM    494  CE1 TYR A  64      -4.587  30.825  22.536  1.00 69.49           C  \\nATOM    495  CE2 TYR A  64      -5.752  29.488  20.911  1.00 69.29           C  \\nATOM    496  CZ  TYR A  64      -5.313  30.718  21.359  1.00 70.05           C  \\nATOM    497  OH  TYR A  64      -5.589  31.852  20.614  1.00 70.67           O  \\nATOM    498  N   SER A  65      -5.095  24.590  24.997  1.00 58.67           N  \\nATOM    499  CA  SER A  65      -5.617  23.230  25.127  1.00 56.93           C  \\nATOM    500  C   SER A  65      -6.614  22.757  24.063  1.00 55.33           C  \\nATOM    501  O   SER A  65      -6.321  22.936  22.876  1.00 54.74           O  \\nATOM    502  CB  SER A  65      -6.190  23.122  26.552  1.00 57.99           C  \\nATOM    503  OG  SER A  65      -7.251  24.011  26.900  1.00 59.80           O  \\nATOM    504  N   ALA A  66      -7.762  22.119  24.359  1.00 52.33           N  \\nATOM    505  CA  ALA A  66      -8.766  21.824  23.335  1.00 48.80           C  \\nATOM    506  C   ALA A  66      -9.138  23.050  22.492  1.00 45.71           C  \\nATOM    507  O   ALA A  66      -9.751  22.862  21.449  1.00 44.33           O  \\nATOM    508  CB  ALA A  66     -10.043  21.300  23.971  1.00 49.14           C  \\nATOM    509  N   MET A  67      -8.829  24.310  22.875  1.00 42.85           N  \\nATOM    510  CA  MET A  67      -8.984  25.465  21.967  1.00 40.01           C  \\nATOM    511  C   MET A  67      -8.093  25.362  20.719  1.00 36.79           C  \\nATOM    512  O   MET A  67      -8.221  26.184  19.805  1.00 36.08           O  \\nATOM    513  CB  MET A  67      -8.594  26.804  22.581  1.00 42.50           C  \\nATOM    514  CG  MET A  67      -9.359  27.330  23.772  1.00 45.93           C  \\nATOM    515  SD  MET A  67     -11.140  27.396  23.483  1.00 48.74           S  \\nATOM    516  CE  MET A  67     -11.526  25.880  24.321  1.00 47.64           C  \\nATOM    517  N   ARG A  68      -7.128  24.418  20.665  1.00 31.59           N  \\nATOM    518  CA  ARG A  68      -6.322  24.217  19.467  1.00 26.34           C  \\nATOM    519  C   ARG A  68      -7.140  23.473  18.424  1.00 23.68           C  \\nATOM    520  O   ARG A  68      -6.846  23.617  17.231  1.00 21.93           O  \\nATOM    521  CB  ARG A  68      -5.057  23.394  19.745  1.00 25.86           C  \\nATOM    522  CG  ARG A  68      -4.161  23.898  20.869  1.00 29.36           C  \\nATOM    523  CD  ARG A  68      -2.881  23.061  20.998  1.00 32.88           C  \\nATOM    524  NE  ARG A  68      -2.221  22.990  22.315  1.00 36.64           N  \\nATOM    525  CZ  ARG A  68      -1.700  24.031  23.001  1.00 39.66           C  \\nATOM    526  NH1 ARG A  68      -1.903  25.315  22.664  1.00 42.06           N  \\nATOM    527  NH2 ARG A  68      -0.933  23.798  24.071  1.00 39.61           N  \\nATOM    528  N   ASP A  69      -8.188  22.721  18.793  1.00 19.14           N  \\nATOM    529  CA  ASP A  69      -8.880  21.905  17.822  1.00 20.43           C  \\nATOM    530  C   ASP A  69      -9.437  22.623  16.617  1.00 22.16           C  \\nATOM    531  O   ASP A  69      -9.453  22.109  15.488  1.00 19.58           O  \\nATOM    532  CB  ASP A  69     -10.025  21.148  18.478  1.00 20.58           C  \\nATOM    533  CG  ASP A  69      -9.598  19.935  19.303  1.00 22.81           C  \\nATOM    534  OD1 ASP A  69      -8.409  19.708  19.538  1.00 24.59           O  \\nATOM    535  OD2 ASP A  69     -10.475  19.176  19.715  1.00 25.28           O  \\nATOM    536  N   GLN A  70      -9.834  23.874  16.840  1.00 23.26           N  \\nATOM    537  CA  GLN A  70     -10.451  24.609  15.759  1.00 23.53           C  \\nATOM    538  C   GLN A  70      -9.437  24.925  14.667  1.00 21.20           C  \\nATOM    539  O   GLN A  70      -9.653  24.591  13.493  1.00 20.90           O  \\nATOM    540  CB  GLN A  70     -11.071  25.881  16.341  1.00 27.73           C  \\nATOM    541  CG  GLN A  70     -12.498  26.019  15.832  1.00 34.11           C  \\nATOM    542  CD  GLN A  70     -13.103  27.407  16.008  1.00 37.87           C  \\nATOM    543  OE1 GLN A  70     -12.532  28.284  16.672  1.00 39.77           O  \\nATOM    544  NE2 GLN A  70     -14.283  27.620  15.406  1.00 37.89           N  \\nATOM    545  N   TYR A  71      -8.292  25.515  15.050  1.00 19.09           N  \\nATOM    546  CA  TYR A  71      -7.329  25.873  14.030  1.00 17.69           C  \\nATOM    547  C   TYR A  71      -6.648  24.658  13.416  1.00 17.24           C  \\nATOM    548  O   TYR A  71      -6.247  24.675  12.245  1.00 17.25           O  \\nATOM    549  CB  TYR A  71      -6.302  26.868  14.625  1.00 17.87           C  \\nATOM    550  CG  TYR A  71      -5.311  26.480  15.725  1.00 18.90           C  \\nATOM    551  CD1 TYR A  71      -4.355  25.510  15.492  1.00 19.74           C  \\nATOM    552  CD2 TYR A  71      -5.328  27.162  16.940  1.00 19.21           C  \\nATOM    553  CE1 TYR A  71      -3.402  25.212  16.450  1.00 20.38           C  \\nATOM    554  CE2 TYR A  71      -4.375  26.868  17.901  1.00 21.93           C  \\nATOM    555  CZ  TYR A  71      -3.422  25.897  17.645  1.00 21.91           C  \\nATOM    556  OH  TYR A  71      -2.463  25.590  18.588  1.00 26.44           O  \\nATOM    557  N   MET A  72      -6.557  23.559  14.166  1.00 14.81           N  \\nATOM    558  CA  MET A  72      -5.958  22.393  13.563  1.00 18.59           C  \\nATOM    559  C   MET A  72      -6.849  21.778  12.472  1.00 19.38           C  \\nATOM    560  O   MET A  72      -6.385  21.198  11.492  1.00 17.67           O  \\nATOM    561  CB  MET A  72      -5.658  21.386  14.669  1.00 17.39           C  \\nATOM    562  CG  MET A  72      -4.479  21.796  15.514  1.00 16.35           C  \\nATOM    563  SD  MET A  72      -4.146  20.600  16.820  1.00 17.04           S  \\nATOM    564  CE  MET A  72      -2.635  21.259  17.460  1.00 17.72           C  \\nATOM    565  N   ARG A  73      -8.174  21.931  12.626  1.00 20.31           N  \\nATOM    566  CA  ARG A  73      -9.120  21.350  11.701  1.00 22.24           C  \\nATOM    567  C   ARG A  73      -9.087  22.084  10.370  1.00 20.94           C  \\nATOM    568  O   ARG A  73      -9.183  21.450   9.326  1.00 20.81           O  \\nATOM    569  CB  ARG A  73     -10.499  21.414  12.369  1.00 25.29           C  \\nATOM    570  CG  ARG A  73     -11.553  20.499  11.767  1.00 30.04           C  \\nATOM    571  CD  ARG A  73     -12.775  20.429  12.687  1.00 31.86           C  \\nATOM    572  NE  ARG A  73     -12.483  19.702  13.917  1.00 34.70           N  \\nATOM    573  CZ  ARG A  73     -12.894  20.122  15.130  1.00 36.49           C  \\nATOM    574  NH1 ARG A  73     -12.579  19.402  16.196  1.00 36.93           N  \\nATOM    575  NH2 ARG A  73     -13.609  21.239  15.311  1.00 38.17           N  \\nATOM    576  N   THR A  74      -8.868  23.401  10.413  1.00 21.33           N  \\nATOM    577  CA  THR A  74      -8.813  24.240   9.219  1.00 24.99           C  \\nATOM    578  C   THR A  74      -7.436  24.351   8.544  1.00 23.10           C  \\nATOM    579  O   THR A  74      -7.367  24.738   7.371  1.00 22.59           O  \\nATOM    580  CB  THR A  74      -9.289  25.676   9.538  1.00 25.73           C  \\nATOM    581  OG1 THR A  74      -8.351  26.302  10.409  1.00 31.08           O  \\nATOM    582  CG2 THR A  74     -10.657  25.644  10.201  1.00 27.58           C  \\nATOM    583  N   GLY A  75      -6.345  24.084   9.289  1.00 19.58           N  \\nATOM    584  CA  GLY A  75      -5.013  24.114   8.730  1.00 15.79           C  \\nATOM    585  C   GLY A  75      -4.823  23.023   7.713  1.00 13.66           C  \\nATOM    586  O   GLY A  75      -5.245  21.880   7.884  1.00 14.69           O  \\nATOM    587  N   GLU A  76      -4.228  23.360   6.573  1.00 13.14           N  \\nATOM    588  CA  GLU A  76      -3.914  22.363   5.546  1.00 12.99           C  \\nATOM    589  C   GLU A  76      -2.514  21.740   5.753  1.00 12.22           C  \\nATOM    590  O   GLU A  76      -2.293  20.576   5.426  1.00 13.59           O  \\nATOM    591  CB  GLU A  76      -3.999  23.014   4.145  1.00 13.07           C  \\nATOM    592  CG  GLU A  76      -5.387  23.377   3.676  1.00 14.35           C  \\nATOM    593  CD  GLU A  76      -5.261  24.069   2.323  1.00 20.40           C  \\nATOM    594  OE1 GLU A  76      -5.461  23.403   1.288  1.00 24.32           O  \\nATOM    595  OE2 GLU A  76      -4.913  25.250   2.284  1.00 18.82           O  \\nATOM    596  N   GLY A  77      -1.553  22.503   6.301  1.00 10.73           N  \\nATOM    597  CA  GLY A  77      -0.216  22.018   6.570  1.00  8.07           C  \\nATOM    598  C   GLY A  77       0.302  22.689   7.819  1.00  8.63           C  \\nATOM    599  O   GLY A  77      -0.066  23.823   8.127  1.00  8.55           O  \\nATOM    600  N   PHE A  78       1.248  22.013   8.479  1.00  9.15           N  \\nATOM    601  CA  PHE A  78       1.754  22.468   9.773  1.00  8.87           C  \\nATOM    602  C   PHE A  78       3.268  22.520   9.854  1.00  9.51           C  \\nATOM    603  O   PHE A  78       3.930  21.542   9.486  1.00  9.15           O  \\nATOM    604  CB  PHE A  78       1.247  21.532  10.904  1.00  9.44           C  \\nATOM    605  CG  PHE A  78      -0.267  21.472  11.000  1.00  9.66           C  \\nATOM    606  CD1 PHE A  78      -0.906  22.310  11.901  1.00 10.71           C  \\nATOM    607  CD2 PHE A  78      -0.994  20.645  10.152  1.00 10.20           C  \\nATOM    608  CE1 PHE A  78      -2.295  22.338  11.940  1.00 10.61           C  \\nATOM    609  CE2 PHE A  78      -2.378  20.678  10.192  1.00 12.55           C  \\nATOM    610  CZ  PHE A  78      -3.022  21.529  11.081  1.00 12.80           C  \\nATOM    611  N   LEU A  79       3.813  23.636  10.354  1.00  8.72           N  \\nATOM    612  CA  LEU A  79       5.230  23.626  10.699  1.00  9.93           C  \\nATOM    613  C   LEU A  79       5.279  23.281  12.181  1.00  9.09           C  \\nATOM    614  O   LEU A  79       4.701  24.008  12.988  1.00 11.58           O  \\nATOM    615  CB  LEU A  79       5.921  24.970  10.567  1.00 12.80           C  \\nATOM    616  CG  LEU A  79       6.155  25.659   9.249  1.00 16.39           C  \\nATOM    617  CD1 LEU A  79       7.007  26.911   9.544  1.00 17.02           C  \\nATOM    618  CD2 LEU A  79       6.904  24.756   8.269  1.00 16.98           C  \\nATOM    619  N   CYS A  80       5.950  22.219  12.579  1.00  9.14           N  \\nATOM    620  CA  CYS A  80       6.074  21.821  13.981  1.00 11.81           C  \\nATOM    621  C   CYS A  80       7.448  22.237  14.434  1.00 11.53           C  \\nATOM    622  O   CYS A  80       8.435  21.640  14.000  1.00 12.21           O  \\nATOM    623  CB  CYS A  80       5.903  20.318  14.106  1.00 13.73           C  \\nATOM    624  SG  CYS A  80       4.158  19.900  13.837  1.00 18.26           S  \\nATOM    625  N   VAL A  81       7.486  23.282  15.269  1.00  7.57           N  \\nATOM    626  CA  VAL A  81       8.730  23.956  15.604  1.00  8.45           C  \\nATOM    627  C   VAL A  81       9.248  23.620  16.996  1.00  8.90           C  \\nATOM    628  O   VAL A  81       8.451  23.680  17.939  1.00  8.49           O  \\nATOM    629  CB  VAL A  81       8.544  25.516  15.507  1.00  7.40           C  \\nATOM    630  CG1 VAL A  81       9.917  26.196  15.588  1.00  9.42           C  \\nATOM    631  CG2 VAL A  81       8.004  25.954  14.167  1.00  8.59           C  \\nATOM    632  N   PHE A  82      10.546  23.237  17.119  1.00  9.49           N  \\nATOM    633  CA  PHE A  82      11.195  23.202  18.442  1.00  9.09           C  \\nATOM    634  C   PHE A  82      12.511  23.994  18.350  1.00  9.83           C  \\nATOM    635  O   PHE A  82      12.914  24.440  17.266  1.00  9.02           O  \\nATOM    636  CB  PHE A  82      11.482  21.755  18.902  1.00  9.76           C  \\nATOM    637  CG  PHE A  82      12.536  21.062  18.071  1.00 10.09           C  \\nATOM    638  CD1 PHE A  82      13.867  21.099  18.486  1.00 10.70           C  \\nATOM    639  CD2 PHE A  82      12.192  20.412  16.889  1.00 10.93           C  \\nATOM    640  CE1 PHE A  82      14.838  20.483  17.699  1.00 11.12           C  \\nATOM    641  CE2 PHE A  82      13.181  19.807  16.125  1.00 10.73           C  \\nATOM    642  CZ  PHE A  82      14.505  19.839  16.523  1.00 10.35           C  \\nATOM    643  N   ALA A  83      13.128  24.269  19.516  1.00  9.75           N  \\nATOM    644  CA  ALA A  83      14.407  24.963  19.569  1.00 10.32           C  \\nATOM    645  C   ALA A  83      15.509  23.960  19.916  1.00  9.48           C  \\nATOM    646  O   ALA A  83      15.362  23.142  20.839  1.00  9.61           O  \\nATOM    647  CB  ALA A  83      14.354  26.083  20.620  1.00  9.75           C  \\nATOM    648  N   ILE A  84      16.624  24.042  19.160  1.00 11.41           N  \\nATOM    649  CA  ILE A  84      17.714  23.070  19.299  1.00 13.15           C  \\nATOM    650  C   ILE A  84      18.446  23.223  20.623  1.00 14.57           C  \\nATOM    651  O   ILE A  84      19.234  22.347  20.971  1.00 13.94           O  \\nATOM    652  CB  ILE A  84      18.775  23.151  18.160  1.00 13.29           C  \\nATOM    653  CG1 ILE A  84      19.403  24.509  17.878  1.00 14.90           C  \\nATOM    654  CG2 ILE A  84      18.018  22.701  16.933  1.00 13.09           C  \\nATOM    655  CD1 ILE A  84      20.578  24.957  18.763  1.00 17.59           C  \\nATOM    656  N   ASN A  85      18.200  24.290  21.390  1.00 13.61           N  \\nATOM    657  CA  ASN A  85      18.794  24.394  22.720  1.00 15.52           C  \\nATOM    658  C   ASN A  85      17.758  24.156  23.828  1.00 15.49           C  \\nATOM    659  O   ASN A  85      17.988  24.519  24.999  1.00 16.13           O  \\nATOM    660  CB  ASN A  85      19.446  25.788  22.885  1.00 15.16           C  \\nATOM    661  CG  ASN A  85      18.464  26.946  22.904  1.00 16.42           C  \\nATOM    662  OD1 ASN A  85      17.355  26.878  22.362  1.00 16.01           O  \\nATOM    663  ND2 ASN A  85      18.808  28.030  23.587  1.00 16.83           N  \\nATOM    664  N   ASN A  86      16.616  23.515  23.509  1.00 14.72           N  \\nATOM    665  CA  ASN A  86      15.553  23.303  24.483  1.00 13.01           C  \\nATOM    666  C   ASN A  86      14.987  21.901  24.309  1.00 13.17           C  \\nATOM    667  O   ASN A  86      14.080  21.668  23.520  1.00 13.04           O  \\nATOM    668  CB  ASN A  86      14.478  24.367  24.256  1.00 16.51           C  \\nATOM    669  CG  ASN A  86      13.267  24.415  25.187  1.00 20.74           C  \\nATOM    670  OD1 ASN A  86      12.370  25.241  25.018  1.00 25.41           O  \\nATOM    671  ND2 ASN A  86      13.123  23.596  26.206  1.00 19.96           N  \\nATOM    672  N   THR A  87      15.488  20.950  25.103  1.00 13.80           N  \\nATOM    673  CA  THR A  87      15.095  19.553  25.026  1.00 15.40           C  \\nATOM    674  C   THR A  87      13.589  19.316  25.266  1.00 14.40           C  \\nATOM    675  O   THR A  87      12.953  18.535  24.546  1.00 14.11           O  \\nATOM    676  CB  THR A  87      15.983  18.791  26.046  1.00 16.19           C  \\nATOM    677  OG1 THR A  87      17.325  18.908  25.571  1.00 18.22           O  \\nATOM    678  CG2 THR A  87      15.626  17.327  26.180  1.00 17.82           C  \\nATOM    679  N   LYS A  88      12.983  19.958  26.272  1.00 14.42           N  \\nATOM    680  CA  LYS A  88      11.552  19.842  26.535  1.00 15.84           C  \\nATOM    681  C   LYS A  88      10.726  20.232  25.278  1.00 15.06           C  \\nATOM    682  O   LYS A  88       9.722  19.582  25.003  1.00 14.61           O  \\nATOM    683  CB  LYS A  88      11.207  20.766  27.717  1.00 19.53           C  \\nATOM    684  CG  LYS A  88       9.937  20.401  28.457  1.00 27.71           C  \\nATOM    685  CD  LYS A  88       9.585  21.452  29.529  1.00 33.26           C  \\nATOM    686  CE  LYS A  88       9.061  22.762  28.877  1.00 36.79           C  \\nATOM    687  NZ  LYS A  88       8.642  23.763  29.852  1.00 37.29           N  \\nATOM    688  N   SER A  89      11.119  21.259  24.475  1.00 12.36           N  \\nATOM    689  CA  SER A  89      10.325  21.666  23.309  1.00 11.31           C  \\nATOM    690  C   SER A  89      10.371  20.572  22.268  1.00 12.51           C  \\nATOM    691  O   SER A  89       9.397  20.388  21.547  1.00 12.88           O  \\nATOM    692  CB  SER A  89      10.844  22.951  22.674  1.00 11.64           C  \\nATOM    693  OG  SER A  89      12.160  22.900  22.124  1.00 11.11           O  \\nATOM    694  N   PHE A  90      11.504  19.842  22.204  1.00 11.27           N  \\nATOM    695  CA  PHE A  90      11.680  18.751  21.262  1.00 12.91           C  \\nATOM    696  C   PHE A  90      10.760  17.601  21.667  1.00 14.37           C  \\nATOM    697  O   PHE A  90      10.034  17.057  20.839  1.00 13.24           O  \\nATOM    698  CB  PHE A  90      13.161  18.312  21.260  1.00 12.34           C  \\nATOM    699  CG  PHE A  90      13.411  17.120  20.351  1.00 14.41           C  \\nATOM    700  CD1 PHE A  90      13.890  15.948  20.903  1.00 15.10           C  \\nATOM    701  CD2 PHE A  90      13.136  17.206  18.990  1.00 14.81           C  \\nATOM    702  CE1 PHE A  90      14.093  14.857  20.088  1.00 16.20           C  \\nATOM    703  CE2 PHE A  90      13.340  16.110  18.179  1.00 14.81           C  \\nATOM    704  CZ  PHE A  90      13.818  14.935  18.731  1.00 15.39           C  \\nATOM    705  N   GLU A  91      10.725  17.231  22.949  1.00 14.40           N  \\nATOM    706  CA  GLU A  91       9.863  16.146  23.407  1.00 15.78           C  \\nATOM    707  C   GLU A  91       8.374  16.466  23.301  1.00 16.18           C  \\nATOM    708  O   GLU A  91       7.533  15.582  23.114  1.00 17.50           O  \\nATOM    709  CB  GLU A  91      10.226  15.846  24.835  1.00 19.84           C  \\nATOM    710  CG  GLU A  91      11.654  15.386  25.074  1.00 24.49           C  \\nATOM    711  CD  GLU A  91      12.090  15.310  26.537  1.00 27.97           C  \\nATOM    712  OE1 GLU A  91      13.199  14.842  26.766  1.00 30.72           O  \\nATOM    713  OE2 GLU A  91      11.371  15.730  27.449  1.00 30.38           O  \\nATOM    714  N   ASP A  92       8.019  17.747  23.433  1.00 16.19           N  \\nATOM    715  CA  ASP A  92       6.650  18.216  23.265  1.00 15.23           C  \\nATOM    716  C   ASP A  92       6.141  18.053  21.839  1.00 14.29           C  \\nATOM    717  O   ASP A  92       4.927  18.074  21.633  1.00 14.78           O  \\nATOM    718  CB  ASP A  92       6.536  19.708  23.646  1.00 17.85           C  \\nATOM    719  CG  ASP A  92       6.706  20.080  25.127  1.00 21.39           C  \\nATOM    720  OD1 ASP A  92       6.648  19.209  25.997  1.00 20.06           O  \\nATOM    721  OD2 ASP A  92       6.887  21.272  25.401  1.00 24.39           O  \\nATOM    722  N   ILE A  93       7.013  17.910  20.826  1.00 11.76           N  \\nATOM    723  CA  ILE A  93       6.559  17.749  19.452  1.00 14.77           C  \\nATOM    724  C   ILE A  93       5.574  16.583  19.354  1.00 14.03           C  \\nATOM    725  O   ILE A  93       4.576  16.695  18.644  1.00 14.93           O  \\nATOM    726  CB  ILE A  93       7.790  17.499  18.485  1.00 14.96           C  \\nATOM    727  CG1 ILE A  93       8.506  18.830  18.293  1.00 15.41           C  \\nATOM    728  CG2 ILE A  93       7.369  16.936  17.091  1.00 14.61           C  \\nATOM    729  CD1 ILE A  93       7.654  19.940  17.637  1.00 20.05           C  \\nATOM    730  N   HIS A  94       5.806  15.471  20.049  1.00 15.23           N  \\nATOM    731  CA  HIS A  94       4.940  14.302  19.956  1.00 18.20           C  \\nATOM    732  C   HIS A  94       3.494  14.635  20.287  1.00 16.36           C  \\nATOM    733  O   HIS A  94       2.576  14.183  19.598  1.00 17.20           O  \\nATOM    734  CB  HIS A  94       5.436  13.194  20.900  1.00 23.57           C  \\nATOM    735  CG  HIS A  94       4.685  11.874  20.678  1.00 29.09           C  \\nATOM    736  ND1 HIS A  94       4.923  11.118  19.602  1.00 31.26           N  \\nATOM    737  CD2 HIS A  94       3.705  11.361  21.498  1.00 30.93           C  \\nATOM    738  CE1 HIS A  94       4.078  10.110  19.746  1.00 31.74           C  \\nATOM    739  NE2 HIS A  94       3.366  10.260  20.861  1.00 32.20           N  \\nATOM    740  N   GLN A  95       3.271  15.499  21.281  1.00 16.91           N  \\nATOM    741  CA  GLN A  95       1.916  15.878  21.673  1.00 18.66           C  \\nATOM    742  C   GLN A  95       1.226  16.736  20.625  1.00 16.97           C  \\nATOM    743  O   GLN A  95       0.012  16.562  20.443  1.00 15.36           O  \\nATOM    744  CB  GLN A  95       1.928  16.626  23.018  1.00 21.56           C  \\nATOM    745  CG  GLN A  95       2.457  15.766  24.180  1.00 29.81           C  \\nATOM    746  CD  GLN A  95       3.370  16.540  25.150  1.00 35.29           C  \\nATOM    747  OE1 GLN A  95       3.037  17.647  25.593  1.00 37.94           O  \\nATOM    748  NE2 GLN A  95       4.546  16.006  25.521  1.00 37.04           N  \\nATOM    749  N   TYR A  96       1.930  17.653  19.921  1.00 16.06           N  \\nATOM    750  CA  TYR A  96       1.276  18.417  18.826  1.00 16.07           C  \\nATOM    751  C   TYR A  96       0.963  17.504  17.658  1.00 14.44           C  \\nATOM    752  O   TYR A  96      -0.138  17.565  17.118  1.00 15.79           O  \\nATOM    753  CB  TYR A  96       2.139  19.554  18.264  1.00 17.94           C  \\nATOM    754  CG  TYR A  96       2.287  20.677  19.277  1.00 19.19           C  \\nATOM    755  CD1 TYR A  96       1.368  21.699  19.338  1.00 19.33           C  \\nATOM    756  CD2 TYR A  96       3.348  20.631  20.161  1.00 19.61           C  \\nATOM    757  CE1 TYR A  96       1.526  22.678  20.308  1.00 21.72           C  \\nATOM    758  CE2 TYR A  96       3.512  21.595  21.129  1.00 21.72           C  \\nATOM    759  CZ  TYR A  96       2.594  22.613  21.190  1.00 21.26           C  \\nATOM    760  OH  TYR A  96       2.776  23.582  22.153  1.00 24.09           O  \\nATOM    761  N   ARG A  97       1.904  16.635  17.295  1.00 13.30           N  \\nATOM    762  CA  ARG A  97       1.699  15.650  16.247  1.00 15.46           C  \\nATOM    763  C   ARG A  97       0.457  14.794  16.492  1.00 14.26           C  \\nATOM    764  O   ARG A  97      -0.420  14.676  15.641  1.00 14.10           O  \\nATOM    765  CB  ARG A  97       2.939  14.796  16.196  1.00 17.97           C  \\nATOM    766  CG  ARG A  97       2.886  13.541  15.357  1.00 23.35           C  \\nATOM    767  CD  ARG A  97       3.333  13.765  13.943  1.00 23.59           C  \\nATOM    768  NE  ARG A  97       3.314  12.471  13.314  1.00 24.08           N  \\nATOM    769  CZ  ARG A  97       3.268  12.320  12.007  1.00 25.74           C  \\nATOM    770  NH1 ARG A  97       3.135  11.073  11.548  1.00 26.76           N  \\nATOM    771  NH2 ARG A  97       3.353  13.380  11.181  1.00 26.95           N  \\nATOM    772  N   GLU A  98       0.332  14.218  17.678  1.00 13.69           N  \\nATOM    773  CA  GLU A  98      -0.807  13.361  17.941  1.00 16.64           C  \\nATOM    774  C   GLU A  98      -2.118  14.108  17.983  1.00 15.57           C  \\nATOM    775  O   GLU A  98      -3.157  13.573  17.607  1.00 15.85           O  \\nATOM    776  CB  GLU A  98      -0.593  12.623  19.260  1.00 21.96           C  \\nATOM    777  CG  GLU A  98       0.417  11.455  19.177  1.00 28.02           C  \\nATOM    778  CD  GLU A  98       0.147  10.460  18.038  1.00 31.04           C  \\nATOM    779  OE1 GLU A  98       1.118  10.123  17.373  1.00 33.58           O  \\nATOM    780  OE2 GLU A  98      -0.992  10.025  17.790  1.00 33.61           O  \\nATOM    781  N   GLN A  99      -2.105  15.362  18.446  1.00 14.66           N  \\nATOM    782  CA  GLN A  99      -3.343  16.104  18.468  1.00 13.22           C  \\nATOM    783  C   GLN A  99      -3.779  16.398  17.049  1.00 12.15           C  \\nATOM    784  O   GLN A  99      -4.974  16.274  16.791  1.00 14.09           O  \\nATOM    785  CB  GLN A  99      -3.158  17.403  19.250  1.00 16.29           C  \\nATOM    786  CG  GLN A  99      -4.513  18.118  19.407  1.00 19.67           C  \\nATOM    787  CD  GLN A  99      -4.517  19.240  20.438  1.00 22.19           C  \\nATOM    788  OE1 GLN A  99      -3.537  19.485  21.138  1.00 23.70           O  \\nATOM    789  NE2 GLN A  99      -5.628  19.961  20.537  1.00 22.72           N  \\nATOM    790  N   ILE A 100      -2.861  16.795  16.142  1.00 11.42           N  \\nATOM    791  CA  ILE A 100      -3.197  17.121  14.743  1.00 10.81           C  \\nATOM    792  C   ILE A 100      -3.761  15.893  14.008  1.00 12.39           C  \\nATOM    793  O   ILE A 100      -4.733  15.967  13.243  1.00 12.70           O  \\nATOM    794  CB  ILE A 100      -1.924  17.659  14.038  1.00 10.32           C  \\nATOM    795  CG1 ILE A 100      -1.642  19.075  14.518  1.00 10.37           C  \\nATOM    796  CG2 ILE A 100      -2.131  17.685  12.512  1.00 11.36           C  \\nATOM    797  CD1 ILE A 100      -0.184  19.481  14.392  1.00 10.69           C  \\nATOM    798  N   LYS A 101      -3.143  14.732  14.255  1.00 14.08           N  \\nATOM    799  CA  LYS A 101      -3.613  13.487  13.637  1.00 15.95           C  \\nATOM    800  C   LYS A 101      -5.021  13.150  14.079  1.00 15.72           C  \\nATOM    801  O   LYS A 101      -5.848  12.770  13.262  1.00 14.25           O  \\nATOM    802  CB  LYS A 101      -2.681  12.317  13.996  1.00 17.44           C  \\nATOM    803  CG  LYS A 101      -1.414  12.284  13.115  1.00 19.24           C  \\nATOM    804  CD  LYS A 101      -0.332  11.418  13.764  1.00 23.28           C  \\nATOM    805  CE  LYS A 101      -0.752  10.000  14.188  1.00 23.82           C  \\nATOM    806  NZ  LYS A 101       0.400   9.230  14.662  1.00 24.00           N  \\nATOM    807  N   ARG A 102      -5.278  13.341  15.377  1.00 16.72           N  \\nATOM    808  CA  ARG A 102      -6.586  13.098  15.962  1.00 18.74           C  \\nATOM    809  C   ARG A 102      -7.643  14.087  15.440  1.00 19.07           C  \\nATOM    810  O   ARG A 102      -8.712  13.628  15.002  1.00 20.67           O  \\nATOM    811  CB  ARG A 102      -6.457  13.197  17.462  1.00 20.62           C  \\nATOM    812  CG  ARG A 102      -7.539  12.409  18.132  1.00 26.89           C  \\nATOM    813  CD  ARG A 102      -7.610  12.887  19.585  1.00 30.60           C  \\nATOM    814  NE  ARG A 102      -8.197  14.224  19.645  1.00 35.83           N  \\nATOM    815  CZ  ARG A 102      -7.633  15.255  20.301  1.00 37.61           C  \\nATOM    816  NH1 ARG A 102      -8.243  16.455  20.267  1.00 38.37           N  \\nATOM    817  NH2 ARG A 102      -6.481  15.110  20.983  1.00 38.39           N  \\nATOM    818  N   VAL A 103      -7.418  15.407  15.395  1.00 15.59           N  \\nATOM    819  CA  VAL A 103      -8.452  16.326  14.881  1.00 18.48           C  \\nATOM    820  C   VAL A 103      -8.665  16.200  13.349  1.00 16.72           C  \\nATOM    821  O   VAL A 103      -9.776  16.383  12.848  1.00 15.90           O  \\nATOM    822  CB  VAL A 103      -8.131  17.840  15.215  1.00 20.10           C  \\nATOM    823  CG1 VAL A 103      -7.896  18.005  16.704  1.00 21.34           C  \\nATOM    824  CG2 VAL A 103      -6.899  18.302  14.512  1.00 23.93           C  \\nATOM    825  N   LYS A 104      -7.622  15.902  12.574  1.00 13.84           N  \\nATOM    826  CA  LYS A 104      -7.804  15.755  11.140  1.00 16.11           C  \\nATOM    827  C   LYS A 104      -8.158  14.329  10.716  1.00 16.70           C  \\nATOM    828  O   LYS A 104      -8.260  14.079   9.508  1.00 17.27           O  \\nATOM    829  CB  LYS A 104      -6.525  16.215  10.404  1.00 18.12           C  \\nATOM    830  CG  LYS A 104      -6.197  17.678  10.744  1.00 19.53           C  \\nATOM    831  CD  LYS A 104      -5.250  18.297   9.736  1.00 21.28           C  \\nATOM    832  CE  LYS A 104      -5.921  18.590   8.390  1.00 20.75           C  \\nATOM    833  NZ  LYS A 104      -6.875  19.673   8.462  1.00 19.79           N  \\nATOM    834  N   ASP A 105      -8.372  13.390  11.669  1.00 15.91           N  \\nATOM    835  CA  ASP A 105      -8.666  11.988  11.359  1.00 15.43           C  \\nATOM    836  C   ASP A 105      -7.705  11.372  10.333  1.00 16.04           C  \\nATOM    837  O   ASP A 105      -8.106  10.654   9.415  1.00 15.38           O  \\nATOM    838  CB  ASP A 105     -10.077  11.847  10.810  1.00 18.74           C  \\nATOM    839  CG  ASP A 105     -11.261  12.121  11.696  1.00 21.15           C  \\nATOM    840  OD1 ASP A 105     -11.183  11.972  12.918  1.00 21.36           O  \\nATOM    841  OD2 ASP A 105     -12.281  12.480  11.113  1.00 22.63           O  \\nATOM    842  N   SER A 106      -6.397  11.591  10.469  1.00 15.54           N  \\nATOM    843  CA  SER A 106      -5.470  11.174   9.442  1.00 17.25           C  \\nATOM    844  C   SER A 106      -4.112  10.882  10.011  1.00 17.31           C  \\nATOM    845  O   SER A 106      -3.650  11.556  10.916  1.00 19.11           O  \\nATOM    846  CB  SER A 106      -5.463  12.314   8.398  1.00 19.27           C  \\nATOM    847  OG  SER A 106      -4.326  12.490   7.573  1.00 24.20           O  \\nATOM    848  N   ASP A 107      -3.454   9.891   9.453  1.00 17.29           N  \\nATOM    849  CA  ASP A 107      -2.109   9.557   9.868  1.00 19.80           C  \\nATOM    850  C   ASP A 107      -1.000  10.220   9.063  1.00 20.36           C  \\nATOM    851  O   ASP A 107       0.197  10.118   9.345  1.00 21.26           O  \\nATOM    852  CB  ASP A 107      -1.970   8.045   9.810  1.00 23.15           C  \\nATOM    853  CG  ASP A 107      -2.863   7.310  10.807  1.00 25.12           C  \\nATOM    854  OD1 ASP A 107      -3.262   6.206  10.462  1.00 29.21           O  \\nATOM    855  OD2 ASP A 107      -3.127   7.812  11.903  1.00 26.59           O  \\nATOM    856  N   ASP A 108      -1.383  10.909   8.005  1.00 20.57           N  \\nATOM    857  CA  ASP A 108      -0.430  11.664   7.216  1.00 22.38           C  \\nATOM    858  C   ASP A 108      -1.061  12.982   6.842  1.00 21.37           C  \\nATOM    859  O   ASP A 108      -1.973  13.096   6.030  1.00 24.36           O  \\nATOM    860  CB  ASP A 108      -0.056  10.900   5.967  1.00 27.28           C  \\nATOM    861  CG  ASP A 108      -1.263  10.278   5.320  1.00 32.24           C  \\nATOM    862  OD1 ASP A 108      -1.714   9.272   5.877  1.00 37.07           O  \\nATOM    863  OD2 ASP A 108      -1.747  10.790   4.301  1.00 36.33           O  \\nATOM    864  N   VAL A 109      -0.598  13.970   7.572  1.00 19.33           N  \\nATOM    865  CA  VAL A 109      -1.009  15.354   7.400  1.00 17.49           C  \\nATOM    866  C   VAL A 109       0.215  16.086   6.819  1.00 14.55           C  \\nATOM    867  O   VAL A 109       1.335  15.772   7.245  1.00 13.64           O  \\nATOM    868  CB  VAL A 109      -1.410  15.870   8.784  1.00 18.40           C  \\nATOM    869  CG1 VAL A 109      -1.645  17.366   8.765  1.00 20.21           C  \\nATOM    870  CG2 VAL A 109      -2.676  15.101   9.214  1.00 20.27           C  \\nATOM    871  N   PRO A 110       0.141  17.001   5.825  1.00 12.41           N  \\nATOM    872  CA  PRO A 110       1.297  17.783   5.384  1.00 11.52           C  \\nATOM    873  C   PRO A 110       1.920  18.480   6.605  1.00 11.79           C  \\nATOM    874  O   PRO A 110       1.289  19.245   7.324  1.00  9.55           O  \\nATOM    875  CB  PRO A 110       0.728  18.721   4.370  1.00 11.31           C  \\nATOM    876  CG  PRO A 110      -0.410  17.926   3.779  1.00 11.38           C  \\nATOM    877  CD  PRO A 110      -1.044  17.309   5.020  1.00 12.05           C  \\nATOM    878  N   MET A 111       3.192  18.201   6.874  1.00 11.92           N  \\nATOM    879  CA  MET A 111       3.878  18.724   8.050  1.00 11.49           C  \\nATOM    880  C   MET A 111       5.379  18.749   7.807  1.00 11.68           C  \\nATOM    881  O   MET A 111       5.903  17.900   7.098  1.00 11.11           O  \\nATOM    882  CB  MET A 111       3.494  17.806   9.211  1.00 14.36           C  \\nATOM    883  CG  MET A 111       3.864  18.110  10.627  1.00 17.41           C  \\nATOM    884  SD  MET A 111       3.231  16.825  11.758  1.00 18.36           S  \\nATOM    885  CE  MET A 111       1.528  17.128  11.498  1.00 17.25           C  \\nATOM    886  N   VAL A 112       6.091  19.706   8.404  1.00 10.73           N  \\nATOM    887  CA  VAL A 112       7.545  19.763   8.348  1.00 10.18           C  \\nATOM    888  C   VAL A 112       7.992  19.875   9.794  1.00  9.11           C  \\nATOM    889  O   VAL A 112       7.404  20.633  10.556  1.00 10.77           O  \\nATOM    890  CB  VAL A 112       8.010  21.024   7.542  1.00 11.80           C  \\nATOM    891  CG1 VAL A 112       9.528  21.213   7.558  1.00 13.24           C  \\nATOM    892  CG2 VAL A 112       7.542  20.843   6.104  1.00 14.56           C  \\nATOM    893  N   LEU A 113       9.032  19.167  10.207  1.00  8.47           N  \\nATOM    894  CA  LEU A 113       9.632  19.332  11.535  1.00  9.53           C  \\nATOM    895  C   LEU A 113      10.738  20.384  11.412  1.00  9.33           C  \\nATOM    896  O   LEU A 113      11.621  20.270  10.555  1.00  9.95           O  \\nATOM    897  CB  LEU A 113      10.214  17.983  12.017  1.00  9.56           C  \\nATOM    898  CG  LEU A 113      10.863  17.976  13.381  1.00 11.58           C  \\nATOM    899  CD1 LEU A 113       9.908  18.379  14.503  1.00 13.17           C  \\nATOM    900  CD2 LEU A 113      11.361  16.564  13.597  1.00 12.84           C  \\nATOM    901  N   VAL A 114      10.705  21.442  12.221  1.00  7.62           N  \\nATOM    902  CA  VAL A 114      11.673  22.541  12.109  1.00  7.64           C  \\nATOM    903  C   VAL A 114      12.436  22.667  13.441  1.00  7.65           C  \\nATOM    904  O   VAL A 114      11.872  22.842  14.541  1.00  9.71           O  \\nATOM    905  CB  VAL A 114      10.896  23.836  11.774  1.00  9.37           C  \\nATOM    906  CG1 VAL A 114      11.814  25.030  11.869  1.00 10.27           C  \\nATOM    907  CG2 VAL A 114      10.254  23.719  10.393  1.00  9.64           C  \\nATOM    908  N   GLY A 115      13.745  22.567  13.331  1.00  6.10           N  \\nATOM    909  CA  GLY A 115      14.603  22.742  14.476  1.00  6.69           C  \\nATOM    910  C   GLY A 115      15.120  24.163  14.407  1.00  8.47           C  \\nATOM    911  O   GLY A 115      16.007  24.439  13.594  1.00  9.16           O  \\nATOM    912  N   ASN A 116      14.593  25.089  15.192  1.00  8.01           N  \\nATOM    913  CA  ASN A 116      15.007  26.475  15.111  1.00  8.24           C  \\nATOM    914  C   ASN A 116      16.112  26.873  16.109  1.00  9.61           C  \\nATOM    915  O   ASN A 116      16.453  26.142  17.051  1.00  9.84           O  \\nATOM    916  CB  ASN A 116      13.721  27.307  15.275  1.00  8.01           C  \\nATOM    917  CG  ASN A 116      13.929  28.788  14.992  1.00  9.48           C  \\nATOM    918  OD1 ASN A 116      14.470  29.205  13.972  1.00  9.94           O  \\nATOM    919  ND2 ASN A 116      13.565  29.644  15.940  1.00 10.75           N  \\nATOM    920  N   LYS A 117      16.660  28.087  15.876  1.00 10.22           N  \\nATOM    921  CA  LYS A 117      17.735  28.715  16.674  1.00 12.36           C  \\nATOM    922  C   LYS A 117      19.094  28.021  16.450  1.00 13.39           C  \\nATOM    923  O   LYS A 117      19.977  27.940  17.324  1.00 11.29           O  \\nATOM    924  CB  LYS A 117      17.391  28.706  18.214  1.00 13.23           C  \\nATOM    925  CG  LYS A 117      16.026  29.272  18.611  1.00 12.90           C  \\nATOM    926  CD  LYS A 117      15.932  29.640  20.097  1.00 12.43           C  \\nATOM    927  CE  LYS A 117      14.551  30.217  20.380  1.00 10.55           C  \\nATOM    928  NZ  LYS A 117      14.491  30.647  21.772  1.00 11.10           N  \\nATOM    929  N   CYS A 118      19.254  27.555  15.188  1.00 13.69           N  \\nATOM    930  CA  CYS A 118      20.449  26.806  14.844  1.00 15.67           C  \\nATOM    931  C   CYS A 118      21.716  27.658  14.796  1.00 15.28           C  \\nATOM    932  O   CYS A 118      22.808  27.113  14.629  1.00 15.30           O  \\nATOM    933  CB  CYS A 118      20.246  26.064  13.492  1.00 17.74           C  \\nATOM    934  SG  CYS A 118      20.353  27.009  11.944  1.00 20.74           S  \\nATOM    935  N   ASP A 119      21.578  28.995  14.931  1.00 15.76           N  \\nATOM    936  CA  ASP A 119      22.722  29.904  15.083  1.00 16.09           C  \\nATOM    937  C   ASP A 119      23.426  29.778  16.445  1.00 17.19           C  \\nATOM    938  O   ASP A 119      24.589  30.178  16.553  1.00 18.76           O  \\nATOM    939  CB  ASP A 119      22.268  31.374  14.888  1.00 15.14           C  \\nATOM    940  CG  ASP A 119      21.153  31.785  15.835  1.00 15.81           C  \\nATOM    941  OD1 ASP A 119      20.030  31.294  15.662  1.00 13.87           O  \\nATOM    942  OD2 ASP A 119      21.407  32.596  16.721  1.00 14.85           O  \\nATOM    943  N   LEU A 120      22.732  29.248  17.480  1.00 16.86           N  \\nATOM    944  CA  LEU A 120      23.230  29.099  18.845  1.00 18.76           C  \\nATOM    945  C   LEU A 120      24.217  27.957  18.979  1.00 19.70           C  \\nATOM    946  O   LEU A 120      23.979  26.885  18.431  1.00 20.53           O  \\nATOM    947  CB  LEU A 120      22.103  28.829  19.831  1.00 17.65           C  \\nATOM    948  CG  LEU A 120      21.065  29.933  20.010  1.00 19.43           C  \\nATOM    949  CD1 LEU A 120      20.044  29.418  21.008  1.00 18.46           C  \\nATOM    950  CD2 LEU A 120      21.688  31.239  20.503  1.00 18.73           C  \\nATOM    951  N   ALA A 121      25.313  28.184  19.705  1.00 19.26           N  \\nATOM    952  CA  ALA A 121      26.303  27.143  19.887  1.00 20.48           C  \\nATOM    953  C   ALA A 121      25.942  26.210  21.036  1.00 21.30           C  \\nATOM    954  O   ALA A 121      26.403  25.069  21.067  1.00 24.04           O  \\nATOM    955  CB  ALA A 121      27.652  27.767  20.164  1.00 22.02           C  \\nATOM    956  N   ALA A 122      25.109  26.635  21.982  1.00 22.09           N  \\nATOM    957  CA  ALA A 122      24.683  25.797  23.094  1.00 23.88           C  \\nATOM    958  C   ALA A 122      23.578  24.811  22.720  1.00 23.77           C  \\nATOM    959  O   ALA A 122      22.492  24.835  23.301  1.00 27.82           O  \\nATOM    960  CB  ALA A 122      24.171  26.684  24.249  1.00 25.67           C  \\nATOM    961  N   ARG A 123      23.843  23.908  21.783  1.00 20.99           N  \\nATOM    962  CA  ARG A 123      22.890  22.905  21.311  1.00 19.81           C  \\nATOM    963  C   ARG A 123      22.619  21.761  22.283  1.00 19.56           C  \\nATOM    964  O   ARG A 123      23.590  21.176  22.774  1.00 20.01           O  \\nATOM    965  CB  ARG A 123      23.421  22.333  19.988  1.00 19.81           C  \\nATOM    966  CG  ARG A 123      22.662  21.166  19.398  1.00 21.43           C  \\nATOM    967  CD  ARG A 123      23.273  20.680  18.073  1.00 22.66           C  \\nATOM    968  NE  ARG A 123      23.050  21.590  16.956  1.00 22.61           N  \\nATOM    969  CZ  ARG A 123      22.019  21.487  16.093  1.00 20.95           C  \\nATOM    970  NH1 ARG A 123      21.045  20.568  16.181  1.00 16.36           N  \\nATOM    971  NH2 ARG A 123      21.989  22.370  15.098  1.00 19.65           N  \\nATOM    972  N   THR A 124      21.364  21.389  22.600  1.00 17.96           N  \\nATOM    973  CA  THR A 124      21.118  20.200  23.415  1.00 17.85           C  \\nATOM    974  C   THR A 124      20.413  19.068  22.656  1.00 17.08           C  \\nATOM    975  O   THR A 124      20.459  17.926  23.091  1.00 19.59           O  \\nATOM    976  CB  THR A 124      20.311  20.578  24.672  1.00 19.42           C  \\nATOM    977  OG1 THR A 124      19.109  21.230  24.277  1.00 20.50           O  \\nATOM    978  CG2 THR A 124      21.155  21.458  25.588  1.00 19.06           C  \\nATOM    979  N   VAL A 125      19.788  19.319  21.495  1.00 16.59           N  \\nATOM    980  CA  VAL A 125      19.133  18.324  20.647  1.00 16.09           C  \\nATOM    981  C   VAL A 125      20.032  18.080  19.413  1.00 16.61           C  \\nATOM    982  O   VAL A 125      20.180  18.980  18.573  1.00 14.07           O  \\nATOM    983  CB  VAL A 125      17.720  18.848  20.179  1.00 15.28           C  \\nATOM    984  CG1 VAL A 125      17.097  17.814  19.265  1.00 14.81           C  \\nATOM    985  CG2 VAL A 125      16.824  19.168  21.366  1.00 15.97           C  \\nATOM    986  N   GLU A 126      20.659  16.889  19.280  1.00 17.05           N  \\nATOM    987  CA  GLU A 126      21.487  16.568  18.124  1.00 19.28           C  \\nATOM    988  C   GLU A 126      20.631  16.496  16.876  1.00 18.51           C  \\nATOM    989  O   GLU A 126      19.511  15.976  16.880  1.00 17.68           O  \\nATOM    990  CB  GLU A 126      22.165  15.215  18.219  1.00 23.29           C  \\nATOM    991  CG  GLU A 126      23.244  15.057  19.265  1.00 30.86           C  \\nATOM    992  CD  GLU A 126      24.402  16.054  19.153  1.00 36.59           C  \\nATOM    993  OE1 GLU A 126      25.139  16.012  18.161  1.00 39.64           O  \\nATOM    994  OE2 GLU A 126      24.581  16.885  20.056  1.00 40.44           O  \\nATOM    995  N   SER A 127      21.167  16.985  15.755  1.00 17.73           N  \\nATOM    996  CA  SER A 127      20.426  16.938  14.493  1.00 16.72           C  \\nATOM    997  C   SER A 127      20.040  15.506  14.158  1.00 16.76           C  \\nATOM    998  O   SER A 127      18.931  15.245  13.690  1.00 17.01           O  \\nATOM    999  CB  SER A 127      21.283  17.518  13.371  1.00 17.67           C  \\nATOM   1000  OG  SER A 127      21.730  18.826  13.768  1.00 19.67           O  \\nATOM   1001  N   ARG A 128      20.916  14.558  14.466  1.00 16.72           N  \\nATOM   1002  CA  ARG A 128      20.667  13.154  14.145  1.00 20.94           C  \\nATOM   1003  C   ARG A 128      19.435  12.617  14.862  1.00 19.62           C  \\nATOM   1004  O   ARG A 128      18.641  11.882  14.278  1.00 18.81           O  \\nATOM   1005  CB  ARG A 128      21.910  12.319  14.510  1.00 24.31           C  \\nATOM   1006  CG  ARG A 128      21.969  11.035  13.692  1.00 30.88           C  \\nATOM   1007  CD  ARG A 128      23.381  10.403  13.536  1.00 35.88           C  \\nATOM   1008  NE  ARG A 128      24.384  10.996  12.621  1.00 39.57           N  \\nATOM   1009  CZ  ARG A 128      24.313  11.018  11.267  1.00 40.85           C  \\nATOM   1010  NH1 ARG A 128      25.399  11.388  10.577  1.00 40.83           N  \\nATOM   1011  NH2 ARG A 128      23.202  10.685  10.576  1.00 41.33           N  \\nATOM   1012  N   GLN A 129      19.209  13.016  16.119  1.00 18.27           N  \\nATOM   1013  CA  GLN A 129      18.032  12.519  16.787  1.00 18.53           C  \\nATOM   1014  C   GLN A 129      16.735  13.151  16.224  1.00 17.11           C  \\nATOM   1015  O   GLN A 129      15.691  12.486  16.098  1.00 15.48           O  \\nATOM   1016  CB  GLN A 129      18.229  12.766  18.306  1.00 22.36           C  \\nATOM   1017  CG  GLN A 129      18.041  14.133  18.911  1.00 24.78           C  \\nATOM   1018  CD  GLN A 129      18.283  14.123  20.411  1.00 27.48           C  \\nATOM   1019  OE1 GLN A 129      19.287  14.643  20.906  1.00 28.14           O  \\nATOM   1020  NE2 GLN A 129      17.367  13.497  21.152  1.00 27.21           N  \\nATOM   1021  N   ALA A 130      16.780  14.446  15.837  1.00 14.73           N  \\nATOM   1022  CA  ALA A 130      15.611  15.044  15.210  1.00 13.87           C  \\nATOM   1023  C   ALA A 130      15.342  14.449  13.826  1.00 13.30           C  \\nATOM   1024  O   ALA A 130      14.199  14.194  13.436  1.00 11.42           O  \\nATOM   1025  CB  ALA A 130      15.822  16.536  15.063  1.00 16.21           C  \\nATOM   1026  N   GLN A 131      16.401  14.189  13.057  1.00 14.61           N  \\nATOM   1027  CA  GLN A 131      16.267  13.529  11.739  1.00 16.01           C  \\nATOM   1028  C   GLN A 131      15.637  12.109  11.821  1.00 16.29           C  \\nATOM   1029  O   GLN A 131      14.777  11.708  11.017  1.00 14.37           O  \\nATOM   1030  CB  GLN A 131      17.673  13.507  11.116  1.00 18.12           C  \\nATOM   1031  CG  GLN A 131      17.702  13.017   9.689  1.00 22.19           C  \\nATOM   1032  CD  GLN A 131      16.889  13.922   8.801  1.00 21.61           C  \\nATOM   1033  OE1 GLN A 131      17.292  15.044   8.529  1.00 24.06           O  \\nATOM   1034  NE2 GLN A 131      15.714  13.485   8.370  1.00 23.37           N  \\nATOM   1035  N   ASP A 132      16.038  11.348  12.853  1.00 16.64           N  \\nATOM   1036  CA  ASP A 132      15.467  10.038  13.173  1.00 18.00           C  \\nATOM   1037  C   ASP A 132      13.984  10.175  13.484  1.00 17.28           C  \\nATOM   1038  O   ASP A 132      13.178   9.395  12.985  1.00 16.45           O  \\nATOM   1039  CB  ASP A 132      16.120   9.415  14.405  1.00 20.10           C  \\nATOM   1040  CG  ASP A 132      17.506   8.816  14.234  1.00 24.07           C  \\nATOM   1041  OD1 ASP A 132      18.120   8.459  15.243  1.00 28.35           O  \\nATOM   1042  OD2 ASP A 132      17.975   8.686  13.106  1.00 25.43           O  \\nATOM   1043  N   LEU A 133      13.614  11.168  14.312  1.00 15.58           N  \\nATOM   1044  CA  LEU A 133      12.213  11.390  14.624  1.00 16.28           C  \\nATOM   1045  C   LEU A 133      11.421  11.729  13.366  1.00 14.96           C  \\nATOM   1046  O   LEU A 133      10.345  11.181  13.128  1.00 15.09           O  \\nATOM   1047  CB  LEU A 133      12.052  12.536  15.647  1.00 17.06           C  \\nATOM   1048  CG  LEU A 133      10.617  12.781  16.136  1.00 18.79           C  \\nATOM   1049  CD1 LEU A 133      10.142  11.554  16.881  1.00 19.96           C  \\nATOM   1050  CD2 LEU A 133      10.544  14.013  17.020  1.00 18.45           C  \\nATOM   1051  N   ALA A 134      11.937  12.642  12.545  1.00 13.16           N  \\nATOM   1052  CA  ALA A 134      11.241  13.016  11.331  1.00 14.89           C  \\nATOM   1053  C   ALA A 134      11.092  11.821  10.414  1.00 16.19           C  \\nATOM   1054  O   ALA A 134      10.015  11.643   9.857  1.00 15.70           O  \\nATOM   1055  CB  ALA A 134      12.021  14.089  10.629  1.00 15.12           C  \\nATOM   1056  N   ARG A 135      12.143  10.995  10.279  1.00 17.00           N  \\nATOM   1057  CA  ARG A 135      12.088   9.828   9.438  1.00 19.17           C  \\nATOM   1058  C   ARG A 135      10.990   8.893   9.916  1.00 18.70           C  \\nATOM   1059  O   ARG A 135      10.269   8.352   9.078  1.00 20.29           O  \\nATOM   1060  CB  ARG A 135      13.425   9.091   9.456  1.00 22.48           C  \\nATOM   1061  CG  ARG A 135      13.499   8.297   8.155  1.00 29.02           C  \\nATOM   1062  CD  ARG A 135      14.619   7.270   7.988  1.00 33.22           C  \\nATOM   1063  NE  ARG A 135      14.117   5.888   8.050  1.00 38.80           N  \\nATOM   1064  CZ  ARG A 135      13.281   5.302   7.148  1.00 39.60           C  \\nATOM   1065  NH1 ARG A 135      12.889   4.044   7.384  1.00 40.80           N  \\nATOM   1066  NH2 ARG A 135      12.827   5.913   6.034  1.00 40.11           N  \\nATOM   1067  N   SER A 136      10.815   8.697  11.242  1.00 19.23           N  \\nATOM   1068  CA  SER A 136       9.757   7.841  11.770  1.00 19.12           C  \\nATOM   1069  C   SER A 136       8.374   8.432  11.528  1.00 19.61           C  \\nATOM   1070  O   SER A 136       7.398   7.690  11.522  1.00 21.01           O  \\nATOM   1071  CB  SER A 136       9.930   7.624  13.280  1.00 19.94           C  \\nATOM   1072  OG  SER A 136       9.459   8.686  14.119  1.00 23.21           O  \\nATOM   1073  N   TYR A 137       8.231   9.760  11.380  1.00 17.33           N  \\nATOM   1074  CA  TYR A 137       6.950  10.375  11.078  1.00 18.12           C  \\nATOM   1075  C   TYR A 137       6.683  10.404   9.570  1.00 18.22           C  \\nATOM   1076  O   TYR A 137       5.561  10.656   9.129  1.00 19.27           O  \\nATOM   1077  CB  TYR A 137       6.907  11.812  11.592  1.00 18.21           C  \\nATOM   1078  CG  TYR A 137       6.883  12.057  13.084  1.00 19.08           C  \\nATOM   1079  CD1 TYR A 137       6.619  11.026  13.955  1.00 20.51           C  \\nATOM   1080  CD2 TYR A 137       7.059  13.355  13.548  1.00 20.43           C  \\nATOM   1081  CE1 TYR A 137       6.510  11.285  15.306  1.00 21.89           C  \\nATOM   1082  CE2 TYR A 137       6.945  13.624  14.899  1.00 21.02           C  \\nATOM   1083  CZ  TYR A 137       6.662  12.574  15.763  1.00 22.13           C  \\nATOM   1084  OH  TYR A 137       6.476  12.782  17.112  1.00 22.16           O  \\nATOM   1085  N   GLY A 138       7.689  10.186   8.737  1.00 16.07           N  \\nATOM   1086  CA  GLY A 138       7.520  10.265   7.294  1.00 17.66           C  \\nATOM   1087  C   GLY A 138       7.476  11.725   6.810  1.00 18.01           C  \\nATOM   1088  O   GLY A 138       6.859  12.010   5.774  1.00 17.85           O  \\nATOM   1089  N   ILE A 139       8.128  12.662   7.540  1.00 14.16           N  \\nATOM   1090  CA  ILE A 139       8.117  14.065   7.145  1.00 13.49           C  \\nATOM   1091  C   ILE A 139       9.526  14.662   7.033  1.00 14.35           C  \\nATOM   1092  O   ILE A 139      10.494  14.152   7.610  1.00 16.19           O  \\nATOM   1093  CB  ILE A 139       7.292  14.896   8.161  1.00 14.40           C  \\nATOM   1094  CG1 ILE A 139       7.925  14.811   9.554  1.00 15.44           C  \\nATOM   1095  CG2 ILE A 139       5.821  14.422   8.104  1.00 14.07           C  \\nATOM   1096  CD1 ILE A 139       7.288  15.765  10.587  1.00 15.35           C  \\nATOM   1097  N   PRO A 140       9.706  15.767   6.308  1.00 13.55           N  \\nATOM   1098  CA  PRO A 140      10.984  16.448   6.205  1.00 13.39           C  \\nATOM   1099  C   PRO A 140      11.424  17.065   7.532  1.00 12.74           C  \\nATOM   1100  O   PRO A 140      10.601  17.530   8.310  1.00 11.42           O  \\nATOM   1101  CB  PRO A 140      10.767  17.498   5.127  1.00 13.47           C  \\nATOM   1102  CG  PRO A 140       9.514  17.084   4.410  1.00 14.09           C  \\nATOM   1103  CD  PRO A 140       8.686  16.467   5.519  1.00 13.14           C  \\nATOM   1104  N   TYR A 141      12.744  17.086   7.756  1.00 11.68           N  \\nATOM   1105  CA  TYR A 141      13.336  17.791   8.890  1.00 10.89           C  \\nATOM   1106  C   TYR A 141      14.175  18.945   8.326  1.00 10.09           C  \\nATOM   1107  O   TYR A 141      15.056  18.720   7.468  1.00 10.27           O  \\nATOM   1108  CB  TYR A 141      14.276  16.897   9.691  1.00 12.54           C  \\nATOM   1109  CG  TYR A 141      15.063  17.676  10.733  1.00 10.92           C  \\nATOM   1110  CD1 TYR A 141      14.350  18.396  11.664  1.00 11.75           C  \\nATOM   1111  CD2 TYR A 141      16.453  17.633  10.784  1.00 12.49           C  \\nATOM   1112  CE1 TYR A 141      14.997  19.074  12.670  1.00 12.85           C  \\nATOM   1113  CE2 TYR A 141      17.112  18.307  11.794  1.00 11.33           C  \\nATOM   1114  CZ  TYR A 141      16.369  19.016  12.728  1.00 12.61           C  \\nATOM   1115  OH  TYR A 141      16.944  19.646  13.813  1.00 13.82           O  \\nATOM   1116  N   ILE A 142      13.892  20.174   8.799  1.00  9.35           N  \\nATOM   1117  CA  ILE A 142      14.636  21.355   8.376  1.00  9.05           C  \\nATOM   1118  C   ILE A 142      15.153  22.163   9.551  1.00  9.25           C  \\nATOM   1119  O   ILE A 142      14.409  22.382  10.499  1.00 10.43           O  \\nATOM   1120  CB  ILE A 142      13.756  22.259   7.472  1.00  9.25           C  \\nATOM   1121  CG1 ILE A 142      13.523  21.470   6.180  1.00 10.51           C  \\nATOM   1122  CG2 ILE A 142      14.450  23.586   7.122  1.00  9.08           C  \\nATOM   1123  CD1 ILE A 142      12.719  22.127   5.028  1.00 13.16           C  \\nATOM   1124  N   GLU A 143      16.457  22.513   9.568  1.00  9.01           N  \\nATOM   1125  CA  GLU A 143      16.979  23.404  10.611  1.00  8.52           C  \\nATOM   1126  C   GLU A 143      16.978  24.864  10.193  1.00 10.31           C  \\nATOM   1127  O   GLU A 143      17.278  25.228   9.040  1.00 10.45           O  \\nATOM   1128  CB  GLU A 143      18.397  23.022  11.013  1.00 10.87           C  \\nATOM   1129  CG  GLU A 143      18.363  21.728  11.801  1.00 10.22           C  \\nATOM   1130  CD  GLU A 143      19.592  21.411  12.633  1.00 13.01           C  \\nATOM   1131  OE1 GLU A 143      20.616  22.082  12.521  1.00 13.65           O  \\nATOM   1132  OE2 GLU A 143      19.511  20.487  13.429  1.00 12.70           O  \\nATOM   1133  N   THR A 144      16.608  25.762  11.117  1.00  9.18           N  \\nATOM   1134  CA  THR A 144      16.422  27.169  10.754  1.00  9.71           C  \\nATOM   1135  C   THR A 144      17.014  28.076  11.789  1.00  8.66           C  \\nATOM   1136  O   THR A 144      17.259  27.698  12.927  1.00  9.13           O  \\nATOM   1137  CB  THR A 144      14.919  27.617  10.635  1.00  9.90           C  \\nATOM   1138  OG1 THR A 144      14.336  27.403  11.931  1.00  9.29           O  \\nATOM   1139  CG2 THR A 144      14.120  26.858   9.557  1.00  9.89           C  \\nATOM   1140  N   SER A 145      17.244  29.288  11.360  1.00 10.91           N  \\nATOM   1141  CA  SER A 145      17.558  30.377  12.290  1.00 12.65           C  \\nATOM   1142  C   SER A 145      16.758  31.602  11.849  1.00 10.37           C  \\nATOM   1143  O   SER A 145      16.930  32.094  10.727  1.00 11.76           O  \\nATOM   1144  CB  SER A 145      19.069  30.768  12.308  1.00 11.79           C  \\nATOM   1145  OG  SER A 145      19.248  31.931  13.125  1.00 12.12           O  \\nATOM   1146  N   ALA A 146      15.838  32.046  12.696  1.00  9.49           N  \\nATOM   1147  CA  ALA A 146      15.131  33.268  12.413  1.00 10.90           C  \\nATOM   1148  C   ALA A 146      16.070  34.455  12.553  1.00 13.53           C  \\nATOM   1149  O   ALA A 146      15.799  35.514  11.995  1.00 16.20           O  \\nATOM   1150  CB  ALA A 146      13.976  33.436  13.367  1.00 12.65           C  \\nATOM   1151  N   LYS A 147      17.183  34.302  13.283  1.00 14.35           N  \\nATOM   1152  CA  LYS A 147      18.150  35.351  13.486  1.00 16.25           C  \\nATOM   1153  C   LYS A 147      18.953  35.568  12.215  1.00 17.58           C  \\nATOM   1154  O   LYS A 147      19.124  36.724  11.825  1.00 20.13           O  \\nATOM   1155  CB  LYS A 147      19.068  34.970  14.617  1.00 17.55           C  \\nATOM   1156  CG  LYS A 147      20.003  36.125  14.958  1.00 22.83           C  \\nATOM   1157  CD  LYS A 147      20.581  35.932  16.341  1.00 26.44           C  \\nATOM   1158  CE  LYS A 147      21.683  36.949  16.507  1.00 30.79           C  \\nATOM   1159  NZ  LYS A 147      22.575  36.534  17.574  1.00 33.81           N  \\nATOM   1160  N   THR A 148      19.479  34.529  11.541  1.00 17.16           N  \\nATOM   1161  CA  THR A 148      20.248  34.735  10.300  1.00 15.93           C  \\nATOM   1162  C   THR A 148      19.422  34.607   9.013  1.00 17.72           C  \\nATOM   1163  O   THR A 148      19.926  34.962   7.945  1.00 18.47           O  \\nATOM   1164  CB  THR A 148      21.444  33.745  10.209  1.00 15.86           C  \\nATOM   1165  OG1 THR A 148      20.886  32.435  10.042  1.00 14.56           O  \\nATOM   1166  CG2 THR A 148      22.377  33.839  11.439  1.00 15.25           C  \\nATOM   1167  N   ARG A 149      18.176  34.108   9.100  1.00 15.18           N  \\nATOM   1168  CA  ARG A 149      17.268  33.815   7.994  1.00 19.20           C  \\nATOM   1169  C   ARG A 149      17.652  32.483   7.318  1.00 17.94           C  \\nATOM   1170  O   ARG A 149      17.031  32.079   6.332  1.00 20.28           O  \\nATOM   1171  CB  ARG A 149      17.251  34.964   6.937  1.00 21.40           C  \\nATOM   1172  CG  ARG A 149      15.929  35.106   6.188  1.00 27.00           C  \\nATOM   1173  CD  ARG A 149      15.946  35.904   4.863  1.00 27.99           C  \\nATOM   1174  NE  ARG A 149      14.707  35.687   4.092  1.00 29.40           N  \\nATOM   1175  CZ  ARG A 149      13.550  36.232   4.492  1.00 29.07           C  \\nATOM   1176  NH1 ARG A 149      12.412  36.015   3.828  1.00 29.56           N  \\nATOM   1177  NH2 ARG A 149      13.527  37.004   5.577  1.00 28.99           N  \\nATOM   1178  N   GLN A 150      18.621  31.701   7.838  1.00 15.80           N  \\nATOM   1179  CA  GLN A 150      18.926  30.402   7.273  1.00 16.36           C  \\nATOM   1180  C   GLN A 150      17.737  29.464   7.455  1.00 15.02           C  \\nATOM   1181  O   GLN A 150      17.119  29.351   8.504  1.00 13.57           O  \\nATOM   1182  CB  GLN A 150      20.152  29.729   7.937  1.00 23.05           C  \\nATOM   1183  CG  GLN A 150      20.578  28.448   7.162  1.00 29.37           C  \\nATOM   1184  CD  GLN A 150      21.604  27.530   7.827  1.00 34.84           C  \\nATOM   1185  OE1 GLN A 150      21.301  26.416   8.280  1.00 38.95           O  \\nATOM   1186  NE2 GLN A 150      22.857  27.967   7.883  1.00 35.22           N  \\nATOM   1187  N   GLY A 151      17.412  28.809   6.348  1.00 13.62           N  \\nATOM   1188  CA  GLY A 151      16.357  27.835   6.296  1.00 13.50           C  \\nATOM   1189  C   GLY A 151      14.943  28.395   6.329  1.00 12.36           C  \\nATOM   1190  O   GLY A 151      14.025  27.591   6.148  1.00 11.07           O  \\nATOM   1191  N   VAL A 152      14.706  29.702   6.510  1.00 12.59           N  \\nATOM   1192  CA  VAL A 152      13.339  30.183   6.715  1.00 14.23           C  \\nATOM   1193  C   VAL A 152      12.435  29.971   5.475  1.00 13.41           C  \\nATOM   1194  O   VAL A 152      11.352  29.362   5.577  1.00 12.94           O  \\nATOM   1195  CB  VAL A 152      13.430  31.691   7.167  1.00 12.57           C  \\nATOM   1196  CG1 VAL A 152      12.059  32.316   7.194  1.00 13.14           C  \\nATOM   1197  CG2 VAL A 152      14.044  31.787   8.543  1.00 12.69           C  \\nATOM   1198  N   GLU A 153      12.831  30.410   4.278  1.00 13.52           N  \\nATOM   1199  CA  GLU A 153      12.001  30.213   3.086  1.00 14.81           C  \\nATOM   1200  C   GLU A 153      11.857  28.756   2.742  1.00 11.88           C  \\nATOM   1201  O   GLU A 153      10.797  28.282   2.344  1.00 13.92           O  \\nATOM   1202  CB  GLU A 153      12.592  30.936   1.881  1.00 17.03           C  \\nATOM   1203  CG  GLU A 153      12.454  32.452   2.015  1.00 21.18           C  \\nATOM   1204  CD  GLU A 153      12.870  33.318   0.817  1.00 25.73           C  \\nATOM   1205  OE1 GLU A 153      13.232  32.781  -0.234  1.00 28.32           O  \\nATOM   1206  OE2 GLU A 153      12.822  34.550   0.936  1.00 26.64           O  \\nATOM   1207  N   ASP A 154      12.959  28.031   2.957  1.00 11.31           N  \\nATOM   1208  CA  ASP A 154      13.023  26.593   2.747  1.00 10.16           C  \\nATOM   1209  C   ASP A 154      11.964  25.828   3.561  1.00 10.90           C  \\nATOM   1210  O   ASP A 154      11.224  24.985   3.035  1.00 11.83           O  \\nATOM   1211  CB  ASP A 154      14.420  26.212   3.117  1.00 15.00           C  \\nATOM   1212  CG  ASP A 154      14.919  24.942   2.524  1.00 19.04           C  \\nATOM   1213  OD1 ASP A 154      14.121  24.195   1.938  1.00 22.96           O  \\nATOM   1214  OD2 ASP A 154      16.122  24.697   2.676  1.00 21.26           O  \\nATOM   1215  N   ALA A 155      11.854  26.107   4.881  1.00  8.83           N  \\nATOM   1216  CA  ALA A 155      10.841  25.450   5.716  1.00 10.00           C  \\nATOM   1217  C   ALA A 155       9.424  25.748   5.203  1.00 10.10           C  \\nATOM   1218  O   ALA A 155       8.653  24.814   4.978  1.00 11.14           O  \\nATOM   1219  CB  ALA A 155      10.923  25.933   7.192  1.00  9.50           C  \\nATOM   1220  N   PHE A 156       9.054  27.011   4.949  1.00 10.59           N  \\nATOM   1221  CA  PHE A 156       7.712  27.345   4.475  1.00 10.51           C  \\nATOM   1222  C   PHE A 156       7.453  26.825   3.069  1.00 11.05           C  \\nATOM   1223  O   PHE A 156       6.368  26.302   2.801  1.00 10.80           O  \\nATOM   1224  CB  PHE A 156       7.492  28.890   4.506  1.00 11.54           C  \\nATOM   1225  CG  PHE A 156       7.266  29.450   5.936  1.00 12.42           C  \\nATOM   1226  CD1 PHE A 156       8.319  29.958   6.707  1.00 12.00           C  \\nATOM   1227  CD2 PHE A 156       5.978  29.434   6.478  1.00 12.57           C  \\nATOM   1228  CE1 PHE A 156       8.065  30.434   8.001  1.00 12.45           C  \\nATOM   1229  CE2 PHE A 156       5.747  29.908   7.757  1.00 12.32           C  \\nATOM   1230  CZ  PHE A 156       6.782  30.406   8.513  1.00 10.18           C  \\nATOM   1231  N   TYR A 157       8.408  26.935   2.151  1.00  9.69           N  \\nATOM   1232  CA  TYR A 157       8.148  26.463   0.798  1.00 12.31           C  \\nATOM   1233  C   TYR A 157       8.091  24.945   0.719  1.00 10.42           C  \\nATOM   1234  O   TYR A 157       7.292  24.403  -0.047  1.00 11.35           O  \\nATOM   1235  CB  TYR A 157       9.233  26.978  -0.156  1.00 15.16           C  \\nATOM   1236  CG  TYR A 157       9.257  28.492  -0.367  1.00 18.73           C  \\nATOM   1237  CD1 TYR A 157       8.380  29.366   0.268  1.00 18.52           C  \\nATOM   1238  CD2 TYR A 157      10.211  28.971  -1.242  1.00 20.44           C  \\nATOM   1239  CE1 TYR A 157       8.466  30.724   0.025  1.00 22.04           C  \\nATOM   1240  CE2 TYR A 157      10.305  30.320  -1.488  1.00 22.70           C  \\nATOM   1241  CZ  TYR A 157       9.436  31.178  -0.852  1.00 23.03           C  \\nATOM   1242  OH  TYR A 157       9.572  32.528  -1.089  1.00 26.15           O  \\nATOM   1243  N   THR A 158       8.940  24.198   1.455  1.00 11.35           N  \\nATOM   1244  CA  THR A 158       8.836  22.727   1.522  1.00 10.11           C  \\nATOM   1245  C   THR A 158       7.440  22.347   2.064  1.00 11.11           C  \\nATOM   1246  O   THR A 158       6.811  21.408   1.558  1.00 11.43           O  \\nATOM   1247  CB  THR A 158       9.935  22.129   2.470  1.00 12.09           C  \\nATOM   1248  OG1 THR A 158      11.216  22.451   1.933  1.00 11.36           O  \\nATOM   1249  CG2 THR A 158       9.791  20.657   2.624  1.00 10.87           C  \\nATOM   1250  N   LEU A 159       6.888  23.056   3.072  1.00  9.25           N  \\nATOM   1251  CA  LEU A 159       5.533  22.754   3.506  1.00  9.93           C  \\nATOM   1252  C   LEU A 159       4.489  22.986   2.388  1.00 10.42           C  \\nATOM   1253  O   LEU A 159       3.560  22.185   2.272  1.00 11.22           O  \\nATOM   1254  CB  LEU A 159       5.171  23.608   4.711  1.00  9.03           C  \\nATOM   1255  CG  LEU A 159       3.789  23.318   5.269  1.00 10.96           C  \\nATOM   1256  CD1 LEU A 159       3.635  21.828   5.651  1.00  9.29           C  \\nATOM   1257  CD2 LEU A 159       3.580  24.223   6.469  1.00 13.71           C  \\nATOM   1258  N   VAL A 160       4.596  24.046   1.548  1.00 10.91           N  \\nATOM   1259  CA  VAL A 160       3.662  24.284   0.439  1.00  9.19           C  \\nATOM   1260  C   VAL A 160       3.803  23.137  -0.556  1.00  9.60           C  \\nATOM   1261  O   VAL A 160       2.781  22.638  -1.031  1.00 10.29           O  \\nATOM   1262  CB  VAL A 160       3.981  25.653  -0.211  1.00  8.50           C  \\nATOM   1263  CG1 VAL A 160       3.402  25.754  -1.622  1.00 10.56           C  \\nATOM   1264  CG2 VAL A 160       3.403  26.741   0.659  1.00  7.98           C  \\nATOM   1265  N   ARG A 161       5.033  22.667  -0.839  1.00  9.10           N  \\nATOM   1266  CA  ARG A 161       5.206  21.554  -1.767  1.00  9.82           C  \\nATOM   1267  C   ARG A 161       4.644  20.240  -1.215  1.00 10.44           C  \\nATOM   1268  O   ARG A 161       4.166  19.395  -1.984  1.00 12.88           O  \\nATOM   1269  CB  ARG A 161       6.679  21.398  -2.115  1.00  9.39           C  \\nATOM   1270  CG  ARG A 161       7.146  22.640  -2.816  1.00  9.70           C  \\nATOM   1271  CD  ARG A 161       8.594  22.398  -3.227  1.00 14.12           C  \\nATOM   1272  NE  ARG A 161       9.122  23.635  -3.780  1.00 14.15           N  \\nATOM   1273  CZ  ARG A 161      10.144  24.313  -3.257  1.00 12.45           C  \\nATOM   1274  NH1 ARG A 161      10.823  23.858  -2.204  1.00 13.24           N  \\nATOM   1275  NH2 ARG A 161      10.485  25.475  -3.819  1.00 13.52           N  \\nATOM   1276  N   GLU A 162       4.663  20.046   0.112  1.00 10.71           N  \\nATOM   1277  CA  GLU A 162       4.054  18.877   0.722  1.00 11.80           C  \\nATOM   1278  C   GLU A 162       2.522  18.887   0.568  1.00 11.20           C  \\nATOM   1279  O   GLU A 162       1.921  17.848   0.274  1.00 12.90           O  \\nATOM   1280  CB  GLU A 162       4.430  18.815   2.216  1.00 13.86           C  \\nATOM   1281  CG  GLU A 162       5.870  18.460   2.580  1.00 14.10           C  \\nATOM   1282  CD  GLU A 162       6.268  17.038   2.224  1.00 17.23           C  \\nATOM   1283  OE1 GLU A 162       5.571  16.116   2.627  1.00 19.29           O  \\nATOM   1284  OE2 GLU A 162       7.278  16.827   1.545  1.00 22.14           O  \\nATOM   1285  N   ILE A 163       1.869  20.045   0.731  1.00 10.13           N  \\nATOM   1286  CA  ILE A 163       0.426  20.175   0.542  1.00 10.58           C  \\nATOM   1287  C   ILE A 163       0.128  19.878  -0.933  1.00 13.43           C  \\nATOM   1288  O   ILE A 163      -0.796  19.118  -1.235  1.00 15.01           O  \\nATOM   1289  CB  ILE A 163      -0.015  21.590   0.891  1.00 11.33           C  \\nATOM   1290  CG1 ILE A 163       0.084  21.777   2.400  1.00 11.27           C  \\nATOM   1291  CG2 ILE A 163      -1.486  21.815   0.437  1.00 12.37           C  \\nATOM   1292  CD1 ILE A 163      -0.006  23.245   2.894  1.00 12.56           C  \\nATOM   1293  N   ARG A 164       0.904  20.456  -1.871  1.00 11.72           N  \\nATOM   1294  CA  ARG A 164       0.770  20.163  -3.283  1.00 13.64           C  \\nATOM   1295  C   ARG A 164       0.813  18.657  -3.626  1.00 17.97           C  \\nATOM   1296  O   ARG A 164       0.150  18.197  -4.559  1.00 18.12           O  \\nATOM   1297  CB  ARG A 164       1.897  20.887  -4.029  1.00 12.51           C  \\nATOM   1298  CG  ARG A 164       1.647  22.371  -4.120  1.00 12.49           C  \\nATOM   1299  CD  ARG A 164       2.801  22.888  -4.950  1.00 14.72           C  \\nATOM   1300  NE  ARG A 164       2.730  24.327  -5.113  1.00 12.82           N  \\nATOM   1301  CZ  ARG A 164       3.800  25.071  -5.405  1.00 13.29           C  \\nATOM   1302  NH1 ARG A 164       5.018  24.539  -5.527  1.00 11.56           N  \\nATOM   1303  NH2 ARG A 164       3.595  26.381  -5.572  1.00 12.55           N  \\nATOM   1304  N   GLN A 165       1.590  17.831  -2.935  1.00 20.23           N  \\nATOM   1305  CA  GLN A 165       1.673  16.435  -3.337  1.00 27.50           C  \\nATOM   1306  C   GLN A 165       0.782  15.499  -2.502  1.00 29.12           C  \\nATOM   1307  O   GLN A 165       0.826  14.274  -2.655  1.00 29.30           O  \\nATOM   1308  CB  GLN A 165       3.168  16.070  -3.286  1.00 30.34           C  \\nATOM   1309  CG  GLN A 165       4.018  16.904  -4.309  1.00 35.17           C  \\nATOM   1310  CD  GLN A 165       4.109  16.470  -5.796  1.00 37.71           C  \\nATOM   1311  OE1 GLN A 165       4.256  17.293  -6.719  1.00 36.70           O  \\nATOM   1312  NE2 GLN A 165       4.079  15.166  -6.094  1.00 38.13           N  \\nATOM   1313  N   HIS A 166      -0.068  16.075  -1.638  1.00 29.28           N  \\nATOM   1314  CA  HIS A 166      -0.982  15.296  -0.820  1.00 31.13           C  \\nATOM   1315  C   HIS A 166      -2.214  14.971  -1.665  1.00 33.30           C  \\nATOM   1316  O   HIS A 166      -2.630  15.786  -2.497  1.00 34.96           O  \\nATOM   1317  CB  HIS A 166      -1.420  16.090   0.428  1.00 30.89           C  \\nATOM   1318  CG  HIS A 166      -2.247  15.319   1.475  1.00 32.17           C  \\nATOM   1319  ND1 HIS A 166      -3.579  15.231   1.603  1.00 33.52           N  \\nATOM   1320  CD2 HIS A 166      -1.712  14.560   2.502  1.00 32.95           C  \\nATOM   1321  CE1 HIS A 166      -3.857  14.464   2.647  1.00 33.07           C  \\nATOM   1322  NE2 HIS A 166      -2.727  14.064   3.184  1.00 33.06           N  \\nATOM   1323  OXT HIS A 166      -2.763  13.888  -1.454  1.00 36.27           O  \\nTER    1324      HIS A 166                                                      \\nHETATM 1325 MG    MG A 168       4.771  33.860  19.213  1.00  9.42          MG  \\nHETATM 1326  PG  GNP A 167       5.113  32.223  21.994  1.00  9.66           P  \\nHETATM 1327  O1G GNP A 167       4.844  32.596  23.390  1.00  9.26           O  \\nHETATM 1328  O2G GNP A 167       4.203  32.910  21.112  1.00 11.20           O  \\nHETATM 1329  O3G GNP A 167       4.640  30.769  21.802  1.00 10.09           O  \\nHETATM 1330  N3B GNP A 167       6.701  32.303  21.621  1.00  8.18           N  \\nHETATM 1331  PB  GNP A 167       7.310  32.034  20.111  1.00  8.77           P  \\nHETATM 1332  O1B GNP A 167       7.468  30.605  19.791  1.00  8.35           O  \\nHETATM 1333  O2B GNP A 167       6.696  32.775  19.021  1.00  9.38           O  \\nHETATM 1334  O3A GNP A 167       8.792  32.643  20.152  1.00 10.15           O  \\nHETATM 1335  PA  GNP A 167       9.280  34.140  19.710  1.00 10.18           P  \\nHETATM 1336  O1A GNP A 167       9.263  34.268  18.220  1.00 10.07           O  \\nHETATM 1337  O2A GNP A 167       8.594  35.173  20.510  1.00  9.33           O  \\nHETATM 1338  O5' GNP A 167      10.826  34.056  20.132  1.00 11.46           O  \\nHETATM 1339  C5' GNP A 167      11.170  33.911  21.535  1.00 10.56           C  \\nHETATM 1340  C4' GNP A 167      12.587  34.445  21.811  1.00 11.96           C  \\nHETATM 1341  O4' GNP A 167      13.560  33.649  21.095  1.00 11.66           O  \\nHETATM 1342  C3' GNP A 167      12.813  35.910  21.359  1.00 12.06           C  \\nHETATM 1343  O3' GNP A 167      13.793  36.538  22.221  1.00 14.52           O  \\nHETATM 1344  C2' GNP A 167      13.325  35.679  19.947  1.00 11.90           C  \\nHETATM 1345  O2' GNP A 167      14.151  36.750  19.499  1.00 14.99           O  \\nHETATM 1346  C1' GNP A 167      14.242  34.478  20.156  1.00 12.90           C  \\nHETATM 1347  N9  GNP A 167      14.454  33.739  18.896  1.00 12.98           N  \\nHETATM 1348  C8  GNP A 167      13.518  33.262  17.993  1.00 11.76           C  \\nHETATM 1349  N7  GNP A 167      14.016  32.636  16.979  1.00 12.34           N  \\nHETATM 1350  C5  GNP A 167      15.413  32.694  17.200  1.00 12.36           C  \\nHETATM 1351  C6  GNP A 167      16.521  32.219  16.402  1.00 12.30           C  \\nHETATM 1352  O6  GNP A 167      16.507  31.591  15.338  1.00 12.16           O  \\nHETATM 1353  N1  GNP A 167      17.735  32.503  16.995  1.00 11.60           N  \\nHETATM 1354  C2  GNP A 167      17.909  33.082  18.236  1.00 13.91           C  \\nHETATM 1355  N2  GNP A 167      19.173  33.280  18.658  1.00 14.60           N  \\nHETATM 1356  N3  GNP A 167      16.884  33.498  18.996  1.00 13.01           N  \\nHETATM 1357  C4  GNP A 167      15.675  33.331  18.398  1.00 14.36           C  \\nHETATM 1358  O   HOH A 169      13.588  43.386  17.779  1.00 31.80           O  \\nHETATM 1359  O   HOH A 170       8.248  36.064  23.209  1.00 13.63           O  \\nHETATM 1360  O   HOH A 171       3.254  37.845  19.454  1.00 18.29           O  \\nHETATM 1361  O   HOH A 172       5.945  35.619  20.105  1.00 11.33           O  \\nHETATM 1362  O   HOH A 173       3.669  32.226  18.172  1.00  8.22           O  \\nHETATM 1363  O   HOH A 174       2.896  37.170  15.500  1.00 21.37           O  \\nHETATM 1364  O   HOH A 175       2.102  32.309  24.122  1.00 27.47           O  \\nHETATM 1365  O   HOH A 176      -1.425  37.650  19.667  1.00 24.98           O  \\nHETATM 1366  O   HOH A 177      -3.225  30.031  -8.217  1.00 23.57           O  \\nHETATM 1367  O   HOH A 178      16.678  26.321  26.916  1.00 36.70           O  \\nHETATM 1368  O   HOH A 179      -2.134  40.975   1.551  1.00 81.53           O  \\nHETATM 1369  O   HOH A 180       0.400  40.675  -0.629  1.00 24.66           O  \\nHETATM 1370  O   HOH A 181      10.944  44.793  16.009  1.00 48.47           O  \\nHETATM 1371  O   HOH A 182      17.207  42.570  16.786  1.00 43.23           O  \\nHETATM 1372  O   HOH A 183      14.999  45.167  19.922  1.00 39.52           O  \\nHETATM 1373  O   HOH A 184      11.598  37.755  13.202  1.00 11.72           O  \\nHETATM 1374  O   HOH A 185       6.958  38.922  18.146  1.00 26.17           O  \\nHETATM 1375  O   HOH A 186      14.078  32.033  27.772  1.00 57.73           O  \\nHETATM 1376  O   HOH A 187       9.126  37.906  19.595  1.00 32.42           O  \\nHETATM 1377  O   HOH A 188      -0.049  36.435  27.259  1.00 50.56           O  \\nHETATM 1378  O   HOH A 189       3.843  30.503  25.485  1.00 46.80           O  \\nHETATM 1379  O   HOH A 190       4.931  28.343  27.587  1.00 29.32           O  \\nHETATM 1380  O   HOH A 191       4.881  25.774  26.247  1.00 28.55           O  \\nHETATM 1381  O   HOH A 193       8.729  26.516  26.645  1.00 37.34           O  \\nHETATM 1382  O   HOH A 194      -5.515  23.731  -4.213  1.00 43.74           O  \\nHETATM 1383  O   HOH A 195      -7.006  26.635   0.729  1.00 25.77           O  \\nHETATM 1384  O   HOH A 196      -2.033  23.281  -4.754  1.00 36.89           O  \\nHETATM 1385  O   HOH A 197      -4.326  24.642  -0.854  1.00 28.59           O  \\nHETATM 1386  O   HOH A 198      -6.858  26.995  -2.808  1.00 28.83           O  \\nHETATM 1387  O   HOH A 200      -3.937  29.225   8.289  1.00 24.08           O  \\nHETATM 1388  O   HOH A 201      -8.934  24.037   2.795  1.00 34.16           O  \\nHETATM 1389  O   HOH A 202       1.444  26.302  19.318  1.00 12.37           O  \\nHETATM 1390  O   HOH A 203       2.031  24.632  27.307  1.00 50.09           O  \\nHETATM 1391  O   HOH A 204      15.125  28.341  23.174  1.00 13.88           O  \\nHETATM 1392  O   HOH A 205      12.808  31.541  25.056  1.00 46.88           O  \\nHETATM 1393  O   HOH A 206       8.329  34.933   0.543  1.00 53.21           O  \\nHETATM 1394  O   HOH A 207       0.760  42.969   2.336  1.00 35.42           O  \\nHETATM 1395  O   HOH A 208       1.270  40.298   7.071  1.00 76.61           O  \\nHETATM 1396  O   HOH A 209      -3.135  39.625   7.649  1.00 35.40           O  \\nHETATM 1397  O   HOH A 210       8.699  39.076   2.399  1.00 33.93           O  \\nHETATM 1398  O   HOH A 212      10.040  41.806   3.447  1.00 37.88           O  \\nHETATM 1399  O   HOH A 213      18.914  29.095   3.704  1.00 25.32           O  \\nHETATM 1400  O   HOH A 214      15.966  29.098   2.957  1.00 18.63           O  \\nHETATM 1401  O   HOH A 215      15.506  31.962   4.015  1.00 18.93           O  \\nHETATM 1402  O   HOH A 216      22.935  30.533  10.845  1.00 39.81           O  \\nHETATM 1403  O   HOH A 217       6.476  43.893  23.325  1.00 49.09           O  \\nHETATM 1404  O   HOH A 218      11.746  41.248  28.612  1.00 36.22           O  \\nHETATM 1405  O   HOH A 219      11.480  42.270  25.715  1.00 29.66           O  \\nHETATM 1406  O   HOH A 220       9.344  34.251  -8.278  1.00 36.61           O  \\nHETATM 1407  O   HOH A 221      -0.111  19.667  -8.066  1.00 44.76           O  \\nHETATM 1408  O   HOH A 222      -0.528  30.460 -10.943  1.00 24.29           O  \\nHETATM 1409  O   HOH A 223     -11.352  24.274  19.491  1.00 31.21           O  \\nHETATM 1410  O   HOH A 224      -4.293  30.642 -16.466  1.00127.55           O  \\nHETATM 1411  O   HOH A 225       3.466  29.729 -16.232  1.00 26.15           O  \\nHETATM 1412  O   HOH A 227      -4.288  18.610   4.198  1.00 34.77           O  \\nHETATM 1413  O   HOH A 228       4.626  16.154   5.151  1.00 14.96           O  \\nHETATM 1414  O   HOH A 229       5.681  11.069   2.580  1.00 59.76           O  \\nHETATM 1415  O   HOH A 230       1.173  14.021   3.406  1.00 32.15           O  \\nHETATM 1416  O   HOH A 231       7.935  22.484  20.504  1.00 10.86           O  \\nHETATM 1417  O   HOH A 232       3.071  10.004  14.546  1.00 23.48           O  \\nHETATM 1418  O   HOH A 233       2.437  13.381   8.401  1.00 17.75           O  \\nHETATM 1419  O   HOH A 234       4.779   7.500   6.154  1.00 71.09           O  \\nHETATM 1420  O   HOH A 235      -4.250   8.748   6.880  1.00 24.76           O  \\nHETATM 1421  O   HOH A 236     -13.381  22.830  17.669  1.00 39.09           O  \\nHETATM 1422  O   HOH A 237      16.969  39.052   8.276  1.00 50.25           O  \\nHETATM 1423  O   HOH A 238      -6.315   9.333  13.897  1.00 36.24           O  \\nHETATM 1424  O   HOH A 240      -0.160   4.395  12.468  1.00 38.79           O  \\nHETATM 1425  O   HOH A 241       1.295   7.931  12.335  1.00 49.03           O  \\nHETATM 1426  O   HOH A 242       4.067  11.551   6.968  1.00 46.97           O  \\nHETATM 1427  O   HOH A 243      -5.060   6.579  -0.216  1.00 50.14           O  \\nHETATM 1428  O   HOH A 244      18.164   7.151   5.573  1.00 71.67           O  \\nHETATM 1429  O   HOH A 245       2.068   8.904   7.636  1.00 45.86           O  \\nHETATM 1430  O   HOH A 246      24.661  23.134  12.757  1.00 34.69           O  \\nHETATM 1431  O   HOH A 247      21.051  23.714   6.239  1.00 24.64           O  \\nHETATM 1432  O   HOH A 248      25.855  31.137  20.712  1.00 34.53           O  \\nHETATM 1433  O   HOH A 249      24.868  30.890  26.223  1.00 48.47           O  \\nHETATM 1434  O   HOH A 250      26.967  20.447  19.448  1.00 47.77           O  \\nHETATM 1435  O   HOH A 252      28.453  14.196  14.602  1.00 39.78           O  \\nHETATM 1436  O   HOH A 253      19.639  16.256   9.760  1.00 37.54           O  \\nHETATM 1437  O   HOH A 254      16.629   7.383  10.526  1.00 60.90           O  \\nHETATM 1438  O   HOH A 255       4.972   7.888  14.276  1.00 54.95           O  \\nHETATM 1439  O   HOH A 256      14.184  14.684   5.643  1.00 31.40           O  \\nHETATM 1440  O   HOH A 257      11.766  17.254   1.278  1.00 41.82           O  \\nHETATM 1441  O   HOH A 258      16.274  16.365   6.342  1.00 17.29           O  \\nHETATM 1442  O   HOH A 259      18.346  24.095   6.620  1.00 17.36           O  \\nHETATM 1443  O   HOH A 260      22.479  35.073  19.694  1.00 43.96           O  \\nHETATM 1444  O   HOH A 261      20.580  31.747   4.906  1.00 59.46           O  \\nHETATM 1445  O   HOH A 262      18.196  25.514   4.261  1.00 28.75           O  \\nHETATM 1446  O   HOH A 263      12.857  20.183   1.149  1.00 21.79           O  \\nHETATM 1447  O   HOH A 264      19.514  15.912  24.943  1.00 55.52           O  \\nHETATM 1448  O   HOH A 265       8.925   7.579   5.621  1.00 77.94           O  \\nHETATM 1449  O   HOH A 266      17.622  21.835  26.964  1.00 27.48           O  \\nHETATM 1450  O   HOH A 267      24.258  19.786  14.155  1.00 20.48           O  \\nHETATM 1451  O   HOH A 268      23.806  15.180  15.136  1.00 19.86           O  \\nHETATM 1452  O   HOH A 269      21.478  14.557  11.134  1.00 29.66           O  \\nHETATM 1453  O   HOH A 270      17.293  45.621  24.753  1.00 39.18           O  \\nHETATM 1454  O   HOH A 271      -3.518  40.206  17.649  1.00 44.75           O  \\nHETATM 1455  O   HOH A 272       2.143  19.852  24.090  1.00 40.10           O  \\nHETATM 1456  O   HOH A 273       7.101  35.918  -4.704  1.00 31.76           O  \\nHETATM 1457  O   HOH A 274       5.462  38.756  -5.521  1.00 27.51           O  \\nHETATM 1458  O   HOH A 275       5.748  20.565  28.638  1.00 58.06           O  \\nHETATM 1459  O   HOH A 276       8.814  31.258  -8.964  1.00 23.79           O  \\nHETATM 1460  O   HOH A 277     -10.427  35.895   4.301  1.00 58.91           O  \\nHETATM 1461  O   HOH A 278      -8.713  28.579   2.333  1.00 69.36           O  \\nHETATM 1462  O   HOH A 279      24.472  24.237  17.316  1.00 32.14           O  \\nHETATM 1463  O   HOH A 281      16.968  32.701  21.815  1.00 23.13           O  \\nHETATM 1464  O   HOH A 282      19.541  34.346  21.581  1.00 53.25           O  \\nHETATM 1465  O   HOH A 283      24.847  23.384  26.621  1.00 61.18           O  \\nHETATM 1466  O   HOH A 284       1.409  11.113   2.873  1.00 44.87           O  \\nHETATM 1467  O   HOH A 285       1.556  11.488  -0.914  1.00 37.86           O  \\nHETATM 1468  O   HOH A 286       9.758  12.945   3.889  1.00 36.47           O  \\nHETATM 1469  O   HOH A 287       3.827  13.375   4.622  1.00 44.90           O  \\nHETATM 1470  O   HOH A 288      14.279   6.175  15.391  1.00 78.25           O  \\nHETATM 1471  O   HOH A 289       7.573  13.968   2.191  1.00 76.47           O  \\nHETATM 1472  O   HOH A 290       2.323  15.482   1.418  1.00 35.15           O  \\nHETATM 1473  O   HOH A 291      11.997  14.573   2.962  1.00 49.86           O  \\nHETATM 1474  O   HOH A 292      23.798  33.550  17.880  1.00 24.64           O  \\nHETATM 1475  O   HOH A 293      14.657  36.046  -0.571  1.00 51.86           O  \\nHETATM 1476  O   HOH A 294      24.456  25.366   4.004  1.00 53.23           O  \\nHETATM 1477  O   HOH A 296      -0.603  12.134  -4.185  1.00 52.38           O  \\nHETATM 1478  O   HOH A 297      -8.372  34.415  -4.471  1.00 74.53           O  \\nHETATM 1479  O   HOH A 300     -14.033  36.501   0.062  1.00 50.21           O  \\nHETATM 1480  O   HOH A 301     -17.102  35.735  -2.806  1.00 42.58           O  \\nHETATM 1481  O   HOH A 303     -17.383  40.542   1.431  1.00 77.16           O  \\nHETATM 1482  O   HOH A 304     -19.024  41.557  -3.411  1.00107.16           O  \\nHETATM 1483  O   HOH A 305      -9.876  37.804   0.839  1.00 90.65           O  \\nHETATM 1484  O   HOH A 306     -14.053  36.466   4.552  1.00 82.05           O  \\nHETATM 1485  O   HOH A 307     -14.183  39.558   0.339  1.00 58.66           O  \\nHETATM 1486  O   HOH A 309     -12.547  28.677   8.301  1.00 74.88           O  \\nHETATM 1487  O   HOH A 311     -14.787  32.746  -2.171  1.00 40.79           O  \\nHETATM 1488  O   HOH A 312      -6.916  30.740   3.095  1.00 36.53           O  \\nHETATM 1489  O   HOH A 315      -8.312  33.027   7.340  1.00 83.28           O  \\nHETATM 1490  O   HOH A 316      -6.280  35.646   7.624  1.00 75.97           O  \\nHETATM 1491  O   HOH A 317      -5.230  31.565   9.609  1.00 55.67           O  \\nHETATM 1492  O   HOH A 318       6.747  37.258   1.208  1.00 57.93           O  \\nHETATM 1493  O   HOH A 319       4.747  42.419  12.683  1.00 57.74           O  \\nHETATM 1494  O   HOH A 320       2.984  41.950  16.191  1.00 41.69           O  \\nHETATM 1495  O   HOH A 321       3.867  49.208  12.056  1.00 24.57           O  \\nHETATM 1496  O   HOH A 322       7.530  42.894  17.776  1.00 79.28           O  \\nHETATM 1497  O   HOH A 324      19.822  39.457  17.826  1.00 66.22           O  \\nHETATM 1498  O   HOH A 325      12.391  45.557  24.973  1.00 48.70           O  \\nHETATM 1499  O   HOH A 326      14.944  43.928  23.794  1.00 62.45           O  \\nHETATM 1500  O   HOH A 327     -12.984  24.008  12.563  1.00 40.31           O  \\nHETATM 1501  O   HOH A 330       0.572  41.133  19.277  1.00 50.77           O  \\nHETATM 1502  O   HOH A 331      -6.196  37.979  23.043  1.00 48.93           O  \\nHETATM 1503  O   HOH A 332      -3.526  30.410  19.065  1.00 21.26           O  \\nHETATM 1504  O   HOH A 333      -7.995  32.093  21.685  1.00 54.11           O  \\nHETATM 1505  O   HOH A 334      -8.483  28.891  20.063  1.00 70.37           O  \\nHETATM 1506  O   HOH A 335     -11.670  31.022  16.981  1.00 76.86           O  \\nHETATM 1507  O   HOH A 336     -11.082  34.450  18.910  1.00 43.07           O  \\nHETATM 1508  O   HOH A 339      -6.521  37.738  15.165  1.00 64.11           O  \\nHETATM 1509  O   HOH A 340     -10.829  35.877  15.629  1.00 58.36           O  \\nHETATM 1510  O   HOH A 341      -6.552  41.166  12.231  1.00 55.67           O  \\nHETATM 1511  O   HOH A 342      -1.242  39.671  12.185  1.00 61.04           O  \\nHETATM 1512  O   HOH A 343      -4.281  38.934  13.464  1.00 32.14           O  \\nHETATM 1513  O   HOH A 344       0.386  41.208  14.039  1.00 40.80           O  \\nHETATM 1514  O   HOH A 345       0.777  39.663  21.603  1.00 67.71           O  \\nHETATM 1515  O   HOH A 346      -7.879  34.848  19.044  1.00 52.60           O  \\nHETATM 1516  O   HOH A 347       1.121  32.594 -11.957  1.00 32.30           O  \\nHETATM 1517  O   HOH A 348       0.924  32.597 -14.716  1.00 48.34           O  \\nHETATM 1518  O   HOH A 349      -2.241  27.348  20.565  1.00 26.54           O  \\nHETATM 1519  O   HOH A 350       3.259  26.214  29.881  1.00 69.22           O  \\nHETATM 1520  O   HOH A 351      -8.318  34.095  28.392  1.00 81.96           O  \\nHETATM 1521  O   HOH A 352      -2.927  29.775  31.412  1.00 80.92           O  \\nHETATM 1522  O   HOH A 353     -10.410  32.034  25.972  1.00 60.17           O  \\nHETATM 1523  O   HOH A 354      -8.122  30.393  25.378  1.00 62.33           O  \\nHETATM 1524  O   HOH A 356      -6.138  18.882  24.644  1.00 49.26           O  \\nHETATM 1525  O   HOH A 358      -1.406  19.033  23.084  1.00 82.05           O  \\nHETATM 1526  O   HOH A 359       4.169   7.339  10.280  1.00 66.03           O  \\nHETATM 1527  O   HOH A 360      -9.761  30.165   7.702  1.00 46.15           O  \\nHETATM 1528  O   HOH A 361     -16.002  27.801  11.700  1.00 41.85           O  \\nHETATM 1529  O   HOH A 362      16.215  30.489  29.264  1.00 67.21           O  \\nHETATM 1530  O   HOH A 364      21.417  27.685  26.140  1.00 49.57           O  \\nHETATM 1531  O   HOH A 366      15.824  34.767  30.662  1.00 25.90           O  \\nHETATM 1532  O   HOH A 367      18.527  35.965  24.111  1.00 68.69           O  \\nHETATM 1533  O   HOH A 369      12.103  35.275  24.945  1.00 51.37           O  \\nHETATM 1534  O   HOH A 370      24.782  11.970  23.641  1.00 55.92           O  \\nHETATM 1535  O   HOH A 371      14.337  21.394  28.627  1.00 30.11           O  \\nHETATM 1536  O   HOH A 372      -1.367  12.600  23.053  1.00 44.43           O  \\nHETATM 1537  O   HOH A 373       7.060   8.567  21.416  1.00 47.99           O  \\nHETATM 1538  O   HOH A 374       7.639   8.040  16.420  1.00 38.76           O  \\nHETATM 1539  O   HOH A 376      -1.282  15.566  25.487  1.00 67.79           O  \\nHETATM 1540  O   HOH A 378      17.945  30.924  23.806  1.00 31.50           O  \\nHETATM 1541  O   HOH A 379      17.360  33.209  33.337  1.00 30.46           O  \\nHETATM 1542  O   HOH A 380       3.971   6.567  22.089  1.00 86.80           O  \\nHETATM 1543  O   HOH A 381      -0.065  10.022  26.928  1.00 61.86           O  \\nHETATM 1544  O   HOH A 382      -4.450  12.966  21.222  1.00 42.82           O  \\nHETATM 1545  O   HOH A 383      -2.584   6.719   5.938  1.00 40.64           O  \\nHETATM 1546  O   HOH A 384      25.567  31.936  12.845  1.00 73.17           O  \\nHETATM 1547  O   HOH A 385      24.446  29.383  22.970  1.00 36.61           O  \\nHETATM 1548  O   HOH A 386      24.865  38.167  18.227  1.00 60.74           O  \\nHETATM 1549  O   HOH A 387      23.135  35.012  14.543  1.00 59.19           O  \\nHETATM 1550  O   HOH A 388      19.325   9.556  18.093  1.00 92.18           O  \\nHETATM 1551  O   HOH A 389      25.326  12.873  16.251  1.00 49.27           O  \\nHETATM 1552  O   HOH A 390      20.219  10.405  10.956  1.00 63.05           O  \\nHETATM 1553  O   HOH A 392       9.452   5.453   8.249  1.00 47.81           O  \\nHETATM 1554  O   HOH A 394      10.139  12.252   0.254  1.00 29.37           O  \\nHETATM 1555  O   HOH A 395      28.004  37.416  13.466  1.00 52.29           O  \\nHETATM 1556  O   HOH A 396      24.283  36.829  12.307  1.00 64.74           O  \\nHETATM 1557  O   HOH A 397      13.748  34.031  -3.019  1.00 47.80           O  \\nHETATM 1558  O   HOH A 398      16.981  34.189   2.006  1.00 80.51           O  \\nHETATM 1559  O   HOH A 399      -4.426  24.223  -7.240  1.00 38.09           O  \\nHETATM 1560  O   HOH A 400      19.676  33.981   3.336  1.00 64.86           O  \\nHETATM 1561  O   HOH A 401      14.633  28.745  -0.317  1.00 66.25           O  \\nHETATM 1562  O   HOH A 402      -2.139  15.576  22.716  1.00 65.01           O  \\nHETATM 1563  O   HOH A 403       9.873  11.090  20.760  1.00 53.88           O  \\nHETATM 1564  O   HOH A 404      18.291  36.379  26.927  1.00 57.23           O  \\nHETATM 1565  O   HOH A 405       0.138   6.145  15.269  1.00 82.80           O  \\nHETATM 1566  O   HOH A 406      -5.515  38.166  18.689  1.00 48.28           O  \\nHETATM 1567  O   HOH A 407      -0.399   4.688   8.809  1.00 58.35           O  \\nHETATM 1568  O   HOH A 408       0.813  25.584  22.307  1.00 40.04           O  \\nCONECT  117 1325                                                                \\nCONECT  265 1325                                                                \\nCONECT 1325  117  265 1328 1333                                                 \\nCONECT 1325 1361 1362                                                           \\nCONECT 1326 1327 1328 1329 1330                                                 \\nCONECT 1327 1326                                                                \\nCONECT 1328 1325 1326                                                           \\nCONECT 1329 1326                                                                \\nCONECT 1330 1326 1331                                                           \\nCONECT 1331 1330 1332 1333 1334                                                 \\nCONECT 1332 1331                                                                \\nCONECT 1333 1325 1331                                                           \\nCONECT 1334 1331 1335                                                           \\nCONECT 1335 1334 1336 1337 1338                                                 \\nCONECT 1336 1335                                                                \\nCONECT 1337 1335                                                                \\nCONECT 1338 1335 1339                                                           \\nCONECT 1339 1338 1340                                                           \\nCONECT 1340 1339 1341 1342                                                      \\nCONECT 1341 1340 1346                                                           \\nCONECT 1342 1340 1343 1344                                                      \\nCONECT 1343 1342                                                                \\nCONECT 1344 1342 1345 1346                                                      \\nCONECT 1345 1344                                                                \\nCONECT 1346 1341 1344 1347                                                      \\nCONECT 1347 1346 1348 1357                                                      \\nCONECT 1348 1347 1349                                                           \\nCONECT 1349 1348 1350                                                           \\nCONECT 1350 1349 1351 1357                                                      \\nCONECT 1351 1350 1352 1353                                                      \\nCONECT 1352 1351                                                                \\nCONECT 1353 1351 1354                                                           \\nCONECT 1354 1353 1355 1356                                                      \\nCONECT 1355 1354                                                                \\nCONECT 1356 1354 1357                                                           \\nCONECT 1357 1347 1350 1356                                                      \\nCONECT 1361 1325                                                                \\nCONECT 1362 1325                                                                \\nMASTER      312    0    2    5    6    0    9    6 1567    1   38   13          \\nEND                                                                             \\n\",\"pdb\");\n\tviewergrid_16835771918631198[1][1].addModel(\"\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n   -3.3668   10.8519   -0.3788 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.6411   10.0411   -1.3780 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8036    8.9757   -0.7409 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.9662    9.5649    0.3292 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8533   10.1694    1.3858 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.5578   11.3266    0.6352 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6427   11.7527   -0.7698 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.0598    8.6821    0.9457 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.0953    8.6757    2.3938 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.7475    7.5578    3.0354 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.3157    7.0379    4.2591 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9245    5.9628    4.8605 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.0101    5.4058    4.1949 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.4261    5.9376    2.9775 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.8191    7.0126    2.3652 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.3828    7.6616    0.8538 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8466    8.0236    0.9100 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.1438    6.6658   -0.2672 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.3900    9.0555    0.4682 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.3087    9.4036    2.9194 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.3799    9.2270   -0.5372 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.6368   10.2707   -0.0818 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.4168   12.4190   -1.3348 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2851   12.2077    0.1321 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4937   11.2223    0.7148 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6561   11.2508    2.0487 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.2320   10.5501    3.1550 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6559   10.8204    4.4367 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5459   11.8421    4.6723 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.0052   12.5925    3.5567 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5634   12.2859    2.2980 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9103   12.8345    1.1310 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7360   14.0298    0.8624 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-4.26887\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n    4.7438    9.3941   -2.7765 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7809    8.3095   -3.0578 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.6564    8.2816   -2.0694 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1960    8.3109   -0.6904 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.9802    9.5804   -0.4847 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1704    9.4225   -1.4630 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.6464    9.2034   -3.2117 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.2256    8.1065    0.4180 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.0475    8.9487    0.3852 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.0962    8.5180    1.1554 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.4110    8.7525    0.7426 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.4990    8.3266    1.4658 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.2361    7.6459    2.6488 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.9202    7.4223    3.0441 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.1767    7.8421    2.3231 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.8059    7.5622    2.8622 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.0338    8.0908    4.2570 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.1263    6.0787    2.8175 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.8878    8.3151    1.7045 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.0271    9.7808   -0.1406 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8036    7.8661    1.7184 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3576   10.5980   -3.4401 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7706   10.8453   -5.3027 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5251   11.7468   -4.1448 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4221   11.6572   -3.3009 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4606   12.6614   -2.4084 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6902   13.1604   -1.3827 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0089   14.2967   -0.6736 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1541   14.9995   -0.9673 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9792   14.5149   -2.0174 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.6216   13.3846   -2.7017 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2305   12.8048   -3.7391 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.5426   13.1058   -4.3475 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-4.20857\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n   -3.4229   10.9599   -0.4989 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.5959   10.3486   -1.5595 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8204    9.1682   -1.0618 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.0981    9.5266    0.1806 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.0903    9.9266    1.2409 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.7226   11.2172    0.6636 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6633   11.9233   -0.7325 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.1342    8.5267    0.7125 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.4149    8.0293    2.0437 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.2093    6.7857    2.4314 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.4142    5.8643    3.2778 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.1717    4.6729    3.6318 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.4345    4.4175    3.1103 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.0400    5.3473    2.2697 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.4583    6.5425    1.9061 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.2571    7.6880    0.8704 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6199    8.0651    1.3970 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.3880    7.1247   -0.5336 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2144    9.0909    0.7219 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.0071    8.5190    2.6592 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.4036    9.5937   -0.1451 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.7130   10.3497   -0.4488 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.5841    9.1661    2.2332 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0134   10.5033    1.7420 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6097   11.0616    0.5328 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1114   12.3039    0.4255 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0561   13.3673   -0.4466 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6731   14.5760   -0.2146 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4012   14.7739    0.9353 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4862   13.6988    1.8602 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8517   12.5157    1.5935 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7727   11.4178    2.3493 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4731   11.0905    3.6083 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.74069\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n    6.8254    3.5394    8.6271 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1124    4.8243    8.7796 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6936    4.7429    8.3077 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6459    4.1462    6.9530 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2142    2.7519    6.9979 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.6997    2.9880    7.3668 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8355    3.6810    8.6168 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.3313    4.1563    6.2575 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.6130    5.4138    6.2896 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.5755    5.6093    5.3036 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2921    6.8612    4.7499 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.3178    7.0442    3.7984 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.3919    5.9158    3.4049 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.0960    4.6775    3.9679 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.8819    4.4841    4.9195 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2066    2.9239    5.6148 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.0359    2.3032    6.2046 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.7965    1.9950    4.5686 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.4595    3.1295    6.8254 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.8232    6.1070    6.9564 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9706    2.2588    6.9702 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.6641    2.7302    9.7925 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3919    4.0219   12.2555 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.8607    3.8901   12.0576 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4523    3.2881   10.9510 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.7859    3.2638   11.1162 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.8984    2.7888   10.4598 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.1778    2.8873   10.9589 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.4041    3.4922   12.1732 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.2817    3.9997   12.8812 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.0258    3.8747   12.3517 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8539    4.2384   12.8784 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.5744    5.0238   14.0980 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.68007\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n    4.1018    9.7254   -3.5001 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.0930    8.7277   -3.9120 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.1826    8.3464   -2.7859 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9810    7.9874   -1.5912 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8013    9.1748   -1.1598 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.7765    9.3795   -2.3455 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9005    9.7209   -4.1345 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.2436    7.4097   -0.4360 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9508    7.4167    0.8282 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.1612    7.2758    2.0296 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.4996    7.9090    3.2291 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.7385    7.7900    4.3670 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.6011    6.9961    4.2786 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.2806    6.3729    3.0757 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.0356    6.4899    1.9288 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.6131    5.6773    0.4509 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.4473    4.1915    0.6554 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.6656    6.2616   -0.1228 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.8642    6.0284   -0.7276 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.9298    7.5135    0.8670 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.5020    5.9443   -1.6774 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6041   11.0517   -3.6799 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5176   11.5731   -6.0080 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.6001   12.2738   -4.6981 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.7249   12.0599   -3.6374 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0257   12.9022   -2.6341 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5468   13.2344   -1.3872 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0829   14.2338   -0.6065 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1609   14.9610   -1.0541 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.6879   14.6463   -2.3354 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1203   13.6493   -3.0820 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4355   13.2413   -4.3137 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.5803   13.6222   -5.1663 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.61611\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n   -0.6465    7.6479    1.0206 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.9213    6.8194    2.2124 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.3723    6.8253    2.5826 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.8705    8.2185    2.6482 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.7202    8.8698    1.2984 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.1851    8.9175    1.0985 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.3367    7.9163    0.9808 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.2454    8.4188    3.1788 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.4494    9.6232    3.9574 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.8095   10.0793    4.1276 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1429   11.4347    4.2042 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4385   11.8693    4.3475 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4223   10.8902    4.4204 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0710    9.5453    4.3431 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7751    9.1006    4.1953 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3749    7.4101    4.1245 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9090    6.6569    5.3179 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9162    6.7952    2.8460 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.6292    7.2793    4.0103 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6920   10.1110    4.3544 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.3592    6.4004    3.5687 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.7784    6.8759   -0.1733 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.2653    7.1445   -0.0415 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.4468    7.5842   -1.2036 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.0638    7.4508   -1.2845 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.3552    7.8730   -2.4897 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.5297    7.9573   -3.2024 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.6024    8.4101   -4.5006 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.4652    8.8175   -5.1581 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.7678    8.7498   -4.4555 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.7963    8.2867   -3.1677 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.8482    8.1035   -2.3658 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2592    8.4950   -2.5615 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.54520\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n    3.6826    7.0817    0.3984 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4242    6.7685   -0.8404 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2179    7.9380   -1.3350 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3574    9.1420   -1.3963 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8441    9.4633   -0.0171 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9454    8.2470    0.3171 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9056    6.4347    0.5326 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9363   10.3527   -2.0374 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4211   11.6214   -1.5648 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2189   12.7977   -1.8232 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2740   13.8772   -0.9366 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.0471   14.9882   -1.1734 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7814   14.9994   -2.3534 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7132   13.9155   -3.2244 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9440   12.7958   -2.9932 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8689   11.4713   -4.1172 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4779   11.9310   -5.5002 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2061   10.7544   -4.1743 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.7294   10.2977   -3.4835 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.5623   11.6796   -1.0871 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9310    9.3650   -3.8438 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4821    6.8078    1.5493 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1103    3.8375    1.9184 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0351    4.5525    2.6577 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7511    5.9072    2.5132 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7799    6.2537    3.3753 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.1036    7.3868    3.7667 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.1598    7.4029    4.7690 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.8384    6.2451    5.4380 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.5116    5.0521    5.0611 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.4483    5.0818    4.0635 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2147    4.0898    3.6034 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1681    2.6455    3.9107 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.47112\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n   -5.7173    4.7949    8.7097 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0079    4.3599   10.0914 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3343    4.8599   10.5742 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4437    6.3166   10.3299 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3416    6.5870    8.8516 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.8950    6.1504    8.5120 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.7164    4.7564    8.5174 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.6232    7.0112   10.9116 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.3117    7.9287   10.0269 N   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.6662    8.2920   10.3735 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -11.6459    8.5453    9.4091 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -12.9390    8.8729    9.7386 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -13.2390    8.9496   11.0936 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -12.2499    8.6947   12.0396 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.9519    8.3616   11.7183 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.7394    8.0719   12.9303 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.6159    9.2311   13.8884 O   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.0588    6.8026   13.6999 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.2243    7.7560   12.1045 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.8744    8.2899    9.2223 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6016    7.1984   12.6892 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2243    3.8513    7.7654 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.5508    2.6643    6.8726 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.3925    3.5419    6.0151 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6193    4.0732    6.4018 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1543    4.7542    5.3740 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3108    5.4409    5.0818 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.5763    5.9972    3.8506 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6610    5.8914    2.8296 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4518    5.1932    3.0919 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.2309    4.6494    4.3284 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.1929    3.9240    4.7519 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9112    3.6333    4.0772 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.41646\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n   -6.0669   11.4001    2.1105 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.7487   11.8823    0.7508 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0078   10.8589   -0.0529 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8573   10.3434    0.7243 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.3561    9.6889    1.9859 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.9778   10.8688    2.7735 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2471   12.1804    2.7422 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9061    9.4510    0.0094 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.3204    9.0185   -1.3096 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.2934    8.5259   -2.1980 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.3406    8.7206   -3.5814 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.3545    8.2682   -4.4247 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.2915    7.5918   -3.8380 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.2621    7.4068   -2.4585 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.2443    7.8606   -1.6050 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.1818    7.5902    0.1114 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.0263    6.1264    0.4433 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.0396    8.3793    0.7262 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.6468    8.2558    0.8102 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.2636    9.0587   -1.5891 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.5033    8.4995    1.7902 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3256   10.7258    2.1263 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.9632   12.8010    3.6641 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.5453   11.4915    4.2335 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8165   10.5307    3.5389 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6716    9.4383    4.3083 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.1329    8.1751    4.2157 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2113    7.2376    5.2211 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8473    7.5390    6.4027 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4138    8.8347    6.5397 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3190    9.7344    5.5126 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.8218   10.9687    5.4301 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.4795   11.7874    6.4691 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.34270\\n\\n$$$$\\n\",\"sdf\");\n\tviewergrid_16835771918631198[1][1].setStyle({\"model\": 0},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewergrid_16835771918631198[1][1].addSurface(1,{\"opacity\": 0.5, \"color\": \"white\"},{\"model\": 0});\n\tviewergrid_16835771918631198[1][1].setStyle({\"model\": 1},{\"stick\": {}});\n\tviewergrid_16835771918631198[1][1].zoomTo();\n\tviewergrid_16835771918631198[0][0].render();\n\tviewergrid_16835771918631198[0][1].render();\n\tviewergrid_16835771918631198[1][0].render();\n\tviewergrid_16835771918631198[1][1].render();\nviewergrid_16835771918631198[1][1].render();\nviewergrid_16835771918631198[1][0].render();\nviewergrid_16835771918631198[0][1].render();\nviewergrid_16835771918631198[0][0].render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_16835771918631198\"  style=\"position: relative; width: 1000px; height: 1000px\">\n",
       "        <p id=\"3dmolwarning_16835771918631198\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n",
       "}\n",
       "\n",
       "var viewer_16835771918631198 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_16835771918631198\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "var viewergrid_16835771918631198 = null;\n",
       "viewergrid_16835771918631198 = $3Dmol.createViewerGrid(document.getElementById(\"3dmolviewer_16835771918631198\"),{rows: 2, cols: 2, control_all: false},{backgroundColor:\"white\"});\n",
       "viewer_16835771918631198 = viewergrid_16835771918631198[0][0];\n",
       "viewergrid_16835771918631198[0][0].zoomTo();viewergrid_16835771918631198[0][1].zoomTo();viewergrid_16835771918631198[1][0].zoomTo();viewergrid_16835771918631198[1][1].zoomTo();\tviewergrid_16835771918631198[0][0].setViewStyle({\"style\": \"outline\", \"color\": \"black\", \"width\": 0.1});\n",
       "\tviewergrid_16835771918631198[0][1].setViewStyle({\"style\": \"outline\", \"color\": \"black\", \"width\": 0.1});\n",
       "\tviewergrid_16835771918631198[1][0].setViewStyle({\"style\": \"outline\", \"color\": \"black\", \"width\": 0.1});\n",
       "\tviewergrid_16835771918631198[1][1].setViewStyle({\"style\": \"outline\", \"color\": \"black\", \"width\": 0.1});\n",
       "\tviewergrid_16835771918631198[0][0].addModel(\"HEADER    ONCOGENE PROTEIN                        30-APR-90   5P21              \\nTITLE     REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION OF H-RAS   \\nTITLE    2 P21 AT 1.35 ANGSTROMS RESOLUTION: IMPLICATIONS FOR THE MECHANISM OF  \\nTITLE    3 GTP HYDROLYSIS                                                       \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: C-H-RAS P21 PROTEIN;                                       \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    ONCOGENE PROTEIN                                                      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    E.F.PAI,A.WITTINGHOFER,W.KABSCH                                       \\nREVDAT   6   29-NOV-17 5P21    1       HELIX                                    \\nREVDAT   5   24-FEB-09 5P21    1       VERSN                                    \\nREVDAT   4   01-APR-03 5P21    1       JRNL                                     \\nREVDAT   3   20-JUL-95 5P21    1       HET                                      \\nREVDAT   2   30-APR-94 5P21    1       FORMUL                                   \\nREVDAT   1   15-JAN-92 5P21    0                                                \\nJRNL        AUTH   E.F.PAI,U.KRENGEL,G.A.PETSKO,R.S.GOODY,W.KABSCH,             \\nJRNL        AUTH 2 A.WITTINGHOFER                                               \\nJRNL        TITL   REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION   \\nJRNL        TITL 2 OF H-RAS P21 AT 1.35 A RESOLUTION: IMPLICATIONS FOR THE      \\nJRNL        TITL 3 MECHANISM OF GTP HYDROLYSIS.                                 \\nJRNL        REF    EMBO J.                       V.   9  2351 1990              \\nJRNL        REFN                   ISSN 0261-4189                               \\nJRNL        PMID   2196171                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,S.C.ALMO,G.RAPP,K.WILSON,K.PETRATOS,A.LENTFER, \\nREMARK   1  AUTH 2 A.WITTINGHOFER,W.KABSCH,E.F.PAI,G.A.PETSKO,R.S.GOODY         \\nREMARK   1  TITL   TIME-RESOLVED X-RAY CRYSTALLOGRAPHIC STUDY OF THE            \\nREMARK   1  TITL 2 CONFORMATIONAL CHANGE IN HA-RAS P21 PROTEIN ON GTP           \\nREMARK   1  TITL 3 HYDROLYSIS                                                   \\nREMARK   1  REF    NATURE                        V. 345   309 1990              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   A.SCHERER,J.JOHN,R.LINKE,R.S.GOODY,A.WITTINGHOFER,E.F.PAI,   \\nREMARK   1  AUTH 2 K.C.HOLMES                                                   \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY ANALYSIS OF THE HUMAN  \\nREMARK   1  TITL 2 C-H-RAS-ONCOGENE PRODUCT P21 COMPLEXED WITH GTP ANALOGUES    \\nREMARK   1  REF    J.MOL.BIOL.                   V. 206   257 1989              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   E.F.PAI,W.KABSCH,U.KRENGEL,K.C.HOLMES,J.JOHN,A.WITTINGHOFER  \\nREMARK   1  TITL   STRUCTURE OF THE GUANINE-NUCLEOTIDE-BINDING DOMAIN OF THE    \\nREMARK   1  TITL 2 HA-RAS ONCOGENE PRODUCT P21 IN THE TRIPHOSPHATE CONFORMATION \\nREMARK   1  REF    NATURE                        V. 341   209 1989              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,G.RAPP,J.JOHN,A.WITTINGHOFER,E.F.PAI,R.S.GOODY \\nREMARK   1  TITL   BIOCHEMICAL AND CRYSTALLOGRAPHIC CHARACTERIZATION OF A       \\nREMARK   1  TITL 2 COMPLEX OF C-HA-RAS P21 AND CAGED GTP WITH FLASH PHOTOLYSIS  \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  86  7687 1989              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 6.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 26806                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.196                           \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1323                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 33                                      \\nREMARK   3   SOLVENT ATOMS            : 211                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.014                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 2.700                           \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  RESIDUES 61 - 65 (GLN - GLU - GLU - TYR - SER) ADOPT                \\nREMARK   3  SEVERAL CONFORMATIONS IN THE CRYSTAL.  THE COORDINATES              \\nREMARK   3  GIVEN APPROXIMATE ONE OF THESE.                                     \\nREMARK   4                                                                      \\nREMARK   4 5P21 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179747.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 38.93                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.01                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 32 2 1                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+2/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+1/3                                           \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z+1/3                                           \\nREMARK 290       6555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000      108.13333            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       54.06667            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       54.06667            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000      108.13333            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025  0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    ARG A  68   NE  -  CZ  -  NH1 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A  97   NE  -  CZ  -  NH1 ANGL. DEV. =  -3.4 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH1 ANGL. DEV. =   4.1 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500    ARG A 128   NE  -  CZ  -  NH2 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A 135   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500    TYR A 157   CB  -  CG  -  CD2 ANGL. DEV. =  -4.8 DEGREES          \\nREMARK 500    ARG A 164   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ASP A  33       75.86   -157.25                                   \\nREMARK 500    ILE A  36      -64.23   -100.39                                   \\nREMARK 500    GLU A  37      123.75   -171.30                                   \\nREMARK 500    SER A  65     -133.70     53.07                                   \\nREMARK 500    LYS A 117       33.80     71.60                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              MG A 168  MG                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 GNP A 167   O2G                                                    \\nREMARK 620 2 GNP A 167   O2B  95.0                                              \\nREMARK 620 3 HOH A 173   O    87.6  91.7                                        \\nREMARK 620 4 SER A  17   OG  172.9  91.0  88.6                                  \\nREMARK 620 5 THR A  35   OG1  91.3 173.4  90.5  82.8                            \\nREMARK 620 6 HOH A 172   O    97.3  88.0 175.0  86.5  89.3                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE MG A 168                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE GNP A 167                 \\nDBREF  5P21 A    1   166  UNP    P01112   RASH_HUMAN       1    166             \\nSEQRES   1 A  166  MET THR GLU TYR LYS LEU VAL VAL VAL GLY ALA GLY GLY          \\nSEQRES   2 A  166  VAL GLY LYS SER ALA LEU THR ILE GLN LEU ILE GLN ASN          \\nSEQRES   3 A  166  HIS PHE VAL ASP GLU TYR ASP PRO THR ILE GLU ASP SER          \\nSEQRES   4 A  166  TYR ARG LYS GLN VAL VAL ILE ASP GLY GLU THR CYS LEU          \\nSEQRES   5 A  166  LEU ASP ILE LEU ASP THR ALA GLY GLN GLU GLU TYR SER          \\nSEQRES   6 A  166  ALA MET ARG ASP GLN TYR MET ARG THR GLY GLU GLY PHE          \\nSEQRES   7 A  166  LEU CYS VAL PHE ALA ILE ASN ASN THR LYS SER PHE GLU          \\nSEQRES   8 A  166  ASP ILE HIS GLN TYR ARG GLU GLN ILE LYS ARG VAL LYS          \\nSEQRES   9 A  166  ASP SER ASP ASP VAL PRO MET VAL LEU VAL GLY ASN LYS          \\nSEQRES  10 A  166  CYS ASP LEU ALA ALA ARG THR VAL GLU SER ARG GLN ALA          \\nSEQRES  11 A  166  GLN ASP LEU ALA ARG SER TYR GLY ILE PRO TYR ILE GLU          \\nSEQRES  12 A  166  THR SER ALA LYS THR ARG GLN GLY VAL GLU ASP ALA PHE          \\nSEQRES  13 A  166  TYR THR LEU VAL ARG GLU ILE ARG GLN HIS                      \\nHET     MG  A 168       1                                                       \\nHET    GNP  A 167      32                                                       \\nHETNAM      MG MAGNESIUM ION                                                    \\nHETNAM     GNP PHOSPHOAMINOPHOSPHONIC ACID-GUANYLATE ESTER                      \\nFORMUL   2   MG    MG 2+                                                        \\nFORMUL   3  GNP    C10 H17 N6 O13 P3                                            \\nFORMUL   4  HOH   *211(H2 O)                                                    \\nHELIX    1  HA LYS A   16  GLN A   25  1                                  10    \\nHELIX    2  HB SER A   65  THR A   74  1RES 65-67 IN 3/10 CONF            10    \\nHELIX    3  HC THR A   87  VAL A  103  1BREAK AT ASP 92                   17    \\nHELIX    4  HD SER A  127  TYR A  137  1                                  11    \\nHELIX    5  HE VAL A  152  ARG A  164  1                                  13    \\nSHEET    1  S1 6 GLU A  37  ILE A  46  0                                        \\nSHEET    2  S1 6 GLU A  49  THR A  58 -1  O  LEU A  53   N  LYS A  42           \\nSHEET    3  S1 6 THR A   2  VAL A   9  1  N  LEU A   6   O  ASP A  54           \\nSHEET    4  S1 6 GLY A  77  ALA A  83  1  N  LEU A  79   O  VAL A   7           \\nSHEET    5  S1 6 MET A 111  ASN A 116  1  N  VAL A 114   O  CYS A  80           \\nSHEET    6  S1 6 TYR A 141  GLU A 143  1  N  ILE A 142   O  LEU A 113           \\nLINK        MG    MG A 168                 O2G GNP A 167     1555   1555  2.20  \\nLINK        MG    MG A 168                 O2B GNP A 167     1555   1555  2.22  \\nLINK        MG    MG A 168                 O   HOH A 173     1555   1555  2.23  \\nLINK        MG    MG A 168                 OG  SER A  17     1555   1555  2.26  \\nLINK        MG    MG A 168                 OG1 THR A  35     1555   1555  2.26  \\nLINK        MG    MG A 168                 O   HOH A 172     1555   1555  2.30  \\nSITE     1 AC1  5 SER A  17  THR A  35  GNP A 167  HOH A 172                    \\nSITE     2 AC1  5 HOH A 173                                                     \\nSITE     1 AC2 28 GLY A  12  GLY A  13  VAL A  14  GLY A  15                    \\nSITE     2 AC2 28 LYS A  16  SER A  17  ALA A  18  PHE A  28                    \\nSITE     3 AC2 28 VAL A  29  ASP A  30  TYR A  32  PRO A  34                    \\nSITE     4 AC2 28 THR A  35  GLY A  60  ASN A 116  LYS A 117                    \\nSITE     5 AC2 28 ASP A 119  SER A 145  ALA A 146  LYS A 147                    \\nSITE     6 AC2 28  MG A 168  HOH A 170  HOH A 172  HOH A 173                    \\nSITE     7 AC2 28 HOH A 175  HOH A 187  HOH A 281  HOH A 366                    \\nCRYST1   40.300   40.300  162.200  90.00  90.00 120.00 P 32 2 1      6          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.024814  0.014326  0.000000        0.00000                         \\nSCALE2      0.000000  0.028653  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.006165        0.00000                         \\nATOM      1  N   MET A   1      -7.186  32.862  -6.632  1.00 13.92           N  \\nATOM      2  CA  MET A   1      -5.921  32.127  -6.684  1.00 17.10           C  \\nATOM      3  C   MET A   1      -5.845  31.045  -5.621  1.00 16.74           C  \\nATOM      4  O   MET A   1      -6.600  31.083  -4.636  1.00 16.60           O  \\nATOM      5  CB  MET A   1      -4.673  33.058  -6.502  1.00 17.00           C  \\nATOM      6  CG  MET A   1      -4.633  33.804  -5.188  1.00 18.08           C  \\nATOM      7  SD  MET A   1      -3.115  34.766  -4.908  1.00 19.73           S  \\nATOM      8  CE  MET A   1      -3.019  35.851  -6.310  1.00 20.41           C  \\nATOM      9  N   THR A   2      -4.910  30.095  -5.820  1.00 14.70           N  \\nATOM     10  CA  THR A   2      -4.661  29.030  -4.859  1.00 16.92           C  \\nATOM     11  C   THR A   2      -4.230  29.655  -3.547  1.00 14.78           C  \\nATOM     12  O   THR A   2      -3.338  30.501  -3.537  1.00 13.93           O  \\nATOM     13  CB  THR A   2      -3.557  28.107  -5.401  1.00 18.03           C  \\nATOM     14  OG1 THR A   2      -4.059  27.565  -6.629  1.00 20.42           O  \\nATOM     15  CG2 THR A   2      -3.193  26.983  -4.443  1.00 18.51           C  \\nATOM     16  N   GLU A   3      -4.911  29.290  -2.472  1.00 14.80           N  \\nATOM     17  CA  GLU A   3      -4.504  29.730  -1.135  1.00 17.71           C  \\nATOM     18  C   GLU A   3      -4.012  28.553  -0.278  1.00 17.33           C  \\nATOM     19  O   GLU A   3      -4.520  27.427  -0.395  1.00 17.54           O  \\nATOM     20  CB  GLU A   3      -5.655  30.402  -0.432  1.00 20.67           C  \\nATOM     21  CG  GLU A   3      -6.213  31.631  -1.186  1.00 25.56           C  \\nATOM     22  CD  GLU A   3      -7.128  32.563  -0.369  1.00 28.46           C  \\nATOM     23  OE1 GLU A   3      -7.578  32.216   0.728  1.00 27.72           O  \\nATOM     24  OE2 GLU A   3      -7.375  33.680  -0.827  1.00 32.49           O  \\nATOM     25  N   TYR A   4      -2.962  28.744   0.542  1.00 14.50           N  \\nATOM     26  CA  TYR A   4      -2.425  27.654   1.358  1.00 12.29           C  \\nATOM     27  C   TYR A   4      -2.534  28.103   2.807  1.00 12.52           C  \\nATOM     28  O   TYR A   4      -2.016  29.143   3.200  1.00 12.11           O  \\nATOM     29  CB  TYR A   4      -0.975  27.385   1.026  1.00 11.46           C  \\nATOM     30  CG  TYR A   4      -0.726  26.846  -0.387  1.00 12.38           C  \\nATOM     31  CD1 TYR A   4      -0.297  27.698  -1.395  1.00 13.00           C  \\nATOM     32  CD2 TYR A   4      -0.933  25.501  -0.649  1.00 14.09           C  \\nATOM     33  CE1 TYR A   4      -0.070  27.208  -2.674  1.00 13.89           C  \\nATOM     34  CE2 TYR A   4      -0.711  24.993  -1.917  1.00 14.65           C  \\nATOM     35  CZ  TYR A   4      -0.279  25.859  -2.917  1.00 14.84           C  \\nATOM     36  OH  TYR A   4      -0.084  25.357  -4.191  1.00 15.07           O  \\nATOM     37  N   LYS A   5      -3.274  27.305   3.568  1.00 13.07           N  \\nATOM     38  CA  LYS A   5      -3.554  27.569   4.970  1.00 12.96           C  \\nATOM     39  C   LYS A   5      -2.556  26.818   5.852  1.00 12.74           C  \\nATOM     40  O   LYS A   5      -2.645  25.611   6.092  1.00 11.58           O  \\nATOM     41  CB  LYS A   5      -4.972  27.131   5.281  1.00 17.65           C  \\nATOM     42  CG  LYS A   5      -6.053  27.833   4.446  1.00 23.49           C  \\nATOM     43  CD  LYS A   5      -7.186  28.310   5.335  1.00 29.10           C  \\nATOM     44  CE  LYS A   5      -6.655  29.309   6.403  1.00 33.68           C  \\nATOM     45  NZ  LYS A   5      -6.597  28.739   7.755  1.00 36.52           N  \\nATOM     46  N   LEU A   6      -1.543  27.574   6.297  1.00 13.08           N  \\nATOM     47  CA  LEU A   6      -0.418  27.011   7.064  1.00 10.34           C  \\nATOM     48  C   LEU A   6      -0.578  27.382   8.535  1.00  9.68           C  \\nATOM     49  O   LEU A   6      -1.034  28.483   8.863  1.00 11.31           O  \\nATOM     50  CB  LEU A   6       0.910  27.571   6.536  1.00 11.34           C  \\nATOM     51  CG  LEU A   6       1.137  27.565   5.017  1.00 11.62           C  \\nATOM     52  CD1 LEU A   6       2.515  28.119   4.720  1.00 13.60           C  \\nATOM     53  CD2 LEU A   6       0.973  26.171   4.475  1.00 12.19           C  \\nATOM     54  N   VAL A   7      -0.196  26.486   9.432  1.00  9.93           N  \\nATOM     55  CA  VAL A   7      -0.333  26.787  10.852  1.00 10.06           C  \\nATOM     56  C   VAL A   7       1.033  26.513  11.495  1.00 10.37           C  \\nATOM     57  O   VAL A   7       1.597  25.441  11.252  1.00  9.51           O  \\nATOM     58  CB  VAL A   7      -1.474  25.884  11.446  1.00 10.97           C  \\nATOM     59  CG1 VAL A   7      -1.610  26.163  12.943  1.00 11.39           C  \\nATOM     60  CG2 VAL A   7      -2.809  26.161  10.777  1.00 10.14           C  \\nATOM     61  N   VAL A   8       1.576  27.455  12.295  1.00  8.35           N  \\nATOM     62  CA  VAL A   8       2.878  27.262  12.944  1.00  6.92           C  \\nATOM     63  C   VAL A   8       2.609  26.896  14.405  1.00  8.84           C  \\nATOM     64  O   VAL A   8       1.930  27.663  15.092  1.00  9.21           O  \\nATOM     65  CB  VAL A   8       3.684  28.565  12.799  1.00  8.08           C  \\nATOM     66  CG1 VAL A   8       5.111  28.324  13.261  1.00  9.52           C  \\nATOM     67  CG2 VAL A   8       3.729  29.011  11.358  1.00  8.43           C  \\nATOM     68  N   VAL A   9       3.024  25.701  14.881  1.00  8.73           N  \\nATOM     69  CA  VAL A   9       2.779  25.234  16.245  1.00  8.41           C  \\nATOM     70  C   VAL A   9       4.075  24.806  16.919  1.00  8.21           C  \\nATOM     71  O   VAL A   9       5.093  24.551  16.287  1.00 10.97           O  \\nATOM     72  CB  VAL A   9       1.824  24.028  16.286  1.00  9.74           C  \\nATOM     73  CG1 VAL A   9       0.545  24.491  15.680  1.00 11.76           C  \\nATOM     74  CG2 VAL A   9       2.349  22.810  15.535  1.00 10.31           C  \\nATOM     75  N   GLY A  10       4.048  24.707  18.238  1.00  7.34           N  \\nATOM     76  CA  GLY A  10       5.231  24.336  19.009  1.00  7.98           C  \\nATOM     77  C   GLY A  10       5.192  25.053  20.355  1.00  8.31           C  \\nATOM     78  O   GLY A  10       4.417  25.994  20.595  1.00  9.89           O  \\nATOM     79  N   ALA A  11       6.099  24.638  21.226  1.00  9.31           N  \\nATOM     80  CA  ALA A  11       6.121  25.135  22.597  1.00  9.50           C  \\nATOM     81  C   ALA A  11       6.440  26.625  22.711  1.00  9.66           C  \\nATOM     82  O   ALA A  11       6.998  27.250  21.806  1.00  9.46           O  \\nATOM     83  CB  ALA A  11       7.145  24.306  23.378  1.00  9.85           C  \\nATOM     84  N   GLY A  12       6.111  27.227  23.847  1.00 10.36           N  \\nATOM     85  CA  GLY A  12       6.406  28.632  24.054  1.00  9.30           C  \\nATOM     86  C   GLY A  12       7.892  28.950  23.964  1.00 10.23           C  \\nATOM     87  O   GLY A  12       8.785  28.249  24.466  1.00 10.57           O  \\nATOM     88  N   GLY A  13       8.159  30.064  23.266  1.00 11.49           N  \\nATOM     89  CA  GLY A  13       9.511  30.602  23.139  1.00 11.29           C  \\nATOM     90  C   GLY A  13      10.436  29.950  22.108  1.00 12.22           C  \\nATOM     91  O   GLY A  13      11.624  30.321  22.060  1.00 13.62           O  \\nATOM     92  N   VAL A  14       9.961  28.991  21.286  1.00  9.23           N  \\nATOM     93  CA  VAL A  14      10.848  28.327  20.317  1.00 10.38           C  \\nATOM     94  C   VAL A  14      11.169  29.168  19.076  1.00 10.47           C  \\nATOM     95  O   VAL A  14      12.108  28.841  18.353  1.00 12.07           O  \\nATOM     96  CB  VAL A  14      10.246  26.970  19.856  1.00  9.98           C  \\nATOM     97  CG1 VAL A  14      10.187  26.074  21.120  1.00  9.93           C  \\nATOM     98  CG2 VAL A  14       8.866  27.127  19.200  1.00  9.52           C  \\nATOM     99  N   GLY A  15      10.423  30.264  18.834  1.00  9.61           N  \\nATOM    100  CA  GLY A  15      10.666  31.160  17.706  1.00  9.70           C  \\nATOM    101  C   GLY A  15       9.615  31.061  16.626  1.00  8.82           C  \\nATOM    102  O   GLY A  15       9.915  31.403  15.491  1.00  8.99           O  \\nATOM    103  N   LYS A  16       8.395  30.591  16.916  1.00  8.18           N  \\nATOM    104  CA  LYS A  16       7.331  30.521  15.913  1.00  8.44           C  \\nATOM    105  C   LYS A  16       7.060  31.899  15.309  1.00  9.01           C  \\nATOM    106  O   LYS A  16       7.020  32.044  14.086  1.00  8.25           O  \\nATOM    107  CB  LYS A  16       6.014  30.000  16.529  1.00  9.27           C  \\nATOM    108  CG  LYS A  16       6.132  28.585  17.086  1.00  9.11           C  \\nATOM    109  CD  LYS A  16       4.820  28.077  17.726  1.00  9.31           C  \\nATOM    110  CE  LYS A  16       4.351  28.934  18.888  1.00 10.05           C  \\nATOM    111  NZ  LYS A  16       5.221  28.829  20.051  1.00  9.64           N  \\nATOM    112  N   SER A  17       6.913  32.942  16.135  1.00  7.74           N  \\nATOM    113  CA  SER A  17       6.661  34.295  15.649  1.00  8.78           C  \\nATOM    114  C   SER A  17       7.854  34.859  14.919  1.00  9.73           C  \\nATOM    115  O   SER A  17       7.653  35.446  13.868  1.00  9.70           O  \\nATOM    116  CB  SER A  17       6.360  35.264  16.769  1.00  8.22           C  \\nATOM    117  OG  SER A  17       5.083  34.853  17.204  1.00  8.79           O  \\nATOM    118  N   ALA A  18       9.072  34.687  15.444  1.00  8.95           N  \\nATOM    119  CA  ALA A  18      10.253  35.199  14.769  1.00  8.95           C  \\nATOM    120  C   ALA A  18      10.438  34.576  13.374  1.00  8.54           C  \\nATOM    121  O   ALA A  18      10.832  35.254  12.430  1.00  8.75           O  \\nATOM    122  CB  ALA A  18      11.521  34.931  15.640  1.00 10.11           C  \\nATOM    123  N   LEU A  19      10.136  33.284  13.176  1.00  8.31           N  \\nATOM    124  CA  LEU A  19      10.225  32.667  11.837  1.00  9.30           C  \\nATOM    125  C   LEU A  19       9.204  33.281  10.884  1.00 10.98           C  \\nATOM    126  O   LEU A  19       9.534  33.652   9.751  1.00 10.47           O  \\nATOM    127  CB  LEU A  19       9.938  31.165  11.885  1.00  9.41           C  \\nATOM    128  CG  LEU A  19      11.035  30.275  12.487  1.00 11.80           C  \\nATOM    129  CD1 LEU A  19      10.491  28.866  12.723  1.00 10.43           C  \\nATOM    130  CD2 LEU A  19      12.256  30.288  11.554  1.00 13.42           C  \\nATOM    131  N   THR A  20       7.950  33.419  11.335  1.00 10.72           N  \\nATOM    132  CA  THR A  20       6.888  34.015  10.529  1.00 11.87           C  \\nATOM    133  C   THR A  20       7.173  35.492  10.161  1.00 12.65           C  \\nATOM    134  O   THR A  20       7.012  35.917   9.009  1.00 12.60           O  \\nATOM    135  CB  THR A  20       5.547  33.888  11.312  1.00 13.43           C  \\nATOM    136  OG1 THR A  20       5.299  32.509  11.544  1.00 12.67           O  \\nATOM    137  CG2 THR A  20       4.346  34.410  10.518  1.00 14.64           C  \\nATOM    138  N   ILE A  21       7.615  36.326  11.098  1.00 10.48           N  \\nATOM    139  CA  ILE A  21       7.945  37.704  10.795  1.00 12.96           C  \\nATOM    140  C   ILE A  21       9.197  37.831   9.927  1.00 14.26           C  \\nATOM    141  O   ILE A  21       9.237  38.691   9.042  1.00 15.83           O  \\nATOM    142  CB  ILE A  21       8.090  38.451  12.121  1.00 12.57           C  \\nATOM    143  CG1 ILE A  21       6.691  38.551  12.737  1.00 13.24           C  \\nATOM    144  CG2 ILE A  21       8.631  39.852  11.910  1.00 13.40           C  \\nATOM    145  CD1 ILE A  21       6.636  39.185  14.129  1.00 16.46           C  \\nATOM    146  N   GLN A  22      10.206  36.979  10.092  1.00 13.58           N  \\nATOM    147  CA  GLN A  22      11.330  36.983   9.179  1.00 14.91           C  \\nATOM    148  C   GLN A  22      10.817  36.729   7.736  1.00 17.29           C  \\nATOM    149  O   GLN A  22      11.120  37.485   6.792  1.00 15.91           O  \\nATOM    150  CB  GLN A  22      12.270  35.930   9.736  1.00 15.90           C  \\nATOM    151  CG  GLN A  22      13.686  35.878   9.230  1.00 19.36           C  \\nATOM    152  CD  GLN A  22      14.372  37.217   9.091  1.00 18.08           C  \\nATOM    153  OE1 GLN A  22      14.160  37.984   8.143  1.00 19.23           O  \\nATOM    154  NE2 GLN A  22      15.224  37.492  10.050  1.00 17.40           N  \\nATOM    155  N   LEU A  23       9.944  35.718   7.523  1.00 15.95           N  \\nATOM    156  CA  LEU A  23       9.381  35.475   6.194  1.00 18.85           C  \\nATOM    157  C   LEU A  23       8.539  36.622   5.636  1.00 18.21           C  \\nATOM    158  O   LEU A  23       8.695  36.958   4.469  1.00 19.03           O  \\nATOM    159  CB  LEU A  23       8.498  34.235   6.196  1.00 18.10           C  \\nATOM    160  CG  LEU A  23       7.916  33.797   4.863  1.00 18.64           C  \\nATOM    161  CD1 LEU A  23       8.921  32.994   4.030  1.00 18.03           C  \\nATOM    162  CD2 LEU A  23       6.680  32.991   5.183  1.00 21.34           C  \\nATOM    163  N   ILE A  24       7.634  37.197   6.428  1.00 19.25           N  \\nATOM    164  CA  ILE A  24       6.682  38.198   5.953  1.00 20.23           C  \\nATOM    165  C   ILE A  24       7.316  39.590   5.922  1.00 21.89           C  \\nATOM    166  O   ILE A  24       7.125  40.318   4.943  1.00 22.93           O  \\nATOM    167  CB  ILE A  24       5.399  38.222   6.881  1.00 21.20           C  \\nATOM    168  CG1 ILE A  24       4.664  36.907   6.821  1.00 21.27           C  \\nATOM    169  CG2 ILE A  24       4.423  39.307   6.446  1.00 20.00           C  \\nATOM    170  CD1 ILE A  24       4.128  36.442   5.475  1.00 23.98           C  \\nATOM    171  N   GLN A  25       8.074  39.998   6.953  1.00 21.07           N  \\nATOM    172  CA  GLN A  25       8.520  41.380   7.066  1.00 22.09           C  \\nATOM    173  C   GLN A  25      10.003  41.559   6.962  1.00 20.75           C  \\nATOM    174  O   GLN A  25      10.495  42.677   7.054  1.00 20.26           O  \\nATOM    175  CB  GLN A  25       8.039  41.968   8.398  1.00 23.96           C  \\nATOM    176  CG  GLN A  25       6.548  42.276   8.381  1.00 28.69           C  \\nATOM    177  CD  GLN A  25       5.931  42.574   9.744  1.00 30.92           C  \\nATOM    178  OE1 GLN A  25       6.537  43.143  10.657  1.00 31.89           O  \\nATOM    179  NE2 GLN A  25       4.664  42.169   9.867  1.00 32.35           N  \\nATOM    180  N   ASN A  26      10.764  40.482   6.795  1.00 22.13           N  \\nATOM    181  CA  ASN A  26      12.198  40.642   6.628  1.00 24.67           C  \\nATOM    182  C   ASN A  26      12.936  41.362   7.734  1.00 23.62           C  \\nATOM    183  O   ASN A  26      13.603  42.372   7.521  1.00 25.10           O  \\nATOM    184  CB  ASN A  26      12.522  41.369   5.315  1.00 27.68           C  \\nATOM    185  CG  ASN A  26      12.942  40.351   4.288  1.00 32.80           C  \\nATOM    186  OD1 ASN A  26      12.135  39.699   3.602  1.00 33.77           O  \\nATOM    187  ND2 ASN A  26      14.268  40.203   4.237  1.00 34.26           N  \\nATOM    188  N   HIS A  27      12.764  40.842   8.945  1.00 19.95           N  \\nATOM    189  CA  HIS A  27      13.593  41.266  10.057  1.00 17.99           C  \\nATOM    190  C   HIS A  27      13.441  40.299  11.215  1.00 16.84           C  \\nATOM    191  O   HIS A  27      12.411  39.623  11.344  1.00 14.72           O  \\nATOM    192  CB  HIS A  27      13.241  42.680  10.551  1.00 19.80           C  \\nATOM    193  CG  HIS A  27      11.926  42.908  11.258  1.00 20.77           C  \\nATOM    194  ND1 HIS A  27      11.936  43.314  12.510  1.00 22.17           N  \\nATOM    195  CD2 HIS A  27      10.674  42.768  10.729  1.00 22.32           C  \\nATOM    196  CE1 HIS A  27      10.657  43.430  12.785  1.00 23.86           C  \\nATOM    197  NE2 HIS A  27       9.905  43.110  11.738  1.00 23.97           N  \\nATOM    198  N   PHE A  28      14.516  40.222  12.012  1.00 15.44           N  \\nATOM    199  CA  PHE A  28      14.528  39.413  13.223  1.00 13.89           C  \\nATOM    200  C   PHE A  28      14.026  40.196  14.440  1.00 15.33           C  \\nATOM    201  O   PHE A  28      14.528  41.255  14.786  1.00 16.80           O  \\nATOM    202  CB  PHE A  28      15.938  38.906  13.470  1.00 12.73           C  \\nATOM    203  CG  PHE A  28      16.083  38.150  14.786  1.00 13.35           C  \\nATOM    204  CD1 PHE A  28      15.354  36.990  15.012  1.00 13.80           C  \\nATOM    205  CD2 PHE A  28      16.974  38.615  15.740  1.00 14.54           C  \\nATOM    206  CE1 PHE A  28      15.534  36.310  16.194  1.00 14.45           C  \\nATOM    207  CE2 PHE A  28      17.147  37.925  16.920  1.00 14.61           C  \\nATOM    208  CZ  PHE A  28      16.420  36.768  17.137  1.00 14.65           C  \\nATOM    209  N   VAL A  29      12.970  39.697  15.089  1.00 17.22           N  \\nATOM    210  CA  VAL A  29      12.467  40.313  16.294  1.00 19.11           C  \\nATOM    211  C   VAL A  29      13.127  39.606  17.476  1.00 19.36           C  \\nATOM    212  O   VAL A  29      13.245  38.376  17.605  1.00 18.14           O  \\nATOM    213  CB  VAL A  29      10.894  40.205  16.412  1.00 22.03           C  \\nATOM    214  CG1 VAL A  29      10.334  40.869  15.143  1.00 19.66           C  \\nATOM    215  CG2 VAL A  29      10.377  38.776  16.631  1.00 23.15           C  \\nATOM    216  N   ASP A  30      13.615  40.484  18.324  1.00 20.14           N  \\nATOM    217  CA  ASP A  30      14.229  40.019  19.518  1.00 24.04           C  \\nATOM    218  C   ASP A  30      13.676  40.746  20.721  1.00 25.93           C  \\nATOM    219  O   ASP A  30      14.244  41.709  21.238  1.00 29.26           O  \\nATOM    220  CB  ASP A  30      15.719  40.203  19.443  1.00 27.10           C  \\nATOM    221  CG  ASP A  30      16.389  39.731  20.732  1.00 32.46           C  \\nATOM    222  OD1 ASP A  30      15.878  38.859  21.464  1.00 34.28           O  \\nATOM    223  OD2 ASP A  30      17.458  40.264  21.002  1.00 36.46           O  \\nATOM    224  N   GLU A  31      12.472  40.347  21.076  1.00 26.14           N  \\nATOM    225  CA  GLU A  31      11.922  40.804  22.318  1.00 28.72           C  \\nATOM    226  C   GLU A  31      10.870  39.836  22.749  1.00 26.20           C  \\nATOM    227  O   GLU A  31      10.466  38.931  22.014  1.00 23.14           O  \\nATOM    228  CB  GLU A  31      11.337  42.210  22.204  1.00 33.25           C  \\nATOM    229  CG  GLU A  31      10.350  42.618  21.152  1.00 40.08           C  \\nATOM    230  CD  GLU A  31      10.572  44.107  20.887  1.00 44.71           C  \\nATOM    231  OE1 GLU A  31      10.930  44.444  19.755  1.00 47.50           O  \\nATOM    232  OE2 GLU A  31      10.420  44.928  21.806  1.00 47.89           O  \\nATOM    233  N   TYR A  32      10.550  40.044  24.028  1.00 22.26           N  \\nATOM    234  CA  TYR A  32       9.597  39.196  24.685  1.00 18.83           C  \\nATOM    235  C   TYR A  32       8.204  39.688  24.317  1.00 17.53           C  \\nATOM    236  O   TYR A  32       7.794  40.792  24.676  1.00 18.45           O  \\nATOM    237  CB  TYR A  32       9.909  39.259  26.199  1.00 16.84           C  \\nATOM    238  CG  TYR A  32       9.076  38.309  27.052  1.00 15.64           C  \\nATOM    239  CD1 TYR A  32       9.598  37.096  27.433  1.00 16.17           C  \\nATOM    240  CD2 TYR A  32       7.814  38.686  27.459  1.00 15.54           C  \\nATOM    241  CE1 TYR A  32       8.845  36.251  28.239  1.00 17.86           C  \\nATOM    242  CE2 TYR A  32       7.051  37.861  28.257  1.00 14.40           C  \\nATOM    243  CZ  TYR A  32       7.572  36.635  28.652  1.00 16.47           C  \\nATOM    244  OH  TYR A  32       6.821  35.789  29.485  1.00 13.59           O  \\nATOM    245  N   ASP A  33       7.461  38.841  23.604  1.00 13.43           N  \\nATOM    246  CA  ASP A  33       6.128  39.200  23.141  1.00 14.00           C  \\nATOM    247  C   ASP A  33       5.262  37.955  22.854  1.00 13.56           C  \\nATOM    248  O   ASP A  33       5.024  37.586  21.681  1.00 12.47           O  \\nATOM    249  CB  ASP A  33       6.271  40.072  21.864  1.00 15.92           C  \\nATOM    250  CG  ASP A  33       4.993  40.654  21.269  1.00 18.19           C  \\nATOM    251  OD1 ASP A  33       3.923  40.631  21.879  1.00 21.46           O  \\nATOM    252  OD2 ASP A  33       5.074  41.164  20.167  1.00 20.34           O  \\nATOM    253  N   PRO A  34       4.752  37.283  23.903  1.00 12.04           N  \\nATOM    254  CA  PRO A  34       3.949  36.070  23.788  1.00 11.09           C  \\nATOM    255  C   PRO A  34       2.720  36.248  22.895  1.00 11.76           C  \\nATOM    256  O   PRO A  34       1.953  37.220  23.006  1.00 12.63           O  \\nATOM    257  CB  PRO A  34       3.590  35.711  25.219  1.00 12.50           C  \\nATOM    258  CG  PRO A  34       4.643  36.418  26.033  1.00 12.14           C  \\nATOM    259  CD  PRO A  34       4.817  37.717  25.311  1.00 12.82           C  \\nATOM    260  N   THR A  35       2.554  35.282  21.972  1.00 11.00           N  \\nATOM    261  CA  THR A  35       1.421  35.320  21.033  1.00 11.54           C  \\nATOM    262  C   THR A  35       0.140  34.746  21.639  1.00 13.51           C  \\nATOM    263  O   THR A  35       0.174  33.820  22.450  1.00 12.45           O  \\nATOM    264  CB  THR A  35       1.730  34.518  19.751  1.00 11.34           C  \\nATOM    265  OG1 THR A  35       2.900  35.134  19.233  1.00 11.20           O  \\nATOM    266  CG2 THR A  35       0.706  34.611  18.637  1.00 13.10           C  \\nATOM    267  N   ILE A  36      -0.996  35.332  21.233  1.00 15.48           N  \\nATOM    268  CA  ILE A  36      -2.308  34.792  21.567  1.00 19.42           C  \\nATOM    269  C   ILE A  36      -2.663  34.079  20.274  1.00 17.36           C  \\nATOM    270  O   ILE A  36      -2.749  32.863  20.256  1.00 18.29           O  \\nATOM    271  CB  ILE A  36      -3.353  35.914  21.879  1.00 21.30           C  \\nATOM    272  CG1 ILE A  36      -2.930  36.823  23.072  1.00 24.32           C  \\nATOM    273  CG2 ILE A  36      -4.670  35.207  22.136  1.00 21.54           C  \\nATOM    274  CD1 ILE A  36      -2.494  36.167  24.413  1.00 24.70           C  \\nATOM    275  N   GLU A  37      -2.818  34.822  19.181  1.00 19.35           N  \\nATOM    276  CA  GLU A  37      -3.059  34.221  17.889  1.00 21.47           C  \\nATOM    277  C   GLU A  37      -2.957  35.362  16.897  1.00 23.77           C  \\nATOM    278  O   GLU A  37      -3.636  36.393  17.031  1.00 25.49           O  \\nATOM    279  CB  GLU A  37      -4.440  33.611  17.880  1.00 25.97           C  \\nATOM    280  CG  GLU A  37      -4.766  32.776  16.667  1.00 33.81           C  \\nATOM    281  CD  GLU A  37      -6.160  32.154  16.777  1.00 38.37           C  \\nATOM    282  OE1 GLU A  37      -7.095  32.664  16.149  1.00 37.86           O  \\nATOM    283  OE2 GLU A  37      -6.298  31.164  17.515  1.00 42.24           O  \\nATOM    284  N   ASP A  38      -2.070  35.212  15.911  1.00 19.96           N  \\nATOM    285  CA  ASP A  38      -1.899  36.178  14.839  1.00 17.94           C  \\nATOM    286  C   ASP A  38      -2.024  35.526  13.473  1.00 18.21           C  \\nATOM    287  O   ASP A  38      -1.582  34.393  13.294  1.00 16.96           O  \\nATOM    288  CB  ASP A  38      -0.540  36.833  14.965  1.00 18.83           C  \\nATOM    289  CG  ASP A  38      -0.507  37.817  16.133  1.00 20.08           C  \\nATOM    290  OD1 ASP A  38      -1.390  38.660  16.208  1.00 25.47           O  \\nATOM    291  OD2 ASP A  38       0.375  37.781  16.967  1.00 20.08           O  \\nATOM    292  N   SER A  39      -2.698  36.170  12.507  1.00 17.82           N  \\nATOM    293  CA  SER A  39      -2.691  35.658  11.133  1.00 18.16           C  \\nATOM    294  C   SER A  39      -2.012  36.627  10.171  1.00 16.95           C  \\nATOM    295  O   SER A  39      -2.053  37.853  10.333  1.00 15.80           O  \\nATOM    296  CB  SER A  39      -4.094  35.402  10.637  1.00 23.01           C  \\nATOM    297  OG  SER A  39      -4.873  36.571  10.754  1.00 27.13           O  \\nATOM    298  N   TYR A  40      -1.336  36.080   9.168  1.00 15.44           N  \\nATOM    299  CA  TYR A  40      -0.607  36.889   8.196  1.00 16.25           C  \\nATOM    300  C   TYR A  40      -1.018  36.383   6.830  1.00 17.06           C  \\nATOM    301  O   TYR A  40      -1.160  35.172   6.638  1.00 18.29           O  \\nATOM    302  CB  TYR A  40       0.909  36.724   8.370  1.00 16.59           C  \\nATOM    303  CG  TYR A  40       1.439  37.242   9.707  1.00 16.10           C  \\nATOM    304  CD1 TYR A  40       1.473  36.412  10.815  1.00 16.77           C  \\nATOM    305  CD2 TYR A  40       1.883  38.549   9.799  1.00 18.02           C  \\nATOM    306  CE1 TYR A  40       1.950  36.893  12.021  1.00 16.86           C  \\nATOM    307  CE2 TYR A  40       2.368  39.032  11.001  1.00 18.70           C  \\nATOM    308  CZ  TYR A  40       2.396  38.199  12.104  1.00 17.62           C  \\nATOM    309  OH  TYR A  40       2.853  38.695  13.304  1.00 18.37           O  \\nATOM    310  N   ARG A  41      -1.270  37.303   5.900  1.00 15.48           N  \\nATOM    311  CA  ARG A  41      -1.660  36.918   4.558  1.00 15.42           C  \\nATOM    312  C   ARG A  41      -0.730  37.606   3.586  1.00 16.76           C  \\nATOM    313  O   ARG A  41      -0.536  38.829   3.633  1.00 17.01           O  \\nATOM    314  CB  ARG A  41      -3.087  37.342   4.277  1.00 17.64           C  \\nATOM    315  CG  ARG A  41      -4.150  36.507   4.947  1.00 18.09           C  \\nATOM    316  CD  ARG A  41      -5.528  37.181   4.849  1.00 19.64           C  \\nATOM    317  NE  ARG A  41      -6.066  37.318   3.489  1.00 20.11           N  \\nATOM    318  CZ  ARG A  41      -6.660  36.329   2.803  1.00 22.17           C  \\nATOM    319  NH1 ARG A  41      -7.219  36.571   1.617  1.00 22.48           N  \\nATOM    320  NH2 ARG A  41      -6.711  35.090   3.274  1.00 23.55           N  \\nATOM    321  N   LYS A  42      -0.135  36.827   2.679  1.00 16.52           N  \\nATOM    322  CA  LYS A  42       0.835  37.405   1.758  1.00 16.59           C  \\nATOM    323  C   LYS A  42       0.740  36.746   0.387  1.00 16.04           C  \\nATOM    324  O   LYS A  42       0.794  35.528   0.307  1.00 14.74           O  \\nATOM    325  CB  LYS A  42       2.208  37.210   2.401  1.00 18.22           C  \\nATOM    326  CG  LYS A  42       3.388  37.784   1.669  1.00 22.68           C  \\nATOM    327  CD  LYS A  42       3.369  39.274   1.792  1.00 28.09           C  \\nATOM    328  CE  LYS A  42       4.695  39.876   1.317  1.00 31.70           C  \\nATOM    329  NZ  LYS A  42       4.828  39.827  -0.128  1.00 35.70           N  \\nATOM    330  N   GLN A  43       0.561  37.510  -0.697  1.00 16.89           N  \\nATOM    331  CA  GLN A  43       0.579  36.950  -2.047  1.00 19.10           C  \\nATOM    332  C   GLN A  43       2.026  36.729  -2.448  1.00 15.08           C  \\nATOM    333  O   GLN A  43       2.828  37.654  -2.362  1.00 15.72           O  \\nATOM    334  CB  GLN A  43      -0.116  37.908  -3.043  1.00 23.32           C  \\nATOM    335  CG  GLN A  43       0.543  38.332  -4.415  1.00 32.83           C  \\nATOM    336  CD  GLN A  43       1.081  37.332  -5.482  1.00 36.11           C  \\nATOM    337  OE1 GLN A  43       0.457  36.335  -5.865  1.00 37.78           O  \\nATOM    338  NE2 GLN A  43       2.271  37.606  -6.053  1.00 35.45           N  \\nATOM    339  N   VAL A  44       2.394  35.532  -2.922  1.00 12.79           N  \\nATOM    340  CA  VAL A  44       3.771  35.242  -3.311  1.00 11.03           C  \\nATOM    341  C   VAL A  44       3.724  34.425  -4.605  1.00 14.28           C  \\nATOM    342  O   VAL A  44       2.633  34.109  -5.090  1.00 14.90           O  \\nATOM    343  CB  VAL A  44       4.508  34.411  -2.223  1.00 13.21           C  \\nATOM    344  CG1 VAL A  44       4.669  35.282  -0.977  1.00 15.04           C  \\nATOM    345  CG2 VAL A  44       3.761  33.116  -1.889  1.00 12.89           C  \\nATOM    346  N   VAL A  45       4.879  34.127  -5.235  1.00 13.20           N  \\nATOM    347  CA  VAL A  45       4.938  33.277  -6.419  1.00 12.28           C  \\nATOM    348  C   VAL A  45       5.836  32.161  -5.970  1.00 11.08           C  \\nATOM    349  O   VAL A  45       6.988  32.447  -5.636  1.00 13.13           O  \\nATOM    350  CB  VAL A  45       5.587  33.979  -7.641  1.00 13.01           C  \\nATOM    351  CG1 VAL A  45       5.549  33.031  -8.810  1.00 14.51           C  \\nATOM    352  CG2 VAL A  45       4.787  35.173  -8.108  1.00 13.54           C  \\nATOM    353  N   ILE A  46       5.374  30.907  -5.915  1.00  9.89           N  \\nATOM    354  CA  ILE A  46       6.274  29.795  -5.491  1.00 11.03           C  \\nATOM    355  C   ILE A  46       6.369  28.800  -6.662  1.00  8.85           C  \\nATOM    356  O   ILE A  46       5.319  28.296  -7.059  1.00 10.25           O  \\nATOM    357  CB  ILE A  46       5.692  29.115  -4.203  1.00 10.64           C  \\nATOM    358  CG1 ILE A  46       5.652  30.096  -3.042  1.00 11.20           C  \\nATOM    359  CG2 ILE A  46       6.599  27.953  -3.757  1.00 12.93           C  \\nATOM    360  CD1 ILE A  46       4.937  29.536  -1.808  1.00  8.90           C  \\nATOM    361  N   ASP A  47       7.565  28.495  -7.224  1.00  9.41           N  \\nATOM    362  CA  ASP A  47       7.767  27.650  -8.426  1.00  9.70           C  \\nATOM    363  C   ASP A  47       6.838  28.031  -9.572  1.00 11.14           C  \\nATOM    364  O   ASP A  47       6.225  27.211 -10.248  1.00 14.09           O  \\nATOM    365  CB  ASP A  47       7.566  26.165  -8.105  1.00 10.73           C  \\nATOM    366  CG  ASP A  47       8.388  25.783  -6.874  1.00 13.52           C  \\nATOM    367  OD1 ASP A  47       7.805  25.291  -5.898  1.00 12.04           O  \\nATOM    368  OD2 ASP A  47       9.598  26.024  -6.872  1.00 11.31           O  \\nATOM    369  N   GLY A  48       6.724  29.353  -9.735  1.00 10.78           N  \\nATOM    370  CA  GLY A  48       5.930  29.961 -10.785  1.00 13.29           C  \\nATOM    371  C   GLY A  48       4.426  29.989 -10.548  1.00 14.28           C  \\nATOM    372  O   GLY A  48       3.677  30.512 -11.386  1.00 15.19           O  \\nATOM    373  N   GLU A  49       3.944  29.422  -9.439  1.00 13.41           N  \\nATOM    374  CA  GLU A  49       2.520  29.457  -9.143  1.00 13.90           C  \\nATOM    375  C   GLU A  49       2.223  30.700  -8.314  1.00 15.34           C  \\nATOM    376  O   GLU A  49       2.860  30.906  -7.275  1.00 14.56           O  \\nATOM    377  CB  GLU A  49       2.124  28.238  -8.340  1.00 15.50           C  \\nATOM    378  CG  GLU A  49       0.630  28.250  -7.990  1.00 18.10           C  \\nATOM    379  CD  GLU A  49       0.204  27.110  -7.100  1.00 19.13           C  \\nATOM    380  OE1 GLU A  49       0.717  27.004  -5.996  1.00 19.53           O  \\nATOM    381  OE2 GLU A  49      -0.630  26.321  -7.529  1.00 22.12           O  \\nATOM    382  N   THR A  50       1.267  31.536  -8.734  1.00 14.06           N  \\nATOM    383  CA  THR A  50       0.901  32.697  -7.927  1.00 16.15           C  \\nATOM    384  C   THR A  50      -0.001  32.169  -6.819  1.00 13.35           C  \\nATOM    385  O   THR A  50      -0.938  31.409  -7.046  1.00 15.13           O  \\nATOM    386  CB  THR A  50       0.203  33.755  -8.846  1.00 17.88           C  \\nATOM    387  OG1 THR A  50      -0.245  34.813  -8.012  1.00 24.33           O  \\nATOM    388  CG2 THR A  50      -1.028  33.236  -9.518  1.00 19.78           C  \\nATOM    389  N   CYS A  51       0.306  32.442  -5.563  1.00 13.28           N  \\nATOM    390  CA  CYS A  51      -0.566  31.959  -4.508  1.00 15.16           C  \\nATOM    391  C   CYS A  51      -0.579  32.892  -3.303  1.00 14.00           C  \\nATOM    392  O   CYS A  51       0.161  33.874  -3.231  1.00 11.81           O  \\nATOM    393  CB  CYS A  51      -0.132  30.537  -4.079  1.00 14.08           C  \\nATOM    394  SG  CYS A  51       1.597  30.409  -3.575  1.00 16.09           S  \\nATOM    395  N   LEU A  52      -1.521  32.609  -2.405  1.00 13.77           N  \\nATOM    396  CA  LEU A  52      -1.573  33.397  -1.199  1.00 16.01           C  \\nATOM    397  C   LEU A  52      -1.260  32.497  -0.025  1.00 14.53           C  \\nATOM    398  O   LEU A  52      -1.868  31.443   0.113  1.00 16.05           O  \\nATOM    399  CB  LEU A  52      -2.957  33.986  -1.014  1.00 19.71           C  \\nATOM    400  CG  LEU A  52      -3.122  34.885   0.214  1.00 20.84           C  \\nATOM    401  CD1 LEU A  52      -3.070  36.329  -0.220  1.00 21.38           C  \\nATOM    402  CD2 LEU A  52      -4.420  34.526   0.923  1.00 20.90           C  \\nATOM    403  N   LEU A  53      -0.319  32.904   0.800  1.00 12.41           N  \\nATOM    404  CA  LEU A  53       0.019  32.232   2.048  1.00 13.70           C  \\nATOM    405  C   LEU A  53      -0.841  32.852   3.148  1.00 13.85           C  \\nATOM    406  O   LEU A  53      -0.733  34.056   3.413  1.00 12.82           O  \\nATOM    407  CB  LEU A  53       1.485  32.464   2.400  1.00 14.51           C  \\nATOM    408  CG  LEU A  53       2.707  31.589   2.031  1.00 19.29           C  \\nATOM    409  CD1 LEU A  53       2.394  30.553   0.962  1.00 18.56           C  \\nATOM    410  CD2 LEU A  53       3.843  32.557   1.741  1.00 20.00           C  \\nATOM    411  N   ASP A  54      -1.699  32.023   3.751  1.00 11.96           N  \\nATOM    412  CA  ASP A  54      -2.476  32.445   4.884  1.00 15.07           C  \\nATOM    413  C   ASP A  54      -1.882  31.675   6.055  1.00 12.67           C  \\nATOM    414  O   ASP A  54      -2.099  30.477   6.234  1.00 13.99           O  \\nATOM    415  CB  ASP A  54      -3.930  32.088   4.644  1.00 19.33           C  \\nATOM    416  CG  ASP A  54      -4.826  32.609   5.746  1.00 24.24           C  \\nATOM    417  OD1 ASP A  54      -4.412  32.652   6.897  1.00 28.17           O  \\nATOM    418  OD2 ASP A  54      -5.958  32.979   5.450  1.00 27.59           O  \\nATOM    419  N   ILE A  55      -1.127  32.389   6.880  1.00 11.78           N  \\nATOM    420  CA  ILE A  55      -0.346  31.769   7.935  1.00 12.15           C  \\nATOM    421  C   ILE A  55      -0.953  32.099   9.296  1.00 13.10           C  \\nATOM    422  O   ILE A  55      -1.195  33.271   9.587  1.00 13.84           O  \\nATOM    423  CB  ILE A  55       1.109  32.282   7.931  1.00 12.42           C  \\nATOM    424  CG1 ILE A  55       1.859  31.979   6.653  1.00 13.99           C  \\nATOM    425  CG2 ILE A  55       1.859  31.505   9.018  1.00 13.24           C  \\nATOM    426  CD1 ILE A  55       3.176  32.768   6.497  1.00 15.68           C  \\nATOM    427  N   LEU A  56      -1.176  31.083  10.134  1.00 11.64           N  \\nATOM    428  CA  LEU A  56      -1.642  31.341  11.490  1.00 11.98           C  \\nATOM    429  C   LEU A  56      -0.524  31.027  12.431  1.00  9.99           C  \\nATOM    430  O   LEU A  56      -0.017  29.895  12.447  1.00  9.72           O  \\nATOM    431  CB  LEU A  56      -2.836  30.463  11.896  1.00 13.80           C  \\nATOM    432  CG  LEU A  56      -3.470  30.741  13.297  1.00 16.13           C  \\nATOM    433  CD1 LEU A  56      -4.347  31.987  13.260  1.00 15.28           C  \\nATOM    434  CD2 LEU A  56      -4.321  29.580  13.692  1.00 16.84           C  \\nATOM    435  N   ASP A  57      -0.188  32.084  13.182  1.00 10.40           N  \\nATOM    436  CA  ASP A  57       0.815  32.014  14.219  1.00 10.30           C  \\nATOM    437  C   ASP A  57       0.127  31.812  15.571  1.00 11.75           C  \\nATOM    438  O   ASP A  57      -0.652  32.631  16.053  1.00 12.91           O  \\nATOM    439  CB  ASP A  57       1.602  33.296  14.173  1.00 10.66           C  \\nATOM    440  CG  ASP A  57       2.778  33.336  15.132  1.00 11.39           C  \\nATOM    441  OD1 ASP A  57       3.306  32.298  15.535  1.00 11.07           O  \\nATOM    442  OD2 ASP A  57       3.168  34.448  15.455  1.00 13.15           O  \\nATOM    443  N   THR A  58       0.447  30.723  16.244  1.00 12.55           N  \\nATOM    444  CA  THR A  58      -0.324  30.325  17.416  1.00 12.98           C  \\nATOM    445  C   THR A  58       0.368  30.541  18.758  1.00 13.48           C  \\nATOM    446  O   THR A  58       1.568  30.812  18.815  1.00 14.00           O  \\nATOM    447  CB  THR A  58      -0.698  28.851  17.219  1.00  9.54           C  \\nATOM    448  OG1 THR A  58       0.491  28.084  17.369  1.00 11.40           O  \\nATOM    449  CG2 THR A  58      -1.313  28.597  15.848  1.00 12.27           C  \\nATOM    450  N   ALA A  59      -0.371  30.382  19.868  1.00 14.38           N  \\nATOM    451  CA  ALA A  59       0.230  30.510  21.194  1.00 14.79           C  \\nATOM    452  C   ALA A  59       0.928  29.210  21.608  1.00 15.41           C  \\nATOM    453  O   ALA A  59       0.448  28.106  21.324  1.00 13.07           O  \\nATOM    454  CB  ALA A  59      -0.845  30.818  22.238  1.00 15.47           C  \\nATOM    455  N   GLY A  60       2.084  29.341  22.271  1.00 17.71           N  \\nATOM    456  CA  GLY A  60       2.807  28.208  22.814  1.00 24.18           C  \\nATOM    457  C   GLY A  60       2.190  27.751  24.130  1.00 31.21           C  \\nATOM    458  O   GLY A  60       2.538  26.688  24.627  1.00 30.44           O  \\nATOM    459  N   GLN A  61       1.260  28.503  24.726  1.00 39.50           N  \\nATOM    460  CA  GLN A  61       0.671  28.099  25.994  1.00 48.84           C  \\nATOM    461  C   GLN A  61      -0.683  27.395  25.985  1.00 51.52           C  \\nATOM    462  O   GLN A  61      -1.606  27.780  25.276  1.00 52.17           O  \\nATOM    463  CB  GLN A  61       0.605  29.327  26.874  1.00 51.56           C  \\nATOM    464  CG  GLN A  61       0.871  28.861  28.312  1.00 56.74           C  \\nATOM    465  CD  GLN A  61       1.506  29.937  29.177  1.00 58.74           C  \\nATOM    466  OE1 GLN A  61       2.647  29.795  29.616  1.00 60.12           O  \\nATOM    467  NE2 GLN A  61       0.783  31.031  29.429  1.00 60.18           N  \\nATOM    468  N   GLU A  62      -0.804  26.349  26.818  1.00 57.40           N  \\nATOM    469  CA  GLU A  62      -2.008  25.527  26.927  1.00 61.96           C  \\nATOM    470  C   GLU A  62      -3.185  26.196  27.631  1.00 62.20           C  \\nATOM    471  O   GLU A  62      -4.138  25.496  27.975  1.00 62.85           O  \\nATOM    472  CB  GLU A  62      -1.629  24.183  27.632  1.00 65.18           C  \\nATOM    473  CG  GLU A  62      -2.622  22.966  27.761  1.00 68.22           C  \\nATOM    474  CD  GLU A  62      -3.032  22.132  26.528  1.00 70.25           C  \\nATOM    475  OE1 GLU A  62      -2.710  22.494  25.395  1.00 71.45           O  \\nATOM    476  OE2 GLU A  62      -3.691  21.101  26.689  1.00 70.00           O  \\nATOM    477  N   GLU A  63      -3.206  27.520  27.904  1.00 62.78           N  \\nATOM    478  CA  GLU A  63      -4.496  28.139  28.254  1.00 63.62           C  \\nATOM    479  C   GLU A  63      -5.457  27.854  27.085  1.00 63.71           C  \\nATOM    480  O   GLU A  63      -6.688  27.814  27.211  1.00 62.97           O  \\nATOM    481  CB  GLU A  63      -4.372  29.666  28.475  1.00 65.13           C  \\nATOM    482  CG  GLU A  63      -3.456  30.553  27.607  1.00 66.91           C  \\nATOM    483  CD  GLU A  63      -4.012  31.235  26.345  1.00 68.05           C  \\nATOM    484  OE1 GLU A  63      -5.220  31.462  26.220  1.00 68.64           O  \\nATOM    485  OE2 GLU A  63      -3.198  31.568  25.481  1.00 67.62           O  \\nATOM    486  N   TYR A  64      -4.801  27.606  25.934  1.00 63.36           N  \\nATOM    487  CA  TYR A  64      -5.420  27.092  24.743  1.00 63.20           C  \\nATOM    488  C   TYR A  64      -5.905  25.653  24.875  1.00 61.76           C  \\nATOM    489  O   TYR A  64      -7.120  25.486  24.858  1.00 63.04           O  \\nATOM    490  CB  TYR A  64      -4.437  27.208  23.613  1.00 64.83           C  \\nATOM    491  CG  TYR A  64      -4.736  28.459  22.818  1.00 66.98           C  \\nATOM    492  CD1 TYR A  64      -4.300  29.686  23.267  1.00 67.55           C  \\nATOM    493  CD2 TYR A  64      -5.459  28.355  21.646  1.00 67.97           C  \\nATOM    494  CE1 TYR A  64      -4.587  30.825  22.536  1.00 69.49           C  \\nATOM    495  CE2 TYR A  64      -5.752  29.488  20.911  1.00 69.29           C  \\nATOM    496  CZ  TYR A  64      -5.313  30.718  21.359  1.00 70.05           C  \\nATOM    497  OH  TYR A  64      -5.589  31.852  20.614  1.00 70.67           O  \\nATOM    498  N   SER A  65      -5.095  24.590  24.997  1.00 58.67           N  \\nATOM    499  CA  SER A  65      -5.617  23.230  25.127  1.00 56.93           C  \\nATOM    500  C   SER A  65      -6.614  22.757  24.063  1.00 55.33           C  \\nATOM    501  O   SER A  65      -6.321  22.936  22.876  1.00 54.74           O  \\nATOM    502  CB  SER A  65      -6.190  23.122  26.552  1.00 57.99           C  \\nATOM    503  OG  SER A  65      -7.251  24.011  26.900  1.00 59.80           O  \\nATOM    504  N   ALA A  66      -7.762  22.119  24.359  1.00 52.33           N  \\nATOM    505  CA  ALA A  66      -8.766  21.824  23.335  1.00 48.80           C  \\nATOM    506  C   ALA A  66      -9.138  23.050  22.492  1.00 45.71           C  \\nATOM    507  O   ALA A  66      -9.751  22.862  21.449  1.00 44.33           O  \\nATOM    508  CB  ALA A  66     -10.043  21.300  23.971  1.00 49.14           C  \\nATOM    509  N   MET A  67      -8.829  24.310  22.875  1.00 42.85           N  \\nATOM    510  CA  MET A  67      -8.984  25.465  21.967  1.00 40.01           C  \\nATOM    511  C   MET A  67      -8.093  25.362  20.719  1.00 36.79           C  \\nATOM    512  O   MET A  67      -8.221  26.184  19.805  1.00 36.08           O  \\nATOM    513  CB  MET A  67      -8.594  26.804  22.581  1.00 42.50           C  \\nATOM    514  CG  MET A  67      -9.359  27.330  23.772  1.00 45.93           C  \\nATOM    515  SD  MET A  67     -11.140  27.396  23.483  1.00 48.74           S  \\nATOM    516  CE  MET A  67     -11.526  25.880  24.321  1.00 47.64           C  \\nATOM    517  N   ARG A  68      -7.128  24.418  20.665  1.00 31.59           N  \\nATOM    518  CA  ARG A  68      -6.322  24.217  19.467  1.00 26.34           C  \\nATOM    519  C   ARG A  68      -7.140  23.473  18.424  1.00 23.68           C  \\nATOM    520  O   ARG A  68      -6.846  23.617  17.231  1.00 21.93           O  \\nATOM    521  CB  ARG A  68      -5.057  23.394  19.745  1.00 25.86           C  \\nATOM    522  CG  ARG A  68      -4.161  23.898  20.869  1.00 29.36           C  \\nATOM    523  CD  ARG A  68      -2.881  23.061  20.998  1.00 32.88           C  \\nATOM    524  NE  ARG A  68      -2.221  22.990  22.315  1.00 36.64           N  \\nATOM    525  CZ  ARG A  68      -1.700  24.031  23.001  1.00 39.66           C  \\nATOM    526  NH1 ARG A  68      -1.903  25.315  22.664  1.00 42.06           N  \\nATOM    527  NH2 ARG A  68      -0.933  23.798  24.071  1.00 39.61           N  \\nATOM    528  N   ASP A  69      -8.188  22.721  18.793  1.00 19.14           N  \\nATOM    529  CA  ASP A  69      -8.880  21.905  17.822  1.00 20.43           C  \\nATOM    530  C   ASP A  69      -9.437  22.623  16.617  1.00 22.16           C  \\nATOM    531  O   ASP A  69      -9.453  22.109  15.488  1.00 19.58           O  \\nATOM    532  CB  ASP A  69     -10.025  21.148  18.478  1.00 20.58           C  \\nATOM    533  CG  ASP A  69      -9.598  19.935  19.303  1.00 22.81           C  \\nATOM    534  OD1 ASP A  69      -8.409  19.708  19.538  1.00 24.59           O  \\nATOM    535  OD2 ASP A  69     -10.475  19.176  19.715  1.00 25.28           O  \\nATOM    536  N   GLN A  70      -9.834  23.874  16.840  1.00 23.26           N  \\nATOM    537  CA  GLN A  70     -10.451  24.609  15.759  1.00 23.53           C  \\nATOM    538  C   GLN A  70      -9.437  24.925  14.667  1.00 21.20           C  \\nATOM    539  O   GLN A  70      -9.653  24.591  13.493  1.00 20.90           O  \\nATOM    540  CB  GLN A  70     -11.071  25.881  16.341  1.00 27.73           C  \\nATOM    541  CG  GLN A  70     -12.498  26.019  15.832  1.00 34.11           C  \\nATOM    542  CD  GLN A  70     -13.103  27.407  16.008  1.00 37.87           C  \\nATOM    543  OE1 GLN A  70     -12.532  28.284  16.672  1.00 39.77           O  \\nATOM    544  NE2 GLN A  70     -14.283  27.620  15.406  1.00 37.89           N  \\nATOM    545  N   TYR A  71      -8.292  25.515  15.050  1.00 19.09           N  \\nATOM    546  CA  TYR A  71      -7.329  25.873  14.030  1.00 17.69           C  \\nATOM    547  C   TYR A  71      -6.648  24.658  13.416  1.00 17.24           C  \\nATOM    548  O   TYR A  71      -6.247  24.675  12.245  1.00 17.25           O  \\nATOM    549  CB  TYR A  71      -6.302  26.868  14.625  1.00 17.87           C  \\nATOM    550  CG  TYR A  71      -5.311  26.480  15.725  1.00 18.90           C  \\nATOM    551  CD1 TYR A  71      -4.355  25.510  15.492  1.00 19.74           C  \\nATOM    552  CD2 TYR A  71      -5.328  27.162  16.940  1.00 19.21           C  \\nATOM    553  CE1 TYR A  71      -3.402  25.212  16.450  1.00 20.38           C  \\nATOM    554  CE2 TYR A  71      -4.375  26.868  17.901  1.00 21.93           C  \\nATOM    555  CZ  TYR A  71      -3.422  25.897  17.645  1.00 21.91           C  \\nATOM    556  OH  TYR A  71      -2.463  25.590  18.588  1.00 26.44           O  \\nATOM    557  N   MET A  72      -6.557  23.559  14.166  1.00 14.81           N  \\nATOM    558  CA  MET A  72      -5.958  22.393  13.563  1.00 18.59           C  \\nATOM    559  C   MET A  72      -6.849  21.778  12.472  1.00 19.38           C  \\nATOM    560  O   MET A  72      -6.385  21.198  11.492  1.00 17.67           O  \\nATOM    561  CB  MET A  72      -5.658  21.386  14.669  1.00 17.39           C  \\nATOM    562  CG  MET A  72      -4.479  21.796  15.514  1.00 16.35           C  \\nATOM    563  SD  MET A  72      -4.146  20.600  16.820  1.00 17.04           S  \\nATOM    564  CE  MET A  72      -2.635  21.259  17.460  1.00 17.72           C  \\nATOM    565  N   ARG A  73      -8.174  21.931  12.626  1.00 20.31           N  \\nATOM    566  CA  ARG A  73      -9.120  21.350  11.701  1.00 22.24           C  \\nATOM    567  C   ARG A  73      -9.087  22.084  10.370  1.00 20.94           C  \\nATOM    568  O   ARG A  73      -9.183  21.450   9.326  1.00 20.81           O  \\nATOM    569  CB  ARG A  73     -10.499  21.414  12.369  1.00 25.29           C  \\nATOM    570  CG  ARG A  73     -11.553  20.499  11.767  1.00 30.04           C  \\nATOM    571  CD  ARG A  73     -12.775  20.429  12.687  1.00 31.86           C  \\nATOM    572  NE  ARG A  73     -12.483  19.702  13.917  1.00 34.70           N  \\nATOM    573  CZ  ARG A  73     -12.894  20.122  15.130  1.00 36.49           C  \\nATOM    574  NH1 ARG A  73     -12.579  19.402  16.196  1.00 36.93           N  \\nATOM    575  NH2 ARG A  73     -13.609  21.239  15.311  1.00 38.17           N  \\nATOM    576  N   THR A  74      -8.868  23.401  10.413  1.00 21.33           N  \\nATOM    577  CA  THR A  74      -8.813  24.240   9.219  1.00 24.99           C  \\nATOM    578  C   THR A  74      -7.436  24.351   8.544  1.00 23.10           C  \\nATOM    579  O   THR A  74      -7.367  24.738   7.371  1.00 22.59           O  \\nATOM    580  CB  THR A  74      -9.289  25.676   9.538  1.00 25.73           C  \\nATOM    581  OG1 THR A  74      -8.351  26.302  10.409  1.00 31.08           O  \\nATOM    582  CG2 THR A  74     -10.657  25.644  10.201  1.00 27.58           C  \\nATOM    583  N   GLY A  75      -6.345  24.084   9.289  1.00 19.58           N  \\nATOM    584  CA  GLY A  75      -5.013  24.114   8.730  1.00 15.79           C  \\nATOM    585  C   GLY A  75      -4.823  23.023   7.713  1.00 13.66           C  \\nATOM    586  O   GLY A  75      -5.245  21.880   7.884  1.00 14.69           O  \\nATOM    587  N   GLU A  76      -4.228  23.360   6.573  1.00 13.14           N  \\nATOM    588  CA  GLU A  76      -3.914  22.363   5.546  1.00 12.99           C  \\nATOM    589  C   GLU A  76      -2.514  21.740   5.753  1.00 12.22           C  \\nATOM    590  O   GLU A  76      -2.293  20.576   5.426  1.00 13.59           O  \\nATOM    591  CB  GLU A  76      -3.999  23.014   4.145  1.00 13.07           C  \\nATOM    592  CG  GLU A  76      -5.387  23.377   3.676  1.00 14.35           C  \\nATOM    593  CD  GLU A  76      -5.261  24.069   2.323  1.00 20.40           C  \\nATOM    594  OE1 GLU A  76      -5.461  23.403   1.288  1.00 24.32           O  \\nATOM    595  OE2 GLU A  76      -4.913  25.250   2.284  1.00 18.82           O  \\nATOM    596  N   GLY A  77      -1.553  22.503   6.301  1.00 10.73           N  \\nATOM    597  CA  GLY A  77      -0.216  22.018   6.570  1.00  8.07           C  \\nATOM    598  C   GLY A  77       0.302  22.689   7.819  1.00  8.63           C  \\nATOM    599  O   GLY A  77      -0.066  23.823   8.127  1.00  8.55           O  \\nATOM    600  N   PHE A  78       1.248  22.013   8.479  1.00  9.15           N  \\nATOM    601  CA  PHE A  78       1.754  22.468   9.773  1.00  8.87           C  \\nATOM    602  C   PHE A  78       3.268  22.520   9.854  1.00  9.51           C  \\nATOM    603  O   PHE A  78       3.930  21.542   9.486  1.00  9.15           O  \\nATOM    604  CB  PHE A  78       1.247  21.532  10.904  1.00  9.44           C  \\nATOM    605  CG  PHE A  78      -0.267  21.472  11.000  1.00  9.66           C  \\nATOM    606  CD1 PHE A  78      -0.906  22.310  11.901  1.00 10.71           C  \\nATOM    607  CD2 PHE A  78      -0.994  20.645  10.152  1.00 10.20           C  \\nATOM    608  CE1 PHE A  78      -2.295  22.338  11.940  1.00 10.61           C  \\nATOM    609  CE2 PHE A  78      -2.378  20.678  10.192  1.00 12.55           C  \\nATOM    610  CZ  PHE A  78      -3.022  21.529  11.081  1.00 12.80           C  \\nATOM    611  N   LEU A  79       3.813  23.636  10.354  1.00  8.72           N  \\nATOM    612  CA  LEU A  79       5.230  23.626  10.699  1.00  9.93           C  \\nATOM    613  C   LEU A  79       5.279  23.281  12.181  1.00  9.09           C  \\nATOM    614  O   LEU A  79       4.701  24.008  12.988  1.00 11.58           O  \\nATOM    615  CB  LEU A  79       5.921  24.970  10.567  1.00 12.80           C  \\nATOM    616  CG  LEU A  79       6.155  25.659   9.249  1.00 16.39           C  \\nATOM    617  CD1 LEU A  79       7.007  26.911   9.544  1.00 17.02           C  \\nATOM    618  CD2 LEU A  79       6.904  24.756   8.269  1.00 16.98           C  \\nATOM    619  N   CYS A  80       5.950  22.219  12.579  1.00  9.14           N  \\nATOM    620  CA  CYS A  80       6.074  21.821  13.981  1.00 11.81           C  \\nATOM    621  C   CYS A  80       7.448  22.237  14.434  1.00 11.53           C  \\nATOM    622  O   CYS A  80       8.435  21.640  14.000  1.00 12.21           O  \\nATOM    623  CB  CYS A  80       5.903  20.318  14.106  1.00 13.73           C  \\nATOM    624  SG  CYS A  80       4.158  19.900  13.837  1.00 18.26           S  \\nATOM    625  N   VAL A  81       7.486  23.282  15.269  1.00  7.57           N  \\nATOM    626  CA  VAL A  81       8.730  23.956  15.604  1.00  8.45           C  \\nATOM    627  C   VAL A  81       9.248  23.620  16.996  1.00  8.90           C  \\nATOM    628  O   VAL A  81       8.451  23.680  17.939  1.00  8.49           O  \\nATOM    629  CB  VAL A  81       8.544  25.516  15.507  1.00  7.40           C  \\nATOM    630  CG1 VAL A  81       9.917  26.196  15.588  1.00  9.42           C  \\nATOM    631  CG2 VAL A  81       8.004  25.954  14.167  1.00  8.59           C  \\nATOM    632  N   PHE A  82      10.546  23.237  17.119  1.00  9.49           N  \\nATOM    633  CA  PHE A  82      11.195  23.202  18.442  1.00  9.09           C  \\nATOM    634  C   PHE A  82      12.511  23.994  18.350  1.00  9.83           C  \\nATOM    635  O   PHE A  82      12.914  24.440  17.266  1.00  9.02           O  \\nATOM    636  CB  PHE A  82      11.482  21.755  18.902  1.00  9.76           C  \\nATOM    637  CG  PHE A  82      12.536  21.062  18.071  1.00 10.09           C  \\nATOM    638  CD1 PHE A  82      13.867  21.099  18.486  1.00 10.70           C  \\nATOM    639  CD2 PHE A  82      12.192  20.412  16.889  1.00 10.93           C  \\nATOM    640  CE1 PHE A  82      14.838  20.483  17.699  1.00 11.12           C  \\nATOM    641  CE2 PHE A  82      13.181  19.807  16.125  1.00 10.73           C  \\nATOM    642  CZ  PHE A  82      14.505  19.839  16.523  1.00 10.35           C  \\nATOM    643  N   ALA A  83      13.128  24.269  19.516  1.00  9.75           N  \\nATOM    644  CA  ALA A  83      14.407  24.963  19.569  1.00 10.32           C  \\nATOM    645  C   ALA A  83      15.509  23.960  19.916  1.00  9.48           C  \\nATOM    646  O   ALA A  83      15.362  23.142  20.839  1.00  9.61           O  \\nATOM    647  CB  ALA A  83      14.354  26.083  20.620  1.00  9.75           C  \\nATOM    648  N   ILE A  84      16.624  24.042  19.160  1.00 11.41           N  \\nATOM    649  CA  ILE A  84      17.714  23.070  19.299  1.00 13.15           C  \\nATOM    650  C   ILE A  84      18.446  23.223  20.623  1.00 14.57           C  \\nATOM    651  O   ILE A  84      19.234  22.347  20.971  1.00 13.94           O  \\nATOM    652  CB  ILE A  84      18.775  23.151  18.160  1.00 13.29           C  \\nATOM    653  CG1 ILE A  84      19.403  24.509  17.878  1.00 14.90           C  \\nATOM    654  CG2 ILE A  84      18.018  22.701  16.933  1.00 13.09           C  \\nATOM    655  CD1 ILE A  84      20.578  24.957  18.763  1.00 17.59           C  \\nATOM    656  N   ASN A  85      18.200  24.290  21.390  1.00 13.61           N  \\nATOM    657  CA  ASN A  85      18.794  24.394  22.720  1.00 15.52           C  \\nATOM    658  C   ASN A  85      17.758  24.156  23.828  1.00 15.49           C  \\nATOM    659  O   ASN A  85      17.988  24.519  24.999  1.00 16.13           O  \\nATOM    660  CB  ASN A  85      19.446  25.788  22.885  1.00 15.16           C  \\nATOM    661  CG  ASN A  85      18.464  26.946  22.904  1.00 16.42           C  \\nATOM    662  OD1 ASN A  85      17.355  26.878  22.362  1.00 16.01           O  \\nATOM    663  ND2 ASN A  85      18.808  28.030  23.587  1.00 16.83           N  \\nATOM    664  N   ASN A  86      16.616  23.515  23.509  1.00 14.72           N  \\nATOM    665  CA  ASN A  86      15.553  23.303  24.483  1.00 13.01           C  \\nATOM    666  C   ASN A  86      14.987  21.901  24.309  1.00 13.17           C  \\nATOM    667  O   ASN A  86      14.080  21.668  23.520  1.00 13.04           O  \\nATOM    668  CB  ASN A  86      14.478  24.367  24.256  1.00 16.51           C  \\nATOM    669  CG  ASN A  86      13.267  24.415  25.187  1.00 20.74           C  \\nATOM    670  OD1 ASN A  86      12.370  25.241  25.018  1.00 25.41           O  \\nATOM    671  ND2 ASN A  86      13.123  23.596  26.206  1.00 19.96           N  \\nATOM    672  N   THR A  87      15.488  20.950  25.103  1.00 13.80           N  \\nATOM    673  CA  THR A  87      15.095  19.553  25.026  1.00 15.40           C  \\nATOM    674  C   THR A  87      13.589  19.316  25.266  1.00 14.40           C  \\nATOM    675  O   THR A  87      12.953  18.535  24.546  1.00 14.11           O  \\nATOM    676  CB  THR A  87      15.983  18.791  26.046  1.00 16.19           C  \\nATOM    677  OG1 THR A  87      17.325  18.908  25.571  1.00 18.22           O  \\nATOM    678  CG2 THR A  87      15.626  17.327  26.180  1.00 17.82           C  \\nATOM    679  N   LYS A  88      12.983  19.958  26.272  1.00 14.42           N  \\nATOM    680  CA  LYS A  88      11.552  19.842  26.535  1.00 15.84           C  \\nATOM    681  C   LYS A  88      10.726  20.232  25.278  1.00 15.06           C  \\nATOM    682  O   LYS A  88       9.722  19.582  25.003  1.00 14.61           O  \\nATOM    683  CB  LYS A  88      11.207  20.766  27.717  1.00 19.53           C  \\nATOM    684  CG  LYS A  88       9.937  20.401  28.457  1.00 27.71           C  \\nATOM    685  CD  LYS A  88       9.585  21.452  29.529  1.00 33.26           C  \\nATOM    686  CE  LYS A  88       9.061  22.762  28.877  1.00 36.79           C  \\nATOM    687  NZ  LYS A  88       8.642  23.763  29.852  1.00 37.29           N  \\nATOM    688  N   SER A  89      11.119  21.259  24.475  1.00 12.36           N  \\nATOM    689  CA  SER A  89      10.325  21.666  23.309  1.00 11.31           C  \\nATOM    690  C   SER A  89      10.371  20.572  22.268  1.00 12.51           C  \\nATOM    691  O   SER A  89       9.397  20.388  21.547  1.00 12.88           O  \\nATOM    692  CB  SER A  89      10.844  22.951  22.674  1.00 11.64           C  \\nATOM    693  OG  SER A  89      12.160  22.900  22.124  1.00 11.11           O  \\nATOM    694  N   PHE A  90      11.504  19.842  22.204  1.00 11.27           N  \\nATOM    695  CA  PHE A  90      11.680  18.751  21.262  1.00 12.91           C  \\nATOM    696  C   PHE A  90      10.760  17.601  21.667  1.00 14.37           C  \\nATOM    697  O   PHE A  90      10.034  17.057  20.839  1.00 13.24           O  \\nATOM    698  CB  PHE A  90      13.161  18.312  21.260  1.00 12.34           C  \\nATOM    699  CG  PHE A  90      13.411  17.120  20.351  1.00 14.41           C  \\nATOM    700  CD1 PHE A  90      13.890  15.948  20.903  1.00 15.10           C  \\nATOM    701  CD2 PHE A  90      13.136  17.206  18.990  1.00 14.81           C  \\nATOM    702  CE1 PHE A  90      14.093  14.857  20.088  1.00 16.20           C  \\nATOM    703  CE2 PHE A  90      13.340  16.110  18.179  1.00 14.81           C  \\nATOM    704  CZ  PHE A  90      13.818  14.935  18.731  1.00 15.39           C  \\nATOM    705  N   GLU A  91      10.725  17.231  22.949  1.00 14.40           N  \\nATOM    706  CA  GLU A  91       9.863  16.146  23.407  1.00 15.78           C  \\nATOM    707  C   GLU A  91       8.374  16.466  23.301  1.00 16.18           C  \\nATOM    708  O   GLU A  91       7.533  15.582  23.114  1.00 17.50           O  \\nATOM    709  CB  GLU A  91      10.226  15.846  24.835  1.00 19.84           C  \\nATOM    710  CG  GLU A  91      11.654  15.386  25.074  1.00 24.49           C  \\nATOM    711  CD  GLU A  91      12.090  15.310  26.537  1.00 27.97           C  \\nATOM    712  OE1 GLU A  91      13.199  14.842  26.766  1.00 30.72           O  \\nATOM    713  OE2 GLU A  91      11.371  15.730  27.449  1.00 30.38           O  \\nATOM    714  N   ASP A  92       8.019  17.747  23.433  1.00 16.19           N  \\nATOM    715  CA  ASP A  92       6.650  18.216  23.265  1.00 15.23           C  \\nATOM    716  C   ASP A  92       6.141  18.053  21.839  1.00 14.29           C  \\nATOM    717  O   ASP A  92       4.927  18.074  21.633  1.00 14.78           O  \\nATOM    718  CB  ASP A  92       6.536  19.708  23.646  1.00 17.85           C  \\nATOM    719  CG  ASP A  92       6.706  20.080  25.127  1.00 21.39           C  \\nATOM    720  OD1 ASP A  92       6.648  19.209  25.997  1.00 20.06           O  \\nATOM    721  OD2 ASP A  92       6.887  21.272  25.401  1.00 24.39           O  \\nATOM    722  N   ILE A  93       7.013  17.910  20.826  1.00 11.76           N  \\nATOM    723  CA  ILE A  93       6.559  17.749  19.452  1.00 14.77           C  \\nATOM    724  C   ILE A  93       5.574  16.583  19.354  1.00 14.03           C  \\nATOM    725  O   ILE A  93       4.576  16.695  18.644  1.00 14.93           O  \\nATOM    726  CB  ILE A  93       7.790  17.499  18.485  1.00 14.96           C  \\nATOM    727  CG1 ILE A  93       8.506  18.830  18.293  1.00 15.41           C  \\nATOM    728  CG2 ILE A  93       7.369  16.936  17.091  1.00 14.61           C  \\nATOM    729  CD1 ILE A  93       7.654  19.940  17.637  1.00 20.05           C  \\nATOM    730  N   HIS A  94       5.806  15.471  20.049  1.00 15.23           N  \\nATOM    731  CA  HIS A  94       4.940  14.302  19.956  1.00 18.20           C  \\nATOM    732  C   HIS A  94       3.494  14.635  20.287  1.00 16.36           C  \\nATOM    733  O   HIS A  94       2.576  14.183  19.598  1.00 17.20           O  \\nATOM    734  CB  HIS A  94       5.436  13.194  20.900  1.00 23.57           C  \\nATOM    735  CG  HIS A  94       4.685  11.874  20.678  1.00 29.09           C  \\nATOM    736  ND1 HIS A  94       4.923  11.118  19.602  1.00 31.26           N  \\nATOM    737  CD2 HIS A  94       3.705  11.361  21.498  1.00 30.93           C  \\nATOM    738  CE1 HIS A  94       4.078  10.110  19.746  1.00 31.74           C  \\nATOM    739  NE2 HIS A  94       3.366  10.260  20.861  1.00 32.20           N  \\nATOM    740  N   GLN A  95       3.271  15.499  21.281  1.00 16.91           N  \\nATOM    741  CA  GLN A  95       1.916  15.878  21.673  1.00 18.66           C  \\nATOM    742  C   GLN A  95       1.226  16.736  20.625  1.00 16.97           C  \\nATOM    743  O   GLN A  95       0.012  16.562  20.443  1.00 15.36           O  \\nATOM    744  CB  GLN A  95       1.928  16.626  23.018  1.00 21.56           C  \\nATOM    745  CG  GLN A  95       2.457  15.766  24.180  1.00 29.81           C  \\nATOM    746  CD  GLN A  95       3.370  16.540  25.150  1.00 35.29           C  \\nATOM    747  OE1 GLN A  95       3.037  17.647  25.593  1.00 37.94           O  \\nATOM    748  NE2 GLN A  95       4.546  16.006  25.521  1.00 37.04           N  \\nATOM    749  N   TYR A  96       1.930  17.653  19.921  1.00 16.06           N  \\nATOM    750  CA  TYR A  96       1.276  18.417  18.826  1.00 16.07           C  \\nATOM    751  C   TYR A  96       0.963  17.504  17.658  1.00 14.44           C  \\nATOM    752  O   TYR A  96      -0.138  17.565  17.118  1.00 15.79           O  \\nATOM    753  CB  TYR A  96       2.139  19.554  18.264  1.00 17.94           C  \\nATOM    754  CG  TYR A  96       2.287  20.677  19.277  1.00 19.19           C  \\nATOM    755  CD1 TYR A  96       1.368  21.699  19.338  1.00 19.33           C  \\nATOM    756  CD2 TYR A  96       3.348  20.631  20.161  1.00 19.61           C  \\nATOM    757  CE1 TYR A  96       1.526  22.678  20.308  1.00 21.72           C  \\nATOM    758  CE2 TYR A  96       3.512  21.595  21.129  1.00 21.72           C  \\nATOM    759  CZ  TYR A  96       2.594  22.613  21.190  1.00 21.26           C  \\nATOM    760  OH  TYR A  96       2.776  23.582  22.153  1.00 24.09           O  \\nATOM    761  N   ARG A  97       1.904  16.635  17.295  1.00 13.30           N  \\nATOM    762  CA  ARG A  97       1.699  15.650  16.247  1.00 15.46           C  \\nATOM    763  C   ARG A  97       0.457  14.794  16.492  1.00 14.26           C  \\nATOM    764  O   ARG A  97      -0.420  14.676  15.641  1.00 14.10           O  \\nATOM    765  CB  ARG A  97       2.939  14.796  16.196  1.00 17.97           C  \\nATOM    766  CG  ARG A  97       2.886  13.541  15.357  1.00 23.35           C  \\nATOM    767  CD  ARG A  97       3.333  13.765  13.943  1.00 23.59           C  \\nATOM    768  NE  ARG A  97       3.314  12.471  13.314  1.00 24.08           N  \\nATOM    769  CZ  ARG A  97       3.268  12.320  12.007  1.00 25.74           C  \\nATOM    770  NH1 ARG A  97       3.135  11.073  11.548  1.00 26.76           N  \\nATOM    771  NH2 ARG A  97       3.353  13.380  11.181  1.00 26.95           N  \\nATOM    772  N   GLU A  98       0.332  14.218  17.678  1.00 13.69           N  \\nATOM    773  CA  GLU A  98      -0.807  13.361  17.941  1.00 16.64           C  \\nATOM    774  C   GLU A  98      -2.118  14.108  17.983  1.00 15.57           C  \\nATOM    775  O   GLU A  98      -3.157  13.573  17.607  1.00 15.85           O  \\nATOM    776  CB  GLU A  98      -0.593  12.623  19.260  1.00 21.96           C  \\nATOM    777  CG  GLU A  98       0.417  11.455  19.177  1.00 28.02           C  \\nATOM    778  CD  GLU A  98       0.147  10.460  18.038  1.00 31.04           C  \\nATOM    779  OE1 GLU A  98       1.118  10.123  17.373  1.00 33.58           O  \\nATOM    780  OE2 GLU A  98      -0.992  10.025  17.790  1.00 33.61           O  \\nATOM    781  N   GLN A  99      -2.105  15.362  18.446  1.00 14.66           N  \\nATOM    782  CA  GLN A  99      -3.343  16.104  18.468  1.00 13.22           C  \\nATOM    783  C   GLN A  99      -3.779  16.398  17.049  1.00 12.15           C  \\nATOM    784  O   GLN A  99      -4.974  16.274  16.791  1.00 14.09           O  \\nATOM    785  CB  GLN A  99      -3.158  17.403  19.250  1.00 16.29           C  \\nATOM    786  CG  GLN A  99      -4.513  18.118  19.407  1.00 19.67           C  \\nATOM    787  CD  GLN A  99      -4.517  19.240  20.438  1.00 22.19           C  \\nATOM    788  OE1 GLN A  99      -3.537  19.485  21.138  1.00 23.70           O  \\nATOM    789  NE2 GLN A  99      -5.628  19.961  20.537  1.00 22.72           N  \\nATOM    790  N   ILE A 100      -2.861  16.795  16.142  1.00 11.42           N  \\nATOM    791  CA  ILE A 100      -3.197  17.121  14.743  1.00 10.81           C  \\nATOM    792  C   ILE A 100      -3.761  15.893  14.008  1.00 12.39           C  \\nATOM    793  O   ILE A 100      -4.733  15.967  13.243  1.00 12.70           O  \\nATOM    794  CB  ILE A 100      -1.924  17.659  14.038  1.00 10.32           C  \\nATOM    795  CG1 ILE A 100      -1.642  19.075  14.518  1.00 10.37           C  \\nATOM    796  CG2 ILE A 100      -2.131  17.685  12.512  1.00 11.36           C  \\nATOM    797  CD1 ILE A 100      -0.184  19.481  14.392  1.00 10.69           C  \\nATOM    798  N   LYS A 101      -3.143  14.732  14.255  1.00 14.08           N  \\nATOM    799  CA  LYS A 101      -3.613  13.487  13.637  1.00 15.95           C  \\nATOM    800  C   LYS A 101      -5.021  13.150  14.079  1.00 15.72           C  \\nATOM    801  O   LYS A 101      -5.848  12.770  13.262  1.00 14.25           O  \\nATOM    802  CB  LYS A 101      -2.681  12.317  13.996  1.00 17.44           C  \\nATOM    803  CG  LYS A 101      -1.414  12.284  13.115  1.00 19.24           C  \\nATOM    804  CD  LYS A 101      -0.332  11.418  13.764  1.00 23.28           C  \\nATOM    805  CE  LYS A 101      -0.752  10.000  14.188  1.00 23.82           C  \\nATOM    806  NZ  LYS A 101       0.400   9.230  14.662  1.00 24.00           N  \\nATOM    807  N   ARG A 102      -5.278  13.341  15.377  1.00 16.72           N  \\nATOM    808  CA  ARG A 102      -6.586  13.098  15.962  1.00 18.74           C  \\nATOM    809  C   ARG A 102      -7.643  14.087  15.440  1.00 19.07           C  \\nATOM    810  O   ARG A 102      -8.712  13.628  15.002  1.00 20.67           O  \\nATOM    811  CB  ARG A 102      -6.457  13.197  17.462  1.00 20.62           C  \\nATOM    812  CG  ARG A 102      -7.539  12.409  18.132  1.00 26.89           C  \\nATOM    813  CD  ARG A 102      -7.610  12.887  19.585  1.00 30.60           C  \\nATOM    814  NE  ARG A 102      -8.197  14.224  19.645  1.00 35.83           N  \\nATOM    815  CZ  ARG A 102      -7.633  15.255  20.301  1.00 37.61           C  \\nATOM    816  NH1 ARG A 102      -8.243  16.455  20.267  1.00 38.37           N  \\nATOM    817  NH2 ARG A 102      -6.481  15.110  20.983  1.00 38.39           N  \\nATOM    818  N   VAL A 103      -7.418  15.407  15.395  1.00 15.59           N  \\nATOM    819  CA  VAL A 103      -8.452  16.326  14.881  1.00 18.48           C  \\nATOM    820  C   VAL A 103      -8.665  16.200  13.349  1.00 16.72           C  \\nATOM    821  O   VAL A 103      -9.776  16.383  12.848  1.00 15.90           O  \\nATOM    822  CB  VAL A 103      -8.131  17.840  15.215  1.00 20.10           C  \\nATOM    823  CG1 VAL A 103      -7.896  18.005  16.704  1.00 21.34           C  \\nATOM    824  CG2 VAL A 103      -6.899  18.302  14.512  1.00 23.93           C  \\nATOM    825  N   LYS A 104      -7.622  15.902  12.574  1.00 13.84           N  \\nATOM    826  CA  LYS A 104      -7.804  15.755  11.140  1.00 16.11           C  \\nATOM    827  C   LYS A 104      -8.158  14.329  10.716  1.00 16.70           C  \\nATOM    828  O   LYS A 104      -8.260  14.079   9.508  1.00 17.27           O  \\nATOM    829  CB  LYS A 104      -6.525  16.215  10.404  1.00 18.12           C  \\nATOM    830  CG  LYS A 104      -6.197  17.678  10.744  1.00 19.53           C  \\nATOM    831  CD  LYS A 104      -5.250  18.297   9.736  1.00 21.28           C  \\nATOM    832  CE  LYS A 104      -5.921  18.590   8.390  1.00 20.75           C  \\nATOM    833  NZ  LYS A 104      -6.875  19.673   8.462  1.00 19.79           N  \\nATOM    834  N   ASP A 105      -8.372  13.390  11.669  1.00 15.91           N  \\nATOM    835  CA  ASP A 105      -8.666  11.988  11.359  1.00 15.43           C  \\nATOM    836  C   ASP A 105      -7.705  11.372  10.333  1.00 16.04           C  \\nATOM    837  O   ASP A 105      -8.106  10.654   9.415  1.00 15.38           O  \\nATOM    838  CB  ASP A 105     -10.077  11.847  10.810  1.00 18.74           C  \\nATOM    839  CG  ASP A 105     -11.261  12.121  11.696  1.00 21.15           C  \\nATOM    840  OD1 ASP A 105     -11.183  11.972  12.918  1.00 21.36           O  \\nATOM    841  OD2 ASP A 105     -12.281  12.480  11.113  1.00 22.63           O  \\nATOM    842  N   SER A 106      -6.397  11.591  10.469  1.00 15.54           N  \\nATOM    843  CA  SER A 106      -5.470  11.174   9.442  1.00 17.25           C  \\nATOM    844  C   SER A 106      -4.112  10.882  10.011  1.00 17.31           C  \\nATOM    845  O   SER A 106      -3.650  11.556  10.916  1.00 19.11           O  \\nATOM    846  CB  SER A 106      -5.463  12.314   8.398  1.00 19.27           C  \\nATOM    847  OG  SER A 106      -4.326  12.490   7.573  1.00 24.20           O  \\nATOM    848  N   ASP A 107      -3.454   9.891   9.453  1.00 17.29           N  \\nATOM    849  CA  ASP A 107      -2.109   9.557   9.868  1.00 19.80           C  \\nATOM    850  C   ASP A 107      -1.000  10.220   9.063  1.00 20.36           C  \\nATOM    851  O   ASP A 107       0.197  10.118   9.345  1.00 21.26           O  \\nATOM    852  CB  ASP A 107      -1.970   8.045   9.810  1.00 23.15           C  \\nATOM    853  CG  ASP A 107      -2.863   7.310  10.807  1.00 25.12           C  \\nATOM    854  OD1 ASP A 107      -3.262   6.206  10.462  1.00 29.21           O  \\nATOM    855  OD2 ASP A 107      -3.127   7.812  11.903  1.00 26.59           O  \\nATOM    856  N   ASP A 108      -1.383  10.909   8.005  1.00 20.57           N  \\nATOM    857  CA  ASP A 108      -0.430  11.664   7.216  1.00 22.38           C  \\nATOM    858  C   ASP A 108      -1.061  12.982   6.842  1.00 21.37           C  \\nATOM    859  O   ASP A 108      -1.973  13.096   6.030  1.00 24.36           O  \\nATOM    860  CB  ASP A 108      -0.056  10.900   5.967  1.00 27.28           C  \\nATOM    861  CG  ASP A 108      -1.263  10.278   5.320  1.00 32.24           C  \\nATOM    862  OD1 ASP A 108      -1.714   9.272   5.877  1.00 37.07           O  \\nATOM    863  OD2 ASP A 108      -1.747  10.790   4.301  1.00 36.33           O  \\nATOM    864  N   VAL A 109      -0.598  13.970   7.572  1.00 19.33           N  \\nATOM    865  CA  VAL A 109      -1.009  15.354   7.400  1.00 17.49           C  \\nATOM    866  C   VAL A 109       0.215  16.086   6.819  1.00 14.55           C  \\nATOM    867  O   VAL A 109       1.335  15.772   7.245  1.00 13.64           O  \\nATOM    868  CB  VAL A 109      -1.410  15.870   8.784  1.00 18.40           C  \\nATOM    869  CG1 VAL A 109      -1.645  17.366   8.765  1.00 20.21           C  \\nATOM    870  CG2 VAL A 109      -2.676  15.101   9.214  1.00 20.27           C  \\nATOM    871  N   PRO A 110       0.141  17.001   5.825  1.00 12.41           N  \\nATOM    872  CA  PRO A 110       1.297  17.783   5.384  1.00 11.52           C  \\nATOM    873  C   PRO A 110       1.920  18.480   6.605  1.00 11.79           C  \\nATOM    874  O   PRO A 110       1.289  19.245   7.324  1.00  9.55           O  \\nATOM    875  CB  PRO A 110       0.728  18.721   4.370  1.00 11.31           C  \\nATOM    876  CG  PRO A 110      -0.410  17.926   3.779  1.00 11.38           C  \\nATOM    877  CD  PRO A 110      -1.044  17.309   5.020  1.00 12.05           C  \\nATOM    878  N   MET A 111       3.192  18.201   6.874  1.00 11.92           N  \\nATOM    879  CA  MET A 111       3.878  18.724   8.050  1.00 11.49           C  \\nATOM    880  C   MET A 111       5.379  18.749   7.807  1.00 11.68           C  \\nATOM    881  O   MET A 111       5.903  17.900   7.098  1.00 11.11           O  \\nATOM    882  CB  MET A 111       3.494  17.806   9.211  1.00 14.36           C  \\nATOM    883  CG  MET A 111       3.864  18.110  10.627  1.00 17.41           C  \\nATOM    884  SD  MET A 111       3.231  16.825  11.758  1.00 18.36           S  \\nATOM    885  CE  MET A 111       1.528  17.128  11.498  1.00 17.25           C  \\nATOM    886  N   VAL A 112       6.091  19.706   8.404  1.00 10.73           N  \\nATOM    887  CA  VAL A 112       7.545  19.763   8.348  1.00 10.18           C  \\nATOM    888  C   VAL A 112       7.992  19.875   9.794  1.00  9.11           C  \\nATOM    889  O   VAL A 112       7.404  20.633  10.556  1.00 10.77           O  \\nATOM    890  CB  VAL A 112       8.010  21.024   7.542  1.00 11.80           C  \\nATOM    891  CG1 VAL A 112       9.528  21.213   7.558  1.00 13.24           C  \\nATOM    892  CG2 VAL A 112       7.542  20.843   6.104  1.00 14.56           C  \\nATOM    893  N   LEU A 113       9.032  19.167  10.207  1.00  8.47           N  \\nATOM    894  CA  LEU A 113       9.632  19.332  11.535  1.00  9.53           C  \\nATOM    895  C   LEU A 113      10.738  20.384  11.412  1.00  9.33           C  \\nATOM    896  O   LEU A 113      11.621  20.270  10.555  1.00  9.95           O  \\nATOM    897  CB  LEU A 113      10.214  17.983  12.017  1.00  9.56           C  \\nATOM    898  CG  LEU A 113      10.863  17.976  13.381  1.00 11.58           C  \\nATOM    899  CD1 LEU A 113       9.908  18.379  14.503  1.00 13.17           C  \\nATOM    900  CD2 LEU A 113      11.361  16.564  13.597  1.00 12.84           C  \\nATOM    901  N   VAL A 114      10.705  21.442  12.221  1.00  7.62           N  \\nATOM    902  CA  VAL A 114      11.673  22.541  12.109  1.00  7.64           C  \\nATOM    903  C   VAL A 114      12.436  22.667  13.441  1.00  7.65           C  \\nATOM    904  O   VAL A 114      11.872  22.842  14.541  1.00  9.71           O  \\nATOM    905  CB  VAL A 114      10.896  23.836  11.774  1.00  9.37           C  \\nATOM    906  CG1 VAL A 114      11.814  25.030  11.869  1.00 10.27           C  \\nATOM    907  CG2 VAL A 114      10.254  23.719  10.393  1.00  9.64           C  \\nATOM    908  N   GLY A 115      13.745  22.567  13.331  1.00  6.10           N  \\nATOM    909  CA  GLY A 115      14.603  22.742  14.476  1.00  6.69           C  \\nATOM    910  C   GLY A 115      15.120  24.163  14.407  1.00  8.47           C  \\nATOM    911  O   GLY A 115      16.007  24.439  13.594  1.00  9.16           O  \\nATOM    912  N   ASN A 116      14.593  25.089  15.192  1.00  8.01           N  \\nATOM    913  CA  ASN A 116      15.007  26.475  15.111  1.00  8.24           C  \\nATOM    914  C   ASN A 116      16.112  26.873  16.109  1.00  9.61           C  \\nATOM    915  O   ASN A 116      16.453  26.142  17.051  1.00  9.84           O  \\nATOM    916  CB  ASN A 116      13.721  27.307  15.275  1.00  8.01           C  \\nATOM    917  CG  ASN A 116      13.929  28.788  14.992  1.00  9.48           C  \\nATOM    918  OD1 ASN A 116      14.470  29.205  13.972  1.00  9.94           O  \\nATOM    919  ND2 ASN A 116      13.565  29.644  15.940  1.00 10.75           N  \\nATOM    920  N   LYS A 117      16.660  28.087  15.876  1.00 10.22           N  \\nATOM    921  CA  LYS A 117      17.735  28.715  16.674  1.00 12.36           C  \\nATOM    922  C   LYS A 117      19.094  28.021  16.450  1.00 13.39           C  \\nATOM    923  O   LYS A 117      19.977  27.940  17.324  1.00 11.29           O  \\nATOM    924  CB  LYS A 117      17.391  28.706  18.214  1.00 13.23           C  \\nATOM    925  CG  LYS A 117      16.026  29.272  18.611  1.00 12.90           C  \\nATOM    926  CD  LYS A 117      15.932  29.640  20.097  1.00 12.43           C  \\nATOM    927  CE  LYS A 117      14.551  30.217  20.380  1.00 10.55           C  \\nATOM    928  NZ  LYS A 117      14.491  30.647  21.772  1.00 11.10           N  \\nATOM    929  N   CYS A 118      19.254  27.555  15.188  1.00 13.69           N  \\nATOM    930  CA  CYS A 118      20.449  26.806  14.844  1.00 15.67           C  \\nATOM    931  C   CYS A 118      21.716  27.658  14.796  1.00 15.28           C  \\nATOM    932  O   CYS A 118      22.808  27.113  14.629  1.00 15.30           O  \\nATOM    933  CB  CYS A 118      20.246  26.064  13.492  1.00 17.74           C  \\nATOM    934  SG  CYS A 118      20.353  27.009  11.944  1.00 20.74           S  \\nATOM    935  N   ASP A 119      21.578  28.995  14.931  1.00 15.76           N  \\nATOM    936  CA  ASP A 119      22.722  29.904  15.083  1.00 16.09           C  \\nATOM    937  C   ASP A 119      23.426  29.778  16.445  1.00 17.19           C  \\nATOM    938  O   ASP A 119      24.589  30.178  16.553  1.00 18.76           O  \\nATOM    939  CB  ASP A 119      22.268  31.374  14.888  1.00 15.14           C  \\nATOM    940  CG  ASP A 119      21.153  31.785  15.835  1.00 15.81           C  \\nATOM    941  OD1 ASP A 119      20.030  31.294  15.662  1.00 13.87           O  \\nATOM    942  OD2 ASP A 119      21.407  32.596  16.721  1.00 14.85           O  \\nATOM    943  N   LEU A 120      22.732  29.248  17.480  1.00 16.86           N  \\nATOM    944  CA  LEU A 120      23.230  29.099  18.845  1.00 18.76           C  \\nATOM    945  C   LEU A 120      24.217  27.957  18.979  1.00 19.70           C  \\nATOM    946  O   LEU A 120      23.979  26.885  18.431  1.00 20.53           O  \\nATOM    947  CB  LEU A 120      22.103  28.829  19.831  1.00 17.65           C  \\nATOM    948  CG  LEU A 120      21.065  29.933  20.010  1.00 19.43           C  \\nATOM    949  CD1 LEU A 120      20.044  29.418  21.008  1.00 18.46           C  \\nATOM    950  CD2 LEU A 120      21.688  31.239  20.503  1.00 18.73           C  \\nATOM    951  N   ALA A 121      25.313  28.184  19.705  1.00 19.26           N  \\nATOM    952  CA  ALA A 121      26.303  27.143  19.887  1.00 20.48           C  \\nATOM    953  C   ALA A 121      25.942  26.210  21.036  1.00 21.30           C  \\nATOM    954  O   ALA A 121      26.403  25.069  21.067  1.00 24.04           O  \\nATOM    955  CB  ALA A 121      27.652  27.767  20.164  1.00 22.02           C  \\nATOM    956  N   ALA A 122      25.109  26.635  21.982  1.00 22.09           N  \\nATOM    957  CA  ALA A 122      24.683  25.797  23.094  1.00 23.88           C  \\nATOM    958  C   ALA A 122      23.578  24.811  22.720  1.00 23.77           C  \\nATOM    959  O   ALA A 122      22.492  24.835  23.301  1.00 27.82           O  \\nATOM    960  CB  ALA A 122      24.171  26.684  24.249  1.00 25.67           C  \\nATOM    961  N   ARG A 123      23.843  23.908  21.783  1.00 20.99           N  \\nATOM    962  CA  ARG A 123      22.890  22.905  21.311  1.00 19.81           C  \\nATOM    963  C   ARG A 123      22.619  21.761  22.283  1.00 19.56           C  \\nATOM    964  O   ARG A 123      23.590  21.176  22.774  1.00 20.01           O  \\nATOM    965  CB  ARG A 123      23.421  22.333  19.988  1.00 19.81           C  \\nATOM    966  CG  ARG A 123      22.662  21.166  19.398  1.00 21.43           C  \\nATOM    967  CD  ARG A 123      23.273  20.680  18.073  1.00 22.66           C  \\nATOM    968  NE  ARG A 123      23.050  21.590  16.956  1.00 22.61           N  \\nATOM    969  CZ  ARG A 123      22.019  21.487  16.093  1.00 20.95           C  \\nATOM    970  NH1 ARG A 123      21.045  20.568  16.181  1.00 16.36           N  \\nATOM    971  NH2 ARG A 123      21.989  22.370  15.098  1.00 19.65           N  \\nATOM    972  N   THR A 124      21.364  21.389  22.600  1.00 17.96           N  \\nATOM    973  CA  THR A 124      21.118  20.200  23.415  1.00 17.85           C  \\nATOM    974  C   THR A 124      20.413  19.068  22.656  1.00 17.08           C  \\nATOM    975  O   THR A 124      20.459  17.926  23.091  1.00 19.59           O  \\nATOM    976  CB  THR A 124      20.311  20.578  24.672  1.00 19.42           C  \\nATOM    977  OG1 THR A 124      19.109  21.230  24.277  1.00 20.50           O  \\nATOM    978  CG2 THR A 124      21.155  21.458  25.588  1.00 19.06           C  \\nATOM    979  N   VAL A 125      19.788  19.319  21.495  1.00 16.59           N  \\nATOM    980  CA  VAL A 125      19.133  18.324  20.647  1.00 16.09           C  \\nATOM    981  C   VAL A 125      20.032  18.080  19.413  1.00 16.61           C  \\nATOM    982  O   VAL A 125      20.180  18.980  18.573  1.00 14.07           O  \\nATOM    983  CB  VAL A 125      17.720  18.848  20.179  1.00 15.28           C  \\nATOM    984  CG1 VAL A 125      17.097  17.814  19.265  1.00 14.81           C  \\nATOM    985  CG2 VAL A 125      16.824  19.168  21.366  1.00 15.97           C  \\nATOM    986  N   GLU A 126      20.659  16.889  19.280  1.00 17.05           N  \\nATOM    987  CA  GLU A 126      21.487  16.568  18.124  1.00 19.28           C  \\nATOM    988  C   GLU A 126      20.631  16.496  16.876  1.00 18.51           C  \\nATOM    989  O   GLU A 126      19.511  15.976  16.880  1.00 17.68           O  \\nATOM    990  CB  GLU A 126      22.165  15.215  18.219  1.00 23.29           C  \\nATOM    991  CG  GLU A 126      23.244  15.057  19.265  1.00 30.86           C  \\nATOM    992  CD  GLU A 126      24.402  16.054  19.153  1.00 36.59           C  \\nATOM    993  OE1 GLU A 126      25.139  16.012  18.161  1.00 39.64           O  \\nATOM    994  OE2 GLU A 126      24.581  16.885  20.056  1.00 40.44           O  \\nATOM    995  N   SER A 127      21.167  16.985  15.755  1.00 17.73           N  \\nATOM    996  CA  SER A 127      20.426  16.938  14.493  1.00 16.72           C  \\nATOM    997  C   SER A 127      20.040  15.506  14.158  1.00 16.76           C  \\nATOM    998  O   SER A 127      18.931  15.245  13.690  1.00 17.01           O  \\nATOM    999  CB  SER A 127      21.283  17.518  13.371  1.00 17.67           C  \\nATOM   1000  OG  SER A 127      21.730  18.826  13.768  1.00 19.67           O  \\nATOM   1001  N   ARG A 128      20.916  14.558  14.466  1.00 16.72           N  \\nATOM   1002  CA  ARG A 128      20.667  13.154  14.145  1.00 20.94           C  \\nATOM   1003  C   ARG A 128      19.435  12.617  14.862  1.00 19.62           C  \\nATOM   1004  O   ARG A 128      18.641  11.882  14.278  1.00 18.81           O  \\nATOM   1005  CB  ARG A 128      21.910  12.319  14.510  1.00 24.31           C  \\nATOM   1006  CG  ARG A 128      21.969  11.035  13.692  1.00 30.88           C  \\nATOM   1007  CD  ARG A 128      23.381  10.403  13.536  1.00 35.88           C  \\nATOM   1008  NE  ARG A 128      24.384  10.996  12.621  1.00 39.57           N  \\nATOM   1009  CZ  ARG A 128      24.313  11.018  11.267  1.00 40.85           C  \\nATOM   1010  NH1 ARG A 128      25.399  11.388  10.577  1.00 40.83           N  \\nATOM   1011  NH2 ARG A 128      23.202  10.685  10.576  1.00 41.33           N  \\nATOM   1012  N   GLN A 129      19.209  13.016  16.119  1.00 18.27           N  \\nATOM   1013  CA  GLN A 129      18.032  12.519  16.787  1.00 18.53           C  \\nATOM   1014  C   GLN A 129      16.735  13.151  16.224  1.00 17.11           C  \\nATOM   1015  O   GLN A 129      15.691  12.486  16.098  1.00 15.48           O  \\nATOM   1016  CB  GLN A 129      18.229  12.766  18.306  1.00 22.36           C  \\nATOM   1017  CG  GLN A 129      18.041  14.133  18.911  1.00 24.78           C  \\nATOM   1018  CD  GLN A 129      18.283  14.123  20.411  1.00 27.48           C  \\nATOM   1019  OE1 GLN A 129      19.287  14.643  20.906  1.00 28.14           O  \\nATOM   1020  NE2 GLN A 129      17.367  13.497  21.152  1.00 27.21           N  \\nATOM   1021  N   ALA A 130      16.780  14.446  15.837  1.00 14.73           N  \\nATOM   1022  CA  ALA A 130      15.611  15.044  15.210  1.00 13.87           C  \\nATOM   1023  C   ALA A 130      15.342  14.449  13.826  1.00 13.30           C  \\nATOM   1024  O   ALA A 130      14.199  14.194  13.436  1.00 11.42           O  \\nATOM   1025  CB  ALA A 130      15.822  16.536  15.063  1.00 16.21           C  \\nATOM   1026  N   GLN A 131      16.401  14.189  13.057  1.00 14.61           N  \\nATOM   1027  CA  GLN A 131      16.267  13.529  11.739  1.00 16.01           C  \\nATOM   1028  C   GLN A 131      15.637  12.109  11.821  1.00 16.29           C  \\nATOM   1029  O   GLN A 131      14.777  11.708  11.017  1.00 14.37           O  \\nATOM   1030  CB  GLN A 131      17.673  13.507  11.116  1.00 18.12           C  \\nATOM   1031  CG  GLN A 131      17.702  13.017   9.689  1.00 22.19           C  \\nATOM   1032  CD  GLN A 131      16.889  13.922   8.801  1.00 21.61           C  \\nATOM   1033  OE1 GLN A 131      17.292  15.044   8.529  1.00 24.06           O  \\nATOM   1034  NE2 GLN A 131      15.714  13.485   8.370  1.00 23.37           N  \\nATOM   1035  N   ASP A 132      16.038  11.348  12.853  1.00 16.64           N  \\nATOM   1036  CA  ASP A 132      15.467  10.038  13.173  1.00 18.00           C  \\nATOM   1037  C   ASP A 132      13.984  10.175  13.484  1.00 17.28           C  \\nATOM   1038  O   ASP A 132      13.178   9.395  12.985  1.00 16.45           O  \\nATOM   1039  CB  ASP A 132      16.120   9.415  14.405  1.00 20.10           C  \\nATOM   1040  CG  ASP A 132      17.506   8.816  14.234  1.00 24.07           C  \\nATOM   1041  OD1 ASP A 132      18.120   8.459  15.243  1.00 28.35           O  \\nATOM   1042  OD2 ASP A 132      17.975   8.686  13.106  1.00 25.43           O  \\nATOM   1043  N   LEU A 133      13.614  11.168  14.312  1.00 15.58           N  \\nATOM   1044  CA  LEU A 133      12.213  11.390  14.624  1.00 16.28           C  \\nATOM   1045  C   LEU A 133      11.421  11.729  13.366  1.00 14.96           C  \\nATOM   1046  O   LEU A 133      10.345  11.181  13.128  1.00 15.09           O  \\nATOM   1047  CB  LEU A 133      12.052  12.536  15.647  1.00 17.06           C  \\nATOM   1048  CG  LEU A 133      10.617  12.781  16.136  1.00 18.79           C  \\nATOM   1049  CD1 LEU A 133      10.142  11.554  16.881  1.00 19.96           C  \\nATOM   1050  CD2 LEU A 133      10.544  14.013  17.020  1.00 18.45           C  \\nATOM   1051  N   ALA A 134      11.937  12.642  12.545  1.00 13.16           N  \\nATOM   1052  CA  ALA A 134      11.241  13.016  11.331  1.00 14.89           C  \\nATOM   1053  C   ALA A 134      11.092  11.821  10.414  1.00 16.19           C  \\nATOM   1054  O   ALA A 134      10.015  11.643   9.857  1.00 15.70           O  \\nATOM   1055  CB  ALA A 134      12.021  14.089  10.629  1.00 15.12           C  \\nATOM   1056  N   ARG A 135      12.143  10.995  10.279  1.00 17.00           N  \\nATOM   1057  CA  ARG A 135      12.088   9.828   9.438  1.00 19.17           C  \\nATOM   1058  C   ARG A 135      10.990   8.893   9.916  1.00 18.70           C  \\nATOM   1059  O   ARG A 135      10.269   8.352   9.078  1.00 20.29           O  \\nATOM   1060  CB  ARG A 135      13.425   9.091   9.456  1.00 22.48           C  \\nATOM   1061  CG  ARG A 135      13.499   8.297   8.155  1.00 29.02           C  \\nATOM   1062  CD  ARG A 135      14.619   7.270   7.988  1.00 33.22           C  \\nATOM   1063  NE  ARG A 135      14.117   5.888   8.050  1.00 38.80           N  \\nATOM   1064  CZ  ARG A 135      13.281   5.302   7.148  1.00 39.60           C  \\nATOM   1065  NH1 ARG A 135      12.889   4.044   7.384  1.00 40.80           N  \\nATOM   1066  NH2 ARG A 135      12.827   5.913   6.034  1.00 40.11           N  \\nATOM   1067  N   SER A 136      10.815   8.697  11.242  1.00 19.23           N  \\nATOM   1068  CA  SER A 136       9.757   7.841  11.770  1.00 19.12           C  \\nATOM   1069  C   SER A 136       8.374   8.432  11.528  1.00 19.61           C  \\nATOM   1070  O   SER A 136       7.398   7.690  11.522  1.00 21.01           O  \\nATOM   1071  CB  SER A 136       9.930   7.624  13.280  1.00 19.94           C  \\nATOM   1072  OG  SER A 136       9.459   8.686  14.119  1.00 23.21           O  \\nATOM   1073  N   TYR A 137       8.231   9.760  11.380  1.00 17.33           N  \\nATOM   1074  CA  TYR A 137       6.950  10.375  11.078  1.00 18.12           C  \\nATOM   1075  C   TYR A 137       6.683  10.404   9.570  1.00 18.22           C  \\nATOM   1076  O   TYR A 137       5.561  10.656   9.129  1.00 19.27           O  \\nATOM   1077  CB  TYR A 137       6.907  11.812  11.592  1.00 18.21           C  \\nATOM   1078  CG  TYR A 137       6.883  12.057  13.084  1.00 19.08           C  \\nATOM   1079  CD1 TYR A 137       6.619  11.026  13.955  1.00 20.51           C  \\nATOM   1080  CD2 TYR A 137       7.059  13.355  13.548  1.00 20.43           C  \\nATOM   1081  CE1 TYR A 137       6.510  11.285  15.306  1.00 21.89           C  \\nATOM   1082  CE2 TYR A 137       6.945  13.624  14.899  1.00 21.02           C  \\nATOM   1083  CZ  TYR A 137       6.662  12.574  15.763  1.00 22.13           C  \\nATOM   1084  OH  TYR A 137       6.476  12.782  17.112  1.00 22.16           O  \\nATOM   1085  N   GLY A 138       7.689  10.186   8.737  1.00 16.07           N  \\nATOM   1086  CA  GLY A 138       7.520  10.265   7.294  1.00 17.66           C  \\nATOM   1087  C   GLY A 138       7.476  11.725   6.810  1.00 18.01           C  \\nATOM   1088  O   GLY A 138       6.859  12.010   5.774  1.00 17.85           O  \\nATOM   1089  N   ILE A 139       8.128  12.662   7.540  1.00 14.16           N  \\nATOM   1090  CA  ILE A 139       8.117  14.065   7.145  1.00 13.49           C  \\nATOM   1091  C   ILE A 139       9.526  14.662   7.033  1.00 14.35           C  \\nATOM   1092  O   ILE A 139      10.494  14.152   7.610  1.00 16.19           O  \\nATOM   1093  CB  ILE A 139       7.292  14.896   8.161  1.00 14.40           C  \\nATOM   1094  CG1 ILE A 139       7.925  14.811   9.554  1.00 15.44           C  \\nATOM   1095  CG2 ILE A 139       5.821  14.422   8.104  1.00 14.07           C  \\nATOM   1096  CD1 ILE A 139       7.288  15.765  10.587  1.00 15.35           C  \\nATOM   1097  N   PRO A 140       9.706  15.767   6.308  1.00 13.55           N  \\nATOM   1098  CA  PRO A 140      10.984  16.448   6.205  1.00 13.39           C  \\nATOM   1099  C   PRO A 140      11.424  17.065   7.532  1.00 12.74           C  \\nATOM   1100  O   PRO A 140      10.601  17.530   8.310  1.00 11.42           O  \\nATOM   1101  CB  PRO A 140      10.767  17.498   5.127  1.00 13.47           C  \\nATOM   1102  CG  PRO A 140       9.514  17.084   4.410  1.00 14.09           C  \\nATOM   1103  CD  PRO A 140       8.686  16.467   5.519  1.00 13.14           C  \\nATOM   1104  N   TYR A 141      12.744  17.086   7.756  1.00 11.68           N  \\nATOM   1105  CA  TYR A 141      13.336  17.791   8.890  1.00 10.89           C  \\nATOM   1106  C   TYR A 141      14.175  18.945   8.326  1.00 10.09           C  \\nATOM   1107  O   TYR A 141      15.056  18.720   7.468  1.00 10.27           O  \\nATOM   1108  CB  TYR A 141      14.276  16.897   9.691  1.00 12.54           C  \\nATOM   1109  CG  TYR A 141      15.063  17.676  10.733  1.00 10.92           C  \\nATOM   1110  CD1 TYR A 141      14.350  18.396  11.664  1.00 11.75           C  \\nATOM   1111  CD2 TYR A 141      16.453  17.633  10.784  1.00 12.49           C  \\nATOM   1112  CE1 TYR A 141      14.997  19.074  12.670  1.00 12.85           C  \\nATOM   1113  CE2 TYR A 141      17.112  18.307  11.794  1.00 11.33           C  \\nATOM   1114  CZ  TYR A 141      16.369  19.016  12.728  1.00 12.61           C  \\nATOM   1115  OH  TYR A 141      16.944  19.646  13.813  1.00 13.82           O  \\nATOM   1116  N   ILE A 142      13.892  20.174   8.799  1.00  9.35           N  \\nATOM   1117  CA  ILE A 142      14.636  21.355   8.376  1.00  9.05           C  \\nATOM   1118  C   ILE A 142      15.153  22.163   9.551  1.00  9.25           C  \\nATOM   1119  O   ILE A 142      14.409  22.382  10.499  1.00 10.43           O  \\nATOM   1120  CB  ILE A 142      13.756  22.259   7.472  1.00  9.25           C  \\nATOM   1121  CG1 ILE A 142      13.523  21.470   6.180  1.00 10.51           C  \\nATOM   1122  CG2 ILE A 142      14.450  23.586   7.122  1.00  9.08           C  \\nATOM   1123  CD1 ILE A 142      12.719  22.127   5.028  1.00 13.16           C  \\nATOM   1124  N   GLU A 143      16.457  22.513   9.568  1.00  9.01           N  \\nATOM   1125  CA  GLU A 143      16.979  23.404  10.611  1.00  8.52           C  \\nATOM   1126  C   GLU A 143      16.978  24.864  10.193  1.00 10.31           C  \\nATOM   1127  O   GLU A 143      17.278  25.228   9.040  1.00 10.45           O  \\nATOM   1128  CB  GLU A 143      18.397  23.022  11.013  1.00 10.87           C  \\nATOM   1129  CG  GLU A 143      18.363  21.728  11.801  1.00 10.22           C  \\nATOM   1130  CD  GLU A 143      19.592  21.411  12.633  1.00 13.01           C  \\nATOM   1131  OE1 GLU A 143      20.616  22.082  12.521  1.00 13.65           O  \\nATOM   1132  OE2 GLU A 143      19.511  20.487  13.429  1.00 12.70           O  \\nATOM   1133  N   THR A 144      16.608  25.762  11.117  1.00  9.18           N  \\nATOM   1134  CA  THR A 144      16.422  27.169  10.754  1.00  9.71           C  \\nATOM   1135  C   THR A 144      17.014  28.076  11.789  1.00  8.66           C  \\nATOM   1136  O   THR A 144      17.259  27.698  12.927  1.00  9.13           O  \\nATOM   1137  CB  THR A 144      14.919  27.617  10.635  1.00  9.90           C  \\nATOM   1138  OG1 THR A 144      14.336  27.403  11.931  1.00  9.29           O  \\nATOM   1139  CG2 THR A 144      14.120  26.858   9.557  1.00  9.89           C  \\nATOM   1140  N   SER A 145      17.244  29.288  11.360  1.00 10.91           N  \\nATOM   1141  CA  SER A 145      17.558  30.377  12.290  1.00 12.65           C  \\nATOM   1142  C   SER A 145      16.758  31.602  11.849  1.00 10.37           C  \\nATOM   1143  O   SER A 145      16.930  32.094  10.727  1.00 11.76           O  \\nATOM   1144  CB  SER A 145      19.069  30.768  12.308  1.00 11.79           C  \\nATOM   1145  OG  SER A 145      19.248  31.931  13.125  1.00 12.12           O  \\nATOM   1146  N   ALA A 146      15.838  32.046  12.696  1.00  9.49           N  \\nATOM   1147  CA  ALA A 146      15.131  33.268  12.413  1.00 10.90           C  \\nATOM   1148  C   ALA A 146      16.070  34.455  12.553  1.00 13.53           C  \\nATOM   1149  O   ALA A 146      15.799  35.514  11.995  1.00 16.20           O  \\nATOM   1150  CB  ALA A 146      13.976  33.436  13.367  1.00 12.65           C  \\nATOM   1151  N   LYS A 147      17.183  34.302  13.283  1.00 14.35           N  \\nATOM   1152  CA  LYS A 147      18.150  35.351  13.486  1.00 16.25           C  \\nATOM   1153  C   LYS A 147      18.953  35.568  12.215  1.00 17.58           C  \\nATOM   1154  O   LYS A 147      19.124  36.724  11.825  1.00 20.13           O  \\nATOM   1155  CB  LYS A 147      19.068  34.970  14.617  1.00 17.55           C  \\nATOM   1156  CG  LYS A 147      20.003  36.125  14.958  1.00 22.83           C  \\nATOM   1157  CD  LYS A 147      20.581  35.932  16.341  1.00 26.44           C  \\nATOM   1158  CE  LYS A 147      21.683  36.949  16.507  1.00 30.79           C  \\nATOM   1159  NZ  LYS A 147      22.575  36.534  17.574  1.00 33.81           N  \\nATOM   1160  N   THR A 148      19.479  34.529  11.541  1.00 17.16           N  \\nATOM   1161  CA  THR A 148      20.248  34.735  10.300  1.00 15.93           C  \\nATOM   1162  C   THR A 148      19.422  34.607   9.013  1.00 17.72           C  \\nATOM   1163  O   THR A 148      19.926  34.962   7.945  1.00 18.47           O  \\nATOM   1164  CB  THR A 148      21.444  33.745  10.209  1.00 15.86           C  \\nATOM   1165  OG1 THR A 148      20.886  32.435  10.042  1.00 14.56           O  \\nATOM   1166  CG2 THR A 148      22.377  33.839  11.439  1.00 15.25           C  \\nATOM   1167  N   ARG A 149      18.176  34.108   9.100  1.00 15.18           N  \\nATOM   1168  CA  ARG A 149      17.268  33.815   7.994  1.00 19.20           C  \\nATOM   1169  C   ARG A 149      17.652  32.483   7.318  1.00 17.94           C  \\nATOM   1170  O   ARG A 149      17.031  32.079   6.332  1.00 20.28           O  \\nATOM   1171  CB  ARG A 149      17.251  34.964   6.937  1.00 21.40           C  \\nATOM   1172  CG  ARG A 149      15.929  35.106   6.188  1.00 27.00           C  \\nATOM   1173  CD  ARG A 149      15.946  35.904   4.863  1.00 27.99           C  \\nATOM   1174  NE  ARG A 149      14.707  35.687   4.092  1.00 29.40           N  \\nATOM   1175  CZ  ARG A 149      13.550  36.232   4.492  1.00 29.07           C  \\nATOM   1176  NH1 ARG A 149      12.412  36.015   3.828  1.00 29.56           N  \\nATOM   1177  NH2 ARG A 149      13.527  37.004   5.577  1.00 28.99           N  \\nATOM   1178  N   GLN A 150      18.621  31.701   7.838  1.00 15.80           N  \\nATOM   1179  CA  GLN A 150      18.926  30.402   7.273  1.00 16.36           C  \\nATOM   1180  C   GLN A 150      17.737  29.464   7.455  1.00 15.02           C  \\nATOM   1181  O   GLN A 150      17.119  29.351   8.504  1.00 13.57           O  \\nATOM   1182  CB  GLN A 150      20.152  29.729   7.937  1.00 23.05           C  \\nATOM   1183  CG  GLN A 150      20.578  28.448   7.162  1.00 29.37           C  \\nATOM   1184  CD  GLN A 150      21.604  27.530   7.827  1.00 34.84           C  \\nATOM   1185  OE1 GLN A 150      21.301  26.416   8.280  1.00 38.95           O  \\nATOM   1186  NE2 GLN A 150      22.857  27.967   7.883  1.00 35.22           N  \\nATOM   1187  N   GLY A 151      17.412  28.809   6.348  1.00 13.62           N  \\nATOM   1188  CA  GLY A 151      16.357  27.835   6.296  1.00 13.50           C  \\nATOM   1189  C   GLY A 151      14.943  28.395   6.329  1.00 12.36           C  \\nATOM   1190  O   GLY A 151      14.025  27.591   6.148  1.00 11.07           O  \\nATOM   1191  N   VAL A 152      14.706  29.702   6.510  1.00 12.59           N  \\nATOM   1192  CA  VAL A 152      13.339  30.183   6.715  1.00 14.23           C  \\nATOM   1193  C   VAL A 152      12.435  29.971   5.475  1.00 13.41           C  \\nATOM   1194  O   VAL A 152      11.352  29.362   5.577  1.00 12.94           O  \\nATOM   1195  CB  VAL A 152      13.430  31.691   7.167  1.00 12.57           C  \\nATOM   1196  CG1 VAL A 152      12.059  32.316   7.194  1.00 13.14           C  \\nATOM   1197  CG2 VAL A 152      14.044  31.787   8.543  1.00 12.69           C  \\nATOM   1198  N   GLU A 153      12.831  30.410   4.278  1.00 13.52           N  \\nATOM   1199  CA  GLU A 153      12.001  30.213   3.086  1.00 14.81           C  \\nATOM   1200  C   GLU A 153      11.857  28.756   2.742  1.00 11.88           C  \\nATOM   1201  O   GLU A 153      10.797  28.282   2.344  1.00 13.92           O  \\nATOM   1202  CB  GLU A 153      12.592  30.936   1.881  1.00 17.03           C  \\nATOM   1203  CG  GLU A 153      12.454  32.452   2.015  1.00 21.18           C  \\nATOM   1204  CD  GLU A 153      12.870  33.318   0.817  1.00 25.73           C  \\nATOM   1205  OE1 GLU A 153      13.232  32.781  -0.234  1.00 28.32           O  \\nATOM   1206  OE2 GLU A 153      12.822  34.550   0.936  1.00 26.64           O  \\nATOM   1207  N   ASP A 154      12.959  28.031   2.957  1.00 11.31           N  \\nATOM   1208  CA  ASP A 154      13.023  26.593   2.747  1.00 10.16           C  \\nATOM   1209  C   ASP A 154      11.964  25.828   3.561  1.00 10.90           C  \\nATOM   1210  O   ASP A 154      11.224  24.985   3.035  1.00 11.83           O  \\nATOM   1211  CB  ASP A 154      14.420  26.212   3.117  1.00 15.00           C  \\nATOM   1212  CG  ASP A 154      14.919  24.942   2.524  1.00 19.04           C  \\nATOM   1213  OD1 ASP A 154      14.121  24.195   1.938  1.00 22.96           O  \\nATOM   1214  OD2 ASP A 154      16.122  24.697   2.676  1.00 21.26           O  \\nATOM   1215  N   ALA A 155      11.854  26.107   4.881  1.00  8.83           N  \\nATOM   1216  CA  ALA A 155      10.841  25.450   5.716  1.00 10.00           C  \\nATOM   1217  C   ALA A 155       9.424  25.748   5.203  1.00 10.10           C  \\nATOM   1218  O   ALA A 155       8.653  24.814   4.978  1.00 11.14           O  \\nATOM   1219  CB  ALA A 155      10.923  25.933   7.192  1.00  9.50           C  \\nATOM   1220  N   PHE A 156       9.054  27.011   4.949  1.00 10.59           N  \\nATOM   1221  CA  PHE A 156       7.712  27.345   4.475  1.00 10.51           C  \\nATOM   1222  C   PHE A 156       7.453  26.825   3.069  1.00 11.05           C  \\nATOM   1223  O   PHE A 156       6.368  26.302   2.801  1.00 10.80           O  \\nATOM   1224  CB  PHE A 156       7.492  28.890   4.506  1.00 11.54           C  \\nATOM   1225  CG  PHE A 156       7.266  29.450   5.936  1.00 12.42           C  \\nATOM   1226  CD1 PHE A 156       8.319  29.958   6.707  1.00 12.00           C  \\nATOM   1227  CD2 PHE A 156       5.978  29.434   6.478  1.00 12.57           C  \\nATOM   1228  CE1 PHE A 156       8.065  30.434   8.001  1.00 12.45           C  \\nATOM   1229  CE2 PHE A 156       5.747  29.908   7.757  1.00 12.32           C  \\nATOM   1230  CZ  PHE A 156       6.782  30.406   8.513  1.00 10.18           C  \\nATOM   1231  N   TYR A 157       8.408  26.935   2.151  1.00  9.69           N  \\nATOM   1232  CA  TYR A 157       8.148  26.463   0.798  1.00 12.31           C  \\nATOM   1233  C   TYR A 157       8.091  24.945   0.719  1.00 10.42           C  \\nATOM   1234  O   TYR A 157       7.292  24.403  -0.047  1.00 11.35           O  \\nATOM   1235  CB  TYR A 157       9.233  26.978  -0.156  1.00 15.16           C  \\nATOM   1236  CG  TYR A 157       9.257  28.492  -0.367  1.00 18.73           C  \\nATOM   1237  CD1 TYR A 157       8.380  29.366   0.268  1.00 18.52           C  \\nATOM   1238  CD2 TYR A 157      10.211  28.971  -1.242  1.00 20.44           C  \\nATOM   1239  CE1 TYR A 157       8.466  30.724   0.025  1.00 22.04           C  \\nATOM   1240  CE2 TYR A 157      10.305  30.320  -1.488  1.00 22.70           C  \\nATOM   1241  CZ  TYR A 157       9.436  31.178  -0.852  1.00 23.03           C  \\nATOM   1242  OH  TYR A 157       9.572  32.528  -1.089  1.00 26.15           O  \\nATOM   1243  N   THR A 158       8.940  24.198   1.455  1.00 11.35           N  \\nATOM   1244  CA  THR A 158       8.836  22.727   1.522  1.00 10.11           C  \\nATOM   1245  C   THR A 158       7.440  22.347   2.064  1.00 11.11           C  \\nATOM   1246  O   THR A 158       6.811  21.408   1.558  1.00 11.43           O  \\nATOM   1247  CB  THR A 158       9.935  22.129   2.470  1.00 12.09           C  \\nATOM   1248  OG1 THR A 158      11.216  22.451   1.933  1.00 11.36           O  \\nATOM   1249  CG2 THR A 158       9.791  20.657   2.624  1.00 10.87           C  \\nATOM   1250  N   LEU A 159       6.888  23.056   3.072  1.00  9.25           N  \\nATOM   1251  CA  LEU A 159       5.533  22.754   3.506  1.00  9.93           C  \\nATOM   1252  C   LEU A 159       4.489  22.986   2.388  1.00 10.42           C  \\nATOM   1253  O   LEU A 159       3.560  22.185   2.272  1.00 11.22           O  \\nATOM   1254  CB  LEU A 159       5.171  23.608   4.711  1.00  9.03           C  \\nATOM   1255  CG  LEU A 159       3.789  23.318   5.269  1.00 10.96           C  \\nATOM   1256  CD1 LEU A 159       3.635  21.828   5.651  1.00  9.29           C  \\nATOM   1257  CD2 LEU A 159       3.580  24.223   6.469  1.00 13.71           C  \\nATOM   1258  N   VAL A 160       4.596  24.046   1.548  1.00 10.91           N  \\nATOM   1259  CA  VAL A 160       3.662  24.284   0.439  1.00  9.19           C  \\nATOM   1260  C   VAL A 160       3.803  23.137  -0.556  1.00  9.60           C  \\nATOM   1261  O   VAL A 160       2.781  22.638  -1.031  1.00 10.29           O  \\nATOM   1262  CB  VAL A 160       3.981  25.653  -0.211  1.00  8.50           C  \\nATOM   1263  CG1 VAL A 160       3.402  25.754  -1.622  1.00 10.56           C  \\nATOM   1264  CG2 VAL A 160       3.403  26.741   0.659  1.00  7.98           C  \\nATOM   1265  N   ARG A 161       5.033  22.667  -0.839  1.00  9.10           N  \\nATOM   1266  CA  ARG A 161       5.206  21.554  -1.767  1.00  9.82           C  \\nATOM   1267  C   ARG A 161       4.644  20.240  -1.215  1.00 10.44           C  \\nATOM   1268  O   ARG A 161       4.166  19.395  -1.984  1.00 12.88           O  \\nATOM   1269  CB  ARG A 161       6.679  21.398  -2.115  1.00  9.39           C  \\nATOM   1270  CG  ARG A 161       7.146  22.640  -2.816  1.00  9.70           C  \\nATOM   1271  CD  ARG A 161       8.594  22.398  -3.227  1.00 14.12           C  \\nATOM   1272  NE  ARG A 161       9.122  23.635  -3.780  1.00 14.15           N  \\nATOM   1273  CZ  ARG A 161      10.144  24.313  -3.257  1.00 12.45           C  \\nATOM   1274  NH1 ARG A 161      10.823  23.858  -2.204  1.00 13.24           N  \\nATOM   1275  NH2 ARG A 161      10.485  25.475  -3.819  1.00 13.52           N  \\nATOM   1276  N   GLU A 162       4.663  20.046   0.112  1.00 10.71           N  \\nATOM   1277  CA  GLU A 162       4.054  18.877   0.722  1.00 11.80           C  \\nATOM   1278  C   GLU A 162       2.522  18.887   0.568  1.00 11.20           C  \\nATOM   1279  O   GLU A 162       1.921  17.848   0.274  1.00 12.90           O  \\nATOM   1280  CB  GLU A 162       4.430  18.815   2.216  1.00 13.86           C  \\nATOM   1281  CG  GLU A 162       5.870  18.460   2.580  1.00 14.10           C  \\nATOM   1282  CD  GLU A 162       6.268  17.038   2.224  1.00 17.23           C  \\nATOM   1283  OE1 GLU A 162       5.571  16.116   2.627  1.00 19.29           O  \\nATOM   1284  OE2 GLU A 162       7.278  16.827   1.545  1.00 22.14           O  \\nATOM   1285  N   ILE A 163       1.869  20.045   0.731  1.00 10.13           N  \\nATOM   1286  CA  ILE A 163       0.426  20.175   0.542  1.00 10.58           C  \\nATOM   1287  C   ILE A 163       0.128  19.878  -0.933  1.00 13.43           C  \\nATOM   1288  O   ILE A 163      -0.796  19.118  -1.235  1.00 15.01           O  \\nATOM   1289  CB  ILE A 163      -0.015  21.590   0.891  1.00 11.33           C  \\nATOM   1290  CG1 ILE A 163       0.084  21.777   2.400  1.00 11.27           C  \\nATOM   1291  CG2 ILE A 163      -1.486  21.815   0.437  1.00 12.37           C  \\nATOM   1292  CD1 ILE A 163      -0.006  23.245   2.894  1.00 12.56           C  \\nATOM   1293  N   ARG A 164       0.904  20.456  -1.871  1.00 11.72           N  \\nATOM   1294  CA  ARG A 164       0.770  20.163  -3.283  1.00 13.64           C  \\nATOM   1295  C   ARG A 164       0.813  18.657  -3.626  1.00 17.97           C  \\nATOM   1296  O   ARG A 164       0.150  18.197  -4.559  1.00 18.12           O  \\nATOM   1297  CB  ARG A 164       1.897  20.887  -4.029  1.00 12.51           C  \\nATOM   1298  CG  ARG A 164       1.647  22.371  -4.120  1.00 12.49           C  \\nATOM   1299  CD  ARG A 164       2.801  22.888  -4.950  1.00 14.72           C  \\nATOM   1300  NE  ARG A 164       2.730  24.327  -5.113  1.00 12.82           N  \\nATOM   1301  CZ  ARG A 164       3.800  25.071  -5.405  1.00 13.29           C  \\nATOM   1302  NH1 ARG A 164       5.018  24.539  -5.527  1.00 11.56           N  \\nATOM   1303  NH2 ARG A 164       3.595  26.381  -5.572  1.00 12.55           N  \\nATOM   1304  N   GLN A 165       1.590  17.831  -2.935  1.00 20.23           N  \\nATOM   1305  CA  GLN A 165       1.673  16.435  -3.337  1.00 27.50           C  \\nATOM   1306  C   GLN A 165       0.782  15.499  -2.502  1.00 29.12           C  \\nATOM   1307  O   GLN A 165       0.826  14.274  -2.655  1.00 29.30           O  \\nATOM   1308  CB  GLN A 165       3.168  16.070  -3.286  1.00 30.34           C  \\nATOM   1309  CG  GLN A 165       4.018  16.904  -4.309  1.00 35.17           C  \\nATOM   1310  CD  GLN A 165       4.109  16.470  -5.796  1.00 37.71           C  \\nATOM   1311  OE1 GLN A 165       4.256  17.293  -6.719  1.00 36.70           O  \\nATOM   1312  NE2 GLN A 165       4.079  15.166  -6.094  1.00 38.13           N  \\nATOM   1313  N   HIS A 166      -0.068  16.075  -1.638  1.00 29.28           N  \\nATOM   1314  CA  HIS A 166      -0.982  15.296  -0.820  1.00 31.13           C  \\nATOM   1315  C   HIS A 166      -2.214  14.971  -1.665  1.00 33.30           C  \\nATOM   1316  O   HIS A 166      -2.630  15.786  -2.497  1.00 34.96           O  \\nATOM   1317  CB  HIS A 166      -1.420  16.090   0.428  1.00 30.89           C  \\nATOM   1318  CG  HIS A 166      -2.247  15.319   1.475  1.00 32.17           C  \\nATOM   1319  ND1 HIS A 166      -3.579  15.231   1.603  1.00 33.52           N  \\nATOM   1320  CD2 HIS A 166      -1.712  14.560   2.502  1.00 32.95           C  \\nATOM   1321  CE1 HIS A 166      -3.857  14.464   2.647  1.00 33.07           C  \\nATOM   1322  NE2 HIS A 166      -2.727  14.064   3.184  1.00 33.06           N  \\nATOM   1323  OXT HIS A 166      -2.763  13.888  -1.454  1.00 36.27           O  \\nTER    1324      HIS A 166                                                      \\nHETATM 1325 MG    MG A 168       4.771  33.860  19.213  1.00  9.42          MG  \\nHETATM 1326  PG  GNP A 167       5.113  32.223  21.994  1.00  9.66           P  \\nHETATM 1327  O1G GNP A 167       4.844  32.596  23.390  1.00  9.26           O  \\nHETATM 1328  O2G GNP A 167       4.203  32.910  21.112  1.00 11.20           O  \\nHETATM 1329  O3G GNP A 167       4.640  30.769  21.802  1.00 10.09           O  \\nHETATM 1330  N3B GNP A 167       6.701  32.303  21.621  1.00  8.18           N  \\nHETATM 1331  PB  GNP A 167       7.310  32.034  20.111  1.00  8.77           P  \\nHETATM 1332  O1B GNP A 167       7.468  30.605  19.791  1.00  8.35           O  \\nHETATM 1333  O2B GNP A 167       6.696  32.775  19.021  1.00  9.38           O  \\nHETATM 1334  O3A GNP A 167       8.792  32.643  20.152  1.00 10.15           O  \\nHETATM 1335  PA  GNP A 167       9.280  34.140  19.710  1.00 10.18           P  \\nHETATM 1336  O1A GNP A 167       9.263  34.268  18.220  1.00 10.07           O  \\nHETATM 1337  O2A GNP A 167       8.594  35.173  20.510  1.00  9.33           O  \\nHETATM 1338  O5' GNP A 167      10.826  34.056  20.132  1.00 11.46           O  \\nHETATM 1339  C5' GNP A 167      11.170  33.911  21.535  1.00 10.56           C  \\nHETATM 1340  C4' GNP A 167      12.587  34.445  21.811  1.00 11.96           C  \\nHETATM 1341  O4' GNP A 167      13.560  33.649  21.095  1.00 11.66           O  \\nHETATM 1342  C3' GNP A 167      12.813  35.910  21.359  1.00 12.06           C  \\nHETATM 1343  O3' GNP A 167      13.793  36.538  22.221  1.00 14.52           O  \\nHETATM 1344  C2' GNP A 167      13.325  35.679  19.947  1.00 11.90           C  \\nHETATM 1345  O2' GNP A 167      14.151  36.750  19.499  1.00 14.99           O  \\nHETATM 1346  C1' GNP A 167      14.242  34.478  20.156  1.00 12.90           C  \\nHETATM 1347  N9  GNP A 167      14.454  33.739  18.896  1.00 12.98           N  \\nHETATM 1348  C8  GNP A 167      13.518  33.262  17.993  1.00 11.76           C  \\nHETATM 1349  N7  GNP A 167      14.016  32.636  16.979  1.00 12.34           N  \\nHETATM 1350  C5  GNP A 167      15.413  32.694  17.200  1.00 12.36           C  \\nHETATM 1351  C6  GNP A 167      16.521  32.219  16.402  1.00 12.30           C  \\nHETATM 1352  O6  GNP A 167      16.507  31.591  15.338  1.00 12.16           O  \\nHETATM 1353  N1  GNP A 167      17.735  32.503  16.995  1.00 11.60           N  \\nHETATM 1354  C2  GNP A 167      17.909  33.082  18.236  1.00 13.91           C  \\nHETATM 1355  N2  GNP A 167      19.173  33.280  18.658  1.00 14.60           N  \\nHETATM 1356  N3  GNP A 167      16.884  33.498  18.996  1.00 13.01           N  \\nHETATM 1357  C4  GNP A 167      15.675  33.331  18.398  1.00 14.36           C  \\nHETATM 1358  O   HOH A 169      13.588  43.386  17.779  1.00 31.80           O  \\nHETATM 1359  O   HOH A 170       8.248  36.064  23.209  1.00 13.63           O  \\nHETATM 1360  O   HOH A 171       3.254  37.845  19.454  1.00 18.29           O  \\nHETATM 1361  O   HOH A 172       5.945  35.619  20.105  1.00 11.33           O  \\nHETATM 1362  O   HOH A 173       3.669  32.226  18.172  1.00  8.22           O  \\nHETATM 1363  O   HOH A 174       2.896  37.170  15.500  1.00 21.37           O  \\nHETATM 1364  O   HOH A 175       2.102  32.309  24.122  1.00 27.47           O  \\nHETATM 1365  O   HOH A 176      -1.425  37.650  19.667  1.00 24.98           O  \\nHETATM 1366  O   HOH A 177      -3.225  30.031  -8.217  1.00 23.57           O  \\nHETATM 1367  O   HOH A 178      16.678  26.321  26.916  1.00 36.70           O  \\nHETATM 1368  O   HOH A 179      -2.134  40.975   1.551  1.00 81.53           O  \\nHETATM 1369  O   HOH A 180       0.400  40.675  -0.629  1.00 24.66           O  \\nHETATM 1370  O   HOH A 181      10.944  44.793  16.009  1.00 48.47           O  \\nHETATM 1371  O   HOH A 182      17.207  42.570  16.786  1.00 43.23           O  \\nHETATM 1372  O   HOH A 183      14.999  45.167  19.922  1.00 39.52           O  \\nHETATM 1373  O   HOH A 184      11.598  37.755  13.202  1.00 11.72           O  \\nHETATM 1374  O   HOH A 185       6.958  38.922  18.146  1.00 26.17           O  \\nHETATM 1375  O   HOH A 186      14.078  32.033  27.772  1.00 57.73           O  \\nHETATM 1376  O   HOH A 187       9.126  37.906  19.595  1.00 32.42           O  \\nHETATM 1377  O   HOH A 188      -0.049  36.435  27.259  1.00 50.56           O  \\nHETATM 1378  O   HOH A 189       3.843  30.503  25.485  1.00 46.80           O  \\nHETATM 1379  O   HOH A 190       4.931  28.343  27.587  1.00 29.32           O  \\nHETATM 1380  O   HOH A 191       4.881  25.774  26.247  1.00 28.55           O  \\nHETATM 1381  O   HOH A 193       8.729  26.516  26.645  1.00 37.34           O  \\nHETATM 1382  O   HOH A 194      -5.515  23.731  -4.213  1.00 43.74           O  \\nHETATM 1383  O   HOH A 195      -7.006  26.635   0.729  1.00 25.77           O  \\nHETATM 1384  O   HOH A 196      -2.033  23.281  -4.754  1.00 36.89           O  \\nHETATM 1385  O   HOH A 197      -4.326  24.642  -0.854  1.00 28.59           O  \\nHETATM 1386  O   HOH A 198      -6.858  26.995  -2.808  1.00 28.83           O  \\nHETATM 1387  O   HOH A 200      -3.937  29.225   8.289  1.00 24.08           O  \\nHETATM 1388  O   HOH A 201      -8.934  24.037   2.795  1.00 34.16           O  \\nHETATM 1389  O   HOH A 202       1.444  26.302  19.318  1.00 12.37           O  \\nHETATM 1390  O   HOH A 203       2.031  24.632  27.307  1.00 50.09           O  \\nHETATM 1391  O   HOH A 204      15.125  28.341  23.174  1.00 13.88           O  \\nHETATM 1392  O   HOH A 205      12.808  31.541  25.056  1.00 46.88           O  \\nHETATM 1393  O   HOH A 206       8.329  34.933   0.543  1.00 53.21           O  \\nHETATM 1394  O   HOH A 207       0.760  42.969   2.336  1.00 35.42           O  \\nHETATM 1395  O   HOH A 208       1.270  40.298   7.071  1.00 76.61           O  \\nHETATM 1396  O   HOH A 209      -3.135  39.625   7.649  1.00 35.40           O  \\nHETATM 1397  O   HOH A 210       8.699  39.076   2.399  1.00 33.93           O  \\nHETATM 1398  O   HOH A 212      10.040  41.806   3.447  1.00 37.88           O  \\nHETATM 1399  O   HOH A 213      18.914  29.095   3.704  1.00 25.32           O  \\nHETATM 1400  O   HOH A 214      15.966  29.098   2.957  1.00 18.63           O  \\nHETATM 1401  O   HOH A 215      15.506  31.962   4.015  1.00 18.93           O  \\nHETATM 1402  O   HOH A 216      22.935  30.533  10.845  1.00 39.81           O  \\nHETATM 1403  O   HOH A 217       6.476  43.893  23.325  1.00 49.09           O  \\nHETATM 1404  O   HOH A 218      11.746  41.248  28.612  1.00 36.22           O  \\nHETATM 1405  O   HOH A 219      11.480  42.270  25.715  1.00 29.66           O  \\nHETATM 1406  O   HOH A 220       9.344  34.251  -8.278  1.00 36.61           O  \\nHETATM 1407  O   HOH A 221      -0.111  19.667  -8.066  1.00 44.76           O  \\nHETATM 1408  O   HOH A 222      -0.528  30.460 -10.943  1.00 24.29           O  \\nHETATM 1409  O   HOH A 223     -11.352  24.274  19.491  1.00 31.21           O  \\nHETATM 1410  O   HOH A 224      -4.293  30.642 -16.466  1.00127.55           O  \\nHETATM 1411  O   HOH A 225       3.466  29.729 -16.232  1.00 26.15           O  \\nHETATM 1412  O   HOH A 227      -4.288  18.610   4.198  1.00 34.77           O  \\nHETATM 1413  O   HOH A 228       4.626  16.154   5.151  1.00 14.96           O  \\nHETATM 1414  O   HOH A 229       5.681  11.069   2.580  1.00 59.76           O  \\nHETATM 1415  O   HOH A 230       1.173  14.021   3.406  1.00 32.15           O  \\nHETATM 1416  O   HOH A 231       7.935  22.484  20.504  1.00 10.86           O  \\nHETATM 1417  O   HOH A 232       3.071  10.004  14.546  1.00 23.48           O  \\nHETATM 1418  O   HOH A 233       2.437  13.381   8.401  1.00 17.75           O  \\nHETATM 1419  O   HOH A 234       4.779   7.500   6.154  1.00 71.09           O  \\nHETATM 1420  O   HOH A 235      -4.250   8.748   6.880  1.00 24.76           O  \\nHETATM 1421  O   HOH A 236     -13.381  22.830  17.669  1.00 39.09           O  \\nHETATM 1422  O   HOH A 237      16.969  39.052   8.276  1.00 50.25           O  \\nHETATM 1423  O   HOH A 238      -6.315   9.333  13.897  1.00 36.24           O  \\nHETATM 1424  O   HOH A 240      -0.160   4.395  12.468  1.00 38.79           O  \\nHETATM 1425  O   HOH A 241       1.295   7.931  12.335  1.00 49.03           O  \\nHETATM 1426  O   HOH A 242       4.067  11.551   6.968  1.00 46.97           O  \\nHETATM 1427  O   HOH A 243      -5.060   6.579  -0.216  1.00 50.14           O  \\nHETATM 1428  O   HOH A 244      18.164   7.151   5.573  1.00 71.67           O  \\nHETATM 1429  O   HOH A 245       2.068   8.904   7.636  1.00 45.86           O  \\nHETATM 1430  O   HOH A 246      24.661  23.134  12.757  1.00 34.69           O  \\nHETATM 1431  O   HOH A 247      21.051  23.714   6.239  1.00 24.64           O  \\nHETATM 1432  O   HOH A 248      25.855  31.137  20.712  1.00 34.53           O  \\nHETATM 1433  O   HOH A 249      24.868  30.890  26.223  1.00 48.47           O  \\nHETATM 1434  O   HOH A 250      26.967  20.447  19.448  1.00 47.77           O  \\nHETATM 1435  O   HOH A 252      28.453  14.196  14.602  1.00 39.78           O  \\nHETATM 1436  O   HOH A 253      19.639  16.256   9.760  1.00 37.54           O  \\nHETATM 1437  O   HOH A 254      16.629   7.383  10.526  1.00 60.90           O  \\nHETATM 1438  O   HOH A 255       4.972   7.888  14.276  1.00 54.95           O  \\nHETATM 1439  O   HOH A 256      14.184  14.684   5.643  1.00 31.40           O  \\nHETATM 1440  O   HOH A 257      11.766  17.254   1.278  1.00 41.82           O  \\nHETATM 1441  O   HOH A 258      16.274  16.365   6.342  1.00 17.29           O  \\nHETATM 1442  O   HOH A 259      18.346  24.095   6.620  1.00 17.36           O  \\nHETATM 1443  O   HOH A 260      22.479  35.073  19.694  1.00 43.96           O  \\nHETATM 1444  O   HOH A 261      20.580  31.747   4.906  1.00 59.46           O  \\nHETATM 1445  O   HOH A 262      18.196  25.514   4.261  1.00 28.75           O  \\nHETATM 1446  O   HOH A 263      12.857  20.183   1.149  1.00 21.79           O  \\nHETATM 1447  O   HOH A 264      19.514  15.912  24.943  1.00 55.52           O  \\nHETATM 1448  O   HOH A 265       8.925   7.579   5.621  1.00 77.94           O  \\nHETATM 1449  O   HOH A 266      17.622  21.835  26.964  1.00 27.48           O  \\nHETATM 1450  O   HOH A 267      24.258  19.786  14.155  1.00 20.48           O  \\nHETATM 1451  O   HOH A 268      23.806  15.180  15.136  1.00 19.86           O  \\nHETATM 1452  O   HOH A 269      21.478  14.557  11.134  1.00 29.66           O  \\nHETATM 1453  O   HOH A 270      17.293  45.621  24.753  1.00 39.18           O  \\nHETATM 1454  O   HOH A 271      -3.518  40.206  17.649  1.00 44.75           O  \\nHETATM 1455  O   HOH A 272       2.143  19.852  24.090  1.00 40.10           O  \\nHETATM 1456  O   HOH A 273       7.101  35.918  -4.704  1.00 31.76           O  \\nHETATM 1457  O   HOH A 274       5.462  38.756  -5.521  1.00 27.51           O  \\nHETATM 1458  O   HOH A 275       5.748  20.565  28.638  1.00 58.06           O  \\nHETATM 1459  O   HOH A 276       8.814  31.258  -8.964  1.00 23.79           O  \\nHETATM 1460  O   HOH A 277     -10.427  35.895   4.301  1.00 58.91           O  \\nHETATM 1461  O   HOH A 278      -8.713  28.579   2.333  1.00 69.36           O  \\nHETATM 1462  O   HOH A 279      24.472  24.237  17.316  1.00 32.14           O  \\nHETATM 1463  O   HOH A 281      16.968  32.701  21.815  1.00 23.13           O  \\nHETATM 1464  O   HOH A 282      19.541  34.346  21.581  1.00 53.25           O  \\nHETATM 1465  O   HOH A 283      24.847  23.384  26.621  1.00 61.18           O  \\nHETATM 1466  O   HOH A 284       1.409  11.113   2.873  1.00 44.87           O  \\nHETATM 1467  O   HOH A 285       1.556  11.488  -0.914  1.00 37.86           O  \\nHETATM 1468  O   HOH A 286       9.758  12.945   3.889  1.00 36.47           O  \\nHETATM 1469  O   HOH A 287       3.827  13.375   4.622  1.00 44.90           O  \\nHETATM 1470  O   HOH A 288      14.279   6.175  15.391  1.00 78.25           O  \\nHETATM 1471  O   HOH A 289       7.573  13.968   2.191  1.00 76.47           O  \\nHETATM 1472  O   HOH A 290       2.323  15.482   1.418  1.00 35.15           O  \\nHETATM 1473  O   HOH A 291      11.997  14.573   2.962  1.00 49.86           O  \\nHETATM 1474  O   HOH A 292      23.798  33.550  17.880  1.00 24.64           O  \\nHETATM 1475  O   HOH A 293      14.657  36.046  -0.571  1.00 51.86           O  \\nHETATM 1476  O   HOH A 294      24.456  25.366   4.004  1.00 53.23           O  \\nHETATM 1477  O   HOH A 296      -0.603  12.134  -4.185  1.00 52.38           O  \\nHETATM 1478  O   HOH A 297      -8.372  34.415  -4.471  1.00 74.53           O  \\nHETATM 1479  O   HOH A 300     -14.033  36.501   0.062  1.00 50.21           O  \\nHETATM 1480  O   HOH A 301     -17.102  35.735  -2.806  1.00 42.58           O  \\nHETATM 1481  O   HOH A 303     -17.383  40.542   1.431  1.00 77.16           O  \\nHETATM 1482  O   HOH A 304     -19.024  41.557  -3.411  1.00107.16           O  \\nHETATM 1483  O   HOH A 305      -9.876  37.804   0.839  1.00 90.65           O  \\nHETATM 1484  O   HOH A 306     -14.053  36.466   4.552  1.00 82.05           O  \\nHETATM 1485  O   HOH A 307     -14.183  39.558   0.339  1.00 58.66           O  \\nHETATM 1486  O   HOH A 309     -12.547  28.677   8.301  1.00 74.88           O  \\nHETATM 1487  O   HOH A 311     -14.787  32.746  -2.171  1.00 40.79           O  \\nHETATM 1488  O   HOH A 312      -6.916  30.740   3.095  1.00 36.53           O  \\nHETATM 1489  O   HOH A 315      -8.312  33.027   7.340  1.00 83.28           O  \\nHETATM 1490  O   HOH A 316      -6.280  35.646   7.624  1.00 75.97           O  \\nHETATM 1491  O   HOH A 317      -5.230  31.565   9.609  1.00 55.67           O  \\nHETATM 1492  O   HOH A 318       6.747  37.258   1.208  1.00 57.93           O  \\nHETATM 1493  O   HOH A 319       4.747  42.419  12.683  1.00 57.74           O  \\nHETATM 1494  O   HOH A 320       2.984  41.950  16.191  1.00 41.69           O  \\nHETATM 1495  O   HOH A 321       3.867  49.208  12.056  1.00 24.57           O  \\nHETATM 1496  O   HOH A 322       7.530  42.894  17.776  1.00 79.28           O  \\nHETATM 1497  O   HOH A 324      19.822  39.457  17.826  1.00 66.22           O  \\nHETATM 1498  O   HOH A 325      12.391  45.557  24.973  1.00 48.70           O  \\nHETATM 1499  O   HOH A 326      14.944  43.928  23.794  1.00 62.45           O  \\nHETATM 1500  O   HOH A 327     -12.984  24.008  12.563  1.00 40.31           O  \\nHETATM 1501  O   HOH A 330       0.572  41.133  19.277  1.00 50.77           O  \\nHETATM 1502  O   HOH A 331      -6.196  37.979  23.043  1.00 48.93           O  \\nHETATM 1503  O   HOH A 332      -3.526  30.410  19.065  1.00 21.26           O  \\nHETATM 1504  O   HOH A 333      -7.995  32.093  21.685  1.00 54.11           O  \\nHETATM 1505  O   HOH A 334      -8.483  28.891  20.063  1.00 70.37           O  \\nHETATM 1506  O   HOH A 335     -11.670  31.022  16.981  1.00 76.86           O  \\nHETATM 1507  O   HOH A 336     -11.082  34.450  18.910  1.00 43.07           O  \\nHETATM 1508  O   HOH A 339      -6.521  37.738  15.165  1.00 64.11           O  \\nHETATM 1509  O   HOH A 340     -10.829  35.877  15.629  1.00 58.36           O  \\nHETATM 1510  O   HOH A 341      -6.552  41.166  12.231  1.00 55.67           O  \\nHETATM 1511  O   HOH A 342      -1.242  39.671  12.185  1.00 61.04           O  \\nHETATM 1512  O   HOH A 343      -4.281  38.934  13.464  1.00 32.14           O  \\nHETATM 1513  O   HOH A 344       0.386  41.208  14.039  1.00 40.80           O  \\nHETATM 1514  O   HOH A 345       0.777  39.663  21.603  1.00 67.71           O  \\nHETATM 1515  O   HOH A 346      -7.879  34.848  19.044  1.00 52.60           O  \\nHETATM 1516  O   HOH A 347       1.121  32.594 -11.957  1.00 32.30           O  \\nHETATM 1517  O   HOH A 348       0.924  32.597 -14.716  1.00 48.34           O  \\nHETATM 1518  O   HOH A 349      -2.241  27.348  20.565  1.00 26.54           O  \\nHETATM 1519  O   HOH A 350       3.259  26.214  29.881  1.00 69.22           O  \\nHETATM 1520  O   HOH A 351      -8.318  34.095  28.392  1.00 81.96           O  \\nHETATM 1521  O   HOH A 352      -2.927  29.775  31.412  1.00 80.92           O  \\nHETATM 1522  O   HOH A 353     -10.410  32.034  25.972  1.00 60.17           O  \\nHETATM 1523  O   HOH A 354      -8.122  30.393  25.378  1.00 62.33           O  \\nHETATM 1524  O   HOH A 356      -6.138  18.882  24.644  1.00 49.26           O  \\nHETATM 1525  O   HOH A 358      -1.406  19.033  23.084  1.00 82.05           O  \\nHETATM 1526  O   HOH A 359       4.169   7.339  10.280  1.00 66.03           O  \\nHETATM 1527  O   HOH A 360      -9.761  30.165   7.702  1.00 46.15           O  \\nHETATM 1528  O   HOH A 361     -16.002  27.801  11.700  1.00 41.85           O  \\nHETATM 1529  O   HOH A 362      16.215  30.489  29.264  1.00 67.21           O  \\nHETATM 1530  O   HOH A 364      21.417  27.685  26.140  1.00 49.57           O  \\nHETATM 1531  O   HOH A 366      15.824  34.767  30.662  1.00 25.90           O  \\nHETATM 1532  O   HOH A 367      18.527  35.965  24.111  1.00 68.69           O  \\nHETATM 1533  O   HOH A 369      12.103  35.275  24.945  1.00 51.37           O  \\nHETATM 1534  O   HOH A 370      24.782  11.970  23.641  1.00 55.92           O  \\nHETATM 1535  O   HOH A 371      14.337  21.394  28.627  1.00 30.11           O  \\nHETATM 1536  O   HOH A 372      -1.367  12.600  23.053  1.00 44.43           O  \\nHETATM 1537  O   HOH A 373       7.060   8.567  21.416  1.00 47.99           O  \\nHETATM 1538  O   HOH A 374       7.639   8.040  16.420  1.00 38.76           O  \\nHETATM 1539  O   HOH A 376      -1.282  15.566  25.487  1.00 67.79           O  \\nHETATM 1540  O   HOH A 378      17.945  30.924  23.806  1.00 31.50           O  \\nHETATM 1541  O   HOH A 379      17.360  33.209  33.337  1.00 30.46           O  \\nHETATM 1542  O   HOH A 380       3.971   6.567  22.089  1.00 86.80           O  \\nHETATM 1543  O   HOH A 381      -0.065  10.022  26.928  1.00 61.86           O  \\nHETATM 1544  O   HOH A 382      -4.450  12.966  21.222  1.00 42.82           O  \\nHETATM 1545  O   HOH A 383      -2.584   6.719   5.938  1.00 40.64           O  \\nHETATM 1546  O   HOH A 384      25.567  31.936  12.845  1.00 73.17           O  \\nHETATM 1547  O   HOH A 385      24.446  29.383  22.970  1.00 36.61           O  \\nHETATM 1548  O   HOH A 386      24.865  38.167  18.227  1.00 60.74           O  \\nHETATM 1549  O   HOH A 387      23.135  35.012  14.543  1.00 59.19           O  \\nHETATM 1550  O   HOH A 388      19.325   9.556  18.093  1.00 92.18           O  \\nHETATM 1551  O   HOH A 389      25.326  12.873  16.251  1.00 49.27           O  \\nHETATM 1552  O   HOH A 390      20.219  10.405  10.956  1.00 63.05           O  \\nHETATM 1553  O   HOH A 392       9.452   5.453   8.249  1.00 47.81           O  \\nHETATM 1554  O   HOH A 394      10.139  12.252   0.254  1.00 29.37           O  \\nHETATM 1555  O   HOH A 395      28.004  37.416  13.466  1.00 52.29           O  \\nHETATM 1556  O   HOH A 396      24.283  36.829  12.307  1.00 64.74           O  \\nHETATM 1557  O   HOH A 397      13.748  34.031  -3.019  1.00 47.80           O  \\nHETATM 1558  O   HOH A 398      16.981  34.189   2.006  1.00 80.51           O  \\nHETATM 1559  O   HOH A 399      -4.426  24.223  -7.240  1.00 38.09           O  \\nHETATM 1560  O   HOH A 400      19.676  33.981   3.336  1.00 64.86           O  \\nHETATM 1561  O   HOH A 401      14.633  28.745  -0.317  1.00 66.25           O  \\nHETATM 1562  O   HOH A 402      -2.139  15.576  22.716  1.00 65.01           O  \\nHETATM 1563  O   HOH A 403       9.873  11.090  20.760  1.00 53.88           O  \\nHETATM 1564  O   HOH A 404      18.291  36.379  26.927  1.00 57.23           O  \\nHETATM 1565  O   HOH A 405       0.138   6.145  15.269  1.00 82.80           O  \\nHETATM 1566  O   HOH A 406      -5.515  38.166  18.689  1.00 48.28           O  \\nHETATM 1567  O   HOH A 407      -0.399   4.688   8.809  1.00 58.35           O  \\nHETATM 1568  O   HOH A 408       0.813  25.584  22.307  1.00 40.04           O  \\nCONECT  117 1325                                                                \\nCONECT  265 1325                                                                \\nCONECT 1325  117  265 1328 1333                                                 \\nCONECT 1325 1361 1362                                                           \\nCONECT 1326 1327 1328 1329 1330                                                 \\nCONECT 1327 1326                                                                \\nCONECT 1328 1325 1326                                                           \\nCONECT 1329 1326                                                                \\nCONECT 1330 1326 1331                                                           \\nCONECT 1331 1330 1332 1333 1334                                                 \\nCONECT 1332 1331                                                                \\nCONECT 1333 1325 1331                                                           \\nCONECT 1334 1331 1335                                                           \\nCONECT 1335 1334 1336 1337 1338                                                 \\nCONECT 1336 1335                                                                \\nCONECT 1337 1335                                                                \\nCONECT 1338 1335 1339                                                           \\nCONECT 1339 1338 1340                                                           \\nCONECT 1340 1339 1341 1342                                                      \\nCONECT 1341 1340 1346                                                           \\nCONECT 1342 1340 1343 1344                                                      \\nCONECT 1343 1342                                                                \\nCONECT 1344 1342 1345 1346                                                      \\nCONECT 1345 1344                                                                \\nCONECT 1346 1341 1344 1347                                                      \\nCONECT 1347 1346 1348 1357                                                      \\nCONECT 1348 1347 1349                                                           \\nCONECT 1349 1348 1350                                                           \\nCONECT 1350 1349 1351 1357                                                      \\nCONECT 1351 1350 1352 1353                                                      \\nCONECT 1352 1351                                                                \\nCONECT 1353 1351 1354                                                           \\nCONECT 1354 1353 1355 1356                                                      \\nCONECT 1355 1354                                                                \\nCONECT 1356 1354 1357                                                           \\nCONECT 1357 1347 1350 1356                                                      \\nCONECT 1361 1325                                                                \\nCONECT 1362 1325                                                                \\nMASTER      312    0    2    5    6    0    9    6 1567    1   38   13          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewergrid_16835771918631198[0][0].addModel(\"\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    5.3278   11.9619   -1.5862 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4918   12.0625   -0.4513 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5527   13.0816    0.5011 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7707   13.0853    1.5453 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4957   14.1194    0.2981 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3069   14.0310   -0.8665 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2721   14.9285   -1.2734 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.7924   14.4907   -2.4207 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.7723   14.9976   -3.2383 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.1222   14.3073   -4.4052 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.4811   13.1363   -4.7128 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4813   12.6157   -3.8820 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1577   13.3143   -2.7429 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2349   13.0415   -1.7643 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8145   11.3598   -0.3222 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1195   12.3585    1.6752 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8318   13.8149    2.2035 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5851   14.8663    0.9332 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7640    9.3703   -2.6645 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8250   10.2866   -3.3296 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1718    9.8621   -4.4958 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2879   10.6862   -5.1442 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.0572   11.9404   -4.6231 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7079   12.3479   -3.4716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5995   11.5321   -2.8072 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-4.68549\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -7.2548    9.0375    4.5817 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0448    8.4019    4.2228 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.8292    9.0547    4.0095 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.7612    8.4021    3.6418 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.7899   10.4573    4.2059 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0025   11.0841    4.6043 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1942   12.4245    4.8676 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4672   12.6023    5.2235 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1730   13.7240    5.5829 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.5292   13.6094    5.9119 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.1308   12.3793    5.8698 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.4149   11.2328    5.5039 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0847   11.3744    5.1862 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.1585   10.4388    4.7983 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0601    7.4234    4.1148 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8013    7.4275    3.5086 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9150    8.8841    3.4976 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9622   10.9735    4.0730 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.7008    7.7755    3.8786 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.2098    7.6907    5.2625 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.9715    6.9952    6.2128 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.5471    6.8921    7.5129 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3553    7.4879    7.8630 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6125    8.1716    6.9162 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0245    8.2848    5.6050 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-4.21517\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -5.2059    9.8913    1.4701 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.4393    8.7205    1.6663 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9500    8.2799    2.8975 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.2150    7.2064    2.9934 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.2783    9.0388    4.0481 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0823   10.1944    3.8472 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.5352   11.0665    4.8152 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2676   12.0083    4.2190 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9393   13.0997    4.7121 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6386   13.9345    3.8321 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6414   13.6492    2.4922 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9578   12.5374    1.9849 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2799   11.7347    2.8716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.5365   10.6013    2.6562 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.2343    8.1735    0.8739 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9938    6.6876    2.1865 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.8815    6.9151    3.8727 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9646    8.7783    4.9442 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0780   11.0707   -1.1110 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2599   10.2829   -0.7287 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3094   10.1417   -1.6483 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4274    9.4138   -1.3304 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4951    8.8250   -0.0867 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4530    8.9717    0.8122 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3219    9.7000    0.5088 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-4.01159\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -6.9498   10.0239    3.9638 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6927    9.4677    4.2914 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.8273    8.8533    3.3843 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6974    8.3277    3.7703 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.2156    8.8174    2.0222 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.4645    9.4132    1.6947 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.0326    9.5098    0.4413 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1973   10.1489    0.5589 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.1492   10.5022   -0.3655 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.2995   11.1814    0.0542 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.4572   11.4825    1.3813 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.4891   11.1240    2.3275 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3695   10.4583    1.8875 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2812    9.9929    2.5820 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4050    9.5174    5.2316 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.4388    8.3591    4.7197 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.1033    7.8998    3.1121 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.6476    8.3998    1.3350 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.9625   10.1830    5.9622 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1934   11.3853    5.6068 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4816   12.5938    6.2575 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7858   13.7364    5.9555 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7975   13.6689    4.9981 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5224   12.4705    4.3628 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2122   11.3124    4.6543 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.94575\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    6.7047   12.9704   -2.2899 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4325   13.2928   -0.9412 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1589   13.3474   -0.3716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9846   13.6911    0.8748 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0535   13.0157   -1.1934 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3391   12.6553   -2.5390 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4301   12.2813   -3.5068 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1009   12.0104   -4.6273 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6860   11.6021   -5.8709 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6341   11.3923   -6.8797 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9614   11.5982   -6.6099 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3889   12.0147   -5.3431 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4366   12.2126   -4.3715 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5684   12.6144   -3.0658 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1999   13.4943   -0.3584 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7609   13.9193    1.4356 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0768   13.7223    1.2545 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1313   13.0340   -0.8491 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.0760   13.9697   -3.4934 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.0229   12.5708   -3.0417 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.1470   11.7530   -3.2282 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.1377   10.4440   -2.8192 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.9984    9.9516   -2.2213 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8955   10.7685   -2.0429 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8879   12.0867   -2.4480 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.69027\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -0.0133    7.1993    3.0780 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9633    8.1816    3.3577 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.9099    8.1021    4.3810 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7566    9.0696    4.6019 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.9237    6.9307    5.1778 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9627    5.9302    4.8653 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.8033    4.7110    5.4906 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.1959    4.0619    4.8915 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.7575    2.8294    5.1174 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8208    2.3984    4.3147 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.2843    3.2138    3.3162 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.7147    4.4715    3.0823 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.6719    4.8690    3.8852 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.0605    6.0297    3.8822 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9772    8.9839    2.7872 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7370    9.8786    4.0412 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4153    8.9908    5.3294 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.5681    6.8148    5.9130 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.3767    8.0951    1.7813 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.3467    7.3852    1.0076 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.5618    7.1538   -0.3589 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.6250    6.4970   -1.1150 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.5318    6.0700   -0.5007 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.7326    6.3043    0.8485 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.1989    6.9630    1.6234 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.62684\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    6.0766   11.2367   -1.2749 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8567   12.0362   -0.1307 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9469   13.0926   -0.0540 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8276   13.8112    1.0283 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1513   13.3627   -1.1948 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3516   12.5291   -2.3293 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6835   12.5959   -3.5342 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1547   11.6324   -4.3269 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8413   11.2642   -5.6121 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5154   10.1899   -6.2056 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4777    9.5230   -5.4943 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8006    9.8951   -4.1835 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1228   10.9531   -3.6255 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2336   11.5243   -2.3825 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3856   11.8328    0.6743 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3862   13.6135    1.8145 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1789   14.5511    1.0589 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4910   14.0929   -1.2017 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.5729    9.9538   -1.7136 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4410    9.1803   -1.1807 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.6171    7.8152   -0.9115 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5868    7.0600   -0.4124 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3752    7.6740   -0.1820 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2137    9.0219   -0.4517 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2390    9.7957   -0.9537 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.57606\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    5.0653   12.1568   -3.9872 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8604   12.4012   -4.6839 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9703   11.4160   -5.1156 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.8943   11.7181   -5.7887 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2724   10.0686   -4.7985 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4672    9.8343   -4.0639 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9490    8.6156   -3.6332 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.0860    8.8271   -2.9690 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9587    7.9704   -2.3445 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0986    8.4851   -1.7151 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.3255    9.8361   -1.7324 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4383   10.7134   -2.3681 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3270   10.1808   -2.9778 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3092   10.7944   -3.6642 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6309   13.3383   -4.8799 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.6954   12.6582   -6.0029 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2813   11.0060   -6.0825 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.6830    9.3295   -5.0736 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.0458   14.1722   -4.2793 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2733   14.0867   -3.0305 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5425   14.9999   -2.0006 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8430   14.9541   -0.8219 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8701   13.9900   -0.6735 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6136   13.0944   -1.6971 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3075   13.1265   -2.8885 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.48728\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -4.9024   11.4840    0.7331 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6126   11.9069    0.3400 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.4179   11.2889    0.7142 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.2607   11.7621    0.3415 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.5003   10.1240    1.5163 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8035    9.6828    1.8758 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.1233    8.5730    2.6300 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4517    8.5025    2.7266 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2812    7.6082    3.3572 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6681    7.7885    3.2902 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1758    8.8551    2.5964 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3337    9.7696    1.9519 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.9762    9.5681    2.0338 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.9361   10.2941    1.5097 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.5518   12.6983   -0.2425 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.2166   12.5724   -0.2159 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.4315   11.3089    0.6177 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.6936    9.6427    1.8109 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9700   13.4305    0.8105 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8997   12.4880   -0.3166 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8688   12.5637   -1.3277 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8367   11.7042   -2.3958 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8305   10.7648   -2.4519 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.8800   10.6999   -1.4480 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.8974   11.5564   -0.3673 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.46691\\n\\n$$$$\\n\",\"sdf\");\n",
       "\tviewergrid_16835771918631198[0][0].setStyle({\"model\": 0},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewergrid_16835771918631198[0][0].addSurface(1,{\"opacity\": 0.5, \"color\": \"white\"},{\"model\": 0});\n",
       "\tviewergrid_16835771918631198[0][0].setStyle({\"model\": 1},{\"stick\": {}});\n",
       "\tviewergrid_16835771918631198[0][0].zoomTo();\n",
       "\tviewergrid_16835771918631198[0][1].addModel(\"HEADER    ONCOGENE PROTEIN                        30-APR-90   5P21              \\nTITLE     REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION OF H-RAS   \\nTITLE    2 P21 AT 1.35 ANGSTROMS RESOLUTION: IMPLICATIONS FOR THE MECHANISM OF  \\nTITLE    3 GTP HYDROLYSIS                                                       \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: C-H-RAS P21 PROTEIN;                                       \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    ONCOGENE PROTEIN                                                      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    E.F.PAI,A.WITTINGHOFER,W.KABSCH                                       \\nREVDAT   6   29-NOV-17 5P21    1       HELIX                                    \\nREVDAT   5   24-FEB-09 5P21    1       VERSN                                    \\nREVDAT   4   01-APR-03 5P21    1       JRNL                                     \\nREVDAT   3   20-JUL-95 5P21    1       HET                                      \\nREVDAT   2   30-APR-94 5P21    1       FORMUL                                   \\nREVDAT   1   15-JAN-92 5P21    0                                                \\nJRNL        AUTH   E.F.PAI,U.KRENGEL,G.A.PETSKO,R.S.GOODY,W.KABSCH,             \\nJRNL        AUTH 2 A.WITTINGHOFER                                               \\nJRNL        TITL   REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION   \\nJRNL        TITL 2 OF H-RAS P21 AT 1.35 A RESOLUTION: IMPLICATIONS FOR THE      \\nJRNL        TITL 3 MECHANISM OF GTP HYDROLYSIS.                                 \\nJRNL        REF    EMBO J.                       V.   9  2351 1990              \\nJRNL        REFN                   ISSN 0261-4189                               \\nJRNL        PMID   2196171                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,S.C.ALMO,G.RAPP,K.WILSON,K.PETRATOS,A.LENTFER, \\nREMARK   1  AUTH 2 A.WITTINGHOFER,W.KABSCH,E.F.PAI,G.A.PETSKO,R.S.GOODY         \\nREMARK   1  TITL   TIME-RESOLVED X-RAY CRYSTALLOGRAPHIC STUDY OF THE            \\nREMARK   1  TITL 2 CONFORMATIONAL CHANGE IN HA-RAS P21 PROTEIN ON GTP           \\nREMARK   1  TITL 3 HYDROLYSIS                                                   \\nREMARK   1  REF    NATURE                        V. 345   309 1990              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   A.SCHERER,J.JOHN,R.LINKE,R.S.GOODY,A.WITTINGHOFER,E.F.PAI,   \\nREMARK   1  AUTH 2 K.C.HOLMES                                                   \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY ANALYSIS OF THE HUMAN  \\nREMARK   1  TITL 2 C-H-RAS-ONCOGENE PRODUCT P21 COMPLEXED WITH GTP ANALOGUES    \\nREMARK   1  REF    J.MOL.BIOL.                   V. 206   257 1989              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   E.F.PAI,W.KABSCH,U.KRENGEL,K.C.HOLMES,J.JOHN,A.WITTINGHOFER  \\nREMARK   1  TITL   STRUCTURE OF THE GUANINE-NUCLEOTIDE-BINDING DOMAIN OF THE    \\nREMARK   1  TITL 2 HA-RAS ONCOGENE PRODUCT P21 IN THE TRIPHOSPHATE CONFORMATION \\nREMARK   1  REF    NATURE                        V. 341   209 1989              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,G.RAPP,J.JOHN,A.WITTINGHOFER,E.F.PAI,R.S.GOODY \\nREMARK   1  TITL   BIOCHEMICAL AND CRYSTALLOGRAPHIC CHARACTERIZATION OF A       \\nREMARK   1  TITL 2 COMPLEX OF C-HA-RAS P21 AND CAGED GTP WITH FLASH PHOTOLYSIS  \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  86  7687 1989              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 6.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 26806                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.196                           \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1323                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 33                                      \\nREMARK   3   SOLVENT ATOMS            : 211                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.014                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 2.700                           \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  RESIDUES 61 - 65 (GLN - GLU - GLU - TYR - SER) ADOPT                \\nREMARK   3  SEVERAL CONFORMATIONS IN THE CRYSTAL.  THE COORDINATES              \\nREMARK   3  GIVEN APPROXIMATE ONE OF THESE.                                     \\nREMARK   4                                                                      \\nREMARK   4 5P21 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179747.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 38.93                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.01                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 32 2 1                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+2/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+1/3                                           \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z+1/3                                           \\nREMARK 290       6555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000      108.13333            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       54.06667            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       54.06667            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000      108.13333            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025  0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    ARG A  68   NE  -  CZ  -  NH1 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A  97   NE  -  CZ  -  NH1 ANGL. DEV. =  -3.4 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH1 ANGL. DEV. =   4.1 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500    ARG A 128   NE  -  CZ  -  NH2 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A 135   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500    TYR A 157   CB  -  CG  -  CD2 ANGL. DEV. =  -4.8 DEGREES          \\nREMARK 500    ARG A 164   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ASP A  33       75.86   -157.25                                   \\nREMARK 500    ILE A  36      -64.23   -100.39                                   \\nREMARK 500    GLU A  37      123.75   -171.30                                   \\nREMARK 500    SER A  65     -133.70     53.07                                   \\nREMARK 500    LYS A 117       33.80     71.60                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              MG A 168  MG                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 GNP A 167   O2G                                                    \\nREMARK 620 2 GNP A 167   O2B  95.0                                              \\nREMARK 620 3 HOH A 173   O    87.6  91.7                                        \\nREMARK 620 4 SER A  17   OG  172.9  91.0  88.6                                  \\nREMARK 620 5 THR A  35   OG1  91.3 173.4  90.5  82.8                            \\nREMARK 620 6 HOH A 172   O    97.3  88.0 175.0  86.5  89.3                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE MG A 168                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE GNP A 167                 \\nDBREF  5P21 A    1   166  UNP    P01112   RASH_HUMAN       1    166             \\nSEQRES   1 A  166  MET THR GLU TYR LYS LEU VAL VAL VAL GLY ALA GLY GLY          \\nSEQRES   2 A  166  VAL GLY LYS SER ALA LEU THR ILE GLN LEU ILE GLN ASN          \\nSEQRES   3 A  166  HIS PHE VAL ASP GLU TYR ASP PRO THR ILE GLU ASP SER          \\nSEQRES   4 A  166  TYR ARG LYS GLN VAL VAL ILE ASP GLY GLU THR CYS LEU          \\nSEQRES   5 A  166  LEU ASP ILE LEU ASP THR ALA GLY GLN GLU GLU TYR SER          \\nSEQRES   6 A  166  ALA MET ARG ASP GLN TYR MET ARG THR GLY GLU GLY PHE          \\nSEQRES   7 A  166  LEU CYS VAL PHE ALA ILE ASN ASN THR LYS SER PHE GLU          \\nSEQRES   8 A  166  ASP ILE HIS GLN TYR ARG GLU GLN ILE LYS ARG VAL LYS          \\nSEQRES   9 A  166  ASP SER ASP ASP VAL PRO MET VAL LEU VAL GLY ASN LYS          \\nSEQRES  10 A  166  CYS ASP LEU ALA ALA ARG THR VAL GLU SER ARG GLN ALA          \\nSEQRES  11 A  166  GLN ASP LEU ALA ARG SER TYR GLY ILE PRO TYR ILE GLU          \\nSEQRES  12 A  166  THR SER ALA LYS THR ARG GLN GLY VAL GLU ASP ALA PHE          \\nSEQRES  13 A  166  TYR THR LEU VAL ARG GLU ILE ARG GLN HIS                      \\nHET     MG  A 168       1                                                       \\nHET    GNP  A 167      32                                                       \\nHETNAM      MG MAGNESIUM ION                                                    \\nHETNAM     GNP PHOSPHOAMINOPHOSPHONIC ACID-GUANYLATE ESTER                      \\nFORMUL   2   MG    MG 2+                                                        \\nFORMUL   3  GNP    C10 H17 N6 O13 P3                                            \\nFORMUL   4  HOH   *211(H2 O)                                                    \\nHELIX    1  HA LYS A   16  GLN A   25  1                                  10    \\nHELIX    2  HB SER A   65  THR A   74  1RES 65-67 IN 3/10 CONF            10    \\nHELIX    3  HC THR A   87  VAL A  103  1BREAK AT ASP 92                   17    \\nHELIX    4  HD SER A  127  TYR A  137  1                                  11    \\nHELIX    5  HE VAL A  152  ARG A  164  1                                  13    \\nSHEET    1  S1 6 GLU A  37  ILE A  46  0                                        \\nSHEET    2  S1 6 GLU A  49  THR A  58 -1  O  LEU A  53   N  LYS A  42           \\nSHEET    3  S1 6 THR A   2  VAL A   9  1  N  LEU A   6   O  ASP A  54           \\nSHEET    4  S1 6 GLY A  77  ALA A  83  1  N  LEU A  79   O  VAL A   7           \\nSHEET    5  S1 6 MET A 111  ASN A 116  1  N  VAL A 114   O  CYS A  80           \\nSHEET    6  S1 6 TYR A 141  GLU A 143  1  N  ILE A 142   O  LEU A 113           \\nLINK        MG    MG A 168                 O2G GNP A 167     1555   1555  2.20  \\nLINK        MG    MG A 168                 O2B GNP A 167     1555   1555  2.22  \\nLINK        MG    MG A 168                 O   HOH A 173     1555   1555  2.23  \\nLINK        MG    MG A 168                 OG  SER A  17     1555   1555  2.26  \\nLINK        MG    MG A 168                 OG1 THR A  35     1555   1555  2.26  \\nLINK        MG    MG A 168                 O   HOH A 172     1555   1555  2.30  \\nSITE     1 AC1  5 SER A  17  THR A  35  GNP A 167  HOH A 172                    \\nSITE     2 AC1  5 HOH A 173                                                     \\nSITE     1 AC2 28 GLY A  12  GLY A  13  VAL A  14  GLY A  15                    \\nSITE     2 AC2 28 LYS A  16  SER A  17  ALA A  18  PHE A  28                    \\nSITE     3 AC2 28 VAL A  29  ASP A  30  TYR A  32  PRO A  34                    \\nSITE     4 AC2 28 THR A  35  GLY A  60  ASN A 116  LYS A 117                    \\nSITE     5 AC2 28 ASP A 119  SER A 145  ALA A 146  LYS A 147                    \\nSITE     6 AC2 28  MG A 168  HOH A 170  HOH A 172  HOH A 173                    \\nSITE     7 AC2 28 HOH A 175  HOH A 187  HOH A 281  HOH A 366                    \\nCRYST1   40.300   40.300  162.200  90.00  90.00 120.00 P 32 2 1      6          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.024814  0.014326  0.000000        0.00000                         \\nSCALE2      0.000000  0.028653  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.006165        0.00000                         \\nATOM      1  N   MET A   1      -7.186  32.862  -6.632  1.00 13.92           N  \\nATOM      2  CA  MET A   1      -5.921  32.127  -6.684  1.00 17.10           C  \\nATOM      3  C   MET A   1      -5.845  31.045  -5.621  1.00 16.74           C  \\nATOM      4  O   MET A   1      -6.600  31.083  -4.636  1.00 16.60           O  \\nATOM      5  CB  MET A   1      -4.673  33.058  -6.502  1.00 17.00           C  \\nATOM      6  CG  MET A   1      -4.633  33.804  -5.188  1.00 18.08           C  \\nATOM      7  SD  MET A   1      -3.115  34.766  -4.908  1.00 19.73           S  \\nATOM      8  CE  MET A   1      -3.019  35.851  -6.310  1.00 20.41           C  \\nATOM      9  N   THR A   2      -4.910  30.095  -5.820  1.00 14.70           N  \\nATOM     10  CA  THR A   2      -4.661  29.030  -4.859  1.00 16.92           C  \\nATOM     11  C   THR A   2      -4.230  29.655  -3.547  1.00 14.78           C  \\nATOM     12  O   THR A   2      -3.338  30.501  -3.537  1.00 13.93           O  \\nATOM     13  CB  THR A   2      -3.557  28.107  -5.401  1.00 18.03           C  \\nATOM     14  OG1 THR A   2      -4.059  27.565  -6.629  1.00 20.42           O  \\nATOM     15  CG2 THR A   2      -3.193  26.983  -4.443  1.00 18.51           C  \\nATOM     16  N   GLU A   3      -4.911  29.290  -2.472  1.00 14.80           N  \\nATOM     17  CA  GLU A   3      -4.504  29.730  -1.135  1.00 17.71           C  \\nATOM     18  C   GLU A   3      -4.012  28.553  -0.278  1.00 17.33           C  \\nATOM     19  O   GLU A   3      -4.520  27.427  -0.395  1.00 17.54           O  \\nATOM     20  CB  GLU A   3      -5.655  30.402  -0.432  1.00 20.67           C  \\nATOM     21  CG  GLU A   3      -6.213  31.631  -1.186  1.00 25.56           C  \\nATOM     22  CD  GLU A   3      -7.128  32.563  -0.369  1.00 28.46           C  \\nATOM     23  OE1 GLU A   3      -7.578  32.216   0.728  1.00 27.72           O  \\nATOM     24  OE2 GLU A   3      -7.375  33.680  -0.827  1.00 32.49           O  \\nATOM     25  N   TYR A   4      -2.962  28.744   0.542  1.00 14.50           N  \\nATOM     26  CA  TYR A   4      -2.425  27.654   1.358  1.00 12.29           C  \\nATOM     27  C   TYR A   4      -2.534  28.103   2.807  1.00 12.52           C  \\nATOM     28  O   TYR A   4      -2.016  29.143   3.200  1.00 12.11           O  \\nATOM     29  CB  TYR A   4      -0.975  27.385   1.026  1.00 11.46           C  \\nATOM     30  CG  TYR A   4      -0.726  26.846  -0.387  1.00 12.38           C  \\nATOM     31  CD1 TYR A   4      -0.297  27.698  -1.395  1.00 13.00           C  \\nATOM     32  CD2 TYR A   4      -0.933  25.501  -0.649  1.00 14.09           C  \\nATOM     33  CE1 TYR A   4      -0.070  27.208  -2.674  1.00 13.89           C  \\nATOM     34  CE2 TYR A   4      -0.711  24.993  -1.917  1.00 14.65           C  \\nATOM     35  CZ  TYR A   4      -0.279  25.859  -2.917  1.00 14.84           C  \\nATOM     36  OH  TYR A   4      -0.084  25.357  -4.191  1.00 15.07           O  \\nATOM     37  N   LYS A   5      -3.274  27.305   3.568  1.00 13.07           N  \\nATOM     38  CA  LYS A   5      -3.554  27.569   4.970  1.00 12.96           C  \\nATOM     39  C   LYS A   5      -2.556  26.818   5.852  1.00 12.74           C  \\nATOM     40  O   LYS A   5      -2.645  25.611   6.092  1.00 11.58           O  \\nATOM     41  CB  LYS A   5      -4.972  27.131   5.281  1.00 17.65           C  \\nATOM     42  CG  LYS A   5      -6.053  27.833   4.446  1.00 23.49           C  \\nATOM     43  CD  LYS A   5      -7.186  28.310   5.335  1.00 29.10           C  \\nATOM     44  CE  LYS A   5      -6.655  29.309   6.403  1.00 33.68           C  \\nATOM     45  NZ  LYS A   5      -6.597  28.739   7.755  1.00 36.52           N  \\nATOM     46  N   LEU A   6      -1.543  27.574   6.297  1.00 13.08           N  \\nATOM     47  CA  LEU A   6      -0.418  27.011   7.064  1.00 10.34           C  \\nATOM     48  C   LEU A   6      -0.578  27.382   8.535  1.00  9.68           C  \\nATOM     49  O   LEU A   6      -1.034  28.483   8.863  1.00 11.31           O  \\nATOM     50  CB  LEU A   6       0.910  27.571   6.536  1.00 11.34           C  \\nATOM     51  CG  LEU A   6       1.137  27.565   5.017  1.00 11.62           C  \\nATOM     52  CD1 LEU A   6       2.515  28.119   4.720  1.00 13.60           C  \\nATOM     53  CD2 LEU A   6       0.973  26.171   4.475  1.00 12.19           C  \\nATOM     54  N   VAL A   7      -0.196  26.486   9.432  1.00  9.93           N  \\nATOM     55  CA  VAL A   7      -0.333  26.787  10.852  1.00 10.06           C  \\nATOM     56  C   VAL A   7       1.033  26.513  11.495  1.00 10.37           C  \\nATOM     57  O   VAL A   7       1.597  25.441  11.252  1.00  9.51           O  \\nATOM     58  CB  VAL A   7      -1.474  25.884  11.446  1.00 10.97           C  \\nATOM     59  CG1 VAL A   7      -1.610  26.163  12.943  1.00 11.39           C  \\nATOM     60  CG2 VAL A   7      -2.809  26.161  10.777  1.00 10.14           C  \\nATOM     61  N   VAL A   8       1.576  27.455  12.295  1.00  8.35           N  \\nATOM     62  CA  VAL A   8       2.878  27.262  12.944  1.00  6.92           C  \\nATOM     63  C   VAL A   8       2.609  26.896  14.405  1.00  8.84           C  \\nATOM     64  O   VAL A   8       1.930  27.663  15.092  1.00  9.21           O  \\nATOM     65  CB  VAL A   8       3.684  28.565  12.799  1.00  8.08           C  \\nATOM     66  CG1 VAL A   8       5.111  28.324  13.261  1.00  9.52           C  \\nATOM     67  CG2 VAL A   8       3.729  29.011  11.358  1.00  8.43           C  \\nATOM     68  N   VAL A   9       3.024  25.701  14.881  1.00  8.73           N  \\nATOM     69  CA  VAL A   9       2.779  25.234  16.245  1.00  8.41           C  \\nATOM     70  C   VAL A   9       4.075  24.806  16.919  1.00  8.21           C  \\nATOM     71  O   VAL A   9       5.093  24.551  16.287  1.00 10.97           O  \\nATOM     72  CB  VAL A   9       1.824  24.028  16.286  1.00  9.74           C  \\nATOM     73  CG1 VAL A   9       0.545  24.491  15.680  1.00 11.76           C  \\nATOM     74  CG2 VAL A   9       2.349  22.810  15.535  1.00 10.31           C  \\nATOM     75  N   GLY A  10       4.048  24.707  18.238  1.00  7.34           N  \\nATOM     76  CA  GLY A  10       5.231  24.336  19.009  1.00  7.98           C  \\nATOM     77  C   GLY A  10       5.192  25.053  20.355  1.00  8.31           C  \\nATOM     78  O   GLY A  10       4.417  25.994  20.595  1.00  9.89           O  \\nATOM     79  N   ALA A  11       6.099  24.638  21.226  1.00  9.31           N  \\nATOM     80  CA  ALA A  11       6.121  25.135  22.597  1.00  9.50           C  \\nATOM     81  C   ALA A  11       6.440  26.625  22.711  1.00  9.66           C  \\nATOM     82  O   ALA A  11       6.998  27.250  21.806  1.00  9.46           O  \\nATOM     83  CB  ALA A  11       7.145  24.306  23.378  1.00  9.85           C  \\nATOM     84  N   GLY A  12       6.111  27.227  23.847  1.00 10.36           N  \\nATOM     85  CA  GLY A  12       6.406  28.632  24.054  1.00  9.30           C  \\nATOM     86  C   GLY A  12       7.892  28.950  23.964  1.00 10.23           C  \\nATOM     87  O   GLY A  12       8.785  28.249  24.466  1.00 10.57           O  \\nATOM     88  N   GLY A  13       8.159  30.064  23.266  1.00 11.49           N  \\nATOM     89  CA  GLY A  13       9.511  30.602  23.139  1.00 11.29           C  \\nATOM     90  C   GLY A  13      10.436  29.950  22.108  1.00 12.22           C  \\nATOM     91  O   GLY A  13      11.624  30.321  22.060  1.00 13.62           O  \\nATOM     92  N   VAL A  14       9.961  28.991  21.286  1.00  9.23           N  \\nATOM     93  CA  VAL A  14      10.848  28.327  20.317  1.00 10.38           C  \\nATOM     94  C   VAL A  14      11.169  29.168  19.076  1.00 10.47           C  \\nATOM     95  O   VAL A  14      12.108  28.841  18.353  1.00 12.07           O  \\nATOM     96  CB  VAL A  14      10.246  26.970  19.856  1.00  9.98           C  \\nATOM     97  CG1 VAL A  14      10.187  26.074  21.120  1.00  9.93           C  \\nATOM     98  CG2 VAL A  14       8.866  27.127  19.200  1.00  9.52           C  \\nATOM     99  N   GLY A  15      10.423  30.264  18.834  1.00  9.61           N  \\nATOM    100  CA  GLY A  15      10.666  31.160  17.706  1.00  9.70           C  \\nATOM    101  C   GLY A  15       9.615  31.061  16.626  1.00  8.82           C  \\nATOM    102  O   GLY A  15       9.915  31.403  15.491  1.00  8.99           O  \\nATOM    103  N   LYS A  16       8.395  30.591  16.916  1.00  8.18           N  \\nATOM    104  CA  LYS A  16       7.331  30.521  15.913  1.00  8.44           C  \\nATOM    105  C   LYS A  16       7.060  31.899  15.309  1.00  9.01           C  \\nATOM    106  O   LYS A  16       7.020  32.044  14.086  1.00  8.25           O  \\nATOM    107  CB  LYS A  16       6.014  30.000  16.529  1.00  9.27           C  \\nATOM    108  CG  LYS A  16       6.132  28.585  17.086  1.00  9.11           C  \\nATOM    109  CD  LYS A  16       4.820  28.077  17.726  1.00  9.31           C  \\nATOM    110  CE  LYS A  16       4.351  28.934  18.888  1.00 10.05           C  \\nATOM    111  NZ  LYS A  16       5.221  28.829  20.051  1.00  9.64           N  \\nATOM    112  N   SER A  17       6.913  32.942  16.135  1.00  7.74           N  \\nATOM    113  CA  SER A  17       6.661  34.295  15.649  1.00  8.78           C  \\nATOM    114  C   SER A  17       7.854  34.859  14.919  1.00  9.73           C  \\nATOM    115  O   SER A  17       7.653  35.446  13.868  1.00  9.70           O  \\nATOM    116  CB  SER A  17       6.360  35.264  16.769  1.00  8.22           C  \\nATOM    117  OG  SER A  17       5.083  34.853  17.204  1.00  8.79           O  \\nATOM    118  N   ALA A  18       9.072  34.687  15.444  1.00  8.95           N  \\nATOM    119  CA  ALA A  18      10.253  35.199  14.769  1.00  8.95           C  \\nATOM    120  C   ALA A  18      10.438  34.576  13.374  1.00  8.54           C  \\nATOM    121  O   ALA A  18      10.832  35.254  12.430  1.00  8.75           O  \\nATOM    122  CB  ALA A  18      11.521  34.931  15.640  1.00 10.11           C  \\nATOM    123  N   LEU A  19      10.136  33.284  13.176  1.00  8.31           N  \\nATOM    124  CA  LEU A  19      10.225  32.667  11.837  1.00  9.30           C  \\nATOM    125  C   LEU A  19       9.204  33.281  10.884  1.00 10.98           C  \\nATOM    126  O   LEU A  19       9.534  33.652   9.751  1.00 10.47           O  \\nATOM    127  CB  LEU A  19       9.938  31.165  11.885  1.00  9.41           C  \\nATOM    128  CG  LEU A  19      11.035  30.275  12.487  1.00 11.80           C  \\nATOM    129  CD1 LEU A  19      10.491  28.866  12.723  1.00 10.43           C  \\nATOM    130  CD2 LEU A  19      12.256  30.288  11.554  1.00 13.42           C  \\nATOM    131  N   THR A  20       7.950  33.419  11.335  1.00 10.72           N  \\nATOM    132  CA  THR A  20       6.888  34.015  10.529  1.00 11.87           C  \\nATOM    133  C   THR A  20       7.173  35.492  10.161  1.00 12.65           C  \\nATOM    134  O   THR A  20       7.012  35.917   9.009  1.00 12.60           O  \\nATOM    135  CB  THR A  20       5.547  33.888  11.312  1.00 13.43           C  \\nATOM    136  OG1 THR A  20       5.299  32.509  11.544  1.00 12.67           O  \\nATOM    137  CG2 THR A  20       4.346  34.410  10.518  1.00 14.64           C  \\nATOM    138  N   ILE A  21       7.615  36.326  11.098  1.00 10.48           N  \\nATOM    139  CA  ILE A  21       7.945  37.704  10.795  1.00 12.96           C  \\nATOM    140  C   ILE A  21       9.197  37.831   9.927  1.00 14.26           C  \\nATOM    141  O   ILE A  21       9.237  38.691   9.042  1.00 15.83           O  \\nATOM    142  CB  ILE A  21       8.090  38.451  12.121  1.00 12.57           C  \\nATOM    143  CG1 ILE A  21       6.691  38.551  12.737  1.00 13.24           C  \\nATOM    144  CG2 ILE A  21       8.631  39.852  11.910  1.00 13.40           C  \\nATOM    145  CD1 ILE A  21       6.636  39.185  14.129  1.00 16.46           C  \\nATOM    146  N   GLN A  22      10.206  36.979  10.092  1.00 13.58           N  \\nATOM    147  CA  GLN A  22      11.330  36.983   9.179  1.00 14.91           C  \\nATOM    148  C   GLN A  22      10.817  36.729   7.736  1.00 17.29           C  \\nATOM    149  O   GLN A  22      11.120  37.485   6.792  1.00 15.91           O  \\nATOM    150  CB  GLN A  22      12.270  35.930   9.736  1.00 15.90           C  \\nATOM    151  CG  GLN A  22      13.686  35.878   9.230  1.00 19.36           C  \\nATOM    152  CD  GLN A  22      14.372  37.217   9.091  1.00 18.08           C  \\nATOM    153  OE1 GLN A  22      14.160  37.984   8.143  1.00 19.23           O  \\nATOM    154  NE2 GLN A  22      15.224  37.492  10.050  1.00 17.40           N  \\nATOM    155  N   LEU A  23       9.944  35.718   7.523  1.00 15.95           N  \\nATOM    156  CA  LEU A  23       9.381  35.475   6.194  1.00 18.85           C  \\nATOM    157  C   LEU A  23       8.539  36.622   5.636  1.00 18.21           C  \\nATOM    158  O   LEU A  23       8.695  36.958   4.469  1.00 19.03           O  \\nATOM    159  CB  LEU A  23       8.498  34.235   6.196  1.00 18.10           C  \\nATOM    160  CG  LEU A  23       7.916  33.797   4.863  1.00 18.64           C  \\nATOM    161  CD1 LEU A  23       8.921  32.994   4.030  1.00 18.03           C  \\nATOM    162  CD2 LEU A  23       6.680  32.991   5.183  1.00 21.34           C  \\nATOM    163  N   ILE A  24       7.634  37.197   6.428  1.00 19.25           N  \\nATOM    164  CA  ILE A  24       6.682  38.198   5.953  1.00 20.23           C  \\nATOM    165  C   ILE A  24       7.316  39.590   5.922  1.00 21.89           C  \\nATOM    166  O   ILE A  24       7.125  40.318   4.943  1.00 22.93           O  \\nATOM    167  CB  ILE A  24       5.399  38.222   6.881  1.00 21.20           C  \\nATOM    168  CG1 ILE A  24       4.664  36.907   6.821  1.00 21.27           C  \\nATOM    169  CG2 ILE A  24       4.423  39.307   6.446  1.00 20.00           C  \\nATOM    170  CD1 ILE A  24       4.128  36.442   5.475  1.00 23.98           C  \\nATOM    171  N   GLN A  25       8.074  39.998   6.953  1.00 21.07           N  \\nATOM    172  CA  GLN A  25       8.520  41.380   7.066  1.00 22.09           C  \\nATOM    173  C   GLN A  25      10.003  41.559   6.962  1.00 20.75           C  \\nATOM    174  O   GLN A  25      10.495  42.677   7.054  1.00 20.26           O  \\nATOM    175  CB  GLN A  25       8.039  41.968   8.398  1.00 23.96           C  \\nATOM    176  CG  GLN A  25       6.548  42.276   8.381  1.00 28.69           C  \\nATOM    177  CD  GLN A  25       5.931  42.574   9.744  1.00 30.92           C  \\nATOM    178  OE1 GLN A  25       6.537  43.143  10.657  1.00 31.89           O  \\nATOM    179  NE2 GLN A  25       4.664  42.169   9.867  1.00 32.35           N  \\nATOM    180  N   ASN A  26      10.764  40.482   6.795  1.00 22.13           N  \\nATOM    181  CA  ASN A  26      12.198  40.642   6.628  1.00 24.67           C  \\nATOM    182  C   ASN A  26      12.936  41.362   7.734  1.00 23.62           C  \\nATOM    183  O   ASN A  26      13.603  42.372   7.521  1.00 25.10           O  \\nATOM    184  CB  ASN A  26      12.522  41.369   5.315  1.00 27.68           C  \\nATOM    185  CG  ASN A  26      12.942  40.351   4.288  1.00 32.80           C  \\nATOM    186  OD1 ASN A  26      12.135  39.699   3.602  1.00 33.77           O  \\nATOM    187  ND2 ASN A  26      14.268  40.203   4.237  1.00 34.26           N  \\nATOM    188  N   HIS A  27      12.764  40.842   8.945  1.00 19.95           N  \\nATOM    189  CA  HIS A  27      13.593  41.266  10.057  1.00 17.99           C  \\nATOM    190  C   HIS A  27      13.441  40.299  11.215  1.00 16.84           C  \\nATOM    191  O   HIS A  27      12.411  39.623  11.344  1.00 14.72           O  \\nATOM    192  CB  HIS A  27      13.241  42.680  10.551  1.00 19.80           C  \\nATOM    193  CG  HIS A  27      11.926  42.908  11.258  1.00 20.77           C  \\nATOM    194  ND1 HIS A  27      11.936  43.314  12.510  1.00 22.17           N  \\nATOM    195  CD2 HIS A  27      10.674  42.768  10.729  1.00 22.32           C  \\nATOM    196  CE1 HIS A  27      10.657  43.430  12.785  1.00 23.86           C  \\nATOM    197  NE2 HIS A  27       9.905  43.110  11.738  1.00 23.97           N  \\nATOM    198  N   PHE A  28      14.516  40.222  12.012  1.00 15.44           N  \\nATOM    199  CA  PHE A  28      14.528  39.413  13.223  1.00 13.89           C  \\nATOM    200  C   PHE A  28      14.026  40.196  14.440  1.00 15.33           C  \\nATOM    201  O   PHE A  28      14.528  41.255  14.786  1.00 16.80           O  \\nATOM    202  CB  PHE A  28      15.938  38.906  13.470  1.00 12.73           C  \\nATOM    203  CG  PHE A  28      16.083  38.150  14.786  1.00 13.35           C  \\nATOM    204  CD1 PHE A  28      15.354  36.990  15.012  1.00 13.80           C  \\nATOM    205  CD2 PHE A  28      16.974  38.615  15.740  1.00 14.54           C  \\nATOM    206  CE1 PHE A  28      15.534  36.310  16.194  1.00 14.45           C  \\nATOM    207  CE2 PHE A  28      17.147  37.925  16.920  1.00 14.61           C  \\nATOM    208  CZ  PHE A  28      16.420  36.768  17.137  1.00 14.65           C  \\nATOM    209  N   VAL A  29      12.970  39.697  15.089  1.00 17.22           N  \\nATOM    210  CA  VAL A  29      12.467  40.313  16.294  1.00 19.11           C  \\nATOM    211  C   VAL A  29      13.127  39.606  17.476  1.00 19.36           C  \\nATOM    212  O   VAL A  29      13.245  38.376  17.605  1.00 18.14           O  \\nATOM    213  CB  VAL A  29      10.894  40.205  16.412  1.00 22.03           C  \\nATOM    214  CG1 VAL A  29      10.334  40.869  15.143  1.00 19.66           C  \\nATOM    215  CG2 VAL A  29      10.377  38.776  16.631  1.00 23.15           C  \\nATOM    216  N   ASP A  30      13.615  40.484  18.324  1.00 20.14           N  \\nATOM    217  CA  ASP A  30      14.229  40.019  19.518  1.00 24.04           C  \\nATOM    218  C   ASP A  30      13.676  40.746  20.721  1.00 25.93           C  \\nATOM    219  O   ASP A  30      14.244  41.709  21.238  1.00 29.26           O  \\nATOM    220  CB  ASP A  30      15.719  40.203  19.443  1.00 27.10           C  \\nATOM    221  CG  ASP A  30      16.389  39.731  20.732  1.00 32.46           C  \\nATOM    222  OD1 ASP A  30      15.878  38.859  21.464  1.00 34.28           O  \\nATOM    223  OD2 ASP A  30      17.458  40.264  21.002  1.00 36.46           O  \\nATOM    224  N   GLU A  31      12.472  40.347  21.076  1.00 26.14           N  \\nATOM    225  CA  GLU A  31      11.922  40.804  22.318  1.00 28.72           C  \\nATOM    226  C   GLU A  31      10.870  39.836  22.749  1.00 26.20           C  \\nATOM    227  O   GLU A  31      10.466  38.931  22.014  1.00 23.14           O  \\nATOM    228  CB  GLU A  31      11.337  42.210  22.204  1.00 33.25           C  \\nATOM    229  CG  GLU A  31      10.350  42.618  21.152  1.00 40.08           C  \\nATOM    230  CD  GLU A  31      10.572  44.107  20.887  1.00 44.71           C  \\nATOM    231  OE1 GLU A  31      10.930  44.444  19.755  1.00 47.50           O  \\nATOM    232  OE2 GLU A  31      10.420  44.928  21.806  1.00 47.89           O  \\nATOM    233  N   TYR A  32      10.550  40.044  24.028  1.00 22.26           N  \\nATOM    234  CA  TYR A  32       9.597  39.196  24.685  1.00 18.83           C  \\nATOM    235  C   TYR A  32       8.204  39.688  24.317  1.00 17.53           C  \\nATOM    236  O   TYR A  32       7.794  40.792  24.676  1.00 18.45           O  \\nATOM    237  CB  TYR A  32       9.909  39.259  26.199  1.00 16.84           C  \\nATOM    238  CG  TYR A  32       9.076  38.309  27.052  1.00 15.64           C  \\nATOM    239  CD1 TYR A  32       9.598  37.096  27.433  1.00 16.17           C  \\nATOM    240  CD2 TYR A  32       7.814  38.686  27.459  1.00 15.54           C  \\nATOM    241  CE1 TYR A  32       8.845  36.251  28.239  1.00 17.86           C  \\nATOM    242  CE2 TYR A  32       7.051  37.861  28.257  1.00 14.40           C  \\nATOM    243  CZ  TYR A  32       7.572  36.635  28.652  1.00 16.47           C  \\nATOM    244  OH  TYR A  32       6.821  35.789  29.485  1.00 13.59           O  \\nATOM    245  N   ASP A  33       7.461  38.841  23.604  1.00 13.43           N  \\nATOM    246  CA  ASP A  33       6.128  39.200  23.141  1.00 14.00           C  \\nATOM    247  C   ASP A  33       5.262  37.955  22.854  1.00 13.56           C  \\nATOM    248  O   ASP A  33       5.024  37.586  21.681  1.00 12.47           O  \\nATOM    249  CB  ASP A  33       6.271  40.072  21.864  1.00 15.92           C  \\nATOM    250  CG  ASP A  33       4.993  40.654  21.269  1.00 18.19           C  \\nATOM    251  OD1 ASP A  33       3.923  40.631  21.879  1.00 21.46           O  \\nATOM    252  OD2 ASP A  33       5.074  41.164  20.167  1.00 20.34           O  \\nATOM    253  N   PRO A  34       4.752  37.283  23.903  1.00 12.04           N  \\nATOM    254  CA  PRO A  34       3.949  36.070  23.788  1.00 11.09           C  \\nATOM    255  C   PRO A  34       2.720  36.248  22.895  1.00 11.76           C  \\nATOM    256  O   PRO A  34       1.953  37.220  23.006  1.00 12.63           O  \\nATOM    257  CB  PRO A  34       3.590  35.711  25.219  1.00 12.50           C  \\nATOM    258  CG  PRO A  34       4.643  36.418  26.033  1.00 12.14           C  \\nATOM    259  CD  PRO A  34       4.817  37.717  25.311  1.00 12.82           C  \\nATOM    260  N   THR A  35       2.554  35.282  21.972  1.00 11.00           N  \\nATOM    261  CA  THR A  35       1.421  35.320  21.033  1.00 11.54           C  \\nATOM    262  C   THR A  35       0.140  34.746  21.639  1.00 13.51           C  \\nATOM    263  O   THR A  35       0.174  33.820  22.450  1.00 12.45           O  \\nATOM    264  CB  THR A  35       1.730  34.518  19.751  1.00 11.34           C  \\nATOM    265  OG1 THR A  35       2.900  35.134  19.233  1.00 11.20           O  \\nATOM    266  CG2 THR A  35       0.706  34.611  18.637  1.00 13.10           C  \\nATOM    267  N   ILE A  36      -0.996  35.332  21.233  1.00 15.48           N  \\nATOM    268  CA  ILE A  36      -2.308  34.792  21.567  1.00 19.42           C  \\nATOM    269  C   ILE A  36      -2.663  34.079  20.274  1.00 17.36           C  \\nATOM    270  O   ILE A  36      -2.749  32.863  20.256  1.00 18.29           O  \\nATOM    271  CB  ILE A  36      -3.353  35.914  21.879  1.00 21.30           C  \\nATOM    272  CG1 ILE A  36      -2.930  36.823  23.072  1.00 24.32           C  \\nATOM    273  CG2 ILE A  36      -4.670  35.207  22.136  1.00 21.54           C  \\nATOM    274  CD1 ILE A  36      -2.494  36.167  24.413  1.00 24.70           C  \\nATOM    275  N   GLU A  37      -2.818  34.822  19.181  1.00 19.35           N  \\nATOM    276  CA  GLU A  37      -3.059  34.221  17.889  1.00 21.47           C  \\nATOM    277  C   GLU A  37      -2.957  35.362  16.897  1.00 23.77           C  \\nATOM    278  O   GLU A  37      -3.636  36.393  17.031  1.00 25.49           O  \\nATOM    279  CB  GLU A  37      -4.440  33.611  17.880  1.00 25.97           C  \\nATOM    280  CG  GLU A  37      -4.766  32.776  16.667  1.00 33.81           C  \\nATOM    281  CD  GLU A  37      -6.160  32.154  16.777  1.00 38.37           C  \\nATOM    282  OE1 GLU A  37      -7.095  32.664  16.149  1.00 37.86           O  \\nATOM    283  OE2 GLU A  37      -6.298  31.164  17.515  1.00 42.24           O  \\nATOM    284  N   ASP A  38      -2.070  35.212  15.911  1.00 19.96           N  \\nATOM    285  CA  ASP A  38      -1.899  36.178  14.839  1.00 17.94           C  \\nATOM    286  C   ASP A  38      -2.024  35.526  13.473  1.00 18.21           C  \\nATOM    287  O   ASP A  38      -1.582  34.393  13.294  1.00 16.96           O  \\nATOM    288  CB  ASP A  38      -0.540  36.833  14.965  1.00 18.83           C  \\nATOM    289  CG  ASP A  38      -0.507  37.817  16.133  1.00 20.08           C  \\nATOM    290  OD1 ASP A  38      -1.390  38.660  16.208  1.00 25.47           O  \\nATOM    291  OD2 ASP A  38       0.375  37.781  16.967  1.00 20.08           O  \\nATOM    292  N   SER A  39      -2.698  36.170  12.507  1.00 17.82           N  \\nATOM    293  CA  SER A  39      -2.691  35.658  11.133  1.00 18.16           C  \\nATOM    294  C   SER A  39      -2.012  36.627  10.171  1.00 16.95           C  \\nATOM    295  O   SER A  39      -2.053  37.853  10.333  1.00 15.80           O  \\nATOM    296  CB  SER A  39      -4.094  35.402  10.637  1.00 23.01           C  \\nATOM    297  OG  SER A  39      -4.873  36.571  10.754  1.00 27.13           O  \\nATOM    298  N   TYR A  40      -1.336  36.080   9.168  1.00 15.44           N  \\nATOM    299  CA  TYR A  40      -0.607  36.889   8.196  1.00 16.25           C  \\nATOM    300  C   TYR A  40      -1.018  36.383   6.830  1.00 17.06           C  \\nATOM    301  O   TYR A  40      -1.160  35.172   6.638  1.00 18.29           O  \\nATOM    302  CB  TYR A  40       0.909  36.724   8.370  1.00 16.59           C  \\nATOM    303  CG  TYR A  40       1.439  37.242   9.707  1.00 16.10           C  \\nATOM    304  CD1 TYR A  40       1.473  36.412  10.815  1.00 16.77           C  \\nATOM    305  CD2 TYR A  40       1.883  38.549   9.799  1.00 18.02           C  \\nATOM    306  CE1 TYR A  40       1.950  36.893  12.021  1.00 16.86           C  \\nATOM    307  CE2 TYR A  40       2.368  39.032  11.001  1.00 18.70           C  \\nATOM    308  CZ  TYR A  40       2.396  38.199  12.104  1.00 17.62           C  \\nATOM    309  OH  TYR A  40       2.853  38.695  13.304  1.00 18.37           O  \\nATOM    310  N   ARG A  41      -1.270  37.303   5.900  1.00 15.48           N  \\nATOM    311  CA  ARG A  41      -1.660  36.918   4.558  1.00 15.42           C  \\nATOM    312  C   ARG A  41      -0.730  37.606   3.586  1.00 16.76           C  \\nATOM    313  O   ARG A  41      -0.536  38.829   3.633  1.00 17.01           O  \\nATOM    314  CB  ARG A  41      -3.087  37.342   4.277  1.00 17.64           C  \\nATOM    315  CG  ARG A  41      -4.150  36.507   4.947  1.00 18.09           C  \\nATOM    316  CD  ARG A  41      -5.528  37.181   4.849  1.00 19.64           C  \\nATOM    317  NE  ARG A  41      -6.066  37.318   3.489  1.00 20.11           N  \\nATOM    318  CZ  ARG A  41      -6.660  36.329   2.803  1.00 22.17           C  \\nATOM    319  NH1 ARG A  41      -7.219  36.571   1.617  1.00 22.48           N  \\nATOM    320  NH2 ARG A  41      -6.711  35.090   3.274  1.00 23.55           N  \\nATOM    321  N   LYS A  42      -0.135  36.827   2.679  1.00 16.52           N  \\nATOM    322  CA  LYS A  42       0.835  37.405   1.758  1.00 16.59           C  \\nATOM    323  C   LYS A  42       0.740  36.746   0.387  1.00 16.04           C  \\nATOM    324  O   LYS A  42       0.794  35.528   0.307  1.00 14.74           O  \\nATOM    325  CB  LYS A  42       2.208  37.210   2.401  1.00 18.22           C  \\nATOM    326  CG  LYS A  42       3.388  37.784   1.669  1.00 22.68           C  \\nATOM    327  CD  LYS A  42       3.369  39.274   1.792  1.00 28.09           C  \\nATOM    328  CE  LYS A  42       4.695  39.876   1.317  1.00 31.70           C  \\nATOM    329  NZ  LYS A  42       4.828  39.827  -0.128  1.00 35.70           N  \\nATOM    330  N   GLN A  43       0.561  37.510  -0.697  1.00 16.89           N  \\nATOM    331  CA  GLN A  43       0.579  36.950  -2.047  1.00 19.10           C  \\nATOM    332  C   GLN A  43       2.026  36.729  -2.448  1.00 15.08           C  \\nATOM    333  O   GLN A  43       2.828  37.654  -2.362  1.00 15.72           O  \\nATOM    334  CB  GLN A  43      -0.116  37.908  -3.043  1.00 23.32           C  \\nATOM    335  CG  GLN A  43       0.543  38.332  -4.415  1.00 32.83           C  \\nATOM    336  CD  GLN A  43       1.081  37.332  -5.482  1.00 36.11           C  \\nATOM    337  OE1 GLN A  43       0.457  36.335  -5.865  1.00 37.78           O  \\nATOM    338  NE2 GLN A  43       2.271  37.606  -6.053  1.00 35.45           N  \\nATOM    339  N   VAL A  44       2.394  35.532  -2.922  1.00 12.79           N  \\nATOM    340  CA  VAL A  44       3.771  35.242  -3.311  1.00 11.03           C  \\nATOM    341  C   VAL A  44       3.724  34.425  -4.605  1.00 14.28           C  \\nATOM    342  O   VAL A  44       2.633  34.109  -5.090  1.00 14.90           O  \\nATOM    343  CB  VAL A  44       4.508  34.411  -2.223  1.00 13.21           C  \\nATOM    344  CG1 VAL A  44       4.669  35.282  -0.977  1.00 15.04           C  \\nATOM    345  CG2 VAL A  44       3.761  33.116  -1.889  1.00 12.89           C  \\nATOM    346  N   VAL A  45       4.879  34.127  -5.235  1.00 13.20           N  \\nATOM    347  CA  VAL A  45       4.938  33.277  -6.419  1.00 12.28           C  \\nATOM    348  C   VAL A  45       5.836  32.161  -5.970  1.00 11.08           C  \\nATOM    349  O   VAL A  45       6.988  32.447  -5.636  1.00 13.13           O  \\nATOM    350  CB  VAL A  45       5.587  33.979  -7.641  1.00 13.01           C  \\nATOM    351  CG1 VAL A  45       5.549  33.031  -8.810  1.00 14.51           C  \\nATOM    352  CG2 VAL A  45       4.787  35.173  -8.108  1.00 13.54           C  \\nATOM    353  N   ILE A  46       5.374  30.907  -5.915  1.00  9.89           N  \\nATOM    354  CA  ILE A  46       6.274  29.795  -5.491  1.00 11.03           C  \\nATOM    355  C   ILE A  46       6.369  28.800  -6.662  1.00  8.85           C  \\nATOM    356  O   ILE A  46       5.319  28.296  -7.059  1.00 10.25           O  \\nATOM    357  CB  ILE A  46       5.692  29.115  -4.203  1.00 10.64           C  \\nATOM    358  CG1 ILE A  46       5.652  30.096  -3.042  1.00 11.20           C  \\nATOM    359  CG2 ILE A  46       6.599  27.953  -3.757  1.00 12.93           C  \\nATOM    360  CD1 ILE A  46       4.937  29.536  -1.808  1.00  8.90           C  \\nATOM    361  N   ASP A  47       7.565  28.495  -7.224  1.00  9.41           N  \\nATOM    362  CA  ASP A  47       7.767  27.650  -8.426  1.00  9.70           C  \\nATOM    363  C   ASP A  47       6.838  28.031  -9.572  1.00 11.14           C  \\nATOM    364  O   ASP A  47       6.225  27.211 -10.248  1.00 14.09           O  \\nATOM    365  CB  ASP A  47       7.566  26.165  -8.105  1.00 10.73           C  \\nATOM    366  CG  ASP A  47       8.388  25.783  -6.874  1.00 13.52           C  \\nATOM    367  OD1 ASP A  47       7.805  25.291  -5.898  1.00 12.04           O  \\nATOM    368  OD2 ASP A  47       9.598  26.024  -6.872  1.00 11.31           O  \\nATOM    369  N   GLY A  48       6.724  29.353  -9.735  1.00 10.78           N  \\nATOM    370  CA  GLY A  48       5.930  29.961 -10.785  1.00 13.29           C  \\nATOM    371  C   GLY A  48       4.426  29.989 -10.548  1.00 14.28           C  \\nATOM    372  O   GLY A  48       3.677  30.512 -11.386  1.00 15.19           O  \\nATOM    373  N   GLU A  49       3.944  29.422  -9.439  1.00 13.41           N  \\nATOM    374  CA  GLU A  49       2.520  29.457  -9.143  1.00 13.90           C  \\nATOM    375  C   GLU A  49       2.223  30.700  -8.314  1.00 15.34           C  \\nATOM    376  O   GLU A  49       2.860  30.906  -7.275  1.00 14.56           O  \\nATOM    377  CB  GLU A  49       2.124  28.238  -8.340  1.00 15.50           C  \\nATOM    378  CG  GLU A  49       0.630  28.250  -7.990  1.00 18.10           C  \\nATOM    379  CD  GLU A  49       0.204  27.110  -7.100  1.00 19.13           C  \\nATOM    380  OE1 GLU A  49       0.717  27.004  -5.996  1.00 19.53           O  \\nATOM    381  OE2 GLU A  49      -0.630  26.321  -7.529  1.00 22.12           O  \\nATOM    382  N   THR A  50       1.267  31.536  -8.734  1.00 14.06           N  \\nATOM    383  CA  THR A  50       0.901  32.697  -7.927  1.00 16.15           C  \\nATOM    384  C   THR A  50      -0.001  32.169  -6.819  1.00 13.35           C  \\nATOM    385  O   THR A  50      -0.938  31.409  -7.046  1.00 15.13           O  \\nATOM    386  CB  THR A  50       0.203  33.755  -8.846  1.00 17.88           C  \\nATOM    387  OG1 THR A  50      -0.245  34.813  -8.012  1.00 24.33           O  \\nATOM    388  CG2 THR A  50      -1.028  33.236  -9.518  1.00 19.78           C  \\nATOM    389  N   CYS A  51       0.306  32.442  -5.563  1.00 13.28           N  \\nATOM    390  CA  CYS A  51      -0.566  31.959  -4.508  1.00 15.16           C  \\nATOM    391  C   CYS A  51      -0.579  32.892  -3.303  1.00 14.00           C  \\nATOM    392  O   CYS A  51       0.161  33.874  -3.231  1.00 11.81           O  \\nATOM    393  CB  CYS A  51      -0.132  30.537  -4.079  1.00 14.08           C  \\nATOM    394  SG  CYS A  51       1.597  30.409  -3.575  1.00 16.09           S  \\nATOM    395  N   LEU A  52      -1.521  32.609  -2.405  1.00 13.77           N  \\nATOM    396  CA  LEU A  52      -1.573  33.397  -1.199  1.00 16.01           C  \\nATOM    397  C   LEU A  52      -1.260  32.497  -0.025  1.00 14.53           C  \\nATOM    398  O   LEU A  52      -1.868  31.443   0.113  1.00 16.05           O  \\nATOM    399  CB  LEU A  52      -2.957  33.986  -1.014  1.00 19.71           C  \\nATOM    400  CG  LEU A  52      -3.122  34.885   0.214  1.00 20.84           C  \\nATOM    401  CD1 LEU A  52      -3.070  36.329  -0.220  1.00 21.38           C  \\nATOM    402  CD2 LEU A  52      -4.420  34.526   0.923  1.00 20.90           C  \\nATOM    403  N   LEU A  53      -0.319  32.904   0.800  1.00 12.41           N  \\nATOM    404  CA  LEU A  53       0.019  32.232   2.048  1.00 13.70           C  \\nATOM    405  C   LEU A  53      -0.841  32.852   3.148  1.00 13.85           C  \\nATOM    406  O   LEU A  53      -0.733  34.056   3.413  1.00 12.82           O  \\nATOM    407  CB  LEU A  53       1.485  32.464   2.400  1.00 14.51           C  \\nATOM    408  CG  LEU A  53       2.707  31.589   2.031  1.00 19.29           C  \\nATOM    409  CD1 LEU A  53       2.394  30.553   0.962  1.00 18.56           C  \\nATOM    410  CD2 LEU A  53       3.843  32.557   1.741  1.00 20.00           C  \\nATOM    411  N   ASP A  54      -1.699  32.023   3.751  1.00 11.96           N  \\nATOM    412  CA  ASP A  54      -2.476  32.445   4.884  1.00 15.07           C  \\nATOM    413  C   ASP A  54      -1.882  31.675   6.055  1.00 12.67           C  \\nATOM    414  O   ASP A  54      -2.099  30.477   6.234  1.00 13.99           O  \\nATOM    415  CB  ASP A  54      -3.930  32.088   4.644  1.00 19.33           C  \\nATOM    416  CG  ASP A  54      -4.826  32.609   5.746  1.00 24.24           C  \\nATOM    417  OD1 ASP A  54      -4.412  32.652   6.897  1.00 28.17           O  \\nATOM    418  OD2 ASP A  54      -5.958  32.979   5.450  1.00 27.59           O  \\nATOM    419  N   ILE A  55      -1.127  32.389   6.880  1.00 11.78           N  \\nATOM    420  CA  ILE A  55      -0.346  31.769   7.935  1.00 12.15           C  \\nATOM    421  C   ILE A  55      -0.953  32.099   9.296  1.00 13.10           C  \\nATOM    422  O   ILE A  55      -1.195  33.271   9.587  1.00 13.84           O  \\nATOM    423  CB  ILE A  55       1.109  32.282   7.931  1.00 12.42           C  \\nATOM    424  CG1 ILE A  55       1.859  31.979   6.653  1.00 13.99           C  \\nATOM    425  CG2 ILE A  55       1.859  31.505   9.018  1.00 13.24           C  \\nATOM    426  CD1 ILE A  55       3.176  32.768   6.497  1.00 15.68           C  \\nATOM    427  N   LEU A  56      -1.176  31.083  10.134  1.00 11.64           N  \\nATOM    428  CA  LEU A  56      -1.642  31.341  11.490  1.00 11.98           C  \\nATOM    429  C   LEU A  56      -0.524  31.027  12.431  1.00  9.99           C  \\nATOM    430  O   LEU A  56      -0.017  29.895  12.447  1.00  9.72           O  \\nATOM    431  CB  LEU A  56      -2.836  30.463  11.896  1.00 13.80           C  \\nATOM    432  CG  LEU A  56      -3.470  30.741  13.297  1.00 16.13           C  \\nATOM    433  CD1 LEU A  56      -4.347  31.987  13.260  1.00 15.28           C  \\nATOM    434  CD2 LEU A  56      -4.321  29.580  13.692  1.00 16.84           C  \\nATOM    435  N   ASP A  57      -0.188  32.084  13.182  1.00 10.40           N  \\nATOM    436  CA  ASP A  57       0.815  32.014  14.219  1.00 10.30           C  \\nATOM    437  C   ASP A  57       0.127  31.812  15.571  1.00 11.75           C  \\nATOM    438  O   ASP A  57      -0.652  32.631  16.053  1.00 12.91           O  \\nATOM    439  CB  ASP A  57       1.602  33.296  14.173  1.00 10.66           C  \\nATOM    440  CG  ASP A  57       2.778  33.336  15.132  1.00 11.39           C  \\nATOM    441  OD1 ASP A  57       3.306  32.298  15.535  1.00 11.07           O  \\nATOM    442  OD2 ASP A  57       3.168  34.448  15.455  1.00 13.15           O  \\nATOM    443  N   THR A  58       0.447  30.723  16.244  1.00 12.55           N  \\nATOM    444  CA  THR A  58      -0.324  30.325  17.416  1.00 12.98           C  \\nATOM    445  C   THR A  58       0.368  30.541  18.758  1.00 13.48           C  \\nATOM    446  O   THR A  58       1.568  30.812  18.815  1.00 14.00           O  \\nATOM    447  CB  THR A  58      -0.698  28.851  17.219  1.00  9.54           C  \\nATOM    448  OG1 THR A  58       0.491  28.084  17.369  1.00 11.40           O  \\nATOM    449  CG2 THR A  58      -1.313  28.597  15.848  1.00 12.27           C  \\nATOM    450  N   ALA A  59      -0.371  30.382  19.868  1.00 14.38           N  \\nATOM    451  CA  ALA A  59       0.230  30.510  21.194  1.00 14.79           C  \\nATOM    452  C   ALA A  59       0.928  29.210  21.608  1.00 15.41           C  \\nATOM    453  O   ALA A  59       0.448  28.106  21.324  1.00 13.07           O  \\nATOM    454  CB  ALA A  59      -0.845  30.818  22.238  1.00 15.47           C  \\nATOM    455  N   GLY A  60       2.084  29.341  22.271  1.00 17.71           N  \\nATOM    456  CA  GLY A  60       2.807  28.208  22.814  1.00 24.18           C  \\nATOM    457  C   GLY A  60       2.190  27.751  24.130  1.00 31.21           C  \\nATOM    458  O   GLY A  60       2.538  26.688  24.627  1.00 30.44           O  \\nATOM    459  N   GLN A  61       1.260  28.503  24.726  1.00 39.50           N  \\nATOM    460  CA  GLN A  61       0.671  28.099  25.994  1.00 48.84           C  \\nATOM    461  C   GLN A  61      -0.683  27.395  25.985  1.00 51.52           C  \\nATOM    462  O   GLN A  61      -1.606  27.780  25.276  1.00 52.17           O  \\nATOM    463  CB  GLN A  61       0.605  29.327  26.874  1.00 51.56           C  \\nATOM    464  CG  GLN A  61       0.871  28.861  28.312  1.00 56.74           C  \\nATOM    465  CD  GLN A  61       1.506  29.937  29.177  1.00 58.74           C  \\nATOM    466  OE1 GLN A  61       2.647  29.795  29.616  1.00 60.12           O  \\nATOM    467  NE2 GLN A  61       0.783  31.031  29.429  1.00 60.18           N  \\nATOM    468  N   GLU A  62      -0.804  26.349  26.818  1.00 57.40           N  \\nATOM    469  CA  GLU A  62      -2.008  25.527  26.927  1.00 61.96           C  \\nATOM    470  C   GLU A  62      -3.185  26.196  27.631  1.00 62.20           C  \\nATOM    471  O   GLU A  62      -4.138  25.496  27.975  1.00 62.85           O  \\nATOM    472  CB  GLU A  62      -1.629  24.183  27.632  1.00 65.18           C  \\nATOM    473  CG  GLU A  62      -2.622  22.966  27.761  1.00 68.22           C  \\nATOM    474  CD  GLU A  62      -3.032  22.132  26.528  1.00 70.25           C  \\nATOM    475  OE1 GLU A  62      -2.710  22.494  25.395  1.00 71.45           O  \\nATOM    476  OE2 GLU A  62      -3.691  21.101  26.689  1.00 70.00           O  \\nATOM    477  N   GLU A  63      -3.206  27.520  27.904  1.00 62.78           N  \\nATOM    478  CA  GLU A  63      -4.496  28.139  28.254  1.00 63.62           C  \\nATOM    479  C   GLU A  63      -5.457  27.854  27.085  1.00 63.71           C  \\nATOM    480  O   GLU A  63      -6.688  27.814  27.211  1.00 62.97           O  \\nATOM    481  CB  GLU A  63      -4.372  29.666  28.475  1.00 65.13           C  \\nATOM    482  CG  GLU A  63      -3.456  30.553  27.607  1.00 66.91           C  \\nATOM    483  CD  GLU A  63      -4.012  31.235  26.345  1.00 68.05           C  \\nATOM    484  OE1 GLU A  63      -5.220  31.462  26.220  1.00 68.64           O  \\nATOM    485  OE2 GLU A  63      -3.198  31.568  25.481  1.00 67.62           O  \\nATOM    486  N   TYR A  64      -4.801  27.606  25.934  1.00 63.36           N  \\nATOM    487  CA  TYR A  64      -5.420  27.092  24.743  1.00 63.20           C  \\nATOM    488  C   TYR A  64      -5.905  25.653  24.875  1.00 61.76           C  \\nATOM    489  O   TYR A  64      -7.120  25.486  24.858  1.00 63.04           O  \\nATOM    490  CB  TYR A  64      -4.437  27.208  23.613  1.00 64.83           C  \\nATOM    491  CG  TYR A  64      -4.736  28.459  22.818  1.00 66.98           C  \\nATOM    492  CD1 TYR A  64      -4.300  29.686  23.267  1.00 67.55           C  \\nATOM    493  CD2 TYR A  64      -5.459  28.355  21.646  1.00 67.97           C  \\nATOM    494  CE1 TYR A  64      -4.587  30.825  22.536  1.00 69.49           C  \\nATOM    495  CE2 TYR A  64      -5.752  29.488  20.911  1.00 69.29           C  \\nATOM    496  CZ  TYR A  64      -5.313  30.718  21.359  1.00 70.05           C  \\nATOM    497  OH  TYR A  64      -5.589  31.852  20.614  1.00 70.67           O  \\nATOM    498  N   SER A  65      -5.095  24.590  24.997  1.00 58.67           N  \\nATOM    499  CA  SER A  65      -5.617  23.230  25.127  1.00 56.93           C  \\nATOM    500  C   SER A  65      -6.614  22.757  24.063  1.00 55.33           C  \\nATOM    501  O   SER A  65      -6.321  22.936  22.876  1.00 54.74           O  \\nATOM    502  CB  SER A  65      -6.190  23.122  26.552  1.00 57.99           C  \\nATOM    503  OG  SER A  65      -7.251  24.011  26.900  1.00 59.80           O  \\nATOM    504  N   ALA A  66      -7.762  22.119  24.359  1.00 52.33           N  \\nATOM    505  CA  ALA A  66      -8.766  21.824  23.335  1.00 48.80           C  \\nATOM    506  C   ALA A  66      -9.138  23.050  22.492  1.00 45.71           C  \\nATOM    507  O   ALA A  66      -9.751  22.862  21.449  1.00 44.33           O  \\nATOM    508  CB  ALA A  66     -10.043  21.300  23.971  1.00 49.14           C  \\nATOM    509  N   MET A  67      -8.829  24.310  22.875  1.00 42.85           N  \\nATOM    510  CA  MET A  67      -8.984  25.465  21.967  1.00 40.01           C  \\nATOM    511  C   MET A  67      -8.093  25.362  20.719  1.00 36.79           C  \\nATOM    512  O   MET A  67      -8.221  26.184  19.805  1.00 36.08           O  \\nATOM    513  CB  MET A  67      -8.594  26.804  22.581  1.00 42.50           C  \\nATOM    514  CG  MET A  67      -9.359  27.330  23.772  1.00 45.93           C  \\nATOM    515  SD  MET A  67     -11.140  27.396  23.483  1.00 48.74           S  \\nATOM    516  CE  MET A  67     -11.526  25.880  24.321  1.00 47.64           C  \\nATOM    517  N   ARG A  68      -7.128  24.418  20.665  1.00 31.59           N  \\nATOM    518  CA  ARG A  68      -6.322  24.217  19.467  1.00 26.34           C  \\nATOM    519  C   ARG A  68      -7.140  23.473  18.424  1.00 23.68           C  \\nATOM    520  O   ARG A  68      -6.846  23.617  17.231  1.00 21.93           O  \\nATOM    521  CB  ARG A  68      -5.057  23.394  19.745  1.00 25.86           C  \\nATOM    522  CG  ARG A  68      -4.161  23.898  20.869  1.00 29.36           C  \\nATOM    523  CD  ARG A  68      -2.881  23.061  20.998  1.00 32.88           C  \\nATOM    524  NE  ARG A  68      -2.221  22.990  22.315  1.00 36.64           N  \\nATOM    525  CZ  ARG A  68      -1.700  24.031  23.001  1.00 39.66           C  \\nATOM    526  NH1 ARG A  68      -1.903  25.315  22.664  1.00 42.06           N  \\nATOM    527  NH2 ARG A  68      -0.933  23.798  24.071  1.00 39.61           N  \\nATOM    528  N   ASP A  69      -8.188  22.721  18.793  1.00 19.14           N  \\nATOM    529  CA  ASP A  69      -8.880  21.905  17.822  1.00 20.43           C  \\nATOM    530  C   ASP A  69      -9.437  22.623  16.617  1.00 22.16           C  \\nATOM    531  O   ASP A  69      -9.453  22.109  15.488  1.00 19.58           O  \\nATOM    532  CB  ASP A  69     -10.025  21.148  18.478  1.00 20.58           C  \\nATOM    533  CG  ASP A  69      -9.598  19.935  19.303  1.00 22.81           C  \\nATOM    534  OD1 ASP A  69      -8.409  19.708  19.538  1.00 24.59           O  \\nATOM    535  OD2 ASP A  69     -10.475  19.176  19.715  1.00 25.28           O  \\nATOM    536  N   GLN A  70      -9.834  23.874  16.840  1.00 23.26           N  \\nATOM    537  CA  GLN A  70     -10.451  24.609  15.759  1.00 23.53           C  \\nATOM    538  C   GLN A  70      -9.437  24.925  14.667  1.00 21.20           C  \\nATOM    539  O   GLN A  70      -9.653  24.591  13.493  1.00 20.90           O  \\nATOM    540  CB  GLN A  70     -11.071  25.881  16.341  1.00 27.73           C  \\nATOM    541  CG  GLN A  70     -12.498  26.019  15.832  1.00 34.11           C  \\nATOM    542  CD  GLN A  70     -13.103  27.407  16.008  1.00 37.87           C  \\nATOM    543  OE1 GLN A  70     -12.532  28.284  16.672  1.00 39.77           O  \\nATOM    544  NE2 GLN A  70     -14.283  27.620  15.406  1.00 37.89           N  \\nATOM    545  N   TYR A  71      -8.292  25.515  15.050  1.00 19.09           N  \\nATOM    546  CA  TYR A  71      -7.329  25.873  14.030  1.00 17.69           C  \\nATOM    547  C   TYR A  71      -6.648  24.658  13.416  1.00 17.24           C  \\nATOM    548  O   TYR A  71      -6.247  24.675  12.245  1.00 17.25           O  \\nATOM    549  CB  TYR A  71      -6.302  26.868  14.625  1.00 17.87           C  \\nATOM    550  CG  TYR A  71      -5.311  26.480  15.725  1.00 18.90           C  \\nATOM    551  CD1 TYR A  71      -4.355  25.510  15.492  1.00 19.74           C  \\nATOM    552  CD2 TYR A  71      -5.328  27.162  16.940  1.00 19.21           C  \\nATOM    553  CE1 TYR A  71      -3.402  25.212  16.450  1.00 20.38           C  \\nATOM    554  CE2 TYR A  71      -4.375  26.868  17.901  1.00 21.93           C  \\nATOM    555  CZ  TYR A  71      -3.422  25.897  17.645  1.00 21.91           C  \\nATOM    556  OH  TYR A  71      -2.463  25.590  18.588  1.00 26.44           O  \\nATOM    557  N   MET A  72      -6.557  23.559  14.166  1.00 14.81           N  \\nATOM    558  CA  MET A  72      -5.958  22.393  13.563  1.00 18.59           C  \\nATOM    559  C   MET A  72      -6.849  21.778  12.472  1.00 19.38           C  \\nATOM    560  O   MET A  72      -6.385  21.198  11.492  1.00 17.67           O  \\nATOM    561  CB  MET A  72      -5.658  21.386  14.669  1.00 17.39           C  \\nATOM    562  CG  MET A  72      -4.479  21.796  15.514  1.00 16.35           C  \\nATOM    563  SD  MET A  72      -4.146  20.600  16.820  1.00 17.04           S  \\nATOM    564  CE  MET A  72      -2.635  21.259  17.460  1.00 17.72           C  \\nATOM    565  N   ARG A  73      -8.174  21.931  12.626  1.00 20.31           N  \\nATOM    566  CA  ARG A  73      -9.120  21.350  11.701  1.00 22.24           C  \\nATOM    567  C   ARG A  73      -9.087  22.084  10.370  1.00 20.94           C  \\nATOM    568  O   ARG A  73      -9.183  21.450   9.326  1.00 20.81           O  \\nATOM    569  CB  ARG A  73     -10.499  21.414  12.369  1.00 25.29           C  \\nATOM    570  CG  ARG A  73     -11.553  20.499  11.767  1.00 30.04           C  \\nATOM    571  CD  ARG A  73     -12.775  20.429  12.687  1.00 31.86           C  \\nATOM    572  NE  ARG A  73     -12.483  19.702  13.917  1.00 34.70           N  \\nATOM    573  CZ  ARG A  73     -12.894  20.122  15.130  1.00 36.49           C  \\nATOM    574  NH1 ARG A  73     -12.579  19.402  16.196  1.00 36.93           N  \\nATOM    575  NH2 ARG A  73     -13.609  21.239  15.311  1.00 38.17           N  \\nATOM    576  N   THR A  74      -8.868  23.401  10.413  1.00 21.33           N  \\nATOM    577  CA  THR A  74      -8.813  24.240   9.219  1.00 24.99           C  \\nATOM    578  C   THR A  74      -7.436  24.351   8.544  1.00 23.10           C  \\nATOM    579  O   THR A  74      -7.367  24.738   7.371  1.00 22.59           O  \\nATOM    580  CB  THR A  74      -9.289  25.676   9.538  1.00 25.73           C  \\nATOM    581  OG1 THR A  74      -8.351  26.302  10.409  1.00 31.08           O  \\nATOM    582  CG2 THR A  74     -10.657  25.644  10.201  1.00 27.58           C  \\nATOM    583  N   GLY A  75      -6.345  24.084   9.289  1.00 19.58           N  \\nATOM    584  CA  GLY A  75      -5.013  24.114   8.730  1.00 15.79           C  \\nATOM    585  C   GLY A  75      -4.823  23.023   7.713  1.00 13.66           C  \\nATOM    586  O   GLY A  75      -5.245  21.880   7.884  1.00 14.69           O  \\nATOM    587  N   GLU A  76      -4.228  23.360   6.573  1.00 13.14           N  \\nATOM    588  CA  GLU A  76      -3.914  22.363   5.546  1.00 12.99           C  \\nATOM    589  C   GLU A  76      -2.514  21.740   5.753  1.00 12.22           C  \\nATOM    590  O   GLU A  76      -2.293  20.576   5.426  1.00 13.59           O  \\nATOM    591  CB  GLU A  76      -3.999  23.014   4.145  1.00 13.07           C  \\nATOM    592  CG  GLU A  76      -5.387  23.377   3.676  1.00 14.35           C  \\nATOM    593  CD  GLU A  76      -5.261  24.069   2.323  1.00 20.40           C  \\nATOM    594  OE1 GLU A  76      -5.461  23.403   1.288  1.00 24.32           O  \\nATOM    595  OE2 GLU A  76      -4.913  25.250   2.284  1.00 18.82           O  \\nATOM    596  N   GLY A  77      -1.553  22.503   6.301  1.00 10.73           N  \\nATOM    597  CA  GLY A  77      -0.216  22.018   6.570  1.00  8.07           C  \\nATOM    598  C   GLY A  77       0.302  22.689   7.819  1.00  8.63           C  \\nATOM    599  O   GLY A  77      -0.066  23.823   8.127  1.00  8.55           O  \\nATOM    600  N   PHE A  78       1.248  22.013   8.479  1.00  9.15           N  \\nATOM    601  CA  PHE A  78       1.754  22.468   9.773  1.00  8.87           C  \\nATOM    602  C   PHE A  78       3.268  22.520   9.854  1.00  9.51           C  \\nATOM    603  O   PHE A  78       3.930  21.542   9.486  1.00  9.15           O  \\nATOM    604  CB  PHE A  78       1.247  21.532  10.904  1.00  9.44           C  \\nATOM    605  CG  PHE A  78      -0.267  21.472  11.000  1.00  9.66           C  \\nATOM    606  CD1 PHE A  78      -0.906  22.310  11.901  1.00 10.71           C  \\nATOM    607  CD2 PHE A  78      -0.994  20.645  10.152  1.00 10.20           C  \\nATOM    608  CE1 PHE A  78      -2.295  22.338  11.940  1.00 10.61           C  \\nATOM    609  CE2 PHE A  78      -2.378  20.678  10.192  1.00 12.55           C  \\nATOM    610  CZ  PHE A  78      -3.022  21.529  11.081  1.00 12.80           C  \\nATOM    611  N   LEU A  79       3.813  23.636  10.354  1.00  8.72           N  \\nATOM    612  CA  LEU A  79       5.230  23.626  10.699  1.00  9.93           C  \\nATOM    613  C   LEU A  79       5.279  23.281  12.181  1.00  9.09           C  \\nATOM    614  O   LEU A  79       4.701  24.008  12.988  1.00 11.58           O  \\nATOM    615  CB  LEU A  79       5.921  24.970  10.567  1.00 12.80           C  \\nATOM    616  CG  LEU A  79       6.155  25.659   9.249  1.00 16.39           C  \\nATOM    617  CD1 LEU A  79       7.007  26.911   9.544  1.00 17.02           C  \\nATOM    618  CD2 LEU A  79       6.904  24.756   8.269  1.00 16.98           C  \\nATOM    619  N   CYS A  80       5.950  22.219  12.579  1.00  9.14           N  \\nATOM    620  CA  CYS A  80       6.074  21.821  13.981  1.00 11.81           C  \\nATOM    621  C   CYS A  80       7.448  22.237  14.434  1.00 11.53           C  \\nATOM    622  O   CYS A  80       8.435  21.640  14.000  1.00 12.21           O  \\nATOM    623  CB  CYS A  80       5.903  20.318  14.106  1.00 13.73           C  \\nATOM    624  SG  CYS A  80       4.158  19.900  13.837  1.00 18.26           S  \\nATOM    625  N   VAL A  81       7.486  23.282  15.269  1.00  7.57           N  \\nATOM    626  CA  VAL A  81       8.730  23.956  15.604  1.00  8.45           C  \\nATOM    627  C   VAL A  81       9.248  23.620  16.996  1.00  8.90           C  \\nATOM    628  O   VAL A  81       8.451  23.680  17.939  1.00  8.49           O  \\nATOM    629  CB  VAL A  81       8.544  25.516  15.507  1.00  7.40           C  \\nATOM    630  CG1 VAL A  81       9.917  26.196  15.588  1.00  9.42           C  \\nATOM    631  CG2 VAL A  81       8.004  25.954  14.167  1.00  8.59           C  \\nATOM    632  N   PHE A  82      10.546  23.237  17.119  1.00  9.49           N  \\nATOM    633  CA  PHE A  82      11.195  23.202  18.442  1.00  9.09           C  \\nATOM    634  C   PHE A  82      12.511  23.994  18.350  1.00  9.83           C  \\nATOM    635  O   PHE A  82      12.914  24.440  17.266  1.00  9.02           O  \\nATOM    636  CB  PHE A  82      11.482  21.755  18.902  1.00  9.76           C  \\nATOM    637  CG  PHE A  82      12.536  21.062  18.071  1.00 10.09           C  \\nATOM    638  CD1 PHE A  82      13.867  21.099  18.486  1.00 10.70           C  \\nATOM    639  CD2 PHE A  82      12.192  20.412  16.889  1.00 10.93           C  \\nATOM    640  CE1 PHE A  82      14.838  20.483  17.699  1.00 11.12           C  \\nATOM    641  CE2 PHE A  82      13.181  19.807  16.125  1.00 10.73           C  \\nATOM    642  CZ  PHE A  82      14.505  19.839  16.523  1.00 10.35           C  \\nATOM    643  N   ALA A  83      13.128  24.269  19.516  1.00  9.75           N  \\nATOM    644  CA  ALA A  83      14.407  24.963  19.569  1.00 10.32           C  \\nATOM    645  C   ALA A  83      15.509  23.960  19.916  1.00  9.48           C  \\nATOM    646  O   ALA A  83      15.362  23.142  20.839  1.00  9.61           O  \\nATOM    647  CB  ALA A  83      14.354  26.083  20.620  1.00  9.75           C  \\nATOM    648  N   ILE A  84      16.624  24.042  19.160  1.00 11.41           N  \\nATOM    649  CA  ILE A  84      17.714  23.070  19.299  1.00 13.15           C  \\nATOM    650  C   ILE A  84      18.446  23.223  20.623  1.00 14.57           C  \\nATOM    651  O   ILE A  84      19.234  22.347  20.971  1.00 13.94           O  \\nATOM    652  CB  ILE A  84      18.775  23.151  18.160  1.00 13.29           C  \\nATOM    653  CG1 ILE A  84      19.403  24.509  17.878  1.00 14.90           C  \\nATOM    654  CG2 ILE A  84      18.018  22.701  16.933  1.00 13.09           C  \\nATOM    655  CD1 ILE A  84      20.578  24.957  18.763  1.00 17.59           C  \\nATOM    656  N   ASN A  85      18.200  24.290  21.390  1.00 13.61           N  \\nATOM    657  CA  ASN A  85      18.794  24.394  22.720  1.00 15.52           C  \\nATOM    658  C   ASN A  85      17.758  24.156  23.828  1.00 15.49           C  \\nATOM    659  O   ASN A  85      17.988  24.519  24.999  1.00 16.13           O  \\nATOM    660  CB  ASN A  85      19.446  25.788  22.885  1.00 15.16           C  \\nATOM    661  CG  ASN A  85      18.464  26.946  22.904  1.00 16.42           C  \\nATOM    662  OD1 ASN A  85      17.355  26.878  22.362  1.00 16.01           O  \\nATOM    663  ND2 ASN A  85      18.808  28.030  23.587  1.00 16.83           N  \\nATOM    664  N   ASN A  86      16.616  23.515  23.509  1.00 14.72           N  \\nATOM    665  CA  ASN A  86      15.553  23.303  24.483  1.00 13.01           C  \\nATOM    666  C   ASN A  86      14.987  21.901  24.309  1.00 13.17           C  \\nATOM    667  O   ASN A  86      14.080  21.668  23.520  1.00 13.04           O  \\nATOM    668  CB  ASN A  86      14.478  24.367  24.256  1.00 16.51           C  \\nATOM    669  CG  ASN A  86      13.267  24.415  25.187  1.00 20.74           C  \\nATOM    670  OD1 ASN A  86      12.370  25.241  25.018  1.00 25.41           O  \\nATOM    671  ND2 ASN A  86      13.123  23.596  26.206  1.00 19.96           N  \\nATOM    672  N   THR A  87      15.488  20.950  25.103  1.00 13.80           N  \\nATOM    673  CA  THR A  87      15.095  19.553  25.026  1.00 15.40           C  \\nATOM    674  C   THR A  87      13.589  19.316  25.266  1.00 14.40           C  \\nATOM    675  O   THR A  87      12.953  18.535  24.546  1.00 14.11           O  \\nATOM    676  CB  THR A  87      15.983  18.791  26.046  1.00 16.19           C  \\nATOM    677  OG1 THR A  87      17.325  18.908  25.571  1.00 18.22           O  \\nATOM    678  CG2 THR A  87      15.626  17.327  26.180  1.00 17.82           C  \\nATOM    679  N   LYS A  88      12.983  19.958  26.272  1.00 14.42           N  \\nATOM    680  CA  LYS A  88      11.552  19.842  26.535  1.00 15.84           C  \\nATOM    681  C   LYS A  88      10.726  20.232  25.278  1.00 15.06           C  \\nATOM    682  O   LYS A  88       9.722  19.582  25.003  1.00 14.61           O  \\nATOM    683  CB  LYS A  88      11.207  20.766  27.717  1.00 19.53           C  \\nATOM    684  CG  LYS A  88       9.937  20.401  28.457  1.00 27.71           C  \\nATOM    685  CD  LYS A  88       9.585  21.452  29.529  1.00 33.26           C  \\nATOM    686  CE  LYS A  88       9.061  22.762  28.877  1.00 36.79           C  \\nATOM    687  NZ  LYS A  88       8.642  23.763  29.852  1.00 37.29           N  \\nATOM    688  N   SER A  89      11.119  21.259  24.475  1.00 12.36           N  \\nATOM    689  CA  SER A  89      10.325  21.666  23.309  1.00 11.31           C  \\nATOM    690  C   SER A  89      10.371  20.572  22.268  1.00 12.51           C  \\nATOM    691  O   SER A  89       9.397  20.388  21.547  1.00 12.88           O  \\nATOM    692  CB  SER A  89      10.844  22.951  22.674  1.00 11.64           C  \\nATOM    693  OG  SER A  89      12.160  22.900  22.124  1.00 11.11           O  \\nATOM    694  N   PHE A  90      11.504  19.842  22.204  1.00 11.27           N  \\nATOM    695  CA  PHE A  90      11.680  18.751  21.262  1.00 12.91           C  \\nATOM    696  C   PHE A  90      10.760  17.601  21.667  1.00 14.37           C  \\nATOM    697  O   PHE A  90      10.034  17.057  20.839  1.00 13.24           O  \\nATOM    698  CB  PHE A  90      13.161  18.312  21.260  1.00 12.34           C  \\nATOM    699  CG  PHE A  90      13.411  17.120  20.351  1.00 14.41           C  \\nATOM    700  CD1 PHE A  90      13.890  15.948  20.903  1.00 15.10           C  \\nATOM    701  CD2 PHE A  90      13.136  17.206  18.990  1.00 14.81           C  \\nATOM    702  CE1 PHE A  90      14.093  14.857  20.088  1.00 16.20           C  \\nATOM    703  CE2 PHE A  90      13.340  16.110  18.179  1.00 14.81           C  \\nATOM    704  CZ  PHE A  90      13.818  14.935  18.731  1.00 15.39           C  \\nATOM    705  N   GLU A  91      10.725  17.231  22.949  1.00 14.40           N  \\nATOM    706  CA  GLU A  91       9.863  16.146  23.407  1.00 15.78           C  \\nATOM    707  C   GLU A  91       8.374  16.466  23.301  1.00 16.18           C  \\nATOM    708  O   GLU A  91       7.533  15.582  23.114  1.00 17.50           O  \\nATOM    709  CB  GLU A  91      10.226  15.846  24.835  1.00 19.84           C  \\nATOM    710  CG  GLU A  91      11.654  15.386  25.074  1.00 24.49           C  \\nATOM    711  CD  GLU A  91      12.090  15.310  26.537  1.00 27.97           C  \\nATOM    712  OE1 GLU A  91      13.199  14.842  26.766  1.00 30.72           O  \\nATOM    713  OE2 GLU A  91      11.371  15.730  27.449  1.00 30.38           O  \\nATOM    714  N   ASP A  92       8.019  17.747  23.433  1.00 16.19           N  \\nATOM    715  CA  ASP A  92       6.650  18.216  23.265  1.00 15.23           C  \\nATOM    716  C   ASP A  92       6.141  18.053  21.839  1.00 14.29           C  \\nATOM    717  O   ASP A  92       4.927  18.074  21.633  1.00 14.78           O  \\nATOM    718  CB  ASP A  92       6.536  19.708  23.646  1.00 17.85           C  \\nATOM    719  CG  ASP A  92       6.706  20.080  25.127  1.00 21.39           C  \\nATOM    720  OD1 ASP A  92       6.648  19.209  25.997  1.00 20.06           O  \\nATOM    721  OD2 ASP A  92       6.887  21.272  25.401  1.00 24.39           O  \\nATOM    722  N   ILE A  93       7.013  17.910  20.826  1.00 11.76           N  \\nATOM    723  CA  ILE A  93       6.559  17.749  19.452  1.00 14.77           C  \\nATOM    724  C   ILE A  93       5.574  16.583  19.354  1.00 14.03           C  \\nATOM    725  O   ILE A  93       4.576  16.695  18.644  1.00 14.93           O  \\nATOM    726  CB  ILE A  93       7.790  17.499  18.485  1.00 14.96           C  \\nATOM    727  CG1 ILE A  93       8.506  18.830  18.293  1.00 15.41           C  \\nATOM    728  CG2 ILE A  93       7.369  16.936  17.091  1.00 14.61           C  \\nATOM    729  CD1 ILE A  93       7.654  19.940  17.637  1.00 20.05           C  \\nATOM    730  N   HIS A  94       5.806  15.471  20.049  1.00 15.23           N  \\nATOM    731  CA  HIS A  94       4.940  14.302  19.956  1.00 18.20           C  \\nATOM    732  C   HIS A  94       3.494  14.635  20.287  1.00 16.36           C  \\nATOM    733  O   HIS A  94       2.576  14.183  19.598  1.00 17.20           O  \\nATOM    734  CB  HIS A  94       5.436  13.194  20.900  1.00 23.57           C  \\nATOM    735  CG  HIS A  94       4.685  11.874  20.678  1.00 29.09           C  \\nATOM    736  ND1 HIS A  94       4.923  11.118  19.602  1.00 31.26           N  \\nATOM    737  CD2 HIS A  94       3.705  11.361  21.498  1.00 30.93           C  \\nATOM    738  CE1 HIS A  94       4.078  10.110  19.746  1.00 31.74           C  \\nATOM    739  NE2 HIS A  94       3.366  10.260  20.861  1.00 32.20           N  \\nATOM    740  N   GLN A  95       3.271  15.499  21.281  1.00 16.91           N  \\nATOM    741  CA  GLN A  95       1.916  15.878  21.673  1.00 18.66           C  \\nATOM    742  C   GLN A  95       1.226  16.736  20.625  1.00 16.97           C  \\nATOM    743  O   GLN A  95       0.012  16.562  20.443  1.00 15.36           O  \\nATOM    744  CB  GLN A  95       1.928  16.626  23.018  1.00 21.56           C  \\nATOM    745  CG  GLN A  95       2.457  15.766  24.180  1.00 29.81           C  \\nATOM    746  CD  GLN A  95       3.370  16.540  25.150  1.00 35.29           C  \\nATOM    747  OE1 GLN A  95       3.037  17.647  25.593  1.00 37.94           O  \\nATOM    748  NE2 GLN A  95       4.546  16.006  25.521  1.00 37.04           N  \\nATOM    749  N   TYR A  96       1.930  17.653  19.921  1.00 16.06           N  \\nATOM    750  CA  TYR A  96       1.276  18.417  18.826  1.00 16.07           C  \\nATOM    751  C   TYR A  96       0.963  17.504  17.658  1.00 14.44           C  \\nATOM    752  O   TYR A  96      -0.138  17.565  17.118  1.00 15.79           O  \\nATOM    753  CB  TYR A  96       2.139  19.554  18.264  1.00 17.94           C  \\nATOM    754  CG  TYR A  96       2.287  20.677  19.277  1.00 19.19           C  \\nATOM    755  CD1 TYR A  96       1.368  21.699  19.338  1.00 19.33           C  \\nATOM    756  CD2 TYR A  96       3.348  20.631  20.161  1.00 19.61           C  \\nATOM    757  CE1 TYR A  96       1.526  22.678  20.308  1.00 21.72           C  \\nATOM    758  CE2 TYR A  96       3.512  21.595  21.129  1.00 21.72           C  \\nATOM    759  CZ  TYR A  96       2.594  22.613  21.190  1.00 21.26           C  \\nATOM    760  OH  TYR A  96       2.776  23.582  22.153  1.00 24.09           O  \\nATOM    761  N   ARG A  97       1.904  16.635  17.295  1.00 13.30           N  \\nATOM    762  CA  ARG A  97       1.699  15.650  16.247  1.00 15.46           C  \\nATOM    763  C   ARG A  97       0.457  14.794  16.492  1.00 14.26           C  \\nATOM    764  O   ARG A  97      -0.420  14.676  15.641  1.00 14.10           O  \\nATOM    765  CB  ARG A  97       2.939  14.796  16.196  1.00 17.97           C  \\nATOM    766  CG  ARG A  97       2.886  13.541  15.357  1.00 23.35           C  \\nATOM    767  CD  ARG A  97       3.333  13.765  13.943  1.00 23.59           C  \\nATOM    768  NE  ARG A  97       3.314  12.471  13.314  1.00 24.08           N  \\nATOM    769  CZ  ARG A  97       3.268  12.320  12.007  1.00 25.74           C  \\nATOM    770  NH1 ARG A  97       3.135  11.073  11.548  1.00 26.76           N  \\nATOM    771  NH2 ARG A  97       3.353  13.380  11.181  1.00 26.95           N  \\nATOM    772  N   GLU A  98       0.332  14.218  17.678  1.00 13.69           N  \\nATOM    773  CA  GLU A  98      -0.807  13.361  17.941  1.00 16.64           C  \\nATOM    774  C   GLU A  98      -2.118  14.108  17.983  1.00 15.57           C  \\nATOM    775  O   GLU A  98      -3.157  13.573  17.607  1.00 15.85           O  \\nATOM    776  CB  GLU A  98      -0.593  12.623  19.260  1.00 21.96           C  \\nATOM    777  CG  GLU A  98       0.417  11.455  19.177  1.00 28.02           C  \\nATOM    778  CD  GLU A  98       0.147  10.460  18.038  1.00 31.04           C  \\nATOM    779  OE1 GLU A  98       1.118  10.123  17.373  1.00 33.58           O  \\nATOM    780  OE2 GLU A  98      -0.992  10.025  17.790  1.00 33.61           O  \\nATOM    781  N   GLN A  99      -2.105  15.362  18.446  1.00 14.66           N  \\nATOM    782  CA  GLN A  99      -3.343  16.104  18.468  1.00 13.22           C  \\nATOM    783  C   GLN A  99      -3.779  16.398  17.049  1.00 12.15           C  \\nATOM    784  O   GLN A  99      -4.974  16.274  16.791  1.00 14.09           O  \\nATOM    785  CB  GLN A  99      -3.158  17.403  19.250  1.00 16.29           C  \\nATOM    786  CG  GLN A  99      -4.513  18.118  19.407  1.00 19.67           C  \\nATOM    787  CD  GLN A  99      -4.517  19.240  20.438  1.00 22.19           C  \\nATOM    788  OE1 GLN A  99      -3.537  19.485  21.138  1.00 23.70           O  \\nATOM    789  NE2 GLN A  99      -5.628  19.961  20.537  1.00 22.72           N  \\nATOM    790  N   ILE A 100      -2.861  16.795  16.142  1.00 11.42           N  \\nATOM    791  CA  ILE A 100      -3.197  17.121  14.743  1.00 10.81           C  \\nATOM    792  C   ILE A 100      -3.761  15.893  14.008  1.00 12.39           C  \\nATOM    793  O   ILE A 100      -4.733  15.967  13.243  1.00 12.70           O  \\nATOM    794  CB  ILE A 100      -1.924  17.659  14.038  1.00 10.32           C  \\nATOM    795  CG1 ILE A 100      -1.642  19.075  14.518  1.00 10.37           C  \\nATOM    796  CG2 ILE A 100      -2.131  17.685  12.512  1.00 11.36           C  \\nATOM    797  CD1 ILE A 100      -0.184  19.481  14.392  1.00 10.69           C  \\nATOM    798  N   LYS A 101      -3.143  14.732  14.255  1.00 14.08           N  \\nATOM    799  CA  LYS A 101      -3.613  13.487  13.637  1.00 15.95           C  \\nATOM    800  C   LYS A 101      -5.021  13.150  14.079  1.00 15.72           C  \\nATOM    801  O   LYS A 101      -5.848  12.770  13.262  1.00 14.25           O  \\nATOM    802  CB  LYS A 101      -2.681  12.317  13.996  1.00 17.44           C  \\nATOM    803  CG  LYS A 101      -1.414  12.284  13.115  1.00 19.24           C  \\nATOM    804  CD  LYS A 101      -0.332  11.418  13.764  1.00 23.28           C  \\nATOM    805  CE  LYS A 101      -0.752  10.000  14.188  1.00 23.82           C  \\nATOM    806  NZ  LYS A 101       0.400   9.230  14.662  1.00 24.00           N  \\nATOM    807  N   ARG A 102      -5.278  13.341  15.377  1.00 16.72           N  \\nATOM    808  CA  ARG A 102      -6.586  13.098  15.962  1.00 18.74           C  \\nATOM    809  C   ARG A 102      -7.643  14.087  15.440  1.00 19.07           C  \\nATOM    810  O   ARG A 102      -8.712  13.628  15.002  1.00 20.67           O  \\nATOM    811  CB  ARG A 102      -6.457  13.197  17.462  1.00 20.62           C  \\nATOM    812  CG  ARG A 102      -7.539  12.409  18.132  1.00 26.89           C  \\nATOM    813  CD  ARG A 102      -7.610  12.887  19.585  1.00 30.60           C  \\nATOM    814  NE  ARG A 102      -8.197  14.224  19.645  1.00 35.83           N  \\nATOM    815  CZ  ARG A 102      -7.633  15.255  20.301  1.00 37.61           C  \\nATOM    816  NH1 ARG A 102      -8.243  16.455  20.267  1.00 38.37           N  \\nATOM    817  NH2 ARG A 102      -6.481  15.110  20.983  1.00 38.39           N  \\nATOM    818  N   VAL A 103      -7.418  15.407  15.395  1.00 15.59           N  \\nATOM    819  CA  VAL A 103      -8.452  16.326  14.881  1.00 18.48           C  \\nATOM    820  C   VAL A 103      -8.665  16.200  13.349  1.00 16.72           C  \\nATOM    821  O   VAL A 103      -9.776  16.383  12.848  1.00 15.90           O  \\nATOM    822  CB  VAL A 103      -8.131  17.840  15.215  1.00 20.10           C  \\nATOM    823  CG1 VAL A 103      -7.896  18.005  16.704  1.00 21.34           C  \\nATOM    824  CG2 VAL A 103      -6.899  18.302  14.512  1.00 23.93           C  \\nATOM    825  N   LYS A 104      -7.622  15.902  12.574  1.00 13.84           N  \\nATOM    826  CA  LYS A 104      -7.804  15.755  11.140  1.00 16.11           C  \\nATOM    827  C   LYS A 104      -8.158  14.329  10.716  1.00 16.70           C  \\nATOM    828  O   LYS A 104      -8.260  14.079   9.508  1.00 17.27           O  \\nATOM    829  CB  LYS A 104      -6.525  16.215  10.404  1.00 18.12           C  \\nATOM    830  CG  LYS A 104      -6.197  17.678  10.744  1.00 19.53           C  \\nATOM    831  CD  LYS A 104      -5.250  18.297   9.736  1.00 21.28           C  \\nATOM    832  CE  LYS A 104      -5.921  18.590   8.390  1.00 20.75           C  \\nATOM    833  NZ  LYS A 104      -6.875  19.673   8.462  1.00 19.79           N  \\nATOM    834  N   ASP A 105      -8.372  13.390  11.669  1.00 15.91           N  \\nATOM    835  CA  ASP A 105      -8.666  11.988  11.359  1.00 15.43           C  \\nATOM    836  C   ASP A 105      -7.705  11.372  10.333  1.00 16.04           C  \\nATOM    837  O   ASP A 105      -8.106  10.654   9.415  1.00 15.38           O  \\nATOM    838  CB  ASP A 105     -10.077  11.847  10.810  1.00 18.74           C  \\nATOM    839  CG  ASP A 105     -11.261  12.121  11.696  1.00 21.15           C  \\nATOM    840  OD1 ASP A 105     -11.183  11.972  12.918  1.00 21.36           O  \\nATOM    841  OD2 ASP A 105     -12.281  12.480  11.113  1.00 22.63           O  \\nATOM    842  N   SER A 106      -6.397  11.591  10.469  1.00 15.54           N  \\nATOM    843  CA  SER A 106      -5.470  11.174   9.442  1.00 17.25           C  \\nATOM    844  C   SER A 106      -4.112  10.882  10.011  1.00 17.31           C  \\nATOM    845  O   SER A 106      -3.650  11.556  10.916  1.00 19.11           O  \\nATOM    846  CB  SER A 106      -5.463  12.314   8.398  1.00 19.27           C  \\nATOM    847  OG  SER A 106      -4.326  12.490   7.573  1.00 24.20           O  \\nATOM    848  N   ASP A 107      -3.454   9.891   9.453  1.00 17.29           N  \\nATOM    849  CA  ASP A 107      -2.109   9.557   9.868  1.00 19.80           C  \\nATOM    850  C   ASP A 107      -1.000  10.220   9.063  1.00 20.36           C  \\nATOM    851  O   ASP A 107       0.197  10.118   9.345  1.00 21.26           O  \\nATOM    852  CB  ASP A 107      -1.970   8.045   9.810  1.00 23.15           C  \\nATOM    853  CG  ASP A 107      -2.863   7.310  10.807  1.00 25.12           C  \\nATOM    854  OD1 ASP A 107      -3.262   6.206  10.462  1.00 29.21           O  \\nATOM    855  OD2 ASP A 107      -3.127   7.812  11.903  1.00 26.59           O  \\nATOM    856  N   ASP A 108      -1.383  10.909   8.005  1.00 20.57           N  \\nATOM    857  CA  ASP A 108      -0.430  11.664   7.216  1.00 22.38           C  \\nATOM    858  C   ASP A 108      -1.061  12.982   6.842  1.00 21.37           C  \\nATOM    859  O   ASP A 108      -1.973  13.096   6.030  1.00 24.36           O  \\nATOM    860  CB  ASP A 108      -0.056  10.900   5.967  1.00 27.28           C  \\nATOM    861  CG  ASP A 108      -1.263  10.278   5.320  1.00 32.24           C  \\nATOM    862  OD1 ASP A 108      -1.714   9.272   5.877  1.00 37.07           O  \\nATOM    863  OD2 ASP A 108      -1.747  10.790   4.301  1.00 36.33           O  \\nATOM    864  N   VAL A 109      -0.598  13.970   7.572  1.00 19.33           N  \\nATOM    865  CA  VAL A 109      -1.009  15.354   7.400  1.00 17.49           C  \\nATOM    866  C   VAL A 109       0.215  16.086   6.819  1.00 14.55           C  \\nATOM    867  O   VAL A 109       1.335  15.772   7.245  1.00 13.64           O  \\nATOM    868  CB  VAL A 109      -1.410  15.870   8.784  1.00 18.40           C  \\nATOM    869  CG1 VAL A 109      -1.645  17.366   8.765  1.00 20.21           C  \\nATOM    870  CG2 VAL A 109      -2.676  15.101   9.214  1.00 20.27           C  \\nATOM    871  N   PRO A 110       0.141  17.001   5.825  1.00 12.41           N  \\nATOM    872  CA  PRO A 110       1.297  17.783   5.384  1.00 11.52           C  \\nATOM    873  C   PRO A 110       1.920  18.480   6.605  1.00 11.79           C  \\nATOM    874  O   PRO A 110       1.289  19.245   7.324  1.00  9.55           O  \\nATOM    875  CB  PRO A 110       0.728  18.721   4.370  1.00 11.31           C  \\nATOM    876  CG  PRO A 110      -0.410  17.926   3.779  1.00 11.38           C  \\nATOM    877  CD  PRO A 110      -1.044  17.309   5.020  1.00 12.05           C  \\nATOM    878  N   MET A 111       3.192  18.201   6.874  1.00 11.92           N  \\nATOM    879  CA  MET A 111       3.878  18.724   8.050  1.00 11.49           C  \\nATOM    880  C   MET A 111       5.379  18.749   7.807  1.00 11.68           C  \\nATOM    881  O   MET A 111       5.903  17.900   7.098  1.00 11.11           O  \\nATOM    882  CB  MET A 111       3.494  17.806   9.211  1.00 14.36           C  \\nATOM    883  CG  MET A 111       3.864  18.110  10.627  1.00 17.41           C  \\nATOM    884  SD  MET A 111       3.231  16.825  11.758  1.00 18.36           S  \\nATOM    885  CE  MET A 111       1.528  17.128  11.498  1.00 17.25           C  \\nATOM    886  N   VAL A 112       6.091  19.706   8.404  1.00 10.73           N  \\nATOM    887  CA  VAL A 112       7.545  19.763   8.348  1.00 10.18           C  \\nATOM    888  C   VAL A 112       7.992  19.875   9.794  1.00  9.11           C  \\nATOM    889  O   VAL A 112       7.404  20.633  10.556  1.00 10.77           O  \\nATOM    890  CB  VAL A 112       8.010  21.024   7.542  1.00 11.80           C  \\nATOM    891  CG1 VAL A 112       9.528  21.213   7.558  1.00 13.24           C  \\nATOM    892  CG2 VAL A 112       7.542  20.843   6.104  1.00 14.56           C  \\nATOM    893  N   LEU A 113       9.032  19.167  10.207  1.00  8.47           N  \\nATOM    894  CA  LEU A 113       9.632  19.332  11.535  1.00  9.53           C  \\nATOM    895  C   LEU A 113      10.738  20.384  11.412  1.00  9.33           C  \\nATOM    896  O   LEU A 113      11.621  20.270  10.555  1.00  9.95           O  \\nATOM    897  CB  LEU A 113      10.214  17.983  12.017  1.00  9.56           C  \\nATOM    898  CG  LEU A 113      10.863  17.976  13.381  1.00 11.58           C  \\nATOM    899  CD1 LEU A 113       9.908  18.379  14.503  1.00 13.17           C  \\nATOM    900  CD2 LEU A 113      11.361  16.564  13.597  1.00 12.84           C  \\nATOM    901  N   VAL A 114      10.705  21.442  12.221  1.00  7.62           N  \\nATOM    902  CA  VAL A 114      11.673  22.541  12.109  1.00  7.64           C  \\nATOM    903  C   VAL A 114      12.436  22.667  13.441  1.00  7.65           C  \\nATOM    904  O   VAL A 114      11.872  22.842  14.541  1.00  9.71           O  \\nATOM    905  CB  VAL A 114      10.896  23.836  11.774  1.00  9.37           C  \\nATOM    906  CG1 VAL A 114      11.814  25.030  11.869  1.00 10.27           C  \\nATOM    907  CG2 VAL A 114      10.254  23.719  10.393  1.00  9.64           C  \\nATOM    908  N   GLY A 115      13.745  22.567  13.331  1.00  6.10           N  \\nATOM    909  CA  GLY A 115      14.603  22.742  14.476  1.00  6.69           C  \\nATOM    910  C   GLY A 115      15.120  24.163  14.407  1.00  8.47           C  \\nATOM    911  O   GLY A 115      16.007  24.439  13.594  1.00  9.16           O  \\nATOM    912  N   ASN A 116      14.593  25.089  15.192  1.00  8.01           N  \\nATOM    913  CA  ASN A 116      15.007  26.475  15.111  1.00  8.24           C  \\nATOM    914  C   ASN A 116      16.112  26.873  16.109  1.00  9.61           C  \\nATOM    915  O   ASN A 116      16.453  26.142  17.051  1.00  9.84           O  \\nATOM    916  CB  ASN A 116      13.721  27.307  15.275  1.00  8.01           C  \\nATOM    917  CG  ASN A 116      13.929  28.788  14.992  1.00  9.48           C  \\nATOM    918  OD1 ASN A 116      14.470  29.205  13.972  1.00  9.94           O  \\nATOM    919  ND2 ASN A 116      13.565  29.644  15.940  1.00 10.75           N  \\nATOM    920  N   LYS A 117      16.660  28.087  15.876  1.00 10.22           N  \\nATOM    921  CA  LYS A 117      17.735  28.715  16.674  1.00 12.36           C  \\nATOM    922  C   LYS A 117      19.094  28.021  16.450  1.00 13.39           C  \\nATOM    923  O   LYS A 117      19.977  27.940  17.324  1.00 11.29           O  \\nATOM    924  CB  LYS A 117      17.391  28.706  18.214  1.00 13.23           C  \\nATOM    925  CG  LYS A 117      16.026  29.272  18.611  1.00 12.90           C  \\nATOM    926  CD  LYS A 117      15.932  29.640  20.097  1.00 12.43           C  \\nATOM    927  CE  LYS A 117      14.551  30.217  20.380  1.00 10.55           C  \\nATOM    928  NZ  LYS A 117      14.491  30.647  21.772  1.00 11.10           N  \\nATOM    929  N   CYS A 118      19.254  27.555  15.188  1.00 13.69           N  \\nATOM    930  CA  CYS A 118      20.449  26.806  14.844  1.00 15.67           C  \\nATOM    931  C   CYS A 118      21.716  27.658  14.796  1.00 15.28           C  \\nATOM    932  O   CYS A 118      22.808  27.113  14.629  1.00 15.30           O  \\nATOM    933  CB  CYS A 118      20.246  26.064  13.492  1.00 17.74           C  \\nATOM    934  SG  CYS A 118      20.353  27.009  11.944  1.00 20.74           S  \\nATOM    935  N   ASP A 119      21.578  28.995  14.931  1.00 15.76           N  \\nATOM    936  CA  ASP A 119      22.722  29.904  15.083  1.00 16.09           C  \\nATOM    937  C   ASP A 119      23.426  29.778  16.445  1.00 17.19           C  \\nATOM    938  O   ASP A 119      24.589  30.178  16.553  1.00 18.76           O  \\nATOM    939  CB  ASP A 119      22.268  31.374  14.888  1.00 15.14           C  \\nATOM    940  CG  ASP A 119      21.153  31.785  15.835  1.00 15.81           C  \\nATOM    941  OD1 ASP A 119      20.030  31.294  15.662  1.00 13.87           O  \\nATOM    942  OD2 ASP A 119      21.407  32.596  16.721  1.00 14.85           O  \\nATOM    943  N   LEU A 120      22.732  29.248  17.480  1.00 16.86           N  \\nATOM    944  CA  LEU A 120      23.230  29.099  18.845  1.00 18.76           C  \\nATOM    945  C   LEU A 120      24.217  27.957  18.979  1.00 19.70           C  \\nATOM    946  O   LEU A 120      23.979  26.885  18.431  1.00 20.53           O  \\nATOM    947  CB  LEU A 120      22.103  28.829  19.831  1.00 17.65           C  \\nATOM    948  CG  LEU A 120      21.065  29.933  20.010  1.00 19.43           C  \\nATOM    949  CD1 LEU A 120      20.044  29.418  21.008  1.00 18.46           C  \\nATOM    950  CD2 LEU A 120      21.688  31.239  20.503  1.00 18.73           C  \\nATOM    951  N   ALA A 121      25.313  28.184  19.705  1.00 19.26           N  \\nATOM    952  CA  ALA A 121      26.303  27.143  19.887  1.00 20.48           C  \\nATOM    953  C   ALA A 121      25.942  26.210  21.036  1.00 21.30           C  \\nATOM    954  O   ALA A 121      26.403  25.069  21.067  1.00 24.04           O  \\nATOM    955  CB  ALA A 121      27.652  27.767  20.164  1.00 22.02           C  \\nATOM    956  N   ALA A 122      25.109  26.635  21.982  1.00 22.09           N  \\nATOM    957  CA  ALA A 122      24.683  25.797  23.094  1.00 23.88           C  \\nATOM    958  C   ALA A 122      23.578  24.811  22.720  1.00 23.77           C  \\nATOM    959  O   ALA A 122      22.492  24.835  23.301  1.00 27.82           O  \\nATOM    960  CB  ALA A 122      24.171  26.684  24.249  1.00 25.67           C  \\nATOM    961  N   ARG A 123      23.843  23.908  21.783  1.00 20.99           N  \\nATOM    962  CA  ARG A 123      22.890  22.905  21.311  1.00 19.81           C  \\nATOM    963  C   ARG A 123      22.619  21.761  22.283  1.00 19.56           C  \\nATOM    964  O   ARG A 123      23.590  21.176  22.774  1.00 20.01           O  \\nATOM    965  CB  ARG A 123      23.421  22.333  19.988  1.00 19.81           C  \\nATOM    966  CG  ARG A 123      22.662  21.166  19.398  1.00 21.43           C  \\nATOM    967  CD  ARG A 123      23.273  20.680  18.073  1.00 22.66           C  \\nATOM    968  NE  ARG A 123      23.050  21.590  16.956  1.00 22.61           N  \\nATOM    969  CZ  ARG A 123      22.019  21.487  16.093  1.00 20.95           C  \\nATOM    970  NH1 ARG A 123      21.045  20.568  16.181  1.00 16.36           N  \\nATOM    971  NH2 ARG A 123      21.989  22.370  15.098  1.00 19.65           N  \\nATOM    972  N   THR A 124      21.364  21.389  22.600  1.00 17.96           N  \\nATOM    973  CA  THR A 124      21.118  20.200  23.415  1.00 17.85           C  \\nATOM    974  C   THR A 124      20.413  19.068  22.656  1.00 17.08           C  \\nATOM    975  O   THR A 124      20.459  17.926  23.091  1.00 19.59           O  \\nATOM    976  CB  THR A 124      20.311  20.578  24.672  1.00 19.42           C  \\nATOM    977  OG1 THR A 124      19.109  21.230  24.277  1.00 20.50           O  \\nATOM    978  CG2 THR A 124      21.155  21.458  25.588  1.00 19.06           C  \\nATOM    979  N   VAL A 125      19.788  19.319  21.495  1.00 16.59           N  \\nATOM    980  CA  VAL A 125      19.133  18.324  20.647  1.00 16.09           C  \\nATOM    981  C   VAL A 125      20.032  18.080  19.413  1.00 16.61           C  \\nATOM    982  O   VAL A 125      20.180  18.980  18.573  1.00 14.07           O  \\nATOM    983  CB  VAL A 125      17.720  18.848  20.179  1.00 15.28           C  \\nATOM    984  CG1 VAL A 125      17.097  17.814  19.265  1.00 14.81           C  \\nATOM    985  CG2 VAL A 125      16.824  19.168  21.366  1.00 15.97           C  \\nATOM    986  N   GLU A 126      20.659  16.889  19.280  1.00 17.05           N  \\nATOM    987  CA  GLU A 126      21.487  16.568  18.124  1.00 19.28           C  \\nATOM    988  C   GLU A 126      20.631  16.496  16.876  1.00 18.51           C  \\nATOM    989  O   GLU A 126      19.511  15.976  16.880  1.00 17.68           O  \\nATOM    990  CB  GLU A 126      22.165  15.215  18.219  1.00 23.29           C  \\nATOM    991  CG  GLU A 126      23.244  15.057  19.265  1.00 30.86           C  \\nATOM    992  CD  GLU A 126      24.402  16.054  19.153  1.00 36.59           C  \\nATOM    993  OE1 GLU A 126      25.139  16.012  18.161  1.00 39.64           O  \\nATOM    994  OE2 GLU A 126      24.581  16.885  20.056  1.00 40.44           O  \\nATOM    995  N   SER A 127      21.167  16.985  15.755  1.00 17.73           N  \\nATOM    996  CA  SER A 127      20.426  16.938  14.493  1.00 16.72           C  \\nATOM    997  C   SER A 127      20.040  15.506  14.158  1.00 16.76           C  \\nATOM    998  O   SER A 127      18.931  15.245  13.690  1.00 17.01           O  \\nATOM    999  CB  SER A 127      21.283  17.518  13.371  1.00 17.67           C  \\nATOM   1000  OG  SER A 127      21.730  18.826  13.768  1.00 19.67           O  \\nATOM   1001  N   ARG A 128      20.916  14.558  14.466  1.00 16.72           N  \\nATOM   1002  CA  ARG A 128      20.667  13.154  14.145  1.00 20.94           C  \\nATOM   1003  C   ARG A 128      19.435  12.617  14.862  1.00 19.62           C  \\nATOM   1004  O   ARG A 128      18.641  11.882  14.278  1.00 18.81           O  \\nATOM   1005  CB  ARG A 128      21.910  12.319  14.510  1.00 24.31           C  \\nATOM   1006  CG  ARG A 128      21.969  11.035  13.692  1.00 30.88           C  \\nATOM   1007  CD  ARG A 128      23.381  10.403  13.536  1.00 35.88           C  \\nATOM   1008  NE  ARG A 128      24.384  10.996  12.621  1.00 39.57           N  \\nATOM   1009  CZ  ARG A 128      24.313  11.018  11.267  1.00 40.85           C  \\nATOM   1010  NH1 ARG A 128      25.399  11.388  10.577  1.00 40.83           N  \\nATOM   1011  NH2 ARG A 128      23.202  10.685  10.576  1.00 41.33           N  \\nATOM   1012  N   GLN A 129      19.209  13.016  16.119  1.00 18.27           N  \\nATOM   1013  CA  GLN A 129      18.032  12.519  16.787  1.00 18.53           C  \\nATOM   1014  C   GLN A 129      16.735  13.151  16.224  1.00 17.11           C  \\nATOM   1015  O   GLN A 129      15.691  12.486  16.098  1.00 15.48           O  \\nATOM   1016  CB  GLN A 129      18.229  12.766  18.306  1.00 22.36           C  \\nATOM   1017  CG  GLN A 129      18.041  14.133  18.911  1.00 24.78           C  \\nATOM   1018  CD  GLN A 129      18.283  14.123  20.411  1.00 27.48           C  \\nATOM   1019  OE1 GLN A 129      19.287  14.643  20.906  1.00 28.14           O  \\nATOM   1020  NE2 GLN A 129      17.367  13.497  21.152  1.00 27.21           N  \\nATOM   1021  N   ALA A 130      16.780  14.446  15.837  1.00 14.73           N  \\nATOM   1022  CA  ALA A 130      15.611  15.044  15.210  1.00 13.87           C  \\nATOM   1023  C   ALA A 130      15.342  14.449  13.826  1.00 13.30           C  \\nATOM   1024  O   ALA A 130      14.199  14.194  13.436  1.00 11.42           O  \\nATOM   1025  CB  ALA A 130      15.822  16.536  15.063  1.00 16.21           C  \\nATOM   1026  N   GLN A 131      16.401  14.189  13.057  1.00 14.61           N  \\nATOM   1027  CA  GLN A 131      16.267  13.529  11.739  1.00 16.01           C  \\nATOM   1028  C   GLN A 131      15.637  12.109  11.821  1.00 16.29           C  \\nATOM   1029  O   GLN A 131      14.777  11.708  11.017  1.00 14.37           O  \\nATOM   1030  CB  GLN A 131      17.673  13.507  11.116  1.00 18.12           C  \\nATOM   1031  CG  GLN A 131      17.702  13.017   9.689  1.00 22.19           C  \\nATOM   1032  CD  GLN A 131      16.889  13.922   8.801  1.00 21.61           C  \\nATOM   1033  OE1 GLN A 131      17.292  15.044   8.529  1.00 24.06           O  \\nATOM   1034  NE2 GLN A 131      15.714  13.485   8.370  1.00 23.37           N  \\nATOM   1035  N   ASP A 132      16.038  11.348  12.853  1.00 16.64           N  \\nATOM   1036  CA  ASP A 132      15.467  10.038  13.173  1.00 18.00           C  \\nATOM   1037  C   ASP A 132      13.984  10.175  13.484  1.00 17.28           C  \\nATOM   1038  O   ASP A 132      13.178   9.395  12.985  1.00 16.45           O  \\nATOM   1039  CB  ASP A 132      16.120   9.415  14.405  1.00 20.10           C  \\nATOM   1040  CG  ASP A 132      17.506   8.816  14.234  1.00 24.07           C  \\nATOM   1041  OD1 ASP A 132      18.120   8.459  15.243  1.00 28.35           O  \\nATOM   1042  OD2 ASP A 132      17.975   8.686  13.106  1.00 25.43           O  \\nATOM   1043  N   LEU A 133      13.614  11.168  14.312  1.00 15.58           N  \\nATOM   1044  CA  LEU A 133      12.213  11.390  14.624  1.00 16.28           C  \\nATOM   1045  C   LEU A 133      11.421  11.729  13.366  1.00 14.96           C  \\nATOM   1046  O   LEU A 133      10.345  11.181  13.128  1.00 15.09           O  \\nATOM   1047  CB  LEU A 133      12.052  12.536  15.647  1.00 17.06           C  \\nATOM   1048  CG  LEU A 133      10.617  12.781  16.136  1.00 18.79           C  \\nATOM   1049  CD1 LEU A 133      10.142  11.554  16.881  1.00 19.96           C  \\nATOM   1050  CD2 LEU A 133      10.544  14.013  17.020  1.00 18.45           C  \\nATOM   1051  N   ALA A 134      11.937  12.642  12.545  1.00 13.16           N  \\nATOM   1052  CA  ALA A 134      11.241  13.016  11.331  1.00 14.89           C  \\nATOM   1053  C   ALA A 134      11.092  11.821  10.414  1.00 16.19           C  \\nATOM   1054  O   ALA A 134      10.015  11.643   9.857  1.00 15.70           O  \\nATOM   1055  CB  ALA A 134      12.021  14.089  10.629  1.00 15.12           C  \\nATOM   1056  N   ARG A 135      12.143  10.995  10.279  1.00 17.00           N  \\nATOM   1057  CA  ARG A 135      12.088   9.828   9.438  1.00 19.17           C  \\nATOM   1058  C   ARG A 135      10.990   8.893   9.916  1.00 18.70           C  \\nATOM   1059  O   ARG A 135      10.269   8.352   9.078  1.00 20.29           O  \\nATOM   1060  CB  ARG A 135      13.425   9.091   9.456  1.00 22.48           C  \\nATOM   1061  CG  ARG A 135      13.499   8.297   8.155  1.00 29.02           C  \\nATOM   1062  CD  ARG A 135      14.619   7.270   7.988  1.00 33.22           C  \\nATOM   1063  NE  ARG A 135      14.117   5.888   8.050  1.00 38.80           N  \\nATOM   1064  CZ  ARG A 135      13.281   5.302   7.148  1.00 39.60           C  \\nATOM   1065  NH1 ARG A 135      12.889   4.044   7.384  1.00 40.80           N  \\nATOM   1066  NH2 ARG A 135      12.827   5.913   6.034  1.00 40.11           N  \\nATOM   1067  N   SER A 136      10.815   8.697  11.242  1.00 19.23           N  \\nATOM   1068  CA  SER A 136       9.757   7.841  11.770  1.00 19.12           C  \\nATOM   1069  C   SER A 136       8.374   8.432  11.528  1.00 19.61           C  \\nATOM   1070  O   SER A 136       7.398   7.690  11.522  1.00 21.01           O  \\nATOM   1071  CB  SER A 136       9.930   7.624  13.280  1.00 19.94           C  \\nATOM   1072  OG  SER A 136       9.459   8.686  14.119  1.00 23.21           O  \\nATOM   1073  N   TYR A 137       8.231   9.760  11.380  1.00 17.33           N  \\nATOM   1074  CA  TYR A 137       6.950  10.375  11.078  1.00 18.12           C  \\nATOM   1075  C   TYR A 137       6.683  10.404   9.570  1.00 18.22           C  \\nATOM   1076  O   TYR A 137       5.561  10.656   9.129  1.00 19.27           O  \\nATOM   1077  CB  TYR A 137       6.907  11.812  11.592  1.00 18.21           C  \\nATOM   1078  CG  TYR A 137       6.883  12.057  13.084  1.00 19.08           C  \\nATOM   1079  CD1 TYR A 137       6.619  11.026  13.955  1.00 20.51           C  \\nATOM   1080  CD2 TYR A 137       7.059  13.355  13.548  1.00 20.43           C  \\nATOM   1081  CE1 TYR A 137       6.510  11.285  15.306  1.00 21.89           C  \\nATOM   1082  CE2 TYR A 137       6.945  13.624  14.899  1.00 21.02           C  \\nATOM   1083  CZ  TYR A 137       6.662  12.574  15.763  1.00 22.13           C  \\nATOM   1084  OH  TYR A 137       6.476  12.782  17.112  1.00 22.16           O  \\nATOM   1085  N   GLY A 138       7.689  10.186   8.737  1.00 16.07           N  \\nATOM   1086  CA  GLY A 138       7.520  10.265   7.294  1.00 17.66           C  \\nATOM   1087  C   GLY A 138       7.476  11.725   6.810  1.00 18.01           C  \\nATOM   1088  O   GLY A 138       6.859  12.010   5.774  1.00 17.85           O  \\nATOM   1089  N   ILE A 139       8.128  12.662   7.540  1.00 14.16           N  \\nATOM   1090  CA  ILE A 139       8.117  14.065   7.145  1.00 13.49           C  \\nATOM   1091  C   ILE A 139       9.526  14.662   7.033  1.00 14.35           C  \\nATOM   1092  O   ILE A 139      10.494  14.152   7.610  1.00 16.19           O  \\nATOM   1093  CB  ILE A 139       7.292  14.896   8.161  1.00 14.40           C  \\nATOM   1094  CG1 ILE A 139       7.925  14.811   9.554  1.00 15.44           C  \\nATOM   1095  CG2 ILE A 139       5.821  14.422   8.104  1.00 14.07           C  \\nATOM   1096  CD1 ILE A 139       7.288  15.765  10.587  1.00 15.35           C  \\nATOM   1097  N   PRO A 140       9.706  15.767   6.308  1.00 13.55           N  \\nATOM   1098  CA  PRO A 140      10.984  16.448   6.205  1.00 13.39           C  \\nATOM   1099  C   PRO A 140      11.424  17.065   7.532  1.00 12.74           C  \\nATOM   1100  O   PRO A 140      10.601  17.530   8.310  1.00 11.42           O  \\nATOM   1101  CB  PRO A 140      10.767  17.498   5.127  1.00 13.47           C  \\nATOM   1102  CG  PRO A 140       9.514  17.084   4.410  1.00 14.09           C  \\nATOM   1103  CD  PRO A 140       8.686  16.467   5.519  1.00 13.14           C  \\nATOM   1104  N   TYR A 141      12.744  17.086   7.756  1.00 11.68           N  \\nATOM   1105  CA  TYR A 141      13.336  17.791   8.890  1.00 10.89           C  \\nATOM   1106  C   TYR A 141      14.175  18.945   8.326  1.00 10.09           C  \\nATOM   1107  O   TYR A 141      15.056  18.720   7.468  1.00 10.27           O  \\nATOM   1108  CB  TYR A 141      14.276  16.897   9.691  1.00 12.54           C  \\nATOM   1109  CG  TYR A 141      15.063  17.676  10.733  1.00 10.92           C  \\nATOM   1110  CD1 TYR A 141      14.350  18.396  11.664  1.00 11.75           C  \\nATOM   1111  CD2 TYR A 141      16.453  17.633  10.784  1.00 12.49           C  \\nATOM   1112  CE1 TYR A 141      14.997  19.074  12.670  1.00 12.85           C  \\nATOM   1113  CE2 TYR A 141      17.112  18.307  11.794  1.00 11.33           C  \\nATOM   1114  CZ  TYR A 141      16.369  19.016  12.728  1.00 12.61           C  \\nATOM   1115  OH  TYR A 141      16.944  19.646  13.813  1.00 13.82           O  \\nATOM   1116  N   ILE A 142      13.892  20.174   8.799  1.00  9.35           N  \\nATOM   1117  CA  ILE A 142      14.636  21.355   8.376  1.00  9.05           C  \\nATOM   1118  C   ILE A 142      15.153  22.163   9.551  1.00  9.25           C  \\nATOM   1119  O   ILE A 142      14.409  22.382  10.499  1.00 10.43           O  \\nATOM   1120  CB  ILE A 142      13.756  22.259   7.472  1.00  9.25           C  \\nATOM   1121  CG1 ILE A 142      13.523  21.470   6.180  1.00 10.51           C  \\nATOM   1122  CG2 ILE A 142      14.450  23.586   7.122  1.00  9.08           C  \\nATOM   1123  CD1 ILE A 142      12.719  22.127   5.028  1.00 13.16           C  \\nATOM   1124  N   GLU A 143      16.457  22.513   9.568  1.00  9.01           N  \\nATOM   1125  CA  GLU A 143      16.979  23.404  10.611  1.00  8.52           C  \\nATOM   1126  C   GLU A 143      16.978  24.864  10.193  1.00 10.31           C  \\nATOM   1127  O   GLU A 143      17.278  25.228   9.040  1.00 10.45           O  \\nATOM   1128  CB  GLU A 143      18.397  23.022  11.013  1.00 10.87           C  \\nATOM   1129  CG  GLU A 143      18.363  21.728  11.801  1.00 10.22           C  \\nATOM   1130  CD  GLU A 143      19.592  21.411  12.633  1.00 13.01           C  \\nATOM   1131  OE1 GLU A 143      20.616  22.082  12.521  1.00 13.65           O  \\nATOM   1132  OE2 GLU A 143      19.511  20.487  13.429  1.00 12.70           O  \\nATOM   1133  N   THR A 144      16.608  25.762  11.117  1.00  9.18           N  \\nATOM   1134  CA  THR A 144      16.422  27.169  10.754  1.00  9.71           C  \\nATOM   1135  C   THR A 144      17.014  28.076  11.789  1.00  8.66           C  \\nATOM   1136  O   THR A 144      17.259  27.698  12.927  1.00  9.13           O  \\nATOM   1137  CB  THR A 144      14.919  27.617  10.635  1.00  9.90           C  \\nATOM   1138  OG1 THR A 144      14.336  27.403  11.931  1.00  9.29           O  \\nATOM   1139  CG2 THR A 144      14.120  26.858   9.557  1.00  9.89           C  \\nATOM   1140  N   SER A 145      17.244  29.288  11.360  1.00 10.91           N  \\nATOM   1141  CA  SER A 145      17.558  30.377  12.290  1.00 12.65           C  \\nATOM   1142  C   SER A 145      16.758  31.602  11.849  1.00 10.37           C  \\nATOM   1143  O   SER A 145      16.930  32.094  10.727  1.00 11.76           O  \\nATOM   1144  CB  SER A 145      19.069  30.768  12.308  1.00 11.79           C  \\nATOM   1145  OG  SER A 145      19.248  31.931  13.125  1.00 12.12           O  \\nATOM   1146  N   ALA A 146      15.838  32.046  12.696  1.00  9.49           N  \\nATOM   1147  CA  ALA A 146      15.131  33.268  12.413  1.00 10.90           C  \\nATOM   1148  C   ALA A 146      16.070  34.455  12.553  1.00 13.53           C  \\nATOM   1149  O   ALA A 146      15.799  35.514  11.995  1.00 16.20           O  \\nATOM   1150  CB  ALA A 146      13.976  33.436  13.367  1.00 12.65           C  \\nATOM   1151  N   LYS A 147      17.183  34.302  13.283  1.00 14.35           N  \\nATOM   1152  CA  LYS A 147      18.150  35.351  13.486  1.00 16.25           C  \\nATOM   1153  C   LYS A 147      18.953  35.568  12.215  1.00 17.58           C  \\nATOM   1154  O   LYS A 147      19.124  36.724  11.825  1.00 20.13           O  \\nATOM   1155  CB  LYS A 147      19.068  34.970  14.617  1.00 17.55           C  \\nATOM   1156  CG  LYS A 147      20.003  36.125  14.958  1.00 22.83           C  \\nATOM   1157  CD  LYS A 147      20.581  35.932  16.341  1.00 26.44           C  \\nATOM   1158  CE  LYS A 147      21.683  36.949  16.507  1.00 30.79           C  \\nATOM   1159  NZ  LYS A 147      22.575  36.534  17.574  1.00 33.81           N  \\nATOM   1160  N   THR A 148      19.479  34.529  11.541  1.00 17.16           N  \\nATOM   1161  CA  THR A 148      20.248  34.735  10.300  1.00 15.93           C  \\nATOM   1162  C   THR A 148      19.422  34.607   9.013  1.00 17.72           C  \\nATOM   1163  O   THR A 148      19.926  34.962   7.945  1.00 18.47           O  \\nATOM   1164  CB  THR A 148      21.444  33.745  10.209  1.00 15.86           C  \\nATOM   1165  OG1 THR A 148      20.886  32.435  10.042  1.00 14.56           O  \\nATOM   1166  CG2 THR A 148      22.377  33.839  11.439  1.00 15.25           C  \\nATOM   1167  N   ARG A 149      18.176  34.108   9.100  1.00 15.18           N  \\nATOM   1168  CA  ARG A 149      17.268  33.815   7.994  1.00 19.20           C  \\nATOM   1169  C   ARG A 149      17.652  32.483   7.318  1.00 17.94           C  \\nATOM   1170  O   ARG A 149      17.031  32.079   6.332  1.00 20.28           O  \\nATOM   1171  CB  ARG A 149      17.251  34.964   6.937  1.00 21.40           C  \\nATOM   1172  CG  ARG A 149      15.929  35.106   6.188  1.00 27.00           C  \\nATOM   1173  CD  ARG A 149      15.946  35.904   4.863  1.00 27.99           C  \\nATOM   1174  NE  ARG A 149      14.707  35.687   4.092  1.00 29.40           N  \\nATOM   1175  CZ  ARG A 149      13.550  36.232   4.492  1.00 29.07           C  \\nATOM   1176  NH1 ARG A 149      12.412  36.015   3.828  1.00 29.56           N  \\nATOM   1177  NH2 ARG A 149      13.527  37.004   5.577  1.00 28.99           N  \\nATOM   1178  N   GLN A 150      18.621  31.701   7.838  1.00 15.80           N  \\nATOM   1179  CA  GLN A 150      18.926  30.402   7.273  1.00 16.36           C  \\nATOM   1180  C   GLN A 150      17.737  29.464   7.455  1.00 15.02           C  \\nATOM   1181  O   GLN A 150      17.119  29.351   8.504  1.00 13.57           O  \\nATOM   1182  CB  GLN A 150      20.152  29.729   7.937  1.00 23.05           C  \\nATOM   1183  CG  GLN A 150      20.578  28.448   7.162  1.00 29.37           C  \\nATOM   1184  CD  GLN A 150      21.604  27.530   7.827  1.00 34.84           C  \\nATOM   1185  OE1 GLN A 150      21.301  26.416   8.280  1.00 38.95           O  \\nATOM   1186  NE2 GLN A 150      22.857  27.967   7.883  1.00 35.22           N  \\nATOM   1187  N   GLY A 151      17.412  28.809   6.348  1.00 13.62           N  \\nATOM   1188  CA  GLY A 151      16.357  27.835   6.296  1.00 13.50           C  \\nATOM   1189  C   GLY A 151      14.943  28.395   6.329  1.00 12.36           C  \\nATOM   1190  O   GLY A 151      14.025  27.591   6.148  1.00 11.07           O  \\nATOM   1191  N   VAL A 152      14.706  29.702   6.510  1.00 12.59           N  \\nATOM   1192  CA  VAL A 152      13.339  30.183   6.715  1.00 14.23           C  \\nATOM   1193  C   VAL A 152      12.435  29.971   5.475  1.00 13.41           C  \\nATOM   1194  O   VAL A 152      11.352  29.362   5.577  1.00 12.94           O  \\nATOM   1195  CB  VAL A 152      13.430  31.691   7.167  1.00 12.57           C  \\nATOM   1196  CG1 VAL A 152      12.059  32.316   7.194  1.00 13.14           C  \\nATOM   1197  CG2 VAL A 152      14.044  31.787   8.543  1.00 12.69           C  \\nATOM   1198  N   GLU A 153      12.831  30.410   4.278  1.00 13.52           N  \\nATOM   1199  CA  GLU A 153      12.001  30.213   3.086  1.00 14.81           C  \\nATOM   1200  C   GLU A 153      11.857  28.756   2.742  1.00 11.88           C  \\nATOM   1201  O   GLU A 153      10.797  28.282   2.344  1.00 13.92           O  \\nATOM   1202  CB  GLU A 153      12.592  30.936   1.881  1.00 17.03           C  \\nATOM   1203  CG  GLU A 153      12.454  32.452   2.015  1.00 21.18           C  \\nATOM   1204  CD  GLU A 153      12.870  33.318   0.817  1.00 25.73           C  \\nATOM   1205  OE1 GLU A 153      13.232  32.781  -0.234  1.00 28.32           O  \\nATOM   1206  OE2 GLU A 153      12.822  34.550   0.936  1.00 26.64           O  \\nATOM   1207  N   ASP A 154      12.959  28.031   2.957  1.00 11.31           N  \\nATOM   1208  CA  ASP A 154      13.023  26.593   2.747  1.00 10.16           C  \\nATOM   1209  C   ASP A 154      11.964  25.828   3.561  1.00 10.90           C  \\nATOM   1210  O   ASP A 154      11.224  24.985   3.035  1.00 11.83           O  \\nATOM   1211  CB  ASP A 154      14.420  26.212   3.117  1.00 15.00           C  \\nATOM   1212  CG  ASP A 154      14.919  24.942   2.524  1.00 19.04           C  \\nATOM   1213  OD1 ASP A 154      14.121  24.195   1.938  1.00 22.96           O  \\nATOM   1214  OD2 ASP A 154      16.122  24.697   2.676  1.00 21.26           O  \\nATOM   1215  N   ALA A 155      11.854  26.107   4.881  1.00  8.83           N  \\nATOM   1216  CA  ALA A 155      10.841  25.450   5.716  1.00 10.00           C  \\nATOM   1217  C   ALA A 155       9.424  25.748   5.203  1.00 10.10           C  \\nATOM   1218  O   ALA A 155       8.653  24.814   4.978  1.00 11.14           O  \\nATOM   1219  CB  ALA A 155      10.923  25.933   7.192  1.00  9.50           C  \\nATOM   1220  N   PHE A 156       9.054  27.011   4.949  1.00 10.59           N  \\nATOM   1221  CA  PHE A 156       7.712  27.345   4.475  1.00 10.51           C  \\nATOM   1222  C   PHE A 156       7.453  26.825   3.069  1.00 11.05           C  \\nATOM   1223  O   PHE A 156       6.368  26.302   2.801  1.00 10.80           O  \\nATOM   1224  CB  PHE A 156       7.492  28.890   4.506  1.00 11.54           C  \\nATOM   1225  CG  PHE A 156       7.266  29.450   5.936  1.00 12.42           C  \\nATOM   1226  CD1 PHE A 156       8.319  29.958   6.707  1.00 12.00           C  \\nATOM   1227  CD2 PHE A 156       5.978  29.434   6.478  1.00 12.57           C  \\nATOM   1228  CE1 PHE A 156       8.065  30.434   8.001  1.00 12.45           C  \\nATOM   1229  CE2 PHE A 156       5.747  29.908   7.757  1.00 12.32           C  \\nATOM   1230  CZ  PHE A 156       6.782  30.406   8.513  1.00 10.18           C  \\nATOM   1231  N   TYR A 157       8.408  26.935   2.151  1.00  9.69           N  \\nATOM   1232  CA  TYR A 157       8.148  26.463   0.798  1.00 12.31           C  \\nATOM   1233  C   TYR A 157       8.091  24.945   0.719  1.00 10.42           C  \\nATOM   1234  O   TYR A 157       7.292  24.403  -0.047  1.00 11.35           O  \\nATOM   1235  CB  TYR A 157       9.233  26.978  -0.156  1.00 15.16           C  \\nATOM   1236  CG  TYR A 157       9.257  28.492  -0.367  1.00 18.73           C  \\nATOM   1237  CD1 TYR A 157       8.380  29.366   0.268  1.00 18.52           C  \\nATOM   1238  CD2 TYR A 157      10.211  28.971  -1.242  1.00 20.44           C  \\nATOM   1239  CE1 TYR A 157       8.466  30.724   0.025  1.00 22.04           C  \\nATOM   1240  CE2 TYR A 157      10.305  30.320  -1.488  1.00 22.70           C  \\nATOM   1241  CZ  TYR A 157       9.436  31.178  -0.852  1.00 23.03           C  \\nATOM   1242  OH  TYR A 157       9.572  32.528  -1.089  1.00 26.15           O  \\nATOM   1243  N   THR A 158       8.940  24.198   1.455  1.00 11.35           N  \\nATOM   1244  CA  THR A 158       8.836  22.727   1.522  1.00 10.11           C  \\nATOM   1245  C   THR A 158       7.440  22.347   2.064  1.00 11.11           C  \\nATOM   1246  O   THR A 158       6.811  21.408   1.558  1.00 11.43           O  \\nATOM   1247  CB  THR A 158       9.935  22.129   2.470  1.00 12.09           C  \\nATOM   1248  OG1 THR A 158      11.216  22.451   1.933  1.00 11.36           O  \\nATOM   1249  CG2 THR A 158       9.791  20.657   2.624  1.00 10.87           C  \\nATOM   1250  N   LEU A 159       6.888  23.056   3.072  1.00  9.25           N  \\nATOM   1251  CA  LEU A 159       5.533  22.754   3.506  1.00  9.93           C  \\nATOM   1252  C   LEU A 159       4.489  22.986   2.388  1.00 10.42           C  \\nATOM   1253  O   LEU A 159       3.560  22.185   2.272  1.00 11.22           O  \\nATOM   1254  CB  LEU A 159       5.171  23.608   4.711  1.00  9.03           C  \\nATOM   1255  CG  LEU A 159       3.789  23.318   5.269  1.00 10.96           C  \\nATOM   1256  CD1 LEU A 159       3.635  21.828   5.651  1.00  9.29           C  \\nATOM   1257  CD2 LEU A 159       3.580  24.223   6.469  1.00 13.71           C  \\nATOM   1258  N   VAL A 160       4.596  24.046   1.548  1.00 10.91           N  \\nATOM   1259  CA  VAL A 160       3.662  24.284   0.439  1.00  9.19           C  \\nATOM   1260  C   VAL A 160       3.803  23.137  -0.556  1.00  9.60           C  \\nATOM   1261  O   VAL A 160       2.781  22.638  -1.031  1.00 10.29           O  \\nATOM   1262  CB  VAL A 160       3.981  25.653  -0.211  1.00  8.50           C  \\nATOM   1263  CG1 VAL A 160       3.402  25.754  -1.622  1.00 10.56           C  \\nATOM   1264  CG2 VAL A 160       3.403  26.741   0.659  1.00  7.98           C  \\nATOM   1265  N   ARG A 161       5.033  22.667  -0.839  1.00  9.10           N  \\nATOM   1266  CA  ARG A 161       5.206  21.554  -1.767  1.00  9.82           C  \\nATOM   1267  C   ARG A 161       4.644  20.240  -1.215  1.00 10.44           C  \\nATOM   1268  O   ARG A 161       4.166  19.395  -1.984  1.00 12.88           O  \\nATOM   1269  CB  ARG A 161       6.679  21.398  -2.115  1.00  9.39           C  \\nATOM   1270  CG  ARG A 161       7.146  22.640  -2.816  1.00  9.70           C  \\nATOM   1271  CD  ARG A 161       8.594  22.398  -3.227  1.00 14.12           C  \\nATOM   1272  NE  ARG A 161       9.122  23.635  -3.780  1.00 14.15           N  \\nATOM   1273  CZ  ARG A 161      10.144  24.313  -3.257  1.00 12.45           C  \\nATOM   1274  NH1 ARG A 161      10.823  23.858  -2.204  1.00 13.24           N  \\nATOM   1275  NH2 ARG A 161      10.485  25.475  -3.819  1.00 13.52           N  \\nATOM   1276  N   GLU A 162       4.663  20.046   0.112  1.00 10.71           N  \\nATOM   1277  CA  GLU A 162       4.054  18.877   0.722  1.00 11.80           C  \\nATOM   1278  C   GLU A 162       2.522  18.887   0.568  1.00 11.20           C  \\nATOM   1279  O   GLU A 162       1.921  17.848   0.274  1.00 12.90           O  \\nATOM   1280  CB  GLU A 162       4.430  18.815   2.216  1.00 13.86           C  \\nATOM   1281  CG  GLU A 162       5.870  18.460   2.580  1.00 14.10           C  \\nATOM   1282  CD  GLU A 162       6.268  17.038   2.224  1.00 17.23           C  \\nATOM   1283  OE1 GLU A 162       5.571  16.116   2.627  1.00 19.29           O  \\nATOM   1284  OE2 GLU A 162       7.278  16.827   1.545  1.00 22.14           O  \\nATOM   1285  N   ILE A 163       1.869  20.045   0.731  1.00 10.13           N  \\nATOM   1286  CA  ILE A 163       0.426  20.175   0.542  1.00 10.58           C  \\nATOM   1287  C   ILE A 163       0.128  19.878  -0.933  1.00 13.43           C  \\nATOM   1288  O   ILE A 163      -0.796  19.118  -1.235  1.00 15.01           O  \\nATOM   1289  CB  ILE A 163      -0.015  21.590   0.891  1.00 11.33           C  \\nATOM   1290  CG1 ILE A 163       0.084  21.777   2.400  1.00 11.27           C  \\nATOM   1291  CG2 ILE A 163      -1.486  21.815   0.437  1.00 12.37           C  \\nATOM   1292  CD1 ILE A 163      -0.006  23.245   2.894  1.00 12.56           C  \\nATOM   1293  N   ARG A 164       0.904  20.456  -1.871  1.00 11.72           N  \\nATOM   1294  CA  ARG A 164       0.770  20.163  -3.283  1.00 13.64           C  \\nATOM   1295  C   ARG A 164       0.813  18.657  -3.626  1.00 17.97           C  \\nATOM   1296  O   ARG A 164       0.150  18.197  -4.559  1.00 18.12           O  \\nATOM   1297  CB  ARG A 164       1.897  20.887  -4.029  1.00 12.51           C  \\nATOM   1298  CG  ARG A 164       1.647  22.371  -4.120  1.00 12.49           C  \\nATOM   1299  CD  ARG A 164       2.801  22.888  -4.950  1.00 14.72           C  \\nATOM   1300  NE  ARG A 164       2.730  24.327  -5.113  1.00 12.82           N  \\nATOM   1301  CZ  ARG A 164       3.800  25.071  -5.405  1.00 13.29           C  \\nATOM   1302  NH1 ARG A 164       5.018  24.539  -5.527  1.00 11.56           N  \\nATOM   1303  NH2 ARG A 164       3.595  26.381  -5.572  1.00 12.55           N  \\nATOM   1304  N   GLN A 165       1.590  17.831  -2.935  1.00 20.23           N  \\nATOM   1305  CA  GLN A 165       1.673  16.435  -3.337  1.00 27.50           C  \\nATOM   1306  C   GLN A 165       0.782  15.499  -2.502  1.00 29.12           C  \\nATOM   1307  O   GLN A 165       0.826  14.274  -2.655  1.00 29.30           O  \\nATOM   1308  CB  GLN A 165       3.168  16.070  -3.286  1.00 30.34           C  \\nATOM   1309  CG  GLN A 165       4.018  16.904  -4.309  1.00 35.17           C  \\nATOM   1310  CD  GLN A 165       4.109  16.470  -5.796  1.00 37.71           C  \\nATOM   1311  OE1 GLN A 165       4.256  17.293  -6.719  1.00 36.70           O  \\nATOM   1312  NE2 GLN A 165       4.079  15.166  -6.094  1.00 38.13           N  \\nATOM   1313  N   HIS A 166      -0.068  16.075  -1.638  1.00 29.28           N  \\nATOM   1314  CA  HIS A 166      -0.982  15.296  -0.820  1.00 31.13           C  \\nATOM   1315  C   HIS A 166      -2.214  14.971  -1.665  1.00 33.30           C  \\nATOM   1316  O   HIS A 166      -2.630  15.786  -2.497  1.00 34.96           O  \\nATOM   1317  CB  HIS A 166      -1.420  16.090   0.428  1.00 30.89           C  \\nATOM   1318  CG  HIS A 166      -2.247  15.319   1.475  1.00 32.17           C  \\nATOM   1319  ND1 HIS A 166      -3.579  15.231   1.603  1.00 33.52           N  \\nATOM   1320  CD2 HIS A 166      -1.712  14.560   2.502  1.00 32.95           C  \\nATOM   1321  CE1 HIS A 166      -3.857  14.464   2.647  1.00 33.07           C  \\nATOM   1322  NE2 HIS A 166      -2.727  14.064   3.184  1.00 33.06           N  \\nATOM   1323  OXT HIS A 166      -2.763  13.888  -1.454  1.00 36.27           O  \\nTER    1324      HIS A 166                                                      \\nHETATM 1325 MG    MG A 168       4.771  33.860  19.213  1.00  9.42          MG  \\nHETATM 1326  PG  GNP A 167       5.113  32.223  21.994  1.00  9.66           P  \\nHETATM 1327  O1G GNP A 167       4.844  32.596  23.390  1.00  9.26           O  \\nHETATM 1328  O2G GNP A 167       4.203  32.910  21.112  1.00 11.20           O  \\nHETATM 1329  O3G GNP A 167       4.640  30.769  21.802  1.00 10.09           O  \\nHETATM 1330  N3B GNP A 167       6.701  32.303  21.621  1.00  8.18           N  \\nHETATM 1331  PB  GNP A 167       7.310  32.034  20.111  1.00  8.77           P  \\nHETATM 1332  O1B GNP A 167       7.468  30.605  19.791  1.00  8.35           O  \\nHETATM 1333  O2B GNP A 167       6.696  32.775  19.021  1.00  9.38           O  \\nHETATM 1334  O3A GNP A 167       8.792  32.643  20.152  1.00 10.15           O  \\nHETATM 1335  PA  GNP A 167       9.280  34.140  19.710  1.00 10.18           P  \\nHETATM 1336  O1A GNP A 167       9.263  34.268  18.220  1.00 10.07           O  \\nHETATM 1337  O2A GNP A 167       8.594  35.173  20.510  1.00  9.33           O  \\nHETATM 1338  O5' GNP A 167      10.826  34.056  20.132  1.00 11.46           O  \\nHETATM 1339  C5' GNP A 167      11.170  33.911  21.535  1.00 10.56           C  \\nHETATM 1340  C4' GNP A 167      12.587  34.445  21.811  1.00 11.96           C  \\nHETATM 1341  O4' GNP A 167      13.560  33.649  21.095  1.00 11.66           O  \\nHETATM 1342  C3' GNP A 167      12.813  35.910  21.359  1.00 12.06           C  \\nHETATM 1343  O3' GNP A 167      13.793  36.538  22.221  1.00 14.52           O  \\nHETATM 1344  C2' GNP A 167      13.325  35.679  19.947  1.00 11.90           C  \\nHETATM 1345  O2' GNP A 167      14.151  36.750  19.499  1.00 14.99           O  \\nHETATM 1346  C1' GNP A 167      14.242  34.478  20.156  1.00 12.90           C  \\nHETATM 1347  N9  GNP A 167      14.454  33.739  18.896  1.00 12.98           N  \\nHETATM 1348  C8  GNP A 167      13.518  33.262  17.993  1.00 11.76           C  \\nHETATM 1349  N7  GNP A 167      14.016  32.636  16.979  1.00 12.34           N  \\nHETATM 1350  C5  GNP A 167      15.413  32.694  17.200  1.00 12.36           C  \\nHETATM 1351  C6  GNP A 167      16.521  32.219  16.402  1.00 12.30           C  \\nHETATM 1352  O6  GNP A 167      16.507  31.591  15.338  1.00 12.16           O  \\nHETATM 1353  N1  GNP A 167      17.735  32.503  16.995  1.00 11.60           N  \\nHETATM 1354  C2  GNP A 167      17.909  33.082  18.236  1.00 13.91           C  \\nHETATM 1355  N2  GNP A 167      19.173  33.280  18.658  1.00 14.60           N  \\nHETATM 1356  N3  GNP A 167      16.884  33.498  18.996  1.00 13.01           N  \\nHETATM 1357  C4  GNP A 167      15.675  33.331  18.398  1.00 14.36           C  \\nHETATM 1358  O   HOH A 169      13.588  43.386  17.779  1.00 31.80           O  \\nHETATM 1359  O   HOH A 170       8.248  36.064  23.209  1.00 13.63           O  \\nHETATM 1360  O   HOH A 171       3.254  37.845  19.454  1.00 18.29           O  \\nHETATM 1361  O   HOH A 172       5.945  35.619  20.105  1.00 11.33           O  \\nHETATM 1362  O   HOH A 173       3.669  32.226  18.172  1.00  8.22           O  \\nHETATM 1363  O   HOH A 174       2.896  37.170  15.500  1.00 21.37           O  \\nHETATM 1364  O   HOH A 175       2.102  32.309  24.122  1.00 27.47           O  \\nHETATM 1365  O   HOH A 176      -1.425  37.650  19.667  1.00 24.98           O  \\nHETATM 1366  O   HOH A 177      -3.225  30.031  -8.217  1.00 23.57           O  \\nHETATM 1367  O   HOH A 178      16.678  26.321  26.916  1.00 36.70           O  \\nHETATM 1368  O   HOH A 179      -2.134  40.975   1.551  1.00 81.53           O  \\nHETATM 1369  O   HOH A 180       0.400  40.675  -0.629  1.00 24.66           O  \\nHETATM 1370  O   HOH A 181      10.944  44.793  16.009  1.00 48.47           O  \\nHETATM 1371  O   HOH A 182      17.207  42.570  16.786  1.00 43.23           O  \\nHETATM 1372  O   HOH A 183      14.999  45.167  19.922  1.00 39.52           O  \\nHETATM 1373  O   HOH A 184      11.598  37.755  13.202  1.00 11.72           O  \\nHETATM 1374  O   HOH A 185       6.958  38.922  18.146  1.00 26.17           O  \\nHETATM 1375  O   HOH A 186      14.078  32.033  27.772  1.00 57.73           O  \\nHETATM 1376  O   HOH A 187       9.126  37.906  19.595  1.00 32.42           O  \\nHETATM 1377  O   HOH A 188      -0.049  36.435  27.259  1.00 50.56           O  \\nHETATM 1378  O   HOH A 189       3.843  30.503  25.485  1.00 46.80           O  \\nHETATM 1379  O   HOH A 190       4.931  28.343  27.587  1.00 29.32           O  \\nHETATM 1380  O   HOH A 191       4.881  25.774  26.247  1.00 28.55           O  \\nHETATM 1381  O   HOH A 193       8.729  26.516  26.645  1.00 37.34           O  \\nHETATM 1382  O   HOH A 194      -5.515  23.731  -4.213  1.00 43.74           O  \\nHETATM 1383  O   HOH A 195      -7.006  26.635   0.729  1.00 25.77           O  \\nHETATM 1384  O   HOH A 196      -2.033  23.281  -4.754  1.00 36.89           O  \\nHETATM 1385  O   HOH A 197      -4.326  24.642  -0.854  1.00 28.59           O  \\nHETATM 1386  O   HOH A 198      -6.858  26.995  -2.808  1.00 28.83           O  \\nHETATM 1387  O   HOH A 200      -3.937  29.225   8.289  1.00 24.08           O  \\nHETATM 1388  O   HOH A 201      -8.934  24.037   2.795  1.00 34.16           O  \\nHETATM 1389  O   HOH A 202       1.444  26.302  19.318  1.00 12.37           O  \\nHETATM 1390  O   HOH A 203       2.031  24.632  27.307  1.00 50.09           O  \\nHETATM 1391  O   HOH A 204      15.125  28.341  23.174  1.00 13.88           O  \\nHETATM 1392  O   HOH A 205      12.808  31.541  25.056  1.00 46.88           O  \\nHETATM 1393  O   HOH A 206       8.329  34.933   0.543  1.00 53.21           O  \\nHETATM 1394  O   HOH A 207       0.760  42.969   2.336  1.00 35.42           O  \\nHETATM 1395  O   HOH A 208       1.270  40.298   7.071  1.00 76.61           O  \\nHETATM 1396  O   HOH A 209      -3.135  39.625   7.649  1.00 35.40           O  \\nHETATM 1397  O   HOH A 210       8.699  39.076   2.399  1.00 33.93           O  \\nHETATM 1398  O   HOH A 212      10.040  41.806   3.447  1.00 37.88           O  \\nHETATM 1399  O   HOH A 213      18.914  29.095   3.704  1.00 25.32           O  \\nHETATM 1400  O   HOH A 214      15.966  29.098   2.957  1.00 18.63           O  \\nHETATM 1401  O   HOH A 215      15.506  31.962   4.015  1.00 18.93           O  \\nHETATM 1402  O   HOH A 216      22.935  30.533  10.845  1.00 39.81           O  \\nHETATM 1403  O   HOH A 217       6.476  43.893  23.325  1.00 49.09           O  \\nHETATM 1404  O   HOH A 218      11.746  41.248  28.612  1.00 36.22           O  \\nHETATM 1405  O   HOH A 219      11.480  42.270  25.715  1.00 29.66           O  \\nHETATM 1406  O   HOH A 220       9.344  34.251  -8.278  1.00 36.61           O  \\nHETATM 1407  O   HOH A 221      -0.111  19.667  -8.066  1.00 44.76           O  \\nHETATM 1408  O   HOH A 222      -0.528  30.460 -10.943  1.00 24.29           O  \\nHETATM 1409  O   HOH A 223     -11.352  24.274  19.491  1.00 31.21           O  \\nHETATM 1410  O   HOH A 224      -4.293  30.642 -16.466  1.00127.55           O  \\nHETATM 1411  O   HOH A 225       3.466  29.729 -16.232  1.00 26.15           O  \\nHETATM 1412  O   HOH A 227      -4.288  18.610   4.198  1.00 34.77           O  \\nHETATM 1413  O   HOH A 228       4.626  16.154   5.151  1.00 14.96           O  \\nHETATM 1414  O   HOH A 229       5.681  11.069   2.580  1.00 59.76           O  \\nHETATM 1415  O   HOH A 230       1.173  14.021   3.406  1.00 32.15           O  \\nHETATM 1416  O   HOH A 231       7.935  22.484  20.504  1.00 10.86           O  \\nHETATM 1417  O   HOH A 232       3.071  10.004  14.546  1.00 23.48           O  \\nHETATM 1418  O   HOH A 233       2.437  13.381   8.401  1.00 17.75           O  \\nHETATM 1419  O   HOH A 234       4.779   7.500   6.154  1.00 71.09           O  \\nHETATM 1420  O   HOH A 235      -4.250   8.748   6.880  1.00 24.76           O  \\nHETATM 1421  O   HOH A 236     -13.381  22.830  17.669  1.00 39.09           O  \\nHETATM 1422  O   HOH A 237      16.969  39.052   8.276  1.00 50.25           O  \\nHETATM 1423  O   HOH A 238      -6.315   9.333  13.897  1.00 36.24           O  \\nHETATM 1424  O   HOH A 240      -0.160   4.395  12.468  1.00 38.79           O  \\nHETATM 1425  O   HOH A 241       1.295   7.931  12.335  1.00 49.03           O  \\nHETATM 1426  O   HOH A 242       4.067  11.551   6.968  1.00 46.97           O  \\nHETATM 1427  O   HOH A 243      -5.060   6.579  -0.216  1.00 50.14           O  \\nHETATM 1428  O   HOH A 244      18.164   7.151   5.573  1.00 71.67           O  \\nHETATM 1429  O   HOH A 245       2.068   8.904   7.636  1.00 45.86           O  \\nHETATM 1430  O   HOH A 246      24.661  23.134  12.757  1.00 34.69           O  \\nHETATM 1431  O   HOH A 247      21.051  23.714   6.239  1.00 24.64           O  \\nHETATM 1432  O   HOH A 248      25.855  31.137  20.712  1.00 34.53           O  \\nHETATM 1433  O   HOH A 249      24.868  30.890  26.223  1.00 48.47           O  \\nHETATM 1434  O   HOH A 250      26.967  20.447  19.448  1.00 47.77           O  \\nHETATM 1435  O   HOH A 252      28.453  14.196  14.602  1.00 39.78           O  \\nHETATM 1436  O   HOH A 253      19.639  16.256   9.760  1.00 37.54           O  \\nHETATM 1437  O   HOH A 254      16.629   7.383  10.526  1.00 60.90           O  \\nHETATM 1438  O   HOH A 255       4.972   7.888  14.276  1.00 54.95           O  \\nHETATM 1439  O   HOH A 256      14.184  14.684   5.643  1.00 31.40           O  \\nHETATM 1440  O   HOH A 257      11.766  17.254   1.278  1.00 41.82           O  \\nHETATM 1441  O   HOH A 258      16.274  16.365   6.342  1.00 17.29           O  \\nHETATM 1442  O   HOH A 259      18.346  24.095   6.620  1.00 17.36           O  \\nHETATM 1443  O   HOH A 260      22.479  35.073  19.694  1.00 43.96           O  \\nHETATM 1444  O   HOH A 261      20.580  31.747   4.906  1.00 59.46           O  \\nHETATM 1445  O   HOH A 262      18.196  25.514   4.261  1.00 28.75           O  \\nHETATM 1446  O   HOH A 263      12.857  20.183   1.149  1.00 21.79           O  \\nHETATM 1447  O   HOH A 264      19.514  15.912  24.943  1.00 55.52           O  \\nHETATM 1448  O   HOH A 265       8.925   7.579   5.621  1.00 77.94           O  \\nHETATM 1449  O   HOH A 266      17.622  21.835  26.964  1.00 27.48           O  \\nHETATM 1450  O   HOH A 267      24.258  19.786  14.155  1.00 20.48           O  \\nHETATM 1451  O   HOH A 268      23.806  15.180  15.136  1.00 19.86           O  \\nHETATM 1452  O   HOH A 269      21.478  14.557  11.134  1.00 29.66           O  \\nHETATM 1453  O   HOH A 270      17.293  45.621  24.753  1.00 39.18           O  \\nHETATM 1454  O   HOH A 271      -3.518  40.206  17.649  1.00 44.75           O  \\nHETATM 1455  O   HOH A 272       2.143  19.852  24.090  1.00 40.10           O  \\nHETATM 1456  O   HOH A 273       7.101  35.918  -4.704  1.00 31.76           O  \\nHETATM 1457  O   HOH A 274       5.462  38.756  -5.521  1.00 27.51           O  \\nHETATM 1458  O   HOH A 275       5.748  20.565  28.638  1.00 58.06           O  \\nHETATM 1459  O   HOH A 276       8.814  31.258  -8.964  1.00 23.79           O  \\nHETATM 1460  O   HOH A 277     -10.427  35.895   4.301  1.00 58.91           O  \\nHETATM 1461  O   HOH A 278      -8.713  28.579   2.333  1.00 69.36           O  \\nHETATM 1462  O   HOH A 279      24.472  24.237  17.316  1.00 32.14           O  \\nHETATM 1463  O   HOH A 281      16.968  32.701  21.815  1.00 23.13           O  \\nHETATM 1464  O   HOH A 282      19.541  34.346  21.581  1.00 53.25           O  \\nHETATM 1465  O   HOH A 283      24.847  23.384  26.621  1.00 61.18           O  \\nHETATM 1466  O   HOH A 284       1.409  11.113   2.873  1.00 44.87           O  \\nHETATM 1467  O   HOH A 285       1.556  11.488  -0.914  1.00 37.86           O  \\nHETATM 1468  O   HOH A 286       9.758  12.945   3.889  1.00 36.47           O  \\nHETATM 1469  O   HOH A 287       3.827  13.375   4.622  1.00 44.90           O  \\nHETATM 1470  O   HOH A 288      14.279   6.175  15.391  1.00 78.25           O  \\nHETATM 1471  O   HOH A 289       7.573  13.968   2.191  1.00 76.47           O  \\nHETATM 1472  O   HOH A 290       2.323  15.482   1.418  1.00 35.15           O  \\nHETATM 1473  O   HOH A 291      11.997  14.573   2.962  1.00 49.86           O  \\nHETATM 1474  O   HOH A 292      23.798  33.550  17.880  1.00 24.64           O  \\nHETATM 1475  O   HOH A 293      14.657  36.046  -0.571  1.00 51.86           O  \\nHETATM 1476  O   HOH A 294      24.456  25.366   4.004  1.00 53.23           O  \\nHETATM 1477  O   HOH A 296      -0.603  12.134  -4.185  1.00 52.38           O  \\nHETATM 1478  O   HOH A 297      -8.372  34.415  -4.471  1.00 74.53           O  \\nHETATM 1479  O   HOH A 300     -14.033  36.501   0.062  1.00 50.21           O  \\nHETATM 1480  O   HOH A 301     -17.102  35.735  -2.806  1.00 42.58           O  \\nHETATM 1481  O   HOH A 303     -17.383  40.542   1.431  1.00 77.16           O  \\nHETATM 1482  O   HOH A 304     -19.024  41.557  -3.411  1.00107.16           O  \\nHETATM 1483  O   HOH A 305      -9.876  37.804   0.839  1.00 90.65           O  \\nHETATM 1484  O   HOH A 306     -14.053  36.466   4.552  1.00 82.05           O  \\nHETATM 1485  O   HOH A 307     -14.183  39.558   0.339  1.00 58.66           O  \\nHETATM 1486  O   HOH A 309     -12.547  28.677   8.301  1.00 74.88           O  \\nHETATM 1487  O   HOH A 311     -14.787  32.746  -2.171  1.00 40.79           O  \\nHETATM 1488  O   HOH A 312      -6.916  30.740   3.095  1.00 36.53           O  \\nHETATM 1489  O   HOH A 315      -8.312  33.027   7.340  1.00 83.28           O  \\nHETATM 1490  O   HOH A 316      -6.280  35.646   7.624  1.00 75.97           O  \\nHETATM 1491  O   HOH A 317      -5.230  31.565   9.609  1.00 55.67           O  \\nHETATM 1492  O   HOH A 318       6.747  37.258   1.208  1.00 57.93           O  \\nHETATM 1493  O   HOH A 319       4.747  42.419  12.683  1.00 57.74           O  \\nHETATM 1494  O   HOH A 320       2.984  41.950  16.191  1.00 41.69           O  \\nHETATM 1495  O   HOH A 321       3.867  49.208  12.056  1.00 24.57           O  \\nHETATM 1496  O   HOH A 322       7.530  42.894  17.776  1.00 79.28           O  \\nHETATM 1497  O   HOH A 324      19.822  39.457  17.826  1.00 66.22           O  \\nHETATM 1498  O   HOH A 325      12.391  45.557  24.973  1.00 48.70           O  \\nHETATM 1499  O   HOH A 326      14.944  43.928  23.794  1.00 62.45           O  \\nHETATM 1500  O   HOH A 327     -12.984  24.008  12.563  1.00 40.31           O  \\nHETATM 1501  O   HOH A 330       0.572  41.133  19.277  1.00 50.77           O  \\nHETATM 1502  O   HOH A 331      -6.196  37.979  23.043  1.00 48.93           O  \\nHETATM 1503  O   HOH A 332      -3.526  30.410  19.065  1.00 21.26           O  \\nHETATM 1504  O   HOH A 333      -7.995  32.093  21.685  1.00 54.11           O  \\nHETATM 1505  O   HOH A 334      -8.483  28.891  20.063  1.00 70.37           O  \\nHETATM 1506  O   HOH A 335     -11.670  31.022  16.981  1.00 76.86           O  \\nHETATM 1507  O   HOH A 336     -11.082  34.450  18.910  1.00 43.07           O  \\nHETATM 1508  O   HOH A 339      -6.521  37.738  15.165  1.00 64.11           O  \\nHETATM 1509  O   HOH A 340     -10.829  35.877  15.629  1.00 58.36           O  \\nHETATM 1510  O   HOH A 341      -6.552  41.166  12.231  1.00 55.67           O  \\nHETATM 1511  O   HOH A 342      -1.242  39.671  12.185  1.00 61.04           O  \\nHETATM 1512  O   HOH A 343      -4.281  38.934  13.464  1.00 32.14           O  \\nHETATM 1513  O   HOH A 344       0.386  41.208  14.039  1.00 40.80           O  \\nHETATM 1514  O   HOH A 345       0.777  39.663  21.603  1.00 67.71           O  \\nHETATM 1515  O   HOH A 346      -7.879  34.848  19.044  1.00 52.60           O  \\nHETATM 1516  O   HOH A 347       1.121  32.594 -11.957  1.00 32.30           O  \\nHETATM 1517  O   HOH A 348       0.924  32.597 -14.716  1.00 48.34           O  \\nHETATM 1518  O   HOH A 349      -2.241  27.348  20.565  1.00 26.54           O  \\nHETATM 1519  O   HOH A 350       3.259  26.214  29.881  1.00 69.22           O  \\nHETATM 1520  O   HOH A 351      -8.318  34.095  28.392  1.00 81.96           O  \\nHETATM 1521  O   HOH A 352      -2.927  29.775  31.412  1.00 80.92           O  \\nHETATM 1522  O   HOH A 353     -10.410  32.034  25.972  1.00 60.17           O  \\nHETATM 1523  O   HOH A 354      -8.122  30.393  25.378  1.00 62.33           O  \\nHETATM 1524  O   HOH A 356      -6.138  18.882  24.644  1.00 49.26           O  \\nHETATM 1525  O   HOH A 358      -1.406  19.033  23.084  1.00 82.05           O  \\nHETATM 1526  O   HOH A 359       4.169   7.339  10.280  1.00 66.03           O  \\nHETATM 1527  O   HOH A 360      -9.761  30.165   7.702  1.00 46.15           O  \\nHETATM 1528  O   HOH A 361     -16.002  27.801  11.700  1.00 41.85           O  \\nHETATM 1529  O   HOH A 362      16.215  30.489  29.264  1.00 67.21           O  \\nHETATM 1530  O   HOH A 364      21.417  27.685  26.140  1.00 49.57           O  \\nHETATM 1531  O   HOH A 366      15.824  34.767  30.662  1.00 25.90           O  \\nHETATM 1532  O   HOH A 367      18.527  35.965  24.111  1.00 68.69           O  \\nHETATM 1533  O   HOH A 369      12.103  35.275  24.945  1.00 51.37           O  \\nHETATM 1534  O   HOH A 370      24.782  11.970  23.641  1.00 55.92           O  \\nHETATM 1535  O   HOH A 371      14.337  21.394  28.627  1.00 30.11           O  \\nHETATM 1536  O   HOH A 372      -1.367  12.600  23.053  1.00 44.43           O  \\nHETATM 1537  O   HOH A 373       7.060   8.567  21.416  1.00 47.99           O  \\nHETATM 1538  O   HOH A 374       7.639   8.040  16.420  1.00 38.76           O  \\nHETATM 1539  O   HOH A 376      -1.282  15.566  25.487  1.00 67.79           O  \\nHETATM 1540  O   HOH A 378      17.945  30.924  23.806  1.00 31.50           O  \\nHETATM 1541  O   HOH A 379      17.360  33.209  33.337  1.00 30.46           O  \\nHETATM 1542  O   HOH A 380       3.971   6.567  22.089  1.00 86.80           O  \\nHETATM 1543  O   HOH A 381      -0.065  10.022  26.928  1.00 61.86           O  \\nHETATM 1544  O   HOH A 382      -4.450  12.966  21.222  1.00 42.82           O  \\nHETATM 1545  O   HOH A 383      -2.584   6.719   5.938  1.00 40.64           O  \\nHETATM 1546  O   HOH A 384      25.567  31.936  12.845  1.00 73.17           O  \\nHETATM 1547  O   HOH A 385      24.446  29.383  22.970  1.00 36.61           O  \\nHETATM 1548  O   HOH A 386      24.865  38.167  18.227  1.00 60.74           O  \\nHETATM 1549  O   HOH A 387      23.135  35.012  14.543  1.00 59.19           O  \\nHETATM 1550  O   HOH A 388      19.325   9.556  18.093  1.00 92.18           O  \\nHETATM 1551  O   HOH A 389      25.326  12.873  16.251  1.00 49.27           O  \\nHETATM 1552  O   HOH A 390      20.219  10.405  10.956  1.00 63.05           O  \\nHETATM 1553  O   HOH A 392       9.452   5.453   8.249  1.00 47.81           O  \\nHETATM 1554  O   HOH A 394      10.139  12.252   0.254  1.00 29.37           O  \\nHETATM 1555  O   HOH A 395      28.004  37.416  13.466  1.00 52.29           O  \\nHETATM 1556  O   HOH A 396      24.283  36.829  12.307  1.00 64.74           O  \\nHETATM 1557  O   HOH A 397      13.748  34.031  -3.019  1.00 47.80           O  \\nHETATM 1558  O   HOH A 398      16.981  34.189   2.006  1.00 80.51           O  \\nHETATM 1559  O   HOH A 399      -4.426  24.223  -7.240  1.00 38.09           O  \\nHETATM 1560  O   HOH A 400      19.676  33.981   3.336  1.00 64.86           O  \\nHETATM 1561  O   HOH A 401      14.633  28.745  -0.317  1.00 66.25           O  \\nHETATM 1562  O   HOH A 402      -2.139  15.576  22.716  1.00 65.01           O  \\nHETATM 1563  O   HOH A 403       9.873  11.090  20.760  1.00 53.88           O  \\nHETATM 1564  O   HOH A 404      18.291  36.379  26.927  1.00 57.23           O  \\nHETATM 1565  O   HOH A 405       0.138   6.145  15.269  1.00 82.80           O  \\nHETATM 1566  O   HOH A 406      -5.515  38.166  18.689  1.00 48.28           O  \\nHETATM 1567  O   HOH A 407      -0.399   4.688   8.809  1.00 58.35           O  \\nHETATM 1568  O   HOH A 408       0.813  25.584  22.307  1.00 40.04           O  \\nCONECT  117 1325                                                                \\nCONECT  265 1325                                                                \\nCONECT 1325  117  265 1328 1333                                                 \\nCONECT 1325 1361 1362                                                           \\nCONECT 1326 1327 1328 1329 1330                                                 \\nCONECT 1327 1326                                                                \\nCONECT 1328 1325 1326                                                           \\nCONECT 1329 1326                                                                \\nCONECT 1330 1326 1331                                                           \\nCONECT 1331 1330 1332 1333 1334                                                 \\nCONECT 1332 1331                                                                \\nCONECT 1333 1325 1331                                                           \\nCONECT 1334 1331 1335                                                           \\nCONECT 1335 1334 1336 1337 1338                                                 \\nCONECT 1336 1335                                                                \\nCONECT 1337 1335                                                                \\nCONECT 1338 1335 1339                                                           \\nCONECT 1339 1338 1340                                                           \\nCONECT 1340 1339 1341 1342                                                      \\nCONECT 1341 1340 1346                                                           \\nCONECT 1342 1340 1343 1344                                                      \\nCONECT 1343 1342                                                                \\nCONECT 1344 1342 1345 1346                                                      \\nCONECT 1345 1344                                                                \\nCONECT 1346 1341 1344 1347                                                      \\nCONECT 1347 1346 1348 1357                                                      \\nCONECT 1348 1347 1349                                                           \\nCONECT 1349 1348 1350                                                           \\nCONECT 1350 1349 1351 1357                                                      \\nCONECT 1351 1350 1352 1353                                                      \\nCONECT 1352 1351                                                                \\nCONECT 1353 1351 1354                                                           \\nCONECT 1354 1353 1355 1356                                                      \\nCONECT 1355 1354                                                                \\nCONECT 1356 1354 1357                                                           \\nCONECT 1357 1347 1350 1356                                                      \\nCONECT 1361 1325                                                                \\nCONECT 1362 1325                                                                \\nMASTER      312    0    2    5    6    0    9    6 1567    1   38   13          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewergrid_16835771918631198[0][1].addModel(\"\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n    8.1498    9.9395    0.8183 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.1695    9.3371    1.5922 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.4103    8.1108    1.4000 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.9316   10.0209    2.5654 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.6967   10.7493    1.1472 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.6864   10.9350    2.8365 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0759    9.0817   -0.2489 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4561    9.6329   -0.2770 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5017    8.5444   -0.1653 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.7693    9.3636   -0.4637 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2842   10.4214   -1.3759 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0242   11.2860   -1.9323 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8490   10.2763   -1.5004 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1270   10.7110   -2.6308 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0018    9.9037   -3.7685 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3069   10.2965   -4.8903 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7101   11.5464   -4.8813 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8178   12.3608   -3.7734 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5166   11.9402   -2.6711 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.0759    9.3652    3.1600 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   12.0671   10.3169    3.7560 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.9955   11.5604    3.1551 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   13.4528    9.7553    3.5843 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   14.4727   10.6536    4.2325 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   14.1515   10.9003    5.6810 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   12.7439   11.3926    5.8652 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.7439   10.4670    5.2278 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   12.2011   11.4757    2.2110 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-4.32704\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n   -5.5591    6.5652    5.3801 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4601    7.2046    4.1219 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3728    6.9851    3.2746 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.4052    8.0696    3.7554 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.7503    6.4196    5.9223 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6134    8.1603    4.3334 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0424    4.3506    9.0315 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1768    5.4043    7.9916 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5582    4.8251    6.6463 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8550    6.1223    5.8745 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3423    7.0404    6.9260 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7729    8.2128    6.7140 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2327    6.3568    8.1977 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0500    6.6586    9.3063 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.3668    6.1884    9.3885 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.1936    6.4652   10.4541 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.6899    7.2434   11.4832 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3963    7.7185   11.4271 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.5976    7.4246   10.3520 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.4994    8.8210    2.5228 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.9032   10.2495    2.7231 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8859   11.1153    2.3654 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.2580   10.4621    4.1700 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.7350   11.8721    4.3970 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8824   12.2209    3.4892 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5596   11.9421    2.0485 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1069   10.5220    1.8454 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.1092   10.9492    2.9221 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-4.22165\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n   -2.0336    7.3002    3.2585 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.0649    8.1827    2.8594 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.1311    8.4991    1.6369 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.0214    8.7315    3.7421 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.7263    7.2945    4.1938 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9237    8.6161    4.7149 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.1505    6.3186    3.2716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.6701    6.1875    3.2975 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.0188    6.9057    2.1353 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.4261    6.4003    2.2883 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.2557    5.0542    2.8758 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.2024    4.2437    3.1029 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.1559    4.8563    3.1297 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.7591    3.5824    3.1699 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.0041    2.9318    4.3858 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.5930    1.6896    4.4628 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.9543    1.0666    3.2798 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.7236    1.6854    2.0689 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.1345    2.9228    2.0258 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.1459    9.4649    3.2031 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.7133   10.4731    4.1546 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.7644   10.8922    5.0692 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8777    9.8598    4.8844 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.5178   10.8671    5.8024 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9231   12.1117    5.0615 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7821   12.6884    4.2721 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1747   11.6688    3.3478 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.4618   10.1332    5.5919 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-4.07662\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n   -7.1997    7.9665    8.2440 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4895    8.3884    6.9251 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0297    7.5540    6.1435 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2111    9.6816    6.4295 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.4990    8.4233    8.7633 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6743   10.3126    6.9614 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3429    6.4030   12.1545 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.0660    6.9950   10.9986 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9185    6.1539    9.7490 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9360    6.8579    8.8347 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.9513    7.3653    9.7822 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.0204    7.9543    9.4433 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4958    7.0654   11.1235 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.3730    6.9003   12.2150 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.9823    5.6669   12.4784 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.8453    5.4702   13.5329 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -11.1149    6.5462   14.3622 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.5276    7.7714   14.1253 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.6717    7.9350   13.0665 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7220   10.0699    5.1330 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9418   11.5450    4.9946 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.7161   11.8372    3.8866 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6084   12.2315    4.8704 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7860   13.7250    4.7975 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.5695   14.2468    5.9705 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.8705   13.5153    6.1433 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.6636   12.0281    6.2352 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.2625   11.5316    3.0855 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-4.06176\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n    7.8347   10.1795    0.6909 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.7621    9.4932    1.5097 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.8104    8.2329    1.4189 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.6337   10.1291    2.4215 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.5151   11.0732    0.9527 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.5346   11.0890    2.6166 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0092   12.0862   -1.9268 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2385   11.4206   -1.4213 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9269   10.1170   -0.7184 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.3521    9.5663   -0.5384 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0788   10.1150   -1.7033 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.2837    9.8487   -1.9897 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1718   10.9844   -2.4230 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2934   11.2507   -3.8023 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7850   10.3656   -4.7615 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.8878   10.5921   -6.1156 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.5236   11.7489   -6.5352 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0325   12.6373   -5.6112 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.9136   12.3832   -4.2690 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.6733    9.3566    3.0658 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.8421   10.1812    3.5104 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.8395   11.4254    2.9066 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   13.1151    9.4574    3.1636 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   14.3126   10.2253    3.6569 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   14.2205   10.5064    5.1316 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   12.9193   11.1641    5.4947 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.7394   10.3666    5.0099 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.9050   11.3181    1.9447 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.99217\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n    2.9603    8.8680    1.1997 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0437    9.3540    0.4305 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0325    8.5864    0.2521 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0879   10.6423   -0.1472 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.3534    9.5001    1.6483 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4022   11.3118    0.0782 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2273    7.0677    4.6963 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.4629    7.4669    3.2840 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.8735    7.1514    2.8354 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7415    7.4343    1.3290 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.3288    7.1046    1.0431 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.8044    7.1268   -0.1098 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.6898    6.7646    2.2971 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.4328    5.9159    2.3833 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.2884    4.5256    2.4720 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.3614    3.6669    2.5551 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.6339    4.2137    2.5498 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.8038    5.5797    2.4639 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.7144    6.4084    2.3829 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1504   10.9687   -1.0731 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1634   12.4090   -1.4843 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0063   12.7502   -2.1605 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3543   12.6568   -2.3702 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4392   14.1116   -2.7493 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4587   14.9994   -1.5354 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3114   14.7065   -0.6105 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2620   13.2519   -0.2300 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.9332   12.2155   -2.9665 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.94954\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n   -9.2444    8.5266    7.4724 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.2625    7.6554    8.0002 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6100    6.9395    7.1874 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9693    7.5361    9.3769 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.9882    8.8310    8.0411 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.5421    7.9746   10.0468 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  -11.4592   11.8798    5.9487 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.3335   10.9796    6.3122 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.5387    9.5764    5.7835 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.1567    8.9738    6.0895 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.2478   10.1352    5.9841 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9871   10.0738    6.0920 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.0568   11.3114    5.7424 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.5709   12.4560    5.0777 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0699   13.5530    5.7900 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.5900   14.6872    5.1746 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6086   14.7335    3.7904 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0962   13.6666    3.0651 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.5668   12.5506    3.7080 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8192    6.7635    9.7930 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8444    6.3878   11.2428 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.9345    7.1315   11.9718 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5195    4.9246   11.3775 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6021    4.4938   12.8179 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9426    4.8200   13.4169 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3052    6.2650   13.2214 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.2360    6.6640   11.7727 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0389    6.9619   11.6402 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.74039\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n    2.6681    5.6237    6.8498 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.3542    4.7346    7.7102 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7488    3.6891    8.0837 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6707    4.9446    8.1774 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9325    6.5712    6.8100 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1233    5.8042    8.0181 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.0009    7.8945    3.6041 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.6382    6.7965    4.3773 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.8179    6.4085    5.5886 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.5716    5.1397    6.0231 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.0916    4.5994    4.7488 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7205    3.5056    4.6348 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.7509    5.5353    3.6979 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.5917    5.1539    2.3499 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.3915    4.5993    1.8877 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.2016    4.2136    0.5798 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2505    4.3865   -0.3081 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.4428    4.9309    0.1212 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.6003    5.3057    1.4307 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3476    3.8845    8.8922 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1056    4.3624   10.0925 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4737    5.6889    9.9605 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.3369    3.5146   10.2662 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0905    3.9221   11.5044 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2130    3.8796   12.7252 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9472    4.6654   12.5311 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2028    4.2245   11.3005 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.0528    5.7885    9.1888 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.71957\\n\\n$$$$\\n\\n\\n\\n 28 30  0  0  0  0  0  0  0  0999 V2000\\n   -8.1030    7.8964    8.9253 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8174    7.7427    7.5482 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9203    6.5853    7.0494 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4235    8.8010    6.6996 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7214    8.6542    9.4248 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2046    9.6842    7.0755 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.6448    8.5241   12.8875 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.5734    8.2653   11.4255 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.6438    7.1162   11.0999 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.9602    6.9356    9.6052 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.3754    7.3514    9.5039 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -11.0689    7.2901    8.4456 O   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.7932    7.8247   10.8069 N   0  0  0  0  0  0  0  0  0  0  0  0\\n  -12.1339    7.7953   11.2425 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -12.9502    8.9306   11.1612 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -14.2620    8.9388   11.5790 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -14.7860    7.7675   12.1005 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -14.0036    6.6353   12.1911 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -12.7003    6.6593   11.7658 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3448    8.5749    5.2729 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7115    9.7060    4.5225 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.3685    9.4688    4.2929 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8686   10.9734    5.3187 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3030   12.1470    4.5637 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9169   12.2731    3.1965 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8277   10.9879    2.4232 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4186    9.8363    3.1897 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.9056    9.3836    5.1411 H   0  0  0  0  0  0  0  0  0  0  0  0\\n  8  7  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  2  4  1  0  0  0\\n  4 20  1  0  0  0\\n 20 21  1  0  0  0\\n 21 22  1  0  0  0\\n 21 23  1  0  0  0\\n 23 24  1  0  0  0\\n 24 25  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  1  0  0  0\\n 10 11  1  0  0  0\\n 11 12  2  0  0  0\\n 11 13  1  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 17 18  1  0  0  0\\n 18 19  2  0  0  0\\n 13  8  1  0  0  0\\n 19 14  1  0  0  0\\n 27 21  1  0  0  0\\n  1  5  1  0  0  0\\n  4  6  1  0  0  0\\n 22 28  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.67500\\n\\n$$$$\\n\",\"sdf\");\n",
       "\tviewergrid_16835771918631198[0][1].setStyle({\"model\": 0},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewergrid_16835771918631198[0][1].addSurface(1,{\"opacity\": 0.5, \"color\": \"white\"},{\"model\": 0});\n",
       "\tviewergrid_16835771918631198[0][1].setStyle({\"model\": 1},{\"stick\": {}});\n",
       "\tviewergrid_16835771918631198[0][1].zoomTo();\n",
       "\tviewergrid_16835771918631198[1][0].addModel(\"HEADER    ONCOGENE PROTEIN                        30-APR-90   5P21              \\nTITLE     REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION OF H-RAS   \\nTITLE    2 P21 AT 1.35 ANGSTROMS RESOLUTION: IMPLICATIONS FOR THE MECHANISM OF  \\nTITLE    3 GTP HYDROLYSIS                                                       \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: C-H-RAS P21 PROTEIN;                                       \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    ONCOGENE PROTEIN                                                      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    E.F.PAI,A.WITTINGHOFER,W.KABSCH                                       \\nREVDAT   6   29-NOV-17 5P21    1       HELIX                                    \\nREVDAT   5   24-FEB-09 5P21    1       VERSN                                    \\nREVDAT   4   01-APR-03 5P21    1       JRNL                                     \\nREVDAT   3   20-JUL-95 5P21    1       HET                                      \\nREVDAT   2   30-APR-94 5P21    1       FORMUL                                   \\nREVDAT   1   15-JAN-92 5P21    0                                                \\nJRNL        AUTH   E.F.PAI,U.KRENGEL,G.A.PETSKO,R.S.GOODY,W.KABSCH,             \\nJRNL        AUTH 2 A.WITTINGHOFER                                               \\nJRNL        TITL   REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION   \\nJRNL        TITL 2 OF H-RAS P21 AT 1.35 A RESOLUTION: IMPLICATIONS FOR THE      \\nJRNL        TITL 3 MECHANISM OF GTP HYDROLYSIS.                                 \\nJRNL        REF    EMBO J.                       V.   9  2351 1990              \\nJRNL        REFN                   ISSN 0261-4189                               \\nJRNL        PMID   2196171                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,S.C.ALMO,G.RAPP,K.WILSON,K.PETRATOS,A.LENTFER, \\nREMARK   1  AUTH 2 A.WITTINGHOFER,W.KABSCH,E.F.PAI,G.A.PETSKO,R.S.GOODY         \\nREMARK   1  TITL   TIME-RESOLVED X-RAY CRYSTALLOGRAPHIC STUDY OF THE            \\nREMARK   1  TITL 2 CONFORMATIONAL CHANGE IN HA-RAS P21 PROTEIN ON GTP           \\nREMARK   1  TITL 3 HYDROLYSIS                                                   \\nREMARK   1  REF    NATURE                        V. 345   309 1990              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   A.SCHERER,J.JOHN,R.LINKE,R.S.GOODY,A.WITTINGHOFER,E.F.PAI,   \\nREMARK   1  AUTH 2 K.C.HOLMES                                                   \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY ANALYSIS OF THE HUMAN  \\nREMARK   1  TITL 2 C-H-RAS-ONCOGENE PRODUCT P21 COMPLEXED WITH GTP ANALOGUES    \\nREMARK   1  REF    J.MOL.BIOL.                   V. 206   257 1989              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   E.F.PAI,W.KABSCH,U.KRENGEL,K.C.HOLMES,J.JOHN,A.WITTINGHOFER  \\nREMARK   1  TITL   STRUCTURE OF THE GUANINE-NUCLEOTIDE-BINDING DOMAIN OF THE    \\nREMARK   1  TITL 2 HA-RAS ONCOGENE PRODUCT P21 IN THE TRIPHOSPHATE CONFORMATION \\nREMARK   1  REF    NATURE                        V. 341   209 1989              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,G.RAPP,J.JOHN,A.WITTINGHOFER,E.F.PAI,R.S.GOODY \\nREMARK   1  TITL   BIOCHEMICAL AND CRYSTALLOGRAPHIC CHARACTERIZATION OF A       \\nREMARK   1  TITL 2 COMPLEX OF C-HA-RAS P21 AND CAGED GTP WITH FLASH PHOTOLYSIS  \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  86  7687 1989              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 6.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 26806                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.196                           \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1323                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 33                                      \\nREMARK   3   SOLVENT ATOMS            : 211                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.014                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 2.700                           \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  RESIDUES 61 - 65 (GLN - GLU - GLU - TYR - SER) ADOPT                \\nREMARK   3  SEVERAL CONFORMATIONS IN THE CRYSTAL.  THE COORDINATES              \\nREMARK   3  GIVEN APPROXIMATE ONE OF THESE.                                     \\nREMARK   4                                                                      \\nREMARK   4 5P21 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179747.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 38.93                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.01                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 32 2 1                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+2/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+1/3                                           \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z+1/3                                           \\nREMARK 290       6555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000      108.13333            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       54.06667            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       54.06667            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000      108.13333            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025  0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    ARG A  68   NE  -  CZ  -  NH1 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A  97   NE  -  CZ  -  NH1 ANGL. DEV. =  -3.4 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH1 ANGL. DEV. =   4.1 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500    ARG A 128   NE  -  CZ  -  NH2 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A 135   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500    TYR A 157   CB  -  CG  -  CD2 ANGL. DEV. =  -4.8 DEGREES          \\nREMARK 500    ARG A 164   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ASP A  33       75.86   -157.25                                   \\nREMARK 500    ILE A  36      -64.23   -100.39                                   \\nREMARK 500    GLU A  37      123.75   -171.30                                   \\nREMARK 500    SER A  65     -133.70     53.07                                   \\nREMARK 500    LYS A 117       33.80     71.60                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              MG A 168  MG                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 GNP A 167   O2G                                                    \\nREMARK 620 2 GNP A 167   O2B  95.0                                              \\nREMARK 620 3 HOH A 173   O    87.6  91.7                                        \\nREMARK 620 4 SER A  17   OG  172.9  91.0  88.6                                  \\nREMARK 620 5 THR A  35   OG1  91.3 173.4  90.5  82.8                            \\nREMARK 620 6 HOH A 172   O    97.3  88.0 175.0  86.5  89.3                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE MG A 168                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE GNP A 167                 \\nDBREF  5P21 A    1   166  UNP    P01112   RASH_HUMAN       1    166             \\nSEQRES   1 A  166  MET THR GLU TYR LYS LEU VAL VAL VAL GLY ALA GLY GLY          \\nSEQRES   2 A  166  VAL GLY LYS SER ALA LEU THR ILE GLN LEU ILE GLN ASN          \\nSEQRES   3 A  166  HIS PHE VAL ASP GLU TYR ASP PRO THR ILE GLU ASP SER          \\nSEQRES   4 A  166  TYR ARG LYS GLN VAL VAL ILE ASP GLY GLU THR CYS LEU          \\nSEQRES   5 A  166  LEU ASP ILE LEU ASP THR ALA GLY GLN GLU GLU TYR SER          \\nSEQRES   6 A  166  ALA MET ARG ASP GLN TYR MET ARG THR GLY GLU GLY PHE          \\nSEQRES   7 A  166  LEU CYS VAL PHE ALA ILE ASN ASN THR LYS SER PHE GLU          \\nSEQRES   8 A  166  ASP ILE HIS GLN TYR ARG GLU GLN ILE LYS ARG VAL LYS          \\nSEQRES   9 A  166  ASP SER ASP ASP VAL PRO MET VAL LEU VAL GLY ASN LYS          \\nSEQRES  10 A  166  CYS ASP LEU ALA ALA ARG THR VAL GLU SER ARG GLN ALA          \\nSEQRES  11 A  166  GLN ASP LEU ALA ARG SER TYR GLY ILE PRO TYR ILE GLU          \\nSEQRES  12 A  166  THR SER ALA LYS THR ARG GLN GLY VAL GLU ASP ALA PHE          \\nSEQRES  13 A  166  TYR THR LEU VAL ARG GLU ILE ARG GLN HIS                      \\nHET     MG  A 168       1                                                       \\nHET    GNP  A 167      32                                                       \\nHETNAM      MG MAGNESIUM ION                                                    \\nHETNAM     GNP PHOSPHOAMINOPHOSPHONIC ACID-GUANYLATE ESTER                      \\nFORMUL   2   MG    MG 2+                                                        \\nFORMUL   3  GNP    C10 H17 N6 O13 P3                                            \\nFORMUL   4  HOH   *211(H2 O)                                                    \\nHELIX    1  HA LYS A   16  GLN A   25  1                                  10    \\nHELIX    2  HB SER A   65  THR A   74  1RES 65-67 IN 3/10 CONF            10    \\nHELIX    3  HC THR A   87  VAL A  103  1BREAK AT ASP 92                   17    \\nHELIX    4  HD SER A  127  TYR A  137  1                                  11    \\nHELIX    5  HE VAL A  152  ARG A  164  1                                  13    \\nSHEET    1  S1 6 GLU A  37  ILE A  46  0                                        \\nSHEET    2  S1 6 GLU A  49  THR A  58 -1  O  LEU A  53   N  LYS A  42           \\nSHEET    3  S1 6 THR A   2  VAL A   9  1  N  LEU A   6   O  ASP A  54           \\nSHEET    4  S1 6 GLY A  77  ALA A  83  1  N  LEU A  79   O  VAL A   7           \\nSHEET    5  S1 6 MET A 111  ASN A 116  1  N  VAL A 114   O  CYS A  80           \\nSHEET    6  S1 6 TYR A 141  GLU A 143  1  N  ILE A 142   O  LEU A 113           \\nLINK        MG    MG A 168                 O2G GNP A 167     1555   1555  2.20  \\nLINK        MG    MG A 168                 O2B GNP A 167     1555   1555  2.22  \\nLINK        MG    MG A 168                 O   HOH A 173     1555   1555  2.23  \\nLINK        MG    MG A 168                 OG  SER A  17     1555   1555  2.26  \\nLINK        MG    MG A 168                 OG1 THR A  35     1555   1555  2.26  \\nLINK        MG    MG A 168                 O   HOH A 172     1555   1555  2.30  \\nSITE     1 AC1  5 SER A  17  THR A  35  GNP A 167  HOH A 172                    \\nSITE     2 AC1  5 HOH A 173                                                     \\nSITE     1 AC2 28 GLY A  12  GLY A  13  VAL A  14  GLY A  15                    \\nSITE     2 AC2 28 LYS A  16  SER A  17  ALA A  18  PHE A  28                    \\nSITE     3 AC2 28 VAL A  29  ASP A  30  TYR A  32  PRO A  34                    \\nSITE     4 AC2 28 THR A  35  GLY A  60  ASN A 116  LYS A 117                    \\nSITE     5 AC2 28 ASP A 119  SER A 145  ALA A 146  LYS A 147                    \\nSITE     6 AC2 28  MG A 168  HOH A 170  HOH A 172  HOH A 173                    \\nSITE     7 AC2 28 HOH A 175  HOH A 187  HOH A 281  HOH A 366                    \\nCRYST1   40.300   40.300  162.200  90.00  90.00 120.00 P 32 2 1      6          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.024814  0.014326  0.000000        0.00000                         \\nSCALE2      0.000000  0.028653  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.006165        0.00000                         \\nATOM      1  N   MET A   1      -7.186  32.862  -6.632  1.00 13.92           N  \\nATOM      2  CA  MET A   1      -5.921  32.127  -6.684  1.00 17.10           C  \\nATOM      3  C   MET A   1      -5.845  31.045  -5.621  1.00 16.74           C  \\nATOM      4  O   MET A   1      -6.600  31.083  -4.636  1.00 16.60           O  \\nATOM      5  CB  MET A   1      -4.673  33.058  -6.502  1.00 17.00           C  \\nATOM      6  CG  MET A   1      -4.633  33.804  -5.188  1.00 18.08           C  \\nATOM      7  SD  MET A   1      -3.115  34.766  -4.908  1.00 19.73           S  \\nATOM      8  CE  MET A   1      -3.019  35.851  -6.310  1.00 20.41           C  \\nATOM      9  N   THR A   2      -4.910  30.095  -5.820  1.00 14.70           N  \\nATOM     10  CA  THR A   2      -4.661  29.030  -4.859  1.00 16.92           C  \\nATOM     11  C   THR A   2      -4.230  29.655  -3.547  1.00 14.78           C  \\nATOM     12  O   THR A   2      -3.338  30.501  -3.537  1.00 13.93           O  \\nATOM     13  CB  THR A   2      -3.557  28.107  -5.401  1.00 18.03           C  \\nATOM     14  OG1 THR A   2      -4.059  27.565  -6.629  1.00 20.42           O  \\nATOM     15  CG2 THR A   2      -3.193  26.983  -4.443  1.00 18.51           C  \\nATOM     16  N   GLU A   3      -4.911  29.290  -2.472  1.00 14.80           N  \\nATOM     17  CA  GLU A   3      -4.504  29.730  -1.135  1.00 17.71           C  \\nATOM     18  C   GLU A   3      -4.012  28.553  -0.278  1.00 17.33           C  \\nATOM     19  O   GLU A   3      -4.520  27.427  -0.395  1.00 17.54           O  \\nATOM     20  CB  GLU A   3      -5.655  30.402  -0.432  1.00 20.67           C  \\nATOM     21  CG  GLU A   3      -6.213  31.631  -1.186  1.00 25.56           C  \\nATOM     22  CD  GLU A   3      -7.128  32.563  -0.369  1.00 28.46           C  \\nATOM     23  OE1 GLU A   3      -7.578  32.216   0.728  1.00 27.72           O  \\nATOM     24  OE2 GLU A   3      -7.375  33.680  -0.827  1.00 32.49           O  \\nATOM     25  N   TYR A   4      -2.962  28.744   0.542  1.00 14.50           N  \\nATOM     26  CA  TYR A   4      -2.425  27.654   1.358  1.00 12.29           C  \\nATOM     27  C   TYR A   4      -2.534  28.103   2.807  1.00 12.52           C  \\nATOM     28  O   TYR A   4      -2.016  29.143   3.200  1.00 12.11           O  \\nATOM     29  CB  TYR A   4      -0.975  27.385   1.026  1.00 11.46           C  \\nATOM     30  CG  TYR A   4      -0.726  26.846  -0.387  1.00 12.38           C  \\nATOM     31  CD1 TYR A   4      -0.297  27.698  -1.395  1.00 13.00           C  \\nATOM     32  CD2 TYR A   4      -0.933  25.501  -0.649  1.00 14.09           C  \\nATOM     33  CE1 TYR A   4      -0.070  27.208  -2.674  1.00 13.89           C  \\nATOM     34  CE2 TYR A   4      -0.711  24.993  -1.917  1.00 14.65           C  \\nATOM     35  CZ  TYR A   4      -0.279  25.859  -2.917  1.00 14.84           C  \\nATOM     36  OH  TYR A   4      -0.084  25.357  -4.191  1.00 15.07           O  \\nATOM     37  N   LYS A   5      -3.274  27.305   3.568  1.00 13.07           N  \\nATOM     38  CA  LYS A   5      -3.554  27.569   4.970  1.00 12.96           C  \\nATOM     39  C   LYS A   5      -2.556  26.818   5.852  1.00 12.74           C  \\nATOM     40  O   LYS A   5      -2.645  25.611   6.092  1.00 11.58           O  \\nATOM     41  CB  LYS A   5      -4.972  27.131   5.281  1.00 17.65           C  \\nATOM     42  CG  LYS A   5      -6.053  27.833   4.446  1.00 23.49           C  \\nATOM     43  CD  LYS A   5      -7.186  28.310   5.335  1.00 29.10           C  \\nATOM     44  CE  LYS A   5      -6.655  29.309   6.403  1.00 33.68           C  \\nATOM     45  NZ  LYS A   5      -6.597  28.739   7.755  1.00 36.52           N  \\nATOM     46  N   LEU A   6      -1.543  27.574   6.297  1.00 13.08           N  \\nATOM     47  CA  LEU A   6      -0.418  27.011   7.064  1.00 10.34           C  \\nATOM     48  C   LEU A   6      -0.578  27.382   8.535  1.00  9.68           C  \\nATOM     49  O   LEU A   6      -1.034  28.483   8.863  1.00 11.31           O  \\nATOM     50  CB  LEU A   6       0.910  27.571   6.536  1.00 11.34           C  \\nATOM     51  CG  LEU A   6       1.137  27.565   5.017  1.00 11.62           C  \\nATOM     52  CD1 LEU A   6       2.515  28.119   4.720  1.00 13.60           C  \\nATOM     53  CD2 LEU A   6       0.973  26.171   4.475  1.00 12.19           C  \\nATOM     54  N   VAL A   7      -0.196  26.486   9.432  1.00  9.93           N  \\nATOM     55  CA  VAL A   7      -0.333  26.787  10.852  1.00 10.06           C  \\nATOM     56  C   VAL A   7       1.033  26.513  11.495  1.00 10.37           C  \\nATOM     57  O   VAL A   7       1.597  25.441  11.252  1.00  9.51           O  \\nATOM     58  CB  VAL A   7      -1.474  25.884  11.446  1.00 10.97           C  \\nATOM     59  CG1 VAL A   7      -1.610  26.163  12.943  1.00 11.39           C  \\nATOM     60  CG2 VAL A   7      -2.809  26.161  10.777  1.00 10.14           C  \\nATOM     61  N   VAL A   8       1.576  27.455  12.295  1.00  8.35           N  \\nATOM     62  CA  VAL A   8       2.878  27.262  12.944  1.00  6.92           C  \\nATOM     63  C   VAL A   8       2.609  26.896  14.405  1.00  8.84           C  \\nATOM     64  O   VAL A   8       1.930  27.663  15.092  1.00  9.21           O  \\nATOM     65  CB  VAL A   8       3.684  28.565  12.799  1.00  8.08           C  \\nATOM     66  CG1 VAL A   8       5.111  28.324  13.261  1.00  9.52           C  \\nATOM     67  CG2 VAL A   8       3.729  29.011  11.358  1.00  8.43           C  \\nATOM     68  N   VAL A   9       3.024  25.701  14.881  1.00  8.73           N  \\nATOM     69  CA  VAL A   9       2.779  25.234  16.245  1.00  8.41           C  \\nATOM     70  C   VAL A   9       4.075  24.806  16.919  1.00  8.21           C  \\nATOM     71  O   VAL A   9       5.093  24.551  16.287  1.00 10.97           O  \\nATOM     72  CB  VAL A   9       1.824  24.028  16.286  1.00  9.74           C  \\nATOM     73  CG1 VAL A   9       0.545  24.491  15.680  1.00 11.76           C  \\nATOM     74  CG2 VAL A   9       2.349  22.810  15.535  1.00 10.31           C  \\nATOM     75  N   GLY A  10       4.048  24.707  18.238  1.00  7.34           N  \\nATOM     76  CA  GLY A  10       5.231  24.336  19.009  1.00  7.98           C  \\nATOM     77  C   GLY A  10       5.192  25.053  20.355  1.00  8.31           C  \\nATOM     78  O   GLY A  10       4.417  25.994  20.595  1.00  9.89           O  \\nATOM     79  N   ALA A  11       6.099  24.638  21.226  1.00  9.31           N  \\nATOM     80  CA  ALA A  11       6.121  25.135  22.597  1.00  9.50           C  \\nATOM     81  C   ALA A  11       6.440  26.625  22.711  1.00  9.66           C  \\nATOM     82  O   ALA A  11       6.998  27.250  21.806  1.00  9.46           O  \\nATOM     83  CB  ALA A  11       7.145  24.306  23.378  1.00  9.85           C  \\nATOM     84  N   GLY A  12       6.111  27.227  23.847  1.00 10.36           N  \\nATOM     85  CA  GLY A  12       6.406  28.632  24.054  1.00  9.30           C  \\nATOM     86  C   GLY A  12       7.892  28.950  23.964  1.00 10.23           C  \\nATOM     87  O   GLY A  12       8.785  28.249  24.466  1.00 10.57           O  \\nATOM     88  N   GLY A  13       8.159  30.064  23.266  1.00 11.49           N  \\nATOM     89  CA  GLY A  13       9.511  30.602  23.139  1.00 11.29           C  \\nATOM     90  C   GLY A  13      10.436  29.950  22.108  1.00 12.22           C  \\nATOM     91  O   GLY A  13      11.624  30.321  22.060  1.00 13.62           O  \\nATOM     92  N   VAL A  14       9.961  28.991  21.286  1.00  9.23           N  \\nATOM     93  CA  VAL A  14      10.848  28.327  20.317  1.00 10.38           C  \\nATOM     94  C   VAL A  14      11.169  29.168  19.076  1.00 10.47           C  \\nATOM     95  O   VAL A  14      12.108  28.841  18.353  1.00 12.07           O  \\nATOM     96  CB  VAL A  14      10.246  26.970  19.856  1.00  9.98           C  \\nATOM     97  CG1 VAL A  14      10.187  26.074  21.120  1.00  9.93           C  \\nATOM     98  CG2 VAL A  14       8.866  27.127  19.200  1.00  9.52           C  \\nATOM     99  N   GLY A  15      10.423  30.264  18.834  1.00  9.61           N  \\nATOM    100  CA  GLY A  15      10.666  31.160  17.706  1.00  9.70           C  \\nATOM    101  C   GLY A  15       9.615  31.061  16.626  1.00  8.82           C  \\nATOM    102  O   GLY A  15       9.915  31.403  15.491  1.00  8.99           O  \\nATOM    103  N   LYS A  16       8.395  30.591  16.916  1.00  8.18           N  \\nATOM    104  CA  LYS A  16       7.331  30.521  15.913  1.00  8.44           C  \\nATOM    105  C   LYS A  16       7.060  31.899  15.309  1.00  9.01           C  \\nATOM    106  O   LYS A  16       7.020  32.044  14.086  1.00  8.25           O  \\nATOM    107  CB  LYS A  16       6.014  30.000  16.529  1.00  9.27           C  \\nATOM    108  CG  LYS A  16       6.132  28.585  17.086  1.00  9.11           C  \\nATOM    109  CD  LYS A  16       4.820  28.077  17.726  1.00  9.31           C  \\nATOM    110  CE  LYS A  16       4.351  28.934  18.888  1.00 10.05           C  \\nATOM    111  NZ  LYS A  16       5.221  28.829  20.051  1.00  9.64           N  \\nATOM    112  N   SER A  17       6.913  32.942  16.135  1.00  7.74           N  \\nATOM    113  CA  SER A  17       6.661  34.295  15.649  1.00  8.78           C  \\nATOM    114  C   SER A  17       7.854  34.859  14.919  1.00  9.73           C  \\nATOM    115  O   SER A  17       7.653  35.446  13.868  1.00  9.70           O  \\nATOM    116  CB  SER A  17       6.360  35.264  16.769  1.00  8.22           C  \\nATOM    117  OG  SER A  17       5.083  34.853  17.204  1.00  8.79           O  \\nATOM    118  N   ALA A  18       9.072  34.687  15.444  1.00  8.95           N  \\nATOM    119  CA  ALA A  18      10.253  35.199  14.769  1.00  8.95           C  \\nATOM    120  C   ALA A  18      10.438  34.576  13.374  1.00  8.54           C  \\nATOM    121  O   ALA A  18      10.832  35.254  12.430  1.00  8.75           O  \\nATOM    122  CB  ALA A  18      11.521  34.931  15.640  1.00 10.11           C  \\nATOM    123  N   LEU A  19      10.136  33.284  13.176  1.00  8.31           N  \\nATOM    124  CA  LEU A  19      10.225  32.667  11.837  1.00  9.30           C  \\nATOM    125  C   LEU A  19       9.204  33.281  10.884  1.00 10.98           C  \\nATOM    126  O   LEU A  19       9.534  33.652   9.751  1.00 10.47           O  \\nATOM    127  CB  LEU A  19       9.938  31.165  11.885  1.00  9.41           C  \\nATOM    128  CG  LEU A  19      11.035  30.275  12.487  1.00 11.80           C  \\nATOM    129  CD1 LEU A  19      10.491  28.866  12.723  1.00 10.43           C  \\nATOM    130  CD2 LEU A  19      12.256  30.288  11.554  1.00 13.42           C  \\nATOM    131  N   THR A  20       7.950  33.419  11.335  1.00 10.72           N  \\nATOM    132  CA  THR A  20       6.888  34.015  10.529  1.00 11.87           C  \\nATOM    133  C   THR A  20       7.173  35.492  10.161  1.00 12.65           C  \\nATOM    134  O   THR A  20       7.012  35.917   9.009  1.00 12.60           O  \\nATOM    135  CB  THR A  20       5.547  33.888  11.312  1.00 13.43           C  \\nATOM    136  OG1 THR A  20       5.299  32.509  11.544  1.00 12.67           O  \\nATOM    137  CG2 THR A  20       4.346  34.410  10.518  1.00 14.64           C  \\nATOM    138  N   ILE A  21       7.615  36.326  11.098  1.00 10.48           N  \\nATOM    139  CA  ILE A  21       7.945  37.704  10.795  1.00 12.96           C  \\nATOM    140  C   ILE A  21       9.197  37.831   9.927  1.00 14.26           C  \\nATOM    141  O   ILE A  21       9.237  38.691   9.042  1.00 15.83           O  \\nATOM    142  CB  ILE A  21       8.090  38.451  12.121  1.00 12.57           C  \\nATOM    143  CG1 ILE A  21       6.691  38.551  12.737  1.00 13.24           C  \\nATOM    144  CG2 ILE A  21       8.631  39.852  11.910  1.00 13.40           C  \\nATOM    145  CD1 ILE A  21       6.636  39.185  14.129  1.00 16.46           C  \\nATOM    146  N   GLN A  22      10.206  36.979  10.092  1.00 13.58           N  \\nATOM    147  CA  GLN A  22      11.330  36.983   9.179  1.00 14.91           C  \\nATOM    148  C   GLN A  22      10.817  36.729   7.736  1.00 17.29           C  \\nATOM    149  O   GLN A  22      11.120  37.485   6.792  1.00 15.91           O  \\nATOM    150  CB  GLN A  22      12.270  35.930   9.736  1.00 15.90           C  \\nATOM    151  CG  GLN A  22      13.686  35.878   9.230  1.00 19.36           C  \\nATOM    152  CD  GLN A  22      14.372  37.217   9.091  1.00 18.08           C  \\nATOM    153  OE1 GLN A  22      14.160  37.984   8.143  1.00 19.23           O  \\nATOM    154  NE2 GLN A  22      15.224  37.492  10.050  1.00 17.40           N  \\nATOM    155  N   LEU A  23       9.944  35.718   7.523  1.00 15.95           N  \\nATOM    156  CA  LEU A  23       9.381  35.475   6.194  1.00 18.85           C  \\nATOM    157  C   LEU A  23       8.539  36.622   5.636  1.00 18.21           C  \\nATOM    158  O   LEU A  23       8.695  36.958   4.469  1.00 19.03           O  \\nATOM    159  CB  LEU A  23       8.498  34.235   6.196  1.00 18.10           C  \\nATOM    160  CG  LEU A  23       7.916  33.797   4.863  1.00 18.64           C  \\nATOM    161  CD1 LEU A  23       8.921  32.994   4.030  1.00 18.03           C  \\nATOM    162  CD2 LEU A  23       6.680  32.991   5.183  1.00 21.34           C  \\nATOM    163  N   ILE A  24       7.634  37.197   6.428  1.00 19.25           N  \\nATOM    164  CA  ILE A  24       6.682  38.198   5.953  1.00 20.23           C  \\nATOM    165  C   ILE A  24       7.316  39.590   5.922  1.00 21.89           C  \\nATOM    166  O   ILE A  24       7.125  40.318   4.943  1.00 22.93           O  \\nATOM    167  CB  ILE A  24       5.399  38.222   6.881  1.00 21.20           C  \\nATOM    168  CG1 ILE A  24       4.664  36.907   6.821  1.00 21.27           C  \\nATOM    169  CG2 ILE A  24       4.423  39.307   6.446  1.00 20.00           C  \\nATOM    170  CD1 ILE A  24       4.128  36.442   5.475  1.00 23.98           C  \\nATOM    171  N   GLN A  25       8.074  39.998   6.953  1.00 21.07           N  \\nATOM    172  CA  GLN A  25       8.520  41.380   7.066  1.00 22.09           C  \\nATOM    173  C   GLN A  25      10.003  41.559   6.962  1.00 20.75           C  \\nATOM    174  O   GLN A  25      10.495  42.677   7.054  1.00 20.26           O  \\nATOM    175  CB  GLN A  25       8.039  41.968   8.398  1.00 23.96           C  \\nATOM    176  CG  GLN A  25       6.548  42.276   8.381  1.00 28.69           C  \\nATOM    177  CD  GLN A  25       5.931  42.574   9.744  1.00 30.92           C  \\nATOM    178  OE1 GLN A  25       6.537  43.143  10.657  1.00 31.89           O  \\nATOM    179  NE2 GLN A  25       4.664  42.169   9.867  1.00 32.35           N  \\nATOM    180  N   ASN A  26      10.764  40.482   6.795  1.00 22.13           N  \\nATOM    181  CA  ASN A  26      12.198  40.642   6.628  1.00 24.67           C  \\nATOM    182  C   ASN A  26      12.936  41.362   7.734  1.00 23.62           C  \\nATOM    183  O   ASN A  26      13.603  42.372   7.521  1.00 25.10           O  \\nATOM    184  CB  ASN A  26      12.522  41.369   5.315  1.00 27.68           C  \\nATOM    185  CG  ASN A  26      12.942  40.351   4.288  1.00 32.80           C  \\nATOM    186  OD1 ASN A  26      12.135  39.699   3.602  1.00 33.77           O  \\nATOM    187  ND2 ASN A  26      14.268  40.203   4.237  1.00 34.26           N  \\nATOM    188  N   HIS A  27      12.764  40.842   8.945  1.00 19.95           N  \\nATOM    189  CA  HIS A  27      13.593  41.266  10.057  1.00 17.99           C  \\nATOM    190  C   HIS A  27      13.441  40.299  11.215  1.00 16.84           C  \\nATOM    191  O   HIS A  27      12.411  39.623  11.344  1.00 14.72           O  \\nATOM    192  CB  HIS A  27      13.241  42.680  10.551  1.00 19.80           C  \\nATOM    193  CG  HIS A  27      11.926  42.908  11.258  1.00 20.77           C  \\nATOM    194  ND1 HIS A  27      11.936  43.314  12.510  1.00 22.17           N  \\nATOM    195  CD2 HIS A  27      10.674  42.768  10.729  1.00 22.32           C  \\nATOM    196  CE1 HIS A  27      10.657  43.430  12.785  1.00 23.86           C  \\nATOM    197  NE2 HIS A  27       9.905  43.110  11.738  1.00 23.97           N  \\nATOM    198  N   PHE A  28      14.516  40.222  12.012  1.00 15.44           N  \\nATOM    199  CA  PHE A  28      14.528  39.413  13.223  1.00 13.89           C  \\nATOM    200  C   PHE A  28      14.026  40.196  14.440  1.00 15.33           C  \\nATOM    201  O   PHE A  28      14.528  41.255  14.786  1.00 16.80           O  \\nATOM    202  CB  PHE A  28      15.938  38.906  13.470  1.00 12.73           C  \\nATOM    203  CG  PHE A  28      16.083  38.150  14.786  1.00 13.35           C  \\nATOM    204  CD1 PHE A  28      15.354  36.990  15.012  1.00 13.80           C  \\nATOM    205  CD2 PHE A  28      16.974  38.615  15.740  1.00 14.54           C  \\nATOM    206  CE1 PHE A  28      15.534  36.310  16.194  1.00 14.45           C  \\nATOM    207  CE2 PHE A  28      17.147  37.925  16.920  1.00 14.61           C  \\nATOM    208  CZ  PHE A  28      16.420  36.768  17.137  1.00 14.65           C  \\nATOM    209  N   VAL A  29      12.970  39.697  15.089  1.00 17.22           N  \\nATOM    210  CA  VAL A  29      12.467  40.313  16.294  1.00 19.11           C  \\nATOM    211  C   VAL A  29      13.127  39.606  17.476  1.00 19.36           C  \\nATOM    212  O   VAL A  29      13.245  38.376  17.605  1.00 18.14           O  \\nATOM    213  CB  VAL A  29      10.894  40.205  16.412  1.00 22.03           C  \\nATOM    214  CG1 VAL A  29      10.334  40.869  15.143  1.00 19.66           C  \\nATOM    215  CG2 VAL A  29      10.377  38.776  16.631  1.00 23.15           C  \\nATOM    216  N   ASP A  30      13.615  40.484  18.324  1.00 20.14           N  \\nATOM    217  CA  ASP A  30      14.229  40.019  19.518  1.00 24.04           C  \\nATOM    218  C   ASP A  30      13.676  40.746  20.721  1.00 25.93           C  \\nATOM    219  O   ASP A  30      14.244  41.709  21.238  1.00 29.26           O  \\nATOM    220  CB  ASP A  30      15.719  40.203  19.443  1.00 27.10           C  \\nATOM    221  CG  ASP A  30      16.389  39.731  20.732  1.00 32.46           C  \\nATOM    222  OD1 ASP A  30      15.878  38.859  21.464  1.00 34.28           O  \\nATOM    223  OD2 ASP A  30      17.458  40.264  21.002  1.00 36.46           O  \\nATOM    224  N   GLU A  31      12.472  40.347  21.076  1.00 26.14           N  \\nATOM    225  CA  GLU A  31      11.922  40.804  22.318  1.00 28.72           C  \\nATOM    226  C   GLU A  31      10.870  39.836  22.749  1.00 26.20           C  \\nATOM    227  O   GLU A  31      10.466  38.931  22.014  1.00 23.14           O  \\nATOM    228  CB  GLU A  31      11.337  42.210  22.204  1.00 33.25           C  \\nATOM    229  CG  GLU A  31      10.350  42.618  21.152  1.00 40.08           C  \\nATOM    230  CD  GLU A  31      10.572  44.107  20.887  1.00 44.71           C  \\nATOM    231  OE1 GLU A  31      10.930  44.444  19.755  1.00 47.50           O  \\nATOM    232  OE2 GLU A  31      10.420  44.928  21.806  1.00 47.89           O  \\nATOM    233  N   TYR A  32      10.550  40.044  24.028  1.00 22.26           N  \\nATOM    234  CA  TYR A  32       9.597  39.196  24.685  1.00 18.83           C  \\nATOM    235  C   TYR A  32       8.204  39.688  24.317  1.00 17.53           C  \\nATOM    236  O   TYR A  32       7.794  40.792  24.676  1.00 18.45           O  \\nATOM    237  CB  TYR A  32       9.909  39.259  26.199  1.00 16.84           C  \\nATOM    238  CG  TYR A  32       9.076  38.309  27.052  1.00 15.64           C  \\nATOM    239  CD1 TYR A  32       9.598  37.096  27.433  1.00 16.17           C  \\nATOM    240  CD2 TYR A  32       7.814  38.686  27.459  1.00 15.54           C  \\nATOM    241  CE1 TYR A  32       8.845  36.251  28.239  1.00 17.86           C  \\nATOM    242  CE2 TYR A  32       7.051  37.861  28.257  1.00 14.40           C  \\nATOM    243  CZ  TYR A  32       7.572  36.635  28.652  1.00 16.47           C  \\nATOM    244  OH  TYR A  32       6.821  35.789  29.485  1.00 13.59           O  \\nATOM    245  N   ASP A  33       7.461  38.841  23.604  1.00 13.43           N  \\nATOM    246  CA  ASP A  33       6.128  39.200  23.141  1.00 14.00           C  \\nATOM    247  C   ASP A  33       5.262  37.955  22.854  1.00 13.56           C  \\nATOM    248  O   ASP A  33       5.024  37.586  21.681  1.00 12.47           O  \\nATOM    249  CB  ASP A  33       6.271  40.072  21.864  1.00 15.92           C  \\nATOM    250  CG  ASP A  33       4.993  40.654  21.269  1.00 18.19           C  \\nATOM    251  OD1 ASP A  33       3.923  40.631  21.879  1.00 21.46           O  \\nATOM    252  OD2 ASP A  33       5.074  41.164  20.167  1.00 20.34           O  \\nATOM    253  N   PRO A  34       4.752  37.283  23.903  1.00 12.04           N  \\nATOM    254  CA  PRO A  34       3.949  36.070  23.788  1.00 11.09           C  \\nATOM    255  C   PRO A  34       2.720  36.248  22.895  1.00 11.76           C  \\nATOM    256  O   PRO A  34       1.953  37.220  23.006  1.00 12.63           O  \\nATOM    257  CB  PRO A  34       3.590  35.711  25.219  1.00 12.50           C  \\nATOM    258  CG  PRO A  34       4.643  36.418  26.033  1.00 12.14           C  \\nATOM    259  CD  PRO A  34       4.817  37.717  25.311  1.00 12.82           C  \\nATOM    260  N   THR A  35       2.554  35.282  21.972  1.00 11.00           N  \\nATOM    261  CA  THR A  35       1.421  35.320  21.033  1.00 11.54           C  \\nATOM    262  C   THR A  35       0.140  34.746  21.639  1.00 13.51           C  \\nATOM    263  O   THR A  35       0.174  33.820  22.450  1.00 12.45           O  \\nATOM    264  CB  THR A  35       1.730  34.518  19.751  1.00 11.34           C  \\nATOM    265  OG1 THR A  35       2.900  35.134  19.233  1.00 11.20           O  \\nATOM    266  CG2 THR A  35       0.706  34.611  18.637  1.00 13.10           C  \\nATOM    267  N   ILE A  36      -0.996  35.332  21.233  1.00 15.48           N  \\nATOM    268  CA  ILE A  36      -2.308  34.792  21.567  1.00 19.42           C  \\nATOM    269  C   ILE A  36      -2.663  34.079  20.274  1.00 17.36           C  \\nATOM    270  O   ILE A  36      -2.749  32.863  20.256  1.00 18.29           O  \\nATOM    271  CB  ILE A  36      -3.353  35.914  21.879  1.00 21.30           C  \\nATOM    272  CG1 ILE A  36      -2.930  36.823  23.072  1.00 24.32           C  \\nATOM    273  CG2 ILE A  36      -4.670  35.207  22.136  1.00 21.54           C  \\nATOM    274  CD1 ILE A  36      -2.494  36.167  24.413  1.00 24.70           C  \\nATOM    275  N   GLU A  37      -2.818  34.822  19.181  1.00 19.35           N  \\nATOM    276  CA  GLU A  37      -3.059  34.221  17.889  1.00 21.47           C  \\nATOM    277  C   GLU A  37      -2.957  35.362  16.897  1.00 23.77           C  \\nATOM    278  O   GLU A  37      -3.636  36.393  17.031  1.00 25.49           O  \\nATOM    279  CB  GLU A  37      -4.440  33.611  17.880  1.00 25.97           C  \\nATOM    280  CG  GLU A  37      -4.766  32.776  16.667  1.00 33.81           C  \\nATOM    281  CD  GLU A  37      -6.160  32.154  16.777  1.00 38.37           C  \\nATOM    282  OE1 GLU A  37      -7.095  32.664  16.149  1.00 37.86           O  \\nATOM    283  OE2 GLU A  37      -6.298  31.164  17.515  1.00 42.24           O  \\nATOM    284  N   ASP A  38      -2.070  35.212  15.911  1.00 19.96           N  \\nATOM    285  CA  ASP A  38      -1.899  36.178  14.839  1.00 17.94           C  \\nATOM    286  C   ASP A  38      -2.024  35.526  13.473  1.00 18.21           C  \\nATOM    287  O   ASP A  38      -1.582  34.393  13.294  1.00 16.96           O  \\nATOM    288  CB  ASP A  38      -0.540  36.833  14.965  1.00 18.83           C  \\nATOM    289  CG  ASP A  38      -0.507  37.817  16.133  1.00 20.08           C  \\nATOM    290  OD1 ASP A  38      -1.390  38.660  16.208  1.00 25.47           O  \\nATOM    291  OD2 ASP A  38       0.375  37.781  16.967  1.00 20.08           O  \\nATOM    292  N   SER A  39      -2.698  36.170  12.507  1.00 17.82           N  \\nATOM    293  CA  SER A  39      -2.691  35.658  11.133  1.00 18.16           C  \\nATOM    294  C   SER A  39      -2.012  36.627  10.171  1.00 16.95           C  \\nATOM    295  O   SER A  39      -2.053  37.853  10.333  1.00 15.80           O  \\nATOM    296  CB  SER A  39      -4.094  35.402  10.637  1.00 23.01           C  \\nATOM    297  OG  SER A  39      -4.873  36.571  10.754  1.00 27.13           O  \\nATOM    298  N   TYR A  40      -1.336  36.080   9.168  1.00 15.44           N  \\nATOM    299  CA  TYR A  40      -0.607  36.889   8.196  1.00 16.25           C  \\nATOM    300  C   TYR A  40      -1.018  36.383   6.830  1.00 17.06           C  \\nATOM    301  O   TYR A  40      -1.160  35.172   6.638  1.00 18.29           O  \\nATOM    302  CB  TYR A  40       0.909  36.724   8.370  1.00 16.59           C  \\nATOM    303  CG  TYR A  40       1.439  37.242   9.707  1.00 16.10           C  \\nATOM    304  CD1 TYR A  40       1.473  36.412  10.815  1.00 16.77           C  \\nATOM    305  CD2 TYR A  40       1.883  38.549   9.799  1.00 18.02           C  \\nATOM    306  CE1 TYR A  40       1.950  36.893  12.021  1.00 16.86           C  \\nATOM    307  CE2 TYR A  40       2.368  39.032  11.001  1.00 18.70           C  \\nATOM    308  CZ  TYR A  40       2.396  38.199  12.104  1.00 17.62           C  \\nATOM    309  OH  TYR A  40       2.853  38.695  13.304  1.00 18.37           O  \\nATOM    310  N   ARG A  41      -1.270  37.303   5.900  1.00 15.48           N  \\nATOM    311  CA  ARG A  41      -1.660  36.918   4.558  1.00 15.42           C  \\nATOM    312  C   ARG A  41      -0.730  37.606   3.586  1.00 16.76           C  \\nATOM    313  O   ARG A  41      -0.536  38.829   3.633  1.00 17.01           O  \\nATOM    314  CB  ARG A  41      -3.087  37.342   4.277  1.00 17.64           C  \\nATOM    315  CG  ARG A  41      -4.150  36.507   4.947  1.00 18.09           C  \\nATOM    316  CD  ARG A  41      -5.528  37.181   4.849  1.00 19.64           C  \\nATOM    317  NE  ARG A  41      -6.066  37.318   3.489  1.00 20.11           N  \\nATOM    318  CZ  ARG A  41      -6.660  36.329   2.803  1.00 22.17           C  \\nATOM    319  NH1 ARG A  41      -7.219  36.571   1.617  1.00 22.48           N  \\nATOM    320  NH2 ARG A  41      -6.711  35.090   3.274  1.00 23.55           N  \\nATOM    321  N   LYS A  42      -0.135  36.827   2.679  1.00 16.52           N  \\nATOM    322  CA  LYS A  42       0.835  37.405   1.758  1.00 16.59           C  \\nATOM    323  C   LYS A  42       0.740  36.746   0.387  1.00 16.04           C  \\nATOM    324  O   LYS A  42       0.794  35.528   0.307  1.00 14.74           O  \\nATOM    325  CB  LYS A  42       2.208  37.210   2.401  1.00 18.22           C  \\nATOM    326  CG  LYS A  42       3.388  37.784   1.669  1.00 22.68           C  \\nATOM    327  CD  LYS A  42       3.369  39.274   1.792  1.00 28.09           C  \\nATOM    328  CE  LYS A  42       4.695  39.876   1.317  1.00 31.70           C  \\nATOM    329  NZ  LYS A  42       4.828  39.827  -0.128  1.00 35.70           N  \\nATOM    330  N   GLN A  43       0.561  37.510  -0.697  1.00 16.89           N  \\nATOM    331  CA  GLN A  43       0.579  36.950  -2.047  1.00 19.10           C  \\nATOM    332  C   GLN A  43       2.026  36.729  -2.448  1.00 15.08           C  \\nATOM    333  O   GLN A  43       2.828  37.654  -2.362  1.00 15.72           O  \\nATOM    334  CB  GLN A  43      -0.116  37.908  -3.043  1.00 23.32           C  \\nATOM    335  CG  GLN A  43       0.543  38.332  -4.415  1.00 32.83           C  \\nATOM    336  CD  GLN A  43       1.081  37.332  -5.482  1.00 36.11           C  \\nATOM    337  OE1 GLN A  43       0.457  36.335  -5.865  1.00 37.78           O  \\nATOM    338  NE2 GLN A  43       2.271  37.606  -6.053  1.00 35.45           N  \\nATOM    339  N   VAL A  44       2.394  35.532  -2.922  1.00 12.79           N  \\nATOM    340  CA  VAL A  44       3.771  35.242  -3.311  1.00 11.03           C  \\nATOM    341  C   VAL A  44       3.724  34.425  -4.605  1.00 14.28           C  \\nATOM    342  O   VAL A  44       2.633  34.109  -5.090  1.00 14.90           O  \\nATOM    343  CB  VAL A  44       4.508  34.411  -2.223  1.00 13.21           C  \\nATOM    344  CG1 VAL A  44       4.669  35.282  -0.977  1.00 15.04           C  \\nATOM    345  CG2 VAL A  44       3.761  33.116  -1.889  1.00 12.89           C  \\nATOM    346  N   VAL A  45       4.879  34.127  -5.235  1.00 13.20           N  \\nATOM    347  CA  VAL A  45       4.938  33.277  -6.419  1.00 12.28           C  \\nATOM    348  C   VAL A  45       5.836  32.161  -5.970  1.00 11.08           C  \\nATOM    349  O   VAL A  45       6.988  32.447  -5.636  1.00 13.13           O  \\nATOM    350  CB  VAL A  45       5.587  33.979  -7.641  1.00 13.01           C  \\nATOM    351  CG1 VAL A  45       5.549  33.031  -8.810  1.00 14.51           C  \\nATOM    352  CG2 VAL A  45       4.787  35.173  -8.108  1.00 13.54           C  \\nATOM    353  N   ILE A  46       5.374  30.907  -5.915  1.00  9.89           N  \\nATOM    354  CA  ILE A  46       6.274  29.795  -5.491  1.00 11.03           C  \\nATOM    355  C   ILE A  46       6.369  28.800  -6.662  1.00  8.85           C  \\nATOM    356  O   ILE A  46       5.319  28.296  -7.059  1.00 10.25           O  \\nATOM    357  CB  ILE A  46       5.692  29.115  -4.203  1.00 10.64           C  \\nATOM    358  CG1 ILE A  46       5.652  30.096  -3.042  1.00 11.20           C  \\nATOM    359  CG2 ILE A  46       6.599  27.953  -3.757  1.00 12.93           C  \\nATOM    360  CD1 ILE A  46       4.937  29.536  -1.808  1.00  8.90           C  \\nATOM    361  N   ASP A  47       7.565  28.495  -7.224  1.00  9.41           N  \\nATOM    362  CA  ASP A  47       7.767  27.650  -8.426  1.00  9.70           C  \\nATOM    363  C   ASP A  47       6.838  28.031  -9.572  1.00 11.14           C  \\nATOM    364  O   ASP A  47       6.225  27.211 -10.248  1.00 14.09           O  \\nATOM    365  CB  ASP A  47       7.566  26.165  -8.105  1.00 10.73           C  \\nATOM    366  CG  ASP A  47       8.388  25.783  -6.874  1.00 13.52           C  \\nATOM    367  OD1 ASP A  47       7.805  25.291  -5.898  1.00 12.04           O  \\nATOM    368  OD2 ASP A  47       9.598  26.024  -6.872  1.00 11.31           O  \\nATOM    369  N   GLY A  48       6.724  29.353  -9.735  1.00 10.78           N  \\nATOM    370  CA  GLY A  48       5.930  29.961 -10.785  1.00 13.29           C  \\nATOM    371  C   GLY A  48       4.426  29.989 -10.548  1.00 14.28           C  \\nATOM    372  O   GLY A  48       3.677  30.512 -11.386  1.00 15.19           O  \\nATOM    373  N   GLU A  49       3.944  29.422  -9.439  1.00 13.41           N  \\nATOM    374  CA  GLU A  49       2.520  29.457  -9.143  1.00 13.90           C  \\nATOM    375  C   GLU A  49       2.223  30.700  -8.314  1.00 15.34           C  \\nATOM    376  O   GLU A  49       2.860  30.906  -7.275  1.00 14.56           O  \\nATOM    377  CB  GLU A  49       2.124  28.238  -8.340  1.00 15.50           C  \\nATOM    378  CG  GLU A  49       0.630  28.250  -7.990  1.00 18.10           C  \\nATOM    379  CD  GLU A  49       0.204  27.110  -7.100  1.00 19.13           C  \\nATOM    380  OE1 GLU A  49       0.717  27.004  -5.996  1.00 19.53           O  \\nATOM    381  OE2 GLU A  49      -0.630  26.321  -7.529  1.00 22.12           O  \\nATOM    382  N   THR A  50       1.267  31.536  -8.734  1.00 14.06           N  \\nATOM    383  CA  THR A  50       0.901  32.697  -7.927  1.00 16.15           C  \\nATOM    384  C   THR A  50      -0.001  32.169  -6.819  1.00 13.35           C  \\nATOM    385  O   THR A  50      -0.938  31.409  -7.046  1.00 15.13           O  \\nATOM    386  CB  THR A  50       0.203  33.755  -8.846  1.00 17.88           C  \\nATOM    387  OG1 THR A  50      -0.245  34.813  -8.012  1.00 24.33           O  \\nATOM    388  CG2 THR A  50      -1.028  33.236  -9.518  1.00 19.78           C  \\nATOM    389  N   CYS A  51       0.306  32.442  -5.563  1.00 13.28           N  \\nATOM    390  CA  CYS A  51      -0.566  31.959  -4.508  1.00 15.16           C  \\nATOM    391  C   CYS A  51      -0.579  32.892  -3.303  1.00 14.00           C  \\nATOM    392  O   CYS A  51       0.161  33.874  -3.231  1.00 11.81           O  \\nATOM    393  CB  CYS A  51      -0.132  30.537  -4.079  1.00 14.08           C  \\nATOM    394  SG  CYS A  51       1.597  30.409  -3.575  1.00 16.09           S  \\nATOM    395  N   LEU A  52      -1.521  32.609  -2.405  1.00 13.77           N  \\nATOM    396  CA  LEU A  52      -1.573  33.397  -1.199  1.00 16.01           C  \\nATOM    397  C   LEU A  52      -1.260  32.497  -0.025  1.00 14.53           C  \\nATOM    398  O   LEU A  52      -1.868  31.443   0.113  1.00 16.05           O  \\nATOM    399  CB  LEU A  52      -2.957  33.986  -1.014  1.00 19.71           C  \\nATOM    400  CG  LEU A  52      -3.122  34.885   0.214  1.00 20.84           C  \\nATOM    401  CD1 LEU A  52      -3.070  36.329  -0.220  1.00 21.38           C  \\nATOM    402  CD2 LEU A  52      -4.420  34.526   0.923  1.00 20.90           C  \\nATOM    403  N   LEU A  53      -0.319  32.904   0.800  1.00 12.41           N  \\nATOM    404  CA  LEU A  53       0.019  32.232   2.048  1.00 13.70           C  \\nATOM    405  C   LEU A  53      -0.841  32.852   3.148  1.00 13.85           C  \\nATOM    406  O   LEU A  53      -0.733  34.056   3.413  1.00 12.82           O  \\nATOM    407  CB  LEU A  53       1.485  32.464   2.400  1.00 14.51           C  \\nATOM    408  CG  LEU A  53       2.707  31.589   2.031  1.00 19.29           C  \\nATOM    409  CD1 LEU A  53       2.394  30.553   0.962  1.00 18.56           C  \\nATOM    410  CD2 LEU A  53       3.843  32.557   1.741  1.00 20.00           C  \\nATOM    411  N   ASP A  54      -1.699  32.023   3.751  1.00 11.96           N  \\nATOM    412  CA  ASP A  54      -2.476  32.445   4.884  1.00 15.07           C  \\nATOM    413  C   ASP A  54      -1.882  31.675   6.055  1.00 12.67           C  \\nATOM    414  O   ASP A  54      -2.099  30.477   6.234  1.00 13.99           O  \\nATOM    415  CB  ASP A  54      -3.930  32.088   4.644  1.00 19.33           C  \\nATOM    416  CG  ASP A  54      -4.826  32.609   5.746  1.00 24.24           C  \\nATOM    417  OD1 ASP A  54      -4.412  32.652   6.897  1.00 28.17           O  \\nATOM    418  OD2 ASP A  54      -5.958  32.979   5.450  1.00 27.59           O  \\nATOM    419  N   ILE A  55      -1.127  32.389   6.880  1.00 11.78           N  \\nATOM    420  CA  ILE A  55      -0.346  31.769   7.935  1.00 12.15           C  \\nATOM    421  C   ILE A  55      -0.953  32.099   9.296  1.00 13.10           C  \\nATOM    422  O   ILE A  55      -1.195  33.271   9.587  1.00 13.84           O  \\nATOM    423  CB  ILE A  55       1.109  32.282   7.931  1.00 12.42           C  \\nATOM    424  CG1 ILE A  55       1.859  31.979   6.653  1.00 13.99           C  \\nATOM    425  CG2 ILE A  55       1.859  31.505   9.018  1.00 13.24           C  \\nATOM    426  CD1 ILE A  55       3.176  32.768   6.497  1.00 15.68           C  \\nATOM    427  N   LEU A  56      -1.176  31.083  10.134  1.00 11.64           N  \\nATOM    428  CA  LEU A  56      -1.642  31.341  11.490  1.00 11.98           C  \\nATOM    429  C   LEU A  56      -0.524  31.027  12.431  1.00  9.99           C  \\nATOM    430  O   LEU A  56      -0.017  29.895  12.447  1.00  9.72           O  \\nATOM    431  CB  LEU A  56      -2.836  30.463  11.896  1.00 13.80           C  \\nATOM    432  CG  LEU A  56      -3.470  30.741  13.297  1.00 16.13           C  \\nATOM    433  CD1 LEU A  56      -4.347  31.987  13.260  1.00 15.28           C  \\nATOM    434  CD2 LEU A  56      -4.321  29.580  13.692  1.00 16.84           C  \\nATOM    435  N   ASP A  57      -0.188  32.084  13.182  1.00 10.40           N  \\nATOM    436  CA  ASP A  57       0.815  32.014  14.219  1.00 10.30           C  \\nATOM    437  C   ASP A  57       0.127  31.812  15.571  1.00 11.75           C  \\nATOM    438  O   ASP A  57      -0.652  32.631  16.053  1.00 12.91           O  \\nATOM    439  CB  ASP A  57       1.602  33.296  14.173  1.00 10.66           C  \\nATOM    440  CG  ASP A  57       2.778  33.336  15.132  1.00 11.39           C  \\nATOM    441  OD1 ASP A  57       3.306  32.298  15.535  1.00 11.07           O  \\nATOM    442  OD2 ASP A  57       3.168  34.448  15.455  1.00 13.15           O  \\nATOM    443  N   THR A  58       0.447  30.723  16.244  1.00 12.55           N  \\nATOM    444  CA  THR A  58      -0.324  30.325  17.416  1.00 12.98           C  \\nATOM    445  C   THR A  58       0.368  30.541  18.758  1.00 13.48           C  \\nATOM    446  O   THR A  58       1.568  30.812  18.815  1.00 14.00           O  \\nATOM    447  CB  THR A  58      -0.698  28.851  17.219  1.00  9.54           C  \\nATOM    448  OG1 THR A  58       0.491  28.084  17.369  1.00 11.40           O  \\nATOM    449  CG2 THR A  58      -1.313  28.597  15.848  1.00 12.27           C  \\nATOM    450  N   ALA A  59      -0.371  30.382  19.868  1.00 14.38           N  \\nATOM    451  CA  ALA A  59       0.230  30.510  21.194  1.00 14.79           C  \\nATOM    452  C   ALA A  59       0.928  29.210  21.608  1.00 15.41           C  \\nATOM    453  O   ALA A  59       0.448  28.106  21.324  1.00 13.07           O  \\nATOM    454  CB  ALA A  59      -0.845  30.818  22.238  1.00 15.47           C  \\nATOM    455  N   GLY A  60       2.084  29.341  22.271  1.00 17.71           N  \\nATOM    456  CA  GLY A  60       2.807  28.208  22.814  1.00 24.18           C  \\nATOM    457  C   GLY A  60       2.190  27.751  24.130  1.00 31.21           C  \\nATOM    458  O   GLY A  60       2.538  26.688  24.627  1.00 30.44           O  \\nATOM    459  N   GLN A  61       1.260  28.503  24.726  1.00 39.50           N  \\nATOM    460  CA  GLN A  61       0.671  28.099  25.994  1.00 48.84           C  \\nATOM    461  C   GLN A  61      -0.683  27.395  25.985  1.00 51.52           C  \\nATOM    462  O   GLN A  61      -1.606  27.780  25.276  1.00 52.17           O  \\nATOM    463  CB  GLN A  61       0.605  29.327  26.874  1.00 51.56           C  \\nATOM    464  CG  GLN A  61       0.871  28.861  28.312  1.00 56.74           C  \\nATOM    465  CD  GLN A  61       1.506  29.937  29.177  1.00 58.74           C  \\nATOM    466  OE1 GLN A  61       2.647  29.795  29.616  1.00 60.12           O  \\nATOM    467  NE2 GLN A  61       0.783  31.031  29.429  1.00 60.18           N  \\nATOM    468  N   GLU A  62      -0.804  26.349  26.818  1.00 57.40           N  \\nATOM    469  CA  GLU A  62      -2.008  25.527  26.927  1.00 61.96           C  \\nATOM    470  C   GLU A  62      -3.185  26.196  27.631  1.00 62.20           C  \\nATOM    471  O   GLU A  62      -4.138  25.496  27.975  1.00 62.85           O  \\nATOM    472  CB  GLU A  62      -1.629  24.183  27.632  1.00 65.18           C  \\nATOM    473  CG  GLU A  62      -2.622  22.966  27.761  1.00 68.22           C  \\nATOM    474  CD  GLU A  62      -3.032  22.132  26.528  1.00 70.25           C  \\nATOM    475  OE1 GLU A  62      -2.710  22.494  25.395  1.00 71.45           O  \\nATOM    476  OE2 GLU A  62      -3.691  21.101  26.689  1.00 70.00           O  \\nATOM    477  N   GLU A  63      -3.206  27.520  27.904  1.00 62.78           N  \\nATOM    478  CA  GLU A  63      -4.496  28.139  28.254  1.00 63.62           C  \\nATOM    479  C   GLU A  63      -5.457  27.854  27.085  1.00 63.71           C  \\nATOM    480  O   GLU A  63      -6.688  27.814  27.211  1.00 62.97           O  \\nATOM    481  CB  GLU A  63      -4.372  29.666  28.475  1.00 65.13           C  \\nATOM    482  CG  GLU A  63      -3.456  30.553  27.607  1.00 66.91           C  \\nATOM    483  CD  GLU A  63      -4.012  31.235  26.345  1.00 68.05           C  \\nATOM    484  OE1 GLU A  63      -5.220  31.462  26.220  1.00 68.64           O  \\nATOM    485  OE2 GLU A  63      -3.198  31.568  25.481  1.00 67.62           O  \\nATOM    486  N   TYR A  64      -4.801  27.606  25.934  1.00 63.36           N  \\nATOM    487  CA  TYR A  64      -5.420  27.092  24.743  1.00 63.20           C  \\nATOM    488  C   TYR A  64      -5.905  25.653  24.875  1.00 61.76           C  \\nATOM    489  O   TYR A  64      -7.120  25.486  24.858  1.00 63.04           O  \\nATOM    490  CB  TYR A  64      -4.437  27.208  23.613  1.00 64.83           C  \\nATOM    491  CG  TYR A  64      -4.736  28.459  22.818  1.00 66.98           C  \\nATOM    492  CD1 TYR A  64      -4.300  29.686  23.267  1.00 67.55           C  \\nATOM    493  CD2 TYR A  64      -5.459  28.355  21.646  1.00 67.97           C  \\nATOM    494  CE1 TYR A  64      -4.587  30.825  22.536  1.00 69.49           C  \\nATOM    495  CE2 TYR A  64      -5.752  29.488  20.911  1.00 69.29           C  \\nATOM    496  CZ  TYR A  64      -5.313  30.718  21.359  1.00 70.05           C  \\nATOM    497  OH  TYR A  64      -5.589  31.852  20.614  1.00 70.67           O  \\nATOM    498  N   SER A  65      -5.095  24.590  24.997  1.00 58.67           N  \\nATOM    499  CA  SER A  65      -5.617  23.230  25.127  1.00 56.93           C  \\nATOM    500  C   SER A  65      -6.614  22.757  24.063  1.00 55.33           C  \\nATOM    501  O   SER A  65      -6.321  22.936  22.876  1.00 54.74           O  \\nATOM    502  CB  SER A  65      -6.190  23.122  26.552  1.00 57.99           C  \\nATOM    503  OG  SER A  65      -7.251  24.011  26.900  1.00 59.80           O  \\nATOM    504  N   ALA A  66      -7.762  22.119  24.359  1.00 52.33           N  \\nATOM    505  CA  ALA A  66      -8.766  21.824  23.335  1.00 48.80           C  \\nATOM    506  C   ALA A  66      -9.138  23.050  22.492  1.00 45.71           C  \\nATOM    507  O   ALA A  66      -9.751  22.862  21.449  1.00 44.33           O  \\nATOM    508  CB  ALA A  66     -10.043  21.300  23.971  1.00 49.14           C  \\nATOM    509  N   MET A  67      -8.829  24.310  22.875  1.00 42.85           N  \\nATOM    510  CA  MET A  67      -8.984  25.465  21.967  1.00 40.01           C  \\nATOM    511  C   MET A  67      -8.093  25.362  20.719  1.00 36.79           C  \\nATOM    512  O   MET A  67      -8.221  26.184  19.805  1.00 36.08           O  \\nATOM    513  CB  MET A  67      -8.594  26.804  22.581  1.00 42.50           C  \\nATOM    514  CG  MET A  67      -9.359  27.330  23.772  1.00 45.93           C  \\nATOM    515  SD  MET A  67     -11.140  27.396  23.483  1.00 48.74           S  \\nATOM    516  CE  MET A  67     -11.526  25.880  24.321  1.00 47.64           C  \\nATOM    517  N   ARG A  68      -7.128  24.418  20.665  1.00 31.59           N  \\nATOM    518  CA  ARG A  68      -6.322  24.217  19.467  1.00 26.34           C  \\nATOM    519  C   ARG A  68      -7.140  23.473  18.424  1.00 23.68           C  \\nATOM    520  O   ARG A  68      -6.846  23.617  17.231  1.00 21.93           O  \\nATOM    521  CB  ARG A  68      -5.057  23.394  19.745  1.00 25.86           C  \\nATOM    522  CG  ARG A  68      -4.161  23.898  20.869  1.00 29.36           C  \\nATOM    523  CD  ARG A  68      -2.881  23.061  20.998  1.00 32.88           C  \\nATOM    524  NE  ARG A  68      -2.221  22.990  22.315  1.00 36.64           N  \\nATOM    525  CZ  ARG A  68      -1.700  24.031  23.001  1.00 39.66           C  \\nATOM    526  NH1 ARG A  68      -1.903  25.315  22.664  1.00 42.06           N  \\nATOM    527  NH2 ARG A  68      -0.933  23.798  24.071  1.00 39.61           N  \\nATOM    528  N   ASP A  69      -8.188  22.721  18.793  1.00 19.14           N  \\nATOM    529  CA  ASP A  69      -8.880  21.905  17.822  1.00 20.43           C  \\nATOM    530  C   ASP A  69      -9.437  22.623  16.617  1.00 22.16           C  \\nATOM    531  O   ASP A  69      -9.453  22.109  15.488  1.00 19.58           O  \\nATOM    532  CB  ASP A  69     -10.025  21.148  18.478  1.00 20.58           C  \\nATOM    533  CG  ASP A  69      -9.598  19.935  19.303  1.00 22.81           C  \\nATOM    534  OD1 ASP A  69      -8.409  19.708  19.538  1.00 24.59           O  \\nATOM    535  OD2 ASP A  69     -10.475  19.176  19.715  1.00 25.28           O  \\nATOM    536  N   GLN A  70      -9.834  23.874  16.840  1.00 23.26           N  \\nATOM    537  CA  GLN A  70     -10.451  24.609  15.759  1.00 23.53           C  \\nATOM    538  C   GLN A  70      -9.437  24.925  14.667  1.00 21.20           C  \\nATOM    539  O   GLN A  70      -9.653  24.591  13.493  1.00 20.90           O  \\nATOM    540  CB  GLN A  70     -11.071  25.881  16.341  1.00 27.73           C  \\nATOM    541  CG  GLN A  70     -12.498  26.019  15.832  1.00 34.11           C  \\nATOM    542  CD  GLN A  70     -13.103  27.407  16.008  1.00 37.87           C  \\nATOM    543  OE1 GLN A  70     -12.532  28.284  16.672  1.00 39.77           O  \\nATOM    544  NE2 GLN A  70     -14.283  27.620  15.406  1.00 37.89           N  \\nATOM    545  N   TYR A  71      -8.292  25.515  15.050  1.00 19.09           N  \\nATOM    546  CA  TYR A  71      -7.329  25.873  14.030  1.00 17.69           C  \\nATOM    547  C   TYR A  71      -6.648  24.658  13.416  1.00 17.24           C  \\nATOM    548  O   TYR A  71      -6.247  24.675  12.245  1.00 17.25           O  \\nATOM    549  CB  TYR A  71      -6.302  26.868  14.625  1.00 17.87           C  \\nATOM    550  CG  TYR A  71      -5.311  26.480  15.725  1.00 18.90           C  \\nATOM    551  CD1 TYR A  71      -4.355  25.510  15.492  1.00 19.74           C  \\nATOM    552  CD2 TYR A  71      -5.328  27.162  16.940  1.00 19.21           C  \\nATOM    553  CE1 TYR A  71      -3.402  25.212  16.450  1.00 20.38           C  \\nATOM    554  CE2 TYR A  71      -4.375  26.868  17.901  1.00 21.93           C  \\nATOM    555  CZ  TYR A  71      -3.422  25.897  17.645  1.00 21.91           C  \\nATOM    556  OH  TYR A  71      -2.463  25.590  18.588  1.00 26.44           O  \\nATOM    557  N   MET A  72      -6.557  23.559  14.166  1.00 14.81           N  \\nATOM    558  CA  MET A  72      -5.958  22.393  13.563  1.00 18.59           C  \\nATOM    559  C   MET A  72      -6.849  21.778  12.472  1.00 19.38           C  \\nATOM    560  O   MET A  72      -6.385  21.198  11.492  1.00 17.67           O  \\nATOM    561  CB  MET A  72      -5.658  21.386  14.669  1.00 17.39           C  \\nATOM    562  CG  MET A  72      -4.479  21.796  15.514  1.00 16.35           C  \\nATOM    563  SD  MET A  72      -4.146  20.600  16.820  1.00 17.04           S  \\nATOM    564  CE  MET A  72      -2.635  21.259  17.460  1.00 17.72           C  \\nATOM    565  N   ARG A  73      -8.174  21.931  12.626  1.00 20.31           N  \\nATOM    566  CA  ARG A  73      -9.120  21.350  11.701  1.00 22.24           C  \\nATOM    567  C   ARG A  73      -9.087  22.084  10.370  1.00 20.94           C  \\nATOM    568  O   ARG A  73      -9.183  21.450   9.326  1.00 20.81           O  \\nATOM    569  CB  ARG A  73     -10.499  21.414  12.369  1.00 25.29           C  \\nATOM    570  CG  ARG A  73     -11.553  20.499  11.767  1.00 30.04           C  \\nATOM    571  CD  ARG A  73     -12.775  20.429  12.687  1.00 31.86           C  \\nATOM    572  NE  ARG A  73     -12.483  19.702  13.917  1.00 34.70           N  \\nATOM    573  CZ  ARG A  73     -12.894  20.122  15.130  1.00 36.49           C  \\nATOM    574  NH1 ARG A  73     -12.579  19.402  16.196  1.00 36.93           N  \\nATOM    575  NH2 ARG A  73     -13.609  21.239  15.311  1.00 38.17           N  \\nATOM    576  N   THR A  74      -8.868  23.401  10.413  1.00 21.33           N  \\nATOM    577  CA  THR A  74      -8.813  24.240   9.219  1.00 24.99           C  \\nATOM    578  C   THR A  74      -7.436  24.351   8.544  1.00 23.10           C  \\nATOM    579  O   THR A  74      -7.367  24.738   7.371  1.00 22.59           O  \\nATOM    580  CB  THR A  74      -9.289  25.676   9.538  1.00 25.73           C  \\nATOM    581  OG1 THR A  74      -8.351  26.302  10.409  1.00 31.08           O  \\nATOM    582  CG2 THR A  74     -10.657  25.644  10.201  1.00 27.58           C  \\nATOM    583  N   GLY A  75      -6.345  24.084   9.289  1.00 19.58           N  \\nATOM    584  CA  GLY A  75      -5.013  24.114   8.730  1.00 15.79           C  \\nATOM    585  C   GLY A  75      -4.823  23.023   7.713  1.00 13.66           C  \\nATOM    586  O   GLY A  75      -5.245  21.880   7.884  1.00 14.69           O  \\nATOM    587  N   GLU A  76      -4.228  23.360   6.573  1.00 13.14           N  \\nATOM    588  CA  GLU A  76      -3.914  22.363   5.546  1.00 12.99           C  \\nATOM    589  C   GLU A  76      -2.514  21.740   5.753  1.00 12.22           C  \\nATOM    590  O   GLU A  76      -2.293  20.576   5.426  1.00 13.59           O  \\nATOM    591  CB  GLU A  76      -3.999  23.014   4.145  1.00 13.07           C  \\nATOM    592  CG  GLU A  76      -5.387  23.377   3.676  1.00 14.35           C  \\nATOM    593  CD  GLU A  76      -5.261  24.069   2.323  1.00 20.40           C  \\nATOM    594  OE1 GLU A  76      -5.461  23.403   1.288  1.00 24.32           O  \\nATOM    595  OE2 GLU A  76      -4.913  25.250   2.284  1.00 18.82           O  \\nATOM    596  N   GLY A  77      -1.553  22.503   6.301  1.00 10.73           N  \\nATOM    597  CA  GLY A  77      -0.216  22.018   6.570  1.00  8.07           C  \\nATOM    598  C   GLY A  77       0.302  22.689   7.819  1.00  8.63           C  \\nATOM    599  O   GLY A  77      -0.066  23.823   8.127  1.00  8.55           O  \\nATOM    600  N   PHE A  78       1.248  22.013   8.479  1.00  9.15           N  \\nATOM    601  CA  PHE A  78       1.754  22.468   9.773  1.00  8.87           C  \\nATOM    602  C   PHE A  78       3.268  22.520   9.854  1.00  9.51           C  \\nATOM    603  O   PHE A  78       3.930  21.542   9.486  1.00  9.15           O  \\nATOM    604  CB  PHE A  78       1.247  21.532  10.904  1.00  9.44           C  \\nATOM    605  CG  PHE A  78      -0.267  21.472  11.000  1.00  9.66           C  \\nATOM    606  CD1 PHE A  78      -0.906  22.310  11.901  1.00 10.71           C  \\nATOM    607  CD2 PHE A  78      -0.994  20.645  10.152  1.00 10.20           C  \\nATOM    608  CE1 PHE A  78      -2.295  22.338  11.940  1.00 10.61           C  \\nATOM    609  CE2 PHE A  78      -2.378  20.678  10.192  1.00 12.55           C  \\nATOM    610  CZ  PHE A  78      -3.022  21.529  11.081  1.00 12.80           C  \\nATOM    611  N   LEU A  79       3.813  23.636  10.354  1.00  8.72           N  \\nATOM    612  CA  LEU A  79       5.230  23.626  10.699  1.00  9.93           C  \\nATOM    613  C   LEU A  79       5.279  23.281  12.181  1.00  9.09           C  \\nATOM    614  O   LEU A  79       4.701  24.008  12.988  1.00 11.58           O  \\nATOM    615  CB  LEU A  79       5.921  24.970  10.567  1.00 12.80           C  \\nATOM    616  CG  LEU A  79       6.155  25.659   9.249  1.00 16.39           C  \\nATOM    617  CD1 LEU A  79       7.007  26.911   9.544  1.00 17.02           C  \\nATOM    618  CD2 LEU A  79       6.904  24.756   8.269  1.00 16.98           C  \\nATOM    619  N   CYS A  80       5.950  22.219  12.579  1.00  9.14           N  \\nATOM    620  CA  CYS A  80       6.074  21.821  13.981  1.00 11.81           C  \\nATOM    621  C   CYS A  80       7.448  22.237  14.434  1.00 11.53           C  \\nATOM    622  O   CYS A  80       8.435  21.640  14.000  1.00 12.21           O  \\nATOM    623  CB  CYS A  80       5.903  20.318  14.106  1.00 13.73           C  \\nATOM    624  SG  CYS A  80       4.158  19.900  13.837  1.00 18.26           S  \\nATOM    625  N   VAL A  81       7.486  23.282  15.269  1.00  7.57           N  \\nATOM    626  CA  VAL A  81       8.730  23.956  15.604  1.00  8.45           C  \\nATOM    627  C   VAL A  81       9.248  23.620  16.996  1.00  8.90           C  \\nATOM    628  O   VAL A  81       8.451  23.680  17.939  1.00  8.49           O  \\nATOM    629  CB  VAL A  81       8.544  25.516  15.507  1.00  7.40           C  \\nATOM    630  CG1 VAL A  81       9.917  26.196  15.588  1.00  9.42           C  \\nATOM    631  CG2 VAL A  81       8.004  25.954  14.167  1.00  8.59           C  \\nATOM    632  N   PHE A  82      10.546  23.237  17.119  1.00  9.49           N  \\nATOM    633  CA  PHE A  82      11.195  23.202  18.442  1.00  9.09           C  \\nATOM    634  C   PHE A  82      12.511  23.994  18.350  1.00  9.83           C  \\nATOM    635  O   PHE A  82      12.914  24.440  17.266  1.00  9.02           O  \\nATOM    636  CB  PHE A  82      11.482  21.755  18.902  1.00  9.76           C  \\nATOM    637  CG  PHE A  82      12.536  21.062  18.071  1.00 10.09           C  \\nATOM    638  CD1 PHE A  82      13.867  21.099  18.486  1.00 10.70           C  \\nATOM    639  CD2 PHE A  82      12.192  20.412  16.889  1.00 10.93           C  \\nATOM    640  CE1 PHE A  82      14.838  20.483  17.699  1.00 11.12           C  \\nATOM    641  CE2 PHE A  82      13.181  19.807  16.125  1.00 10.73           C  \\nATOM    642  CZ  PHE A  82      14.505  19.839  16.523  1.00 10.35           C  \\nATOM    643  N   ALA A  83      13.128  24.269  19.516  1.00  9.75           N  \\nATOM    644  CA  ALA A  83      14.407  24.963  19.569  1.00 10.32           C  \\nATOM    645  C   ALA A  83      15.509  23.960  19.916  1.00  9.48           C  \\nATOM    646  O   ALA A  83      15.362  23.142  20.839  1.00  9.61           O  \\nATOM    647  CB  ALA A  83      14.354  26.083  20.620  1.00  9.75           C  \\nATOM    648  N   ILE A  84      16.624  24.042  19.160  1.00 11.41           N  \\nATOM    649  CA  ILE A  84      17.714  23.070  19.299  1.00 13.15           C  \\nATOM    650  C   ILE A  84      18.446  23.223  20.623  1.00 14.57           C  \\nATOM    651  O   ILE A  84      19.234  22.347  20.971  1.00 13.94           O  \\nATOM    652  CB  ILE A  84      18.775  23.151  18.160  1.00 13.29           C  \\nATOM    653  CG1 ILE A  84      19.403  24.509  17.878  1.00 14.90           C  \\nATOM    654  CG2 ILE A  84      18.018  22.701  16.933  1.00 13.09           C  \\nATOM    655  CD1 ILE A  84      20.578  24.957  18.763  1.00 17.59           C  \\nATOM    656  N   ASN A  85      18.200  24.290  21.390  1.00 13.61           N  \\nATOM    657  CA  ASN A  85      18.794  24.394  22.720  1.00 15.52           C  \\nATOM    658  C   ASN A  85      17.758  24.156  23.828  1.00 15.49           C  \\nATOM    659  O   ASN A  85      17.988  24.519  24.999  1.00 16.13           O  \\nATOM    660  CB  ASN A  85      19.446  25.788  22.885  1.00 15.16           C  \\nATOM    661  CG  ASN A  85      18.464  26.946  22.904  1.00 16.42           C  \\nATOM    662  OD1 ASN A  85      17.355  26.878  22.362  1.00 16.01           O  \\nATOM    663  ND2 ASN A  85      18.808  28.030  23.587  1.00 16.83           N  \\nATOM    664  N   ASN A  86      16.616  23.515  23.509  1.00 14.72           N  \\nATOM    665  CA  ASN A  86      15.553  23.303  24.483  1.00 13.01           C  \\nATOM    666  C   ASN A  86      14.987  21.901  24.309  1.00 13.17           C  \\nATOM    667  O   ASN A  86      14.080  21.668  23.520  1.00 13.04           O  \\nATOM    668  CB  ASN A  86      14.478  24.367  24.256  1.00 16.51           C  \\nATOM    669  CG  ASN A  86      13.267  24.415  25.187  1.00 20.74           C  \\nATOM    670  OD1 ASN A  86      12.370  25.241  25.018  1.00 25.41           O  \\nATOM    671  ND2 ASN A  86      13.123  23.596  26.206  1.00 19.96           N  \\nATOM    672  N   THR A  87      15.488  20.950  25.103  1.00 13.80           N  \\nATOM    673  CA  THR A  87      15.095  19.553  25.026  1.00 15.40           C  \\nATOM    674  C   THR A  87      13.589  19.316  25.266  1.00 14.40           C  \\nATOM    675  O   THR A  87      12.953  18.535  24.546  1.00 14.11           O  \\nATOM    676  CB  THR A  87      15.983  18.791  26.046  1.00 16.19           C  \\nATOM    677  OG1 THR A  87      17.325  18.908  25.571  1.00 18.22           O  \\nATOM    678  CG2 THR A  87      15.626  17.327  26.180  1.00 17.82           C  \\nATOM    679  N   LYS A  88      12.983  19.958  26.272  1.00 14.42           N  \\nATOM    680  CA  LYS A  88      11.552  19.842  26.535  1.00 15.84           C  \\nATOM    681  C   LYS A  88      10.726  20.232  25.278  1.00 15.06           C  \\nATOM    682  O   LYS A  88       9.722  19.582  25.003  1.00 14.61           O  \\nATOM    683  CB  LYS A  88      11.207  20.766  27.717  1.00 19.53           C  \\nATOM    684  CG  LYS A  88       9.937  20.401  28.457  1.00 27.71           C  \\nATOM    685  CD  LYS A  88       9.585  21.452  29.529  1.00 33.26           C  \\nATOM    686  CE  LYS A  88       9.061  22.762  28.877  1.00 36.79           C  \\nATOM    687  NZ  LYS A  88       8.642  23.763  29.852  1.00 37.29           N  \\nATOM    688  N   SER A  89      11.119  21.259  24.475  1.00 12.36           N  \\nATOM    689  CA  SER A  89      10.325  21.666  23.309  1.00 11.31           C  \\nATOM    690  C   SER A  89      10.371  20.572  22.268  1.00 12.51           C  \\nATOM    691  O   SER A  89       9.397  20.388  21.547  1.00 12.88           O  \\nATOM    692  CB  SER A  89      10.844  22.951  22.674  1.00 11.64           C  \\nATOM    693  OG  SER A  89      12.160  22.900  22.124  1.00 11.11           O  \\nATOM    694  N   PHE A  90      11.504  19.842  22.204  1.00 11.27           N  \\nATOM    695  CA  PHE A  90      11.680  18.751  21.262  1.00 12.91           C  \\nATOM    696  C   PHE A  90      10.760  17.601  21.667  1.00 14.37           C  \\nATOM    697  O   PHE A  90      10.034  17.057  20.839  1.00 13.24           O  \\nATOM    698  CB  PHE A  90      13.161  18.312  21.260  1.00 12.34           C  \\nATOM    699  CG  PHE A  90      13.411  17.120  20.351  1.00 14.41           C  \\nATOM    700  CD1 PHE A  90      13.890  15.948  20.903  1.00 15.10           C  \\nATOM    701  CD2 PHE A  90      13.136  17.206  18.990  1.00 14.81           C  \\nATOM    702  CE1 PHE A  90      14.093  14.857  20.088  1.00 16.20           C  \\nATOM    703  CE2 PHE A  90      13.340  16.110  18.179  1.00 14.81           C  \\nATOM    704  CZ  PHE A  90      13.818  14.935  18.731  1.00 15.39           C  \\nATOM    705  N   GLU A  91      10.725  17.231  22.949  1.00 14.40           N  \\nATOM    706  CA  GLU A  91       9.863  16.146  23.407  1.00 15.78           C  \\nATOM    707  C   GLU A  91       8.374  16.466  23.301  1.00 16.18           C  \\nATOM    708  O   GLU A  91       7.533  15.582  23.114  1.00 17.50           O  \\nATOM    709  CB  GLU A  91      10.226  15.846  24.835  1.00 19.84           C  \\nATOM    710  CG  GLU A  91      11.654  15.386  25.074  1.00 24.49           C  \\nATOM    711  CD  GLU A  91      12.090  15.310  26.537  1.00 27.97           C  \\nATOM    712  OE1 GLU A  91      13.199  14.842  26.766  1.00 30.72           O  \\nATOM    713  OE2 GLU A  91      11.371  15.730  27.449  1.00 30.38           O  \\nATOM    714  N   ASP A  92       8.019  17.747  23.433  1.00 16.19           N  \\nATOM    715  CA  ASP A  92       6.650  18.216  23.265  1.00 15.23           C  \\nATOM    716  C   ASP A  92       6.141  18.053  21.839  1.00 14.29           C  \\nATOM    717  O   ASP A  92       4.927  18.074  21.633  1.00 14.78           O  \\nATOM    718  CB  ASP A  92       6.536  19.708  23.646  1.00 17.85           C  \\nATOM    719  CG  ASP A  92       6.706  20.080  25.127  1.00 21.39           C  \\nATOM    720  OD1 ASP A  92       6.648  19.209  25.997  1.00 20.06           O  \\nATOM    721  OD2 ASP A  92       6.887  21.272  25.401  1.00 24.39           O  \\nATOM    722  N   ILE A  93       7.013  17.910  20.826  1.00 11.76           N  \\nATOM    723  CA  ILE A  93       6.559  17.749  19.452  1.00 14.77           C  \\nATOM    724  C   ILE A  93       5.574  16.583  19.354  1.00 14.03           C  \\nATOM    725  O   ILE A  93       4.576  16.695  18.644  1.00 14.93           O  \\nATOM    726  CB  ILE A  93       7.790  17.499  18.485  1.00 14.96           C  \\nATOM    727  CG1 ILE A  93       8.506  18.830  18.293  1.00 15.41           C  \\nATOM    728  CG2 ILE A  93       7.369  16.936  17.091  1.00 14.61           C  \\nATOM    729  CD1 ILE A  93       7.654  19.940  17.637  1.00 20.05           C  \\nATOM    730  N   HIS A  94       5.806  15.471  20.049  1.00 15.23           N  \\nATOM    731  CA  HIS A  94       4.940  14.302  19.956  1.00 18.20           C  \\nATOM    732  C   HIS A  94       3.494  14.635  20.287  1.00 16.36           C  \\nATOM    733  O   HIS A  94       2.576  14.183  19.598  1.00 17.20           O  \\nATOM    734  CB  HIS A  94       5.436  13.194  20.900  1.00 23.57           C  \\nATOM    735  CG  HIS A  94       4.685  11.874  20.678  1.00 29.09           C  \\nATOM    736  ND1 HIS A  94       4.923  11.118  19.602  1.00 31.26           N  \\nATOM    737  CD2 HIS A  94       3.705  11.361  21.498  1.00 30.93           C  \\nATOM    738  CE1 HIS A  94       4.078  10.110  19.746  1.00 31.74           C  \\nATOM    739  NE2 HIS A  94       3.366  10.260  20.861  1.00 32.20           N  \\nATOM    740  N   GLN A  95       3.271  15.499  21.281  1.00 16.91           N  \\nATOM    741  CA  GLN A  95       1.916  15.878  21.673  1.00 18.66           C  \\nATOM    742  C   GLN A  95       1.226  16.736  20.625  1.00 16.97           C  \\nATOM    743  O   GLN A  95       0.012  16.562  20.443  1.00 15.36           O  \\nATOM    744  CB  GLN A  95       1.928  16.626  23.018  1.00 21.56           C  \\nATOM    745  CG  GLN A  95       2.457  15.766  24.180  1.00 29.81           C  \\nATOM    746  CD  GLN A  95       3.370  16.540  25.150  1.00 35.29           C  \\nATOM    747  OE1 GLN A  95       3.037  17.647  25.593  1.00 37.94           O  \\nATOM    748  NE2 GLN A  95       4.546  16.006  25.521  1.00 37.04           N  \\nATOM    749  N   TYR A  96       1.930  17.653  19.921  1.00 16.06           N  \\nATOM    750  CA  TYR A  96       1.276  18.417  18.826  1.00 16.07           C  \\nATOM    751  C   TYR A  96       0.963  17.504  17.658  1.00 14.44           C  \\nATOM    752  O   TYR A  96      -0.138  17.565  17.118  1.00 15.79           O  \\nATOM    753  CB  TYR A  96       2.139  19.554  18.264  1.00 17.94           C  \\nATOM    754  CG  TYR A  96       2.287  20.677  19.277  1.00 19.19           C  \\nATOM    755  CD1 TYR A  96       1.368  21.699  19.338  1.00 19.33           C  \\nATOM    756  CD2 TYR A  96       3.348  20.631  20.161  1.00 19.61           C  \\nATOM    757  CE1 TYR A  96       1.526  22.678  20.308  1.00 21.72           C  \\nATOM    758  CE2 TYR A  96       3.512  21.595  21.129  1.00 21.72           C  \\nATOM    759  CZ  TYR A  96       2.594  22.613  21.190  1.00 21.26           C  \\nATOM    760  OH  TYR A  96       2.776  23.582  22.153  1.00 24.09           O  \\nATOM    761  N   ARG A  97       1.904  16.635  17.295  1.00 13.30           N  \\nATOM    762  CA  ARG A  97       1.699  15.650  16.247  1.00 15.46           C  \\nATOM    763  C   ARG A  97       0.457  14.794  16.492  1.00 14.26           C  \\nATOM    764  O   ARG A  97      -0.420  14.676  15.641  1.00 14.10           O  \\nATOM    765  CB  ARG A  97       2.939  14.796  16.196  1.00 17.97           C  \\nATOM    766  CG  ARG A  97       2.886  13.541  15.357  1.00 23.35           C  \\nATOM    767  CD  ARG A  97       3.333  13.765  13.943  1.00 23.59           C  \\nATOM    768  NE  ARG A  97       3.314  12.471  13.314  1.00 24.08           N  \\nATOM    769  CZ  ARG A  97       3.268  12.320  12.007  1.00 25.74           C  \\nATOM    770  NH1 ARG A  97       3.135  11.073  11.548  1.00 26.76           N  \\nATOM    771  NH2 ARG A  97       3.353  13.380  11.181  1.00 26.95           N  \\nATOM    772  N   GLU A  98       0.332  14.218  17.678  1.00 13.69           N  \\nATOM    773  CA  GLU A  98      -0.807  13.361  17.941  1.00 16.64           C  \\nATOM    774  C   GLU A  98      -2.118  14.108  17.983  1.00 15.57           C  \\nATOM    775  O   GLU A  98      -3.157  13.573  17.607  1.00 15.85           O  \\nATOM    776  CB  GLU A  98      -0.593  12.623  19.260  1.00 21.96           C  \\nATOM    777  CG  GLU A  98       0.417  11.455  19.177  1.00 28.02           C  \\nATOM    778  CD  GLU A  98       0.147  10.460  18.038  1.00 31.04           C  \\nATOM    779  OE1 GLU A  98       1.118  10.123  17.373  1.00 33.58           O  \\nATOM    780  OE2 GLU A  98      -0.992  10.025  17.790  1.00 33.61           O  \\nATOM    781  N   GLN A  99      -2.105  15.362  18.446  1.00 14.66           N  \\nATOM    782  CA  GLN A  99      -3.343  16.104  18.468  1.00 13.22           C  \\nATOM    783  C   GLN A  99      -3.779  16.398  17.049  1.00 12.15           C  \\nATOM    784  O   GLN A  99      -4.974  16.274  16.791  1.00 14.09           O  \\nATOM    785  CB  GLN A  99      -3.158  17.403  19.250  1.00 16.29           C  \\nATOM    786  CG  GLN A  99      -4.513  18.118  19.407  1.00 19.67           C  \\nATOM    787  CD  GLN A  99      -4.517  19.240  20.438  1.00 22.19           C  \\nATOM    788  OE1 GLN A  99      -3.537  19.485  21.138  1.00 23.70           O  \\nATOM    789  NE2 GLN A  99      -5.628  19.961  20.537  1.00 22.72           N  \\nATOM    790  N   ILE A 100      -2.861  16.795  16.142  1.00 11.42           N  \\nATOM    791  CA  ILE A 100      -3.197  17.121  14.743  1.00 10.81           C  \\nATOM    792  C   ILE A 100      -3.761  15.893  14.008  1.00 12.39           C  \\nATOM    793  O   ILE A 100      -4.733  15.967  13.243  1.00 12.70           O  \\nATOM    794  CB  ILE A 100      -1.924  17.659  14.038  1.00 10.32           C  \\nATOM    795  CG1 ILE A 100      -1.642  19.075  14.518  1.00 10.37           C  \\nATOM    796  CG2 ILE A 100      -2.131  17.685  12.512  1.00 11.36           C  \\nATOM    797  CD1 ILE A 100      -0.184  19.481  14.392  1.00 10.69           C  \\nATOM    798  N   LYS A 101      -3.143  14.732  14.255  1.00 14.08           N  \\nATOM    799  CA  LYS A 101      -3.613  13.487  13.637  1.00 15.95           C  \\nATOM    800  C   LYS A 101      -5.021  13.150  14.079  1.00 15.72           C  \\nATOM    801  O   LYS A 101      -5.848  12.770  13.262  1.00 14.25           O  \\nATOM    802  CB  LYS A 101      -2.681  12.317  13.996  1.00 17.44           C  \\nATOM    803  CG  LYS A 101      -1.414  12.284  13.115  1.00 19.24           C  \\nATOM    804  CD  LYS A 101      -0.332  11.418  13.764  1.00 23.28           C  \\nATOM    805  CE  LYS A 101      -0.752  10.000  14.188  1.00 23.82           C  \\nATOM    806  NZ  LYS A 101       0.400   9.230  14.662  1.00 24.00           N  \\nATOM    807  N   ARG A 102      -5.278  13.341  15.377  1.00 16.72           N  \\nATOM    808  CA  ARG A 102      -6.586  13.098  15.962  1.00 18.74           C  \\nATOM    809  C   ARG A 102      -7.643  14.087  15.440  1.00 19.07           C  \\nATOM    810  O   ARG A 102      -8.712  13.628  15.002  1.00 20.67           O  \\nATOM    811  CB  ARG A 102      -6.457  13.197  17.462  1.00 20.62           C  \\nATOM    812  CG  ARG A 102      -7.539  12.409  18.132  1.00 26.89           C  \\nATOM    813  CD  ARG A 102      -7.610  12.887  19.585  1.00 30.60           C  \\nATOM    814  NE  ARG A 102      -8.197  14.224  19.645  1.00 35.83           N  \\nATOM    815  CZ  ARG A 102      -7.633  15.255  20.301  1.00 37.61           C  \\nATOM    816  NH1 ARG A 102      -8.243  16.455  20.267  1.00 38.37           N  \\nATOM    817  NH2 ARG A 102      -6.481  15.110  20.983  1.00 38.39           N  \\nATOM    818  N   VAL A 103      -7.418  15.407  15.395  1.00 15.59           N  \\nATOM    819  CA  VAL A 103      -8.452  16.326  14.881  1.00 18.48           C  \\nATOM    820  C   VAL A 103      -8.665  16.200  13.349  1.00 16.72           C  \\nATOM    821  O   VAL A 103      -9.776  16.383  12.848  1.00 15.90           O  \\nATOM    822  CB  VAL A 103      -8.131  17.840  15.215  1.00 20.10           C  \\nATOM    823  CG1 VAL A 103      -7.896  18.005  16.704  1.00 21.34           C  \\nATOM    824  CG2 VAL A 103      -6.899  18.302  14.512  1.00 23.93           C  \\nATOM    825  N   LYS A 104      -7.622  15.902  12.574  1.00 13.84           N  \\nATOM    826  CA  LYS A 104      -7.804  15.755  11.140  1.00 16.11           C  \\nATOM    827  C   LYS A 104      -8.158  14.329  10.716  1.00 16.70           C  \\nATOM    828  O   LYS A 104      -8.260  14.079   9.508  1.00 17.27           O  \\nATOM    829  CB  LYS A 104      -6.525  16.215  10.404  1.00 18.12           C  \\nATOM    830  CG  LYS A 104      -6.197  17.678  10.744  1.00 19.53           C  \\nATOM    831  CD  LYS A 104      -5.250  18.297   9.736  1.00 21.28           C  \\nATOM    832  CE  LYS A 104      -5.921  18.590   8.390  1.00 20.75           C  \\nATOM    833  NZ  LYS A 104      -6.875  19.673   8.462  1.00 19.79           N  \\nATOM    834  N   ASP A 105      -8.372  13.390  11.669  1.00 15.91           N  \\nATOM    835  CA  ASP A 105      -8.666  11.988  11.359  1.00 15.43           C  \\nATOM    836  C   ASP A 105      -7.705  11.372  10.333  1.00 16.04           C  \\nATOM    837  O   ASP A 105      -8.106  10.654   9.415  1.00 15.38           O  \\nATOM    838  CB  ASP A 105     -10.077  11.847  10.810  1.00 18.74           C  \\nATOM    839  CG  ASP A 105     -11.261  12.121  11.696  1.00 21.15           C  \\nATOM    840  OD1 ASP A 105     -11.183  11.972  12.918  1.00 21.36           O  \\nATOM    841  OD2 ASP A 105     -12.281  12.480  11.113  1.00 22.63           O  \\nATOM    842  N   SER A 106      -6.397  11.591  10.469  1.00 15.54           N  \\nATOM    843  CA  SER A 106      -5.470  11.174   9.442  1.00 17.25           C  \\nATOM    844  C   SER A 106      -4.112  10.882  10.011  1.00 17.31           C  \\nATOM    845  O   SER A 106      -3.650  11.556  10.916  1.00 19.11           O  \\nATOM    846  CB  SER A 106      -5.463  12.314   8.398  1.00 19.27           C  \\nATOM    847  OG  SER A 106      -4.326  12.490   7.573  1.00 24.20           O  \\nATOM    848  N   ASP A 107      -3.454   9.891   9.453  1.00 17.29           N  \\nATOM    849  CA  ASP A 107      -2.109   9.557   9.868  1.00 19.80           C  \\nATOM    850  C   ASP A 107      -1.000  10.220   9.063  1.00 20.36           C  \\nATOM    851  O   ASP A 107       0.197  10.118   9.345  1.00 21.26           O  \\nATOM    852  CB  ASP A 107      -1.970   8.045   9.810  1.00 23.15           C  \\nATOM    853  CG  ASP A 107      -2.863   7.310  10.807  1.00 25.12           C  \\nATOM    854  OD1 ASP A 107      -3.262   6.206  10.462  1.00 29.21           O  \\nATOM    855  OD2 ASP A 107      -3.127   7.812  11.903  1.00 26.59           O  \\nATOM    856  N   ASP A 108      -1.383  10.909   8.005  1.00 20.57           N  \\nATOM    857  CA  ASP A 108      -0.430  11.664   7.216  1.00 22.38           C  \\nATOM    858  C   ASP A 108      -1.061  12.982   6.842  1.00 21.37           C  \\nATOM    859  O   ASP A 108      -1.973  13.096   6.030  1.00 24.36           O  \\nATOM    860  CB  ASP A 108      -0.056  10.900   5.967  1.00 27.28           C  \\nATOM    861  CG  ASP A 108      -1.263  10.278   5.320  1.00 32.24           C  \\nATOM    862  OD1 ASP A 108      -1.714   9.272   5.877  1.00 37.07           O  \\nATOM    863  OD2 ASP A 108      -1.747  10.790   4.301  1.00 36.33           O  \\nATOM    864  N   VAL A 109      -0.598  13.970   7.572  1.00 19.33           N  \\nATOM    865  CA  VAL A 109      -1.009  15.354   7.400  1.00 17.49           C  \\nATOM    866  C   VAL A 109       0.215  16.086   6.819  1.00 14.55           C  \\nATOM    867  O   VAL A 109       1.335  15.772   7.245  1.00 13.64           O  \\nATOM    868  CB  VAL A 109      -1.410  15.870   8.784  1.00 18.40           C  \\nATOM    869  CG1 VAL A 109      -1.645  17.366   8.765  1.00 20.21           C  \\nATOM    870  CG2 VAL A 109      -2.676  15.101   9.214  1.00 20.27           C  \\nATOM    871  N   PRO A 110       0.141  17.001   5.825  1.00 12.41           N  \\nATOM    872  CA  PRO A 110       1.297  17.783   5.384  1.00 11.52           C  \\nATOM    873  C   PRO A 110       1.920  18.480   6.605  1.00 11.79           C  \\nATOM    874  O   PRO A 110       1.289  19.245   7.324  1.00  9.55           O  \\nATOM    875  CB  PRO A 110       0.728  18.721   4.370  1.00 11.31           C  \\nATOM    876  CG  PRO A 110      -0.410  17.926   3.779  1.00 11.38           C  \\nATOM    877  CD  PRO A 110      -1.044  17.309   5.020  1.00 12.05           C  \\nATOM    878  N   MET A 111       3.192  18.201   6.874  1.00 11.92           N  \\nATOM    879  CA  MET A 111       3.878  18.724   8.050  1.00 11.49           C  \\nATOM    880  C   MET A 111       5.379  18.749   7.807  1.00 11.68           C  \\nATOM    881  O   MET A 111       5.903  17.900   7.098  1.00 11.11           O  \\nATOM    882  CB  MET A 111       3.494  17.806   9.211  1.00 14.36           C  \\nATOM    883  CG  MET A 111       3.864  18.110  10.627  1.00 17.41           C  \\nATOM    884  SD  MET A 111       3.231  16.825  11.758  1.00 18.36           S  \\nATOM    885  CE  MET A 111       1.528  17.128  11.498  1.00 17.25           C  \\nATOM    886  N   VAL A 112       6.091  19.706   8.404  1.00 10.73           N  \\nATOM    887  CA  VAL A 112       7.545  19.763   8.348  1.00 10.18           C  \\nATOM    888  C   VAL A 112       7.992  19.875   9.794  1.00  9.11           C  \\nATOM    889  O   VAL A 112       7.404  20.633  10.556  1.00 10.77           O  \\nATOM    890  CB  VAL A 112       8.010  21.024   7.542  1.00 11.80           C  \\nATOM    891  CG1 VAL A 112       9.528  21.213   7.558  1.00 13.24           C  \\nATOM    892  CG2 VAL A 112       7.542  20.843   6.104  1.00 14.56           C  \\nATOM    893  N   LEU A 113       9.032  19.167  10.207  1.00  8.47           N  \\nATOM    894  CA  LEU A 113       9.632  19.332  11.535  1.00  9.53           C  \\nATOM    895  C   LEU A 113      10.738  20.384  11.412  1.00  9.33           C  \\nATOM    896  O   LEU A 113      11.621  20.270  10.555  1.00  9.95           O  \\nATOM    897  CB  LEU A 113      10.214  17.983  12.017  1.00  9.56           C  \\nATOM    898  CG  LEU A 113      10.863  17.976  13.381  1.00 11.58           C  \\nATOM    899  CD1 LEU A 113       9.908  18.379  14.503  1.00 13.17           C  \\nATOM    900  CD2 LEU A 113      11.361  16.564  13.597  1.00 12.84           C  \\nATOM    901  N   VAL A 114      10.705  21.442  12.221  1.00  7.62           N  \\nATOM    902  CA  VAL A 114      11.673  22.541  12.109  1.00  7.64           C  \\nATOM    903  C   VAL A 114      12.436  22.667  13.441  1.00  7.65           C  \\nATOM    904  O   VAL A 114      11.872  22.842  14.541  1.00  9.71           O  \\nATOM    905  CB  VAL A 114      10.896  23.836  11.774  1.00  9.37           C  \\nATOM    906  CG1 VAL A 114      11.814  25.030  11.869  1.00 10.27           C  \\nATOM    907  CG2 VAL A 114      10.254  23.719  10.393  1.00  9.64           C  \\nATOM    908  N   GLY A 115      13.745  22.567  13.331  1.00  6.10           N  \\nATOM    909  CA  GLY A 115      14.603  22.742  14.476  1.00  6.69           C  \\nATOM    910  C   GLY A 115      15.120  24.163  14.407  1.00  8.47           C  \\nATOM    911  O   GLY A 115      16.007  24.439  13.594  1.00  9.16           O  \\nATOM    912  N   ASN A 116      14.593  25.089  15.192  1.00  8.01           N  \\nATOM    913  CA  ASN A 116      15.007  26.475  15.111  1.00  8.24           C  \\nATOM    914  C   ASN A 116      16.112  26.873  16.109  1.00  9.61           C  \\nATOM    915  O   ASN A 116      16.453  26.142  17.051  1.00  9.84           O  \\nATOM    916  CB  ASN A 116      13.721  27.307  15.275  1.00  8.01           C  \\nATOM    917  CG  ASN A 116      13.929  28.788  14.992  1.00  9.48           C  \\nATOM    918  OD1 ASN A 116      14.470  29.205  13.972  1.00  9.94           O  \\nATOM    919  ND2 ASN A 116      13.565  29.644  15.940  1.00 10.75           N  \\nATOM    920  N   LYS A 117      16.660  28.087  15.876  1.00 10.22           N  \\nATOM    921  CA  LYS A 117      17.735  28.715  16.674  1.00 12.36           C  \\nATOM    922  C   LYS A 117      19.094  28.021  16.450  1.00 13.39           C  \\nATOM    923  O   LYS A 117      19.977  27.940  17.324  1.00 11.29           O  \\nATOM    924  CB  LYS A 117      17.391  28.706  18.214  1.00 13.23           C  \\nATOM    925  CG  LYS A 117      16.026  29.272  18.611  1.00 12.90           C  \\nATOM    926  CD  LYS A 117      15.932  29.640  20.097  1.00 12.43           C  \\nATOM    927  CE  LYS A 117      14.551  30.217  20.380  1.00 10.55           C  \\nATOM    928  NZ  LYS A 117      14.491  30.647  21.772  1.00 11.10           N  \\nATOM    929  N   CYS A 118      19.254  27.555  15.188  1.00 13.69           N  \\nATOM    930  CA  CYS A 118      20.449  26.806  14.844  1.00 15.67           C  \\nATOM    931  C   CYS A 118      21.716  27.658  14.796  1.00 15.28           C  \\nATOM    932  O   CYS A 118      22.808  27.113  14.629  1.00 15.30           O  \\nATOM    933  CB  CYS A 118      20.246  26.064  13.492  1.00 17.74           C  \\nATOM    934  SG  CYS A 118      20.353  27.009  11.944  1.00 20.74           S  \\nATOM    935  N   ASP A 119      21.578  28.995  14.931  1.00 15.76           N  \\nATOM    936  CA  ASP A 119      22.722  29.904  15.083  1.00 16.09           C  \\nATOM    937  C   ASP A 119      23.426  29.778  16.445  1.00 17.19           C  \\nATOM    938  O   ASP A 119      24.589  30.178  16.553  1.00 18.76           O  \\nATOM    939  CB  ASP A 119      22.268  31.374  14.888  1.00 15.14           C  \\nATOM    940  CG  ASP A 119      21.153  31.785  15.835  1.00 15.81           C  \\nATOM    941  OD1 ASP A 119      20.030  31.294  15.662  1.00 13.87           O  \\nATOM    942  OD2 ASP A 119      21.407  32.596  16.721  1.00 14.85           O  \\nATOM    943  N   LEU A 120      22.732  29.248  17.480  1.00 16.86           N  \\nATOM    944  CA  LEU A 120      23.230  29.099  18.845  1.00 18.76           C  \\nATOM    945  C   LEU A 120      24.217  27.957  18.979  1.00 19.70           C  \\nATOM    946  O   LEU A 120      23.979  26.885  18.431  1.00 20.53           O  \\nATOM    947  CB  LEU A 120      22.103  28.829  19.831  1.00 17.65           C  \\nATOM    948  CG  LEU A 120      21.065  29.933  20.010  1.00 19.43           C  \\nATOM    949  CD1 LEU A 120      20.044  29.418  21.008  1.00 18.46           C  \\nATOM    950  CD2 LEU A 120      21.688  31.239  20.503  1.00 18.73           C  \\nATOM    951  N   ALA A 121      25.313  28.184  19.705  1.00 19.26           N  \\nATOM    952  CA  ALA A 121      26.303  27.143  19.887  1.00 20.48           C  \\nATOM    953  C   ALA A 121      25.942  26.210  21.036  1.00 21.30           C  \\nATOM    954  O   ALA A 121      26.403  25.069  21.067  1.00 24.04           O  \\nATOM    955  CB  ALA A 121      27.652  27.767  20.164  1.00 22.02           C  \\nATOM    956  N   ALA A 122      25.109  26.635  21.982  1.00 22.09           N  \\nATOM    957  CA  ALA A 122      24.683  25.797  23.094  1.00 23.88           C  \\nATOM    958  C   ALA A 122      23.578  24.811  22.720  1.00 23.77           C  \\nATOM    959  O   ALA A 122      22.492  24.835  23.301  1.00 27.82           O  \\nATOM    960  CB  ALA A 122      24.171  26.684  24.249  1.00 25.67           C  \\nATOM    961  N   ARG A 123      23.843  23.908  21.783  1.00 20.99           N  \\nATOM    962  CA  ARG A 123      22.890  22.905  21.311  1.00 19.81           C  \\nATOM    963  C   ARG A 123      22.619  21.761  22.283  1.00 19.56           C  \\nATOM    964  O   ARG A 123      23.590  21.176  22.774  1.00 20.01           O  \\nATOM    965  CB  ARG A 123      23.421  22.333  19.988  1.00 19.81           C  \\nATOM    966  CG  ARG A 123      22.662  21.166  19.398  1.00 21.43           C  \\nATOM    967  CD  ARG A 123      23.273  20.680  18.073  1.00 22.66           C  \\nATOM    968  NE  ARG A 123      23.050  21.590  16.956  1.00 22.61           N  \\nATOM    969  CZ  ARG A 123      22.019  21.487  16.093  1.00 20.95           C  \\nATOM    970  NH1 ARG A 123      21.045  20.568  16.181  1.00 16.36           N  \\nATOM    971  NH2 ARG A 123      21.989  22.370  15.098  1.00 19.65           N  \\nATOM    972  N   THR A 124      21.364  21.389  22.600  1.00 17.96           N  \\nATOM    973  CA  THR A 124      21.118  20.200  23.415  1.00 17.85           C  \\nATOM    974  C   THR A 124      20.413  19.068  22.656  1.00 17.08           C  \\nATOM    975  O   THR A 124      20.459  17.926  23.091  1.00 19.59           O  \\nATOM    976  CB  THR A 124      20.311  20.578  24.672  1.00 19.42           C  \\nATOM    977  OG1 THR A 124      19.109  21.230  24.277  1.00 20.50           O  \\nATOM    978  CG2 THR A 124      21.155  21.458  25.588  1.00 19.06           C  \\nATOM    979  N   VAL A 125      19.788  19.319  21.495  1.00 16.59           N  \\nATOM    980  CA  VAL A 125      19.133  18.324  20.647  1.00 16.09           C  \\nATOM    981  C   VAL A 125      20.032  18.080  19.413  1.00 16.61           C  \\nATOM    982  O   VAL A 125      20.180  18.980  18.573  1.00 14.07           O  \\nATOM    983  CB  VAL A 125      17.720  18.848  20.179  1.00 15.28           C  \\nATOM    984  CG1 VAL A 125      17.097  17.814  19.265  1.00 14.81           C  \\nATOM    985  CG2 VAL A 125      16.824  19.168  21.366  1.00 15.97           C  \\nATOM    986  N   GLU A 126      20.659  16.889  19.280  1.00 17.05           N  \\nATOM    987  CA  GLU A 126      21.487  16.568  18.124  1.00 19.28           C  \\nATOM    988  C   GLU A 126      20.631  16.496  16.876  1.00 18.51           C  \\nATOM    989  O   GLU A 126      19.511  15.976  16.880  1.00 17.68           O  \\nATOM    990  CB  GLU A 126      22.165  15.215  18.219  1.00 23.29           C  \\nATOM    991  CG  GLU A 126      23.244  15.057  19.265  1.00 30.86           C  \\nATOM    992  CD  GLU A 126      24.402  16.054  19.153  1.00 36.59           C  \\nATOM    993  OE1 GLU A 126      25.139  16.012  18.161  1.00 39.64           O  \\nATOM    994  OE2 GLU A 126      24.581  16.885  20.056  1.00 40.44           O  \\nATOM    995  N   SER A 127      21.167  16.985  15.755  1.00 17.73           N  \\nATOM    996  CA  SER A 127      20.426  16.938  14.493  1.00 16.72           C  \\nATOM    997  C   SER A 127      20.040  15.506  14.158  1.00 16.76           C  \\nATOM    998  O   SER A 127      18.931  15.245  13.690  1.00 17.01           O  \\nATOM    999  CB  SER A 127      21.283  17.518  13.371  1.00 17.67           C  \\nATOM   1000  OG  SER A 127      21.730  18.826  13.768  1.00 19.67           O  \\nATOM   1001  N   ARG A 128      20.916  14.558  14.466  1.00 16.72           N  \\nATOM   1002  CA  ARG A 128      20.667  13.154  14.145  1.00 20.94           C  \\nATOM   1003  C   ARG A 128      19.435  12.617  14.862  1.00 19.62           C  \\nATOM   1004  O   ARG A 128      18.641  11.882  14.278  1.00 18.81           O  \\nATOM   1005  CB  ARG A 128      21.910  12.319  14.510  1.00 24.31           C  \\nATOM   1006  CG  ARG A 128      21.969  11.035  13.692  1.00 30.88           C  \\nATOM   1007  CD  ARG A 128      23.381  10.403  13.536  1.00 35.88           C  \\nATOM   1008  NE  ARG A 128      24.384  10.996  12.621  1.00 39.57           N  \\nATOM   1009  CZ  ARG A 128      24.313  11.018  11.267  1.00 40.85           C  \\nATOM   1010  NH1 ARG A 128      25.399  11.388  10.577  1.00 40.83           N  \\nATOM   1011  NH2 ARG A 128      23.202  10.685  10.576  1.00 41.33           N  \\nATOM   1012  N   GLN A 129      19.209  13.016  16.119  1.00 18.27           N  \\nATOM   1013  CA  GLN A 129      18.032  12.519  16.787  1.00 18.53           C  \\nATOM   1014  C   GLN A 129      16.735  13.151  16.224  1.00 17.11           C  \\nATOM   1015  O   GLN A 129      15.691  12.486  16.098  1.00 15.48           O  \\nATOM   1016  CB  GLN A 129      18.229  12.766  18.306  1.00 22.36           C  \\nATOM   1017  CG  GLN A 129      18.041  14.133  18.911  1.00 24.78           C  \\nATOM   1018  CD  GLN A 129      18.283  14.123  20.411  1.00 27.48           C  \\nATOM   1019  OE1 GLN A 129      19.287  14.643  20.906  1.00 28.14           O  \\nATOM   1020  NE2 GLN A 129      17.367  13.497  21.152  1.00 27.21           N  \\nATOM   1021  N   ALA A 130      16.780  14.446  15.837  1.00 14.73           N  \\nATOM   1022  CA  ALA A 130      15.611  15.044  15.210  1.00 13.87           C  \\nATOM   1023  C   ALA A 130      15.342  14.449  13.826  1.00 13.30           C  \\nATOM   1024  O   ALA A 130      14.199  14.194  13.436  1.00 11.42           O  \\nATOM   1025  CB  ALA A 130      15.822  16.536  15.063  1.00 16.21           C  \\nATOM   1026  N   GLN A 131      16.401  14.189  13.057  1.00 14.61           N  \\nATOM   1027  CA  GLN A 131      16.267  13.529  11.739  1.00 16.01           C  \\nATOM   1028  C   GLN A 131      15.637  12.109  11.821  1.00 16.29           C  \\nATOM   1029  O   GLN A 131      14.777  11.708  11.017  1.00 14.37           O  \\nATOM   1030  CB  GLN A 131      17.673  13.507  11.116  1.00 18.12           C  \\nATOM   1031  CG  GLN A 131      17.702  13.017   9.689  1.00 22.19           C  \\nATOM   1032  CD  GLN A 131      16.889  13.922   8.801  1.00 21.61           C  \\nATOM   1033  OE1 GLN A 131      17.292  15.044   8.529  1.00 24.06           O  \\nATOM   1034  NE2 GLN A 131      15.714  13.485   8.370  1.00 23.37           N  \\nATOM   1035  N   ASP A 132      16.038  11.348  12.853  1.00 16.64           N  \\nATOM   1036  CA  ASP A 132      15.467  10.038  13.173  1.00 18.00           C  \\nATOM   1037  C   ASP A 132      13.984  10.175  13.484  1.00 17.28           C  \\nATOM   1038  O   ASP A 132      13.178   9.395  12.985  1.00 16.45           O  \\nATOM   1039  CB  ASP A 132      16.120   9.415  14.405  1.00 20.10           C  \\nATOM   1040  CG  ASP A 132      17.506   8.816  14.234  1.00 24.07           C  \\nATOM   1041  OD1 ASP A 132      18.120   8.459  15.243  1.00 28.35           O  \\nATOM   1042  OD2 ASP A 132      17.975   8.686  13.106  1.00 25.43           O  \\nATOM   1043  N   LEU A 133      13.614  11.168  14.312  1.00 15.58           N  \\nATOM   1044  CA  LEU A 133      12.213  11.390  14.624  1.00 16.28           C  \\nATOM   1045  C   LEU A 133      11.421  11.729  13.366  1.00 14.96           C  \\nATOM   1046  O   LEU A 133      10.345  11.181  13.128  1.00 15.09           O  \\nATOM   1047  CB  LEU A 133      12.052  12.536  15.647  1.00 17.06           C  \\nATOM   1048  CG  LEU A 133      10.617  12.781  16.136  1.00 18.79           C  \\nATOM   1049  CD1 LEU A 133      10.142  11.554  16.881  1.00 19.96           C  \\nATOM   1050  CD2 LEU A 133      10.544  14.013  17.020  1.00 18.45           C  \\nATOM   1051  N   ALA A 134      11.937  12.642  12.545  1.00 13.16           N  \\nATOM   1052  CA  ALA A 134      11.241  13.016  11.331  1.00 14.89           C  \\nATOM   1053  C   ALA A 134      11.092  11.821  10.414  1.00 16.19           C  \\nATOM   1054  O   ALA A 134      10.015  11.643   9.857  1.00 15.70           O  \\nATOM   1055  CB  ALA A 134      12.021  14.089  10.629  1.00 15.12           C  \\nATOM   1056  N   ARG A 135      12.143  10.995  10.279  1.00 17.00           N  \\nATOM   1057  CA  ARG A 135      12.088   9.828   9.438  1.00 19.17           C  \\nATOM   1058  C   ARG A 135      10.990   8.893   9.916  1.00 18.70           C  \\nATOM   1059  O   ARG A 135      10.269   8.352   9.078  1.00 20.29           O  \\nATOM   1060  CB  ARG A 135      13.425   9.091   9.456  1.00 22.48           C  \\nATOM   1061  CG  ARG A 135      13.499   8.297   8.155  1.00 29.02           C  \\nATOM   1062  CD  ARG A 135      14.619   7.270   7.988  1.00 33.22           C  \\nATOM   1063  NE  ARG A 135      14.117   5.888   8.050  1.00 38.80           N  \\nATOM   1064  CZ  ARG A 135      13.281   5.302   7.148  1.00 39.60           C  \\nATOM   1065  NH1 ARG A 135      12.889   4.044   7.384  1.00 40.80           N  \\nATOM   1066  NH2 ARG A 135      12.827   5.913   6.034  1.00 40.11           N  \\nATOM   1067  N   SER A 136      10.815   8.697  11.242  1.00 19.23           N  \\nATOM   1068  CA  SER A 136       9.757   7.841  11.770  1.00 19.12           C  \\nATOM   1069  C   SER A 136       8.374   8.432  11.528  1.00 19.61           C  \\nATOM   1070  O   SER A 136       7.398   7.690  11.522  1.00 21.01           O  \\nATOM   1071  CB  SER A 136       9.930   7.624  13.280  1.00 19.94           C  \\nATOM   1072  OG  SER A 136       9.459   8.686  14.119  1.00 23.21           O  \\nATOM   1073  N   TYR A 137       8.231   9.760  11.380  1.00 17.33           N  \\nATOM   1074  CA  TYR A 137       6.950  10.375  11.078  1.00 18.12           C  \\nATOM   1075  C   TYR A 137       6.683  10.404   9.570  1.00 18.22           C  \\nATOM   1076  O   TYR A 137       5.561  10.656   9.129  1.00 19.27           O  \\nATOM   1077  CB  TYR A 137       6.907  11.812  11.592  1.00 18.21           C  \\nATOM   1078  CG  TYR A 137       6.883  12.057  13.084  1.00 19.08           C  \\nATOM   1079  CD1 TYR A 137       6.619  11.026  13.955  1.00 20.51           C  \\nATOM   1080  CD2 TYR A 137       7.059  13.355  13.548  1.00 20.43           C  \\nATOM   1081  CE1 TYR A 137       6.510  11.285  15.306  1.00 21.89           C  \\nATOM   1082  CE2 TYR A 137       6.945  13.624  14.899  1.00 21.02           C  \\nATOM   1083  CZ  TYR A 137       6.662  12.574  15.763  1.00 22.13           C  \\nATOM   1084  OH  TYR A 137       6.476  12.782  17.112  1.00 22.16           O  \\nATOM   1085  N   GLY A 138       7.689  10.186   8.737  1.00 16.07           N  \\nATOM   1086  CA  GLY A 138       7.520  10.265   7.294  1.00 17.66           C  \\nATOM   1087  C   GLY A 138       7.476  11.725   6.810  1.00 18.01           C  \\nATOM   1088  O   GLY A 138       6.859  12.010   5.774  1.00 17.85           O  \\nATOM   1089  N   ILE A 139       8.128  12.662   7.540  1.00 14.16           N  \\nATOM   1090  CA  ILE A 139       8.117  14.065   7.145  1.00 13.49           C  \\nATOM   1091  C   ILE A 139       9.526  14.662   7.033  1.00 14.35           C  \\nATOM   1092  O   ILE A 139      10.494  14.152   7.610  1.00 16.19           O  \\nATOM   1093  CB  ILE A 139       7.292  14.896   8.161  1.00 14.40           C  \\nATOM   1094  CG1 ILE A 139       7.925  14.811   9.554  1.00 15.44           C  \\nATOM   1095  CG2 ILE A 139       5.821  14.422   8.104  1.00 14.07           C  \\nATOM   1096  CD1 ILE A 139       7.288  15.765  10.587  1.00 15.35           C  \\nATOM   1097  N   PRO A 140       9.706  15.767   6.308  1.00 13.55           N  \\nATOM   1098  CA  PRO A 140      10.984  16.448   6.205  1.00 13.39           C  \\nATOM   1099  C   PRO A 140      11.424  17.065   7.532  1.00 12.74           C  \\nATOM   1100  O   PRO A 140      10.601  17.530   8.310  1.00 11.42           O  \\nATOM   1101  CB  PRO A 140      10.767  17.498   5.127  1.00 13.47           C  \\nATOM   1102  CG  PRO A 140       9.514  17.084   4.410  1.00 14.09           C  \\nATOM   1103  CD  PRO A 140       8.686  16.467   5.519  1.00 13.14           C  \\nATOM   1104  N   TYR A 141      12.744  17.086   7.756  1.00 11.68           N  \\nATOM   1105  CA  TYR A 141      13.336  17.791   8.890  1.00 10.89           C  \\nATOM   1106  C   TYR A 141      14.175  18.945   8.326  1.00 10.09           C  \\nATOM   1107  O   TYR A 141      15.056  18.720   7.468  1.00 10.27           O  \\nATOM   1108  CB  TYR A 141      14.276  16.897   9.691  1.00 12.54           C  \\nATOM   1109  CG  TYR A 141      15.063  17.676  10.733  1.00 10.92           C  \\nATOM   1110  CD1 TYR A 141      14.350  18.396  11.664  1.00 11.75           C  \\nATOM   1111  CD2 TYR A 141      16.453  17.633  10.784  1.00 12.49           C  \\nATOM   1112  CE1 TYR A 141      14.997  19.074  12.670  1.00 12.85           C  \\nATOM   1113  CE2 TYR A 141      17.112  18.307  11.794  1.00 11.33           C  \\nATOM   1114  CZ  TYR A 141      16.369  19.016  12.728  1.00 12.61           C  \\nATOM   1115  OH  TYR A 141      16.944  19.646  13.813  1.00 13.82           O  \\nATOM   1116  N   ILE A 142      13.892  20.174   8.799  1.00  9.35           N  \\nATOM   1117  CA  ILE A 142      14.636  21.355   8.376  1.00  9.05           C  \\nATOM   1118  C   ILE A 142      15.153  22.163   9.551  1.00  9.25           C  \\nATOM   1119  O   ILE A 142      14.409  22.382  10.499  1.00 10.43           O  \\nATOM   1120  CB  ILE A 142      13.756  22.259   7.472  1.00  9.25           C  \\nATOM   1121  CG1 ILE A 142      13.523  21.470   6.180  1.00 10.51           C  \\nATOM   1122  CG2 ILE A 142      14.450  23.586   7.122  1.00  9.08           C  \\nATOM   1123  CD1 ILE A 142      12.719  22.127   5.028  1.00 13.16           C  \\nATOM   1124  N   GLU A 143      16.457  22.513   9.568  1.00  9.01           N  \\nATOM   1125  CA  GLU A 143      16.979  23.404  10.611  1.00  8.52           C  \\nATOM   1126  C   GLU A 143      16.978  24.864  10.193  1.00 10.31           C  \\nATOM   1127  O   GLU A 143      17.278  25.228   9.040  1.00 10.45           O  \\nATOM   1128  CB  GLU A 143      18.397  23.022  11.013  1.00 10.87           C  \\nATOM   1129  CG  GLU A 143      18.363  21.728  11.801  1.00 10.22           C  \\nATOM   1130  CD  GLU A 143      19.592  21.411  12.633  1.00 13.01           C  \\nATOM   1131  OE1 GLU A 143      20.616  22.082  12.521  1.00 13.65           O  \\nATOM   1132  OE2 GLU A 143      19.511  20.487  13.429  1.00 12.70           O  \\nATOM   1133  N   THR A 144      16.608  25.762  11.117  1.00  9.18           N  \\nATOM   1134  CA  THR A 144      16.422  27.169  10.754  1.00  9.71           C  \\nATOM   1135  C   THR A 144      17.014  28.076  11.789  1.00  8.66           C  \\nATOM   1136  O   THR A 144      17.259  27.698  12.927  1.00  9.13           O  \\nATOM   1137  CB  THR A 144      14.919  27.617  10.635  1.00  9.90           C  \\nATOM   1138  OG1 THR A 144      14.336  27.403  11.931  1.00  9.29           O  \\nATOM   1139  CG2 THR A 144      14.120  26.858   9.557  1.00  9.89           C  \\nATOM   1140  N   SER A 145      17.244  29.288  11.360  1.00 10.91           N  \\nATOM   1141  CA  SER A 145      17.558  30.377  12.290  1.00 12.65           C  \\nATOM   1142  C   SER A 145      16.758  31.602  11.849  1.00 10.37           C  \\nATOM   1143  O   SER A 145      16.930  32.094  10.727  1.00 11.76           O  \\nATOM   1144  CB  SER A 145      19.069  30.768  12.308  1.00 11.79           C  \\nATOM   1145  OG  SER A 145      19.248  31.931  13.125  1.00 12.12           O  \\nATOM   1146  N   ALA A 146      15.838  32.046  12.696  1.00  9.49           N  \\nATOM   1147  CA  ALA A 146      15.131  33.268  12.413  1.00 10.90           C  \\nATOM   1148  C   ALA A 146      16.070  34.455  12.553  1.00 13.53           C  \\nATOM   1149  O   ALA A 146      15.799  35.514  11.995  1.00 16.20           O  \\nATOM   1150  CB  ALA A 146      13.976  33.436  13.367  1.00 12.65           C  \\nATOM   1151  N   LYS A 147      17.183  34.302  13.283  1.00 14.35           N  \\nATOM   1152  CA  LYS A 147      18.150  35.351  13.486  1.00 16.25           C  \\nATOM   1153  C   LYS A 147      18.953  35.568  12.215  1.00 17.58           C  \\nATOM   1154  O   LYS A 147      19.124  36.724  11.825  1.00 20.13           O  \\nATOM   1155  CB  LYS A 147      19.068  34.970  14.617  1.00 17.55           C  \\nATOM   1156  CG  LYS A 147      20.003  36.125  14.958  1.00 22.83           C  \\nATOM   1157  CD  LYS A 147      20.581  35.932  16.341  1.00 26.44           C  \\nATOM   1158  CE  LYS A 147      21.683  36.949  16.507  1.00 30.79           C  \\nATOM   1159  NZ  LYS A 147      22.575  36.534  17.574  1.00 33.81           N  \\nATOM   1160  N   THR A 148      19.479  34.529  11.541  1.00 17.16           N  \\nATOM   1161  CA  THR A 148      20.248  34.735  10.300  1.00 15.93           C  \\nATOM   1162  C   THR A 148      19.422  34.607   9.013  1.00 17.72           C  \\nATOM   1163  O   THR A 148      19.926  34.962   7.945  1.00 18.47           O  \\nATOM   1164  CB  THR A 148      21.444  33.745  10.209  1.00 15.86           C  \\nATOM   1165  OG1 THR A 148      20.886  32.435  10.042  1.00 14.56           O  \\nATOM   1166  CG2 THR A 148      22.377  33.839  11.439  1.00 15.25           C  \\nATOM   1167  N   ARG A 149      18.176  34.108   9.100  1.00 15.18           N  \\nATOM   1168  CA  ARG A 149      17.268  33.815   7.994  1.00 19.20           C  \\nATOM   1169  C   ARG A 149      17.652  32.483   7.318  1.00 17.94           C  \\nATOM   1170  O   ARG A 149      17.031  32.079   6.332  1.00 20.28           O  \\nATOM   1171  CB  ARG A 149      17.251  34.964   6.937  1.00 21.40           C  \\nATOM   1172  CG  ARG A 149      15.929  35.106   6.188  1.00 27.00           C  \\nATOM   1173  CD  ARG A 149      15.946  35.904   4.863  1.00 27.99           C  \\nATOM   1174  NE  ARG A 149      14.707  35.687   4.092  1.00 29.40           N  \\nATOM   1175  CZ  ARG A 149      13.550  36.232   4.492  1.00 29.07           C  \\nATOM   1176  NH1 ARG A 149      12.412  36.015   3.828  1.00 29.56           N  \\nATOM   1177  NH2 ARG A 149      13.527  37.004   5.577  1.00 28.99           N  \\nATOM   1178  N   GLN A 150      18.621  31.701   7.838  1.00 15.80           N  \\nATOM   1179  CA  GLN A 150      18.926  30.402   7.273  1.00 16.36           C  \\nATOM   1180  C   GLN A 150      17.737  29.464   7.455  1.00 15.02           C  \\nATOM   1181  O   GLN A 150      17.119  29.351   8.504  1.00 13.57           O  \\nATOM   1182  CB  GLN A 150      20.152  29.729   7.937  1.00 23.05           C  \\nATOM   1183  CG  GLN A 150      20.578  28.448   7.162  1.00 29.37           C  \\nATOM   1184  CD  GLN A 150      21.604  27.530   7.827  1.00 34.84           C  \\nATOM   1185  OE1 GLN A 150      21.301  26.416   8.280  1.00 38.95           O  \\nATOM   1186  NE2 GLN A 150      22.857  27.967   7.883  1.00 35.22           N  \\nATOM   1187  N   GLY A 151      17.412  28.809   6.348  1.00 13.62           N  \\nATOM   1188  CA  GLY A 151      16.357  27.835   6.296  1.00 13.50           C  \\nATOM   1189  C   GLY A 151      14.943  28.395   6.329  1.00 12.36           C  \\nATOM   1190  O   GLY A 151      14.025  27.591   6.148  1.00 11.07           O  \\nATOM   1191  N   VAL A 152      14.706  29.702   6.510  1.00 12.59           N  \\nATOM   1192  CA  VAL A 152      13.339  30.183   6.715  1.00 14.23           C  \\nATOM   1193  C   VAL A 152      12.435  29.971   5.475  1.00 13.41           C  \\nATOM   1194  O   VAL A 152      11.352  29.362   5.577  1.00 12.94           O  \\nATOM   1195  CB  VAL A 152      13.430  31.691   7.167  1.00 12.57           C  \\nATOM   1196  CG1 VAL A 152      12.059  32.316   7.194  1.00 13.14           C  \\nATOM   1197  CG2 VAL A 152      14.044  31.787   8.543  1.00 12.69           C  \\nATOM   1198  N   GLU A 153      12.831  30.410   4.278  1.00 13.52           N  \\nATOM   1199  CA  GLU A 153      12.001  30.213   3.086  1.00 14.81           C  \\nATOM   1200  C   GLU A 153      11.857  28.756   2.742  1.00 11.88           C  \\nATOM   1201  O   GLU A 153      10.797  28.282   2.344  1.00 13.92           O  \\nATOM   1202  CB  GLU A 153      12.592  30.936   1.881  1.00 17.03           C  \\nATOM   1203  CG  GLU A 153      12.454  32.452   2.015  1.00 21.18           C  \\nATOM   1204  CD  GLU A 153      12.870  33.318   0.817  1.00 25.73           C  \\nATOM   1205  OE1 GLU A 153      13.232  32.781  -0.234  1.00 28.32           O  \\nATOM   1206  OE2 GLU A 153      12.822  34.550   0.936  1.00 26.64           O  \\nATOM   1207  N   ASP A 154      12.959  28.031   2.957  1.00 11.31           N  \\nATOM   1208  CA  ASP A 154      13.023  26.593   2.747  1.00 10.16           C  \\nATOM   1209  C   ASP A 154      11.964  25.828   3.561  1.00 10.90           C  \\nATOM   1210  O   ASP A 154      11.224  24.985   3.035  1.00 11.83           O  \\nATOM   1211  CB  ASP A 154      14.420  26.212   3.117  1.00 15.00           C  \\nATOM   1212  CG  ASP A 154      14.919  24.942   2.524  1.00 19.04           C  \\nATOM   1213  OD1 ASP A 154      14.121  24.195   1.938  1.00 22.96           O  \\nATOM   1214  OD2 ASP A 154      16.122  24.697   2.676  1.00 21.26           O  \\nATOM   1215  N   ALA A 155      11.854  26.107   4.881  1.00  8.83           N  \\nATOM   1216  CA  ALA A 155      10.841  25.450   5.716  1.00 10.00           C  \\nATOM   1217  C   ALA A 155       9.424  25.748   5.203  1.00 10.10           C  \\nATOM   1218  O   ALA A 155       8.653  24.814   4.978  1.00 11.14           O  \\nATOM   1219  CB  ALA A 155      10.923  25.933   7.192  1.00  9.50           C  \\nATOM   1220  N   PHE A 156       9.054  27.011   4.949  1.00 10.59           N  \\nATOM   1221  CA  PHE A 156       7.712  27.345   4.475  1.00 10.51           C  \\nATOM   1222  C   PHE A 156       7.453  26.825   3.069  1.00 11.05           C  \\nATOM   1223  O   PHE A 156       6.368  26.302   2.801  1.00 10.80           O  \\nATOM   1224  CB  PHE A 156       7.492  28.890   4.506  1.00 11.54           C  \\nATOM   1225  CG  PHE A 156       7.266  29.450   5.936  1.00 12.42           C  \\nATOM   1226  CD1 PHE A 156       8.319  29.958   6.707  1.00 12.00           C  \\nATOM   1227  CD2 PHE A 156       5.978  29.434   6.478  1.00 12.57           C  \\nATOM   1228  CE1 PHE A 156       8.065  30.434   8.001  1.00 12.45           C  \\nATOM   1229  CE2 PHE A 156       5.747  29.908   7.757  1.00 12.32           C  \\nATOM   1230  CZ  PHE A 156       6.782  30.406   8.513  1.00 10.18           C  \\nATOM   1231  N   TYR A 157       8.408  26.935   2.151  1.00  9.69           N  \\nATOM   1232  CA  TYR A 157       8.148  26.463   0.798  1.00 12.31           C  \\nATOM   1233  C   TYR A 157       8.091  24.945   0.719  1.00 10.42           C  \\nATOM   1234  O   TYR A 157       7.292  24.403  -0.047  1.00 11.35           O  \\nATOM   1235  CB  TYR A 157       9.233  26.978  -0.156  1.00 15.16           C  \\nATOM   1236  CG  TYR A 157       9.257  28.492  -0.367  1.00 18.73           C  \\nATOM   1237  CD1 TYR A 157       8.380  29.366   0.268  1.00 18.52           C  \\nATOM   1238  CD2 TYR A 157      10.211  28.971  -1.242  1.00 20.44           C  \\nATOM   1239  CE1 TYR A 157       8.466  30.724   0.025  1.00 22.04           C  \\nATOM   1240  CE2 TYR A 157      10.305  30.320  -1.488  1.00 22.70           C  \\nATOM   1241  CZ  TYR A 157       9.436  31.178  -0.852  1.00 23.03           C  \\nATOM   1242  OH  TYR A 157       9.572  32.528  -1.089  1.00 26.15           O  \\nATOM   1243  N   THR A 158       8.940  24.198   1.455  1.00 11.35           N  \\nATOM   1244  CA  THR A 158       8.836  22.727   1.522  1.00 10.11           C  \\nATOM   1245  C   THR A 158       7.440  22.347   2.064  1.00 11.11           C  \\nATOM   1246  O   THR A 158       6.811  21.408   1.558  1.00 11.43           O  \\nATOM   1247  CB  THR A 158       9.935  22.129   2.470  1.00 12.09           C  \\nATOM   1248  OG1 THR A 158      11.216  22.451   1.933  1.00 11.36           O  \\nATOM   1249  CG2 THR A 158       9.791  20.657   2.624  1.00 10.87           C  \\nATOM   1250  N   LEU A 159       6.888  23.056   3.072  1.00  9.25           N  \\nATOM   1251  CA  LEU A 159       5.533  22.754   3.506  1.00  9.93           C  \\nATOM   1252  C   LEU A 159       4.489  22.986   2.388  1.00 10.42           C  \\nATOM   1253  O   LEU A 159       3.560  22.185   2.272  1.00 11.22           O  \\nATOM   1254  CB  LEU A 159       5.171  23.608   4.711  1.00  9.03           C  \\nATOM   1255  CG  LEU A 159       3.789  23.318   5.269  1.00 10.96           C  \\nATOM   1256  CD1 LEU A 159       3.635  21.828   5.651  1.00  9.29           C  \\nATOM   1257  CD2 LEU A 159       3.580  24.223   6.469  1.00 13.71           C  \\nATOM   1258  N   VAL A 160       4.596  24.046   1.548  1.00 10.91           N  \\nATOM   1259  CA  VAL A 160       3.662  24.284   0.439  1.00  9.19           C  \\nATOM   1260  C   VAL A 160       3.803  23.137  -0.556  1.00  9.60           C  \\nATOM   1261  O   VAL A 160       2.781  22.638  -1.031  1.00 10.29           O  \\nATOM   1262  CB  VAL A 160       3.981  25.653  -0.211  1.00  8.50           C  \\nATOM   1263  CG1 VAL A 160       3.402  25.754  -1.622  1.00 10.56           C  \\nATOM   1264  CG2 VAL A 160       3.403  26.741   0.659  1.00  7.98           C  \\nATOM   1265  N   ARG A 161       5.033  22.667  -0.839  1.00  9.10           N  \\nATOM   1266  CA  ARG A 161       5.206  21.554  -1.767  1.00  9.82           C  \\nATOM   1267  C   ARG A 161       4.644  20.240  -1.215  1.00 10.44           C  \\nATOM   1268  O   ARG A 161       4.166  19.395  -1.984  1.00 12.88           O  \\nATOM   1269  CB  ARG A 161       6.679  21.398  -2.115  1.00  9.39           C  \\nATOM   1270  CG  ARG A 161       7.146  22.640  -2.816  1.00  9.70           C  \\nATOM   1271  CD  ARG A 161       8.594  22.398  -3.227  1.00 14.12           C  \\nATOM   1272  NE  ARG A 161       9.122  23.635  -3.780  1.00 14.15           N  \\nATOM   1273  CZ  ARG A 161      10.144  24.313  -3.257  1.00 12.45           C  \\nATOM   1274  NH1 ARG A 161      10.823  23.858  -2.204  1.00 13.24           N  \\nATOM   1275  NH2 ARG A 161      10.485  25.475  -3.819  1.00 13.52           N  \\nATOM   1276  N   GLU A 162       4.663  20.046   0.112  1.00 10.71           N  \\nATOM   1277  CA  GLU A 162       4.054  18.877   0.722  1.00 11.80           C  \\nATOM   1278  C   GLU A 162       2.522  18.887   0.568  1.00 11.20           C  \\nATOM   1279  O   GLU A 162       1.921  17.848   0.274  1.00 12.90           O  \\nATOM   1280  CB  GLU A 162       4.430  18.815   2.216  1.00 13.86           C  \\nATOM   1281  CG  GLU A 162       5.870  18.460   2.580  1.00 14.10           C  \\nATOM   1282  CD  GLU A 162       6.268  17.038   2.224  1.00 17.23           C  \\nATOM   1283  OE1 GLU A 162       5.571  16.116   2.627  1.00 19.29           O  \\nATOM   1284  OE2 GLU A 162       7.278  16.827   1.545  1.00 22.14           O  \\nATOM   1285  N   ILE A 163       1.869  20.045   0.731  1.00 10.13           N  \\nATOM   1286  CA  ILE A 163       0.426  20.175   0.542  1.00 10.58           C  \\nATOM   1287  C   ILE A 163       0.128  19.878  -0.933  1.00 13.43           C  \\nATOM   1288  O   ILE A 163      -0.796  19.118  -1.235  1.00 15.01           O  \\nATOM   1289  CB  ILE A 163      -0.015  21.590   0.891  1.00 11.33           C  \\nATOM   1290  CG1 ILE A 163       0.084  21.777   2.400  1.00 11.27           C  \\nATOM   1291  CG2 ILE A 163      -1.486  21.815   0.437  1.00 12.37           C  \\nATOM   1292  CD1 ILE A 163      -0.006  23.245   2.894  1.00 12.56           C  \\nATOM   1293  N   ARG A 164       0.904  20.456  -1.871  1.00 11.72           N  \\nATOM   1294  CA  ARG A 164       0.770  20.163  -3.283  1.00 13.64           C  \\nATOM   1295  C   ARG A 164       0.813  18.657  -3.626  1.00 17.97           C  \\nATOM   1296  O   ARG A 164       0.150  18.197  -4.559  1.00 18.12           O  \\nATOM   1297  CB  ARG A 164       1.897  20.887  -4.029  1.00 12.51           C  \\nATOM   1298  CG  ARG A 164       1.647  22.371  -4.120  1.00 12.49           C  \\nATOM   1299  CD  ARG A 164       2.801  22.888  -4.950  1.00 14.72           C  \\nATOM   1300  NE  ARG A 164       2.730  24.327  -5.113  1.00 12.82           N  \\nATOM   1301  CZ  ARG A 164       3.800  25.071  -5.405  1.00 13.29           C  \\nATOM   1302  NH1 ARG A 164       5.018  24.539  -5.527  1.00 11.56           N  \\nATOM   1303  NH2 ARG A 164       3.595  26.381  -5.572  1.00 12.55           N  \\nATOM   1304  N   GLN A 165       1.590  17.831  -2.935  1.00 20.23           N  \\nATOM   1305  CA  GLN A 165       1.673  16.435  -3.337  1.00 27.50           C  \\nATOM   1306  C   GLN A 165       0.782  15.499  -2.502  1.00 29.12           C  \\nATOM   1307  O   GLN A 165       0.826  14.274  -2.655  1.00 29.30           O  \\nATOM   1308  CB  GLN A 165       3.168  16.070  -3.286  1.00 30.34           C  \\nATOM   1309  CG  GLN A 165       4.018  16.904  -4.309  1.00 35.17           C  \\nATOM   1310  CD  GLN A 165       4.109  16.470  -5.796  1.00 37.71           C  \\nATOM   1311  OE1 GLN A 165       4.256  17.293  -6.719  1.00 36.70           O  \\nATOM   1312  NE2 GLN A 165       4.079  15.166  -6.094  1.00 38.13           N  \\nATOM   1313  N   HIS A 166      -0.068  16.075  -1.638  1.00 29.28           N  \\nATOM   1314  CA  HIS A 166      -0.982  15.296  -0.820  1.00 31.13           C  \\nATOM   1315  C   HIS A 166      -2.214  14.971  -1.665  1.00 33.30           C  \\nATOM   1316  O   HIS A 166      -2.630  15.786  -2.497  1.00 34.96           O  \\nATOM   1317  CB  HIS A 166      -1.420  16.090   0.428  1.00 30.89           C  \\nATOM   1318  CG  HIS A 166      -2.247  15.319   1.475  1.00 32.17           C  \\nATOM   1319  ND1 HIS A 166      -3.579  15.231   1.603  1.00 33.52           N  \\nATOM   1320  CD2 HIS A 166      -1.712  14.560   2.502  1.00 32.95           C  \\nATOM   1321  CE1 HIS A 166      -3.857  14.464   2.647  1.00 33.07           C  \\nATOM   1322  NE2 HIS A 166      -2.727  14.064   3.184  1.00 33.06           N  \\nATOM   1323  OXT HIS A 166      -2.763  13.888  -1.454  1.00 36.27           O  \\nTER    1324      HIS A 166                                                      \\nHETATM 1325 MG    MG A 168       4.771  33.860  19.213  1.00  9.42          MG  \\nHETATM 1326  PG  GNP A 167       5.113  32.223  21.994  1.00  9.66           P  \\nHETATM 1327  O1G GNP A 167       4.844  32.596  23.390  1.00  9.26           O  \\nHETATM 1328  O2G GNP A 167       4.203  32.910  21.112  1.00 11.20           O  \\nHETATM 1329  O3G GNP A 167       4.640  30.769  21.802  1.00 10.09           O  \\nHETATM 1330  N3B GNP A 167       6.701  32.303  21.621  1.00  8.18           N  \\nHETATM 1331  PB  GNP A 167       7.310  32.034  20.111  1.00  8.77           P  \\nHETATM 1332  O1B GNP A 167       7.468  30.605  19.791  1.00  8.35           O  \\nHETATM 1333  O2B GNP A 167       6.696  32.775  19.021  1.00  9.38           O  \\nHETATM 1334  O3A GNP A 167       8.792  32.643  20.152  1.00 10.15           O  \\nHETATM 1335  PA  GNP A 167       9.280  34.140  19.710  1.00 10.18           P  \\nHETATM 1336  O1A GNP A 167       9.263  34.268  18.220  1.00 10.07           O  \\nHETATM 1337  O2A GNP A 167       8.594  35.173  20.510  1.00  9.33           O  \\nHETATM 1338  O5' GNP A 167      10.826  34.056  20.132  1.00 11.46           O  \\nHETATM 1339  C5' GNP A 167      11.170  33.911  21.535  1.00 10.56           C  \\nHETATM 1340  C4' GNP A 167      12.587  34.445  21.811  1.00 11.96           C  \\nHETATM 1341  O4' GNP A 167      13.560  33.649  21.095  1.00 11.66           O  \\nHETATM 1342  C3' GNP A 167      12.813  35.910  21.359  1.00 12.06           C  \\nHETATM 1343  O3' GNP A 167      13.793  36.538  22.221  1.00 14.52           O  \\nHETATM 1344  C2' GNP A 167      13.325  35.679  19.947  1.00 11.90           C  \\nHETATM 1345  O2' GNP A 167      14.151  36.750  19.499  1.00 14.99           O  \\nHETATM 1346  C1' GNP A 167      14.242  34.478  20.156  1.00 12.90           C  \\nHETATM 1347  N9  GNP A 167      14.454  33.739  18.896  1.00 12.98           N  \\nHETATM 1348  C8  GNP A 167      13.518  33.262  17.993  1.00 11.76           C  \\nHETATM 1349  N7  GNP A 167      14.016  32.636  16.979  1.00 12.34           N  \\nHETATM 1350  C5  GNP A 167      15.413  32.694  17.200  1.00 12.36           C  \\nHETATM 1351  C6  GNP A 167      16.521  32.219  16.402  1.00 12.30           C  \\nHETATM 1352  O6  GNP A 167      16.507  31.591  15.338  1.00 12.16           O  \\nHETATM 1353  N1  GNP A 167      17.735  32.503  16.995  1.00 11.60           N  \\nHETATM 1354  C2  GNP A 167      17.909  33.082  18.236  1.00 13.91           C  \\nHETATM 1355  N2  GNP A 167      19.173  33.280  18.658  1.00 14.60           N  \\nHETATM 1356  N3  GNP A 167      16.884  33.498  18.996  1.00 13.01           N  \\nHETATM 1357  C4  GNP A 167      15.675  33.331  18.398  1.00 14.36           C  \\nHETATM 1358  O   HOH A 169      13.588  43.386  17.779  1.00 31.80           O  \\nHETATM 1359  O   HOH A 170       8.248  36.064  23.209  1.00 13.63           O  \\nHETATM 1360  O   HOH A 171       3.254  37.845  19.454  1.00 18.29           O  \\nHETATM 1361  O   HOH A 172       5.945  35.619  20.105  1.00 11.33           O  \\nHETATM 1362  O   HOH A 173       3.669  32.226  18.172  1.00  8.22           O  \\nHETATM 1363  O   HOH A 174       2.896  37.170  15.500  1.00 21.37           O  \\nHETATM 1364  O   HOH A 175       2.102  32.309  24.122  1.00 27.47           O  \\nHETATM 1365  O   HOH A 176      -1.425  37.650  19.667  1.00 24.98           O  \\nHETATM 1366  O   HOH A 177      -3.225  30.031  -8.217  1.00 23.57           O  \\nHETATM 1367  O   HOH A 178      16.678  26.321  26.916  1.00 36.70           O  \\nHETATM 1368  O   HOH A 179      -2.134  40.975   1.551  1.00 81.53           O  \\nHETATM 1369  O   HOH A 180       0.400  40.675  -0.629  1.00 24.66           O  \\nHETATM 1370  O   HOH A 181      10.944  44.793  16.009  1.00 48.47           O  \\nHETATM 1371  O   HOH A 182      17.207  42.570  16.786  1.00 43.23           O  \\nHETATM 1372  O   HOH A 183      14.999  45.167  19.922  1.00 39.52           O  \\nHETATM 1373  O   HOH A 184      11.598  37.755  13.202  1.00 11.72           O  \\nHETATM 1374  O   HOH A 185       6.958  38.922  18.146  1.00 26.17           O  \\nHETATM 1375  O   HOH A 186      14.078  32.033  27.772  1.00 57.73           O  \\nHETATM 1376  O   HOH A 187       9.126  37.906  19.595  1.00 32.42           O  \\nHETATM 1377  O   HOH A 188      -0.049  36.435  27.259  1.00 50.56           O  \\nHETATM 1378  O   HOH A 189       3.843  30.503  25.485  1.00 46.80           O  \\nHETATM 1379  O   HOH A 190       4.931  28.343  27.587  1.00 29.32           O  \\nHETATM 1380  O   HOH A 191       4.881  25.774  26.247  1.00 28.55           O  \\nHETATM 1381  O   HOH A 193       8.729  26.516  26.645  1.00 37.34           O  \\nHETATM 1382  O   HOH A 194      -5.515  23.731  -4.213  1.00 43.74           O  \\nHETATM 1383  O   HOH A 195      -7.006  26.635   0.729  1.00 25.77           O  \\nHETATM 1384  O   HOH A 196      -2.033  23.281  -4.754  1.00 36.89           O  \\nHETATM 1385  O   HOH A 197      -4.326  24.642  -0.854  1.00 28.59           O  \\nHETATM 1386  O   HOH A 198      -6.858  26.995  -2.808  1.00 28.83           O  \\nHETATM 1387  O   HOH A 200      -3.937  29.225   8.289  1.00 24.08           O  \\nHETATM 1388  O   HOH A 201      -8.934  24.037   2.795  1.00 34.16           O  \\nHETATM 1389  O   HOH A 202       1.444  26.302  19.318  1.00 12.37           O  \\nHETATM 1390  O   HOH A 203       2.031  24.632  27.307  1.00 50.09           O  \\nHETATM 1391  O   HOH A 204      15.125  28.341  23.174  1.00 13.88           O  \\nHETATM 1392  O   HOH A 205      12.808  31.541  25.056  1.00 46.88           O  \\nHETATM 1393  O   HOH A 206       8.329  34.933   0.543  1.00 53.21           O  \\nHETATM 1394  O   HOH A 207       0.760  42.969   2.336  1.00 35.42           O  \\nHETATM 1395  O   HOH A 208       1.270  40.298   7.071  1.00 76.61           O  \\nHETATM 1396  O   HOH A 209      -3.135  39.625   7.649  1.00 35.40           O  \\nHETATM 1397  O   HOH A 210       8.699  39.076   2.399  1.00 33.93           O  \\nHETATM 1398  O   HOH A 212      10.040  41.806   3.447  1.00 37.88           O  \\nHETATM 1399  O   HOH A 213      18.914  29.095   3.704  1.00 25.32           O  \\nHETATM 1400  O   HOH A 214      15.966  29.098   2.957  1.00 18.63           O  \\nHETATM 1401  O   HOH A 215      15.506  31.962   4.015  1.00 18.93           O  \\nHETATM 1402  O   HOH A 216      22.935  30.533  10.845  1.00 39.81           O  \\nHETATM 1403  O   HOH A 217       6.476  43.893  23.325  1.00 49.09           O  \\nHETATM 1404  O   HOH A 218      11.746  41.248  28.612  1.00 36.22           O  \\nHETATM 1405  O   HOH A 219      11.480  42.270  25.715  1.00 29.66           O  \\nHETATM 1406  O   HOH A 220       9.344  34.251  -8.278  1.00 36.61           O  \\nHETATM 1407  O   HOH A 221      -0.111  19.667  -8.066  1.00 44.76           O  \\nHETATM 1408  O   HOH A 222      -0.528  30.460 -10.943  1.00 24.29           O  \\nHETATM 1409  O   HOH A 223     -11.352  24.274  19.491  1.00 31.21           O  \\nHETATM 1410  O   HOH A 224      -4.293  30.642 -16.466  1.00127.55           O  \\nHETATM 1411  O   HOH A 225       3.466  29.729 -16.232  1.00 26.15           O  \\nHETATM 1412  O   HOH A 227      -4.288  18.610   4.198  1.00 34.77           O  \\nHETATM 1413  O   HOH A 228       4.626  16.154   5.151  1.00 14.96           O  \\nHETATM 1414  O   HOH A 229       5.681  11.069   2.580  1.00 59.76           O  \\nHETATM 1415  O   HOH A 230       1.173  14.021   3.406  1.00 32.15           O  \\nHETATM 1416  O   HOH A 231       7.935  22.484  20.504  1.00 10.86           O  \\nHETATM 1417  O   HOH A 232       3.071  10.004  14.546  1.00 23.48           O  \\nHETATM 1418  O   HOH A 233       2.437  13.381   8.401  1.00 17.75           O  \\nHETATM 1419  O   HOH A 234       4.779   7.500   6.154  1.00 71.09           O  \\nHETATM 1420  O   HOH A 235      -4.250   8.748   6.880  1.00 24.76           O  \\nHETATM 1421  O   HOH A 236     -13.381  22.830  17.669  1.00 39.09           O  \\nHETATM 1422  O   HOH A 237      16.969  39.052   8.276  1.00 50.25           O  \\nHETATM 1423  O   HOH A 238      -6.315   9.333  13.897  1.00 36.24           O  \\nHETATM 1424  O   HOH A 240      -0.160   4.395  12.468  1.00 38.79           O  \\nHETATM 1425  O   HOH A 241       1.295   7.931  12.335  1.00 49.03           O  \\nHETATM 1426  O   HOH A 242       4.067  11.551   6.968  1.00 46.97           O  \\nHETATM 1427  O   HOH A 243      -5.060   6.579  -0.216  1.00 50.14           O  \\nHETATM 1428  O   HOH A 244      18.164   7.151   5.573  1.00 71.67           O  \\nHETATM 1429  O   HOH A 245       2.068   8.904   7.636  1.00 45.86           O  \\nHETATM 1430  O   HOH A 246      24.661  23.134  12.757  1.00 34.69           O  \\nHETATM 1431  O   HOH A 247      21.051  23.714   6.239  1.00 24.64           O  \\nHETATM 1432  O   HOH A 248      25.855  31.137  20.712  1.00 34.53           O  \\nHETATM 1433  O   HOH A 249      24.868  30.890  26.223  1.00 48.47           O  \\nHETATM 1434  O   HOH A 250      26.967  20.447  19.448  1.00 47.77           O  \\nHETATM 1435  O   HOH A 252      28.453  14.196  14.602  1.00 39.78           O  \\nHETATM 1436  O   HOH A 253      19.639  16.256   9.760  1.00 37.54           O  \\nHETATM 1437  O   HOH A 254      16.629   7.383  10.526  1.00 60.90           O  \\nHETATM 1438  O   HOH A 255       4.972   7.888  14.276  1.00 54.95           O  \\nHETATM 1439  O   HOH A 256      14.184  14.684   5.643  1.00 31.40           O  \\nHETATM 1440  O   HOH A 257      11.766  17.254   1.278  1.00 41.82           O  \\nHETATM 1441  O   HOH A 258      16.274  16.365   6.342  1.00 17.29           O  \\nHETATM 1442  O   HOH A 259      18.346  24.095   6.620  1.00 17.36           O  \\nHETATM 1443  O   HOH A 260      22.479  35.073  19.694  1.00 43.96           O  \\nHETATM 1444  O   HOH A 261      20.580  31.747   4.906  1.00 59.46           O  \\nHETATM 1445  O   HOH A 262      18.196  25.514   4.261  1.00 28.75           O  \\nHETATM 1446  O   HOH A 263      12.857  20.183   1.149  1.00 21.79           O  \\nHETATM 1447  O   HOH A 264      19.514  15.912  24.943  1.00 55.52           O  \\nHETATM 1448  O   HOH A 265       8.925   7.579   5.621  1.00 77.94           O  \\nHETATM 1449  O   HOH A 266      17.622  21.835  26.964  1.00 27.48           O  \\nHETATM 1450  O   HOH A 267      24.258  19.786  14.155  1.00 20.48           O  \\nHETATM 1451  O   HOH A 268      23.806  15.180  15.136  1.00 19.86           O  \\nHETATM 1452  O   HOH A 269      21.478  14.557  11.134  1.00 29.66           O  \\nHETATM 1453  O   HOH A 270      17.293  45.621  24.753  1.00 39.18           O  \\nHETATM 1454  O   HOH A 271      -3.518  40.206  17.649  1.00 44.75           O  \\nHETATM 1455  O   HOH A 272       2.143  19.852  24.090  1.00 40.10           O  \\nHETATM 1456  O   HOH A 273       7.101  35.918  -4.704  1.00 31.76           O  \\nHETATM 1457  O   HOH A 274       5.462  38.756  -5.521  1.00 27.51           O  \\nHETATM 1458  O   HOH A 275       5.748  20.565  28.638  1.00 58.06           O  \\nHETATM 1459  O   HOH A 276       8.814  31.258  -8.964  1.00 23.79           O  \\nHETATM 1460  O   HOH A 277     -10.427  35.895   4.301  1.00 58.91           O  \\nHETATM 1461  O   HOH A 278      -8.713  28.579   2.333  1.00 69.36           O  \\nHETATM 1462  O   HOH A 279      24.472  24.237  17.316  1.00 32.14           O  \\nHETATM 1463  O   HOH A 281      16.968  32.701  21.815  1.00 23.13           O  \\nHETATM 1464  O   HOH A 282      19.541  34.346  21.581  1.00 53.25           O  \\nHETATM 1465  O   HOH A 283      24.847  23.384  26.621  1.00 61.18           O  \\nHETATM 1466  O   HOH A 284       1.409  11.113   2.873  1.00 44.87           O  \\nHETATM 1467  O   HOH A 285       1.556  11.488  -0.914  1.00 37.86           O  \\nHETATM 1468  O   HOH A 286       9.758  12.945   3.889  1.00 36.47           O  \\nHETATM 1469  O   HOH A 287       3.827  13.375   4.622  1.00 44.90           O  \\nHETATM 1470  O   HOH A 288      14.279   6.175  15.391  1.00 78.25           O  \\nHETATM 1471  O   HOH A 289       7.573  13.968   2.191  1.00 76.47           O  \\nHETATM 1472  O   HOH A 290       2.323  15.482   1.418  1.00 35.15           O  \\nHETATM 1473  O   HOH A 291      11.997  14.573   2.962  1.00 49.86           O  \\nHETATM 1474  O   HOH A 292      23.798  33.550  17.880  1.00 24.64           O  \\nHETATM 1475  O   HOH A 293      14.657  36.046  -0.571  1.00 51.86           O  \\nHETATM 1476  O   HOH A 294      24.456  25.366   4.004  1.00 53.23           O  \\nHETATM 1477  O   HOH A 296      -0.603  12.134  -4.185  1.00 52.38           O  \\nHETATM 1478  O   HOH A 297      -8.372  34.415  -4.471  1.00 74.53           O  \\nHETATM 1479  O   HOH A 300     -14.033  36.501   0.062  1.00 50.21           O  \\nHETATM 1480  O   HOH A 301     -17.102  35.735  -2.806  1.00 42.58           O  \\nHETATM 1481  O   HOH A 303     -17.383  40.542   1.431  1.00 77.16           O  \\nHETATM 1482  O   HOH A 304     -19.024  41.557  -3.411  1.00107.16           O  \\nHETATM 1483  O   HOH A 305      -9.876  37.804   0.839  1.00 90.65           O  \\nHETATM 1484  O   HOH A 306     -14.053  36.466   4.552  1.00 82.05           O  \\nHETATM 1485  O   HOH A 307     -14.183  39.558   0.339  1.00 58.66           O  \\nHETATM 1486  O   HOH A 309     -12.547  28.677   8.301  1.00 74.88           O  \\nHETATM 1487  O   HOH A 311     -14.787  32.746  -2.171  1.00 40.79           O  \\nHETATM 1488  O   HOH A 312      -6.916  30.740   3.095  1.00 36.53           O  \\nHETATM 1489  O   HOH A 315      -8.312  33.027   7.340  1.00 83.28           O  \\nHETATM 1490  O   HOH A 316      -6.280  35.646   7.624  1.00 75.97           O  \\nHETATM 1491  O   HOH A 317      -5.230  31.565   9.609  1.00 55.67           O  \\nHETATM 1492  O   HOH A 318       6.747  37.258   1.208  1.00 57.93           O  \\nHETATM 1493  O   HOH A 319       4.747  42.419  12.683  1.00 57.74           O  \\nHETATM 1494  O   HOH A 320       2.984  41.950  16.191  1.00 41.69           O  \\nHETATM 1495  O   HOH A 321       3.867  49.208  12.056  1.00 24.57           O  \\nHETATM 1496  O   HOH A 322       7.530  42.894  17.776  1.00 79.28           O  \\nHETATM 1497  O   HOH A 324      19.822  39.457  17.826  1.00 66.22           O  \\nHETATM 1498  O   HOH A 325      12.391  45.557  24.973  1.00 48.70           O  \\nHETATM 1499  O   HOH A 326      14.944  43.928  23.794  1.00 62.45           O  \\nHETATM 1500  O   HOH A 327     -12.984  24.008  12.563  1.00 40.31           O  \\nHETATM 1501  O   HOH A 330       0.572  41.133  19.277  1.00 50.77           O  \\nHETATM 1502  O   HOH A 331      -6.196  37.979  23.043  1.00 48.93           O  \\nHETATM 1503  O   HOH A 332      -3.526  30.410  19.065  1.00 21.26           O  \\nHETATM 1504  O   HOH A 333      -7.995  32.093  21.685  1.00 54.11           O  \\nHETATM 1505  O   HOH A 334      -8.483  28.891  20.063  1.00 70.37           O  \\nHETATM 1506  O   HOH A 335     -11.670  31.022  16.981  1.00 76.86           O  \\nHETATM 1507  O   HOH A 336     -11.082  34.450  18.910  1.00 43.07           O  \\nHETATM 1508  O   HOH A 339      -6.521  37.738  15.165  1.00 64.11           O  \\nHETATM 1509  O   HOH A 340     -10.829  35.877  15.629  1.00 58.36           O  \\nHETATM 1510  O   HOH A 341      -6.552  41.166  12.231  1.00 55.67           O  \\nHETATM 1511  O   HOH A 342      -1.242  39.671  12.185  1.00 61.04           O  \\nHETATM 1512  O   HOH A 343      -4.281  38.934  13.464  1.00 32.14           O  \\nHETATM 1513  O   HOH A 344       0.386  41.208  14.039  1.00 40.80           O  \\nHETATM 1514  O   HOH A 345       0.777  39.663  21.603  1.00 67.71           O  \\nHETATM 1515  O   HOH A 346      -7.879  34.848  19.044  1.00 52.60           O  \\nHETATM 1516  O   HOH A 347       1.121  32.594 -11.957  1.00 32.30           O  \\nHETATM 1517  O   HOH A 348       0.924  32.597 -14.716  1.00 48.34           O  \\nHETATM 1518  O   HOH A 349      -2.241  27.348  20.565  1.00 26.54           O  \\nHETATM 1519  O   HOH A 350       3.259  26.214  29.881  1.00 69.22           O  \\nHETATM 1520  O   HOH A 351      -8.318  34.095  28.392  1.00 81.96           O  \\nHETATM 1521  O   HOH A 352      -2.927  29.775  31.412  1.00 80.92           O  \\nHETATM 1522  O   HOH A 353     -10.410  32.034  25.972  1.00 60.17           O  \\nHETATM 1523  O   HOH A 354      -8.122  30.393  25.378  1.00 62.33           O  \\nHETATM 1524  O   HOH A 356      -6.138  18.882  24.644  1.00 49.26           O  \\nHETATM 1525  O   HOH A 358      -1.406  19.033  23.084  1.00 82.05           O  \\nHETATM 1526  O   HOH A 359       4.169   7.339  10.280  1.00 66.03           O  \\nHETATM 1527  O   HOH A 360      -9.761  30.165   7.702  1.00 46.15           O  \\nHETATM 1528  O   HOH A 361     -16.002  27.801  11.700  1.00 41.85           O  \\nHETATM 1529  O   HOH A 362      16.215  30.489  29.264  1.00 67.21           O  \\nHETATM 1530  O   HOH A 364      21.417  27.685  26.140  1.00 49.57           O  \\nHETATM 1531  O   HOH A 366      15.824  34.767  30.662  1.00 25.90           O  \\nHETATM 1532  O   HOH A 367      18.527  35.965  24.111  1.00 68.69           O  \\nHETATM 1533  O   HOH A 369      12.103  35.275  24.945  1.00 51.37           O  \\nHETATM 1534  O   HOH A 370      24.782  11.970  23.641  1.00 55.92           O  \\nHETATM 1535  O   HOH A 371      14.337  21.394  28.627  1.00 30.11           O  \\nHETATM 1536  O   HOH A 372      -1.367  12.600  23.053  1.00 44.43           O  \\nHETATM 1537  O   HOH A 373       7.060   8.567  21.416  1.00 47.99           O  \\nHETATM 1538  O   HOH A 374       7.639   8.040  16.420  1.00 38.76           O  \\nHETATM 1539  O   HOH A 376      -1.282  15.566  25.487  1.00 67.79           O  \\nHETATM 1540  O   HOH A 378      17.945  30.924  23.806  1.00 31.50           O  \\nHETATM 1541  O   HOH A 379      17.360  33.209  33.337  1.00 30.46           O  \\nHETATM 1542  O   HOH A 380       3.971   6.567  22.089  1.00 86.80           O  \\nHETATM 1543  O   HOH A 381      -0.065  10.022  26.928  1.00 61.86           O  \\nHETATM 1544  O   HOH A 382      -4.450  12.966  21.222  1.00 42.82           O  \\nHETATM 1545  O   HOH A 383      -2.584   6.719   5.938  1.00 40.64           O  \\nHETATM 1546  O   HOH A 384      25.567  31.936  12.845  1.00 73.17           O  \\nHETATM 1547  O   HOH A 385      24.446  29.383  22.970  1.00 36.61           O  \\nHETATM 1548  O   HOH A 386      24.865  38.167  18.227  1.00 60.74           O  \\nHETATM 1549  O   HOH A 387      23.135  35.012  14.543  1.00 59.19           O  \\nHETATM 1550  O   HOH A 388      19.325   9.556  18.093  1.00 92.18           O  \\nHETATM 1551  O   HOH A 389      25.326  12.873  16.251  1.00 49.27           O  \\nHETATM 1552  O   HOH A 390      20.219  10.405  10.956  1.00 63.05           O  \\nHETATM 1553  O   HOH A 392       9.452   5.453   8.249  1.00 47.81           O  \\nHETATM 1554  O   HOH A 394      10.139  12.252   0.254  1.00 29.37           O  \\nHETATM 1555  O   HOH A 395      28.004  37.416  13.466  1.00 52.29           O  \\nHETATM 1556  O   HOH A 396      24.283  36.829  12.307  1.00 64.74           O  \\nHETATM 1557  O   HOH A 397      13.748  34.031  -3.019  1.00 47.80           O  \\nHETATM 1558  O   HOH A 398      16.981  34.189   2.006  1.00 80.51           O  \\nHETATM 1559  O   HOH A 399      -4.426  24.223  -7.240  1.00 38.09           O  \\nHETATM 1560  O   HOH A 400      19.676  33.981   3.336  1.00 64.86           O  \\nHETATM 1561  O   HOH A 401      14.633  28.745  -0.317  1.00 66.25           O  \\nHETATM 1562  O   HOH A 402      -2.139  15.576  22.716  1.00 65.01           O  \\nHETATM 1563  O   HOH A 403       9.873  11.090  20.760  1.00 53.88           O  \\nHETATM 1564  O   HOH A 404      18.291  36.379  26.927  1.00 57.23           O  \\nHETATM 1565  O   HOH A 405       0.138   6.145  15.269  1.00 82.80           O  \\nHETATM 1566  O   HOH A 406      -5.515  38.166  18.689  1.00 48.28           O  \\nHETATM 1567  O   HOH A 407      -0.399   4.688   8.809  1.00 58.35           O  \\nHETATM 1568  O   HOH A 408       0.813  25.584  22.307  1.00 40.04           O  \\nCONECT  117 1325                                                                \\nCONECT  265 1325                                                                \\nCONECT 1325  117  265 1328 1333                                                 \\nCONECT 1325 1361 1362                                                           \\nCONECT 1326 1327 1328 1329 1330                                                 \\nCONECT 1327 1326                                                                \\nCONECT 1328 1325 1326                                                           \\nCONECT 1329 1326                                                                \\nCONECT 1330 1326 1331                                                           \\nCONECT 1331 1330 1332 1333 1334                                                 \\nCONECT 1332 1331                                                                \\nCONECT 1333 1325 1331                                                           \\nCONECT 1334 1331 1335                                                           \\nCONECT 1335 1334 1336 1337 1338                                                 \\nCONECT 1336 1335                                                                \\nCONECT 1337 1335                                                                \\nCONECT 1338 1335 1339                                                           \\nCONECT 1339 1338 1340                                                           \\nCONECT 1340 1339 1341 1342                                                      \\nCONECT 1341 1340 1346                                                           \\nCONECT 1342 1340 1343 1344                                                      \\nCONECT 1343 1342                                                                \\nCONECT 1344 1342 1345 1346                                                      \\nCONECT 1345 1344                                                                \\nCONECT 1346 1341 1344 1347                                                      \\nCONECT 1347 1346 1348 1357                                                      \\nCONECT 1348 1347 1349                                                           \\nCONECT 1349 1348 1350                                                           \\nCONECT 1350 1349 1351 1357                                                      \\nCONECT 1351 1350 1352 1353                                                      \\nCONECT 1352 1351                                                                \\nCONECT 1353 1351 1354                                                           \\nCONECT 1354 1353 1355 1356                                                      \\nCONECT 1355 1354                                                                \\nCONECT 1356 1354 1357                                                           \\nCONECT 1357 1347 1350 1356                                                      \\nCONECT 1361 1325                                                                \\nCONECT 1362 1325                                                                \\nMASTER      312    0    2    5    6    0    9    6 1567    1   38   13          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewergrid_16835771918631198[1][0].addModel(\"\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n   -9.4114   12.4715    7.2804 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4411   12.3024    6.2602 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7949   13.4149    5.7632 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8582   13.2305    4.7753 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5557   11.9767    4.2780 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.7162   11.5413    3.3522 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.7686   10.2482    3.2133 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6926    9.8146    4.0964 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.1966   10.8621    4.7693 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1359   11.0475    5.7603 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.6142   13.3721    7.6225 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.8823   11.6882    7.6466 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0255    9.4196    2.3278 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.3159    8.4875    2.2011 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8700    9.8820    1.6098 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.8639    8.7148    1.5301 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.6979    7.8986    2.5605 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8107    6.8678    2.4813 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.1439    7.0309    0.8493 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.1103    8.4783    0.3910 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.21977\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n   -5.9911    6.7304   12.3325 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9191    6.9212   11.2774 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.2647    6.7642   11.5359 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.1479    6.9534   10.5007 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.7239    7.2913    9.2292 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.3552    7.5235    8.0893 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.5317    7.8152    7.1245 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2947    7.7714    7.6628 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3827    7.4498    8.9640 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.4972    7.2600   10.0026 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3110    6.4898   13.2320 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0269    6.8384   12.1656 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.8272    8.1235    5.7679 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.8559    7.3880    5.1141 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.0857    9.4602    5.3088 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7450   10.0707    4.8498 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4635   11.3409    5.1007 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2760   11.8718    4.6959 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4727   10.5094    3.9001 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7969    9.3211    4.1710 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.13625\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n   -9.2752   13.0914    6.8146 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4983   12.1348    6.1129 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.8716   10.8079    6.1603 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1063    9.8965    5.4740 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9907   10.2745    4.7505 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1083    9.6096    4.0219 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.2035   10.3945    3.5126 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.5111   11.6404    3.9310 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6124   11.5968    4.6987 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3803   12.5106    5.3872 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.0666   12.8041    7.3249 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.0231   14.0428    6.7916 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.1003   10.0556    2.6814 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.5163    9.3118    2.9553 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.7997   10.7501    1.4601 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9148    9.8273    0.5962 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.1986    9.6277   -0.6825 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.4103    8.8148   -1.4399 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.1690    8.2752   -0.2986 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8029    9.1854    1.1187 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.05286\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n    3.8277   12.8535    0.7057 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8945   13.2451   -0.1425 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0823   14.5865   -0.4024 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1212   14.9457   -1.2265 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9663   14.0083   -1.7903 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0127   14.0920   -2.5962 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.5106   12.9221   -2.8747 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.7484   12.0247   -2.2149 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7848   12.6680   -1.5353 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7427   12.3081   -0.7086 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2355   13.5350    1.0984 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6791   11.9003    0.9018 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.6260   12.6053   -3.6985 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.1601   11.8076   -3.4803 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.0081   13.3972   -4.8348 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.9410   13.1976   -5.9313 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.7807   12.0148   -6.5063 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8337   11.8451   -7.4709 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.0965   13.4495   -7.6023 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.1191   14.2339   -6.3461 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-3.03419\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n    3.8796   12.8280    0.5341 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6839   13.3276   -0.5216 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3512   14.5215   -0.3442 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1254   14.9888   -1.3785 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2471   14.3003   -2.5709 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9120   14.5294   -3.6921 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7287   13.5869   -4.5709 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8988   12.6931   -3.9929 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5842   13.1083   -2.7548 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8078   12.6399   -1.7171 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8044   13.3319    1.3765 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.3963   11.9773    0.4245 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2696   13.4781   -5.8818 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.9442   14.1354   -6.1681 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.8756   12.4518   -6.8069 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5310   13.1310   -8.1489 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.8433   14.4013   -8.3593 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5395   14.9931   -9.5482 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7557   13.6934  -10.4598 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8944   12.4447   -9.1711 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-2.95533\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n    3.7671   12.7959    0.6769 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8542   13.1811   -0.1483 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0647   14.5222   -0.3915 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1230   14.8752   -1.1933 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9653   13.9318   -1.7512 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0271   14.0091   -2.5375 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.5142   12.8353   -2.8186 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.7282   11.9420   -2.1814 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7614   12.5918   -1.5127 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.6998   12.2382   -0.7083 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1773   13.4816    1.0657 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6023   11.8429    0.8609 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.6398   12.5112   -3.6254 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.7621   12.9910   -4.4764 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.6064   11.5151   -3.2543 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   12.0024   12.1710   -3.2972 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   12.2320   13.3064   -2.6541 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   13.4683   13.8776   -2.6932 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   14.4000   12.7594   -3.7013 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   13.0544   11.6015   -3.9974 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-2.89640\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n    4.0202   13.0896    0.3514 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8424   13.4745   -0.7380 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5672   14.6443   -0.6461 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3577   15.0000   -1.7119 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4401   14.2246   -2.8532 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1096   14.3421   -3.9888 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.8767   13.3503   -4.7990 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.0078   12.5403   -4.1582 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7197   13.0560   -2.9519 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9270   12.6995   -1.8826 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.9736   13.6543    1.1565 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4961   12.2577    0.3027 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4052   13.1239   -6.0999 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.3793   13.1849   -6.2289 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5811   12.8100   -7.2343 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7451   11.3058   -7.5368 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7932   10.6405   -8.1745 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9468    9.3134   -8.4417 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.5376    8.9435   -7.7581 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8812   10.6062   -7.1611 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-2.74515\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n   12.9015    6.6939    2.9198 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.8323    7.6012    2.7084 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.6940    8.6789    3.5579 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.6520    9.5464    3.3367 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.7560    9.3652    2.2998 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.6955   10.0385    1.8832 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.1434    9.4863    0.8419 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.8831    8.3924    0.5625 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.8880    8.2921    1.4481 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.9333    7.4222    1.6704 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   13.5302    6.8387    3.6635 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   13.0151    5.9193    2.3229 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9971    9.9109    0.1148 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1080    9.7643    0.5112 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.0786   10.5444   -1.1722 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4416   11.9446   -1.0518 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1787   12.0786   -0.6741 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6243   13.3184   -0.5662 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9515   14.3975   -1.0230 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1593   13.0982   -1.3267 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-2.73742\\n\\n$$$$\\n\\n\\n\\n 20 22  0  0  0  0  0  0  0  0999 V2000\\n   -8.9631   12.4957    7.1107 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1675   11.9876    6.0525 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.5706   12.8747    5.1814 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8038   12.3631    4.1627 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6216   11.0031    3.9957 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.9490   10.2743    3.1192 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0633    8.9990    3.3532 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8534    8.8880    4.4418 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2141   10.1120    4.8612 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.9830   10.6256    5.8828 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.0820   13.4674    7.2144 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.3965   11.8766    7.7417 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4922    7.9105    2.6376 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6511    7.8503    1.6679 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.6940    6.8948    3.2666 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6150    5.6984    3.5848 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5858    5.3565    2.7504 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3924    4.2965    3.0362 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7654    3.7159    4.5868 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4596    4.9452    4.7380 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  1  2  1  0  0  0\\n  2  3  2  0  0  0\\n  3  4  1  0  0  0\\n  4  5  2  0  0  0\\n  5  6  1  0  0  0\\n  6  7  1  0  0  0\\n  7 13  1  0  0  0\\n 13 15  1  0  0  0\\n 15 16  1  0  0  0\\n 16 17  1  0  0  0\\n 17 18  2  0  0  0\\n 18 19  1  0  0  0\\n 19 20  1  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  2  0  0  0\\n 10  2  1  0  0  0\\n  9  5  1  0  0  0\\n 20 16  2  0  0  0\\n  1 11  1  0  0  0\\n  1 12  1  0  0  0\\n 13 14  1  0  0  0\\nM  END\\n> <minimizedAffinity>\\n-2.73579\\n\\n$$$$\\n\",\"sdf\");\n",
       "\tviewergrid_16835771918631198[1][0].setStyle({\"model\": 0},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewergrid_16835771918631198[1][0].addSurface(1,{\"opacity\": 0.5, \"color\": \"white\"},{\"model\": 0});\n",
       "\tviewergrid_16835771918631198[1][0].setStyle({\"model\": 1},{\"stick\": {}});\n",
       "\tviewergrid_16835771918631198[1][0].zoomTo();\n",
       "\tviewergrid_16835771918631198[1][1].addModel(\"HEADER    ONCOGENE PROTEIN                        30-APR-90   5P21              \\nTITLE     REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION OF H-RAS   \\nTITLE    2 P21 AT 1.35 ANGSTROMS RESOLUTION: IMPLICATIONS FOR THE MECHANISM OF  \\nTITLE    3 GTP HYDROLYSIS                                                       \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: C-H-RAS P21 PROTEIN;                                       \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    ONCOGENE PROTEIN                                                      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    E.F.PAI,A.WITTINGHOFER,W.KABSCH                                       \\nREVDAT   6   29-NOV-17 5P21    1       HELIX                                    \\nREVDAT   5   24-FEB-09 5P21    1       VERSN                                    \\nREVDAT   4   01-APR-03 5P21    1       JRNL                                     \\nREVDAT   3   20-JUL-95 5P21    1       HET                                      \\nREVDAT   2   30-APR-94 5P21    1       FORMUL                                   \\nREVDAT   1   15-JAN-92 5P21    0                                                \\nJRNL        AUTH   E.F.PAI,U.KRENGEL,G.A.PETSKO,R.S.GOODY,W.KABSCH,             \\nJRNL        AUTH 2 A.WITTINGHOFER                                               \\nJRNL        TITL   REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION   \\nJRNL        TITL 2 OF H-RAS P21 AT 1.35 A RESOLUTION: IMPLICATIONS FOR THE      \\nJRNL        TITL 3 MECHANISM OF GTP HYDROLYSIS.                                 \\nJRNL        REF    EMBO J.                       V.   9  2351 1990              \\nJRNL        REFN                   ISSN 0261-4189                               \\nJRNL        PMID   2196171                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,S.C.ALMO,G.RAPP,K.WILSON,K.PETRATOS,A.LENTFER, \\nREMARK   1  AUTH 2 A.WITTINGHOFER,W.KABSCH,E.F.PAI,G.A.PETSKO,R.S.GOODY         \\nREMARK   1  TITL   TIME-RESOLVED X-RAY CRYSTALLOGRAPHIC STUDY OF THE            \\nREMARK   1  TITL 2 CONFORMATIONAL CHANGE IN HA-RAS P21 PROTEIN ON GTP           \\nREMARK   1  TITL 3 HYDROLYSIS                                                   \\nREMARK   1  REF    NATURE                        V. 345   309 1990              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   A.SCHERER,J.JOHN,R.LINKE,R.S.GOODY,A.WITTINGHOFER,E.F.PAI,   \\nREMARK   1  AUTH 2 K.C.HOLMES                                                   \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY ANALYSIS OF THE HUMAN  \\nREMARK   1  TITL 2 C-H-RAS-ONCOGENE PRODUCT P21 COMPLEXED WITH GTP ANALOGUES    \\nREMARK   1  REF    J.MOL.BIOL.                   V. 206   257 1989              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   E.F.PAI,W.KABSCH,U.KRENGEL,K.C.HOLMES,J.JOHN,A.WITTINGHOFER  \\nREMARK   1  TITL   STRUCTURE OF THE GUANINE-NUCLEOTIDE-BINDING DOMAIN OF THE    \\nREMARK   1  TITL 2 HA-RAS ONCOGENE PRODUCT P21 IN THE TRIPHOSPHATE CONFORMATION \\nREMARK   1  REF    NATURE                        V. 341   209 1989              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,G.RAPP,J.JOHN,A.WITTINGHOFER,E.F.PAI,R.S.GOODY \\nREMARK   1  TITL   BIOCHEMICAL AND CRYSTALLOGRAPHIC CHARACTERIZATION OF A       \\nREMARK   1  TITL 2 COMPLEX OF C-HA-RAS P21 AND CAGED GTP WITH FLASH PHOTOLYSIS  \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  86  7687 1989              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 6.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 26806                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.196                           \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1323                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 33                                      \\nREMARK   3   SOLVENT ATOMS            : 211                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.014                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 2.700                           \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  RESIDUES 61 - 65 (GLN - GLU - GLU - TYR - SER) ADOPT                \\nREMARK   3  SEVERAL CONFORMATIONS IN THE CRYSTAL.  THE COORDINATES              \\nREMARK   3  GIVEN APPROXIMATE ONE OF THESE.                                     \\nREMARK   4                                                                      \\nREMARK   4 5P21 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179747.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 38.93                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.01                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 32 2 1                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+2/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+1/3                                           \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z+1/3                                           \\nREMARK 290       6555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000      108.13333            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       54.06667            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       54.06667            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000      108.13333            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025  0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    ARG A  68   NE  -  CZ  -  NH1 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A  97   NE  -  CZ  -  NH1 ANGL. DEV. =  -3.4 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH1 ANGL. DEV. =   4.1 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500    ARG A 128   NE  -  CZ  -  NH2 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A 135   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500    TYR A 157   CB  -  CG  -  CD2 ANGL. DEV. =  -4.8 DEGREES          \\nREMARK 500    ARG A 164   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ASP A  33       75.86   -157.25                                   \\nREMARK 500    ILE A  36      -64.23   -100.39                                   \\nREMARK 500    GLU A  37      123.75   -171.30                                   \\nREMARK 500    SER A  65     -133.70     53.07                                   \\nREMARK 500    LYS A 117       33.80     71.60                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              MG A 168  MG                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 GNP A 167   O2G                                                    \\nREMARK 620 2 GNP A 167   O2B  95.0                                              \\nREMARK 620 3 HOH A 173   O    87.6  91.7                                        \\nREMARK 620 4 SER A  17   OG  172.9  91.0  88.6                                  \\nREMARK 620 5 THR A  35   OG1  91.3 173.4  90.5  82.8                            \\nREMARK 620 6 HOH A 172   O    97.3  88.0 175.0  86.5  89.3                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE MG A 168                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE GNP A 167                 \\nDBREF  5P21 A    1   166  UNP    P01112   RASH_HUMAN       1    166             \\nSEQRES   1 A  166  MET THR GLU TYR LYS LEU VAL VAL VAL GLY ALA GLY GLY          \\nSEQRES   2 A  166  VAL GLY LYS SER ALA LEU THR ILE GLN LEU ILE GLN ASN          \\nSEQRES   3 A  166  HIS PHE VAL ASP GLU TYR ASP PRO THR ILE GLU ASP SER          \\nSEQRES   4 A  166  TYR ARG LYS GLN VAL VAL ILE ASP GLY GLU THR CYS LEU          \\nSEQRES   5 A  166  LEU ASP ILE LEU ASP THR ALA GLY GLN GLU GLU TYR SER          \\nSEQRES   6 A  166  ALA MET ARG ASP GLN TYR MET ARG THR GLY GLU GLY PHE          \\nSEQRES   7 A  166  LEU CYS VAL PHE ALA ILE ASN ASN THR LYS SER PHE GLU          \\nSEQRES   8 A  166  ASP ILE HIS GLN TYR ARG GLU GLN ILE LYS ARG VAL LYS          \\nSEQRES   9 A  166  ASP SER ASP ASP VAL PRO MET VAL LEU VAL GLY ASN LYS          \\nSEQRES  10 A  166  CYS ASP LEU ALA ALA ARG THR VAL GLU SER ARG GLN ALA          \\nSEQRES  11 A  166  GLN ASP LEU ALA ARG SER TYR GLY ILE PRO TYR ILE GLU          \\nSEQRES  12 A  166  THR SER ALA LYS THR ARG GLN GLY VAL GLU ASP ALA PHE          \\nSEQRES  13 A  166  TYR THR LEU VAL ARG GLU ILE ARG GLN HIS                      \\nHET     MG  A 168       1                                                       \\nHET    GNP  A 167      32                                                       \\nHETNAM      MG MAGNESIUM ION                                                    \\nHETNAM     GNP PHOSPHOAMINOPHOSPHONIC ACID-GUANYLATE ESTER                      \\nFORMUL   2   MG    MG 2+                                                        \\nFORMUL   3  GNP    C10 H17 N6 O13 P3                                            \\nFORMUL   4  HOH   *211(H2 O)                                                    \\nHELIX    1  HA LYS A   16  GLN A   25  1                                  10    \\nHELIX    2  HB SER A   65  THR A   74  1RES 65-67 IN 3/10 CONF            10    \\nHELIX    3  HC THR A   87  VAL A  103  1BREAK AT ASP 92                   17    \\nHELIX    4  HD SER A  127  TYR A  137  1                                  11    \\nHELIX    5  HE VAL A  152  ARG A  164  1                                  13    \\nSHEET    1  S1 6 GLU A  37  ILE A  46  0                                        \\nSHEET    2  S1 6 GLU A  49  THR A  58 -1  O  LEU A  53   N  LYS A  42           \\nSHEET    3  S1 6 THR A   2  VAL A   9  1  N  LEU A   6   O  ASP A  54           \\nSHEET    4  S1 6 GLY A  77  ALA A  83  1  N  LEU A  79   O  VAL A   7           \\nSHEET    5  S1 6 MET A 111  ASN A 116  1  N  VAL A 114   O  CYS A  80           \\nSHEET    6  S1 6 TYR A 141  GLU A 143  1  N  ILE A 142   O  LEU A 113           \\nLINK        MG    MG A 168                 O2G GNP A 167     1555   1555  2.20  \\nLINK        MG    MG A 168                 O2B GNP A 167     1555   1555  2.22  \\nLINK        MG    MG A 168                 O   HOH A 173     1555   1555  2.23  \\nLINK        MG    MG A 168                 OG  SER A  17     1555   1555  2.26  \\nLINK        MG    MG A 168                 OG1 THR A  35     1555   1555  2.26  \\nLINK        MG    MG A 168                 O   HOH A 172     1555   1555  2.30  \\nSITE     1 AC1  5 SER A  17  THR A  35  GNP A 167  HOH A 172                    \\nSITE     2 AC1  5 HOH A 173                                                     \\nSITE     1 AC2 28 GLY A  12  GLY A  13  VAL A  14  GLY A  15                    \\nSITE     2 AC2 28 LYS A  16  SER A  17  ALA A  18  PHE A  28                    \\nSITE     3 AC2 28 VAL A  29  ASP A  30  TYR A  32  PRO A  34                    \\nSITE     4 AC2 28 THR A  35  GLY A  60  ASN A 116  LYS A 117                    \\nSITE     5 AC2 28 ASP A 119  SER A 145  ALA A 146  LYS A 147                    \\nSITE     6 AC2 28  MG A 168  HOH A 170  HOH A 172  HOH A 173                    \\nSITE     7 AC2 28 HOH A 175  HOH A 187  HOH A 281  HOH A 366                    \\nCRYST1   40.300   40.300  162.200  90.00  90.00 120.00 P 32 2 1      6          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.024814  0.014326  0.000000        0.00000                         \\nSCALE2      0.000000  0.028653  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.006165        0.00000                         \\nATOM      1  N   MET A   1      -7.186  32.862  -6.632  1.00 13.92           N  \\nATOM      2  CA  MET A   1      -5.921  32.127  -6.684  1.00 17.10           C  \\nATOM      3  C   MET A   1      -5.845  31.045  -5.621  1.00 16.74           C  \\nATOM      4  O   MET A   1      -6.600  31.083  -4.636  1.00 16.60           O  \\nATOM      5  CB  MET A   1      -4.673  33.058  -6.502  1.00 17.00           C  \\nATOM      6  CG  MET A   1      -4.633  33.804  -5.188  1.00 18.08           C  \\nATOM      7  SD  MET A   1      -3.115  34.766  -4.908  1.00 19.73           S  \\nATOM      8  CE  MET A   1      -3.019  35.851  -6.310  1.00 20.41           C  \\nATOM      9  N   THR A   2      -4.910  30.095  -5.820  1.00 14.70           N  \\nATOM     10  CA  THR A   2      -4.661  29.030  -4.859  1.00 16.92           C  \\nATOM     11  C   THR A   2      -4.230  29.655  -3.547  1.00 14.78           C  \\nATOM     12  O   THR A   2      -3.338  30.501  -3.537  1.00 13.93           O  \\nATOM     13  CB  THR A   2      -3.557  28.107  -5.401  1.00 18.03           C  \\nATOM     14  OG1 THR A   2      -4.059  27.565  -6.629  1.00 20.42           O  \\nATOM     15  CG2 THR A   2      -3.193  26.983  -4.443  1.00 18.51           C  \\nATOM     16  N   GLU A   3      -4.911  29.290  -2.472  1.00 14.80           N  \\nATOM     17  CA  GLU A   3      -4.504  29.730  -1.135  1.00 17.71           C  \\nATOM     18  C   GLU A   3      -4.012  28.553  -0.278  1.00 17.33           C  \\nATOM     19  O   GLU A   3      -4.520  27.427  -0.395  1.00 17.54           O  \\nATOM     20  CB  GLU A   3      -5.655  30.402  -0.432  1.00 20.67           C  \\nATOM     21  CG  GLU A   3      -6.213  31.631  -1.186  1.00 25.56           C  \\nATOM     22  CD  GLU A   3      -7.128  32.563  -0.369  1.00 28.46           C  \\nATOM     23  OE1 GLU A   3      -7.578  32.216   0.728  1.00 27.72           O  \\nATOM     24  OE2 GLU A   3      -7.375  33.680  -0.827  1.00 32.49           O  \\nATOM     25  N   TYR A   4      -2.962  28.744   0.542  1.00 14.50           N  \\nATOM     26  CA  TYR A   4      -2.425  27.654   1.358  1.00 12.29           C  \\nATOM     27  C   TYR A   4      -2.534  28.103   2.807  1.00 12.52           C  \\nATOM     28  O   TYR A   4      -2.016  29.143   3.200  1.00 12.11           O  \\nATOM     29  CB  TYR A   4      -0.975  27.385   1.026  1.00 11.46           C  \\nATOM     30  CG  TYR A   4      -0.726  26.846  -0.387  1.00 12.38           C  \\nATOM     31  CD1 TYR A   4      -0.297  27.698  -1.395  1.00 13.00           C  \\nATOM     32  CD2 TYR A   4      -0.933  25.501  -0.649  1.00 14.09           C  \\nATOM     33  CE1 TYR A   4      -0.070  27.208  -2.674  1.00 13.89           C  \\nATOM     34  CE2 TYR A   4      -0.711  24.993  -1.917  1.00 14.65           C  \\nATOM     35  CZ  TYR A   4      -0.279  25.859  -2.917  1.00 14.84           C  \\nATOM     36  OH  TYR A   4      -0.084  25.357  -4.191  1.00 15.07           O  \\nATOM     37  N   LYS A   5      -3.274  27.305   3.568  1.00 13.07           N  \\nATOM     38  CA  LYS A   5      -3.554  27.569   4.970  1.00 12.96           C  \\nATOM     39  C   LYS A   5      -2.556  26.818   5.852  1.00 12.74           C  \\nATOM     40  O   LYS A   5      -2.645  25.611   6.092  1.00 11.58           O  \\nATOM     41  CB  LYS A   5      -4.972  27.131   5.281  1.00 17.65           C  \\nATOM     42  CG  LYS A   5      -6.053  27.833   4.446  1.00 23.49           C  \\nATOM     43  CD  LYS A   5      -7.186  28.310   5.335  1.00 29.10           C  \\nATOM     44  CE  LYS A   5      -6.655  29.309   6.403  1.00 33.68           C  \\nATOM     45  NZ  LYS A   5      -6.597  28.739   7.755  1.00 36.52           N  \\nATOM     46  N   LEU A   6      -1.543  27.574   6.297  1.00 13.08           N  \\nATOM     47  CA  LEU A   6      -0.418  27.011   7.064  1.00 10.34           C  \\nATOM     48  C   LEU A   6      -0.578  27.382   8.535  1.00  9.68           C  \\nATOM     49  O   LEU A   6      -1.034  28.483   8.863  1.00 11.31           O  \\nATOM     50  CB  LEU A   6       0.910  27.571   6.536  1.00 11.34           C  \\nATOM     51  CG  LEU A   6       1.137  27.565   5.017  1.00 11.62           C  \\nATOM     52  CD1 LEU A   6       2.515  28.119   4.720  1.00 13.60           C  \\nATOM     53  CD2 LEU A   6       0.973  26.171   4.475  1.00 12.19           C  \\nATOM     54  N   VAL A   7      -0.196  26.486   9.432  1.00  9.93           N  \\nATOM     55  CA  VAL A   7      -0.333  26.787  10.852  1.00 10.06           C  \\nATOM     56  C   VAL A   7       1.033  26.513  11.495  1.00 10.37           C  \\nATOM     57  O   VAL A   7       1.597  25.441  11.252  1.00  9.51           O  \\nATOM     58  CB  VAL A   7      -1.474  25.884  11.446  1.00 10.97           C  \\nATOM     59  CG1 VAL A   7      -1.610  26.163  12.943  1.00 11.39           C  \\nATOM     60  CG2 VAL A   7      -2.809  26.161  10.777  1.00 10.14           C  \\nATOM     61  N   VAL A   8       1.576  27.455  12.295  1.00  8.35           N  \\nATOM     62  CA  VAL A   8       2.878  27.262  12.944  1.00  6.92           C  \\nATOM     63  C   VAL A   8       2.609  26.896  14.405  1.00  8.84           C  \\nATOM     64  O   VAL A   8       1.930  27.663  15.092  1.00  9.21           O  \\nATOM     65  CB  VAL A   8       3.684  28.565  12.799  1.00  8.08           C  \\nATOM     66  CG1 VAL A   8       5.111  28.324  13.261  1.00  9.52           C  \\nATOM     67  CG2 VAL A   8       3.729  29.011  11.358  1.00  8.43           C  \\nATOM     68  N   VAL A   9       3.024  25.701  14.881  1.00  8.73           N  \\nATOM     69  CA  VAL A   9       2.779  25.234  16.245  1.00  8.41           C  \\nATOM     70  C   VAL A   9       4.075  24.806  16.919  1.00  8.21           C  \\nATOM     71  O   VAL A   9       5.093  24.551  16.287  1.00 10.97           O  \\nATOM     72  CB  VAL A   9       1.824  24.028  16.286  1.00  9.74           C  \\nATOM     73  CG1 VAL A   9       0.545  24.491  15.680  1.00 11.76           C  \\nATOM     74  CG2 VAL A   9       2.349  22.810  15.535  1.00 10.31           C  \\nATOM     75  N   GLY A  10       4.048  24.707  18.238  1.00  7.34           N  \\nATOM     76  CA  GLY A  10       5.231  24.336  19.009  1.00  7.98           C  \\nATOM     77  C   GLY A  10       5.192  25.053  20.355  1.00  8.31           C  \\nATOM     78  O   GLY A  10       4.417  25.994  20.595  1.00  9.89           O  \\nATOM     79  N   ALA A  11       6.099  24.638  21.226  1.00  9.31           N  \\nATOM     80  CA  ALA A  11       6.121  25.135  22.597  1.00  9.50           C  \\nATOM     81  C   ALA A  11       6.440  26.625  22.711  1.00  9.66           C  \\nATOM     82  O   ALA A  11       6.998  27.250  21.806  1.00  9.46           O  \\nATOM     83  CB  ALA A  11       7.145  24.306  23.378  1.00  9.85           C  \\nATOM     84  N   GLY A  12       6.111  27.227  23.847  1.00 10.36           N  \\nATOM     85  CA  GLY A  12       6.406  28.632  24.054  1.00  9.30           C  \\nATOM     86  C   GLY A  12       7.892  28.950  23.964  1.00 10.23           C  \\nATOM     87  O   GLY A  12       8.785  28.249  24.466  1.00 10.57           O  \\nATOM     88  N   GLY A  13       8.159  30.064  23.266  1.00 11.49           N  \\nATOM     89  CA  GLY A  13       9.511  30.602  23.139  1.00 11.29           C  \\nATOM     90  C   GLY A  13      10.436  29.950  22.108  1.00 12.22           C  \\nATOM     91  O   GLY A  13      11.624  30.321  22.060  1.00 13.62           O  \\nATOM     92  N   VAL A  14       9.961  28.991  21.286  1.00  9.23           N  \\nATOM     93  CA  VAL A  14      10.848  28.327  20.317  1.00 10.38           C  \\nATOM     94  C   VAL A  14      11.169  29.168  19.076  1.00 10.47           C  \\nATOM     95  O   VAL A  14      12.108  28.841  18.353  1.00 12.07           O  \\nATOM     96  CB  VAL A  14      10.246  26.970  19.856  1.00  9.98           C  \\nATOM     97  CG1 VAL A  14      10.187  26.074  21.120  1.00  9.93           C  \\nATOM     98  CG2 VAL A  14       8.866  27.127  19.200  1.00  9.52           C  \\nATOM     99  N   GLY A  15      10.423  30.264  18.834  1.00  9.61           N  \\nATOM    100  CA  GLY A  15      10.666  31.160  17.706  1.00  9.70           C  \\nATOM    101  C   GLY A  15       9.615  31.061  16.626  1.00  8.82           C  \\nATOM    102  O   GLY A  15       9.915  31.403  15.491  1.00  8.99           O  \\nATOM    103  N   LYS A  16       8.395  30.591  16.916  1.00  8.18           N  \\nATOM    104  CA  LYS A  16       7.331  30.521  15.913  1.00  8.44           C  \\nATOM    105  C   LYS A  16       7.060  31.899  15.309  1.00  9.01           C  \\nATOM    106  O   LYS A  16       7.020  32.044  14.086  1.00  8.25           O  \\nATOM    107  CB  LYS A  16       6.014  30.000  16.529  1.00  9.27           C  \\nATOM    108  CG  LYS A  16       6.132  28.585  17.086  1.00  9.11           C  \\nATOM    109  CD  LYS A  16       4.820  28.077  17.726  1.00  9.31           C  \\nATOM    110  CE  LYS A  16       4.351  28.934  18.888  1.00 10.05           C  \\nATOM    111  NZ  LYS A  16       5.221  28.829  20.051  1.00  9.64           N  \\nATOM    112  N   SER A  17       6.913  32.942  16.135  1.00  7.74           N  \\nATOM    113  CA  SER A  17       6.661  34.295  15.649  1.00  8.78           C  \\nATOM    114  C   SER A  17       7.854  34.859  14.919  1.00  9.73           C  \\nATOM    115  O   SER A  17       7.653  35.446  13.868  1.00  9.70           O  \\nATOM    116  CB  SER A  17       6.360  35.264  16.769  1.00  8.22           C  \\nATOM    117  OG  SER A  17       5.083  34.853  17.204  1.00  8.79           O  \\nATOM    118  N   ALA A  18       9.072  34.687  15.444  1.00  8.95           N  \\nATOM    119  CA  ALA A  18      10.253  35.199  14.769  1.00  8.95           C  \\nATOM    120  C   ALA A  18      10.438  34.576  13.374  1.00  8.54           C  \\nATOM    121  O   ALA A  18      10.832  35.254  12.430  1.00  8.75           O  \\nATOM    122  CB  ALA A  18      11.521  34.931  15.640  1.00 10.11           C  \\nATOM    123  N   LEU A  19      10.136  33.284  13.176  1.00  8.31           N  \\nATOM    124  CA  LEU A  19      10.225  32.667  11.837  1.00  9.30           C  \\nATOM    125  C   LEU A  19       9.204  33.281  10.884  1.00 10.98           C  \\nATOM    126  O   LEU A  19       9.534  33.652   9.751  1.00 10.47           O  \\nATOM    127  CB  LEU A  19       9.938  31.165  11.885  1.00  9.41           C  \\nATOM    128  CG  LEU A  19      11.035  30.275  12.487  1.00 11.80           C  \\nATOM    129  CD1 LEU A  19      10.491  28.866  12.723  1.00 10.43           C  \\nATOM    130  CD2 LEU A  19      12.256  30.288  11.554  1.00 13.42           C  \\nATOM    131  N   THR A  20       7.950  33.419  11.335  1.00 10.72           N  \\nATOM    132  CA  THR A  20       6.888  34.015  10.529  1.00 11.87           C  \\nATOM    133  C   THR A  20       7.173  35.492  10.161  1.00 12.65           C  \\nATOM    134  O   THR A  20       7.012  35.917   9.009  1.00 12.60           O  \\nATOM    135  CB  THR A  20       5.547  33.888  11.312  1.00 13.43           C  \\nATOM    136  OG1 THR A  20       5.299  32.509  11.544  1.00 12.67           O  \\nATOM    137  CG2 THR A  20       4.346  34.410  10.518  1.00 14.64           C  \\nATOM    138  N   ILE A  21       7.615  36.326  11.098  1.00 10.48           N  \\nATOM    139  CA  ILE A  21       7.945  37.704  10.795  1.00 12.96           C  \\nATOM    140  C   ILE A  21       9.197  37.831   9.927  1.00 14.26           C  \\nATOM    141  O   ILE A  21       9.237  38.691   9.042  1.00 15.83           O  \\nATOM    142  CB  ILE A  21       8.090  38.451  12.121  1.00 12.57           C  \\nATOM    143  CG1 ILE A  21       6.691  38.551  12.737  1.00 13.24           C  \\nATOM    144  CG2 ILE A  21       8.631  39.852  11.910  1.00 13.40           C  \\nATOM    145  CD1 ILE A  21       6.636  39.185  14.129  1.00 16.46           C  \\nATOM    146  N   GLN A  22      10.206  36.979  10.092  1.00 13.58           N  \\nATOM    147  CA  GLN A  22      11.330  36.983   9.179  1.00 14.91           C  \\nATOM    148  C   GLN A  22      10.817  36.729   7.736  1.00 17.29           C  \\nATOM    149  O   GLN A  22      11.120  37.485   6.792  1.00 15.91           O  \\nATOM    150  CB  GLN A  22      12.270  35.930   9.736  1.00 15.90           C  \\nATOM    151  CG  GLN A  22      13.686  35.878   9.230  1.00 19.36           C  \\nATOM    152  CD  GLN A  22      14.372  37.217   9.091  1.00 18.08           C  \\nATOM    153  OE1 GLN A  22      14.160  37.984   8.143  1.00 19.23           O  \\nATOM    154  NE2 GLN A  22      15.224  37.492  10.050  1.00 17.40           N  \\nATOM    155  N   LEU A  23       9.944  35.718   7.523  1.00 15.95           N  \\nATOM    156  CA  LEU A  23       9.381  35.475   6.194  1.00 18.85           C  \\nATOM    157  C   LEU A  23       8.539  36.622   5.636  1.00 18.21           C  \\nATOM    158  O   LEU A  23       8.695  36.958   4.469  1.00 19.03           O  \\nATOM    159  CB  LEU A  23       8.498  34.235   6.196  1.00 18.10           C  \\nATOM    160  CG  LEU A  23       7.916  33.797   4.863  1.00 18.64           C  \\nATOM    161  CD1 LEU A  23       8.921  32.994   4.030  1.00 18.03           C  \\nATOM    162  CD2 LEU A  23       6.680  32.991   5.183  1.00 21.34           C  \\nATOM    163  N   ILE A  24       7.634  37.197   6.428  1.00 19.25           N  \\nATOM    164  CA  ILE A  24       6.682  38.198   5.953  1.00 20.23           C  \\nATOM    165  C   ILE A  24       7.316  39.590   5.922  1.00 21.89           C  \\nATOM    166  O   ILE A  24       7.125  40.318   4.943  1.00 22.93           O  \\nATOM    167  CB  ILE A  24       5.399  38.222   6.881  1.00 21.20           C  \\nATOM    168  CG1 ILE A  24       4.664  36.907   6.821  1.00 21.27           C  \\nATOM    169  CG2 ILE A  24       4.423  39.307   6.446  1.00 20.00           C  \\nATOM    170  CD1 ILE A  24       4.128  36.442   5.475  1.00 23.98           C  \\nATOM    171  N   GLN A  25       8.074  39.998   6.953  1.00 21.07           N  \\nATOM    172  CA  GLN A  25       8.520  41.380   7.066  1.00 22.09           C  \\nATOM    173  C   GLN A  25      10.003  41.559   6.962  1.00 20.75           C  \\nATOM    174  O   GLN A  25      10.495  42.677   7.054  1.00 20.26           O  \\nATOM    175  CB  GLN A  25       8.039  41.968   8.398  1.00 23.96           C  \\nATOM    176  CG  GLN A  25       6.548  42.276   8.381  1.00 28.69           C  \\nATOM    177  CD  GLN A  25       5.931  42.574   9.744  1.00 30.92           C  \\nATOM    178  OE1 GLN A  25       6.537  43.143  10.657  1.00 31.89           O  \\nATOM    179  NE2 GLN A  25       4.664  42.169   9.867  1.00 32.35           N  \\nATOM    180  N   ASN A  26      10.764  40.482   6.795  1.00 22.13           N  \\nATOM    181  CA  ASN A  26      12.198  40.642   6.628  1.00 24.67           C  \\nATOM    182  C   ASN A  26      12.936  41.362   7.734  1.00 23.62           C  \\nATOM    183  O   ASN A  26      13.603  42.372   7.521  1.00 25.10           O  \\nATOM    184  CB  ASN A  26      12.522  41.369   5.315  1.00 27.68           C  \\nATOM    185  CG  ASN A  26      12.942  40.351   4.288  1.00 32.80           C  \\nATOM    186  OD1 ASN A  26      12.135  39.699   3.602  1.00 33.77           O  \\nATOM    187  ND2 ASN A  26      14.268  40.203   4.237  1.00 34.26           N  \\nATOM    188  N   HIS A  27      12.764  40.842   8.945  1.00 19.95           N  \\nATOM    189  CA  HIS A  27      13.593  41.266  10.057  1.00 17.99           C  \\nATOM    190  C   HIS A  27      13.441  40.299  11.215  1.00 16.84           C  \\nATOM    191  O   HIS A  27      12.411  39.623  11.344  1.00 14.72           O  \\nATOM    192  CB  HIS A  27      13.241  42.680  10.551  1.00 19.80           C  \\nATOM    193  CG  HIS A  27      11.926  42.908  11.258  1.00 20.77           C  \\nATOM    194  ND1 HIS A  27      11.936  43.314  12.510  1.00 22.17           N  \\nATOM    195  CD2 HIS A  27      10.674  42.768  10.729  1.00 22.32           C  \\nATOM    196  CE1 HIS A  27      10.657  43.430  12.785  1.00 23.86           C  \\nATOM    197  NE2 HIS A  27       9.905  43.110  11.738  1.00 23.97           N  \\nATOM    198  N   PHE A  28      14.516  40.222  12.012  1.00 15.44           N  \\nATOM    199  CA  PHE A  28      14.528  39.413  13.223  1.00 13.89           C  \\nATOM    200  C   PHE A  28      14.026  40.196  14.440  1.00 15.33           C  \\nATOM    201  O   PHE A  28      14.528  41.255  14.786  1.00 16.80           O  \\nATOM    202  CB  PHE A  28      15.938  38.906  13.470  1.00 12.73           C  \\nATOM    203  CG  PHE A  28      16.083  38.150  14.786  1.00 13.35           C  \\nATOM    204  CD1 PHE A  28      15.354  36.990  15.012  1.00 13.80           C  \\nATOM    205  CD2 PHE A  28      16.974  38.615  15.740  1.00 14.54           C  \\nATOM    206  CE1 PHE A  28      15.534  36.310  16.194  1.00 14.45           C  \\nATOM    207  CE2 PHE A  28      17.147  37.925  16.920  1.00 14.61           C  \\nATOM    208  CZ  PHE A  28      16.420  36.768  17.137  1.00 14.65           C  \\nATOM    209  N   VAL A  29      12.970  39.697  15.089  1.00 17.22           N  \\nATOM    210  CA  VAL A  29      12.467  40.313  16.294  1.00 19.11           C  \\nATOM    211  C   VAL A  29      13.127  39.606  17.476  1.00 19.36           C  \\nATOM    212  O   VAL A  29      13.245  38.376  17.605  1.00 18.14           O  \\nATOM    213  CB  VAL A  29      10.894  40.205  16.412  1.00 22.03           C  \\nATOM    214  CG1 VAL A  29      10.334  40.869  15.143  1.00 19.66           C  \\nATOM    215  CG2 VAL A  29      10.377  38.776  16.631  1.00 23.15           C  \\nATOM    216  N   ASP A  30      13.615  40.484  18.324  1.00 20.14           N  \\nATOM    217  CA  ASP A  30      14.229  40.019  19.518  1.00 24.04           C  \\nATOM    218  C   ASP A  30      13.676  40.746  20.721  1.00 25.93           C  \\nATOM    219  O   ASP A  30      14.244  41.709  21.238  1.00 29.26           O  \\nATOM    220  CB  ASP A  30      15.719  40.203  19.443  1.00 27.10           C  \\nATOM    221  CG  ASP A  30      16.389  39.731  20.732  1.00 32.46           C  \\nATOM    222  OD1 ASP A  30      15.878  38.859  21.464  1.00 34.28           O  \\nATOM    223  OD2 ASP A  30      17.458  40.264  21.002  1.00 36.46           O  \\nATOM    224  N   GLU A  31      12.472  40.347  21.076  1.00 26.14           N  \\nATOM    225  CA  GLU A  31      11.922  40.804  22.318  1.00 28.72           C  \\nATOM    226  C   GLU A  31      10.870  39.836  22.749  1.00 26.20           C  \\nATOM    227  O   GLU A  31      10.466  38.931  22.014  1.00 23.14           O  \\nATOM    228  CB  GLU A  31      11.337  42.210  22.204  1.00 33.25           C  \\nATOM    229  CG  GLU A  31      10.350  42.618  21.152  1.00 40.08           C  \\nATOM    230  CD  GLU A  31      10.572  44.107  20.887  1.00 44.71           C  \\nATOM    231  OE1 GLU A  31      10.930  44.444  19.755  1.00 47.50           O  \\nATOM    232  OE2 GLU A  31      10.420  44.928  21.806  1.00 47.89           O  \\nATOM    233  N   TYR A  32      10.550  40.044  24.028  1.00 22.26           N  \\nATOM    234  CA  TYR A  32       9.597  39.196  24.685  1.00 18.83           C  \\nATOM    235  C   TYR A  32       8.204  39.688  24.317  1.00 17.53           C  \\nATOM    236  O   TYR A  32       7.794  40.792  24.676  1.00 18.45           O  \\nATOM    237  CB  TYR A  32       9.909  39.259  26.199  1.00 16.84           C  \\nATOM    238  CG  TYR A  32       9.076  38.309  27.052  1.00 15.64           C  \\nATOM    239  CD1 TYR A  32       9.598  37.096  27.433  1.00 16.17           C  \\nATOM    240  CD2 TYR A  32       7.814  38.686  27.459  1.00 15.54           C  \\nATOM    241  CE1 TYR A  32       8.845  36.251  28.239  1.00 17.86           C  \\nATOM    242  CE2 TYR A  32       7.051  37.861  28.257  1.00 14.40           C  \\nATOM    243  CZ  TYR A  32       7.572  36.635  28.652  1.00 16.47           C  \\nATOM    244  OH  TYR A  32       6.821  35.789  29.485  1.00 13.59           O  \\nATOM    245  N   ASP A  33       7.461  38.841  23.604  1.00 13.43           N  \\nATOM    246  CA  ASP A  33       6.128  39.200  23.141  1.00 14.00           C  \\nATOM    247  C   ASP A  33       5.262  37.955  22.854  1.00 13.56           C  \\nATOM    248  O   ASP A  33       5.024  37.586  21.681  1.00 12.47           O  \\nATOM    249  CB  ASP A  33       6.271  40.072  21.864  1.00 15.92           C  \\nATOM    250  CG  ASP A  33       4.993  40.654  21.269  1.00 18.19           C  \\nATOM    251  OD1 ASP A  33       3.923  40.631  21.879  1.00 21.46           O  \\nATOM    252  OD2 ASP A  33       5.074  41.164  20.167  1.00 20.34           O  \\nATOM    253  N   PRO A  34       4.752  37.283  23.903  1.00 12.04           N  \\nATOM    254  CA  PRO A  34       3.949  36.070  23.788  1.00 11.09           C  \\nATOM    255  C   PRO A  34       2.720  36.248  22.895  1.00 11.76           C  \\nATOM    256  O   PRO A  34       1.953  37.220  23.006  1.00 12.63           O  \\nATOM    257  CB  PRO A  34       3.590  35.711  25.219  1.00 12.50           C  \\nATOM    258  CG  PRO A  34       4.643  36.418  26.033  1.00 12.14           C  \\nATOM    259  CD  PRO A  34       4.817  37.717  25.311  1.00 12.82           C  \\nATOM    260  N   THR A  35       2.554  35.282  21.972  1.00 11.00           N  \\nATOM    261  CA  THR A  35       1.421  35.320  21.033  1.00 11.54           C  \\nATOM    262  C   THR A  35       0.140  34.746  21.639  1.00 13.51           C  \\nATOM    263  O   THR A  35       0.174  33.820  22.450  1.00 12.45           O  \\nATOM    264  CB  THR A  35       1.730  34.518  19.751  1.00 11.34           C  \\nATOM    265  OG1 THR A  35       2.900  35.134  19.233  1.00 11.20           O  \\nATOM    266  CG2 THR A  35       0.706  34.611  18.637  1.00 13.10           C  \\nATOM    267  N   ILE A  36      -0.996  35.332  21.233  1.00 15.48           N  \\nATOM    268  CA  ILE A  36      -2.308  34.792  21.567  1.00 19.42           C  \\nATOM    269  C   ILE A  36      -2.663  34.079  20.274  1.00 17.36           C  \\nATOM    270  O   ILE A  36      -2.749  32.863  20.256  1.00 18.29           O  \\nATOM    271  CB  ILE A  36      -3.353  35.914  21.879  1.00 21.30           C  \\nATOM    272  CG1 ILE A  36      -2.930  36.823  23.072  1.00 24.32           C  \\nATOM    273  CG2 ILE A  36      -4.670  35.207  22.136  1.00 21.54           C  \\nATOM    274  CD1 ILE A  36      -2.494  36.167  24.413  1.00 24.70           C  \\nATOM    275  N   GLU A  37      -2.818  34.822  19.181  1.00 19.35           N  \\nATOM    276  CA  GLU A  37      -3.059  34.221  17.889  1.00 21.47           C  \\nATOM    277  C   GLU A  37      -2.957  35.362  16.897  1.00 23.77           C  \\nATOM    278  O   GLU A  37      -3.636  36.393  17.031  1.00 25.49           O  \\nATOM    279  CB  GLU A  37      -4.440  33.611  17.880  1.00 25.97           C  \\nATOM    280  CG  GLU A  37      -4.766  32.776  16.667  1.00 33.81           C  \\nATOM    281  CD  GLU A  37      -6.160  32.154  16.777  1.00 38.37           C  \\nATOM    282  OE1 GLU A  37      -7.095  32.664  16.149  1.00 37.86           O  \\nATOM    283  OE2 GLU A  37      -6.298  31.164  17.515  1.00 42.24           O  \\nATOM    284  N   ASP A  38      -2.070  35.212  15.911  1.00 19.96           N  \\nATOM    285  CA  ASP A  38      -1.899  36.178  14.839  1.00 17.94           C  \\nATOM    286  C   ASP A  38      -2.024  35.526  13.473  1.00 18.21           C  \\nATOM    287  O   ASP A  38      -1.582  34.393  13.294  1.00 16.96           O  \\nATOM    288  CB  ASP A  38      -0.540  36.833  14.965  1.00 18.83           C  \\nATOM    289  CG  ASP A  38      -0.507  37.817  16.133  1.00 20.08           C  \\nATOM    290  OD1 ASP A  38      -1.390  38.660  16.208  1.00 25.47           O  \\nATOM    291  OD2 ASP A  38       0.375  37.781  16.967  1.00 20.08           O  \\nATOM    292  N   SER A  39      -2.698  36.170  12.507  1.00 17.82           N  \\nATOM    293  CA  SER A  39      -2.691  35.658  11.133  1.00 18.16           C  \\nATOM    294  C   SER A  39      -2.012  36.627  10.171  1.00 16.95           C  \\nATOM    295  O   SER A  39      -2.053  37.853  10.333  1.00 15.80           O  \\nATOM    296  CB  SER A  39      -4.094  35.402  10.637  1.00 23.01           C  \\nATOM    297  OG  SER A  39      -4.873  36.571  10.754  1.00 27.13           O  \\nATOM    298  N   TYR A  40      -1.336  36.080   9.168  1.00 15.44           N  \\nATOM    299  CA  TYR A  40      -0.607  36.889   8.196  1.00 16.25           C  \\nATOM    300  C   TYR A  40      -1.018  36.383   6.830  1.00 17.06           C  \\nATOM    301  O   TYR A  40      -1.160  35.172   6.638  1.00 18.29           O  \\nATOM    302  CB  TYR A  40       0.909  36.724   8.370  1.00 16.59           C  \\nATOM    303  CG  TYR A  40       1.439  37.242   9.707  1.00 16.10           C  \\nATOM    304  CD1 TYR A  40       1.473  36.412  10.815  1.00 16.77           C  \\nATOM    305  CD2 TYR A  40       1.883  38.549   9.799  1.00 18.02           C  \\nATOM    306  CE1 TYR A  40       1.950  36.893  12.021  1.00 16.86           C  \\nATOM    307  CE2 TYR A  40       2.368  39.032  11.001  1.00 18.70           C  \\nATOM    308  CZ  TYR A  40       2.396  38.199  12.104  1.00 17.62           C  \\nATOM    309  OH  TYR A  40       2.853  38.695  13.304  1.00 18.37           O  \\nATOM    310  N   ARG A  41      -1.270  37.303   5.900  1.00 15.48           N  \\nATOM    311  CA  ARG A  41      -1.660  36.918   4.558  1.00 15.42           C  \\nATOM    312  C   ARG A  41      -0.730  37.606   3.586  1.00 16.76           C  \\nATOM    313  O   ARG A  41      -0.536  38.829   3.633  1.00 17.01           O  \\nATOM    314  CB  ARG A  41      -3.087  37.342   4.277  1.00 17.64           C  \\nATOM    315  CG  ARG A  41      -4.150  36.507   4.947  1.00 18.09           C  \\nATOM    316  CD  ARG A  41      -5.528  37.181   4.849  1.00 19.64           C  \\nATOM    317  NE  ARG A  41      -6.066  37.318   3.489  1.00 20.11           N  \\nATOM    318  CZ  ARG A  41      -6.660  36.329   2.803  1.00 22.17           C  \\nATOM    319  NH1 ARG A  41      -7.219  36.571   1.617  1.00 22.48           N  \\nATOM    320  NH2 ARG A  41      -6.711  35.090   3.274  1.00 23.55           N  \\nATOM    321  N   LYS A  42      -0.135  36.827   2.679  1.00 16.52           N  \\nATOM    322  CA  LYS A  42       0.835  37.405   1.758  1.00 16.59           C  \\nATOM    323  C   LYS A  42       0.740  36.746   0.387  1.00 16.04           C  \\nATOM    324  O   LYS A  42       0.794  35.528   0.307  1.00 14.74           O  \\nATOM    325  CB  LYS A  42       2.208  37.210   2.401  1.00 18.22           C  \\nATOM    326  CG  LYS A  42       3.388  37.784   1.669  1.00 22.68           C  \\nATOM    327  CD  LYS A  42       3.369  39.274   1.792  1.00 28.09           C  \\nATOM    328  CE  LYS A  42       4.695  39.876   1.317  1.00 31.70           C  \\nATOM    329  NZ  LYS A  42       4.828  39.827  -0.128  1.00 35.70           N  \\nATOM    330  N   GLN A  43       0.561  37.510  -0.697  1.00 16.89           N  \\nATOM    331  CA  GLN A  43       0.579  36.950  -2.047  1.00 19.10           C  \\nATOM    332  C   GLN A  43       2.026  36.729  -2.448  1.00 15.08           C  \\nATOM    333  O   GLN A  43       2.828  37.654  -2.362  1.00 15.72           O  \\nATOM    334  CB  GLN A  43      -0.116  37.908  -3.043  1.00 23.32           C  \\nATOM    335  CG  GLN A  43       0.543  38.332  -4.415  1.00 32.83           C  \\nATOM    336  CD  GLN A  43       1.081  37.332  -5.482  1.00 36.11           C  \\nATOM    337  OE1 GLN A  43       0.457  36.335  -5.865  1.00 37.78           O  \\nATOM    338  NE2 GLN A  43       2.271  37.606  -6.053  1.00 35.45           N  \\nATOM    339  N   VAL A  44       2.394  35.532  -2.922  1.00 12.79           N  \\nATOM    340  CA  VAL A  44       3.771  35.242  -3.311  1.00 11.03           C  \\nATOM    341  C   VAL A  44       3.724  34.425  -4.605  1.00 14.28           C  \\nATOM    342  O   VAL A  44       2.633  34.109  -5.090  1.00 14.90           O  \\nATOM    343  CB  VAL A  44       4.508  34.411  -2.223  1.00 13.21           C  \\nATOM    344  CG1 VAL A  44       4.669  35.282  -0.977  1.00 15.04           C  \\nATOM    345  CG2 VAL A  44       3.761  33.116  -1.889  1.00 12.89           C  \\nATOM    346  N   VAL A  45       4.879  34.127  -5.235  1.00 13.20           N  \\nATOM    347  CA  VAL A  45       4.938  33.277  -6.419  1.00 12.28           C  \\nATOM    348  C   VAL A  45       5.836  32.161  -5.970  1.00 11.08           C  \\nATOM    349  O   VAL A  45       6.988  32.447  -5.636  1.00 13.13           O  \\nATOM    350  CB  VAL A  45       5.587  33.979  -7.641  1.00 13.01           C  \\nATOM    351  CG1 VAL A  45       5.549  33.031  -8.810  1.00 14.51           C  \\nATOM    352  CG2 VAL A  45       4.787  35.173  -8.108  1.00 13.54           C  \\nATOM    353  N   ILE A  46       5.374  30.907  -5.915  1.00  9.89           N  \\nATOM    354  CA  ILE A  46       6.274  29.795  -5.491  1.00 11.03           C  \\nATOM    355  C   ILE A  46       6.369  28.800  -6.662  1.00  8.85           C  \\nATOM    356  O   ILE A  46       5.319  28.296  -7.059  1.00 10.25           O  \\nATOM    357  CB  ILE A  46       5.692  29.115  -4.203  1.00 10.64           C  \\nATOM    358  CG1 ILE A  46       5.652  30.096  -3.042  1.00 11.20           C  \\nATOM    359  CG2 ILE A  46       6.599  27.953  -3.757  1.00 12.93           C  \\nATOM    360  CD1 ILE A  46       4.937  29.536  -1.808  1.00  8.90           C  \\nATOM    361  N   ASP A  47       7.565  28.495  -7.224  1.00  9.41           N  \\nATOM    362  CA  ASP A  47       7.767  27.650  -8.426  1.00  9.70           C  \\nATOM    363  C   ASP A  47       6.838  28.031  -9.572  1.00 11.14           C  \\nATOM    364  O   ASP A  47       6.225  27.211 -10.248  1.00 14.09           O  \\nATOM    365  CB  ASP A  47       7.566  26.165  -8.105  1.00 10.73           C  \\nATOM    366  CG  ASP A  47       8.388  25.783  -6.874  1.00 13.52           C  \\nATOM    367  OD1 ASP A  47       7.805  25.291  -5.898  1.00 12.04           O  \\nATOM    368  OD2 ASP A  47       9.598  26.024  -6.872  1.00 11.31           O  \\nATOM    369  N   GLY A  48       6.724  29.353  -9.735  1.00 10.78           N  \\nATOM    370  CA  GLY A  48       5.930  29.961 -10.785  1.00 13.29           C  \\nATOM    371  C   GLY A  48       4.426  29.989 -10.548  1.00 14.28           C  \\nATOM    372  O   GLY A  48       3.677  30.512 -11.386  1.00 15.19           O  \\nATOM    373  N   GLU A  49       3.944  29.422  -9.439  1.00 13.41           N  \\nATOM    374  CA  GLU A  49       2.520  29.457  -9.143  1.00 13.90           C  \\nATOM    375  C   GLU A  49       2.223  30.700  -8.314  1.00 15.34           C  \\nATOM    376  O   GLU A  49       2.860  30.906  -7.275  1.00 14.56           O  \\nATOM    377  CB  GLU A  49       2.124  28.238  -8.340  1.00 15.50           C  \\nATOM    378  CG  GLU A  49       0.630  28.250  -7.990  1.00 18.10           C  \\nATOM    379  CD  GLU A  49       0.204  27.110  -7.100  1.00 19.13           C  \\nATOM    380  OE1 GLU A  49       0.717  27.004  -5.996  1.00 19.53           O  \\nATOM    381  OE2 GLU A  49      -0.630  26.321  -7.529  1.00 22.12           O  \\nATOM    382  N   THR A  50       1.267  31.536  -8.734  1.00 14.06           N  \\nATOM    383  CA  THR A  50       0.901  32.697  -7.927  1.00 16.15           C  \\nATOM    384  C   THR A  50      -0.001  32.169  -6.819  1.00 13.35           C  \\nATOM    385  O   THR A  50      -0.938  31.409  -7.046  1.00 15.13           O  \\nATOM    386  CB  THR A  50       0.203  33.755  -8.846  1.00 17.88           C  \\nATOM    387  OG1 THR A  50      -0.245  34.813  -8.012  1.00 24.33           O  \\nATOM    388  CG2 THR A  50      -1.028  33.236  -9.518  1.00 19.78           C  \\nATOM    389  N   CYS A  51       0.306  32.442  -5.563  1.00 13.28           N  \\nATOM    390  CA  CYS A  51      -0.566  31.959  -4.508  1.00 15.16           C  \\nATOM    391  C   CYS A  51      -0.579  32.892  -3.303  1.00 14.00           C  \\nATOM    392  O   CYS A  51       0.161  33.874  -3.231  1.00 11.81           O  \\nATOM    393  CB  CYS A  51      -0.132  30.537  -4.079  1.00 14.08           C  \\nATOM    394  SG  CYS A  51       1.597  30.409  -3.575  1.00 16.09           S  \\nATOM    395  N   LEU A  52      -1.521  32.609  -2.405  1.00 13.77           N  \\nATOM    396  CA  LEU A  52      -1.573  33.397  -1.199  1.00 16.01           C  \\nATOM    397  C   LEU A  52      -1.260  32.497  -0.025  1.00 14.53           C  \\nATOM    398  O   LEU A  52      -1.868  31.443   0.113  1.00 16.05           O  \\nATOM    399  CB  LEU A  52      -2.957  33.986  -1.014  1.00 19.71           C  \\nATOM    400  CG  LEU A  52      -3.122  34.885   0.214  1.00 20.84           C  \\nATOM    401  CD1 LEU A  52      -3.070  36.329  -0.220  1.00 21.38           C  \\nATOM    402  CD2 LEU A  52      -4.420  34.526   0.923  1.00 20.90           C  \\nATOM    403  N   LEU A  53      -0.319  32.904   0.800  1.00 12.41           N  \\nATOM    404  CA  LEU A  53       0.019  32.232   2.048  1.00 13.70           C  \\nATOM    405  C   LEU A  53      -0.841  32.852   3.148  1.00 13.85           C  \\nATOM    406  O   LEU A  53      -0.733  34.056   3.413  1.00 12.82           O  \\nATOM    407  CB  LEU A  53       1.485  32.464   2.400  1.00 14.51           C  \\nATOM    408  CG  LEU A  53       2.707  31.589   2.031  1.00 19.29           C  \\nATOM    409  CD1 LEU A  53       2.394  30.553   0.962  1.00 18.56           C  \\nATOM    410  CD2 LEU A  53       3.843  32.557   1.741  1.00 20.00           C  \\nATOM    411  N   ASP A  54      -1.699  32.023   3.751  1.00 11.96           N  \\nATOM    412  CA  ASP A  54      -2.476  32.445   4.884  1.00 15.07           C  \\nATOM    413  C   ASP A  54      -1.882  31.675   6.055  1.00 12.67           C  \\nATOM    414  O   ASP A  54      -2.099  30.477   6.234  1.00 13.99           O  \\nATOM    415  CB  ASP A  54      -3.930  32.088   4.644  1.00 19.33           C  \\nATOM    416  CG  ASP A  54      -4.826  32.609   5.746  1.00 24.24           C  \\nATOM    417  OD1 ASP A  54      -4.412  32.652   6.897  1.00 28.17           O  \\nATOM    418  OD2 ASP A  54      -5.958  32.979   5.450  1.00 27.59           O  \\nATOM    419  N   ILE A  55      -1.127  32.389   6.880  1.00 11.78           N  \\nATOM    420  CA  ILE A  55      -0.346  31.769   7.935  1.00 12.15           C  \\nATOM    421  C   ILE A  55      -0.953  32.099   9.296  1.00 13.10           C  \\nATOM    422  O   ILE A  55      -1.195  33.271   9.587  1.00 13.84           O  \\nATOM    423  CB  ILE A  55       1.109  32.282   7.931  1.00 12.42           C  \\nATOM    424  CG1 ILE A  55       1.859  31.979   6.653  1.00 13.99           C  \\nATOM    425  CG2 ILE A  55       1.859  31.505   9.018  1.00 13.24           C  \\nATOM    426  CD1 ILE A  55       3.176  32.768   6.497  1.00 15.68           C  \\nATOM    427  N   LEU A  56      -1.176  31.083  10.134  1.00 11.64           N  \\nATOM    428  CA  LEU A  56      -1.642  31.341  11.490  1.00 11.98           C  \\nATOM    429  C   LEU A  56      -0.524  31.027  12.431  1.00  9.99           C  \\nATOM    430  O   LEU A  56      -0.017  29.895  12.447  1.00  9.72           O  \\nATOM    431  CB  LEU A  56      -2.836  30.463  11.896  1.00 13.80           C  \\nATOM    432  CG  LEU A  56      -3.470  30.741  13.297  1.00 16.13           C  \\nATOM    433  CD1 LEU A  56      -4.347  31.987  13.260  1.00 15.28           C  \\nATOM    434  CD2 LEU A  56      -4.321  29.580  13.692  1.00 16.84           C  \\nATOM    435  N   ASP A  57      -0.188  32.084  13.182  1.00 10.40           N  \\nATOM    436  CA  ASP A  57       0.815  32.014  14.219  1.00 10.30           C  \\nATOM    437  C   ASP A  57       0.127  31.812  15.571  1.00 11.75           C  \\nATOM    438  O   ASP A  57      -0.652  32.631  16.053  1.00 12.91           O  \\nATOM    439  CB  ASP A  57       1.602  33.296  14.173  1.00 10.66           C  \\nATOM    440  CG  ASP A  57       2.778  33.336  15.132  1.00 11.39           C  \\nATOM    441  OD1 ASP A  57       3.306  32.298  15.535  1.00 11.07           O  \\nATOM    442  OD2 ASP A  57       3.168  34.448  15.455  1.00 13.15           O  \\nATOM    443  N   THR A  58       0.447  30.723  16.244  1.00 12.55           N  \\nATOM    444  CA  THR A  58      -0.324  30.325  17.416  1.00 12.98           C  \\nATOM    445  C   THR A  58       0.368  30.541  18.758  1.00 13.48           C  \\nATOM    446  O   THR A  58       1.568  30.812  18.815  1.00 14.00           O  \\nATOM    447  CB  THR A  58      -0.698  28.851  17.219  1.00  9.54           C  \\nATOM    448  OG1 THR A  58       0.491  28.084  17.369  1.00 11.40           O  \\nATOM    449  CG2 THR A  58      -1.313  28.597  15.848  1.00 12.27           C  \\nATOM    450  N   ALA A  59      -0.371  30.382  19.868  1.00 14.38           N  \\nATOM    451  CA  ALA A  59       0.230  30.510  21.194  1.00 14.79           C  \\nATOM    452  C   ALA A  59       0.928  29.210  21.608  1.00 15.41           C  \\nATOM    453  O   ALA A  59       0.448  28.106  21.324  1.00 13.07           O  \\nATOM    454  CB  ALA A  59      -0.845  30.818  22.238  1.00 15.47           C  \\nATOM    455  N   GLY A  60       2.084  29.341  22.271  1.00 17.71           N  \\nATOM    456  CA  GLY A  60       2.807  28.208  22.814  1.00 24.18           C  \\nATOM    457  C   GLY A  60       2.190  27.751  24.130  1.00 31.21           C  \\nATOM    458  O   GLY A  60       2.538  26.688  24.627  1.00 30.44           O  \\nATOM    459  N   GLN A  61       1.260  28.503  24.726  1.00 39.50           N  \\nATOM    460  CA  GLN A  61       0.671  28.099  25.994  1.00 48.84           C  \\nATOM    461  C   GLN A  61      -0.683  27.395  25.985  1.00 51.52           C  \\nATOM    462  O   GLN A  61      -1.606  27.780  25.276  1.00 52.17           O  \\nATOM    463  CB  GLN A  61       0.605  29.327  26.874  1.00 51.56           C  \\nATOM    464  CG  GLN A  61       0.871  28.861  28.312  1.00 56.74           C  \\nATOM    465  CD  GLN A  61       1.506  29.937  29.177  1.00 58.74           C  \\nATOM    466  OE1 GLN A  61       2.647  29.795  29.616  1.00 60.12           O  \\nATOM    467  NE2 GLN A  61       0.783  31.031  29.429  1.00 60.18           N  \\nATOM    468  N   GLU A  62      -0.804  26.349  26.818  1.00 57.40           N  \\nATOM    469  CA  GLU A  62      -2.008  25.527  26.927  1.00 61.96           C  \\nATOM    470  C   GLU A  62      -3.185  26.196  27.631  1.00 62.20           C  \\nATOM    471  O   GLU A  62      -4.138  25.496  27.975  1.00 62.85           O  \\nATOM    472  CB  GLU A  62      -1.629  24.183  27.632  1.00 65.18           C  \\nATOM    473  CG  GLU A  62      -2.622  22.966  27.761  1.00 68.22           C  \\nATOM    474  CD  GLU A  62      -3.032  22.132  26.528  1.00 70.25           C  \\nATOM    475  OE1 GLU A  62      -2.710  22.494  25.395  1.00 71.45           O  \\nATOM    476  OE2 GLU A  62      -3.691  21.101  26.689  1.00 70.00           O  \\nATOM    477  N   GLU A  63      -3.206  27.520  27.904  1.00 62.78           N  \\nATOM    478  CA  GLU A  63      -4.496  28.139  28.254  1.00 63.62           C  \\nATOM    479  C   GLU A  63      -5.457  27.854  27.085  1.00 63.71           C  \\nATOM    480  O   GLU A  63      -6.688  27.814  27.211  1.00 62.97           O  \\nATOM    481  CB  GLU A  63      -4.372  29.666  28.475  1.00 65.13           C  \\nATOM    482  CG  GLU A  63      -3.456  30.553  27.607  1.00 66.91           C  \\nATOM    483  CD  GLU A  63      -4.012  31.235  26.345  1.00 68.05           C  \\nATOM    484  OE1 GLU A  63      -5.220  31.462  26.220  1.00 68.64           O  \\nATOM    485  OE2 GLU A  63      -3.198  31.568  25.481  1.00 67.62           O  \\nATOM    486  N   TYR A  64      -4.801  27.606  25.934  1.00 63.36           N  \\nATOM    487  CA  TYR A  64      -5.420  27.092  24.743  1.00 63.20           C  \\nATOM    488  C   TYR A  64      -5.905  25.653  24.875  1.00 61.76           C  \\nATOM    489  O   TYR A  64      -7.120  25.486  24.858  1.00 63.04           O  \\nATOM    490  CB  TYR A  64      -4.437  27.208  23.613  1.00 64.83           C  \\nATOM    491  CG  TYR A  64      -4.736  28.459  22.818  1.00 66.98           C  \\nATOM    492  CD1 TYR A  64      -4.300  29.686  23.267  1.00 67.55           C  \\nATOM    493  CD2 TYR A  64      -5.459  28.355  21.646  1.00 67.97           C  \\nATOM    494  CE1 TYR A  64      -4.587  30.825  22.536  1.00 69.49           C  \\nATOM    495  CE2 TYR A  64      -5.752  29.488  20.911  1.00 69.29           C  \\nATOM    496  CZ  TYR A  64      -5.313  30.718  21.359  1.00 70.05           C  \\nATOM    497  OH  TYR A  64      -5.589  31.852  20.614  1.00 70.67           O  \\nATOM    498  N   SER A  65      -5.095  24.590  24.997  1.00 58.67           N  \\nATOM    499  CA  SER A  65      -5.617  23.230  25.127  1.00 56.93           C  \\nATOM    500  C   SER A  65      -6.614  22.757  24.063  1.00 55.33           C  \\nATOM    501  O   SER A  65      -6.321  22.936  22.876  1.00 54.74           O  \\nATOM    502  CB  SER A  65      -6.190  23.122  26.552  1.00 57.99           C  \\nATOM    503  OG  SER A  65      -7.251  24.011  26.900  1.00 59.80           O  \\nATOM    504  N   ALA A  66      -7.762  22.119  24.359  1.00 52.33           N  \\nATOM    505  CA  ALA A  66      -8.766  21.824  23.335  1.00 48.80           C  \\nATOM    506  C   ALA A  66      -9.138  23.050  22.492  1.00 45.71           C  \\nATOM    507  O   ALA A  66      -9.751  22.862  21.449  1.00 44.33           O  \\nATOM    508  CB  ALA A  66     -10.043  21.300  23.971  1.00 49.14           C  \\nATOM    509  N   MET A  67      -8.829  24.310  22.875  1.00 42.85           N  \\nATOM    510  CA  MET A  67      -8.984  25.465  21.967  1.00 40.01           C  \\nATOM    511  C   MET A  67      -8.093  25.362  20.719  1.00 36.79           C  \\nATOM    512  O   MET A  67      -8.221  26.184  19.805  1.00 36.08           O  \\nATOM    513  CB  MET A  67      -8.594  26.804  22.581  1.00 42.50           C  \\nATOM    514  CG  MET A  67      -9.359  27.330  23.772  1.00 45.93           C  \\nATOM    515  SD  MET A  67     -11.140  27.396  23.483  1.00 48.74           S  \\nATOM    516  CE  MET A  67     -11.526  25.880  24.321  1.00 47.64           C  \\nATOM    517  N   ARG A  68      -7.128  24.418  20.665  1.00 31.59           N  \\nATOM    518  CA  ARG A  68      -6.322  24.217  19.467  1.00 26.34           C  \\nATOM    519  C   ARG A  68      -7.140  23.473  18.424  1.00 23.68           C  \\nATOM    520  O   ARG A  68      -6.846  23.617  17.231  1.00 21.93           O  \\nATOM    521  CB  ARG A  68      -5.057  23.394  19.745  1.00 25.86           C  \\nATOM    522  CG  ARG A  68      -4.161  23.898  20.869  1.00 29.36           C  \\nATOM    523  CD  ARG A  68      -2.881  23.061  20.998  1.00 32.88           C  \\nATOM    524  NE  ARG A  68      -2.221  22.990  22.315  1.00 36.64           N  \\nATOM    525  CZ  ARG A  68      -1.700  24.031  23.001  1.00 39.66           C  \\nATOM    526  NH1 ARG A  68      -1.903  25.315  22.664  1.00 42.06           N  \\nATOM    527  NH2 ARG A  68      -0.933  23.798  24.071  1.00 39.61           N  \\nATOM    528  N   ASP A  69      -8.188  22.721  18.793  1.00 19.14           N  \\nATOM    529  CA  ASP A  69      -8.880  21.905  17.822  1.00 20.43           C  \\nATOM    530  C   ASP A  69      -9.437  22.623  16.617  1.00 22.16           C  \\nATOM    531  O   ASP A  69      -9.453  22.109  15.488  1.00 19.58           O  \\nATOM    532  CB  ASP A  69     -10.025  21.148  18.478  1.00 20.58           C  \\nATOM    533  CG  ASP A  69      -9.598  19.935  19.303  1.00 22.81           C  \\nATOM    534  OD1 ASP A  69      -8.409  19.708  19.538  1.00 24.59           O  \\nATOM    535  OD2 ASP A  69     -10.475  19.176  19.715  1.00 25.28           O  \\nATOM    536  N   GLN A  70      -9.834  23.874  16.840  1.00 23.26           N  \\nATOM    537  CA  GLN A  70     -10.451  24.609  15.759  1.00 23.53           C  \\nATOM    538  C   GLN A  70      -9.437  24.925  14.667  1.00 21.20           C  \\nATOM    539  O   GLN A  70      -9.653  24.591  13.493  1.00 20.90           O  \\nATOM    540  CB  GLN A  70     -11.071  25.881  16.341  1.00 27.73           C  \\nATOM    541  CG  GLN A  70     -12.498  26.019  15.832  1.00 34.11           C  \\nATOM    542  CD  GLN A  70     -13.103  27.407  16.008  1.00 37.87           C  \\nATOM    543  OE1 GLN A  70     -12.532  28.284  16.672  1.00 39.77           O  \\nATOM    544  NE2 GLN A  70     -14.283  27.620  15.406  1.00 37.89           N  \\nATOM    545  N   TYR A  71      -8.292  25.515  15.050  1.00 19.09           N  \\nATOM    546  CA  TYR A  71      -7.329  25.873  14.030  1.00 17.69           C  \\nATOM    547  C   TYR A  71      -6.648  24.658  13.416  1.00 17.24           C  \\nATOM    548  O   TYR A  71      -6.247  24.675  12.245  1.00 17.25           O  \\nATOM    549  CB  TYR A  71      -6.302  26.868  14.625  1.00 17.87           C  \\nATOM    550  CG  TYR A  71      -5.311  26.480  15.725  1.00 18.90           C  \\nATOM    551  CD1 TYR A  71      -4.355  25.510  15.492  1.00 19.74           C  \\nATOM    552  CD2 TYR A  71      -5.328  27.162  16.940  1.00 19.21           C  \\nATOM    553  CE1 TYR A  71      -3.402  25.212  16.450  1.00 20.38           C  \\nATOM    554  CE2 TYR A  71      -4.375  26.868  17.901  1.00 21.93           C  \\nATOM    555  CZ  TYR A  71      -3.422  25.897  17.645  1.00 21.91           C  \\nATOM    556  OH  TYR A  71      -2.463  25.590  18.588  1.00 26.44           O  \\nATOM    557  N   MET A  72      -6.557  23.559  14.166  1.00 14.81           N  \\nATOM    558  CA  MET A  72      -5.958  22.393  13.563  1.00 18.59           C  \\nATOM    559  C   MET A  72      -6.849  21.778  12.472  1.00 19.38           C  \\nATOM    560  O   MET A  72      -6.385  21.198  11.492  1.00 17.67           O  \\nATOM    561  CB  MET A  72      -5.658  21.386  14.669  1.00 17.39           C  \\nATOM    562  CG  MET A  72      -4.479  21.796  15.514  1.00 16.35           C  \\nATOM    563  SD  MET A  72      -4.146  20.600  16.820  1.00 17.04           S  \\nATOM    564  CE  MET A  72      -2.635  21.259  17.460  1.00 17.72           C  \\nATOM    565  N   ARG A  73      -8.174  21.931  12.626  1.00 20.31           N  \\nATOM    566  CA  ARG A  73      -9.120  21.350  11.701  1.00 22.24           C  \\nATOM    567  C   ARG A  73      -9.087  22.084  10.370  1.00 20.94           C  \\nATOM    568  O   ARG A  73      -9.183  21.450   9.326  1.00 20.81           O  \\nATOM    569  CB  ARG A  73     -10.499  21.414  12.369  1.00 25.29           C  \\nATOM    570  CG  ARG A  73     -11.553  20.499  11.767  1.00 30.04           C  \\nATOM    571  CD  ARG A  73     -12.775  20.429  12.687  1.00 31.86           C  \\nATOM    572  NE  ARG A  73     -12.483  19.702  13.917  1.00 34.70           N  \\nATOM    573  CZ  ARG A  73     -12.894  20.122  15.130  1.00 36.49           C  \\nATOM    574  NH1 ARG A  73     -12.579  19.402  16.196  1.00 36.93           N  \\nATOM    575  NH2 ARG A  73     -13.609  21.239  15.311  1.00 38.17           N  \\nATOM    576  N   THR A  74      -8.868  23.401  10.413  1.00 21.33           N  \\nATOM    577  CA  THR A  74      -8.813  24.240   9.219  1.00 24.99           C  \\nATOM    578  C   THR A  74      -7.436  24.351   8.544  1.00 23.10           C  \\nATOM    579  O   THR A  74      -7.367  24.738   7.371  1.00 22.59           O  \\nATOM    580  CB  THR A  74      -9.289  25.676   9.538  1.00 25.73           C  \\nATOM    581  OG1 THR A  74      -8.351  26.302  10.409  1.00 31.08           O  \\nATOM    582  CG2 THR A  74     -10.657  25.644  10.201  1.00 27.58           C  \\nATOM    583  N   GLY A  75      -6.345  24.084   9.289  1.00 19.58           N  \\nATOM    584  CA  GLY A  75      -5.013  24.114   8.730  1.00 15.79           C  \\nATOM    585  C   GLY A  75      -4.823  23.023   7.713  1.00 13.66           C  \\nATOM    586  O   GLY A  75      -5.245  21.880   7.884  1.00 14.69           O  \\nATOM    587  N   GLU A  76      -4.228  23.360   6.573  1.00 13.14           N  \\nATOM    588  CA  GLU A  76      -3.914  22.363   5.546  1.00 12.99           C  \\nATOM    589  C   GLU A  76      -2.514  21.740   5.753  1.00 12.22           C  \\nATOM    590  O   GLU A  76      -2.293  20.576   5.426  1.00 13.59           O  \\nATOM    591  CB  GLU A  76      -3.999  23.014   4.145  1.00 13.07           C  \\nATOM    592  CG  GLU A  76      -5.387  23.377   3.676  1.00 14.35           C  \\nATOM    593  CD  GLU A  76      -5.261  24.069   2.323  1.00 20.40           C  \\nATOM    594  OE1 GLU A  76      -5.461  23.403   1.288  1.00 24.32           O  \\nATOM    595  OE2 GLU A  76      -4.913  25.250   2.284  1.00 18.82           O  \\nATOM    596  N   GLY A  77      -1.553  22.503   6.301  1.00 10.73           N  \\nATOM    597  CA  GLY A  77      -0.216  22.018   6.570  1.00  8.07           C  \\nATOM    598  C   GLY A  77       0.302  22.689   7.819  1.00  8.63           C  \\nATOM    599  O   GLY A  77      -0.066  23.823   8.127  1.00  8.55           O  \\nATOM    600  N   PHE A  78       1.248  22.013   8.479  1.00  9.15           N  \\nATOM    601  CA  PHE A  78       1.754  22.468   9.773  1.00  8.87           C  \\nATOM    602  C   PHE A  78       3.268  22.520   9.854  1.00  9.51           C  \\nATOM    603  O   PHE A  78       3.930  21.542   9.486  1.00  9.15           O  \\nATOM    604  CB  PHE A  78       1.247  21.532  10.904  1.00  9.44           C  \\nATOM    605  CG  PHE A  78      -0.267  21.472  11.000  1.00  9.66           C  \\nATOM    606  CD1 PHE A  78      -0.906  22.310  11.901  1.00 10.71           C  \\nATOM    607  CD2 PHE A  78      -0.994  20.645  10.152  1.00 10.20           C  \\nATOM    608  CE1 PHE A  78      -2.295  22.338  11.940  1.00 10.61           C  \\nATOM    609  CE2 PHE A  78      -2.378  20.678  10.192  1.00 12.55           C  \\nATOM    610  CZ  PHE A  78      -3.022  21.529  11.081  1.00 12.80           C  \\nATOM    611  N   LEU A  79       3.813  23.636  10.354  1.00  8.72           N  \\nATOM    612  CA  LEU A  79       5.230  23.626  10.699  1.00  9.93           C  \\nATOM    613  C   LEU A  79       5.279  23.281  12.181  1.00  9.09           C  \\nATOM    614  O   LEU A  79       4.701  24.008  12.988  1.00 11.58           O  \\nATOM    615  CB  LEU A  79       5.921  24.970  10.567  1.00 12.80           C  \\nATOM    616  CG  LEU A  79       6.155  25.659   9.249  1.00 16.39           C  \\nATOM    617  CD1 LEU A  79       7.007  26.911   9.544  1.00 17.02           C  \\nATOM    618  CD2 LEU A  79       6.904  24.756   8.269  1.00 16.98           C  \\nATOM    619  N   CYS A  80       5.950  22.219  12.579  1.00  9.14           N  \\nATOM    620  CA  CYS A  80       6.074  21.821  13.981  1.00 11.81           C  \\nATOM    621  C   CYS A  80       7.448  22.237  14.434  1.00 11.53           C  \\nATOM    622  O   CYS A  80       8.435  21.640  14.000  1.00 12.21           O  \\nATOM    623  CB  CYS A  80       5.903  20.318  14.106  1.00 13.73           C  \\nATOM    624  SG  CYS A  80       4.158  19.900  13.837  1.00 18.26           S  \\nATOM    625  N   VAL A  81       7.486  23.282  15.269  1.00  7.57           N  \\nATOM    626  CA  VAL A  81       8.730  23.956  15.604  1.00  8.45           C  \\nATOM    627  C   VAL A  81       9.248  23.620  16.996  1.00  8.90           C  \\nATOM    628  O   VAL A  81       8.451  23.680  17.939  1.00  8.49           O  \\nATOM    629  CB  VAL A  81       8.544  25.516  15.507  1.00  7.40           C  \\nATOM    630  CG1 VAL A  81       9.917  26.196  15.588  1.00  9.42           C  \\nATOM    631  CG2 VAL A  81       8.004  25.954  14.167  1.00  8.59           C  \\nATOM    632  N   PHE A  82      10.546  23.237  17.119  1.00  9.49           N  \\nATOM    633  CA  PHE A  82      11.195  23.202  18.442  1.00  9.09           C  \\nATOM    634  C   PHE A  82      12.511  23.994  18.350  1.00  9.83           C  \\nATOM    635  O   PHE A  82      12.914  24.440  17.266  1.00  9.02           O  \\nATOM    636  CB  PHE A  82      11.482  21.755  18.902  1.00  9.76           C  \\nATOM    637  CG  PHE A  82      12.536  21.062  18.071  1.00 10.09           C  \\nATOM    638  CD1 PHE A  82      13.867  21.099  18.486  1.00 10.70           C  \\nATOM    639  CD2 PHE A  82      12.192  20.412  16.889  1.00 10.93           C  \\nATOM    640  CE1 PHE A  82      14.838  20.483  17.699  1.00 11.12           C  \\nATOM    641  CE2 PHE A  82      13.181  19.807  16.125  1.00 10.73           C  \\nATOM    642  CZ  PHE A  82      14.505  19.839  16.523  1.00 10.35           C  \\nATOM    643  N   ALA A  83      13.128  24.269  19.516  1.00  9.75           N  \\nATOM    644  CA  ALA A  83      14.407  24.963  19.569  1.00 10.32           C  \\nATOM    645  C   ALA A  83      15.509  23.960  19.916  1.00  9.48           C  \\nATOM    646  O   ALA A  83      15.362  23.142  20.839  1.00  9.61           O  \\nATOM    647  CB  ALA A  83      14.354  26.083  20.620  1.00  9.75           C  \\nATOM    648  N   ILE A  84      16.624  24.042  19.160  1.00 11.41           N  \\nATOM    649  CA  ILE A  84      17.714  23.070  19.299  1.00 13.15           C  \\nATOM    650  C   ILE A  84      18.446  23.223  20.623  1.00 14.57           C  \\nATOM    651  O   ILE A  84      19.234  22.347  20.971  1.00 13.94           O  \\nATOM    652  CB  ILE A  84      18.775  23.151  18.160  1.00 13.29           C  \\nATOM    653  CG1 ILE A  84      19.403  24.509  17.878  1.00 14.90           C  \\nATOM    654  CG2 ILE A  84      18.018  22.701  16.933  1.00 13.09           C  \\nATOM    655  CD1 ILE A  84      20.578  24.957  18.763  1.00 17.59           C  \\nATOM    656  N   ASN A  85      18.200  24.290  21.390  1.00 13.61           N  \\nATOM    657  CA  ASN A  85      18.794  24.394  22.720  1.00 15.52           C  \\nATOM    658  C   ASN A  85      17.758  24.156  23.828  1.00 15.49           C  \\nATOM    659  O   ASN A  85      17.988  24.519  24.999  1.00 16.13           O  \\nATOM    660  CB  ASN A  85      19.446  25.788  22.885  1.00 15.16           C  \\nATOM    661  CG  ASN A  85      18.464  26.946  22.904  1.00 16.42           C  \\nATOM    662  OD1 ASN A  85      17.355  26.878  22.362  1.00 16.01           O  \\nATOM    663  ND2 ASN A  85      18.808  28.030  23.587  1.00 16.83           N  \\nATOM    664  N   ASN A  86      16.616  23.515  23.509  1.00 14.72           N  \\nATOM    665  CA  ASN A  86      15.553  23.303  24.483  1.00 13.01           C  \\nATOM    666  C   ASN A  86      14.987  21.901  24.309  1.00 13.17           C  \\nATOM    667  O   ASN A  86      14.080  21.668  23.520  1.00 13.04           O  \\nATOM    668  CB  ASN A  86      14.478  24.367  24.256  1.00 16.51           C  \\nATOM    669  CG  ASN A  86      13.267  24.415  25.187  1.00 20.74           C  \\nATOM    670  OD1 ASN A  86      12.370  25.241  25.018  1.00 25.41           O  \\nATOM    671  ND2 ASN A  86      13.123  23.596  26.206  1.00 19.96           N  \\nATOM    672  N   THR A  87      15.488  20.950  25.103  1.00 13.80           N  \\nATOM    673  CA  THR A  87      15.095  19.553  25.026  1.00 15.40           C  \\nATOM    674  C   THR A  87      13.589  19.316  25.266  1.00 14.40           C  \\nATOM    675  O   THR A  87      12.953  18.535  24.546  1.00 14.11           O  \\nATOM    676  CB  THR A  87      15.983  18.791  26.046  1.00 16.19           C  \\nATOM    677  OG1 THR A  87      17.325  18.908  25.571  1.00 18.22           O  \\nATOM    678  CG2 THR A  87      15.626  17.327  26.180  1.00 17.82           C  \\nATOM    679  N   LYS A  88      12.983  19.958  26.272  1.00 14.42           N  \\nATOM    680  CA  LYS A  88      11.552  19.842  26.535  1.00 15.84           C  \\nATOM    681  C   LYS A  88      10.726  20.232  25.278  1.00 15.06           C  \\nATOM    682  O   LYS A  88       9.722  19.582  25.003  1.00 14.61           O  \\nATOM    683  CB  LYS A  88      11.207  20.766  27.717  1.00 19.53           C  \\nATOM    684  CG  LYS A  88       9.937  20.401  28.457  1.00 27.71           C  \\nATOM    685  CD  LYS A  88       9.585  21.452  29.529  1.00 33.26           C  \\nATOM    686  CE  LYS A  88       9.061  22.762  28.877  1.00 36.79           C  \\nATOM    687  NZ  LYS A  88       8.642  23.763  29.852  1.00 37.29           N  \\nATOM    688  N   SER A  89      11.119  21.259  24.475  1.00 12.36           N  \\nATOM    689  CA  SER A  89      10.325  21.666  23.309  1.00 11.31           C  \\nATOM    690  C   SER A  89      10.371  20.572  22.268  1.00 12.51           C  \\nATOM    691  O   SER A  89       9.397  20.388  21.547  1.00 12.88           O  \\nATOM    692  CB  SER A  89      10.844  22.951  22.674  1.00 11.64           C  \\nATOM    693  OG  SER A  89      12.160  22.900  22.124  1.00 11.11           O  \\nATOM    694  N   PHE A  90      11.504  19.842  22.204  1.00 11.27           N  \\nATOM    695  CA  PHE A  90      11.680  18.751  21.262  1.00 12.91           C  \\nATOM    696  C   PHE A  90      10.760  17.601  21.667  1.00 14.37           C  \\nATOM    697  O   PHE A  90      10.034  17.057  20.839  1.00 13.24           O  \\nATOM    698  CB  PHE A  90      13.161  18.312  21.260  1.00 12.34           C  \\nATOM    699  CG  PHE A  90      13.411  17.120  20.351  1.00 14.41           C  \\nATOM    700  CD1 PHE A  90      13.890  15.948  20.903  1.00 15.10           C  \\nATOM    701  CD2 PHE A  90      13.136  17.206  18.990  1.00 14.81           C  \\nATOM    702  CE1 PHE A  90      14.093  14.857  20.088  1.00 16.20           C  \\nATOM    703  CE2 PHE A  90      13.340  16.110  18.179  1.00 14.81           C  \\nATOM    704  CZ  PHE A  90      13.818  14.935  18.731  1.00 15.39           C  \\nATOM    705  N   GLU A  91      10.725  17.231  22.949  1.00 14.40           N  \\nATOM    706  CA  GLU A  91       9.863  16.146  23.407  1.00 15.78           C  \\nATOM    707  C   GLU A  91       8.374  16.466  23.301  1.00 16.18           C  \\nATOM    708  O   GLU A  91       7.533  15.582  23.114  1.00 17.50           O  \\nATOM    709  CB  GLU A  91      10.226  15.846  24.835  1.00 19.84           C  \\nATOM    710  CG  GLU A  91      11.654  15.386  25.074  1.00 24.49           C  \\nATOM    711  CD  GLU A  91      12.090  15.310  26.537  1.00 27.97           C  \\nATOM    712  OE1 GLU A  91      13.199  14.842  26.766  1.00 30.72           O  \\nATOM    713  OE2 GLU A  91      11.371  15.730  27.449  1.00 30.38           O  \\nATOM    714  N   ASP A  92       8.019  17.747  23.433  1.00 16.19           N  \\nATOM    715  CA  ASP A  92       6.650  18.216  23.265  1.00 15.23           C  \\nATOM    716  C   ASP A  92       6.141  18.053  21.839  1.00 14.29           C  \\nATOM    717  O   ASP A  92       4.927  18.074  21.633  1.00 14.78           O  \\nATOM    718  CB  ASP A  92       6.536  19.708  23.646  1.00 17.85           C  \\nATOM    719  CG  ASP A  92       6.706  20.080  25.127  1.00 21.39           C  \\nATOM    720  OD1 ASP A  92       6.648  19.209  25.997  1.00 20.06           O  \\nATOM    721  OD2 ASP A  92       6.887  21.272  25.401  1.00 24.39           O  \\nATOM    722  N   ILE A  93       7.013  17.910  20.826  1.00 11.76           N  \\nATOM    723  CA  ILE A  93       6.559  17.749  19.452  1.00 14.77           C  \\nATOM    724  C   ILE A  93       5.574  16.583  19.354  1.00 14.03           C  \\nATOM    725  O   ILE A  93       4.576  16.695  18.644  1.00 14.93           O  \\nATOM    726  CB  ILE A  93       7.790  17.499  18.485  1.00 14.96           C  \\nATOM    727  CG1 ILE A  93       8.506  18.830  18.293  1.00 15.41           C  \\nATOM    728  CG2 ILE A  93       7.369  16.936  17.091  1.00 14.61           C  \\nATOM    729  CD1 ILE A  93       7.654  19.940  17.637  1.00 20.05           C  \\nATOM    730  N   HIS A  94       5.806  15.471  20.049  1.00 15.23           N  \\nATOM    731  CA  HIS A  94       4.940  14.302  19.956  1.00 18.20           C  \\nATOM    732  C   HIS A  94       3.494  14.635  20.287  1.00 16.36           C  \\nATOM    733  O   HIS A  94       2.576  14.183  19.598  1.00 17.20           O  \\nATOM    734  CB  HIS A  94       5.436  13.194  20.900  1.00 23.57           C  \\nATOM    735  CG  HIS A  94       4.685  11.874  20.678  1.00 29.09           C  \\nATOM    736  ND1 HIS A  94       4.923  11.118  19.602  1.00 31.26           N  \\nATOM    737  CD2 HIS A  94       3.705  11.361  21.498  1.00 30.93           C  \\nATOM    738  CE1 HIS A  94       4.078  10.110  19.746  1.00 31.74           C  \\nATOM    739  NE2 HIS A  94       3.366  10.260  20.861  1.00 32.20           N  \\nATOM    740  N   GLN A  95       3.271  15.499  21.281  1.00 16.91           N  \\nATOM    741  CA  GLN A  95       1.916  15.878  21.673  1.00 18.66           C  \\nATOM    742  C   GLN A  95       1.226  16.736  20.625  1.00 16.97           C  \\nATOM    743  O   GLN A  95       0.012  16.562  20.443  1.00 15.36           O  \\nATOM    744  CB  GLN A  95       1.928  16.626  23.018  1.00 21.56           C  \\nATOM    745  CG  GLN A  95       2.457  15.766  24.180  1.00 29.81           C  \\nATOM    746  CD  GLN A  95       3.370  16.540  25.150  1.00 35.29           C  \\nATOM    747  OE1 GLN A  95       3.037  17.647  25.593  1.00 37.94           O  \\nATOM    748  NE2 GLN A  95       4.546  16.006  25.521  1.00 37.04           N  \\nATOM    749  N   TYR A  96       1.930  17.653  19.921  1.00 16.06           N  \\nATOM    750  CA  TYR A  96       1.276  18.417  18.826  1.00 16.07           C  \\nATOM    751  C   TYR A  96       0.963  17.504  17.658  1.00 14.44           C  \\nATOM    752  O   TYR A  96      -0.138  17.565  17.118  1.00 15.79           O  \\nATOM    753  CB  TYR A  96       2.139  19.554  18.264  1.00 17.94           C  \\nATOM    754  CG  TYR A  96       2.287  20.677  19.277  1.00 19.19           C  \\nATOM    755  CD1 TYR A  96       1.368  21.699  19.338  1.00 19.33           C  \\nATOM    756  CD2 TYR A  96       3.348  20.631  20.161  1.00 19.61           C  \\nATOM    757  CE1 TYR A  96       1.526  22.678  20.308  1.00 21.72           C  \\nATOM    758  CE2 TYR A  96       3.512  21.595  21.129  1.00 21.72           C  \\nATOM    759  CZ  TYR A  96       2.594  22.613  21.190  1.00 21.26           C  \\nATOM    760  OH  TYR A  96       2.776  23.582  22.153  1.00 24.09           O  \\nATOM    761  N   ARG A  97       1.904  16.635  17.295  1.00 13.30           N  \\nATOM    762  CA  ARG A  97       1.699  15.650  16.247  1.00 15.46           C  \\nATOM    763  C   ARG A  97       0.457  14.794  16.492  1.00 14.26           C  \\nATOM    764  O   ARG A  97      -0.420  14.676  15.641  1.00 14.10           O  \\nATOM    765  CB  ARG A  97       2.939  14.796  16.196  1.00 17.97           C  \\nATOM    766  CG  ARG A  97       2.886  13.541  15.357  1.00 23.35           C  \\nATOM    767  CD  ARG A  97       3.333  13.765  13.943  1.00 23.59           C  \\nATOM    768  NE  ARG A  97       3.314  12.471  13.314  1.00 24.08           N  \\nATOM    769  CZ  ARG A  97       3.268  12.320  12.007  1.00 25.74           C  \\nATOM    770  NH1 ARG A  97       3.135  11.073  11.548  1.00 26.76           N  \\nATOM    771  NH2 ARG A  97       3.353  13.380  11.181  1.00 26.95           N  \\nATOM    772  N   GLU A  98       0.332  14.218  17.678  1.00 13.69           N  \\nATOM    773  CA  GLU A  98      -0.807  13.361  17.941  1.00 16.64           C  \\nATOM    774  C   GLU A  98      -2.118  14.108  17.983  1.00 15.57           C  \\nATOM    775  O   GLU A  98      -3.157  13.573  17.607  1.00 15.85           O  \\nATOM    776  CB  GLU A  98      -0.593  12.623  19.260  1.00 21.96           C  \\nATOM    777  CG  GLU A  98       0.417  11.455  19.177  1.00 28.02           C  \\nATOM    778  CD  GLU A  98       0.147  10.460  18.038  1.00 31.04           C  \\nATOM    779  OE1 GLU A  98       1.118  10.123  17.373  1.00 33.58           O  \\nATOM    780  OE2 GLU A  98      -0.992  10.025  17.790  1.00 33.61           O  \\nATOM    781  N   GLN A  99      -2.105  15.362  18.446  1.00 14.66           N  \\nATOM    782  CA  GLN A  99      -3.343  16.104  18.468  1.00 13.22           C  \\nATOM    783  C   GLN A  99      -3.779  16.398  17.049  1.00 12.15           C  \\nATOM    784  O   GLN A  99      -4.974  16.274  16.791  1.00 14.09           O  \\nATOM    785  CB  GLN A  99      -3.158  17.403  19.250  1.00 16.29           C  \\nATOM    786  CG  GLN A  99      -4.513  18.118  19.407  1.00 19.67           C  \\nATOM    787  CD  GLN A  99      -4.517  19.240  20.438  1.00 22.19           C  \\nATOM    788  OE1 GLN A  99      -3.537  19.485  21.138  1.00 23.70           O  \\nATOM    789  NE2 GLN A  99      -5.628  19.961  20.537  1.00 22.72           N  \\nATOM    790  N   ILE A 100      -2.861  16.795  16.142  1.00 11.42           N  \\nATOM    791  CA  ILE A 100      -3.197  17.121  14.743  1.00 10.81           C  \\nATOM    792  C   ILE A 100      -3.761  15.893  14.008  1.00 12.39           C  \\nATOM    793  O   ILE A 100      -4.733  15.967  13.243  1.00 12.70           O  \\nATOM    794  CB  ILE A 100      -1.924  17.659  14.038  1.00 10.32           C  \\nATOM    795  CG1 ILE A 100      -1.642  19.075  14.518  1.00 10.37           C  \\nATOM    796  CG2 ILE A 100      -2.131  17.685  12.512  1.00 11.36           C  \\nATOM    797  CD1 ILE A 100      -0.184  19.481  14.392  1.00 10.69           C  \\nATOM    798  N   LYS A 101      -3.143  14.732  14.255  1.00 14.08           N  \\nATOM    799  CA  LYS A 101      -3.613  13.487  13.637  1.00 15.95           C  \\nATOM    800  C   LYS A 101      -5.021  13.150  14.079  1.00 15.72           C  \\nATOM    801  O   LYS A 101      -5.848  12.770  13.262  1.00 14.25           O  \\nATOM    802  CB  LYS A 101      -2.681  12.317  13.996  1.00 17.44           C  \\nATOM    803  CG  LYS A 101      -1.414  12.284  13.115  1.00 19.24           C  \\nATOM    804  CD  LYS A 101      -0.332  11.418  13.764  1.00 23.28           C  \\nATOM    805  CE  LYS A 101      -0.752  10.000  14.188  1.00 23.82           C  \\nATOM    806  NZ  LYS A 101       0.400   9.230  14.662  1.00 24.00           N  \\nATOM    807  N   ARG A 102      -5.278  13.341  15.377  1.00 16.72           N  \\nATOM    808  CA  ARG A 102      -6.586  13.098  15.962  1.00 18.74           C  \\nATOM    809  C   ARG A 102      -7.643  14.087  15.440  1.00 19.07           C  \\nATOM    810  O   ARG A 102      -8.712  13.628  15.002  1.00 20.67           O  \\nATOM    811  CB  ARG A 102      -6.457  13.197  17.462  1.00 20.62           C  \\nATOM    812  CG  ARG A 102      -7.539  12.409  18.132  1.00 26.89           C  \\nATOM    813  CD  ARG A 102      -7.610  12.887  19.585  1.00 30.60           C  \\nATOM    814  NE  ARG A 102      -8.197  14.224  19.645  1.00 35.83           N  \\nATOM    815  CZ  ARG A 102      -7.633  15.255  20.301  1.00 37.61           C  \\nATOM    816  NH1 ARG A 102      -8.243  16.455  20.267  1.00 38.37           N  \\nATOM    817  NH2 ARG A 102      -6.481  15.110  20.983  1.00 38.39           N  \\nATOM    818  N   VAL A 103      -7.418  15.407  15.395  1.00 15.59           N  \\nATOM    819  CA  VAL A 103      -8.452  16.326  14.881  1.00 18.48           C  \\nATOM    820  C   VAL A 103      -8.665  16.200  13.349  1.00 16.72           C  \\nATOM    821  O   VAL A 103      -9.776  16.383  12.848  1.00 15.90           O  \\nATOM    822  CB  VAL A 103      -8.131  17.840  15.215  1.00 20.10           C  \\nATOM    823  CG1 VAL A 103      -7.896  18.005  16.704  1.00 21.34           C  \\nATOM    824  CG2 VAL A 103      -6.899  18.302  14.512  1.00 23.93           C  \\nATOM    825  N   LYS A 104      -7.622  15.902  12.574  1.00 13.84           N  \\nATOM    826  CA  LYS A 104      -7.804  15.755  11.140  1.00 16.11           C  \\nATOM    827  C   LYS A 104      -8.158  14.329  10.716  1.00 16.70           C  \\nATOM    828  O   LYS A 104      -8.260  14.079   9.508  1.00 17.27           O  \\nATOM    829  CB  LYS A 104      -6.525  16.215  10.404  1.00 18.12           C  \\nATOM    830  CG  LYS A 104      -6.197  17.678  10.744  1.00 19.53           C  \\nATOM    831  CD  LYS A 104      -5.250  18.297   9.736  1.00 21.28           C  \\nATOM    832  CE  LYS A 104      -5.921  18.590   8.390  1.00 20.75           C  \\nATOM    833  NZ  LYS A 104      -6.875  19.673   8.462  1.00 19.79           N  \\nATOM    834  N   ASP A 105      -8.372  13.390  11.669  1.00 15.91           N  \\nATOM    835  CA  ASP A 105      -8.666  11.988  11.359  1.00 15.43           C  \\nATOM    836  C   ASP A 105      -7.705  11.372  10.333  1.00 16.04           C  \\nATOM    837  O   ASP A 105      -8.106  10.654   9.415  1.00 15.38           O  \\nATOM    838  CB  ASP A 105     -10.077  11.847  10.810  1.00 18.74           C  \\nATOM    839  CG  ASP A 105     -11.261  12.121  11.696  1.00 21.15           C  \\nATOM    840  OD1 ASP A 105     -11.183  11.972  12.918  1.00 21.36           O  \\nATOM    841  OD2 ASP A 105     -12.281  12.480  11.113  1.00 22.63           O  \\nATOM    842  N   SER A 106      -6.397  11.591  10.469  1.00 15.54           N  \\nATOM    843  CA  SER A 106      -5.470  11.174   9.442  1.00 17.25           C  \\nATOM    844  C   SER A 106      -4.112  10.882  10.011  1.00 17.31           C  \\nATOM    845  O   SER A 106      -3.650  11.556  10.916  1.00 19.11           O  \\nATOM    846  CB  SER A 106      -5.463  12.314   8.398  1.00 19.27           C  \\nATOM    847  OG  SER A 106      -4.326  12.490   7.573  1.00 24.20           O  \\nATOM    848  N   ASP A 107      -3.454   9.891   9.453  1.00 17.29           N  \\nATOM    849  CA  ASP A 107      -2.109   9.557   9.868  1.00 19.80           C  \\nATOM    850  C   ASP A 107      -1.000  10.220   9.063  1.00 20.36           C  \\nATOM    851  O   ASP A 107       0.197  10.118   9.345  1.00 21.26           O  \\nATOM    852  CB  ASP A 107      -1.970   8.045   9.810  1.00 23.15           C  \\nATOM    853  CG  ASP A 107      -2.863   7.310  10.807  1.00 25.12           C  \\nATOM    854  OD1 ASP A 107      -3.262   6.206  10.462  1.00 29.21           O  \\nATOM    855  OD2 ASP A 107      -3.127   7.812  11.903  1.00 26.59           O  \\nATOM    856  N   ASP A 108      -1.383  10.909   8.005  1.00 20.57           N  \\nATOM    857  CA  ASP A 108      -0.430  11.664   7.216  1.00 22.38           C  \\nATOM    858  C   ASP A 108      -1.061  12.982   6.842  1.00 21.37           C  \\nATOM    859  O   ASP A 108      -1.973  13.096   6.030  1.00 24.36           O  \\nATOM    860  CB  ASP A 108      -0.056  10.900   5.967  1.00 27.28           C  \\nATOM    861  CG  ASP A 108      -1.263  10.278   5.320  1.00 32.24           C  \\nATOM    862  OD1 ASP A 108      -1.714   9.272   5.877  1.00 37.07           O  \\nATOM    863  OD2 ASP A 108      -1.747  10.790   4.301  1.00 36.33           O  \\nATOM    864  N   VAL A 109      -0.598  13.970   7.572  1.00 19.33           N  \\nATOM    865  CA  VAL A 109      -1.009  15.354   7.400  1.00 17.49           C  \\nATOM    866  C   VAL A 109       0.215  16.086   6.819  1.00 14.55           C  \\nATOM    867  O   VAL A 109       1.335  15.772   7.245  1.00 13.64           O  \\nATOM    868  CB  VAL A 109      -1.410  15.870   8.784  1.00 18.40           C  \\nATOM    869  CG1 VAL A 109      -1.645  17.366   8.765  1.00 20.21           C  \\nATOM    870  CG2 VAL A 109      -2.676  15.101   9.214  1.00 20.27           C  \\nATOM    871  N   PRO A 110       0.141  17.001   5.825  1.00 12.41           N  \\nATOM    872  CA  PRO A 110       1.297  17.783   5.384  1.00 11.52           C  \\nATOM    873  C   PRO A 110       1.920  18.480   6.605  1.00 11.79           C  \\nATOM    874  O   PRO A 110       1.289  19.245   7.324  1.00  9.55           O  \\nATOM    875  CB  PRO A 110       0.728  18.721   4.370  1.00 11.31           C  \\nATOM    876  CG  PRO A 110      -0.410  17.926   3.779  1.00 11.38           C  \\nATOM    877  CD  PRO A 110      -1.044  17.309   5.020  1.00 12.05           C  \\nATOM    878  N   MET A 111       3.192  18.201   6.874  1.00 11.92           N  \\nATOM    879  CA  MET A 111       3.878  18.724   8.050  1.00 11.49           C  \\nATOM    880  C   MET A 111       5.379  18.749   7.807  1.00 11.68           C  \\nATOM    881  O   MET A 111       5.903  17.900   7.098  1.00 11.11           O  \\nATOM    882  CB  MET A 111       3.494  17.806   9.211  1.00 14.36           C  \\nATOM    883  CG  MET A 111       3.864  18.110  10.627  1.00 17.41           C  \\nATOM    884  SD  MET A 111       3.231  16.825  11.758  1.00 18.36           S  \\nATOM    885  CE  MET A 111       1.528  17.128  11.498  1.00 17.25           C  \\nATOM    886  N   VAL A 112       6.091  19.706   8.404  1.00 10.73           N  \\nATOM    887  CA  VAL A 112       7.545  19.763   8.348  1.00 10.18           C  \\nATOM    888  C   VAL A 112       7.992  19.875   9.794  1.00  9.11           C  \\nATOM    889  O   VAL A 112       7.404  20.633  10.556  1.00 10.77           O  \\nATOM    890  CB  VAL A 112       8.010  21.024   7.542  1.00 11.80           C  \\nATOM    891  CG1 VAL A 112       9.528  21.213   7.558  1.00 13.24           C  \\nATOM    892  CG2 VAL A 112       7.542  20.843   6.104  1.00 14.56           C  \\nATOM    893  N   LEU A 113       9.032  19.167  10.207  1.00  8.47           N  \\nATOM    894  CA  LEU A 113       9.632  19.332  11.535  1.00  9.53           C  \\nATOM    895  C   LEU A 113      10.738  20.384  11.412  1.00  9.33           C  \\nATOM    896  O   LEU A 113      11.621  20.270  10.555  1.00  9.95           O  \\nATOM    897  CB  LEU A 113      10.214  17.983  12.017  1.00  9.56           C  \\nATOM    898  CG  LEU A 113      10.863  17.976  13.381  1.00 11.58           C  \\nATOM    899  CD1 LEU A 113       9.908  18.379  14.503  1.00 13.17           C  \\nATOM    900  CD2 LEU A 113      11.361  16.564  13.597  1.00 12.84           C  \\nATOM    901  N   VAL A 114      10.705  21.442  12.221  1.00  7.62           N  \\nATOM    902  CA  VAL A 114      11.673  22.541  12.109  1.00  7.64           C  \\nATOM    903  C   VAL A 114      12.436  22.667  13.441  1.00  7.65           C  \\nATOM    904  O   VAL A 114      11.872  22.842  14.541  1.00  9.71           O  \\nATOM    905  CB  VAL A 114      10.896  23.836  11.774  1.00  9.37           C  \\nATOM    906  CG1 VAL A 114      11.814  25.030  11.869  1.00 10.27           C  \\nATOM    907  CG2 VAL A 114      10.254  23.719  10.393  1.00  9.64           C  \\nATOM    908  N   GLY A 115      13.745  22.567  13.331  1.00  6.10           N  \\nATOM    909  CA  GLY A 115      14.603  22.742  14.476  1.00  6.69           C  \\nATOM    910  C   GLY A 115      15.120  24.163  14.407  1.00  8.47           C  \\nATOM    911  O   GLY A 115      16.007  24.439  13.594  1.00  9.16           O  \\nATOM    912  N   ASN A 116      14.593  25.089  15.192  1.00  8.01           N  \\nATOM    913  CA  ASN A 116      15.007  26.475  15.111  1.00  8.24           C  \\nATOM    914  C   ASN A 116      16.112  26.873  16.109  1.00  9.61           C  \\nATOM    915  O   ASN A 116      16.453  26.142  17.051  1.00  9.84           O  \\nATOM    916  CB  ASN A 116      13.721  27.307  15.275  1.00  8.01           C  \\nATOM    917  CG  ASN A 116      13.929  28.788  14.992  1.00  9.48           C  \\nATOM    918  OD1 ASN A 116      14.470  29.205  13.972  1.00  9.94           O  \\nATOM    919  ND2 ASN A 116      13.565  29.644  15.940  1.00 10.75           N  \\nATOM    920  N   LYS A 117      16.660  28.087  15.876  1.00 10.22           N  \\nATOM    921  CA  LYS A 117      17.735  28.715  16.674  1.00 12.36           C  \\nATOM    922  C   LYS A 117      19.094  28.021  16.450  1.00 13.39           C  \\nATOM    923  O   LYS A 117      19.977  27.940  17.324  1.00 11.29           O  \\nATOM    924  CB  LYS A 117      17.391  28.706  18.214  1.00 13.23           C  \\nATOM    925  CG  LYS A 117      16.026  29.272  18.611  1.00 12.90           C  \\nATOM    926  CD  LYS A 117      15.932  29.640  20.097  1.00 12.43           C  \\nATOM    927  CE  LYS A 117      14.551  30.217  20.380  1.00 10.55           C  \\nATOM    928  NZ  LYS A 117      14.491  30.647  21.772  1.00 11.10           N  \\nATOM    929  N   CYS A 118      19.254  27.555  15.188  1.00 13.69           N  \\nATOM    930  CA  CYS A 118      20.449  26.806  14.844  1.00 15.67           C  \\nATOM    931  C   CYS A 118      21.716  27.658  14.796  1.00 15.28           C  \\nATOM    932  O   CYS A 118      22.808  27.113  14.629  1.00 15.30           O  \\nATOM    933  CB  CYS A 118      20.246  26.064  13.492  1.00 17.74           C  \\nATOM    934  SG  CYS A 118      20.353  27.009  11.944  1.00 20.74           S  \\nATOM    935  N   ASP A 119      21.578  28.995  14.931  1.00 15.76           N  \\nATOM    936  CA  ASP A 119      22.722  29.904  15.083  1.00 16.09           C  \\nATOM    937  C   ASP A 119      23.426  29.778  16.445  1.00 17.19           C  \\nATOM    938  O   ASP A 119      24.589  30.178  16.553  1.00 18.76           O  \\nATOM    939  CB  ASP A 119      22.268  31.374  14.888  1.00 15.14           C  \\nATOM    940  CG  ASP A 119      21.153  31.785  15.835  1.00 15.81           C  \\nATOM    941  OD1 ASP A 119      20.030  31.294  15.662  1.00 13.87           O  \\nATOM    942  OD2 ASP A 119      21.407  32.596  16.721  1.00 14.85           O  \\nATOM    943  N   LEU A 120      22.732  29.248  17.480  1.00 16.86           N  \\nATOM    944  CA  LEU A 120      23.230  29.099  18.845  1.00 18.76           C  \\nATOM    945  C   LEU A 120      24.217  27.957  18.979  1.00 19.70           C  \\nATOM    946  O   LEU A 120      23.979  26.885  18.431  1.00 20.53           O  \\nATOM    947  CB  LEU A 120      22.103  28.829  19.831  1.00 17.65           C  \\nATOM    948  CG  LEU A 120      21.065  29.933  20.010  1.00 19.43           C  \\nATOM    949  CD1 LEU A 120      20.044  29.418  21.008  1.00 18.46           C  \\nATOM    950  CD2 LEU A 120      21.688  31.239  20.503  1.00 18.73           C  \\nATOM    951  N   ALA A 121      25.313  28.184  19.705  1.00 19.26           N  \\nATOM    952  CA  ALA A 121      26.303  27.143  19.887  1.00 20.48           C  \\nATOM    953  C   ALA A 121      25.942  26.210  21.036  1.00 21.30           C  \\nATOM    954  O   ALA A 121      26.403  25.069  21.067  1.00 24.04           O  \\nATOM    955  CB  ALA A 121      27.652  27.767  20.164  1.00 22.02           C  \\nATOM    956  N   ALA A 122      25.109  26.635  21.982  1.00 22.09           N  \\nATOM    957  CA  ALA A 122      24.683  25.797  23.094  1.00 23.88           C  \\nATOM    958  C   ALA A 122      23.578  24.811  22.720  1.00 23.77           C  \\nATOM    959  O   ALA A 122      22.492  24.835  23.301  1.00 27.82           O  \\nATOM    960  CB  ALA A 122      24.171  26.684  24.249  1.00 25.67           C  \\nATOM    961  N   ARG A 123      23.843  23.908  21.783  1.00 20.99           N  \\nATOM    962  CA  ARG A 123      22.890  22.905  21.311  1.00 19.81           C  \\nATOM    963  C   ARG A 123      22.619  21.761  22.283  1.00 19.56           C  \\nATOM    964  O   ARG A 123      23.590  21.176  22.774  1.00 20.01           O  \\nATOM    965  CB  ARG A 123      23.421  22.333  19.988  1.00 19.81           C  \\nATOM    966  CG  ARG A 123      22.662  21.166  19.398  1.00 21.43           C  \\nATOM    967  CD  ARG A 123      23.273  20.680  18.073  1.00 22.66           C  \\nATOM    968  NE  ARG A 123      23.050  21.590  16.956  1.00 22.61           N  \\nATOM    969  CZ  ARG A 123      22.019  21.487  16.093  1.00 20.95           C  \\nATOM    970  NH1 ARG A 123      21.045  20.568  16.181  1.00 16.36           N  \\nATOM    971  NH2 ARG A 123      21.989  22.370  15.098  1.00 19.65           N  \\nATOM    972  N   THR A 124      21.364  21.389  22.600  1.00 17.96           N  \\nATOM    973  CA  THR A 124      21.118  20.200  23.415  1.00 17.85           C  \\nATOM    974  C   THR A 124      20.413  19.068  22.656  1.00 17.08           C  \\nATOM    975  O   THR A 124      20.459  17.926  23.091  1.00 19.59           O  \\nATOM    976  CB  THR A 124      20.311  20.578  24.672  1.00 19.42           C  \\nATOM    977  OG1 THR A 124      19.109  21.230  24.277  1.00 20.50           O  \\nATOM    978  CG2 THR A 124      21.155  21.458  25.588  1.00 19.06           C  \\nATOM    979  N   VAL A 125      19.788  19.319  21.495  1.00 16.59           N  \\nATOM    980  CA  VAL A 125      19.133  18.324  20.647  1.00 16.09           C  \\nATOM    981  C   VAL A 125      20.032  18.080  19.413  1.00 16.61           C  \\nATOM    982  O   VAL A 125      20.180  18.980  18.573  1.00 14.07           O  \\nATOM    983  CB  VAL A 125      17.720  18.848  20.179  1.00 15.28           C  \\nATOM    984  CG1 VAL A 125      17.097  17.814  19.265  1.00 14.81           C  \\nATOM    985  CG2 VAL A 125      16.824  19.168  21.366  1.00 15.97           C  \\nATOM    986  N   GLU A 126      20.659  16.889  19.280  1.00 17.05           N  \\nATOM    987  CA  GLU A 126      21.487  16.568  18.124  1.00 19.28           C  \\nATOM    988  C   GLU A 126      20.631  16.496  16.876  1.00 18.51           C  \\nATOM    989  O   GLU A 126      19.511  15.976  16.880  1.00 17.68           O  \\nATOM    990  CB  GLU A 126      22.165  15.215  18.219  1.00 23.29           C  \\nATOM    991  CG  GLU A 126      23.244  15.057  19.265  1.00 30.86           C  \\nATOM    992  CD  GLU A 126      24.402  16.054  19.153  1.00 36.59           C  \\nATOM    993  OE1 GLU A 126      25.139  16.012  18.161  1.00 39.64           O  \\nATOM    994  OE2 GLU A 126      24.581  16.885  20.056  1.00 40.44           O  \\nATOM    995  N   SER A 127      21.167  16.985  15.755  1.00 17.73           N  \\nATOM    996  CA  SER A 127      20.426  16.938  14.493  1.00 16.72           C  \\nATOM    997  C   SER A 127      20.040  15.506  14.158  1.00 16.76           C  \\nATOM    998  O   SER A 127      18.931  15.245  13.690  1.00 17.01           O  \\nATOM    999  CB  SER A 127      21.283  17.518  13.371  1.00 17.67           C  \\nATOM   1000  OG  SER A 127      21.730  18.826  13.768  1.00 19.67           O  \\nATOM   1001  N   ARG A 128      20.916  14.558  14.466  1.00 16.72           N  \\nATOM   1002  CA  ARG A 128      20.667  13.154  14.145  1.00 20.94           C  \\nATOM   1003  C   ARG A 128      19.435  12.617  14.862  1.00 19.62           C  \\nATOM   1004  O   ARG A 128      18.641  11.882  14.278  1.00 18.81           O  \\nATOM   1005  CB  ARG A 128      21.910  12.319  14.510  1.00 24.31           C  \\nATOM   1006  CG  ARG A 128      21.969  11.035  13.692  1.00 30.88           C  \\nATOM   1007  CD  ARG A 128      23.381  10.403  13.536  1.00 35.88           C  \\nATOM   1008  NE  ARG A 128      24.384  10.996  12.621  1.00 39.57           N  \\nATOM   1009  CZ  ARG A 128      24.313  11.018  11.267  1.00 40.85           C  \\nATOM   1010  NH1 ARG A 128      25.399  11.388  10.577  1.00 40.83           N  \\nATOM   1011  NH2 ARG A 128      23.202  10.685  10.576  1.00 41.33           N  \\nATOM   1012  N   GLN A 129      19.209  13.016  16.119  1.00 18.27           N  \\nATOM   1013  CA  GLN A 129      18.032  12.519  16.787  1.00 18.53           C  \\nATOM   1014  C   GLN A 129      16.735  13.151  16.224  1.00 17.11           C  \\nATOM   1015  O   GLN A 129      15.691  12.486  16.098  1.00 15.48           O  \\nATOM   1016  CB  GLN A 129      18.229  12.766  18.306  1.00 22.36           C  \\nATOM   1017  CG  GLN A 129      18.041  14.133  18.911  1.00 24.78           C  \\nATOM   1018  CD  GLN A 129      18.283  14.123  20.411  1.00 27.48           C  \\nATOM   1019  OE1 GLN A 129      19.287  14.643  20.906  1.00 28.14           O  \\nATOM   1020  NE2 GLN A 129      17.367  13.497  21.152  1.00 27.21           N  \\nATOM   1021  N   ALA A 130      16.780  14.446  15.837  1.00 14.73           N  \\nATOM   1022  CA  ALA A 130      15.611  15.044  15.210  1.00 13.87           C  \\nATOM   1023  C   ALA A 130      15.342  14.449  13.826  1.00 13.30           C  \\nATOM   1024  O   ALA A 130      14.199  14.194  13.436  1.00 11.42           O  \\nATOM   1025  CB  ALA A 130      15.822  16.536  15.063  1.00 16.21           C  \\nATOM   1026  N   GLN A 131      16.401  14.189  13.057  1.00 14.61           N  \\nATOM   1027  CA  GLN A 131      16.267  13.529  11.739  1.00 16.01           C  \\nATOM   1028  C   GLN A 131      15.637  12.109  11.821  1.00 16.29           C  \\nATOM   1029  O   GLN A 131      14.777  11.708  11.017  1.00 14.37           O  \\nATOM   1030  CB  GLN A 131      17.673  13.507  11.116  1.00 18.12           C  \\nATOM   1031  CG  GLN A 131      17.702  13.017   9.689  1.00 22.19           C  \\nATOM   1032  CD  GLN A 131      16.889  13.922   8.801  1.00 21.61           C  \\nATOM   1033  OE1 GLN A 131      17.292  15.044   8.529  1.00 24.06           O  \\nATOM   1034  NE2 GLN A 131      15.714  13.485   8.370  1.00 23.37           N  \\nATOM   1035  N   ASP A 132      16.038  11.348  12.853  1.00 16.64           N  \\nATOM   1036  CA  ASP A 132      15.467  10.038  13.173  1.00 18.00           C  \\nATOM   1037  C   ASP A 132      13.984  10.175  13.484  1.00 17.28           C  \\nATOM   1038  O   ASP A 132      13.178   9.395  12.985  1.00 16.45           O  \\nATOM   1039  CB  ASP A 132      16.120   9.415  14.405  1.00 20.10           C  \\nATOM   1040  CG  ASP A 132      17.506   8.816  14.234  1.00 24.07           C  \\nATOM   1041  OD1 ASP A 132      18.120   8.459  15.243  1.00 28.35           O  \\nATOM   1042  OD2 ASP A 132      17.975   8.686  13.106  1.00 25.43           O  \\nATOM   1043  N   LEU A 133      13.614  11.168  14.312  1.00 15.58           N  \\nATOM   1044  CA  LEU A 133      12.213  11.390  14.624  1.00 16.28           C  \\nATOM   1045  C   LEU A 133      11.421  11.729  13.366  1.00 14.96           C  \\nATOM   1046  O   LEU A 133      10.345  11.181  13.128  1.00 15.09           O  \\nATOM   1047  CB  LEU A 133      12.052  12.536  15.647  1.00 17.06           C  \\nATOM   1048  CG  LEU A 133      10.617  12.781  16.136  1.00 18.79           C  \\nATOM   1049  CD1 LEU A 133      10.142  11.554  16.881  1.00 19.96           C  \\nATOM   1050  CD2 LEU A 133      10.544  14.013  17.020  1.00 18.45           C  \\nATOM   1051  N   ALA A 134      11.937  12.642  12.545  1.00 13.16           N  \\nATOM   1052  CA  ALA A 134      11.241  13.016  11.331  1.00 14.89           C  \\nATOM   1053  C   ALA A 134      11.092  11.821  10.414  1.00 16.19           C  \\nATOM   1054  O   ALA A 134      10.015  11.643   9.857  1.00 15.70           O  \\nATOM   1055  CB  ALA A 134      12.021  14.089  10.629  1.00 15.12           C  \\nATOM   1056  N   ARG A 135      12.143  10.995  10.279  1.00 17.00           N  \\nATOM   1057  CA  ARG A 135      12.088   9.828   9.438  1.00 19.17           C  \\nATOM   1058  C   ARG A 135      10.990   8.893   9.916  1.00 18.70           C  \\nATOM   1059  O   ARG A 135      10.269   8.352   9.078  1.00 20.29           O  \\nATOM   1060  CB  ARG A 135      13.425   9.091   9.456  1.00 22.48           C  \\nATOM   1061  CG  ARG A 135      13.499   8.297   8.155  1.00 29.02           C  \\nATOM   1062  CD  ARG A 135      14.619   7.270   7.988  1.00 33.22           C  \\nATOM   1063  NE  ARG A 135      14.117   5.888   8.050  1.00 38.80           N  \\nATOM   1064  CZ  ARG A 135      13.281   5.302   7.148  1.00 39.60           C  \\nATOM   1065  NH1 ARG A 135      12.889   4.044   7.384  1.00 40.80           N  \\nATOM   1066  NH2 ARG A 135      12.827   5.913   6.034  1.00 40.11           N  \\nATOM   1067  N   SER A 136      10.815   8.697  11.242  1.00 19.23           N  \\nATOM   1068  CA  SER A 136       9.757   7.841  11.770  1.00 19.12           C  \\nATOM   1069  C   SER A 136       8.374   8.432  11.528  1.00 19.61           C  \\nATOM   1070  O   SER A 136       7.398   7.690  11.522  1.00 21.01           O  \\nATOM   1071  CB  SER A 136       9.930   7.624  13.280  1.00 19.94           C  \\nATOM   1072  OG  SER A 136       9.459   8.686  14.119  1.00 23.21           O  \\nATOM   1073  N   TYR A 137       8.231   9.760  11.380  1.00 17.33           N  \\nATOM   1074  CA  TYR A 137       6.950  10.375  11.078  1.00 18.12           C  \\nATOM   1075  C   TYR A 137       6.683  10.404   9.570  1.00 18.22           C  \\nATOM   1076  O   TYR A 137       5.561  10.656   9.129  1.00 19.27           O  \\nATOM   1077  CB  TYR A 137       6.907  11.812  11.592  1.00 18.21           C  \\nATOM   1078  CG  TYR A 137       6.883  12.057  13.084  1.00 19.08           C  \\nATOM   1079  CD1 TYR A 137       6.619  11.026  13.955  1.00 20.51           C  \\nATOM   1080  CD2 TYR A 137       7.059  13.355  13.548  1.00 20.43           C  \\nATOM   1081  CE1 TYR A 137       6.510  11.285  15.306  1.00 21.89           C  \\nATOM   1082  CE2 TYR A 137       6.945  13.624  14.899  1.00 21.02           C  \\nATOM   1083  CZ  TYR A 137       6.662  12.574  15.763  1.00 22.13           C  \\nATOM   1084  OH  TYR A 137       6.476  12.782  17.112  1.00 22.16           O  \\nATOM   1085  N   GLY A 138       7.689  10.186   8.737  1.00 16.07           N  \\nATOM   1086  CA  GLY A 138       7.520  10.265   7.294  1.00 17.66           C  \\nATOM   1087  C   GLY A 138       7.476  11.725   6.810  1.00 18.01           C  \\nATOM   1088  O   GLY A 138       6.859  12.010   5.774  1.00 17.85           O  \\nATOM   1089  N   ILE A 139       8.128  12.662   7.540  1.00 14.16           N  \\nATOM   1090  CA  ILE A 139       8.117  14.065   7.145  1.00 13.49           C  \\nATOM   1091  C   ILE A 139       9.526  14.662   7.033  1.00 14.35           C  \\nATOM   1092  O   ILE A 139      10.494  14.152   7.610  1.00 16.19           O  \\nATOM   1093  CB  ILE A 139       7.292  14.896   8.161  1.00 14.40           C  \\nATOM   1094  CG1 ILE A 139       7.925  14.811   9.554  1.00 15.44           C  \\nATOM   1095  CG2 ILE A 139       5.821  14.422   8.104  1.00 14.07           C  \\nATOM   1096  CD1 ILE A 139       7.288  15.765  10.587  1.00 15.35           C  \\nATOM   1097  N   PRO A 140       9.706  15.767   6.308  1.00 13.55           N  \\nATOM   1098  CA  PRO A 140      10.984  16.448   6.205  1.00 13.39           C  \\nATOM   1099  C   PRO A 140      11.424  17.065   7.532  1.00 12.74           C  \\nATOM   1100  O   PRO A 140      10.601  17.530   8.310  1.00 11.42           O  \\nATOM   1101  CB  PRO A 140      10.767  17.498   5.127  1.00 13.47           C  \\nATOM   1102  CG  PRO A 140       9.514  17.084   4.410  1.00 14.09           C  \\nATOM   1103  CD  PRO A 140       8.686  16.467   5.519  1.00 13.14           C  \\nATOM   1104  N   TYR A 141      12.744  17.086   7.756  1.00 11.68           N  \\nATOM   1105  CA  TYR A 141      13.336  17.791   8.890  1.00 10.89           C  \\nATOM   1106  C   TYR A 141      14.175  18.945   8.326  1.00 10.09           C  \\nATOM   1107  O   TYR A 141      15.056  18.720   7.468  1.00 10.27           O  \\nATOM   1108  CB  TYR A 141      14.276  16.897   9.691  1.00 12.54           C  \\nATOM   1109  CG  TYR A 141      15.063  17.676  10.733  1.00 10.92           C  \\nATOM   1110  CD1 TYR A 141      14.350  18.396  11.664  1.00 11.75           C  \\nATOM   1111  CD2 TYR A 141      16.453  17.633  10.784  1.00 12.49           C  \\nATOM   1112  CE1 TYR A 141      14.997  19.074  12.670  1.00 12.85           C  \\nATOM   1113  CE2 TYR A 141      17.112  18.307  11.794  1.00 11.33           C  \\nATOM   1114  CZ  TYR A 141      16.369  19.016  12.728  1.00 12.61           C  \\nATOM   1115  OH  TYR A 141      16.944  19.646  13.813  1.00 13.82           O  \\nATOM   1116  N   ILE A 142      13.892  20.174   8.799  1.00  9.35           N  \\nATOM   1117  CA  ILE A 142      14.636  21.355   8.376  1.00  9.05           C  \\nATOM   1118  C   ILE A 142      15.153  22.163   9.551  1.00  9.25           C  \\nATOM   1119  O   ILE A 142      14.409  22.382  10.499  1.00 10.43           O  \\nATOM   1120  CB  ILE A 142      13.756  22.259   7.472  1.00  9.25           C  \\nATOM   1121  CG1 ILE A 142      13.523  21.470   6.180  1.00 10.51           C  \\nATOM   1122  CG2 ILE A 142      14.450  23.586   7.122  1.00  9.08           C  \\nATOM   1123  CD1 ILE A 142      12.719  22.127   5.028  1.00 13.16           C  \\nATOM   1124  N   GLU A 143      16.457  22.513   9.568  1.00  9.01           N  \\nATOM   1125  CA  GLU A 143      16.979  23.404  10.611  1.00  8.52           C  \\nATOM   1126  C   GLU A 143      16.978  24.864  10.193  1.00 10.31           C  \\nATOM   1127  O   GLU A 143      17.278  25.228   9.040  1.00 10.45           O  \\nATOM   1128  CB  GLU A 143      18.397  23.022  11.013  1.00 10.87           C  \\nATOM   1129  CG  GLU A 143      18.363  21.728  11.801  1.00 10.22           C  \\nATOM   1130  CD  GLU A 143      19.592  21.411  12.633  1.00 13.01           C  \\nATOM   1131  OE1 GLU A 143      20.616  22.082  12.521  1.00 13.65           O  \\nATOM   1132  OE2 GLU A 143      19.511  20.487  13.429  1.00 12.70           O  \\nATOM   1133  N   THR A 144      16.608  25.762  11.117  1.00  9.18           N  \\nATOM   1134  CA  THR A 144      16.422  27.169  10.754  1.00  9.71           C  \\nATOM   1135  C   THR A 144      17.014  28.076  11.789  1.00  8.66           C  \\nATOM   1136  O   THR A 144      17.259  27.698  12.927  1.00  9.13           O  \\nATOM   1137  CB  THR A 144      14.919  27.617  10.635  1.00  9.90           C  \\nATOM   1138  OG1 THR A 144      14.336  27.403  11.931  1.00  9.29           O  \\nATOM   1139  CG2 THR A 144      14.120  26.858   9.557  1.00  9.89           C  \\nATOM   1140  N   SER A 145      17.244  29.288  11.360  1.00 10.91           N  \\nATOM   1141  CA  SER A 145      17.558  30.377  12.290  1.00 12.65           C  \\nATOM   1142  C   SER A 145      16.758  31.602  11.849  1.00 10.37           C  \\nATOM   1143  O   SER A 145      16.930  32.094  10.727  1.00 11.76           O  \\nATOM   1144  CB  SER A 145      19.069  30.768  12.308  1.00 11.79           C  \\nATOM   1145  OG  SER A 145      19.248  31.931  13.125  1.00 12.12           O  \\nATOM   1146  N   ALA A 146      15.838  32.046  12.696  1.00  9.49           N  \\nATOM   1147  CA  ALA A 146      15.131  33.268  12.413  1.00 10.90           C  \\nATOM   1148  C   ALA A 146      16.070  34.455  12.553  1.00 13.53           C  \\nATOM   1149  O   ALA A 146      15.799  35.514  11.995  1.00 16.20           O  \\nATOM   1150  CB  ALA A 146      13.976  33.436  13.367  1.00 12.65           C  \\nATOM   1151  N   LYS A 147      17.183  34.302  13.283  1.00 14.35           N  \\nATOM   1152  CA  LYS A 147      18.150  35.351  13.486  1.00 16.25           C  \\nATOM   1153  C   LYS A 147      18.953  35.568  12.215  1.00 17.58           C  \\nATOM   1154  O   LYS A 147      19.124  36.724  11.825  1.00 20.13           O  \\nATOM   1155  CB  LYS A 147      19.068  34.970  14.617  1.00 17.55           C  \\nATOM   1156  CG  LYS A 147      20.003  36.125  14.958  1.00 22.83           C  \\nATOM   1157  CD  LYS A 147      20.581  35.932  16.341  1.00 26.44           C  \\nATOM   1158  CE  LYS A 147      21.683  36.949  16.507  1.00 30.79           C  \\nATOM   1159  NZ  LYS A 147      22.575  36.534  17.574  1.00 33.81           N  \\nATOM   1160  N   THR A 148      19.479  34.529  11.541  1.00 17.16           N  \\nATOM   1161  CA  THR A 148      20.248  34.735  10.300  1.00 15.93           C  \\nATOM   1162  C   THR A 148      19.422  34.607   9.013  1.00 17.72           C  \\nATOM   1163  O   THR A 148      19.926  34.962   7.945  1.00 18.47           O  \\nATOM   1164  CB  THR A 148      21.444  33.745  10.209  1.00 15.86           C  \\nATOM   1165  OG1 THR A 148      20.886  32.435  10.042  1.00 14.56           O  \\nATOM   1166  CG2 THR A 148      22.377  33.839  11.439  1.00 15.25           C  \\nATOM   1167  N   ARG A 149      18.176  34.108   9.100  1.00 15.18           N  \\nATOM   1168  CA  ARG A 149      17.268  33.815   7.994  1.00 19.20           C  \\nATOM   1169  C   ARG A 149      17.652  32.483   7.318  1.00 17.94           C  \\nATOM   1170  O   ARG A 149      17.031  32.079   6.332  1.00 20.28           O  \\nATOM   1171  CB  ARG A 149      17.251  34.964   6.937  1.00 21.40           C  \\nATOM   1172  CG  ARG A 149      15.929  35.106   6.188  1.00 27.00           C  \\nATOM   1173  CD  ARG A 149      15.946  35.904   4.863  1.00 27.99           C  \\nATOM   1174  NE  ARG A 149      14.707  35.687   4.092  1.00 29.40           N  \\nATOM   1175  CZ  ARG A 149      13.550  36.232   4.492  1.00 29.07           C  \\nATOM   1176  NH1 ARG A 149      12.412  36.015   3.828  1.00 29.56           N  \\nATOM   1177  NH2 ARG A 149      13.527  37.004   5.577  1.00 28.99           N  \\nATOM   1178  N   GLN A 150      18.621  31.701   7.838  1.00 15.80           N  \\nATOM   1179  CA  GLN A 150      18.926  30.402   7.273  1.00 16.36           C  \\nATOM   1180  C   GLN A 150      17.737  29.464   7.455  1.00 15.02           C  \\nATOM   1181  O   GLN A 150      17.119  29.351   8.504  1.00 13.57           O  \\nATOM   1182  CB  GLN A 150      20.152  29.729   7.937  1.00 23.05           C  \\nATOM   1183  CG  GLN A 150      20.578  28.448   7.162  1.00 29.37           C  \\nATOM   1184  CD  GLN A 150      21.604  27.530   7.827  1.00 34.84           C  \\nATOM   1185  OE1 GLN A 150      21.301  26.416   8.280  1.00 38.95           O  \\nATOM   1186  NE2 GLN A 150      22.857  27.967   7.883  1.00 35.22           N  \\nATOM   1187  N   GLY A 151      17.412  28.809   6.348  1.00 13.62           N  \\nATOM   1188  CA  GLY A 151      16.357  27.835   6.296  1.00 13.50           C  \\nATOM   1189  C   GLY A 151      14.943  28.395   6.329  1.00 12.36           C  \\nATOM   1190  O   GLY A 151      14.025  27.591   6.148  1.00 11.07           O  \\nATOM   1191  N   VAL A 152      14.706  29.702   6.510  1.00 12.59           N  \\nATOM   1192  CA  VAL A 152      13.339  30.183   6.715  1.00 14.23           C  \\nATOM   1193  C   VAL A 152      12.435  29.971   5.475  1.00 13.41           C  \\nATOM   1194  O   VAL A 152      11.352  29.362   5.577  1.00 12.94           O  \\nATOM   1195  CB  VAL A 152      13.430  31.691   7.167  1.00 12.57           C  \\nATOM   1196  CG1 VAL A 152      12.059  32.316   7.194  1.00 13.14           C  \\nATOM   1197  CG2 VAL A 152      14.044  31.787   8.543  1.00 12.69           C  \\nATOM   1198  N   GLU A 153      12.831  30.410   4.278  1.00 13.52           N  \\nATOM   1199  CA  GLU A 153      12.001  30.213   3.086  1.00 14.81           C  \\nATOM   1200  C   GLU A 153      11.857  28.756   2.742  1.00 11.88           C  \\nATOM   1201  O   GLU A 153      10.797  28.282   2.344  1.00 13.92           O  \\nATOM   1202  CB  GLU A 153      12.592  30.936   1.881  1.00 17.03           C  \\nATOM   1203  CG  GLU A 153      12.454  32.452   2.015  1.00 21.18           C  \\nATOM   1204  CD  GLU A 153      12.870  33.318   0.817  1.00 25.73           C  \\nATOM   1205  OE1 GLU A 153      13.232  32.781  -0.234  1.00 28.32           O  \\nATOM   1206  OE2 GLU A 153      12.822  34.550   0.936  1.00 26.64           O  \\nATOM   1207  N   ASP A 154      12.959  28.031   2.957  1.00 11.31           N  \\nATOM   1208  CA  ASP A 154      13.023  26.593   2.747  1.00 10.16           C  \\nATOM   1209  C   ASP A 154      11.964  25.828   3.561  1.00 10.90           C  \\nATOM   1210  O   ASP A 154      11.224  24.985   3.035  1.00 11.83           O  \\nATOM   1211  CB  ASP A 154      14.420  26.212   3.117  1.00 15.00           C  \\nATOM   1212  CG  ASP A 154      14.919  24.942   2.524  1.00 19.04           C  \\nATOM   1213  OD1 ASP A 154      14.121  24.195   1.938  1.00 22.96           O  \\nATOM   1214  OD2 ASP A 154      16.122  24.697   2.676  1.00 21.26           O  \\nATOM   1215  N   ALA A 155      11.854  26.107   4.881  1.00  8.83           N  \\nATOM   1216  CA  ALA A 155      10.841  25.450   5.716  1.00 10.00           C  \\nATOM   1217  C   ALA A 155       9.424  25.748   5.203  1.00 10.10           C  \\nATOM   1218  O   ALA A 155       8.653  24.814   4.978  1.00 11.14           O  \\nATOM   1219  CB  ALA A 155      10.923  25.933   7.192  1.00  9.50           C  \\nATOM   1220  N   PHE A 156       9.054  27.011   4.949  1.00 10.59           N  \\nATOM   1221  CA  PHE A 156       7.712  27.345   4.475  1.00 10.51           C  \\nATOM   1222  C   PHE A 156       7.453  26.825   3.069  1.00 11.05           C  \\nATOM   1223  O   PHE A 156       6.368  26.302   2.801  1.00 10.80           O  \\nATOM   1224  CB  PHE A 156       7.492  28.890   4.506  1.00 11.54           C  \\nATOM   1225  CG  PHE A 156       7.266  29.450   5.936  1.00 12.42           C  \\nATOM   1226  CD1 PHE A 156       8.319  29.958   6.707  1.00 12.00           C  \\nATOM   1227  CD2 PHE A 156       5.978  29.434   6.478  1.00 12.57           C  \\nATOM   1228  CE1 PHE A 156       8.065  30.434   8.001  1.00 12.45           C  \\nATOM   1229  CE2 PHE A 156       5.747  29.908   7.757  1.00 12.32           C  \\nATOM   1230  CZ  PHE A 156       6.782  30.406   8.513  1.00 10.18           C  \\nATOM   1231  N   TYR A 157       8.408  26.935   2.151  1.00  9.69           N  \\nATOM   1232  CA  TYR A 157       8.148  26.463   0.798  1.00 12.31           C  \\nATOM   1233  C   TYR A 157       8.091  24.945   0.719  1.00 10.42           C  \\nATOM   1234  O   TYR A 157       7.292  24.403  -0.047  1.00 11.35           O  \\nATOM   1235  CB  TYR A 157       9.233  26.978  -0.156  1.00 15.16           C  \\nATOM   1236  CG  TYR A 157       9.257  28.492  -0.367  1.00 18.73           C  \\nATOM   1237  CD1 TYR A 157       8.380  29.366   0.268  1.00 18.52           C  \\nATOM   1238  CD2 TYR A 157      10.211  28.971  -1.242  1.00 20.44           C  \\nATOM   1239  CE1 TYR A 157       8.466  30.724   0.025  1.00 22.04           C  \\nATOM   1240  CE2 TYR A 157      10.305  30.320  -1.488  1.00 22.70           C  \\nATOM   1241  CZ  TYR A 157       9.436  31.178  -0.852  1.00 23.03           C  \\nATOM   1242  OH  TYR A 157       9.572  32.528  -1.089  1.00 26.15           O  \\nATOM   1243  N   THR A 158       8.940  24.198   1.455  1.00 11.35           N  \\nATOM   1244  CA  THR A 158       8.836  22.727   1.522  1.00 10.11           C  \\nATOM   1245  C   THR A 158       7.440  22.347   2.064  1.00 11.11           C  \\nATOM   1246  O   THR A 158       6.811  21.408   1.558  1.00 11.43           O  \\nATOM   1247  CB  THR A 158       9.935  22.129   2.470  1.00 12.09           C  \\nATOM   1248  OG1 THR A 158      11.216  22.451   1.933  1.00 11.36           O  \\nATOM   1249  CG2 THR A 158       9.791  20.657   2.624  1.00 10.87           C  \\nATOM   1250  N   LEU A 159       6.888  23.056   3.072  1.00  9.25           N  \\nATOM   1251  CA  LEU A 159       5.533  22.754   3.506  1.00  9.93           C  \\nATOM   1252  C   LEU A 159       4.489  22.986   2.388  1.00 10.42           C  \\nATOM   1253  O   LEU A 159       3.560  22.185   2.272  1.00 11.22           O  \\nATOM   1254  CB  LEU A 159       5.171  23.608   4.711  1.00  9.03           C  \\nATOM   1255  CG  LEU A 159       3.789  23.318   5.269  1.00 10.96           C  \\nATOM   1256  CD1 LEU A 159       3.635  21.828   5.651  1.00  9.29           C  \\nATOM   1257  CD2 LEU A 159       3.580  24.223   6.469  1.00 13.71           C  \\nATOM   1258  N   VAL A 160       4.596  24.046   1.548  1.00 10.91           N  \\nATOM   1259  CA  VAL A 160       3.662  24.284   0.439  1.00  9.19           C  \\nATOM   1260  C   VAL A 160       3.803  23.137  -0.556  1.00  9.60           C  \\nATOM   1261  O   VAL A 160       2.781  22.638  -1.031  1.00 10.29           O  \\nATOM   1262  CB  VAL A 160       3.981  25.653  -0.211  1.00  8.50           C  \\nATOM   1263  CG1 VAL A 160       3.402  25.754  -1.622  1.00 10.56           C  \\nATOM   1264  CG2 VAL A 160       3.403  26.741   0.659  1.00  7.98           C  \\nATOM   1265  N   ARG A 161       5.033  22.667  -0.839  1.00  9.10           N  \\nATOM   1266  CA  ARG A 161       5.206  21.554  -1.767  1.00  9.82           C  \\nATOM   1267  C   ARG A 161       4.644  20.240  -1.215  1.00 10.44           C  \\nATOM   1268  O   ARG A 161       4.166  19.395  -1.984  1.00 12.88           O  \\nATOM   1269  CB  ARG A 161       6.679  21.398  -2.115  1.00  9.39           C  \\nATOM   1270  CG  ARG A 161       7.146  22.640  -2.816  1.00  9.70           C  \\nATOM   1271  CD  ARG A 161       8.594  22.398  -3.227  1.00 14.12           C  \\nATOM   1272  NE  ARG A 161       9.122  23.635  -3.780  1.00 14.15           N  \\nATOM   1273  CZ  ARG A 161      10.144  24.313  -3.257  1.00 12.45           C  \\nATOM   1274  NH1 ARG A 161      10.823  23.858  -2.204  1.00 13.24           N  \\nATOM   1275  NH2 ARG A 161      10.485  25.475  -3.819  1.00 13.52           N  \\nATOM   1276  N   GLU A 162       4.663  20.046   0.112  1.00 10.71           N  \\nATOM   1277  CA  GLU A 162       4.054  18.877   0.722  1.00 11.80           C  \\nATOM   1278  C   GLU A 162       2.522  18.887   0.568  1.00 11.20           C  \\nATOM   1279  O   GLU A 162       1.921  17.848   0.274  1.00 12.90           O  \\nATOM   1280  CB  GLU A 162       4.430  18.815   2.216  1.00 13.86           C  \\nATOM   1281  CG  GLU A 162       5.870  18.460   2.580  1.00 14.10           C  \\nATOM   1282  CD  GLU A 162       6.268  17.038   2.224  1.00 17.23           C  \\nATOM   1283  OE1 GLU A 162       5.571  16.116   2.627  1.00 19.29           O  \\nATOM   1284  OE2 GLU A 162       7.278  16.827   1.545  1.00 22.14           O  \\nATOM   1285  N   ILE A 163       1.869  20.045   0.731  1.00 10.13           N  \\nATOM   1286  CA  ILE A 163       0.426  20.175   0.542  1.00 10.58           C  \\nATOM   1287  C   ILE A 163       0.128  19.878  -0.933  1.00 13.43           C  \\nATOM   1288  O   ILE A 163      -0.796  19.118  -1.235  1.00 15.01           O  \\nATOM   1289  CB  ILE A 163      -0.015  21.590   0.891  1.00 11.33           C  \\nATOM   1290  CG1 ILE A 163       0.084  21.777   2.400  1.00 11.27           C  \\nATOM   1291  CG2 ILE A 163      -1.486  21.815   0.437  1.00 12.37           C  \\nATOM   1292  CD1 ILE A 163      -0.006  23.245   2.894  1.00 12.56           C  \\nATOM   1293  N   ARG A 164       0.904  20.456  -1.871  1.00 11.72           N  \\nATOM   1294  CA  ARG A 164       0.770  20.163  -3.283  1.00 13.64           C  \\nATOM   1295  C   ARG A 164       0.813  18.657  -3.626  1.00 17.97           C  \\nATOM   1296  O   ARG A 164       0.150  18.197  -4.559  1.00 18.12           O  \\nATOM   1297  CB  ARG A 164       1.897  20.887  -4.029  1.00 12.51           C  \\nATOM   1298  CG  ARG A 164       1.647  22.371  -4.120  1.00 12.49           C  \\nATOM   1299  CD  ARG A 164       2.801  22.888  -4.950  1.00 14.72           C  \\nATOM   1300  NE  ARG A 164       2.730  24.327  -5.113  1.00 12.82           N  \\nATOM   1301  CZ  ARG A 164       3.800  25.071  -5.405  1.00 13.29           C  \\nATOM   1302  NH1 ARG A 164       5.018  24.539  -5.527  1.00 11.56           N  \\nATOM   1303  NH2 ARG A 164       3.595  26.381  -5.572  1.00 12.55           N  \\nATOM   1304  N   GLN A 165       1.590  17.831  -2.935  1.00 20.23           N  \\nATOM   1305  CA  GLN A 165       1.673  16.435  -3.337  1.00 27.50           C  \\nATOM   1306  C   GLN A 165       0.782  15.499  -2.502  1.00 29.12           C  \\nATOM   1307  O   GLN A 165       0.826  14.274  -2.655  1.00 29.30           O  \\nATOM   1308  CB  GLN A 165       3.168  16.070  -3.286  1.00 30.34           C  \\nATOM   1309  CG  GLN A 165       4.018  16.904  -4.309  1.00 35.17           C  \\nATOM   1310  CD  GLN A 165       4.109  16.470  -5.796  1.00 37.71           C  \\nATOM   1311  OE1 GLN A 165       4.256  17.293  -6.719  1.00 36.70           O  \\nATOM   1312  NE2 GLN A 165       4.079  15.166  -6.094  1.00 38.13           N  \\nATOM   1313  N   HIS A 166      -0.068  16.075  -1.638  1.00 29.28           N  \\nATOM   1314  CA  HIS A 166      -0.982  15.296  -0.820  1.00 31.13           C  \\nATOM   1315  C   HIS A 166      -2.214  14.971  -1.665  1.00 33.30           C  \\nATOM   1316  O   HIS A 166      -2.630  15.786  -2.497  1.00 34.96           O  \\nATOM   1317  CB  HIS A 166      -1.420  16.090   0.428  1.00 30.89           C  \\nATOM   1318  CG  HIS A 166      -2.247  15.319   1.475  1.00 32.17           C  \\nATOM   1319  ND1 HIS A 166      -3.579  15.231   1.603  1.00 33.52           N  \\nATOM   1320  CD2 HIS A 166      -1.712  14.560   2.502  1.00 32.95           C  \\nATOM   1321  CE1 HIS A 166      -3.857  14.464   2.647  1.00 33.07           C  \\nATOM   1322  NE2 HIS A 166      -2.727  14.064   3.184  1.00 33.06           N  \\nATOM   1323  OXT HIS A 166      -2.763  13.888  -1.454  1.00 36.27           O  \\nTER    1324      HIS A 166                                                      \\nHETATM 1325 MG    MG A 168       4.771  33.860  19.213  1.00  9.42          MG  \\nHETATM 1326  PG  GNP A 167       5.113  32.223  21.994  1.00  9.66           P  \\nHETATM 1327  O1G GNP A 167       4.844  32.596  23.390  1.00  9.26           O  \\nHETATM 1328  O2G GNP A 167       4.203  32.910  21.112  1.00 11.20           O  \\nHETATM 1329  O3G GNP A 167       4.640  30.769  21.802  1.00 10.09           O  \\nHETATM 1330  N3B GNP A 167       6.701  32.303  21.621  1.00  8.18           N  \\nHETATM 1331  PB  GNP A 167       7.310  32.034  20.111  1.00  8.77           P  \\nHETATM 1332  O1B GNP A 167       7.468  30.605  19.791  1.00  8.35           O  \\nHETATM 1333  O2B GNP A 167       6.696  32.775  19.021  1.00  9.38           O  \\nHETATM 1334  O3A GNP A 167       8.792  32.643  20.152  1.00 10.15           O  \\nHETATM 1335  PA  GNP A 167       9.280  34.140  19.710  1.00 10.18           P  \\nHETATM 1336  O1A GNP A 167       9.263  34.268  18.220  1.00 10.07           O  \\nHETATM 1337  O2A GNP A 167       8.594  35.173  20.510  1.00  9.33           O  \\nHETATM 1338  O5' GNP A 167      10.826  34.056  20.132  1.00 11.46           O  \\nHETATM 1339  C5' GNP A 167      11.170  33.911  21.535  1.00 10.56           C  \\nHETATM 1340  C4' GNP A 167      12.587  34.445  21.811  1.00 11.96           C  \\nHETATM 1341  O4' GNP A 167      13.560  33.649  21.095  1.00 11.66           O  \\nHETATM 1342  C3' GNP A 167      12.813  35.910  21.359  1.00 12.06           C  \\nHETATM 1343  O3' GNP A 167      13.793  36.538  22.221  1.00 14.52           O  \\nHETATM 1344  C2' GNP A 167      13.325  35.679  19.947  1.00 11.90           C  \\nHETATM 1345  O2' GNP A 167      14.151  36.750  19.499  1.00 14.99           O  \\nHETATM 1346  C1' GNP A 167      14.242  34.478  20.156  1.00 12.90           C  \\nHETATM 1347  N9  GNP A 167      14.454  33.739  18.896  1.00 12.98           N  \\nHETATM 1348  C8  GNP A 167      13.518  33.262  17.993  1.00 11.76           C  \\nHETATM 1349  N7  GNP A 167      14.016  32.636  16.979  1.00 12.34           N  \\nHETATM 1350  C5  GNP A 167      15.413  32.694  17.200  1.00 12.36           C  \\nHETATM 1351  C6  GNP A 167      16.521  32.219  16.402  1.00 12.30           C  \\nHETATM 1352  O6  GNP A 167      16.507  31.591  15.338  1.00 12.16           O  \\nHETATM 1353  N1  GNP A 167      17.735  32.503  16.995  1.00 11.60           N  \\nHETATM 1354  C2  GNP A 167      17.909  33.082  18.236  1.00 13.91           C  \\nHETATM 1355  N2  GNP A 167      19.173  33.280  18.658  1.00 14.60           N  \\nHETATM 1356  N3  GNP A 167      16.884  33.498  18.996  1.00 13.01           N  \\nHETATM 1357  C4  GNP A 167      15.675  33.331  18.398  1.00 14.36           C  \\nHETATM 1358  O   HOH A 169      13.588  43.386  17.779  1.00 31.80           O  \\nHETATM 1359  O   HOH A 170       8.248  36.064  23.209  1.00 13.63           O  \\nHETATM 1360  O   HOH A 171       3.254  37.845  19.454  1.00 18.29           O  \\nHETATM 1361  O   HOH A 172       5.945  35.619  20.105  1.00 11.33           O  \\nHETATM 1362  O   HOH A 173       3.669  32.226  18.172  1.00  8.22           O  \\nHETATM 1363  O   HOH A 174       2.896  37.170  15.500  1.00 21.37           O  \\nHETATM 1364  O   HOH A 175       2.102  32.309  24.122  1.00 27.47           O  \\nHETATM 1365  O   HOH A 176      -1.425  37.650  19.667  1.00 24.98           O  \\nHETATM 1366  O   HOH A 177      -3.225  30.031  -8.217  1.00 23.57           O  \\nHETATM 1367  O   HOH A 178      16.678  26.321  26.916  1.00 36.70           O  \\nHETATM 1368  O   HOH A 179      -2.134  40.975   1.551  1.00 81.53           O  \\nHETATM 1369  O   HOH A 180       0.400  40.675  -0.629  1.00 24.66           O  \\nHETATM 1370  O   HOH A 181      10.944  44.793  16.009  1.00 48.47           O  \\nHETATM 1371  O   HOH A 182      17.207  42.570  16.786  1.00 43.23           O  \\nHETATM 1372  O   HOH A 183      14.999  45.167  19.922  1.00 39.52           O  \\nHETATM 1373  O   HOH A 184      11.598  37.755  13.202  1.00 11.72           O  \\nHETATM 1374  O   HOH A 185       6.958  38.922  18.146  1.00 26.17           O  \\nHETATM 1375  O   HOH A 186      14.078  32.033  27.772  1.00 57.73           O  \\nHETATM 1376  O   HOH A 187       9.126  37.906  19.595  1.00 32.42           O  \\nHETATM 1377  O   HOH A 188      -0.049  36.435  27.259  1.00 50.56           O  \\nHETATM 1378  O   HOH A 189       3.843  30.503  25.485  1.00 46.80           O  \\nHETATM 1379  O   HOH A 190       4.931  28.343  27.587  1.00 29.32           O  \\nHETATM 1380  O   HOH A 191       4.881  25.774  26.247  1.00 28.55           O  \\nHETATM 1381  O   HOH A 193       8.729  26.516  26.645  1.00 37.34           O  \\nHETATM 1382  O   HOH A 194      -5.515  23.731  -4.213  1.00 43.74           O  \\nHETATM 1383  O   HOH A 195      -7.006  26.635   0.729  1.00 25.77           O  \\nHETATM 1384  O   HOH A 196      -2.033  23.281  -4.754  1.00 36.89           O  \\nHETATM 1385  O   HOH A 197      -4.326  24.642  -0.854  1.00 28.59           O  \\nHETATM 1386  O   HOH A 198      -6.858  26.995  -2.808  1.00 28.83           O  \\nHETATM 1387  O   HOH A 200      -3.937  29.225   8.289  1.00 24.08           O  \\nHETATM 1388  O   HOH A 201      -8.934  24.037   2.795  1.00 34.16           O  \\nHETATM 1389  O   HOH A 202       1.444  26.302  19.318  1.00 12.37           O  \\nHETATM 1390  O   HOH A 203       2.031  24.632  27.307  1.00 50.09           O  \\nHETATM 1391  O   HOH A 204      15.125  28.341  23.174  1.00 13.88           O  \\nHETATM 1392  O   HOH A 205      12.808  31.541  25.056  1.00 46.88           O  \\nHETATM 1393  O   HOH A 206       8.329  34.933   0.543  1.00 53.21           O  \\nHETATM 1394  O   HOH A 207       0.760  42.969   2.336  1.00 35.42           O  \\nHETATM 1395  O   HOH A 208       1.270  40.298   7.071  1.00 76.61           O  \\nHETATM 1396  O   HOH A 209      -3.135  39.625   7.649  1.00 35.40           O  \\nHETATM 1397  O   HOH A 210       8.699  39.076   2.399  1.00 33.93           O  \\nHETATM 1398  O   HOH A 212      10.040  41.806   3.447  1.00 37.88           O  \\nHETATM 1399  O   HOH A 213      18.914  29.095   3.704  1.00 25.32           O  \\nHETATM 1400  O   HOH A 214      15.966  29.098   2.957  1.00 18.63           O  \\nHETATM 1401  O   HOH A 215      15.506  31.962   4.015  1.00 18.93           O  \\nHETATM 1402  O   HOH A 216      22.935  30.533  10.845  1.00 39.81           O  \\nHETATM 1403  O   HOH A 217       6.476  43.893  23.325  1.00 49.09           O  \\nHETATM 1404  O   HOH A 218      11.746  41.248  28.612  1.00 36.22           O  \\nHETATM 1405  O   HOH A 219      11.480  42.270  25.715  1.00 29.66           O  \\nHETATM 1406  O   HOH A 220       9.344  34.251  -8.278  1.00 36.61           O  \\nHETATM 1407  O   HOH A 221      -0.111  19.667  -8.066  1.00 44.76           O  \\nHETATM 1408  O   HOH A 222      -0.528  30.460 -10.943  1.00 24.29           O  \\nHETATM 1409  O   HOH A 223     -11.352  24.274  19.491  1.00 31.21           O  \\nHETATM 1410  O   HOH A 224      -4.293  30.642 -16.466  1.00127.55           O  \\nHETATM 1411  O   HOH A 225       3.466  29.729 -16.232  1.00 26.15           O  \\nHETATM 1412  O   HOH A 227      -4.288  18.610   4.198  1.00 34.77           O  \\nHETATM 1413  O   HOH A 228       4.626  16.154   5.151  1.00 14.96           O  \\nHETATM 1414  O   HOH A 229       5.681  11.069   2.580  1.00 59.76           O  \\nHETATM 1415  O   HOH A 230       1.173  14.021   3.406  1.00 32.15           O  \\nHETATM 1416  O   HOH A 231       7.935  22.484  20.504  1.00 10.86           O  \\nHETATM 1417  O   HOH A 232       3.071  10.004  14.546  1.00 23.48           O  \\nHETATM 1418  O   HOH A 233       2.437  13.381   8.401  1.00 17.75           O  \\nHETATM 1419  O   HOH A 234       4.779   7.500   6.154  1.00 71.09           O  \\nHETATM 1420  O   HOH A 235      -4.250   8.748   6.880  1.00 24.76           O  \\nHETATM 1421  O   HOH A 236     -13.381  22.830  17.669  1.00 39.09           O  \\nHETATM 1422  O   HOH A 237      16.969  39.052   8.276  1.00 50.25           O  \\nHETATM 1423  O   HOH A 238      -6.315   9.333  13.897  1.00 36.24           O  \\nHETATM 1424  O   HOH A 240      -0.160   4.395  12.468  1.00 38.79           O  \\nHETATM 1425  O   HOH A 241       1.295   7.931  12.335  1.00 49.03           O  \\nHETATM 1426  O   HOH A 242       4.067  11.551   6.968  1.00 46.97           O  \\nHETATM 1427  O   HOH A 243      -5.060   6.579  -0.216  1.00 50.14           O  \\nHETATM 1428  O   HOH A 244      18.164   7.151   5.573  1.00 71.67           O  \\nHETATM 1429  O   HOH A 245       2.068   8.904   7.636  1.00 45.86           O  \\nHETATM 1430  O   HOH A 246      24.661  23.134  12.757  1.00 34.69           O  \\nHETATM 1431  O   HOH A 247      21.051  23.714   6.239  1.00 24.64           O  \\nHETATM 1432  O   HOH A 248      25.855  31.137  20.712  1.00 34.53           O  \\nHETATM 1433  O   HOH A 249      24.868  30.890  26.223  1.00 48.47           O  \\nHETATM 1434  O   HOH A 250      26.967  20.447  19.448  1.00 47.77           O  \\nHETATM 1435  O   HOH A 252      28.453  14.196  14.602  1.00 39.78           O  \\nHETATM 1436  O   HOH A 253      19.639  16.256   9.760  1.00 37.54           O  \\nHETATM 1437  O   HOH A 254      16.629   7.383  10.526  1.00 60.90           O  \\nHETATM 1438  O   HOH A 255       4.972   7.888  14.276  1.00 54.95           O  \\nHETATM 1439  O   HOH A 256      14.184  14.684   5.643  1.00 31.40           O  \\nHETATM 1440  O   HOH A 257      11.766  17.254   1.278  1.00 41.82           O  \\nHETATM 1441  O   HOH A 258      16.274  16.365   6.342  1.00 17.29           O  \\nHETATM 1442  O   HOH A 259      18.346  24.095   6.620  1.00 17.36           O  \\nHETATM 1443  O   HOH A 260      22.479  35.073  19.694  1.00 43.96           O  \\nHETATM 1444  O   HOH A 261      20.580  31.747   4.906  1.00 59.46           O  \\nHETATM 1445  O   HOH A 262      18.196  25.514   4.261  1.00 28.75           O  \\nHETATM 1446  O   HOH A 263      12.857  20.183   1.149  1.00 21.79           O  \\nHETATM 1447  O   HOH A 264      19.514  15.912  24.943  1.00 55.52           O  \\nHETATM 1448  O   HOH A 265       8.925   7.579   5.621  1.00 77.94           O  \\nHETATM 1449  O   HOH A 266      17.622  21.835  26.964  1.00 27.48           O  \\nHETATM 1450  O   HOH A 267      24.258  19.786  14.155  1.00 20.48           O  \\nHETATM 1451  O   HOH A 268      23.806  15.180  15.136  1.00 19.86           O  \\nHETATM 1452  O   HOH A 269      21.478  14.557  11.134  1.00 29.66           O  \\nHETATM 1453  O   HOH A 270      17.293  45.621  24.753  1.00 39.18           O  \\nHETATM 1454  O   HOH A 271      -3.518  40.206  17.649  1.00 44.75           O  \\nHETATM 1455  O   HOH A 272       2.143  19.852  24.090  1.00 40.10           O  \\nHETATM 1456  O   HOH A 273       7.101  35.918  -4.704  1.00 31.76           O  \\nHETATM 1457  O   HOH A 274       5.462  38.756  -5.521  1.00 27.51           O  \\nHETATM 1458  O   HOH A 275       5.748  20.565  28.638  1.00 58.06           O  \\nHETATM 1459  O   HOH A 276       8.814  31.258  -8.964  1.00 23.79           O  \\nHETATM 1460  O   HOH A 277     -10.427  35.895   4.301  1.00 58.91           O  \\nHETATM 1461  O   HOH A 278      -8.713  28.579   2.333  1.00 69.36           O  \\nHETATM 1462  O   HOH A 279      24.472  24.237  17.316  1.00 32.14           O  \\nHETATM 1463  O   HOH A 281      16.968  32.701  21.815  1.00 23.13           O  \\nHETATM 1464  O   HOH A 282      19.541  34.346  21.581  1.00 53.25           O  \\nHETATM 1465  O   HOH A 283      24.847  23.384  26.621  1.00 61.18           O  \\nHETATM 1466  O   HOH A 284       1.409  11.113   2.873  1.00 44.87           O  \\nHETATM 1467  O   HOH A 285       1.556  11.488  -0.914  1.00 37.86           O  \\nHETATM 1468  O   HOH A 286       9.758  12.945   3.889  1.00 36.47           O  \\nHETATM 1469  O   HOH A 287       3.827  13.375   4.622  1.00 44.90           O  \\nHETATM 1470  O   HOH A 288      14.279   6.175  15.391  1.00 78.25           O  \\nHETATM 1471  O   HOH A 289       7.573  13.968   2.191  1.00 76.47           O  \\nHETATM 1472  O   HOH A 290       2.323  15.482   1.418  1.00 35.15           O  \\nHETATM 1473  O   HOH A 291      11.997  14.573   2.962  1.00 49.86           O  \\nHETATM 1474  O   HOH A 292      23.798  33.550  17.880  1.00 24.64           O  \\nHETATM 1475  O   HOH A 293      14.657  36.046  -0.571  1.00 51.86           O  \\nHETATM 1476  O   HOH A 294      24.456  25.366   4.004  1.00 53.23           O  \\nHETATM 1477  O   HOH A 296      -0.603  12.134  -4.185  1.00 52.38           O  \\nHETATM 1478  O   HOH A 297      -8.372  34.415  -4.471  1.00 74.53           O  \\nHETATM 1479  O   HOH A 300     -14.033  36.501   0.062  1.00 50.21           O  \\nHETATM 1480  O   HOH A 301     -17.102  35.735  -2.806  1.00 42.58           O  \\nHETATM 1481  O   HOH A 303     -17.383  40.542   1.431  1.00 77.16           O  \\nHETATM 1482  O   HOH A 304     -19.024  41.557  -3.411  1.00107.16           O  \\nHETATM 1483  O   HOH A 305      -9.876  37.804   0.839  1.00 90.65           O  \\nHETATM 1484  O   HOH A 306     -14.053  36.466   4.552  1.00 82.05           O  \\nHETATM 1485  O   HOH A 307     -14.183  39.558   0.339  1.00 58.66           O  \\nHETATM 1486  O   HOH A 309     -12.547  28.677   8.301  1.00 74.88           O  \\nHETATM 1487  O   HOH A 311     -14.787  32.746  -2.171  1.00 40.79           O  \\nHETATM 1488  O   HOH A 312      -6.916  30.740   3.095  1.00 36.53           O  \\nHETATM 1489  O   HOH A 315      -8.312  33.027   7.340  1.00 83.28           O  \\nHETATM 1490  O   HOH A 316      -6.280  35.646   7.624  1.00 75.97           O  \\nHETATM 1491  O   HOH A 317      -5.230  31.565   9.609  1.00 55.67           O  \\nHETATM 1492  O   HOH A 318       6.747  37.258   1.208  1.00 57.93           O  \\nHETATM 1493  O   HOH A 319       4.747  42.419  12.683  1.00 57.74           O  \\nHETATM 1494  O   HOH A 320       2.984  41.950  16.191  1.00 41.69           O  \\nHETATM 1495  O   HOH A 321       3.867  49.208  12.056  1.00 24.57           O  \\nHETATM 1496  O   HOH A 322       7.530  42.894  17.776  1.00 79.28           O  \\nHETATM 1497  O   HOH A 324      19.822  39.457  17.826  1.00 66.22           O  \\nHETATM 1498  O   HOH A 325      12.391  45.557  24.973  1.00 48.70           O  \\nHETATM 1499  O   HOH A 326      14.944  43.928  23.794  1.00 62.45           O  \\nHETATM 1500  O   HOH A 327     -12.984  24.008  12.563  1.00 40.31           O  \\nHETATM 1501  O   HOH A 330       0.572  41.133  19.277  1.00 50.77           O  \\nHETATM 1502  O   HOH A 331      -6.196  37.979  23.043  1.00 48.93           O  \\nHETATM 1503  O   HOH A 332      -3.526  30.410  19.065  1.00 21.26           O  \\nHETATM 1504  O   HOH A 333      -7.995  32.093  21.685  1.00 54.11           O  \\nHETATM 1505  O   HOH A 334      -8.483  28.891  20.063  1.00 70.37           O  \\nHETATM 1506  O   HOH A 335     -11.670  31.022  16.981  1.00 76.86           O  \\nHETATM 1507  O   HOH A 336     -11.082  34.450  18.910  1.00 43.07           O  \\nHETATM 1508  O   HOH A 339      -6.521  37.738  15.165  1.00 64.11           O  \\nHETATM 1509  O   HOH A 340     -10.829  35.877  15.629  1.00 58.36           O  \\nHETATM 1510  O   HOH A 341      -6.552  41.166  12.231  1.00 55.67           O  \\nHETATM 1511  O   HOH A 342      -1.242  39.671  12.185  1.00 61.04           O  \\nHETATM 1512  O   HOH A 343      -4.281  38.934  13.464  1.00 32.14           O  \\nHETATM 1513  O   HOH A 344       0.386  41.208  14.039  1.00 40.80           O  \\nHETATM 1514  O   HOH A 345       0.777  39.663  21.603  1.00 67.71           O  \\nHETATM 1515  O   HOH A 346      -7.879  34.848  19.044  1.00 52.60           O  \\nHETATM 1516  O   HOH A 347       1.121  32.594 -11.957  1.00 32.30           O  \\nHETATM 1517  O   HOH A 348       0.924  32.597 -14.716  1.00 48.34           O  \\nHETATM 1518  O   HOH A 349      -2.241  27.348  20.565  1.00 26.54           O  \\nHETATM 1519  O   HOH A 350       3.259  26.214  29.881  1.00 69.22           O  \\nHETATM 1520  O   HOH A 351      -8.318  34.095  28.392  1.00 81.96           O  \\nHETATM 1521  O   HOH A 352      -2.927  29.775  31.412  1.00 80.92           O  \\nHETATM 1522  O   HOH A 353     -10.410  32.034  25.972  1.00 60.17           O  \\nHETATM 1523  O   HOH A 354      -8.122  30.393  25.378  1.00 62.33           O  \\nHETATM 1524  O   HOH A 356      -6.138  18.882  24.644  1.00 49.26           O  \\nHETATM 1525  O   HOH A 358      -1.406  19.033  23.084  1.00 82.05           O  \\nHETATM 1526  O   HOH A 359       4.169   7.339  10.280  1.00 66.03           O  \\nHETATM 1527  O   HOH A 360      -9.761  30.165   7.702  1.00 46.15           O  \\nHETATM 1528  O   HOH A 361     -16.002  27.801  11.700  1.00 41.85           O  \\nHETATM 1529  O   HOH A 362      16.215  30.489  29.264  1.00 67.21           O  \\nHETATM 1530  O   HOH A 364      21.417  27.685  26.140  1.00 49.57           O  \\nHETATM 1531  O   HOH A 366      15.824  34.767  30.662  1.00 25.90           O  \\nHETATM 1532  O   HOH A 367      18.527  35.965  24.111  1.00 68.69           O  \\nHETATM 1533  O   HOH A 369      12.103  35.275  24.945  1.00 51.37           O  \\nHETATM 1534  O   HOH A 370      24.782  11.970  23.641  1.00 55.92           O  \\nHETATM 1535  O   HOH A 371      14.337  21.394  28.627  1.00 30.11           O  \\nHETATM 1536  O   HOH A 372      -1.367  12.600  23.053  1.00 44.43           O  \\nHETATM 1537  O   HOH A 373       7.060   8.567  21.416  1.00 47.99           O  \\nHETATM 1538  O   HOH A 374       7.639   8.040  16.420  1.00 38.76           O  \\nHETATM 1539  O   HOH A 376      -1.282  15.566  25.487  1.00 67.79           O  \\nHETATM 1540  O   HOH A 378      17.945  30.924  23.806  1.00 31.50           O  \\nHETATM 1541  O   HOH A 379      17.360  33.209  33.337  1.00 30.46           O  \\nHETATM 1542  O   HOH A 380       3.971   6.567  22.089  1.00 86.80           O  \\nHETATM 1543  O   HOH A 381      -0.065  10.022  26.928  1.00 61.86           O  \\nHETATM 1544  O   HOH A 382      -4.450  12.966  21.222  1.00 42.82           O  \\nHETATM 1545  O   HOH A 383      -2.584   6.719   5.938  1.00 40.64           O  \\nHETATM 1546  O   HOH A 384      25.567  31.936  12.845  1.00 73.17           O  \\nHETATM 1547  O   HOH A 385      24.446  29.383  22.970  1.00 36.61           O  \\nHETATM 1548  O   HOH A 386      24.865  38.167  18.227  1.00 60.74           O  \\nHETATM 1549  O   HOH A 387      23.135  35.012  14.543  1.00 59.19           O  \\nHETATM 1550  O   HOH A 388      19.325   9.556  18.093  1.00 92.18           O  \\nHETATM 1551  O   HOH A 389      25.326  12.873  16.251  1.00 49.27           O  \\nHETATM 1552  O   HOH A 390      20.219  10.405  10.956  1.00 63.05           O  \\nHETATM 1553  O   HOH A 392       9.452   5.453   8.249  1.00 47.81           O  \\nHETATM 1554  O   HOH A 394      10.139  12.252   0.254  1.00 29.37           O  \\nHETATM 1555  O   HOH A 395      28.004  37.416  13.466  1.00 52.29           O  \\nHETATM 1556  O   HOH A 396      24.283  36.829  12.307  1.00 64.74           O  \\nHETATM 1557  O   HOH A 397      13.748  34.031  -3.019  1.00 47.80           O  \\nHETATM 1558  O   HOH A 398      16.981  34.189   2.006  1.00 80.51           O  \\nHETATM 1559  O   HOH A 399      -4.426  24.223  -7.240  1.00 38.09           O  \\nHETATM 1560  O   HOH A 400      19.676  33.981   3.336  1.00 64.86           O  \\nHETATM 1561  O   HOH A 401      14.633  28.745  -0.317  1.00 66.25           O  \\nHETATM 1562  O   HOH A 402      -2.139  15.576  22.716  1.00 65.01           O  \\nHETATM 1563  O   HOH A 403       9.873  11.090  20.760  1.00 53.88           O  \\nHETATM 1564  O   HOH A 404      18.291  36.379  26.927  1.00 57.23           O  \\nHETATM 1565  O   HOH A 405       0.138   6.145  15.269  1.00 82.80           O  \\nHETATM 1566  O   HOH A 406      -5.515  38.166  18.689  1.00 48.28           O  \\nHETATM 1567  O   HOH A 407      -0.399   4.688   8.809  1.00 58.35           O  \\nHETATM 1568  O   HOH A 408       0.813  25.584  22.307  1.00 40.04           O  \\nCONECT  117 1325                                                                \\nCONECT  265 1325                                                                \\nCONECT 1325  117  265 1328 1333                                                 \\nCONECT 1325 1361 1362                                                           \\nCONECT 1326 1327 1328 1329 1330                                                 \\nCONECT 1327 1326                                                                \\nCONECT 1328 1325 1326                                                           \\nCONECT 1329 1326                                                                \\nCONECT 1330 1326 1331                                                           \\nCONECT 1331 1330 1332 1333 1334                                                 \\nCONECT 1332 1331                                                                \\nCONECT 1333 1325 1331                                                           \\nCONECT 1334 1331 1335                                                           \\nCONECT 1335 1334 1336 1337 1338                                                 \\nCONECT 1336 1335                                                                \\nCONECT 1337 1335                                                                \\nCONECT 1338 1335 1339                                                           \\nCONECT 1339 1338 1340                                                           \\nCONECT 1340 1339 1341 1342                                                      \\nCONECT 1341 1340 1346                                                           \\nCONECT 1342 1340 1343 1344                                                      \\nCONECT 1343 1342                                                                \\nCONECT 1344 1342 1345 1346                                                      \\nCONECT 1345 1344                                                                \\nCONECT 1346 1341 1344 1347                                                      \\nCONECT 1347 1346 1348 1357                                                      \\nCONECT 1348 1347 1349                                                           \\nCONECT 1349 1348 1350                                                           \\nCONECT 1350 1349 1351 1357                                                      \\nCONECT 1351 1350 1352 1353                                                      \\nCONECT 1352 1351                                                                \\nCONECT 1353 1351 1354                                                           \\nCONECT 1354 1353 1355 1356                                                      \\nCONECT 1355 1354                                                                \\nCONECT 1356 1354 1357                                                           \\nCONECT 1357 1347 1350 1356                                                      \\nCONECT 1361 1325                                                                \\nCONECT 1362 1325                                                                \\nMASTER      312    0    2    5    6    0    9    6 1567    1   38   13          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewergrid_16835771918631198[1][1].addModel(\"\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n   -3.3668   10.8519   -0.3788 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.6411   10.0411   -1.3780 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8036    8.9757   -0.7409 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.9662    9.5649    0.3292 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8533   10.1694    1.3858 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.5578   11.3266    0.6352 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6427   11.7527   -0.7698 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.0598    8.6821    0.9457 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.0953    8.6757    2.3938 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.7475    7.5578    3.0354 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.3157    7.0379    4.2591 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9245    5.9628    4.8605 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.0101    5.4058    4.1949 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.4261    5.9376    2.9775 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.8191    7.0126    2.3652 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.3828    7.6616    0.8538 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8466    8.0236    0.9100 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.1438    6.6658   -0.2672 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.3900    9.0555    0.4682 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.3087    9.4036    2.9194 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.3799    9.2270   -0.5372 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.6368   10.2707   -0.0818 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.4168   12.4190   -1.3348 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2851   12.2077    0.1321 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4937   11.2223    0.7148 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6561   11.2508    2.0487 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.2320   10.5501    3.1550 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6559   10.8204    4.4367 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5459   11.8421    4.6723 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.0052   12.5925    3.5567 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5634   12.2859    2.2980 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9103   12.8345    1.1310 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7360   14.0298    0.8624 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-4.26887\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n    4.7438    9.3941   -2.7765 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7809    8.3095   -3.0578 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.6564    8.2816   -2.0694 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1960    8.3109   -0.6904 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.9802    9.5804   -0.4847 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1704    9.4225   -1.4630 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.6464    9.2034   -3.2117 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.2256    8.1065    0.4180 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.0475    8.9487    0.3852 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.0962    8.5180    1.1554 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.4110    8.7525    0.7426 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.4990    8.3266    1.4658 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.2361    7.6459    2.6488 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.9202    7.4223    3.0441 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.1767    7.8421    2.3231 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.8059    7.5622    2.8622 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.0338    8.0908    4.2570 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.1263    6.0787    2.8175 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.8878    8.3151    1.7045 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.0271    9.7808   -0.1406 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8036    7.8661    1.7184 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3576   10.5980   -3.4401 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7706   10.8453   -5.3027 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5251   11.7468   -4.1448 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4221   11.6572   -3.3009 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4606   12.6614   -2.4084 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6902   13.1604   -1.3827 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0089   14.2967   -0.6736 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1541   14.9995   -0.9673 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9792   14.5149   -2.0174 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.6216   13.3846   -2.7017 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2305   12.8048   -3.7391 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.5426   13.1058   -4.3475 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-4.20857\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n   -3.4229   10.9599   -0.4989 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.5959   10.3486   -1.5595 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8204    9.1682   -1.0618 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.0981    9.5266    0.1806 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.0903    9.9266    1.2409 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.7226   11.2172    0.6636 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6633   11.9233   -0.7325 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.1342    8.5267    0.7125 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.4149    8.0293    2.0437 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.2093    6.7857    2.4314 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.4142    5.8643    3.2778 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.1717    4.6729    3.6318 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.4345    4.4175    3.1103 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.0400    5.3473    2.2697 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.4583    6.5425    1.9061 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.2571    7.6880    0.8704 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6199    8.0651    1.3970 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.3880    7.1247   -0.5336 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2144    9.0909    0.7219 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.0071    8.5190    2.6592 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.4036    9.5937   -0.1451 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.7130   10.3497   -0.4488 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.5841    9.1661    2.2332 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0134   10.5033    1.7420 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6097   11.0616    0.5328 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1114   12.3039    0.4255 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0561   13.3673   -0.4466 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6731   14.5760   -0.2146 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4012   14.7739    0.9353 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4862   13.6988    1.8602 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8517   12.5157    1.5935 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7727   11.4178    2.3493 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4731   11.0905    3.6083 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.74069\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n    6.8254    3.5394    8.6271 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1124    4.8243    8.7796 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6936    4.7429    8.3077 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6459    4.1462    6.9530 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2142    2.7519    6.9979 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.6997    2.9880    7.3668 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8355    3.6810    8.6168 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.3313    4.1563    6.2575 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.6130    5.4138    6.2896 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.5755    5.6093    5.3036 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2921    6.8612    4.7499 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.3178    7.0442    3.7984 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.3919    5.9158    3.4049 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.0960    4.6775    3.9679 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.8819    4.4841    4.9195 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2066    2.9239    5.6148 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.0359    2.3032    6.2046 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.7965    1.9950    4.5686 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.4595    3.1295    6.8254 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.8232    6.1070    6.9564 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9706    2.2588    6.9702 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.6641    2.7302    9.7925 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3919    4.0219   12.2555 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.8607    3.8901   12.0576 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4523    3.2881   10.9510 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.7859    3.2638   11.1162 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.8984    2.7888   10.4598 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.1778    2.8873   10.9589 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   11.4041    3.4922   12.1732 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.2817    3.9997   12.8812 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.0258    3.8747   12.3517 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8539    4.2384   12.8784 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.5744    5.0238   14.0980 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.68007\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n    4.1018    9.7254   -3.5001 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.0930    8.7277   -3.9120 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.1826    8.3464   -2.7859 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9810    7.9874   -1.5912 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8013    9.1748   -1.1598 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.7765    9.3795   -2.3455 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9005    9.7209   -4.1345 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.2436    7.4097   -0.4360 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9508    7.4167    0.8282 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.1612    7.2758    2.0296 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.4996    7.9090    3.2291 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.7385    7.7900    4.3670 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.6011    6.9961    4.2786 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.2806    6.3729    3.0757 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.0356    6.4899    1.9288 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.6131    5.6773    0.4509 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.4473    4.1915    0.6554 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.6656    6.2616   -0.1228 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.8642    6.0284   -0.7276 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.9298    7.5135    0.8670 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.5020    5.9443   -1.6774 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6041   11.0517   -3.6799 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5176   11.5731   -6.0080 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.6001   12.2738   -4.6981 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.7249   12.0599   -3.6374 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0257   12.9022   -2.6341 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5468   13.2344   -1.3872 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.0829   14.2338   -0.6065 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1609   14.9610   -1.0541 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.6879   14.6463   -2.3354 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.1203   13.6493   -3.0820 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4355   13.2413   -4.3137 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.5803   13.6222   -5.1663 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.61611\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n   -0.6465    7.6479    1.0206 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.9213    6.8194    2.2124 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.3723    6.8253    2.5826 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.8705    8.2185    2.6482 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.7202    8.8698    1.2984 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.1851    8.9175    1.0985 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.3367    7.9163    0.9808 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.2454    8.4188    3.1788 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.4494    9.6232    3.9574 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.8095   10.0793    4.1276 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1429   11.4347    4.2042 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4385   11.8693    4.3475 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4223   10.8902    4.4204 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0710    9.5453    4.3431 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7751    9.1006    4.1953 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3749    7.4101    4.1245 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9090    6.6569    5.3179 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9162    6.7952    2.8460 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.6292    7.2793    4.0103 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6920   10.1110    4.3544 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.3592    6.4004    3.5687 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.7784    6.8759   -0.1733 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.2653    7.1445   -0.0415 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.4468    7.5842   -1.2036 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.0638    7.4508   -1.2845 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.3552    7.8730   -2.4897 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.5297    7.9573   -3.2024 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.6024    8.4101   -4.5006 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.4652    8.8175   -5.1581 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.7678    8.7498   -4.4555 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.7963    8.2867   -3.1677 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.8482    8.1035   -2.3658 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2592    8.4950   -2.5615 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.54520\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n    3.6826    7.0817    0.3984 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4242    6.7685   -0.8404 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2179    7.9380   -1.3350 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3574    9.1420   -1.3963 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8441    9.4633   -0.0171 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9454    8.2470    0.3171 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9056    6.4347    0.5326 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9363   10.3527   -2.0374 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4211   11.6214   -1.5648 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2189   12.7977   -1.8232 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2740   13.8772   -0.9366 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.0471   14.9882   -1.1734 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7814   14.9994   -2.3534 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.7132   13.9155   -3.2244 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9440   12.7958   -2.9932 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8689   11.4713   -4.1172 S   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4779   11.9310   -5.5002 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2061   10.7544   -4.1743 O   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.7294   10.2977   -3.4835 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.5623   11.6796   -1.0871 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9310    9.3650   -3.8438 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4821    6.8078    1.5493 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1103    3.8375    1.9184 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0351    4.5525    2.6577 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7511    5.9072    2.5132 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7799    6.2537    3.3753 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.1036    7.3868    3.7667 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.1598    7.4029    4.7690 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.8384    6.2451    5.4380 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.5116    5.0521    5.0611 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.4483    5.0818    4.0635 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2147    4.0898    3.6034 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1681    2.6455    3.9107 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.47112\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n   -5.7173    4.7949    8.7097 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0079    4.3599   10.0914 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3343    4.8599   10.5742 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4437    6.3166   10.3299 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3416    6.5870    8.8516 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.8950    6.1504    8.5120 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.7164    4.7564    8.5174 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.6232    7.0112   10.9116 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.3117    7.9287   10.0269 N   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.6662    8.2920   10.3735 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -11.6459    8.5453    9.4091 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -12.9390    8.8729    9.7386 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -13.2390    8.9496   11.0936 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -12.2499    8.6947   12.0396 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.9519    8.3616   11.7183 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.7394    8.0719   12.9303 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.6159    9.2311   13.8884 O   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.0588    6.8026   13.6999 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.2243    7.7560   12.1045 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.8744    8.2899    9.2223 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6016    7.1984   12.6892 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2243    3.8513    7.7654 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.5508    2.6643    6.8726 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.3925    3.5419    6.0151 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6193    4.0732    6.4018 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1543    4.7542    5.3740 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3108    5.4409    5.0818 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.5763    5.9972    3.8506 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.6610    5.8914    2.8296 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4518    5.1932    3.0919 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.2309    4.6494    4.3284 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.1929    3.9240    4.7519 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9112    3.6333    4.0772 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.41646\\n\\n$$$$\\n\\n\\n\\n 33 37  0  0  0  0  0  0  0  0999 V2000\\n   -6.0669   11.4001    2.1105 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.7487   11.8823    0.7508 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0078   10.8589   -0.0529 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8573   10.3434    0.7243 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.3561    9.6889    1.9859 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.9778   10.8688    2.7735 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2471   12.1804    2.7422 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9061    9.4510    0.0094 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.3204    9.0185   -1.3096 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.2934    8.5259   -2.1980 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.3406    8.7206   -3.5814 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.3545    8.2682   -4.4247 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.2915    7.5918   -3.8380 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.2621    7.4068   -2.4585 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.2443    7.8606   -1.6050 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.1818    7.5902    0.1114 S   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.0263    6.1264    0.4433 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.0396    8.3793    0.7262 O   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.6468    8.2558    0.8102 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.2636    9.0587   -1.5891 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.5033    8.4995    1.7902 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3256   10.7258    2.1263 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.9632   12.8010    3.6641 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.5453   11.4915    4.2335 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8165   10.5307    3.5389 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6716    9.4383    4.3083 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.1329    8.1751    4.2157 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2113    7.2376    5.2211 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8473    7.5390    6.4027 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4138    8.8347    6.5397 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3190    9.7344    5.5126 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.8218   10.9687    5.4301 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.4795   11.7874    6.4691 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n 25 22  1  0  0  0\\n 22  1  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  1  0  0  0\\n  8  4  1  0  0  0\\n  8  9  1  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 14 15  2  0  0  0\\n 15 16  1  0  0  0\\n 16 17  2  0  0  0\\n 16 18  2  0  0  0\\n 16 19  1  0  0  0\\n  4  5  1  0  0  0\\n  5  6  1  0  0  0\\n 25 26  1  0  0  0\\n 26 27  2  0  0  0\\n 27 28  1  0  0  0\\n 28 29  2  0  0  0\\n 29 30  1  0  0  0\\n 30 31  2  0  0  0\\n 31 32  1  0  0  0\\n 32 33  1  0  0  0\\n 32 24  1  0  0  0\\n  6  1  1  0  0  0\\n 31 26  1  0  0  0\\n 19  8  1  0  0  0\\n 15 10  1  0  0  0\\n  1  7  1  0  0  0\\n  9 20  1  0  0  0\\n 19 21  1  0  0  0\\nM  CHG 1   1   1\\nM  END\\n> <minimizedAffinity>\\n-3.34270\\n\\n$$$$\\n\",\"sdf\");\n",
       "\tviewergrid_16835771918631198[1][1].setStyle({\"model\": 0},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewergrid_16835771918631198[1][1].addSurface(1,{\"opacity\": 0.5, \"color\": \"white\"},{\"model\": 0});\n",
       "\tviewergrid_16835771918631198[1][1].setStyle({\"model\": 1},{\"stick\": {}});\n",
       "\tviewergrid_16835771918631198[1][1].zoomTo();\n",
       "\tviewergrid_16835771918631198[0][0].render();\n",
       "\tviewergrid_16835771918631198[0][1].render();\n",
       "\tviewergrid_16835771918631198[1][0].render();\n",
       "\tviewergrid_16835771918631198[1][1].render();\n",
       "viewergrid_16835771918631198[1][1].render();\n",
       "viewergrid_16835771918631198[1][0].render();\n",
       "viewergrid_16835771918631198[0][1].render();\n",
       "viewergrid_16835771918631198[0][0].render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<py3Dmol.view at 0x7efbfcf8abd0>"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Load protein data\n",
    "with open('data/ras/ras_protein.pdb', 'r') as f:\n",
    "    chA_data = f.read()\n",
    "\n",
    "# Load ligand data\n",
    "ligands = []\n",
    "for i in [282, 309, 884, 214]:\n",
    "    with open(f'data/temp/docking_results_{i}.sdf', 'r') as f:\n",
    "        ligands.append(f.read())\n",
    "\n",
    "view = py3Dmol.view(width=1000, height=1000, viewergrid=(2, 2), linked=False)\n",
    "view.setViewStyle({'style': 'outline', 'color': 'black', 'width': 0.1})\n",
    "\n",
    "for i, ligand in enumerate(ligands):\n",
    "    viewer = (i // 2, i % 2)\n",
    "    view.addModel(chA_data, 'pdb', viewer=viewer)\n",
    "    view.addModel(ligand, 'sdf', viewer=viewer)\n",
    "\n",
    "    chA = {'model': 0}\n",
    "    chB = {'model': 1}\n",
    "    \n",
    "    view.setStyle(chA, {'cartoon': {'color': 'spectrum'}}, viewer=viewer)\n",
    "    view.addSurface(py3Dmol.VDW, {'opacity': 0.5, 'color': 'white'}, chA, viewer=viewer)\n",
    "    view.setStyle(chB, {'stick': {}}, viewer=viewer)\n",
    "\n",
    "    view.zoomTo(viewer=viewer)\n",
    "\n",
    "view.render()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1d350e04-4757-4a43-b770-13856f372d54",
   "metadata": {},
   "source": [
    "#### Highest Affinity Ligand"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "93e7b5fa-c85c-4164-a341-ac63faa4e0de",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16835771918828518\"  style=\"position: relative; width: 1000px; height: 1000px\">\n        <p id=\"3dmolwarning_16835771918828518\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n}\n\nvar viewer_16835771918828518 = null;\nvar warn = document.getElementById(\"3dmolwarning_16835771918828518\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nvar viewergrid_16835771918828518 = null;\nviewergrid_16835771918828518 = $3Dmol.createViewerGrid(document.getElementById(\"3dmolviewer_16835771918828518\"),{rows: 1, cols: 1, control_all: false},{backgroundColor:\"white\"});\nviewer_16835771918828518 = viewergrid_16835771918828518[0][0];\nviewergrid_16835771918828518[0][0].zoomTo();\tviewergrid_16835771918828518[0][0].setViewStyle({\"style\": \"outline\", \"color\": \"black\", \"width\": 0.1});\n\tviewergrid_16835771918828518[0][0].addModel(\"HEADER    ONCOGENE PROTEIN                        30-APR-90   5P21              \\nTITLE     REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION OF H-RAS   \\nTITLE    2 P21 AT 1.35 ANGSTROMS RESOLUTION: IMPLICATIONS FOR THE MECHANISM OF  \\nTITLE    3 GTP HYDROLYSIS                                                       \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: C-H-RAS P21 PROTEIN;                                       \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    ONCOGENE PROTEIN                                                      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    E.F.PAI,A.WITTINGHOFER,W.KABSCH                                       \\nREVDAT   6   29-NOV-17 5P21    1       HELIX                                    \\nREVDAT   5   24-FEB-09 5P21    1       VERSN                                    \\nREVDAT   4   01-APR-03 5P21    1       JRNL                                     \\nREVDAT   3   20-JUL-95 5P21    1       HET                                      \\nREVDAT   2   30-APR-94 5P21    1       FORMUL                                   \\nREVDAT   1   15-JAN-92 5P21    0                                                \\nJRNL        AUTH   E.F.PAI,U.KRENGEL,G.A.PETSKO,R.S.GOODY,W.KABSCH,             \\nJRNL        AUTH 2 A.WITTINGHOFER                                               \\nJRNL        TITL   REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION   \\nJRNL        TITL 2 OF H-RAS P21 AT 1.35 A RESOLUTION: IMPLICATIONS FOR THE      \\nJRNL        TITL 3 MECHANISM OF GTP HYDROLYSIS.                                 \\nJRNL        REF    EMBO J.                       V.   9  2351 1990              \\nJRNL        REFN                   ISSN 0261-4189                               \\nJRNL        PMID   2196171                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,S.C.ALMO,G.RAPP,K.WILSON,K.PETRATOS,A.LENTFER, \\nREMARK   1  AUTH 2 A.WITTINGHOFER,W.KABSCH,E.F.PAI,G.A.PETSKO,R.S.GOODY         \\nREMARK   1  TITL   TIME-RESOLVED X-RAY CRYSTALLOGRAPHIC STUDY OF THE            \\nREMARK   1  TITL 2 CONFORMATIONAL CHANGE IN HA-RAS P21 PROTEIN ON GTP           \\nREMARK   1  TITL 3 HYDROLYSIS                                                   \\nREMARK   1  REF    NATURE                        V. 345   309 1990              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   A.SCHERER,J.JOHN,R.LINKE,R.S.GOODY,A.WITTINGHOFER,E.F.PAI,   \\nREMARK   1  AUTH 2 K.C.HOLMES                                                   \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY ANALYSIS OF THE HUMAN  \\nREMARK   1  TITL 2 C-H-RAS-ONCOGENE PRODUCT P21 COMPLEXED WITH GTP ANALOGUES    \\nREMARK   1  REF    J.MOL.BIOL.                   V. 206   257 1989              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   E.F.PAI,W.KABSCH,U.KRENGEL,K.C.HOLMES,J.JOHN,A.WITTINGHOFER  \\nREMARK   1  TITL   STRUCTURE OF THE GUANINE-NUCLEOTIDE-BINDING DOMAIN OF THE    \\nREMARK   1  TITL 2 HA-RAS ONCOGENE PRODUCT P21 IN THE TRIPHOSPHATE CONFORMATION \\nREMARK   1  REF    NATURE                        V. 341   209 1989              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,G.RAPP,J.JOHN,A.WITTINGHOFER,E.F.PAI,R.S.GOODY \\nREMARK   1  TITL   BIOCHEMICAL AND CRYSTALLOGRAPHIC CHARACTERIZATION OF A       \\nREMARK   1  TITL 2 COMPLEX OF C-HA-RAS P21 AND CAGED GTP WITH FLASH PHOTOLYSIS  \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  86  7687 1989              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 6.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 26806                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.196                           \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1323                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 33                                      \\nREMARK   3   SOLVENT ATOMS            : 211                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.014                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 2.700                           \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  RESIDUES 61 - 65 (GLN - GLU - GLU - TYR - SER) ADOPT                \\nREMARK   3  SEVERAL CONFORMATIONS IN THE CRYSTAL.  THE COORDINATES              \\nREMARK   3  GIVEN APPROXIMATE ONE OF THESE.                                     \\nREMARK   4                                                                      \\nREMARK   4 5P21 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179747.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 38.93                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.01                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 32 2 1                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+2/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+1/3                                           \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z+1/3                                           \\nREMARK 290       6555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000      108.13333            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       54.06667            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       54.06667            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000      108.13333            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025  0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    ARG A  68   NE  -  CZ  -  NH1 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A  97   NE  -  CZ  -  NH1 ANGL. DEV. =  -3.4 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH1 ANGL. DEV. =   4.1 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500    ARG A 128   NE  -  CZ  -  NH2 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A 135   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500    TYR A 157   CB  -  CG  -  CD2 ANGL. DEV. =  -4.8 DEGREES          \\nREMARK 500    ARG A 164   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ASP A  33       75.86   -157.25                                   \\nREMARK 500    ILE A  36      -64.23   -100.39                                   \\nREMARK 500    GLU A  37      123.75   -171.30                                   \\nREMARK 500    SER A  65     -133.70     53.07                                   \\nREMARK 500    LYS A 117       33.80     71.60                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              MG A 168  MG                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 GNP A 167   O2G                                                    \\nREMARK 620 2 GNP A 167   O2B  95.0                                              \\nREMARK 620 3 HOH A 173   O    87.6  91.7                                        \\nREMARK 620 4 SER A  17   OG  172.9  91.0  88.6                                  \\nREMARK 620 5 THR A  35   OG1  91.3 173.4  90.5  82.8                            \\nREMARK 620 6 HOH A 172   O    97.3  88.0 175.0  86.5  89.3                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE MG A 168                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE GNP A 167                 \\nDBREF  5P21 A    1   166  UNP    P01112   RASH_HUMAN       1    166             \\nSEQRES   1 A  166  MET THR GLU TYR LYS LEU VAL VAL VAL GLY ALA GLY GLY          \\nSEQRES   2 A  166  VAL GLY LYS SER ALA LEU THR ILE GLN LEU ILE GLN ASN          \\nSEQRES   3 A  166  HIS PHE VAL ASP GLU TYR ASP PRO THR ILE GLU ASP SER          \\nSEQRES   4 A  166  TYR ARG LYS GLN VAL VAL ILE ASP GLY GLU THR CYS LEU          \\nSEQRES   5 A  166  LEU ASP ILE LEU ASP THR ALA GLY GLN GLU GLU TYR SER          \\nSEQRES   6 A  166  ALA MET ARG ASP GLN TYR MET ARG THR GLY GLU GLY PHE          \\nSEQRES   7 A  166  LEU CYS VAL PHE ALA ILE ASN ASN THR LYS SER PHE GLU          \\nSEQRES   8 A  166  ASP ILE HIS GLN TYR ARG GLU GLN ILE LYS ARG VAL LYS          \\nSEQRES   9 A  166  ASP SER ASP ASP VAL PRO MET VAL LEU VAL GLY ASN LYS          \\nSEQRES  10 A  166  CYS ASP LEU ALA ALA ARG THR VAL GLU SER ARG GLN ALA          \\nSEQRES  11 A  166  GLN ASP LEU ALA ARG SER TYR GLY ILE PRO TYR ILE GLU          \\nSEQRES  12 A  166  THR SER ALA LYS THR ARG GLN GLY VAL GLU ASP ALA PHE          \\nSEQRES  13 A  166  TYR THR LEU VAL ARG GLU ILE ARG GLN HIS                      \\nHET     MG  A 168       1                                                       \\nHET    GNP  A 167      32                                                       \\nHETNAM      MG MAGNESIUM ION                                                    \\nHETNAM     GNP PHOSPHOAMINOPHOSPHONIC ACID-GUANYLATE ESTER                      \\nFORMUL   2   MG    MG 2+                                                        \\nFORMUL   3  GNP    C10 H17 N6 O13 P3                                            \\nFORMUL   4  HOH   *211(H2 O)                                                    \\nHELIX    1  HA LYS A   16  GLN A   25  1                                  10    \\nHELIX    2  HB SER A   65  THR A   74  1RES 65-67 IN 3/10 CONF            10    \\nHELIX    3  HC THR A   87  VAL A  103  1BREAK AT ASP 92                   17    \\nHELIX    4  HD SER A  127  TYR A  137  1                                  11    \\nHELIX    5  HE VAL A  152  ARG A  164  1                                  13    \\nSHEET    1  S1 6 GLU A  37  ILE A  46  0                                        \\nSHEET    2  S1 6 GLU A  49  THR A  58 -1  O  LEU A  53   N  LYS A  42           \\nSHEET    3  S1 6 THR A   2  VAL A   9  1  N  LEU A   6   O  ASP A  54           \\nSHEET    4  S1 6 GLY A  77  ALA A  83  1  N  LEU A  79   O  VAL A   7           \\nSHEET    5  S1 6 MET A 111  ASN A 116  1  N  VAL A 114   O  CYS A  80           \\nSHEET    6  S1 6 TYR A 141  GLU A 143  1  N  ILE A 142   O  LEU A 113           \\nLINK        MG    MG A 168                 O2G GNP A 167     1555   1555  2.20  \\nLINK        MG    MG A 168                 O2B GNP A 167     1555   1555  2.22  \\nLINK        MG    MG A 168                 O   HOH A 173     1555   1555  2.23  \\nLINK        MG    MG A 168                 OG  SER A  17     1555   1555  2.26  \\nLINK        MG    MG A 168                 OG1 THR A  35     1555   1555  2.26  \\nLINK        MG    MG A 168                 O   HOH A 172     1555   1555  2.30  \\nSITE     1 AC1  5 SER A  17  THR A  35  GNP A 167  HOH A 172                    \\nSITE     2 AC1  5 HOH A 173                                                     \\nSITE     1 AC2 28 GLY A  12  GLY A  13  VAL A  14  GLY A  15                    \\nSITE     2 AC2 28 LYS A  16  SER A  17  ALA A  18  PHE A  28                    \\nSITE     3 AC2 28 VAL A  29  ASP A  30  TYR A  32  PRO A  34                    \\nSITE     4 AC2 28 THR A  35  GLY A  60  ASN A 116  LYS A 117                    \\nSITE     5 AC2 28 ASP A 119  SER A 145  ALA A 146  LYS A 147                    \\nSITE     6 AC2 28  MG A 168  HOH A 170  HOH A 172  HOH A 173                    \\nSITE     7 AC2 28 HOH A 175  HOH A 187  HOH A 281  HOH A 366                    \\nCRYST1   40.300   40.300  162.200  90.00  90.00 120.00 P 32 2 1      6          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.024814  0.014326  0.000000        0.00000                         \\nSCALE2      0.000000  0.028653  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.006165        0.00000                         \\nATOM      1  N   MET A   1      -7.186  32.862  -6.632  1.00 13.92           N  \\nATOM      2  CA  MET A   1      -5.921  32.127  -6.684  1.00 17.10           C  \\nATOM      3  C   MET A   1      -5.845  31.045  -5.621  1.00 16.74           C  \\nATOM      4  O   MET A   1      -6.600  31.083  -4.636  1.00 16.60           O  \\nATOM      5  CB  MET A   1      -4.673  33.058  -6.502  1.00 17.00           C  \\nATOM      6  CG  MET A   1      -4.633  33.804  -5.188  1.00 18.08           C  \\nATOM      7  SD  MET A   1      -3.115  34.766  -4.908  1.00 19.73           S  \\nATOM      8  CE  MET A   1      -3.019  35.851  -6.310  1.00 20.41           C  \\nATOM      9  N   THR A   2      -4.910  30.095  -5.820  1.00 14.70           N  \\nATOM     10  CA  THR A   2      -4.661  29.030  -4.859  1.00 16.92           C  \\nATOM     11  C   THR A   2      -4.230  29.655  -3.547  1.00 14.78           C  \\nATOM     12  O   THR A   2      -3.338  30.501  -3.537  1.00 13.93           O  \\nATOM     13  CB  THR A   2      -3.557  28.107  -5.401  1.00 18.03           C  \\nATOM     14  OG1 THR A   2      -4.059  27.565  -6.629  1.00 20.42           O  \\nATOM     15  CG2 THR A   2      -3.193  26.983  -4.443  1.00 18.51           C  \\nATOM     16  N   GLU A   3      -4.911  29.290  -2.472  1.00 14.80           N  \\nATOM     17  CA  GLU A   3      -4.504  29.730  -1.135  1.00 17.71           C  \\nATOM     18  C   GLU A   3      -4.012  28.553  -0.278  1.00 17.33           C  \\nATOM     19  O   GLU A   3      -4.520  27.427  -0.395  1.00 17.54           O  \\nATOM     20  CB  GLU A   3      -5.655  30.402  -0.432  1.00 20.67           C  \\nATOM     21  CG  GLU A   3      -6.213  31.631  -1.186  1.00 25.56           C  \\nATOM     22  CD  GLU A   3      -7.128  32.563  -0.369  1.00 28.46           C  \\nATOM     23  OE1 GLU A   3      -7.578  32.216   0.728  1.00 27.72           O  \\nATOM     24  OE2 GLU A   3      -7.375  33.680  -0.827  1.00 32.49           O  \\nATOM     25  N   TYR A   4      -2.962  28.744   0.542  1.00 14.50           N  \\nATOM     26  CA  TYR A   4      -2.425  27.654   1.358  1.00 12.29           C  \\nATOM     27  C   TYR A   4      -2.534  28.103   2.807  1.00 12.52           C  \\nATOM     28  O   TYR A   4      -2.016  29.143   3.200  1.00 12.11           O  \\nATOM     29  CB  TYR A   4      -0.975  27.385   1.026  1.00 11.46           C  \\nATOM     30  CG  TYR A   4      -0.726  26.846  -0.387  1.00 12.38           C  \\nATOM     31  CD1 TYR A   4      -0.297  27.698  -1.395  1.00 13.00           C  \\nATOM     32  CD2 TYR A   4      -0.933  25.501  -0.649  1.00 14.09           C  \\nATOM     33  CE1 TYR A   4      -0.070  27.208  -2.674  1.00 13.89           C  \\nATOM     34  CE2 TYR A   4      -0.711  24.993  -1.917  1.00 14.65           C  \\nATOM     35  CZ  TYR A   4      -0.279  25.859  -2.917  1.00 14.84           C  \\nATOM     36  OH  TYR A   4      -0.084  25.357  -4.191  1.00 15.07           O  \\nATOM     37  N   LYS A   5      -3.274  27.305   3.568  1.00 13.07           N  \\nATOM     38  CA  LYS A   5      -3.554  27.569   4.970  1.00 12.96           C  \\nATOM     39  C   LYS A   5      -2.556  26.818   5.852  1.00 12.74           C  \\nATOM     40  O   LYS A   5      -2.645  25.611   6.092  1.00 11.58           O  \\nATOM     41  CB  LYS A   5      -4.972  27.131   5.281  1.00 17.65           C  \\nATOM     42  CG  LYS A   5      -6.053  27.833   4.446  1.00 23.49           C  \\nATOM     43  CD  LYS A   5      -7.186  28.310   5.335  1.00 29.10           C  \\nATOM     44  CE  LYS A   5      -6.655  29.309   6.403  1.00 33.68           C  \\nATOM     45  NZ  LYS A   5      -6.597  28.739   7.755  1.00 36.52           N  \\nATOM     46  N   LEU A   6      -1.543  27.574   6.297  1.00 13.08           N  \\nATOM     47  CA  LEU A   6      -0.418  27.011   7.064  1.00 10.34           C  \\nATOM     48  C   LEU A   6      -0.578  27.382   8.535  1.00  9.68           C  \\nATOM     49  O   LEU A   6      -1.034  28.483   8.863  1.00 11.31           O  \\nATOM     50  CB  LEU A   6       0.910  27.571   6.536  1.00 11.34           C  \\nATOM     51  CG  LEU A   6       1.137  27.565   5.017  1.00 11.62           C  \\nATOM     52  CD1 LEU A   6       2.515  28.119   4.720  1.00 13.60           C  \\nATOM     53  CD2 LEU A   6       0.973  26.171   4.475  1.00 12.19           C  \\nATOM     54  N   VAL A   7      -0.196  26.486   9.432  1.00  9.93           N  \\nATOM     55  CA  VAL A   7      -0.333  26.787  10.852  1.00 10.06           C  \\nATOM     56  C   VAL A   7       1.033  26.513  11.495  1.00 10.37           C  \\nATOM     57  O   VAL A   7       1.597  25.441  11.252  1.00  9.51           O  \\nATOM     58  CB  VAL A   7      -1.474  25.884  11.446  1.00 10.97           C  \\nATOM     59  CG1 VAL A   7      -1.610  26.163  12.943  1.00 11.39           C  \\nATOM     60  CG2 VAL A   7      -2.809  26.161  10.777  1.00 10.14           C  \\nATOM     61  N   VAL A   8       1.576  27.455  12.295  1.00  8.35           N  \\nATOM     62  CA  VAL A   8       2.878  27.262  12.944  1.00  6.92           C  \\nATOM     63  C   VAL A   8       2.609  26.896  14.405  1.00  8.84           C  \\nATOM     64  O   VAL A   8       1.930  27.663  15.092  1.00  9.21           O  \\nATOM     65  CB  VAL A   8       3.684  28.565  12.799  1.00  8.08           C  \\nATOM     66  CG1 VAL A   8       5.111  28.324  13.261  1.00  9.52           C  \\nATOM     67  CG2 VAL A   8       3.729  29.011  11.358  1.00  8.43           C  \\nATOM     68  N   VAL A   9       3.024  25.701  14.881  1.00  8.73           N  \\nATOM     69  CA  VAL A   9       2.779  25.234  16.245  1.00  8.41           C  \\nATOM     70  C   VAL A   9       4.075  24.806  16.919  1.00  8.21           C  \\nATOM     71  O   VAL A   9       5.093  24.551  16.287  1.00 10.97           O  \\nATOM     72  CB  VAL A   9       1.824  24.028  16.286  1.00  9.74           C  \\nATOM     73  CG1 VAL A   9       0.545  24.491  15.680  1.00 11.76           C  \\nATOM     74  CG2 VAL A   9       2.349  22.810  15.535  1.00 10.31           C  \\nATOM     75  N   GLY A  10       4.048  24.707  18.238  1.00  7.34           N  \\nATOM     76  CA  GLY A  10       5.231  24.336  19.009  1.00  7.98           C  \\nATOM     77  C   GLY A  10       5.192  25.053  20.355  1.00  8.31           C  \\nATOM     78  O   GLY A  10       4.417  25.994  20.595  1.00  9.89           O  \\nATOM     79  N   ALA A  11       6.099  24.638  21.226  1.00  9.31           N  \\nATOM     80  CA  ALA A  11       6.121  25.135  22.597  1.00  9.50           C  \\nATOM     81  C   ALA A  11       6.440  26.625  22.711  1.00  9.66           C  \\nATOM     82  O   ALA A  11       6.998  27.250  21.806  1.00  9.46           O  \\nATOM     83  CB  ALA A  11       7.145  24.306  23.378  1.00  9.85           C  \\nATOM     84  N   GLY A  12       6.111  27.227  23.847  1.00 10.36           N  \\nATOM     85  CA  GLY A  12       6.406  28.632  24.054  1.00  9.30           C  \\nATOM     86  C   GLY A  12       7.892  28.950  23.964  1.00 10.23           C  \\nATOM     87  O   GLY A  12       8.785  28.249  24.466  1.00 10.57           O  \\nATOM     88  N   GLY A  13       8.159  30.064  23.266  1.00 11.49           N  \\nATOM     89  CA  GLY A  13       9.511  30.602  23.139  1.00 11.29           C  \\nATOM     90  C   GLY A  13      10.436  29.950  22.108  1.00 12.22           C  \\nATOM     91  O   GLY A  13      11.624  30.321  22.060  1.00 13.62           O  \\nATOM     92  N   VAL A  14       9.961  28.991  21.286  1.00  9.23           N  \\nATOM     93  CA  VAL A  14      10.848  28.327  20.317  1.00 10.38           C  \\nATOM     94  C   VAL A  14      11.169  29.168  19.076  1.00 10.47           C  \\nATOM     95  O   VAL A  14      12.108  28.841  18.353  1.00 12.07           O  \\nATOM     96  CB  VAL A  14      10.246  26.970  19.856  1.00  9.98           C  \\nATOM     97  CG1 VAL A  14      10.187  26.074  21.120  1.00  9.93           C  \\nATOM     98  CG2 VAL A  14       8.866  27.127  19.200  1.00  9.52           C  \\nATOM     99  N   GLY A  15      10.423  30.264  18.834  1.00  9.61           N  \\nATOM    100  CA  GLY A  15      10.666  31.160  17.706  1.00  9.70           C  \\nATOM    101  C   GLY A  15       9.615  31.061  16.626  1.00  8.82           C  \\nATOM    102  O   GLY A  15       9.915  31.403  15.491  1.00  8.99           O  \\nATOM    103  N   LYS A  16       8.395  30.591  16.916  1.00  8.18           N  \\nATOM    104  CA  LYS A  16       7.331  30.521  15.913  1.00  8.44           C  \\nATOM    105  C   LYS A  16       7.060  31.899  15.309  1.00  9.01           C  \\nATOM    106  O   LYS A  16       7.020  32.044  14.086  1.00  8.25           O  \\nATOM    107  CB  LYS A  16       6.014  30.000  16.529  1.00  9.27           C  \\nATOM    108  CG  LYS A  16       6.132  28.585  17.086  1.00  9.11           C  \\nATOM    109  CD  LYS A  16       4.820  28.077  17.726  1.00  9.31           C  \\nATOM    110  CE  LYS A  16       4.351  28.934  18.888  1.00 10.05           C  \\nATOM    111  NZ  LYS A  16       5.221  28.829  20.051  1.00  9.64           N  \\nATOM    112  N   SER A  17       6.913  32.942  16.135  1.00  7.74           N  \\nATOM    113  CA  SER A  17       6.661  34.295  15.649  1.00  8.78           C  \\nATOM    114  C   SER A  17       7.854  34.859  14.919  1.00  9.73           C  \\nATOM    115  O   SER A  17       7.653  35.446  13.868  1.00  9.70           O  \\nATOM    116  CB  SER A  17       6.360  35.264  16.769  1.00  8.22           C  \\nATOM    117  OG  SER A  17       5.083  34.853  17.204  1.00  8.79           O  \\nATOM    118  N   ALA A  18       9.072  34.687  15.444  1.00  8.95           N  \\nATOM    119  CA  ALA A  18      10.253  35.199  14.769  1.00  8.95           C  \\nATOM    120  C   ALA A  18      10.438  34.576  13.374  1.00  8.54           C  \\nATOM    121  O   ALA A  18      10.832  35.254  12.430  1.00  8.75           O  \\nATOM    122  CB  ALA A  18      11.521  34.931  15.640  1.00 10.11           C  \\nATOM    123  N   LEU A  19      10.136  33.284  13.176  1.00  8.31           N  \\nATOM    124  CA  LEU A  19      10.225  32.667  11.837  1.00  9.30           C  \\nATOM    125  C   LEU A  19       9.204  33.281  10.884  1.00 10.98           C  \\nATOM    126  O   LEU A  19       9.534  33.652   9.751  1.00 10.47           O  \\nATOM    127  CB  LEU A  19       9.938  31.165  11.885  1.00  9.41           C  \\nATOM    128  CG  LEU A  19      11.035  30.275  12.487  1.00 11.80           C  \\nATOM    129  CD1 LEU A  19      10.491  28.866  12.723  1.00 10.43           C  \\nATOM    130  CD2 LEU A  19      12.256  30.288  11.554  1.00 13.42           C  \\nATOM    131  N   THR A  20       7.950  33.419  11.335  1.00 10.72           N  \\nATOM    132  CA  THR A  20       6.888  34.015  10.529  1.00 11.87           C  \\nATOM    133  C   THR A  20       7.173  35.492  10.161  1.00 12.65           C  \\nATOM    134  O   THR A  20       7.012  35.917   9.009  1.00 12.60           O  \\nATOM    135  CB  THR A  20       5.547  33.888  11.312  1.00 13.43           C  \\nATOM    136  OG1 THR A  20       5.299  32.509  11.544  1.00 12.67           O  \\nATOM    137  CG2 THR A  20       4.346  34.410  10.518  1.00 14.64           C  \\nATOM    138  N   ILE A  21       7.615  36.326  11.098  1.00 10.48           N  \\nATOM    139  CA  ILE A  21       7.945  37.704  10.795  1.00 12.96           C  \\nATOM    140  C   ILE A  21       9.197  37.831   9.927  1.00 14.26           C  \\nATOM    141  O   ILE A  21       9.237  38.691   9.042  1.00 15.83           O  \\nATOM    142  CB  ILE A  21       8.090  38.451  12.121  1.00 12.57           C  \\nATOM    143  CG1 ILE A  21       6.691  38.551  12.737  1.00 13.24           C  \\nATOM    144  CG2 ILE A  21       8.631  39.852  11.910  1.00 13.40           C  \\nATOM    145  CD1 ILE A  21       6.636  39.185  14.129  1.00 16.46           C  \\nATOM    146  N   GLN A  22      10.206  36.979  10.092  1.00 13.58           N  \\nATOM    147  CA  GLN A  22      11.330  36.983   9.179  1.00 14.91           C  \\nATOM    148  C   GLN A  22      10.817  36.729   7.736  1.00 17.29           C  \\nATOM    149  O   GLN A  22      11.120  37.485   6.792  1.00 15.91           O  \\nATOM    150  CB  GLN A  22      12.270  35.930   9.736  1.00 15.90           C  \\nATOM    151  CG  GLN A  22      13.686  35.878   9.230  1.00 19.36           C  \\nATOM    152  CD  GLN A  22      14.372  37.217   9.091  1.00 18.08           C  \\nATOM    153  OE1 GLN A  22      14.160  37.984   8.143  1.00 19.23           O  \\nATOM    154  NE2 GLN A  22      15.224  37.492  10.050  1.00 17.40           N  \\nATOM    155  N   LEU A  23       9.944  35.718   7.523  1.00 15.95           N  \\nATOM    156  CA  LEU A  23       9.381  35.475   6.194  1.00 18.85           C  \\nATOM    157  C   LEU A  23       8.539  36.622   5.636  1.00 18.21           C  \\nATOM    158  O   LEU A  23       8.695  36.958   4.469  1.00 19.03           O  \\nATOM    159  CB  LEU A  23       8.498  34.235   6.196  1.00 18.10           C  \\nATOM    160  CG  LEU A  23       7.916  33.797   4.863  1.00 18.64           C  \\nATOM    161  CD1 LEU A  23       8.921  32.994   4.030  1.00 18.03           C  \\nATOM    162  CD2 LEU A  23       6.680  32.991   5.183  1.00 21.34           C  \\nATOM    163  N   ILE A  24       7.634  37.197   6.428  1.00 19.25           N  \\nATOM    164  CA  ILE A  24       6.682  38.198   5.953  1.00 20.23           C  \\nATOM    165  C   ILE A  24       7.316  39.590   5.922  1.00 21.89           C  \\nATOM    166  O   ILE A  24       7.125  40.318   4.943  1.00 22.93           O  \\nATOM    167  CB  ILE A  24       5.399  38.222   6.881  1.00 21.20           C  \\nATOM    168  CG1 ILE A  24       4.664  36.907   6.821  1.00 21.27           C  \\nATOM    169  CG2 ILE A  24       4.423  39.307   6.446  1.00 20.00           C  \\nATOM    170  CD1 ILE A  24       4.128  36.442   5.475  1.00 23.98           C  \\nATOM    171  N   GLN A  25       8.074  39.998   6.953  1.00 21.07           N  \\nATOM    172  CA  GLN A  25       8.520  41.380   7.066  1.00 22.09           C  \\nATOM    173  C   GLN A  25      10.003  41.559   6.962  1.00 20.75           C  \\nATOM    174  O   GLN A  25      10.495  42.677   7.054  1.00 20.26           O  \\nATOM    175  CB  GLN A  25       8.039  41.968   8.398  1.00 23.96           C  \\nATOM    176  CG  GLN A  25       6.548  42.276   8.381  1.00 28.69           C  \\nATOM    177  CD  GLN A  25       5.931  42.574   9.744  1.00 30.92           C  \\nATOM    178  OE1 GLN A  25       6.537  43.143  10.657  1.00 31.89           O  \\nATOM    179  NE2 GLN A  25       4.664  42.169   9.867  1.00 32.35           N  \\nATOM    180  N   ASN A  26      10.764  40.482   6.795  1.00 22.13           N  \\nATOM    181  CA  ASN A  26      12.198  40.642   6.628  1.00 24.67           C  \\nATOM    182  C   ASN A  26      12.936  41.362   7.734  1.00 23.62           C  \\nATOM    183  O   ASN A  26      13.603  42.372   7.521  1.00 25.10           O  \\nATOM    184  CB  ASN A  26      12.522  41.369   5.315  1.00 27.68           C  \\nATOM    185  CG  ASN A  26      12.942  40.351   4.288  1.00 32.80           C  \\nATOM    186  OD1 ASN A  26      12.135  39.699   3.602  1.00 33.77           O  \\nATOM    187  ND2 ASN A  26      14.268  40.203   4.237  1.00 34.26           N  \\nATOM    188  N   HIS A  27      12.764  40.842   8.945  1.00 19.95           N  \\nATOM    189  CA  HIS A  27      13.593  41.266  10.057  1.00 17.99           C  \\nATOM    190  C   HIS A  27      13.441  40.299  11.215  1.00 16.84           C  \\nATOM    191  O   HIS A  27      12.411  39.623  11.344  1.00 14.72           O  \\nATOM    192  CB  HIS A  27      13.241  42.680  10.551  1.00 19.80           C  \\nATOM    193  CG  HIS A  27      11.926  42.908  11.258  1.00 20.77           C  \\nATOM    194  ND1 HIS A  27      11.936  43.314  12.510  1.00 22.17           N  \\nATOM    195  CD2 HIS A  27      10.674  42.768  10.729  1.00 22.32           C  \\nATOM    196  CE1 HIS A  27      10.657  43.430  12.785  1.00 23.86           C  \\nATOM    197  NE2 HIS A  27       9.905  43.110  11.738  1.00 23.97           N  \\nATOM    198  N   PHE A  28      14.516  40.222  12.012  1.00 15.44           N  \\nATOM    199  CA  PHE A  28      14.528  39.413  13.223  1.00 13.89           C  \\nATOM    200  C   PHE A  28      14.026  40.196  14.440  1.00 15.33           C  \\nATOM    201  O   PHE A  28      14.528  41.255  14.786  1.00 16.80           O  \\nATOM    202  CB  PHE A  28      15.938  38.906  13.470  1.00 12.73           C  \\nATOM    203  CG  PHE A  28      16.083  38.150  14.786  1.00 13.35           C  \\nATOM    204  CD1 PHE A  28      15.354  36.990  15.012  1.00 13.80           C  \\nATOM    205  CD2 PHE A  28      16.974  38.615  15.740  1.00 14.54           C  \\nATOM    206  CE1 PHE A  28      15.534  36.310  16.194  1.00 14.45           C  \\nATOM    207  CE2 PHE A  28      17.147  37.925  16.920  1.00 14.61           C  \\nATOM    208  CZ  PHE A  28      16.420  36.768  17.137  1.00 14.65           C  \\nATOM    209  N   VAL A  29      12.970  39.697  15.089  1.00 17.22           N  \\nATOM    210  CA  VAL A  29      12.467  40.313  16.294  1.00 19.11           C  \\nATOM    211  C   VAL A  29      13.127  39.606  17.476  1.00 19.36           C  \\nATOM    212  O   VAL A  29      13.245  38.376  17.605  1.00 18.14           O  \\nATOM    213  CB  VAL A  29      10.894  40.205  16.412  1.00 22.03           C  \\nATOM    214  CG1 VAL A  29      10.334  40.869  15.143  1.00 19.66           C  \\nATOM    215  CG2 VAL A  29      10.377  38.776  16.631  1.00 23.15           C  \\nATOM    216  N   ASP A  30      13.615  40.484  18.324  1.00 20.14           N  \\nATOM    217  CA  ASP A  30      14.229  40.019  19.518  1.00 24.04           C  \\nATOM    218  C   ASP A  30      13.676  40.746  20.721  1.00 25.93           C  \\nATOM    219  O   ASP A  30      14.244  41.709  21.238  1.00 29.26           O  \\nATOM    220  CB  ASP A  30      15.719  40.203  19.443  1.00 27.10           C  \\nATOM    221  CG  ASP A  30      16.389  39.731  20.732  1.00 32.46           C  \\nATOM    222  OD1 ASP A  30      15.878  38.859  21.464  1.00 34.28           O  \\nATOM    223  OD2 ASP A  30      17.458  40.264  21.002  1.00 36.46           O  \\nATOM    224  N   GLU A  31      12.472  40.347  21.076  1.00 26.14           N  \\nATOM    225  CA  GLU A  31      11.922  40.804  22.318  1.00 28.72           C  \\nATOM    226  C   GLU A  31      10.870  39.836  22.749  1.00 26.20           C  \\nATOM    227  O   GLU A  31      10.466  38.931  22.014  1.00 23.14           O  \\nATOM    228  CB  GLU A  31      11.337  42.210  22.204  1.00 33.25           C  \\nATOM    229  CG  GLU A  31      10.350  42.618  21.152  1.00 40.08           C  \\nATOM    230  CD  GLU A  31      10.572  44.107  20.887  1.00 44.71           C  \\nATOM    231  OE1 GLU A  31      10.930  44.444  19.755  1.00 47.50           O  \\nATOM    232  OE2 GLU A  31      10.420  44.928  21.806  1.00 47.89           O  \\nATOM    233  N   TYR A  32      10.550  40.044  24.028  1.00 22.26           N  \\nATOM    234  CA  TYR A  32       9.597  39.196  24.685  1.00 18.83           C  \\nATOM    235  C   TYR A  32       8.204  39.688  24.317  1.00 17.53           C  \\nATOM    236  O   TYR A  32       7.794  40.792  24.676  1.00 18.45           O  \\nATOM    237  CB  TYR A  32       9.909  39.259  26.199  1.00 16.84           C  \\nATOM    238  CG  TYR A  32       9.076  38.309  27.052  1.00 15.64           C  \\nATOM    239  CD1 TYR A  32       9.598  37.096  27.433  1.00 16.17           C  \\nATOM    240  CD2 TYR A  32       7.814  38.686  27.459  1.00 15.54           C  \\nATOM    241  CE1 TYR A  32       8.845  36.251  28.239  1.00 17.86           C  \\nATOM    242  CE2 TYR A  32       7.051  37.861  28.257  1.00 14.40           C  \\nATOM    243  CZ  TYR A  32       7.572  36.635  28.652  1.00 16.47           C  \\nATOM    244  OH  TYR A  32       6.821  35.789  29.485  1.00 13.59           O  \\nATOM    245  N   ASP A  33       7.461  38.841  23.604  1.00 13.43           N  \\nATOM    246  CA  ASP A  33       6.128  39.200  23.141  1.00 14.00           C  \\nATOM    247  C   ASP A  33       5.262  37.955  22.854  1.00 13.56           C  \\nATOM    248  O   ASP A  33       5.024  37.586  21.681  1.00 12.47           O  \\nATOM    249  CB  ASP A  33       6.271  40.072  21.864  1.00 15.92           C  \\nATOM    250  CG  ASP A  33       4.993  40.654  21.269  1.00 18.19           C  \\nATOM    251  OD1 ASP A  33       3.923  40.631  21.879  1.00 21.46           O  \\nATOM    252  OD2 ASP A  33       5.074  41.164  20.167  1.00 20.34           O  \\nATOM    253  N   PRO A  34       4.752  37.283  23.903  1.00 12.04           N  \\nATOM    254  CA  PRO A  34       3.949  36.070  23.788  1.00 11.09           C  \\nATOM    255  C   PRO A  34       2.720  36.248  22.895  1.00 11.76           C  \\nATOM    256  O   PRO A  34       1.953  37.220  23.006  1.00 12.63           O  \\nATOM    257  CB  PRO A  34       3.590  35.711  25.219  1.00 12.50           C  \\nATOM    258  CG  PRO A  34       4.643  36.418  26.033  1.00 12.14           C  \\nATOM    259  CD  PRO A  34       4.817  37.717  25.311  1.00 12.82           C  \\nATOM    260  N   THR A  35       2.554  35.282  21.972  1.00 11.00           N  \\nATOM    261  CA  THR A  35       1.421  35.320  21.033  1.00 11.54           C  \\nATOM    262  C   THR A  35       0.140  34.746  21.639  1.00 13.51           C  \\nATOM    263  O   THR A  35       0.174  33.820  22.450  1.00 12.45           O  \\nATOM    264  CB  THR A  35       1.730  34.518  19.751  1.00 11.34           C  \\nATOM    265  OG1 THR A  35       2.900  35.134  19.233  1.00 11.20           O  \\nATOM    266  CG2 THR A  35       0.706  34.611  18.637  1.00 13.10           C  \\nATOM    267  N   ILE A  36      -0.996  35.332  21.233  1.00 15.48           N  \\nATOM    268  CA  ILE A  36      -2.308  34.792  21.567  1.00 19.42           C  \\nATOM    269  C   ILE A  36      -2.663  34.079  20.274  1.00 17.36           C  \\nATOM    270  O   ILE A  36      -2.749  32.863  20.256  1.00 18.29           O  \\nATOM    271  CB  ILE A  36      -3.353  35.914  21.879  1.00 21.30           C  \\nATOM    272  CG1 ILE A  36      -2.930  36.823  23.072  1.00 24.32           C  \\nATOM    273  CG2 ILE A  36      -4.670  35.207  22.136  1.00 21.54           C  \\nATOM    274  CD1 ILE A  36      -2.494  36.167  24.413  1.00 24.70           C  \\nATOM    275  N   GLU A  37      -2.818  34.822  19.181  1.00 19.35           N  \\nATOM    276  CA  GLU A  37      -3.059  34.221  17.889  1.00 21.47           C  \\nATOM    277  C   GLU A  37      -2.957  35.362  16.897  1.00 23.77           C  \\nATOM    278  O   GLU A  37      -3.636  36.393  17.031  1.00 25.49           O  \\nATOM    279  CB  GLU A  37      -4.440  33.611  17.880  1.00 25.97           C  \\nATOM    280  CG  GLU A  37      -4.766  32.776  16.667  1.00 33.81           C  \\nATOM    281  CD  GLU A  37      -6.160  32.154  16.777  1.00 38.37           C  \\nATOM    282  OE1 GLU A  37      -7.095  32.664  16.149  1.00 37.86           O  \\nATOM    283  OE2 GLU A  37      -6.298  31.164  17.515  1.00 42.24           O  \\nATOM    284  N   ASP A  38      -2.070  35.212  15.911  1.00 19.96           N  \\nATOM    285  CA  ASP A  38      -1.899  36.178  14.839  1.00 17.94           C  \\nATOM    286  C   ASP A  38      -2.024  35.526  13.473  1.00 18.21           C  \\nATOM    287  O   ASP A  38      -1.582  34.393  13.294  1.00 16.96           O  \\nATOM    288  CB  ASP A  38      -0.540  36.833  14.965  1.00 18.83           C  \\nATOM    289  CG  ASP A  38      -0.507  37.817  16.133  1.00 20.08           C  \\nATOM    290  OD1 ASP A  38      -1.390  38.660  16.208  1.00 25.47           O  \\nATOM    291  OD2 ASP A  38       0.375  37.781  16.967  1.00 20.08           O  \\nATOM    292  N   SER A  39      -2.698  36.170  12.507  1.00 17.82           N  \\nATOM    293  CA  SER A  39      -2.691  35.658  11.133  1.00 18.16           C  \\nATOM    294  C   SER A  39      -2.012  36.627  10.171  1.00 16.95           C  \\nATOM    295  O   SER A  39      -2.053  37.853  10.333  1.00 15.80           O  \\nATOM    296  CB  SER A  39      -4.094  35.402  10.637  1.00 23.01           C  \\nATOM    297  OG  SER A  39      -4.873  36.571  10.754  1.00 27.13           O  \\nATOM    298  N   TYR A  40      -1.336  36.080   9.168  1.00 15.44           N  \\nATOM    299  CA  TYR A  40      -0.607  36.889   8.196  1.00 16.25           C  \\nATOM    300  C   TYR A  40      -1.018  36.383   6.830  1.00 17.06           C  \\nATOM    301  O   TYR A  40      -1.160  35.172   6.638  1.00 18.29           O  \\nATOM    302  CB  TYR A  40       0.909  36.724   8.370  1.00 16.59           C  \\nATOM    303  CG  TYR A  40       1.439  37.242   9.707  1.00 16.10           C  \\nATOM    304  CD1 TYR A  40       1.473  36.412  10.815  1.00 16.77           C  \\nATOM    305  CD2 TYR A  40       1.883  38.549   9.799  1.00 18.02           C  \\nATOM    306  CE1 TYR A  40       1.950  36.893  12.021  1.00 16.86           C  \\nATOM    307  CE2 TYR A  40       2.368  39.032  11.001  1.00 18.70           C  \\nATOM    308  CZ  TYR A  40       2.396  38.199  12.104  1.00 17.62           C  \\nATOM    309  OH  TYR A  40       2.853  38.695  13.304  1.00 18.37           O  \\nATOM    310  N   ARG A  41      -1.270  37.303   5.900  1.00 15.48           N  \\nATOM    311  CA  ARG A  41      -1.660  36.918   4.558  1.00 15.42           C  \\nATOM    312  C   ARG A  41      -0.730  37.606   3.586  1.00 16.76           C  \\nATOM    313  O   ARG A  41      -0.536  38.829   3.633  1.00 17.01           O  \\nATOM    314  CB  ARG A  41      -3.087  37.342   4.277  1.00 17.64           C  \\nATOM    315  CG  ARG A  41      -4.150  36.507   4.947  1.00 18.09           C  \\nATOM    316  CD  ARG A  41      -5.528  37.181   4.849  1.00 19.64           C  \\nATOM    317  NE  ARG A  41      -6.066  37.318   3.489  1.00 20.11           N  \\nATOM    318  CZ  ARG A  41      -6.660  36.329   2.803  1.00 22.17           C  \\nATOM    319  NH1 ARG A  41      -7.219  36.571   1.617  1.00 22.48           N  \\nATOM    320  NH2 ARG A  41      -6.711  35.090   3.274  1.00 23.55           N  \\nATOM    321  N   LYS A  42      -0.135  36.827   2.679  1.00 16.52           N  \\nATOM    322  CA  LYS A  42       0.835  37.405   1.758  1.00 16.59           C  \\nATOM    323  C   LYS A  42       0.740  36.746   0.387  1.00 16.04           C  \\nATOM    324  O   LYS A  42       0.794  35.528   0.307  1.00 14.74           O  \\nATOM    325  CB  LYS A  42       2.208  37.210   2.401  1.00 18.22           C  \\nATOM    326  CG  LYS A  42       3.388  37.784   1.669  1.00 22.68           C  \\nATOM    327  CD  LYS A  42       3.369  39.274   1.792  1.00 28.09           C  \\nATOM    328  CE  LYS A  42       4.695  39.876   1.317  1.00 31.70           C  \\nATOM    329  NZ  LYS A  42       4.828  39.827  -0.128  1.00 35.70           N  \\nATOM    330  N   GLN A  43       0.561  37.510  -0.697  1.00 16.89           N  \\nATOM    331  CA  GLN A  43       0.579  36.950  -2.047  1.00 19.10           C  \\nATOM    332  C   GLN A  43       2.026  36.729  -2.448  1.00 15.08           C  \\nATOM    333  O   GLN A  43       2.828  37.654  -2.362  1.00 15.72           O  \\nATOM    334  CB  GLN A  43      -0.116  37.908  -3.043  1.00 23.32           C  \\nATOM    335  CG  GLN A  43       0.543  38.332  -4.415  1.00 32.83           C  \\nATOM    336  CD  GLN A  43       1.081  37.332  -5.482  1.00 36.11           C  \\nATOM    337  OE1 GLN A  43       0.457  36.335  -5.865  1.00 37.78           O  \\nATOM    338  NE2 GLN A  43       2.271  37.606  -6.053  1.00 35.45           N  \\nATOM    339  N   VAL A  44       2.394  35.532  -2.922  1.00 12.79           N  \\nATOM    340  CA  VAL A  44       3.771  35.242  -3.311  1.00 11.03           C  \\nATOM    341  C   VAL A  44       3.724  34.425  -4.605  1.00 14.28           C  \\nATOM    342  O   VAL A  44       2.633  34.109  -5.090  1.00 14.90           O  \\nATOM    343  CB  VAL A  44       4.508  34.411  -2.223  1.00 13.21           C  \\nATOM    344  CG1 VAL A  44       4.669  35.282  -0.977  1.00 15.04           C  \\nATOM    345  CG2 VAL A  44       3.761  33.116  -1.889  1.00 12.89           C  \\nATOM    346  N   VAL A  45       4.879  34.127  -5.235  1.00 13.20           N  \\nATOM    347  CA  VAL A  45       4.938  33.277  -6.419  1.00 12.28           C  \\nATOM    348  C   VAL A  45       5.836  32.161  -5.970  1.00 11.08           C  \\nATOM    349  O   VAL A  45       6.988  32.447  -5.636  1.00 13.13           O  \\nATOM    350  CB  VAL A  45       5.587  33.979  -7.641  1.00 13.01           C  \\nATOM    351  CG1 VAL A  45       5.549  33.031  -8.810  1.00 14.51           C  \\nATOM    352  CG2 VAL A  45       4.787  35.173  -8.108  1.00 13.54           C  \\nATOM    353  N   ILE A  46       5.374  30.907  -5.915  1.00  9.89           N  \\nATOM    354  CA  ILE A  46       6.274  29.795  -5.491  1.00 11.03           C  \\nATOM    355  C   ILE A  46       6.369  28.800  -6.662  1.00  8.85           C  \\nATOM    356  O   ILE A  46       5.319  28.296  -7.059  1.00 10.25           O  \\nATOM    357  CB  ILE A  46       5.692  29.115  -4.203  1.00 10.64           C  \\nATOM    358  CG1 ILE A  46       5.652  30.096  -3.042  1.00 11.20           C  \\nATOM    359  CG2 ILE A  46       6.599  27.953  -3.757  1.00 12.93           C  \\nATOM    360  CD1 ILE A  46       4.937  29.536  -1.808  1.00  8.90           C  \\nATOM    361  N   ASP A  47       7.565  28.495  -7.224  1.00  9.41           N  \\nATOM    362  CA  ASP A  47       7.767  27.650  -8.426  1.00  9.70           C  \\nATOM    363  C   ASP A  47       6.838  28.031  -9.572  1.00 11.14           C  \\nATOM    364  O   ASP A  47       6.225  27.211 -10.248  1.00 14.09           O  \\nATOM    365  CB  ASP A  47       7.566  26.165  -8.105  1.00 10.73           C  \\nATOM    366  CG  ASP A  47       8.388  25.783  -6.874  1.00 13.52           C  \\nATOM    367  OD1 ASP A  47       7.805  25.291  -5.898  1.00 12.04           O  \\nATOM    368  OD2 ASP A  47       9.598  26.024  -6.872  1.00 11.31           O  \\nATOM    369  N   GLY A  48       6.724  29.353  -9.735  1.00 10.78           N  \\nATOM    370  CA  GLY A  48       5.930  29.961 -10.785  1.00 13.29           C  \\nATOM    371  C   GLY A  48       4.426  29.989 -10.548  1.00 14.28           C  \\nATOM    372  O   GLY A  48       3.677  30.512 -11.386  1.00 15.19           O  \\nATOM    373  N   GLU A  49       3.944  29.422  -9.439  1.00 13.41           N  \\nATOM    374  CA  GLU A  49       2.520  29.457  -9.143  1.00 13.90           C  \\nATOM    375  C   GLU A  49       2.223  30.700  -8.314  1.00 15.34           C  \\nATOM    376  O   GLU A  49       2.860  30.906  -7.275  1.00 14.56           O  \\nATOM    377  CB  GLU A  49       2.124  28.238  -8.340  1.00 15.50           C  \\nATOM    378  CG  GLU A  49       0.630  28.250  -7.990  1.00 18.10           C  \\nATOM    379  CD  GLU A  49       0.204  27.110  -7.100  1.00 19.13           C  \\nATOM    380  OE1 GLU A  49       0.717  27.004  -5.996  1.00 19.53           O  \\nATOM    381  OE2 GLU A  49      -0.630  26.321  -7.529  1.00 22.12           O  \\nATOM    382  N   THR A  50       1.267  31.536  -8.734  1.00 14.06           N  \\nATOM    383  CA  THR A  50       0.901  32.697  -7.927  1.00 16.15           C  \\nATOM    384  C   THR A  50      -0.001  32.169  -6.819  1.00 13.35           C  \\nATOM    385  O   THR A  50      -0.938  31.409  -7.046  1.00 15.13           O  \\nATOM    386  CB  THR A  50       0.203  33.755  -8.846  1.00 17.88           C  \\nATOM    387  OG1 THR A  50      -0.245  34.813  -8.012  1.00 24.33           O  \\nATOM    388  CG2 THR A  50      -1.028  33.236  -9.518  1.00 19.78           C  \\nATOM    389  N   CYS A  51       0.306  32.442  -5.563  1.00 13.28           N  \\nATOM    390  CA  CYS A  51      -0.566  31.959  -4.508  1.00 15.16           C  \\nATOM    391  C   CYS A  51      -0.579  32.892  -3.303  1.00 14.00           C  \\nATOM    392  O   CYS A  51       0.161  33.874  -3.231  1.00 11.81           O  \\nATOM    393  CB  CYS A  51      -0.132  30.537  -4.079  1.00 14.08           C  \\nATOM    394  SG  CYS A  51       1.597  30.409  -3.575  1.00 16.09           S  \\nATOM    395  N   LEU A  52      -1.521  32.609  -2.405  1.00 13.77           N  \\nATOM    396  CA  LEU A  52      -1.573  33.397  -1.199  1.00 16.01           C  \\nATOM    397  C   LEU A  52      -1.260  32.497  -0.025  1.00 14.53           C  \\nATOM    398  O   LEU A  52      -1.868  31.443   0.113  1.00 16.05           O  \\nATOM    399  CB  LEU A  52      -2.957  33.986  -1.014  1.00 19.71           C  \\nATOM    400  CG  LEU A  52      -3.122  34.885   0.214  1.00 20.84           C  \\nATOM    401  CD1 LEU A  52      -3.070  36.329  -0.220  1.00 21.38           C  \\nATOM    402  CD2 LEU A  52      -4.420  34.526   0.923  1.00 20.90           C  \\nATOM    403  N   LEU A  53      -0.319  32.904   0.800  1.00 12.41           N  \\nATOM    404  CA  LEU A  53       0.019  32.232   2.048  1.00 13.70           C  \\nATOM    405  C   LEU A  53      -0.841  32.852   3.148  1.00 13.85           C  \\nATOM    406  O   LEU A  53      -0.733  34.056   3.413  1.00 12.82           O  \\nATOM    407  CB  LEU A  53       1.485  32.464   2.400  1.00 14.51           C  \\nATOM    408  CG  LEU A  53       2.707  31.589   2.031  1.00 19.29           C  \\nATOM    409  CD1 LEU A  53       2.394  30.553   0.962  1.00 18.56           C  \\nATOM    410  CD2 LEU A  53       3.843  32.557   1.741  1.00 20.00           C  \\nATOM    411  N   ASP A  54      -1.699  32.023   3.751  1.00 11.96           N  \\nATOM    412  CA  ASP A  54      -2.476  32.445   4.884  1.00 15.07           C  \\nATOM    413  C   ASP A  54      -1.882  31.675   6.055  1.00 12.67           C  \\nATOM    414  O   ASP A  54      -2.099  30.477   6.234  1.00 13.99           O  \\nATOM    415  CB  ASP A  54      -3.930  32.088   4.644  1.00 19.33           C  \\nATOM    416  CG  ASP A  54      -4.826  32.609   5.746  1.00 24.24           C  \\nATOM    417  OD1 ASP A  54      -4.412  32.652   6.897  1.00 28.17           O  \\nATOM    418  OD2 ASP A  54      -5.958  32.979   5.450  1.00 27.59           O  \\nATOM    419  N   ILE A  55      -1.127  32.389   6.880  1.00 11.78           N  \\nATOM    420  CA  ILE A  55      -0.346  31.769   7.935  1.00 12.15           C  \\nATOM    421  C   ILE A  55      -0.953  32.099   9.296  1.00 13.10           C  \\nATOM    422  O   ILE A  55      -1.195  33.271   9.587  1.00 13.84           O  \\nATOM    423  CB  ILE A  55       1.109  32.282   7.931  1.00 12.42           C  \\nATOM    424  CG1 ILE A  55       1.859  31.979   6.653  1.00 13.99           C  \\nATOM    425  CG2 ILE A  55       1.859  31.505   9.018  1.00 13.24           C  \\nATOM    426  CD1 ILE A  55       3.176  32.768   6.497  1.00 15.68           C  \\nATOM    427  N   LEU A  56      -1.176  31.083  10.134  1.00 11.64           N  \\nATOM    428  CA  LEU A  56      -1.642  31.341  11.490  1.00 11.98           C  \\nATOM    429  C   LEU A  56      -0.524  31.027  12.431  1.00  9.99           C  \\nATOM    430  O   LEU A  56      -0.017  29.895  12.447  1.00  9.72           O  \\nATOM    431  CB  LEU A  56      -2.836  30.463  11.896  1.00 13.80           C  \\nATOM    432  CG  LEU A  56      -3.470  30.741  13.297  1.00 16.13           C  \\nATOM    433  CD1 LEU A  56      -4.347  31.987  13.260  1.00 15.28           C  \\nATOM    434  CD2 LEU A  56      -4.321  29.580  13.692  1.00 16.84           C  \\nATOM    435  N   ASP A  57      -0.188  32.084  13.182  1.00 10.40           N  \\nATOM    436  CA  ASP A  57       0.815  32.014  14.219  1.00 10.30           C  \\nATOM    437  C   ASP A  57       0.127  31.812  15.571  1.00 11.75           C  \\nATOM    438  O   ASP A  57      -0.652  32.631  16.053  1.00 12.91           O  \\nATOM    439  CB  ASP A  57       1.602  33.296  14.173  1.00 10.66           C  \\nATOM    440  CG  ASP A  57       2.778  33.336  15.132  1.00 11.39           C  \\nATOM    441  OD1 ASP A  57       3.306  32.298  15.535  1.00 11.07           O  \\nATOM    442  OD2 ASP A  57       3.168  34.448  15.455  1.00 13.15           O  \\nATOM    443  N   THR A  58       0.447  30.723  16.244  1.00 12.55           N  \\nATOM    444  CA  THR A  58      -0.324  30.325  17.416  1.00 12.98           C  \\nATOM    445  C   THR A  58       0.368  30.541  18.758  1.00 13.48           C  \\nATOM    446  O   THR A  58       1.568  30.812  18.815  1.00 14.00           O  \\nATOM    447  CB  THR A  58      -0.698  28.851  17.219  1.00  9.54           C  \\nATOM    448  OG1 THR A  58       0.491  28.084  17.369  1.00 11.40           O  \\nATOM    449  CG2 THR A  58      -1.313  28.597  15.848  1.00 12.27           C  \\nATOM    450  N   ALA A  59      -0.371  30.382  19.868  1.00 14.38           N  \\nATOM    451  CA  ALA A  59       0.230  30.510  21.194  1.00 14.79           C  \\nATOM    452  C   ALA A  59       0.928  29.210  21.608  1.00 15.41           C  \\nATOM    453  O   ALA A  59       0.448  28.106  21.324  1.00 13.07           O  \\nATOM    454  CB  ALA A  59      -0.845  30.818  22.238  1.00 15.47           C  \\nATOM    455  N   GLY A  60       2.084  29.341  22.271  1.00 17.71           N  \\nATOM    456  CA  GLY A  60       2.807  28.208  22.814  1.00 24.18           C  \\nATOM    457  C   GLY A  60       2.190  27.751  24.130  1.00 31.21           C  \\nATOM    458  O   GLY A  60       2.538  26.688  24.627  1.00 30.44           O  \\nATOM    459  N   GLN A  61       1.260  28.503  24.726  1.00 39.50           N  \\nATOM    460  CA  GLN A  61       0.671  28.099  25.994  1.00 48.84           C  \\nATOM    461  C   GLN A  61      -0.683  27.395  25.985  1.00 51.52           C  \\nATOM    462  O   GLN A  61      -1.606  27.780  25.276  1.00 52.17           O  \\nATOM    463  CB  GLN A  61       0.605  29.327  26.874  1.00 51.56           C  \\nATOM    464  CG  GLN A  61       0.871  28.861  28.312  1.00 56.74           C  \\nATOM    465  CD  GLN A  61       1.506  29.937  29.177  1.00 58.74           C  \\nATOM    466  OE1 GLN A  61       2.647  29.795  29.616  1.00 60.12           O  \\nATOM    467  NE2 GLN A  61       0.783  31.031  29.429  1.00 60.18           N  \\nATOM    468  N   GLU A  62      -0.804  26.349  26.818  1.00 57.40           N  \\nATOM    469  CA  GLU A  62      -2.008  25.527  26.927  1.00 61.96           C  \\nATOM    470  C   GLU A  62      -3.185  26.196  27.631  1.00 62.20           C  \\nATOM    471  O   GLU A  62      -4.138  25.496  27.975  1.00 62.85           O  \\nATOM    472  CB  GLU A  62      -1.629  24.183  27.632  1.00 65.18           C  \\nATOM    473  CG  GLU A  62      -2.622  22.966  27.761  1.00 68.22           C  \\nATOM    474  CD  GLU A  62      -3.032  22.132  26.528  1.00 70.25           C  \\nATOM    475  OE1 GLU A  62      -2.710  22.494  25.395  1.00 71.45           O  \\nATOM    476  OE2 GLU A  62      -3.691  21.101  26.689  1.00 70.00           O  \\nATOM    477  N   GLU A  63      -3.206  27.520  27.904  1.00 62.78           N  \\nATOM    478  CA  GLU A  63      -4.496  28.139  28.254  1.00 63.62           C  \\nATOM    479  C   GLU A  63      -5.457  27.854  27.085  1.00 63.71           C  \\nATOM    480  O   GLU A  63      -6.688  27.814  27.211  1.00 62.97           O  \\nATOM    481  CB  GLU A  63      -4.372  29.666  28.475  1.00 65.13           C  \\nATOM    482  CG  GLU A  63      -3.456  30.553  27.607  1.00 66.91           C  \\nATOM    483  CD  GLU A  63      -4.012  31.235  26.345  1.00 68.05           C  \\nATOM    484  OE1 GLU A  63      -5.220  31.462  26.220  1.00 68.64           O  \\nATOM    485  OE2 GLU A  63      -3.198  31.568  25.481  1.00 67.62           O  \\nATOM    486  N   TYR A  64      -4.801  27.606  25.934  1.00 63.36           N  \\nATOM    487  CA  TYR A  64      -5.420  27.092  24.743  1.00 63.20           C  \\nATOM    488  C   TYR A  64      -5.905  25.653  24.875  1.00 61.76           C  \\nATOM    489  O   TYR A  64      -7.120  25.486  24.858  1.00 63.04           O  \\nATOM    490  CB  TYR A  64      -4.437  27.208  23.613  1.00 64.83           C  \\nATOM    491  CG  TYR A  64      -4.736  28.459  22.818  1.00 66.98           C  \\nATOM    492  CD1 TYR A  64      -4.300  29.686  23.267  1.00 67.55           C  \\nATOM    493  CD2 TYR A  64      -5.459  28.355  21.646  1.00 67.97           C  \\nATOM    494  CE1 TYR A  64      -4.587  30.825  22.536  1.00 69.49           C  \\nATOM    495  CE2 TYR A  64      -5.752  29.488  20.911  1.00 69.29           C  \\nATOM    496  CZ  TYR A  64      -5.313  30.718  21.359  1.00 70.05           C  \\nATOM    497  OH  TYR A  64      -5.589  31.852  20.614  1.00 70.67           O  \\nATOM    498  N   SER A  65      -5.095  24.590  24.997  1.00 58.67           N  \\nATOM    499  CA  SER A  65      -5.617  23.230  25.127  1.00 56.93           C  \\nATOM    500  C   SER A  65      -6.614  22.757  24.063  1.00 55.33           C  \\nATOM    501  O   SER A  65      -6.321  22.936  22.876  1.00 54.74           O  \\nATOM    502  CB  SER A  65      -6.190  23.122  26.552  1.00 57.99           C  \\nATOM    503  OG  SER A  65      -7.251  24.011  26.900  1.00 59.80           O  \\nATOM    504  N   ALA A  66      -7.762  22.119  24.359  1.00 52.33           N  \\nATOM    505  CA  ALA A  66      -8.766  21.824  23.335  1.00 48.80           C  \\nATOM    506  C   ALA A  66      -9.138  23.050  22.492  1.00 45.71           C  \\nATOM    507  O   ALA A  66      -9.751  22.862  21.449  1.00 44.33           O  \\nATOM    508  CB  ALA A  66     -10.043  21.300  23.971  1.00 49.14           C  \\nATOM    509  N   MET A  67      -8.829  24.310  22.875  1.00 42.85           N  \\nATOM    510  CA  MET A  67      -8.984  25.465  21.967  1.00 40.01           C  \\nATOM    511  C   MET A  67      -8.093  25.362  20.719  1.00 36.79           C  \\nATOM    512  O   MET A  67      -8.221  26.184  19.805  1.00 36.08           O  \\nATOM    513  CB  MET A  67      -8.594  26.804  22.581  1.00 42.50           C  \\nATOM    514  CG  MET A  67      -9.359  27.330  23.772  1.00 45.93           C  \\nATOM    515  SD  MET A  67     -11.140  27.396  23.483  1.00 48.74           S  \\nATOM    516  CE  MET A  67     -11.526  25.880  24.321  1.00 47.64           C  \\nATOM    517  N   ARG A  68      -7.128  24.418  20.665  1.00 31.59           N  \\nATOM    518  CA  ARG A  68      -6.322  24.217  19.467  1.00 26.34           C  \\nATOM    519  C   ARG A  68      -7.140  23.473  18.424  1.00 23.68           C  \\nATOM    520  O   ARG A  68      -6.846  23.617  17.231  1.00 21.93           O  \\nATOM    521  CB  ARG A  68      -5.057  23.394  19.745  1.00 25.86           C  \\nATOM    522  CG  ARG A  68      -4.161  23.898  20.869  1.00 29.36           C  \\nATOM    523  CD  ARG A  68      -2.881  23.061  20.998  1.00 32.88           C  \\nATOM    524  NE  ARG A  68      -2.221  22.990  22.315  1.00 36.64           N  \\nATOM    525  CZ  ARG A  68      -1.700  24.031  23.001  1.00 39.66           C  \\nATOM    526  NH1 ARG A  68      -1.903  25.315  22.664  1.00 42.06           N  \\nATOM    527  NH2 ARG A  68      -0.933  23.798  24.071  1.00 39.61           N  \\nATOM    528  N   ASP A  69      -8.188  22.721  18.793  1.00 19.14           N  \\nATOM    529  CA  ASP A  69      -8.880  21.905  17.822  1.00 20.43           C  \\nATOM    530  C   ASP A  69      -9.437  22.623  16.617  1.00 22.16           C  \\nATOM    531  O   ASP A  69      -9.453  22.109  15.488  1.00 19.58           O  \\nATOM    532  CB  ASP A  69     -10.025  21.148  18.478  1.00 20.58           C  \\nATOM    533  CG  ASP A  69      -9.598  19.935  19.303  1.00 22.81           C  \\nATOM    534  OD1 ASP A  69      -8.409  19.708  19.538  1.00 24.59           O  \\nATOM    535  OD2 ASP A  69     -10.475  19.176  19.715  1.00 25.28           O  \\nATOM    536  N   GLN A  70      -9.834  23.874  16.840  1.00 23.26           N  \\nATOM    537  CA  GLN A  70     -10.451  24.609  15.759  1.00 23.53           C  \\nATOM    538  C   GLN A  70      -9.437  24.925  14.667  1.00 21.20           C  \\nATOM    539  O   GLN A  70      -9.653  24.591  13.493  1.00 20.90           O  \\nATOM    540  CB  GLN A  70     -11.071  25.881  16.341  1.00 27.73           C  \\nATOM    541  CG  GLN A  70     -12.498  26.019  15.832  1.00 34.11           C  \\nATOM    542  CD  GLN A  70     -13.103  27.407  16.008  1.00 37.87           C  \\nATOM    543  OE1 GLN A  70     -12.532  28.284  16.672  1.00 39.77           O  \\nATOM    544  NE2 GLN A  70     -14.283  27.620  15.406  1.00 37.89           N  \\nATOM    545  N   TYR A  71      -8.292  25.515  15.050  1.00 19.09           N  \\nATOM    546  CA  TYR A  71      -7.329  25.873  14.030  1.00 17.69           C  \\nATOM    547  C   TYR A  71      -6.648  24.658  13.416  1.00 17.24           C  \\nATOM    548  O   TYR A  71      -6.247  24.675  12.245  1.00 17.25           O  \\nATOM    549  CB  TYR A  71      -6.302  26.868  14.625  1.00 17.87           C  \\nATOM    550  CG  TYR A  71      -5.311  26.480  15.725  1.00 18.90           C  \\nATOM    551  CD1 TYR A  71      -4.355  25.510  15.492  1.00 19.74           C  \\nATOM    552  CD2 TYR A  71      -5.328  27.162  16.940  1.00 19.21           C  \\nATOM    553  CE1 TYR A  71      -3.402  25.212  16.450  1.00 20.38           C  \\nATOM    554  CE2 TYR A  71      -4.375  26.868  17.901  1.00 21.93           C  \\nATOM    555  CZ  TYR A  71      -3.422  25.897  17.645  1.00 21.91           C  \\nATOM    556  OH  TYR A  71      -2.463  25.590  18.588  1.00 26.44           O  \\nATOM    557  N   MET A  72      -6.557  23.559  14.166  1.00 14.81           N  \\nATOM    558  CA  MET A  72      -5.958  22.393  13.563  1.00 18.59           C  \\nATOM    559  C   MET A  72      -6.849  21.778  12.472  1.00 19.38           C  \\nATOM    560  O   MET A  72      -6.385  21.198  11.492  1.00 17.67           O  \\nATOM    561  CB  MET A  72      -5.658  21.386  14.669  1.00 17.39           C  \\nATOM    562  CG  MET A  72      -4.479  21.796  15.514  1.00 16.35           C  \\nATOM    563  SD  MET A  72      -4.146  20.600  16.820  1.00 17.04           S  \\nATOM    564  CE  MET A  72      -2.635  21.259  17.460  1.00 17.72           C  \\nATOM    565  N   ARG A  73      -8.174  21.931  12.626  1.00 20.31           N  \\nATOM    566  CA  ARG A  73      -9.120  21.350  11.701  1.00 22.24           C  \\nATOM    567  C   ARG A  73      -9.087  22.084  10.370  1.00 20.94           C  \\nATOM    568  O   ARG A  73      -9.183  21.450   9.326  1.00 20.81           O  \\nATOM    569  CB  ARG A  73     -10.499  21.414  12.369  1.00 25.29           C  \\nATOM    570  CG  ARG A  73     -11.553  20.499  11.767  1.00 30.04           C  \\nATOM    571  CD  ARG A  73     -12.775  20.429  12.687  1.00 31.86           C  \\nATOM    572  NE  ARG A  73     -12.483  19.702  13.917  1.00 34.70           N  \\nATOM    573  CZ  ARG A  73     -12.894  20.122  15.130  1.00 36.49           C  \\nATOM    574  NH1 ARG A  73     -12.579  19.402  16.196  1.00 36.93           N  \\nATOM    575  NH2 ARG A  73     -13.609  21.239  15.311  1.00 38.17           N  \\nATOM    576  N   THR A  74      -8.868  23.401  10.413  1.00 21.33           N  \\nATOM    577  CA  THR A  74      -8.813  24.240   9.219  1.00 24.99           C  \\nATOM    578  C   THR A  74      -7.436  24.351   8.544  1.00 23.10           C  \\nATOM    579  O   THR A  74      -7.367  24.738   7.371  1.00 22.59           O  \\nATOM    580  CB  THR A  74      -9.289  25.676   9.538  1.00 25.73           C  \\nATOM    581  OG1 THR A  74      -8.351  26.302  10.409  1.00 31.08           O  \\nATOM    582  CG2 THR A  74     -10.657  25.644  10.201  1.00 27.58           C  \\nATOM    583  N   GLY A  75      -6.345  24.084   9.289  1.00 19.58           N  \\nATOM    584  CA  GLY A  75      -5.013  24.114   8.730  1.00 15.79           C  \\nATOM    585  C   GLY A  75      -4.823  23.023   7.713  1.00 13.66           C  \\nATOM    586  O   GLY A  75      -5.245  21.880   7.884  1.00 14.69           O  \\nATOM    587  N   GLU A  76      -4.228  23.360   6.573  1.00 13.14           N  \\nATOM    588  CA  GLU A  76      -3.914  22.363   5.546  1.00 12.99           C  \\nATOM    589  C   GLU A  76      -2.514  21.740   5.753  1.00 12.22           C  \\nATOM    590  O   GLU A  76      -2.293  20.576   5.426  1.00 13.59           O  \\nATOM    591  CB  GLU A  76      -3.999  23.014   4.145  1.00 13.07           C  \\nATOM    592  CG  GLU A  76      -5.387  23.377   3.676  1.00 14.35           C  \\nATOM    593  CD  GLU A  76      -5.261  24.069   2.323  1.00 20.40           C  \\nATOM    594  OE1 GLU A  76      -5.461  23.403   1.288  1.00 24.32           O  \\nATOM    595  OE2 GLU A  76      -4.913  25.250   2.284  1.00 18.82           O  \\nATOM    596  N   GLY A  77      -1.553  22.503   6.301  1.00 10.73           N  \\nATOM    597  CA  GLY A  77      -0.216  22.018   6.570  1.00  8.07           C  \\nATOM    598  C   GLY A  77       0.302  22.689   7.819  1.00  8.63           C  \\nATOM    599  O   GLY A  77      -0.066  23.823   8.127  1.00  8.55           O  \\nATOM    600  N   PHE A  78       1.248  22.013   8.479  1.00  9.15           N  \\nATOM    601  CA  PHE A  78       1.754  22.468   9.773  1.00  8.87           C  \\nATOM    602  C   PHE A  78       3.268  22.520   9.854  1.00  9.51           C  \\nATOM    603  O   PHE A  78       3.930  21.542   9.486  1.00  9.15           O  \\nATOM    604  CB  PHE A  78       1.247  21.532  10.904  1.00  9.44           C  \\nATOM    605  CG  PHE A  78      -0.267  21.472  11.000  1.00  9.66           C  \\nATOM    606  CD1 PHE A  78      -0.906  22.310  11.901  1.00 10.71           C  \\nATOM    607  CD2 PHE A  78      -0.994  20.645  10.152  1.00 10.20           C  \\nATOM    608  CE1 PHE A  78      -2.295  22.338  11.940  1.00 10.61           C  \\nATOM    609  CE2 PHE A  78      -2.378  20.678  10.192  1.00 12.55           C  \\nATOM    610  CZ  PHE A  78      -3.022  21.529  11.081  1.00 12.80           C  \\nATOM    611  N   LEU A  79       3.813  23.636  10.354  1.00  8.72           N  \\nATOM    612  CA  LEU A  79       5.230  23.626  10.699  1.00  9.93           C  \\nATOM    613  C   LEU A  79       5.279  23.281  12.181  1.00  9.09           C  \\nATOM    614  O   LEU A  79       4.701  24.008  12.988  1.00 11.58           O  \\nATOM    615  CB  LEU A  79       5.921  24.970  10.567  1.00 12.80           C  \\nATOM    616  CG  LEU A  79       6.155  25.659   9.249  1.00 16.39           C  \\nATOM    617  CD1 LEU A  79       7.007  26.911   9.544  1.00 17.02           C  \\nATOM    618  CD2 LEU A  79       6.904  24.756   8.269  1.00 16.98           C  \\nATOM    619  N   CYS A  80       5.950  22.219  12.579  1.00  9.14           N  \\nATOM    620  CA  CYS A  80       6.074  21.821  13.981  1.00 11.81           C  \\nATOM    621  C   CYS A  80       7.448  22.237  14.434  1.00 11.53           C  \\nATOM    622  O   CYS A  80       8.435  21.640  14.000  1.00 12.21           O  \\nATOM    623  CB  CYS A  80       5.903  20.318  14.106  1.00 13.73           C  \\nATOM    624  SG  CYS A  80       4.158  19.900  13.837  1.00 18.26           S  \\nATOM    625  N   VAL A  81       7.486  23.282  15.269  1.00  7.57           N  \\nATOM    626  CA  VAL A  81       8.730  23.956  15.604  1.00  8.45           C  \\nATOM    627  C   VAL A  81       9.248  23.620  16.996  1.00  8.90           C  \\nATOM    628  O   VAL A  81       8.451  23.680  17.939  1.00  8.49           O  \\nATOM    629  CB  VAL A  81       8.544  25.516  15.507  1.00  7.40           C  \\nATOM    630  CG1 VAL A  81       9.917  26.196  15.588  1.00  9.42           C  \\nATOM    631  CG2 VAL A  81       8.004  25.954  14.167  1.00  8.59           C  \\nATOM    632  N   PHE A  82      10.546  23.237  17.119  1.00  9.49           N  \\nATOM    633  CA  PHE A  82      11.195  23.202  18.442  1.00  9.09           C  \\nATOM    634  C   PHE A  82      12.511  23.994  18.350  1.00  9.83           C  \\nATOM    635  O   PHE A  82      12.914  24.440  17.266  1.00  9.02           O  \\nATOM    636  CB  PHE A  82      11.482  21.755  18.902  1.00  9.76           C  \\nATOM    637  CG  PHE A  82      12.536  21.062  18.071  1.00 10.09           C  \\nATOM    638  CD1 PHE A  82      13.867  21.099  18.486  1.00 10.70           C  \\nATOM    639  CD2 PHE A  82      12.192  20.412  16.889  1.00 10.93           C  \\nATOM    640  CE1 PHE A  82      14.838  20.483  17.699  1.00 11.12           C  \\nATOM    641  CE2 PHE A  82      13.181  19.807  16.125  1.00 10.73           C  \\nATOM    642  CZ  PHE A  82      14.505  19.839  16.523  1.00 10.35           C  \\nATOM    643  N   ALA A  83      13.128  24.269  19.516  1.00  9.75           N  \\nATOM    644  CA  ALA A  83      14.407  24.963  19.569  1.00 10.32           C  \\nATOM    645  C   ALA A  83      15.509  23.960  19.916  1.00  9.48           C  \\nATOM    646  O   ALA A  83      15.362  23.142  20.839  1.00  9.61           O  \\nATOM    647  CB  ALA A  83      14.354  26.083  20.620  1.00  9.75           C  \\nATOM    648  N   ILE A  84      16.624  24.042  19.160  1.00 11.41           N  \\nATOM    649  CA  ILE A  84      17.714  23.070  19.299  1.00 13.15           C  \\nATOM    650  C   ILE A  84      18.446  23.223  20.623  1.00 14.57           C  \\nATOM    651  O   ILE A  84      19.234  22.347  20.971  1.00 13.94           O  \\nATOM    652  CB  ILE A  84      18.775  23.151  18.160  1.00 13.29           C  \\nATOM    653  CG1 ILE A  84      19.403  24.509  17.878  1.00 14.90           C  \\nATOM    654  CG2 ILE A  84      18.018  22.701  16.933  1.00 13.09           C  \\nATOM    655  CD1 ILE A  84      20.578  24.957  18.763  1.00 17.59           C  \\nATOM    656  N   ASN A  85      18.200  24.290  21.390  1.00 13.61           N  \\nATOM    657  CA  ASN A  85      18.794  24.394  22.720  1.00 15.52           C  \\nATOM    658  C   ASN A  85      17.758  24.156  23.828  1.00 15.49           C  \\nATOM    659  O   ASN A  85      17.988  24.519  24.999  1.00 16.13           O  \\nATOM    660  CB  ASN A  85      19.446  25.788  22.885  1.00 15.16           C  \\nATOM    661  CG  ASN A  85      18.464  26.946  22.904  1.00 16.42           C  \\nATOM    662  OD1 ASN A  85      17.355  26.878  22.362  1.00 16.01           O  \\nATOM    663  ND2 ASN A  85      18.808  28.030  23.587  1.00 16.83           N  \\nATOM    664  N   ASN A  86      16.616  23.515  23.509  1.00 14.72           N  \\nATOM    665  CA  ASN A  86      15.553  23.303  24.483  1.00 13.01           C  \\nATOM    666  C   ASN A  86      14.987  21.901  24.309  1.00 13.17           C  \\nATOM    667  O   ASN A  86      14.080  21.668  23.520  1.00 13.04           O  \\nATOM    668  CB  ASN A  86      14.478  24.367  24.256  1.00 16.51           C  \\nATOM    669  CG  ASN A  86      13.267  24.415  25.187  1.00 20.74           C  \\nATOM    670  OD1 ASN A  86      12.370  25.241  25.018  1.00 25.41           O  \\nATOM    671  ND2 ASN A  86      13.123  23.596  26.206  1.00 19.96           N  \\nATOM    672  N   THR A  87      15.488  20.950  25.103  1.00 13.80           N  \\nATOM    673  CA  THR A  87      15.095  19.553  25.026  1.00 15.40           C  \\nATOM    674  C   THR A  87      13.589  19.316  25.266  1.00 14.40           C  \\nATOM    675  O   THR A  87      12.953  18.535  24.546  1.00 14.11           O  \\nATOM    676  CB  THR A  87      15.983  18.791  26.046  1.00 16.19           C  \\nATOM    677  OG1 THR A  87      17.325  18.908  25.571  1.00 18.22           O  \\nATOM    678  CG2 THR A  87      15.626  17.327  26.180  1.00 17.82           C  \\nATOM    679  N   LYS A  88      12.983  19.958  26.272  1.00 14.42           N  \\nATOM    680  CA  LYS A  88      11.552  19.842  26.535  1.00 15.84           C  \\nATOM    681  C   LYS A  88      10.726  20.232  25.278  1.00 15.06           C  \\nATOM    682  O   LYS A  88       9.722  19.582  25.003  1.00 14.61           O  \\nATOM    683  CB  LYS A  88      11.207  20.766  27.717  1.00 19.53           C  \\nATOM    684  CG  LYS A  88       9.937  20.401  28.457  1.00 27.71           C  \\nATOM    685  CD  LYS A  88       9.585  21.452  29.529  1.00 33.26           C  \\nATOM    686  CE  LYS A  88       9.061  22.762  28.877  1.00 36.79           C  \\nATOM    687  NZ  LYS A  88       8.642  23.763  29.852  1.00 37.29           N  \\nATOM    688  N   SER A  89      11.119  21.259  24.475  1.00 12.36           N  \\nATOM    689  CA  SER A  89      10.325  21.666  23.309  1.00 11.31           C  \\nATOM    690  C   SER A  89      10.371  20.572  22.268  1.00 12.51           C  \\nATOM    691  O   SER A  89       9.397  20.388  21.547  1.00 12.88           O  \\nATOM    692  CB  SER A  89      10.844  22.951  22.674  1.00 11.64           C  \\nATOM    693  OG  SER A  89      12.160  22.900  22.124  1.00 11.11           O  \\nATOM    694  N   PHE A  90      11.504  19.842  22.204  1.00 11.27           N  \\nATOM    695  CA  PHE A  90      11.680  18.751  21.262  1.00 12.91           C  \\nATOM    696  C   PHE A  90      10.760  17.601  21.667  1.00 14.37           C  \\nATOM    697  O   PHE A  90      10.034  17.057  20.839  1.00 13.24           O  \\nATOM    698  CB  PHE A  90      13.161  18.312  21.260  1.00 12.34           C  \\nATOM    699  CG  PHE A  90      13.411  17.120  20.351  1.00 14.41           C  \\nATOM    700  CD1 PHE A  90      13.890  15.948  20.903  1.00 15.10           C  \\nATOM    701  CD2 PHE A  90      13.136  17.206  18.990  1.00 14.81           C  \\nATOM    702  CE1 PHE A  90      14.093  14.857  20.088  1.00 16.20           C  \\nATOM    703  CE2 PHE A  90      13.340  16.110  18.179  1.00 14.81           C  \\nATOM    704  CZ  PHE A  90      13.818  14.935  18.731  1.00 15.39           C  \\nATOM    705  N   GLU A  91      10.725  17.231  22.949  1.00 14.40           N  \\nATOM    706  CA  GLU A  91       9.863  16.146  23.407  1.00 15.78           C  \\nATOM    707  C   GLU A  91       8.374  16.466  23.301  1.00 16.18           C  \\nATOM    708  O   GLU A  91       7.533  15.582  23.114  1.00 17.50           O  \\nATOM    709  CB  GLU A  91      10.226  15.846  24.835  1.00 19.84           C  \\nATOM    710  CG  GLU A  91      11.654  15.386  25.074  1.00 24.49           C  \\nATOM    711  CD  GLU A  91      12.090  15.310  26.537  1.00 27.97           C  \\nATOM    712  OE1 GLU A  91      13.199  14.842  26.766  1.00 30.72           O  \\nATOM    713  OE2 GLU A  91      11.371  15.730  27.449  1.00 30.38           O  \\nATOM    714  N   ASP A  92       8.019  17.747  23.433  1.00 16.19           N  \\nATOM    715  CA  ASP A  92       6.650  18.216  23.265  1.00 15.23           C  \\nATOM    716  C   ASP A  92       6.141  18.053  21.839  1.00 14.29           C  \\nATOM    717  O   ASP A  92       4.927  18.074  21.633  1.00 14.78           O  \\nATOM    718  CB  ASP A  92       6.536  19.708  23.646  1.00 17.85           C  \\nATOM    719  CG  ASP A  92       6.706  20.080  25.127  1.00 21.39           C  \\nATOM    720  OD1 ASP A  92       6.648  19.209  25.997  1.00 20.06           O  \\nATOM    721  OD2 ASP A  92       6.887  21.272  25.401  1.00 24.39           O  \\nATOM    722  N   ILE A  93       7.013  17.910  20.826  1.00 11.76           N  \\nATOM    723  CA  ILE A  93       6.559  17.749  19.452  1.00 14.77           C  \\nATOM    724  C   ILE A  93       5.574  16.583  19.354  1.00 14.03           C  \\nATOM    725  O   ILE A  93       4.576  16.695  18.644  1.00 14.93           O  \\nATOM    726  CB  ILE A  93       7.790  17.499  18.485  1.00 14.96           C  \\nATOM    727  CG1 ILE A  93       8.506  18.830  18.293  1.00 15.41           C  \\nATOM    728  CG2 ILE A  93       7.369  16.936  17.091  1.00 14.61           C  \\nATOM    729  CD1 ILE A  93       7.654  19.940  17.637  1.00 20.05           C  \\nATOM    730  N   HIS A  94       5.806  15.471  20.049  1.00 15.23           N  \\nATOM    731  CA  HIS A  94       4.940  14.302  19.956  1.00 18.20           C  \\nATOM    732  C   HIS A  94       3.494  14.635  20.287  1.00 16.36           C  \\nATOM    733  O   HIS A  94       2.576  14.183  19.598  1.00 17.20           O  \\nATOM    734  CB  HIS A  94       5.436  13.194  20.900  1.00 23.57           C  \\nATOM    735  CG  HIS A  94       4.685  11.874  20.678  1.00 29.09           C  \\nATOM    736  ND1 HIS A  94       4.923  11.118  19.602  1.00 31.26           N  \\nATOM    737  CD2 HIS A  94       3.705  11.361  21.498  1.00 30.93           C  \\nATOM    738  CE1 HIS A  94       4.078  10.110  19.746  1.00 31.74           C  \\nATOM    739  NE2 HIS A  94       3.366  10.260  20.861  1.00 32.20           N  \\nATOM    740  N   GLN A  95       3.271  15.499  21.281  1.00 16.91           N  \\nATOM    741  CA  GLN A  95       1.916  15.878  21.673  1.00 18.66           C  \\nATOM    742  C   GLN A  95       1.226  16.736  20.625  1.00 16.97           C  \\nATOM    743  O   GLN A  95       0.012  16.562  20.443  1.00 15.36           O  \\nATOM    744  CB  GLN A  95       1.928  16.626  23.018  1.00 21.56           C  \\nATOM    745  CG  GLN A  95       2.457  15.766  24.180  1.00 29.81           C  \\nATOM    746  CD  GLN A  95       3.370  16.540  25.150  1.00 35.29           C  \\nATOM    747  OE1 GLN A  95       3.037  17.647  25.593  1.00 37.94           O  \\nATOM    748  NE2 GLN A  95       4.546  16.006  25.521  1.00 37.04           N  \\nATOM    749  N   TYR A  96       1.930  17.653  19.921  1.00 16.06           N  \\nATOM    750  CA  TYR A  96       1.276  18.417  18.826  1.00 16.07           C  \\nATOM    751  C   TYR A  96       0.963  17.504  17.658  1.00 14.44           C  \\nATOM    752  O   TYR A  96      -0.138  17.565  17.118  1.00 15.79           O  \\nATOM    753  CB  TYR A  96       2.139  19.554  18.264  1.00 17.94           C  \\nATOM    754  CG  TYR A  96       2.287  20.677  19.277  1.00 19.19           C  \\nATOM    755  CD1 TYR A  96       1.368  21.699  19.338  1.00 19.33           C  \\nATOM    756  CD2 TYR A  96       3.348  20.631  20.161  1.00 19.61           C  \\nATOM    757  CE1 TYR A  96       1.526  22.678  20.308  1.00 21.72           C  \\nATOM    758  CE2 TYR A  96       3.512  21.595  21.129  1.00 21.72           C  \\nATOM    759  CZ  TYR A  96       2.594  22.613  21.190  1.00 21.26           C  \\nATOM    760  OH  TYR A  96       2.776  23.582  22.153  1.00 24.09           O  \\nATOM    761  N   ARG A  97       1.904  16.635  17.295  1.00 13.30           N  \\nATOM    762  CA  ARG A  97       1.699  15.650  16.247  1.00 15.46           C  \\nATOM    763  C   ARG A  97       0.457  14.794  16.492  1.00 14.26           C  \\nATOM    764  O   ARG A  97      -0.420  14.676  15.641  1.00 14.10           O  \\nATOM    765  CB  ARG A  97       2.939  14.796  16.196  1.00 17.97           C  \\nATOM    766  CG  ARG A  97       2.886  13.541  15.357  1.00 23.35           C  \\nATOM    767  CD  ARG A  97       3.333  13.765  13.943  1.00 23.59           C  \\nATOM    768  NE  ARG A  97       3.314  12.471  13.314  1.00 24.08           N  \\nATOM    769  CZ  ARG A  97       3.268  12.320  12.007  1.00 25.74           C  \\nATOM    770  NH1 ARG A  97       3.135  11.073  11.548  1.00 26.76           N  \\nATOM    771  NH2 ARG A  97       3.353  13.380  11.181  1.00 26.95           N  \\nATOM    772  N   GLU A  98       0.332  14.218  17.678  1.00 13.69           N  \\nATOM    773  CA  GLU A  98      -0.807  13.361  17.941  1.00 16.64           C  \\nATOM    774  C   GLU A  98      -2.118  14.108  17.983  1.00 15.57           C  \\nATOM    775  O   GLU A  98      -3.157  13.573  17.607  1.00 15.85           O  \\nATOM    776  CB  GLU A  98      -0.593  12.623  19.260  1.00 21.96           C  \\nATOM    777  CG  GLU A  98       0.417  11.455  19.177  1.00 28.02           C  \\nATOM    778  CD  GLU A  98       0.147  10.460  18.038  1.00 31.04           C  \\nATOM    779  OE1 GLU A  98       1.118  10.123  17.373  1.00 33.58           O  \\nATOM    780  OE2 GLU A  98      -0.992  10.025  17.790  1.00 33.61           O  \\nATOM    781  N   GLN A  99      -2.105  15.362  18.446  1.00 14.66           N  \\nATOM    782  CA  GLN A  99      -3.343  16.104  18.468  1.00 13.22           C  \\nATOM    783  C   GLN A  99      -3.779  16.398  17.049  1.00 12.15           C  \\nATOM    784  O   GLN A  99      -4.974  16.274  16.791  1.00 14.09           O  \\nATOM    785  CB  GLN A  99      -3.158  17.403  19.250  1.00 16.29           C  \\nATOM    786  CG  GLN A  99      -4.513  18.118  19.407  1.00 19.67           C  \\nATOM    787  CD  GLN A  99      -4.517  19.240  20.438  1.00 22.19           C  \\nATOM    788  OE1 GLN A  99      -3.537  19.485  21.138  1.00 23.70           O  \\nATOM    789  NE2 GLN A  99      -5.628  19.961  20.537  1.00 22.72           N  \\nATOM    790  N   ILE A 100      -2.861  16.795  16.142  1.00 11.42           N  \\nATOM    791  CA  ILE A 100      -3.197  17.121  14.743  1.00 10.81           C  \\nATOM    792  C   ILE A 100      -3.761  15.893  14.008  1.00 12.39           C  \\nATOM    793  O   ILE A 100      -4.733  15.967  13.243  1.00 12.70           O  \\nATOM    794  CB  ILE A 100      -1.924  17.659  14.038  1.00 10.32           C  \\nATOM    795  CG1 ILE A 100      -1.642  19.075  14.518  1.00 10.37           C  \\nATOM    796  CG2 ILE A 100      -2.131  17.685  12.512  1.00 11.36           C  \\nATOM    797  CD1 ILE A 100      -0.184  19.481  14.392  1.00 10.69           C  \\nATOM    798  N   LYS A 101      -3.143  14.732  14.255  1.00 14.08           N  \\nATOM    799  CA  LYS A 101      -3.613  13.487  13.637  1.00 15.95           C  \\nATOM    800  C   LYS A 101      -5.021  13.150  14.079  1.00 15.72           C  \\nATOM    801  O   LYS A 101      -5.848  12.770  13.262  1.00 14.25           O  \\nATOM    802  CB  LYS A 101      -2.681  12.317  13.996  1.00 17.44           C  \\nATOM    803  CG  LYS A 101      -1.414  12.284  13.115  1.00 19.24           C  \\nATOM    804  CD  LYS A 101      -0.332  11.418  13.764  1.00 23.28           C  \\nATOM    805  CE  LYS A 101      -0.752  10.000  14.188  1.00 23.82           C  \\nATOM    806  NZ  LYS A 101       0.400   9.230  14.662  1.00 24.00           N  \\nATOM    807  N   ARG A 102      -5.278  13.341  15.377  1.00 16.72           N  \\nATOM    808  CA  ARG A 102      -6.586  13.098  15.962  1.00 18.74           C  \\nATOM    809  C   ARG A 102      -7.643  14.087  15.440  1.00 19.07           C  \\nATOM    810  O   ARG A 102      -8.712  13.628  15.002  1.00 20.67           O  \\nATOM    811  CB  ARG A 102      -6.457  13.197  17.462  1.00 20.62           C  \\nATOM    812  CG  ARG A 102      -7.539  12.409  18.132  1.00 26.89           C  \\nATOM    813  CD  ARG A 102      -7.610  12.887  19.585  1.00 30.60           C  \\nATOM    814  NE  ARG A 102      -8.197  14.224  19.645  1.00 35.83           N  \\nATOM    815  CZ  ARG A 102      -7.633  15.255  20.301  1.00 37.61           C  \\nATOM    816  NH1 ARG A 102      -8.243  16.455  20.267  1.00 38.37           N  \\nATOM    817  NH2 ARG A 102      -6.481  15.110  20.983  1.00 38.39           N  \\nATOM    818  N   VAL A 103      -7.418  15.407  15.395  1.00 15.59           N  \\nATOM    819  CA  VAL A 103      -8.452  16.326  14.881  1.00 18.48           C  \\nATOM    820  C   VAL A 103      -8.665  16.200  13.349  1.00 16.72           C  \\nATOM    821  O   VAL A 103      -9.776  16.383  12.848  1.00 15.90           O  \\nATOM    822  CB  VAL A 103      -8.131  17.840  15.215  1.00 20.10           C  \\nATOM    823  CG1 VAL A 103      -7.896  18.005  16.704  1.00 21.34           C  \\nATOM    824  CG2 VAL A 103      -6.899  18.302  14.512  1.00 23.93           C  \\nATOM    825  N   LYS A 104      -7.622  15.902  12.574  1.00 13.84           N  \\nATOM    826  CA  LYS A 104      -7.804  15.755  11.140  1.00 16.11           C  \\nATOM    827  C   LYS A 104      -8.158  14.329  10.716  1.00 16.70           C  \\nATOM    828  O   LYS A 104      -8.260  14.079   9.508  1.00 17.27           O  \\nATOM    829  CB  LYS A 104      -6.525  16.215  10.404  1.00 18.12           C  \\nATOM    830  CG  LYS A 104      -6.197  17.678  10.744  1.00 19.53           C  \\nATOM    831  CD  LYS A 104      -5.250  18.297   9.736  1.00 21.28           C  \\nATOM    832  CE  LYS A 104      -5.921  18.590   8.390  1.00 20.75           C  \\nATOM    833  NZ  LYS A 104      -6.875  19.673   8.462  1.00 19.79           N  \\nATOM    834  N   ASP A 105      -8.372  13.390  11.669  1.00 15.91           N  \\nATOM    835  CA  ASP A 105      -8.666  11.988  11.359  1.00 15.43           C  \\nATOM    836  C   ASP A 105      -7.705  11.372  10.333  1.00 16.04           C  \\nATOM    837  O   ASP A 105      -8.106  10.654   9.415  1.00 15.38           O  \\nATOM    838  CB  ASP A 105     -10.077  11.847  10.810  1.00 18.74           C  \\nATOM    839  CG  ASP A 105     -11.261  12.121  11.696  1.00 21.15           C  \\nATOM    840  OD1 ASP A 105     -11.183  11.972  12.918  1.00 21.36           O  \\nATOM    841  OD2 ASP A 105     -12.281  12.480  11.113  1.00 22.63           O  \\nATOM    842  N   SER A 106      -6.397  11.591  10.469  1.00 15.54           N  \\nATOM    843  CA  SER A 106      -5.470  11.174   9.442  1.00 17.25           C  \\nATOM    844  C   SER A 106      -4.112  10.882  10.011  1.00 17.31           C  \\nATOM    845  O   SER A 106      -3.650  11.556  10.916  1.00 19.11           O  \\nATOM    846  CB  SER A 106      -5.463  12.314   8.398  1.00 19.27           C  \\nATOM    847  OG  SER A 106      -4.326  12.490   7.573  1.00 24.20           O  \\nATOM    848  N   ASP A 107      -3.454   9.891   9.453  1.00 17.29           N  \\nATOM    849  CA  ASP A 107      -2.109   9.557   9.868  1.00 19.80           C  \\nATOM    850  C   ASP A 107      -1.000  10.220   9.063  1.00 20.36           C  \\nATOM    851  O   ASP A 107       0.197  10.118   9.345  1.00 21.26           O  \\nATOM    852  CB  ASP A 107      -1.970   8.045   9.810  1.00 23.15           C  \\nATOM    853  CG  ASP A 107      -2.863   7.310  10.807  1.00 25.12           C  \\nATOM    854  OD1 ASP A 107      -3.262   6.206  10.462  1.00 29.21           O  \\nATOM    855  OD2 ASP A 107      -3.127   7.812  11.903  1.00 26.59           O  \\nATOM    856  N   ASP A 108      -1.383  10.909   8.005  1.00 20.57           N  \\nATOM    857  CA  ASP A 108      -0.430  11.664   7.216  1.00 22.38           C  \\nATOM    858  C   ASP A 108      -1.061  12.982   6.842  1.00 21.37           C  \\nATOM    859  O   ASP A 108      -1.973  13.096   6.030  1.00 24.36           O  \\nATOM    860  CB  ASP A 108      -0.056  10.900   5.967  1.00 27.28           C  \\nATOM    861  CG  ASP A 108      -1.263  10.278   5.320  1.00 32.24           C  \\nATOM    862  OD1 ASP A 108      -1.714   9.272   5.877  1.00 37.07           O  \\nATOM    863  OD2 ASP A 108      -1.747  10.790   4.301  1.00 36.33           O  \\nATOM    864  N   VAL A 109      -0.598  13.970   7.572  1.00 19.33           N  \\nATOM    865  CA  VAL A 109      -1.009  15.354   7.400  1.00 17.49           C  \\nATOM    866  C   VAL A 109       0.215  16.086   6.819  1.00 14.55           C  \\nATOM    867  O   VAL A 109       1.335  15.772   7.245  1.00 13.64           O  \\nATOM    868  CB  VAL A 109      -1.410  15.870   8.784  1.00 18.40           C  \\nATOM    869  CG1 VAL A 109      -1.645  17.366   8.765  1.00 20.21           C  \\nATOM    870  CG2 VAL A 109      -2.676  15.101   9.214  1.00 20.27           C  \\nATOM    871  N   PRO A 110       0.141  17.001   5.825  1.00 12.41           N  \\nATOM    872  CA  PRO A 110       1.297  17.783   5.384  1.00 11.52           C  \\nATOM    873  C   PRO A 110       1.920  18.480   6.605  1.00 11.79           C  \\nATOM    874  O   PRO A 110       1.289  19.245   7.324  1.00  9.55           O  \\nATOM    875  CB  PRO A 110       0.728  18.721   4.370  1.00 11.31           C  \\nATOM    876  CG  PRO A 110      -0.410  17.926   3.779  1.00 11.38           C  \\nATOM    877  CD  PRO A 110      -1.044  17.309   5.020  1.00 12.05           C  \\nATOM    878  N   MET A 111       3.192  18.201   6.874  1.00 11.92           N  \\nATOM    879  CA  MET A 111       3.878  18.724   8.050  1.00 11.49           C  \\nATOM    880  C   MET A 111       5.379  18.749   7.807  1.00 11.68           C  \\nATOM    881  O   MET A 111       5.903  17.900   7.098  1.00 11.11           O  \\nATOM    882  CB  MET A 111       3.494  17.806   9.211  1.00 14.36           C  \\nATOM    883  CG  MET A 111       3.864  18.110  10.627  1.00 17.41           C  \\nATOM    884  SD  MET A 111       3.231  16.825  11.758  1.00 18.36           S  \\nATOM    885  CE  MET A 111       1.528  17.128  11.498  1.00 17.25           C  \\nATOM    886  N   VAL A 112       6.091  19.706   8.404  1.00 10.73           N  \\nATOM    887  CA  VAL A 112       7.545  19.763   8.348  1.00 10.18           C  \\nATOM    888  C   VAL A 112       7.992  19.875   9.794  1.00  9.11           C  \\nATOM    889  O   VAL A 112       7.404  20.633  10.556  1.00 10.77           O  \\nATOM    890  CB  VAL A 112       8.010  21.024   7.542  1.00 11.80           C  \\nATOM    891  CG1 VAL A 112       9.528  21.213   7.558  1.00 13.24           C  \\nATOM    892  CG2 VAL A 112       7.542  20.843   6.104  1.00 14.56           C  \\nATOM    893  N   LEU A 113       9.032  19.167  10.207  1.00  8.47           N  \\nATOM    894  CA  LEU A 113       9.632  19.332  11.535  1.00  9.53           C  \\nATOM    895  C   LEU A 113      10.738  20.384  11.412  1.00  9.33           C  \\nATOM    896  O   LEU A 113      11.621  20.270  10.555  1.00  9.95           O  \\nATOM    897  CB  LEU A 113      10.214  17.983  12.017  1.00  9.56           C  \\nATOM    898  CG  LEU A 113      10.863  17.976  13.381  1.00 11.58           C  \\nATOM    899  CD1 LEU A 113       9.908  18.379  14.503  1.00 13.17           C  \\nATOM    900  CD2 LEU A 113      11.361  16.564  13.597  1.00 12.84           C  \\nATOM    901  N   VAL A 114      10.705  21.442  12.221  1.00  7.62           N  \\nATOM    902  CA  VAL A 114      11.673  22.541  12.109  1.00  7.64           C  \\nATOM    903  C   VAL A 114      12.436  22.667  13.441  1.00  7.65           C  \\nATOM    904  O   VAL A 114      11.872  22.842  14.541  1.00  9.71           O  \\nATOM    905  CB  VAL A 114      10.896  23.836  11.774  1.00  9.37           C  \\nATOM    906  CG1 VAL A 114      11.814  25.030  11.869  1.00 10.27           C  \\nATOM    907  CG2 VAL A 114      10.254  23.719  10.393  1.00  9.64           C  \\nATOM    908  N   GLY A 115      13.745  22.567  13.331  1.00  6.10           N  \\nATOM    909  CA  GLY A 115      14.603  22.742  14.476  1.00  6.69           C  \\nATOM    910  C   GLY A 115      15.120  24.163  14.407  1.00  8.47           C  \\nATOM    911  O   GLY A 115      16.007  24.439  13.594  1.00  9.16           O  \\nATOM    912  N   ASN A 116      14.593  25.089  15.192  1.00  8.01           N  \\nATOM    913  CA  ASN A 116      15.007  26.475  15.111  1.00  8.24           C  \\nATOM    914  C   ASN A 116      16.112  26.873  16.109  1.00  9.61           C  \\nATOM    915  O   ASN A 116      16.453  26.142  17.051  1.00  9.84           O  \\nATOM    916  CB  ASN A 116      13.721  27.307  15.275  1.00  8.01           C  \\nATOM    917  CG  ASN A 116      13.929  28.788  14.992  1.00  9.48           C  \\nATOM    918  OD1 ASN A 116      14.470  29.205  13.972  1.00  9.94           O  \\nATOM    919  ND2 ASN A 116      13.565  29.644  15.940  1.00 10.75           N  \\nATOM    920  N   LYS A 117      16.660  28.087  15.876  1.00 10.22           N  \\nATOM    921  CA  LYS A 117      17.735  28.715  16.674  1.00 12.36           C  \\nATOM    922  C   LYS A 117      19.094  28.021  16.450  1.00 13.39           C  \\nATOM    923  O   LYS A 117      19.977  27.940  17.324  1.00 11.29           O  \\nATOM    924  CB  LYS A 117      17.391  28.706  18.214  1.00 13.23           C  \\nATOM    925  CG  LYS A 117      16.026  29.272  18.611  1.00 12.90           C  \\nATOM    926  CD  LYS A 117      15.932  29.640  20.097  1.00 12.43           C  \\nATOM    927  CE  LYS A 117      14.551  30.217  20.380  1.00 10.55           C  \\nATOM    928  NZ  LYS A 117      14.491  30.647  21.772  1.00 11.10           N  \\nATOM    929  N   CYS A 118      19.254  27.555  15.188  1.00 13.69           N  \\nATOM    930  CA  CYS A 118      20.449  26.806  14.844  1.00 15.67           C  \\nATOM    931  C   CYS A 118      21.716  27.658  14.796  1.00 15.28           C  \\nATOM    932  O   CYS A 118      22.808  27.113  14.629  1.00 15.30           O  \\nATOM    933  CB  CYS A 118      20.246  26.064  13.492  1.00 17.74           C  \\nATOM    934  SG  CYS A 118      20.353  27.009  11.944  1.00 20.74           S  \\nATOM    935  N   ASP A 119      21.578  28.995  14.931  1.00 15.76           N  \\nATOM    936  CA  ASP A 119      22.722  29.904  15.083  1.00 16.09           C  \\nATOM    937  C   ASP A 119      23.426  29.778  16.445  1.00 17.19           C  \\nATOM    938  O   ASP A 119      24.589  30.178  16.553  1.00 18.76           O  \\nATOM    939  CB  ASP A 119      22.268  31.374  14.888  1.00 15.14           C  \\nATOM    940  CG  ASP A 119      21.153  31.785  15.835  1.00 15.81           C  \\nATOM    941  OD1 ASP A 119      20.030  31.294  15.662  1.00 13.87           O  \\nATOM    942  OD2 ASP A 119      21.407  32.596  16.721  1.00 14.85           O  \\nATOM    943  N   LEU A 120      22.732  29.248  17.480  1.00 16.86           N  \\nATOM    944  CA  LEU A 120      23.230  29.099  18.845  1.00 18.76           C  \\nATOM    945  C   LEU A 120      24.217  27.957  18.979  1.00 19.70           C  \\nATOM    946  O   LEU A 120      23.979  26.885  18.431  1.00 20.53           O  \\nATOM    947  CB  LEU A 120      22.103  28.829  19.831  1.00 17.65           C  \\nATOM    948  CG  LEU A 120      21.065  29.933  20.010  1.00 19.43           C  \\nATOM    949  CD1 LEU A 120      20.044  29.418  21.008  1.00 18.46           C  \\nATOM    950  CD2 LEU A 120      21.688  31.239  20.503  1.00 18.73           C  \\nATOM    951  N   ALA A 121      25.313  28.184  19.705  1.00 19.26           N  \\nATOM    952  CA  ALA A 121      26.303  27.143  19.887  1.00 20.48           C  \\nATOM    953  C   ALA A 121      25.942  26.210  21.036  1.00 21.30           C  \\nATOM    954  O   ALA A 121      26.403  25.069  21.067  1.00 24.04           O  \\nATOM    955  CB  ALA A 121      27.652  27.767  20.164  1.00 22.02           C  \\nATOM    956  N   ALA A 122      25.109  26.635  21.982  1.00 22.09           N  \\nATOM    957  CA  ALA A 122      24.683  25.797  23.094  1.00 23.88           C  \\nATOM    958  C   ALA A 122      23.578  24.811  22.720  1.00 23.77           C  \\nATOM    959  O   ALA A 122      22.492  24.835  23.301  1.00 27.82           O  \\nATOM    960  CB  ALA A 122      24.171  26.684  24.249  1.00 25.67           C  \\nATOM    961  N   ARG A 123      23.843  23.908  21.783  1.00 20.99           N  \\nATOM    962  CA  ARG A 123      22.890  22.905  21.311  1.00 19.81           C  \\nATOM    963  C   ARG A 123      22.619  21.761  22.283  1.00 19.56           C  \\nATOM    964  O   ARG A 123      23.590  21.176  22.774  1.00 20.01           O  \\nATOM    965  CB  ARG A 123      23.421  22.333  19.988  1.00 19.81           C  \\nATOM    966  CG  ARG A 123      22.662  21.166  19.398  1.00 21.43           C  \\nATOM    967  CD  ARG A 123      23.273  20.680  18.073  1.00 22.66           C  \\nATOM    968  NE  ARG A 123      23.050  21.590  16.956  1.00 22.61           N  \\nATOM    969  CZ  ARG A 123      22.019  21.487  16.093  1.00 20.95           C  \\nATOM    970  NH1 ARG A 123      21.045  20.568  16.181  1.00 16.36           N  \\nATOM    971  NH2 ARG A 123      21.989  22.370  15.098  1.00 19.65           N  \\nATOM    972  N   THR A 124      21.364  21.389  22.600  1.00 17.96           N  \\nATOM    973  CA  THR A 124      21.118  20.200  23.415  1.00 17.85           C  \\nATOM    974  C   THR A 124      20.413  19.068  22.656  1.00 17.08           C  \\nATOM    975  O   THR A 124      20.459  17.926  23.091  1.00 19.59           O  \\nATOM    976  CB  THR A 124      20.311  20.578  24.672  1.00 19.42           C  \\nATOM    977  OG1 THR A 124      19.109  21.230  24.277  1.00 20.50           O  \\nATOM    978  CG2 THR A 124      21.155  21.458  25.588  1.00 19.06           C  \\nATOM    979  N   VAL A 125      19.788  19.319  21.495  1.00 16.59           N  \\nATOM    980  CA  VAL A 125      19.133  18.324  20.647  1.00 16.09           C  \\nATOM    981  C   VAL A 125      20.032  18.080  19.413  1.00 16.61           C  \\nATOM    982  O   VAL A 125      20.180  18.980  18.573  1.00 14.07           O  \\nATOM    983  CB  VAL A 125      17.720  18.848  20.179  1.00 15.28           C  \\nATOM    984  CG1 VAL A 125      17.097  17.814  19.265  1.00 14.81           C  \\nATOM    985  CG2 VAL A 125      16.824  19.168  21.366  1.00 15.97           C  \\nATOM    986  N   GLU A 126      20.659  16.889  19.280  1.00 17.05           N  \\nATOM    987  CA  GLU A 126      21.487  16.568  18.124  1.00 19.28           C  \\nATOM    988  C   GLU A 126      20.631  16.496  16.876  1.00 18.51           C  \\nATOM    989  O   GLU A 126      19.511  15.976  16.880  1.00 17.68           O  \\nATOM    990  CB  GLU A 126      22.165  15.215  18.219  1.00 23.29           C  \\nATOM    991  CG  GLU A 126      23.244  15.057  19.265  1.00 30.86           C  \\nATOM    992  CD  GLU A 126      24.402  16.054  19.153  1.00 36.59           C  \\nATOM    993  OE1 GLU A 126      25.139  16.012  18.161  1.00 39.64           O  \\nATOM    994  OE2 GLU A 126      24.581  16.885  20.056  1.00 40.44           O  \\nATOM    995  N   SER A 127      21.167  16.985  15.755  1.00 17.73           N  \\nATOM    996  CA  SER A 127      20.426  16.938  14.493  1.00 16.72           C  \\nATOM    997  C   SER A 127      20.040  15.506  14.158  1.00 16.76           C  \\nATOM    998  O   SER A 127      18.931  15.245  13.690  1.00 17.01           O  \\nATOM    999  CB  SER A 127      21.283  17.518  13.371  1.00 17.67           C  \\nATOM   1000  OG  SER A 127      21.730  18.826  13.768  1.00 19.67           O  \\nATOM   1001  N   ARG A 128      20.916  14.558  14.466  1.00 16.72           N  \\nATOM   1002  CA  ARG A 128      20.667  13.154  14.145  1.00 20.94           C  \\nATOM   1003  C   ARG A 128      19.435  12.617  14.862  1.00 19.62           C  \\nATOM   1004  O   ARG A 128      18.641  11.882  14.278  1.00 18.81           O  \\nATOM   1005  CB  ARG A 128      21.910  12.319  14.510  1.00 24.31           C  \\nATOM   1006  CG  ARG A 128      21.969  11.035  13.692  1.00 30.88           C  \\nATOM   1007  CD  ARG A 128      23.381  10.403  13.536  1.00 35.88           C  \\nATOM   1008  NE  ARG A 128      24.384  10.996  12.621  1.00 39.57           N  \\nATOM   1009  CZ  ARG A 128      24.313  11.018  11.267  1.00 40.85           C  \\nATOM   1010  NH1 ARG A 128      25.399  11.388  10.577  1.00 40.83           N  \\nATOM   1011  NH2 ARG A 128      23.202  10.685  10.576  1.00 41.33           N  \\nATOM   1012  N   GLN A 129      19.209  13.016  16.119  1.00 18.27           N  \\nATOM   1013  CA  GLN A 129      18.032  12.519  16.787  1.00 18.53           C  \\nATOM   1014  C   GLN A 129      16.735  13.151  16.224  1.00 17.11           C  \\nATOM   1015  O   GLN A 129      15.691  12.486  16.098  1.00 15.48           O  \\nATOM   1016  CB  GLN A 129      18.229  12.766  18.306  1.00 22.36           C  \\nATOM   1017  CG  GLN A 129      18.041  14.133  18.911  1.00 24.78           C  \\nATOM   1018  CD  GLN A 129      18.283  14.123  20.411  1.00 27.48           C  \\nATOM   1019  OE1 GLN A 129      19.287  14.643  20.906  1.00 28.14           O  \\nATOM   1020  NE2 GLN A 129      17.367  13.497  21.152  1.00 27.21           N  \\nATOM   1021  N   ALA A 130      16.780  14.446  15.837  1.00 14.73           N  \\nATOM   1022  CA  ALA A 130      15.611  15.044  15.210  1.00 13.87           C  \\nATOM   1023  C   ALA A 130      15.342  14.449  13.826  1.00 13.30           C  \\nATOM   1024  O   ALA A 130      14.199  14.194  13.436  1.00 11.42           O  \\nATOM   1025  CB  ALA A 130      15.822  16.536  15.063  1.00 16.21           C  \\nATOM   1026  N   GLN A 131      16.401  14.189  13.057  1.00 14.61           N  \\nATOM   1027  CA  GLN A 131      16.267  13.529  11.739  1.00 16.01           C  \\nATOM   1028  C   GLN A 131      15.637  12.109  11.821  1.00 16.29           C  \\nATOM   1029  O   GLN A 131      14.777  11.708  11.017  1.00 14.37           O  \\nATOM   1030  CB  GLN A 131      17.673  13.507  11.116  1.00 18.12           C  \\nATOM   1031  CG  GLN A 131      17.702  13.017   9.689  1.00 22.19           C  \\nATOM   1032  CD  GLN A 131      16.889  13.922   8.801  1.00 21.61           C  \\nATOM   1033  OE1 GLN A 131      17.292  15.044   8.529  1.00 24.06           O  \\nATOM   1034  NE2 GLN A 131      15.714  13.485   8.370  1.00 23.37           N  \\nATOM   1035  N   ASP A 132      16.038  11.348  12.853  1.00 16.64           N  \\nATOM   1036  CA  ASP A 132      15.467  10.038  13.173  1.00 18.00           C  \\nATOM   1037  C   ASP A 132      13.984  10.175  13.484  1.00 17.28           C  \\nATOM   1038  O   ASP A 132      13.178   9.395  12.985  1.00 16.45           O  \\nATOM   1039  CB  ASP A 132      16.120   9.415  14.405  1.00 20.10           C  \\nATOM   1040  CG  ASP A 132      17.506   8.816  14.234  1.00 24.07           C  \\nATOM   1041  OD1 ASP A 132      18.120   8.459  15.243  1.00 28.35           O  \\nATOM   1042  OD2 ASP A 132      17.975   8.686  13.106  1.00 25.43           O  \\nATOM   1043  N   LEU A 133      13.614  11.168  14.312  1.00 15.58           N  \\nATOM   1044  CA  LEU A 133      12.213  11.390  14.624  1.00 16.28           C  \\nATOM   1045  C   LEU A 133      11.421  11.729  13.366  1.00 14.96           C  \\nATOM   1046  O   LEU A 133      10.345  11.181  13.128  1.00 15.09           O  \\nATOM   1047  CB  LEU A 133      12.052  12.536  15.647  1.00 17.06           C  \\nATOM   1048  CG  LEU A 133      10.617  12.781  16.136  1.00 18.79           C  \\nATOM   1049  CD1 LEU A 133      10.142  11.554  16.881  1.00 19.96           C  \\nATOM   1050  CD2 LEU A 133      10.544  14.013  17.020  1.00 18.45           C  \\nATOM   1051  N   ALA A 134      11.937  12.642  12.545  1.00 13.16           N  \\nATOM   1052  CA  ALA A 134      11.241  13.016  11.331  1.00 14.89           C  \\nATOM   1053  C   ALA A 134      11.092  11.821  10.414  1.00 16.19           C  \\nATOM   1054  O   ALA A 134      10.015  11.643   9.857  1.00 15.70           O  \\nATOM   1055  CB  ALA A 134      12.021  14.089  10.629  1.00 15.12           C  \\nATOM   1056  N   ARG A 135      12.143  10.995  10.279  1.00 17.00           N  \\nATOM   1057  CA  ARG A 135      12.088   9.828   9.438  1.00 19.17           C  \\nATOM   1058  C   ARG A 135      10.990   8.893   9.916  1.00 18.70           C  \\nATOM   1059  O   ARG A 135      10.269   8.352   9.078  1.00 20.29           O  \\nATOM   1060  CB  ARG A 135      13.425   9.091   9.456  1.00 22.48           C  \\nATOM   1061  CG  ARG A 135      13.499   8.297   8.155  1.00 29.02           C  \\nATOM   1062  CD  ARG A 135      14.619   7.270   7.988  1.00 33.22           C  \\nATOM   1063  NE  ARG A 135      14.117   5.888   8.050  1.00 38.80           N  \\nATOM   1064  CZ  ARG A 135      13.281   5.302   7.148  1.00 39.60           C  \\nATOM   1065  NH1 ARG A 135      12.889   4.044   7.384  1.00 40.80           N  \\nATOM   1066  NH2 ARG A 135      12.827   5.913   6.034  1.00 40.11           N  \\nATOM   1067  N   SER A 136      10.815   8.697  11.242  1.00 19.23           N  \\nATOM   1068  CA  SER A 136       9.757   7.841  11.770  1.00 19.12           C  \\nATOM   1069  C   SER A 136       8.374   8.432  11.528  1.00 19.61           C  \\nATOM   1070  O   SER A 136       7.398   7.690  11.522  1.00 21.01           O  \\nATOM   1071  CB  SER A 136       9.930   7.624  13.280  1.00 19.94           C  \\nATOM   1072  OG  SER A 136       9.459   8.686  14.119  1.00 23.21           O  \\nATOM   1073  N   TYR A 137       8.231   9.760  11.380  1.00 17.33           N  \\nATOM   1074  CA  TYR A 137       6.950  10.375  11.078  1.00 18.12           C  \\nATOM   1075  C   TYR A 137       6.683  10.404   9.570  1.00 18.22           C  \\nATOM   1076  O   TYR A 137       5.561  10.656   9.129  1.00 19.27           O  \\nATOM   1077  CB  TYR A 137       6.907  11.812  11.592  1.00 18.21           C  \\nATOM   1078  CG  TYR A 137       6.883  12.057  13.084  1.00 19.08           C  \\nATOM   1079  CD1 TYR A 137       6.619  11.026  13.955  1.00 20.51           C  \\nATOM   1080  CD2 TYR A 137       7.059  13.355  13.548  1.00 20.43           C  \\nATOM   1081  CE1 TYR A 137       6.510  11.285  15.306  1.00 21.89           C  \\nATOM   1082  CE2 TYR A 137       6.945  13.624  14.899  1.00 21.02           C  \\nATOM   1083  CZ  TYR A 137       6.662  12.574  15.763  1.00 22.13           C  \\nATOM   1084  OH  TYR A 137       6.476  12.782  17.112  1.00 22.16           O  \\nATOM   1085  N   GLY A 138       7.689  10.186   8.737  1.00 16.07           N  \\nATOM   1086  CA  GLY A 138       7.520  10.265   7.294  1.00 17.66           C  \\nATOM   1087  C   GLY A 138       7.476  11.725   6.810  1.00 18.01           C  \\nATOM   1088  O   GLY A 138       6.859  12.010   5.774  1.00 17.85           O  \\nATOM   1089  N   ILE A 139       8.128  12.662   7.540  1.00 14.16           N  \\nATOM   1090  CA  ILE A 139       8.117  14.065   7.145  1.00 13.49           C  \\nATOM   1091  C   ILE A 139       9.526  14.662   7.033  1.00 14.35           C  \\nATOM   1092  O   ILE A 139      10.494  14.152   7.610  1.00 16.19           O  \\nATOM   1093  CB  ILE A 139       7.292  14.896   8.161  1.00 14.40           C  \\nATOM   1094  CG1 ILE A 139       7.925  14.811   9.554  1.00 15.44           C  \\nATOM   1095  CG2 ILE A 139       5.821  14.422   8.104  1.00 14.07           C  \\nATOM   1096  CD1 ILE A 139       7.288  15.765  10.587  1.00 15.35           C  \\nATOM   1097  N   PRO A 140       9.706  15.767   6.308  1.00 13.55           N  \\nATOM   1098  CA  PRO A 140      10.984  16.448   6.205  1.00 13.39           C  \\nATOM   1099  C   PRO A 140      11.424  17.065   7.532  1.00 12.74           C  \\nATOM   1100  O   PRO A 140      10.601  17.530   8.310  1.00 11.42           O  \\nATOM   1101  CB  PRO A 140      10.767  17.498   5.127  1.00 13.47           C  \\nATOM   1102  CG  PRO A 140       9.514  17.084   4.410  1.00 14.09           C  \\nATOM   1103  CD  PRO A 140       8.686  16.467   5.519  1.00 13.14           C  \\nATOM   1104  N   TYR A 141      12.744  17.086   7.756  1.00 11.68           N  \\nATOM   1105  CA  TYR A 141      13.336  17.791   8.890  1.00 10.89           C  \\nATOM   1106  C   TYR A 141      14.175  18.945   8.326  1.00 10.09           C  \\nATOM   1107  O   TYR A 141      15.056  18.720   7.468  1.00 10.27           O  \\nATOM   1108  CB  TYR A 141      14.276  16.897   9.691  1.00 12.54           C  \\nATOM   1109  CG  TYR A 141      15.063  17.676  10.733  1.00 10.92           C  \\nATOM   1110  CD1 TYR A 141      14.350  18.396  11.664  1.00 11.75           C  \\nATOM   1111  CD2 TYR A 141      16.453  17.633  10.784  1.00 12.49           C  \\nATOM   1112  CE1 TYR A 141      14.997  19.074  12.670  1.00 12.85           C  \\nATOM   1113  CE2 TYR A 141      17.112  18.307  11.794  1.00 11.33           C  \\nATOM   1114  CZ  TYR A 141      16.369  19.016  12.728  1.00 12.61           C  \\nATOM   1115  OH  TYR A 141      16.944  19.646  13.813  1.00 13.82           O  \\nATOM   1116  N   ILE A 142      13.892  20.174   8.799  1.00  9.35           N  \\nATOM   1117  CA  ILE A 142      14.636  21.355   8.376  1.00  9.05           C  \\nATOM   1118  C   ILE A 142      15.153  22.163   9.551  1.00  9.25           C  \\nATOM   1119  O   ILE A 142      14.409  22.382  10.499  1.00 10.43           O  \\nATOM   1120  CB  ILE A 142      13.756  22.259   7.472  1.00  9.25           C  \\nATOM   1121  CG1 ILE A 142      13.523  21.470   6.180  1.00 10.51           C  \\nATOM   1122  CG2 ILE A 142      14.450  23.586   7.122  1.00  9.08           C  \\nATOM   1123  CD1 ILE A 142      12.719  22.127   5.028  1.00 13.16           C  \\nATOM   1124  N   GLU A 143      16.457  22.513   9.568  1.00  9.01           N  \\nATOM   1125  CA  GLU A 143      16.979  23.404  10.611  1.00  8.52           C  \\nATOM   1126  C   GLU A 143      16.978  24.864  10.193  1.00 10.31           C  \\nATOM   1127  O   GLU A 143      17.278  25.228   9.040  1.00 10.45           O  \\nATOM   1128  CB  GLU A 143      18.397  23.022  11.013  1.00 10.87           C  \\nATOM   1129  CG  GLU A 143      18.363  21.728  11.801  1.00 10.22           C  \\nATOM   1130  CD  GLU A 143      19.592  21.411  12.633  1.00 13.01           C  \\nATOM   1131  OE1 GLU A 143      20.616  22.082  12.521  1.00 13.65           O  \\nATOM   1132  OE2 GLU A 143      19.511  20.487  13.429  1.00 12.70           O  \\nATOM   1133  N   THR A 144      16.608  25.762  11.117  1.00  9.18           N  \\nATOM   1134  CA  THR A 144      16.422  27.169  10.754  1.00  9.71           C  \\nATOM   1135  C   THR A 144      17.014  28.076  11.789  1.00  8.66           C  \\nATOM   1136  O   THR A 144      17.259  27.698  12.927  1.00  9.13           O  \\nATOM   1137  CB  THR A 144      14.919  27.617  10.635  1.00  9.90           C  \\nATOM   1138  OG1 THR A 144      14.336  27.403  11.931  1.00  9.29           O  \\nATOM   1139  CG2 THR A 144      14.120  26.858   9.557  1.00  9.89           C  \\nATOM   1140  N   SER A 145      17.244  29.288  11.360  1.00 10.91           N  \\nATOM   1141  CA  SER A 145      17.558  30.377  12.290  1.00 12.65           C  \\nATOM   1142  C   SER A 145      16.758  31.602  11.849  1.00 10.37           C  \\nATOM   1143  O   SER A 145      16.930  32.094  10.727  1.00 11.76           O  \\nATOM   1144  CB  SER A 145      19.069  30.768  12.308  1.00 11.79           C  \\nATOM   1145  OG  SER A 145      19.248  31.931  13.125  1.00 12.12           O  \\nATOM   1146  N   ALA A 146      15.838  32.046  12.696  1.00  9.49           N  \\nATOM   1147  CA  ALA A 146      15.131  33.268  12.413  1.00 10.90           C  \\nATOM   1148  C   ALA A 146      16.070  34.455  12.553  1.00 13.53           C  \\nATOM   1149  O   ALA A 146      15.799  35.514  11.995  1.00 16.20           O  \\nATOM   1150  CB  ALA A 146      13.976  33.436  13.367  1.00 12.65           C  \\nATOM   1151  N   LYS A 147      17.183  34.302  13.283  1.00 14.35           N  \\nATOM   1152  CA  LYS A 147      18.150  35.351  13.486  1.00 16.25           C  \\nATOM   1153  C   LYS A 147      18.953  35.568  12.215  1.00 17.58           C  \\nATOM   1154  O   LYS A 147      19.124  36.724  11.825  1.00 20.13           O  \\nATOM   1155  CB  LYS A 147      19.068  34.970  14.617  1.00 17.55           C  \\nATOM   1156  CG  LYS A 147      20.003  36.125  14.958  1.00 22.83           C  \\nATOM   1157  CD  LYS A 147      20.581  35.932  16.341  1.00 26.44           C  \\nATOM   1158  CE  LYS A 147      21.683  36.949  16.507  1.00 30.79           C  \\nATOM   1159  NZ  LYS A 147      22.575  36.534  17.574  1.00 33.81           N  \\nATOM   1160  N   THR A 148      19.479  34.529  11.541  1.00 17.16           N  \\nATOM   1161  CA  THR A 148      20.248  34.735  10.300  1.00 15.93           C  \\nATOM   1162  C   THR A 148      19.422  34.607   9.013  1.00 17.72           C  \\nATOM   1163  O   THR A 148      19.926  34.962   7.945  1.00 18.47           O  \\nATOM   1164  CB  THR A 148      21.444  33.745  10.209  1.00 15.86           C  \\nATOM   1165  OG1 THR A 148      20.886  32.435  10.042  1.00 14.56           O  \\nATOM   1166  CG2 THR A 148      22.377  33.839  11.439  1.00 15.25           C  \\nATOM   1167  N   ARG A 149      18.176  34.108   9.100  1.00 15.18           N  \\nATOM   1168  CA  ARG A 149      17.268  33.815   7.994  1.00 19.20           C  \\nATOM   1169  C   ARG A 149      17.652  32.483   7.318  1.00 17.94           C  \\nATOM   1170  O   ARG A 149      17.031  32.079   6.332  1.00 20.28           O  \\nATOM   1171  CB  ARG A 149      17.251  34.964   6.937  1.00 21.40           C  \\nATOM   1172  CG  ARG A 149      15.929  35.106   6.188  1.00 27.00           C  \\nATOM   1173  CD  ARG A 149      15.946  35.904   4.863  1.00 27.99           C  \\nATOM   1174  NE  ARG A 149      14.707  35.687   4.092  1.00 29.40           N  \\nATOM   1175  CZ  ARG A 149      13.550  36.232   4.492  1.00 29.07           C  \\nATOM   1176  NH1 ARG A 149      12.412  36.015   3.828  1.00 29.56           N  \\nATOM   1177  NH2 ARG A 149      13.527  37.004   5.577  1.00 28.99           N  \\nATOM   1178  N   GLN A 150      18.621  31.701   7.838  1.00 15.80           N  \\nATOM   1179  CA  GLN A 150      18.926  30.402   7.273  1.00 16.36           C  \\nATOM   1180  C   GLN A 150      17.737  29.464   7.455  1.00 15.02           C  \\nATOM   1181  O   GLN A 150      17.119  29.351   8.504  1.00 13.57           O  \\nATOM   1182  CB  GLN A 150      20.152  29.729   7.937  1.00 23.05           C  \\nATOM   1183  CG  GLN A 150      20.578  28.448   7.162  1.00 29.37           C  \\nATOM   1184  CD  GLN A 150      21.604  27.530   7.827  1.00 34.84           C  \\nATOM   1185  OE1 GLN A 150      21.301  26.416   8.280  1.00 38.95           O  \\nATOM   1186  NE2 GLN A 150      22.857  27.967   7.883  1.00 35.22           N  \\nATOM   1187  N   GLY A 151      17.412  28.809   6.348  1.00 13.62           N  \\nATOM   1188  CA  GLY A 151      16.357  27.835   6.296  1.00 13.50           C  \\nATOM   1189  C   GLY A 151      14.943  28.395   6.329  1.00 12.36           C  \\nATOM   1190  O   GLY A 151      14.025  27.591   6.148  1.00 11.07           O  \\nATOM   1191  N   VAL A 152      14.706  29.702   6.510  1.00 12.59           N  \\nATOM   1192  CA  VAL A 152      13.339  30.183   6.715  1.00 14.23           C  \\nATOM   1193  C   VAL A 152      12.435  29.971   5.475  1.00 13.41           C  \\nATOM   1194  O   VAL A 152      11.352  29.362   5.577  1.00 12.94           O  \\nATOM   1195  CB  VAL A 152      13.430  31.691   7.167  1.00 12.57           C  \\nATOM   1196  CG1 VAL A 152      12.059  32.316   7.194  1.00 13.14           C  \\nATOM   1197  CG2 VAL A 152      14.044  31.787   8.543  1.00 12.69           C  \\nATOM   1198  N   GLU A 153      12.831  30.410   4.278  1.00 13.52           N  \\nATOM   1199  CA  GLU A 153      12.001  30.213   3.086  1.00 14.81           C  \\nATOM   1200  C   GLU A 153      11.857  28.756   2.742  1.00 11.88           C  \\nATOM   1201  O   GLU A 153      10.797  28.282   2.344  1.00 13.92           O  \\nATOM   1202  CB  GLU A 153      12.592  30.936   1.881  1.00 17.03           C  \\nATOM   1203  CG  GLU A 153      12.454  32.452   2.015  1.00 21.18           C  \\nATOM   1204  CD  GLU A 153      12.870  33.318   0.817  1.00 25.73           C  \\nATOM   1205  OE1 GLU A 153      13.232  32.781  -0.234  1.00 28.32           O  \\nATOM   1206  OE2 GLU A 153      12.822  34.550   0.936  1.00 26.64           O  \\nATOM   1207  N   ASP A 154      12.959  28.031   2.957  1.00 11.31           N  \\nATOM   1208  CA  ASP A 154      13.023  26.593   2.747  1.00 10.16           C  \\nATOM   1209  C   ASP A 154      11.964  25.828   3.561  1.00 10.90           C  \\nATOM   1210  O   ASP A 154      11.224  24.985   3.035  1.00 11.83           O  \\nATOM   1211  CB  ASP A 154      14.420  26.212   3.117  1.00 15.00           C  \\nATOM   1212  CG  ASP A 154      14.919  24.942   2.524  1.00 19.04           C  \\nATOM   1213  OD1 ASP A 154      14.121  24.195   1.938  1.00 22.96           O  \\nATOM   1214  OD2 ASP A 154      16.122  24.697   2.676  1.00 21.26           O  \\nATOM   1215  N   ALA A 155      11.854  26.107   4.881  1.00  8.83           N  \\nATOM   1216  CA  ALA A 155      10.841  25.450   5.716  1.00 10.00           C  \\nATOM   1217  C   ALA A 155       9.424  25.748   5.203  1.00 10.10           C  \\nATOM   1218  O   ALA A 155       8.653  24.814   4.978  1.00 11.14           O  \\nATOM   1219  CB  ALA A 155      10.923  25.933   7.192  1.00  9.50           C  \\nATOM   1220  N   PHE A 156       9.054  27.011   4.949  1.00 10.59           N  \\nATOM   1221  CA  PHE A 156       7.712  27.345   4.475  1.00 10.51           C  \\nATOM   1222  C   PHE A 156       7.453  26.825   3.069  1.00 11.05           C  \\nATOM   1223  O   PHE A 156       6.368  26.302   2.801  1.00 10.80           O  \\nATOM   1224  CB  PHE A 156       7.492  28.890   4.506  1.00 11.54           C  \\nATOM   1225  CG  PHE A 156       7.266  29.450   5.936  1.00 12.42           C  \\nATOM   1226  CD1 PHE A 156       8.319  29.958   6.707  1.00 12.00           C  \\nATOM   1227  CD2 PHE A 156       5.978  29.434   6.478  1.00 12.57           C  \\nATOM   1228  CE1 PHE A 156       8.065  30.434   8.001  1.00 12.45           C  \\nATOM   1229  CE2 PHE A 156       5.747  29.908   7.757  1.00 12.32           C  \\nATOM   1230  CZ  PHE A 156       6.782  30.406   8.513  1.00 10.18           C  \\nATOM   1231  N   TYR A 157       8.408  26.935   2.151  1.00  9.69           N  \\nATOM   1232  CA  TYR A 157       8.148  26.463   0.798  1.00 12.31           C  \\nATOM   1233  C   TYR A 157       8.091  24.945   0.719  1.00 10.42           C  \\nATOM   1234  O   TYR A 157       7.292  24.403  -0.047  1.00 11.35           O  \\nATOM   1235  CB  TYR A 157       9.233  26.978  -0.156  1.00 15.16           C  \\nATOM   1236  CG  TYR A 157       9.257  28.492  -0.367  1.00 18.73           C  \\nATOM   1237  CD1 TYR A 157       8.380  29.366   0.268  1.00 18.52           C  \\nATOM   1238  CD2 TYR A 157      10.211  28.971  -1.242  1.00 20.44           C  \\nATOM   1239  CE1 TYR A 157       8.466  30.724   0.025  1.00 22.04           C  \\nATOM   1240  CE2 TYR A 157      10.305  30.320  -1.488  1.00 22.70           C  \\nATOM   1241  CZ  TYR A 157       9.436  31.178  -0.852  1.00 23.03           C  \\nATOM   1242  OH  TYR A 157       9.572  32.528  -1.089  1.00 26.15           O  \\nATOM   1243  N   THR A 158       8.940  24.198   1.455  1.00 11.35           N  \\nATOM   1244  CA  THR A 158       8.836  22.727   1.522  1.00 10.11           C  \\nATOM   1245  C   THR A 158       7.440  22.347   2.064  1.00 11.11           C  \\nATOM   1246  O   THR A 158       6.811  21.408   1.558  1.00 11.43           O  \\nATOM   1247  CB  THR A 158       9.935  22.129   2.470  1.00 12.09           C  \\nATOM   1248  OG1 THR A 158      11.216  22.451   1.933  1.00 11.36           O  \\nATOM   1249  CG2 THR A 158       9.791  20.657   2.624  1.00 10.87           C  \\nATOM   1250  N   LEU A 159       6.888  23.056   3.072  1.00  9.25           N  \\nATOM   1251  CA  LEU A 159       5.533  22.754   3.506  1.00  9.93           C  \\nATOM   1252  C   LEU A 159       4.489  22.986   2.388  1.00 10.42           C  \\nATOM   1253  O   LEU A 159       3.560  22.185   2.272  1.00 11.22           O  \\nATOM   1254  CB  LEU A 159       5.171  23.608   4.711  1.00  9.03           C  \\nATOM   1255  CG  LEU A 159       3.789  23.318   5.269  1.00 10.96           C  \\nATOM   1256  CD1 LEU A 159       3.635  21.828   5.651  1.00  9.29           C  \\nATOM   1257  CD2 LEU A 159       3.580  24.223   6.469  1.00 13.71           C  \\nATOM   1258  N   VAL A 160       4.596  24.046   1.548  1.00 10.91           N  \\nATOM   1259  CA  VAL A 160       3.662  24.284   0.439  1.00  9.19           C  \\nATOM   1260  C   VAL A 160       3.803  23.137  -0.556  1.00  9.60           C  \\nATOM   1261  O   VAL A 160       2.781  22.638  -1.031  1.00 10.29           O  \\nATOM   1262  CB  VAL A 160       3.981  25.653  -0.211  1.00  8.50           C  \\nATOM   1263  CG1 VAL A 160       3.402  25.754  -1.622  1.00 10.56           C  \\nATOM   1264  CG2 VAL A 160       3.403  26.741   0.659  1.00  7.98           C  \\nATOM   1265  N   ARG A 161       5.033  22.667  -0.839  1.00  9.10           N  \\nATOM   1266  CA  ARG A 161       5.206  21.554  -1.767  1.00  9.82           C  \\nATOM   1267  C   ARG A 161       4.644  20.240  -1.215  1.00 10.44           C  \\nATOM   1268  O   ARG A 161       4.166  19.395  -1.984  1.00 12.88           O  \\nATOM   1269  CB  ARG A 161       6.679  21.398  -2.115  1.00  9.39           C  \\nATOM   1270  CG  ARG A 161       7.146  22.640  -2.816  1.00  9.70           C  \\nATOM   1271  CD  ARG A 161       8.594  22.398  -3.227  1.00 14.12           C  \\nATOM   1272  NE  ARG A 161       9.122  23.635  -3.780  1.00 14.15           N  \\nATOM   1273  CZ  ARG A 161      10.144  24.313  -3.257  1.00 12.45           C  \\nATOM   1274  NH1 ARG A 161      10.823  23.858  -2.204  1.00 13.24           N  \\nATOM   1275  NH2 ARG A 161      10.485  25.475  -3.819  1.00 13.52           N  \\nATOM   1276  N   GLU A 162       4.663  20.046   0.112  1.00 10.71           N  \\nATOM   1277  CA  GLU A 162       4.054  18.877   0.722  1.00 11.80           C  \\nATOM   1278  C   GLU A 162       2.522  18.887   0.568  1.00 11.20           C  \\nATOM   1279  O   GLU A 162       1.921  17.848   0.274  1.00 12.90           O  \\nATOM   1280  CB  GLU A 162       4.430  18.815   2.216  1.00 13.86           C  \\nATOM   1281  CG  GLU A 162       5.870  18.460   2.580  1.00 14.10           C  \\nATOM   1282  CD  GLU A 162       6.268  17.038   2.224  1.00 17.23           C  \\nATOM   1283  OE1 GLU A 162       5.571  16.116   2.627  1.00 19.29           O  \\nATOM   1284  OE2 GLU A 162       7.278  16.827   1.545  1.00 22.14           O  \\nATOM   1285  N   ILE A 163       1.869  20.045   0.731  1.00 10.13           N  \\nATOM   1286  CA  ILE A 163       0.426  20.175   0.542  1.00 10.58           C  \\nATOM   1287  C   ILE A 163       0.128  19.878  -0.933  1.00 13.43           C  \\nATOM   1288  O   ILE A 163      -0.796  19.118  -1.235  1.00 15.01           O  \\nATOM   1289  CB  ILE A 163      -0.015  21.590   0.891  1.00 11.33           C  \\nATOM   1290  CG1 ILE A 163       0.084  21.777   2.400  1.00 11.27           C  \\nATOM   1291  CG2 ILE A 163      -1.486  21.815   0.437  1.00 12.37           C  \\nATOM   1292  CD1 ILE A 163      -0.006  23.245   2.894  1.00 12.56           C  \\nATOM   1293  N   ARG A 164       0.904  20.456  -1.871  1.00 11.72           N  \\nATOM   1294  CA  ARG A 164       0.770  20.163  -3.283  1.00 13.64           C  \\nATOM   1295  C   ARG A 164       0.813  18.657  -3.626  1.00 17.97           C  \\nATOM   1296  O   ARG A 164       0.150  18.197  -4.559  1.00 18.12           O  \\nATOM   1297  CB  ARG A 164       1.897  20.887  -4.029  1.00 12.51           C  \\nATOM   1298  CG  ARG A 164       1.647  22.371  -4.120  1.00 12.49           C  \\nATOM   1299  CD  ARG A 164       2.801  22.888  -4.950  1.00 14.72           C  \\nATOM   1300  NE  ARG A 164       2.730  24.327  -5.113  1.00 12.82           N  \\nATOM   1301  CZ  ARG A 164       3.800  25.071  -5.405  1.00 13.29           C  \\nATOM   1302  NH1 ARG A 164       5.018  24.539  -5.527  1.00 11.56           N  \\nATOM   1303  NH2 ARG A 164       3.595  26.381  -5.572  1.00 12.55           N  \\nATOM   1304  N   GLN A 165       1.590  17.831  -2.935  1.00 20.23           N  \\nATOM   1305  CA  GLN A 165       1.673  16.435  -3.337  1.00 27.50           C  \\nATOM   1306  C   GLN A 165       0.782  15.499  -2.502  1.00 29.12           C  \\nATOM   1307  O   GLN A 165       0.826  14.274  -2.655  1.00 29.30           O  \\nATOM   1308  CB  GLN A 165       3.168  16.070  -3.286  1.00 30.34           C  \\nATOM   1309  CG  GLN A 165       4.018  16.904  -4.309  1.00 35.17           C  \\nATOM   1310  CD  GLN A 165       4.109  16.470  -5.796  1.00 37.71           C  \\nATOM   1311  OE1 GLN A 165       4.256  17.293  -6.719  1.00 36.70           O  \\nATOM   1312  NE2 GLN A 165       4.079  15.166  -6.094  1.00 38.13           N  \\nATOM   1313  N   HIS A 166      -0.068  16.075  -1.638  1.00 29.28           N  \\nATOM   1314  CA  HIS A 166      -0.982  15.296  -0.820  1.00 31.13           C  \\nATOM   1315  C   HIS A 166      -2.214  14.971  -1.665  1.00 33.30           C  \\nATOM   1316  O   HIS A 166      -2.630  15.786  -2.497  1.00 34.96           O  \\nATOM   1317  CB  HIS A 166      -1.420  16.090   0.428  1.00 30.89           C  \\nATOM   1318  CG  HIS A 166      -2.247  15.319   1.475  1.00 32.17           C  \\nATOM   1319  ND1 HIS A 166      -3.579  15.231   1.603  1.00 33.52           N  \\nATOM   1320  CD2 HIS A 166      -1.712  14.560   2.502  1.00 32.95           C  \\nATOM   1321  CE1 HIS A 166      -3.857  14.464   2.647  1.00 33.07           C  \\nATOM   1322  NE2 HIS A 166      -2.727  14.064   3.184  1.00 33.06           N  \\nATOM   1323  OXT HIS A 166      -2.763  13.888  -1.454  1.00 36.27           O  \\nTER    1324      HIS A 166                                                      \\nHETATM 1325 MG    MG A 168       4.771  33.860  19.213  1.00  9.42          MG  \\nHETATM 1326  PG  GNP A 167       5.113  32.223  21.994  1.00  9.66           P  \\nHETATM 1327  O1G GNP A 167       4.844  32.596  23.390  1.00  9.26           O  \\nHETATM 1328  O2G GNP A 167       4.203  32.910  21.112  1.00 11.20           O  \\nHETATM 1329  O3G GNP A 167       4.640  30.769  21.802  1.00 10.09           O  \\nHETATM 1330  N3B GNP A 167       6.701  32.303  21.621  1.00  8.18           N  \\nHETATM 1331  PB  GNP A 167       7.310  32.034  20.111  1.00  8.77           P  \\nHETATM 1332  O1B GNP A 167       7.468  30.605  19.791  1.00  8.35           O  \\nHETATM 1333  O2B GNP A 167       6.696  32.775  19.021  1.00  9.38           O  \\nHETATM 1334  O3A GNP A 167       8.792  32.643  20.152  1.00 10.15           O  \\nHETATM 1335  PA  GNP A 167       9.280  34.140  19.710  1.00 10.18           P  \\nHETATM 1336  O1A GNP A 167       9.263  34.268  18.220  1.00 10.07           O  \\nHETATM 1337  O2A GNP A 167       8.594  35.173  20.510  1.00  9.33           O  \\nHETATM 1338  O5' GNP A 167      10.826  34.056  20.132  1.00 11.46           O  \\nHETATM 1339  C5' GNP A 167      11.170  33.911  21.535  1.00 10.56           C  \\nHETATM 1340  C4' GNP A 167      12.587  34.445  21.811  1.00 11.96           C  \\nHETATM 1341  O4' GNP A 167      13.560  33.649  21.095  1.00 11.66           O  \\nHETATM 1342  C3' GNP A 167      12.813  35.910  21.359  1.00 12.06           C  \\nHETATM 1343  O3' GNP A 167      13.793  36.538  22.221  1.00 14.52           O  \\nHETATM 1344  C2' GNP A 167      13.325  35.679  19.947  1.00 11.90           C  \\nHETATM 1345  O2' GNP A 167      14.151  36.750  19.499  1.00 14.99           O  \\nHETATM 1346  C1' GNP A 167      14.242  34.478  20.156  1.00 12.90           C  \\nHETATM 1347  N9  GNP A 167      14.454  33.739  18.896  1.00 12.98           N  \\nHETATM 1348  C8  GNP A 167      13.518  33.262  17.993  1.00 11.76           C  \\nHETATM 1349  N7  GNP A 167      14.016  32.636  16.979  1.00 12.34           N  \\nHETATM 1350  C5  GNP A 167      15.413  32.694  17.200  1.00 12.36           C  \\nHETATM 1351  C6  GNP A 167      16.521  32.219  16.402  1.00 12.30           C  \\nHETATM 1352  O6  GNP A 167      16.507  31.591  15.338  1.00 12.16           O  \\nHETATM 1353  N1  GNP A 167      17.735  32.503  16.995  1.00 11.60           N  \\nHETATM 1354  C2  GNP A 167      17.909  33.082  18.236  1.00 13.91           C  \\nHETATM 1355  N2  GNP A 167      19.173  33.280  18.658  1.00 14.60           N  \\nHETATM 1356  N3  GNP A 167      16.884  33.498  18.996  1.00 13.01           N  \\nHETATM 1357  C4  GNP A 167      15.675  33.331  18.398  1.00 14.36           C  \\nHETATM 1358  O   HOH A 169      13.588  43.386  17.779  1.00 31.80           O  \\nHETATM 1359  O   HOH A 170       8.248  36.064  23.209  1.00 13.63           O  \\nHETATM 1360  O   HOH A 171       3.254  37.845  19.454  1.00 18.29           O  \\nHETATM 1361  O   HOH A 172       5.945  35.619  20.105  1.00 11.33           O  \\nHETATM 1362  O   HOH A 173       3.669  32.226  18.172  1.00  8.22           O  \\nHETATM 1363  O   HOH A 174       2.896  37.170  15.500  1.00 21.37           O  \\nHETATM 1364  O   HOH A 175       2.102  32.309  24.122  1.00 27.47           O  \\nHETATM 1365  O   HOH A 176      -1.425  37.650  19.667  1.00 24.98           O  \\nHETATM 1366  O   HOH A 177      -3.225  30.031  -8.217  1.00 23.57           O  \\nHETATM 1367  O   HOH A 178      16.678  26.321  26.916  1.00 36.70           O  \\nHETATM 1368  O   HOH A 179      -2.134  40.975   1.551  1.00 81.53           O  \\nHETATM 1369  O   HOH A 180       0.400  40.675  -0.629  1.00 24.66           O  \\nHETATM 1370  O   HOH A 181      10.944  44.793  16.009  1.00 48.47           O  \\nHETATM 1371  O   HOH A 182      17.207  42.570  16.786  1.00 43.23           O  \\nHETATM 1372  O   HOH A 183      14.999  45.167  19.922  1.00 39.52           O  \\nHETATM 1373  O   HOH A 184      11.598  37.755  13.202  1.00 11.72           O  \\nHETATM 1374  O   HOH A 185       6.958  38.922  18.146  1.00 26.17           O  \\nHETATM 1375  O   HOH A 186      14.078  32.033  27.772  1.00 57.73           O  \\nHETATM 1376  O   HOH A 187       9.126  37.906  19.595  1.00 32.42           O  \\nHETATM 1377  O   HOH A 188      -0.049  36.435  27.259  1.00 50.56           O  \\nHETATM 1378  O   HOH A 189       3.843  30.503  25.485  1.00 46.80           O  \\nHETATM 1379  O   HOH A 190       4.931  28.343  27.587  1.00 29.32           O  \\nHETATM 1380  O   HOH A 191       4.881  25.774  26.247  1.00 28.55           O  \\nHETATM 1381  O   HOH A 193       8.729  26.516  26.645  1.00 37.34           O  \\nHETATM 1382  O   HOH A 194      -5.515  23.731  -4.213  1.00 43.74           O  \\nHETATM 1383  O   HOH A 195      -7.006  26.635   0.729  1.00 25.77           O  \\nHETATM 1384  O   HOH A 196      -2.033  23.281  -4.754  1.00 36.89           O  \\nHETATM 1385  O   HOH A 197      -4.326  24.642  -0.854  1.00 28.59           O  \\nHETATM 1386  O   HOH A 198      -6.858  26.995  -2.808  1.00 28.83           O  \\nHETATM 1387  O   HOH A 200      -3.937  29.225   8.289  1.00 24.08           O  \\nHETATM 1388  O   HOH A 201      -8.934  24.037   2.795  1.00 34.16           O  \\nHETATM 1389  O   HOH A 202       1.444  26.302  19.318  1.00 12.37           O  \\nHETATM 1390  O   HOH A 203       2.031  24.632  27.307  1.00 50.09           O  \\nHETATM 1391  O   HOH A 204      15.125  28.341  23.174  1.00 13.88           O  \\nHETATM 1392  O   HOH A 205      12.808  31.541  25.056  1.00 46.88           O  \\nHETATM 1393  O   HOH A 206       8.329  34.933   0.543  1.00 53.21           O  \\nHETATM 1394  O   HOH A 207       0.760  42.969   2.336  1.00 35.42           O  \\nHETATM 1395  O   HOH A 208       1.270  40.298   7.071  1.00 76.61           O  \\nHETATM 1396  O   HOH A 209      -3.135  39.625   7.649  1.00 35.40           O  \\nHETATM 1397  O   HOH A 210       8.699  39.076   2.399  1.00 33.93           O  \\nHETATM 1398  O   HOH A 212      10.040  41.806   3.447  1.00 37.88           O  \\nHETATM 1399  O   HOH A 213      18.914  29.095   3.704  1.00 25.32           O  \\nHETATM 1400  O   HOH A 214      15.966  29.098   2.957  1.00 18.63           O  \\nHETATM 1401  O   HOH A 215      15.506  31.962   4.015  1.00 18.93           O  \\nHETATM 1402  O   HOH A 216      22.935  30.533  10.845  1.00 39.81           O  \\nHETATM 1403  O   HOH A 217       6.476  43.893  23.325  1.00 49.09           O  \\nHETATM 1404  O   HOH A 218      11.746  41.248  28.612  1.00 36.22           O  \\nHETATM 1405  O   HOH A 219      11.480  42.270  25.715  1.00 29.66           O  \\nHETATM 1406  O   HOH A 220       9.344  34.251  -8.278  1.00 36.61           O  \\nHETATM 1407  O   HOH A 221      -0.111  19.667  -8.066  1.00 44.76           O  \\nHETATM 1408  O   HOH A 222      -0.528  30.460 -10.943  1.00 24.29           O  \\nHETATM 1409  O   HOH A 223     -11.352  24.274  19.491  1.00 31.21           O  \\nHETATM 1410  O   HOH A 224      -4.293  30.642 -16.466  1.00127.55           O  \\nHETATM 1411  O   HOH A 225       3.466  29.729 -16.232  1.00 26.15           O  \\nHETATM 1412  O   HOH A 227      -4.288  18.610   4.198  1.00 34.77           O  \\nHETATM 1413  O   HOH A 228       4.626  16.154   5.151  1.00 14.96           O  \\nHETATM 1414  O   HOH A 229       5.681  11.069   2.580  1.00 59.76           O  \\nHETATM 1415  O   HOH A 230       1.173  14.021   3.406  1.00 32.15           O  \\nHETATM 1416  O   HOH A 231       7.935  22.484  20.504  1.00 10.86           O  \\nHETATM 1417  O   HOH A 232       3.071  10.004  14.546  1.00 23.48           O  \\nHETATM 1418  O   HOH A 233       2.437  13.381   8.401  1.00 17.75           O  \\nHETATM 1419  O   HOH A 234       4.779   7.500   6.154  1.00 71.09           O  \\nHETATM 1420  O   HOH A 235      -4.250   8.748   6.880  1.00 24.76           O  \\nHETATM 1421  O   HOH A 236     -13.381  22.830  17.669  1.00 39.09           O  \\nHETATM 1422  O   HOH A 237      16.969  39.052   8.276  1.00 50.25           O  \\nHETATM 1423  O   HOH A 238      -6.315   9.333  13.897  1.00 36.24           O  \\nHETATM 1424  O   HOH A 240      -0.160   4.395  12.468  1.00 38.79           O  \\nHETATM 1425  O   HOH A 241       1.295   7.931  12.335  1.00 49.03           O  \\nHETATM 1426  O   HOH A 242       4.067  11.551   6.968  1.00 46.97           O  \\nHETATM 1427  O   HOH A 243      -5.060   6.579  -0.216  1.00 50.14           O  \\nHETATM 1428  O   HOH A 244      18.164   7.151   5.573  1.00 71.67           O  \\nHETATM 1429  O   HOH A 245       2.068   8.904   7.636  1.00 45.86           O  \\nHETATM 1430  O   HOH A 246      24.661  23.134  12.757  1.00 34.69           O  \\nHETATM 1431  O   HOH A 247      21.051  23.714   6.239  1.00 24.64           O  \\nHETATM 1432  O   HOH A 248      25.855  31.137  20.712  1.00 34.53           O  \\nHETATM 1433  O   HOH A 249      24.868  30.890  26.223  1.00 48.47           O  \\nHETATM 1434  O   HOH A 250      26.967  20.447  19.448  1.00 47.77           O  \\nHETATM 1435  O   HOH A 252      28.453  14.196  14.602  1.00 39.78           O  \\nHETATM 1436  O   HOH A 253      19.639  16.256   9.760  1.00 37.54           O  \\nHETATM 1437  O   HOH A 254      16.629   7.383  10.526  1.00 60.90           O  \\nHETATM 1438  O   HOH A 255       4.972   7.888  14.276  1.00 54.95           O  \\nHETATM 1439  O   HOH A 256      14.184  14.684   5.643  1.00 31.40           O  \\nHETATM 1440  O   HOH A 257      11.766  17.254   1.278  1.00 41.82           O  \\nHETATM 1441  O   HOH A 258      16.274  16.365   6.342  1.00 17.29           O  \\nHETATM 1442  O   HOH A 259      18.346  24.095   6.620  1.00 17.36           O  \\nHETATM 1443  O   HOH A 260      22.479  35.073  19.694  1.00 43.96           O  \\nHETATM 1444  O   HOH A 261      20.580  31.747   4.906  1.00 59.46           O  \\nHETATM 1445  O   HOH A 262      18.196  25.514   4.261  1.00 28.75           O  \\nHETATM 1446  O   HOH A 263      12.857  20.183   1.149  1.00 21.79           O  \\nHETATM 1447  O   HOH A 264      19.514  15.912  24.943  1.00 55.52           O  \\nHETATM 1448  O   HOH A 265       8.925   7.579   5.621  1.00 77.94           O  \\nHETATM 1449  O   HOH A 266      17.622  21.835  26.964  1.00 27.48           O  \\nHETATM 1450  O   HOH A 267      24.258  19.786  14.155  1.00 20.48           O  \\nHETATM 1451  O   HOH A 268      23.806  15.180  15.136  1.00 19.86           O  \\nHETATM 1452  O   HOH A 269      21.478  14.557  11.134  1.00 29.66           O  \\nHETATM 1453  O   HOH A 270      17.293  45.621  24.753  1.00 39.18           O  \\nHETATM 1454  O   HOH A 271      -3.518  40.206  17.649  1.00 44.75           O  \\nHETATM 1455  O   HOH A 272       2.143  19.852  24.090  1.00 40.10           O  \\nHETATM 1456  O   HOH A 273       7.101  35.918  -4.704  1.00 31.76           O  \\nHETATM 1457  O   HOH A 274       5.462  38.756  -5.521  1.00 27.51           O  \\nHETATM 1458  O   HOH A 275       5.748  20.565  28.638  1.00 58.06           O  \\nHETATM 1459  O   HOH A 276       8.814  31.258  -8.964  1.00 23.79           O  \\nHETATM 1460  O   HOH A 277     -10.427  35.895   4.301  1.00 58.91           O  \\nHETATM 1461  O   HOH A 278      -8.713  28.579   2.333  1.00 69.36           O  \\nHETATM 1462  O   HOH A 279      24.472  24.237  17.316  1.00 32.14           O  \\nHETATM 1463  O   HOH A 281      16.968  32.701  21.815  1.00 23.13           O  \\nHETATM 1464  O   HOH A 282      19.541  34.346  21.581  1.00 53.25           O  \\nHETATM 1465  O   HOH A 283      24.847  23.384  26.621  1.00 61.18           O  \\nHETATM 1466  O   HOH A 284       1.409  11.113   2.873  1.00 44.87           O  \\nHETATM 1467  O   HOH A 285       1.556  11.488  -0.914  1.00 37.86           O  \\nHETATM 1468  O   HOH A 286       9.758  12.945   3.889  1.00 36.47           O  \\nHETATM 1469  O   HOH A 287       3.827  13.375   4.622  1.00 44.90           O  \\nHETATM 1470  O   HOH A 288      14.279   6.175  15.391  1.00 78.25           O  \\nHETATM 1471  O   HOH A 289       7.573  13.968   2.191  1.00 76.47           O  \\nHETATM 1472  O   HOH A 290       2.323  15.482   1.418  1.00 35.15           O  \\nHETATM 1473  O   HOH A 291      11.997  14.573   2.962  1.00 49.86           O  \\nHETATM 1474  O   HOH A 292      23.798  33.550  17.880  1.00 24.64           O  \\nHETATM 1475  O   HOH A 293      14.657  36.046  -0.571  1.00 51.86           O  \\nHETATM 1476  O   HOH A 294      24.456  25.366   4.004  1.00 53.23           O  \\nHETATM 1477  O   HOH A 296      -0.603  12.134  -4.185  1.00 52.38           O  \\nHETATM 1478  O   HOH A 297      -8.372  34.415  -4.471  1.00 74.53           O  \\nHETATM 1479  O   HOH A 300     -14.033  36.501   0.062  1.00 50.21           O  \\nHETATM 1480  O   HOH A 301     -17.102  35.735  -2.806  1.00 42.58           O  \\nHETATM 1481  O   HOH A 303     -17.383  40.542   1.431  1.00 77.16           O  \\nHETATM 1482  O   HOH A 304     -19.024  41.557  -3.411  1.00107.16           O  \\nHETATM 1483  O   HOH A 305      -9.876  37.804   0.839  1.00 90.65           O  \\nHETATM 1484  O   HOH A 306     -14.053  36.466   4.552  1.00 82.05           O  \\nHETATM 1485  O   HOH A 307     -14.183  39.558   0.339  1.00 58.66           O  \\nHETATM 1486  O   HOH A 309     -12.547  28.677   8.301  1.00 74.88           O  \\nHETATM 1487  O   HOH A 311     -14.787  32.746  -2.171  1.00 40.79           O  \\nHETATM 1488  O   HOH A 312      -6.916  30.740   3.095  1.00 36.53           O  \\nHETATM 1489  O   HOH A 315      -8.312  33.027   7.340  1.00 83.28           O  \\nHETATM 1490  O   HOH A 316      -6.280  35.646   7.624  1.00 75.97           O  \\nHETATM 1491  O   HOH A 317      -5.230  31.565   9.609  1.00 55.67           O  \\nHETATM 1492  O   HOH A 318       6.747  37.258   1.208  1.00 57.93           O  \\nHETATM 1493  O   HOH A 319       4.747  42.419  12.683  1.00 57.74           O  \\nHETATM 1494  O   HOH A 320       2.984  41.950  16.191  1.00 41.69           O  \\nHETATM 1495  O   HOH A 321       3.867  49.208  12.056  1.00 24.57           O  \\nHETATM 1496  O   HOH A 322       7.530  42.894  17.776  1.00 79.28           O  \\nHETATM 1497  O   HOH A 324      19.822  39.457  17.826  1.00 66.22           O  \\nHETATM 1498  O   HOH A 325      12.391  45.557  24.973  1.00 48.70           O  \\nHETATM 1499  O   HOH A 326      14.944  43.928  23.794  1.00 62.45           O  \\nHETATM 1500  O   HOH A 327     -12.984  24.008  12.563  1.00 40.31           O  \\nHETATM 1501  O   HOH A 330       0.572  41.133  19.277  1.00 50.77           O  \\nHETATM 1502  O   HOH A 331      -6.196  37.979  23.043  1.00 48.93           O  \\nHETATM 1503  O   HOH A 332      -3.526  30.410  19.065  1.00 21.26           O  \\nHETATM 1504  O   HOH A 333      -7.995  32.093  21.685  1.00 54.11           O  \\nHETATM 1505  O   HOH A 334      -8.483  28.891  20.063  1.00 70.37           O  \\nHETATM 1506  O   HOH A 335     -11.670  31.022  16.981  1.00 76.86           O  \\nHETATM 1507  O   HOH A 336     -11.082  34.450  18.910  1.00 43.07           O  \\nHETATM 1508  O   HOH A 339      -6.521  37.738  15.165  1.00 64.11           O  \\nHETATM 1509  O   HOH A 340     -10.829  35.877  15.629  1.00 58.36           O  \\nHETATM 1510  O   HOH A 341      -6.552  41.166  12.231  1.00 55.67           O  \\nHETATM 1511  O   HOH A 342      -1.242  39.671  12.185  1.00 61.04           O  \\nHETATM 1512  O   HOH A 343      -4.281  38.934  13.464  1.00 32.14           O  \\nHETATM 1513  O   HOH A 344       0.386  41.208  14.039  1.00 40.80           O  \\nHETATM 1514  O   HOH A 345       0.777  39.663  21.603  1.00 67.71           O  \\nHETATM 1515  O   HOH A 346      -7.879  34.848  19.044  1.00 52.60           O  \\nHETATM 1516  O   HOH A 347       1.121  32.594 -11.957  1.00 32.30           O  \\nHETATM 1517  O   HOH A 348       0.924  32.597 -14.716  1.00 48.34           O  \\nHETATM 1518  O   HOH A 349      -2.241  27.348  20.565  1.00 26.54           O  \\nHETATM 1519  O   HOH A 350       3.259  26.214  29.881  1.00 69.22           O  \\nHETATM 1520  O   HOH A 351      -8.318  34.095  28.392  1.00 81.96           O  \\nHETATM 1521  O   HOH A 352      -2.927  29.775  31.412  1.00 80.92           O  \\nHETATM 1522  O   HOH A 353     -10.410  32.034  25.972  1.00 60.17           O  \\nHETATM 1523  O   HOH A 354      -8.122  30.393  25.378  1.00 62.33           O  \\nHETATM 1524  O   HOH A 356      -6.138  18.882  24.644  1.00 49.26           O  \\nHETATM 1525  O   HOH A 358      -1.406  19.033  23.084  1.00 82.05           O  \\nHETATM 1526  O   HOH A 359       4.169   7.339  10.280  1.00 66.03           O  \\nHETATM 1527  O   HOH A 360      -9.761  30.165   7.702  1.00 46.15           O  \\nHETATM 1528  O   HOH A 361     -16.002  27.801  11.700  1.00 41.85           O  \\nHETATM 1529  O   HOH A 362      16.215  30.489  29.264  1.00 67.21           O  \\nHETATM 1530  O   HOH A 364      21.417  27.685  26.140  1.00 49.57           O  \\nHETATM 1531  O   HOH A 366      15.824  34.767  30.662  1.00 25.90           O  \\nHETATM 1532  O   HOH A 367      18.527  35.965  24.111  1.00 68.69           O  \\nHETATM 1533  O   HOH A 369      12.103  35.275  24.945  1.00 51.37           O  \\nHETATM 1534  O   HOH A 370      24.782  11.970  23.641  1.00 55.92           O  \\nHETATM 1535  O   HOH A 371      14.337  21.394  28.627  1.00 30.11           O  \\nHETATM 1536  O   HOH A 372      -1.367  12.600  23.053  1.00 44.43           O  \\nHETATM 1537  O   HOH A 373       7.060   8.567  21.416  1.00 47.99           O  \\nHETATM 1538  O   HOH A 374       7.639   8.040  16.420  1.00 38.76           O  \\nHETATM 1539  O   HOH A 376      -1.282  15.566  25.487  1.00 67.79           O  \\nHETATM 1540  O   HOH A 378      17.945  30.924  23.806  1.00 31.50           O  \\nHETATM 1541  O   HOH A 379      17.360  33.209  33.337  1.00 30.46           O  \\nHETATM 1542  O   HOH A 380       3.971   6.567  22.089  1.00 86.80           O  \\nHETATM 1543  O   HOH A 381      -0.065  10.022  26.928  1.00 61.86           O  \\nHETATM 1544  O   HOH A 382      -4.450  12.966  21.222  1.00 42.82           O  \\nHETATM 1545  O   HOH A 383      -2.584   6.719   5.938  1.00 40.64           O  \\nHETATM 1546  O   HOH A 384      25.567  31.936  12.845  1.00 73.17           O  \\nHETATM 1547  O   HOH A 385      24.446  29.383  22.970  1.00 36.61           O  \\nHETATM 1548  O   HOH A 386      24.865  38.167  18.227  1.00 60.74           O  \\nHETATM 1549  O   HOH A 387      23.135  35.012  14.543  1.00 59.19           O  \\nHETATM 1550  O   HOH A 388      19.325   9.556  18.093  1.00 92.18           O  \\nHETATM 1551  O   HOH A 389      25.326  12.873  16.251  1.00 49.27           O  \\nHETATM 1552  O   HOH A 390      20.219  10.405  10.956  1.00 63.05           O  \\nHETATM 1553  O   HOH A 392       9.452   5.453   8.249  1.00 47.81           O  \\nHETATM 1554  O   HOH A 394      10.139  12.252   0.254  1.00 29.37           O  \\nHETATM 1555  O   HOH A 395      28.004  37.416  13.466  1.00 52.29           O  \\nHETATM 1556  O   HOH A 396      24.283  36.829  12.307  1.00 64.74           O  \\nHETATM 1557  O   HOH A 397      13.748  34.031  -3.019  1.00 47.80           O  \\nHETATM 1558  O   HOH A 398      16.981  34.189   2.006  1.00 80.51           O  \\nHETATM 1559  O   HOH A 399      -4.426  24.223  -7.240  1.00 38.09           O  \\nHETATM 1560  O   HOH A 400      19.676  33.981   3.336  1.00 64.86           O  \\nHETATM 1561  O   HOH A 401      14.633  28.745  -0.317  1.00 66.25           O  \\nHETATM 1562  O   HOH A 402      -2.139  15.576  22.716  1.00 65.01           O  \\nHETATM 1563  O   HOH A 403       9.873  11.090  20.760  1.00 53.88           O  \\nHETATM 1564  O   HOH A 404      18.291  36.379  26.927  1.00 57.23           O  \\nHETATM 1565  O   HOH A 405       0.138   6.145  15.269  1.00 82.80           O  \\nHETATM 1566  O   HOH A 406      -5.515  38.166  18.689  1.00 48.28           O  \\nHETATM 1567  O   HOH A 407      -0.399   4.688   8.809  1.00 58.35           O  \\nHETATM 1568  O   HOH A 408       0.813  25.584  22.307  1.00 40.04           O  \\nCONECT  117 1325                                                                \\nCONECT  265 1325                                                                \\nCONECT 1325  117  265 1328 1333                                                 \\nCONECT 1325 1361 1362                                                           \\nCONECT 1326 1327 1328 1329 1330                                                 \\nCONECT 1327 1326                                                                \\nCONECT 1328 1325 1326                                                           \\nCONECT 1329 1326                                                                \\nCONECT 1330 1326 1331                                                           \\nCONECT 1331 1330 1332 1333 1334                                                 \\nCONECT 1332 1331                                                                \\nCONECT 1333 1325 1331                                                           \\nCONECT 1334 1331 1335                                                           \\nCONECT 1335 1334 1336 1337 1338                                                 \\nCONECT 1336 1335                                                                \\nCONECT 1337 1335                                                                \\nCONECT 1338 1335 1339                                                           \\nCONECT 1339 1338 1340                                                           \\nCONECT 1340 1339 1341 1342                                                      \\nCONECT 1341 1340 1346                                                           \\nCONECT 1342 1340 1343 1344                                                      \\nCONECT 1343 1342                                                                \\nCONECT 1344 1342 1345 1346                                                      \\nCONECT 1345 1344                                                                \\nCONECT 1346 1341 1344 1347                                                      \\nCONECT 1347 1346 1348 1357                                                      \\nCONECT 1348 1347 1349                                                           \\nCONECT 1349 1348 1350                                                           \\nCONECT 1350 1349 1351 1357                                                      \\nCONECT 1351 1350 1352 1353                                                      \\nCONECT 1352 1351                                                                \\nCONECT 1353 1351 1354                                                           \\nCONECT 1354 1353 1355 1356                                                      \\nCONECT 1355 1354                                                                \\nCONECT 1356 1354 1357                                                           \\nCONECT 1357 1347 1350 1356                                                      \\nCONECT 1361 1325                                                                \\nCONECT 1362 1325                                                                \\nMASTER      312    0    2    5    6    0    9    6 1567    1   38   13          \\nEND                                                                             \\n\",\"pdb\");\n\tviewergrid_16835771918828518[0][0].addModel(\"\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    5.3278   11.9619   -1.5862 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4918   12.0625   -0.4513 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5527   13.0816    0.5011 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7707   13.0853    1.5453 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4957   14.1194    0.2981 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3069   14.0310   -0.8665 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2721   14.9285   -1.2734 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.7924   14.4907   -2.4207 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.7723   14.9976   -3.2383 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.1222   14.3073   -4.4052 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.4811   13.1363   -4.7128 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4813   12.6157   -3.8820 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1577   13.3143   -2.7429 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2349   13.0415   -1.7643 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8145   11.3598   -0.3222 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1195   12.3585    1.6752 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8318   13.8149    2.2035 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5851   14.8663    0.9332 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7640    9.3703   -2.6645 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8250   10.2866   -3.3296 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1718    9.8621   -4.4958 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2879   10.6862   -5.1442 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.0572   11.9404   -4.6231 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7079   12.3479   -3.4716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5995   11.5321   -2.8072 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-4.68549\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -7.2548    9.0375    4.5817 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0448    8.4019    4.2228 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.8292    9.0547    4.0095 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.7612    8.4021    3.6418 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.7899   10.4573    4.2059 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0025   11.0841    4.6043 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1942   12.4245    4.8676 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4672   12.6023    5.2235 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1730   13.7240    5.5829 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.5292   13.6094    5.9119 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.1308   12.3793    5.8698 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.4149   11.2328    5.5039 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0847   11.3744    5.1862 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.1585   10.4388    4.7983 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0601    7.4234    4.1148 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8013    7.4275    3.5086 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9150    8.8841    3.4976 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9622   10.9735    4.0730 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.7008    7.7755    3.8786 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.2098    7.6907    5.2625 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.9715    6.9952    6.2128 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.5471    6.8921    7.5129 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3553    7.4879    7.8630 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6125    8.1716    6.9162 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0245    8.2848    5.6050 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-4.21517\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -5.2059    9.8913    1.4701 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.4393    8.7205    1.6663 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9500    8.2799    2.8975 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.2150    7.2064    2.9934 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.2783    9.0388    4.0481 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0823   10.1944    3.8472 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.5352   11.0665    4.8152 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2676   12.0083    4.2190 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9393   13.0997    4.7121 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6386   13.9345    3.8321 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6414   13.6492    2.4922 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9578   12.5374    1.9849 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2799   11.7347    2.8716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.5365   10.6013    2.6562 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.2343    8.1735    0.8739 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9938    6.6876    2.1865 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.8815    6.9151    3.8727 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9646    8.7783    4.9442 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0780   11.0707   -1.1110 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2599   10.2829   -0.7287 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3094   10.1417   -1.6483 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4274    9.4138   -1.3304 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4951    8.8250   -0.0867 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4530    8.9717    0.8122 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3219    9.7000    0.5088 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-4.01159\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -6.9498   10.0239    3.9638 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6927    9.4677    4.2914 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.8273    8.8533    3.3843 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6974    8.3277    3.7703 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.2156    8.8174    2.0222 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.4645    9.4132    1.6947 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.0326    9.5098    0.4413 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1973   10.1489    0.5589 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.1492   10.5022   -0.3655 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.2995   11.1814    0.0542 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.4572   11.4825    1.3813 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.4891   11.1240    2.3275 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3695   10.4583    1.8875 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2812    9.9929    2.5820 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4050    9.5174    5.2316 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.4388    8.3591    4.7197 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.1033    7.8998    3.1121 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.6476    8.3998    1.3350 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.9625   10.1830    5.9622 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1934   11.3853    5.6068 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4816   12.5938    6.2575 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7858   13.7364    5.9555 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7975   13.6689    4.9981 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5224   12.4705    4.3628 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2122   11.3124    4.6543 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.94575\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    6.7047   12.9704   -2.2899 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4325   13.2928   -0.9412 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1589   13.3474   -0.3716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9846   13.6911    0.8748 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0535   13.0157   -1.1934 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3391   12.6553   -2.5390 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4301   12.2813   -3.5068 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1009   12.0104   -4.6273 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6860   11.6021   -5.8709 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6341   11.3923   -6.8797 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9614   11.5982   -6.6099 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3889   12.0147   -5.3431 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4366   12.2126   -4.3715 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5684   12.6144   -3.0658 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1999   13.4943   -0.3584 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7609   13.9193    1.4356 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0768   13.7223    1.2545 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1313   13.0340   -0.8491 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.0760   13.9697   -3.4934 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.0229   12.5708   -3.0417 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.1470   11.7530   -3.2282 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.1377   10.4440   -2.8192 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.9984    9.9516   -2.2213 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8955   10.7685   -2.0429 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8879   12.0867   -2.4480 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.69027\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -0.0133    7.1993    3.0780 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9633    8.1816    3.3577 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.9099    8.1021    4.3810 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7566    9.0696    4.6019 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.9237    6.9307    5.1778 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9627    5.9302    4.8653 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.8033    4.7110    5.4906 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.1959    4.0619    4.8915 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.7575    2.8294    5.1174 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8208    2.3984    4.3147 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.2843    3.2138    3.3162 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.7147    4.4715    3.0823 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.6719    4.8690    3.8852 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.0605    6.0297    3.8822 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9772    8.9839    2.7872 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7370    9.8786    4.0412 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4153    8.9908    5.3294 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.5681    6.8148    5.9130 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.3767    8.0951    1.7813 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.3467    7.3852    1.0076 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.5618    7.1538   -0.3589 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.6250    6.4970   -1.1150 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.5318    6.0700   -0.5007 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.7326    6.3043    0.8485 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.1989    6.9630    1.6234 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.62684\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    6.0766   11.2367   -1.2749 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8567   12.0362   -0.1307 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9469   13.0926   -0.0540 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8276   13.8112    1.0283 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1513   13.3627   -1.1948 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3516   12.5291   -2.3293 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6835   12.5959   -3.5342 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1547   11.6324   -4.3269 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8413   11.2642   -5.6121 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5154   10.1899   -6.2056 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4777    9.5230   -5.4943 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8006    9.8951   -4.1835 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1228   10.9531   -3.6255 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2336   11.5243   -2.3825 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3856   11.8328    0.6743 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3862   13.6135    1.8145 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1789   14.5511    1.0589 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4910   14.0929   -1.2017 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.5729    9.9538   -1.7136 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4410    9.1803   -1.1807 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.6171    7.8152   -0.9115 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5868    7.0600   -0.4124 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3752    7.6740   -0.1820 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2137    9.0219   -0.4517 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2390    9.7957   -0.9537 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.57606\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    5.0653   12.1568   -3.9872 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8604   12.4012   -4.6839 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9703   11.4160   -5.1156 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.8943   11.7181   -5.7887 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2724   10.0686   -4.7985 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4672    9.8343   -4.0639 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9490    8.6156   -3.6332 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.0860    8.8271   -2.9690 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9587    7.9704   -2.3445 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0986    8.4851   -1.7151 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.3255    9.8361   -1.7324 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4383   10.7134   -2.3681 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3270   10.1808   -2.9778 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3092   10.7944   -3.6642 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6309   13.3383   -4.8799 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.6954   12.6582   -6.0029 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2813   11.0060   -6.0825 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.6830    9.3295   -5.0736 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.0458   14.1722   -4.2793 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2733   14.0867   -3.0305 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5425   14.9999   -2.0006 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8430   14.9541   -0.8219 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8701   13.9900   -0.6735 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6136   13.0944   -1.6971 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3075   13.1265   -2.8885 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.48728\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -4.9024   11.4840    0.7331 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6126   11.9069    0.3400 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.4179   11.2889    0.7142 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.2607   11.7621    0.3415 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.5003   10.1240    1.5163 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8035    9.6828    1.8758 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.1233    8.5730    2.6300 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4517    8.5025    2.7266 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2812    7.6082    3.3572 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6681    7.7885    3.2902 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1758    8.8551    2.5964 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3337    9.7696    1.9519 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.9762    9.5681    2.0338 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.9361   10.2941    1.5097 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.5518   12.6983   -0.2425 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.2166   12.5724   -0.2159 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.4315   11.3089    0.6177 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.6936    9.6427    1.8109 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9700   13.4305    0.8105 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8997   12.4880   -0.3166 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8688   12.5637   -1.3277 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8367   11.7042   -2.3958 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8305   10.7648   -2.4519 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.8800   10.6999   -1.4480 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.8974   11.5564   -0.3673 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.46691\\n\\n$$$$\\n\",\"sdf\");\n\tviewergrid_16835771918828518[0][0].setStyle({\"model\": 0},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewergrid_16835771918828518[0][0].addSurface(1,{\"opacity\": 0.7, \"color\": \"white\"},{\"model\": 0});\n\tviewergrid_16835771918828518[0][0].setStyle({\"model\": 1},{\"stick\": {}});\n\tviewergrid_16835771918828518[0][0].zoomTo();\n\tviewergrid_16835771918828518[0][0].render();\nviewergrid_16835771918828518[0][0].render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_16835771918828518\"  style=\"position: relative; width: 1000px; height: 1000px\">\n",
       "        <p id=\"3dmolwarning_16835771918828518\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n",
       "}\n",
       "\n",
       "var viewer_16835771918828518 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_16835771918828518\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "var viewergrid_16835771918828518 = null;\n",
       "viewergrid_16835771918828518 = $3Dmol.createViewerGrid(document.getElementById(\"3dmolviewer_16835771918828518\"),{rows: 1, cols: 1, control_all: false},{backgroundColor:\"white\"});\n",
       "viewer_16835771918828518 = viewergrid_16835771918828518[0][0];\n",
       "viewergrid_16835771918828518[0][0].zoomTo();\tviewergrid_16835771918828518[0][0].setViewStyle({\"style\": \"outline\", \"color\": \"black\", \"width\": 0.1});\n",
       "\tviewergrid_16835771918828518[0][0].addModel(\"HEADER    ONCOGENE PROTEIN                        30-APR-90   5P21              \\nTITLE     REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION OF H-RAS   \\nTITLE    2 P21 AT 1.35 ANGSTROMS RESOLUTION: IMPLICATIONS FOR THE MECHANISM OF  \\nTITLE    3 GTP HYDROLYSIS                                                       \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: C-H-RAS P21 PROTEIN;                                       \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: HOMO SAPIENS;                                   \\nSOURCE   3 ORGANISM_COMMON: HUMAN;                                              \\nSOURCE   4 ORGANISM_TAXID: 9606                                                 \\nKEYWDS    ONCOGENE PROTEIN                                                      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    E.F.PAI,A.WITTINGHOFER,W.KABSCH                                       \\nREVDAT   6   29-NOV-17 5P21    1       HELIX                                    \\nREVDAT   5   24-FEB-09 5P21    1       VERSN                                    \\nREVDAT   4   01-APR-03 5P21    1       JRNL                                     \\nREVDAT   3   20-JUL-95 5P21    1       HET                                      \\nREVDAT   2   30-APR-94 5P21    1       FORMUL                                   \\nREVDAT   1   15-JAN-92 5P21    0                                                \\nJRNL        AUTH   E.F.PAI,U.KRENGEL,G.A.PETSKO,R.S.GOODY,W.KABSCH,             \\nJRNL        AUTH 2 A.WITTINGHOFER                                               \\nJRNL        TITL   REFINED CRYSTAL STRUCTURE OF THE TRIPHOSPHATE CONFORMATION   \\nJRNL        TITL 2 OF H-RAS P21 AT 1.35 A RESOLUTION: IMPLICATIONS FOR THE      \\nJRNL        TITL 3 MECHANISM OF GTP HYDROLYSIS.                                 \\nJRNL        REF    EMBO J.                       V.   9  2351 1990              \\nJRNL        REFN                   ISSN 0261-4189                               \\nJRNL        PMID   2196171                                                      \\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,S.C.ALMO,G.RAPP,K.WILSON,K.PETRATOS,A.LENTFER, \\nREMARK   1  AUTH 2 A.WITTINGHOFER,W.KABSCH,E.F.PAI,G.A.PETSKO,R.S.GOODY         \\nREMARK   1  TITL   TIME-RESOLVED X-RAY CRYSTALLOGRAPHIC STUDY OF THE            \\nREMARK   1  TITL 2 CONFORMATIONAL CHANGE IN HA-RAS P21 PROTEIN ON GTP           \\nREMARK   1  TITL 3 HYDROLYSIS                                                   \\nREMARK   1  REF    NATURE                        V. 345   309 1990              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 2                                                          \\nREMARK   1  AUTH   A.SCHERER,J.JOHN,R.LINKE,R.S.GOODY,A.WITTINGHOFER,E.F.PAI,   \\nREMARK   1  AUTH 2 K.C.HOLMES                                                   \\nREMARK   1  TITL   CRYSTALLIZATION AND PRELIMINARY X-RAY ANALYSIS OF THE HUMAN  \\nREMARK   1  TITL 2 C-H-RAS-ONCOGENE PRODUCT P21 COMPLEXED WITH GTP ANALOGUES    \\nREMARK   1  REF    J.MOL.BIOL.                   V. 206   257 1989              \\nREMARK   1  REFN                   ISSN 0022-2836                               \\nREMARK   1 REFERENCE 3                                                          \\nREMARK   1  AUTH   E.F.PAI,W.KABSCH,U.KRENGEL,K.C.HOLMES,J.JOHN,A.WITTINGHOFER  \\nREMARK   1  TITL   STRUCTURE OF THE GUANINE-NUCLEOTIDE-BINDING DOMAIN OF THE    \\nREMARK   1  TITL 2 HA-RAS ONCOGENE PRODUCT P21 IN THE TRIPHOSPHATE CONFORMATION \\nREMARK   1  REF    NATURE                        V. 341   209 1989              \\nREMARK   1  REFN                   ISSN 0028-0836                               \\nREMARK   1 REFERENCE 4                                                          \\nREMARK   1  AUTH   I.SCHLICHTING,G.RAPP,J.JOHN,A.WITTINGHOFER,E.F.PAI,R.S.GOODY \\nREMARK   1  TITL   BIOCHEMICAL AND CRYSTALLOGRAPHIC CHARACTERIZATION OF A       \\nREMARK   1  TITL 2 COMPLEX OF C-HA-RAS P21 AND CAGED GTP WITH FLASH PHOTOLYSIS  \\nREMARK   1  REF    PROC.NATL.ACAD.SCI.USA        V.  86  7687 1989              \\nREMARK   1  REFN                   ISSN 0027-8424                               \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.35 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.35                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 6.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 26806                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.196                           \\nREMARK   3   FREE R VALUE                     : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 1323                                    \\nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \\nREMARK   3   HETEROGEN ATOMS          : 33                                      \\nREMARK   3   SOLVENT ATOMS            : 211                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.014                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 2.700                           \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS:                                           \\nREMARK   3  RESIDUES 61 - 65 (GLN - GLU - GLU - TYR - SER) ADOPT                \\nREMARK   3  SEVERAL CONFORMATIONS IN THE CRYSTAL.  THE COORDINATES              \\nREMARK   3  GIVEN APPROXIMATE ONE OF THESE.                                     \\nREMARK   4                                                                      \\nREMARK   4 5P21 COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000179747.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : NULL                               \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : NULL                               \\nREMARK 200  DETECTOR MANUFACTURER          : NULL                               \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : NULL                               \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 38.93                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.01                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: NULL                                     \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 32 2 1                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+2/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+1/3                                           \\nREMARK 290       4555   Y,X,-Z                                                  \\nREMARK 290       5555   X-Y,-Y,-Z+1/3                                           \\nREMARK 290       6555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000      108.13333            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       54.06667            \\nREMARK 290   SMTRY1   4 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       54.06667            \\nREMARK 290   SMTRY1   6 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000      108.13333            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 350   BIOMT2   2  0.866025  0.500000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500    ARG A  68   NE  -  CZ  -  NH1 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A  97   NE  -  CZ  -  NH1 ANGL. DEV. =  -3.4 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH1 ANGL. DEV. =   4.1 DEGREES          \\nREMARK 500    ARG A 123   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500    ARG A 128   NE  -  CZ  -  NH2 ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500    ARG A 135   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500    TYR A 157   CB  -  CG  -  CD2 ANGL. DEV. =  -4.8 DEGREES          \\nREMARK 500    ARG A 164   NE  -  CZ  -  NH2 ANGL. DEV. =  -3.5 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ASP A  33       75.86   -157.25                                   \\nREMARK 500    ILE A  36      -64.23   -100.39                                   \\nREMARK 500    GLU A  37      123.75   -171.30                                   \\nREMARK 500    SER A  65     -133.70     53.07                                   \\nREMARK 500    LYS A 117       33.80     71.60                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 620                                                                      \\nREMARK 620 METAL COORDINATION                                                   \\nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \\nREMARK 620                                                                      \\nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \\nREMARK 620                              MG A 168  MG                            \\nREMARK 620 N RES CSSEQI ATOM                                                    \\nREMARK 620 1 GNP A 167   O2G                                                    \\nREMARK 620 2 GNP A 167   O2B  95.0                                              \\nREMARK 620 3 HOH A 173   O    87.6  91.7                                        \\nREMARK 620 4 SER A  17   OG  172.9  91.0  88.6                                  \\nREMARK 620 5 THR A  35   OG1  91.3 173.4  90.5  82.8                            \\nREMARK 620 6 HOH A 172   O    97.3  88.0 175.0  86.5  89.3                      \\nREMARK 620 N                    1     2     3     4     5                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE MG A 168                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE GNP A 167                 \\nDBREF  5P21 A    1   166  UNP    P01112   RASH_HUMAN       1    166             \\nSEQRES   1 A  166  MET THR GLU TYR LYS LEU VAL VAL VAL GLY ALA GLY GLY          \\nSEQRES   2 A  166  VAL GLY LYS SER ALA LEU THR ILE GLN LEU ILE GLN ASN          \\nSEQRES   3 A  166  HIS PHE VAL ASP GLU TYR ASP PRO THR ILE GLU ASP SER          \\nSEQRES   4 A  166  TYR ARG LYS GLN VAL VAL ILE ASP GLY GLU THR CYS LEU          \\nSEQRES   5 A  166  LEU ASP ILE LEU ASP THR ALA GLY GLN GLU GLU TYR SER          \\nSEQRES   6 A  166  ALA MET ARG ASP GLN TYR MET ARG THR GLY GLU GLY PHE          \\nSEQRES   7 A  166  LEU CYS VAL PHE ALA ILE ASN ASN THR LYS SER PHE GLU          \\nSEQRES   8 A  166  ASP ILE HIS GLN TYR ARG GLU GLN ILE LYS ARG VAL LYS          \\nSEQRES   9 A  166  ASP SER ASP ASP VAL PRO MET VAL LEU VAL GLY ASN LYS          \\nSEQRES  10 A  166  CYS ASP LEU ALA ALA ARG THR VAL GLU SER ARG GLN ALA          \\nSEQRES  11 A  166  GLN ASP LEU ALA ARG SER TYR GLY ILE PRO TYR ILE GLU          \\nSEQRES  12 A  166  THR SER ALA LYS THR ARG GLN GLY VAL GLU ASP ALA PHE          \\nSEQRES  13 A  166  TYR THR LEU VAL ARG GLU ILE ARG GLN HIS                      \\nHET     MG  A 168       1                                                       \\nHET    GNP  A 167      32                                                       \\nHETNAM      MG MAGNESIUM ION                                                    \\nHETNAM     GNP PHOSPHOAMINOPHOSPHONIC ACID-GUANYLATE ESTER                      \\nFORMUL   2   MG    MG 2+                                                        \\nFORMUL   3  GNP    C10 H17 N6 O13 P3                                            \\nFORMUL   4  HOH   *211(H2 O)                                                    \\nHELIX    1  HA LYS A   16  GLN A   25  1                                  10    \\nHELIX    2  HB SER A   65  THR A   74  1RES 65-67 IN 3/10 CONF            10    \\nHELIX    3  HC THR A   87  VAL A  103  1BREAK AT ASP 92                   17    \\nHELIX    4  HD SER A  127  TYR A  137  1                                  11    \\nHELIX    5  HE VAL A  152  ARG A  164  1                                  13    \\nSHEET    1  S1 6 GLU A  37  ILE A  46  0                                        \\nSHEET    2  S1 6 GLU A  49  THR A  58 -1  O  LEU A  53   N  LYS A  42           \\nSHEET    3  S1 6 THR A   2  VAL A   9  1  N  LEU A   6   O  ASP A  54           \\nSHEET    4  S1 6 GLY A  77  ALA A  83  1  N  LEU A  79   O  VAL A   7           \\nSHEET    5  S1 6 MET A 111  ASN A 116  1  N  VAL A 114   O  CYS A  80           \\nSHEET    6  S1 6 TYR A 141  GLU A 143  1  N  ILE A 142   O  LEU A 113           \\nLINK        MG    MG A 168                 O2G GNP A 167     1555   1555  2.20  \\nLINK        MG    MG A 168                 O2B GNP A 167     1555   1555  2.22  \\nLINK        MG    MG A 168                 O   HOH A 173     1555   1555  2.23  \\nLINK        MG    MG A 168                 OG  SER A  17     1555   1555  2.26  \\nLINK        MG    MG A 168                 OG1 THR A  35     1555   1555  2.26  \\nLINK        MG    MG A 168                 O   HOH A 172     1555   1555  2.30  \\nSITE     1 AC1  5 SER A  17  THR A  35  GNP A 167  HOH A 172                    \\nSITE     2 AC1  5 HOH A 173                                                     \\nSITE     1 AC2 28 GLY A  12  GLY A  13  VAL A  14  GLY A  15                    \\nSITE     2 AC2 28 LYS A  16  SER A  17  ALA A  18  PHE A  28                    \\nSITE     3 AC2 28 VAL A  29  ASP A  30  TYR A  32  PRO A  34                    \\nSITE     4 AC2 28 THR A  35  GLY A  60  ASN A 116  LYS A 117                    \\nSITE     5 AC2 28 ASP A 119  SER A 145  ALA A 146  LYS A 147                    \\nSITE     6 AC2 28  MG A 168  HOH A 170  HOH A 172  HOH A 173                    \\nSITE     7 AC2 28 HOH A 175  HOH A 187  HOH A 281  HOH A 366                    \\nCRYST1   40.300   40.300  162.200  90.00  90.00 120.00 P 32 2 1      6          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.024814  0.014326  0.000000        0.00000                         \\nSCALE2      0.000000  0.028653  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.006165        0.00000                         \\nATOM      1  N   MET A   1      -7.186  32.862  -6.632  1.00 13.92           N  \\nATOM      2  CA  MET A   1      -5.921  32.127  -6.684  1.00 17.10           C  \\nATOM      3  C   MET A   1      -5.845  31.045  -5.621  1.00 16.74           C  \\nATOM      4  O   MET A   1      -6.600  31.083  -4.636  1.00 16.60           O  \\nATOM      5  CB  MET A   1      -4.673  33.058  -6.502  1.00 17.00           C  \\nATOM      6  CG  MET A   1      -4.633  33.804  -5.188  1.00 18.08           C  \\nATOM      7  SD  MET A   1      -3.115  34.766  -4.908  1.00 19.73           S  \\nATOM      8  CE  MET A   1      -3.019  35.851  -6.310  1.00 20.41           C  \\nATOM      9  N   THR A   2      -4.910  30.095  -5.820  1.00 14.70           N  \\nATOM     10  CA  THR A   2      -4.661  29.030  -4.859  1.00 16.92           C  \\nATOM     11  C   THR A   2      -4.230  29.655  -3.547  1.00 14.78           C  \\nATOM     12  O   THR A   2      -3.338  30.501  -3.537  1.00 13.93           O  \\nATOM     13  CB  THR A   2      -3.557  28.107  -5.401  1.00 18.03           C  \\nATOM     14  OG1 THR A   2      -4.059  27.565  -6.629  1.00 20.42           O  \\nATOM     15  CG2 THR A   2      -3.193  26.983  -4.443  1.00 18.51           C  \\nATOM     16  N   GLU A   3      -4.911  29.290  -2.472  1.00 14.80           N  \\nATOM     17  CA  GLU A   3      -4.504  29.730  -1.135  1.00 17.71           C  \\nATOM     18  C   GLU A   3      -4.012  28.553  -0.278  1.00 17.33           C  \\nATOM     19  O   GLU A   3      -4.520  27.427  -0.395  1.00 17.54           O  \\nATOM     20  CB  GLU A   3      -5.655  30.402  -0.432  1.00 20.67           C  \\nATOM     21  CG  GLU A   3      -6.213  31.631  -1.186  1.00 25.56           C  \\nATOM     22  CD  GLU A   3      -7.128  32.563  -0.369  1.00 28.46           C  \\nATOM     23  OE1 GLU A   3      -7.578  32.216   0.728  1.00 27.72           O  \\nATOM     24  OE2 GLU A   3      -7.375  33.680  -0.827  1.00 32.49           O  \\nATOM     25  N   TYR A   4      -2.962  28.744   0.542  1.00 14.50           N  \\nATOM     26  CA  TYR A   4      -2.425  27.654   1.358  1.00 12.29           C  \\nATOM     27  C   TYR A   4      -2.534  28.103   2.807  1.00 12.52           C  \\nATOM     28  O   TYR A   4      -2.016  29.143   3.200  1.00 12.11           O  \\nATOM     29  CB  TYR A   4      -0.975  27.385   1.026  1.00 11.46           C  \\nATOM     30  CG  TYR A   4      -0.726  26.846  -0.387  1.00 12.38           C  \\nATOM     31  CD1 TYR A   4      -0.297  27.698  -1.395  1.00 13.00           C  \\nATOM     32  CD2 TYR A   4      -0.933  25.501  -0.649  1.00 14.09           C  \\nATOM     33  CE1 TYR A   4      -0.070  27.208  -2.674  1.00 13.89           C  \\nATOM     34  CE2 TYR A   4      -0.711  24.993  -1.917  1.00 14.65           C  \\nATOM     35  CZ  TYR A   4      -0.279  25.859  -2.917  1.00 14.84           C  \\nATOM     36  OH  TYR A   4      -0.084  25.357  -4.191  1.00 15.07           O  \\nATOM     37  N   LYS A   5      -3.274  27.305   3.568  1.00 13.07           N  \\nATOM     38  CA  LYS A   5      -3.554  27.569   4.970  1.00 12.96           C  \\nATOM     39  C   LYS A   5      -2.556  26.818   5.852  1.00 12.74           C  \\nATOM     40  O   LYS A   5      -2.645  25.611   6.092  1.00 11.58           O  \\nATOM     41  CB  LYS A   5      -4.972  27.131   5.281  1.00 17.65           C  \\nATOM     42  CG  LYS A   5      -6.053  27.833   4.446  1.00 23.49           C  \\nATOM     43  CD  LYS A   5      -7.186  28.310   5.335  1.00 29.10           C  \\nATOM     44  CE  LYS A   5      -6.655  29.309   6.403  1.00 33.68           C  \\nATOM     45  NZ  LYS A   5      -6.597  28.739   7.755  1.00 36.52           N  \\nATOM     46  N   LEU A   6      -1.543  27.574   6.297  1.00 13.08           N  \\nATOM     47  CA  LEU A   6      -0.418  27.011   7.064  1.00 10.34           C  \\nATOM     48  C   LEU A   6      -0.578  27.382   8.535  1.00  9.68           C  \\nATOM     49  O   LEU A   6      -1.034  28.483   8.863  1.00 11.31           O  \\nATOM     50  CB  LEU A   6       0.910  27.571   6.536  1.00 11.34           C  \\nATOM     51  CG  LEU A   6       1.137  27.565   5.017  1.00 11.62           C  \\nATOM     52  CD1 LEU A   6       2.515  28.119   4.720  1.00 13.60           C  \\nATOM     53  CD2 LEU A   6       0.973  26.171   4.475  1.00 12.19           C  \\nATOM     54  N   VAL A   7      -0.196  26.486   9.432  1.00  9.93           N  \\nATOM     55  CA  VAL A   7      -0.333  26.787  10.852  1.00 10.06           C  \\nATOM     56  C   VAL A   7       1.033  26.513  11.495  1.00 10.37           C  \\nATOM     57  O   VAL A   7       1.597  25.441  11.252  1.00  9.51           O  \\nATOM     58  CB  VAL A   7      -1.474  25.884  11.446  1.00 10.97           C  \\nATOM     59  CG1 VAL A   7      -1.610  26.163  12.943  1.00 11.39           C  \\nATOM     60  CG2 VAL A   7      -2.809  26.161  10.777  1.00 10.14           C  \\nATOM     61  N   VAL A   8       1.576  27.455  12.295  1.00  8.35           N  \\nATOM     62  CA  VAL A   8       2.878  27.262  12.944  1.00  6.92           C  \\nATOM     63  C   VAL A   8       2.609  26.896  14.405  1.00  8.84           C  \\nATOM     64  O   VAL A   8       1.930  27.663  15.092  1.00  9.21           O  \\nATOM     65  CB  VAL A   8       3.684  28.565  12.799  1.00  8.08           C  \\nATOM     66  CG1 VAL A   8       5.111  28.324  13.261  1.00  9.52           C  \\nATOM     67  CG2 VAL A   8       3.729  29.011  11.358  1.00  8.43           C  \\nATOM     68  N   VAL A   9       3.024  25.701  14.881  1.00  8.73           N  \\nATOM     69  CA  VAL A   9       2.779  25.234  16.245  1.00  8.41           C  \\nATOM     70  C   VAL A   9       4.075  24.806  16.919  1.00  8.21           C  \\nATOM     71  O   VAL A   9       5.093  24.551  16.287  1.00 10.97           O  \\nATOM     72  CB  VAL A   9       1.824  24.028  16.286  1.00  9.74           C  \\nATOM     73  CG1 VAL A   9       0.545  24.491  15.680  1.00 11.76           C  \\nATOM     74  CG2 VAL A   9       2.349  22.810  15.535  1.00 10.31           C  \\nATOM     75  N   GLY A  10       4.048  24.707  18.238  1.00  7.34           N  \\nATOM     76  CA  GLY A  10       5.231  24.336  19.009  1.00  7.98           C  \\nATOM     77  C   GLY A  10       5.192  25.053  20.355  1.00  8.31           C  \\nATOM     78  O   GLY A  10       4.417  25.994  20.595  1.00  9.89           O  \\nATOM     79  N   ALA A  11       6.099  24.638  21.226  1.00  9.31           N  \\nATOM     80  CA  ALA A  11       6.121  25.135  22.597  1.00  9.50           C  \\nATOM     81  C   ALA A  11       6.440  26.625  22.711  1.00  9.66           C  \\nATOM     82  O   ALA A  11       6.998  27.250  21.806  1.00  9.46           O  \\nATOM     83  CB  ALA A  11       7.145  24.306  23.378  1.00  9.85           C  \\nATOM     84  N   GLY A  12       6.111  27.227  23.847  1.00 10.36           N  \\nATOM     85  CA  GLY A  12       6.406  28.632  24.054  1.00  9.30           C  \\nATOM     86  C   GLY A  12       7.892  28.950  23.964  1.00 10.23           C  \\nATOM     87  O   GLY A  12       8.785  28.249  24.466  1.00 10.57           O  \\nATOM     88  N   GLY A  13       8.159  30.064  23.266  1.00 11.49           N  \\nATOM     89  CA  GLY A  13       9.511  30.602  23.139  1.00 11.29           C  \\nATOM     90  C   GLY A  13      10.436  29.950  22.108  1.00 12.22           C  \\nATOM     91  O   GLY A  13      11.624  30.321  22.060  1.00 13.62           O  \\nATOM     92  N   VAL A  14       9.961  28.991  21.286  1.00  9.23           N  \\nATOM     93  CA  VAL A  14      10.848  28.327  20.317  1.00 10.38           C  \\nATOM     94  C   VAL A  14      11.169  29.168  19.076  1.00 10.47           C  \\nATOM     95  O   VAL A  14      12.108  28.841  18.353  1.00 12.07           O  \\nATOM     96  CB  VAL A  14      10.246  26.970  19.856  1.00  9.98           C  \\nATOM     97  CG1 VAL A  14      10.187  26.074  21.120  1.00  9.93           C  \\nATOM     98  CG2 VAL A  14       8.866  27.127  19.200  1.00  9.52           C  \\nATOM     99  N   GLY A  15      10.423  30.264  18.834  1.00  9.61           N  \\nATOM    100  CA  GLY A  15      10.666  31.160  17.706  1.00  9.70           C  \\nATOM    101  C   GLY A  15       9.615  31.061  16.626  1.00  8.82           C  \\nATOM    102  O   GLY A  15       9.915  31.403  15.491  1.00  8.99           O  \\nATOM    103  N   LYS A  16       8.395  30.591  16.916  1.00  8.18           N  \\nATOM    104  CA  LYS A  16       7.331  30.521  15.913  1.00  8.44           C  \\nATOM    105  C   LYS A  16       7.060  31.899  15.309  1.00  9.01           C  \\nATOM    106  O   LYS A  16       7.020  32.044  14.086  1.00  8.25           O  \\nATOM    107  CB  LYS A  16       6.014  30.000  16.529  1.00  9.27           C  \\nATOM    108  CG  LYS A  16       6.132  28.585  17.086  1.00  9.11           C  \\nATOM    109  CD  LYS A  16       4.820  28.077  17.726  1.00  9.31           C  \\nATOM    110  CE  LYS A  16       4.351  28.934  18.888  1.00 10.05           C  \\nATOM    111  NZ  LYS A  16       5.221  28.829  20.051  1.00  9.64           N  \\nATOM    112  N   SER A  17       6.913  32.942  16.135  1.00  7.74           N  \\nATOM    113  CA  SER A  17       6.661  34.295  15.649  1.00  8.78           C  \\nATOM    114  C   SER A  17       7.854  34.859  14.919  1.00  9.73           C  \\nATOM    115  O   SER A  17       7.653  35.446  13.868  1.00  9.70           O  \\nATOM    116  CB  SER A  17       6.360  35.264  16.769  1.00  8.22           C  \\nATOM    117  OG  SER A  17       5.083  34.853  17.204  1.00  8.79           O  \\nATOM    118  N   ALA A  18       9.072  34.687  15.444  1.00  8.95           N  \\nATOM    119  CA  ALA A  18      10.253  35.199  14.769  1.00  8.95           C  \\nATOM    120  C   ALA A  18      10.438  34.576  13.374  1.00  8.54           C  \\nATOM    121  O   ALA A  18      10.832  35.254  12.430  1.00  8.75           O  \\nATOM    122  CB  ALA A  18      11.521  34.931  15.640  1.00 10.11           C  \\nATOM    123  N   LEU A  19      10.136  33.284  13.176  1.00  8.31           N  \\nATOM    124  CA  LEU A  19      10.225  32.667  11.837  1.00  9.30           C  \\nATOM    125  C   LEU A  19       9.204  33.281  10.884  1.00 10.98           C  \\nATOM    126  O   LEU A  19       9.534  33.652   9.751  1.00 10.47           O  \\nATOM    127  CB  LEU A  19       9.938  31.165  11.885  1.00  9.41           C  \\nATOM    128  CG  LEU A  19      11.035  30.275  12.487  1.00 11.80           C  \\nATOM    129  CD1 LEU A  19      10.491  28.866  12.723  1.00 10.43           C  \\nATOM    130  CD2 LEU A  19      12.256  30.288  11.554  1.00 13.42           C  \\nATOM    131  N   THR A  20       7.950  33.419  11.335  1.00 10.72           N  \\nATOM    132  CA  THR A  20       6.888  34.015  10.529  1.00 11.87           C  \\nATOM    133  C   THR A  20       7.173  35.492  10.161  1.00 12.65           C  \\nATOM    134  O   THR A  20       7.012  35.917   9.009  1.00 12.60           O  \\nATOM    135  CB  THR A  20       5.547  33.888  11.312  1.00 13.43           C  \\nATOM    136  OG1 THR A  20       5.299  32.509  11.544  1.00 12.67           O  \\nATOM    137  CG2 THR A  20       4.346  34.410  10.518  1.00 14.64           C  \\nATOM    138  N   ILE A  21       7.615  36.326  11.098  1.00 10.48           N  \\nATOM    139  CA  ILE A  21       7.945  37.704  10.795  1.00 12.96           C  \\nATOM    140  C   ILE A  21       9.197  37.831   9.927  1.00 14.26           C  \\nATOM    141  O   ILE A  21       9.237  38.691   9.042  1.00 15.83           O  \\nATOM    142  CB  ILE A  21       8.090  38.451  12.121  1.00 12.57           C  \\nATOM    143  CG1 ILE A  21       6.691  38.551  12.737  1.00 13.24           C  \\nATOM    144  CG2 ILE A  21       8.631  39.852  11.910  1.00 13.40           C  \\nATOM    145  CD1 ILE A  21       6.636  39.185  14.129  1.00 16.46           C  \\nATOM    146  N   GLN A  22      10.206  36.979  10.092  1.00 13.58           N  \\nATOM    147  CA  GLN A  22      11.330  36.983   9.179  1.00 14.91           C  \\nATOM    148  C   GLN A  22      10.817  36.729   7.736  1.00 17.29           C  \\nATOM    149  O   GLN A  22      11.120  37.485   6.792  1.00 15.91           O  \\nATOM    150  CB  GLN A  22      12.270  35.930   9.736  1.00 15.90           C  \\nATOM    151  CG  GLN A  22      13.686  35.878   9.230  1.00 19.36           C  \\nATOM    152  CD  GLN A  22      14.372  37.217   9.091  1.00 18.08           C  \\nATOM    153  OE1 GLN A  22      14.160  37.984   8.143  1.00 19.23           O  \\nATOM    154  NE2 GLN A  22      15.224  37.492  10.050  1.00 17.40           N  \\nATOM    155  N   LEU A  23       9.944  35.718   7.523  1.00 15.95           N  \\nATOM    156  CA  LEU A  23       9.381  35.475   6.194  1.00 18.85           C  \\nATOM    157  C   LEU A  23       8.539  36.622   5.636  1.00 18.21           C  \\nATOM    158  O   LEU A  23       8.695  36.958   4.469  1.00 19.03           O  \\nATOM    159  CB  LEU A  23       8.498  34.235   6.196  1.00 18.10           C  \\nATOM    160  CG  LEU A  23       7.916  33.797   4.863  1.00 18.64           C  \\nATOM    161  CD1 LEU A  23       8.921  32.994   4.030  1.00 18.03           C  \\nATOM    162  CD2 LEU A  23       6.680  32.991   5.183  1.00 21.34           C  \\nATOM    163  N   ILE A  24       7.634  37.197   6.428  1.00 19.25           N  \\nATOM    164  CA  ILE A  24       6.682  38.198   5.953  1.00 20.23           C  \\nATOM    165  C   ILE A  24       7.316  39.590   5.922  1.00 21.89           C  \\nATOM    166  O   ILE A  24       7.125  40.318   4.943  1.00 22.93           O  \\nATOM    167  CB  ILE A  24       5.399  38.222   6.881  1.00 21.20           C  \\nATOM    168  CG1 ILE A  24       4.664  36.907   6.821  1.00 21.27           C  \\nATOM    169  CG2 ILE A  24       4.423  39.307   6.446  1.00 20.00           C  \\nATOM    170  CD1 ILE A  24       4.128  36.442   5.475  1.00 23.98           C  \\nATOM    171  N   GLN A  25       8.074  39.998   6.953  1.00 21.07           N  \\nATOM    172  CA  GLN A  25       8.520  41.380   7.066  1.00 22.09           C  \\nATOM    173  C   GLN A  25      10.003  41.559   6.962  1.00 20.75           C  \\nATOM    174  O   GLN A  25      10.495  42.677   7.054  1.00 20.26           O  \\nATOM    175  CB  GLN A  25       8.039  41.968   8.398  1.00 23.96           C  \\nATOM    176  CG  GLN A  25       6.548  42.276   8.381  1.00 28.69           C  \\nATOM    177  CD  GLN A  25       5.931  42.574   9.744  1.00 30.92           C  \\nATOM    178  OE1 GLN A  25       6.537  43.143  10.657  1.00 31.89           O  \\nATOM    179  NE2 GLN A  25       4.664  42.169   9.867  1.00 32.35           N  \\nATOM    180  N   ASN A  26      10.764  40.482   6.795  1.00 22.13           N  \\nATOM    181  CA  ASN A  26      12.198  40.642   6.628  1.00 24.67           C  \\nATOM    182  C   ASN A  26      12.936  41.362   7.734  1.00 23.62           C  \\nATOM    183  O   ASN A  26      13.603  42.372   7.521  1.00 25.10           O  \\nATOM    184  CB  ASN A  26      12.522  41.369   5.315  1.00 27.68           C  \\nATOM    185  CG  ASN A  26      12.942  40.351   4.288  1.00 32.80           C  \\nATOM    186  OD1 ASN A  26      12.135  39.699   3.602  1.00 33.77           O  \\nATOM    187  ND2 ASN A  26      14.268  40.203   4.237  1.00 34.26           N  \\nATOM    188  N   HIS A  27      12.764  40.842   8.945  1.00 19.95           N  \\nATOM    189  CA  HIS A  27      13.593  41.266  10.057  1.00 17.99           C  \\nATOM    190  C   HIS A  27      13.441  40.299  11.215  1.00 16.84           C  \\nATOM    191  O   HIS A  27      12.411  39.623  11.344  1.00 14.72           O  \\nATOM    192  CB  HIS A  27      13.241  42.680  10.551  1.00 19.80           C  \\nATOM    193  CG  HIS A  27      11.926  42.908  11.258  1.00 20.77           C  \\nATOM    194  ND1 HIS A  27      11.936  43.314  12.510  1.00 22.17           N  \\nATOM    195  CD2 HIS A  27      10.674  42.768  10.729  1.00 22.32           C  \\nATOM    196  CE1 HIS A  27      10.657  43.430  12.785  1.00 23.86           C  \\nATOM    197  NE2 HIS A  27       9.905  43.110  11.738  1.00 23.97           N  \\nATOM    198  N   PHE A  28      14.516  40.222  12.012  1.00 15.44           N  \\nATOM    199  CA  PHE A  28      14.528  39.413  13.223  1.00 13.89           C  \\nATOM    200  C   PHE A  28      14.026  40.196  14.440  1.00 15.33           C  \\nATOM    201  O   PHE A  28      14.528  41.255  14.786  1.00 16.80           O  \\nATOM    202  CB  PHE A  28      15.938  38.906  13.470  1.00 12.73           C  \\nATOM    203  CG  PHE A  28      16.083  38.150  14.786  1.00 13.35           C  \\nATOM    204  CD1 PHE A  28      15.354  36.990  15.012  1.00 13.80           C  \\nATOM    205  CD2 PHE A  28      16.974  38.615  15.740  1.00 14.54           C  \\nATOM    206  CE1 PHE A  28      15.534  36.310  16.194  1.00 14.45           C  \\nATOM    207  CE2 PHE A  28      17.147  37.925  16.920  1.00 14.61           C  \\nATOM    208  CZ  PHE A  28      16.420  36.768  17.137  1.00 14.65           C  \\nATOM    209  N   VAL A  29      12.970  39.697  15.089  1.00 17.22           N  \\nATOM    210  CA  VAL A  29      12.467  40.313  16.294  1.00 19.11           C  \\nATOM    211  C   VAL A  29      13.127  39.606  17.476  1.00 19.36           C  \\nATOM    212  O   VAL A  29      13.245  38.376  17.605  1.00 18.14           O  \\nATOM    213  CB  VAL A  29      10.894  40.205  16.412  1.00 22.03           C  \\nATOM    214  CG1 VAL A  29      10.334  40.869  15.143  1.00 19.66           C  \\nATOM    215  CG2 VAL A  29      10.377  38.776  16.631  1.00 23.15           C  \\nATOM    216  N   ASP A  30      13.615  40.484  18.324  1.00 20.14           N  \\nATOM    217  CA  ASP A  30      14.229  40.019  19.518  1.00 24.04           C  \\nATOM    218  C   ASP A  30      13.676  40.746  20.721  1.00 25.93           C  \\nATOM    219  O   ASP A  30      14.244  41.709  21.238  1.00 29.26           O  \\nATOM    220  CB  ASP A  30      15.719  40.203  19.443  1.00 27.10           C  \\nATOM    221  CG  ASP A  30      16.389  39.731  20.732  1.00 32.46           C  \\nATOM    222  OD1 ASP A  30      15.878  38.859  21.464  1.00 34.28           O  \\nATOM    223  OD2 ASP A  30      17.458  40.264  21.002  1.00 36.46           O  \\nATOM    224  N   GLU A  31      12.472  40.347  21.076  1.00 26.14           N  \\nATOM    225  CA  GLU A  31      11.922  40.804  22.318  1.00 28.72           C  \\nATOM    226  C   GLU A  31      10.870  39.836  22.749  1.00 26.20           C  \\nATOM    227  O   GLU A  31      10.466  38.931  22.014  1.00 23.14           O  \\nATOM    228  CB  GLU A  31      11.337  42.210  22.204  1.00 33.25           C  \\nATOM    229  CG  GLU A  31      10.350  42.618  21.152  1.00 40.08           C  \\nATOM    230  CD  GLU A  31      10.572  44.107  20.887  1.00 44.71           C  \\nATOM    231  OE1 GLU A  31      10.930  44.444  19.755  1.00 47.50           O  \\nATOM    232  OE2 GLU A  31      10.420  44.928  21.806  1.00 47.89           O  \\nATOM    233  N   TYR A  32      10.550  40.044  24.028  1.00 22.26           N  \\nATOM    234  CA  TYR A  32       9.597  39.196  24.685  1.00 18.83           C  \\nATOM    235  C   TYR A  32       8.204  39.688  24.317  1.00 17.53           C  \\nATOM    236  O   TYR A  32       7.794  40.792  24.676  1.00 18.45           O  \\nATOM    237  CB  TYR A  32       9.909  39.259  26.199  1.00 16.84           C  \\nATOM    238  CG  TYR A  32       9.076  38.309  27.052  1.00 15.64           C  \\nATOM    239  CD1 TYR A  32       9.598  37.096  27.433  1.00 16.17           C  \\nATOM    240  CD2 TYR A  32       7.814  38.686  27.459  1.00 15.54           C  \\nATOM    241  CE1 TYR A  32       8.845  36.251  28.239  1.00 17.86           C  \\nATOM    242  CE2 TYR A  32       7.051  37.861  28.257  1.00 14.40           C  \\nATOM    243  CZ  TYR A  32       7.572  36.635  28.652  1.00 16.47           C  \\nATOM    244  OH  TYR A  32       6.821  35.789  29.485  1.00 13.59           O  \\nATOM    245  N   ASP A  33       7.461  38.841  23.604  1.00 13.43           N  \\nATOM    246  CA  ASP A  33       6.128  39.200  23.141  1.00 14.00           C  \\nATOM    247  C   ASP A  33       5.262  37.955  22.854  1.00 13.56           C  \\nATOM    248  O   ASP A  33       5.024  37.586  21.681  1.00 12.47           O  \\nATOM    249  CB  ASP A  33       6.271  40.072  21.864  1.00 15.92           C  \\nATOM    250  CG  ASP A  33       4.993  40.654  21.269  1.00 18.19           C  \\nATOM    251  OD1 ASP A  33       3.923  40.631  21.879  1.00 21.46           O  \\nATOM    252  OD2 ASP A  33       5.074  41.164  20.167  1.00 20.34           O  \\nATOM    253  N   PRO A  34       4.752  37.283  23.903  1.00 12.04           N  \\nATOM    254  CA  PRO A  34       3.949  36.070  23.788  1.00 11.09           C  \\nATOM    255  C   PRO A  34       2.720  36.248  22.895  1.00 11.76           C  \\nATOM    256  O   PRO A  34       1.953  37.220  23.006  1.00 12.63           O  \\nATOM    257  CB  PRO A  34       3.590  35.711  25.219  1.00 12.50           C  \\nATOM    258  CG  PRO A  34       4.643  36.418  26.033  1.00 12.14           C  \\nATOM    259  CD  PRO A  34       4.817  37.717  25.311  1.00 12.82           C  \\nATOM    260  N   THR A  35       2.554  35.282  21.972  1.00 11.00           N  \\nATOM    261  CA  THR A  35       1.421  35.320  21.033  1.00 11.54           C  \\nATOM    262  C   THR A  35       0.140  34.746  21.639  1.00 13.51           C  \\nATOM    263  O   THR A  35       0.174  33.820  22.450  1.00 12.45           O  \\nATOM    264  CB  THR A  35       1.730  34.518  19.751  1.00 11.34           C  \\nATOM    265  OG1 THR A  35       2.900  35.134  19.233  1.00 11.20           O  \\nATOM    266  CG2 THR A  35       0.706  34.611  18.637  1.00 13.10           C  \\nATOM    267  N   ILE A  36      -0.996  35.332  21.233  1.00 15.48           N  \\nATOM    268  CA  ILE A  36      -2.308  34.792  21.567  1.00 19.42           C  \\nATOM    269  C   ILE A  36      -2.663  34.079  20.274  1.00 17.36           C  \\nATOM    270  O   ILE A  36      -2.749  32.863  20.256  1.00 18.29           O  \\nATOM    271  CB  ILE A  36      -3.353  35.914  21.879  1.00 21.30           C  \\nATOM    272  CG1 ILE A  36      -2.930  36.823  23.072  1.00 24.32           C  \\nATOM    273  CG2 ILE A  36      -4.670  35.207  22.136  1.00 21.54           C  \\nATOM    274  CD1 ILE A  36      -2.494  36.167  24.413  1.00 24.70           C  \\nATOM    275  N   GLU A  37      -2.818  34.822  19.181  1.00 19.35           N  \\nATOM    276  CA  GLU A  37      -3.059  34.221  17.889  1.00 21.47           C  \\nATOM    277  C   GLU A  37      -2.957  35.362  16.897  1.00 23.77           C  \\nATOM    278  O   GLU A  37      -3.636  36.393  17.031  1.00 25.49           O  \\nATOM    279  CB  GLU A  37      -4.440  33.611  17.880  1.00 25.97           C  \\nATOM    280  CG  GLU A  37      -4.766  32.776  16.667  1.00 33.81           C  \\nATOM    281  CD  GLU A  37      -6.160  32.154  16.777  1.00 38.37           C  \\nATOM    282  OE1 GLU A  37      -7.095  32.664  16.149  1.00 37.86           O  \\nATOM    283  OE2 GLU A  37      -6.298  31.164  17.515  1.00 42.24           O  \\nATOM    284  N   ASP A  38      -2.070  35.212  15.911  1.00 19.96           N  \\nATOM    285  CA  ASP A  38      -1.899  36.178  14.839  1.00 17.94           C  \\nATOM    286  C   ASP A  38      -2.024  35.526  13.473  1.00 18.21           C  \\nATOM    287  O   ASP A  38      -1.582  34.393  13.294  1.00 16.96           O  \\nATOM    288  CB  ASP A  38      -0.540  36.833  14.965  1.00 18.83           C  \\nATOM    289  CG  ASP A  38      -0.507  37.817  16.133  1.00 20.08           C  \\nATOM    290  OD1 ASP A  38      -1.390  38.660  16.208  1.00 25.47           O  \\nATOM    291  OD2 ASP A  38       0.375  37.781  16.967  1.00 20.08           O  \\nATOM    292  N   SER A  39      -2.698  36.170  12.507  1.00 17.82           N  \\nATOM    293  CA  SER A  39      -2.691  35.658  11.133  1.00 18.16           C  \\nATOM    294  C   SER A  39      -2.012  36.627  10.171  1.00 16.95           C  \\nATOM    295  O   SER A  39      -2.053  37.853  10.333  1.00 15.80           O  \\nATOM    296  CB  SER A  39      -4.094  35.402  10.637  1.00 23.01           C  \\nATOM    297  OG  SER A  39      -4.873  36.571  10.754  1.00 27.13           O  \\nATOM    298  N   TYR A  40      -1.336  36.080   9.168  1.00 15.44           N  \\nATOM    299  CA  TYR A  40      -0.607  36.889   8.196  1.00 16.25           C  \\nATOM    300  C   TYR A  40      -1.018  36.383   6.830  1.00 17.06           C  \\nATOM    301  O   TYR A  40      -1.160  35.172   6.638  1.00 18.29           O  \\nATOM    302  CB  TYR A  40       0.909  36.724   8.370  1.00 16.59           C  \\nATOM    303  CG  TYR A  40       1.439  37.242   9.707  1.00 16.10           C  \\nATOM    304  CD1 TYR A  40       1.473  36.412  10.815  1.00 16.77           C  \\nATOM    305  CD2 TYR A  40       1.883  38.549   9.799  1.00 18.02           C  \\nATOM    306  CE1 TYR A  40       1.950  36.893  12.021  1.00 16.86           C  \\nATOM    307  CE2 TYR A  40       2.368  39.032  11.001  1.00 18.70           C  \\nATOM    308  CZ  TYR A  40       2.396  38.199  12.104  1.00 17.62           C  \\nATOM    309  OH  TYR A  40       2.853  38.695  13.304  1.00 18.37           O  \\nATOM    310  N   ARG A  41      -1.270  37.303   5.900  1.00 15.48           N  \\nATOM    311  CA  ARG A  41      -1.660  36.918   4.558  1.00 15.42           C  \\nATOM    312  C   ARG A  41      -0.730  37.606   3.586  1.00 16.76           C  \\nATOM    313  O   ARG A  41      -0.536  38.829   3.633  1.00 17.01           O  \\nATOM    314  CB  ARG A  41      -3.087  37.342   4.277  1.00 17.64           C  \\nATOM    315  CG  ARG A  41      -4.150  36.507   4.947  1.00 18.09           C  \\nATOM    316  CD  ARG A  41      -5.528  37.181   4.849  1.00 19.64           C  \\nATOM    317  NE  ARG A  41      -6.066  37.318   3.489  1.00 20.11           N  \\nATOM    318  CZ  ARG A  41      -6.660  36.329   2.803  1.00 22.17           C  \\nATOM    319  NH1 ARG A  41      -7.219  36.571   1.617  1.00 22.48           N  \\nATOM    320  NH2 ARG A  41      -6.711  35.090   3.274  1.00 23.55           N  \\nATOM    321  N   LYS A  42      -0.135  36.827   2.679  1.00 16.52           N  \\nATOM    322  CA  LYS A  42       0.835  37.405   1.758  1.00 16.59           C  \\nATOM    323  C   LYS A  42       0.740  36.746   0.387  1.00 16.04           C  \\nATOM    324  O   LYS A  42       0.794  35.528   0.307  1.00 14.74           O  \\nATOM    325  CB  LYS A  42       2.208  37.210   2.401  1.00 18.22           C  \\nATOM    326  CG  LYS A  42       3.388  37.784   1.669  1.00 22.68           C  \\nATOM    327  CD  LYS A  42       3.369  39.274   1.792  1.00 28.09           C  \\nATOM    328  CE  LYS A  42       4.695  39.876   1.317  1.00 31.70           C  \\nATOM    329  NZ  LYS A  42       4.828  39.827  -0.128  1.00 35.70           N  \\nATOM    330  N   GLN A  43       0.561  37.510  -0.697  1.00 16.89           N  \\nATOM    331  CA  GLN A  43       0.579  36.950  -2.047  1.00 19.10           C  \\nATOM    332  C   GLN A  43       2.026  36.729  -2.448  1.00 15.08           C  \\nATOM    333  O   GLN A  43       2.828  37.654  -2.362  1.00 15.72           O  \\nATOM    334  CB  GLN A  43      -0.116  37.908  -3.043  1.00 23.32           C  \\nATOM    335  CG  GLN A  43       0.543  38.332  -4.415  1.00 32.83           C  \\nATOM    336  CD  GLN A  43       1.081  37.332  -5.482  1.00 36.11           C  \\nATOM    337  OE1 GLN A  43       0.457  36.335  -5.865  1.00 37.78           O  \\nATOM    338  NE2 GLN A  43       2.271  37.606  -6.053  1.00 35.45           N  \\nATOM    339  N   VAL A  44       2.394  35.532  -2.922  1.00 12.79           N  \\nATOM    340  CA  VAL A  44       3.771  35.242  -3.311  1.00 11.03           C  \\nATOM    341  C   VAL A  44       3.724  34.425  -4.605  1.00 14.28           C  \\nATOM    342  O   VAL A  44       2.633  34.109  -5.090  1.00 14.90           O  \\nATOM    343  CB  VAL A  44       4.508  34.411  -2.223  1.00 13.21           C  \\nATOM    344  CG1 VAL A  44       4.669  35.282  -0.977  1.00 15.04           C  \\nATOM    345  CG2 VAL A  44       3.761  33.116  -1.889  1.00 12.89           C  \\nATOM    346  N   VAL A  45       4.879  34.127  -5.235  1.00 13.20           N  \\nATOM    347  CA  VAL A  45       4.938  33.277  -6.419  1.00 12.28           C  \\nATOM    348  C   VAL A  45       5.836  32.161  -5.970  1.00 11.08           C  \\nATOM    349  O   VAL A  45       6.988  32.447  -5.636  1.00 13.13           O  \\nATOM    350  CB  VAL A  45       5.587  33.979  -7.641  1.00 13.01           C  \\nATOM    351  CG1 VAL A  45       5.549  33.031  -8.810  1.00 14.51           C  \\nATOM    352  CG2 VAL A  45       4.787  35.173  -8.108  1.00 13.54           C  \\nATOM    353  N   ILE A  46       5.374  30.907  -5.915  1.00  9.89           N  \\nATOM    354  CA  ILE A  46       6.274  29.795  -5.491  1.00 11.03           C  \\nATOM    355  C   ILE A  46       6.369  28.800  -6.662  1.00  8.85           C  \\nATOM    356  O   ILE A  46       5.319  28.296  -7.059  1.00 10.25           O  \\nATOM    357  CB  ILE A  46       5.692  29.115  -4.203  1.00 10.64           C  \\nATOM    358  CG1 ILE A  46       5.652  30.096  -3.042  1.00 11.20           C  \\nATOM    359  CG2 ILE A  46       6.599  27.953  -3.757  1.00 12.93           C  \\nATOM    360  CD1 ILE A  46       4.937  29.536  -1.808  1.00  8.90           C  \\nATOM    361  N   ASP A  47       7.565  28.495  -7.224  1.00  9.41           N  \\nATOM    362  CA  ASP A  47       7.767  27.650  -8.426  1.00  9.70           C  \\nATOM    363  C   ASP A  47       6.838  28.031  -9.572  1.00 11.14           C  \\nATOM    364  O   ASP A  47       6.225  27.211 -10.248  1.00 14.09           O  \\nATOM    365  CB  ASP A  47       7.566  26.165  -8.105  1.00 10.73           C  \\nATOM    366  CG  ASP A  47       8.388  25.783  -6.874  1.00 13.52           C  \\nATOM    367  OD1 ASP A  47       7.805  25.291  -5.898  1.00 12.04           O  \\nATOM    368  OD2 ASP A  47       9.598  26.024  -6.872  1.00 11.31           O  \\nATOM    369  N   GLY A  48       6.724  29.353  -9.735  1.00 10.78           N  \\nATOM    370  CA  GLY A  48       5.930  29.961 -10.785  1.00 13.29           C  \\nATOM    371  C   GLY A  48       4.426  29.989 -10.548  1.00 14.28           C  \\nATOM    372  O   GLY A  48       3.677  30.512 -11.386  1.00 15.19           O  \\nATOM    373  N   GLU A  49       3.944  29.422  -9.439  1.00 13.41           N  \\nATOM    374  CA  GLU A  49       2.520  29.457  -9.143  1.00 13.90           C  \\nATOM    375  C   GLU A  49       2.223  30.700  -8.314  1.00 15.34           C  \\nATOM    376  O   GLU A  49       2.860  30.906  -7.275  1.00 14.56           O  \\nATOM    377  CB  GLU A  49       2.124  28.238  -8.340  1.00 15.50           C  \\nATOM    378  CG  GLU A  49       0.630  28.250  -7.990  1.00 18.10           C  \\nATOM    379  CD  GLU A  49       0.204  27.110  -7.100  1.00 19.13           C  \\nATOM    380  OE1 GLU A  49       0.717  27.004  -5.996  1.00 19.53           O  \\nATOM    381  OE2 GLU A  49      -0.630  26.321  -7.529  1.00 22.12           O  \\nATOM    382  N   THR A  50       1.267  31.536  -8.734  1.00 14.06           N  \\nATOM    383  CA  THR A  50       0.901  32.697  -7.927  1.00 16.15           C  \\nATOM    384  C   THR A  50      -0.001  32.169  -6.819  1.00 13.35           C  \\nATOM    385  O   THR A  50      -0.938  31.409  -7.046  1.00 15.13           O  \\nATOM    386  CB  THR A  50       0.203  33.755  -8.846  1.00 17.88           C  \\nATOM    387  OG1 THR A  50      -0.245  34.813  -8.012  1.00 24.33           O  \\nATOM    388  CG2 THR A  50      -1.028  33.236  -9.518  1.00 19.78           C  \\nATOM    389  N   CYS A  51       0.306  32.442  -5.563  1.00 13.28           N  \\nATOM    390  CA  CYS A  51      -0.566  31.959  -4.508  1.00 15.16           C  \\nATOM    391  C   CYS A  51      -0.579  32.892  -3.303  1.00 14.00           C  \\nATOM    392  O   CYS A  51       0.161  33.874  -3.231  1.00 11.81           O  \\nATOM    393  CB  CYS A  51      -0.132  30.537  -4.079  1.00 14.08           C  \\nATOM    394  SG  CYS A  51       1.597  30.409  -3.575  1.00 16.09           S  \\nATOM    395  N   LEU A  52      -1.521  32.609  -2.405  1.00 13.77           N  \\nATOM    396  CA  LEU A  52      -1.573  33.397  -1.199  1.00 16.01           C  \\nATOM    397  C   LEU A  52      -1.260  32.497  -0.025  1.00 14.53           C  \\nATOM    398  O   LEU A  52      -1.868  31.443   0.113  1.00 16.05           O  \\nATOM    399  CB  LEU A  52      -2.957  33.986  -1.014  1.00 19.71           C  \\nATOM    400  CG  LEU A  52      -3.122  34.885   0.214  1.00 20.84           C  \\nATOM    401  CD1 LEU A  52      -3.070  36.329  -0.220  1.00 21.38           C  \\nATOM    402  CD2 LEU A  52      -4.420  34.526   0.923  1.00 20.90           C  \\nATOM    403  N   LEU A  53      -0.319  32.904   0.800  1.00 12.41           N  \\nATOM    404  CA  LEU A  53       0.019  32.232   2.048  1.00 13.70           C  \\nATOM    405  C   LEU A  53      -0.841  32.852   3.148  1.00 13.85           C  \\nATOM    406  O   LEU A  53      -0.733  34.056   3.413  1.00 12.82           O  \\nATOM    407  CB  LEU A  53       1.485  32.464   2.400  1.00 14.51           C  \\nATOM    408  CG  LEU A  53       2.707  31.589   2.031  1.00 19.29           C  \\nATOM    409  CD1 LEU A  53       2.394  30.553   0.962  1.00 18.56           C  \\nATOM    410  CD2 LEU A  53       3.843  32.557   1.741  1.00 20.00           C  \\nATOM    411  N   ASP A  54      -1.699  32.023   3.751  1.00 11.96           N  \\nATOM    412  CA  ASP A  54      -2.476  32.445   4.884  1.00 15.07           C  \\nATOM    413  C   ASP A  54      -1.882  31.675   6.055  1.00 12.67           C  \\nATOM    414  O   ASP A  54      -2.099  30.477   6.234  1.00 13.99           O  \\nATOM    415  CB  ASP A  54      -3.930  32.088   4.644  1.00 19.33           C  \\nATOM    416  CG  ASP A  54      -4.826  32.609   5.746  1.00 24.24           C  \\nATOM    417  OD1 ASP A  54      -4.412  32.652   6.897  1.00 28.17           O  \\nATOM    418  OD2 ASP A  54      -5.958  32.979   5.450  1.00 27.59           O  \\nATOM    419  N   ILE A  55      -1.127  32.389   6.880  1.00 11.78           N  \\nATOM    420  CA  ILE A  55      -0.346  31.769   7.935  1.00 12.15           C  \\nATOM    421  C   ILE A  55      -0.953  32.099   9.296  1.00 13.10           C  \\nATOM    422  O   ILE A  55      -1.195  33.271   9.587  1.00 13.84           O  \\nATOM    423  CB  ILE A  55       1.109  32.282   7.931  1.00 12.42           C  \\nATOM    424  CG1 ILE A  55       1.859  31.979   6.653  1.00 13.99           C  \\nATOM    425  CG2 ILE A  55       1.859  31.505   9.018  1.00 13.24           C  \\nATOM    426  CD1 ILE A  55       3.176  32.768   6.497  1.00 15.68           C  \\nATOM    427  N   LEU A  56      -1.176  31.083  10.134  1.00 11.64           N  \\nATOM    428  CA  LEU A  56      -1.642  31.341  11.490  1.00 11.98           C  \\nATOM    429  C   LEU A  56      -0.524  31.027  12.431  1.00  9.99           C  \\nATOM    430  O   LEU A  56      -0.017  29.895  12.447  1.00  9.72           O  \\nATOM    431  CB  LEU A  56      -2.836  30.463  11.896  1.00 13.80           C  \\nATOM    432  CG  LEU A  56      -3.470  30.741  13.297  1.00 16.13           C  \\nATOM    433  CD1 LEU A  56      -4.347  31.987  13.260  1.00 15.28           C  \\nATOM    434  CD2 LEU A  56      -4.321  29.580  13.692  1.00 16.84           C  \\nATOM    435  N   ASP A  57      -0.188  32.084  13.182  1.00 10.40           N  \\nATOM    436  CA  ASP A  57       0.815  32.014  14.219  1.00 10.30           C  \\nATOM    437  C   ASP A  57       0.127  31.812  15.571  1.00 11.75           C  \\nATOM    438  O   ASP A  57      -0.652  32.631  16.053  1.00 12.91           O  \\nATOM    439  CB  ASP A  57       1.602  33.296  14.173  1.00 10.66           C  \\nATOM    440  CG  ASP A  57       2.778  33.336  15.132  1.00 11.39           C  \\nATOM    441  OD1 ASP A  57       3.306  32.298  15.535  1.00 11.07           O  \\nATOM    442  OD2 ASP A  57       3.168  34.448  15.455  1.00 13.15           O  \\nATOM    443  N   THR A  58       0.447  30.723  16.244  1.00 12.55           N  \\nATOM    444  CA  THR A  58      -0.324  30.325  17.416  1.00 12.98           C  \\nATOM    445  C   THR A  58       0.368  30.541  18.758  1.00 13.48           C  \\nATOM    446  O   THR A  58       1.568  30.812  18.815  1.00 14.00           O  \\nATOM    447  CB  THR A  58      -0.698  28.851  17.219  1.00  9.54           C  \\nATOM    448  OG1 THR A  58       0.491  28.084  17.369  1.00 11.40           O  \\nATOM    449  CG2 THR A  58      -1.313  28.597  15.848  1.00 12.27           C  \\nATOM    450  N   ALA A  59      -0.371  30.382  19.868  1.00 14.38           N  \\nATOM    451  CA  ALA A  59       0.230  30.510  21.194  1.00 14.79           C  \\nATOM    452  C   ALA A  59       0.928  29.210  21.608  1.00 15.41           C  \\nATOM    453  O   ALA A  59       0.448  28.106  21.324  1.00 13.07           O  \\nATOM    454  CB  ALA A  59      -0.845  30.818  22.238  1.00 15.47           C  \\nATOM    455  N   GLY A  60       2.084  29.341  22.271  1.00 17.71           N  \\nATOM    456  CA  GLY A  60       2.807  28.208  22.814  1.00 24.18           C  \\nATOM    457  C   GLY A  60       2.190  27.751  24.130  1.00 31.21           C  \\nATOM    458  O   GLY A  60       2.538  26.688  24.627  1.00 30.44           O  \\nATOM    459  N   GLN A  61       1.260  28.503  24.726  1.00 39.50           N  \\nATOM    460  CA  GLN A  61       0.671  28.099  25.994  1.00 48.84           C  \\nATOM    461  C   GLN A  61      -0.683  27.395  25.985  1.00 51.52           C  \\nATOM    462  O   GLN A  61      -1.606  27.780  25.276  1.00 52.17           O  \\nATOM    463  CB  GLN A  61       0.605  29.327  26.874  1.00 51.56           C  \\nATOM    464  CG  GLN A  61       0.871  28.861  28.312  1.00 56.74           C  \\nATOM    465  CD  GLN A  61       1.506  29.937  29.177  1.00 58.74           C  \\nATOM    466  OE1 GLN A  61       2.647  29.795  29.616  1.00 60.12           O  \\nATOM    467  NE2 GLN A  61       0.783  31.031  29.429  1.00 60.18           N  \\nATOM    468  N   GLU A  62      -0.804  26.349  26.818  1.00 57.40           N  \\nATOM    469  CA  GLU A  62      -2.008  25.527  26.927  1.00 61.96           C  \\nATOM    470  C   GLU A  62      -3.185  26.196  27.631  1.00 62.20           C  \\nATOM    471  O   GLU A  62      -4.138  25.496  27.975  1.00 62.85           O  \\nATOM    472  CB  GLU A  62      -1.629  24.183  27.632  1.00 65.18           C  \\nATOM    473  CG  GLU A  62      -2.622  22.966  27.761  1.00 68.22           C  \\nATOM    474  CD  GLU A  62      -3.032  22.132  26.528  1.00 70.25           C  \\nATOM    475  OE1 GLU A  62      -2.710  22.494  25.395  1.00 71.45           O  \\nATOM    476  OE2 GLU A  62      -3.691  21.101  26.689  1.00 70.00           O  \\nATOM    477  N   GLU A  63      -3.206  27.520  27.904  1.00 62.78           N  \\nATOM    478  CA  GLU A  63      -4.496  28.139  28.254  1.00 63.62           C  \\nATOM    479  C   GLU A  63      -5.457  27.854  27.085  1.00 63.71           C  \\nATOM    480  O   GLU A  63      -6.688  27.814  27.211  1.00 62.97           O  \\nATOM    481  CB  GLU A  63      -4.372  29.666  28.475  1.00 65.13           C  \\nATOM    482  CG  GLU A  63      -3.456  30.553  27.607  1.00 66.91           C  \\nATOM    483  CD  GLU A  63      -4.012  31.235  26.345  1.00 68.05           C  \\nATOM    484  OE1 GLU A  63      -5.220  31.462  26.220  1.00 68.64           O  \\nATOM    485  OE2 GLU A  63      -3.198  31.568  25.481  1.00 67.62           O  \\nATOM    486  N   TYR A  64      -4.801  27.606  25.934  1.00 63.36           N  \\nATOM    487  CA  TYR A  64      -5.420  27.092  24.743  1.00 63.20           C  \\nATOM    488  C   TYR A  64      -5.905  25.653  24.875  1.00 61.76           C  \\nATOM    489  O   TYR A  64      -7.120  25.486  24.858  1.00 63.04           O  \\nATOM    490  CB  TYR A  64      -4.437  27.208  23.613  1.00 64.83           C  \\nATOM    491  CG  TYR A  64      -4.736  28.459  22.818  1.00 66.98           C  \\nATOM    492  CD1 TYR A  64      -4.300  29.686  23.267  1.00 67.55           C  \\nATOM    493  CD2 TYR A  64      -5.459  28.355  21.646  1.00 67.97           C  \\nATOM    494  CE1 TYR A  64      -4.587  30.825  22.536  1.00 69.49           C  \\nATOM    495  CE2 TYR A  64      -5.752  29.488  20.911  1.00 69.29           C  \\nATOM    496  CZ  TYR A  64      -5.313  30.718  21.359  1.00 70.05           C  \\nATOM    497  OH  TYR A  64      -5.589  31.852  20.614  1.00 70.67           O  \\nATOM    498  N   SER A  65      -5.095  24.590  24.997  1.00 58.67           N  \\nATOM    499  CA  SER A  65      -5.617  23.230  25.127  1.00 56.93           C  \\nATOM    500  C   SER A  65      -6.614  22.757  24.063  1.00 55.33           C  \\nATOM    501  O   SER A  65      -6.321  22.936  22.876  1.00 54.74           O  \\nATOM    502  CB  SER A  65      -6.190  23.122  26.552  1.00 57.99           C  \\nATOM    503  OG  SER A  65      -7.251  24.011  26.900  1.00 59.80           O  \\nATOM    504  N   ALA A  66      -7.762  22.119  24.359  1.00 52.33           N  \\nATOM    505  CA  ALA A  66      -8.766  21.824  23.335  1.00 48.80           C  \\nATOM    506  C   ALA A  66      -9.138  23.050  22.492  1.00 45.71           C  \\nATOM    507  O   ALA A  66      -9.751  22.862  21.449  1.00 44.33           O  \\nATOM    508  CB  ALA A  66     -10.043  21.300  23.971  1.00 49.14           C  \\nATOM    509  N   MET A  67      -8.829  24.310  22.875  1.00 42.85           N  \\nATOM    510  CA  MET A  67      -8.984  25.465  21.967  1.00 40.01           C  \\nATOM    511  C   MET A  67      -8.093  25.362  20.719  1.00 36.79           C  \\nATOM    512  O   MET A  67      -8.221  26.184  19.805  1.00 36.08           O  \\nATOM    513  CB  MET A  67      -8.594  26.804  22.581  1.00 42.50           C  \\nATOM    514  CG  MET A  67      -9.359  27.330  23.772  1.00 45.93           C  \\nATOM    515  SD  MET A  67     -11.140  27.396  23.483  1.00 48.74           S  \\nATOM    516  CE  MET A  67     -11.526  25.880  24.321  1.00 47.64           C  \\nATOM    517  N   ARG A  68      -7.128  24.418  20.665  1.00 31.59           N  \\nATOM    518  CA  ARG A  68      -6.322  24.217  19.467  1.00 26.34           C  \\nATOM    519  C   ARG A  68      -7.140  23.473  18.424  1.00 23.68           C  \\nATOM    520  O   ARG A  68      -6.846  23.617  17.231  1.00 21.93           O  \\nATOM    521  CB  ARG A  68      -5.057  23.394  19.745  1.00 25.86           C  \\nATOM    522  CG  ARG A  68      -4.161  23.898  20.869  1.00 29.36           C  \\nATOM    523  CD  ARG A  68      -2.881  23.061  20.998  1.00 32.88           C  \\nATOM    524  NE  ARG A  68      -2.221  22.990  22.315  1.00 36.64           N  \\nATOM    525  CZ  ARG A  68      -1.700  24.031  23.001  1.00 39.66           C  \\nATOM    526  NH1 ARG A  68      -1.903  25.315  22.664  1.00 42.06           N  \\nATOM    527  NH2 ARG A  68      -0.933  23.798  24.071  1.00 39.61           N  \\nATOM    528  N   ASP A  69      -8.188  22.721  18.793  1.00 19.14           N  \\nATOM    529  CA  ASP A  69      -8.880  21.905  17.822  1.00 20.43           C  \\nATOM    530  C   ASP A  69      -9.437  22.623  16.617  1.00 22.16           C  \\nATOM    531  O   ASP A  69      -9.453  22.109  15.488  1.00 19.58           O  \\nATOM    532  CB  ASP A  69     -10.025  21.148  18.478  1.00 20.58           C  \\nATOM    533  CG  ASP A  69      -9.598  19.935  19.303  1.00 22.81           C  \\nATOM    534  OD1 ASP A  69      -8.409  19.708  19.538  1.00 24.59           O  \\nATOM    535  OD2 ASP A  69     -10.475  19.176  19.715  1.00 25.28           O  \\nATOM    536  N   GLN A  70      -9.834  23.874  16.840  1.00 23.26           N  \\nATOM    537  CA  GLN A  70     -10.451  24.609  15.759  1.00 23.53           C  \\nATOM    538  C   GLN A  70      -9.437  24.925  14.667  1.00 21.20           C  \\nATOM    539  O   GLN A  70      -9.653  24.591  13.493  1.00 20.90           O  \\nATOM    540  CB  GLN A  70     -11.071  25.881  16.341  1.00 27.73           C  \\nATOM    541  CG  GLN A  70     -12.498  26.019  15.832  1.00 34.11           C  \\nATOM    542  CD  GLN A  70     -13.103  27.407  16.008  1.00 37.87           C  \\nATOM    543  OE1 GLN A  70     -12.532  28.284  16.672  1.00 39.77           O  \\nATOM    544  NE2 GLN A  70     -14.283  27.620  15.406  1.00 37.89           N  \\nATOM    545  N   TYR A  71      -8.292  25.515  15.050  1.00 19.09           N  \\nATOM    546  CA  TYR A  71      -7.329  25.873  14.030  1.00 17.69           C  \\nATOM    547  C   TYR A  71      -6.648  24.658  13.416  1.00 17.24           C  \\nATOM    548  O   TYR A  71      -6.247  24.675  12.245  1.00 17.25           O  \\nATOM    549  CB  TYR A  71      -6.302  26.868  14.625  1.00 17.87           C  \\nATOM    550  CG  TYR A  71      -5.311  26.480  15.725  1.00 18.90           C  \\nATOM    551  CD1 TYR A  71      -4.355  25.510  15.492  1.00 19.74           C  \\nATOM    552  CD2 TYR A  71      -5.328  27.162  16.940  1.00 19.21           C  \\nATOM    553  CE1 TYR A  71      -3.402  25.212  16.450  1.00 20.38           C  \\nATOM    554  CE2 TYR A  71      -4.375  26.868  17.901  1.00 21.93           C  \\nATOM    555  CZ  TYR A  71      -3.422  25.897  17.645  1.00 21.91           C  \\nATOM    556  OH  TYR A  71      -2.463  25.590  18.588  1.00 26.44           O  \\nATOM    557  N   MET A  72      -6.557  23.559  14.166  1.00 14.81           N  \\nATOM    558  CA  MET A  72      -5.958  22.393  13.563  1.00 18.59           C  \\nATOM    559  C   MET A  72      -6.849  21.778  12.472  1.00 19.38           C  \\nATOM    560  O   MET A  72      -6.385  21.198  11.492  1.00 17.67           O  \\nATOM    561  CB  MET A  72      -5.658  21.386  14.669  1.00 17.39           C  \\nATOM    562  CG  MET A  72      -4.479  21.796  15.514  1.00 16.35           C  \\nATOM    563  SD  MET A  72      -4.146  20.600  16.820  1.00 17.04           S  \\nATOM    564  CE  MET A  72      -2.635  21.259  17.460  1.00 17.72           C  \\nATOM    565  N   ARG A  73      -8.174  21.931  12.626  1.00 20.31           N  \\nATOM    566  CA  ARG A  73      -9.120  21.350  11.701  1.00 22.24           C  \\nATOM    567  C   ARG A  73      -9.087  22.084  10.370  1.00 20.94           C  \\nATOM    568  O   ARG A  73      -9.183  21.450   9.326  1.00 20.81           O  \\nATOM    569  CB  ARG A  73     -10.499  21.414  12.369  1.00 25.29           C  \\nATOM    570  CG  ARG A  73     -11.553  20.499  11.767  1.00 30.04           C  \\nATOM    571  CD  ARG A  73     -12.775  20.429  12.687  1.00 31.86           C  \\nATOM    572  NE  ARG A  73     -12.483  19.702  13.917  1.00 34.70           N  \\nATOM    573  CZ  ARG A  73     -12.894  20.122  15.130  1.00 36.49           C  \\nATOM    574  NH1 ARG A  73     -12.579  19.402  16.196  1.00 36.93           N  \\nATOM    575  NH2 ARG A  73     -13.609  21.239  15.311  1.00 38.17           N  \\nATOM    576  N   THR A  74      -8.868  23.401  10.413  1.00 21.33           N  \\nATOM    577  CA  THR A  74      -8.813  24.240   9.219  1.00 24.99           C  \\nATOM    578  C   THR A  74      -7.436  24.351   8.544  1.00 23.10           C  \\nATOM    579  O   THR A  74      -7.367  24.738   7.371  1.00 22.59           O  \\nATOM    580  CB  THR A  74      -9.289  25.676   9.538  1.00 25.73           C  \\nATOM    581  OG1 THR A  74      -8.351  26.302  10.409  1.00 31.08           O  \\nATOM    582  CG2 THR A  74     -10.657  25.644  10.201  1.00 27.58           C  \\nATOM    583  N   GLY A  75      -6.345  24.084   9.289  1.00 19.58           N  \\nATOM    584  CA  GLY A  75      -5.013  24.114   8.730  1.00 15.79           C  \\nATOM    585  C   GLY A  75      -4.823  23.023   7.713  1.00 13.66           C  \\nATOM    586  O   GLY A  75      -5.245  21.880   7.884  1.00 14.69           O  \\nATOM    587  N   GLU A  76      -4.228  23.360   6.573  1.00 13.14           N  \\nATOM    588  CA  GLU A  76      -3.914  22.363   5.546  1.00 12.99           C  \\nATOM    589  C   GLU A  76      -2.514  21.740   5.753  1.00 12.22           C  \\nATOM    590  O   GLU A  76      -2.293  20.576   5.426  1.00 13.59           O  \\nATOM    591  CB  GLU A  76      -3.999  23.014   4.145  1.00 13.07           C  \\nATOM    592  CG  GLU A  76      -5.387  23.377   3.676  1.00 14.35           C  \\nATOM    593  CD  GLU A  76      -5.261  24.069   2.323  1.00 20.40           C  \\nATOM    594  OE1 GLU A  76      -5.461  23.403   1.288  1.00 24.32           O  \\nATOM    595  OE2 GLU A  76      -4.913  25.250   2.284  1.00 18.82           O  \\nATOM    596  N   GLY A  77      -1.553  22.503   6.301  1.00 10.73           N  \\nATOM    597  CA  GLY A  77      -0.216  22.018   6.570  1.00  8.07           C  \\nATOM    598  C   GLY A  77       0.302  22.689   7.819  1.00  8.63           C  \\nATOM    599  O   GLY A  77      -0.066  23.823   8.127  1.00  8.55           O  \\nATOM    600  N   PHE A  78       1.248  22.013   8.479  1.00  9.15           N  \\nATOM    601  CA  PHE A  78       1.754  22.468   9.773  1.00  8.87           C  \\nATOM    602  C   PHE A  78       3.268  22.520   9.854  1.00  9.51           C  \\nATOM    603  O   PHE A  78       3.930  21.542   9.486  1.00  9.15           O  \\nATOM    604  CB  PHE A  78       1.247  21.532  10.904  1.00  9.44           C  \\nATOM    605  CG  PHE A  78      -0.267  21.472  11.000  1.00  9.66           C  \\nATOM    606  CD1 PHE A  78      -0.906  22.310  11.901  1.00 10.71           C  \\nATOM    607  CD2 PHE A  78      -0.994  20.645  10.152  1.00 10.20           C  \\nATOM    608  CE1 PHE A  78      -2.295  22.338  11.940  1.00 10.61           C  \\nATOM    609  CE2 PHE A  78      -2.378  20.678  10.192  1.00 12.55           C  \\nATOM    610  CZ  PHE A  78      -3.022  21.529  11.081  1.00 12.80           C  \\nATOM    611  N   LEU A  79       3.813  23.636  10.354  1.00  8.72           N  \\nATOM    612  CA  LEU A  79       5.230  23.626  10.699  1.00  9.93           C  \\nATOM    613  C   LEU A  79       5.279  23.281  12.181  1.00  9.09           C  \\nATOM    614  O   LEU A  79       4.701  24.008  12.988  1.00 11.58           O  \\nATOM    615  CB  LEU A  79       5.921  24.970  10.567  1.00 12.80           C  \\nATOM    616  CG  LEU A  79       6.155  25.659   9.249  1.00 16.39           C  \\nATOM    617  CD1 LEU A  79       7.007  26.911   9.544  1.00 17.02           C  \\nATOM    618  CD2 LEU A  79       6.904  24.756   8.269  1.00 16.98           C  \\nATOM    619  N   CYS A  80       5.950  22.219  12.579  1.00  9.14           N  \\nATOM    620  CA  CYS A  80       6.074  21.821  13.981  1.00 11.81           C  \\nATOM    621  C   CYS A  80       7.448  22.237  14.434  1.00 11.53           C  \\nATOM    622  O   CYS A  80       8.435  21.640  14.000  1.00 12.21           O  \\nATOM    623  CB  CYS A  80       5.903  20.318  14.106  1.00 13.73           C  \\nATOM    624  SG  CYS A  80       4.158  19.900  13.837  1.00 18.26           S  \\nATOM    625  N   VAL A  81       7.486  23.282  15.269  1.00  7.57           N  \\nATOM    626  CA  VAL A  81       8.730  23.956  15.604  1.00  8.45           C  \\nATOM    627  C   VAL A  81       9.248  23.620  16.996  1.00  8.90           C  \\nATOM    628  O   VAL A  81       8.451  23.680  17.939  1.00  8.49           O  \\nATOM    629  CB  VAL A  81       8.544  25.516  15.507  1.00  7.40           C  \\nATOM    630  CG1 VAL A  81       9.917  26.196  15.588  1.00  9.42           C  \\nATOM    631  CG2 VAL A  81       8.004  25.954  14.167  1.00  8.59           C  \\nATOM    632  N   PHE A  82      10.546  23.237  17.119  1.00  9.49           N  \\nATOM    633  CA  PHE A  82      11.195  23.202  18.442  1.00  9.09           C  \\nATOM    634  C   PHE A  82      12.511  23.994  18.350  1.00  9.83           C  \\nATOM    635  O   PHE A  82      12.914  24.440  17.266  1.00  9.02           O  \\nATOM    636  CB  PHE A  82      11.482  21.755  18.902  1.00  9.76           C  \\nATOM    637  CG  PHE A  82      12.536  21.062  18.071  1.00 10.09           C  \\nATOM    638  CD1 PHE A  82      13.867  21.099  18.486  1.00 10.70           C  \\nATOM    639  CD2 PHE A  82      12.192  20.412  16.889  1.00 10.93           C  \\nATOM    640  CE1 PHE A  82      14.838  20.483  17.699  1.00 11.12           C  \\nATOM    641  CE2 PHE A  82      13.181  19.807  16.125  1.00 10.73           C  \\nATOM    642  CZ  PHE A  82      14.505  19.839  16.523  1.00 10.35           C  \\nATOM    643  N   ALA A  83      13.128  24.269  19.516  1.00  9.75           N  \\nATOM    644  CA  ALA A  83      14.407  24.963  19.569  1.00 10.32           C  \\nATOM    645  C   ALA A  83      15.509  23.960  19.916  1.00  9.48           C  \\nATOM    646  O   ALA A  83      15.362  23.142  20.839  1.00  9.61           O  \\nATOM    647  CB  ALA A  83      14.354  26.083  20.620  1.00  9.75           C  \\nATOM    648  N   ILE A  84      16.624  24.042  19.160  1.00 11.41           N  \\nATOM    649  CA  ILE A  84      17.714  23.070  19.299  1.00 13.15           C  \\nATOM    650  C   ILE A  84      18.446  23.223  20.623  1.00 14.57           C  \\nATOM    651  O   ILE A  84      19.234  22.347  20.971  1.00 13.94           O  \\nATOM    652  CB  ILE A  84      18.775  23.151  18.160  1.00 13.29           C  \\nATOM    653  CG1 ILE A  84      19.403  24.509  17.878  1.00 14.90           C  \\nATOM    654  CG2 ILE A  84      18.018  22.701  16.933  1.00 13.09           C  \\nATOM    655  CD1 ILE A  84      20.578  24.957  18.763  1.00 17.59           C  \\nATOM    656  N   ASN A  85      18.200  24.290  21.390  1.00 13.61           N  \\nATOM    657  CA  ASN A  85      18.794  24.394  22.720  1.00 15.52           C  \\nATOM    658  C   ASN A  85      17.758  24.156  23.828  1.00 15.49           C  \\nATOM    659  O   ASN A  85      17.988  24.519  24.999  1.00 16.13           O  \\nATOM    660  CB  ASN A  85      19.446  25.788  22.885  1.00 15.16           C  \\nATOM    661  CG  ASN A  85      18.464  26.946  22.904  1.00 16.42           C  \\nATOM    662  OD1 ASN A  85      17.355  26.878  22.362  1.00 16.01           O  \\nATOM    663  ND2 ASN A  85      18.808  28.030  23.587  1.00 16.83           N  \\nATOM    664  N   ASN A  86      16.616  23.515  23.509  1.00 14.72           N  \\nATOM    665  CA  ASN A  86      15.553  23.303  24.483  1.00 13.01           C  \\nATOM    666  C   ASN A  86      14.987  21.901  24.309  1.00 13.17           C  \\nATOM    667  O   ASN A  86      14.080  21.668  23.520  1.00 13.04           O  \\nATOM    668  CB  ASN A  86      14.478  24.367  24.256  1.00 16.51           C  \\nATOM    669  CG  ASN A  86      13.267  24.415  25.187  1.00 20.74           C  \\nATOM    670  OD1 ASN A  86      12.370  25.241  25.018  1.00 25.41           O  \\nATOM    671  ND2 ASN A  86      13.123  23.596  26.206  1.00 19.96           N  \\nATOM    672  N   THR A  87      15.488  20.950  25.103  1.00 13.80           N  \\nATOM    673  CA  THR A  87      15.095  19.553  25.026  1.00 15.40           C  \\nATOM    674  C   THR A  87      13.589  19.316  25.266  1.00 14.40           C  \\nATOM    675  O   THR A  87      12.953  18.535  24.546  1.00 14.11           O  \\nATOM    676  CB  THR A  87      15.983  18.791  26.046  1.00 16.19           C  \\nATOM    677  OG1 THR A  87      17.325  18.908  25.571  1.00 18.22           O  \\nATOM    678  CG2 THR A  87      15.626  17.327  26.180  1.00 17.82           C  \\nATOM    679  N   LYS A  88      12.983  19.958  26.272  1.00 14.42           N  \\nATOM    680  CA  LYS A  88      11.552  19.842  26.535  1.00 15.84           C  \\nATOM    681  C   LYS A  88      10.726  20.232  25.278  1.00 15.06           C  \\nATOM    682  O   LYS A  88       9.722  19.582  25.003  1.00 14.61           O  \\nATOM    683  CB  LYS A  88      11.207  20.766  27.717  1.00 19.53           C  \\nATOM    684  CG  LYS A  88       9.937  20.401  28.457  1.00 27.71           C  \\nATOM    685  CD  LYS A  88       9.585  21.452  29.529  1.00 33.26           C  \\nATOM    686  CE  LYS A  88       9.061  22.762  28.877  1.00 36.79           C  \\nATOM    687  NZ  LYS A  88       8.642  23.763  29.852  1.00 37.29           N  \\nATOM    688  N   SER A  89      11.119  21.259  24.475  1.00 12.36           N  \\nATOM    689  CA  SER A  89      10.325  21.666  23.309  1.00 11.31           C  \\nATOM    690  C   SER A  89      10.371  20.572  22.268  1.00 12.51           C  \\nATOM    691  O   SER A  89       9.397  20.388  21.547  1.00 12.88           O  \\nATOM    692  CB  SER A  89      10.844  22.951  22.674  1.00 11.64           C  \\nATOM    693  OG  SER A  89      12.160  22.900  22.124  1.00 11.11           O  \\nATOM    694  N   PHE A  90      11.504  19.842  22.204  1.00 11.27           N  \\nATOM    695  CA  PHE A  90      11.680  18.751  21.262  1.00 12.91           C  \\nATOM    696  C   PHE A  90      10.760  17.601  21.667  1.00 14.37           C  \\nATOM    697  O   PHE A  90      10.034  17.057  20.839  1.00 13.24           O  \\nATOM    698  CB  PHE A  90      13.161  18.312  21.260  1.00 12.34           C  \\nATOM    699  CG  PHE A  90      13.411  17.120  20.351  1.00 14.41           C  \\nATOM    700  CD1 PHE A  90      13.890  15.948  20.903  1.00 15.10           C  \\nATOM    701  CD2 PHE A  90      13.136  17.206  18.990  1.00 14.81           C  \\nATOM    702  CE1 PHE A  90      14.093  14.857  20.088  1.00 16.20           C  \\nATOM    703  CE2 PHE A  90      13.340  16.110  18.179  1.00 14.81           C  \\nATOM    704  CZ  PHE A  90      13.818  14.935  18.731  1.00 15.39           C  \\nATOM    705  N   GLU A  91      10.725  17.231  22.949  1.00 14.40           N  \\nATOM    706  CA  GLU A  91       9.863  16.146  23.407  1.00 15.78           C  \\nATOM    707  C   GLU A  91       8.374  16.466  23.301  1.00 16.18           C  \\nATOM    708  O   GLU A  91       7.533  15.582  23.114  1.00 17.50           O  \\nATOM    709  CB  GLU A  91      10.226  15.846  24.835  1.00 19.84           C  \\nATOM    710  CG  GLU A  91      11.654  15.386  25.074  1.00 24.49           C  \\nATOM    711  CD  GLU A  91      12.090  15.310  26.537  1.00 27.97           C  \\nATOM    712  OE1 GLU A  91      13.199  14.842  26.766  1.00 30.72           O  \\nATOM    713  OE2 GLU A  91      11.371  15.730  27.449  1.00 30.38           O  \\nATOM    714  N   ASP A  92       8.019  17.747  23.433  1.00 16.19           N  \\nATOM    715  CA  ASP A  92       6.650  18.216  23.265  1.00 15.23           C  \\nATOM    716  C   ASP A  92       6.141  18.053  21.839  1.00 14.29           C  \\nATOM    717  O   ASP A  92       4.927  18.074  21.633  1.00 14.78           O  \\nATOM    718  CB  ASP A  92       6.536  19.708  23.646  1.00 17.85           C  \\nATOM    719  CG  ASP A  92       6.706  20.080  25.127  1.00 21.39           C  \\nATOM    720  OD1 ASP A  92       6.648  19.209  25.997  1.00 20.06           O  \\nATOM    721  OD2 ASP A  92       6.887  21.272  25.401  1.00 24.39           O  \\nATOM    722  N   ILE A  93       7.013  17.910  20.826  1.00 11.76           N  \\nATOM    723  CA  ILE A  93       6.559  17.749  19.452  1.00 14.77           C  \\nATOM    724  C   ILE A  93       5.574  16.583  19.354  1.00 14.03           C  \\nATOM    725  O   ILE A  93       4.576  16.695  18.644  1.00 14.93           O  \\nATOM    726  CB  ILE A  93       7.790  17.499  18.485  1.00 14.96           C  \\nATOM    727  CG1 ILE A  93       8.506  18.830  18.293  1.00 15.41           C  \\nATOM    728  CG2 ILE A  93       7.369  16.936  17.091  1.00 14.61           C  \\nATOM    729  CD1 ILE A  93       7.654  19.940  17.637  1.00 20.05           C  \\nATOM    730  N   HIS A  94       5.806  15.471  20.049  1.00 15.23           N  \\nATOM    731  CA  HIS A  94       4.940  14.302  19.956  1.00 18.20           C  \\nATOM    732  C   HIS A  94       3.494  14.635  20.287  1.00 16.36           C  \\nATOM    733  O   HIS A  94       2.576  14.183  19.598  1.00 17.20           O  \\nATOM    734  CB  HIS A  94       5.436  13.194  20.900  1.00 23.57           C  \\nATOM    735  CG  HIS A  94       4.685  11.874  20.678  1.00 29.09           C  \\nATOM    736  ND1 HIS A  94       4.923  11.118  19.602  1.00 31.26           N  \\nATOM    737  CD2 HIS A  94       3.705  11.361  21.498  1.00 30.93           C  \\nATOM    738  CE1 HIS A  94       4.078  10.110  19.746  1.00 31.74           C  \\nATOM    739  NE2 HIS A  94       3.366  10.260  20.861  1.00 32.20           N  \\nATOM    740  N   GLN A  95       3.271  15.499  21.281  1.00 16.91           N  \\nATOM    741  CA  GLN A  95       1.916  15.878  21.673  1.00 18.66           C  \\nATOM    742  C   GLN A  95       1.226  16.736  20.625  1.00 16.97           C  \\nATOM    743  O   GLN A  95       0.012  16.562  20.443  1.00 15.36           O  \\nATOM    744  CB  GLN A  95       1.928  16.626  23.018  1.00 21.56           C  \\nATOM    745  CG  GLN A  95       2.457  15.766  24.180  1.00 29.81           C  \\nATOM    746  CD  GLN A  95       3.370  16.540  25.150  1.00 35.29           C  \\nATOM    747  OE1 GLN A  95       3.037  17.647  25.593  1.00 37.94           O  \\nATOM    748  NE2 GLN A  95       4.546  16.006  25.521  1.00 37.04           N  \\nATOM    749  N   TYR A  96       1.930  17.653  19.921  1.00 16.06           N  \\nATOM    750  CA  TYR A  96       1.276  18.417  18.826  1.00 16.07           C  \\nATOM    751  C   TYR A  96       0.963  17.504  17.658  1.00 14.44           C  \\nATOM    752  O   TYR A  96      -0.138  17.565  17.118  1.00 15.79           O  \\nATOM    753  CB  TYR A  96       2.139  19.554  18.264  1.00 17.94           C  \\nATOM    754  CG  TYR A  96       2.287  20.677  19.277  1.00 19.19           C  \\nATOM    755  CD1 TYR A  96       1.368  21.699  19.338  1.00 19.33           C  \\nATOM    756  CD2 TYR A  96       3.348  20.631  20.161  1.00 19.61           C  \\nATOM    757  CE1 TYR A  96       1.526  22.678  20.308  1.00 21.72           C  \\nATOM    758  CE2 TYR A  96       3.512  21.595  21.129  1.00 21.72           C  \\nATOM    759  CZ  TYR A  96       2.594  22.613  21.190  1.00 21.26           C  \\nATOM    760  OH  TYR A  96       2.776  23.582  22.153  1.00 24.09           O  \\nATOM    761  N   ARG A  97       1.904  16.635  17.295  1.00 13.30           N  \\nATOM    762  CA  ARG A  97       1.699  15.650  16.247  1.00 15.46           C  \\nATOM    763  C   ARG A  97       0.457  14.794  16.492  1.00 14.26           C  \\nATOM    764  O   ARG A  97      -0.420  14.676  15.641  1.00 14.10           O  \\nATOM    765  CB  ARG A  97       2.939  14.796  16.196  1.00 17.97           C  \\nATOM    766  CG  ARG A  97       2.886  13.541  15.357  1.00 23.35           C  \\nATOM    767  CD  ARG A  97       3.333  13.765  13.943  1.00 23.59           C  \\nATOM    768  NE  ARG A  97       3.314  12.471  13.314  1.00 24.08           N  \\nATOM    769  CZ  ARG A  97       3.268  12.320  12.007  1.00 25.74           C  \\nATOM    770  NH1 ARG A  97       3.135  11.073  11.548  1.00 26.76           N  \\nATOM    771  NH2 ARG A  97       3.353  13.380  11.181  1.00 26.95           N  \\nATOM    772  N   GLU A  98       0.332  14.218  17.678  1.00 13.69           N  \\nATOM    773  CA  GLU A  98      -0.807  13.361  17.941  1.00 16.64           C  \\nATOM    774  C   GLU A  98      -2.118  14.108  17.983  1.00 15.57           C  \\nATOM    775  O   GLU A  98      -3.157  13.573  17.607  1.00 15.85           O  \\nATOM    776  CB  GLU A  98      -0.593  12.623  19.260  1.00 21.96           C  \\nATOM    777  CG  GLU A  98       0.417  11.455  19.177  1.00 28.02           C  \\nATOM    778  CD  GLU A  98       0.147  10.460  18.038  1.00 31.04           C  \\nATOM    779  OE1 GLU A  98       1.118  10.123  17.373  1.00 33.58           O  \\nATOM    780  OE2 GLU A  98      -0.992  10.025  17.790  1.00 33.61           O  \\nATOM    781  N   GLN A  99      -2.105  15.362  18.446  1.00 14.66           N  \\nATOM    782  CA  GLN A  99      -3.343  16.104  18.468  1.00 13.22           C  \\nATOM    783  C   GLN A  99      -3.779  16.398  17.049  1.00 12.15           C  \\nATOM    784  O   GLN A  99      -4.974  16.274  16.791  1.00 14.09           O  \\nATOM    785  CB  GLN A  99      -3.158  17.403  19.250  1.00 16.29           C  \\nATOM    786  CG  GLN A  99      -4.513  18.118  19.407  1.00 19.67           C  \\nATOM    787  CD  GLN A  99      -4.517  19.240  20.438  1.00 22.19           C  \\nATOM    788  OE1 GLN A  99      -3.537  19.485  21.138  1.00 23.70           O  \\nATOM    789  NE2 GLN A  99      -5.628  19.961  20.537  1.00 22.72           N  \\nATOM    790  N   ILE A 100      -2.861  16.795  16.142  1.00 11.42           N  \\nATOM    791  CA  ILE A 100      -3.197  17.121  14.743  1.00 10.81           C  \\nATOM    792  C   ILE A 100      -3.761  15.893  14.008  1.00 12.39           C  \\nATOM    793  O   ILE A 100      -4.733  15.967  13.243  1.00 12.70           O  \\nATOM    794  CB  ILE A 100      -1.924  17.659  14.038  1.00 10.32           C  \\nATOM    795  CG1 ILE A 100      -1.642  19.075  14.518  1.00 10.37           C  \\nATOM    796  CG2 ILE A 100      -2.131  17.685  12.512  1.00 11.36           C  \\nATOM    797  CD1 ILE A 100      -0.184  19.481  14.392  1.00 10.69           C  \\nATOM    798  N   LYS A 101      -3.143  14.732  14.255  1.00 14.08           N  \\nATOM    799  CA  LYS A 101      -3.613  13.487  13.637  1.00 15.95           C  \\nATOM    800  C   LYS A 101      -5.021  13.150  14.079  1.00 15.72           C  \\nATOM    801  O   LYS A 101      -5.848  12.770  13.262  1.00 14.25           O  \\nATOM    802  CB  LYS A 101      -2.681  12.317  13.996  1.00 17.44           C  \\nATOM    803  CG  LYS A 101      -1.414  12.284  13.115  1.00 19.24           C  \\nATOM    804  CD  LYS A 101      -0.332  11.418  13.764  1.00 23.28           C  \\nATOM    805  CE  LYS A 101      -0.752  10.000  14.188  1.00 23.82           C  \\nATOM    806  NZ  LYS A 101       0.400   9.230  14.662  1.00 24.00           N  \\nATOM    807  N   ARG A 102      -5.278  13.341  15.377  1.00 16.72           N  \\nATOM    808  CA  ARG A 102      -6.586  13.098  15.962  1.00 18.74           C  \\nATOM    809  C   ARG A 102      -7.643  14.087  15.440  1.00 19.07           C  \\nATOM    810  O   ARG A 102      -8.712  13.628  15.002  1.00 20.67           O  \\nATOM    811  CB  ARG A 102      -6.457  13.197  17.462  1.00 20.62           C  \\nATOM    812  CG  ARG A 102      -7.539  12.409  18.132  1.00 26.89           C  \\nATOM    813  CD  ARG A 102      -7.610  12.887  19.585  1.00 30.60           C  \\nATOM    814  NE  ARG A 102      -8.197  14.224  19.645  1.00 35.83           N  \\nATOM    815  CZ  ARG A 102      -7.633  15.255  20.301  1.00 37.61           C  \\nATOM    816  NH1 ARG A 102      -8.243  16.455  20.267  1.00 38.37           N  \\nATOM    817  NH2 ARG A 102      -6.481  15.110  20.983  1.00 38.39           N  \\nATOM    818  N   VAL A 103      -7.418  15.407  15.395  1.00 15.59           N  \\nATOM    819  CA  VAL A 103      -8.452  16.326  14.881  1.00 18.48           C  \\nATOM    820  C   VAL A 103      -8.665  16.200  13.349  1.00 16.72           C  \\nATOM    821  O   VAL A 103      -9.776  16.383  12.848  1.00 15.90           O  \\nATOM    822  CB  VAL A 103      -8.131  17.840  15.215  1.00 20.10           C  \\nATOM    823  CG1 VAL A 103      -7.896  18.005  16.704  1.00 21.34           C  \\nATOM    824  CG2 VAL A 103      -6.899  18.302  14.512  1.00 23.93           C  \\nATOM    825  N   LYS A 104      -7.622  15.902  12.574  1.00 13.84           N  \\nATOM    826  CA  LYS A 104      -7.804  15.755  11.140  1.00 16.11           C  \\nATOM    827  C   LYS A 104      -8.158  14.329  10.716  1.00 16.70           C  \\nATOM    828  O   LYS A 104      -8.260  14.079   9.508  1.00 17.27           O  \\nATOM    829  CB  LYS A 104      -6.525  16.215  10.404  1.00 18.12           C  \\nATOM    830  CG  LYS A 104      -6.197  17.678  10.744  1.00 19.53           C  \\nATOM    831  CD  LYS A 104      -5.250  18.297   9.736  1.00 21.28           C  \\nATOM    832  CE  LYS A 104      -5.921  18.590   8.390  1.00 20.75           C  \\nATOM    833  NZ  LYS A 104      -6.875  19.673   8.462  1.00 19.79           N  \\nATOM    834  N   ASP A 105      -8.372  13.390  11.669  1.00 15.91           N  \\nATOM    835  CA  ASP A 105      -8.666  11.988  11.359  1.00 15.43           C  \\nATOM    836  C   ASP A 105      -7.705  11.372  10.333  1.00 16.04           C  \\nATOM    837  O   ASP A 105      -8.106  10.654   9.415  1.00 15.38           O  \\nATOM    838  CB  ASP A 105     -10.077  11.847  10.810  1.00 18.74           C  \\nATOM    839  CG  ASP A 105     -11.261  12.121  11.696  1.00 21.15           C  \\nATOM    840  OD1 ASP A 105     -11.183  11.972  12.918  1.00 21.36           O  \\nATOM    841  OD2 ASP A 105     -12.281  12.480  11.113  1.00 22.63           O  \\nATOM    842  N   SER A 106      -6.397  11.591  10.469  1.00 15.54           N  \\nATOM    843  CA  SER A 106      -5.470  11.174   9.442  1.00 17.25           C  \\nATOM    844  C   SER A 106      -4.112  10.882  10.011  1.00 17.31           C  \\nATOM    845  O   SER A 106      -3.650  11.556  10.916  1.00 19.11           O  \\nATOM    846  CB  SER A 106      -5.463  12.314   8.398  1.00 19.27           C  \\nATOM    847  OG  SER A 106      -4.326  12.490   7.573  1.00 24.20           O  \\nATOM    848  N   ASP A 107      -3.454   9.891   9.453  1.00 17.29           N  \\nATOM    849  CA  ASP A 107      -2.109   9.557   9.868  1.00 19.80           C  \\nATOM    850  C   ASP A 107      -1.000  10.220   9.063  1.00 20.36           C  \\nATOM    851  O   ASP A 107       0.197  10.118   9.345  1.00 21.26           O  \\nATOM    852  CB  ASP A 107      -1.970   8.045   9.810  1.00 23.15           C  \\nATOM    853  CG  ASP A 107      -2.863   7.310  10.807  1.00 25.12           C  \\nATOM    854  OD1 ASP A 107      -3.262   6.206  10.462  1.00 29.21           O  \\nATOM    855  OD2 ASP A 107      -3.127   7.812  11.903  1.00 26.59           O  \\nATOM    856  N   ASP A 108      -1.383  10.909   8.005  1.00 20.57           N  \\nATOM    857  CA  ASP A 108      -0.430  11.664   7.216  1.00 22.38           C  \\nATOM    858  C   ASP A 108      -1.061  12.982   6.842  1.00 21.37           C  \\nATOM    859  O   ASP A 108      -1.973  13.096   6.030  1.00 24.36           O  \\nATOM    860  CB  ASP A 108      -0.056  10.900   5.967  1.00 27.28           C  \\nATOM    861  CG  ASP A 108      -1.263  10.278   5.320  1.00 32.24           C  \\nATOM    862  OD1 ASP A 108      -1.714   9.272   5.877  1.00 37.07           O  \\nATOM    863  OD2 ASP A 108      -1.747  10.790   4.301  1.00 36.33           O  \\nATOM    864  N   VAL A 109      -0.598  13.970   7.572  1.00 19.33           N  \\nATOM    865  CA  VAL A 109      -1.009  15.354   7.400  1.00 17.49           C  \\nATOM    866  C   VAL A 109       0.215  16.086   6.819  1.00 14.55           C  \\nATOM    867  O   VAL A 109       1.335  15.772   7.245  1.00 13.64           O  \\nATOM    868  CB  VAL A 109      -1.410  15.870   8.784  1.00 18.40           C  \\nATOM    869  CG1 VAL A 109      -1.645  17.366   8.765  1.00 20.21           C  \\nATOM    870  CG2 VAL A 109      -2.676  15.101   9.214  1.00 20.27           C  \\nATOM    871  N   PRO A 110       0.141  17.001   5.825  1.00 12.41           N  \\nATOM    872  CA  PRO A 110       1.297  17.783   5.384  1.00 11.52           C  \\nATOM    873  C   PRO A 110       1.920  18.480   6.605  1.00 11.79           C  \\nATOM    874  O   PRO A 110       1.289  19.245   7.324  1.00  9.55           O  \\nATOM    875  CB  PRO A 110       0.728  18.721   4.370  1.00 11.31           C  \\nATOM    876  CG  PRO A 110      -0.410  17.926   3.779  1.00 11.38           C  \\nATOM    877  CD  PRO A 110      -1.044  17.309   5.020  1.00 12.05           C  \\nATOM    878  N   MET A 111       3.192  18.201   6.874  1.00 11.92           N  \\nATOM    879  CA  MET A 111       3.878  18.724   8.050  1.00 11.49           C  \\nATOM    880  C   MET A 111       5.379  18.749   7.807  1.00 11.68           C  \\nATOM    881  O   MET A 111       5.903  17.900   7.098  1.00 11.11           O  \\nATOM    882  CB  MET A 111       3.494  17.806   9.211  1.00 14.36           C  \\nATOM    883  CG  MET A 111       3.864  18.110  10.627  1.00 17.41           C  \\nATOM    884  SD  MET A 111       3.231  16.825  11.758  1.00 18.36           S  \\nATOM    885  CE  MET A 111       1.528  17.128  11.498  1.00 17.25           C  \\nATOM    886  N   VAL A 112       6.091  19.706   8.404  1.00 10.73           N  \\nATOM    887  CA  VAL A 112       7.545  19.763   8.348  1.00 10.18           C  \\nATOM    888  C   VAL A 112       7.992  19.875   9.794  1.00  9.11           C  \\nATOM    889  O   VAL A 112       7.404  20.633  10.556  1.00 10.77           O  \\nATOM    890  CB  VAL A 112       8.010  21.024   7.542  1.00 11.80           C  \\nATOM    891  CG1 VAL A 112       9.528  21.213   7.558  1.00 13.24           C  \\nATOM    892  CG2 VAL A 112       7.542  20.843   6.104  1.00 14.56           C  \\nATOM    893  N   LEU A 113       9.032  19.167  10.207  1.00  8.47           N  \\nATOM    894  CA  LEU A 113       9.632  19.332  11.535  1.00  9.53           C  \\nATOM    895  C   LEU A 113      10.738  20.384  11.412  1.00  9.33           C  \\nATOM    896  O   LEU A 113      11.621  20.270  10.555  1.00  9.95           O  \\nATOM    897  CB  LEU A 113      10.214  17.983  12.017  1.00  9.56           C  \\nATOM    898  CG  LEU A 113      10.863  17.976  13.381  1.00 11.58           C  \\nATOM    899  CD1 LEU A 113       9.908  18.379  14.503  1.00 13.17           C  \\nATOM    900  CD2 LEU A 113      11.361  16.564  13.597  1.00 12.84           C  \\nATOM    901  N   VAL A 114      10.705  21.442  12.221  1.00  7.62           N  \\nATOM    902  CA  VAL A 114      11.673  22.541  12.109  1.00  7.64           C  \\nATOM    903  C   VAL A 114      12.436  22.667  13.441  1.00  7.65           C  \\nATOM    904  O   VAL A 114      11.872  22.842  14.541  1.00  9.71           O  \\nATOM    905  CB  VAL A 114      10.896  23.836  11.774  1.00  9.37           C  \\nATOM    906  CG1 VAL A 114      11.814  25.030  11.869  1.00 10.27           C  \\nATOM    907  CG2 VAL A 114      10.254  23.719  10.393  1.00  9.64           C  \\nATOM    908  N   GLY A 115      13.745  22.567  13.331  1.00  6.10           N  \\nATOM    909  CA  GLY A 115      14.603  22.742  14.476  1.00  6.69           C  \\nATOM    910  C   GLY A 115      15.120  24.163  14.407  1.00  8.47           C  \\nATOM    911  O   GLY A 115      16.007  24.439  13.594  1.00  9.16           O  \\nATOM    912  N   ASN A 116      14.593  25.089  15.192  1.00  8.01           N  \\nATOM    913  CA  ASN A 116      15.007  26.475  15.111  1.00  8.24           C  \\nATOM    914  C   ASN A 116      16.112  26.873  16.109  1.00  9.61           C  \\nATOM    915  O   ASN A 116      16.453  26.142  17.051  1.00  9.84           O  \\nATOM    916  CB  ASN A 116      13.721  27.307  15.275  1.00  8.01           C  \\nATOM    917  CG  ASN A 116      13.929  28.788  14.992  1.00  9.48           C  \\nATOM    918  OD1 ASN A 116      14.470  29.205  13.972  1.00  9.94           O  \\nATOM    919  ND2 ASN A 116      13.565  29.644  15.940  1.00 10.75           N  \\nATOM    920  N   LYS A 117      16.660  28.087  15.876  1.00 10.22           N  \\nATOM    921  CA  LYS A 117      17.735  28.715  16.674  1.00 12.36           C  \\nATOM    922  C   LYS A 117      19.094  28.021  16.450  1.00 13.39           C  \\nATOM    923  O   LYS A 117      19.977  27.940  17.324  1.00 11.29           O  \\nATOM    924  CB  LYS A 117      17.391  28.706  18.214  1.00 13.23           C  \\nATOM    925  CG  LYS A 117      16.026  29.272  18.611  1.00 12.90           C  \\nATOM    926  CD  LYS A 117      15.932  29.640  20.097  1.00 12.43           C  \\nATOM    927  CE  LYS A 117      14.551  30.217  20.380  1.00 10.55           C  \\nATOM    928  NZ  LYS A 117      14.491  30.647  21.772  1.00 11.10           N  \\nATOM    929  N   CYS A 118      19.254  27.555  15.188  1.00 13.69           N  \\nATOM    930  CA  CYS A 118      20.449  26.806  14.844  1.00 15.67           C  \\nATOM    931  C   CYS A 118      21.716  27.658  14.796  1.00 15.28           C  \\nATOM    932  O   CYS A 118      22.808  27.113  14.629  1.00 15.30           O  \\nATOM    933  CB  CYS A 118      20.246  26.064  13.492  1.00 17.74           C  \\nATOM    934  SG  CYS A 118      20.353  27.009  11.944  1.00 20.74           S  \\nATOM    935  N   ASP A 119      21.578  28.995  14.931  1.00 15.76           N  \\nATOM    936  CA  ASP A 119      22.722  29.904  15.083  1.00 16.09           C  \\nATOM    937  C   ASP A 119      23.426  29.778  16.445  1.00 17.19           C  \\nATOM    938  O   ASP A 119      24.589  30.178  16.553  1.00 18.76           O  \\nATOM    939  CB  ASP A 119      22.268  31.374  14.888  1.00 15.14           C  \\nATOM    940  CG  ASP A 119      21.153  31.785  15.835  1.00 15.81           C  \\nATOM    941  OD1 ASP A 119      20.030  31.294  15.662  1.00 13.87           O  \\nATOM    942  OD2 ASP A 119      21.407  32.596  16.721  1.00 14.85           O  \\nATOM    943  N   LEU A 120      22.732  29.248  17.480  1.00 16.86           N  \\nATOM    944  CA  LEU A 120      23.230  29.099  18.845  1.00 18.76           C  \\nATOM    945  C   LEU A 120      24.217  27.957  18.979  1.00 19.70           C  \\nATOM    946  O   LEU A 120      23.979  26.885  18.431  1.00 20.53           O  \\nATOM    947  CB  LEU A 120      22.103  28.829  19.831  1.00 17.65           C  \\nATOM    948  CG  LEU A 120      21.065  29.933  20.010  1.00 19.43           C  \\nATOM    949  CD1 LEU A 120      20.044  29.418  21.008  1.00 18.46           C  \\nATOM    950  CD2 LEU A 120      21.688  31.239  20.503  1.00 18.73           C  \\nATOM    951  N   ALA A 121      25.313  28.184  19.705  1.00 19.26           N  \\nATOM    952  CA  ALA A 121      26.303  27.143  19.887  1.00 20.48           C  \\nATOM    953  C   ALA A 121      25.942  26.210  21.036  1.00 21.30           C  \\nATOM    954  O   ALA A 121      26.403  25.069  21.067  1.00 24.04           O  \\nATOM    955  CB  ALA A 121      27.652  27.767  20.164  1.00 22.02           C  \\nATOM    956  N   ALA A 122      25.109  26.635  21.982  1.00 22.09           N  \\nATOM    957  CA  ALA A 122      24.683  25.797  23.094  1.00 23.88           C  \\nATOM    958  C   ALA A 122      23.578  24.811  22.720  1.00 23.77           C  \\nATOM    959  O   ALA A 122      22.492  24.835  23.301  1.00 27.82           O  \\nATOM    960  CB  ALA A 122      24.171  26.684  24.249  1.00 25.67           C  \\nATOM    961  N   ARG A 123      23.843  23.908  21.783  1.00 20.99           N  \\nATOM    962  CA  ARG A 123      22.890  22.905  21.311  1.00 19.81           C  \\nATOM    963  C   ARG A 123      22.619  21.761  22.283  1.00 19.56           C  \\nATOM    964  O   ARG A 123      23.590  21.176  22.774  1.00 20.01           O  \\nATOM    965  CB  ARG A 123      23.421  22.333  19.988  1.00 19.81           C  \\nATOM    966  CG  ARG A 123      22.662  21.166  19.398  1.00 21.43           C  \\nATOM    967  CD  ARG A 123      23.273  20.680  18.073  1.00 22.66           C  \\nATOM    968  NE  ARG A 123      23.050  21.590  16.956  1.00 22.61           N  \\nATOM    969  CZ  ARG A 123      22.019  21.487  16.093  1.00 20.95           C  \\nATOM    970  NH1 ARG A 123      21.045  20.568  16.181  1.00 16.36           N  \\nATOM    971  NH2 ARG A 123      21.989  22.370  15.098  1.00 19.65           N  \\nATOM    972  N   THR A 124      21.364  21.389  22.600  1.00 17.96           N  \\nATOM    973  CA  THR A 124      21.118  20.200  23.415  1.00 17.85           C  \\nATOM    974  C   THR A 124      20.413  19.068  22.656  1.00 17.08           C  \\nATOM    975  O   THR A 124      20.459  17.926  23.091  1.00 19.59           O  \\nATOM    976  CB  THR A 124      20.311  20.578  24.672  1.00 19.42           C  \\nATOM    977  OG1 THR A 124      19.109  21.230  24.277  1.00 20.50           O  \\nATOM    978  CG2 THR A 124      21.155  21.458  25.588  1.00 19.06           C  \\nATOM    979  N   VAL A 125      19.788  19.319  21.495  1.00 16.59           N  \\nATOM    980  CA  VAL A 125      19.133  18.324  20.647  1.00 16.09           C  \\nATOM    981  C   VAL A 125      20.032  18.080  19.413  1.00 16.61           C  \\nATOM    982  O   VAL A 125      20.180  18.980  18.573  1.00 14.07           O  \\nATOM    983  CB  VAL A 125      17.720  18.848  20.179  1.00 15.28           C  \\nATOM    984  CG1 VAL A 125      17.097  17.814  19.265  1.00 14.81           C  \\nATOM    985  CG2 VAL A 125      16.824  19.168  21.366  1.00 15.97           C  \\nATOM    986  N   GLU A 126      20.659  16.889  19.280  1.00 17.05           N  \\nATOM    987  CA  GLU A 126      21.487  16.568  18.124  1.00 19.28           C  \\nATOM    988  C   GLU A 126      20.631  16.496  16.876  1.00 18.51           C  \\nATOM    989  O   GLU A 126      19.511  15.976  16.880  1.00 17.68           O  \\nATOM    990  CB  GLU A 126      22.165  15.215  18.219  1.00 23.29           C  \\nATOM    991  CG  GLU A 126      23.244  15.057  19.265  1.00 30.86           C  \\nATOM    992  CD  GLU A 126      24.402  16.054  19.153  1.00 36.59           C  \\nATOM    993  OE1 GLU A 126      25.139  16.012  18.161  1.00 39.64           O  \\nATOM    994  OE2 GLU A 126      24.581  16.885  20.056  1.00 40.44           O  \\nATOM    995  N   SER A 127      21.167  16.985  15.755  1.00 17.73           N  \\nATOM    996  CA  SER A 127      20.426  16.938  14.493  1.00 16.72           C  \\nATOM    997  C   SER A 127      20.040  15.506  14.158  1.00 16.76           C  \\nATOM    998  O   SER A 127      18.931  15.245  13.690  1.00 17.01           O  \\nATOM    999  CB  SER A 127      21.283  17.518  13.371  1.00 17.67           C  \\nATOM   1000  OG  SER A 127      21.730  18.826  13.768  1.00 19.67           O  \\nATOM   1001  N   ARG A 128      20.916  14.558  14.466  1.00 16.72           N  \\nATOM   1002  CA  ARG A 128      20.667  13.154  14.145  1.00 20.94           C  \\nATOM   1003  C   ARG A 128      19.435  12.617  14.862  1.00 19.62           C  \\nATOM   1004  O   ARG A 128      18.641  11.882  14.278  1.00 18.81           O  \\nATOM   1005  CB  ARG A 128      21.910  12.319  14.510  1.00 24.31           C  \\nATOM   1006  CG  ARG A 128      21.969  11.035  13.692  1.00 30.88           C  \\nATOM   1007  CD  ARG A 128      23.381  10.403  13.536  1.00 35.88           C  \\nATOM   1008  NE  ARG A 128      24.384  10.996  12.621  1.00 39.57           N  \\nATOM   1009  CZ  ARG A 128      24.313  11.018  11.267  1.00 40.85           C  \\nATOM   1010  NH1 ARG A 128      25.399  11.388  10.577  1.00 40.83           N  \\nATOM   1011  NH2 ARG A 128      23.202  10.685  10.576  1.00 41.33           N  \\nATOM   1012  N   GLN A 129      19.209  13.016  16.119  1.00 18.27           N  \\nATOM   1013  CA  GLN A 129      18.032  12.519  16.787  1.00 18.53           C  \\nATOM   1014  C   GLN A 129      16.735  13.151  16.224  1.00 17.11           C  \\nATOM   1015  O   GLN A 129      15.691  12.486  16.098  1.00 15.48           O  \\nATOM   1016  CB  GLN A 129      18.229  12.766  18.306  1.00 22.36           C  \\nATOM   1017  CG  GLN A 129      18.041  14.133  18.911  1.00 24.78           C  \\nATOM   1018  CD  GLN A 129      18.283  14.123  20.411  1.00 27.48           C  \\nATOM   1019  OE1 GLN A 129      19.287  14.643  20.906  1.00 28.14           O  \\nATOM   1020  NE2 GLN A 129      17.367  13.497  21.152  1.00 27.21           N  \\nATOM   1021  N   ALA A 130      16.780  14.446  15.837  1.00 14.73           N  \\nATOM   1022  CA  ALA A 130      15.611  15.044  15.210  1.00 13.87           C  \\nATOM   1023  C   ALA A 130      15.342  14.449  13.826  1.00 13.30           C  \\nATOM   1024  O   ALA A 130      14.199  14.194  13.436  1.00 11.42           O  \\nATOM   1025  CB  ALA A 130      15.822  16.536  15.063  1.00 16.21           C  \\nATOM   1026  N   GLN A 131      16.401  14.189  13.057  1.00 14.61           N  \\nATOM   1027  CA  GLN A 131      16.267  13.529  11.739  1.00 16.01           C  \\nATOM   1028  C   GLN A 131      15.637  12.109  11.821  1.00 16.29           C  \\nATOM   1029  O   GLN A 131      14.777  11.708  11.017  1.00 14.37           O  \\nATOM   1030  CB  GLN A 131      17.673  13.507  11.116  1.00 18.12           C  \\nATOM   1031  CG  GLN A 131      17.702  13.017   9.689  1.00 22.19           C  \\nATOM   1032  CD  GLN A 131      16.889  13.922   8.801  1.00 21.61           C  \\nATOM   1033  OE1 GLN A 131      17.292  15.044   8.529  1.00 24.06           O  \\nATOM   1034  NE2 GLN A 131      15.714  13.485   8.370  1.00 23.37           N  \\nATOM   1035  N   ASP A 132      16.038  11.348  12.853  1.00 16.64           N  \\nATOM   1036  CA  ASP A 132      15.467  10.038  13.173  1.00 18.00           C  \\nATOM   1037  C   ASP A 132      13.984  10.175  13.484  1.00 17.28           C  \\nATOM   1038  O   ASP A 132      13.178   9.395  12.985  1.00 16.45           O  \\nATOM   1039  CB  ASP A 132      16.120   9.415  14.405  1.00 20.10           C  \\nATOM   1040  CG  ASP A 132      17.506   8.816  14.234  1.00 24.07           C  \\nATOM   1041  OD1 ASP A 132      18.120   8.459  15.243  1.00 28.35           O  \\nATOM   1042  OD2 ASP A 132      17.975   8.686  13.106  1.00 25.43           O  \\nATOM   1043  N   LEU A 133      13.614  11.168  14.312  1.00 15.58           N  \\nATOM   1044  CA  LEU A 133      12.213  11.390  14.624  1.00 16.28           C  \\nATOM   1045  C   LEU A 133      11.421  11.729  13.366  1.00 14.96           C  \\nATOM   1046  O   LEU A 133      10.345  11.181  13.128  1.00 15.09           O  \\nATOM   1047  CB  LEU A 133      12.052  12.536  15.647  1.00 17.06           C  \\nATOM   1048  CG  LEU A 133      10.617  12.781  16.136  1.00 18.79           C  \\nATOM   1049  CD1 LEU A 133      10.142  11.554  16.881  1.00 19.96           C  \\nATOM   1050  CD2 LEU A 133      10.544  14.013  17.020  1.00 18.45           C  \\nATOM   1051  N   ALA A 134      11.937  12.642  12.545  1.00 13.16           N  \\nATOM   1052  CA  ALA A 134      11.241  13.016  11.331  1.00 14.89           C  \\nATOM   1053  C   ALA A 134      11.092  11.821  10.414  1.00 16.19           C  \\nATOM   1054  O   ALA A 134      10.015  11.643   9.857  1.00 15.70           O  \\nATOM   1055  CB  ALA A 134      12.021  14.089  10.629  1.00 15.12           C  \\nATOM   1056  N   ARG A 135      12.143  10.995  10.279  1.00 17.00           N  \\nATOM   1057  CA  ARG A 135      12.088   9.828   9.438  1.00 19.17           C  \\nATOM   1058  C   ARG A 135      10.990   8.893   9.916  1.00 18.70           C  \\nATOM   1059  O   ARG A 135      10.269   8.352   9.078  1.00 20.29           O  \\nATOM   1060  CB  ARG A 135      13.425   9.091   9.456  1.00 22.48           C  \\nATOM   1061  CG  ARG A 135      13.499   8.297   8.155  1.00 29.02           C  \\nATOM   1062  CD  ARG A 135      14.619   7.270   7.988  1.00 33.22           C  \\nATOM   1063  NE  ARG A 135      14.117   5.888   8.050  1.00 38.80           N  \\nATOM   1064  CZ  ARG A 135      13.281   5.302   7.148  1.00 39.60           C  \\nATOM   1065  NH1 ARG A 135      12.889   4.044   7.384  1.00 40.80           N  \\nATOM   1066  NH2 ARG A 135      12.827   5.913   6.034  1.00 40.11           N  \\nATOM   1067  N   SER A 136      10.815   8.697  11.242  1.00 19.23           N  \\nATOM   1068  CA  SER A 136       9.757   7.841  11.770  1.00 19.12           C  \\nATOM   1069  C   SER A 136       8.374   8.432  11.528  1.00 19.61           C  \\nATOM   1070  O   SER A 136       7.398   7.690  11.522  1.00 21.01           O  \\nATOM   1071  CB  SER A 136       9.930   7.624  13.280  1.00 19.94           C  \\nATOM   1072  OG  SER A 136       9.459   8.686  14.119  1.00 23.21           O  \\nATOM   1073  N   TYR A 137       8.231   9.760  11.380  1.00 17.33           N  \\nATOM   1074  CA  TYR A 137       6.950  10.375  11.078  1.00 18.12           C  \\nATOM   1075  C   TYR A 137       6.683  10.404   9.570  1.00 18.22           C  \\nATOM   1076  O   TYR A 137       5.561  10.656   9.129  1.00 19.27           O  \\nATOM   1077  CB  TYR A 137       6.907  11.812  11.592  1.00 18.21           C  \\nATOM   1078  CG  TYR A 137       6.883  12.057  13.084  1.00 19.08           C  \\nATOM   1079  CD1 TYR A 137       6.619  11.026  13.955  1.00 20.51           C  \\nATOM   1080  CD2 TYR A 137       7.059  13.355  13.548  1.00 20.43           C  \\nATOM   1081  CE1 TYR A 137       6.510  11.285  15.306  1.00 21.89           C  \\nATOM   1082  CE2 TYR A 137       6.945  13.624  14.899  1.00 21.02           C  \\nATOM   1083  CZ  TYR A 137       6.662  12.574  15.763  1.00 22.13           C  \\nATOM   1084  OH  TYR A 137       6.476  12.782  17.112  1.00 22.16           O  \\nATOM   1085  N   GLY A 138       7.689  10.186   8.737  1.00 16.07           N  \\nATOM   1086  CA  GLY A 138       7.520  10.265   7.294  1.00 17.66           C  \\nATOM   1087  C   GLY A 138       7.476  11.725   6.810  1.00 18.01           C  \\nATOM   1088  O   GLY A 138       6.859  12.010   5.774  1.00 17.85           O  \\nATOM   1089  N   ILE A 139       8.128  12.662   7.540  1.00 14.16           N  \\nATOM   1090  CA  ILE A 139       8.117  14.065   7.145  1.00 13.49           C  \\nATOM   1091  C   ILE A 139       9.526  14.662   7.033  1.00 14.35           C  \\nATOM   1092  O   ILE A 139      10.494  14.152   7.610  1.00 16.19           O  \\nATOM   1093  CB  ILE A 139       7.292  14.896   8.161  1.00 14.40           C  \\nATOM   1094  CG1 ILE A 139       7.925  14.811   9.554  1.00 15.44           C  \\nATOM   1095  CG2 ILE A 139       5.821  14.422   8.104  1.00 14.07           C  \\nATOM   1096  CD1 ILE A 139       7.288  15.765  10.587  1.00 15.35           C  \\nATOM   1097  N   PRO A 140       9.706  15.767   6.308  1.00 13.55           N  \\nATOM   1098  CA  PRO A 140      10.984  16.448   6.205  1.00 13.39           C  \\nATOM   1099  C   PRO A 140      11.424  17.065   7.532  1.00 12.74           C  \\nATOM   1100  O   PRO A 140      10.601  17.530   8.310  1.00 11.42           O  \\nATOM   1101  CB  PRO A 140      10.767  17.498   5.127  1.00 13.47           C  \\nATOM   1102  CG  PRO A 140       9.514  17.084   4.410  1.00 14.09           C  \\nATOM   1103  CD  PRO A 140       8.686  16.467   5.519  1.00 13.14           C  \\nATOM   1104  N   TYR A 141      12.744  17.086   7.756  1.00 11.68           N  \\nATOM   1105  CA  TYR A 141      13.336  17.791   8.890  1.00 10.89           C  \\nATOM   1106  C   TYR A 141      14.175  18.945   8.326  1.00 10.09           C  \\nATOM   1107  O   TYR A 141      15.056  18.720   7.468  1.00 10.27           O  \\nATOM   1108  CB  TYR A 141      14.276  16.897   9.691  1.00 12.54           C  \\nATOM   1109  CG  TYR A 141      15.063  17.676  10.733  1.00 10.92           C  \\nATOM   1110  CD1 TYR A 141      14.350  18.396  11.664  1.00 11.75           C  \\nATOM   1111  CD2 TYR A 141      16.453  17.633  10.784  1.00 12.49           C  \\nATOM   1112  CE1 TYR A 141      14.997  19.074  12.670  1.00 12.85           C  \\nATOM   1113  CE2 TYR A 141      17.112  18.307  11.794  1.00 11.33           C  \\nATOM   1114  CZ  TYR A 141      16.369  19.016  12.728  1.00 12.61           C  \\nATOM   1115  OH  TYR A 141      16.944  19.646  13.813  1.00 13.82           O  \\nATOM   1116  N   ILE A 142      13.892  20.174   8.799  1.00  9.35           N  \\nATOM   1117  CA  ILE A 142      14.636  21.355   8.376  1.00  9.05           C  \\nATOM   1118  C   ILE A 142      15.153  22.163   9.551  1.00  9.25           C  \\nATOM   1119  O   ILE A 142      14.409  22.382  10.499  1.00 10.43           O  \\nATOM   1120  CB  ILE A 142      13.756  22.259   7.472  1.00  9.25           C  \\nATOM   1121  CG1 ILE A 142      13.523  21.470   6.180  1.00 10.51           C  \\nATOM   1122  CG2 ILE A 142      14.450  23.586   7.122  1.00  9.08           C  \\nATOM   1123  CD1 ILE A 142      12.719  22.127   5.028  1.00 13.16           C  \\nATOM   1124  N   GLU A 143      16.457  22.513   9.568  1.00  9.01           N  \\nATOM   1125  CA  GLU A 143      16.979  23.404  10.611  1.00  8.52           C  \\nATOM   1126  C   GLU A 143      16.978  24.864  10.193  1.00 10.31           C  \\nATOM   1127  O   GLU A 143      17.278  25.228   9.040  1.00 10.45           O  \\nATOM   1128  CB  GLU A 143      18.397  23.022  11.013  1.00 10.87           C  \\nATOM   1129  CG  GLU A 143      18.363  21.728  11.801  1.00 10.22           C  \\nATOM   1130  CD  GLU A 143      19.592  21.411  12.633  1.00 13.01           C  \\nATOM   1131  OE1 GLU A 143      20.616  22.082  12.521  1.00 13.65           O  \\nATOM   1132  OE2 GLU A 143      19.511  20.487  13.429  1.00 12.70           O  \\nATOM   1133  N   THR A 144      16.608  25.762  11.117  1.00  9.18           N  \\nATOM   1134  CA  THR A 144      16.422  27.169  10.754  1.00  9.71           C  \\nATOM   1135  C   THR A 144      17.014  28.076  11.789  1.00  8.66           C  \\nATOM   1136  O   THR A 144      17.259  27.698  12.927  1.00  9.13           O  \\nATOM   1137  CB  THR A 144      14.919  27.617  10.635  1.00  9.90           C  \\nATOM   1138  OG1 THR A 144      14.336  27.403  11.931  1.00  9.29           O  \\nATOM   1139  CG2 THR A 144      14.120  26.858   9.557  1.00  9.89           C  \\nATOM   1140  N   SER A 145      17.244  29.288  11.360  1.00 10.91           N  \\nATOM   1141  CA  SER A 145      17.558  30.377  12.290  1.00 12.65           C  \\nATOM   1142  C   SER A 145      16.758  31.602  11.849  1.00 10.37           C  \\nATOM   1143  O   SER A 145      16.930  32.094  10.727  1.00 11.76           O  \\nATOM   1144  CB  SER A 145      19.069  30.768  12.308  1.00 11.79           C  \\nATOM   1145  OG  SER A 145      19.248  31.931  13.125  1.00 12.12           O  \\nATOM   1146  N   ALA A 146      15.838  32.046  12.696  1.00  9.49           N  \\nATOM   1147  CA  ALA A 146      15.131  33.268  12.413  1.00 10.90           C  \\nATOM   1148  C   ALA A 146      16.070  34.455  12.553  1.00 13.53           C  \\nATOM   1149  O   ALA A 146      15.799  35.514  11.995  1.00 16.20           O  \\nATOM   1150  CB  ALA A 146      13.976  33.436  13.367  1.00 12.65           C  \\nATOM   1151  N   LYS A 147      17.183  34.302  13.283  1.00 14.35           N  \\nATOM   1152  CA  LYS A 147      18.150  35.351  13.486  1.00 16.25           C  \\nATOM   1153  C   LYS A 147      18.953  35.568  12.215  1.00 17.58           C  \\nATOM   1154  O   LYS A 147      19.124  36.724  11.825  1.00 20.13           O  \\nATOM   1155  CB  LYS A 147      19.068  34.970  14.617  1.00 17.55           C  \\nATOM   1156  CG  LYS A 147      20.003  36.125  14.958  1.00 22.83           C  \\nATOM   1157  CD  LYS A 147      20.581  35.932  16.341  1.00 26.44           C  \\nATOM   1158  CE  LYS A 147      21.683  36.949  16.507  1.00 30.79           C  \\nATOM   1159  NZ  LYS A 147      22.575  36.534  17.574  1.00 33.81           N  \\nATOM   1160  N   THR A 148      19.479  34.529  11.541  1.00 17.16           N  \\nATOM   1161  CA  THR A 148      20.248  34.735  10.300  1.00 15.93           C  \\nATOM   1162  C   THR A 148      19.422  34.607   9.013  1.00 17.72           C  \\nATOM   1163  O   THR A 148      19.926  34.962   7.945  1.00 18.47           O  \\nATOM   1164  CB  THR A 148      21.444  33.745  10.209  1.00 15.86           C  \\nATOM   1165  OG1 THR A 148      20.886  32.435  10.042  1.00 14.56           O  \\nATOM   1166  CG2 THR A 148      22.377  33.839  11.439  1.00 15.25           C  \\nATOM   1167  N   ARG A 149      18.176  34.108   9.100  1.00 15.18           N  \\nATOM   1168  CA  ARG A 149      17.268  33.815   7.994  1.00 19.20           C  \\nATOM   1169  C   ARG A 149      17.652  32.483   7.318  1.00 17.94           C  \\nATOM   1170  O   ARG A 149      17.031  32.079   6.332  1.00 20.28           O  \\nATOM   1171  CB  ARG A 149      17.251  34.964   6.937  1.00 21.40           C  \\nATOM   1172  CG  ARG A 149      15.929  35.106   6.188  1.00 27.00           C  \\nATOM   1173  CD  ARG A 149      15.946  35.904   4.863  1.00 27.99           C  \\nATOM   1174  NE  ARG A 149      14.707  35.687   4.092  1.00 29.40           N  \\nATOM   1175  CZ  ARG A 149      13.550  36.232   4.492  1.00 29.07           C  \\nATOM   1176  NH1 ARG A 149      12.412  36.015   3.828  1.00 29.56           N  \\nATOM   1177  NH2 ARG A 149      13.527  37.004   5.577  1.00 28.99           N  \\nATOM   1178  N   GLN A 150      18.621  31.701   7.838  1.00 15.80           N  \\nATOM   1179  CA  GLN A 150      18.926  30.402   7.273  1.00 16.36           C  \\nATOM   1180  C   GLN A 150      17.737  29.464   7.455  1.00 15.02           C  \\nATOM   1181  O   GLN A 150      17.119  29.351   8.504  1.00 13.57           O  \\nATOM   1182  CB  GLN A 150      20.152  29.729   7.937  1.00 23.05           C  \\nATOM   1183  CG  GLN A 150      20.578  28.448   7.162  1.00 29.37           C  \\nATOM   1184  CD  GLN A 150      21.604  27.530   7.827  1.00 34.84           C  \\nATOM   1185  OE1 GLN A 150      21.301  26.416   8.280  1.00 38.95           O  \\nATOM   1186  NE2 GLN A 150      22.857  27.967   7.883  1.00 35.22           N  \\nATOM   1187  N   GLY A 151      17.412  28.809   6.348  1.00 13.62           N  \\nATOM   1188  CA  GLY A 151      16.357  27.835   6.296  1.00 13.50           C  \\nATOM   1189  C   GLY A 151      14.943  28.395   6.329  1.00 12.36           C  \\nATOM   1190  O   GLY A 151      14.025  27.591   6.148  1.00 11.07           O  \\nATOM   1191  N   VAL A 152      14.706  29.702   6.510  1.00 12.59           N  \\nATOM   1192  CA  VAL A 152      13.339  30.183   6.715  1.00 14.23           C  \\nATOM   1193  C   VAL A 152      12.435  29.971   5.475  1.00 13.41           C  \\nATOM   1194  O   VAL A 152      11.352  29.362   5.577  1.00 12.94           O  \\nATOM   1195  CB  VAL A 152      13.430  31.691   7.167  1.00 12.57           C  \\nATOM   1196  CG1 VAL A 152      12.059  32.316   7.194  1.00 13.14           C  \\nATOM   1197  CG2 VAL A 152      14.044  31.787   8.543  1.00 12.69           C  \\nATOM   1198  N   GLU A 153      12.831  30.410   4.278  1.00 13.52           N  \\nATOM   1199  CA  GLU A 153      12.001  30.213   3.086  1.00 14.81           C  \\nATOM   1200  C   GLU A 153      11.857  28.756   2.742  1.00 11.88           C  \\nATOM   1201  O   GLU A 153      10.797  28.282   2.344  1.00 13.92           O  \\nATOM   1202  CB  GLU A 153      12.592  30.936   1.881  1.00 17.03           C  \\nATOM   1203  CG  GLU A 153      12.454  32.452   2.015  1.00 21.18           C  \\nATOM   1204  CD  GLU A 153      12.870  33.318   0.817  1.00 25.73           C  \\nATOM   1205  OE1 GLU A 153      13.232  32.781  -0.234  1.00 28.32           O  \\nATOM   1206  OE2 GLU A 153      12.822  34.550   0.936  1.00 26.64           O  \\nATOM   1207  N   ASP A 154      12.959  28.031   2.957  1.00 11.31           N  \\nATOM   1208  CA  ASP A 154      13.023  26.593   2.747  1.00 10.16           C  \\nATOM   1209  C   ASP A 154      11.964  25.828   3.561  1.00 10.90           C  \\nATOM   1210  O   ASP A 154      11.224  24.985   3.035  1.00 11.83           O  \\nATOM   1211  CB  ASP A 154      14.420  26.212   3.117  1.00 15.00           C  \\nATOM   1212  CG  ASP A 154      14.919  24.942   2.524  1.00 19.04           C  \\nATOM   1213  OD1 ASP A 154      14.121  24.195   1.938  1.00 22.96           O  \\nATOM   1214  OD2 ASP A 154      16.122  24.697   2.676  1.00 21.26           O  \\nATOM   1215  N   ALA A 155      11.854  26.107   4.881  1.00  8.83           N  \\nATOM   1216  CA  ALA A 155      10.841  25.450   5.716  1.00 10.00           C  \\nATOM   1217  C   ALA A 155       9.424  25.748   5.203  1.00 10.10           C  \\nATOM   1218  O   ALA A 155       8.653  24.814   4.978  1.00 11.14           O  \\nATOM   1219  CB  ALA A 155      10.923  25.933   7.192  1.00  9.50           C  \\nATOM   1220  N   PHE A 156       9.054  27.011   4.949  1.00 10.59           N  \\nATOM   1221  CA  PHE A 156       7.712  27.345   4.475  1.00 10.51           C  \\nATOM   1222  C   PHE A 156       7.453  26.825   3.069  1.00 11.05           C  \\nATOM   1223  O   PHE A 156       6.368  26.302   2.801  1.00 10.80           O  \\nATOM   1224  CB  PHE A 156       7.492  28.890   4.506  1.00 11.54           C  \\nATOM   1225  CG  PHE A 156       7.266  29.450   5.936  1.00 12.42           C  \\nATOM   1226  CD1 PHE A 156       8.319  29.958   6.707  1.00 12.00           C  \\nATOM   1227  CD2 PHE A 156       5.978  29.434   6.478  1.00 12.57           C  \\nATOM   1228  CE1 PHE A 156       8.065  30.434   8.001  1.00 12.45           C  \\nATOM   1229  CE2 PHE A 156       5.747  29.908   7.757  1.00 12.32           C  \\nATOM   1230  CZ  PHE A 156       6.782  30.406   8.513  1.00 10.18           C  \\nATOM   1231  N   TYR A 157       8.408  26.935   2.151  1.00  9.69           N  \\nATOM   1232  CA  TYR A 157       8.148  26.463   0.798  1.00 12.31           C  \\nATOM   1233  C   TYR A 157       8.091  24.945   0.719  1.00 10.42           C  \\nATOM   1234  O   TYR A 157       7.292  24.403  -0.047  1.00 11.35           O  \\nATOM   1235  CB  TYR A 157       9.233  26.978  -0.156  1.00 15.16           C  \\nATOM   1236  CG  TYR A 157       9.257  28.492  -0.367  1.00 18.73           C  \\nATOM   1237  CD1 TYR A 157       8.380  29.366   0.268  1.00 18.52           C  \\nATOM   1238  CD2 TYR A 157      10.211  28.971  -1.242  1.00 20.44           C  \\nATOM   1239  CE1 TYR A 157       8.466  30.724   0.025  1.00 22.04           C  \\nATOM   1240  CE2 TYR A 157      10.305  30.320  -1.488  1.00 22.70           C  \\nATOM   1241  CZ  TYR A 157       9.436  31.178  -0.852  1.00 23.03           C  \\nATOM   1242  OH  TYR A 157       9.572  32.528  -1.089  1.00 26.15           O  \\nATOM   1243  N   THR A 158       8.940  24.198   1.455  1.00 11.35           N  \\nATOM   1244  CA  THR A 158       8.836  22.727   1.522  1.00 10.11           C  \\nATOM   1245  C   THR A 158       7.440  22.347   2.064  1.00 11.11           C  \\nATOM   1246  O   THR A 158       6.811  21.408   1.558  1.00 11.43           O  \\nATOM   1247  CB  THR A 158       9.935  22.129   2.470  1.00 12.09           C  \\nATOM   1248  OG1 THR A 158      11.216  22.451   1.933  1.00 11.36           O  \\nATOM   1249  CG2 THR A 158       9.791  20.657   2.624  1.00 10.87           C  \\nATOM   1250  N   LEU A 159       6.888  23.056   3.072  1.00  9.25           N  \\nATOM   1251  CA  LEU A 159       5.533  22.754   3.506  1.00  9.93           C  \\nATOM   1252  C   LEU A 159       4.489  22.986   2.388  1.00 10.42           C  \\nATOM   1253  O   LEU A 159       3.560  22.185   2.272  1.00 11.22           O  \\nATOM   1254  CB  LEU A 159       5.171  23.608   4.711  1.00  9.03           C  \\nATOM   1255  CG  LEU A 159       3.789  23.318   5.269  1.00 10.96           C  \\nATOM   1256  CD1 LEU A 159       3.635  21.828   5.651  1.00  9.29           C  \\nATOM   1257  CD2 LEU A 159       3.580  24.223   6.469  1.00 13.71           C  \\nATOM   1258  N   VAL A 160       4.596  24.046   1.548  1.00 10.91           N  \\nATOM   1259  CA  VAL A 160       3.662  24.284   0.439  1.00  9.19           C  \\nATOM   1260  C   VAL A 160       3.803  23.137  -0.556  1.00  9.60           C  \\nATOM   1261  O   VAL A 160       2.781  22.638  -1.031  1.00 10.29           O  \\nATOM   1262  CB  VAL A 160       3.981  25.653  -0.211  1.00  8.50           C  \\nATOM   1263  CG1 VAL A 160       3.402  25.754  -1.622  1.00 10.56           C  \\nATOM   1264  CG2 VAL A 160       3.403  26.741   0.659  1.00  7.98           C  \\nATOM   1265  N   ARG A 161       5.033  22.667  -0.839  1.00  9.10           N  \\nATOM   1266  CA  ARG A 161       5.206  21.554  -1.767  1.00  9.82           C  \\nATOM   1267  C   ARG A 161       4.644  20.240  -1.215  1.00 10.44           C  \\nATOM   1268  O   ARG A 161       4.166  19.395  -1.984  1.00 12.88           O  \\nATOM   1269  CB  ARG A 161       6.679  21.398  -2.115  1.00  9.39           C  \\nATOM   1270  CG  ARG A 161       7.146  22.640  -2.816  1.00  9.70           C  \\nATOM   1271  CD  ARG A 161       8.594  22.398  -3.227  1.00 14.12           C  \\nATOM   1272  NE  ARG A 161       9.122  23.635  -3.780  1.00 14.15           N  \\nATOM   1273  CZ  ARG A 161      10.144  24.313  -3.257  1.00 12.45           C  \\nATOM   1274  NH1 ARG A 161      10.823  23.858  -2.204  1.00 13.24           N  \\nATOM   1275  NH2 ARG A 161      10.485  25.475  -3.819  1.00 13.52           N  \\nATOM   1276  N   GLU A 162       4.663  20.046   0.112  1.00 10.71           N  \\nATOM   1277  CA  GLU A 162       4.054  18.877   0.722  1.00 11.80           C  \\nATOM   1278  C   GLU A 162       2.522  18.887   0.568  1.00 11.20           C  \\nATOM   1279  O   GLU A 162       1.921  17.848   0.274  1.00 12.90           O  \\nATOM   1280  CB  GLU A 162       4.430  18.815   2.216  1.00 13.86           C  \\nATOM   1281  CG  GLU A 162       5.870  18.460   2.580  1.00 14.10           C  \\nATOM   1282  CD  GLU A 162       6.268  17.038   2.224  1.00 17.23           C  \\nATOM   1283  OE1 GLU A 162       5.571  16.116   2.627  1.00 19.29           O  \\nATOM   1284  OE2 GLU A 162       7.278  16.827   1.545  1.00 22.14           O  \\nATOM   1285  N   ILE A 163       1.869  20.045   0.731  1.00 10.13           N  \\nATOM   1286  CA  ILE A 163       0.426  20.175   0.542  1.00 10.58           C  \\nATOM   1287  C   ILE A 163       0.128  19.878  -0.933  1.00 13.43           C  \\nATOM   1288  O   ILE A 163      -0.796  19.118  -1.235  1.00 15.01           O  \\nATOM   1289  CB  ILE A 163      -0.015  21.590   0.891  1.00 11.33           C  \\nATOM   1290  CG1 ILE A 163       0.084  21.777   2.400  1.00 11.27           C  \\nATOM   1291  CG2 ILE A 163      -1.486  21.815   0.437  1.00 12.37           C  \\nATOM   1292  CD1 ILE A 163      -0.006  23.245   2.894  1.00 12.56           C  \\nATOM   1293  N   ARG A 164       0.904  20.456  -1.871  1.00 11.72           N  \\nATOM   1294  CA  ARG A 164       0.770  20.163  -3.283  1.00 13.64           C  \\nATOM   1295  C   ARG A 164       0.813  18.657  -3.626  1.00 17.97           C  \\nATOM   1296  O   ARG A 164       0.150  18.197  -4.559  1.00 18.12           O  \\nATOM   1297  CB  ARG A 164       1.897  20.887  -4.029  1.00 12.51           C  \\nATOM   1298  CG  ARG A 164       1.647  22.371  -4.120  1.00 12.49           C  \\nATOM   1299  CD  ARG A 164       2.801  22.888  -4.950  1.00 14.72           C  \\nATOM   1300  NE  ARG A 164       2.730  24.327  -5.113  1.00 12.82           N  \\nATOM   1301  CZ  ARG A 164       3.800  25.071  -5.405  1.00 13.29           C  \\nATOM   1302  NH1 ARG A 164       5.018  24.539  -5.527  1.00 11.56           N  \\nATOM   1303  NH2 ARG A 164       3.595  26.381  -5.572  1.00 12.55           N  \\nATOM   1304  N   GLN A 165       1.590  17.831  -2.935  1.00 20.23           N  \\nATOM   1305  CA  GLN A 165       1.673  16.435  -3.337  1.00 27.50           C  \\nATOM   1306  C   GLN A 165       0.782  15.499  -2.502  1.00 29.12           C  \\nATOM   1307  O   GLN A 165       0.826  14.274  -2.655  1.00 29.30           O  \\nATOM   1308  CB  GLN A 165       3.168  16.070  -3.286  1.00 30.34           C  \\nATOM   1309  CG  GLN A 165       4.018  16.904  -4.309  1.00 35.17           C  \\nATOM   1310  CD  GLN A 165       4.109  16.470  -5.796  1.00 37.71           C  \\nATOM   1311  OE1 GLN A 165       4.256  17.293  -6.719  1.00 36.70           O  \\nATOM   1312  NE2 GLN A 165       4.079  15.166  -6.094  1.00 38.13           N  \\nATOM   1313  N   HIS A 166      -0.068  16.075  -1.638  1.00 29.28           N  \\nATOM   1314  CA  HIS A 166      -0.982  15.296  -0.820  1.00 31.13           C  \\nATOM   1315  C   HIS A 166      -2.214  14.971  -1.665  1.00 33.30           C  \\nATOM   1316  O   HIS A 166      -2.630  15.786  -2.497  1.00 34.96           O  \\nATOM   1317  CB  HIS A 166      -1.420  16.090   0.428  1.00 30.89           C  \\nATOM   1318  CG  HIS A 166      -2.247  15.319   1.475  1.00 32.17           C  \\nATOM   1319  ND1 HIS A 166      -3.579  15.231   1.603  1.00 33.52           N  \\nATOM   1320  CD2 HIS A 166      -1.712  14.560   2.502  1.00 32.95           C  \\nATOM   1321  CE1 HIS A 166      -3.857  14.464   2.647  1.00 33.07           C  \\nATOM   1322  NE2 HIS A 166      -2.727  14.064   3.184  1.00 33.06           N  \\nATOM   1323  OXT HIS A 166      -2.763  13.888  -1.454  1.00 36.27           O  \\nTER    1324      HIS A 166                                                      \\nHETATM 1325 MG    MG A 168       4.771  33.860  19.213  1.00  9.42          MG  \\nHETATM 1326  PG  GNP A 167       5.113  32.223  21.994  1.00  9.66           P  \\nHETATM 1327  O1G GNP A 167       4.844  32.596  23.390  1.00  9.26           O  \\nHETATM 1328  O2G GNP A 167       4.203  32.910  21.112  1.00 11.20           O  \\nHETATM 1329  O3G GNP A 167       4.640  30.769  21.802  1.00 10.09           O  \\nHETATM 1330  N3B GNP A 167       6.701  32.303  21.621  1.00  8.18           N  \\nHETATM 1331  PB  GNP A 167       7.310  32.034  20.111  1.00  8.77           P  \\nHETATM 1332  O1B GNP A 167       7.468  30.605  19.791  1.00  8.35           O  \\nHETATM 1333  O2B GNP A 167       6.696  32.775  19.021  1.00  9.38           O  \\nHETATM 1334  O3A GNP A 167       8.792  32.643  20.152  1.00 10.15           O  \\nHETATM 1335  PA  GNP A 167       9.280  34.140  19.710  1.00 10.18           P  \\nHETATM 1336  O1A GNP A 167       9.263  34.268  18.220  1.00 10.07           O  \\nHETATM 1337  O2A GNP A 167       8.594  35.173  20.510  1.00  9.33           O  \\nHETATM 1338  O5' GNP A 167      10.826  34.056  20.132  1.00 11.46           O  \\nHETATM 1339  C5' GNP A 167      11.170  33.911  21.535  1.00 10.56           C  \\nHETATM 1340  C4' GNP A 167      12.587  34.445  21.811  1.00 11.96           C  \\nHETATM 1341  O4' GNP A 167      13.560  33.649  21.095  1.00 11.66           O  \\nHETATM 1342  C3' GNP A 167      12.813  35.910  21.359  1.00 12.06           C  \\nHETATM 1343  O3' GNP A 167      13.793  36.538  22.221  1.00 14.52           O  \\nHETATM 1344  C2' GNP A 167      13.325  35.679  19.947  1.00 11.90           C  \\nHETATM 1345  O2' GNP A 167      14.151  36.750  19.499  1.00 14.99           O  \\nHETATM 1346  C1' GNP A 167      14.242  34.478  20.156  1.00 12.90           C  \\nHETATM 1347  N9  GNP A 167      14.454  33.739  18.896  1.00 12.98           N  \\nHETATM 1348  C8  GNP A 167      13.518  33.262  17.993  1.00 11.76           C  \\nHETATM 1349  N7  GNP A 167      14.016  32.636  16.979  1.00 12.34           N  \\nHETATM 1350  C5  GNP A 167      15.413  32.694  17.200  1.00 12.36           C  \\nHETATM 1351  C6  GNP A 167      16.521  32.219  16.402  1.00 12.30           C  \\nHETATM 1352  O6  GNP A 167      16.507  31.591  15.338  1.00 12.16           O  \\nHETATM 1353  N1  GNP A 167      17.735  32.503  16.995  1.00 11.60           N  \\nHETATM 1354  C2  GNP A 167      17.909  33.082  18.236  1.00 13.91           C  \\nHETATM 1355  N2  GNP A 167      19.173  33.280  18.658  1.00 14.60           N  \\nHETATM 1356  N3  GNP A 167      16.884  33.498  18.996  1.00 13.01           N  \\nHETATM 1357  C4  GNP A 167      15.675  33.331  18.398  1.00 14.36           C  \\nHETATM 1358  O   HOH A 169      13.588  43.386  17.779  1.00 31.80           O  \\nHETATM 1359  O   HOH A 170       8.248  36.064  23.209  1.00 13.63           O  \\nHETATM 1360  O   HOH A 171       3.254  37.845  19.454  1.00 18.29           O  \\nHETATM 1361  O   HOH A 172       5.945  35.619  20.105  1.00 11.33           O  \\nHETATM 1362  O   HOH A 173       3.669  32.226  18.172  1.00  8.22           O  \\nHETATM 1363  O   HOH A 174       2.896  37.170  15.500  1.00 21.37           O  \\nHETATM 1364  O   HOH A 175       2.102  32.309  24.122  1.00 27.47           O  \\nHETATM 1365  O   HOH A 176      -1.425  37.650  19.667  1.00 24.98           O  \\nHETATM 1366  O   HOH A 177      -3.225  30.031  -8.217  1.00 23.57           O  \\nHETATM 1367  O   HOH A 178      16.678  26.321  26.916  1.00 36.70           O  \\nHETATM 1368  O   HOH A 179      -2.134  40.975   1.551  1.00 81.53           O  \\nHETATM 1369  O   HOH A 180       0.400  40.675  -0.629  1.00 24.66           O  \\nHETATM 1370  O   HOH A 181      10.944  44.793  16.009  1.00 48.47           O  \\nHETATM 1371  O   HOH A 182      17.207  42.570  16.786  1.00 43.23           O  \\nHETATM 1372  O   HOH A 183      14.999  45.167  19.922  1.00 39.52           O  \\nHETATM 1373  O   HOH A 184      11.598  37.755  13.202  1.00 11.72           O  \\nHETATM 1374  O   HOH A 185       6.958  38.922  18.146  1.00 26.17           O  \\nHETATM 1375  O   HOH A 186      14.078  32.033  27.772  1.00 57.73           O  \\nHETATM 1376  O   HOH A 187       9.126  37.906  19.595  1.00 32.42           O  \\nHETATM 1377  O   HOH A 188      -0.049  36.435  27.259  1.00 50.56           O  \\nHETATM 1378  O   HOH A 189       3.843  30.503  25.485  1.00 46.80           O  \\nHETATM 1379  O   HOH A 190       4.931  28.343  27.587  1.00 29.32           O  \\nHETATM 1380  O   HOH A 191       4.881  25.774  26.247  1.00 28.55           O  \\nHETATM 1381  O   HOH A 193       8.729  26.516  26.645  1.00 37.34           O  \\nHETATM 1382  O   HOH A 194      -5.515  23.731  -4.213  1.00 43.74           O  \\nHETATM 1383  O   HOH A 195      -7.006  26.635   0.729  1.00 25.77           O  \\nHETATM 1384  O   HOH A 196      -2.033  23.281  -4.754  1.00 36.89           O  \\nHETATM 1385  O   HOH A 197      -4.326  24.642  -0.854  1.00 28.59           O  \\nHETATM 1386  O   HOH A 198      -6.858  26.995  -2.808  1.00 28.83           O  \\nHETATM 1387  O   HOH A 200      -3.937  29.225   8.289  1.00 24.08           O  \\nHETATM 1388  O   HOH A 201      -8.934  24.037   2.795  1.00 34.16           O  \\nHETATM 1389  O   HOH A 202       1.444  26.302  19.318  1.00 12.37           O  \\nHETATM 1390  O   HOH A 203       2.031  24.632  27.307  1.00 50.09           O  \\nHETATM 1391  O   HOH A 204      15.125  28.341  23.174  1.00 13.88           O  \\nHETATM 1392  O   HOH A 205      12.808  31.541  25.056  1.00 46.88           O  \\nHETATM 1393  O   HOH A 206       8.329  34.933   0.543  1.00 53.21           O  \\nHETATM 1394  O   HOH A 207       0.760  42.969   2.336  1.00 35.42           O  \\nHETATM 1395  O   HOH A 208       1.270  40.298   7.071  1.00 76.61           O  \\nHETATM 1396  O   HOH A 209      -3.135  39.625   7.649  1.00 35.40           O  \\nHETATM 1397  O   HOH A 210       8.699  39.076   2.399  1.00 33.93           O  \\nHETATM 1398  O   HOH A 212      10.040  41.806   3.447  1.00 37.88           O  \\nHETATM 1399  O   HOH A 213      18.914  29.095   3.704  1.00 25.32           O  \\nHETATM 1400  O   HOH A 214      15.966  29.098   2.957  1.00 18.63           O  \\nHETATM 1401  O   HOH A 215      15.506  31.962   4.015  1.00 18.93           O  \\nHETATM 1402  O   HOH A 216      22.935  30.533  10.845  1.00 39.81           O  \\nHETATM 1403  O   HOH A 217       6.476  43.893  23.325  1.00 49.09           O  \\nHETATM 1404  O   HOH A 218      11.746  41.248  28.612  1.00 36.22           O  \\nHETATM 1405  O   HOH A 219      11.480  42.270  25.715  1.00 29.66           O  \\nHETATM 1406  O   HOH A 220       9.344  34.251  -8.278  1.00 36.61           O  \\nHETATM 1407  O   HOH A 221      -0.111  19.667  -8.066  1.00 44.76           O  \\nHETATM 1408  O   HOH A 222      -0.528  30.460 -10.943  1.00 24.29           O  \\nHETATM 1409  O   HOH A 223     -11.352  24.274  19.491  1.00 31.21           O  \\nHETATM 1410  O   HOH A 224      -4.293  30.642 -16.466  1.00127.55           O  \\nHETATM 1411  O   HOH A 225       3.466  29.729 -16.232  1.00 26.15           O  \\nHETATM 1412  O   HOH A 227      -4.288  18.610   4.198  1.00 34.77           O  \\nHETATM 1413  O   HOH A 228       4.626  16.154   5.151  1.00 14.96           O  \\nHETATM 1414  O   HOH A 229       5.681  11.069   2.580  1.00 59.76           O  \\nHETATM 1415  O   HOH A 230       1.173  14.021   3.406  1.00 32.15           O  \\nHETATM 1416  O   HOH A 231       7.935  22.484  20.504  1.00 10.86           O  \\nHETATM 1417  O   HOH A 232       3.071  10.004  14.546  1.00 23.48           O  \\nHETATM 1418  O   HOH A 233       2.437  13.381   8.401  1.00 17.75           O  \\nHETATM 1419  O   HOH A 234       4.779   7.500   6.154  1.00 71.09           O  \\nHETATM 1420  O   HOH A 235      -4.250   8.748   6.880  1.00 24.76           O  \\nHETATM 1421  O   HOH A 236     -13.381  22.830  17.669  1.00 39.09           O  \\nHETATM 1422  O   HOH A 237      16.969  39.052   8.276  1.00 50.25           O  \\nHETATM 1423  O   HOH A 238      -6.315   9.333  13.897  1.00 36.24           O  \\nHETATM 1424  O   HOH A 240      -0.160   4.395  12.468  1.00 38.79           O  \\nHETATM 1425  O   HOH A 241       1.295   7.931  12.335  1.00 49.03           O  \\nHETATM 1426  O   HOH A 242       4.067  11.551   6.968  1.00 46.97           O  \\nHETATM 1427  O   HOH A 243      -5.060   6.579  -0.216  1.00 50.14           O  \\nHETATM 1428  O   HOH A 244      18.164   7.151   5.573  1.00 71.67           O  \\nHETATM 1429  O   HOH A 245       2.068   8.904   7.636  1.00 45.86           O  \\nHETATM 1430  O   HOH A 246      24.661  23.134  12.757  1.00 34.69           O  \\nHETATM 1431  O   HOH A 247      21.051  23.714   6.239  1.00 24.64           O  \\nHETATM 1432  O   HOH A 248      25.855  31.137  20.712  1.00 34.53           O  \\nHETATM 1433  O   HOH A 249      24.868  30.890  26.223  1.00 48.47           O  \\nHETATM 1434  O   HOH A 250      26.967  20.447  19.448  1.00 47.77           O  \\nHETATM 1435  O   HOH A 252      28.453  14.196  14.602  1.00 39.78           O  \\nHETATM 1436  O   HOH A 253      19.639  16.256   9.760  1.00 37.54           O  \\nHETATM 1437  O   HOH A 254      16.629   7.383  10.526  1.00 60.90           O  \\nHETATM 1438  O   HOH A 255       4.972   7.888  14.276  1.00 54.95           O  \\nHETATM 1439  O   HOH A 256      14.184  14.684   5.643  1.00 31.40           O  \\nHETATM 1440  O   HOH A 257      11.766  17.254   1.278  1.00 41.82           O  \\nHETATM 1441  O   HOH A 258      16.274  16.365   6.342  1.00 17.29           O  \\nHETATM 1442  O   HOH A 259      18.346  24.095   6.620  1.00 17.36           O  \\nHETATM 1443  O   HOH A 260      22.479  35.073  19.694  1.00 43.96           O  \\nHETATM 1444  O   HOH A 261      20.580  31.747   4.906  1.00 59.46           O  \\nHETATM 1445  O   HOH A 262      18.196  25.514   4.261  1.00 28.75           O  \\nHETATM 1446  O   HOH A 263      12.857  20.183   1.149  1.00 21.79           O  \\nHETATM 1447  O   HOH A 264      19.514  15.912  24.943  1.00 55.52           O  \\nHETATM 1448  O   HOH A 265       8.925   7.579   5.621  1.00 77.94           O  \\nHETATM 1449  O   HOH A 266      17.622  21.835  26.964  1.00 27.48           O  \\nHETATM 1450  O   HOH A 267      24.258  19.786  14.155  1.00 20.48           O  \\nHETATM 1451  O   HOH A 268      23.806  15.180  15.136  1.00 19.86           O  \\nHETATM 1452  O   HOH A 269      21.478  14.557  11.134  1.00 29.66           O  \\nHETATM 1453  O   HOH A 270      17.293  45.621  24.753  1.00 39.18           O  \\nHETATM 1454  O   HOH A 271      -3.518  40.206  17.649  1.00 44.75           O  \\nHETATM 1455  O   HOH A 272       2.143  19.852  24.090  1.00 40.10           O  \\nHETATM 1456  O   HOH A 273       7.101  35.918  -4.704  1.00 31.76           O  \\nHETATM 1457  O   HOH A 274       5.462  38.756  -5.521  1.00 27.51           O  \\nHETATM 1458  O   HOH A 275       5.748  20.565  28.638  1.00 58.06           O  \\nHETATM 1459  O   HOH A 276       8.814  31.258  -8.964  1.00 23.79           O  \\nHETATM 1460  O   HOH A 277     -10.427  35.895   4.301  1.00 58.91           O  \\nHETATM 1461  O   HOH A 278      -8.713  28.579   2.333  1.00 69.36           O  \\nHETATM 1462  O   HOH A 279      24.472  24.237  17.316  1.00 32.14           O  \\nHETATM 1463  O   HOH A 281      16.968  32.701  21.815  1.00 23.13           O  \\nHETATM 1464  O   HOH A 282      19.541  34.346  21.581  1.00 53.25           O  \\nHETATM 1465  O   HOH A 283      24.847  23.384  26.621  1.00 61.18           O  \\nHETATM 1466  O   HOH A 284       1.409  11.113   2.873  1.00 44.87           O  \\nHETATM 1467  O   HOH A 285       1.556  11.488  -0.914  1.00 37.86           O  \\nHETATM 1468  O   HOH A 286       9.758  12.945   3.889  1.00 36.47           O  \\nHETATM 1469  O   HOH A 287       3.827  13.375   4.622  1.00 44.90           O  \\nHETATM 1470  O   HOH A 288      14.279   6.175  15.391  1.00 78.25           O  \\nHETATM 1471  O   HOH A 289       7.573  13.968   2.191  1.00 76.47           O  \\nHETATM 1472  O   HOH A 290       2.323  15.482   1.418  1.00 35.15           O  \\nHETATM 1473  O   HOH A 291      11.997  14.573   2.962  1.00 49.86           O  \\nHETATM 1474  O   HOH A 292      23.798  33.550  17.880  1.00 24.64           O  \\nHETATM 1475  O   HOH A 293      14.657  36.046  -0.571  1.00 51.86           O  \\nHETATM 1476  O   HOH A 294      24.456  25.366   4.004  1.00 53.23           O  \\nHETATM 1477  O   HOH A 296      -0.603  12.134  -4.185  1.00 52.38           O  \\nHETATM 1478  O   HOH A 297      -8.372  34.415  -4.471  1.00 74.53           O  \\nHETATM 1479  O   HOH A 300     -14.033  36.501   0.062  1.00 50.21           O  \\nHETATM 1480  O   HOH A 301     -17.102  35.735  -2.806  1.00 42.58           O  \\nHETATM 1481  O   HOH A 303     -17.383  40.542   1.431  1.00 77.16           O  \\nHETATM 1482  O   HOH A 304     -19.024  41.557  -3.411  1.00107.16           O  \\nHETATM 1483  O   HOH A 305      -9.876  37.804   0.839  1.00 90.65           O  \\nHETATM 1484  O   HOH A 306     -14.053  36.466   4.552  1.00 82.05           O  \\nHETATM 1485  O   HOH A 307     -14.183  39.558   0.339  1.00 58.66           O  \\nHETATM 1486  O   HOH A 309     -12.547  28.677   8.301  1.00 74.88           O  \\nHETATM 1487  O   HOH A 311     -14.787  32.746  -2.171  1.00 40.79           O  \\nHETATM 1488  O   HOH A 312      -6.916  30.740   3.095  1.00 36.53           O  \\nHETATM 1489  O   HOH A 315      -8.312  33.027   7.340  1.00 83.28           O  \\nHETATM 1490  O   HOH A 316      -6.280  35.646   7.624  1.00 75.97           O  \\nHETATM 1491  O   HOH A 317      -5.230  31.565   9.609  1.00 55.67           O  \\nHETATM 1492  O   HOH A 318       6.747  37.258   1.208  1.00 57.93           O  \\nHETATM 1493  O   HOH A 319       4.747  42.419  12.683  1.00 57.74           O  \\nHETATM 1494  O   HOH A 320       2.984  41.950  16.191  1.00 41.69           O  \\nHETATM 1495  O   HOH A 321       3.867  49.208  12.056  1.00 24.57           O  \\nHETATM 1496  O   HOH A 322       7.530  42.894  17.776  1.00 79.28           O  \\nHETATM 1497  O   HOH A 324      19.822  39.457  17.826  1.00 66.22           O  \\nHETATM 1498  O   HOH A 325      12.391  45.557  24.973  1.00 48.70           O  \\nHETATM 1499  O   HOH A 326      14.944  43.928  23.794  1.00 62.45           O  \\nHETATM 1500  O   HOH A 327     -12.984  24.008  12.563  1.00 40.31           O  \\nHETATM 1501  O   HOH A 330       0.572  41.133  19.277  1.00 50.77           O  \\nHETATM 1502  O   HOH A 331      -6.196  37.979  23.043  1.00 48.93           O  \\nHETATM 1503  O   HOH A 332      -3.526  30.410  19.065  1.00 21.26           O  \\nHETATM 1504  O   HOH A 333      -7.995  32.093  21.685  1.00 54.11           O  \\nHETATM 1505  O   HOH A 334      -8.483  28.891  20.063  1.00 70.37           O  \\nHETATM 1506  O   HOH A 335     -11.670  31.022  16.981  1.00 76.86           O  \\nHETATM 1507  O   HOH A 336     -11.082  34.450  18.910  1.00 43.07           O  \\nHETATM 1508  O   HOH A 339      -6.521  37.738  15.165  1.00 64.11           O  \\nHETATM 1509  O   HOH A 340     -10.829  35.877  15.629  1.00 58.36           O  \\nHETATM 1510  O   HOH A 341      -6.552  41.166  12.231  1.00 55.67           O  \\nHETATM 1511  O   HOH A 342      -1.242  39.671  12.185  1.00 61.04           O  \\nHETATM 1512  O   HOH A 343      -4.281  38.934  13.464  1.00 32.14           O  \\nHETATM 1513  O   HOH A 344       0.386  41.208  14.039  1.00 40.80           O  \\nHETATM 1514  O   HOH A 345       0.777  39.663  21.603  1.00 67.71           O  \\nHETATM 1515  O   HOH A 346      -7.879  34.848  19.044  1.00 52.60           O  \\nHETATM 1516  O   HOH A 347       1.121  32.594 -11.957  1.00 32.30           O  \\nHETATM 1517  O   HOH A 348       0.924  32.597 -14.716  1.00 48.34           O  \\nHETATM 1518  O   HOH A 349      -2.241  27.348  20.565  1.00 26.54           O  \\nHETATM 1519  O   HOH A 350       3.259  26.214  29.881  1.00 69.22           O  \\nHETATM 1520  O   HOH A 351      -8.318  34.095  28.392  1.00 81.96           O  \\nHETATM 1521  O   HOH A 352      -2.927  29.775  31.412  1.00 80.92           O  \\nHETATM 1522  O   HOH A 353     -10.410  32.034  25.972  1.00 60.17           O  \\nHETATM 1523  O   HOH A 354      -8.122  30.393  25.378  1.00 62.33           O  \\nHETATM 1524  O   HOH A 356      -6.138  18.882  24.644  1.00 49.26           O  \\nHETATM 1525  O   HOH A 358      -1.406  19.033  23.084  1.00 82.05           O  \\nHETATM 1526  O   HOH A 359       4.169   7.339  10.280  1.00 66.03           O  \\nHETATM 1527  O   HOH A 360      -9.761  30.165   7.702  1.00 46.15           O  \\nHETATM 1528  O   HOH A 361     -16.002  27.801  11.700  1.00 41.85           O  \\nHETATM 1529  O   HOH A 362      16.215  30.489  29.264  1.00 67.21           O  \\nHETATM 1530  O   HOH A 364      21.417  27.685  26.140  1.00 49.57           O  \\nHETATM 1531  O   HOH A 366      15.824  34.767  30.662  1.00 25.90           O  \\nHETATM 1532  O   HOH A 367      18.527  35.965  24.111  1.00 68.69           O  \\nHETATM 1533  O   HOH A 369      12.103  35.275  24.945  1.00 51.37           O  \\nHETATM 1534  O   HOH A 370      24.782  11.970  23.641  1.00 55.92           O  \\nHETATM 1535  O   HOH A 371      14.337  21.394  28.627  1.00 30.11           O  \\nHETATM 1536  O   HOH A 372      -1.367  12.600  23.053  1.00 44.43           O  \\nHETATM 1537  O   HOH A 373       7.060   8.567  21.416  1.00 47.99           O  \\nHETATM 1538  O   HOH A 374       7.639   8.040  16.420  1.00 38.76           O  \\nHETATM 1539  O   HOH A 376      -1.282  15.566  25.487  1.00 67.79           O  \\nHETATM 1540  O   HOH A 378      17.945  30.924  23.806  1.00 31.50           O  \\nHETATM 1541  O   HOH A 379      17.360  33.209  33.337  1.00 30.46           O  \\nHETATM 1542  O   HOH A 380       3.971   6.567  22.089  1.00 86.80           O  \\nHETATM 1543  O   HOH A 381      -0.065  10.022  26.928  1.00 61.86           O  \\nHETATM 1544  O   HOH A 382      -4.450  12.966  21.222  1.00 42.82           O  \\nHETATM 1545  O   HOH A 383      -2.584   6.719   5.938  1.00 40.64           O  \\nHETATM 1546  O   HOH A 384      25.567  31.936  12.845  1.00 73.17           O  \\nHETATM 1547  O   HOH A 385      24.446  29.383  22.970  1.00 36.61           O  \\nHETATM 1548  O   HOH A 386      24.865  38.167  18.227  1.00 60.74           O  \\nHETATM 1549  O   HOH A 387      23.135  35.012  14.543  1.00 59.19           O  \\nHETATM 1550  O   HOH A 388      19.325   9.556  18.093  1.00 92.18           O  \\nHETATM 1551  O   HOH A 389      25.326  12.873  16.251  1.00 49.27           O  \\nHETATM 1552  O   HOH A 390      20.219  10.405  10.956  1.00 63.05           O  \\nHETATM 1553  O   HOH A 392       9.452   5.453   8.249  1.00 47.81           O  \\nHETATM 1554  O   HOH A 394      10.139  12.252   0.254  1.00 29.37           O  \\nHETATM 1555  O   HOH A 395      28.004  37.416  13.466  1.00 52.29           O  \\nHETATM 1556  O   HOH A 396      24.283  36.829  12.307  1.00 64.74           O  \\nHETATM 1557  O   HOH A 397      13.748  34.031  -3.019  1.00 47.80           O  \\nHETATM 1558  O   HOH A 398      16.981  34.189   2.006  1.00 80.51           O  \\nHETATM 1559  O   HOH A 399      -4.426  24.223  -7.240  1.00 38.09           O  \\nHETATM 1560  O   HOH A 400      19.676  33.981   3.336  1.00 64.86           O  \\nHETATM 1561  O   HOH A 401      14.633  28.745  -0.317  1.00 66.25           O  \\nHETATM 1562  O   HOH A 402      -2.139  15.576  22.716  1.00 65.01           O  \\nHETATM 1563  O   HOH A 403       9.873  11.090  20.760  1.00 53.88           O  \\nHETATM 1564  O   HOH A 404      18.291  36.379  26.927  1.00 57.23           O  \\nHETATM 1565  O   HOH A 405       0.138   6.145  15.269  1.00 82.80           O  \\nHETATM 1566  O   HOH A 406      -5.515  38.166  18.689  1.00 48.28           O  \\nHETATM 1567  O   HOH A 407      -0.399   4.688   8.809  1.00 58.35           O  \\nHETATM 1568  O   HOH A 408       0.813  25.584  22.307  1.00 40.04           O  \\nCONECT  117 1325                                                                \\nCONECT  265 1325                                                                \\nCONECT 1325  117  265 1328 1333                                                 \\nCONECT 1325 1361 1362                                                           \\nCONECT 1326 1327 1328 1329 1330                                                 \\nCONECT 1327 1326                                                                \\nCONECT 1328 1325 1326                                                           \\nCONECT 1329 1326                                                                \\nCONECT 1330 1326 1331                                                           \\nCONECT 1331 1330 1332 1333 1334                                                 \\nCONECT 1332 1331                                                                \\nCONECT 1333 1325 1331                                                           \\nCONECT 1334 1331 1335                                                           \\nCONECT 1335 1334 1336 1337 1338                                                 \\nCONECT 1336 1335                                                                \\nCONECT 1337 1335                                                                \\nCONECT 1338 1335 1339                                                           \\nCONECT 1339 1338 1340                                                           \\nCONECT 1340 1339 1341 1342                                                      \\nCONECT 1341 1340 1346                                                           \\nCONECT 1342 1340 1343 1344                                                      \\nCONECT 1343 1342                                                                \\nCONECT 1344 1342 1345 1346                                                      \\nCONECT 1345 1344                                                                \\nCONECT 1346 1341 1344 1347                                                      \\nCONECT 1347 1346 1348 1357                                                      \\nCONECT 1348 1347 1349                                                           \\nCONECT 1349 1348 1350                                                           \\nCONECT 1350 1349 1351 1357                                                      \\nCONECT 1351 1350 1352 1353                                                      \\nCONECT 1352 1351                                                                \\nCONECT 1353 1351 1354                                                           \\nCONECT 1354 1353 1355 1356                                                      \\nCONECT 1355 1354                                                                \\nCONECT 1356 1354 1357                                                           \\nCONECT 1357 1347 1350 1356                                                      \\nCONECT 1361 1325                                                                \\nCONECT 1362 1325                                                                \\nMASTER      312    0    2    5    6    0    9    6 1567    1   38   13          \\nEND                                                                             \\n\",\"pdb\");\n",
       "\tviewergrid_16835771918828518[0][0].addModel(\"\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    5.3278   11.9619   -1.5862 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4918   12.0625   -0.4513 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5527   13.0816    0.5011 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7707   13.0853    1.5453 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4957   14.1194    0.2981 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3069   14.0310   -0.8665 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.2721   14.9285   -1.2734 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.7924   14.4907   -2.4207 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.7723   14.9976   -3.2383 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.1222   14.3073   -4.4052 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.4811   13.1363   -4.7128 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4813   12.6157   -3.8820 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1577   13.3143   -2.7429 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2349   13.0415   -1.7643 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8145   11.3598   -0.3222 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1195   12.3585    1.6752 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8318   13.8149    2.2035 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5851   14.8663    0.9332 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7640    9.3703   -2.6645 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8250   10.2866   -3.3296 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1718    9.8621   -4.4958 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2879   10.6862   -5.1442 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.0572   11.9404   -4.6231 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.7079   12.3479   -3.4716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5995   11.5321   -2.8072 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-4.68549\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -7.2548    9.0375    4.5817 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0448    8.4019    4.2228 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.8292    9.0547    4.0095 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.7612    8.4021    3.6418 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.7899   10.4573    4.2059 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0025   11.0841    4.6043 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.1942   12.4245    4.8676 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4672   12.6023    5.2235 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1730   13.7240    5.5829 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.5292   13.6094    5.9119 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.1308   12.3793    5.8698 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.4149   11.2328    5.5039 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0847   11.3744    5.1862 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.1585   10.4388    4.7983 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.0601    7.4234    4.1148 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8013    7.4275    3.5086 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9150    8.8841    3.4976 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9622   10.9735    4.0730 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.7008    7.7755    3.8786 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.2098    7.6907    5.2625 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.9715    6.9952    6.2128 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.5471    6.8921    7.5129 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3553    7.4879    7.8630 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6125    8.1716    6.9162 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.0245    8.2848    5.6050 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-4.21517\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -5.2059    9.8913    1.4701 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.4393    8.7205    1.6663 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9500    8.2799    2.8975 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.2150    7.2064    2.9934 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.2783    9.0388    4.0481 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0823   10.1944    3.8472 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.5352   11.0665    4.8152 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2676   12.0083    4.2190 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9393   13.0997    4.7121 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6386   13.9345    3.8321 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6414   13.6492    2.4922 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9578   12.5374    1.9849 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2799   11.7347    2.8716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.5365   10.6013    2.6562 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.2343    8.1735    0.8739 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.9938    6.6876    2.1865 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.8815    6.9151    3.8727 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.9646    8.7783    4.9442 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.0780   11.0707   -1.1110 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2599   10.2829   -0.7287 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3094   10.1417   -1.6483 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4274    9.4138   -1.3304 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4951    8.8250   -0.0867 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.4530    8.9717    0.8122 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.3219    9.7000    0.5088 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-4.01159\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -6.9498   10.0239    3.9638 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.6927    9.4677    4.2914 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.8273    8.8533    3.3843 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6974    8.3277    3.7703 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.2156    8.8174    2.0222 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.4645    9.4132    1.6947 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.0326    9.5098    0.4413 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1973   10.1489    0.5589 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.1492   10.5022   -0.3655 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.2995   11.1814    0.0542 C   0  0  0  0  0  0  0  0  0  0  0  0\\n  -10.4572   11.4825    1.3813 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -9.4891   11.1240    2.3275 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.3695   10.4583    1.8875 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2812    9.9929    2.5820 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4050    9.5174    5.2316 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.4388    8.3591    4.7197 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.1033    7.8998    3.1121 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.6476    8.3998    1.3350 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.9625   10.1830    5.9622 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1934   11.3853    5.6068 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.4816   12.5938    6.2575 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.7858   13.7364    5.9555 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.7975   13.6689    4.9981 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.5224   12.4705    4.3628 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.2122   11.3124    4.6543 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.94575\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    6.7047   12.9704   -2.2899 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.4325   13.2928   -0.9412 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1589   13.3474   -0.3716 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9846   13.6911    0.8748 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0535   13.0157   -1.1934 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3391   12.6553   -2.5390 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4301   12.2813   -3.5068 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1009   12.0104   -4.6273 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6860   11.6021   -5.8709 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6341   11.3923   -6.8797 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.9614   11.5982   -6.6099 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3889   12.0147   -5.3431 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4366   12.2126   -4.3715 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.5684   12.6144   -3.0658 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.1999   13.4943   -0.3584 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.7609   13.9193    1.4356 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.0768   13.7223    1.2545 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.1313   13.0340   -0.8491 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.0760   13.9697   -3.4934 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    9.0229   12.5708   -3.0417 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.1470   11.7530   -3.2282 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   10.1377   10.4440   -2.8192 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.9984    9.9516   -2.2213 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8955   10.7685   -2.0429 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.8879   12.0867   -2.4480 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.69027\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -0.0133    7.1993    3.0780 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9633    8.1816    3.3577 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.9099    8.1021    4.3810 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7566    9.0696    4.6019 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.9237    6.9307    5.1778 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9627    5.9302    4.8653 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.8033    4.7110    5.4906 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.1959    4.0619    4.8915 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.7575    2.8294    5.1174 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.8208    2.3984    4.3147 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.2843    3.2138    3.3162 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.7147    4.4715    3.0823 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.6719    4.8690    3.8852 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.0605    6.0297    3.8822 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.9772    8.9839    2.7872 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.7370    9.8786    4.0412 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4153    8.9908    5.3294 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.5681    6.8148    5.9130 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.3767    8.0951    1.7813 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.3467    7.3852    1.0076 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.5618    7.1538   -0.3589 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.6250    6.4970   -1.1150 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.5318    6.0700   -0.5007 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    0.7326    6.3043    0.8485 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.1989    6.9630    1.6234 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.62684\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    6.0766   11.2367   -1.2749 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8567   12.0362   -0.1307 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9469   13.0926   -0.0540 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8276   13.8112    1.0283 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1513   13.3627   -1.1948 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.3516   12.5291   -2.3293 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6835   12.5959   -3.5342 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1547   11.6324   -4.3269 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8413   11.2642   -5.6121 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.5154   10.1899   -6.2056 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.4777    9.5230   -5.4943 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8006    9.8951   -4.1835 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.1228   10.9531   -3.6255 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2336   11.5243   -2.3825 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3856   11.8328    0.6743 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3862   13.6135    1.8145 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.1789   14.5511    1.0589 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.4910   14.0929   -1.2017 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.5729    9.9538   -1.7136 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4410    9.1803   -1.1807 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.6171    7.8152   -0.9115 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5868    7.0600   -0.4124 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3752    7.6740   -0.1820 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.2137    9.0219   -0.4517 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2390    9.7957   -0.9537 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.57606\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n    5.0653   12.1568   -3.9872 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.8604   12.4012   -4.6839 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.9703   11.4160   -5.1156 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.8943   11.7181   -5.7887 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.2724   10.0686   -4.7985 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.4672    9.8343   -4.0639 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.9490    8.6156   -3.6332 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.0860    8.8271   -2.9690 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.9587    7.9704   -2.3445 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.0986    8.4851   -1.7151 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    8.3255    9.8361   -1.7324 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.4383   10.7134   -2.3681 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.3270   10.1808   -2.9778 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3092   10.7944   -3.6642 N   0  0  0  0  0  0  0  0  0  0  0  0\\n    3.6309   13.3383   -4.8799 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.6954   12.6582   -6.0029 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    1.2813   11.0060   -6.0825 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    2.6830    9.3295   -5.0736 H   0  0  0  0  0  0  0  0  0  0  0  0\\n    7.0458   14.1722   -4.2793 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.2733   14.0867   -3.0305 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    6.5425   14.9999   -2.0006 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.8430   14.9541   -0.8219 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.8701   13.9900   -0.6735 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    4.6136   13.0944   -1.6971 C   0  0  0  0  0  0  0  0  0  0  0  0\\n    5.3075   13.1265   -2.8885 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.48728\\n\\n$$$$\\n\\n\\n\\n 25 28  0  0  0  0  0  0  0  0999 V2000\\n   -4.9024   11.4840    0.7331 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.6126   11.9069    0.3400 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.4179   11.2889    0.7142 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.2607   11.7621    0.3415 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -2.5003   10.1240    1.5163 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.8035    9.6828    1.8758 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.1233    8.5730    2.6300 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.4517    8.5025    2.7266 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.2812    7.6082    3.3572 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.6681    7.7885    3.2902 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -8.1758    8.8551    2.5964 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.3337    9.7696    1.9519 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.9762    9.5681    2.0338 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -4.9361   10.2941    1.5097 N   0  0  0  0  0  0  0  0  0  0  0  0\\n   -3.5518   12.6983   -0.2425 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.2166   12.5724   -0.2159 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -0.4315   11.3089    0.6177 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -1.6936    9.6427    1.8109 H   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.9700   13.4305    0.8105 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8997   12.4880   -0.3166 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8688   12.5637   -1.3277 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -7.8367   11.7042   -2.3958 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -6.8305   10.7648   -2.4519 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.8800   10.6999   -1.4480 C   0  0  0  0  0  0  0  0  0  0  0  0\\n   -5.8974   11.5564   -0.3673 C   0  0  0  0  0  0  0  0  0  0  0  0\\n 19 20  1  0  0  0\\n 20 21  2  0  0  0\\n 21 22  1  0  0  0\\n 22 23  2  0  0  0\\n 23 24  1  0  0  0\\n 24 25  2  0  0  0\\n  1 25  1  0  0  0\\n  1  2  1  0  0  0\\n  2  3  1  0  0  0\\n  3  4  2  0  0  0\\n  3  5  1  0  0  0\\n  5  6  1  0  0  0\\n  6  7  2  0  0  0\\n  7  8  2  0  0  0\\n  8  9  2  0  0  0\\n  9 10  1  0  0  0\\n 10 11  2  0  0  0\\n 11 12  1  0  0  0\\n 12 13  2  0  0  0\\n 13 14  1  0  0  0\\n 25 20  1  0  0  0\\n 14  1  1  0  0  0\\n 14  6  1  0  0  0\\n 13  8  1  0  0  0\\n  2 15  1  0  0  0\\n  4 16  1  0  0  0\\n  4 17  1  0  0  0\\n  5 18  1  0  0  0\\nM  CHG 1   4   1\\nM  END\\n> <minimizedAffinity>\\n-3.46691\\n\\n$$$$\\n\",\"sdf\");\n",
       "\tviewergrid_16835771918828518[0][0].setStyle({\"model\": 0},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewergrid_16835771918828518[0][0].addSurface(1,{\"opacity\": 0.7, \"color\": \"white\"},{\"model\": 0});\n",
       "\tviewergrid_16835771918828518[0][0].setStyle({\"model\": 1},{\"stick\": {}});\n",
       "\tviewergrid_16835771918828518[0][0].zoomTo();\n",
       "\tviewergrid_16835771918828518[0][0].render();\n",
       "viewergrid_16835771918828518[0][0].render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<py3Dmol.view at 0x7efc8c0f44d0>"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Load protein data\n",
    "with open('data/ras/ras_protein.pdb', 'r') as f:\n",
    "    chA_data = f.read()\n",
    "\n",
    "# Load ligand data\n",
    "ligands = []\n",
    "for i in [282]:  # Changed to only include the ligand with index 282\n",
    "    with open(f'data/temp/docking_results_{i}.sdf', 'r') as f:\n",
    "        ligands.append(f.read())\n",
    "\n",
    "view = py3Dmol.view(width=1000, height=1000, viewergrid=(1, 1), linked=False)  # Changed viewergrid to (1, 1)\n",
    "view.setViewStyle({'style': 'outline', 'color': 'black', 'width': 0.1})\n",
    "\n",
    "for i, ligand in enumerate(ligands):\n",
    "    viewer = (0, 0)  # Changed to a single viewer (0, 0)\n",
    "    view.addModel(chA_data, 'pdb', viewer=viewer)\n",
    "    view.addModel(ligand, 'sdf', viewer=viewer)\n",
    "\n",
    "    chA = {'model': 0}\n",
    "    chB = {'model': 1}\n",
    "    \n",
    "    view.setStyle(chA, {'cartoon': {'color': 'spectrum'}}, viewer=viewer)\n",
    "    view.addSurface(py3Dmol.VDW, {'opacity': 0.7, 'color': 'white'}, chA, viewer=viewer)\n",
    "    view.setStyle(chB, {'stick': {}}, viewer=viewer)\n",
    "\n",
    "    view.zoomTo(viewer=viewer)\n",
    "\n",
    "view.render()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "29a38430-862e-4c42-b090-ce4359265bfa",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
